## Evolutionary and Genomic Analysis of SARS-CoV-2

### Abstract

### Introduction

### Initial Characterization of SARS-CoV-2

The first genome sequence of the virus was released on January 3, 2020 and revealed that the cluster of pneumonia cases seen in Wuhan were caused by a novel coronavirus [@doi:10.46234/ccdcw2020.017].
Multiple research groups have drafted the genome sequence of SARS-CoV-2 based on sequences developed from clinical samples collected from the lower respiratory tract, namely bronchoalveolar lavage fluid (BALF), and the upper respiratory tract, in the form of throat swabs [@doi:10.1038/s41586-020-2008-3; @doi:10.1038/s41586-020-2012-7; @doi:10/ggjr43].
Analysis of the SARS-CoV-2 genome revealed significant sequence homology with two coronaviruses known to infect humans, with about 79% identity to SARS-CoV-1 and 50% to MERS-CoV [@doi:10/ggjr43].
However, in this analysis, the highest degree of similarity was observed between SARS-CoV-2 and bat-derived SARS-like coronaviruses (bat-SL-CoVZC45 and bat-SL-CoVZXC21) [@doi:10/ggjr43].
Other analyses have reported even greater similarity between SARS-CoV-2 and the bat coronavirus BatCoV-RaTG13, with identity as high as 96.2% [@doi:10.1038/s41586-020-2012-7; @doi:10.1093/nsr/nwaa036], and the closely related pangolin coronavirus [@doi:10.1101/2020.02.19.950253].
This evidence therefore suggests the SARS-CoV-2 virus is the result of zoonotic transfer of a virus from bats to humans.
Nevertheless, some fragments between SARS-CoV-2 and RATG13 differ by up to 17%, suggesting a complex natural selection process during zoonotic transfer.
While the _S_ region is highly similar to that of viruses found in pangolins [@doi:10.1101/2020.02.19.950253], there is no consensus about the origin of _S_ in SARS-CoV-2, as it could potentially be the result either of recombination or coevolution [@doi:10.1093/nsr/nwaa036; @doi:10.1126/sciadv.abb9153].
Though the intermediate host serving as the source for the zoonotic introduction of SARS-CoV-2 to human populations has not yet been identified, the SARS-CoV-2 virus has been placed within the coronavirus phylogeny through comparative genomic analyses.
Genomic analyses and comparisons to other known coronaviruses suggest that SARS-CoV-2 is unlikely to have originated in a laboratory -- either purposely engineered and released, or escaped -- and instead evolved naturally in an animal host [@doi:10.1038/s41591-020-0820-9].
While the position of the SARS-CoV-2 virus within the coronavirus phylogeny has been largely resolved, the functional consequences of molecular variation between this virus and other viruses, such as its bat and pangolin sister taxa or SARS-CoV-1, remain unknown [@doi:10.1038/s41591-020-0820-9, @doi:10.1038/s41586-020-2012-7].
Fortunately, the basic genome structure of coronaviruses is highly conserved, and insight into the mechanisms the virus uses to enter cells, replicate, and spread is likely to be available from prior research in coronaviruses.

### Coronaviruses and Humans Hosts

Coronaviruses have long been known to infect animals and have been the subject of veterinary medical investigations and vaccine development efforts due to their effect on the health of companion and agricultural animals [@doi:10/ckfn8b].
Most coronaviruses show little to no transmission in humans.
However, today it is thought that approximately one-third of common cold infections are caused by four human coronaviruses (HCoV): _Human coronavirus 229E_ (HCoV-229E), _Human coronavirus NL63_ (HCoV-NL63), _Human coronavirus OC43_ (HCoV-OC43), and _Human coronavirus HKU1_ (HCoV-HKU1) [@doi:10.1038/s41579-018-0118-9; @doi:10.3390/diseases4030026].
The first HCoV were identified in the 1960s: HCoV-229E in 1965 [@doi:10.3181/00379727-121-30734] and HCoV-OC43 in 1967 [@doi:10.1073/pnas.57.4.933].
Both of these viruses typically cause cold-like symptoms, including upper and lower respiratory infections [@doi:10.1186/1743-422X-1-7; @pmid:20700397; @doi:10.1186/s40779-020-00279-z], but they have also been associated with gastrointestinal symptoms [@doi:10.1111/j.1440-1754.2007.01246.x].
Two additional HCoV were subsequently identified [@doi:10.1038/nm1024; @doi:10.1128/JVI.79.2.884-895.2005].
In 2003, HCoV-NL63 [@doi:10.1038/nm1024] was first identified in a 7-month-old infant and then in clinical specimens collected from seven additional patients, five of whom were infants younger than 1 year old and the remainder of whom were adults.
CoV-HKU1 was identified in samples collected from a 71-year-old pneumonia patient in 2004 and then found in samples collected from a second adult patient [@doi:10.1128/JVI.79.2.884-895.2005].
These viruses are associated with respiratory diseases of varying severity, ranging from common cold to severe pneumonia, with severe symptoms mostly observed in immunocompromised individuals [@doi:10.1086/377612], and also have gastrointestinal involvement in some cases [@doi:10.1111/j.1440-1754.2007.01246.x].
In addition to these relatively mild HCoV, however, highly pathogenic human coronaviruses have been identified, including _Severe acute respiratory syndrome-related coronavirus_ (SARS-CoV or SARS-CoV-1) and _Middle East respiratory syndrome-related coronavirus_ (MERS-CoV) [@doi:10.1038/nrmicro.2016.81; @doi:10.1038/s41579-018-0118-9; @doi:10.1016/bs.aivir.2018.01.001].

### Conclusions

As of October 2020 the SARS-CoV-2 virus remains a serious worldwide threat.
The scientific community has responded by rapidly collecting and disseminating information about the SARS-CoV-2 virus and the associated illness, COVID-19.
The rapid identification of the genomic sequence of the virus allowed for early contextualization of SARS-CoV-2 among other known respiratory viruses.
The pathogen is a coronavirus that is closely related to SARS-CoV-1, which caused the SARS pandemics of the early 2000s.
Knowing the phylogenetic context and genomic sequence of the virus then allowed for rapid insights into its structure and pathogenesis.
