<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en-US" xml:lang="en-US">
<head>
  <meta charset="utf-8" />
  <meta name="generator" content="pandoc" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
  <meta name="author" content="Halie M. Rando" />
  <meta name="author" content="Casey S. Greene" />
  <meta name="author" content="Michael P. Robson" />
  <meta name="author" content="Simina M. Boca" />
  <meta name="author" content="Nils Wellhausen" />
  <meta name="author" content="Ronan Lordan" />
  <meta name="author" content="Christian Brueffer" />
  <meta name="author" content="Sadipan Ray" />
  <meta name="author" content="Lucy D'Agostino McGowan" />
  <meta name="author" content="Anthony Gitter" />
  <meta name="author" content="Ronnie M. Russell" />
  <meta name="author" content="Anna Ada Dattoli" />
  <meta name="author" content="Ryan Velazquez" />
  <meta name="author" content="John P. Barton" />
  <meta name="author" content="Jeffrey M. Field" />
  <meta name="author" content="Bharath Ramsundar" />
  <meta name="author" content="Adam L. MacLean" />
  <meta name="author" content="Alexandra J. Lee" />
  <meta name="author" content="Immunology Institute of the Icahn School of Medicine" />
  <meta name="author" content="Fengling Hu" />
  <meta name="author" content="Nafisa M. Jadavji" />
  <meta name="author" content="Elizabeth Sell" />
  <meta name="author" content="Jinhui Wang" />
  <meta name="author" content="Diane N. Rafizadeh" />
  <meta name="author" content="Ashwin N. Skelly" />
  <meta name="author" content="Marouen Ben Guebila" />
  <meta name="author" content="Likhitha Kolla" />
  <meta name="author" content="David Manheim" />
  <meta name="author" content="Soumita Ghosh" />
  <meta name="author" content="Matthias Fax" />
  <meta name="author" content="James Brian Byrd" />
  <meta name="author" content="YoSon Park" />
  <meta name="author" content="Yael Evelyn Marshall" />
  <meta name="author" content="Vikas Bansal" />
  <meta name="author" content="Stephen Capone" />
  <meta name="author" content="John J. Dziak" />
  <meta name="author" content="YuCheng Sun" />
  <meta name="author" content="Yanjun Qi" />
  <meta name="author" content="Lamonica Shinholster" />
  <meta name="dcterms.date" content="2020-07-31" />
  <meta name="keywords" content="covid-19, coronavirus, pandemic, viral infection" />
  <title>SARS-CoV-2 and COVID-19: An Evolving Review of Diagnostics and Therapeutics</title>
  <style>
    code{white-space: pre-wrap;}
    span.smallcaps{font-variant: small-caps;}
    span.underline{text-decoration: underline;}
    div.column{display: inline-block; vertical-align: top; width: 50%;}
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
    ul.task-list{list-style: none;}
  </style>
  <!--[if lt IE 9]>
    <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv-printshiv.min.js"></script>
  <![endif]-->
  <!--
  Manubot generated metadata rendered from header-includes-template.html.
  Suggest improvements at https://github.com/manubot/manubot/blob/master/manubot/process/header-includes-template.html
  -->
  <meta name="dc.format" content="text/html" />
  <meta name="dc.title" content="SARS-CoV-2 and COVID-19: An Evolving Review of Diagnostics and Therapeutics" />
  <meta name="citation_title" content="SARS-CoV-2 and COVID-19: An Evolving Review of Diagnostics and Therapeutics" />
  <meta property="og:title" content="SARS-CoV-2 and COVID-19: An Evolving Review of Diagnostics and Therapeutics" />
  <meta property="twitter:title" content="SARS-CoV-2 and COVID-19: An Evolving Review of Diagnostics and Therapeutics" />
  <meta name="dc.date" content="2020-07-31" />
  <meta name="citation_publication_date" content="2020-07-31" />
  <meta name="dc.language" content="en-US" />
  <meta name="citation_language" content="en-US" />
  <meta name="dc.relation.ispartof" content="Manubot" />
  <meta name="dc.publisher" content="Manubot" />
  <meta name="citation_journal_title" content="Manubot" />
  <meta name="citation_technical_report_institution" content="Manubot" />
  <meta name="citation_author" content="Halie M. Rando" />
  <meta name="citation_author_institution" content="Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America" />
  <meta name="citation_author_orcid" content="0000-0001-7688-1770" />
  <meta name="twitter:creator" content="@tamefoxtime" />
  <meta name="citation_author" content="Casey S. Greene" />
  <meta name="citation_author_institution" content="Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America" />
  <meta name="citation_author_institution" content="Childhood Cancer Data Lab, Alex&#39;s Lemonade Stand Foundation, Philadelphia, Pennsylvania, United States of America" />
  <meta name="citation_author_orcid" content="0000-0001-8713-9213" />
  <meta name="twitter:creator" content="@GreeneScientist" />
  <meta name="citation_author" content="Michael P. Robson" />
  <meta name="citation_author_institution" content="Department of Computing Sciences, Villanova University, Villanova, Pennsylvania, United States of America" />
  <meta name="citation_author_orcid" content="0000-0002-4859-0033" />
  <meta name="citation_author" content="Simina M. Boca" />
  <meta name="citation_author_institution" content="Innovation Center for Biomedical Informatics, Georgetown University Medical Center, Washington, District of Columbia, United States of America" />
  <meta name="citation_author_orcid" content="0000-0002-1400-3398" />
  <meta name="citation_author" content="Nils Wellhausen" />
  <meta name="citation_author_institution" content="Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America" />
  <meta name="citation_author_orcid" content="0000-0001-8955-7582" />
  <meta name="citation_author" content="Ronan Lordan" />
  <meta name="citation_author_institution" content="Institute for Translational Medicine and Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104-5158, USA" />
  <meta name="citation_author_orcid" content="0000-0001-9668-3368" />
  <meta name="twitter:creator" content="@el_ronan" />
  <meta name="citation_author" content="Christian Brueffer" />
  <meta name="citation_author_institution" content="Department of Clinical Sciences, Lund University, Lund, Sweden" />
  <meta name="citation_author_orcid" content="0000-0002-3826-0989" />
  <meta name="twitter:creator" content="@cbrueffer" />
  <meta name="citation_author" content="Sadipan Ray" />
  <meta name="citation_author_institution" content="Department of Systems Pharmacology &amp; Translational Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA" />
  <meta name="citation_author_institution" content="Institute for Translational Medicine and Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA" />
  <meta name="citation_author_orcid" content="0000-0002-9960-5768" />
  <meta name="citation_author" content="Lucy D\&#39;Agostino McGowan" />
  <meta name="citation_author_institution" content="Department of Mathematics and Statistics, Wake Forest University, Winston-Salem, North Carolina, United States of America" />
  <meta name="citation_author_orcid" content="0000-0001-7297-9359" />
  <meta name="twitter:creator" content="@LucyStats" />
  <meta name="citation_author" content="Anthony Gitter" />
  <meta name="citation_author_institution" content="Department of Biostatistics and Medical Informatics, University of Wisconsin-Madison, Madison, Wisconsin, United States of America" />
  <meta name="citation_author_institution" content="Morgridge Institute for Research, Madison, Wisconsin, United States of America" />
  <meta name="citation_author_orcid" content="0000-0002-5324-9833" />
  <meta name="twitter:creator" content="@anthonygitter" />
  <meta name="citation_author" content="Ronnie M. Russell" />
  <meta name="citation_author_institution" content="Department of Microbiology, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA" />
  <meta name="citation_author_orcid" content="0000-0003-1484-4207" />
  <meta name="citation_author" content="Anna Ada Dattoli" />
  <meta name="citation_author_institution" content="Department of Systems Pharmacology &amp; Translational Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA" />
  <meta name="citation_author_orcid" content="0000-0003-1462-831X" />
  <meta name="twitter:creator" content="@aadattoli" />
  <meta name="citation_author" content="Ryan Velazquez" />
  <meta name="citation_author_institution" content="Azimuth1, McLean, VA" />
  <meta name="citation_author_orcid" content="0000-0002-3655-3403" />
  <meta name="citation_author" content="John P. Barton" />
  <meta name="citation_author_institution" content="Department of Physics and Astronomy, University of California-Riverside, Riverside, California, United States of America" />
  <meta name="citation_author_orcid" content="0000-0003-1467-421X" />
  <meta name="twitter:creator" content="@_jpbarton" />
  <meta name="citation_author" content="Jeffrey M. Field" />
  <meta name="citation_author_institution" content="Department of Pharmacology, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA" />
  <meta name="citation_author_orcid" content="0000-0001-7161-7284" />
  <meta name="citation_author" content="Bharath Ramsundar" />
  <meta name="citation_author_institution" content="The DeepChem Project, https://deepchem.io/" />
  <meta name="citation_author_orcid" content="0000-0001-8450-4262" />
  <meta name="twitter:creator" content="@rbhar90" />
  <meta name="citation_author" content="Adam L. MacLean" />
  <meta name="citation_author_institution" content="Department of Biological Sciences, University of Southern California, Los Angeles, California, United States of America" />
  <meta name="citation_author_orcid" content="0000-0003-0689-7907" />
  <meta name="twitter:creator" content="@adamlmaclean" />
  <meta name="citation_author" content="Alexandra J. Lee" />
  <meta name="citation_author_institution" content="Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America" />
  <meta name="citation_author_orcid" content="0000-0002-0208-3730" />
  <meta name="citation_author" content="Immunology Institute of the Icahn School of Medicine" />
  <meta name="citation_author_institution" content="Immunology Institute of the Icahn School of Medicine" />
  <meta name="citation_author" content="Fengling Hu" />
  <meta name="citation_author_institution" content="Department of Biostatistics, Epidemiology and Informatics, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America" />
  <meta name="citation_author_orcid" content="0000-0003-1081-5038" />
  <meta name="twitter:creator" content="@hufengling" />
  <meta name="citation_author" content="Nafisa M. Jadavji" />
  <meta name="citation_author_institution" content="Biomedical Science, Midwestern University, Glendale, AZ, United States of America" />
  <meta name="citation_author_institution" content="Department of Neuroscience, Carleton University, Ottawa, Ontario, Canada" />
  <meta name="citation_author_orcid" content="0000-0002-3557-7307" />
  <meta name="twitter:creator" content="@nafisajadavji" />
  <meta name="citation_author" content="Elizabeth Sell" />
  <meta name="citation_author_institution" content="Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America" />
  <meta name="citation_author_orcid" content="0000-0002-9658-1107" />
  <meta name="citation_author" content="Jinhui Wang" />
  <meta name="citation_author_institution" content="Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America" />
  <meta name="citation_author_orcid" content="0000-0002-5796-8130" />
  <meta name="citation_author" content="Diane N. Rafizadeh" />
  <meta name="citation_author_institution" content="Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America" />
  <meta name="citation_author_institution" content="Department of Chemistry, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America" />
  <meta name="citation_author_orcid" content="0000-0002-2838-067X" />
  <meta name="citation_author" content="Ashwin N. Skelly" />
  <meta name="citation_author_institution" content="Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America" />
  <meta name="citation_author_institution" content="Institute for Immunology, University of Pennsylvania Perelman School of Medicine, Philadelphia, United States of America" />
  <meta name="citation_author_orcid" content="0000-0002-1565-3376" />
  <meta name="citation_author" content="Marouen Ben Guebila" />
  <meta name="citation_author_institution" content="Department of Biostatistics, Harvard School of Public Health, Boston, Massachusetts, United States of America" />
  <meta name="citation_author_orcid" content="0000-0001-5934-966X" />
  <meta name="twitter:creator" content="@marouenbg" />
  <meta name="citation_author" content="Likhitha Kolla" />
  <meta name="citation_author_institution" content="Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America" />
  <meta name="citation_author_orcid" content="0000-0002-1169-906X" />
  <meta name="twitter:creator" content="@lkolla2018" />
  <meta name="citation_author" content="David Manheim" />
  <meta name="citation_author_institution" content="Risk and Health Communication Research Center, School of Public Health, University of Haifa, Haifa, Israel" />
  <meta name="citation_author_orcid" content="0000-0001-8599-8380" />
  <meta name="twitter:creator" content="@davidmanheim" />
  <meta name="citation_author" content="Soumita Ghosh" />
  <meta name="citation_author_institution" content="Institute of Translational Medicine and Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America" />
  <meta name="citation_author_orcid" content="0000-0002-2783-2750" />
  <meta name="citation_author" content="Matthias Fax" />
  <meta name="citation_author_institution" content="greensolid technologies, Bavaria, Germany" />
  <meta name="citation_author_institution" content="IEEE Reliability Society, Vienna, Austria" />
  <meta name="citation_author_orcid" content="0000-0002-0064-395X" />
  <meta name="citation_author" content="James Brian Byrd" />
  <meta name="citation_author_institution" content="University of Michigan School of Medicine, Ann Arbor, Michigan, United States of America" />
  <meta name="citation_author_orcid" content="0000-0002-0509-3520" />
  <meta name="twitter:creator" content="@thebyrdlab" />
  <meta name="citation_author" content="YoSon Park" />
  <meta name="citation_author_institution" content="Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America" />
  <meta name="citation_author_orcid" content="0000-0002-0465-4744" />
  <meta name="twitter:creator" content="@__yoson__" />
  <meta name="citation_author" content="Yael Evelyn Marshall" />
  <meta name="citation_author_institution" content="recent PhD, Princeton University, Princeton, NJ, USA" />
  <meta name="citation_author_institution" content="freelance copyeditor, multiple locations, USA" />
  <meta name="citation_author_orcid" content="0000-0003-3723-9683" />
  <meta name="citation_author" content="Vikas Bansal" />
  <meta name="citation_author_institution" content="Biomedical Data Science and Machine Learning Group, DZNE, Tübingen, Germany" />
  <meta name="citation_author_orcid" content="0000-0002-0944-7226" />
  <meta name="twitter:creator" content="@VikasBansal1989" />
  <meta name="citation_author" content="Stephen Capone" />
  <meta name="citation_author_institution" content="St. George&#39;s University School of Medicine, St. George&#39;s, Grenada" />
  <meta name="citation_author_orcid" content="0000-0001-7231-1535" />
  <meta name="citation_author" content="John J. Dziak" />
  <meta name="citation_author_institution" content="Edna Bennett Pierce Prevention Research Center, The Pennsylvania State University, University Park, PA, United States of America" />
  <meta name="citation_author_orcid" content="0000-0003-0762-5495" />
  <meta name="citation_author" content="YuCheng Sun" />
  <meta name="citation_author_institution" content="Department of Computer Science, University of Virginia, Charlottesville, VA, United States of America" />
  <meta name="citation_author" content="Yanjun Qi" />
  <meta name="citation_author_institution" content="Department of Computer Science, University of Virginia, Charlottesville, VA, United States of America" />
  <meta name="citation_author_orcid" content="0000-0002-5796-7453" />
  <meta name="citation_author" content="Lamonica Shinholster" />
  <meta name="citation_author_institution" content="Mercer University, Macon, GA, United States of America" />
  <meta name="citation_author_orcid" content="0000-0001-6285-005X" />
  <link rel="canonical" href="https://greenelab.github.io/covid19-review/" />
  <meta property="og:url" content="https://greenelab.github.io/covid19-review/" />
  <meta property="twitter:url" content="https://greenelab.github.io/covid19-review/" />
  <meta name="citation_fulltext_html_url" content="https://greenelab.github.io/covid19-review/" />
  <meta name="citation_pdf_url" content="https://greenelab.github.io/covid19-review/manuscript.pdf" />
  <link rel="alternate" type="application/pdf" href="https://greenelab.github.io/covid19-review/manuscript.pdf" />
  <link rel="alternate" type="text/html" href="https://greenelab.github.io/covid19-review/v/7eaea7d5410ae632e2e07351cf66810fe308a226/" />
  <meta name="manubot_html_url_versioned" content="https://greenelab.github.io/covid19-review/v/7eaea7d5410ae632e2e07351cf66810fe308a226/" />
  <meta name="manubot_pdf_url_versioned" content="https://greenelab.github.io/covid19-review/v/7eaea7d5410ae632e2e07351cf66810fe308a226/manuscript.pdf" />
  <meta property="og:type" content="article" />
  <meta property="twitter:card" content="summary_large_image" />
  <link rel="icon" type="image/png" sizes="192x192" href="https://manubot.org/favicon-192x192.png" />
  <link rel="mask-icon" href="https://manubot.org/safari-pinned-tab.svg" color="#ad1457" />
  <meta name="theme-color" content="#ad1457" />
  <!-- end Manubot generated metadata -->
</head>
<body>
<header id="title-block-header">
<h1 class="title">SARS-CoV-2 and COVID-19: An Evolving Review of Diagnostics and Therapeutics</h1>
</header>
<p><small><em>
This manuscript
(<a href="https://greenelab.github.io/covid19-review/v/7eaea7d5410ae632e2e07351cf66810fe308a226/">permalink</a>)
was automatically generated
from <a href="https://github.com/greenelab/covid19-review/tree/7eaea7d5410ae632e2e07351cf66810fe308a226">greenelab/covid19-review@7eaea7d</a>
on July 31, 2020.
</em></small></p>
<!-- include the Font Awesome library, per: https://fontawesome.com/start -->
<p><link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.13.0/css/all.css">
<span class="banner lightred"><span class="fas fa-info-circle fa-lg"></span> <strong>This in progress manuscript is not intended for the general public.</strong><br>
This is a review paper that is authored by scientists for an audience of scientists to discuss research that is in progress.
If you are interested in guidelines on testing, therapies, or other issues related to your health, you should not use this document.
Instead, you should collect information from your local health department, the <a href="https://www.cdc.gov/coronavirus/2019-ncov/index.html">CDC’s guidance</a>, or your own government.</span></p>
<h1 class="unnumbered" data-number="" id="authors">Authors</h1>
<ul>
<li><p><strong>Halie M. Rando</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-7688-1770">0000-0001-7688-1770</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/rando2">rando2</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/tamefoxtime">tamefoxtime</a><br>
<small>
Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America
· Funded by the Gordon and Betty Moore Foundation (GBMF 4552)
</small></p></li>
<li><p><strong>Casey S. Greene</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-8713-9213">0000-0001-8713-9213</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/cgreene">cgreene</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/GreeneScientist">GreeneScientist</a><br>
<small>
Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America; Childhood Cancer Data Lab, Alex’s Lemonade Stand Foundation, Philadelphia, Pennsylvania, United States of America
· Funded by the Gordon and Betty Moore Foundation (GBMF 4552)
</small></p></li>
<li><p><strong>Michael P. Robson</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-4859-0033">0000-0002-4859-0033</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/mprobson">mprobson</a><br>
<small>
Department of Computing Sciences, Villanova University, Villanova, Pennsylvania, United States of America
</small></p></li>
<li><p><strong>Simina M. Boca</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-1400-3398">0000-0002-1400-3398</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/SiminaB">SiminaB</a><br>
<small>
Innovation Center for Biomedical Informatics, Georgetown University Medical Center, Washington, District of Columbia, United States of America
</small></p></li>
<li><p><strong>Nils Wellhausen</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-8955-7582">0000-0001-8955-7582</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/nilswellhausen">nilswellhausen</a><br>
<small>
Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America
</small></p></li>
<li><p><strong>Ronan Lordan</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-9668-3368">0000-0001-9668-3368</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/RLordan">RLordan</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/el_ronan">el_ronan</a><br>
<small>
Institute for Translational Medicine and Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104-5158, USA
</small></p></li>
<li><p><strong>Christian Brueffer</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-3826-0989">0000-0002-3826-0989</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/cbrueffer">cbrueffer</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/cbrueffer">cbrueffer</a><br>
<small>
Department of Clinical Sciences, Lund University, Lund, Sweden
</small></p></li>
<li><p><strong>Sadipan Ray</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-9960-5768">0000-0002-9960-5768</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/rays1987">rays1987</a><br>
<small>
Department of Systems Pharmacology &amp; Translational Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA; Institute for Translational Medicine and Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA
</small></p></li>
<li><p><strong>Lucy D'Agostino McGowan</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-7297-9359">0000-0001-7297-9359</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/LucyMcGowan">LucyMcGowan</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/LucyStats">LucyStats</a><br>
<small>
Department of Mathematics and Statistics, Wake Forest University, Winston-Salem, North Carolina, United States of America
</small></p></li>
<li><p><strong>Anthony Gitter</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-5324-9833">0000-0002-5324-9833</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/agitter">agitter</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/anthonygitter">anthonygitter</a><br>
<small>
Department of Biostatistics and Medical Informatics, University of Wisconsin-Madison, Madison, Wisconsin, United States of America; Morgridge Institute for Research, Madison, Wisconsin, United States of America
</small></p></li>
<li><p><strong>Ronnie M. Russell</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-1484-4207">0000-0003-1484-4207</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/rmrussell">rmrussell</a><br>
<small>
Department of Microbiology, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA
</small></p></li>
<li><p><strong>Anna Ada Dattoli</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-1462-831X">0000-0003-1462-831X</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/aadattoli">aadattoli</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/aadattoli">aadattoli</a><br>
<small>
Department of Systems Pharmacology &amp; Translational Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA
</small></p></li>
<li><p><strong>Ryan Velazquez</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-3655-3403">0000-0002-3655-3403</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/rdvelazquez">rdvelazquez</a><br>
<small>
Azimuth1, McLean, VA
</small></p></li>
<li><p><strong>John P. Barton</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-1467-421X">0000-0003-1467-421X</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/johnbarton">johnbarton</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/_jpbarton">_jpbarton</a><br>
<small>
Department of Physics and Astronomy, University of California-Riverside, Riverside, California, United States of America
</small></p></li>
<li><p><strong>Jeffrey M. Field</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-7161-7284">0000-0001-7161-7284</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/Jeff-Field">Jeff-Field</a><br>
<small>
Department of Pharmacology, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA
</small></p></li>
<li><p><strong>Bharath Ramsundar</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-8450-4262">0000-0001-8450-4262</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/rbharath">rbharath</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/rbhar90">rbhar90</a><br>
<small>
The DeepChem Project, https://deepchem.io/
</small></p></li>
<li><p><strong>Adam L. MacLean</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-0689-7907">0000-0003-0689-7907</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/alavendelm">alavendelm</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/adamlmaclean">adamlmaclean</a><br>
<small>
Department of Biological Sciences, University of Southern California, Los Angeles, California, United States of America
</small></p></li>
<li><p><strong>Alexandra J. Lee</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-0208-3730">0000-0002-0208-3730</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/ajlee21">ajlee21</a><br>
<small>
Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America
· Funded by the Gordon and Betty Moore Foundation (GBMF 4552)
</small></p></li>
<li><p><strong>Immunology Institute of the Icahn School of Medicine</strong><br>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/ismms-himc">ismms-himc</a><br>
<small>
Immunology Institute of the Icahn School of Medicine
</small></p></li>
<li><p><strong>Fengling Hu</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-1081-5038">0000-0003-1081-5038</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/hufengling">hufengling</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/hufengling">hufengling</a><br>
<small>
Department of Biostatistics, Epidemiology and Informatics, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America
</small></p></li>
<li><p><strong>Nafisa M. Jadavji</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-3557-7307">0000-0002-3557-7307</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/nafisajadavji">nafisajadavji</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/nafisajadavji">nafisajadavji</a><br>
<small>
Biomedical Science, Midwestern University, Glendale, AZ, United States of America; Department of Neuroscience, Carleton University, Ottawa, Ontario, Canada
· Funded by the American Heart Association (20AIREA35050015)
</small></p></li>
<li><p><strong>Elizabeth Sell</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-9658-1107">0000-0002-9658-1107</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/esell17">esell17</a><br>
<small>
Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America
</small></p></li>
<li><p><strong>Jinhui Wang</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-5796-8130">0000-0002-5796-8130</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/jinhui2">jinhui2</a><br>
<small>
Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America
</small></p></li>
<li><p><strong>Diane N. Rafizadeh</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-2838-067X">0000-0002-2838-067X</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/dianerafi">dianerafi</a><br>
<small>
Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America; Department of Chemistry, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America
· Funded by NIH Medical Scientist Training Program T32 GM07170
</small></p></li>
<li><p><strong>Ashwin N. Skelly</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-1565-3376">0000-0002-1565-3376</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/anskelly">anskelly</a><br>
<small>
Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America; Institute for Immunology, University of Pennsylvania Perelman School of Medicine, Philadelphia, United States of America
· Funded by NIH Medical Scientist Training Program T32 GM07170
</small></p></li>
<li><p><strong>Marouen Ben Guebila</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-5934-966X">0000-0001-5934-966X</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/marouenbg">marouenbg</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/marouenbg">marouenbg</a><br>
<small>
Department of Biostatistics, Harvard School of Public Health, Boston, Massachusetts, United States of America
</small></p></li>
<li><p><strong>Likhitha Kolla</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-1169-906X">0000-0002-1169-906X</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/likhithakolla">likhithakolla</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/lkolla2018">lkolla2018</a><br>
<small>
Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America
· Funded by NIH Medical Scientist Training Program T32 GM07170
</small></p></li>
<li><p><strong>David Manheim</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-8599-8380">0000-0001-8599-8380</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/davidmanheim">davidmanheim</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/davidmanheim">davidmanheim</a><br>
<small>
Risk and Health Communication Research Center, School of Public Health, University of Haifa, Haifa, Israel
</small></p></li>
<li><p><strong>Soumita Ghosh</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-2783-2750">0000-0002-2783-2750</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/soumitagh">soumitagh</a><br>
<small>
Institute of Translational Medicine and Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America
</small></p></li>
<li><p><strong>Matthias Fax</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-0064-395X">0000-0002-0064-395X</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/matfax">matfax</a><br>
<small>
greensolid technologies, Bavaria, Germany; IEEE Reliability Society, Vienna, Austria
</small></p></li>
<li><p><strong>James Brian Byrd</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-0509-3520">0000-0002-0509-3520</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/byrdjb">byrdjb</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/thebyrdlab">thebyrdlab</a><br>
<small>
University of Michigan School of Medicine, Ann Arbor, Michigan, United States of America
· Funded by NIH K23HL128909; FastGrants
</small></p></li>
<li><p><strong>YoSon Park</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-0465-4744">0000-0002-0465-4744</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/ypar">ypar</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/__yoson__"><strong>yoson</strong></a><br>
<small>
Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America
· Funded by NHGRI R01 HG10067
</small></p></li>
<li><p><strong>Yael Evelyn Marshall</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-3723-9683">0000-0003-3723-9683</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/yemarshall">yemarshall</a><br>
<small>
recent PhD, Princeton University, Princeton, NJ, USA; freelance copyeditor, multiple locations, USA
</small></p></li>
<li><p><strong>Vikas Bansal</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-0944-7226">0000-0002-0944-7226</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/bansalvi">bansalvi</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/VikasBansal1989">VikasBansal1989</a><br>
<small>
Biomedical Data Science and Machine Learning Group, DZNE, Tübingen, Germany
</small></p></li>
<li><p><strong>Stephen Capone</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-7231-1535">0000-0001-7231-1535</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/scapone01">scapone01</a><br>
<small>
St. George’s University School of Medicine, St. George’s, Grenada
</small></p></li>
<li><p><strong>John J. Dziak</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-0762-5495">0000-0003-0762-5495</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/dziakj1">dziakj1</a><br>
<small>
Edna Bennett Pierce Prevention Research Center, The Pennsylvania State University, University Park, PA, United States of America
</small></p></li>
<li><p><strong>YuCheng Sun</strong><br>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/kevinsunofficial">kevinsunofficial</a><br>
<small>
Department of Computer Science, University of Virginia, Charlottesville, VA, United States of America
</small></p></li>
<li><p><strong>Yanjun Qi</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-5796-7453">0000-0002-5796-7453</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/qiyanjun">qiyanjun</a><br>
<small>
Department of Computer Science, University of Virginia, Charlottesville, VA, United States of America
</small></p></li>
<li><p><strong>Lamonica Shinholster</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-6285-005X">0000-0001-6285-005X</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/LSH2126">LSH2126</a><br>
<small>
Mercer University, Macon, GA, United States of America
</small></p></li>
</ul>
<h1 class="page_break_before unnumbered" data-number="" id="abstract">Abstract</h1>
<p>Since late 2019, Coronavirus disease 2019 (COVID-19) has spread around the world, resulting in the declaration of a pandemic by the World Health Organization (WHO).
This infectious disease is caused by the newly identified severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2).
Research on the virus SARS-CoV-2 and the disease it causes is emerging rapidly through global scientific efforts.
The development of approaches for the diagnosis and treatment of the disease will be critical to mitigating the impact of the virus.
Scientific discussion of new and existing technologies and methods under investigation must be contextualized alongside a solid fundamental understanding of the virus and the disease it causes.</p>
<p>This manuscript represents a collaborative effort to organize and consolidate the rapidly emerging scientific literature related to SARS-CoV-2 and COVID-19.
We present information about the virus in the context of what is known about related viruses, describe the pathogenesis of COVID-19, and synthesize studies emerging about the diagnosis and treatment of COVID-19 alongside literature about related illnesses.
A broad scientific effort to understand this pandemic and related viruses and diseases will be fundamental to efforts to predict possible interventions.
This text is an evolving and collaborative document that seeks to incorporate the ever-expanding body of information related to SARS-CoV-2 and COVID-19.</p>
<h1 class="unnumbered" data-number="" id="where-to-contribute">Where to Contribute</h1>
<p>Introduce Yourself (GitHub Issue) <a href="https://github.com/greenelab/covid19-review/issues/17">https://github.com/greenelab/covid19-review/issues/17</a></p>
<p>Community Chat (Gitter Room) <a href="https://gitter.im/covid19-review/community">https://gitter.im/covid19-review/community</a></p>
<p>More Info (GitHub Readme) <a href="https://github.com/greenelab/covid19-review#sars-cov-2-and-covid-19-an-evolving-review-of-diagnostics-and-therapeutics">https://github.com/greenelab/covid19-review#sars-cov-2-and-covid-19-an-evolving-review-of-diagnostics-and-therapeutics</a></p>
<h1 class="page_break_before" data-number="1" id="introduction"><span class="header-section-number">1</span> Introduction</h1>
<h2 data-number="1.1" id="general-background"><span class="header-section-number">1.1</span> General Background</h2>
<p>On January 21, 2020, the World Health Organization (WHO) released its first report concerning what is now known as the Coronavirus Disease 2019 (COVID-19) <span class="citation" data-cites="Vb5KFWnO">[<a href="#ref-Vb5KFWnO" role="doc-biblioref">1</a>]</span>.
This infectious disease came to international attention on December 31, 2019 following an announcement by national officials in China describing 44 cases of a respiratory infection of unknown cause.
The first known cases were located in Wuhan City within the Hubei province of China, but the disease spread rapidly beyond Wuhan to throughout China and subsequently around the world.
At the time of the WHO’s first situation report <span class="citation" data-cites="Vb5KFWnO">[<a href="#ref-Vb5KFWnO" role="doc-biblioref">1</a>]</span>, 282 confirmed cases had been identified.
Most of these cases were in China, but 1-2 exported cases had also been identified in each of several neighboring countries (Thailand, Japan, and the Republic of Korea).
One week later, 4,593 confirmed cases had been identified, spanning not only Asia, but also Australia, North America, and Europe <span class="citation" data-cites="fLTiwToY">[<a href="#ref-fLTiwToY" role="doc-biblioref">2</a>]</span>.
On March 11, 2020, WHO formally classified the situation as a pandemic <span class="citation" data-cites="x9975iBK">[<a href="#ref-x9975iBK" role="doc-biblioref">3</a>]</span>.
On April 4, 2020, the WHO reported that the global number of confirmed cases had surpassed one million <span class="citation" data-cites="1GACP7OZY">[<a href="#ref-1GACP7OZY" role="doc-biblioref">4</a>]</span>.
667,011 COVID-19 deaths had been reported worldwide as of July 29, 2020 (Figure <a href="#fig:csse-deaths">1</a>).</p>
<div id="fig:csse-deaths" class="fignos">
<figure>
<img src="https://github.com/greenelab/covid19-review/raw/3ebd63d13ddfc066804429b847fd10caba7a26c3/csse/csse-deaths.svg" alt="" /><figcaption><span>Figure 1:</span> <strong>Cumulative global COVID-19 deaths since January 22, 2020.</strong>
Data are from the COVID-19 Data Repository by the Center for Systems Science and Engineering at Johns Hopkins University <span class="citation" data-cites="MrwDDw9R">[<a href="#ref-MrwDDw9R" role="doc-biblioref">5</a>]</span>.</figcaption>
</figure>
</div>
<p>Several review articles on aspects of COVID-19 have already been published.
These have included reviews on the disease epidemiology <span class="citation" data-cites="I2EsJmfs">[<a href="#ref-I2EsJmfs" role="doc-biblioref">6</a>]</span>, immunological response <span class="citation" data-cites="5x25saIz">[<a href="#ref-5x25saIz" role="doc-biblioref">7</a>]</span>, diagnostics <span class="citation" data-cites="evtsR3C5">[<a href="#ref-evtsR3C5" role="doc-biblioref">8</a>]</span>, and pharmacological treatments <span class="citation" data-cites="5x25saIz 18eCxyLhx">[<a href="#ref-5x25saIz" role="doc-biblioref">7</a>,<a href="#ref-18eCxyLhx" role="doc-biblioref">9</a>]</span>.
<span class="citation" data-cites="SAE5ME3N">[<a href="#ref-SAE5ME3N" role="doc-biblioref">10</a>]</span> and <span class="citation" data-cites="xOs5ctsW">[<a href="#ref-xOs5ctsW" role="doc-biblioref">11</a>]</span> provide relatively brief narrative reviews of progress on some important ongoing COVID-19 research questions.
However, research on these topics is proceeding so quickly that any static review is likely to quickly become dated.</p>
<p>In this review, we seek to consolidate information about the virus in the context of related viruses and to synthesize what is known about the diagnosis and treatment of COVID-19 and related diseases.
Further, we aim to amplify the true signal out of the vast noise produced by thousands of publications on the topic <span class="citation" data-cites="1HZeeO4Cs">[<a href="#ref-1HZeeO4Cs" role="doc-biblioref">12</a>]</span>.
We will critique, sort and distill informative content out of the overwhelming flood of information and help the larger scientific community to be better educated on this critical subject.
Thus, our approach has been to develop a real-time, collaborative effort that welcomes submissions from scientists worldwide.</p>
<h2 data-number="1.2" id="about-coronaviruses"><span class="header-section-number">1.2</span> About Coronaviruses</h2>
<h3 data-number="1.2.1" id="coronavirus-genome-and-structure"><span class="header-section-number">1.2.1</span> Coronavirus Genome and Structure</h3>
<p>Coronaviruses (CoVs; order <em>Nidovirales</em>, family <em>Coronaviridae</em>, subfamily <em>Orthocoronavirinae</em>) are enveloped viruses with some of the largest viral RNA genomes, ranging from 27 kilobases (Kb) to 32 Kb in length.
The SARS-CoV-2 genome lies in the middle of this range at 29,903 bp <span class="citation" data-cites="1Fqilxaum">[<a href="#ref-1Fqilxaum" role="doc-biblioref">13</a>]</span>.
There are several fundamental genomic characteristics shared by all viruses in the <em>Nidovirales</em> order, including being enveloped and non-segmented and having positive-sense (ssRNA+) genomes that contain a large number of non-structural genes originating through ribosomal frameshifting <span class="citation" data-cites="47hnjvDb">[<a href="#ref-47hnjvDb" role="doc-biblioref">14</a>]</span>.
Genome organization is highly conserved within the order <span class="citation" data-cites="47hnjvDb">[<a href="#ref-47hnjvDb" role="doc-biblioref">14</a>]</span>.
A replicase gene comprises about two-thirds of the genome of coronaviruses.
This polypeptide is translated into 16 non-structural proteins (nsp1-16, except in Gammacoronaviruses, where nsp1 is absent), which form replication machinery used to synthesize viral RNA <span class="citation" data-cites="18GflyMj">[<a href="#ref-18GflyMj" role="doc-biblioref">15</a>]</span>.
The remaining third of the genome encodes structural proteins, including the spike (S), membrane (M), envelope (E), and nucleocapsid (N) proteins.
Additional accessory genes are sometimes present between these two regions, depending on the species or strain.
<!-- To do: Add a figure showing the structure of the genome.-->
<!-- Also, everything focuses on S! Should we also talk about others?--></p>
<p>Coronaviruses are classified into four genera: alpha, beta, delta and gamma coronaviruses.
Among them, alpha and beta coronaviruses infect mammalian species, gamma coronaviruses infect avian species and delta coronaviruses infect both mammalian and avian species <span class="citation" data-cites="17DSmRo9H">[<a href="#ref-17DSmRo9H" role="doc-biblioref">16</a>]</span>.
The viruses were initially subdivided into these genera based on antigenic relationships of the spike (S), membrane (M), envelope (E), and nucleocapsid (N) proteins but are now divided by phylogenetic clustering.
<!-- this section may be better as a figure panel + legend-->
Phylogenetic analysis of a PCR amplicon fragment from five patients along with the total virus genome of 29.8 Kb indicates that SARS-CoV-2 is a novel betacoronavirus belonging to the B lineage, also known as sarbecovirus, which also includes the human SARS coronavirus <span class="citation" data-cites="IFLf8rsY">[<a href="#ref-IFLf8rsY" role="doc-biblioref">17</a>]</span>.</p>
<p>Most coronaviruses are considered zoonotic viruses with little to no transmission observed in humans.
A major group of coronaviruses include human coronavirus (HCoV) strains associated with multiple respiratory diseases of varying severity, ranging from common cold to severe pneumonia, with severe symptoms mostly observed in immunocompromised individuals <span class="citation" data-cites="rDtyU6cv">[<a href="#ref-rDtyU6cv" role="doc-biblioref">18</a>]</span>.
Approximately one-third of common cold infections in humans is attributable to four out of six previously known human coronaviruses (HCoV-229E, HCoV-NL63, HCoV-OC43 and HCoV-HKU1) that are globally circulating in the human population <span class="citation" data-cites="UBTUyMGV YAhVzI0A">[<a href="#ref-UBTUyMGV" role="doc-biblioref">19</a>,<a href="#ref-YAhVzI0A" role="doc-biblioref">20</a>]</span>.
In the past two decades, however, highly pathogenic human coronaviruses have been identified, including the severe acute respiratory syndrome coronavirus 1 (SARS-CoV-1) and the Middle East respiratory syndrome coronavirus (MERS-CoV) although both infections were confined to specific geographic regions <span class="citation" data-cites="G5NJrE75 UBTUyMGV">[<a href="#ref-UBTUyMGV" role="doc-biblioref">19</a>,<a href="#ref-G5NJrE75" role="doc-biblioref">21</a>]</span>.
The current pandemic of COVID-19, caused by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), represents an acute and rapidly spreading global health crisis with symptoms for reported cases ranging from mild to severe or fatal <span class="citation" data-cites="gHWlMufv">[<a href="#ref-gHWlMufv" role="doc-biblioref">22</a>]</span> and including outcomes such as acute respiratory distress, acute lung injury and other pulmonary complications.
The possibility of asymptomatic cases is also being investigated.
The transmission and mortality rate estimations of COVID-19 remain to be determined.
<!-- I think there are better estimates now (about 50% asymptomatic I think, once you take out the presymptomatic).-->
<!-- Most credible IFR estimates I've seen are between 0.5-1.5%, depending on the population. CFRs will be greater and vary a lot.--></p>
<h3 data-number="1.2.2" id="transmission-and-evolution"><span class="header-section-number">1.2.2</span> Transmission and Evolution</h3>
<p>SARS-CoV-2 is a novel human coronavirus, following SARS-CoV and MERS-CoV to become the third coronavirus causing severe pneumonia in humans to emerge in the 21st century <span class="citation" data-cites="uXEsLlzb">[<a href="#ref-uXEsLlzb" role="doc-biblioref">23</a>]</span>.
Though the intermediate host serving as the source for the zoonotic introduction of SARS-CoV-2 to human populations has not yet been identified, the SARS-CoV-2 virus has been placed within the coronavirus phylogeny through comparative genomic analyses.
Genomic analyses and comparisons to other known coronaviruses suggest that SARS-CoV-2 is unlikely to have originated in a laboratory – either purposely engineered and released, or escaped – and instead evolved naturally in an animal host <span class="citation" data-cites="NEIoHDjq">[<a href="#ref-NEIoHDjq" role="doc-biblioref">24</a>]</span>.
Among known coronaviruses, SARS-CoV-2 has the closest overall sequence similarity to RaTG13 (~96%) found in a <em>Rhinolophus affinis</em> bat <span class="citation" data-cites="VSkK7CeP">[<a href="#ref-VSkK7CeP" role="doc-biblioref">25</a>]</span>, while the receptor binding domain (RBD) is highly similar to that of viruses found in pangolins <span class="citation" data-cites="eQIU1Nbs">[<a href="#ref-eQIU1Nbs" role="doc-biblioref">26</a>]</span>.
This suggests that SARS-CoV-2 may have originated in viral reservoirs of similar hosts; however, current evidence cannot discriminate an origin of the virus before or after zoonotic transfer to humans <span class="citation" data-cites="NEIoHDjq">[<a href="#ref-NEIoHDjq" role="doc-biblioref">24</a>]</span>.</p>
<p>In general, respiratory viruses like coronavirus can have multiple routes of person-to-person transmission including droplet transmission (i.e. inhalation for cough, sneeze), aerosol transmission (i.e. virus suspended in air), and contact transmission (i.e. contact with oral, nasal, and eye mucous membranes).
Other modes of transmission, such as through touching surfaces or objects and then touching mucous membranes, can also be investigated.
While droplet-based and contact were initially considered to be the primary modes of SARS-CoV-2 transmission <span class="citation" data-cites="vP6yTZ0y">[<a href="#ref-vP6yTZ0y" role="doc-biblioref">27</a>]</span>, as additional information has emerged, the possibility of aerosol transmission has also been raised <span class="citation" data-cites="16SDeiudC yXKcviw8 9ouDX5IN">[<a href="#ref-16SDeiudC" role="doc-biblioref">28</a>,<a href="#ref-yXKcviw8" role="doc-biblioref">29</a>,<a href="#ref-9ouDX5IN" role="doc-biblioref">30</a>]</span>.
Other aspects of transmission to investigate are the relationship between infectiousness and virus shedding with disease period or symptoms and also the proportions of cases that are attributable to various types of transmission events, such as transmission between relatives, nosocomial transmissions, and other possible types of interactions.
Some information about these characteristics of transmission are available for the highly pathogenic coronaviruses SARS-CoV and MERS-CoV <span class="citation" data-cites="rjVw7V94 G5NJrE75">[<a href="#ref-G5NJrE75" role="doc-biblioref">21</a>,<a href="#ref-rjVw7V94" role="doc-biblioref">31</a>]</span>.
For SARS-CoV-2 it is still being investigated whether, in addition to being spread by people who show symptoms, the virus can be transmitted by people who do not show symptoms <span class="citation" data-cites="Oyy1OUTT">[<a href="#ref-Oyy1OUTT" role="doc-biblioref">32</a>]</span>, such as during the pre-symptomatic stage of infection or in people with asymptomatic infections.
<!-- I think there are some more recent papers that look at the possibility of infection from asymptomatic individuals.--></p>
<h3 data-number="1.2.3" id="cell-entry-and-replication"><span class="header-section-number">1.2.3</span> Cell Entry and Replication</h3>
<p>Coronavirus virions are spherical with diameters ranging from 100 to 160 nanometers (nm).
The virion is made up of a lipid envelope in which peplomers of 2-3 spike (S) glycoproteins are anchored, creating a distinctive “crown” shape for which the family of viruses was named <span class="citation" data-cites="RGAUx68v DlBAegL1">[<a href="#ref-RGAUx68v" role="doc-biblioref">33</a>,<a href="#ref-DlBAegL1" role="doc-biblioref">34</a>]</span>.
The replication process is initiated by the viral spike protein: first, the virus binds to a host cell, and then the viral membrane fuses with the endosomal membrane to release the viral genome into the host cytoplasm.
<!-- It seems like this info is on SARS-CoV, not on SARS-CoV-2. I think we should only include the parts that are relevant or maybe note that some things may not be known about SARS-CoV-2, in which case, we make assumptions based on SARS-CoV.-->
The coronavirus spike protein is structured in three segments: the ectodomain, a transmembrane anchor, and an intracellular tail <span class="citation" data-cites="skHKTFMe">[<a href="#ref-skHKTFMe" role="doc-biblioref">35</a>]</span>.
The ectodomain contains two subdomains known as the S1 and S2 subunits, with the S1 (N-terminal) subunit guiding the binding of the virus to a host cell receptor and the S2 (C-terminal) subunit guiding the fusion process <span class="citation" data-cites="skHKTFMe">[<a href="#ref-skHKTFMe" role="doc-biblioref">35</a>]</span>.
<!-- Is it worth going into more detail about siganl peptides and secretory pathways? Not sure whether this is relevant information.-->
The ectodomain forms the crown-like structures on the viral membrane, with the S1 domain forming the head of the crown and containing the Receptor Binding Motif (RBM) that binds a host receptor, while the S2 domain forms the stalk that supports the head <span class="citation" data-cites="ueMPuUMe">[<a href="#ref-ueMPuUMe" role="doc-biblioref">36</a>]</span>.
Among betacoronaviruses, some are known to bind to CEACAM1, Neu 5,9 Ac2, and ACE2 <span class="citation" data-cites="13wWdgODZ">[<a href="#ref-13wWdgODZ" role="doc-biblioref">37</a>]</span>.
Both SARS-CoV and SARS-CoV-2 bind to ACE2.</p>
<p>After the virus binds to a host cell receptor, the viral and plasma membranes must then fuse.
Many coronaviruses are cleaved at the S1-S2 boundary and remain non-covalently bound until fusion to the host cell is achieved, with the S1 subunit stabilizing the S2 subunit during the membrane fusion process <span class="citation" data-cites="qcVbT0w4">[<a href="#ref-qcVbT0w4" role="doc-biblioref">38</a>]</span>.
This priming of the S protein is induced by the binding of cellular proteases <span class="citation" data-cites="kCVy8vjj YeKaMzdJ">[<a href="#ref-kCVy8vjj" role="doc-biblioref">39</a>,<a href="#ref-YeKaMzdJ" role="doc-biblioref">40</a>]</span>.
Angiotensin-Converting Enzyme 2 (ACE2) and Transmembrane Serine Protease 2 (TMPRSS2) have been identified as a prime receptor and a critical protease, respectively, facilitating SARS-CoV/CoV-2 entry into a target cell <span class="citation" data-cites="2QTH37Xi CqQVkaqj MXWaV7SO 15l3di3Wj JjrO6YbD">[<a href="#ref-2QTH37Xi" role="doc-biblioref">41</a>,<a href="#ref-CqQVkaqj" role="doc-biblioref">42</a>,<a href="#ref-MXWaV7SO" role="doc-biblioref">43</a>,<a href="#ref-15l3di3Wj" role="doc-biblioref">44</a>,<a href="#ref-JjrO6YbD" role="doc-biblioref">45</a>]</span>.
This finding has led to a hypothesized role for ACE2 and TMPRSS2 expression in determining which cells, tissues and organs are most likely to be infected by SARS-CoV-2.
A second cleavage site within S2 is thought to activate <em>S</em> for fusion by inducing conformational change <span class="citation" data-cites="qcVbT0w4">[<a href="#ref-qcVbT0w4" role="doc-biblioref">38</a>]</span>.
Electron microscopy suggests that in some coronaviruses, including SARS-CoV and MERS-CoV, a six-helix bundle separates the two subunits in the postfusion conformation, and the unusual length of this bundle facilitates membrane fusion through the release of additional energy <span class="citation" data-cites="17DSmRo9H">[<a href="#ref-17DSmRo9H" role="doc-biblioref">16</a>]</span>.
However, research in SARS-CoV suggests that, in addition to fusion with the plasma membrane, viruses may also be taken up by cells through endocytosis <span class="citation" data-cites="FFwktXgx">[<a href="#ref-FFwktXgx" role="doc-biblioref">46</a>]</span>.
<!-- This is very interesting and seems like it could be relevant to virulence (?), but I don't know enough cell biology to expand on it. Another reference that seems useful is doi.org/10.1146/annurev-biochem-060208-104626-->
Once the virus enters a host cell, the replicase gene is translated and assembled into the viral replicase complex, which can synthesize dsRNA genome from the genomic ssRNA(+).
Finally, the dsRNA genome is transcribed and replicated to create viral mRNAs and new ssRNA(+) genomes <span class="citation" data-cites="47hnjvDb ZUuGRxk9">[<a href="#ref-47hnjvDb" role="doc-biblioref">14</a>,<a href="#ref-ZUuGRxk9" role="doc-biblioref">47</a>]</span>.</p>
<p>Thus, most of the viral entry process is determined by the viral genome, but the initial step, when the virus binds to a host receptor, also depends on the virus being able to recognize the host receptor.
Coronaviruses such as SARS-CoV, SARS-CoV-2, and MERS-CoV may therefore be engaged in an evolutionary arms race with the human immune system <span class="citation" data-cites="EGPffQf6 Ovrrh22d ZBvTot1X">[<a href="#ref-EGPffQf6" role="doc-biblioref">48</a>,<a href="#ref-Ovrrh22d" role="doc-biblioref">49</a>,<a href="#ref-ZBvTot1X" role="doc-biblioref">50</a>]</span>.
Two “hot spots” within human ACE2 have been identified as driving the strong binding affinity between this receptor and two sites in SARS-CoV <span class="citation" data-cites="17DSmRo9H">[<a href="#ref-17DSmRo9H" role="doc-biblioref">16</a>]</span>.
Coronaviruses bind to a range of host receptors <span class="citation" data-cites="13wWdgODZ OVsxrEuX">[<a href="#ref-13wWdgODZ" role="doc-biblioref">37</a>,<a href="#ref-OVsxrEuX" role="doc-biblioref">51</a>]</span>, with binding conserved only at the genus level <span class="citation" data-cites="17DSmRo9H">[<a href="#ref-17DSmRo9H" role="doc-biblioref">16</a>]</span>.
The S1 domain of the spike protein, which contains the RBM, evolves more rapidly than <em>S</em>’s S2 domain <span class="citation" data-cites="13wWdgODZ OVsxrEuX">[<a href="#ref-13wWdgODZ" role="doc-biblioref">37</a>,<a href="#ref-OVsxrEuX" role="doc-biblioref">51</a>]</span>.
However, even within the S1 domain, some regions are more conserved than others, with the receptors in S1’s N-terminal domain (S1-NTD) evolving more rapidly than those in its C-terminal domain (S1-CTD) <span class="citation" data-cites="OVsxrEuX">[<a href="#ref-OVsxrEuX" role="doc-biblioref">51</a>]</span>.
Both S1-NTD and S1-CTD are involved in receptor binding and can function as receptor binding domains (RBDs) to bind proteins and sugars <span class="citation" data-cites="13wWdgODZ">[<a href="#ref-13wWdgODZ" role="doc-biblioref">37</a>]</span>, but RBDs in the S1-NTD typically bind to sugars, while those in the S1-CTD recognize protein receptors <span class="citation" data-cites="17DSmRo9H">[<a href="#ref-17DSmRo9H" role="doc-biblioref">16</a>]</span>.
Viral receptors show higher affinity with protein receptors than sugar receptors <span class="citation" data-cites="17DSmRo9H">[<a href="#ref-17DSmRo9H" role="doc-biblioref">16</a>]</span>, which suggests that positive selection on or relaxed conservation of the S1-NTD preferentially might reduce the risk of a deleterious mutation that would prevent binding.
<!--also positive selection on MHC/HLA? Seems worth mentioning somewhere, possibly here?...-->
Cell entry by the virus is a critical component to pathogenesis and therefore an important process to understand when examining possible therapeutics.
Pathogenesis is made up of three major components: entry, replication, and spread <span class="citation" data-cites="jsMmdH4D">[<a href="#ref-jsMmdH4D" role="doc-biblioref">52</a>]</span>.
<!-- We should probably mention more about spread in this section & add spread to the title... or make a spread section--></p>
<h2 data-number="1.3" id="covid-19-infection-and-presentation"><span class="header-section-number">1.3</span> COVID-19: Infection and Presentation</h2>
<h3 data-number="1.3.1" id="immunological-mechanisms-of-coronavirus-driven-disease-in-humans"><span class="header-section-number">1.3.1</span> Immunological Mechanisms of Coronavirus-driven Disease in Humans</h3>
<p>One way that humans respond to viral threats is through the immune response.
The human immune system utilizes a variety of innate and adaptive responses to protect against the pathogens it encounters.
The innate immune system consists of barriers, such as the skin, mucous secretions, neutrophils, macrophages, and dendritic cells.
It also includes cell-surface receptors that can recognize the molecular patterns of pathogens.
The adaptive immune system utilizes antigen-specific receptors that are expressed on B and T lymphocytes.
These components of the immune system typically act together; the innate response acts first, and the adaptive response begins to act several days after initial infection following the clonal expansion of T and B cells <span class="citation" data-cites="icy0PeD7">[<a href="#ref-icy0PeD7" role="doc-biblioref">53</a>]</span>.</p>
<p>After a virus enters into a host cell, its antigen is presented by major histocompatibility complex 1 (MHC 1) molecules and is then recognized by cytotoxic T lymphocytes.
One of the main immune responses that contribute to the onset of Acute Respiratory Distress Syndrome (ARDS) in COVID-19 patients is the cytokine storm, which causes an extreme inflammatory response due to a release of pro-inflammatory cytokines and chemokines by immune effector cells.
In addition to respiratory distress, this mechanism leads to organ failure and death in severe COVID-19 cases <span class="citation" data-cites="OlEbK4fc">[<a href="#ref-OlEbK4fc" role="doc-biblioref">54</a>]</span>.</p>
<h3 data-number="1.3.2" id="clinical-presentation-of-covid-19"><span class="header-section-number">1.3.2</span> Clinical Presentation of COVID-19</h3>
<p>A great diversity of symptom profiles has been observed for COVID-19, although a large study from Wuhan, China suggests fever and cough as the two most common symptoms on admission <span class="citation" data-cites="15IDA5kX6">[<a href="#ref-15IDA5kX6" role="doc-biblioref">55</a>]</span>.
One early retrospective study in China described the clinical presentations of patients infected with SARS-CoV-2 as including lower respiratory tract infection with fever, dry cough, and dyspnea <span class="citation" data-cites="10THxyeCg">[<a href="#ref-10THxyeCg" role="doc-biblioref">56</a>]</span>.
This study <span class="citation" data-cites="10THxyeCg">[<a href="#ref-10THxyeCg" role="doc-biblioref">56</a>]</span> noted that upper respiratory tract symptoms were less common, which suggests that the virus targets cells located in the lower respiratory tract.
However, data from the New York City region <span class="citation" data-cites="19ytsiSpq 1H8gSsSkn">[<a href="#ref-19ytsiSpq" role="doc-biblioref">57</a>,<a href="#ref-1H8gSsSkn" role="doc-biblioref">58</a>]</span> showed variable rates of fever as a presenting symptom, suggesting that symptoms may not be consistent across samples.
These differences are present when comparing both between institutions in similar locations and between different regions experiencing COVID-19 outbreaks, leading to conflicting reports of the frequency of fever as a presenting symptom for patients upon hospital admission.
For example, even within New York City, one study <span class="citation" data-cites="19ytsiSpq">[<a href="#ref-19ytsiSpq" role="doc-biblioref">57</a>]</span> identified low oxygen saturation (&lt;90% without the use of supplemental oxygen or ventilation support) in a significant percentage of patients on presentation, while another study <span class="citation" data-cites="1H8gSsSkn">[<a href="#ref-1H8gSsSkn" role="doc-biblioref">58</a>]</span> reported cough, fever, and dyspnea as the most common presenting symptoms.
The variability of both which symptoms present and their severity makes it difficult for public health agencies to provide clear recommendations for citizens regarding what symptoms indicate SARS-CoV-2 infection and should prompt isolation.</p>
<p>The significant range of individual outcomes observed has led to interest in which factors influence disease severity.
The reported hospitalization rates have been wide-ranging and appear to be influenced by age and location, with several underlying health conditions being disproportionately reported among hospitalized patients <span class="citation" data-cites="xZBEFXiG">[<a href="#ref-xZBEFXiG" role="doc-biblioref">59</a>]</span>.
<!--Should emphasize that the risk goes up a lot with age, with maleness and diabetes/obesity also being risk factors.-->
Among patients who are admitted to the hospital, outcomes have generally been poor, with rates of admission to the intensive care unit (ICU) upwards of 15% in both Wuhan, China and Italy <span class="citation" data-cites="15IDA5kX6 11xfoqpEl qlJ8hCUs">[<a href="#ref-15IDA5kX6" role="doc-biblioref">55</a>,<a href="#ref-11xfoqpEl" role="doc-biblioref">60</a>,<a href="#ref-qlJ8hCUs" role="doc-biblioref">61</a>]</span>.
Several studies have also investigated which factors are likely to influence COVID-19 outcomes.
For example, a Chinese retrospective study <span class="citation" data-cites="10THxyeCg">[<a href="#ref-10THxyeCg" role="doc-biblioref">56</a>]</span> found that a higher probability of mortality was associated with older age and higher Sequential Organ Failure Assessment scores, as well as high levels of d-dimer.
Mortality might be associated with other biomarkers measured in blood samples including lactate dehydrogenase and cardiac troponin I, although these analyses may not have been appropriately corrected for multiple testing.
<span class="citation" data-cites="10THxyeCg">[<a href="#ref-10THxyeCg" role="doc-biblioref">56</a>]</span> reported that survivors continued to shed the virus for a median of 20 days and a maximum of at least 37 days.
A later study reported radiographic findings such as ground-glass opacity and bilateral patchy shadowing in the lungs of many hospitalized patients, and most COVID-19 patients had lymphocytopenia, meaning they had low levels of lymphocytes (a type of white blood cell) <span class="citation" data-cites="15IDA5kX6">[<a href="#ref-15IDA5kX6" role="doc-biblioref">55</a>]</span>.</p>
<p>COVID-19 can affect diverse body systems in addition to causing respiratory problems <span class="citation" data-cites="wK2afyL8">[<a href="#ref-wK2afyL8" role="doc-biblioref">62</a>]</span>.
For example, COVID-19 can lead to acute kidney injury, especially in patients with severe respiratory symptoms or certain preexisting conditions <span class="citation" data-cites="17hdoiBo9">[<a href="#ref-17hdoiBo9" role="doc-biblioref">63</a>]</span>.
It can also cause neurological complications <span class="citation" data-cites="1CqoZGSKK 1EF9EpbZ5">[<a href="#ref-1CqoZGSKK" role="doc-biblioref">64</a>,<a href="#ref-1EF9EpbZ5" role="doc-biblioref">65</a>]</span>, potentially including stroke, seizures or meningitis <span class="citation" data-cites="nBGGUV9s LLm9d62d">[<a href="#ref-nBGGUV9s" role="doc-biblioref">66</a>,<a href="#ref-LLm9d62d" role="doc-biblioref">67</a>]</span>.</p>
<p>COVID-19 has also been associated with an increased incidence of large vessel stroke, particularly in patients under the age of 40 <span class="citation" data-cites="aRNqjkMg">[<a href="#ref-aRNqjkMg" role="doc-biblioref">68</a>]</span>, and other thrombotic events including pulmonary embolism and deep vein thrombosis <span class="citation" data-cites="3UBbDrG2">[<a href="#ref-3UBbDrG2" role="doc-biblioref">69</a>]</span>.
The mechanism behind these complications has been suggested to be related to coagulopathy with reports of antiphospholipid antibodies present <span class="citation" data-cites="8OnbWuhF">[<a href="#ref-8OnbWuhF" role="doc-biblioref">70</a>]</span> and elevated levels of d-dimer and fibrinogen degradation products (FDP) elevated in deceased patients <span class="citation" data-cites="dUfws1q0">[<a href="#ref-dUfws1q0" role="doc-biblioref">71</a>]</span>.
Other viral infections have been associated with coagulation defects; notably SARS was also found to lead to disseminated intravascular coagulation (DIC) and was associated with both pulmonary embolism and deep vein thrombosis <span class="citation" data-cites="p5EhN9Qd">[<a href="#ref-p5EhN9Qd" role="doc-biblioref">72</a>]</span>.
A wide range of viral infections can affect the coagulation cascade.
The mechanism behind these insults has been suggested to be related to inflammation-induced increases in the von Willebrand factor (VWF) clotting protein, leading to a pro-coagulative state <span class="citation" data-cites="p5EhN9Qd">[<a href="#ref-p5EhN9Qd" role="doc-biblioref">72</a>]</span>.
Abnormal clotting (thromboinflammation or coagulopathy) has been increasingly discussed recently as a possible key mechanism in many cases of severe COVID-19, and may be associated with the high d-dimer levels often observed in severe cases <span class="citation" data-cites="v1EIzwfx 18AiyvhO8 PyKMLraw">[<a href="#ref-v1EIzwfx" role="doc-biblioref">73</a>,<a href="#ref-18AiyvhO8" role="doc-biblioref">74</a>,<a href="#ref-PyKMLraw" role="doc-biblioref">75</a>]</span>.
This excessive clotting in lung capillaries has been suggested to be related to a dysregulated activation of the complement system, part of the innate immune system <span class="citation" data-cites="14dyYQY7s 17y6YeJ6R">[<a href="#ref-14dyYQY7s" role="doc-biblioref">76</a>,<a href="#ref-17y6YeJ6R" role="doc-biblioref">77</a>]</span>.</p>
<p>The presentation of COVID-19 infection can vary greatly among pediatric patients and in some cases manifests in distinct ways from COVID-19 in adults.
<span class="citation" data-cites="1BhN2uq3f">[<a href="#ref-1BhN2uq3f" role="doc-biblioref">78</a>]</span> reviewed 17 studies on children infected with COVID-19 during the early months of the COVID-19 epidemic in China and one study from Singapore.
Of the more than a thousand cases described, the most commonly reports were for mild symptoms such as fever, dry cough, fatigue, nasal congestion and/or runny nose, while three children were reported to be asymptomatic.
Severe lower respiratory infection was described in only one of the pediatric cases reviewed.
Gastrointestinal symptoms such as vomiting or diarrhea were occasionally reported.
Radiologic findings were not always reported in the case studies reviewed by <span class="citation" data-cites="1BhN2uq3f">[<a href="#ref-1BhN2uq3f" role="doc-biblioref">78</a>]</span>, but when they were mentioned they included bronchial thickening, ground-glass opacities, and/or inflammatory lesions.
Neurological symptoms have also been reported <span class="citation" data-cites="12lVfu2Qe">[<a href="#ref-12lVfu2Qe" role="doc-biblioref">79</a>]</span>.</p>
<p>These analyses indicate that generally, most pediatric cases of COVID-19 are not severe.<br />
However, serious complications and, in rare cases, death have occurred <span class="citation" data-cites="MCVe25tf">[<a href="#ref-MCVe25tf" role="doc-biblioref">80</a>]</span>.
Of particular interest, children have occasionally experienced a serious inflammatory syndrome, multisystem inflammatory syndrome in children (MIS-C), after COVID-19 infection; this syndrome is similar in some respects to Kawasaki disease or to Kawasaki disease shock syndrome <span class="citation" data-cites="fZwrv8M AmvwCWm3 68JorBeU">[<a href="#ref-fZwrv8M" role="doc-biblioref">81</a>,<a href="#ref-AmvwCWm3" role="doc-biblioref">82</a>,<a href="#ref-68JorBeU" role="doc-biblioref">83</a>]</span>.
MIS-C has been associated with heart failure in some cases <span class="citation" data-cites="FYgN6gUL">[<a href="#ref-FYgN6gUL" role="doc-biblioref">84</a>]</span>.
<span class="citation" data-cites="71To4laE">[<a href="#ref-71To4laE" role="doc-biblioref">85</a>]</span> reported a single case of an adult who appeared to show symptoms similar to MIS-C after exposure to COVID-19, but cautioned against broad conclusions; <span class="citation" data-cites="UBDAxWWv">[<a href="#ref-UBDAxWWv" role="doc-biblioref">86</a>]</span> reported another such possible adult case.</p>
<h3 data-number="1.3.3" id="subpopulations-of-special-concern"><span class="header-section-number">1.3.3</span> Subpopulations of Special Concern</h3>
<p>In the context of the United States, persons diagnosed with COVID-19 are more likely to require hospitalization if they are of male sex, of older age, and/or of Black/African American background <span class="citation" data-cites="1D6lVXWbJ Zk5CfYeW">[<a href="#ref-1D6lVXWbJ" role="doc-biblioref">87</a>,<a href="#ref-Zk5CfYeW" role="doc-biblioref">88</a>]</span>.
African Americans have also been reported to have disproportionate risk of kidney complications from COVID-19 <span class="citation" data-cites="17hdoiBo9">[<a href="#ref-17hdoiBo9" role="doc-biblioref">63</a>]</span>.
In addition to African Americans, disproportionate harm and mortality from COVID-19 has also been noted in Latino/Hispanic communities and in Native American / Alaskan Native communities such as the Navajo nation <span class="citation" data-cites="yK5bczRb u4MMV8aJ JAo2QxY 12WZObWRf Pp8GWnQm">[<a href="#ref-yK5bczRb" role="doc-biblioref">89</a>,<a href="#ref-u4MMV8aJ" role="doc-biblioref">90</a>,<a href="#ref-JAo2QxY" role="doc-biblioref">91</a>,<a href="#ref-12WZObWRf" role="doc-biblioref">92</a>,<a href="#ref-Pp8GWnQm" role="doc-biblioref">93</a>]</span>.
In Brazil, where the pandemic has also been severe, indigenous communities are likewise at special risk <span class="citation" data-cites="ODm1CaEq">[<a href="#ref-ODm1CaEq" role="doc-biblioref">94</a>]</span>.
The sizable racial disparities observed may be due to a number of factors, including different levels of pre-existing co-morbidities, such as hypertension, diabetes or lung diseases.
They may also be influenced by the disproportionate socioeconomic burdens placed on many people of color, which can correspond to greater economic difficulties, more hazardous or crowded work or living conditions, or reduced access to health care <span class="citation" data-cites="1D6lVXWbJ 17hdoiBo9 fxDYekIg">[<a href="#ref-17hdoiBo9" role="doc-biblioref">63</a>,<a href="#ref-1D6lVXWbJ" role="doc-biblioref">87</a>,<a href="#ref-fxDYekIg" role="doc-biblioref">95</a>]</span>.
This might cause infections to be less likely to be diagnosed unless or until they were very severe; in the sample studied by <span class="citation" data-cites="1D6lVXWbJ">[<a href="#ref-1D6lVXWbJ" role="doc-biblioref">87</a>]</span>, African Americans were more likely to be diagnosed in hospital, while other groups were more likely to have been diagnosed in ambulatory settings in the community.
More research is needed for analyzing and remediating these disparities <span class="citation" data-cites="Pp8GWnQm">[<a href="#ref-Pp8GWnQm" role="doc-biblioref">93</a>]</span>.</p>
<p>People living in certain locations may be especially vulnerable to harm.
In a preprint, <span class="citation" data-cites="5BMDcXz3">[<a href="#ref-5BMDcXz3" role="doc-biblioref">96</a>]</span> provided observational evidence that geographical areas in the United States that suffer from worse air pollution by fine particulate matter have also suffered more COVID-19 deaths per capita, after adjusting for demographic covariates.
Although lack of individual-level exposure data and the impossibility of randomization make it difficult to elucidate the exact causal mechanism, this finding would be consistent with similar findings for all-cause mortality (e.g., <span class="citation" data-cites="vImXUPNS">[<a href="#ref-vImXUPNS" role="doc-biblioref">97</a>]</span>).
Individuals in nursing homes / skilled nursing facilities <span class="citation" data-cites="aM2XPE3J">[<a href="#ref-aM2XPE3J" role="doc-biblioref">98</a>]</span> and in some prisons and detention centers <span class="citation" data-cites="BeedQx3I 2uUcva6Y">[<a href="#ref-BeedQx3I" role="doc-biblioref">99</a>,<a href="#ref-2uUcva6Y" role="doc-biblioref">100</a>]</span> have also been exposed to higher risk of infection.
Certain occupations, such as health care work, can put individuals at increased risk <span class="citation" data-cites="HihYbn6k F4laSOXM hAYohR8m DOyiuaup 13qkVV6eN">[<a href="#ref-HihYbn6k" role="doc-biblioref">101</a>,<a href="#ref-F4laSOXM" role="doc-biblioref">102</a>,<a href="#ref-hAYohR8m" role="doc-biblioref">103</a>,<a href="#ref-DOyiuaup" role="doc-biblioref">104</a>,<a href="#ref-13qkVV6eN" role="doc-biblioref">105</a>]</span>.</p>
<p>Many other factors influence risk of serious COVID-19 outcomes.
Genetic factors may play a role in risk of respiratory failure for COVID-19 <span class="citation" data-cites="owevcpwb nn67UTjV I2mIU0wn">[<a href="#ref-owevcpwb" role="doc-biblioref">106</a>,<a href="#ref-nn67UTjV" role="doc-biblioref">107</a>,<a href="#ref-I2mIU0wn" role="doc-biblioref">108</a>]</span>.
Diabetes may increase the risk of lengthy hospitalization <span class="citation" data-cites="9NfXSHjH">[<a href="#ref-9NfXSHjH" role="doc-biblioref">109</a>]</span> or of death <span class="citation" data-cites="9NfXSHjH 13BpwZaaq">[<a href="#ref-9NfXSHjH" role="doc-biblioref">109</a>,<a href="#ref-13BpwZaaq" role="doc-biblioref">110</a>]</span>.
<span class="citation" data-cites="UkCdtLwF">[<a href="#ref-UkCdtLwF" role="doc-biblioref">111</a>]</span> and <span class="citation" data-cites="OT5GEOej">[<a href="#ref-OT5GEOej" role="doc-biblioref">112</a>]</span> discuss possible ways in which COVID-19 and diabetes may interact.
Obesity also appears to be associated with higher risk of severe outcomes from SARS-CoV-2 <span class="citation" data-cites="ZLmNMDrD 2Ahpn6R2">[<a href="#ref-ZLmNMDrD" role="doc-biblioref">113</a>,<a href="#ref-2Ahpn6R2" role="doc-biblioref">114</a>]</span>.
Obesity is considered an underlying risk factor for other health problems, and the mechanism for its contributions to COVID-19 hospitalization or mortality is not yet clear <span class="citation" data-cites="1AmBg80Un">[<a href="#ref-1AmBg80Un" role="doc-biblioref">115</a>]</span>.</p>
<p>Study of disparities and differences among groups in rates of hospitalization or death from COVID-19 are complicated by the fact that risk may be elevated in multiple ways.
An individual may be more likely to be exposed to the virus, more likely to get infected once exposed, more likely to have serious complications once infected, be less likely to get adequate help once they are seriously ill.
Although these are difficult to disaggregate, they suggest different possible interventions and each deserves consideration.
The epidemiological characteristics and clinical presentation of COVID-19 may differ somewhat in different areas of the world, presumably due to differential reporting, different age structures, and/or different risk factors <span class="citation" data-cites="13BpwZaaq">[<a href="#ref-13BpwZaaq" role="doc-biblioref">110</a>]</span>.
Furthermore, because different subpopulations may have somewhat different vulnerabilities, needs, and resources, we recommend that researchers publishing studies on diagnostics and therapeutics take extra care to be clear about the demographic and medical characteristics of their sample, in order to facilitate discussions of the degree to which results may generalize or differ in other settings.</p>
<p>Several studies on disparities in COVID-19 have compared groups in terms of their probability of a severe outcome (like hospitalization or death) given that a person is diagnosed.<br />
The larger population of all exposed or all infected people usually cannot directly be studied, or even identified, because of the existence of undetected infections.<br />
In contrast, <span class="citation" data-cites="17gnxmOnf">[<a href="#ref-17gnxmOnf" role="doc-biblioref">116</a>]</span> were able to compare overall rates of death from COVID-19, not simply in the subsample of diagnosed individuals, thus combining the probability of infection with the probability of death given infection.<br />
This was done using a dataset of 17 million adults in the United Kingdom.<br />
In this administrative dataset, <span class="citation" data-cites="17gnxmOnf">[<a href="#ref-17gnxmOnf" role="doc-biblioref">116</a>]</span> found that male sex, older age, economic deprivation, nonwhite ethnicity (principally Black or South Asian), obesity, diabetes, asthma, and several other comorbid conditions were associated with a higher risk of death from COVID-19.
They cautioned that despite the large sample size, their observational data did not directly provide information on the causal mechanisms which led to these statistical associations.<br />
However, the ethnic and economic disparities did not vanish when adjusting for observed preexisting conditions.<br />
<span class="citation" data-cites="17gnxmOnf">[<a href="#ref-17gnxmOnf" role="doc-biblioref">116</a>]</span> also found that the predictive value of socioeconomic status seemed to increase over time, perhaps because many higher-paid workers were able to transition to working online.</p>
<p>Besides the direct harms caused by infection, many populations are in indirect risk of serious harm due to the social and economic effects of the pandemic and of the efforts needed to fight it.<br />
These might include individuals with substance use disorder <span class="citation" data-cites="xYVGtq0L">[<a href="#ref-xYVGtq0L" role="doc-biblioref">117</a>]</span>, victims of natural disasters <span class="citation" data-cites="qyQhe3zX">[<a href="#ref-qyQhe3zX" role="doc-biblioref">118</a>]</span>, and victims of human trafficking <span class="citation" data-cites="o9LSGYWO">[<a href="#ref-o9LSGYWO" role="doc-biblioref">119</a>]</span>.
The pandemic could also delay the fight against other major infectious diseases, such as HIV, malaria and tuberculosis, potentially leading to a further increase of mortality <span class="citation" data-cites="7TVDQB9o">[<a href="#ref-7TVDQB9o" role="doc-biblioref">120</a>]</span>.
Although they are beyond the scope of this paper, further research is needed in order to prevent these harms.</p>
<h3 data-number="1.3.4" id="molecular-mechanisms-of-covid-19"><span class="header-section-number">1.3.4</span> Molecular Mechanisms of COVID-19</h3>
<!-- How can we evaluate the human response to SARS-CoV-2 on a molecular level?-->
<!-- What can omics technologies tell us about changes in the body following SARS-CoV-2 infection?-->
<p>Most of our current knowledge of the molecular mechanisms through which SARS-CoV-2 infects cells is obtained from studies on previously identified coronaviruses.
The entry of a coronavirus into a target cell is mediated by the binding of the viral spike (S) protein to a specific cellular receptor and the subsequent priming/cleavage of the S protein by cellular proteases <span class="citation" data-cites="kCVy8vjj YeKaMzdJ">[<a href="#ref-kCVy8vjj" role="doc-biblioref">39</a>,<a href="#ref-YeKaMzdJ" role="doc-biblioref">40</a>]</span>.
Angiotensin-Converting Enzyme 2 (ACE2) and Transmembrane Serine Protease 2 (TMPRSS2) have been identified as a prime receptor and a critical protease, respectively, facilitating SARS-CoV/CoV-2 entry into a target cell <span class="citation" data-cites="2QTH37Xi CqQVkaqj MXWaV7SO 15l3di3Wj JjrO6YbD">[<a href="#ref-2QTH37Xi" role="doc-biblioref">41</a>,<a href="#ref-CqQVkaqj" role="doc-biblioref">42</a>,<a href="#ref-MXWaV7SO" role="doc-biblioref">43</a>,<a href="#ref-15l3di3Wj" role="doc-biblioref">44</a>,<a href="#ref-JjrO6YbD" role="doc-biblioref">45</a>]</span>.
This finding has led to a hypothesized role for ACE2 and TMPRSS2 expression in determining which cells, tissues and organs are most likely to be infected by SARS-CoV-2.</p>
<p>Recent clinical investigations of COVID-19 patients have detected SARS-CoV-2 transcripts in bronchoalveolar lavage fluid (93% of specimens), sputum (72%), nasal swabs (63%), fibrobronchoscopy brush biopsies (46%), pharyngeal swabs (32%), feces (29%) and blood (1%) <span class="citation" data-cites="vr7AH83b">[<a href="#ref-vr7AH83b" role="doc-biblioref">121</a>]</span>.
Two studies reported that SARS-CoV-2 could not be detected in the urine specimens <span class="citation" data-cites="vr7AH83b azgJqujy">[<a href="#ref-vr7AH83b" role="doc-biblioref">121</a>,<a href="#ref-azgJqujy" role="doc-biblioref">122</a>]</span>; however, a third study identified four urine samples (out of 58) that were positive for SARS-CoV-2 nucleic acids <span class="citation" data-cites="F7nSMvZk">[<a href="#ref-F7nSMvZk" role="doc-biblioref">123</a>]</span>.
Respiratory failure remains the leading cause of death for COVID-19 patients <span class="citation" data-cites="1AJFJxzmJ">[<a href="#ref-1AJFJxzmJ" role="doc-biblioref">124</a>]</span>.
Besides major pulmonary damage, SARS-CoV-2 infection can damage many other organ systems including the heart <span class="citation" data-cites="5gnCuPzp">[<a href="#ref-5gnCuPzp" role="doc-biblioref">125</a>]</span>, kidney <span class="citation" data-cites="5ET1D3cK w5B6qRKv">[<a href="#ref-5ET1D3cK" role="doc-biblioref">126</a>,<a href="#ref-w5B6qRKv" role="doc-biblioref">127</a>]</span>, liver <span class="citation" data-cites="TTSFlLVC">[<a href="#ref-TTSFlLVC" role="doc-biblioref">128</a>]</span>, and gastrointestinal tract <span class="citation" data-cites="PmE9xedP 3Ak4Mata">[<a href="#ref-PmE9xedP" role="doc-biblioref">129</a>,<a href="#ref-3Ak4Mata" role="doc-biblioref">130</a>]</span>.
As it becomes clearer that SARS-CoV-2 infection can damage multiple organs, the scientific community is pursuing multiple avenues of investigation in order to build a consensus about how the virus affects the human body.
To quickly associate the clinical outcomes with SARS-CoV-2 infection, researchers are taking advantage of “omics” technologies to profile the expression of coronavirus entry factors across the tissues.</p>
<h2 data-number="1.4" id="approaches-to-understanding-covid-19"><span class="header-section-number">1.4</span> Approaches to Understanding COVID-19</h2>
<p>Scientific characterization of the SARS-CoV-2 virus and of the COVID-19 disease it causes is critical to controlling the current pandemic.
Several broad areas of research interact with each other, offering different pieces of information critical to understanding the virus and disease.
A comprehensive understanding of the epidemic must unify basic scientific and medical research with public health and biotechnology.</p>
<h3 data-number="1.4.1" id="science-medicine"><span class="header-section-number">1.4.1</span> Science &amp; Medicine</h3>
<p>Understanding how the virus functions and interacts with the host is foundational to understanding pathogenesis and disease progression and to identifying available and novel approaches to treatment.
Therefore, the fields of virology, immunology, and molecular biology are fundamental to characterizing SARS-CoV-2 and COVID-19.
These topics can be approached using a range of techniques, including characterization of the host response from the cellular to systems level.
Contextualizing SARS-CoV-2 in relation to other viruses that infect humans and other animals can further serve to elucidate the reaction of the human host to viral exposure.
This information, when combined with an understanding of the biology of pharmaceutical and medical interventions, can guide new approaches to treatment.</p>
<h3 data-number="1.4.2" id="public-health"><span class="header-section-number">1.4.2</span> Public Health</h3>
<p>One necessary component of determining how to manage the outbreak is to understand epidemiological factors related to the transmission of the SARS-CoV-2 virus.
These can include characteristics such as when infected individuals are contagious, how the virus is transmitted between individuals, the range of symptoms associated with infection and/or contagiousness in different individuals, and how rapidly the virus propagates between individuals, etc.
The development of diagnostic tools is critical to this goal.
Accurate diagnoses on a large scale is necessary to collect the data needed to develop epidemiological models.
Other areas of public health that address resource availability, inequity, human behavior, and other components that influence people’s exposure to pathogens and ability to manage illness will also be critical to mounting a global response to the pandemic.
Public health strategies to manage epidemics include anticipation and early detection of emerging diseases (aided by rapid development of diagnostics), containment (using strategies to test, trace and isolate such as widespread testing, contact tracing, quarantining and isolation), control and mitigation of spread (including social distancing), and elimination and eradication.
An effective public health response requires response coordination, disease surveillance and intervention monitoring, risk communication and health education (including education about personal protective equipment and hand hygiene and containment of “infodemics” of false information), non-pharmaceutical measures to prevent and reduce transmission, and health interventions such as vaccines and other pharmaceuticals that can reduce transmission, curb morbidity and mortality, and mitigate the effects on health systems and other sectors of society <span class="citation" data-cites="13CJl7iIO">[<a href="#ref-13CJl7iIO" role="doc-biblioref">131</a>]</span>.
<!--I think non-pharmaceutical interventions are already in categories above (tracing, social distancing, lockdowns etc)-->
Currently, this manuscript focuses primarily on contextualizing epidemiological characteristics such as reproduction number and dynamics of transmission that intersect with the fundamental biology of the virus or the development of therapeutic and diagnostic technologies.</p>
<h3 data-number="1.4.3" id="biotechnology"><span class="header-section-number">1.4.3</span> Biotechnology</h3>
<h4 data-number="1.4.3.1" id="diagnostics"><span class="header-section-number">1.4.3.1</span> Diagnostics</h4>
<p>Two major concerns within diagnosis include the detection of current infections in individuals with and without symptoms, and the detection of past exposure without an active infection.
In the latter category, identifying whether individuals can develop or have developed sustained immunity is also a major consideration.
The development of high-throughput, affordable methods for detecting active infections and sustained immunity will be critical to understanding and controlling the disease.</p>
<!-- What are approaches that allow us to detect current infection or past exposure for other viruses?-->
<!-- What is sustained immunity and what are the indicators?-->
<h4 data-number="1.4.3.2" id="therapeutics"><span class="header-section-number">1.4.3.2</span> Therapeutics</h4>
<p>The identification of interventions that can mitigate the effect of the virus on exposed and infected individuals is a significant research priority.
Some possible approaches include the identification of existing pharmaceuticals that reduce the severity of infection, either by reducing the virus’ virulence (e.g., antivirals) or managing the most severe symptoms of infection.
Due to the long timeline for the development of novel pharmaceuticals, in most cases, research surrounding possible pharmaceutical interventions focuses on the identification and investigation of existing compounds whose mechanisms may be relevant to COVID-19.
Other foci of current research include the identification of antibodies produced by survivors of COVID-19 and the development of vaccines.
Understanding the mechanisms describing host-virus interactions between humans and SARS-CoV-2 is thus critical to identifying candidate therapeutics.</p>
<h2 data-number="1.5" id="summary"><span class="header-section-number">1.5</span> Summary</h2>
<p>In this review, we seek to consolidate information about efforts to develop strategies for diagnosis and therapeutics as new information is released by the scientific community.
We include information from both traditional peer-reviewed scientific literature and from preprints, which typically have not undergone peer review but have been critically evaluated by the scientists involved in this effort.
The goal of this manuscript is to present preliminary findings within the broader context of COVID-19 research and to identify the broad interpretations of new research, as well as limitations to interpretability.</p>
<h1 data-number="2" id="pathogenesis"><span class="header-section-number">2</span> Pathogenesis</h1>
<h2 data-number="2.1" id="mechanism-of-host-infection-by-sars-cov-2"><span class="header-section-number">2.1</span> Mechanism of Host Infection by SARS-CoV-2</h2>
<p>This section would also be great for the introduction of zoonotic diseases which has been shown to be the origin of SARS-CoV2.</p>
<h3 data-number="2.1.1" id="primary-transmission-and-viral-entry"><span class="header-section-number">2.1.1</span> Primary Transmission and Viral Entry</h3>
<p>Like that of SARS-CoV, SARS-CoV-2 entry into host cells is mediated by the interaction between the viral spike glycoprotein (S) and human angiotensin-converting enzyme 2 (ACE2) in humans and other animals <span class="citation" data-cites="15EIBRhef qcVbT0w4 12TZ7hPMA wCbhn23d VX8OWaGj 15l3di3Wj 15Q2XgkK7 t1e4CW9A">[<a href="#ref-qcVbT0w4" role="doc-biblioref">38</a>,<a href="#ref-15l3di3Wj" role="doc-biblioref">44</a>,<a href="#ref-15EIBRhef" role="doc-biblioref">132</a>,<a href="#ref-12TZ7hPMA" role="doc-biblioref">133</a>,<a href="#ref-wCbhn23d" role="doc-biblioref">134</a>,<a href="#ref-VX8OWaGj" role="doc-biblioref">135</a>,<a href="#ref-15Q2XgkK7" role="doc-biblioref">136</a>,<a href="#ref-t1e4CW9A" role="doc-biblioref">137</a>]</span>.
The ACE2 receptor is expressed in numerous organs, such as the heart, kidney, and intestine, but most prominently in alveolar epithelial cells, which is expected to contribute to the virus’ association with lung pathology <span class="citation" data-cites="UwVweB2M So6ceUwB">[<a href="#ref-UwVweB2M" role="doc-biblioref">138</a>,<a href="#ref-So6ceUwB" role="doc-biblioref">139</a>]</span>.
The S protein is a highly glycosylated trimer that requires two proteolytic cleavage events, leading to substantial conformational changes, to achieve viral fusion with the host cell membrane <span class="citation" data-cites="qcVbT0w4 CqQVkaqj">[<a href="#ref-qcVbT0w4" role="doc-biblioref">38</a>,<a href="#ref-CqQVkaqj" role="doc-biblioref">42</a>]</span>.
Each protomer is composed of an S1 and an S2 subunit, which mediate receptor binding and viral fusion, respectively.
The priming proteolytic events occur sequentially, first at the S1/S2 junction and then at the S2’ site, ultimately resulting in the shedding of the S1 subunit and transitioning of the S2 subunit to a more stable, fusion-conducive conformation.</p>
<p>Similar to SARS-CoV, SARS-CoV-2 exhibits redundancy in which host proteases it can use to cleave the S protein <span class="citation" data-cites="15l3di3Wj">[<a href="#ref-15l3di3Wj" role="doc-biblioref">44</a>]</span>.
Specifically, both transmembrane protease serine protease-2 (TMPRSS2) and cathepsins B/L have been shown to mediate SARS-CoV-2 S protein proteolytic priming, and small molecule inhibition of these enzymes fully inhibited viral entry <em>in vitro</em> <span class="citation" data-cites="15l3di3Wj JOJ2n3gC">[<a href="#ref-15l3di3Wj" role="doc-biblioref">44</a>,<a href="#ref-JOJ2n3gC" role="doc-biblioref">140</a>]</span>.
Interestingly, SARS-CoV-2 S protein also contains a RRAR furin recognition site at the S1/S2 junction <span class="citation" data-cites="qcVbT0w4 CqQVkaqj">[<a href="#ref-qcVbT0w4" role="doc-biblioref">38</a>,<a href="#ref-CqQVkaqj" role="doc-biblioref">42</a>]</span>, setting it apart from bat coronavirus RaTG13, with which it shares 96% genome sequence identity, and SARS-CoV <span class="citation" data-cites="VSkK7CeP">[<a href="#ref-VSkK7CeP" role="doc-biblioref">25</a>]</span>.
Such furin cleavage sites are commonly found in highly virulent influenza viruses, and as such may contribute to the heightened pathogenicity of SARS-CoV-2 <span class="citation" data-cites="NsORsLig vhHB3yyS">[<a href="#ref-NsORsLig" role="doc-biblioref">141</a>,<a href="#ref-vhHB3yyS" role="doc-biblioref">142</a>]</span>.
Differences in S protein sequence between SARS-CoV and SARS-CoV-2 may also partially account for the increased transmissibility seen in the current COVID-19 pandemic.
Recent studies have reported conflicting binding constants for the S protein-hACE2 interaction, though they agree that SARS-CoV-2 S protein binds with equal if not greater affinity than does SARS-CoV S protein <span class="citation" data-cites="qcVbT0w4 VX8OWaGj CqQVkaqj">[<a href="#ref-qcVbT0w4" role="doc-biblioref">38</a>,<a href="#ref-CqQVkaqj" role="doc-biblioref">42</a>,<a href="#ref-VX8OWaGj" role="doc-biblioref">135</a>]</span>.
The C-terminal domain of the SARS-CoV-2 S protein in particular was identified as the key region of the virus that interacts with the human ACE2 (hACE2) receptor, and the crystal structure of the C-terminal domain of SARS-CoV-2 S protein in complex with human ACE2 reveals stronger interaction and higher affinity for receptor binding than SARS-CoV [doi:10.1016/j.cell.2020.03.045].
Among the 14 key binding residues identified in the SARS-CoV S protein, 8 are conserved in SARS-CoV-2 and the remaining 6 are semi-conservatively substituted, potentially explaining variation in binding affinity <span class="citation" data-cites="qcVbT0w4 VX8OWaGj">[<a href="#ref-qcVbT0w4" role="doc-biblioref">38</a>,<a href="#ref-VX8OWaGj" role="doc-biblioref">135</a>]</span>.
Recent crystal structures have shown that the receptor-binding domain (RBD) of SARS-CoV-2 S protein, like that of other coronaviruses, undergoes stochastic hinge-like movement that flips it from a “closed” conformation, in which key binding residues are hidden at the interface between protomers, to an “open” one <span class="citation" data-cites="qcVbT0w4 CqQVkaqj">[<a href="#ref-qcVbT0w4" role="doc-biblioref">38</a>,<a href="#ref-CqQVkaqj" role="doc-biblioref">42</a>]</span>.
Because the RBD plays such a critical role in viral entry, blocking its interaction with ACE2 represents a promising therapeutic approach. Nevertheless, despite the high structural homology between SARS-CoV-2 RBD and that from SARS-CoV, monoclonal antibodies targeting SARS-CoV-RBD failed to bind SARS-CoV-2-RBD <span class="citation" data-cites="CqQVkaqj">[<a href="#ref-CqQVkaqj" role="doc-biblioref">42</a>]</span>.
Promisingly though, sera from convalescent SARS patients inhibited SARS-CoV-2 viral entry in vitro, albeit with lower efficiency than it inhibited SARS-CoV <span class="citation" data-cites="15l3di3Wj">[<a href="#ref-15l3di3Wj" role="doc-biblioref">44</a>]</span>.</p>
<h3 data-number="2.1.2" id="viral-replication-and-spreading"><span class="header-section-number">2.1.2</span> Viral Replication and Spreading</h3>
<ul>
<li>Basic introduction into replication cycle</li>
<li>What are the routes of transmission</li>
</ul>
<h4 data-number="2.1.2.1" id="mechanism-of-viral-replication-within-cells"><span class="header-section-number">2.1.2.1</span> Mechanism of viral replication within cells</h4>
<h4 data-number="2.1.2.2" id="mechanism-of-viral-spreading-to-neighbor-cells"><span class="header-section-number">2.1.2.2</span> Mechanism of viral spreading to neighbor cells</h4>
<h4 data-number="2.1.2.3" id="factors-enhancing-viral-spreading"><span class="header-section-number">2.1.2.3</span> Factors enhancing viral spreading</h4>
<p>Viral progression may be enhanced by active upregulation of ACE2 on cell surfaces following or during a response to infection.
In several preliminary assays and an analysis of previous microarray data, Wang et al. reported that ACE2 is significantly upregulated following infection by other coronaviruses, including SARS-CoV and MERS-CoV, as well as viruses such as rhinovirus and influenza virus <span class="citation" data-cites="UwVweB2M">[<a href="#ref-UwVweB2M" role="doc-biblioref">138</a>]</span>.
Additionally, direct stimulation with inflammatory cytokines such as type I interferons resulted in the upregulation of ACE2, with treated groups showing 4-fold higher ACE2 expression than control groups at 18 hours post-treatment <span class="citation" data-cites="UwVweB2M">[<a href="#ref-UwVweB2M" role="doc-biblioref">138</a>]</span>.
Though whether SARS-CoV-2 infection facilitates positive regulation of its own transmission between host cells is still unclear, the host immune response itself likely plays a key role in mediating infection-associated pathologies.
One severe example includes reports of cytokine storm-like responses in patients with particularly severe infections, in which the overproduction of inflammatory cytokines leads to systemic inflammation and potentially multi-organ failure, which may very well accelerate the spread of virus in the host <span class="citation" data-cites="UwVweB2M 3HdlV9Vf">[<a href="#ref-UwVweB2M" role="doc-biblioref">138</a>,<a href="#ref-3HdlV9Vf" role="doc-biblioref">143</a>]</span>.
<span class="citation" data-cites="cyUG1Zi2">[<a href="#ref-cyUG1Zi2" role="doc-biblioref">144</a>]</span> describe the concept of the cytokine storm in detail, although <span class="citation" data-cites="Xcrxlxnp">[<a href="#ref-Xcrxlxnp" role="doc-biblioref">145</a>]</span> claim that the widespread use of the term can be misleading.</p>
<h4 data-number="2.1.2.4" id="person-to-person-transmission"><span class="header-section-number">2.1.2.4</span> Person-to-person transmission</h4>
<h3 data-number="2.1.3" id="reproduction-number-and-dynamics-of-transmission"><span class="header-section-number">2.1.3</span> Reproduction Number and Dynamics of Transmission</h3>
<p>Accurate estimates of the reproduction number of a virus are crucial to understanding the dynamics of infection and to predict the effects of different interventions.
The basic reproduction number and the timescale of infection (measured by the infectious period and the exposed period) govern population-level epidemic dynamics, with R<sub>0</sub> being one of most critical epidemiological parameters <span class="citation" data-cites="14l4fvU71">[<a href="#ref-14l4fvU71" role="doc-biblioref">146</a>]</span>.
The basic reproduction number, R<sub>0</sub>, is the expected number of new (secondary) infections caused by one infected person, assuming no time dependence and a wholly susceptible population <span class="citation" data-cites="16CD4voW1">[<a href="#ref-16CD4voW1" role="doc-biblioref">147</a>]</span>.
R<sub>0</sub> is a unitless number which in mechanistic models is a combination of parameters principally related to the transmission rate (rate of infection transmitting interactions for an infectious individual) and the infectious period <span class="citation" data-cites="YubbhU7G 14l4fvU71">[<a href="#ref-14l4fvU71" role="doc-biblioref">146</a>,<a href="#ref-YubbhU7G" role="doc-biblioref">148</a>]</span>.
A pathogen can invade a susceptible population only if R<sub>0</sub> &gt; 1 <span class="citation" data-cites="14l4fvU71 JRaVgcNd">[<a href="#ref-14l4fvU71" role="doc-biblioref">146</a>,<a href="#ref-JRaVgcNd" role="doc-biblioref">149</a>]</span>.
The effective reproduction number, R<sub>t</sub>, describes how the reproduction number may change over time, and is used to quantify deviations in R from R<sub>0</sub> that would occur, for example, as some fraction of the population became infected or as interventions were put into place.
When R<sub>t</sub> is greater than 1, an epidemic grows (i.e., the proportion of the population that is infectious increases); when R<sub>t</sub> is less than 1, the proportion of the population that is infectious decreases.
R<sub>0</sub> and R<sub>t</sub> can be estimated directly from epidemiological data or inferred using mathematical modeling.
Modeling approaches are typically based upon a classic epidemiological model structure: the susceptible-infected-recovered (SIR) model and its extensions <span class="citation" data-cites="1Aozf7KBW SPInPew0">[<a href="#ref-1Aozf7KBW" role="doc-biblioref">150</a>,<a href="#ref-SPInPew0" role="doc-biblioref">151</a>]</span>.</p>
<p>Estimates of R<sub>0</sub> for COVID-19 lie in the range R<sub>0</sub>=1.4-6.5 <span class="citation" data-cites="1E0r4uZy9 LHtuVmaq mHYmt0mv">[<a href="#ref-1E0r4uZy9" role="doc-biblioref">152</a>,<a href="#ref-LHtuVmaq" role="doc-biblioref">153</a>,<a href="#ref-mHYmt0mv" role="doc-biblioref">154</a>]</span>.
Variation in R<sub>0</sub> is expected between different populations and the estimated values of R<sub>0</sub> discussed below are for specific populations in specific environments. The different estimates of R<sub>0</sub> should not be interpreted as a range of estimates of the same underlying parameter.
Data-derived estimates (i.e. those that do not incorporate SIR-type models into their analysis) typically predict lower values of R<sub>0</sub>.
For data-derived estimates, in one study of international cases, the predicted value is R<sub>0</sub>=1.7 <span class="citation" data-cites="vITui6ac">[<a href="#ref-vITui6ac" role="doc-biblioref">155</a>]</span>, in China (both Hubei province and nationwide), the value is predicted to lie in the range R<sub>0</sub>=2.0-3.6 <span class="citation" data-cites="nxM0rP5R WLc2UMgQ 1E0r4uZy9">[<a href="#ref-1E0r4uZy9" role="doc-biblioref">152</a>,<a href="#ref-nxM0rP5R" role="doc-biblioref">156</a>,<a href="#ref-WLc2UMgQ" role="doc-biblioref">157</a>]</span>, and on a cruise ship where an outbreak occurred, predicted R<sub>0</sub>=2.28 <span class="citation" data-cites="10bBqMHH7">[<a href="#ref-10bBqMHH7" role="doc-biblioref">158</a>]</span>.
SIR model-derived estimates of R<sub>0</sub> range from 2.0 - 6.5 in China <span class="citation" data-cites="ITh0Anof 1BMU7sKbs Yj8Xh4Wz ZzrrVDoE">[<a href="#ref-ITh0Anof" role="doc-biblioref">159</a>,<a href="#ref-1BMU7sKbs" role="doc-biblioref">160</a>,<a href="#ref-Yj8Xh4Wz" role="doc-biblioref">161</a>,<a href="#ref-ZzrrVDoE" role="doc-biblioref">162</a>]</span> to R<sub>0</sub>=4.8 in France <span class="citation" data-cites="itj26agd">[<a href="#ref-itj26agd" role="doc-biblioref">163</a>]</span>.
Using the same model as for the French population, this study estimated R<sub>0</sub>=2.6 in South Korea <span class="citation" data-cites="itj26agd">[<a href="#ref-itj26agd" role="doc-biblioref">163</a>]</span>, which is consistent with other studies <span class="citation" data-cites="YfOcGRRa">[<a href="#ref-YfOcGRRa" role="doc-biblioref">164</a>]</span>.
From a meta-analysis of studies estimating R<sub>0</sub>, <span class="citation" data-cites="LHtuVmaq">[<a href="#ref-LHtuVmaq" role="doc-biblioref">153</a>]</span> predict the median as R<sub>0</sub>=2.79.</p>
<p>Inference of the effective reproduction number can provide insight into how populations respond to an infection, and the effectiveness of interventions.
In China, R<sub>t</sub> was predicted to lie in the range 1.6-2.6 in Jan 2020, before travel restrictions <span class="citation" data-cites="tiRfUgvs">[<a href="#ref-tiRfUgvs" role="doc-biblioref">165</a>]</span>.
R<sub>t</sub> decreased from 2.35 one week before travel restrictions were imposed (Jan 23, 2020), to 1.05 one week after.
Using their model, the authors also estimate the probability of new outbreaks occurring: the probability of a single individual exporting virus causing a large outbreak is 17-25% assuming MERS-like or SARS-like transmission, and the probability of a large outbreak occurring after ≥4 infections exist at a new location is greater than 50%.
An independent study came to similar conclusions: in a two-week period before Jan 23 finding R<sub>t</sub>=2.38, and decreasing to R<sub>t</sub> = 1.34 (using data from Jan 24 to Feb 3) or R<sub>t</sub>=0.98 (using data from Jan 24 to Feb 8) <span class="citation" data-cites="mHYmt0mv">[<a href="#ref-mHYmt0mv" role="doc-biblioref">154</a>]</span>.
In South Korea, R<sub>t</sub> was inferred for Feb-Mar 2020 in two cities: Daegu (the center of the outbreak), and Seoul <span class="citation" data-cites="YfOcGRRa">[<a href="#ref-YfOcGRRa" role="doc-biblioref">164</a>]</span>.
Metro data was also analyzed to estimate the effects of social distancing measures.
R<sub>t</sub> decreased in Daegu from around 3 to &lt;1 over the period that social distancing measures were introduced.
In Seoul, R<sub>t</sub> decreased slightly, but remained close to 1 (and larger than R<sub>t</sub> in Daegu).
This highlights that social distancing measures appeared to work to contain the infection in Daegu, but that in Seoul, R<sub>t</sub> remains above 1, thus secondary outbreaks are possible.
It also shows the importance of region-specific analysis: the large decline in case load nationwide is mainly due to the Daegu region, and could hide persistence of the epidemic in other regions, such as Seoul and Gyeonggi-do.
In Iran, estimates of R<sub>t</sub> declined from 4.86 in the first week to 2.1 by the fourth week after the first cases were reported <span class="citation" data-cites="TH4ymqvJ">[<a href="#ref-TH4ymqvJ" role="doc-biblioref">166</a>]</span>.
In Europe, analysis of 11 countries inferred the dynamics of R<sub>t</sub> over a time range from the beginning of the outbreak until March 28, 2020, by which point most countries had implemented major interventions (such as school closures, public gathering bans, and stay-at-home orders) <span class="citation" data-cites="Frya4XA4">[<a href="#ref-Frya4XA4" role="doc-biblioref">167</a>]</span>.
Across all countries, the mean R<sub>t</sub> before interventions began was estimated as 3.87; R<sub>t</sub> varied considerably, from below 3 in Norway to above 4.5 in Spain.
After interventions, R<sub>t</sub> decreased by an average of 64% across all countries, with mean R<sub>t</sub>=1.43.
The lowest predicted value was 0.97 for Norway and the highest was 2.64 for Sweden (note that this is in part because Sweden did not implement social distancing measures on the same scale as other countries).
The study concludes that while large changes in R<sub>t</sub> are observed, it is too early to tell whether the interventions put into place are sufficient to decrease R<sub>t</sub> below 1.</p>
<p>More generally, population-level epidemic dynamics can be both observed and modelled <span class="citation" data-cites="YubbhU7G">[<a href="#ref-YubbhU7G" role="doc-biblioref">148</a>]</span>.
Data and empirically determined biological mechanisms inform models, while models can be used to try to understand data and systems of interest or to make predictions about possible future dynamics, such as the estimation of capacity needs <span class="citation" data-cites="NCcssjLV">[<a href="#ref-NCcssjLV" role="doc-biblioref">168</a>]</span> or the comparison of predicted outcomes among prevention and control strategies <span class="citation" data-cites="11CWFYFDL J4RCpntH">[<a href="#ref-11CWFYFDL" role="doc-biblioref">169</a>,<a href="#ref-J4RCpntH" role="doc-biblioref">170</a>]</span>.
Many current efforts to model R<sub>t</sub> have led to tools that assist the visualization of estimates in real time (or over recent intervals) <span class="citation" data-cites="WA1R5R95 17CQnstjh">[<a href="#ref-WA1R5R95" role="doc-biblioref">171</a>,<a href="#ref-17CQnstjh" role="doc-biblioref">172</a>]</span>.
While these may be valuable resources, it is important to note that the estimates arise from models containing many assumptions and are dependent on the quality of the data they use, which varies widely by region.</p>
<h2 data-number="2.2" id="immune-response-to-sars-cov-2"><span class="header-section-number">2.2</span> Immune Response to SARS-CoV-2</h2>
<ul>
<li>Cellular responses to SARS-CoV-2 infection</li>
<li>What is causing neutropenia and lymphopenia observed in COVID-19 patients</li>
<li>Antibody production against SARS-CoV-2 by patient who recovered vs patient who did not recover</li>
<li>Cytokines and other soluble factors contribution to immune response</li>
</ul>
<h2 data-number="2.3" id="systems-level-approaches-for-understanding-sars-cov-2-pathogenesis"><span class="header-section-number">2.3</span> Systems-level approaches for understanding SARS-CoV-2 pathogenesis</h2>
<p>Systems biology provides a cross-disciplinary analytical platform integrating the different omics (genomics, transcriptomics, proteomics, metabolomics, and other omics approaches), bioinformatics, and computational strategies.
These cutting-edge research approaches have enormous potential to study the complexity of biological systems and human diseases <span class="citation" data-cites="5MhhHQO0">[<a href="#ref-5MhhHQO0" role="doc-biblioref">173</a>]</span>.
Over the last decade, systems biology approaches have been used widely to study the pathogenesis of diverse types of life-threatening acute and chronic infectious diseases <span class="citation" data-cites="8esIIWt9">[<a href="#ref-8esIIWt9" role="doc-biblioref">174</a>]</span>.
Omics-based studies also provided meaningful information regarding host immune responses and surrogate protein markers in several viral, bacterial and protozoan infections <span class="citation" data-cites="8vUHj9cm">[<a href="#ref-8vUHj9cm" role="doc-biblioref">175</a>]</span>.</p>
<p>The complex pathogenesis and clinical manifestations of SARS-CoV-2 infection are not understood adequately yet.
A significant breakthrough in SARS-CoV-2 research was achieved through the successful full-length genome sequencing of the pathogen <span class="citation" data-cites="vHq0hOWQ 1Fqilxaum VSkK7CeP">[<a href="#ref-1Fqilxaum" role="doc-biblioref">13</a>,<a href="#ref-VSkK7CeP" role="doc-biblioref">25</a>,<a href="#ref-vHq0hOWQ" role="doc-biblioref">176</a>]</span>.
Multiple research groups have drafted the genome sequence of SARS-CoV-2 based on sequencing of clinical samples collected from bronchoalveolar lavage fluid (BALF) <span class="citation" data-cites="vHq0hOWQ VSkK7CeP">[<a href="#ref-VSkK7CeP" role="doc-biblioref">25</a>,<a href="#ref-vHq0hOWQ" role="doc-biblioref">176</a>]</span> or from BALF, throat swabs, or isolates of the virus cultured from BALF <span class="citation" data-cites="1Fqilxaum">[<a href="#ref-1Fqilxaum" role="doc-biblioref">13</a>]</span>.
Importantly, SARS-CoV-2 has significant sequence homology with SARS-CoV (about 79%) and also to some extent with MERS-CoV (about 50%) <span class="citation" data-cites="1Fqilxaum">[<a href="#ref-1Fqilxaum" role="doc-biblioref">13</a>]</span>.
However, a higher level of similarity (about 90%) has been observed between SARS-CoV-2 and bat-derived SARS-like coronaviruses (bat-SL-CoVZC45 and bat-SL-CoVZXC21), indicating a possible origin in bats <span class="citation" data-cites="1Fqilxaum VSkK7CeP">[<a href="#ref-1Fqilxaum" role="doc-biblioref">13</a>,<a href="#ref-VSkK7CeP" role="doc-biblioref">25</a>]</span>.</p>
<p>The genome sequence of the pathogen subsequently allowed its phylogenetic characterization and prediction of its protein expression profile, which is crucial for understanding the pathogenesis and virulence of this novel viral infection.
Availability of the genome sequence of SARS-CoV-2 enhances the potential for subsequent proteome-level studies to provide further mechanistic insights into the virus’ complex pathogenesis.
Of note, the cryo-electron microscopy structure of the SARS-CoV-2 spike (S) glycoprotein, which plays an important role in the early steps of viral infection, was reported very recently <span class="citation" data-cites="CqQVkaqj">[<a href="#ref-CqQVkaqj" role="doc-biblioref">42</a>]</span>.
Even though no comprehensive proteomic analysis of the pathogen or of patients suffering from its infection has yet been reported, one forthcoming study has demonstrated SARS-CoV-2 infected host cell proteomics using human Caco–2 cells as an infection model <span class="citation" data-cites="11xZWeHN3">[<a href="#ref-11xZWeHN3" role="doc-biblioref">177</a>]</span>.
The authors observed SARS-CoV-2 induced alterations in multiple vital physiological pathways, including translation, splicing, carbon metabolism and nucleic acid metabolism in the host cells.</p>
<p>There is a high level of sequence homology between SARS-CoV-2 and SARS-CoV, and sera from convalescent SARS-CoV patients might show some efficacy to cross-neutralize SARS-CoV-2-S-driven entry <span class="citation" data-cites="15l3di3Wj">[<a href="#ref-15l3di3Wj" role="doc-biblioref">44</a>]</span>.
However, despite the high level of sequence homology, certain protein structures might be immunologically distinct, prohibiting effective cross-neutralization across different SARS strains <span class="citation" data-cites="GhJYjnft">[<a href="#ref-GhJYjnft" role="doc-biblioref">178</a>]</span>.
Consequently, earlier proteome-level studies on SARS-CoV can also provide some essential information regarding the new pathogen <span class="citation" data-cites="GpnngtWK JLTf2Fwb">[<a href="#ref-GpnngtWK" role="doc-biblioref">179</a>,<a href="#ref-JLTf2Fwb" role="doc-biblioref">180</a>]</span>.
Considering the paucity of omics-level big data sets for SARS-CoV-2 up until now, existing data hubs that contain information for other coronaviruses such as UniProt, NCBI Genome Database, The Immune Epitope Database and Analysis Resource (IEDB), and The Virus Pathogen Resource (ViPR) will serve as useful resources for computational and bioinformatics research on SARS-CoV-2.
Using such databases, the systems level reconstruction of the PPI (Protein-Protein Interaction) enabled the generation of hypotheses on the mechanism of action of SARS-CoV-2 and suggested drug targets.</p>
<h3 data-number="2.3.1" id="protein-protein-interaction-networks"><span class="header-section-number">2.3.1</span> Protein-protein interaction networks</h3>
<p>In an initial study <span class="citation" data-cites="phJM8g2Y">[<a href="#ref-phJM8g2Y" role="doc-biblioref">181</a>]</span>, 26 of the 29 SARS-CoV-2 proteins were cloned and expressed in HEK293T kidney cells, allowing for the identification of 332 high-confidence human proteins that interact with them.
Notably, this study suggested that SARS-CoV-2 interacts with innate immunity pathways.
The ranking of pathogens with respect to their interactome’s similarity to SARS-CoV-2 suggested <em>West Nile Virus</em>, <em>Mycobacterium tuberculosis</em>, and <em>Human papillomavirus</em> as the top three hits.
However, given the lung symptoms associated with COVID-19, the <em>Mycobacterium tuberculosis</em> host-pathogen interactome could also provide new insights to the mechanism of SARS-CoV-2 infection.
In addition, it was suggested that the envelope protein E could disrupt the host bromodomain-containing proteins, i.e., BRD2 and BRD4, binding to histone.
The Spike protein S could likely intervene in the virus fusion through modulating the GOLGA7-ZDHHC5 acyl-transferase complex to increase palmitoylation.</p>
<p>Another study <span class="citation" data-cites="NLoN4aYj">[<a href="#ref-NLoN4aYj" role="doc-biblioref">182</a>]</span>, used patient-derived peripheral blood mononuclear cells (PBMCs) to identify 251 host proteins targeted by SARS-CoV-2 and the disruption of more than 200 host proteins following the infection.
The network analysis showed in particular that non-structural proteins 9 and 10 (nsp9 and nsp10) interacted with NKRF that usually represses NFKB.
This finding could explain the exacerbation of the immune response that shape the pathology and the high cytokine levels possibly due to the chemotaxis of neutrophils mediated by IL-8 and IL-6.
Finally, it was suggested <span class="citation" data-cites="1ccnm0N9">[<a href="#ref-1ccnm0N9" role="doc-biblioref">183</a>]</span> that protein E of both SARS-CoV and SARS-CoV-2 has a conserved Bcl-2 Homology 3 (BH3)-like motif, that could inhibit anti-apoptosis proteins BCL2 and trigger the apoptosis of T cells.
Several known compounds were suggested to disrupt the host-pathogen protein interactome were suggested mostly through the inhibition of host proteins.</p>
<h3 data-number="2.3.2" id="transcriptomics-of-covid-19"><span class="header-section-number">2.3.2</span> Transcriptomics of COVID-19</h3>
<p>To quickly associate the clinical outcomes with SARS-CoV-2 infection, researchers are taking advantage of “omics” technologies to profile the expression of coronavirus entry factors across the tissues/cells and also to measure the host transcriptional response after virus infection.
Although we make an effort to evaluate all relevant studies, we realize that this area is rapidly evolving.</p>
<p>Two studies profiled expression after SARS-CoV-2 infection in multiple human cell lines (Figure <a href="#fig:cell-lines-moi">2</a>A) <span class="citation" data-cites="pHrstqMQ Gj8vlc0W">[<a href="#ref-pHrstqMQ" role="doc-biblioref">184</a>,<a href="#ref-Gj8vlc0W" role="doc-biblioref">185</a>]</span>.
The first study, which is by Blanco-Melo et al., performed an in-depth analysis of the transcriptional response to SARS-CoV-2 in comparison to other respiratory viruses, including MERS-CoV, SARS-CoV, human parainfluenza virus 3 (HPIV3), respiratory syncytial virus (RSV), and influenza A virus (IAV).
They analyzed the responses of three human cell lines: A549 (adenocarcinomic human alveolar basal epithelial cells), Calu-3 (human airway epithelial cells derived from human bronchial submucosal glands), and MRC-5 (human fetal lung fibroblast cells).
As the viral receptor ACE2 has low expression in A549 cells, authors also supplemented A549 cells with adenovirus (AdV)-based vectors expressing mCherry (a fluorescent protein used as a control) or ACE2 (A549-ACE2).
All infections were performed at a high multiplicity of infections (MOI, 2–5), with the exception of A549-ACE2 cells, which were infected with SARS-CoV-2 at both low (0.2 MOI) and high MOI (2 MOI).
Poly(A) bulk RNA sequencing (RNA-seq) of host cells was performed at 24 hours post-infection (hpi) of all the viruses except IAV (9-12 hours).
The authors also measured host transcriptional responses to SARS-CoV-2 in primary normal human bronchial epithelial cells (HBEC or NHBE cells), nasal washes of ferrets as an animal model, and lung samples from two COVID-19 patients.
Additionally, they measured transcriptional responses to IAV in NHBE cells and nasal washes of ferrets.
Differential expression (DE) analysis was then carried out to compare infected cells with control cells that underwent only a mock treatment.
In general, the interferons (IFNs) activate intracellular antimicrobial programs and are the major first line of defense against viruses <span class="citation" data-cites="1228YjPRv">[<a href="#ref-1228YjPRv" role="doc-biblioref">186</a>]</span>.
IFNs are a type of cytokines along with chemokines, interleukins (IL), lymphokines and tumor necrosis factor (TNF).
Cytokines can be classified based on the nature of the immune response or based on their specific roles, as reviewed by <span class="citation" data-cites="A4XFZlP">[<a href="#ref-A4XFZlP" role="doc-biblioref">187</a>]</span>).
In the hosts where SARS-CoV-2 was able to replicate efficiently, DE analysis revealed that the transcriptional response was significantly different from the response to all of the other viruses tested.
A unique proinflammatory cytokine signature associated with SARS-CoV-2 was present under both high- and low-MOI conditions.
The cytokines IL-6 and IL1RA were uniquely elevated in response to SARS-CoV-2.
However, A549-ACE2 cells in the low-MOI condition showed no significant IFN-I or IFN-III expression, but did in cells at high MOI.
This finding suggests that IFN induction is MOI dependent.
Similarly, in cells from the NHBE line, ferrets, and COVID-19 patients, they found significant enrichment of chemokine signaling but no significant induction of IFN-I or IFN-III.
Together, these results suggest that in contrast to common respiratory viruses, SARS-CoV-2 induces a limited antiviral state with low IFN-I or IFN-III expression and a moderate IFN-stimulated genes (ISG) response.
However, the low IFN induction in ACE2-expressing A549 cells could be overcome by using a 10-fold increase in SARS-CoV-2 (MOI, 2), suggesting a SARS-CoV-2 antagonist that is rendered ineffective under high-MOI conditions <span class="citation" data-cites="pHrstqMQ">[<a href="#ref-pHrstqMQ" role="doc-biblioref">184</a>]</span>.
This hypothesis was further supported by a recent study <span class="citation" data-cites="PJgYPcKM">[<a href="#ref-PJgYPcKM" role="doc-biblioref">188</a>]</span> that showed that the SARS-CoV-2 <em>ORF3b</em> gene suppresses IFNB1 promoter activity (IFN-I induction) more efficiently than the SARS-CoV_ORF3b_ gene, which is considerably longer.</p>
<p>A second study by Wyler et al. <span class="citation" data-cites="Gj8vlc0W">[<a href="#ref-Gj8vlc0W" role="doc-biblioref">185</a>]</span> examined three human cell lines: H1299 (human non-small cell lung carcinoma cell line), Calu-3, and Caco-2 (human epithelial colorectal adenocarcinoma cell line).
They performed a comprehensive analysis of the transcriptional response to SARS-CoV-2 and SARS-CoV at different hpi (4-36 hours) with an MOI of 0.3.
Using poly(A) bulk RNA-seq, the authors found negligible susceptibility of H1299 cells (&lt; 0.08 viral read percentage of total reads) in contrast to Caco-2 and Calu-3 cells (&gt;10% of viral reads), suggesting a role of cell-type specific host factors.
Based on visual inspection (microscopy images) and transcriptional profiling, the authors showed distinct responses associated with different hosts following infection.
In contrast to Caco-2, SARS-CoV-2 infected Calu-3 cells showed signs of impaired growth and cell death at 24 hpi, as well as moderate IFN induction with a strong up-regulation of ISGs.
Interestingly, the response is similar to SARS-CoV-2 infected Calu-3 cells at ~7-fold higher MOI performed by Blanco-Melo et al., suggesting that IFN induction is not MOI dependent in Calu-3 cells, in contrast to reports using A549-ACE2 cells.
The discrepancy could be explained by the observations that Calu-3 cells are highly susceptible to SARS-CoV-2 with rapid virus replication <span class="citation" data-cites="JOJ2n3gC">[<a href="#ref-JOJ2n3gC" role="doc-biblioref">140</a>]</span>, whereas A549 cells are incompatible with SARS-CoV-2 infection <span class="citation" data-cites="16EFyBURq">[<a href="#ref-16EFyBURq" role="doc-biblioref">189</a>]</span>.
We performed a correlation analysis of the host expression response (RNA fold change compared to the mock infection) using the publicly available analyzed data from Blanco-Melo et al. and Wyler et al. (Figure <a href="#fig:cell-lines-moi">2</a>B) <span class="citation" data-cites="pHrstqMQ Gj8vlc0W">[<a href="#ref-pHrstqMQ" role="doc-biblioref">184</a>,<a href="#ref-Gj8vlc0W" role="doc-biblioref">185</a>]</span>.
In general, we found high correlations among the same host irrespective of the study and the virus infected, suggesting that the response is host-specific.
For instance, Calu-3 cells infected with SARS-CoV by Wyler et al. and Calu-3 cells infected with SARS-CoV-2 by Blanco-Melo et al. had a Pearson’s correlation coefficient of 0.8 (Figure <a href="#fig:cell-lines-moi">2</a>B).
Moreover, COVID-19 patients had the lowest correlations among all the cells used, with the highest value being 0.29.
This discrepancy suggests that the <em>in vitro</em> models used may not necessarily imitate the human response, underscoring the importance of follow-up in additional models.</p>
<div id="fig:cell-lines-moi" class="fignos">
<figure>
<img src="images/cell-lines-moi.jpg" alt="" /><figcaption><span>Figure 2:</span> <strong>Profiling and correlation of expression response after respiratory viruses infection in various hosts.</strong> (A) Overview of hosts and respiratory viruses used in two studies <span class="citation" data-cites="pHrstqMQ Gj8vlc0W">[<a href="#ref-pHrstqMQ" role="doc-biblioref">184</a>,<a href="#ref-Gj8vlc0W" role="doc-biblioref">185</a>]</span>. MOI:Multiplicity of infections; PFU:Plaque-forming unit; a:ACE2-expressing A549 cells; b:Data used from GSE56192. (B) Pearson’s correlation of expression response (Log2 fold change) between various hosts. Note that we used the data from bulk RNA-seq produced after the infection of unmodified viruses.</figcaption>
</figure>
</div>
<!-- Start with scRNA-seq studies in healthy tissues (may be put this before Blanco-Melo et al), then summarize scRNA-seq studies in COVID-19 patients] -->
<h1 data-number="3" id="diagnostics-1"><span class="header-section-number">3</span> Diagnostics</h1>
<p>Identifying individuals who have contracted COVID-19 is crucial to slowing down the global pandemic.
Given the high transmissibility of SARS-CoV-2, the development of reliable assays to detect SARS-CoV-2 infection even in asymptomatic carriers is vitally important.
For instance, the deployment of wide-scale diagnostic testing followed by the isolation of infected people has been a key factor in South Korea’s successful strategy for controlling the spread of the virus.
Following the first release of the genetic sequence of the virus by Chinese officials on January 10 2020, the first test was released about 13 days later <span class="citation" data-cites="1AHtg71LX">[<a href="#ref-1AHtg71LX" role="doc-biblioref">190</a>]</span>.
A range of diagnostic approaches from a methodological standpoint are being or could possibly be developed.</p>
<p>There are two main classes of diagnostic tests: molecular tests, which can diagnose an active infection by identifying the presence of SARS-CoV-2, and serological tests, which can assess whether an individual was infected in the past via the presence or absence of antibodies against SARS-CoV-2.
Molecular tests are essential for identifying individuals for treatment and alerting their contacts to quarantine and be alert for possible symptoms.
While serological tests may be of interest to individuals who wish to confirm they were infected with SARS-CoV-2 in the past based on disease symptoms, their potential for false positives means that they are not currently recommended for this use.
However, serological tests are critical at the population level from an epidemiological perspective, as they can be used to estimate the extent of the infection in a given area.
Thus, they may be used to better understand the percent of infected cases that develop severe disease as well as to guide public health and economic decisions regarding resource allocation and counter-disease measures.</p>
<h2 data-number="3.1" id="molecular-tests"><span class="header-section-number">3.1</span> Molecular Tests</h2>
<p>Molecular tests are used to identify distinct genomic subsequences of a viral molecule in a sample and thus to diagnose an active viral infection.
This first requires identifying biospecimens that are likely to contain the virus in infected individuals and then acquiring these samples from the patient(s) to be tested.
Common sources for a sample used in a molecular test include nasopharyngeal cavity samples, including throat wash and saliva <span class="citation" data-cites="fCZXxoas">[<a href="#ref-fCZXxoas" role="doc-biblioref">191</a>]</span>, or stool samples <span class="citation" data-cites="oNFnjRBI">[<a href="#ref-oNFnjRBI" role="doc-biblioref">192</a>]</span>.
Given a sample from a patient, molecular tests involve a number of steps to analyze a sample and produce results.
When testing for RNA viruses like SARS-CoV-2, pre-processing is needed in order to create DNA, which can then be replicated during PCR, from the initial RNA sample.
The DNA can then be amplified with PCR.
Some tests use the results of the PCR to determine presence or absence of the pathogen, but in other cases, it may be necessary to sequence the amplified DNA.
Sequencing requires an additional pre-processing step: library preparation.
Library preparation is the process of preparing the sample for sequencing, typically by fragmenting the sequences and adding adapters <span class="citation" data-cites="jr3z6jGU">[<a href="#ref-jr3z6jGU" role="doc-biblioref">193</a>]</span>.
In some cases, library preparation can involve other modifications of the sample, such as adding “barcoding” to identify a particular sample in the sequence data, which is useful for pooling samples from multiple sources.
There are different reagents used for library preparation that are specific to identifying one or more target sections with PCR <span class="citation" data-cites="43modeVu">[<a href="#ref-43modeVu" role="doc-biblioref">194</a>]</span>.
Sequential pattern matching is then used to identify unique subsequences of the virus that identify it in specific.
If sufficient subsequences are found, the test is considered positive.</p>
<h3 data-number="3.1.1" id="rt-pcr"><span class="header-section-number">3.1.1</span> RT-PCR</h3>
<p>Real-Time Polymerase Chain Reaction (RT-PCR) tests determine whether a target is present by measuring the rate of amplification during PCR compared to a standard.
When the target is RNA, such as in the case of RNA viruses, the RNA must be converted into complementary DNA during pre-processing.
There are different reagents used for library preparation that are specific to identifying one or more target sections with PCR <span class="citation" data-cites="43modeVu">[<a href="#ref-43modeVu" role="doc-biblioref">194</a>]</span>.
The Drosten Lab, from Germany, was the first lab to establish and validate a diagnostic test to detect SARS-CoV-2.
This test uses RT-PCR with reverse transcription <span class="citation" data-cites="1AHtg71LX">[<a href="#ref-1AHtg71LX" role="doc-biblioref">190</a>]</span> to detect several regions of the viral genome: the <em>ORF1b</em> of the RNA dependent RNA polymerase (RdRP), the Envelope protein gene (<em>E</em>), and the Nucleocapsid protein gene (<em>N</em>).
In collaboration with several other labs in Europe and in China, the researchers confirmed the specificity of this test with respect to other coronaviruses against specimens from 297 patients infected with a broad range of respiratory agents.
Specifically this test utilizes two probes against RdRP of which one is specific to SARS-CoV-2 <span class="citation" data-cites="1AHtg71LX">[<a href="#ref-1AHtg71LX" role="doc-biblioref">190</a>]</span>.
Importantly, this assay did not give any false positive results.</p>
<h3 data-number="3.1.2" id="qrt-pcr"><span class="header-section-number">3.1.2</span> qRT-PCR</h3>
<p>Chinese researchers developed a quantitative real-time reverse transcription PCR (qRT-PCR) test to identify two gene regions of the viral genome, <em>ORF1b</em> and <em>N</em> <span class="citation" data-cites="COEfjKo5">[<a href="#ref-COEfjKo5" role="doc-biblioref">195</a>]</span>.
Specifically, this assay was tested on samples coming from two COVID-19 patients, including a panel of positive and negative controls consisting of RNA extracted from several cultured viruses.
The assay uses the <em>N</em> gene to screen patients, while the <em>ORF1b</em> gene region is used to confirm the infection <span class="citation" data-cites="COEfjKo5">[<a href="#ref-COEfjKo5" role="doc-biblioref">195</a>]</span>.
In this case the test was designed to detect sequences conserved across sarbecoviruses, or viruses within the same subgenus as SARS-CoV-2.
Considering that no other sarbecoviruses are currently known to infect humans, a positive test indicates that the patient is infected with SARS-CoV-2.
However, this test is not able to discriminate the genetics of viruses within the sarbecovirus clade.</p>
<h4 data-number="3.1.2.1" id="dpcr"><span class="header-section-number">3.1.2.1</span> dPCR</h4>
<p>Digital PCR (dPCR) is a new generation of PCR technologies offering an alternative to traditional real-time quantitative PCR.
In dPCR, a sample is partitioned into thousands of compartments, such as nanodroplets (droplet dPCR or ddPCR) or nanowells, and a PCR reaction takes place in each compartment.
This leads to a digital read-out where each partition is either positive or negative for the nucleic acid sequence being tested for, allowing for much higher throughput.
While dPCR equipment is not yet as common as that for RT-PCR, dPCR for DNA targets generally achieves higher sensitivity than other PCR technologies while maintaining high specificity, though sensitivity is slightly lower for RNA targets <span class="citation" data-cites="Hl2pASeg">[<a href="#ref-Hl2pASeg" role="doc-biblioref">196</a>]</span>.
High sensitivity is particularly relevant for SARS-CoV-2 detection, since low viral load in clinical samples can lead to false negatives.
Suo et al. performed a double-blind evaluation of ddPCR for SARS-CoV-2 detection on 57 samples–43 samples from suspected positive patients, and 14 from supposed convalescents–that had all tested negative for SARS-CoV-2 using RT-PCR.
Despite the initial negative results, 33 out of 35 (94.3%) patients were later clinically confirmed positive.
All of these individuals tested positive using ddPCR.
Additionally, of 14 supposed convalescents who had received two consecutive negative RT-PCR tests, nine (64.2%) tested positive for SARS-CoV-2 using ddPCR.
Two symptomatic patients tested negative with both RT-PCR and ddPCR, but were later clinically diagnosed positive, and 5 of the 14 suspected convalescents tested negative by ddPCR <span class="citation" data-cites="2WgqQLF8">[<a href="#ref-2WgqQLF8" role="doc-biblioref">197</a>]</span>.
While not a complete head-to-head comparison to RT-PCR in all aspects– e.g. no samples testing positive using RT-PCR were evaluated by ddPCR– the study shows the potential of dPCR for viral detection in highly diluted samples.
In a second study, Dong et al. <span class="citation" data-cites="UYD5T2mp">[<a href="#ref-UYD5T2mp" role="doc-biblioref">198</a>]</span> compared the results of qRT-PCR and ddPCR testing for SARS-CoV-2 in 194 samples, including 103 samples from suspected patients, 75 from contacts and close contacts, and 16 from suspected convalescents.
Of the 103 suspected patient samples, 29 were reported as positive, 25 as negative, and 49 as suspected by qRT-PCR; all patients were later confirmed to be SARS-CoV-2 positive.
Of the qRT-PCR negative or suspected samples, a total of 61 (17 negative and 44 suspected) were later confirmed to be positive by ddPCR, improving the overall detection rate among these patients from 28.2% to 87.4%.
Of 75 patient samples from contacts and close contacts, 48 were tested negative with both methods and patients indeed remained healthy.
Within the remaining 27 patient samples, 10 tested positive, 1 negative, 16 suspect by qRT-PCR.
15 out of 16 suspect samples and the negative test result were overturned by ddPCR, decreasing the rate of suspect cases from 21% to 1%.
Importantly, all samples that tested positive using qRT-PCR also tested positive using ddPCR.
Among the 16 convalescent patients, qRT-PCR identified 12 as positive, three as suspect, and one as negative, but RT-dPCR identified all 16 as positive.
This evidence further indicates that the lower limit of detection made possible by ddPCR may be useful for identifying when COVID-19 patients are cleared of the virus.
Overall, these studies suggest that dPCR is a promising tool for overcoming the problem of false-negative SARS-CoV-2 testing.</p>
<h3 data-number="3.1.3" id="pooled-and-automated-pcr-testing"><span class="header-section-number">3.1.3</span> Pooled and Automated PCR Testing</h3>
<p>Due to limited supplies and the need for more tests, several labs have found ways to pool or otherwise strategically design tests to increase throughput.
The first such result came from Yelin et al. <span class="citation" data-cites="93NBANq2">[<a href="#ref-93NBANq2" role="doc-biblioref">199</a>]</span>, who found they could pool up to 32 samples in a single qPCR run.
This was followed by larger-scale pooling with slightly different methods <span class="citation" data-cites="IQ5kUQEO">[<a href="#ref-IQ5kUQEO" role="doc-biblioref">200</a>]</span>.
Although these approaches are also PCR based, they allow for more rapid scaling and higher efficiency for testing than the initial PCR-based methods developed.</p>
<h4 data-number="3.1.3.1" id="crispr-based-detection"><span class="header-section-number">3.1.3.1</span> CRISPR-based detection</h4>
<p>Two American companies, Mammoth Biosciences and Sherlock Biosciences, adapted their CRISPR-based detection technology <span class="citation" data-cites="1272qWb3a">[<a href="#ref-1272qWb3a" role="doc-biblioref">201</a>]</span> for COVID-19 diagnostics to increase testing throughput and accessibility <span class="citation" data-cites="1dxrDub1">[<a href="#ref-1dxrDub1" role="doc-biblioref">202</a>]</span>.
Their methodology involves purification of RNA extracted from patient specimens, amplification of extracted RNAs by loop-mediated amplification, a rapid, isothermal nucleic acid amplification technique, and application of their CRISPR-Cas12-based technology.
In the assay designed by Mammoth Biosciences, guide RNAs were designed to recognize portions of sequences corresponding to the SARS-CoV-2 genome, specifically the N, E and RdRP regions.
In the presence of SARS-CoV-2 genetic material, sequence recognition by the guide RNAs results in double-stranded DNA cleavage by Cas12, as well as cleavage of a single-stranded DNA molecular beacon.
The cleavage of this molecular beacon acts as a colorimetric reporter that is subsequently read out in a lateral flow assay and indicates the positive presence of SARS-CoV-2 genetic material and therefore SARS-CoV-2 infection <span class="citation" data-cites="1dxrDub1">[<a href="#ref-1dxrDub1" role="doc-biblioref">202</a>]</span>.
This assay has been reported to have a sensitivity as high as detection of 70-300 copies of the target RNA/µl, requires simple, accessible equipment, and can output an easily-interpretable result in approximately 30 minutes <span class="citation" data-cites="NT5mrVGY">[<a href="#ref-NT5mrVGY" role="doc-biblioref">203</a>]</span>.
Initial testing with patient samples (n = 23) demonstrated a positive predictive value of 100% and a negative predictive value of 91.7%, which is highly competitive with the CDC’s current testing standard using qRT-PCR.
This test seeks to develop a practical solution for rapid, low-barrier testing in areas that are at greater risk of infection, such as airports and local community hospitals.</p>
<h3 data-number="3.1.4" id="limitation-of-molecular-tests"><span class="header-section-number">3.1.4</span> Limitation of Molecular Tests</h3>
<p>Tests that identify SARS-CoV-2 using nucleic-acid-based technologies will identify only individuals with current infections and are not appropriate for identifying individuals who have recovered from a previous infection.
Within this category, different types of tests have different limitations.
For example, PCR-based test can be highly sensitive, but in high-throughput settings they can show several problems:</p>
<ol type="1">
<li>False-negative responses, which can present a significant problem to large-scale testing.
To reduce occurrence of false negatives, correct execution of the analysis is crucial <span class="citation" data-cites="Fdc7seLs">[<a href="#ref-Fdc7seLs" role="doc-biblioref">204</a>]</span>.</li>
<li>Uncertainty surrounding the SARS-CoV-2 viral shedding kinetics, which could affect the result of a test depending on when it was taken <span class="citation" data-cites="Fdc7seLs">[<a href="#ref-Fdc7seLs" role="doc-biblioref">204</a>]</span>.</li>
<li>Type of specimen, as it is not clear which clinical samples are best to detect the virus <span class="citation" data-cites="Fdc7seLs">[<a href="#ref-Fdc7seLs" role="doc-biblioref">204</a>]</span>.</li>
<li>Expensive machinery, which might be present in major hospitals and/or diagnostic centers but is often not available to smaller facilities <span class="citation" data-cites="NT5mrVGY">[<a href="#ref-NT5mrVGY" role="doc-biblioref">203</a>]</span>.</li>
<li>Timing of the test, which might take up to 4 days to give results <span class="citation" data-cites="NT5mrVGY">[<a href="#ref-NT5mrVGY" role="doc-biblioref">203</a>]</span>.</li>
<li>The availability of supplies for testing, including swabs and testing media, has been limited <span class="citation" data-cites="AvQI3GSm">[<a href="#ref-AvQI3GSm" role="doc-biblioref">205</a>]</span>.</li>
<li>Because the guide RNA can recognize other interspersed sequences on the patient’s genome, false positives and a loss of specificity can occur.</li>
</ol>
<p>Similarly, in tests that use CRISPR, false positives can occur due to the specificity of the technique, as the guide RNA can recognize other interspersed sequences on the patient’s genome.
As noted above, false negatives are a significant concern for several reason.
Importantly, clinical reports indicate that it is imperative to exercise caution when interpreting the results of molecular tests for SARS-CoV-2 because negative results do not necessarily mean a patient is virus-free <span class="citation" data-cites="11vIuqmS2">[<a href="#ref-11vIuqmS2" role="doc-biblioref">206</a>]</span>.</p>
<h2 data-number="3.2" id="serological-tests"><span class="header-section-number">3.2</span> Serological Tests</h2>
<p>Although diagnostic tests based on the detection of the genetic material can be quite sensitive, they cannot provide information about the extent of the disease over time.
Most importantly, they would not work on a patient who has fully recovered from the virus at the time of sample collection.
In this context, serological tests, which use serum to test for the presence of antibodies against SARS-CoV-2, are significantly more informative.
Additionally, they can help scientists to understand why the disease has a different course among patients, as well as what strategy might work to manage the spread of the infection.
Furthermore, serological tests hold significant interest at present because they can provide information relevant to advancing economic recovery and allowing reopenings.
For instance, people that have developed antibodies can plausibly return to work prior to the others, based on (still-unproven) protective immunity <span class="citation" data-cites="EUrJt06r">[<a href="#ref-EUrJt06r" role="doc-biblioref">207</a>]</span>.
On a related note, for some infectious agents an epidemic can be stopped from growing through herd immunity in which enough of the population is immune to infection through vaccination and/or prior experience with infection.
A simple SIR model predicts that to achieve the required level of exposure for herd immunity to be effective, at least (1-(1/R0)) fraction of the population must be immune or, equivalently, less than (1/R0) fraction of the population susceptible <span class="citation" data-cites="JRaVgcNd">[<a href="#ref-JRaVgcNd" role="doc-biblioref">149</a>]</span>.
However, for SARS-CoV-2 and COVID-19, because of the calculated R0 and mortality and the recorded per-region deaths and accumulated cases with the estimated factor of undetected cases, relying on herd immunity without vaccines and/or proven treatment options and/or strong non-pharmaceutical measures of prevention and control would likely result in a significant loss of life.</p>
<h3 data-number="3.2.1" id="current-approaches"><span class="header-section-number">3.2.1</span> Current Approaches</h3>
<p>Several countries are now focused on implementing antibody tests, and in the United States, the FDA recently approved a serological test by Cellex for use under emergency conditions <span class="citation" data-cites="n6sKSpIk">[<a href="#ref-n6sKSpIk" role="doc-biblioref">208</a>]</span>.
Specifically, the Cellex qSARS-CoV-2 IgG/IgM Rapid Test is a chromatographic immunoassay designed to qualitatively detect IgM and IgG antibodies against SARS-CoV-2 in the plasma of patients (blood sample) suspected to have developed the infection <span class="citation" data-cites="n6sKSpIk">[<a href="#ref-n6sKSpIk" role="doc-biblioref">208</a>]</span>.
Such tests allow for the progress of the viral disease to be understood, as IgM are the first antibodies produced by the body and indicate that the infection is active.
Once the body has responded to the infection, IgG are produced and gradually replace IgM, indicating that the body has developed immunogenic memory <span class="citation" data-cites="FFMfMDRN">[<a href="#ref-FFMfMDRN" role="doc-biblioref">209</a>]</span>.
The test cassette contains a pad of SARS-CoV-2 antigens and a nitrocellulose strip with lines for each of IgG and IgM, as well as a control (goat IgG) <span class="citation" data-cites="n6sKSpIk">[<a href="#ref-n6sKSpIk" role="doc-biblioref">208</a>]</span>.
In a specimen that contains antibodies against the SARS-CoV-2 antigen, the antibodies will bind to the strip and be captured by the IgM and/or IgG line(s), resulting in a change of color <span class="citation" data-cites="n6sKSpIk">[<a href="#ref-n6sKSpIk" role="doc-biblioref">208</a>]</span>.
With this particular assay results can be read within 15-20 minutes <span class="citation" data-cites="n6sKSpIk">[<a href="#ref-n6sKSpIk" role="doc-biblioref">208</a>]</span>.
Other research groups, such as the Krammer lab of the Icahn School of Medicine at Mount Sinai proposed an ELISA test that detects IgG and IgM that react against the receptor binding domain (RBD) of the spike proteins (S) of the virus <span class="citation" data-cites="yJW1RVQb">[<a href="#ref-yJW1RVQb" role="doc-biblioref">210</a>]</span>.
The authors are now working to get the assay into clinical use <span class="citation" data-cites="Gk9WwhYB">[<a href="#ref-Gk9WwhYB" role="doc-biblioref">211</a>]</span>.</p>
<h3 data-number="3.2.2" id="limitations-of-serological-tests"><span class="header-section-number">3.2.2</span> Limitations of Serological Tests</h3>
<p>Importantly, false-positives can occur due to the cross-reactivity with other antibodies according to the clinical condition of the patient <span class="citation" data-cites="n6sKSpIk">[<a href="#ref-n6sKSpIk" role="doc-biblioref">208</a>]</span>.
Therefore, this test should be used in combination with RNA detection tests <span class="citation" data-cites="n6sKSpIk">[<a href="#ref-n6sKSpIk" role="doc-biblioref">208</a>]</span>.
Due to the long incubation times and delayed immune responses of infected patients, serological tests are insufficiently sensitive for a diagnosis in the early stages of an infection.
The limitations due to timing make serological tests far less useful for enabling test-and-trace strategies.</p>
<h2 data-number="3.3" id="possible-alternatives-to-current-practices-for-identifying-active-cases"><span class="header-section-number">3.3</span> Possible Alternatives to Current Practices for Identifying Active Cases</h2>
<p>Clinical symptoms are too similar to other types of pneumonia to be sufficient as a sole diagnostics criterion.
In addition, as noted above, identifying asymptomatic cases is critical.
Even among mildly symptomatic patients, a predictive model based on clinical symptoms had a sensitivity of only 56% and a specificity of 91% <span class="citation" data-cites="91dto3z3">[<a href="#ref-91dto3z3" role="doc-biblioref">212</a>]</span>.
More problematic is that clinical symptom-based tests are only able to identify already symptomatic cases, not presymptomatic or asymptomatic cases.
They may still be important for clinical practice, and for reducing tests needed for patients deemed unlikely to have COVID-19.</p>
<p>X-ray diagnostics have been reported to have high sensitivity but low specificity in some studies <span class="citation" data-cites="itCGVb5b">[<a href="#ref-itCGVb5b" role="doc-biblioref">213</a>]</span>.
Other studies have shown that specificity varies between radiologists <span class="citation" data-cites="tkuUFNIX">[<a href="#ref-tkuUFNIX" role="doc-biblioref">214</a>]</span>, though the sensitivity reported here was lower than that published in the previous paper.
However, preliminary machine-learning results have shown far higher sensitivity and specificity from analyzing chest X-rays than was possible with clinical examination <span class="citation" data-cites="nN9zfzPe">[<a href="#ref-nN9zfzPe" role="doc-biblioref">215</a>]</span>.
X-ray tests with machine learning can potentially detect asymptomatic or presymptomatic infections that show lung manifestations.
This approach would still not recognize entirely asymptomatic cases.
Given the above, the widespread use of X-ray tests on otherwise healthy adults is likely inadvisable.</p>
<h2 data-number="3.4" id="challenges-to-diagnostic-approaches"><span class="header-section-number">3.4</span> Challenges to Diagnostic Approaches</h2>
<h3 data-number="3.4.1" id="limitations-to-implementation-of-large-scale-testing"><span class="header-section-number">3.4.1</span> Limitations to Implementation of Large-Scale Testing</h3>
<p>More information to follow.</p>
<h3 data-number="3.4.2" id="strategies-and-considerations-for-determining-whom-to-test"><span class="header-section-number">3.4.2</span> Strategies and Considerations for Determining Whom to Test</h3>
<p>Currently, Coronavirus tests are limited to people that are in danger of serious illness <span class="citation" data-cites="OB8BJiwj">[<a href="#ref-OB8BJiwj" role="doc-biblioref">216</a>]</span>.
Specifically, the individuals at risk include:</p>
<ul>
<li>people with severe symptoms</li>
<li>people showing mild symptoms that have been in contact with a person who has developed the infection</li>
<li>people with underlying health conditions</li>
</ul>
<p>However, this method of testing administration does not detect a high proportion of infections and does not allow for test-and-trace methods to be used.
Individuals who are asymptomatic (i.e. potential spreaders) and individuals who are able to recover at home are therefore often unaware of their status.
For instance, a recent study from Imperial College estimates that in Italy the true number of infections is around 5.9 million in a total population ~60 million, compared to the 70,000 detected as of March 28th <span class="citation" data-cites="Frya4XA4">[<a href="#ref-Frya4XA4" role="doc-biblioref">167</a>]</span>.
Another analysis, which examined New York state, indicated that as of May 2020, approximately 300,000 cases had been reported in a total population of approximately 20 million, corresponding to ~1.5% of the population, but 12% of individuals tested statewide and up to ~20% in some places tested positive when evaluated with antibody tests <span class="citation" data-cites="IldxlHwA">[<a href="#ref-IldxlHwA" role="doc-biblioref">217</a>]</span>.</p>
<h1 data-number="4" id="therapeutics-and-prophylactics"><span class="header-section-number">4</span> Therapeutics and Prophylactics</h1>
<p>Given the observed and predicted spread of COVID-19, the development of interventions will be critical to mitigating its effect on health and the mortality rate.
Such interventions fall into two categories: therapeutics, which are meant to treat an already existing disease, and prophylactics, which are meant to prevent a disease from occurring.
For infectious diseases such as COVID-19, the main prophylactics are vaccines; several types of vaccines are currently under development, as detailed below.
While vaccines would be expected to save the largest number of lives by bolstering the immune response of the at-risk population broadly to the virus, which would result in a lower rate of infection, the vaccine development process is long, and they fail to provide immediate prophylactic protection or treat ongoing infections <span class="citation" data-cites="181QWa7HL">[<a href="#ref-181QWa7HL" role="doc-biblioref">218</a>]</span>.
This means that there is also an immediate need for treatments that palliate symptoms to avoid the most severe outcomes from infection.
Therapeutics can generally either be considered for the treatment and reduction of symptoms — to reduce the severity and risks associated with an active infection — or as a more direct way of targeting the virus (“antivirals”) — to inhibit the development of the virus once an individual is infected.
In the context of COVID-19, there is uncertainty surrounding the exact mechanism of action, as most therapies have secondary or off-target effects.
Thus, for this section, we will classify both therapeutics and prophylactics according to their biological properties, specifically whether they are biologics (produced from components of organisms) or small molecules.
Biologics include antibodies, interferons, and vaccines, while small molecules include drugs targeted at viral particles, drugs targeted at host proteins, and broad-based pharmaceuticals.
Broad-based pharmaceuticals include the much-discussed drugs hydroxychloroquine and chloroquine.
We also describe nutraceuticals, which are dietary supplement interventions that may prime an individual’s immune system to prevent or lessen the impact of RNA virus infections <span class="citation" data-cites="eyBAUUCl GT7yDBjv">[<a href="#ref-eyBAUUCl" role="doc-biblioref">219</a>,<a href="#ref-GT7yDBjv" role="doc-biblioref">220</a>]</span>; they have been considered for both treatment and prevention of COVID-19.
In the following sections, we critically appraise the literature and clinical trials (Figure <a href="#fig:ebm-trials">3</a>) surrounding the repurposing of existing treatments and development of novel approaches for the prevention, mitigation, and treatment of coronavirus infections.</p>
<div id="fig:ebm-trials" class="fignos">
<figure>
<img src="https://github.com/greenelab/covid19-review/raw/3ebd63d13ddfc066804429b847fd10caba7a26c3/ebmdatalab/ebmdatalab-trials.svg" alt="" /><figcaption><span>Figure 3:</span> <strong>COVID-19 clinical trials.</strong>
There are 4,282 COVID-19 clinical trials and 104 trials with results as of July 17, 2020.
The recruitment statuses and trial phases are shown only for trials in which the status or phase is recorded.
The study types include only types used in at least five trials.
The common interventions are all interventions used in at least ten trials.
Combinations of interventions, such as Hydroxychloroquine + Azithromycin, are tallied separately from the individual interventions.
Trials data are from the University of Oxford Evidence-Based Medicine Data Lab’s COVID-19 TrialsTracker <span class="citation" data-cites="SSbnPnzT">[<a href="#ref-SSbnPnzT" role="doc-biblioref">221</a>]</span>.</figcaption>
</figure>
</div>
<h2 data-number="4.1" id="treatment-of-symptoms"><span class="header-section-number">4.1</span> Treatment of Symptoms</h2>
<!--
I think it's OK to leave in a section like this to explain what this means.
It's different from eg palliative care in cancer.
We can make sure it fits better into the rest of the section.
-->
<p>The clinical picture of SARS-CoV-2 infection differs dramatically between individuals.
Some are asymptomatic, and many experience mild COVID-19 symptoms.
Mild symptoms commonly include fever and respiratory symptoms such as cough and sore throat, and, less commonly, gastrointestinal symptoms such as loss of appetite and vomiting <span class="citation" data-cites="4oZGCeCo">[<a href="#ref-4oZGCeCo" role="doc-biblioref">222</a>]</span>; some patients experience a combination of respiratory and gastrointestinal symptoms.
The most severe cases of COVID-19 include severe complications such as pneumonia and Acute Respiratory Distress Syndrome (ARDS), which can lead to respiratory failure and death <span class="citation" data-cites="vEFSIErd">[<a href="#ref-vEFSIErd" role="doc-biblioref">223</a>]</span>.
Thus, specific drugs may be considered to alleviate these severe symptoms and reduce the risk of death.</p>
<!--
Should include other emerging symptoms like cardiac complications, blood clots, etc.
-->
<h2 data-number="4.2" id="small-molecule-drugs"><span class="header-section-number">4.2</span> Small Molecule Drugs</h2>
<h3 data-number="4.2.1" id="small-molecule-drugs-for-targeting-sars-cov-2"><span class="header-section-number">4.2.1</span> Small Molecule Drugs for Targeting SARS-CoV-2</h3>
<p>The replication cycle of a virus within an epithelial host cell includes six basic steps that can be summarized as follows: i) attachment of the virus to the host cell; ii) penetration by endocytosis; iii) uncoating, classically defined as the release of viral contents into the host cell; iv) biosynthesis, during which the viral genetic material enters the nucleus where it gets replicated; v) assembly, where viral proteins are translated and new viral particles are assembled; vi) release, when the new viruses are released into the extracellular environment <span class="citation" data-cites="14gthSuhm">[<a href="#ref-14gthSuhm" role="doc-biblioref">224</a>]</span>.
Antiviral drugs do not kill the virus, rather they inhibit its amplification by impairing one of these steps.
Nowadays, many of these drugs act during the biosynthesis step in order to inhibit the replication of viral genetic material.
Importantly, SARS-CoV-2 is an RNA virus.
In contrast to DNA viruses, which can use the host enzymes to propagate themselves, RNA viruses depends on their own polymerase, the RNA-dependent RNA polymerase (RdRP), for replication <span class="citation" data-cites="tn6j6IZ0 Sh1FwWps">[<a href="#ref-tn6j6IZ0" role="doc-biblioref">225</a>,<a href="#ref-Sh1FwWps" role="doc-biblioref">226</a>]</span>.
As noted above, even if a drug is meant to target the virus, it can also impact other processes in the host.</p>
<h4 data-number="4.2.1.1" id="nucleoside-and-nucleotide-analogs"><span class="header-section-number">4.2.1.1</span> Nucleoside and Nucleotide Analogs</h4>
<h5 data-number="4.2.1.1.1" id="favipiravir"><span class="header-section-number">4.2.1.1.1</span> Favipiravir</h5>
<p>Favipiravir (Avigan) was discovered by Toyama Chemical Co., Ltd. <span class="citation" data-cites="IkAU62Pd">[<a href="#ref-IkAU62Pd" role="doc-biblioref">227</a>]</span>.
The drug was found to effective at blocking viral amplification in several Influenza subtypes as well as other RNA viruses, such as Flaviviridae and Picornaviridae, through a reduction in plaque formation <span class="citation" data-cites="EgoJKtW4">[<a href="#ref-EgoJKtW4" role="doc-biblioref">228</a>]</span> and viral replication in MDCK cells <span class="citation" data-cites="rG5RCRdJ">[<a href="#ref-rG5RCRdJ" role="doc-biblioref">229</a>]</span>.
Furthermore, inoculation of mice with favipiravir was shown to increase survivability.
In 2014, the drug was approved in Japan for the treatment of patients infected with influenza that was resistant to conventional treatments like neuraminidase inhibitors <span class="citation" data-cites="14p3btkRH">[<a href="#ref-14p3btkRH" role="doc-biblioref">230</a>]</span>.</p>
<p><strong>Anticipated Mechanism.</strong>
Favipiravir (6-fluoro-3-hydroxy-2-pyrazinecarboxamide) acts as a purine and purine nucleoside analogue that inhibits viral RNA polymerase in a dose-dependent manner across a range of RNA viruses, including Influenza virus <span class="citation" data-cites="DpdKJbki 9j7NVPJS IYEoCGHy 7d1TnpE3 15kQ4QYbx">[<a href="#ref-DpdKJbki" role="doc-biblioref">231</a>,<a href="#ref-9j7NVPJS" role="doc-biblioref">232</a>,<a href="#ref-IYEoCGHy" role="doc-biblioref">233</a>,<a href="#ref-7d1TnpE3" role="doc-biblioref">234</a>,<a href="#ref-15kQ4QYbx" role="doc-biblioref">235</a>]</span>.
Nucleotide/side are the natural building blocks for RNA synthesis.
Because of this, modifications to these nucleotides/sides can disrupt key processes including replication <span class="citation" data-cites="MPJdSF1p">[<a href="#ref-MPJdSF1p" role="doc-biblioref">236</a>]</span>.
Biochemical experiments showed that favipiravir was recognized as a purine nucleoside analogue and incorporated into the viral RNA template.
A single incorporation does not influence RNA transcription; however, multiple events of incorporation lead to the arrest of RNA synthesis <span class="citation" data-cites="ukZRKllk">[<a href="#ref-ukZRKllk" role="doc-biblioref">237</a>]</span>.
Evidence for T-705 inhibiting viral RNA polymerase are based on time-of-drug addition studies that found that viral loads were reduced with the addition of Favipiravir in early times post-infection <span class="citation" data-cites="DpdKJbki 7d1TnpE3 15kQ4QYbx">[<a href="#ref-DpdKJbki" role="doc-biblioref">231</a>,<a href="#ref-7d1TnpE3" role="doc-biblioref">234</a>,<a href="#ref-15kQ4QYbx" role="doc-biblioref">235</a>]</span>.</p>
<p><strong>Current Evidence.</strong>
The effectiveness of favipiravir for treating patients with COVID-19 is currently under investigation.
An open-label, nonrandomized, before-after controlled study was recently conducted <span class="citation" data-cites="13Oz7yRPx">[<a href="#ref-13Oz7yRPx" role="doc-biblioref">238</a>]</span>.
The study included 80 COVID-19 patients (35 treated with favipiravir, 45 control) from the isolation ward of the National Clinical Research Center for Infectious Diseases (The Third People’s Hospital of Shenzhen), Shenzhen, China.
The patients in the control group were treated with other antivirals, such as lopinavir and ritonavir.
Treatment was applied on days 2-14; treatment stopped either when viral clearance was confirmed or at day 14.
The efficacy of the treatment was measured by 1) the time until viral clearance using Kaplan-Meier survival curves, and 2) the improvement rate of chest computed tomography (CT) scans on day 14 after treatment.
The study found that favipiravir increased the speed of recovery (measured as viral clearance from the patient by RT-PCR) to 4 days compared to 11 days using other antivirals such as lopinavir and ritonavir.
Additionally, the lung CT scans of patients treated with favipiravir showed significantly higher improvement rates (91%) on day 14 compared to control patients (62%).
However, there were adverse side effects in 4 (11%) favipiravir-treated patients and 25 (56%) control patients.
The adverse side effects included: diarrhea, vomiting, nausea, rash, and liver and kidney injury.
Overall, despite the study reporting clinical improvement in favipiravir-treated patients, due to some issues with study design, it cannot be determined whether treatment with favipiravir had an effect or whether these patients would have recovered regardless of any treatment.
For example, although there were significant differences between the two treatment groups, follow-up analysis is necessary due to the small sample size.
The selection of patients did not take into consideration important factors such as previous clinical conditions or sex, and there was no age categorization.
The study was not randomized or blinded, and the baseline control group was another antiviral instead of a placebo.
Therefore, randomized controlled trials are still required.</p>
<h5 data-number="4.2.1.1.2" id="remdesivir"><span class="header-section-number">4.2.1.1.2</span> Remdesivir</h5>
<p>Remdesivir (GS-5734) was developed by Gilead Sciences to treat Ebola Virus Disease.
At the outset of the COVID-19 pandemic, it did not have any have any FDA-approved use.
However, on May 1, 2020, the FDA issued an Emergency Use Authorization (EUA) for remdesivir for the treatment of hospitalized COVID-19 patients <span class="citation" data-cites="1BYoe0y2H">[<a href="#ref-1BYoe0y2H" role="doc-biblioref">239</a>]</span>.
The EUA was based on information from two clinical trials, NCT04280705 and NCT04292899 <span class="citation" data-cites="yTCAmOyt 1ATTPbCCO DtpdBrgt 16zLDIOyk">[<a href="#ref-yTCAmOyt" role="doc-biblioref">240</a>,<a href="#ref-1ATTPbCCO" role="doc-biblioref">241</a>,<a href="#ref-DtpdBrgt" role="doc-biblioref">242</a>,<a href="#ref-16zLDIOyk" role="doc-biblioref">243</a>]</span>.</p>
<p><strong>Anticipated Mechanism.</strong>
Remdesivir is metabolized to GS-441524, an adenosine analog that inhibits a broad range of polymerases and then evades exonuclease repair, causing chain termination <span class="citation" data-cites="YaxKcqq8 lIko9Imd IhdcvbzC">[<a href="#ref-YaxKcqq8" role="doc-biblioref">244</a>,<a href="#ref-lIko9Imd" role="doc-biblioref">245</a>,<a href="#ref-IhdcvbzC" role="doc-biblioref">246</a>]</span>.
Although it was developed against Ebola, it also inhibits polymerase and replication of the coronaviruses MERS-CoV and SARS-CoV in cell culture assays with submicromolar IC50s <span class="citation" data-cites="1H0NPTMw9">[<a href="#ref-1H0NPTMw9" role="doc-biblioref">247</a>]</span>.
It also inhibits SARS-CoV-2, showing synergy with chloroquine <em>in vitro</em> <span class="citation" data-cites="IhdcvbzC">[<a href="#ref-IhdcvbzC" role="doc-biblioref">246</a>]</span>.</p>
<p><strong>Current Evidence.</strong>
In addition to the previous work showing remdesivir to be an effective treatment for viral pathogens such as SARS-CoV and MERS-CoV in cultured cells and animal models, a recent study found that administration of remdesivir to non-human primate models resulted in 100% protection against infection by the Ebola virus.
Although a clinical trial in the Democratic Republic of Congo found some evidence of effectiveness against Ebola, two antibody preparations were found to be more effective, and remdesivir was not pursued <span class="citation" data-cites="gnfviuGp">[<a href="#ref-gnfviuGp" role="doc-biblioref">248</a>]</span>.
Remdesivir has also been reported to inhibit SARS-CoV-2 infection in a human cell line sensitive to the virus <span class="citation" data-cites="IhdcvbzC">[<a href="#ref-IhdcvbzC" role="doc-biblioref">246</a>]</span>.</p>
<p>The effectiveness of remdesivir for treating patients with COVID-19 is currently under investigation.
Remdesivir was first used on some COVID-19 patients under compassionate use guidelines <span class="citation" data-cites="D4HwvRMd zoLViMNg">[<a href="#ref-D4HwvRMd" role="doc-biblioref">249</a>,<a href="#ref-zoLViMNg" role="doc-biblioref">250</a>]</span>.
All were in late stages of COVID-19 infection, and these reports were inconclusive about the drug’s efficacy.
Gilead Sciences, the maker of remdesivir, led a recent publication that reported outcomes for compassionate use of the drug in 61 patients hospitalized with confirmed COVID-19.
Here, 200mg of remdesivir was administered intravenously on day 1, followed by a further 100mg/day for 9 days <span class="citation" data-cites="16zLDIOyk">[<a href="#ref-16zLDIOyk" role="doc-biblioref">243</a>]</span>.
There were significant issues with the study design or lack thereof.
There was no randomized control group.
The inclusion criteria were variable: some patients only required low doses of oxygen, others required ventilation.
The study included many sites, potentially with variable inclusion criteria and treatment protocols.
Patients analyzed had mixed demographics.
There was a short follow-up period of investigation.
Some patients worsened, some patients died, and eight were excluded from the analysis mainly due to missing post-baseline information, thus their health is unaccounted for.
Therefore, even though the study reported clinical improvement in 68% of the 53 patients ultimately evaluated, due to the significant issues with study design, it could not be determined whether treatment with remdesivir had an effect or whether these patients would have recovered regardless of treatment.
Another study comparing 5 and 10 day treatment regimens had similar results but was also limited because of the lack of a placebo control <span class="citation" data-cites="w5y1LWUR">[<a href="#ref-w5y1LWUR" role="doc-biblioref">252</a>]</span>.
The study did not alter our understanding of the efficacy of remdesivir in treating COVID-19, but the encouraging results motivated placebo controlled studies.</p>
<p>Remdesivir was later tested in a double-blind placebo-controlled phase 3 clinical trial performed at 60 trial sites, 45 of which were in the United states.
The trial recruited 1,063 patients and randomly assigned them to placebo treatment or treatment with remdesivir.
The treatment was 200 mg on day 1, followed by 100 mg on days 2 through 10.
Data was analyzed from 1,059 patients (538 assigned to remdesivir and 521 to placebo).
The two groups were well matched demographically and clinically at baseline.
Those who received remdesivir had a median recovery time of 11 days (95% confidence interval [CI], 9 to 12), as compared with 15 days (95% CI, 13 to 19) in those who received placebo (rate ratio for recovery, 1.32; 95% CI, 1.12 to 1.55; P&lt;0.001).
The Kaplan-Meier estimates of mortality by 14 days were 7.1% with remdesivir and 11.9% with placebo (hazard ratio for death, 0.70; 95% CI, 0.47 to 1.04).
Though mortality was lower in the remdesivir group, it was not significant.
Serious adverse events were reported for 114 of the 541 patients in the remdesivir group who underwent randomization (21.1%) and 141 of the 522 patients in the placebo group who underwent randomization (27.0%).
The median time to recovery in patients in the subgroup receiving invasive mechanical ventilation or extracorporeal membrane oxygenation (ECMO) could not be established which may indicate that the follow up time was too short for this group (272 patients).
Largely on the results of this trial, the FDA issued the Emergency Use Authorization (EUA) for remdesivir for the treatment of hospitalized COVID-19 patients.</p>
<p>As of July 2020, there are ten clinical trials underway using remdesivir to treat COVID-19 patients at both early and late stages of infection and in combinations with other drugs including <span class="citation" data-cites="IhdcvbzC oso9r21L 1q4PmqTm rUTsIlZs JSoOOX98">[<a href="#ref-IhdcvbzC" role="doc-biblioref">246</a>,<a href="#ref-oso9r21L" role="doc-biblioref">253</a>,<a href="#ref-1q4PmqTm" role="doc-biblioref">254</a>,<a href="#ref-rUTsIlZs" role="doc-biblioref">255</a>,<a href="#ref-JSoOOX98" role="doc-biblioref">256</a>]</span>.</p>
<p><strong>Summary.</strong>
Remdesivir is a first in class to receive FDA approval, currently as an Emergency Use Authorization.
It establishes proof of principle that drugs targeting the virus can benefit patients.
It also shows proof of principle that the virus can be targeted at the level of viral replication, since it targets the viral RNA polymerase at high potency.
Moreover, one of the most successful therapies for viral diseases is to target the viral replication machinery, which are typically virally encoded polymerases.
Small molecule drugs targeting viral polymerases are the backbones of treatments for other viral diseases including HIV and Herpes.
Note that the HIV and Herpes polymerases are a reverse transcriptase and a DNA polymerase respectively, whereas SARS-CoV-2 encodes an RNA dependent RNA polymerase, so most of the commonly used polymerase inhibitors are not likely to be active against SARS-CoV-2.
In clinical use, polymerase inhibitors show short term benefits for HIV patients but for long term benefits they must be part of combination regimens.
They are typically combined with protease inhibitors, integrase inhibitors and even other polymerase inhibitors.
Additional clinical trials of remdesivir on different patient pools and in combination with other therapies will refine its use in the clinic.</p>
<h4 data-number="4.2.1.2" id="protease-inhibitors"><span class="header-section-number">4.2.1.2</span> Protease Inhibitors</h4>
<p>Several studies showed that viral proteases play an important role in the life cycle of (corona)viruses by modulating the cleavage of viral polyprotein precursors <span class="citation" data-cites="26dpptLw">[<a href="#ref-26dpptLw" role="doc-biblioref">257</a>]</span>.
Several FDA-approved drugs target proteases, including lopinavir and ritonavir for HIV infection and simeprevir for hepatitis C virus infection.
In particular, serine protease inhibitors were suggested for the treatment of SARS and MERS viruses <span class="citation" data-cites="f7csoyEu">[<a href="#ref-f7csoyEu" role="doc-biblioref">258</a>]</span>.
Recently, a study <span class="citation" data-cites="15l3di3Wj">[<a href="#ref-15l3di3Wj" role="doc-biblioref">44</a>]</span> suggested that camostat mesylate, an FDA-approved protease inhibitor (PI) could block the entry of SARS-CoV-2 into lung cells in vitro.
However, to test the efficacy of PIs in patients, randomized clinical trials have to be conducted on patients and healthy volunteers.</p>
<h5 data-number="4.2.1.2.1" id="n3"><span class="header-section-number">4.2.1.2.1</span> N3</h5>
<p>N3 is an inhibitor of Mpro, a 33.8-kDa SARS-CoV-2 protease that is involved in viral replication and transcription.</p>
<p><strong>Anticipated Mechanism.</strong>
N3 inhibits Mpro through binding to its substrate pocket.</p>
<p><strong>Current Evidence.</strong>
N3 was first designed computationally <span class="citation" data-cites="15JYG0AIU">[<a href="#ref-15JYG0AIU" role="doc-biblioref">259</a>]</span> to bind in the substrate binding pocket of the Mpro protease of SARS-like coronaviruses <span class="citation" data-cites="XfOTPQPJ">[<a href="#ref-XfOTPQPJ" role="doc-biblioref">260</a>]</span>.
Subsequently, the structure of N3-bound SARS-CoV-2 Mpro was solved <span class="citation" data-cites="4Sja6dIz">[<a href="#ref-4Sja6dIz" role="doc-biblioref">261</a>]</span>, confirming the computational prediction.
Finally, N3 reduced the viral load in samples taken from patients.</p>
<p><strong>Summary.</strong>
N3 is a computationally designed molecule that inhibits the viral transcription through inhibiting Mpro.
Although N3 is a strong inhibitor of SARS-CoV-2 in vitro, its safety and efficacy have to be tested in healthy volunteers and patients.</p>
<h5 data-number="4.2.1.2.2" id="ebselen"><span class="header-section-number">4.2.1.2.2</span> Ebselen</h5>
<p>Ebselen identified as Mpro protease inhibitor.
It is currently investigated as an anti-oxidant drug <span class="citation" data-cites="17EErh01r">[<a href="#ref-17EErh01r" role="doc-biblioref">262</a>]</span>.</p>
<p><strong>Anticipated Mechanism.</strong>
Ebselen inhibits Mpro through binding to its substrate pocket.</p>
<p><strong>Current Evidence.</strong>
After the design and confirmation of N3 as a highly potent Michael acceptor inhibitor and the identification of Mpro structure <span class="citation" data-cites="4Sja6dIz 10NcfBSiT">[<a href="#ref-4Sja6dIz" role="doc-biblioref">261</a>,<a href="#ref-10NcfBSiT" role="doc-biblioref">263</a>]</span>, 10,000 compounds were screened for their in vitro anti-Mpro activity.
The six leads that were identified were Ebselen, Disulfiram, Tideglusib, Carmofour, PX-12.
When the compounds were further assayed on patient viral samples, Ebselen had the strongest potency in reducing the viral load.
However, the authors cautioned that these compounds are likely promiscuous binders, which would diminish their therapeutic potential.</p>
<p><strong>Summary.</strong>
Ebselen is both a strong Mpro inhibitor and strong inhibitor of viral replication in vitro.
The reduction of the viral load after exposure to Ebselen was even larger than N3.
Ebselen is a very promising compound since its safety has been demonstrated in other indications.
However, Ebselen is likely a false positive since it is a promiscuous compound that can have many targets <span class="citation" data-cites="vXFGEybh">[<a href="#ref-vXFGEybh" role="doc-biblioref">264</a>]</span>.
Therefore, compounds with higher specificity are required to effectively translate to clinical trials.</p>
<h4 data-number="4.2.1.3" id="molecules-targeting-the-viral-envelope"><span class="header-section-number">4.2.1.3</span> Molecules Targeting the Viral Envelope</h4>
<p>Why it may be useful</p>
<!--##### Viral Envelope Targeter 1

**Anticipated Mechanism.**

**Current Evidence.**

A list of current studies and their results, using carefully the information requested in the therapeutic paper tickets.

**Summary.**

Summarize the state of the antiviral approach.
-->
<h3 data-number="4.2.2" id="drugs-targeting-host-proteins"><span class="header-section-number">4.2.2</span> Drugs Targeting Host Proteins</h3>
<p>Brief background on the therapeutic.</p>
<h4 data-number="4.2.2.1" id="viral-entry-receptors"><span class="header-section-number">4.2.2.1</span> Viral Entry Receptors</h4>
<p>Entry of SARS-CoV-2 into the cell depends on the ACE2 receptor and the enzyme encoded by <em>TMPRSS2</em> <span class="citation" data-cites="15l3di3Wj">[<a href="#ref-15l3di3Wj" role="doc-biblioref">44</a>]</span>.
In principle, drugs that reduce the expression of these proteins or sterically hinder viral interactions with them might reduce viral entry into cells.</p>
<p><strong>Current Evidence.</strong>
Angiotensin-converting enzyme (ACE) inhibitors and angiotensin II receptor blockers (ARBs) are among the most commonly prescribed medications <span class="citation" data-cites="CIeWPFHG pbfxOvnK">[<a href="#ref-CIeWPFHG" role="doc-biblioref">265</a>,<a href="#ref-pbfxOvnK" role="doc-biblioref">266</a>]</span>.
In the United States, for example, they are prescribed well over 100,000,000 times annually.
Data from some animal models suggest several, but not all, ACE inhibitors and several ARBs increase ACE2 expression in the cells of some organs <span class="citation" data-cites="17xTGqKp2">[<a href="#ref-17xTGqKp2" role="doc-biblioref">267</a>]</span>.
Clinical studies have not established whether plasma ACE2 expression is increased in humans treated with these medications <span class="citation" data-cites="ZhhV4ztO">[<a href="#ref-ZhhV4ztO" role="doc-biblioref">268</a>]</span>.
While randomized clinical trials are ongoing, a variety of observational studies have examined the relationship between exposure to ACE inhibitors or ARBs and outcomes in patients with COVID-19.
An observational study of the association of ACE inhibitor or ARB exposure on outcomes in COVID-19 was retracted from the <em>New England Journal of Medicine</em> <span class="citation" data-cites="10q7vcaWw">[<a href="#ref-10q7vcaWw" role="doc-biblioref">269</a>]</span>.
Moreover, because observational studies are subject to confounding, randomized controlled trials are the standard means of assessing the effects of medications, and the findings of the various observational studies bearing on this topic cannot be interpreted as indicating a protective effect of the drug <span class="citation" data-cites="19quOF2kK k5xEN3Ke">[<a href="#ref-19quOF2kK" role="doc-biblioref">270</a>,<a href="#ref-k5xEN3Ke" role="doc-biblioref">271</a>]</span>.
Clinical trials testing the effects of ACE inhibitors or ARBs on COVID-19 outcomes are ongoing <span class="citation" data-cites="H0oEjZR8 kuUcA1G9 3ccJAyrG NCb9Wq2z 5fkIarR7 c7V9rAFR">[<a href="#ref-H0oEjZR8" role="doc-biblioref">272</a>,<a href="#ref-kuUcA1G9" role="doc-biblioref">273</a>,<a href="#ref-3ccJAyrG" role="doc-biblioref">274</a>,<a href="#ref-NCb9Wq2z" role="doc-biblioref">275</a>,<a href="#ref-5fkIarR7" role="doc-biblioref">276</a>,<a href="#ref-c7V9rAFR" role="doc-biblioref">277</a>]</span>.
These studies of randomized intervention will provide important data for understanding whether exposure to ACEis or ARBs is associated with COVID-19 outcomes.
Additional information about ACE2, observational studies of ACE inhibitors and ARBs in COVID-19, and clinical trials on this topic have been summarized <span class="citation" data-cites="1Hrsy8Hiq">[<a href="#ref-1Hrsy8Hiq" role="doc-biblioref">278</a>]</span>.</p>
<p><strong>Summary.</strong>
Summarize the state of the antiviral approach.</p>
<h3 data-number="4.2.3" id="broad-spectrum-pharmaceuticals"><span class="header-section-number">4.2.3</span> Broad-Spectrum Pharmaceuticals</h3>
<ul>
<li>Add some background on broad-spectrum pharmaceuticals</li>
</ul>
<h4 data-number="4.2.3.1" id="hydroxychloroquine-and-chloroquine"><span class="header-section-number">4.2.3.1</span> Hydroxychloroquine and Chloroquine</h4>
<p><strong>Potential Mechanism.</strong>
Chloroquine (CQ) and hydroxychloroquine (HCQ) are lysosomotropic agents, meaning they are weak bases that can pass through the plasma membrane.
Both drugs increase cellular pH by accumulating in their protonated form inside lysosomes <span class="citation" data-cites="ggYRK4FU 16IRfGFKb">[<a href="#ref-ggYRK4FU" role="doc-biblioref">279</a>,<a href="#ref-16IRfGFKb" role="doc-biblioref">280</a>]</span>.
This shift in pH inhibits the breakdown of proteins and peptides by the lysosomes during the process of proteolysis <span class="citation" data-cites="16IRfGFKb">[<a href="#ref-16IRfGFKb" role="doc-biblioref">280</a>]</span>.
A number of mechanisms have been proposed through which these drugs could influence the immune response to pathogen challenge.
For example, CQ/HCQ can interfere with digestion of antigens within the lysosome and inhibit CD4 T-cell stimulation while promoting the stimulation of CD8 T-cells <span class="citation" data-cites="16IRfGFKb">[<a href="#ref-16IRfGFKb" role="doc-biblioref">280</a>]</span>.
CQ/HCQ can also decrease the production of certain key cytokines involved in the immune response including IL-6 and inhibit the stimulation of toll-like receptors (TLR) and TLR signaling <span class="citation" data-cites="16IRfGFKb">[<a href="#ref-16IRfGFKb" role="doc-biblioref">280</a>]</span>.
The drugs also have anti-inflammatory and photoprotective effects and may also affect rates of cell death, blood clotting, glucose tolerance, and cholesterol levels <span class="citation" data-cites="16IRfGFKb">[<a href="#ref-16IRfGFKb" role="doc-biblioref">280</a>]</span>.</p>
<p>Interest in CQ and HCQ for treating COVID-19 was catalyzed by a mechanism observed in <em>in vitro</em> studies of both SARS-CoV and SARS-CoV-2.
In one study, CQ inhibited viral entry of SARS-CoV into Vero E6 cells, a cell line derived from the epithelial cells of an African green monkey kidney, through the elevation of endosomal pH and the terminal glycosylation of angiotensin-converting enzyme 2 (ACE2), which is the cellular entry receptor <span class="citation" data-cites="x6nInyCf">[<a href="#ref-x6nInyCf" role="doc-biblioref">281</a>]</span>.
Increased pH within the cell, as discussed above, inhibits proteolysis, and terminal glycosylation of ACE2 is thought to interfere with virus-receptor binding.
An <em>in vitro</em> study of SARS-CoV-2 infection of Vero cells, a line from which the Vero E6 clone has been separated since 1968, found both HCQ and CQ to be effective in inhibiting viral replication, with HCQ being more potent <span class="citation" data-cites="1GBsnkeLK">[<a href="#ref-1GBsnkeLK" role="doc-biblioref">282</a>]</span>.
Additionally, an early case study of three COVID-19 patients reported the presence of antiphospholipid antibodies in all three patients <span class="citation" data-cites="8OnbWuhF">[<a href="#ref-8OnbWuhF" role="doc-biblioref">70</a>]</span>.
Antiphospholipid antibodies are central to the diagnosis of the antiphospholipid syndrome, a disorder that HCQ has often been used to treat <span class="citation" data-cites="nEaG534R rL8owM5Y 134vCyU3E">[<a href="#ref-nEaG534R" role="doc-biblioref">283</a>,<a href="#ref-rL8owM5Y" role="doc-biblioref">284</a>,<a href="#ref-134vCyU3E" role="doc-biblioref">285</a>]</span>.
Together, these studies triggered initial enthusiasm about the therapeutic potential for HCQ and CQ against COVID-19.
HCQ/CQ has been proposed both as a treatment for COVID-19 and a prophylaxis against SARS-CoV-2 exposure.
Additionally, HCQ/CQ are sometimes administered with azithromycin (AZ) and/or zinc supplementation.
This section will discuss the current information available on the administration of CQ and HCQ as a treatment for or prophylaxis against COVID-19.</p>
<!--
rishirajgoel to add text on Azithromycin
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4262884/
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5918160/
https://www.ncbi.nlm.nih.gov/pubmed/28099856
https://www.ncbi.nlm.nih.gov/pubmed/27911847
-->
<p><strong>Controversy.</strong>
The initial study evaluating HCQ as a treatment for COVID-19 patients was published on March 20, 2020 by Gautret et al.
This non-randomized, non-blinded, non-placebo clinical trial compared HCQ to standard of care in 42 hospitalized patients in southern France <span class="citation" data-cites="ovHLMvCi">[<a href="#ref-ovHLMvCi" role="doc-biblioref">286</a>]</span>.
They reported that patients who received HCQ showed higher rates of virological clearance by nasopharyngeal swab on Days 3-6 when compared to standard care.
This study also treated six patients with both HCQ + AZ and found this combination therapy to be more effective than HCQ alone.
This study showed design and analysis weaknesses that severely limit interpretability of results, including the lack of randomization, lack of blinding, lack of placebo, lack of Intention-To-Treat analysis, lack of correction for sequential multiple comparisons, trial arms entirely confounded by hospital, false negatives in outcome measurements, lack of trial pre-registration, and small sample size.
Two of these weaknesses are due to inappropriate data analysis and can therefore be corrected post-hoc by recalculating p-values (lack of Intention-To-Treat analysis and multiple comparisons.)
However, all other weaknesses are fundamental design flaws and cannot be corrected for.
Thus, conclusions cannot be generalized outside of the study.
The International Society of Antimicrobial Chemotherapy, the scientific organization that publishes <em>International Journal of Antimicrobial Agents</em> where the article appeared, has announced that the article does not meet its expected standard for publications <span class="citation" data-cites="10J3hBiMo">[<a href="#ref-10J3hBiMo" role="doc-biblioref">287</a>]</span>, although it has not been officially retracted.
Because of the preliminary data presented in this study, the use of HCQ in COVID-19 treatment has subsequently been explored by other researchers.
About one week later, a follow-up case study reported that 11 consecutive patients were treated with HCQ + AZ using the same dosing regimen <span class="citation" data-cites="18KmvBOFr">[<a href="#ref-18KmvBOFr" role="doc-biblioref">288</a>]</span>.
One patient died, two were transferred to the ICU, and one developed a prolonged QT interval, leading to discontinuation of HCQ + AZ.
As in the Gautret et al. study, the outcome assessed was virological clearance at Day 6 post-treatment, as measured in nasopharyngeal swabs.
Of the ten living patients on Day 6, eight remained positive for SARS-CoV-2 RNA.
Like in the original study, interpretability was severely limited by lack of comparison group and the small sample size.
However, these results stand in contrast to the claims by Gautret et al. that all six patients treated with HCQ + AZ tested negative for SARS-CoV-2 RNA by Day 6 post-treatment.
This case study illustrated the need for further investigation using robust study design before the original proposed benefits of HCQ could be widely accepted.</p>
<p>On April 10, 2020, a randomized, non-placebo trial of 62 COVID-19 patients at the Renmin Hospital of Wuhan University was released <span class="citation" data-cites="wzHb7Mj5">[<a href="#ref-wzHb7Mj5" role="doc-biblioref">289</a>]</span>.
This study investigated whether HCQ decreased time to fever break or time to cough relief when compared to standard of care <span class="citation" data-cites="wzHb7Mj5">[<a href="#ref-wzHb7Mj5" role="doc-biblioref">289</a>]</span>.
This trial found HCQ decreased both average time to fever break and average time to cough relief, defined as mild or no cough.
While this study improved on some of the methodological flaws in Gautret et al. by randomizing patients, it also had several flaws in trial design and data analysis that prevent generalization of the results.
These weaknesses include the lack of placebo, lack of correction for multiple primary outcomes, inappropriate choice of outcomes, lack of sufficient detail to understand analysis, drastic disparities between pre-registration and published protocol, and small sample size.
The choice of outcomes may be inappropriate as both fevers and cough may break periodically without resolution of illness.
Additionally, for these outcomes, the authors reported that 23 of 62 patients did not have a fever and 25 of 62 patients did not have a cough at the start of the study, but the authors failed to describe how these patients were included in a study assessing time to fever break and time to cough relief.
It is important to note here that the authors claimed “neither the research performers nor the patients were aware of the treatment assignments.”
This blinding seems impossible in a non-placebo trial because at the very least, providers would know whether they were administering a medication or not, and this knowledge could lead to systematic differences in the administration of care.
Correction for multiple primary outcomes can be adjusted post-hoc by recalculating p-values, but all of the other issues were design and statistical weaknesses that cannot be corrected for.
Additionally, the observation of drastic disparities between pre-registration and published protocol could indicate p-hacking.
The design limitations mean that the conclusions cannot be generalized outside of the study.
A second randomized trial, conducted by the Shanghai Public Health Clinical Center, analyzed whether HCQ increased rates of virological clearance at day 7 in respiratory pharyngeal swabs compared to standard care <span class="citation" data-cites="o5gwQA7W">[<a href="#ref-o5gwQA7W" role="doc-biblioref">290</a>]</span>.
This trial was published in Chinese along with an abstract in English, and only the English abstract was read and interpreted for this review.
The trial found comparable outcomes in virological clearance rate, time to virological clearance, and time to body temperature normalization between the treatment and control groups.
A known weakness is small sample size, with only 30 patients enrolled and 15 in each arm.
This problem suggests the study is underpowered to detect potentially useful differences and precludes interpretation of results.
Additionally, because only the abstract could be read, other design and analysis issues could be present.
Thus, though these studies added randomization to their assessment of HCQ, their conclusions should be interpreted very cautiously.
These two studies assessed different outcomes and reached differing conclusions about the efficacy of HCQ for treating COVID-19; the designs of both studies, especially with respect to sample size, meant that no general conclusions can be made about the efficacy of the drug.</p>
<p>Several widely reported studies on HCQ have issues with data integrity and/or provenance.
A Letter to the Editor published in <em>BioScience Trends</em> on March 16, 2020 claimed that numerous clinical trials have shown that HCQ is superior to control treatment in inhibiting the exacerbation of COVID-19 pneumonia <span class="citation" data-cites="aiItPxSY">[<a href="#ref-aiItPxSY" role="doc-biblioref">291</a>]</span>.
This letter has been cited by numerous primary literature, review articles, and media alike <span class="citation" data-cites="1BdQKtXnY z8XnLYWM">[<a href="#ref-1BdQKtXnY" role="doc-biblioref">292</a>,<a href="#ref-z8XnLYWM" role="doc-biblioref">293</a>]</span>.
However, the letter referred to 15 pre-registration identifiers from the Chinese Clinical Trial Registry.
When these identifiers are followed back to the registry, most trials claim they are not yet recruiting patients or are currently recruiting patients.
For all of these 15 identifiers, no data uploads or links to publications could be located on the pre-registrations.
At the very least, the lack of availability of the primary data means the claim that HCQ is efficacious against COVID-19 pneumonia cannot be verified.
Similarly, a recent multinational registry analysis <span class="citation" data-cites="1BApBNHvd">[<a href="#ref-1BApBNHvd" role="doc-biblioref">294</a>]</span> analyzed the efficacy of CQ and HCQ with and without a macrolide, which is a class of antibiotics that includes Azithromycin, for the treatment of COVID-19.
The study observed 96,032 patients split into a control and four treatment conditions (CQ with and without a macrolide; HCQ with and without a macrolide).
They concluded that treatment with CQ and HCQ was associated with increased risk of <em>de novo</em> ventricular arrhythmia during hospitalization.
However, this study has since been retracted by <em>The Lancet</em> due to an inability to validate the data used <span class="citation" data-cites="11FsxWalS">[<a href="#ref-11FsxWalS" role="doc-biblioref">295</a>]</span>.
These studies demonstrate that increased skepticism in evaluation of the HCQ/CQ and COVID-19 literature may be warranted, possible because of the significant attention HCQ and CQ have received as possible treatments for COVID-19 and the politicization of these drugs.</p>
<p>Despite the fact that the study suggesting that CQ/HCQ increased risk of ventricular arrhythmia in COVID-19 patients has now been retracted, previous studies have identified risks with HCQ/CQ.
A patient with systemic lupus erythematosus developed a prolonged QT interval that was likely exacerbated by use of HCQ in combination with renal failure <span class="citation" data-cites="dY9kXLAS">[<a href="#ref-dY9kXLAS" role="doc-biblioref">296</a>]</span>.
A prolonged QT interval is associated with ventricular arrhythmia <span class="citation" data-cites="wy3S7Cpa">[<a href="#ref-wy3S7Cpa" role="doc-biblioref">297</a>]</span>.
Furthermore, a separate study <span class="citation" data-cites="OQ0b31BA">[<a href="#ref-OQ0b31BA" role="doc-biblioref">298</a>]</span> investigated the safety associated with the use of HCQ with and without macrolides between 2000 and 2020.
The study involved 900,000 cases treated with HCQ and 300,000 cases treated with HCQ + AZ.
The results indicated that short-term use of HCQ was not associated with additional risk, but that HCQ + AZ was associated with an enhanced risk of cardiovascular complications (15-20% increased risk of chest pain) and a two-fold increased risk of mortality.
Therefore, whether studies utilize HCQ alone or HCQ in combination with a macrolide may be an important consideration in assessing risk.
As results from initial investigations of these drug combinations have emerged, concerns about the efficacy and risks of treating COVID-19 with HCQ and CQ has led to the removal of CQ/HCQ from standard of care practices in several countries <span class="citation" data-cites="1CgZR55GZ 1EcscrT32">[<a href="#ref-1CgZR55GZ" role="doc-biblioref">299</a>,<a href="#ref-1EcscrT32" role="doc-biblioref">300</a>]</span>.
As of May 25, 2020, WHO had suspended administration of HCQ as part of the worldwide Solidarity Trial <span class="citation" data-cites="AMQ5402v">[<a href="#ref-AMQ5402v" role="doc-biblioref">301</a>]</span>.</p>
<p><strong>Current Evidence.</strong>
As additional research has emerged, it remains unclear whether HCQ and/or CQ affect COVID-19 outcomes.
A randomized, open-label, non-placebo trial of 150 COVID-19 patients was conducted in parallel at 16 government-designated COVID-19 centers in China to assess the safety and efficacy of HCQ <span class="citation" data-cites="RXr9BKKT">[<a href="#ref-RXr9BKKT" role="doc-biblioref">302</a>]</span>.
The trial compared treatment with HCQ in conjunction with standard of care (SOC) to SOC alone in 150 infected patients who were assigned randomly to the two groups (75 per group).
The primary endpoint of the study was the negative conversion rate of SARS-CoV-2 in 28 days, and the investigators found no difference in this parameter between the groups.
The secondary endpoints were an amelioration of the symptoms of the disease such as axillary temperature ≤36.6°C, SpO2 &gt;94% on room air, and disappearance of symptoms like shortness of breath, cough, and sore throat.
The median time to symptom alleviation was similar across different conditions (19 days in HCQ+SOC vs. 21 days in SOC).
However, the investigators reported an interesting finding revealed in <em>post hoc</em> analysis: controlling for the administration of antivirals yielded an effect of HCQ on the alleviation of symptoms, with a reported hazard ratio of 8.83, although the 95% confidence interval of 1.09 to 71.3 suggests that this analysis may be underpowered and should be interpreted cautiously, as only 28 patients total (14 in each group) met this criterion.
Additionally, there was a non-significant trend towards a greater number of lymphocytes in the SOC+HCQ group compared to the SOC-alone group (mean of 0.062 versus 0.008; p=0.57).
Given the improvement in CRP levels and the possible improvement in lymphocyte count, the authors hypothesized that the addition of HCQ to the current SOC could decrease the inflammatory response, which could help to prevent multiorgan failure and death.
However, one of the key results of this trial was that the 30% of the patients receiving SOC+HCQ reported adverse outcomes compared to 8.8% of patients receiving only SOC.
The most common adverse outcome in the SOC+HCQ group was diarrhea (10% vs. 0% in the SOC group, p=0.004).</p>
<p>The adverse effects of administering HCQ to mild and moderate COVID-19 cases were found to be manageable.
Furthermore, there are several factors that limit the interpretability of this study.
Most of the enrolled patients had mild-to-moderate symptoms (98%), and the average age was 46.
Although the authors claimed that HCQ could be beneficial in hindering disease progression, multiorgan failure, and death in COVID-19, this finding cannot be extrapolated to older patients, who are known to be at higher risk, or to severe cases.
Additionally, a larger sample size is needed to validate whether lymphocyte counts increase in patients treated with SOC+HCQ; although the result in the present analysis appeared promising, it lacked statistical significance.
Likewise, in this study, SOC included the use of antivirals (Lopinavir-Ritonavir, Arbidol, Oseltamivir, Virazole, Entecavir, Ganciclovir, and Interferon alfa), which appeared to introduce confounding effects.
Thus, to isolate the effect of HCQ, SOC would need to exclude the use of antivirals.
In this trial, the samples used to test for the presence of the SARS-CoV-2 virus were collected from the upper respiratory tract, and the authors indicated that the use of upper respiratory samples may have introduced false negatives (e.g., <span class="citation" data-cites="vr7AH83b">[<a href="#ref-vr7AH83b" role="doc-biblioref">121</a>]</span>); thus, the identification of biomarkers that can be collected non-invasively would be valuable to studies such as this one.
Another limitation of the study that the authors acknowledge was that the HCQ treatment began, on average, at a 16-day delay from the symptom onset.
An investigation of earlier intervention with HCQ would help to elucidate how the drug affects early disease management.
Overall, the study provides promising data, although all of the findings still need to be validated in independent population cohorts.
The fact that this study was open-label and lacked a placebo limits interpretation, and additional analysis is required to determine whether HCQ reduces inflammatory response.</p>
<p>Additional evidence comes from a retrospective analysis <span class="citation" data-cites="MENbRfOl">[<a href="#ref-MENbRfOl" role="doc-biblioref">303</a>]</span> that examined data from 368 COVID-19 patients across all United States Veteran Health Administration medical centers.
The study retrospectively investigated the effect of the administration of HCQ (n=97), HCQ + AZ (n=113), and no HCQ (n=158) on 368 patients.
The primary outcomes assessed were death and the need for mechanical ventilation.
Standard supportive care was rendered to all patients.
Due to the low representation of women (N=17) in the available data, the study included only men, and the median age was 65 years.
The rate of death in the HCQ-only treatment condition was 27.8% and in the HCQ + AZ treatment condition, it was 22.1%.
In comparison to the 14.1% rate of death in the no-HCQ cohort, these data indicated a statistically significant elevation in the risk of death for the HCQ-only group compared to the no-HCQ group (adjusted hazard ratio: 2.61, p=0.03), but not for the HCQ + AZ group compared to the no-HCQ group (adjusted hazard ratio: 1.14; p=0.72).
Further, the risk of ventilation was similar across all three groups (adjusted hazard ratio: 1.43, p=0.48 (HCQ) and 0.43, p=0.09 (HCQ + AZ) compared to no HCQ).
The study thus showed evidence of an association between increased mortality and HCQ in this cohort of COVID-19 patients but no change in rates of mechanical ventilation among the treatment conditions.
The study had a few limitations: it was not randomized, and the baseline vital signs, laboratory tests, and prescription drug use were significantly different among the three groups.
All of these factors could potentially influence treatment outcome.
Furthermore, the authors acknowledge that the effect of the drugs might be different in females and pediatric subjects, since these subjects were not part of the study.
The reported result that HCQ + AZ is safer than HCQ contradicts the findings of the previous large-scale analysis of twenty years of records that found HCQ + AZ to be more frequently associated with cardiac arrhythmia than HCQ alone <span class="citation" data-cites="OQ0b31BA">[<a href="#ref-OQ0b31BA" role="doc-biblioref">298</a>]</span>; whether this discrepancy is caused by the pathology of COVID-19, is influenced by age or sex, or is a statistical artifact is not presently known.</p>
<p>Thus, both of the above studies have significant limitations but analyzed data acquired broadly from centers within a country’s health system in China and the United States, respectively.
Neither study reported robust improvements in patients treated with HCQ, although the Tang et al. study identified potential improvements associated with HCQ in <em>post hoc</em> analysis.
In contrast, the VA study found that HCQ was associated with increased risk of mortality in COVID-19 patients.
These two studies examined different populations, especially since age and sex have both been found to influence the severity of COVID-19.
Additional analyses to increase sample size and encompassing a variety of ages and sexes could help to resolve the potential benefits and risks of HCQ administration to COVID-19 patients.
A randomized, blinded study with a placebo would serve to overcome many of the limitations in the design of these studies.</p>
<p>One study did utilize a randomized, double-blind, placebo-controlled design, although this study analyzed the administration of HCQ prophylactically rather than therapeutically <span class="citation" data-cites="NiVdIN7Z">[<a href="#ref-NiVdIN7Z" role="doc-biblioref">304</a>]</span>.
Asymptomatic adults in the United States and Canada who had been exposed to SARS-CoV-2 within the past four days were enrolled in an online study to see whether administration of HCQ over five days would influence the probability of developing COVID-19 symptoms over a 14-day period.
Of the participants, 414 received HCQ and 407 received a placebo.
Other than location, the demographics of this study were more comparable to the Yang et al. study above, as 51.6% of participants were women and the median age was 40 years.
They found no significant difference in the rate of symptomatic illness between the two groups (11.8% HCQ, 14.3% placebo, p=0.35).
The HCQ condition was associated with side effects, with 40.1% of patients reporting side effects compared to 16.8% in the control group (p&lt;0.001).
However, likely due to the high enrollment of healthcare workers (66% of participants) and the well-known side effects associated with HCQ, a large number of participants were able to correctly identify whether they were receiving HCQ or a placebo (46.5% and 35.7%, respectively).
Furthermore, due to a lack of availability of diagnostic testing, only 20 of the 107 cases were confirmed with a PCR-based test to be positive for SARS-CoV-2.
The rest were categorized as “probable” or “possible” cases by a panel of four physicians who were blind to the treatment status.
However, a patient presenting one or more symptoms, which included diarrhea, was defined as a “possible” case, but diarrhea is also a common side effect of HCQ.
Additionally, four of the twenty PCR-confirmed cases did not develop symptoms until after the observation period had completed, suggesting that the 14-day trial period may not have been long enough or the initial exposure may not have accounted for secondary exposure, given that most of the patients were exposed through situations that were likely to be recurrent: by a close family member with a COVID-19 diagnosis or through their work in healthcare.
Finally, in addition to the young age of the participants in this study, which ranged from 32 to 51, there were possible impediments to generalization introduced by the selection process, as the 2,237 patients who were eligible but had already developed symptoms by day 4 were enrolled in a separate study.
It is therefore likely that asymptomatic cases were over-represented in this sample, which would not have been detected based on the diagnostic criteria used.
Therefore, while this study does represent the first effort to conduct a randomized, double-blind, placebo-controlled investigation of HCQ’s effect on COVID-19 symptoms in a large sample, the lack of PCR tests significantly impedes interpretation of the results.</p>
<p><strong>Summary.</strong>
<em>In vitro</em> evidence suggests that HCQ may be an effective therapeutic against SARS-CoV-2 and COVID-19.
Because the 90% effective concentration (EC<sub>90</sub>) of CQ in Vero E6 cells (6.90 μM) can be achieved in and tolerated by rheumatoid arthritis patients, it was hypothesized that it might also be possible to achieve the effective concentration in COVID-19 patients <span class="citation" data-cites="WFGcbZhe">[<a href="#ref-WFGcbZhe" role="doc-biblioref">305</a>]</span>.
Additionally, HCQ has been found to be effective in treating HIV <span class="citation" data-cites="hFGjQQN8">[<a href="#ref-hFGjQQN8" role="doc-biblioref">306</a>]</span> and chronic Hepatitis C <span class="citation" data-cites="rQNzKxRd">[<a href="#ref-rQNzKxRd" role="doc-biblioref">307</a>]</span> patients.
Therefore, initially it was thought that CQ/HCQ could be effective against SARS-COV-2.
CQ and HCQ have both been found to inhibit the expression of CD154 in T-cells and to reduce toll-like receptor signaling that leads to the production of pro-inflammatory cytokines <span class="citation" data-cites="j3rGUH1f">[<a href="#ref-j3rGUH1f" role="doc-biblioref">308</a>]</span>.
Clinical trials for COVID-19 have more often used HCQ rather than CQ because it offers the advantages of being cheaper and having fewer side effects than CQ.
Several countries have removed CQ from their standard of care for COVID-19 due to the frequency of adverse effects.
However, a large analysis of patients receiving HCQ from January 2000 through March 2020 reported that the combination of HCQ and azithromycin, but not other macrolides, was associated with an elevated risk of cardiovascular complications and mortality.
Multiple clinical studies have already been carried out to assess HCQ as a therapeutic agent for COVID-19, and many more are in progress.
To date, none of these studies have used randomized, double-blind, placebo-controlled designs with a large sample size, which would be the gold standard.
The one exception was an analysis of the prophylactic potential of HCQ <span class="citation" data-cites="NiVdIN7Z">[<a href="#ref-NiVdIN7Z" role="doc-biblioref">304</a>]</span>, which was designed to meet all these criteria, but masking was not adequately maintained during the study and the outcomes were not accurately assessed due to the lack of availability of PCR-based testing in the United States and Canada.
Thus, interpretation is severely limited and must be done cautiously.
Due to inconsistency in the outcomes assessed and results reported from study to study, it remains unclear whether HCQ, or HCQ + AZ as a combination therapy, holds any therapeutic or prophylactic value against COVID-19.
Additionally, one study identified an increased risk of mortality in older men receiving HCQ, and administration of HCQ and HCQ+AZ did not decrease the use of mechanical ventilation in these patients <span class="citation" data-cites="MENbRfOl">[<a href="#ref-MENbRfOl" role="doc-biblioref">303</a>]</span>.
HCQ use for COVID-19 also leads to shortages for anti-malarial or anti-rheumatic use, where it has been definitively proven to be effective.
Further investigation of HCQ in large, rigorous, multi-center clinical trials encompassing different sex and ages is required.</p>
<h3 data-number="4.2.4" id="nutraceuticals"><span class="header-section-number">4.2.4</span> Nutraceuticals</h3>
<p>Considering the current pandemic, scientists and the medical community are scrambling to repurpose or discover novel host-directed therapies for which nutraceuticals hold some promise.
Nutraceuticals are classified as supplements with health benefits beyond their basic nutritional value <span class="citation" data-cites="xLPeszeO 2Us0gNMr">[<a href="#ref-xLPeszeO" role="doc-biblioref">309</a>,<a href="#ref-2Us0gNMr" role="doc-biblioref">310</a>]</span>.
The key difference between a dietary supplement and a nutraceutical is that nutraceuticals should not only supplement the diet, but they must also aid in the prophylaxis and/or treatment of a disorder or disease <span class="citation" data-cites="X0jEogv4">[<a href="#ref-X0jEogv4" role="doc-biblioref">311</a>]</span>.
Unlike pharmaceuticals, nutraceuticals do not fall under the responsibility of the FDA, but they are monitored as dietary supplements according to the Dietary Supplement, Health and Education Act 1994 (DSHEA) <span class="citation" data-cites="10Tnnl6nF">[<a href="#ref-10Tnnl6nF" role="doc-biblioref">312</a>]</span> and the Drug Administration Modernization Act 1997 <span class="citation" data-cites="16eEQf0HM">[<a href="#ref-16eEQf0HM" role="doc-biblioref">313</a>]</span>.
However, there is significant concern that these acts do not adequately protect the consumer as they ascribe responsibility on the manufacturers to ensure safety of the product before manufacturing or marketing <span class="citation" data-cites="5TmtJWVQ">[<a href="#ref-5TmtJWVQ" role="doc-biblioref">314</a>]</span>.
Likewise, manufacturers are not required to register or seek approval from the FDA to produce or sell food supplements or nutraceuticals.
Health or nutrient content claims for labeling purposes are approved based on an authoritative statement from the Academy of Sciences or relevant federal authorities once the FDA has been notified and on the basis that the information is known to be true and not deceptive <span class="citation" data-cites="5TmtJWVQ">[<a href="#ref-5TmtJWVQ" role="doc-biblioref">314</a>]</span>.
In Europe, health claims are only permitted on a product label following authorization according to the European Food Safety Authority (EFSA) guidelines on nutrition and health claims <span class="citation" data-cites="dzOveIQY">[<a href="#ref-dzOveIQY" role="doc-biblioref">315</a>]</span>.
EFSA does not currently distinguish between food supplements and nutraceuticals for health claim applications of new products as claim authorization is dependent on the availability of clinical data in order to substantiate efficacy <span class="citation" data-cites="MhvDTmeu">[<a href="#ref-MhvDTmeu" role="doc-biblioref">316</a>]</span>.
These guidelines seem to provide more protection to consumers.
Currently, there is a debate among scientists and regulatory authorities to develop a regulatory framework in order to deal with the challenges of safety and health claim substantiation for nutraceuticals <span class="citation" data-cites="MhvDTmeu 5TmtJWVQ">[<a href="#ref-5TmtJWVQ" role="doc-biblioref">314</a>,<a href="#ref-MhvDTmeu" role="doc-biblioref">316</a>]</span>.</p>
<p>Nutraceuticals purported to boost the immune response, reduce immunopathology, exhibit antiviral activities or prevent acute respiratory distress syndrome (ARDS) are being considered for their potential therapeutic value <span class="citation" data-cites="GT7yDBjv">[<a href="#ref-GT7yDBjv" role="doc-biblioref">220</a>]</span>.
A host of potential candidates have been highlighted in the literature that target various aspects of the COVID-19 viral pathology, while others are thought to prime the host immune system.
These candidates include vitamins and minerals along with extracts and omega-3 polyunsaturated fatty acids (n-3 PUFA) <span class="citation" data-cites="2hMFPQJR">[<a href="#ref-2hMFPQJR" role="doc-biblioref">317</a>]</span>.
Considerable evidence <em>in vitro</em> and <em>in vivo</em> suggests that nutraceuticals containing phycocyanobilin, N-acetylcysteine, glucosamine, selenium or phase 2 inductive nutraceuticals (e.g. ferulic acid, lipoic acid, or sulforaphane) can prevent or modulate RNA virus infections via amplification of the signaling activity of mitochondrial antiviral-signaling protein (MAVS) and activation of toll-like receptor 7 (TLR7) <span class="citation" data-cites="eyBAUUCl">[<a href="#ref-eyBAUUCl" role="doc-biblioref">219</a>]</span>.
While promising, further animal and human studies are required to assess the therapeutic potential of these various nutraceuticals against COVID-19.</p>
<h4 data-number="4.2.4.1" id="n-3-pufa"><span class="header-section-number">4.2.4.1</span> n-3 PUFA</h4>
<p>Another potential nutraceutical that has exhibited beneficial effects against various viral infections is n-3 PUFA <span class="citation" data-cites="2hMFPQJR">[<a href="#ref-2hMFPQJR" role="doc-biblioref">317</a>]</span>, such as eicosapentaenoic acid (EPA) and docosahexaenoic acid (DHA).
EPA and DHA intake can come from a diet high in fish or through dietary supplementation with fish oils or purified oils <span class="citation" data-cites="Ufvsmc3N">[<a href="#ref-Ufvsmc3N" role="doc-biblioref">318</a>]</span>.</p>
<p><strong>Potential Mechanisms.</strong>
n-3 PUFA nutraceuticals can mediate inflammation, and therefore they may have the capacity to modulate the adaptive immune response <span class="citation" data-cites="JNyhLAfe Ufvsmc3N 2Us0gNMr">[<a href="#ref-2Us0gNMr" role="doc-biblioref">310</a>,<a href="#ref-Ufvsmc3N" role="doc-biblioref">318</a>,<a href="#ref-JNyhLAfe" role="doc-biblioref">319</a>]</span>.
Another potential mechanism through which n-3 PUFA could exert beneficial effects against viral infections is by acting as precursor molecules for the biosynthesis of endogenous specialized proresolving mediators (SPM), such as protectins and resolvins, that actively resolve inflammation and infection <span class="citation" data-cites="BIVORG2X">[<a href="#ref-BIVORG2X" role="doc-biblioref">320</a>]</span>.
Finally, some COVID-19 patients, particularly those with co-morbidities, are at a significant risk of thrombotic complications including arterial and venous thrombosis <span class="citation" data-cites="17jegDTvF PyKMLraw">[<a href="#ref-PyKMLraw" role="doc-biblioref">75</a>,<a href="#ref-17jegDTvF" role="doc-biblioref">321</a>]</span>.
Therefore, the use of prophylactic and therapeutic anticoagulants and antithrombotic agents is under consideration <span class="citation" data-cites="SxPCbOB1">[<a href="#ref-SxPCbOB1" role="doc-biblioref">322</a>]</span>.</p>
<p><strong>Current Evidence.</strong>
SPM have exhibited beneficial effects against a variety of lung infections including RNA viruses <span class="citation" data-cites="uPOTvvjR">[<a href="#ref-uPOTvvjR" role="doc-biblioref">323</a>]</span>.
Indeed, protectin D1 has been shown to increase survival from H1N1 viral infection in mice by affecting the viral replication machinery <span class="citation" data-cites="fIo3ESPM">[<a href="#ref-fIo3ESPM" role="doc-biblioref">324</a>]</span>.
Several mechanisms for SPM have been proposed, including preventing the release of pro-inflammatory cytokines and chemokines or increasing phagocytosis of cellular debris by macrophages <span class="citation" data-cites="OSEnHt6H">[<a href="#ref-OSEnHt6H" role="doc-biblioref">325</a>]</span>.
In influenza, SPM promote antiviral B lymphocytic activities <span class="citation" data-cites="MK0GQ55y">[<a href="#ref-MK0GQ55y" role="doc-biblioref">326</a>]</span>, and protectin D1 has been shown to increase survival from H1N1 viral infection in mice by affecting the viral replication machinery <span class="citation" data-cites="fIo3ESPM">[<a href="#ref-fIo3ESPM" role="doc-biblioref">324</a>]</span>.
It is hypothesized that SPM may aid in the resolution of the cytokine storm and pulmonary inflammation associated with COVID-19 <span class="citation" data-cites="14LjKv9ZH">[<a href="#ref-14LjKv9ZH" role="doc-biblioref">327</a>]</span>.
However, not all studies are in agreement that n-3 PUFA are effective against infections <span class="citation" data-cites="B8toe6Mr">[<a href="#ref-B8toe6Mr" role="doc-biblioref">328</a>]</span>.
The effectiveness of n-3 PUFA against infections would be dependent on the dosage, timing, and the specific pathogens responsible <span class="citation" data-cites="3Xht24k3">[<a href="#ref-3Xht24k3" role="doc-biblioref">329</a>]</span>.
On another level, there is still the question of whether fish oils can raise the levels of SPM levels upon ingestion and in response to acute inflammation in humans <span class="citation" data-cites="Ge00xeR3">[<a href="#ref-Ge00xeR3" role="doc-biblioref">330</a>]</span>.</p>
<p>The increased risk of thrombotic complications in COVID-19 infected patients was reported relatively late in comparison to other COVID-19 manifestations <span class="citation" data-cites="3UBbDrG2 17jegDTvF">[<a href="#ref-3UBbDrG2" role="doc-biblioref">69</a>,<a href="#ref-17jegDTvF" role="doc-biblioref">321</a>]</span>.
Considering that there is significant evidence that n-3 fatty acids and other fish oil-derived lipids possess antithrombotic properties and anti-inflammatory properties <span class="citation" data-cites="umzPtQG7 V9MDMfpj">[<a href="#ref-umzPtQG7" role="doc-biblioref">331</a>,<a href="#ref-V9MDMfpj" role="doc-biblioref">332</a>]</span>, they may have therapeutic value against the prothrombotic complications of COVID-19.
Based on concern among the medical community that the use of investigational therapeutics on COVID-19 patients already on antiplatelet therapies due to a pre-existing comorbidities may lead to issues with dosing and drug choice, and/or negative drug-drug interactions <span class="citation" data-cites="SxPCbOB1">[<a href="#ref-SxPCbOB1" role="doc-biblioref">322</a>]</span>, this supplementation may be particularly important for patients already receiving pharmaceutical antiplatelet therapies who may become infected.
As a result, the use of other therapeutics such as dietary sources of n-3 fatty acids or nutraceuticals with antiplatelet activities may be beneficial and warrant further investigation.
A new clinical trial <span class="citation" data-cites="QKHdIx2w">[<a href="#ref-QKHdIx2w" role="doc-biblioref">333</a>]</span> is currently recruiting COVID-19 positive patients to investigate the anti-inflammatory activity of a recently developed, highly purified derivative of EPA known as icosapent ethyl (Vascepa <sup>TM</sup>) <span class="citation" data-cites="12Q4bY0vF">[<a href="#ref-12Q4bY0vF" role="doc-biblioref">334</a>]</span>.
Other randomized controlled trials are in the preparatory stages with the intention of investigating the administration of EPA and other bioactive compounds to COVID-19 positive patients to determine whether anti-inflammatory effects or disease state improvements are observed <span class="citation" data-cites="8lPjdnE7 1AGVAbQuf">[<a href="#ref-8lPjdnE7" role="doc-biblioref">335</a>,<a href="#ref-1AGVAbQuf" role="doc-biblioref">336</a>]</span>.
Finally, while there have been studies investigating the therapeutic value of n-3 fatty acids against ARDS in humans, there is still limited evidence of their effectiveness <span class="citation" data-cites="XHtzuR5c">[<a href="#ref-XHtzuR5c" role="doc-biblioref">337</a>]</span>.</p>
<p><strong>Summary.</strong>
The overall lack of human studies in this area means there is limited evidence as to whether these nutraceuticals could affect COVID-19 infection.
Consequently, clinical trials that have been proposed and are in the preparatory stages will investigate the anti-inflammatory potential of n-3 PUFA and their derivatives for the treatment of COVID-19.</p>
<h4 data-number="4.2.4.2" id="zinc"><span class="header-section-number">4.2.4.2</span> Zinc</h4>
<p>There is evidence that nutrient supplements may exhibit some benefit against RNA viral infections.
Zinc is a trace metal obtained from dietary sources or supplementation that is important for the maintenance of immune cells involved in adaptive and innate immunity <span class="citation" data-cites="VMI1lOFR">[<a href="#ref-VMI1lOFR" role="doc-biblioref">338</a>]</span>.
Zinc supplements can be administered orally as a tablet or as a lozenge and they are available in many forms, such as zinc picolinate, zinc acetate, and zinc citrate.
Zinc is also available from dietary sources including meat, seafood, nuts, seeds, legumes, and dairy.</p>
<p><strong>Potential Mechanisms.</strong>
The role of zinc in immune function has been extensively reviewed <span class="citation" data-cites="VMI1lOFR">[<a href="#ref-VMI1lOFR" role="doc-biblioref">338</a>]</span>.
Zinc is an important signaling molecule and zinc levels can alter host defense systems.
In inflammatory situations such as an infection, zinc can regulate leukocyte immune responses and it can activate the nuclear factor kappa-light-chain-enhancer of activated B cells (NF-kB), thus altering cytokine production <span class="citation" data-cites="Qif5U0Ig 1CEzU1G34">[<a href="#ref-Qif5U0Ig" role="doc-biblioref">339</a>,<a href="#ref-1CEzU1G34" role="doc-biblioref">340</a>]</span>.
In particular, zinc supplementation can increase natural killer cell levels, which are important cells for host defense against viral infections <span class="citation" data-cites="7ElRN12v VMI1lOFR">[<a href="#ref-VMI1lOFR" role="doc-biblioref">338</a>,<a href="#ref-7ElRN12v" role="doc-biblioref">341</a>]</span>.</p>
<p><strong>Current Evidence.</strong>
Adequate zinc intake has been associated with reduced incidence of infection <span class="citation" data-cites="kxuSwrCI">[<a href="#ref-kxuSwrCI" role="doc-biblioref">342</a>]</span> and antiviral immunity <span class="citation" data-cites="IAOnO0Gj">[<a href="#ref-IAOnO0Gj" role="doc-biblioref">343</a>]</span>.
Similarly, a randomized, double-blind, placebo-controlled trial that administered zinc supplementation to elderly subjects over the course of a year found zinc deficiency to be associated with increased susceptibility to infection and that zinc deficiency could be prevented through supplementation <span class="citation" data-cites="kxuSwrCI">[<a href="#ref-kxuSwrCI" role="doc-biblioref">342</a>]</span>.
Clinical trial data supports the utility of zinc to diminish the duration and severity of symptoms associated with common colds when it is provided within 24 hours of the onset of symptoms <span class="citation" data-cites="sgr0mlJK lPexOhy6">[<a href="#ref-sgr0mlJK" role="doc-biblioref">344</a>,<a href="#ref-lPexOhy6" role="doc-biblioref">345</a>]</span>.
In coronaviruses specifically, in vitro evidence demonstrates that the combination of zinc (Zn2+) and zinc ionophores (pyrithione) can interrupt the replication mechanisms of SARS-CoV-GFP (a fluorescently tagged SARS-CoV) and a variety of other RNA viruses <span class="citation" data-cites="6aOUJdHa NB1udFHF">[<a href="#ref-6aOUJdHa" role="doc-biblioref">346</a>,<a href="#ref-NB1udFHF" role="doc-biblioref">347</a>]</span>.
Currently, there are over ten clinical trials registered with the intention to use zinc in a preventative or therapeutic manner.
Many of these trials have proposed the use of zinc in conjunction with hydroxychloroquine and azithromycin <span class="citation" data-cites="XJrj3MiZ 1EqFW3G4X Hc9mjVB2 T0aXNTIp">[<a href="#ref-XJrj3MiZ" role="doc-biblioref">348</a>,<a href="#ref-1EqFW3G4X" role="doc-biblioref">349</a>,<a href="#ref-Hc9mjVB2" role="doc-biblioref">350</a>,<a href="#ref-T0aXNTIp" role="doc-biblioref">351</a>]</span>.
However, it is not known how these trials will respond to the emerging lack of evidence supporting the use of hydroxychloroquine.
Other trials are investigating zinc in conjunction with other nutrients such as vitamin C or n-3 PUFA <span class="citation" data-cites="1GGbb9IZt 1AGVAbQuf">[<a href="#ref-1AGVAbQuf" role="doc-biblioref">336</a>,<a href="#ref-1GGbb9IZt" role="doc-biblioref">352</a>]</span>.</p>
<p><strong>Summary.</strong></p>
<p>Though there is, overall, encouraging data for zinc supplementation against the common cold and viral infections, there is currently limited evidence to suggest zinc supplementation has any beneficial effects against the current novel COVID-19; thus, the clinical trials that are currently underway will provide vital information on the efficacious use of zinc in COVID-19 prevention and/or treatment.
However, given the limited risk, maintaining a healthy diet to ensure an adequate zinc status may be advisable for individuals seeking to reduce their likelihood of infection.</p>
<h4 data-number="4.2.4.3" id="vitamin-c"><span class="header-section-number">4.2.4.3</span> Vitamin C</h4>
<p>Vitamins B, C, D, and E have also been suggested as potential nutrient supplement interventions for COVID-19 <span class="citation" data-cites="2hMFPQJR 3RCeQRRc">[<a href="#ref-2hMFPQJR" role="doc-biblioref">317</a>,<a href="#ref-3RCeQRRc" role="doc-biblioref">353</a>]</span>.
In particular vitamin C has been proposed as a potential therapeutic agent against COVID-19.
Vitamin C can be obtained via dietary sources such as fruit and vegetable or via supplementation.</p>
<p><strong>Potential Mechanisms.</strong>
Vitamin C plays a significant role in promoting immune function due to its effects on various immune cells.
Vitamin C affects inflammation by modulating cytokine production, decreasing histamine levels, enhancing the differentiation and proliferation of T- and B-lymphocytes, increasing antibody levels, and protecting against the negative effects of reactive oxygen species amongst other effects <span class="citation" data-cites="9czQt5cR RjFxOURM 13M3VAfY9">[<a href="#ref-9czQt5cR" role="doc-biblioref">354</a>,<a href="#ref-RjFxOURM" role="doc-biblioref">355</a>,<a href="#ref-13M3VAfY9" role="doc-biblioref">356</a>]</span>.
During viral infections vitamin C is utilized as evinced by lower concentrations in leukocytes and lower concentrations of urinary vitamin C.
Post-infection, these levels return to baseline ranges <span class="citation" data-cites="uQFCDDIy 10nBT1HbX 1Bj8VMlSk aEEZ8Pq9 gZRPoTM9">[<a href="#ref-uQFCDDIy" role="doc-biblioref">357</a>,<a href="#ref-10nBT1HbX" role="doc-biblioref">358</a>,<a href="#ref-1Bj8VMlSk" role="doc-biblioref">359</a>,<a href="#ref-aEEZ8Pq9" role="doc-biblioref">360</a>,<a href="#ref-gZRPoTM9" role="doc-biblioref">361</a>]</span>.</p>
<p><strong>Current Evidence.</strong>
A recent meta-analysis found consistent support for regular vitamin C supplementation reducing the duration of the common cold, but that supplementation with vitamin C (&gt; 200 mg) failed to reduce the incidence of colds <span class="citation" data-cites="Pb5PCH7y">[<a href="#ref-Pb5PCH7y" role="doc-biblioref">362</a>]</span>.
Individual studies have found Vitamin C to reduce the susceptibility of patients to lower respiratory tract infections such as pneumonia <span class="citation" data-cites="JARvpqRw">[<a href="#ref-JARvpqRw" role="doc-biblioref">363</a>]</span>.
Another meta-analysis has demonstrated in twelve trials that vitamin C supplementation reduced the length of stay of patients in intensive care units (ICUs) by 7.8% (95% CI: 4.2% to 11.2%; p = 0.00003).
Furthermore, high doses (1-3 g/day) significantly reduced the length of an ICU stay by 8.6% in six trials (p = 0.003).
Vitamin C also shortened the duration of mechanical ventilation by 18.2% in three trials in which patients required intervention for over 24 hours (95% CI 7.7% to 27%; p = 0.001) <span class="citation" data-cites="DPhRTQw0">[<a href="#ref-DPhRTQw0" role="doc-biblioref">364</a>]</span>.
Despite these findings, the CITRUS ALI study failed to show a benefit of a 96-hour infusion of vitamin C to treat ARDS, which is a severe complication of COVID-19 infection <span class="citation" data-cites="Rt5Aik4p">[<a href="#ref-Rt5Aik4p" role="doc-biblioref">365</a>]</span>.
Nevertheless, a randomized placebo-controlled trial <span class="citation" data-cites="wBwLIQkU">[<a href="#ref-wBwLIQkU" role="doc-biblioref">366</a>]</span> has begun in Wuhan, China to investigate the intravenous infusion of vitamin C to treat pneumonia in 140 severe COVID-19 infected patients.
As summarized by Carr <span class="citation" data-cites="rSyBZFhE">[<a href="#ref-rSyBZFhE" role="doc-biblioref">367</a>]</span> the trial will not be completed until September 2020.
Another trial in Italy <span class="citation" data-cites="16435N02u">[<a href="#ref-16435N02u" role="doc-biblioref">368</a>]</span> intends to deliver a 10 g infusion of vitamin C to 500 severe COVID-19 patients with pneumonia to assess in-hospital mortality over a 72 hr period as the primary outcome.
The trial is currently recruiting and is due to run until March 2021.
We will not know how effective vitamin C is as a therapeutic for quite some time due to the length of both trials.
These are not the only trials investigating the potential value of vitamin C, as there are currently (as of June 2020) over fifteen trials registered with clinicaltrials.gov that are either recruiting or are currently in preparation.
When completed, the trials will provide crucial evidence on the efficacy of vitamin C as a therapeutic for COVID-19 infection.</p>
<p><strong>Summary.</strong>
Some evidence suggests that vitamin C supplementation or infusion can shorten the duration of a cold, reduce an individual’s susceptibility to infections, and shorten a patient’s stay in an ICU when administered at high doses.
We don’t yet understand if these findings apply to COVID-19.
There are ongoing trials in China and Italy that will inform our understanding of the therapeutic value of vitamin C supplementation for COVID-19.</p>
<h4 data-number="4.2.4.4" id="vitamin-d"><span class="header-section-number">4.2.4.4</span> Vitamin D</h4>
<p>In terms of other dietary supplements, vitamin D can modulate the adaptive and innate immune system and has been associated with various aspects of health.
Vitamin D can be sourced through diet or supplementation, but it is mainly biosynthesized by the body on exposure to sunlight.</p>
<p><strong>Potential Mechanisms.</strong>
Vitamin D deficiency is associated with an increased susceptibility to infection <span class="citation" data-cites="n3v9Thlz">[<a href="#ref-n3v9Thlz" role="doc-biblioref">369</a>]</span>.
In particular, vitamin D deficient patients are at risk of developing acute respiratory infections <span class="citation" data-cites="iWLsflrG">[<a href="#ref-iWLsflrG" role="doc-biblioref">370</a>]</span> and ARDS <span class="citation" data-cites="iWLsflrG">[<a href="#ref-iWLsflrG" role="doc-biblioref">370</a>]</span>.
1,25-dihydroxyvitamin D3 (25-hydroxyvitamin D) is the active form of vitamin D that is involved in adaptive and innate responses, whereby the vitamin D receptor is expressed in various immune cells and vitamin D is an immunomodulator of antigen presenting cells, dendritic cells, macrophages, monocytes, and T- and B-lymphocytes <span class="citation" data-cites="n3v9Thlz nwEQBjV4">[<a href="#ref-n3v9Thlz" role="doc-biblioref">369</a>,<a href="#ref-nwEQBjV4" role="doc-biblioref">371</a>]</span>.
Due to its potential immunomodulating properties, vitamin D supplementation may be advantageous to maintain a healthy immune system.</p>
<p><strong>Current Evidence.</strong>
A recent review postulated that an individual’s vitamin D status may significantly affect their risk of developing COVID-19 <span class="citation" data-cites="9ac0GYir">[<a href="#ref-9ac0GYir" role="doc-biblioref">372</a>]</span>.
This hypothesis was derived from the fact that the current pandemic emerged in winter in Wuhan China when 25-hydroxyvitamin D concentrations are at their lowest due to a lack of sunlight, whereas in the Southern Hemisphere, where it was nearing the end of the summer and higher 25-hydroxyvitamin D concentrations would be higher, the number of cases was low.
The authors suggest that people at risk of developing COVID-19 should increase their vitamin D3 intake to reach 25-hydroxyvitamin D plasma concentrations above 40–60 ng/ml.
The authors also suggest high-dose supplementation of vitamin D to treat infected patients and to prevent infection in hospital staff <span class="citation" data-cites="9ac0GYir">[<a href="#ref-9ac0GYir" role="doc-biblioref">372</a>]</span>.
While vitamin D is relatively inexpensive and safe to consume, caution is warranted when interpreting this review as it has yet to be determined whether vitamin D levels affect COVID-19 outside of this geographic/climatic correlation.
Likewise, though it is assumed that COVID-19 may be seasonal, multiple other factors that can affect vitamin D levels should also be considered.
These factors include an individual’s nutritional status, their age, their occupation, skin pigmentation, potential comorbidities, and the variation of exposure to sunlight due to latitude amongst others.
As the pandemic evolves, further research has investigated some of the potential links identified in the Grant et al. review <span class="citation" data-cites="9ac0GYir">[<a href="#ref-9ac0GYir" role="doc-biblioref">372</a>]</span> between vitamin D and COVID-19 and sought to shed light on whether there is any prophylactic and/or therapeutic relationship.
A study in Switzerland demonstrated that 27 SARS-CoV-2 positive patients exhibited 25-hydroxyvitamin D plasma concentrations that were significantly lower (11.1 ng/ml) than those of SARS-CoV-2 negative patients (24.6 ng/ml; p = 0.004), an association that held when stratifying for patients greater than 70 years old <span class="citation" data-cites="1Gy7Svl72">[<a href="#ref-1Gy7Svl72" role="doc-biblioref">373</a>]</span>.
These findings seem to be supported by a Belgian observational study of 186 SARS-CoV-2 positive patients exhibiting symptoms of pneumonia, where 25-hydroxyvitamin D plasma concentrations were measured and a CT scan of the lungs was obtained upon hospitalization <span class="citation" data-cites="GMAEaWD1">[<a href="#ref-GMAEaWD1" role="doc-biblioref">374</a>]</span>.
A significant difference in 25-hydroxyvitamin D levels was observed between the SARS-CoV-2 patients and 2,717 season-matched diseased controls.
Both female and male patients possessed lower median 25-hydroxyvitamin D concentrations than the control group (18.6 ng/ml versus 21.5 ng/ml; p = 0.0016) and a higher rate of vitamin D deficiency (58.6% versus 42.5%).
Evidence of sexual dimorphism was apparent, as female patients had equivalent levels of 23-hydroxyvitamin D to the control group, whereas male patients were deficient in 25-hydroxyvitamin D relative to male controls (67% versus 49%; p = 0.0006).
Notably, vitamin D deficiency was progressively lower in males with advancing radiological disease stages (p = 0.001).
Despite these two observational studies potentially linking vitamin D with COVID-19, an examination of the UK Biobank did not support this thesis <span class="citation" data-cites="FZ8yXmxQ">[<a href="#ref-FZ8yXmxQ" role="doc-biblioref">375</a>]</span>.
This analysis examined 25-hydroxyvitamin D concentrations in 348,598 UK Biobank participants, of which 449 were SARS-CoV-2 positive.
There is significant interest in the link between vitamin D and COVID-19, hence the multitude of studies published in the literature of varying quality and the clinical trials underway.
One trial is currently investigating the utility of vitamin D as an immune-modulating agent by monitoring whether administration of vitamin D precipitates an improvement of health status in non-severe symptomatic patients infected with COVID-19 or whether vitamin D prevents patient deterioration <span class="citation" data-cites="162t7zdGv">[<a href="#ref-162t7zdGv" role="doc-biblioref">376</a>]</span>.
Other trials are also underway examining various factors including mortality, symptom recovery, severity of disease, rates of ventilation, inflammatory markers such as C-reactive protein (CRP) and IL-6, blood cell counts, and the prophylactic capacity of vitamin D administration <span class="citation" data-cites="2K0jV5Bn 162t7zdGv rBiojRHk 6kLoX5qT">[<a href="#ref-162t7zdGv" role="doc-biblioref">376</a>,<a href="#ref-2K0jV5Bn" role="doc-biblioref">377</a>,<a href="#ref-rBiojRHk" role="doc-biblioref">378</a>,<a href="#ref-6kLoX5qT" role="doc-biblioref">379</a>]</span>.
Concomitant administration of vitamin D with pharmaceuticals such as aspirin <span class="citation" data-cites="32C2erIe">[<a href="#ref-32C2erIe" role="doc-biblioref">380</a>]</span> and bioactive molecules such as resveratrol <span class="citation" data-cites="tLVliODQ">[<a href="#ref-tLVliODQ" role="doc-biblioref">381</a>]</span> are also under investigation.</p>
<p><strong>Summary.</strong>
Supplementation of vitamin D and maintaining a healthy diet for optimum vitamin D status warrants further investigation.
This is particularly important considering ‘stay in place’ guidance has been implemented in many densely populated cities around the world.
This measure is likely to limit people’s exposure to sunlight and thus reduce endogenous synthesis of vitamin D, potentially weakening the immune system and increasing the risk of COVID-19 infection.</p>
<h4 data-number="4.2.4.5" id="probiotics"><span class="header-section-number">4.2.4.5</span> Probiotics</h4>
<p>Probiotics are “live microorganisms that, when administered in adequate amounts, confer a health benefit on the host” <span class="citation" data-cites="1FyHVZ9yx">[<a href="#ref-1FyHVZ9yx" role="doc-biblioref">382</a>]</span>.
Some studies suggest that probiotics are beneficial against common viral infections and there is modest evidence to suggest that they can modulate the immune response <span class="citation" data-cites="pvZU55k9 HKeroTXj">[<a href="#ref-pvZU55k9" role="doc-biblioref">383</a>,<a href="#ref-HKeroTXj" role="doc-biblioref">384</a>]</span>, as a result it has been hypothesized that probiotics may have therapeutic value worthy of investigation against SARS-CoV-2 <span class="citation" data-cites="10e8cHDs8">[<a href="#ref-10e8cHDs8" role="doc-biblioref">385</a>]</span>.</p>
<p><strong>Potential Mechanisms.</strong>
Probiotics and next-generation probiotics, which are more akin to pharmacological-grade supplements, have been associated with multiple potential beneficial effects for allergies, digestive tract disorders, and even metabolic diseases through their anti-inflammatory and immunomodulatory effects <span class="citation" data-cites="5fU7dexB 1GCrGCp5P">[<a href="#ref-5fU7dexB" role="doc-biblioref">386</a>,<a href="#ref-1GCrGCp5P" role="doc-biblioref">387</a>]</span>.
However, the mechanisms by which probiotics affect these various conditions would likely differ among strains, with the ultimate effect of the probiotic depending on the heterogeneous set of bacteria present <span class="citation" data-cites="1GCrGCp5P">[<a href="#ref-1GCrGCp5P" role="doc-biblioref">387</a>]</span>.
Some of the beneficial effects of probiotics include reducing inflammation by promoting the expression of anti-inflammatory mediators, inhibiting toll-like receptors (TLR) 2 and 4, direct competition with pathogens, the synthesis of antimicrobial substances or other metabolites, improving intestinal barrier function, and/or favorably altering the gut microbiota and the brain-gut axis <span class="citation" data-cites="B2GeSSQo 1GCrGCp5P nmXF8vkS">[<a href="#ref-1GCrGCp5P" role="doc-biblioref">387</a>,<a href="#ref-B2GeSSQo" role="doc-biblioref">388</a>,<a href="#ref-nmXF8vkS" role="doc-biblioref">389</a>]</span>.
However, there is also a bi-directional relationship between the lungs and gut microbiota known as the gut-lung axis <span class="citation" data-cites="13AWoYvqB">[<a href="#ref-13AWoYvqB" role="doc-biblioref">390</a>]</span>, whereby gut microbial metabolites and endotoxins may affect the lungs via the circulatory system and the lung microbiota in return may affect the gut <span class="citation" data-cites="Smqu1EYN">[<a href="#ref-Smqu1EYN" role="doc-biblioref">391</a>]</span>.
Therefore, the gut-lung axis may play role in our future understanding of COVID-19 pathogenesis and become a target for probiotic treatments <span class="citation" data-cites="d27mQ4kO">[<a href="#ref-d27mQ4kO" role="doc-biblioref">392</a>]</span>.</p>
<p><strong>Current Evidence.</strong>
Probiotics have tentatively been associated with the reduction of risk and duration of viral upper respiratory tract infections <span class="citation" data-cites="1MIGrWIP 58eXvYEV kjavvCzp">[<a href="#ref-1MIGrWIP" role="doc-biblioref">393</a>,<a href="#ref-58eXvYEV" role="doc-biblioref">394</a>,<a href="#ref-kjavvCzp" role="doc-biblioref">395</a>]</span>.
Some meta-analyses that have assessed the efficacy of probiotics in viral respiratory infections have reported moderate reductions in the incidence and duration of infection <span class="citation" data-cites="58eXvYEV dJIGDt4k">[<a href="#ref-58eXvYEV" role="doc-biblioref">394</a>,<a href="#ref-dJIGDt4k" role="doc-biblioref">396</a>]</span>.
Indeed, randomized controlled trials have shown that administering <em>Bacillus subtilis</em> and <em>Enterococcus faecalis</em> <span class="citation" data-cites="AJav6c61">[<a href="#ref-AJav6c61" role="doc-biblioref">397</a>]</span>, <em>Lactobacillus rhamnosus GG</em> <span class="citation" data-cites="1E2aDJp4T">[<a href="#ref-1E2aDJp4T" role="doc-biblioref">398</a>]</span>, or <em>Lactobacillus casei</em> and <em>Bifidobacterium breve</em> with galactooligosaccharides <span class="citation" data-cites="zccFgajh">[<a href="#ref-zccFgajh" role="doc-biblioref">399</a>]</span> via the nasogastric tube to ventilated patients reduced the occurrence of ventilator-associated pneumonia in comparison to the respective control groups in studies of viral infections and sepsis.
These findings are supported by a recently published meta-analysis <span class="citation" data-cites="1DI06gFPc">[<a href="#ref-1DI06gFPc" role="doc-biblioref">400</a>]</span>.
There is a significant risk of ventilator-associated bacterial pneumonia in COVID-19 patients <span class="citation" data-cites="164mVBFZx">[<a href="#ref-164mVBFZx" role="doc-biblioref">401</a>]</span>, but it can be challenging for clinicians to diagnose this infection due to the fact that severe COVID-19 infection presents with the symptoms of pneumonia <span class="citation" data-cites="w2VFo0h1">[<a href="#ref-w2VFo0h1" role="doc-biblioref">402</a>]</span>.
Therefore, an effective prophylactic therapy for ventilator-associated pneumonia in severe COVID-19 patients would be of significant therapeutic value.</p>
<p>Probiotics are generally synonymous with the treatment of gastrointestinal issues due to their supposed anti-inflammatory and immunomodulatory effects <span class="citation" data-cites="gcyQ73v7">[<a href="#ref-gcyQ73v7" role="doc-biblioref">403</a>]</span>.
Notably, gastrointestinal symptoms commonly occur in COVID-19 patients <span class="citation" data-cites="67ihxDiV">[<a href="#ref-67ihxDiV" role="doc-biblioref">404</a>]</span>, and the ACE2 receptor is highly expressed in enterocytes of the ileum and colon, suggesting that these organs may be a potential route of infection <span class="citation" data-cites="8Hvgbl3n Nhxoefru">[<a href="#ref-8Hvgbl3n" role="doc-biblioref">405</a>,<a href="#ref-Nhxoefru" role="doc-biblioref">406</a>]</span>.
Indeed, SARS-CoV-2 viral RNA has been detected in human feces <span class="citation" data-cites="TPWaxApy">[<a href="#ref-TPWaxApy" role="doc-biblioref">407</a>]</span> and fecal-oral transmission of the virus has not yet been ruled out <span class="citation" data-cites="r9HNMqTS">[<a href="#ref-r9HNMqTS" role="doc-biblioref">408</a>]</span>.
Rectal swabs of some SARS-CoV-2 positive pediatric patients persistently tested positive for several days despite negative nasopharyngeal tests, indicating the potential for fecal viral shedding <span class="citation" data-cites="eOcc1BUx">[<a href="#ref-eOcc1BUx" role="doc-biblioref">409</a>]</span>.
However, there is conflicting evidence for the therapeutic value of various probiotics against the incidence or severity of gastrointestinal symptoms in viral or bacterial infections such as gastroenteritis <span class="citation" data-cites="VwStn9Ps UtAKolza">[<a href="#ref-VwStn9Ps" role="doc-biblioref">410</a>,<a href="#ref-UtAKolza" role="doc-biblioref">411</a>]</span>.
Nevertheless, it has been proposed that the administration of probiotics to COVID-19 patients and healthcare workers may prevent or ameliorate the gastrointestinal symptoms of COVID-19, a hypothesis that several clinical trials are now preparing to investigate <span class="citation" data-cites="1AqF1ESKq 9UYsmETk">[<a href="#ref-1AqF1ESKq" role="doc-biblioref">412</a>,<a href="#ref-9UYsmETk" role="doc-biblioref">413</a>]</span>.
Other studies are investigating whether probiotics may affect patient outcomes following SARS-CoV-2 infection <span class="citation" data-cites="15pDlYtom">[<a href="#ref-15pDlYtom" role="doc-biblioref">414</a>]</span>.</p>
<p><strong>Summary.</strong>
Generally, the efficacy of probiotic use is a controversial topic among scientists.
In Europe, EFSA has banned the term probiotics on products labels, which has elicited either criticism for EFSA or support for probiotics from researchers in the field <span class="citation" data-cites="m3SexO9F 5FTOk7ol 1FyHVZ9yx">[<a href="#ref-1FyHVZ9yx" role="doc-biblioref">382</a>,<a href="#ref-m3SexO9F" role="doc-biblioref">415</a>,<a href="#ref-5FTOk7ol" role="doc-biblioref">416</a>]</span>.
This is due to the hyperbolic claims placed on the labels of various probiotic products, which lack rigorous scientific data to support their efficacy.
Overall, the data supporting probiotics in the treatment or prevention of many different disorders and diseases is not conclusive as the quality of the evidence is generally considered low <span class="citation" data-cites="1MIGrWIP">[<a href="#ref-1MIGrWIP" role="doc-biblioref">393</a>]</span>.
However, in the case of probiotics and respiratory infections, the evidence seems to be supportive of their potential therapeutic value.
Consequently, several investigations are underway to investigate the prophylactic and therapeutic potential of probiotics for COVID-19.
However, blind use of conventional probiotics for COVID-19 is cautioned against until the pathogenesis of SARS-CoV-2 is further established <span class="citation" data-cites="kv94VRLF">[<a href="#ref-kv94VRLF" role="doc-biblioref">417</a>]</span>.
Until clinical trials investigating the prophylactic and therapeutic potential of probiotics for COVID-19 are complete, it is not possible to provide an evidence-based recommendation for their use.</p>
<h4 data-number="4.2.4.6" id="nutraceutical-conclusions"><span class="header-section-number">4.2.4.6</span> Nutraceutical Conclusions</h4>
<p>Despite all the potential benefits of nutraceutical and dietary supplement interventions presented, currently there is a paucity of clinical evidence to support their use for the prevention or mitigation of COVID-19 infections.
Nevertheless, optimal nutritional status will undoubtedly prime an individual’s immune system to protect against the effects of acute respiratory viral infections by supporting normal maintenance of the immune system <span class="citation" data-cites="5tHMedSi Lyd8s4Mk">[<a href="#ref-5tHMedSi" role="doc-biblioref">418</a>,<a href="#ref-Lyd8s4Mk" role="doc-biblioref">419</a>]</span>.
Nutritional strategies and the use of nutraceuticals will also undoubtedly play a role in the treatment of hospitalized patients as malnutrition is a risk to COVID-19 patients <span class="citation" data-cites="1DmkRczqw">[<a href="#ref-1DmkRczqw" role="doc-biblioref">420</a>]</span>.
Overall, supplementation of vitamin C, vitamin D, and zinc may be an effective method of ensuring their adequate intake to maintain optimal immune function, which may also convey beneficial effects against viral infections due to their immunomodulatory effects.
As a result, the administration of zinc, vitamin C, and vitamin D in conjunction with hydroxychloroquine are being investigated for their prophylactic effects in healthcare workers and their first-degree relatives in Turkey <span class="citation" data-cites="4vqbVndZ">[<a href="#ref-4vqbVndZ" role="doc-biblioref">421</a>]</span>.
However, many supplements and nutraceuticals designed for various ailments are available in the United States and beyond that are not strictly regulated <span class="citation" data-cites="lMhFSIiF">[<a href="#ref-lMhFSIiF" role="doc-biblioref">422</a>]</span>.
Indeed, there can be safety and efficacy concerns associated with many of these products.
Often, the vulnerable members of society can be exploited in this regard and unfortunately the COVID-19 pandemic is no different.
The Food and Drug Administration (FDA) has issued warnings to several companies for advertising falsified claims in relation to the preventative and therapeutic capabilities of their products against COVID-19 <span class="citation" data-cites="fez8aZAu">[<a href="#ref-fez8aZAu" role="doc-biblioref">423</a>]</span>.
In light of these serious occurrences, it is pertinent to clarify that the nutraceuticals discussed in this review have been selected because of their possible relevance to the biological mechanisms that can beneficially affect viral and respiratory infections and because they are currently under clinical investigation.
Therefore, further intensive investigation is required to establish the effects of these nutraceuticals, if any, against COVID-19.
Until effective therapeutics are established, the most effective mitigation strategies consist of encouraging standard public health practices such as regular hand washing with soap, wearing a face mask, and covering a cough with your elbow <span class="citation" data-cites="2Xu90WtD">[<a href="#ref-2Xu90WtD" role="doc-biblioref">424</a>]</span>, along with following social distancing measures, “stay in place” guidelines, expansive testing, and contact tracing <span class="citation" data-cites="EhbZIMo0 12Met0WvX">[<a href="#ref-EhbZIMo0" role="doc-biblioref">425</a>,<a href="#ref-12Met0WvX" role="doc-biblioref">426</a>]</span>.</p>
<h2 data-number="4.3" id="biologics"><span class="header-section-number">4.3</span> Biologics</h2>
<p>Biologics are produced from components of living organisms or viruses. They include antibodies such as the humanized monoclonal antibody
(mAb) tocilizumab (TCZ), neutralizing antibodies (nAbs), and vaccines.</p>
<h3 data-number="4.3.1" id="tocilizumab"><span class="header-section-number">4.3.1</span> Tocilizumab</h3>
<p>A recent study carried out on a sample of 191 adult COVID-19 in-patients at two Wuhan hospitals found that blood samples taken at admission contained significantly higher concentrations of interleukin-6 (IL-6) in patients who ultimately deceased compared to those who survived; average concentrations of IL-6 remained higher in the deceased group than the surviving group throughout hospitalization <span class="citation" data-cites="10THxyeCg">[<a href="#ref-10THxyeCg" role="doc-biblioref">56</a>]</span>.
This suggests that these individuals may be experiencing a “cytokine storm”, which refers to an excessive inflammatory response.
IL-6 plays a key role in this response <span class="citation" data-cites="bZMKqj6e">[<a href="#ref-bZMKqj6e" role="doc-biblioref">427</a>]</span>.
IL-6 is a pro-inflammatory cytokine belonging to the family of interleukins, which are immune system regulators that are primarily responsible for immune cell differentiation.
Specifically, IL-6 promotes the differentiation of activated B cells into immunoglobulin-producing plasma cells <span class="citation" data-cites="cVGAh42z">[<a href="#ref-cVGAh42z" role="doc-biblioref">428</a>]</span> and acts as a growth factor for hybridoma and myeloma cells <span class="citation" data-cites="1HPUdhrBf g6GOVWD3">[<a href="#ref-1HPUdhrBf" role="doc-biblioref">429</a>,<a href="#ref-g6GOVWD3" role="doc-biblioref">430</a>]</span>.
In addition, IL-6 also induces the differentiation of naïve CD4+ T cells into effector T-cell subsets <span class="citation" data-cites="eW4MC6Do">[<a href="#ref-eW4MC6Do" role="doc-biblioref">431</a>]</span>.
In this way interleukins regulate both the pro- and anti-inflammatory responses.
In this context, the observation of elevated IL-6 in patients who died may reflect an over-production of proinflammatory interleukins.</p>
<p>In a healthy situation the lung respiratory epithelium together with alveolar macrophages limits the activation of the immune system, ensuring homeostasis.
The introduction of the S-protein from SARS-CoV to mouse macrophages was found to increase production of IL-6 and TNF-α <span class="citation" data-cites="QkTGQUcj">[<a href="#ref-QkTGQUcj" role="doc-biblioref">432</a>]</span>, and deceased SARS-CoV patients were found to have intermediate levels of IL-6, IL-1𝛽, and TNF-α expressed in a number of ACE2-expressing cell types sampled from the lung and bronchial tissues during autopsy <span class="citation" data-cites="NNFGje9g">[<a href="#ref-NNFGje9g" role="doc-biblioref">433</a>]</span>.
However, other reports found the severe respiratory condition ARDS to be associated with elevated concentrations of IL-6 in BALF, but that concentrations of Tumor Necrosis Factor α (TNF-α) and IL-1𝛽 decreased with the onset of ARDS <span class="citation" data-cites="f61jsRKY">[<a href="#ref-f61jsRKY" role="doc-biblioref">434</a>]</span>.
These cytokines enhance the pro-inflammatory reaction by increase acute-phase signaling, trafficking of immune cells to the site of primary infection, epithelial cell activation, and secondary cytokine production.
The acute phase response to infection results in the heavy damage of the endothelium of blood vessels, which disrupts the balance between the pro- and anti-inflammatory response <span class="citation" data-cites="f61jsRKY">[<a href="#ref-f61jsRKY" role="doc-biblioref">434</a>]</span>.
Thus, the holes generated allow not just for the passage of neutrophils, macrophages and lymphocytes to the site of the infection but also the accumulation of liquids into the lungs, which is the ultimate cause of the death in ARDS and Severe Acute Respiratory Syndrome (SARS) <span class="citation" data-cites="suIFBW">[<a href="#ref-suIFBW" role="doc-biblioref">435</a>]</span>, also caused by the new coronavirus.
Recently Chinese and Italian doctors have found that Tocilizumab (TCZ), or actemra by Roche, a drug commonly used to treat rheumatoid arthritis (RA), may palliate the most severe symptoms associated with COVID-19.</p>
<h4 data-number="4.3.1.1" id="anticipated-mechanism"><span class="header-section-number">4.3.1.1</span> Anticipated Mechanism</h4>
<p>Human IL-6 is a glycoprotein of 26 kDa that consists of 184 amino acids containing 2 potential N-glycosylation sites and four cysteine residues.
IL-6 binds to its receptor either in the insoluble (IL-6R) and soluble (sIL-6R) form.
The receptor specificity determines the type of signaling.
Specifically, the binding of IL-6 to the cell membrane receptor IL-6R gives rise to the “classical transduction of the signaling”, while its binding to sIL-6R generates the so-called “trans-signaling” <span class="citation" data-cites="1BxkPZYnj 1EEloBjzn">[<a href="#ref-1BxkPZYnj" role="doc-biblioref">436</a>,<a href="#ref-1EEloBjzn" role="doc-biblioref">437</a>]</span>.
IL-6 signaling occurs through 3 independent pathways: the Janus-activated kinase (JAK)-STAT3 pathway, the Ras/Mitogen-Activated Protein Kinases (MAPK) pathway and the Phosphoinositol-3 Kinase (PI3K)/Akt pathway <span class="citation" data-cites="VXDsRwHC">[<a href="#ref-VXDsRwHC" role="doc-biblioref">438</a>]</span>.
The ultimate result of the IL-6 cascade is to direct transcriptional activity of various promoters of pro-inflammatory cytokines, such as IL-1 and TFN, including IL-6 own regulation through the activity of NF-κB <span class="citation" data-cites="VXDsRwHC">[<a href="#ref-VXDsRwHC" role="doc-biblioref">438</a>]</span>.
Particularly, IL-6 synthesis is tightly regulated both transcriptionally and post-transcriptionally.
In this context, it has been shown that viral proteins can enhance transcription of the IL-6 gene, via strengthening the DNA-binding activity between several transcription factors and IL-6 gene-cis-regulatory elements <span class="citation" data-cites="1H8o8TFtM">[<a href="#ref-1H8o8TFtM" role="doc-biblioref">439</a>]</span>.
TCZ is a humanized monoclonal antibody that binds both to the insoluble and soluble receptor of IL-6, de facto inhibiting the IL-6 immune cascade.</p>
<p><strong>Current Evidence.</strong>
Chinese doctors have started a trial enrolling 188 patients of which 14 with severe lung disease have shown clear signs of improvements, according to their results <span class="citation" data-cites="JgXCBFEo">[<a href="#ref-JgXCBFEo" role="doc-biblioref">440</a>]</span>.
Also, AIFA (the Italian Drug Agency) approved the start of a new trial on March 19 recruiting patients at the initial stage of the infection <span class="citation" data-cites="123fXWICW">[<a href="#ref-123fXWICW" role="doc-biblioref">441</a>]</span>.
In addition to these independent trials, Roche, in collaboration with the FDA, will start a randomized, double-blind, placebo-controlled phase III trial in early April.
The trial will enroll 330 patients globally, which will be followed for 60 days upon use of the drug via injection to analyze its efficiency/safety <span class="citation" data-cites="16AWSEc1x">[<a href="#ref-16AWSEc1x" role="doc-biblioref">442</a>]</span>.
However, previous studies of RA showed that the rate of incident infections in clinical practice patients treated with TCZ was higher than the rate observed during clinical trial <span class="citation" data-cites="TgV8gb30">[<a href="#ref-TgV8gb30" role="doc-biblioref">443</a>]</span>.
Also, RA patients with chronic hepatitis B (HB) infection showed high risk of HB virus reactivation upon TCZ administration in combination with other RA drugs <span class="citation" data-cites="IJzmtJOr">[<a href="#ref-IJzmtJOr" role="doc-biblioref">444</a>]</span>.
These last findings highlight the need to search for a balance between impairing a harmful immune response, such as the one generated by the cytokine storm, and preventing the worsening of the clinical picture of the patients by potential new viral infections.
This aspect should be investigated further in upcoming trials.</p>
<p>Perhaps, the TCZ treatment would best suit patients with severely compromised lungs due to the COVID-19 infection and are therefore at greater risk of death, in order to stop the uncontrolled immune response before it’s too late.</p>
<p><strong>Summary.</strong>
On April 29, 2020 UC San Diego Health started phase III clinical trial to establish whether TCZ, a drug commonly used to treat rheumatoid arthritis and similar diseases, also has therapeutic effect on COVID-19 patients <span class="citation" data-cites="JMM4eCZ3">[<a href="#ref-JMM4eCZ3" role="doc-biblioref">445</a>]</span>.
Approximately 25% of coronavirus patients develop ARDS, which is caused by an excessive early response of the immune system also known as the cytokine storm <span class="citation" data-cites="SiOKIbl3">[<a href="#ref-SiOKIbl3" role="doc-biblioref">446</a>]</span>.
This overwhelming inflammation is triggered by IL-6.
TCZ is an inhibitor of IL-6 and therefore may be able to neutralize the inflammatory pathway that leads to the cytokine storm.
According to the health commission of China, TCZ can be used in patients with high concentration of IL-6 in their plasma (greater than 20 pg/ml) or with extensive lung damage.
The recommended dose is 400mg of TCZ diluted to 100 ml with 0.9% normal saline (NaCl).
Patients who developed a fever within 12 hours or for whom the initial dose showed poor efficacy were administered an additional dose <span class="citation" data-cites="bj2feMy4 SiOKIbl3">[<a href="#ref-SiOKIbl3" role="doc-biblioref">446</a>,<a href="#ref-bj2feMy4" role="doc-biblioref">447</a>]</span>.
It is important to note that TCZ is contraindicated in patients with active infections such as tuberculosis <span class="citation" data-cites="SiOKIbl3">[<a href="#ref-SiOKIbl3" role="doc-biblioref">446</a>]</span>, though a recent study shows that the drug is safe for pregnant and breastfeeding women <span class="citation" data-cites="6xLhs9st">[<a href="#ref-6xLhs9st" role="doc-biblioref">448</a>]</span>.</p>
<h3 data-number="4.3.2" id="neutralizing-antibodies"><span class="header-section-number">4.3.2</span> Neutralizing Antibodies</h3>
<p>Monoclonal antibodies have revolutionized the way we treat human diseases.
As a result, they have become some of the best-selling drugs in the pharmaceutical market in recent years <span class="citation" data-cites="1AKi0FYUB">[<a href="#ref-1AKi0FYUB" role="doc-biblioref">449</a>]</span>.
There are currently 79 FDA approved mAbs on the market including antibodies for viral infections (e.g. Ibalizumab for HIV and Palivizumab for RSV) <span class="citation" data-cites="1AKi0FYUB yumBaJ6U">[<a href="#ref-1AKi0FYUB" role="doc-biblioref">449</a>,<a href="#ref-yumBaJ6U" role="doc-biblioref">450</a>]</span>.
For that reason, neutralizing antibodies have emerged to address these shortcomings.
Virus-specific neutralizing antibodies commonly target viral surface glycoproteins or host structures, thereby inhibiting viral entry through receptor binding interference <span class="citation" data-cites="dUnB3gD6 cwbHiM1o">[<a href="#ref-dUnB3gD6" role="doc-biblioref">451</a>,<a href="#ref-cwbHiM1o" role="doc-biblioref">452</a>]</span>.
This section discusses current efforts in developing neutralizing antibodies against SARS-CoV-2 and how expertise gained from previous approaches for MERS-CoV and SARS-CoV may benefit antibody development.</p>
<h4 data-number="4.3.2.1" id="spike-s-neutralizing-antibody"><span class="header-section-number">4.3.2.1</span> Spike (S) Neutralizing Antibody</h4>
<p>During the first SARS epidemic in 2002, nAbs were found in SARS-CoV infected patients <span class="citation" data-cites="3gp7t98X KULxo48U">[<a href="#ref-3gp7t98X" role="doc-biblioref">453</a>,<a href="#ref-KULxo48U" role="doc-biblioref">454</a>]</span>.
Several studies following up on these findings identified various S glycoprotein epitopes as the major targets of neutralizing antibodies against SARS-CoV <span class="citation" data-cites="MTcTiS7i">[<a href="#ref-MTcTiS7i" role="doc-biblioref">455</a>]</span>.
The passive transfer of immune serum containing nAbs from SARS-CoV-infected mice resulted in protection of naïve mice from viral lower respiratory tract infection upon intranasal challenge <span class="citation" data-cites="rXzkS3gA">[<a href="#ref-rXzkS3gA" role="doc-biblioref">456</a>]</span>.
Similarly, a meta-analysis suggested that administration of plasma from recovered SARS-CoV patients reduced mortality upon SARS-CoV infection <span class="citation" data-cites="RRx6YovP">[<a href="#ref-RRx6YovP" role="doc-biblioref">457</a>]</span>.</p>
<p>Similar results have been observed for MERS-CoV infections, which emerged as the second coronavirus-related epidemic.
Neutralizing antibodies have been identified against various epitopes of the RBD of the S glycoprotein [<span class="citation" data-cites="Ou3P9pGV"><a href="#ref-Ou3P9pGV" role="doc-biblioref">458</a></span>; doi:10.1128/JVI.00912-14].</p>
<p><strong>Anticipated Mechanisms.</strong>
Coronaviruses use trimeric spike (S) glycoproteins on their surface to bind to host cell receptors, such as ACE2, allowing for cell entry <span class="citation" data-cites="15l3di3Wj qcVbT0w4">[<a href="#ref-qcVbT0w4" role="doc-biblioref">38</a>,<a href="#ref-15l3di3Wj" role="doc-biblioref">44</a>]</span>.
Each S glycoprotein protomer is comprised of an S1 domain, also called the receptor binding domain (RBD), and an S2 domain.
The S1 domain binds to host cell receptors while the S2 domain facilitates the fusion between the viral envelope and host cell membranes <span class="citation" data-cites="MTcTiS7i">[<a href="#ref-MTcTiS7i" role="doc-biblioref">455</a>]</span>.
Although targeting of the host cell receptor ACE2 shows efficacy in inhibiting SARS-CoV-2 infection <span class="citation" data-cites="2QTH37Xi">[<a href="#ref-2QTH37Xi" role="doc-biblioref">41</a>]</span>, given the physiological relevance of ACE2 <span class="citation" data-cites="1GLr0EJU">[<a href="#ref-1GLr0EJU" role="doc-biblioref">459</a>]</span>, it would be favorable to target virus-specific structures rather than host receptors.
This forms the rationale of developing neutralizing antibodies against the S glycoprotein, disrupting its interaction with ACE2 and other receptors and thereby inhibiting viral entry.</p>
<p><strong>Current Evidence.</strong>
The first human neutralizing antibody against SARS-CoV-2 targeting the trimeric spike (S) glycoproteins has been developed using hybridoma technology, <span class="citation" data-cites="XcuzhxrJ">[<a href="#ref-XcuzhxrJ" role="doc-biblioref">460</a>]</span>, where antibody-producing B-cells developed by mice can be inserted into myeloma cells to produce a hybrid cell line (the hybridoma) that is grown in culture.
The 47D11 clone was able to cross-neutralize SARS-CoV and SARS-CoV2 by a mechanism that is different from receptor binding interference.
The exact mechanism of how this clone neutralizes SARS-CoV-2 and inhibits infection in vitro remains unknown, but a potential mechanism might be antibody induced destabilization of the membrane prefusion structure <span class="citation" data-cites="XcuzhxrJ Ftbm1M9p">[<a href="#ref-XcuzhxrJ" role="doc-biblioref">460</a>,<a href="#ref-Ftbm1M9p" role="doc-biblioref">461</a>]</span>.
The ability of this antibody to prevent infection at a feasible dose needs to be validated in vivo, especially since in vitro neutralization effects have been shown to not be reflective of in vivo efficacy <span class="citation" data-cites="O1whWg6Q">[<a href="#ref-O1whWg6Q" role="doc-biblioref">462</a>]</span>.
Only a week later, a different group successfully isolated multiple nAbs targeting the RBD of the S glycoprotein from blood samples taken from COVID-19 patients in China <span class="citation" data-cites="GhJYjnft">[<a href="#ref-GhJYjnft" role="doc-biblioref">178</a>]</span>.
Interestingly, the patient isolated antibodies did not cross-react with RBDs from SARS-CoV and MERS-CoV, although cross-reactivity to the trimeric spike proteins of SARS-CoV and MERS-CoV was observed.
This suggests that the RBDs between the three coronavirus species are immunologically distinct and that the isolated nAbs targeting the RBD of SARS-CoV-2 are species specific.
While this specificity is desirable, it also raises the question of whether these antibodies are more susceptible to viral escape mechanisms.
Viral escape is a common resistance mechanism to nAbs therapy due to selective pressure from neutralizing antibodies <span class="citation" data-cites="pUnzB8wV 14motyCOm">[<a href="#ref-pUnzB8wV" role="doc-biblioref">463</a>,<a href="#ref-14motyCOm" role="doc-biblioref">464</a>]</span>.
For HIV, broadly neutralizing antibodies (bnAbs) targeting the CD4 binding site (CD4bs) show greater neutralization breadth than monoclonal antibodies, which target only specific HIV strains <span class="citation" data-cites="oGHFF5Cm">[<a href="#ref-oGHFF5Cm" role="doc-biblioref">465</a>]</span>.
For MERS-CoV, a combination of multiple neutralizing antibodies targeting different antigenic sites prevented neutralization escape <span class="citation" data-cites="hvjXMm9H">[<a href="#ref-hvjXMm9H" role="doc-biblioref">466</a>]</span>.
It was found that the different antibody isolates did not target the same epitopes, suggesting that using them in combination might produce a synergistic effect that prevents viral escape <span class="citation" data-cites="GhJYjnft">[<a href="#ref-GhJYjnft" role="doc-biblioref">178</a>]</span>.
It was also demonstrated that binding affinity of the antibodies does not reflect their capability to compete with ACE2 binding.
Furthermore, no conclusions about correlations between the severity of disease and the ability to produce neutralizing antibodies can be drawn at this point.
Rather, higher neutralizing antibody titers were more frequently found in patients with severe disease.
Correspondingly, higher levels of anti-spike IgG were observed in patients that deceased from infection compared to patient that recovered <span class="citation" data-cites="tC5vJmwj">[<a href="#ref-tC5vJmwj" role="doc-biblioref">467</a>]</span>.</p>
<p><strong>Summary.</strong></p>
<p>Results from the SARS-CoV and MERS-CoV epidemics can provide valuable lessons for the design of neutralizing antibodies for the current outbreak.
The findings for SARS-CoV and MERS can aid in identifying which structures constitute suitable targets for nAbs, despite the fact that the RBD appears to be distinct between the three coronavirus species.
These studies also suggest that a combination of nAbs targeting distinct antigens might be necessary to provide protection <span class="citation" data-cites="hvjXMm9H">[<a href="#ref-hvjXMm9H" role="doc-biblioref">466</a>]</span>.
The biggest challenge remains identifying antibodies that not only bind to their target, but also prove to be beneficial for disease management.
On that note, a recently published study indicates that anti-spike antibodies could make the disease worse rather than eliminating the virus <span class="citation" data-cites="tC5vJmwj">[<a href="#ref-tC5vJmwj" role="doc-biblioref">467</a>]</span>.
These findings underscores our current lack of understanding the full immune response to SARS-CoV-2.</p>
<!-- Use the format below to add additional sections
**Anticipated Mechanism.**
Why it may be useful

**Current Evidence.**
A list of current studies and their results, using carefully the information requested in the therapeutic paper tickets.

**Summary.**
Summarize the state of the neutralizing antibody approach.
-->
<h3 data-number="4.3.3" id="interferons"><span class="header-section-number">4.3.3</span> Interferons</h3>
<p>Interferons (IFNs) are a family of cytokines crucial to activate the first (innate) immune system response against viral infections.
Interferons are classified into three categories based on their receptor specificity: type I, II and III <span class="citation" data-cites="bZMKqj6e">[<a href="#ref-bZMKqj6e" role="doc-biblioref">427</a>]</span>.
Specifically, IFNs I (IFN-𝛼 and 𝛽) and II (IFN-𝛾) induce the expression of antiviral proteins that bring the viral RNA to degradation <span class="citation" data-cites="LtFtUo2P">[<a href="#ref-LtFtUo2P" role="doc-biblioref">468</a>]</span>.
Among these IFNs, IFN-𝛽 has already been found to strongly inhibit the replication of other coronaviruses, such as SARS-CoV, in cell culture, while IFN-𝛼 and 𝛾 were shown to be less effective in this context <span class="citation" data-cites="LtFtUo2P">[<a href="#ref-LtFtUo2P" role="doc-biblioref">468</a>]</span>.
There is evidence that patients with higher susceptibility to ARDS indeed show deficiency in IFN- 𝛽.
For instance, infection with other coronaviruses impairs IFN-𝛽 expression and synthesis, allowing the virus to escape the innate immune response <span class="citation" data-cites="Nfu9kiae">[<a href="#ref-Nfu9kiae" role="doc-biblioref">469</a>]</span>.</p>
<p>On March 18 2020 Synairgen plc has received approval to start a phase II trial for SNG001, an IFN-𝛽-1a formulation to be delivered to the lungs via inhalation.
SNG001 was already shown to be effective reducing viral load in swine flu in vivo model, as well as it has been shown to be effective in the protection from other Corona virus infection in vitro (Synairgen plc, press release).</p>
<p><strong>Anticipated Mechanism.</strong>
Why it may be useful</p>
<p><strong>Current Evidence.</strong></p>
<p>A list of current studies and their results, using carefully the information requested in the therapeutic paper tickets.</p>
<p><strong>Summary.</strong>
Summarize the state of interferons.</p>
<h3 data-number="4.3.4" id="vaccines"><span class="header-section-number">4.3.4</span> Vaccines</h3>
<h4 data-number="4.3.4.1" id="strategies-for-and-challenges-to-vaccine-development"><span class="header-section-number">4.3.4.1</span> Strategies for and challenges to vaccine development</h4>
<p>Today, the first step in producing a vaccine often is characterizing the target.
The genetic sequence of SARS-CoV-2 was published on January 11, 2020, which aided the global effort to develop a vaccine to prevent COVID-19.
The Coalition for Epidemic Preparedness Innovations (CEPI) is coordinating global health agencies and pharmaceutical companies to develop vaccines against SARS-CoV-2.
As of April 8, 2020, there were 115 vaccine candidates to prevent COVID-19, of which 78 were active.
Of the 78 active vaccine programs, 73 were in the preclinical or exploratory stage <span class="citation" data-cites="1GA95MF2m">[<a href="#ref-1GA95MF2m" role="doc-biblioref">470</a>]</span>.</p>
<p>Historically, an H1N1 influenza vaccine was developed relatively efficiently, mainly because influenza-vaccine technology had already been developed and regulatory agencies had already decided that vaccines produced using egg- and cell-based platforms could be licensed under the regulations used for a strain change.
Critiques of the experience producing and distributing the H1N1 vaccine have stressed the need for alternative development-and-manufacturing platforms that can be readily adapted to new pathogens.
Although a monovalent H1N1 vaccine was not available before the pandemic peaked in the United States and Europe, it was available soon afterward as a stand-alone vaccine that was eventually incorporated into the commercially available seasonal influenza vaccines <span class="citation" data-cites="HyYY2agc">[<a href="#ref-HyYY2agc" role="doc-biblioref">471</a>]</span>.
If H1N1 vaccine development provides any indication, considering developing and manufacturing platforms for promising COVID-19 vaccine trials early could hasten the emergence of an effective prophylactic vaccine against SARS-CoV-2.</p>
<p>Unlike many global vaccine development programs previously, such as with H1N1, the vaccine development landscape for COVID-19 includes vaccines produced by a wide array of technologies.
Experience in the field of oncology is encouraging COVID-19 vaccine developers to use next-generation approaches to vaccine development, which have led to the great diversity of vaccine development programs <span class="citation" data-cites="wPl93ATP">[<a href="#ref-wPl93ATP" role="doc-biblioref">472</a>]</span>.
Diverse technology platforms include DNA, RNA, virus-like particle, recombinant protein, both replicating and non-replicating viral vectors, live attenuated virus, and inactivated virus approaches.
Given the wide range of vaccines under development, it is possible that some vaccine products may eventually be shown to be more effective in certain subpopulations, such as children, pregnant women, immunocompromised patients, the elderly, etc.
The requirements for a successful vaccine trial and deployment are complex and may require coordination between government, industry, academia, and philanthropic entities <span class="citation" data-cites="plfPrQP7">[<a href="#ref-plfPrQP7" role="doc-biblioref">473</a>]</span>.</p>
<h5 data-number="4.3.4.1.1" id="dna-vaccines"><span class="header-section-number">4.3.4.1.1</span> DNA Vaccines</h5>
<p>This vaccination method involves the direct introduction of a plasmid containing a DNA sequence encoding the antigen(s) against which an immune response is sought into appropriate tissues <span class="citation" data-cites="XnrBoKVk">[<a href="#ref-XnrBoKVk" role="doc-biblioref">474</a>]</span>.</p>
<p><strong>Anticipated Mechanism.</strong>
This approach may offer several advantages over traditional vaccination approaches, such as the stimulation of both B- as well as T-cell responses and the absence of any infectious agent.</p>
<p><strong>Current Evidence.</strong>
Currently, a Phase I safety and immunogenicity clinical trial of INO-4800, a prophylactic vaccine against SARS-CoV-2, is underway <span class="citation" data-cites="xuzLfS0y">[<a href="#ref-xuzLfS0y" role="doc-biblioref">475</a>]</span>.
The vaccine developer Inovio Pharmaceuticals Technology is overseeing administration of INO-4800 by intradermal injection followed by electroporation with the CELLECTRA® device to healthy volunteers.
Electroporation is the application of brief electric pulses to tissues in order to permeabilize cell membranes in a transient and reversible manner.
It has been shown that electroporation can enhance vaccine efficacy by up to 100-fold, as measured by increases in antigen-specific antibody titers <span class="citation" data-cites="H6tWVs5R">[<a href="#ref-H6tWVs5R" role="doc-biblioref">476</a>]</span>.
The safety of the CELLECTRA® device has been studied for over seven years, and these studies support the further development of electroporation as a safe vaccine delivery method <span class="citation" data-cites="1Hsm2J1sc">[<a href="#ref-1Hsm2J1sc" role="doc-biblioref">477</a>]</span>.
The temporary formation of pores through electroporation facilitates the successful transportation of macromolecules into cells, allowing cells to robustly take up INO-4800 for the production of an antibody response.</p>
<p>Approved by the U.S. Food and Drug Administration (FDA) on April 6, 2020, the Phase I study is enrolling up to 40 healthy adult volunteers in Philadelphia, PA at the Perelman School of Medicine and at the Center for Pharmaceutical Research in Kansas City, MO.
The trial has two experimental arms corresponding to the two locations.
Participants in Experimental Group 1 will receive one intradermal injection of 1.0 milligram (mg) of INO-4800 followed by electroporation using the CELLECTRA® 2000 device twice, administered at Day 0 and Week 4.
Participants in Experimental Group 2 will receive two intradermal injections of 1.0 mg (total 2.0 mg per dosing visit) of INO-4800 followed by electroporation using the CELLECTRA® 2000 device, administered at Day 0 and Week 4.
Safety data and the initial immune responses of participants from the trial are expected by the end of the summer of 2020.</p>
<p><strong>Summary.</strong>
The development of a DNA vaccine against SARS-CoV-2 by Inovio could be an important step forward in the world’s search for a COVID-19 vaccine.
Although exciting, the cost of vaccine manufacturing and electroporation may make scaling the use of this technology for prophylactic use for the general public difficult.</p>
<h5 data-number="4.3.4.1.2" id="rna-vaccines"><span class="header-section-number">4.3.4.1.2</span> RNA Vaccines</h5>
<p>RNA vaccines are nucleic-acid based modalities that code for viral antigens against which the human body elicits a humoral and cellular immune response.
The mRNA technology is transcribed <em>in vitro</em> and delivered to cells via lipid nanoparticles (LNP).
They are recognized by ribosomes <em>in vivo</em> and then translated and modified into functional proteins <span class="citation" data-cites="K0Ltu31S">[<a href="#ref-K0Ltu31S" role="doc-biblioref">478</a>]</span>.
The resulting intracellular viral proteins are displayed on surface MHC proteins, provoking a strong CD8+ T cell response as well as a CD4+ T cell and B cell-associated antibody responses <span class="citation" data-cites="K0Ltu31S">[<a href="#ref-K0Ltu31S" role="doc-biblioref">478</a>]</span>.</p>
<p>Naturally, mRNA is not very stable and can degrade quickly in the extracellular environment or the cytoplasm.
The LNP covering protects the mRNA from enzymatic degradation outside of the cell <span class="citation" data-cites="zNKWlCwE">[<a href="#ref-zNKWlCwE" role="doc-biblioref">479</a>]</span>.
Codon optimization to prevent secondary structure formation and modifications of the poly-A tail as well as the 5’ untranslated region to promote ribosomal complex binding can increase mRNA expression in cells.
Furthermore, purifying out dsRNA and immature RNA with FPLC (fast performance liquid chromatography) and HPLC (high performance liquid chromatography) technology will improve translation of the mRNA in the cell <span class="citation" data-cites="K0Ltu31S pRoqjur8">[<a href="#ref-K0Ltu31S" role="doc-biblioref">478</a>,<a href="#ref-pRoqjur8" role="doc-biblioref">480</a>]</span>.</p>
<p>mRNA vaccines confer many advantages over traditional viral vectored vaccines and DNA vaccines.
In comparison to live attenuated viruses, mRNA vaccines are non-infectious and can be synthetically produced in an egg-free, cell-free environment, thereby reducing the risk of a detrimental immune response in the host <span class="citation" data-cites="wYZ6qJMu">[<a href="#ref-wYZ6qJMu" role="doc-biblioref">481</a>]</span>.
Unlike DNA vaccines, mRNA technologies are naturally degradable and non-integrating, and they do not need to cross the nuclear membrane in addition to the plasma membrane for their effects to be seen <span class="citation" data-cites="K0Ltu31S">[<a href="#ref-K0Ltu31S" role="doc-biblioref">478</a>]</span>.
Furthermore, mRNA vaccines are easily, affordably, and rapidly scalable.</p>
<p>Although mRNA vaccines have been developed for therapeutic and prophylactic purposes, none have been licensed or commercialized thus far.
Nevertheless, they have shown promise in animal models and preliminary clinical trials for several indications, including rabies, coronavirus, influenza, and cytomegalovirus <span class="citation" data-cites="3EUiWZdN">[<a href="#ref-3EUiWZdN" role="doc-biblioref">482</a>]</span>.
Preclinical data from Pardi et al. identified effective antibody generation against full-length FPLC-purified influenza hemagglutinin stalk-encoding mRNA in mice, rabbits, and ferrets <span class="citation" data-cites="6wZy2mn8">[<a href="#ref-6wZy2mn8" role="doc-biblioref">483</a>]</span>.
Similar immunological responses for mRNA vaccines were observed in humans in Phase I and II clinical trials operated by the pharmaceutical-development companies Curevac and Moderna for rabies, flu, and zika <span class="citation" data-cites="pRoqjur8">[<a href="#ref-pRoqjur8" role="doc-biblioref">480</a>]</span>.</p>
<p><strong>Anticipated Mechanism.</strong>
Positively charged bilayer LNPs carrying the mRNA attract negatively charged cell membranes, endocytose into the cytoplasm <span class="citation" data-cites="zNKWlCwE">[<a href="#ref-zNKWlCwE" role="doc-biblioref">479</a>]</span>, and facilitate endosomal escape.
LNPs can be coated with modalities recognized and engulfed by specific cell types.
LNPs 150nm or less effectively enter into lymphatic vessels.</p>
<p>There are three types of RNA vaccines: non-replicating, <em>in vivo</em> self-replicating, and <em>in vitro</em> dendritic cell non-replicating <span class="citation" data-cites="1EM5nGaYd">[<a href="#ref-1EM5nGaYd" role="doc-biblioref">484</a>]</span>.
Non-replicating mRNA vaccines consist of a simple open reading frame (ORF) for the viral antigen flanked by the 5’ UTR and 3’ poly-A tail.
<em>In vivo</em> self-replicating vaccines encode a modified viral genome derived from single-stranded, positive sense RNA alphaviruses <span class="citation" data-cites="K0Ltu31S pRoqjur8">[<a href="#ref-K0Ltu31S" role="doc-biblioref">478</a>,<a href="#ref-pRoqjur8" role="doc-biblioref">480</a>]</span>.
The RNA genome encodes the viral antigen along with proteins of the genome replication machinery, including an RNA polymerase.
Structural proteins required for viral assembly are not included in the engineered genome <span class="citation" data-cites="K0Ltu31S">[<a href="#ref-K0Ltu31S" role="doc-biblioref">478</a>]</span>.
Self-replicating vaccines produce more viral antigens over a longer period of time, thereby evoking a more robust immune response <span class="citation" data-cites="1EM5nGaYd">[<a href="#ref-1EM5nGaYd" role="doc-biblioref">484</a>]</span>.
Finally, <em>in vitro</em> dendritic cell non-replicating RNA vaccines limit transfection to dendritic cells.
Dendritic cells are potent antigen-presenting immune cells that easily take up mRNA and present fragments of the translated peptide on their MHC proteins, which can then interact with T cell receptors.
Ultimately, primed T follicular helper cells can stimulate germinal center B cells that also present the viral antigen to produce antibodies against the virus <span class="citation" data-cites="3LMMW7F0">[<a href="#ref-3LMMW7F0" role="doc-biblioref">485</a>]</span>.
These cells are isolated from the patient, grown and transfected <em>ex vivo</em>, and reintroduced to the patient <span class="citation" data-cites="ENBWnhAh">[<a href="#ref-ENBWnhAh" role="doc-biblioref">486</a>]</span>.</p>
<p><strong>Current Evidence.</strong>
mRNA-1273 is the first COVID-19 vaccine to enter a phase I clinical in the United States.
ModernaTX, Inc. is currently spearheading an investigation on the immunogenicity and reactogenicity of mRNA-1273, a conventional lipid nanoparticle encapsulated RNA encoding a full-length prefusion stabilized spike (S) protein for SARS-CoV-2 <span class="citation" data-cites="Biu1CQeQ">[<a href="#ref-Biu1CQeQ" role="doc-biblioref">487</a>]</span>.
Forty-five participants will be enrolled in the study and given an intramuscular injection of mRNA-1273 in their deltoid muscle on Day 1 and Day 29, and then followed for the next twelve months.
Healthy males and non-pregnant females aged 18-55 years are being recruited and will be divided in three dosage groups receiving either 25 micrograms, 100 mcg, or 250 mcg of the vaccine.</p>
<p>The study started on March 3rd, 2020, and is expected to complete by June 1st, 2021.
Emory Vaccine Center and Kaiser Permanent Washington Health Research Institute are currently recruiting participants with NIH Clinical Center expecting to recruit soon.
Reports on patient safety and reactogenicity will be recorded soon.
IgG ELISA assays on patient serology samples will study the immunogenicity of the vaccine <span class="citation" data-cites="Biu1CQeQ">[<a href="#ref-Biu1CQeQ" role="doc-biblioref">487</a>]</span>.</p>
<p><strong>Summary.</strong>
mRNA vaccines are promising tools in the prevention and control of pandemics.
mRNA-1273 is the only RNA vaccine for SARS-CoV-2 currently being tested in clinical trials and results are expected soon.</p>
<h5 data-number="4.3.4.1.3" id="viral-particle-vaccines"><span class="header-section-number">4.3.4.1.3</span> Viral Particle Vaccines</h5>
<p>Brief background on the therapeutic.</p>
<!--##### Anticipated Mechanism

Why it may be useful

**Current Evidence.**
A list of current studies and their results, using carefully the information requested in the therapeutic paper tickets.

**Summary.**
Summarize the state of the vaccine approach.
-->
<h4 data-number="4.3.4.2" id="oligonucleotide-therapies"><span class="header-section-number">4.3.4.2</span> Oligonucleotide Therapies</h4>
<p>Add background and other information below</p>
<!--##### Anticipated Mechanism

Why it may be useful

**Current Evidence.**
A list of current studies and their results, using carefully the information requested in the therapeutic paper tickets.

**Summary.**
Summarize the state of the neutralizing antibody approach.
-->
<h4 data-number="4.3.4.3" id="adjuvants-for-vaccines"><span class="header-section-number">4.3.4.3</span> Adjuvants for Vaccines</h4>
<p>Adjuvants include a variety of molecules or larger microbial-related products which sometimes are or include immunostimulants or immunomodulators or more generally have an effect on the immune system or immune responses of interest.
Adjuvants are sometimes included within vaccines, especially vaccines other than live attenuated and inactivated viruses, in order to enhance the immune response.
A review on the development of SARS-CoV-2 vaccines <span class="citation" data-cites="ouOXy0wH">[<a href="#ref-ouOXy0wH" role="doc-biblioref">488</a>]</span> also included a brief summary of considerations for adjuvants to research for the vaccines including a brief description of some already commonly used adjuvants.
Different adjuvants can regulate different types of immune responses, so the type of adjuvant or combination of adjuvants for use in a vaccine depends on the type of vaccine and the research into what works with respect to efficacy and safety.
A variety of possible mechanisms for adjuvants have been researched <span class="citation" data-cites="13bVbfc5h 122h6fIxE uO0uqhxc">[<a href="#ref-13bVbfc5h" role="doc-biblioref">489</a>,<a href="#ref-122h6fIxE" role="doc-biblioref">490</a>,<a href="#ref-uO0uqhxc" role="doc-biblioref">491</a>]</span> including the following: induction of DAMPs (damage-associated molecular patterns) which can be recognized by certain pattern recognition receptors (PRRs) of the innate immune system; being or including or functioning as PAMPs (pathogen-associated molecular patterns) which can also be recognized by certain pattern recognition receptors (PRRs); and more generally enhancing humoral or cellular immune responses.
When choosing an adjuvant or combination of adjuvants, considerations include promotion of advantageous effects of the components and immune response and avoiding or inhibition of possible deleterious effects of the components and immune response.
There are also considerations, technologies, and research into the effects of different ways to deliver (or co-deliver or combine) the adjuvant and antigen components of a vaccine.</p>
<h2 data-number="4.4" id="underexplored-therapeutics"><span class="header-section-number">4.4</span> Underexplored Therapeutics</h2>
<p>The majority of current clinical trials and lines of investigation have focused on repurposing existing therapies to counter SARS-CoV-2 and treat its symptoms.
This is necessary given the urgency of the situation as well as the extensive time required for developing and testing new therapies.
However, in the long-term, new drugs specific for treatment of COVID-19 may also enter development.
There is thus value in investigating two lines of inquiry for treatment of COVID-19: 1) new therapeutics specific for treatment of COVID-19 or its symptoms, and 2) repurposing of existing therapeutics for treatment of COVID-19 or its symptoms.
Here we consider further avenues that scientific investigators may explore in the development of therapies for COVID-19.</p>
<p>Given the great focus in investigating hydroxychloroquine (HCQ) as a potential antiviral treatment for SARS-CoV-2, it may be of interest to researchers to explore related alternatives.
For example, hydroxyferroquine derivatives of HCQ have been described as a class of bioorganometallic compounds that exert antiviral effects with some selectivity for SARS-CoV <span class="citation" data-cites="Cd5uMaAr">[<a href="#ref-Cd5uMaAr" role="doc-biblioref">492</a>]</span>.
Future work could explore whether these compounds exert antiviral effects against SARS-CoV-2 and whether they are safe for use in animals and humans.</p>
<p>The tocilizumab trial described in an above section <span class="citation" data-cites="10THxyeCg">[<a href="#ref-10THxyeCg" role="doc-biblioref">56</a>]</span> studies the possibility of using an anti-inflammatory agent typically used for the treatment of autoimmune disease to counter the effects of the “cytokine storm” induced by the virus.
Another anti-IL-6 antibody, sarilumab, is also being investigated <span class="citation" data-cites="4DAIEbQF amy3rDsj">[<a href="#ref-4DAIEbQF" role="doc-biblioref">493</a>,<a href="#ref-amy3rDsj" role="doc-biblioref">494</a>]</span>.
Typically, immunosuppressive drugs such as these are contraindicated in the setting of infection <span class="citation" data-cites="nbiwprc">[<a href="#ref-nbiwprc" role="doc-biblioref">495</a>]</span>.
However, COVID-19 results in hyperinflammation that appears to contribute to mortality via lung damage, suggesting that immunosuppression may be a helpful approach to treatment <span class="citation" data-cites="3HdlV9Vf">[<a href="#ref-3HdlV9Vf" role="doc-biblioref">143</a>]</span>.
The decision of whether and/or when to counter hyperinflammation with immunosuppression in the setting of COVID-19 remains in debate as the risks of inhibiting antiviral immunity continue to be weighed against the beneficial anti-inflammatory effects <span class="citation" data-cites="suHq6Qnm">[<a href="#ref-suHq6Qnm" role="doc-biblioref">496</a>]</span>.
If the need to curtail the “cytokine storm” inflammatory response to the virus transcends the risks of immunosuppression, exploration of more anti-inflammatory agents may be warranted; these agents are considered here.
While tocilizumab targets IL-6, several other inflammatory markers could be potential targets, including TNF-alpha. Inhibition of TNF-alpha by an inhibitor such as Etanercept has been previously suggested for treatment of SARS-CoV <span class="citation" data-cites="LDQlolY3">[<a href="#ref-LDQlolY3" role="doc-biblioref">497</a>]</span> and may be relevant for SARS-CoV-2 as well.
Baricitinib and other small molecule inhibitors of the JAK kinase pathway also curtail the inflammatory response and have been suggested as potential options for SARS-CoV-2 infections <span class="citation" data-cites="WF7ymA4m">[<a href="#ref-WF7ymA4m" role="doc-biblioref">498</a>]</span>.
Baricitinib in particular may be able to reduce the ability of SARS-CoV-2 to infect lung cells <span class="citation" data-cites="uCns3aFw">[<a href="#ref-uCns3aFw" role="doc-biblioref">499</a>]</span>.
Clinical trials studying baricitinib in COVID-19 have already begun in the US and in Italy <span class="citation" data-cites="jKRGxazA JCMhG5r9">[<a href="#ref-jKRGxazA" role="doc-biblioref">500</a>,<a href="#ref-JCMhG5r9" role="doc-biblioref">501</a>]</span>.
Identification and targeting of further inflammatory markers that are relevant in SARS-CoV-2 infection may be of value for curtailing the inflammatory response and lung damage.
Lastly, it is also worth noting the high costs of tocilizumab therapy and other biologics: at doses used for rheumatoid arthritis patients, the cost for tocilizumab ranges from $179.20 to $896 per dose for the IV form and $355 for the pre-filled syringe <span class="citation" data-cites="7fZ9eyMk">[<a href="#ref-7fZ9eyMk" role="doc-biblioref">502</a>]</span>.
Cyclosporine may be a more cost-effective and readily-available alternative than biologics <span class="citation" data-cites="trWLoLFz">[<a href="#ref-trWLoLFz" role="doc-biblioref">503</a>]</span>, if it proves effective against the cytokine storm induced by SARS-CoV-2.</p>
<p>Another approach is the development of antivirals, which could be broad-spectrum, specific to coronaviruses, or targeted to SARS-CoV-2.
Given the increasingly apparent role of the cytokine storm in disease pathogenesis, it is possible that antivirals could be less effective in more severe cases of COVID-19, but this is not yet known; regardless, it is likely that early-stage patients could benefit from antiviral therapy.
The potential for remdesivir as an antiviral has already been described in an above section.
Development of new antivirals is complicated by the fact that none have yet been approved for human coronaviruses.
Intriguing new options are emerging, however.
Beta-D-N4-hydroxycytidine (NHC) is an orally bioavailable ribonucleotide analog showing broad-spectrum activity against RNA viruses, which may inhibit SARS-CoV-2 replication <span class="citation" data-cites="TZleuwHX">[<a href="#ref-TZleuwHX" role="doc-biblioref">504</a>]</span>.
Various other antivirals are in development.
Development of antivirals will be further facilitated as research reveals more information about the interaction of SARS-CoV-2 with the host cell and host cell genome, mechanisms of viral replication, mechanisms of viral assembly, and mechanisms of viral release to other cells; this can allow researchers to target specific stages and structures of the viral life cycle.</p>
<p>Antibodies against viruses, also known as antiviral monoclonal antibodies, could be an alternative as well and are described in detail in an above section.
The goal of antiviral antibodies is to neutralize viruses through either cell-killing activity or blocking of viral replication <span class="citation" data-cites="1FTnvyZlf">[<a href="#ref-1FTnvyZlf" role="doc-biblioref">505</a>]</span>.
They may also engage the host immune response, encouraging the immune system to hone in on the virus.
Given the cytokine storm that results from immune system activation in response to the virus, which has been implicated in worsening of the disease, a neutralizing antibody (nAb) may be preferable.
Upcoming work may explore the specificity of nAbs for their target, mechanisms by which the nAbs impede the virus, and improvements to antibody structure that may enhance the ability of the antibody to block viral activity.</p>
<p>There is also some research into possible potential therapeutics or prophylactics which interact with components of the innate immune response.
For example, there are a variety of toll-like receptors (TLRs) which are examples of pattern recognition receptors (PRRs), innate immune response components which recognize pathogen-associated molecular patterns (PAMPs) and damage-associated molecular patterns (DAMPs).
Note that TLRs form a part of innate immune recognition and can more generally contribute to promoting both innate and adaptive responses <span class="citation" data-cites="o6BQnEt7">[<a href="#ref-o6BQnEt7" role="doc-biblioref">506</a>]</span>.
In mouse models, poly(I:C) and lipopolysaccharide, which are agonists of toll-like receptors TLR3 and TLR4, respectively, showed protective effects when administered prior to SARS-CoV exposure <span class="citation" data-cites="Ull2rQ5L">[<a href="#ref-Ull2rQ5L" role="doc-biblioref">507</a>]</span>.
Therefore, TLR agonists and antagonists hold some potential for broad-spectrum prophylaxis.
Another type of approach which is being investigated is the possible use of what is termed trained immunity, in particular as elicited by non-SARS-CoV-2 whole-microorganism vaccines (or other microbial stimuli), as a tool which might be shown to generate heterologous protective effects with respect to SARS-CoV-2 susceptibility or severity <span class="citation" data-cites="Vu1VILWK">[<a href="#ref-Vu1VILWK" role="doc-biblioref">508</a>]</span>.
In a recent review <span class="citation" data-cites="103fS7Kz2">[<a href="#ref-103fS7Kz2" role="doc-biblioref">509</a>]</span>, trained immunity was defined as forms of memory which are temporary (e.g., months or years, and reversible), displayed by innate immune cells and innate immune features of other cells, displayed as increased responsiveness to future same or heterologous pathogen infection or sometimes decreased responsiveness or immunological tolerance, and established through epigenetic and metabolic mechanisms.
One type of stimulus which research indicates can induce trained immunity is bacillus Calmette-Guerin (BCG) vaccination.
BCG is an attenuated form of bacteria <em>Mycobacterium bovis</em>.
The vaccine is most commonly administered for the prevention of tuberculosis in humans.
With respect to SARS-CoV-2 specifically, clinical trials in non-SARS-CoV-2-infected adults have been setup to assess the possible efficacy of BCG vaccination – e.g., to assess if it may be effective as a potential prophylactic or potential partial prophylactic in (1) reducing susceptibility / preventing infection and (2) reducing disease severity (for descriptions of trials with BCG vaccine or the related vaccine VPM1002, see <span class="citation" data-cites="Vu1VILWK 9m3rP633 y9IYdfM3 xdqxBruc 962rELVS EuwTWcPi dQtUeruv DjXsPR8O 10OE6y3Pv 86OjIybR ITO15LIz VkZGZxLn 13JVjMfQI nk2MVsld 1E2t9tr8h">[<a href="#ref-Vu1VILWK" role="doc-biblioref">508</a>,<a href="#ref-9m3rP633" role="doc-biblioref">510</a>,<a href="#ref-y9IYdfM3" role="doc-biblioref">511</a>,<a href="#ref-xdqxBruc" role="doc-biblioref">512</a>,<a href="#ref-962rELVS" role="doc-biblioref">513</a>,<a href="#ref-EuwTWcPi" role="doc-biblioref">514</a>,<a href="#ref-dQtUeruv" role="doc-biblioref">515</a>,<a href="#ref-DjXsPR8O" role="doc-biblioref">516</a>,<a href="#ref-10OE6y3Pv" role="doc-biblioref">517</a>,<a href="#ref-86OjIybR" role="doc-biblioref">518</a>,<a href="#ref-ITO15LIz" role="doc-biblioref">519</a>,<a href="#ref-VkZGZxLn" role="doc-biblioref">520</a>,<a href="#ref-13JVjMfQI" role="doc-biblioref">521</a>,<a href="#ref-nk2MVsld" role="doc-biblioref">522</a>,<a href="#ref-1E2t9tr8h" role="doc-biblioref">523</a>]</span>).
Some trials enroll healthcare workers, other trials hospitalized elderly adults without immunosuppression who get vaccinated with placebo or BCG at hospital discharge, and yet another set of trials older adults (&gt;50 years) under chronic care for conditions like hypertension and diabetes.
One set of trials, for example, uses time until first infection as the primary study endpoint; more generally, outcomes measured in some of these trials are related to incidence of disease and disease severity or symptoms.
An article <span class="citation" data-cites="Vu1VILWK">[<a href="#ref-Vu1VILWK" role="doc-biblioref">508</a>]</span> that very briefly summarizes the setup of these trials also mentions some data analyses which showed possible correlations between countries which have more BCG vaccination (or BCG vaccination policies) and the severity of COVID-19 in those countries, but (1) it is unclear whether this correlation indicates an interaction because for example of many other possible factors or confounding factors (country age distribution, detection efficiency, stochastic epidemic dynamic effects, differences in healthcare capacity over time in relation to epidemic dynamics, and various other factors) and (2) also it is unclear what is the relation between BCG vaccination at different ages and at longer times before the start of the SARS-CoV-2 epidemic and BCG vaccination at different ages during the SARS-CoV-2 epidemic at shorter times before the risk of possible infection; at least some of these considerations are also made or pointed out in the data analyses sited and in the article.
The article <span class="citation" data-cites="Vu1VILWK">[<a href="#ref-Vu1VILWK" role="doc-biblioref">508</a>]</span> also includes various related considerations such as efficacy in related animal and human studies and the safety of the vaccine, both generally and specifically with respect to SARS-CoV-2.
Additionally, we review here one of the most important considerations: severe SARS-CoV-2 has been characterized so far to possibly exhibit dysregulated immune responses and whether or when some immune responses are protective or pathogenic is still under research (e.g., <span class="citation" data-cites="i5k18bpX 1GnFL9zeN Vu1VILWK 1228YjPRv">[<a href="#ref-1228YjPRv" role="doc-biblioref">186</a>,<a href="#ref-Vu1VILWK" role="doc-biblioref">508</a>,<a href="#ref-i5k18bpX" role="doc-biblioref">524</a>,<a href="#ref-1GnFL9zeN" role="doc-biblioref">525</a>]</span>); also, trained immunity itself may sometimes be related to either balanced, or too much or too little, or useful or harmful immunological responses.
The article <span class="citation" data-cites="Vu1VILWK">[<a href="#ref-Vu1VILWK" role="doc-biblioref">508</a>]</span> proposes that trained immunity might lead to an earlier stronger response which could reduce viremia and avoid (possibly with the additional support of another therapeutic administered later) the later detrimental immunopathology seen with severe cases; while this is a possibility, additional research is required to assess its validity.</p>
<p>In the longer term, as more information becomes available about the structures of SARS-CoV-2 components, small molecule inhibitors of those components may become candidates for drug discovery.
For example, crystal structures of the SARS-CoV-2 main protease have recently been resolved <span class="citation" data-cites="v08gazBj AJLaaguT">[<a href="#ref-v08gazBj" role="doc-biblioref">526</a>,<a href="#ref-AJLaaguT" role="doc-biblioref">527</a>]</span>.
Efforts have already been in place to perform screens for small molecule inhibitors of the main protease, yielding potential hits <span class="citation" data-cites="v08gazBj">[<a href="#ref-v08gazBj" role="doc-biblioref">526</a>]</span>.
Much work remains to be done to determine further crystal structures of other viral components, understand the relative utility of targeting different viral components, perform additional small molecule inhibitor screens, and determine the safety and efficacy of the potential inhibitors.
While still nascent, work in this area is promising.</p>
<h1 data-number="5" id="discussion"><span class="header-section-number">5</span> Discussion</h1>
<!-- This section is a stub. Additional work is needed to flesh this section out. Outlining or writing help is welcome!-->
<p>As the COVID-19 pandemic continues to develop, the scientific community has responded by rapidly collecting and disseminating information about the SARS-CoV-2 virus and its ability to infect humans and other animals.
This fundamental information has allowed for innovations in the areas of diagnostics and therapeutics that continue to be proposed and developed upon.
In this review, we seek to explain the scientific rationale underlying these technologies and to critically evaluate the literature available about them.</p>
<h2 data-number="5.1" id="current-state-of-diagnostics"><span class="header-section-number">5.1</span> Current State of Diagnostics</h2>
<h2 data-number="5.2" id="current-state-of-therapeutics"><span class="header-section-number">5.2</span> Current State of Therapeutics</h2>
<h2 data-number="5.3" id="concerns-about-equity-in-healthcare"><span class="header-section-number">5.3</span> Concerns about Equity in Healthcare</h2>
<p>Scientific and medical research broadly is shaped by a number of biases.
Some concerns include how clinical trials recruit and operate.</p>
<!--Risk of comorbid health conditions associated with more severe outcomes may be influenced by long-term damage caused by chronic stress related to traumatic social experiences [@doi:10.1007/s12170-013-0338-5], perhaps mediated by cardiovascular risk factors [@doi:10.1101/2020.05.10.20097253], although the effects of chronic stress have not yet been researched in the specific case of COVID-19 disparities.-->
<h1 data-number="6" id="methods"><span class="header-section-number">6</span> Methods</h1>
<h2 data-number="6.1" id="article-selection-and-evaluation"><span class="header-section-number">6.1</span> Article Selection and Evaluation</h2>
<p>The authors solicited relevant articles to be submitted via <a href="https://github.com/greenelab/covid19-review">GitHub</a> for review.
Articles were classified as <em>diagnostic</em>, <em>therapeutic</em>, or <em>other</em>.
Following a framework often used for assessing medical literature, the review consisted of examining methods used in each relevant article, assignment (whether the study was observational or randomized), assessment, results, interpretation, and how well the study extrapolates <span class="citation" data-cites="17OQtAY4l">[<a href="#ref-17OQtAY4l" role="doc-biblioref">528</a>]</span>.</p>
<h3 data-number="6.1.1" id="diagnostic-papers"><span class="header-section-number">6.1.1</span> Diagnostic Papers</h3>
<h4 data-number="6.1.1.1" id="methods-1"><span class="header-section-number">6.1.1.1</span> Methods</h4>
<p>Reviewers began by describing the study question(s) being investigated by the article.
They then described the study population, the sample size, the prevalence of the disease in the study population, countries / regions considered in case of human subjects, demographics of participants, the setting, and any remaining inclusion / exclusion criteria considered.
They then described the reference test or “gold standard,” if one was utilized.</p>
<h4 data-number="6.1.1.2" id="assignment"><span class="header-section-number">6.1.1.2</span> Assignment</h4>
<p>Reviewers described how new and reference tests were assigned, including additional relevant details about the study design.
For example, reviewers were asked whether the diagnostic test resulted in rigorous assignments of case status or was biased towards sicker or healthier individuals.</p>
<h4 data-number="6.1.1.3" id="assessment"><span class="header-section-number">6.1.1.3</span> Assessment</h4>
<p>Reviewers described how the test was performed.
For example, for both standard and reference tests, reviewers described technical details of assays used, when measurements were taken and by whom.
Subsequently, they described how individuals were classified as positive or negative cases and whether results were precise and reproducible with repeated tests.
Reviewers described whether there were any missing data, whether some participants underwent only one test, or whether there were individuals with inconclusive results.</p>
<h4 data-number="6.1.1.4" id="results"><span class="header-section-number">6.1.1.4</span> Results</h4>
<p>Reviewers reported the estimated sensitivity, specificity, positive predictive value (PPV), and negative predicted value (NPV), as well as confidence bounds around these measures, if provided.</p>
<h4 data-number="6.1.1.5" id="interpretation"><span class="header-section-number">6.1.1.5</span> Interpretation</h4>
<p>Reviewers reported how well the test ruled in or ruled out disease based on the population, if there were identified side effects, and patient adherence.</p>
<h4 data-number="6.1.1.6" id="extrapolation"><span class="header-section-number">6.1.1.6</span> Extrapolation</h4>
<p>Reviewers described how well this test will extrapolate outside the measured population.</p>
<h3 data-number="6.1.2" id="therapeutic-papers"><span class="header-section-number">6.1.2</span> Therapeutic Papers</h3>
<h4 data-number="6.1.2.1" id="methods-2"><span class="header-section-number">6.1.2.1</span> Methods</h4>
<p>Reviewers began by describing the study question(s) being investigated by the article.
They then described the study population, the sample size, the prevalence of the disease in the study population, countries / regions considered in case of human subjects, demographics of participants, the setting, and any remaining inclusion / exclusion criteria considered.</p>
<h4 data-number="6.1.2.2" id="assignment-1"><span class="header-section-number">6.1.2.2</span> Assignment</h4>
<p>Reviewers described how the treatment is assigned, whether it was an interventional or observational study, whether randomization took place, etc.</p>
<h4 data-number="6.1.2.3" id="assessment-1"><span class="header-section-number">6.1.2.3</span> Assessment</h4>
<h5 data-number="6.1.2.3.1" id="outcome-assessment"><span class="header-section-number">6.1.2.3.1</span> Outcome Assessment</h5>
<p>Reviewers described the outcome that was assessed and evaluated whether it was appropriate given the underlying study question.
They described whether there were any missing data such as whether there were individuals lost to follow up.
They then describe whether there were any potential sources of bias such as lack of blinding in a randomized controlled trial.</p>
<h5 data-number="6.1.2.3.2" id="statistical-methods-assessment"><span class="header-section-number">6.1.2.3.2</span> Statistical Methods Assessment</h5>
<p>Reviewers described which statistical methods were used for inference and whether applied methods were appropriate for the study.
They then described whether adjustments were made for possible confounders.</p>
<h4 data-number="6.1.2.4" id="results-1"><span class="header-section-number">6.1.2.4</span> Results</h4>
<p>Reviewers described the estimated association between the treatment and outcome.
They described measures of confidence or statistical significance, if provided.</p>
<h4 data-number="6.1.2.5" id="interpretation-1"><span class="header-section-number">6.1.2.5</span> Interpretation</h4>
<p>Reviewers described whether a causal claim could be made.
They described whether any side effects or interactions with other drugs were identified, as well as any subgroup findings.</p>
<h4 data-number="6.1.2.6" id="extrapolation-1"><span class="header-section-number">6.1.2.6</span> Extrapolation</h4>
<p>Reviewers describe how the study may extrapolate to a different species or population.</p>
<h2 data-number="6.2" id="collaborative-writing"><span class="header-section-number">6.2</span> Collaborative Writing</h2>
<p>Crowd-sourced writing with Manubot <span class="citation" data-cites="YuJbg3zO">[<a href="#ref-YuJbg3zO" role="doc-biblioref">529</a>]</span>.</p>
<h1 class="page_break_before" data-number="7" id="additional-items"><span class="header-section-number">7</span> Additional Items</h1>
<h2 data-number="7.1" id="competing-interests"><span class="header-section-number">7.1</span> Competing Interests</h2>
<table>
<colgroup>
<col style="width: 33%" />
<col style="width: 33%" />
<col style="width: 33%" />
</colgroup>
<thead>
<tr class="header">
<th>Author</th>
<th>Competing Interests</th>
<th>Last Reviewed</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Halie M. Rando</td>
<td>None</td>
<td>2020-03-22</td>
</tr>
<tr class="even">
<td>Casey S. Greene</td>
<td>None</td>
<td>2020-03-22</td>
</tr>
<tr class="odd">
<td>Michael P. Robson</td>
<td>None</td>
<td>2020-03-23</td>
</tr>
<tr class="even">
<td>Simina M. Boca</td>
<td>None</td>
<td>2020-03-23</td>
</tr>
<tr class="odd">
<td>Nils Wellhausen</td>
<td>None</td>
<td>2020-03-22</td>
</tr>
<tr class="even">
<td>Ronan Lordan</td>
<td>None</td>
<td>2020-03-25</td>
</tr>
<tr class="odd">
<td>Christian Brueffer</td>
<td>Employee and shareholder of SAGA Diagnostics AB.</td>
<td>2020-04-14</td>
</tr>
<tr class="even">
<td>Sadipan Ray</td>
<td>None</td>
<td>2020-03-25</td>
</tr>
<tr class="odd">
<td>Lucy D'Agostino McGowan</td>
<td>None</td>
<td>2020-03-26</td>
</tr>
<tr class="even">
<td>Anthony Gitter</td>
<td>None</td>
<td>2020-03-26</td>
</tr>
<tr class="odd">
<td>Ronnie M. Russell</td>
<td>None</td>
<td>2020-04-07</td>
</tr>
<tr class="even">
<td>Anna Ada Dattoli</td>
<td>None</td>
<td>2020-03-26</td>
</tr>
<tr class="odd">
<td>Ryan Velazquez</td>
<td>None</td>
<td>2020-04-04</td>
</tr>
<tr class="even">
<td>John P. Barton</td>
<td>None</td>
<td>2020-04-06</td>
</tr>
<tr class="odd">
<td>Jeffrey M. Field</td>
<td>None</td>
<td>2020-03-30</td>
</tr>
<tr class="even">
<td>Bharath Ramsundar</td>
<td>None</td>
<td>2020-04-06</td>
</tr>
<tr class="odd">
<td>Adam L. MacLean</td>
<td>None</td>
<td>2020-04-06</td>
</tr>
<tr class="even">
<td>Alexandra J. Lee</td>
<td>None</td>
<td>2020-04-07</td>
</tr>
<tr class="odd">
<td>Immunology Institute of the Icahn School of Medicine</td>
<td>None</td>
<td>2020-04-07</td>
</tr>
<tr class="even">
<td>Fengling Hu</td>
<td>None</td>
<td>2020-04-08</td>
</tr>
<tr class="odd">
<td>Nafisa M. Jadavji</td>
<td>None</td>
<td>2020-04-09</td>
</tr>
<tr class="even">
<td>Elizabeth Sell</td>
<td>None</td>
<td>2020-04-10</td>
</tr>
<tr class="odd">
<td>Jinhui Wang</td>
<td>None</td>
<td>2020-04-13</td>
</tr>
<tr class="even">
<td>Diane N. Rafizadeh</td>
<td>None</td>
<td>2020-04-14</td>
</tr>
<tr class="odd">
<td>Ashwin N. Skelly</td>
<td>None</td>
<td>2020-04-16</td>
</tr>
<tr class="even">
<td>Marouen Ben Guebila</td>
<td>None</td>
<td>2020-04-17</td>
</tr>
<tr class="odd">
<td>Likhitha Kolla</td>
<td>None</td>
<td>2020-04-23</td>
</tr>
<tr class="even">
<td>David Manheim</td>
<td>None</td>
<td>2020-04-28</td>
</tr>
<tr class="odd">
<td>Soumita Ghosh</td>
<td>None</td>
<td>2020-04-28</td>
</tr>
<tr class="even">
<td>Matthias Fax</td>
<td>None</td>
<td>2020-04-30</td>
</tr>
<tr class="odd">
<td>James Brian Byrd</td>
<td>Funded by FastGrants to conduct a COVID-19-related clinical trial</td>
<td>2020-04-23</td>
</tr>
<tr class="even">
<td>YoSon Park</td>
<td>None</td>
<td>2020-04-30</td>
</tr>
<tr class="odd">
<td>Yael Evelyn Marshall</td>
<td>None</td>
<td></td>
</tr>
<tr class="even">
<td>Vikas Bansal</td>
<td>None</td>
<td>2020-05-26</td>
</tr>
<tr class="odd">
<td>Stephen Capone</td>
<td>None</td>
<td>2020-06-23</td>
</tr>
<tr class="even">
<td>John J. Dziak</td>
<td>None</td>
<td>2020-06-23</td>
</tr>
<tr class="odd">
<td>YuCheng Sun</td>
<td>None</td>
<td>2020-07-09</td>
</tr>
<tr class="even">
<td>Yanjun Qi</td>
<td>None</td>
<td>2020-07-09</td>
</tr>
<tr class="odd">
<td>Lamonica Shinholster</td>
<td>None</td>
<td>2020-07-22</td>
</tr>
</tbody>
</table>
<h2 data-number="7.2" id="author-contributions"><span class="header-section-number">7.2</span> Author Contributions</h2>
<table>
<colgroup>
<col style="width: 50%" />
<col style="width: 50%" />
</colgroup>
<thead>
<tr class="header">
<th>Author</th>
<th>Contributions</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Halie M. Rando</td>
<td>Project Administration, Writing - Original Draft, Writing - Review &amp; Editing, Methodology</td>
</tr>
<tr class="even">
<td>Casey S. Greene</td>
<td>Conceptualization, Software</td>
</tr>
<tr class="odd">
<td>Michael P. Robson</td>
<td>Software</td>
</tr>
<tr class="even">
<td>Simina M. Boca</td>
<td>Methodology, Writing - Review &amp; Editing</td>
</tr>
<tr class="odd">
<td>Nils Wellhausen</td>
<td>Writing - Original Draft, Writing - Review &amp; Editing, Project Administration, Visualization</td>
</tr>
<tr class="even">
<td>Ronan Lordan</td>
<td>Writing - Original Draft, Writing - Review &amp; Editing</td>
</tr>
<tr class="odd">
<td>Christian Brueffer</td>
<td>Writing - Original Draft, Writing - Review &amp; Editing, Project Administration</td>
</tr>
<tr class="even">
<td>Sadipan Ray</td>
<td>Writing - Original Draft</td>
</tr>
<tr class="odd">
<td>Lucy D'Agostino McGowan</td>
<td>Methodology, Writing - Original Draft</td>
</tr>
<tr class="even">
<td>Anthony Gitter</td>
<td>Methodology, Software, Project Administration</td>
</tr>
<tr class="odd">
<td>Ronnie M. Russell</td>
<td>Writing - Original Draft, Writing - Review &amp; Editing</td>
</tr>
<tr class="even">
<td>Anna Ada Dattoli</td>
<td>Writing - Original Draft</td>
</tr>
<tr class="odd">
<td>Ryan Velazquez</td>
<td>Methodology, Software</td>
</tr>
<tr class="even">
<td>John P. Barton</td>
<td>Writing - Original Draft, Writing - Review &amp; Editing</td>
</tr>
<tr class="odd">
<td>Jeffrey M. Field</td>
<td>Writing - Original Draft</td>
</tr>
<tr class="even">
<td>Bharath Ramsundar</td>
<td>Investigation, Writing - Review &amp; Editing</td>
</tr>
<tr class="odd">
<td>Adam L. MacLean</td>
<td>Writing - Original Draft</td>
</tr>
<tr class="even">
<td>Alexandra J. Lee</td>
<td>Writing - Original Draft</td>
</tr>
<tr class="odd">
<td>Immunology Institute of the Icahn School of Medicine</td>
<td>Data Curation</td>
</tr>
<tr class="even">
<td>Fengling Hu</td>
<td>Writing - Original Draft, Writing - Review &amp; Editing</td>
</tr>
<tr class="odd">
<td>Nafisa M. Jadavji</td>
<td>Writing - Original Draft, Writing - Review &amp; Editing</td>
</tr>
<tr class="even">
<td>Elizabeth Sell</td>
<td>Writing - Original Draft, Writing - Review &amp; Editing</td>
</tr>
<tr class="odd">
<td>Jinhui Wang</td>
<td>Writing - Revising &amp; Editing</td>
</tr>
<tr class="even">
<td>Diane N. Rafizadeh</td>
<td>Writing - Original Draft, Writing - Review &amp; Editing</td>
</tr>
<tr class="odd">
<td>Ashwin N. Skelly</td>
<td>Writing - Original Draft, Writing - Review &amp; Editing</td>
</tr>
<tr class="even">
<td>Marouen Ben Guebila</td>
<td>Writing - Original Draft</td>
</tr>
<tr class="odd">
<td>Likhitha Kolla</td>
<td>Writing - Original Draft</td>
</tr>
<tr class="even">
<td>David Manheim</td>
<td>Writing - Original Draft, Investigation</td>
</tr>
<tr class="odd">
<td>Soumita Ghosh</td>
<td>Writing - Original Draft</td>
</tr>
<tr class="even">
<td>Matthias Fax</td>
<td>Writing - Review &amp; Editing</td>
</tr>
<tr class="odd">
<td>James Brian Byrd</td>
<td>Writing - Original Draft, Writing - Review &amp; Editing</td>
</tr>
<tr class="even">
<td>YoSon Park</td>
<td>Writing - Original Draft, Writing - Review &amp; Editing, Investigation</td>
</tr>
<tr class="odd">
<td>Yael Evelyn Marshall</td>
<td>Writing - Original Draft, Writing - Review &amp; Editing</td>
</tr>
<tr class="even">
<td>Vikas Bansal</td>
<td>Writing - Original Draft, Investigation</td>
</tr>
<tr class="odd">
<td>Stephen Capone</td>
<td>Writing - Review &amp; Editing, Writing - Original Draft</td>
</tr>
<tr class="even">
<td>John J. Dziak</td>
<td>Writing - Original Draft</td>
</tr>
<tr class="odd">
<td>YuCheng Sun</td>
<td>Visualization</td>
</tr>
<tr class="even">
<td>Yanjun Qi</td>
<td>Visualization</td>
</tr>
<tr class="odd">
<td>Lamonica Shinholster</td>
<td>Writing - Original Draft</td>
</tr>
</tbody>
</table>
<h2 data-number="7.3" id="acknowledgements"><span class="header-section-number">7.3</span> Acknowledgements</h2>
<p>We thank Nick DeVito for assistance with the Evidence-Based Medicine Data Lab COVID-19 TrialsTracker data.
We are grateful to the following contributors for reviewing pieces of the text: Nadia Danilova, James Eberwine and Ipsita Krishnan.</p>
<h1 class="page_break_before" data-number="8" id="references"><span class="header-section-number">8</span> References</h1>
<!-- Explicitly insert bibliography here -->
<div id="refs" class="references hanging-indent" role="doc-bibliography">
<div id="ref-Vb5KFWnO">
<p>1. <strong>Novel Coronavirus (2019-nCoV) SITUATION REPORT - 1</strong><br />
World Health Organization<br />
(2020-01-21) <a href="https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200121-sitrep-1-2019-ncov.pdf">https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200121-sitrep-1-2019-ncov.pdf</a></p>
</div>
<div id="ref-fLTiwToY">
<p>2. <strong>Novel Coronavirus (2019-nCoV) Situation Report - 8</strong><br />
World Health Organization<br />
(2020-01-28) <a href="https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200128-sitrep-8-ncov-cleared.pdf">https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200128-sitrep-8-ncov-cleared.pdf</a></p>
</div>
<div id="ref-x9975iBK">
<p>3. <strong>Novel Coronavirus (2019-nCoV) Situation Report - 51</strong><br />
World Health Organization<br />
(2020-03-11) <a href="https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200311-sitrep-51-covid-19.pdf">https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200311-sitrep-51-covid-19.pdf</a></p>
</div>
<div id="ref-1GACP7OZY">
<p>4. <strong>Novel Coronavirus (2019-nCoV) Situation Report - 75</strong><br />
World Health Organization<br />
(2020-04-04) <a href="https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200404-sitrep-75-covid-19.pdf">https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200404-sitrep-75-covid-19.pdf</a></p>
</div>
<div id="ref-MrwDDw9R">
<p>5. <strong>COVID-19 Data Repository</strong><br />
Center for Systems Science and Engineering at Johns Hopkins University<br />
<em>GitHub</em> <a href="https://github.com/CSSEGISandData/COVID-19/tree/master/csse_covid_19_data/csse_covid_19_time_series">https://github.com/CSSEGISandData/COVID-19/tree/master/csse_covid_19_data/csse_covid_19_time_series</a></p>
</div>
<div id="ref-I2EsJmfs">
<p>6. <strong>COVID-19: Epidemiology, Evolution, and Cross-Disciplinary Perspectives</strong><br />
Jiumeng Sun, Wan-Ting He, Lifang Wang, Alexander Lai, Xiang Ji, Xiaofeng Zhai, Gairu Li, Marc A. Suchard, Jin Tian, Jiyong Zhou, … Shuo Su<br />
<em>Trends in Molecular Medicine</em> (2020-03) <a href="https://doi.org/ggqgwq">https://doi.org/ggqgwq</a><br />
DOI: <a href="https://doi.org/10.1016/j.molmed.2020.02.008">10.1016/j.molmed.2020.02.008</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32359479">32359479</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7118693">PMC7118693</a></p>
</div>
<div id="ref-5x25saIz">
<p>7. <strong>Immunology of COVID-19: Current State of the Science</strong><br />
Nicolas Vabret, Graham J. Britton, Conor Gruber, Samarth Hegde, Joel Kim, Maria Kuksin, Rachel Levantovsky, Louise Malle, Alvaro Moreira, Matthew D. Park, … Uri Laserson<br />
<em>Immunity</em> (2020-06) <a href="https://doi.org/ggt54g">https://doi.org/ggt54g</a><br />
DOI: <a href="https://doi.org/10.1016/j.immuni.2020.05.002">10.1016/j.immuni.2020.05.002</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32505227">32505227</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7200337">PMC7200337</a></p>
</div>
<div id="ref-evtsR3C5">
<p>8. <strong>COVID-19 diagnostics in context</strong><br />
Ralph Weissleder, Hakho Lee, Jina Ko, Mikael J. Pittet<br />
<em>Science Translational Medicine</em> (2020-06-03) <a href="https://doi.org/gg339m">https://doi.org/gg339m</a><br />
DOI: <a href="https://doi.org/10.1126/scitranslmed.abc1931">10.1126/scitranslmed.abc1931</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32493791">32493791</a></p>
</div>
<div id="ref-18eCxyLhx">
<p>9. <strong>Pharmacologic Treatments for Coronavirus Disease 2019 (COVID-19)</strong><br />
James M. Sanders, Marguerite L. Monogue, Tomasz Z. Jodlowski, James B. Cutrell<br />
<em>JAMA</em> (2020-04-13) <a href="https://doi.org/ggr27x">https://doi.org/ggr27x</a><br />
DOI: <a href="https://doi.org/10.1001/jama.2020.6019">10.1001/jama.2020.6019</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32282022">32282022</a></p>
</div>
<div id="ref-SAE5ME3N">
<p>10. <strong>COVID-19 Research in Brief: December, 2019 to June, 2020</strong><br />
Thiago Carvalho<br />
<em>Nature Medicine</em> (2020-06-26) <a href="https://doi.org/gg3kd2">https://doi.org/gg3kd2</a><br />
DOI: <a href="https://doi.org/10.1038/d41591-020-00026-w">10.1038/d41591-020-00026-w</a></p>
</div>
<div id="ref-xOs5ctsW">
<p>11. <strong>Pathophysiology, Transmission, Diagnosis, and Treatment of Coronavirus Disease 2019 (COVID-19)</strong><br />
W. Joost Wiersinga, Andrew Rhodes, Allen C. Cheng, Sharon J. Peacock, Hallie C. Prescott<br />
<em>JAMA</em> (2020-07-10) <a href="https://doi.org/gg4ht4">https://doi.org/gg4ht4</a><br />
DOI: <a href="https://doi.org/10.1001/jama.2020.12839">10.1001/jama.2020.12839</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32648899">32648899</a></p>
</div>
<div id="ref-1HZeeO4Cs">
<p>12. <strong>How you can help with COVID-19 modelling</strong><br />
Julia R. Gog<br />
<em>Nature Reviews Physics</em> (2020-04-08) <a href="https://doi.org/ggrsq3">https://doi.org/ggrsq3</a><br />
DOI: <a href="https://doi.org/10.1038/s42254-020-0175-7">10.1038/s42254-020-0175-7</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7144181">PMC7144181</a></p>
</div>
<div id="ref-1Fqilxaum">
<p>13. <strong>Genomic characterisation and epidemiology of 2019 novel coronavirus: implications for virus origins and receptor binding</strong><br />
Roujian Lu, Xiang Zhao, Juan Li, Peihua Niu, Bo Yang, Honglong Wu, Wenling Wang, Hao Song, Baoying Huang, Na Zhu, … Wenjie Tan<br />
<em>The Lancet</em> (2020-02) <a href="https://doi.org/ggjr43">https://doi.org/ggjr43</a><br />
DOI: <a href="https://doi.org/10.1016/s0140-6736(20)30251-8">10.1016/s0140-6736(20)30251-8</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32007145">32007145</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7159086">PMC7159086</a></p>
</div>
<div id="ref-47hnjvDb">
<p>14. <strong>Coronaviruses: An Overview of Their Replication and Pathogenesis</strong><br />
Anthony R. Fehr, Stanley Perlman<br />
<em>Methods in Molecular Biology</em> (2015) <a href="https://doi.org/ggpc6n">https://doi.org/ggpc6n</a><br />
DOI: <a href="https://doi.org/10.1007/978-1-4939-2438-7_1">10.1007/978-1-4939-2438-7_1</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25720466">25720466</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4369385">PMC4369385</a></p>
</div>
<div id="ref-18GflyMj">
<p>15. <strong>Emerging coronaviruses: Genome structure, replication, and pathogenesis</strong><br />
Yu Chen, Qianyun Liu, Deyin Guo<br />
<em>Journal of Medical Virology</em> (2020-02-07) <a href="https://doi.org/ggjvwj">https://doi.org/ggjvwj</a><br />
DOI: <a href="https://doi.org/10.1002/jmv.25681">10.1002/jmv.25681</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31967327">31967327</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7167049">PMC7167049</a></p>
</div>
<div id="ref-17DSmRo9H">
<p>16. <strong>Structure, Function, and Evolution of Coronavirus Spike Proteins</strong><br />
Fang Li<br />
<em>Annual Review of Virology</em> (2016-09-29) <a href="https://doi.org/ggr7gv">https://doi.org/ggr7gv</a><br />
DOI: <a href="https://doi.org/10.1146/annurev-virology-110615-042301">10.1146/annurev-virology-110615-042301</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27578435">27578435</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5457962">PMC5457962</a></p>
</div>
<div id="ref-IFLf8rsY">
<p>17. <strong>A familial cluster of pneumonia associated with the 2019 novel coronavirus indicating person-to-person transmission: a study of a family cluster</strong><br />
Jasper Fuk-Woo Chan, Shuofeng Yuan, Kin-Hang Kok, Kelvin Kai-Wang To, Hin Chu, Jin Yang, Fanfan Xing, Jieling Liu, Cyril Chik-Yan Yip, Rosana Wing-Shan Poon, … Kwok-Yung Yuen<br />
<em>The Lancet</em> (2020-02) <a href="https://doi.org/ggjs7j">https://doi.org/ggjs7j</a><br />
DOI: <a href="https://doi.org/10.1016/s0140-6736(20)30154-9">10.1016/s0140-6736(20)30154-9</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31986261">31986261</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7159286">PMC7159286</a></p>
</div>
<div id="ref-rDtyU6cv">
<p>18. <strong>Coronavirus 229E-Related Pneumonia in Immunocompromised Patients</strong><br />
F. Pene, A. Merlat, A. Vabret, F. Rozenberg, A. Buzyn, F. Dreyfus, A. Cariou, F. Freymuth, P. Lebon<br />
<em>Clinical Infectious Diseases</em> (2003-10-01) <a href="https://doi.org/dcjk64">https://doi.org/dcjk64</a><br />
DOI: <a href="https://doi.org/10.1086/377612">10.1086/377612</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/13130404">13130404</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7107892">PMC7107892</a></p>
</div>
<div id="ref-UBTUyMGV">
<p>19. <strong>Origin and evolution of pathogenic coronaviruses</strong><br />
Jie Cui, Fang Li, Zheng-Li Shi<br />
<em>Nature Reviews Microbiology</em> (2018-12-10) <a href="https://doi.org/ggh4vb">https://doi.org/ggh4vb</a><br />
DOI: <a href="https://doi.org/10.1038/s41579-018-0118-9">10.1038/s41579-018-0118-9</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30531947">30531947</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7097006">PMC7097006</a></p>
</div>
<div id="ref-YAhVzI0A">
<p>20. <strong>Human Coronaviruses: A Review of Virus–Host Interactions</strong><br />
Yvonne Lim, Yan Ng, James Tam, Ding Liu<br />
<em>Diseases</em> (2016-07-25) <a href="https://doi.org/ggjs23">https://doi.org/ggjs23</a><br />
DOI: <a href="https://doi.org/10.3390/diseases4030026">10.3390/diseases4030026</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28933406">28933406</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5456285">PMC5456285</a></p>
</div>
<div id="ref-G5NJrE75">
<p>21. <strong>SARS and MERS: recent insights into emerging coronaviruses</strong><br />
Emmie de Wit, Neeltje van Doremalen, Darryl Falzarano, Vincent J. Munster<br />
<em>Nature Reviews Microbiology</em> (2016-06-27) <a href="https://doi.org/f8v5cv">https://doi.org/f8v5cv</a><br />
DOI: <a href="https://doi.org/10.1038/nrmicro.2016.81">10.1038/nrmicro.2016.81</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27344959">27344959</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7097822">PMC7097822</a></p>
</div>
<div id="ref-gHWlMufv">
<p>22. <strong>Coronavirus Disease 2019 (COVID-19) – Symptoms</strong><br />
CDC<br />
<em>Centers for Disease Control and Prevention</em> (2020-05-13) <a href="https://www.cdc.gov/coronavirus/2019-ncov/symptoms-testing/symptoms.html">https://www.cdc.gov/coronavirus/2019-ncov/symptoms-testing/symptoms.html</a></p>
</div>
<div id="ref-uXEsLlzb">
<p>23. <strong>Hosts and Sources of Endemic Human Coronaviruses</strong><br />
Victor M. Corman, Doreen Muth, Daniela Niemeyer, Christian Drosten<br />
<em>Advances in Virus Research</em> (2018) <a href="https://doi.org/ggwx4j">https://doi.org/ggwx4j</a><br />
DOI: <a href="https://doi.org/10.1016/bs.aivir.2018.01.001">10.1016/bs.aivir.2018.01.001</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29551135">29551135</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7112090">PMC7112090</a></p>
</div>
<div id="ref-NEIoHDjq">
<p>24. <strong>The proximal origin of SARS-CoV-2</strong><br />
Kristian G. Andersen, Andrew Rambaut, W. Ian Lipkin, Edward C. Holmes, Robert F. Garry<br />
<em>Nature Medicine</em> (2020-03-17) <a href="https://doi.org/ggn4dn">https://doi.org/ggn4dn</a><br />
DOI: <a href="https://doi.org/10.1038/s41591-020-0820-9">10.1038/s41591-020-0820-9</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32284615">32284615</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7095063">PMC7095063</a></p>
</div>
<div id="ref-VSkK7CeP">
<p>25. <strong>A pneumonia outbreak associated with a new coronavirus of probable bat origin</strong><br />
Peng Zhou, Xing-Lou Yang, Xian-Guang Wang, Ben Hu, Lei Zhang, Wei Zhang, Hao-Rui Si, Yan Zhu, Bei Li, Chao-Lin Huang, … Zheng-Li Shi<br />
<em>Nature</em> (2020-02-03) <a href="https://doi.org/ggj5cg">https://doi.org/ggj5cg</a><br />
DOI: <a href="https://doi.org/10.1038/s41586-020-2012-7">10.1038/s41586-020-2012-7</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32015507">32015507</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7095418">PMC7095418</a></p>
</div>
<div id="ref-eQIU1Nbs">
<p>26. <strong>Pangolin homology associated with 2019-nCoV</strong><br />
Tao Zhang, Qunfu Wu, Zhigang Zhang<br />
<em>bioRxiv</em> (2020-02-20) <a href="https://doi.org/ggpvpt">https://doi.org/ggpvpt</a><br />
DOI: <a href="https://doi.org/10.1101/2020.02.19.950253">10.1101/2020.02.19.950253</a></p>
</div>
<div id="ref-vP6yTZ0y">
<p>27. <strong>Transmission routes of 2019-nCoV and controls in dental practice</strong><br />
Xian Peng, Xin Xu, Yuqing Li, Lei Cheng, Xuedong Zhou, Biao Ren<br />
<em>International Journal of Oral Science</em> (2020-03-03) <a href="https://doi.org/ggnf47">https://doi.org/ggnf47</a><br />
DOI: <a href="https://doi.org/10.1038/s41368-020-0075-9">10.1038/s41368-020-0075-9</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32127517">32127517</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7054527">PMC7054527</a></p>
</div>
<div id="ref-16SDeiudC">
<p>28. <strong>Reducing transmission of SARS-CoV-2</strong><br />
Kimberly A. Prather, Chia C. Wang, Robert T. Schooley<br />
<em>Science</em> (2020-06-26) <a href="https://doi.org/ggxp9w">https://doi.org/ggxp9w</a><br />
DOI: <a href="https://doi.org/10.1126/science.abc6197">10.1126/science.abc6197</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32461212">32461212</a></p>
</div>
<div id="ref-yXKcviw8">
<p>29. <strong>Aerosol and Surface Stability of SARS-CoV-2 as Compared with SARS-CoV-1</strong><br />
Neeltje van Doremalen, Trenton Bushmaker, Dylan H. Morris, Myndi G. Holbrook, Amandine Gamble, Brandi N. Williamson, Azaibi Tamin, Jennifer L. Harcourt, Natalie J. Thornburg, Susan I. Gerber, … Vincent J. Munster<br />
<em>New England Journal of Medicine</em> (2020-03-17) <a href="https://doi.org/ggn88w">https://doi.org/ggn88w</a><br />
DOI: <a href="https://doi.org/10.1056/nejmc2004973">10.1056/nejmc2004973</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32182409">32182409</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7121658">PMC7121658</a></p>
</div>
<div id="ref-9ouDX5IN">
<p>30. <strong>It is Time to Address Airborne Transmission of COVID-19</strong><br />
Lidia Morawska, Donald K Milton<br />
<em>Clinical Infectious Diseases</em> (2020-07-06) <a href="https://doi.org/gg34zn">https://doi.org/gg34zn</a><br />
DOI: <a href="https://doi.org/10.1093/cid/ciaa939">10.1093/cid/ciaa939</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32628269">32628269</a></p>
</div>
<div id="ref-rjVw7V94">
<p>31. <strong>Coronavirus Pathogenesis</strong><br />
Susan R. Weiss, Julian L. Leibowitz<br />
<em>Advances in Virus Research</em> (2011) <a href="https://doi.org/ggvvd7">https://doi.org/ggvvd7</a><br />
DOI: <a href="https://doi.org/10.1016/b978-0-12-385885-6.00009-2">10.1016/b978-0-12-385885-6.00009-2</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22094080">22094080</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7149603">PMC7149603</a></p>
</div>
<div id="ref-Oyy1OUTT">
<p>32. <strong>Coronavirus Disease 2019 (COVID-19) - Transmission</strong><br />
CDC<br />
<em>Centers for Disease Control and Prevention</em> (2020-06-16) <a href="https://www.cdc.gov/coronavirus/2019-ncov/prevent-getting-sick/how-covid-spreads.html">https://www.cdc.gov/coronavirus/2019-ncov/prevent-getting-sick/how-covid-spreads.html</a></p>
</div>
<div id="ref-RGAUx68v">
<p>33. <strong>Fields virology</strong><br />
Bernard N. Fields, David M. Knipe, Peter M. Howley (editors)<br />
<em>Wolters Kluwer Health/Lippincott Williams &amp; Wilkins</em> (2007)<br />
ISBN: <a href="https://worldcat.org/isbn/9780781760607">9780781760607</a></p>
</div>
<div id="ref-DlBAegL1">
<p>34. <strong>Important Role for the Transmembrane Domain of Severe Acute Respiratory Syndrome Coronavirus Spike Protein during Entry</strong><br />
Rene Broer, Bertrand Boson, Willy Spaan, François-Loïc Cosset, Jeroen Corver<br />
<em>Journal of Virology</em> (2006-02-01) <a href="https://doi.org/dvvg2h">https://doi.org/dvvg2h</a><br />
DOI: <a href="https://doi.org/10.1128/jvi.80.3.1302-1310.2006">10.1128/jvi.80.3.1302-1310.2006</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16415007">16415007</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1346921">PMC1346921</a></p>
</div>
<div id="ref-skHKTFMe">
<p>35. <strong>Receptor Recognition Mechanisms of Coronaviruses: a Decade of Structural Studies</strong><br />
Fang Li<br />
<em>Journal of Virology</em> (2015-02-15) <a href="https://doi.org/f633jb">https://doi.org/f633jb</a><br />
DOI: <a href="https://doi.org/10.1128/jvi.02615-14">10.1128/jvi.02615-14</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25428871">25428871</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4338876">PMC4338876</a></p>
</div>
<div id="ref-ueMPuUMe">
<p>36. <strong>The spike protein of SARS-CoV — a target for vaccine and therapeutic development</strong><br />
Lanying Du, Yuxian He, Yusen Zhou, Shuwen Liu, Bo-Jian Zheng, Shibo Jiang<br />
<em>Nature Reviews Microbiology</em> (2009-02-09) <a href="https://doi.org/d4tq4t">https://doi.org/d4tq4t</a><br />
DOI: <a href="https://doi.org/10.1038/nrmicro2090">10.1038/nrmicro2090</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19198616">19198616</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2750777">PMC2750777</a></p>
</div>
<div id="ref-13wWdgODZ">
<p>37. <strong>Mechanisms of Coronavirus Cell Entry Mediated by the Viral Spike Protein</strong><br />
Sandrine Belouzard, Jean K. Millet, Beth N. Licitra, Gary R. Whittaker<br />
<em>Viruses</em> (2012-06-20) <a href="https://doi.org/gbbktb">https://doi.org/gbbktb</a><br />
DOI: <a href="https://doi.org/10.3390/v4061011">10.3390/v4061011</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22816037">22816037</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3397359">PMC3397359</a></p>
</div>
<div id="ref-qcVbT0w4">
<p>38. <strong>Structure, Function, and Antigenicity of the SARS-CoV-2 Spike Glycoprotein</strong><br />
Alexandra C. Walls, Young-Jun Park, M. Alejandra Tortorici, Abigail Wall, Andrew T. McGuire, David Veesler<br />
<em>Cell</em> (2020-03) <a href="https://doi.org/dpvh">https://doi.org/dpvh</a><br />
DOI: <a href="https://doi.org/10.1016/j.cell.2020.02.058">10.1016/j.cell.2020.02.058</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32155444">32155444</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7102599">PMC7102599</a></p>
</div>
<div id="ref-kCVy8vjj">
<p>39. <strong>Molecular Interactions in the Assembly of Coronaviruses</strong><br />
Cornelis A. M. de Haan, Peter J. M. Rottier<br />
<em>Advances in Virus Research</em> (2005) <a href="https://doi.org/cf8chz">https://doi.org/cf8chz</a><br />
DOI: <a href="https://doi.org/10.1016/s0065-3527(05)64006-7">10.1016/s0065-3527(05)64006-7</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16139595">16139595</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7112327">PMC7112327</a></p>
</div>
<div id="ref-YeKaMzdJ">
<p>40. <strong>Coronavirus membrane fusion mechanism offers a potential target for antiviral development</strong><br />
Tiffany Tang, Miya Bidon, Javier A. Jaimes, Gary R. Whittaker, Susan Daniel<br />
<em>Antiviral Research</em> (2020-06) <a href="https://doi.org/ggr23b">https://doi.org/ggr23b</a><br />
DOI: <a href="https://doi.org/10.1016/j.antiviral.2020.104792">10.1016/j.antiviral.2020.104792</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32272173">32272173</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7194977">PMC7194977</a></p>
</div>
<div id="ref-2QTH37Xi">
<p>41. <strong>Angiotensin-converting enzyme 2 is a functional receptor for the SARS coronavirus</strong><br />
Wenhui Li, Michael J. Moore, Natalya Vasilieva, Jianhua Sui, Swee Kee Wong, Michael A. Berne, Mohan Somasundaran, John L. Sullivan, Katherine Luzuriaga, Thomas C. Greenough, … Michael Farzan<br />
<em>Nature</em> (2003-11) <a href="https://doi.org/bqvpjh">https://doi.org/bqvpjh</a><br />
DOI: <a href="https://doi.org/10.1038/nature02145">10.1038/nature02145</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/14647384">14647384</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7095016">PMC7095016</a></p>
</div>
<div id="ref-CqQVkaqj">
<p>42. <strong>Cryo-EM structure of the 2019-nCoV spike in the prefusion conformation</strong><br />
Daniel Wrapp, Nianshuang Wang, Kizzmekia S. Corbett, Jory A. Goldsmith, Ching-Lin Hsieh, Olubukola Abiona, Barney S. Graham, Jason S. McLellan<br />
<em>Science</em> (2020-02-19) <a href="https://doi.org/ggmtk2">https://doi.org/ggmtk2</a><br />
DOI: <a href="https://doi.org/10.1126/science.abb2507">10.1126/science.abb2507</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32075877">32075877</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7164637">PMC7164637</a></p>
</div>
<div id="ref-MXWaV7SO">
<p>43. <strong>Efficient Activation of the Severe Acute Respiratory Syndrome Coronavirus Spike Protein by the Transmembrane Protease TMPRSS2</strong><br />
S. Matsuyama, N. Nagata, K. Shirato, M. Kawase, M. Takeda, F. Taguchi<br />
<em>Journal of Virology</em> (2010-10-06) <a href="https://doi.org/d4hnfr">https://doi.org/d4hnfr</a><br />
DOI: <a href="https://doi.org/10.1128/jvi.01542-10">10.1128/jvi.01542-10</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20926566">20926566</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3004351">PMC3004351</a></p>
</div>
<div id="ref-15l3di3Wj">
<p>44. <strong>SARS-CoV-2 Cell Entry Depends on ACE2 and TMPRSS2 and Is Blocked by a Clinically Proven Protease Inhibitor</strong><br />
Markus Hoffmann, Hannah Kleine-Weber, Simon Schroeder, Nadine Krüger, Tanja Herrler, Sandra Erichsen, Tobias S. Schiergens, Georg Herrler, Nai-Huei Wu, Andreas Nitsche, … Stefan Pöhlmann<br />
<em>Cell</em> (2020-03) <a href="https://doi.org/ggnq74">https://doi.org/ggnq74</a><br />
DOI: <a href="https://doi.org/10.1016/j.cell.2020.02.052">10.1016/j.cell.2020.02.052</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32142651">32142651</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7102627">PMC7102627</a></p>
</div>
<div id="ref-JjrO6YbD">
<p>45. <strong>Evidence that TMPRSS2 Activates the Severe Acute Respiratory Syndrome Coronavirus Spike Protein for Membrane Fusion and Reduces Viral Control by the Humoral Immune Response</strong><br />
I. Glowacka, S. Bertram, M. A. Muller, P. Allen, E. Soilleux, S. Pfefferle, I. Steffen, T. S. Tsegaye, Y. He, K. Gnirss, … S. Pohlmann<br />
<em>Journal of Virology</em> (2011-02-16) <a href="https://doi.org/bg97wb">https://doi.org/bg97wb</a><br />
DOI: <a href="https://doi.org/10.1128/jvi.02232-10">10.1128/jvi.02232-10</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21325420">21325420</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3126222">PMC3126222</a></p>
</div>
<div id="ref-FFwktXgx">
<p>46. <strong>SARS coronavirus entry into host cells through a novel clathrin- and caveolae-independent endocytic pathway</strong><br />
Hongliang Wang, Peng Yang, Kangtai Liu, Feng Guo, Yanli Zhang, Gongyi Zhang, Chengyu Jiang<br />
<em>Cell Research</em> (2008-01-29) <a href="https://doi.org/bp9275">https://doi.org/bp9275</a><br />
DOI: <a href="https://doi.org/10.1038/cr.2008.15">10.1038/cr.2008.15</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/18227861">18227861</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7091891">PMC7091891</a></p>
</div>
<div id="ref-ZUuGRxk9">
<p>47. <strong>Coronaviridae ~ ViralZone page</strong><a href="https://viralzone.expasy.org/30?outline">https://viralzone.expasy.org/30?outline</a></p>
</div>
<div id="ref-EGPffQf6">
<p>48. <strong>Evolutionary arms race between virus and host drives genetic diversity in bat SARS related coronavirus spike genes</strong><br />
Hua Guo, Bing-Jie Hu, Xing-Lou Yang, Lei-Ping Zeng, Bei Li, Song-Ying Ouyang, Zheng-Li Shi<br />
<em>bioRxiv</em> (2020-05-13) <a href="https://doi.org/d3dh">https://doi.org/d3dh</a><br />
DOI: <a href="https://doi.org/10.1101/2020.05.13.093658">10.1101/2020.05.13.093658</a></p>
</div>
<div id="ref-Ovrrh22d">
<p>49. <strong>The Red Queen’s Crown: an evolutionary arms race between coronaviruses and mammalian species reflected in positive selection of the ACE2 receptor among many species</strong><br />
Mehrdad Hajibabaei, Gregory A. C. Singer<br />
<em>bioRxiv</em> (2020-05-14) <a href="https://doi.org/gg562h">https://doi.org/gg562h</a><br />
DOI: <a href="https://doi.org/10.1101/2020.05.14.096131">10.1101/2020.05.14.096131</a></p>
</div>
<div id="ref-ZBvTot1X">
<p>50. <strong>Adaptive evolution of bat dipeptidyl peptidase 4 (dpp4): implications for the origin and emergence of Middle East respiratory syndrome coronavirus</strong><br />
Jie Cui, John-Sebastian Eden, Edward C Holmes, Lin-Fa Wang<br />
<em>Virology Journal</em> (2013-10-10) <a href="https://doi.org/gbd8fm">https://doi.org/gbd8fm</a><br />
DOI: <a href="https://doi.org/10.1186/1743-422x-10-304">10.1186/1743-422x-10-304</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24107353">24107353</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3852826">PMC3852826</a></p>
</div>
<div id="ref-OVsxrEuX">
<p>51. <strong>Phylogenetic Analysis and Structural Modeling of SARS-CoV-2 Spike Protein Reveals an Evolutionary Distinct and Proteolytically Sensitive Activation Loop</strong><br />
Javier A. Jaimes, Nicole M. André, Joshua S. Chappie, Jean K. Millet, Gary R. Whittaker<br />
<em>Journal of Molecular Biology</em> (2020-05) <a href="https://doi.org/ggtxhr">https://doi.org/ggtxhr</a><br />
DOI: <a href="https://doi.org/10.1016/j.jmb.2020.04.009">10.1016/j.jmb.2020.04.009</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32320687">32320687</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7166309">PMC7166309</a></p>
</div>
<div id="ref-jsMmdH4D">
<p>52. <strong>Medical microbiology</strong><br />
Samuel Baron (editor)<br />
<em>University of Texas Medical Branch at Galveston</em> (1996)<br />
ISBN: <a href="https://worldcat.org/isbn/9780963117212">9780963117212</a></p>
</div>
<div id="ref-icy0PeD7">
<p>53. <strong>1. Overview of the human immune response</strong><br />
D CHAPLIN<br />
<em>Journal of Allergy and Clinical Immunology</em> (2006-02) <a href="https://doi.org/b6zghf">https://doi.org/b6zghf</a><br />
DOI: <a href="https://doi.org/10.1016/j.jaci.2005.09.034">10.1016/j.jaci.2005.09.034</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16455341">16455341</a></p>
</div>
<div id="ref-OlEbK4fc">
<p>54. <strong>Molecular immune pathogenesis and diagnosis of COVID-19</strong><br />
Xiaowei Li, Manman Geng, Yizhao Peng, Liesu Meng, Shemin Lu<br />
<em>Journal of Pharmaceutical Analysis</em> (2020-03) <a href="https://doi.org/ggppqg">https://doi.org/ggppqg</a><br />
DOI: <a href="https://doi.org/10.1016/j.jpha.2020.03.001">10.1016/j.jpha.2020.03.001</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32282863">32282863</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7104082">PMC7104082</a></p>
</div>
<div id="ref-15IDA5kX6">
<p>55. <strong>Clinical Characteristics of Coronavirus Disease 2019 in China</strong><br />
Wei-jie Guan, Zheng-yi Ni, Yu Hu, Wen-hua Liang, Chun-quan Ou, Jian-xing He, Lei Liu, Hong Shan, Chun-liang Lei, David S. C. Hui, … Nan-shan Zhong<br />
<em>New England Journal of Medicine</em> (2020-02-28) <a href="https://doi.org/ggm6dh">https://doi.org/ggm6dh</a><br />
DOI: <a href="https://doi.org/10.1056/nejmoa2002032">10.1056/nejmoa2002032</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32109013">32109013</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7092819">PMC7092819</a></p>
</div>
<div id="ref-10THxyeCg">
<p>56. <strong>Clinical course and risk factors for mortality of adult inpatients with COVID-19 in Wuhan, China: a retrospective cohort study</strong><br />
Fei Zhou, Ting Yu, Ronghui Du, Guohui Fan, Ying Liu, Zhibo Liu, Jie Xiang, Yeming Wang, Bin Song, Xiaoying Gu, … Bin Cao<br />
<em>The Lancet</em> (2020-03) <a href="https://doi.org/ggnxb3">https://doi.org/ggnxb3</a><br />
DOI: <a href="https://doi.org/10.1016/s0140-6736(20)30566-3">10.1016/s0140-6736(20)30566-3</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32171076">32171076</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7270627">PMC7270627</a></p>
</div>
<div id="ref-19ytsiSpq">
<p>57. <strong>Presenting Characteristics, Comorbidities, and Outcomes Among 5700 Patients Hospitalized With COVID-19 in the New York City Area</strong><br />
Safiya Richardson, Jamie S. Hirsch, Mangala Narasimhan, James M. Crawford, Thomas McGinn, Karina W. Davidson, Douglas P. Barnaby, Lance B. Becker, John D. Chelico, Stuart L. Cohen, … and the Northwell COVID-19 Research Consortium<br />
<em>JAMA</em> (2020-05-26) <a href="https://doi.org/ggsrkd">https://doi.org/ggsrkd</a><br />
DOI: <a href="https://doi.org/10.1001/jama.2020.6775">10.1001/jama.2020.6775</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32320003">32320003</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7177629">PMC7177629</a></p>
</div>
<div id="ref-1H8gSsSkn">
<p>58. <strong>Clinical Characteristics of Covid-19 in New York City</strong><br />
Parag Goyal, Justin J. Choi, Laura C. Pinheiro, Edward J. Schenck, Ruijun Chen, Assem Jabri, Michael J. Satlin, Thomas R. Campion, Musarrat Nahid, Joanna B. Ringel, … Monika M. Safford<br />
<em>New England Journal of Medicine</em> (2020-06-11) <a href="https://doi.org/ggtsjc">https://doi.org/ggtsjc</a><br />
DOI: <a href="https://doi.org/10.1056/nejmc2010419">10.1056/nejmc2010419</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32302078">32302078</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7182018">PMC7182018</a></p>
</div>
<div id="ref-xZBEFXiG">
<p>59. <strong>Hospitalization Rates and Characteristics of Patients Hospitalized with Laboratory-Confirmed Coronavirus Disease 2019 — COVID-NET, 14 States, March 1–30, 2020</strong><br />
Shikha Garg, Lindsay Kim, Michael Whitaker, Alissa O’Halloran, Charisse Cummings, Rachel Holstein, Mila Prill, Shua J. Chai, Pam D. Kirley, Nisha B. Alden, … Alicia Fry<br />
<em>MMWR. Morbidity and Mortality Weekly Report</em> (2020-04-17) <a href="https://doi.org/ggsppz">https://doi.org/ggsppz</a><br />
DOI: <a href="https://doi.org/10.15585/mmwr.mm6915e3">10.15585/mmwr.mm6915e3</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32298251">32298251</a></p>
</div>
<div id="ref-11xfoqpEl">
<p>60. <strong>Characteristics of and Important Lessons From the Coronavirus Disease 2019 (COVID-19) Outbreak in China</strong><br />
Zunyou Wu, Jennifer M. McGoogan<br />
<em>JAMA</em> (2020-02-24) <a href="https://doi.org/ggmq43">https://doi.org/ggmq43</a><br />
DOI: <a href="https://doi.org/10.1001/jama.2020.2648">10.1001/jama.2020.2648</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32091533">32091533</a></p>
</div>
<div id="ref-qlJ8hCUs">
<p>61. <strong>Critical Care Utilization for the COVID-19 Outbreak in Lombardy, Italy</strong><br />
Giacomo Grasselli, Antonio Pesenti, Maurizio Cecconi<br />
<em>JAMA</em> (2020-03-13) <a href="https://doi.org/ggqf6g">https://doi.org/ggqf6g</a><br />
DOI: <a href="https://doi.org/10.1001/jama.2020.4031">10.1001/jama.2020.4031</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32167538">32167538</a></p>
</div>
<div id="ref-wK2afyL8">
<p>62. <strong>Extrapulmonary manifestations of COVID-19</strong><br />
Aakriti Gupta, Mahesh V. Madhavan, Kartik Sehgal, Nandini Nair, Shiwani Mahajan, Tejasav S. Sehrawat, Behnood Bikdeli, Neha Ahluwalia, John C. Ausiello, Elaine Y. Wan, … Donald W. Landry<br />
<em>Nature Medicine</em> (2020-07-10) <a href="https://doi.org/gg4r37">https://doi.org/gg4r37</a><br />
DOI: <a href="https://doi.org/10.1038/s41591-020-0968-3">10.1038/s41591-020-0968-3</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32651579">32651579</a></p>
</div>
<div id="ref-17hdoiBo9">
<p>63. <strong>Acute kidney injury in patients hospitalized with COVID-19</strong><br />
Jamie S. Hirsch, Jia H. Ng, Daniel W. Ross, Purva Sharma, Hitesh H. Shah, Richard L. Barnett, Azzour D. Hazzan, Steven Fishbane, Kenar D. Jhaveri, Mersema Abate, … Jia Hwei. Ng<br />
<em>Kidney International</em> (2020-07) <a href="https://doi.org/ggx24k">https://doi.org/ggx24k</a><br />
DOI: <a href="https://doi.org/10.1016/j.kint.2020.05.006">10.1016/j.kint.2020.05.006</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32416116">32416116</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7229463">PMC7229463</a></p>
</div>
<div id="ref-1CqoZGSKK">
<p>64. <strong>Nervous system involvement after infection with COVID-19 and other coronaviruses</strong><br />
Yeshun Wu, Xiaolin Xu, Zijun Chen, Jiahao Duan, Kenji Hashimoto, Ling Yang, Cunming Liu, Chun Yang<br />
<em>Brain, Behavior, and Immunity</em> (2020-07) <a href="https://doi.org/ggq7s2">https://doi.org/ggq7s2</a><br />
DOI: <a href="https://doi.org/10.1016/j.bbi.2020.03.031">10.1016/j.bbi.2020.03.031</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32240762">32240762</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7146689">PMC7146689</a></p>
</div>
<div id="ref-1EF9EpbZ5">
<p>65. <strong>Neurological associations of COVID-19</strong><br />
Mark A Ellul, Laura Benjamin, Bhagteshwar Singh, Suzannah Lant, Benedict Daniel Michael, Ava Easton, Rachel Kneen, Sylviane Defres, Jim Sejvar, Tom Solomon<br />
<em>The Lancet Neurology</em> (2020-07) <a href="https://doi.org/d259">https://doi.org/d259</a><br />
DOI: <a href="https://doi.org/10.1016/s1474-4422(20)30221-0">10.1016/s1474-4422(20)30221-0</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32622375">32622375</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7332267">PMC7332267</a></p>
</div>
<div id="ref-nBGGUV9s">
<p>66. <strong>Update on the neurology of COVID‐19</strong><br />
Josef Finsterer, Claudia Stollberger<br />
<em>Journal of Medical Virology</em> (2020-06-02) <a href="https://doi.org/gg2qnn">https://doi.org/gg2qnn</a><br />
DOI: <a href="https://doi.org/10.1002/jmv.26000">10.1002/jmv.26000</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32401352">32401352</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7272942">PMC7272942</a></p>
</div>
<div id="ref-LLm9d62d">
<p>67. <strong>‐19: A Global Threat to the Nervous System</strong><br />
Igor J. Koralnik, Kenneth L. Tyler<br />
<em>Annals of Neurology</em> (2020-06-23) <a href="https://doi.org/gg3hzh">https://doi.org/gg3hzh</a><br />
DOI: <a href="https://doi.org/10.1002/ana.25807">10.1002/ana.25807</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32506549">32506549</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7300753">PMC7300753</a></p>
</div>
<div id="ref-aRNqjkMg">
<p>68. <strong>Large-Vessel Stroke as a Presenting Feature of Covid-19 in the Young</strong><br />
Thomas J. Oxley, J. Mocco, Shahram Majidi, Christopher P. Kellner, Hazem Shoirah, I. Paul Singh, Reade A. De Leacy, Tomoyoshi Shigematsu, Travis R. Ladner, Kurt A. Yaeger, … Johanna T. Fifi<br />
<em>New England Journal of Medicine</em> (2020-05-14) <a href="https://doi.org/ggtsjg">https://doi.org/ggtsjg</a><br />
DOI: <a href="https://doi.org/10.1056/nejmc2009787">10.1056/nejmc2009787</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32343504">32343504</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7207073">PMC7207073</a></p>
</div>
<div id="ref-3UBbDrG2">
<p>69. <strong>Incidence of thrombotic complications in critically ill ICU patients with COVID-19</strong><br />
F. A. Klok, M. J. H. A. Kruip, N. J. M. van der Meer, M. S. Arbous, D. A. M. P. J. Gommers, K. M. Kant, F. H. J. Kaptein, J. van Paassen, M. A. M. Stals, M. V. Huisman, H. Endeman<br />
<em>Thrombosis Research</em> (2020-07) <a href="https://doi.org/dt2q">https://doi.org/dt2q</a><br />
DOI: <a href="https://doi.org/10.1016/j.thromres.2020.04.013">10.1016/j.thromres.2020.04.013</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32291094">32291094</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7146714">PMC7146714</a></p>
</div>
<div id="ref-8OnbWuhF">
<p>70. <strong>Coagulopathy and Antiphospholipid Antibodies in Patients with Covid-19</strong><br />
Yan Zhang, Meng Xiao, Shulan Zhang, Peng Xia, Wei Cao, Wei Jiang, Huan Chen, Xin Ding, Hua Zhao, Hongmin Zhang, … Shuyang Zhang<br />
<em>New England Journal of Medicine</em> (2020-04-23) <a href="https://doi.org/ggrgz7">https://doi.org/ggrgz7</a><br />
DOI: <a href="https://doi.org/10.1056/nejmc2007575">10.1056/nejmc2007575</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32268022">32268022</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7161262">PMC7161262</a></p>
</div>
<div id="ref-dUfws1q0">
<p>71. <strong>Abnormal coagulation parameters are associated with poor prognosis in patients with novel coronavirus pneumonia</strong><br />
Ning Tang, Dengju Li, Xiong Wang, Ziyong Sun<br />
<em>Journal of Thrombosis and Haemostasis</em> (2020-04) <a href="https://doi.org/ggqxf6">https://doi.org/ggqxf6</a><br />
DOI: <a href="https://doi.org/10.1111/jth.14768">10.1111/jth.14768</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32073213">32073213</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7166509">PMC7166509</a></p>
</div>
<div id="ref-p5EhN9Qd">
<p>72. <strong>Review: Viral infections and mechanisms of thrombosis and bleeding</strong><br />
M. Goeijenbier, M. van Wissen, C. van de Weg, E. Jong, V. E. A. Gerdes, J. C. M. Meijers, D. P. M. Brandjes, E. C. M. van Gorp<br />
<em>Journal of Medical Virology</em> (2012-10) <a href="https://doi.org/f37tfr">https://doi.org/f37tfr</a><br />
DOI: <a href="https://doi.org/10.1002/jmv.23354">10.1002/jmv.23354</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22930518">22930518</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7166625">PMC7166625</a></p>
</div>
<div id="ref-v1EIzwfx">
<p>73. <strong>Immune mechanisms of pulmonary intravascular coagulopathy in COVID-19 pneumonia</strong><br />
Dennis McGonagle, James S O’Donnell, Kassem Sharif, Paul Emery, Charles Bridgewood<br />
<em>The Lancet Rheumatology</em> (2020-07) <a href="https://doi.org/ggvd74">https://doi.org/ggvd74</a><br />
DOI: <a href="https://doi.org/10.1016/s2665-9913(20)30121-1">10.1016/s2665-9913(20)30121-1</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7252093">PMC7252093</a></p>
</div>
<div id="ref-18AiyvhO8">
<p>74. <strong>“War to the knife” against thromboinflammation to protect endothelial function of COVID-19 patients</strong><br />
Gabriele Guglielmetti, Marco Quaglia, Pier Paolo Sainaghi, Luigi Mario Castello, Rosanna Vaschetto, Mario Pirisi, Francesco Della Corte, Gian Carlo Avanzi, Piero Stratta, Vincenzo Cantaluppi<br />
<em>Critical Care</em> (2020-06-19) <a href="https://doi.org/gg35w7">https://doi.org/gg35w7</a><br />
DOI: <a href="https://doi.org/10.1186/s13054-020-03060-9">10.1186/s13054-020-03060-9</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32560665">32560665</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7303575">PMC7303575</a></p>
</div>
<div id="ref-PyKMLraw">
<p>75. <strong>COVID-19 update: Covid-19-associated coagulopathy</strong><br />
Richard C. Becker<br />
<em>Journal of Thrombosis and Thrombolysis</em> (2020-05-15) <a href="https://doi.org/ggwpp5">https://doi.org/ggwpp5</a><br />
DOI: <a href="https://doi.org/10.1007/s11239-020-02134-3">10.1007/s11239-020-02134-3</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32415579">32415579</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7225095">PMC7225095</a></p>
</div>
<div id="ref-14dyYQY7s">
<p>76. <strong>The complement system in COVID-19: friend and foe?</strong><br />
Anuja Java, Anthony J. Apicelli, M. Kathryn Liszewski, Ariella Coler-Reilly, John P. Atkinson, Alfred H. J. Kim, Hrishikesh S. Kulkarni<br />
<em>JCI Insight</em> (2020-06-18) <a href="https://doi.org/gg4b5b">https://doi.org/gg4b5b</a><br />
DOI: <a href="https://doi.org/10.1172/jci.insight.140711">10.1172/jci.insight.140711</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32554923">32554923</a></p>
</div>
<div id="ref-17y6YeJ6R">
<p>77. <strong>COVID-19, microangiopathy, hemostatic activation, and complement</strong><br />
Wen-Chao Song, Garret A. FitzGerald<br />
<em>Journal of Clinical Investigation</em> (2020-06-22) <a href="https://doi.org/gg4b5c">https://doi.org/gg4b5c</a><br />
DOI: <a href="https://doi.org/10.1172/jci140183">10.1172/jci140183</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32459663">32459663</a></p>
</div>
<div id="ref-1BhN2uq3f">
<p>78. <strong>Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) Infection in Children and Adolescents</strong><br />
Riccardo Castagnoli, Martina Votto, Amelia Licari, Ilaria Brambilla, Raffaele Bruno, Stefano Perlini, Francesca Rovida, Fausto Baldanti, Gian Luigi Marseglia<br />
<em>JAMA Pediatrics</em> (2020-04-22) <a href="https://doi.org/dswz">https://doi.org/dswz</a><br />
DOI: <a href="https://doi.org/10.1001/jamapediatrics.2020.1467">10.1001/jamapediatrics.2020.1467</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32320004">32320004</a></p>
</div>
<div id="ref-12lVfu2Qe">
<p>79. <strong>Neurologic and Radiographic Findings Associated With COVID-19 Infection in Children</strong><br />
Omar Abdel-Mannan, Michael Eyre, Ulrike Löbel, Alasdair Bamford, Christin Eltze, Biju Hameed, Cheryl Hemingway, Yael Hacohen<br />
<em>JAMA Neurology</em> (2020-07-01) <a href="https://doi.org/gg339f">https://doi.org/gg339f</a><br />
DOI: <a href="https://doi.org/10.1001/jamaneurol.2020.2687">10.1001/jamaneurol.2020.2687</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32609336">32609336</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7330822">PMC7330822</a></p>
</div>
<div id="ref-MCVe25tf">
<p>80. <strong>COVID-19 in 7780 pediatric patients: A systematic review</strong><br />
Ansel Hoang, Kevin Chorath, Axel Moreira, Mary Evans, Finn Burmeister-Morton, Fiona Burmeister, Rija Naqvi, Matthew Petershack, Alvaro Moreira<br />
<em>EClinicalMedicine</em> (2020-07) <a href="https://doi.org/gg4hn2">https://doi.org/gg4hn2</a><br />
DOI: <a href="https://doi.org/10.1016/j.eclinm.2020.100433">10.1016/j.eclinm.2020.100433</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7318942">PMC7318942</a></p>
</div>
<div id="ref-fZwrv8M">
<p>81. <strong>Multisystem Inflammatory Syndrome in Children During the Coronavirus 2019 Pandemic: A Case Series</strong><br />
Kathleen Chiotos, Hamid Bassiri, Edward M Behrens, Allison M Blatz, Joyce Chang, Caroline Diorio, Julie C Fitzgerald, Alexis Topjian, Audrey R Odom John<br />
<em>Journal of the Pediatric Infectious Diseases Society</em> (2020-07) <a href="https://doi.org/ggx4pd">https://doi.org/ggx4pd</a><br />
DOI: <a href="https://doi.org/10.1093/jpids/piaa069">10.1093/jpids/piaa069</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32463092">32463092</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7313950">PMC7313950</a></p>
</div>
<div id="ref-AmvwCWm3">
<p>82. <strong>Clinical Characteristics of 58 Children With a Pediatric Inflammatory Multisystem Syndrome Temporally Associated With SARS-CoV-2</strong><br />
Elizabeth Whittaker, Alasdair Bamford, Julia Kenny, Myrsini Kaforou, Christine E. Jones, Priyen Shah, Padmanabhan Ramnarayan, Alain Fraisse, Owen Miller, Patrick Davies, … for the PIMS-TS Study Group and EUCLIDS and PERFORM Consortia<br />
<em>JAMA</em> (2020-07-21) <a href="https://doi.org/gg2v75">https://doi.org/gg2v75</a><br />
DOI: <a href="https://doi.org/10.1001/jama.2020.10369">10.1001/jama.2020.10369</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32511692">32511692</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7281356">PMC7281356</a></p>
</div>
<div id="ref-68JorBeU">
<p>83. <strong>Toxic shock-like syndrome and COVID-19: A case report of multisystem inflammatory syndrome in children (MIS-C)</strong><br />
Andrea G. Greene, Mona Saleh, Eric Roseman, Richard Sinert<br />
<em>The American Journal of Emergency Medicine</em> (2020-06) <a href="https://doi.org/gg2586">https://doi.org/gg2586</a><br />
DOI: <a href="https://doi.org/10.1016/j.ajem.2020.05.117">10.1016/j.ajem.2020.05.117</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32532619">32532619</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7274960">PMC7274960</a></p>
</div>
<div id="ref-FYgN6gUL">
<p>84. <strong>Acute heart failure in multisystem inflammatory syndrome in children (MIS-C) in the context of global SARS-CoV-2 pandemic</strong><br />
Zahra Belhadjer, Mathilde Méot, Fanny Bajolle, Diala Khraiche, Antoine Legendre, Samya Abakka, Johanne Auriau, Marion Grimaud, Mehdi Oualha, Maurice Beghetti, … Damien Bonnet<br />
<em>Circulation</em> (2020-05-17) <a href="https://doi.org/ggwkv6">https://doi.org/ggwkv6</a><br />
DOI: <a href="https://doi.org/10.1161/circulationaha.120.048360">10.1161/circulationaha.120.048360</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32418446">32418446</a></p>
</div>
<div id="ref-71To4laE">
<p>85. <strong>An adult with Kawasaki-like multisystem inflammatory syndrome associated with COVID-19</strong><br />
Sheila Shaigany, Marlis Gnirke, Allison Guttmann, Hong Chong, Shane Meehan, Vanessa Raabe, Eddie Louie, Bruce Solitar, Alisa Femia<br />
<em>The Lancet</em> (2020-07) <a href="https://doi.org/gg4sd6">https://doi.org/gg4sd6</a><br />
DOI: <a href="https://doi.org/10.1016/s0140-6736(20)31526-9">10.1016/s0140-6736(20)31526-9</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32659211">32659211</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7351414">PMC7351414</a></p>
</div>
<div id="ref-UBDAxWWv">
<p>86. <strong>COVID-19 associated Kawasaki-like multisystem inflammatory disease in an adult</strong><br />
Sabrina Sokolovsky, Parita Soni, Taryn Hoffman, Philip Kahn, Joshua Scheers-Masters<br />
<em>The American Journal of Emergency Medicine</em> (2020-06) <a href="https://doi.org/gg5tf4">https://doi.org/gg5tf4</a><br />
DOI: <a href="https://doi.org/10.1016/j.ajem.2020.06.053">10.1016/j.ajem.2020.06.053</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32631771">32631771</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7315983">PMC7315983</a></p>
</div>
<div id="ref-1D6lVXWbJ">
<p>87. <strong>Disparities In Outcomes Among COVID-19 Patients In A Large Health Care System In California</strong><br />
Kristen M. J. Azar, Zijun Shen, Robert J. Romanelli, Stephen H. Lockhart, Kelly Smits, Sarah Robinson, Stephanie Brown, Alice R. Pressman<br />
<em>Health Affairs</em> (2020-07-01) <a href="https://doi.org/ggx4mf">https://doi.org/ggx4mf</a><br />
DOI: <a href="https://doi.org/10.1377/hlthaff.2020.00598">10.1377/hlthaff.2020.00598</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32437224">32437224</a></p>
</div>
<div id="ref-Zk5CfYeW">
<p>88. <strong>Characteristics Associated with Hospitalization Among Patients with COVID-19 — Metropolitan Atlanta, Georgia, March–April 2020</strong><br />
Marie E. Killerby, Ruth Link-Gelles, Sarah C. Haight, Caroline A. Schrodt, Lucinda England, Danica J. Gomes, Mays Shamout, Kristen Pettrone, Kevin O’Laughlin, Anne Kimball, … CDC COVID-19 Response Clinical Team<br />
<em>MMWR. Morbidity and Mortality Weekly Report</em> (2020-06-26) <a href="https://doi.org/gg3k6h">https://doi.org/gg3k6h</a><br />
DOI: <a href="https://doi.org/10.15585/mmwr.mm6925e1">10.15585/mmwr.mm6925e1</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32584797">32584797</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7316317">PMC7316317</a></p>
</div>
<div id="ref-yK5bczRb">
<p>89. <strong>COVID-19 and Racial/Ethnic Disparities</strong><br />
Monica Webb Hooper, Anna María Nápoles, Eliseo J. Pérez-Stable<br />
<em>JAMA</em> (2020-06-23) <a href="https://doi.org/ggvzqn">https://doi.org/ggvzqn</a><br />
DOI: <a href="https://doi.org/10.1001/jama.2020.8598">10.1001/jama.2020.8598</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32391864">32391864</a></p>
</div>
<div id="ref-u4MMV8aJ">
<p>90. <strong>Covid-19: Black people and other minorities are hardest hit in US</strong><br />
Owen Dyer<br />
<em>BMJ</em> (2020-04-14) <a href="https://doi.org/ggv5br">https://doi.org/ggv5br</a><br />
DOI: <a href="https://doi.org/10.1136/bmj.m1483">10.1136/bmj.m1483</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32291262">32291262</a></p>
</div>
<div id="ref-JAo2QxY">
<p>91. <strong>Susceptibility of Southwestern American Indian Tribes to Coronavirus Disease 2019 (COVID‐19)</strong><br />
Monika Kakol, Dona Upson, Akshay Sood<br />
<em>The Journal of Rural Health</em> (2020-06) <a href="https://doi.org/ggtzkq">https://doi.org/ggtzkq</a><br />
DOI: <a href="https://doi.org/10.1111/jrh.12451">10.1111/jrh.12451</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32304251">32304251</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7264672">PMC7264672</a></p>
</div>
<div id="ref-12WZObWRf">
<p>92. <strong>The Fullest Look Yet at the Racial Inequity of Coronavirus</strong><br />
Richard A. Oppel Jr, Robert Gebeloff, K. K. Rebecca Lai, Will Wright, Mitch Smith<br />
<em>The New York Times</em> (2020-07-05) <a href="https://www.nytimes.com/interactive/2020/07/05/us/coronavirus-latinos-african-americans-cdc-data.html">https://www.nytimes.com/interactive/2020/07/05/us/coronavirus-latinos-african-americans-cdc-data.html</a></p>
</div>
<div id="ref-Pp8GWnQm">
<p>93. <strong>Addressing inequities in COVID-19 morbidity and mortality: research and policy recommendations</strong><br />
Monica L Wang, Pamela Behrman, Akilah Dulin, Monica L Baskin, Joanna Buscemi, Kassandra I Alcaraz, Carly M Goldstein, Tiffany L Carson, Megan Shen, Marian Fitzgibbon<br />
<em>Translational Behavioral Medicine</em> (2020-06-16) <a href="https://doi.org/gg3389">https://doi.org/gg3389</a><br />
DOI: <a href="https://doi.org/10.1093/tbm/ibaa055">10.1093/tbm/ibaa055</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32542349">32542349</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7337775">PMC7337775</a></p>
</div>
<div id="ref-ODm1CaEq">
<p>94. <strong>Protect Indigenous peoples from COVID-19</strong><br />
Lucas Ferrante, Philip M. Fearnside<br />
<em>Science</em> (2020-04-16) <a href="https://doi.org/gg3k6f">https://doi.org/gg3k6f</a><br />
DOI: <a href="https://doi.org/10.1126/science.abc0073">10.1126/science.abc0073</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32299940">32299940</a></p>
</div>
<div id="ref-fxDYekIg">
<p>95. <strong>Disparities in Coronavirus 2019 Reported Incidence, Knowledge, and Behavior Among US Adults</strong><br />
Marcella Alsan, Stefanie Stantcheva, David Yang, David Cutler<br />
<em>JAMA Network Open</em> (2020-06-18) <a href="https://doi.org/gg3388">https://doi.org/gg3388</a><br />
DOI: <a href="https://doi.org/10.1001/jamanetworkopen.2020.12403">10.1001/jamanetworkopen.2020.12403</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32556260">32556260</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7303811">PMC7303811</a></p>
</div>
<div id="ref-5BMDcXz3">
<p>96. <strong>Exposure to air pollution and COVID-19 mortality in the United States: A nationwide cross-sectional study</strong><br />
Xiao Wu, Rachel C. Nethery, Benjamin M. Sabath, Danielle Braun, Francesca Dominici<br />
<em>medRxiv</em> (2020-04-27) <a href="https://doi.org/ggrpcj">https://doi.org/ggrpcj</a><br />
DOI: <a href="https://doi.org/10.1101/2020.04.05.20054502">10.1101/2020.04.05.20054502</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32511651">32511651</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7277007">PMC7277007</a></p>
</div>
<div id="ref-vImXUPNS">
<p>97. <strong>Global estimates of mortality associated with long-term exposure to outdoor fine particulate matter</strong><br />
Richard Burnett, Hong Chen, Mieczysław Szyszkowicz, Neal Fann, Bryan Hubbell, C. Arden Pope, Joshua S. Apte, Michael Brauer, Aaron Cohen, Scott Weichenthal, … Joseph V. Spadaro<br />
<em>Proceedings of the National Academy of Sciences</em> (2018-09-18) <a href="https://doi.org/gfgbcx">https://doi.org/gfgbcx</a><br />
DOI: <a href="https://doi.org/10.1073/pnas.1803222115">10.1073/pnas.1803222115</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30181279">30181279</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6156628">PMC6156628</a></p>
</div>
<div id="ref-aM2XPE3J">
<p>98. <strong>Mortality, Admissions, and Patient Census at SNFs in 3 US Cities During the COVID-19 Pandemic</strong><br />
Michael L. Barnett, Lissy Hu, Thomas Martin, David C. Grabowski<br />
<em>JAMA</em> (2020-06-24) <a href="https://doi.org/gg3387">https://doi.org/gg3387</a><br />
DOI: <a href="https://doi.org/10.1001/jama.2020.11642">10.1001/jama.2020.11642</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32579161">32579161</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7315390">PMC7315390</a></p>
</div>
<div id="ref-BeedQx3I">
<p>99. <strong>COVID-19 in Prisons and Jails in the United States</strong><br />
Laura Hawks, Steffie Woolhandler, Danny McCormick<br />
<em>JAMA Internal Medicine</em> (2020-04-28) <a href="https://doi.org/ggtxw6">https://doi.org/ggtxw6</a><br />
DOI: <a href="https://doi.org/10.1001/jamainternmed.2020.1856">10.1001/jamainternmed.2020.1856</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32343355">32343355</a></p>
</div>
<div id="ref-2uUcva6Y">
<p>100. <strong>COVID-19 Cases and Deaths in Federal and State Prisons</strong><br />
Brendan Saloner, Kalind Parish, Julie A. Ward, Grace DiLaura, Sharon Dolovich<br />
<em>JAMA</em> (2020-07-08) <a href="https://doi.org/gg4dcv">https://doi.org/gg4dcv</a><br />
DOI: <a href="https://doi.org/10.1001/jama.2020.12528">10.1001/jama.2020.12528</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32639537">32639537</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7344796">PMC7344796</a></p>
</div>
<div id="ref-HihYbn6k">
<p>101. <strong>Differential occupational risk for COVID‐19 and other infection exposure according to race and ethnicity</strong><br />
Devan Hawkins<br />
<em>American Journal of Industrial Medicine</em> (2020-06-15) <a href="https://doi.org/gg3rb2">https://doi.org/gg3rb2</a><br />
DOI: <a href="https://doi.org/10.1002/ajim.23145">10.1002/ajim.23145</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32539166">32539166</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7323065">PMC7323065</a></p>
</div>
<div id="ref-F4laSOXM">
<p>102. <strong>Estimating the burden of United States workers exposed to infection or disease: A key factor in containing risk of COVID-19 infection</strong><br />
Marissa G. Baker, Trevor K. Peckham, Noah S. Seixas<br />
<em>PLOS ONE</em> (2020-04-28) <a href="https://doi.org/ggtx7c">https://doi.org/ggtx7c</a><br />
DOI: <a href="https://doi.org/10.1371/journal.pone.0232452">10.1371/journal.pone.0232452</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32343747">32343747</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7188235">PMC7188235</a></p>
</div>
<div id="ref-hAYohR8m">
<p>103. <strong>Coronavirus (COVID-19) related deaths by occupation, England and Wales: deaths registered up to and including 20 April 2020</strong><br />
Ben Windsor-Shellard, Jasveer Kaur<br />
(2020-05-11) <a href="https://www.ons.gov.uk/peoplepopulationandcommunity/healthandsocialcare/causesofdeath/bulletins/coronaviruscovid19relateddeathsbyoccupationenglandandwales/deathsregistereduptoandincluding20april2020">https://www.ons.gov.uk/peoplepopulationandcommunity/healthandsocialcare/causesofdeath/bulletins/coronaviruscovid19relateddeathsbyoccupationenglandandwales/deathsregistereduptoandincluding20april2020</a></p>
</div>
<div id="ref-DOyiuaup">
<p>104. <strong>Which occupations have the highest potential exposure to the coronavirus (COVID-19)?</strong><br />
Office for National Statistics<br />
(2020-05-11) <a href="https://www.ons.gov.uk/employmentandlabourmarket/peopleinwork/employmentandemployeetypes/articles/whichoccupationshavethehighestpotentialexposuretothecoronaviruscovid19/2020-05-11">https://www.ons.gov.uk/employmentandlabourmarket/peopleinwork/employmentandemployeetypes/articles/whichoccupationshavethehighestpotentialexposuretothecoronaviruscovid19/2020-05-11</a></p>
</div>
<div id="ref-13qkVV6eN">
<p>105. <strong>Disparities in the risk and outcomes from COVID-19</strong><br />
Public Health England<br />
(2020-06-12) <a href="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/892085/disparities_review.pdf">https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/892085/disparities_review.pdf</a></p>
</div>
<div id="ref-owevcpwb">
<p>106. <strong>Genomewide Association Study of Severe Covid-19 with Respiratory Failure</strong><br />
David Ellinghaus, Frauke Degenhardt, Luis Bujanda, Maria Buti, Agustín Albillos, Pietro Invernizzi, Javier Fernández, Daniele Prati, Guido Baselli, Rosanna Asselta, … Tom H. Karlsen<br />
<em>New England Journal of Medicine</em> (2020-06-17) <a href="https://doi.org/gg2pqx">https://doi.org/gg2pqx</a><br />
DOI: <a href="https://doi.org/10.1056/nejmoa2020283">10.1056/nejmoa2020283</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32558485">32558485</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7315890">PMC7315890</a></p>
</div>
<div id="ref-nn67UTjV">
<p>107. <strong>APOE e4 Genotype Predicts Severe COVID-19 in the UK Biobank Community Cohort</strong><br />
Chia-Ling Kuo, Luke C Pilling, Janice L Atkins, Jane AH Masoli, João Delgado, George A Kuchel, David Melzer<br />
<em>The Journals of Gerontology: Series A</em> (2020-05-26) <a href="https://doi.org/ggx4ng">https://doi.org/ggx4ng</a><br />
DOI: <a href="https://doi.org/10.1093/gerona/glaa131">10.1093/gerona/glaa131</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32451547">32451547</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7314139">PMC7314139</a></p>
</div>
<div id="ref-I2mIU0wn">
<p>108. <strong>Genome-wide CRISPR screen reveals host genes that regulate SARS-CoV-2 infection</strong><br />
Jin Wei, Mia Madel Alfajaro, Ruth E. Hanna, Peter C. DeWeirdt, Madison S. Strine, William J. Lu-Culligan, Shang-Min Zhang, Vincent R. Graziano, Cameron O. Schmitz, Jennifer S. Chen, … Craig B. Wilen<br />
<em>bioRxiv</em> (2020-06-17) <a href="https://doi.org/dzz3">https://doi.org/dzz3</a><br />
DOI: <a href="https://doi.org/10.1101/2020.06.16.155101">10.1101/2020.06.16.155101</a></p>
</div>
<div id="ref-9NfXSHjH">
<p>109. <strong>Association of Blood Glucose Control and Outcomes in Patients with COVID-19 and Pre-existing Type 2 Diabetes</strong><br />
Lihua Zhu, Zhi-Gang She, Xu Cheng, Juan-Juan Qin, Xiao-Jing Zhang, Jingjing Cai, Fang Lei, Haitao Wang, Jing Xie, Wenxin Wang, … Hongliang Li<br />
<em>Cell Metabolism</em> (2020-06) <a href="https://doi.org/ggvcc9">https://doi.org/ggvcc9</a><br />
DOI: <a href="https://doi.org/10.1016/j.cmet.2020.04.021">10.1016/j.cmet.2020.04.021</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32369736">32369736</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7252168">PMC7252168</a></p>
</div>
<div id="ref-13BpwZaaq">
<p>110. <strong>Diabetes increases the mortality of patients with COVID-19: a meta-analysis</strong><br />
Zeng-hong Wu, Yun Tang, Qing Cheng<br />
<em>Acta Diabetologica</em> (2020-06-24) <a href="https://doi.org/gg3k55">https://doi.org/gg3k55</a><br />
DOI: <a href="https://doi.org/10.1007/s00592-020-01546-0">10.1007/s00592-020-01546-0</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32583078">32583078</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7311595">PMC7311595</a></p>
</div>
<div id="ref-UkCdtLwF">
<p>111. <strong>COVID-19 infection may cause ketosis and ketoacidosis</strong><br />
Juyi Li, Xiufang Wang, Jian Chen, Xiuran Zuo, Hongmei Zhang, Aiping Deng<br />
<em>Diabetes, Obesity and Metabolism</em> (2020-05-18) <a href="https://doi.org/ggv4tm">https://doi.org/ggv4tm</a><br />
DOI: <a href="https://doi.org/10.1111/dom.14057">10.1111/dom.14057</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32314455">32314455</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7264681">PMC7264681</a></p>
</div>
<div id="ref-OT5GEOej">
<p>112. <strong>COVID-19 pandemic, coronaviruses, and diabetes mellitus</strong><br />
Ranganath Muniyappa, Sriram Gubbi<br />
<em>American Journal of Physiology-Endocrinology and Metabolism</em> (2020-05-01) <a href="https://doi.org/ggq79v">https://doi.org/ggq79v</a><br />
DOI: <a href="https://doi.org/10.1152/ajpendo.00124.2020">10.1152/ajpendo.00124.2020</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32228322">32228322</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7191633">PMC7191633</a></p>
</div>
<div id="ref-ZLmNMDrD">
<p>113. <strong>Severe obesity, increasing age and male sex are independently associated with worse in-hospital outcomes, and higher in-hospital mortality, in a cohort of patients with COVID-19 in the Bronx, New York</strong><br />
Leonidas Palaiodimos, Damianos G. Kokkinidis, Weijia Li, Dimitrios Karamanis, Jennifer Ognibene, Shitij Arora, William N. Southern, Christos S. Mantzoros<br />
<em>Metabolism</em> (2020-07) <a href="https://doi.org/ggx229">https://doi.org/ggx229</a><br />
DOI: <a href="https://doi.org/10.1016/j.metabol.2020.154262">10.1016/j.metabol.2020.154262</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32422233">32422233</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7228874">PMC7228874</a></p>
</div>
<div id="ref-2Ahpn6R2">
<p>114. <strong>Features of 16,749 hospitalised UK patients with COVID-19 using the ISARIC WHO Clinical Characterisation Protocol</strong><br />
Annemarie B Docherty, Ewen M Harrison, Christopher A Green, Hayley E Hardwick, Riinu Pius, Lisa Norman, Karl A Holden, Jonathan M Read, Frank Dondelinger, Gail Carson, … Malcolm Gracie Semple<br />
<em>medRxiv</em> (2020-04-28) <a href="https://doi.org/ggtdtb">https://doi.org/ggtdtb</a><br />
DOI: <a href="https://doi.org/10.1101/2020.04.23.20076042">10.1101/2020.04.23.20076042</a></p>
</div>
<div id="ref-1AmBg80Un">
<p>115. <strong>When Two Pandemics Meet: Why Is Obesity Associated with Increased COVID-19 Mortality?</strong><br />
Sam M. Lockhart, Stephen O’Rahilly<br />
<em>Med</em> (2020-06) <a href="https://doi.org/gg3k57">https://doi.org/gg3k57</a><br />
DOI: <a href="https://doi.org/10.1016/j.medj.2020.06.005">10.1016/j.medj.2020.06.005</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7323660">PMC7323660</a></p>
</div>
<div id="ref-17gnxmOnf">
<p>116. <strong>OpenSAFELY: factors associated with COVID-19 death in 17 million patients</strong><br />
Elizabeth J. Williamson, Alex J. Walker, Krishnan Bhaskaran, Seb Bacon, Chris Bates, Caroline E. Morton, Helen J. Curtis, Amir Mehrkar, David Evans, Peter Inglesby, … Ben Goldacre<br />
<em>Nature</em> (2020-07-08) <a href="https://doi.org/gg39n7">https://doi.org/gg39n7</a><br />
DOI: <a href="https://doi.org/10.1038/s41586-020-2521-4">10.1038/s41586-020-2521-4</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32640463">32640463</a></p>
</div>
<div id="ref-xYVGtq0L">
<p>117. <strong>An Epidemic in the Midst of a Pandemic: Opioid Use Disorder and COVID-19</strong><br />
G. Caleb Alexander, Kenneth B. Stoller, Rebecca L. Haffajee, Brendan Saloner<br />
<em>Annals of Internal Medicine</em> (2020-07-07) <a href="https://doi.org/ggq7t2">https://doi.org/ggq7t2</a><br />
DOI: <a href="https://doi.org/10.7326/m20-1141">10.7326/m20-1141</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32240283">32240283</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7138407">PMC7138407</a></p>
</div>
<div id="ref-qyQhe3zX">
<p>118. <strong>Compound climate risks in the COVID-19 pandemic</strong><br />
Carly A. Phillips, Astrid Caldas, Rachel Cleetus, Kristina A. Dahl, Juan Declet-Barreto, Rachel Licker, L. Delta Merner, J. Pablo Ortiz-Partida, Alexandra L. Phelan, Erika Spanger-Siegfried, … Colin J. Carlson<br />
<em>Nature Climate Change</em> (2020-05-15) <a href="https://doi.org/ggv7m5">https://doi.org/ggv7m5</a><br />
DOI: <a href="https://doi.org/10.1038/s41558-020-0804-2">10.1038/s41558-020-0804-2</a></p>
</div>
<div id="ref-o9LSGYWO">
<p>119. <strong>COVID-19: Compounding the health-related harms of human trafficking</strong><br />
Richard Armitage, Laura B Nellums<br />
<em>EClinicalMedicine</em> (2020-07) <a href="https://doi.org/ggzqf5">https://doi.org/ggzqf5</a><br />
DOI: <a href="https://doi.org/10.1016/j.eclinm.2020.100409">10.1016/j.eclinm.2020.100409</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7274576">PMC7274576</a></p>
</div>
<div id="ref-7TVDQB9o">
<p>120. <strong>Report 19: The potential impact of the COVID-19 epidemic on HIV, TB and malaria in low- and middle-income countries</strong><br />
A Hogan, B Jewell, E Sherrard-Smith, J Vesga, O Watson, C Whittaker, A Hamlet, J Smith, K Ainslie, M Baguelin, … T Hallett<br />
<em>Imperial College London</em> (2020-05-01) <a href="https://doi.org/gg339b">https://doi.org/gg339b</a><br />
DOI: <a href="https://doi.org/10.25561/78670">10.25561/78670</a></p>
</div>
<div id="ref-vr7AH83b">
<p>121. <strong>Detection of SARS-CoV-2 in Different Types of Clinical Specimens</strong><br />
Wenling Wang, Yanli Xu, Ruqin Gao, Roujian Lu, Kai Han, Guizhen Wu, Wenjie Tan<br />
<em>JAMA</em> (2020-03-11) <a href="https://doi.org/ggpp6h">https://doi.org/ggpp6h</a><br />
DOI: <a href="https://doi.org/10.1001/jama.2020.3786">10.1001/jama.2020.3786</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32159775">32159775</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7066521">PMC7066521</a></p>
</div>
<div id="ref-azgJqujy">
<p>122. <strong>Epidemiologic Features and Clinical Course of Patients Infected With SARS-CoV-2 in Singapore</strong><br />
Barnaby Edward Young, Sean Wei Xiang Ong, Shirin Kalimuddin, Jenny G. Low, Seow Yen Tan, Jiashen Loh, Oon-Tek Ng, Kalisvar Marimuthu, Li Wei Ang, Tze Minn Mak, … for the Singapore 2019 Novel Coronavirus Outbreak Research Team<br />
<em>JAMA</em> (2020-03-03) <a href="https://doi.org/ggnb37">https://doi.org/ggnb37</a><br />
DOI: <a href="https://doi.org/10.1001/jama.2020.3204">10.1001/jama.2020.3204</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32125362">32125362</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7054855">PMC7054855</a></p>
</div>
<div id="ref-F7nSMvZk">
<p>123. <strong>Persistence and clearance of viral RNA in 2019 novel coronavirus disease rehabilitation patients</strong><br />
Yun Ling, Shui-Bao Xu, Yi-Xiao Lin, Di Tian, Zhao-Qin Zhu, Fa-Hui Dai, Fan Wu, Zhi-Gang Song, Wei Huang, Jun Chen, … Hong-Zhou Lu<br />
<em>Chinese Medical Journal</em> (2020-02) <a href="https://doi.org/ggnnz8">https://doi.org/ggnnz8</a><br />
DOI: <a href="https://doi.org/10.1097/cm9.0000000000000774">10.1097/cm9.0000000000000774</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32118639">32118639</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7147278">PMC7147278</a></p>
</div>
<div id="ref-1AJFJxzmJ">
<p>124. <strong>Postmortem examination of COVID‐19 patients reveals diffuse alveolar damage with severe capillary congestion and variegated findings in lungs and other organs suggesting vascular dysfunction</strong><br />
Thomas Menter, Jasmin D Haslbauer, Ronny Nienhold, Spasenija Savic, Helmut Deigendesch, Stephan Frank, Daniel Turek, Niels Willi, Hans Pargger, Stefano Bassetti, … Alexandar Tzankov<br />
<em>Histopathology</em> (2020-07-05) <a href="https://doi.org/ggwr32">https://doi.org/ggwr32</a><br />
DOI: <a href="https://doi.org/10.1111/his.14134">10.1111/his.14134</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32364264">32364264</a></p>
</div>
<div id="ref-5gnCuPzp">
<p>125. <strong>Association of Cardiac Injury With Mortality in Hospitalized Patients With COVID-19 in Wuhan, China</strong><br />
Shaobo Shi, Mu Qin, Bo Shen, Yuli Cai, Tao Liu, Fan Yang, Wei Gong, Xu Liu, Jinjun Liang, Qinyan Zhao, … Congxin Huang<br />
<em>JAMA Cardiology</em> (2020-03-25) <a href="https://doi.org/ggq8qf">https://doi.org/ggq8qf</a><br />
DOI: <a href="https://doi.org/10.1001/jamacardio.2020.0950">10.1001/jamacardio.2020.0950</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32211816">32211816</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7097841">PMC7097841</a></p>
</div>
<div id="ref-5ET1D3cK">
<p>126. <strong>The need for urogenital tract monitoring in COVID-19</strong><br />
Shangqian Wang, Xiang Zhou, Tongtong Zhang, Zengjun Wang<br />
<em>Nature Reviews Urology</em> (2020-04-20) <a href="https://doi.org/ggv4xb">https://doi.org/ggv4xb</a><br />
DOI: <a href="https://doi.org/10.1038/s41585-020-0319-7">10.1038/s41585-020-0319-7</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32313110">32313110</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7186932">PMC7186932</a></p>
</div>
<div id="ref-w5B6qRKv">
<p>127. <strong>Acute kidney injury in SARS-CoV-2 infected patients</strong><br />
Vito Fanelli, Marco Fiorentino, Vincenzo Cantaluppi, Loreto Gesualdo, Giovanni Stallone, Claudio Ronco, Giuseppe Castellano<br />
<em>Critical Care</em> (2020-04-16) <a href="https://doi.org/ggv45f">https://doi.org/ggv45f</a><br />
DOI: <a href="https://doi.org/10.1186/s13054-020-02872-z">10.1186/s13054-020-02872-z</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32299479">32299479</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7161433">PMC7161433</a></p>
</div>
<div id="ref-TTSFlLVC">
<p>128. <strong>Liver injury in COVID-19: management and challenges</strong><br />
Chao Zhang, Lei Shi, Fu-Sheng Wang<br />
<em>The Lancet Gastroenterology &amp; Hepatology</em> (2020-03) <a href="https://doi.org/ggpx6s">https://doi.org/ggpx6s</a><br />
DOI: <a href="https://doi.org/10.1016/s2468-1253(20)30057-1">10.1016/s2468-1253(20)30057-1</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32145190">32145190</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7129165">PMC7129165</a></p>
</div>
<div id="ref-PmE9xedP">
<p>129. <strong>Evidence for gastrointestinal infection of SARS-CoV-2</strong><br />
Fei Xiao, Meiwen Tang, Xiaobin Zheng, Ye Liu, Xiaofeng Li, Hong Shan<br />
<em>Gastroenterology</em> (2020-03) <a href="https://doi.org/ggpx27">https://doi.org/ggpx27</a><br />
DOI: <a href="https://doi.org/10.1053/j.gastro.2020.02.055">10.1053/j.gastro.2020.02.055</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32142773">32142773</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7130181">PMC7130181</a></p>
</div>
<div id="ref-3Ak4Mata">
<p>130. <strong>2019 Novel coronavirus infection and gastrointestinal tract</strong><br />
Qin Yan Gao, Ying Xuan Chen, Jing Yuan Fang<br />
<em>Journal of Digestive Diseases</em> (2020-03-10) <a href="https://doi.org/ggqr86">https://doi.org/ggqr86</a><br />
DOI: <a href="https://doi.org/10.1111/1751-2980.12851">10.1111/1751-2980.12851</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32096611">32096611</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7162053">PMC7162053</a></p>
</div>
<div id="ref-13CJl7iIO">
<p>131. <strong>Managing epidemics: key facts about major deadly diseases.</strong><br />
World Health Organization<br />
<em>World Health Organization</em> (2018)<br />
ISBN: <a href="https://worldcat.org/isbn/9789241565530">9789241565530</a></p>
</div>
<div id="ref-15EIBRhef">
<p>132. <strong>Structure of SARS Coronavirus Spike Receptor-Binding Domain Complexed with Receptor</strong><br />
F. Li<br />
<em>Science</em> (2005-09-16) <a href="https://doi.org/fww324">https://doi.org/fww324</a><br />
DOI: <a href="https://doi.org/10.1126/science.1116480">10.1126/science.1116480</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16166518">16166518</a></p>
</div>
<div id="ref-12TZ7hPMA">
<p>133. <strong>Structural basis for the recognition of SARS-CoV-2 by full-length human ACE2</strong><br />
Renhong Yan, Yuanyuan Zhang, Yaning Li, Lu Xia, Yingying Guo, Qiang Zhou<br />
<em>Science</em> (2020-03-27) <a href="https://doi.org/ggpxc8">https://doi.org/ggpxc8</a><br />
DOI: <a href="https://doi.org/10.1126/science.abb2762">10.1126/science.abb2762</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32132184">32132184</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7164635">PMC7164635</a></p>
</div>
<div id="ref-wCbhn23d">
<p>134. <strong>Structural basis of receptor recognition by SARS-CoV-2</strong><br />
Jian Shang, Gang Ye, Ke Shi, Yushun Wan, Chuming Luo, Hideki Aihara, Qibin Geng, Ashley Auerbach, Fang Li<br />
<em>Nature</em> (2020-03-30) <a href="https://doi.org/ggqspv">https://doi.org/ggqspv</a><br />
DOI: <a href="https://doi.org/10.1038/s41586-020-2179-y">10.1038/s41586-020-2179-y</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32225175">32225175</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7328981">PMC7328981</a></p>
</div>
<div id="ref-VX8OWaGj">
<p>135. <strong>Crystal structure of the 2019-nCoV spike receptor-binding domain bound with the ACE2 receptor</strong><br />
Jun Lan, Jiwan Ge, Jinfang Yu, Sisi Shan, Huan Zhou, Shilong Fan, Qi Zhang, Xuanling Shi, Qisheng Wang, Linqi Zhang, Xinquan Wang<br />
<em>bioRxiv</em> (2020-02-20) <a href="https://doi.org/ggqzp5">https://doi.org/ggqzp5</a><br />
DOI: <a href="https://doi.org/10.1101/2020.02.19.956235">10.1101/2020.02.19.956235</a></p>
</div>
<div id="ref-15Q2XgkK7">
<p>136. <strong>Structural and Functional Basis of SARS-CoV-2 Entry by Using Human ACE2</strong><br />
Qihui Wang, Yanfang Zhang, Lili Wu, Sheng Niu, Chunli Song, Zengyuan Zhang, Guangwen Lu, Chengpeng Qiao, Yu Hu, Kwok-Yung Yuen, … Jianxun Qi<br />
<em>Cell</em> (2020-05) <a href="https://doi.org/ggr2cz">https://doi.org/ggr2cz</a><br />
DOI: <a href="https://doi.org/10.1016/j.cell.2020.03.045">10.1016/j.cell.2020.03.045</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32275855">32275855</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7144619">PMC7144619</a></p>
</div>
<div id="ref-t1e4CW9A">
<p>137. <strong>Receptor recognition by novel coronavirus from Wuhan: An analysis based on decade-long structural studies of SARS</strong><br />
Yushun Wan, Jian Shang, Rachel Graham, Ralph S. Baric, Fang Li<br />
<em>Journal of Virology</em> (2020-01-29) <a href="https://doi.org/ggjvwn">https://doi.org/ggjvwn</a><br />
DOI: <a href="https://doi.org/10.1128/jvi.00127-20">10.1128/jvi.00127-20</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31996437">31996437</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7081895">PMC7081895</a></p>
</div>
<div id="ref-UwVweB2M">
<p>138. <strong>Increasing Host Cellular Receptor—Angiotensin-Converting Enzyme 2 (ACE2) Expression by Coronavirus may Facilitate 2019-nCoV Infection</strong><br />
Pei-Hui Wang, Yun Cheng<br />
<em>bioRxiv</em> (2020-02-27) <a href="https://doi.org/ggscwd">https://doi.org/ggscwd</a><br />
DOI: <a href="https://doi.org/10.1101/2020.02.24.963348">10.1101/2020.02.24.963348</a></p>
</div>
<div id="ref-So6ceUwB">
<p>139. <strong>Angiotensin-converting enzyme 2 (ACE2) as a SARS-CoV-2 receptor: molecular mechanisms and potential therapeutic target</strong><br />
Haibo Zhang, Josef M. Penninger, Yimin Li, Nanshan Zhong, Arthur S. Slutsky<br />
<em>Intensive Care Medicine</em> (2020-03-03) <a href="https://doi.org/ggpx6p">https://doi.org/ggpx6p</a><br />
DOI: <a href="https://doi.org/10.1007/s00134-020-05985-9">10.1007/s00134-020-05985-9</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32125455">32125455</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7079879">PMC7079879</a></p>
</div>
<div id="ref-JOJ2n3gC">
<p>140. <strong>Characterization of spike glycoprotein of SARS-CoV-2 on virus entry and its immune cross-reactivity with SARS-CoV</strong><br />
Xiuyuan Ou, Yan Liu, Xiaobo Lei, Pei Li, Dan Mi, Lili Ren, Li Guo, Ruixuan Guo, Ting Chen, Jiaxin Hu, … Zhaohui Qian<br />
<em>Nature Communications</em> (2020-03-27) <a href="https://doi.org/ggqsrf">https://doi.org/ggqsrf</a><br />
DOI: <a href="https://doi.org/10.1038/s41467-020-15562-9">10.1038/s41467-020-15562-9</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32221306">32221306</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7100515">PMC7100515</a></p>
</div>
<div id="ref-NsORsLig">
<p>141. <strong>Structure of the Hemagglutinin Precursor Cleavage Site, a Determinant of Influenza Pathogenicity and the Origin of the Labile Conformation</strong><br />
Jue Chen, Kon Ho Lee, David A Steinhauer, David J Stevens, John J Skehel, Don C Wiley<br />
<em>Cell</em> (1998-10) <a href="https://doi.org/bvgh5b">https://doi.org/bvgh5b</a><br />
DOI: <a href="https://doi.org/10.1016/s0092-8674(00)81771-7">10.1016/s0092-8674(00)81771-7</a></p>
</div>
<div id="ref-vhHB3yyS">
<p>142. <strong>Role of Hemagglutinin Cleavage for the Pathogenicity of Influenza Virus</strong><br />
David A. Steinhauer<br />
<em>Virology</em> (1999-05) <a href="https://doi.org/fw3jz4">https://doi.org/fw3jz4</a><br />
DOI: <a href="https://doi.org/10.1006/viro.1999.9716">10.1006/viro.1999.9716</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/10329563">10329563</a></p>
</div>
<div id="ref-3HdlV9Vf">
<p>143. <strong>COVID-19: consider cytokine storm syndromes and immunosuppression</strong><br />
Puja Mehta, Daniel F McAuley, Michael Brown, Emilie Sanchez, Rachel S Tattersall, Jessica J Manson<br />
<em>The Lancet</em> (2020-03) <a href="https://doi.org/ggnzmc">https://doi.org/ggnzmc</a><br />
DOI: <a href="https://doi.org/10.1016/s0140-6736(20)30628-0">10.1016/s0140-6736(20)30628-0</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32192578">32192578</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7270045">PMC7270045</a></p>
</div>
<div id="ref-cyUG1Zi2">
<p>144. <strong>Cytokine Storms: Understanding COVID-19</strong><br />
Nilam Mangalmurti, Christopher A. Hunter<br />
<em>Immunity</em> (2020-07) <a href="https://doi.org/gg4fd7">https://doi.org/gg4fd7</a><br />
DOI: <a href="https://doi.org/10.1016/j.immuni.2020.06.017">10.1016/j.immuni.2020.06.017</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32610079">32610079</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7321048">PMC7321048</a></p>
</div>
<div id="ref-Xcrxlxnp">
<p>145. <strong>Is a “Cytokine Storm” Relevant to COVID-19?</strong><br />
Pratik Sinha, Michael A. Matthay, Carolyn S. Calfee<br />
<em>JAMA Internal Medicine</em> (2020-06-30) <a href="https://doi.org/gg3k6r">https://doi.org/gg3k6r</a><br />
DOI: <a href="https://doi.org/10.1001/jamainternmed.2020.3313">10.1001/jamainternmed.2020.3313</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32602883">32602883</a></p>
</div>
<div id="ref-14l4fvU71">
<p>146. <strong>Modeling infectious diseases in humans and animals</strong><br />
Matthew James Keeling, Pejman Rohani<br />
<em>Princeton University Press</em> (2008)<br />
ISBN: <a href="https://worldcat.org/isbn/9780691116174">9780691116174</a></p>
</div>
<div id="ref-16CD4voW1">
<p>147. <strong>The concept of R <sub>o</sub> in epidemic theory</strong><br />
J. A. P. Heesterbeek, K. Dietz<br />
<em>Statistica Neerlandica</em> (1996-03) <a href="https://doi.org/d29ch4">https://doi.org/d29ch4</a><br />
DOI: <a href="https://doi.org/10.1111/j.1467-9574.1996.tb01482.x">10.1111/j.1467-9574.1996.tb01482.x</a></p>
</div>
<div id="ref-YubbhU7G">
<p>148. <strong>Modeling infectious disease dynamics</strong><br />
Sarah Cobey<br />
<em>Science</em> (2020-05-15) <a href="https://doi.org/ggsztw">https://doi.org/ggsztw</a><br />
DOI: <a href="https://doi.org/10.1126/science.abb5659">10.1126/science.abb5659</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32332062">32332062</a></p>
</div>
<div id="ref-JRaVgcNd">
<p>149. <strong>Theoretical ecology: principles and applications</strong><br />
Robert M. May, Angela R. McLean (editors)<br />
<em>Oxford University Press</em> (2007)<br />
ISBN: <a href="https://worldcat.org/isbn/9780199209989">9780199209989</a></p>
</div>
<div id="ref-1Aozf7KBW">
<p>150. <strong>A contribution to the mathematical theory of epidemics</strong><br />
William Ogilvy Kermack, A. G. McKendrick<br />
<em>Proceedings of the Royal Society of London. Series A, Containing Papers of a Mathematical and Physical Character</em> (1927-08) <a href="https://doi.org/10.1098/rspa.1927.0118">https://doi.org/10.1098/rspa.1927.0118</a><br />
DOI: <a href="https://doi.org/10.1098/rspa.1927.0118">10.1098/rspa.1927.0118</a></p>
</div>
<div id="ref-SPInPew0">
<p>151. <strong>Population biology of infectious diseases: Part I</strong><br />
Roy M. Anderson, Robert M. May<br />
<em>Nature</em> (1979-08) <a href="https://doi.org/b6z9hc">https://doi.org/b6z9hc</a><br />
DOI: <a href="https://doi.org/10.1038/280361a0">10.1038/280361a0</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/460412">460412</a></p>
</div>
<div id="ref-1E0r4uZy9">
<p>152. <strong>Nowcasting and forecasting the potential domestic and international spread of the 2019-nCoV outbreak originating in Wuhan, China: a modelling study</strong><br />
Joseph T Wu, Kathy Leung, Gabriel M Leung<br />
<em>The Lancet</em> (2020-01) <a href="https://doi.org/ggjvr7">https://doi.org/ggjvr7</a><br />
DOI: <a href="https://doi.org/10.1016/s0140-6736(20)30260-9">10.1016/s0140-6736(20)30260-9</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32014114">32014114</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7159271">PMC7159271</a></p>
</div>
<div id="ref-LHtuVmaq">
<p>153. <strong>The reproductive number of COVID-19 is higher compared to SARS coronavirus</strong><br />
Ying Liu, Albert A Gayle, Annelies Wilder-Smith, Joacim Rocklöv<br />
<em>Journal of Travel Medicine</em> (2020-02-13) <a href="https://doi.org/ggnntv">https://doi.org/ggnntv</a><br />
DOI: <a href="https://doi.org/10.1093/jtm/taaa021">10.1093/jtm/taaa021</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32052846">32052846</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7074654">PMC7074654</a></p>
</div>
<div id="ref-mHYmt0mv">
<p>154. <strong>Substantial undocumented infection facilitates the rapid dissemination of novel coronavirus (SARS-CoV2)</strong><br />
Ruiyun Li, Sen Pei, Bin Chen, Yimeng Song, Tao Zhang, Wan Yang, Jeffrey Shaman<br />
<em>Science</em> (2020-03-16) <a href="https://doi.org/ggn6c2">https://doi.org/ggn6c2</a><br />
DOI: <a href="https://doi.org/10.1126/science.abb3221">10.1126/science.abb3221</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32179701">32179701</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7164387">PMC7164387</a></p>
</div>
<div id="ref-vITui6ac">
<p>155. <strong>Epidemiological parameters of coronavirus disease 2019: a pooled analysis of publicly reported individual data of 1155 cases from seven countries</strong><br />
Shujuan Ma, Jiayue Zhang, Minyan Zeng, Qingping Yun, Wei Guo, Yixiang Zheng, Shi Zhao, Maggie H Wang, Zuyao Yang<br />
<em>medRxiv</em> (2020-03-24) <a href="https://doi.org/ggqhzz">https://doi.org/ggqhzz</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.21.20040329">10.1101/2020.03.21.20040329</a></p>
</div>
<div id="ref-nxM0rP5R">
<p>156. <strong>Early Transmissibility Assessment of a Novel Coronavirus in Wuhan, China</strong><br />
Maimuna Majumder, Kenneth D. Mandl<br />
<em>SSRN Electronic Journal</em> (2020) <a href="https://doi.org/ggqhz3">https://doi.org/ggqhz3</a><br />
DOI: <a href="https://doi.org/10.2139/ssrn.3524675">10.2139/ssrn.3524675</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32714102">32714102</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7366781">PMC7366781</a></p>
</div>
<div id="ref-WLc2UMgQ">
<p>157. <strong>Time-varying transmission dynamics of Novel Coronavirus Pneumonia in China</strong><br />
Tao Liu, Jianxiong Hu, Jianpeng Xiao, Guanhao He, Min Kang, Zuhua Rong, Lifeng Lin, Haojie Zhong, Qiong Huang, Aiping Deng, … Wenjun Ma<br />
<em>bioRxiv</em> (2020-02-13) <a href="https://doi.org/dkx9">https://doi.org/dkx9</a><br />
DOI: <a href="https://doi.org/10.1101/2020.01.25.919787">10.1101/2020.01.25.919787</a></p>
</div>
<div id="ref-10bBqMHH7">
<p>158. <strong>Estimation of the reproductive number of novel coronavirus (COVID-19) and the probable outbreak size on the Diamond Princess cruise ship: A data-driven analysis</strong><br />
Sheng Zhang, MengYuan Diao, Wenbo Yu, Lei Pei, Zhaofen Lin, Dechang Chen<br />
<em>International Journal of Infectious Diseases</em> (2020-04) <a href="https://doi.org/ggpx56">https://doi.org/ggpx56</a><br />
DOI: <a href="https://doi.org/10.1016/j.ijid.2020.02.033">10.1016/j.ijid.2020.02.033</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32097725">32097725</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7110591">PMC7110591</a></p>
</div>
<div id="ref-ITh0Anof">
<p>159. <strong>Estimation of the Transmission Risk of the 2019-nCoV and Its Implication for Public Health Interventions</strong><br />
Tang, Wang, Li, Bragazzi, Tang, Xiao, Wu<br />
<em>Journal of Clinical Medicine</em> (2020-02-07) <a href="https://doi.org/ggmkf4">https://doi.org/ggmkf4</a><br />
DOI: <a href="https://doi.org/10.3390/jcm9020462">10.3390/jcm9020462</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32046137">32046137</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7074281">PMC7074281</a></p>
</div>
<div id="ref-1BMU7sKbs">
<p>160. <strong>Estimating the effective reproduction number of the 2019-nCoV in China</strong><br />
Zhidong Cao, Qingpeng Zhang, Xin Lu, Dirk Pfeiffer, Zhongwei Jia, Hongbing Song, Daniel Dajun Zeng<br />
<em>medRxiv</em> (2020-01) <a href="https://www.medrxiv.org/content/10.1101/2020.01.27.20018952v1">https://www.medrxiv.org/content/10.1101/2020.01.27.20018952v1</a><br />
DOI: <a href="https://doi.org/10.1101/2020.01.27.20018952">10.1101/2020.01.27.20018952</a></p>
</div>
<div id="ref-Yj8Xh4Wz">
<p>161. <strong>Modelling the epidemic trend of the 2019 novel coronavirus outbreak in China</strong><br />
Mingwang Shen, Zhihang Peng, Yanni Xiao, Lei Zhang<br />
<em>bioRxiv</em> (2020-01-25) <a href="https://doi.org/ggqhzw">https://doi.org/ggqhzw</a><br />
DOI: <a href="https://doi.org/10.1101/2020.01.23.916726">10.1101/2020.01.23.916726</a></p>
</div>
<div id="ref-ZzrrVDoE">
<p>162. <strong>Novel coronavirus 2019-nCoV: early estimation of epidemiological parameters and epidemic predictions</strong><br />
Jonathan M Read, Jessica RE Bridgen, Derek AT Cummings, Antonia Ho, Chris P Jewell<br />
<em>medRxiv</em> (2020-01-28) <a href="https://doi.org/dkzb">https://doi.org/dkzb</a><br />
DOI: <a href="https://doi.org/10.1101/2020.01.23.20018549">10.1101/2020.01.23.20018549</a></p>
</div>
<div id="ref-itj26agd">
<p>163. <strong>Using early data to estimate the actual infection fatality ratio from COVID-19 in France</strong><br />
Lionel Roques, Etienne Klein, Julien Papaix, Antoine Sar, Samuel Soubeyrand<br />
<em>medRxiv</em> (2020-05-07) <a href="https://doi.org/ggqhz2">https://doi.org/ggqhz2</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.22.20040915">10.1101/2020.03.22.20040915</a></p>
</div>
<div id="ref-YfOcGRRa">
<p>164. <strong>Potential roles of social distancing in mitigating the spread of coronavirus disease 2019 (COVID-19) in South Korea</strong><br />
Sang Woo Park, Kaiyuan Sun, Cécile Viboud, Bryan T Grenfell, Jonathan Dushoff<br />
<em>GitHub</em> (2020) <a href="https://github.com/parksw3/Korea-analysis/blob/master/v1/korea.pdf">https://github.com/parksw3/Korea-analysis/blob/master/v1/korea.pdf</a></p>
</div>
<div id="ref-tiRfUgvs">
<p>165. <strong>Early dynamics of transmission and control of COVID-19: a mathematical modelling study</strong><br />
Adam J Kucharski, Timothy W Russell, Charlie Diamond, Yang Liu, John Edmunds, Sebastian Funk, Rosalind M Eggo, Fiona Sun, Mark Jit, James D Munday, … Stefan Flasche<br />
<em>The Lancet Infectious Diseases</em> (2020-03) <a href="https://doi.org/ggptcf">https://doi.org/ggptcf</a><br />
DOI: <a href="https://doi.org/10.1016/s1473-3099(20)30144-4">10.1016/s1473-3099(20)30144-4</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32171059">32171059</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7158569">PMC7158569</a></p>
</div>
<div id="ref-TH4ymqvJ">
<p>166. <strong>Estimating the reproduction number of COVID-19 in Iran using epidemic modeling</strong><br />
Ebrahim Sahafizadeh, Samaneh Sartoli<br />
<em>medRxiv</em> (2020-04-23) <a href="https://doi.org/ggqhzx">https://doi.org/ggqhzx</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.20.20038422">10.1101/2020.03.20.20038422</a></p>
</div>
<div id="ref-Frya4XA4">
<p>167. <strong>Report 13: Estimating the number of infections and the impact of non-pharmaceutical interventions on COVID-19 in 11 European countries</strong><br />
S Flaxman, S Mishra, A Gandy, H Unwin, H Coupland, T Mellan, H Zhu, T Berah, J Eaton, P Perez Guzman, … S Bhatt<br />
<em>Imperial College London</em> (2020-03-30) <a href="https://doi.org/ggrbmf">https://doi.org/ggrbmf</a><br />
DOI: <a href="https://doi.org/10.25561/77731">10.25561/77731</a></p>
</div>
<div id="ref-NCcssjLV">
<p>168. <strong>Projecting hospital utilization during the COVID-19 outbreaks in the United States</strong><br />
Seyed M. Moghadas, Affan Shoukat, Meagan C. Fitzpatrick, Chad R. Wells, Pratha Sah, Abhishek Pandey, Jeffrey D. Sachs, Zheng Wang, Lauren A. Meyers, Burton H. Singer, Alison P. Galvani<br />
<em>Proceedings of the National Academy of Sciences</em> (2020-04-21) <a href="https://doi.org/ggq7jc">https://doi.org/ggq7jc</a><br />
DOI: <a href="https://doi.org/10.1073/pnas.2004064117">10.1073/pnas.2004064117</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32245814">32245814</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7183199">PMC7183199</a></p>
</div>
<div id="ref-11CWFYFDL">
<p>169. <strong>The effect of control strategies to reduce social mixing on outcomes of the COVID-19 epidemic in Wuhan, China: a modelling study</strong><br />
Kiesha Prem, Yang Liu, Timothy W Russell, Adam J Kucharski, Rosalind M Eggo, Nicholas Davies, Mark Jit, Petra Klepac, Stefan Flasche, Samuel Clifford, … Joel Hellewell<br />
<em>The Lancet Public Health</em> (2020-03) <a href="https://doi.org/ggp3xq">https://doi.org/ggp3xq</a><br />
DOI: <a href="https://doi.org/10.1016/s2468-2667(20)30073-6">10.1016/s2468-2667(20)30073-6</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32220655">32220655</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7158905">PMC7158905</a></p>
</div>
<div id="ref-J4RCpntH">
<p>170. <strong>Spread and dynamics of the COVID-19 epidemic in Italy: Effects of emergency containment measures</strong><br />
Marino Gatto, Enrico Bertuzzo, Lorenzo Mari, Stefano Miccoli, Luca Carraro, Renato Casagrandi, Andrea Rinaldo<br />
<em>Proceedings of the National Academy of Sciences</em> (2020-05-12) <a href="https://doi.org/ggv4j6">https://doi.org/ggv4j6</a><br />
DOI: <a href="https://doi.org/10.1073/pnas.2004978117">10.1073/pnas.2004978117</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32327608">32327608</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7229754">PMC7229754</a></p>
</div>
<div id="ref-WA1R5R95">
<p>171. <strong>Covid-19: Temporal variation in transmission during the COVID-19 outbreak</strong><br />
EpiForecasts and the CMMID Covid working group<br />
<a href="https://epiforecasts.io/covid/">https://epiforecasts.io/covid/</a></p>
</div>
<div id="ref-17CQnstjh">
<p>172. <strong>Rt COVID-19</strong><br />
Kevin Systrom, Thomas Vladeck, Mike Krieger<br />
<a href="https://rt.live/">https://rt.live/</a></p>
</div>
<div id="ref-5MhhHQO0">
<p>173. <strong>Systems Approaches to Biology and Disease Enable Translational Systems Medicine</strong><br />
Leroy Hood, Qiang Tian<br />
<em>Genomics, Proteomics &amp; Bioinformatics</em> (2012-08) <a href="https://doi.org/f4f599">https://doi.org/f4f599</a><br />
DOI: <a href="https://doi.org/10.1016/j.gpb.2012.08.004">10.1016/j.gpb.2012.08.004</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23084773">23084773</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3844613">PMC3844613</a></p>
</div>
<div id="ref-8esIIWt9">
<p>174. <strong>A systems approach to infectious disease</strong><br />
Manon Eckhardt, Judd F. Hultquist, Robyn M. Kaake, Ruth Hüttenhain, Nevan J. Krogan<br />
<em>Nature Reviews Genetics</em> (2020-02-14) <a href="https://doi.org/ggnv63">https://doi.org/ggnv63</a><br />
DOI: <a href="https://doi.org/10.1038/s41576-020-0212-5">10.1038/s41576-020-0212-5</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32060427">32060427</a></p>
</div>
<div id="ref-8vUHj9cm">
<p>175. <strong>Differential expression of serum/plasma proteins in various infectious diseases: Specific or nonspecific signatures</strong><br />
Sandipan Ray, Sandip K. Patel, Vipin Kumar, Jagruti Damahe, Sanjeeva Srivastava<br />
<em>PROTEOMICS - Clinical Applications</em> (2014-02) <a href="https://doi.org/f2px3h">https://doi.org/f2px3h</a><br />
DOI: <a href="https://doi.org/10.1002/prca.201300074">10.1002/prca.201300074</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24293340">24293340</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7168033">PMC7168033</a></p>
</div>
<div id="ref-vHq0hOWQ">
<p>176. <strong>A new coronavirus associated with human respiratory disease in China</strong><br />
Fan Wu, Su Zhao, Bin Yu, Yan-Mei Chen, Wen Wang, Zhi-Gang Song, Yi Hu, Zhao-Wu Tao, Jun-Hua Tian, Yuan-Yuan Pei, … Yong-Zhen Zhang<br />
<em>Nature</em> (2020-02-03) <a href="https://doi.org/dk2w">https://doi.org/dk2w</a><br />
DOI: <a href="https://doi.org/10.1038/s41586-020-2008-3">10.1038/s41586-020-2008-3</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32015508">32015508</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7094943">PMC7094943</a></p>
</div>
<div id="ref-11xZWeHN3">
<p>177. <strong>Proteomics of SARS-CoV-2-infected host cells reveals therapy targets</strong><br />
Denisa Bojkova, Kevin Klann, Benjamin Koch, Marek Widera, David Krause, Sandra Ciesek, Jindrich Cinatl, Christian Münch<br />
<em>Nature</em> (2020-05-14) <a href="https://doi.org/dw7s">https://doi.org/dw7s</a><br />
DOI: <a href="https://doi.org/10.1038/s41586-020-2332-7">10.1038/s41586-020-2332-7</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32408336">32408336</a></p>
</div>
<div id="ref-GhJYjnft">
<p>178. <strong>Potent human neutralizing antibodies elicited by SARS-CoV-2 infection</strong><br />
Bin Ju, Qi Zhang, Xiangyang Ge, Ruoke Wang, Jiazhen Yu, Sisi Shan, Bing Zhou, Shuo Song, Xian Tang, Jinfang Yu, … Linqi Zhang<br />
<em>bioRxiv</em> (2020-03-26) <a href="https://doi.org/ggp7t4">https://doi.org/ggp7t4</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.21.990770">10.1101/2020.03.21.990770</a></p>
</div>
<div id="ref-GpnngtWK">
<p>179. <strong>Plasma proteome of severe acute respiratory syndrome analyzed by two-dimensional gel electrophoresis and mass spectrometry</strong><br />
J.-H. Chen, Y.-W. Chang, C.-W. Yao, T.-S. Chiueh, S.-C. Huang, K.-Y. Chien, A. Chen, F.-Y. Chang, C.-H. Wong, Y.-J. Chen<br />
<em>Proceedings of the National Academy of Sciences</em> (2004-11-30) <a href="https://doi.org/dtv8sx">https://doi.org/dtv8sx</a><br />
DOI: <a href="https://doi.org/10.1073/pnas.0407992101">10.1073/pnas.0407992101</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15572443">15572443</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC535397">PMC535397</a></p>
</div>
<div id="ref-JLTf2Fwb">
<p>180. <strong>Analysis of multimerization of the SARS coronavirus nucleocapsid protein</strong><br />
Runtao He, Frederick Dobie, Melissa Ballantine, Andrew Leeson, Yan Li, Nathalie Bastien, Todd Cutts, Anton Andonov, Jingxin Cao, Timothy F. Booth, … Xuguang Li<br />
<em>Biochemical and Biophysical Research Communications</em> (2004-04) <a href="https://doi.org/dbfwr9">https://doi.org/dbfwr9</a><br />
DOI: <a href="https://doi.org/10.1016/j.bbrc.2004.02.074">10.1016/j.bbrc.2004.02.074</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15020242">15020242</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7111152">PMC7111152</a></p>
</div>
<div id="ref-phJM8g2Y">
<p>181. <strong>A SARS-CoV-2-Human Protein-Protein Interaction Map Reveals Drug Targets and Potential Drug-Repurposing</strong><br />
David E. Gordon, Gwendolyn M. Jang, Mehdi Bouhaddou, Jiewei Xu, Kirsten Obernier, Matthew J. O’Meara, Jeffrey Z. Guo, Danielle L. Swaney, Tia A. Tummino, Ruth Hüttenhain, … Nevan J. Krogan<br />
<em>bioRxiv</em> (2020-03-22) <a href="https://doi.org/ggpptg">https://doi.org/ggpptg</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.22.002386">10.1101/2020.03.22.002386</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32511329">32511329</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7239059">PMC7239059</a></p>
</div>
<div id="ref-NLoN4aYj">
<p>182. <strong>Virus-host interactome and proteomic survey of PMBCs from COVID-19 patients reveal potential virulence factors influencing SARS-CoV-2 pathogenesis</strong><br />
Jingjiao Li, Mingquan Guo, Xiaoxu Tian, Chengrong Liu, Xin Wang, Xing Yang, Ping Wu, Zixuan Xiao, Yafei Qu, Yue Yin, … Qiming Liang<br />
<em>bioRxiv</em> (2020-04-02) <a href="https://doi.org/ggrgbv">https://doi.org/ggrgbv</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.31.019216">10.1101/2020.03.31.019216</a></p>
</div>
<div id="ref-1ccnm0N9">
<p>183. <strong>The severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) envelope (E) protein harbors a conserved BH3-like sequence</strong><br />
Vincent Navratil, Loïc Lionnard, Sonia Longhi, J. Marie Hardwick, Christophe Combet, Abdel Aouacheria<br />
<em>bioRxiv</em> (2020-06-09) <a href="https://doi.org/ggrp43">https://doi.org/ggrp43</a><br />
DOI: <a href="https://doi.org/10.1101/2020.04.09.033522">10.1101/2020.04.09.033522</a></p>
</div>
<div id="ref-pHrstqMQ">
<p>184. <strong>Imbalanced Host Response to SARS-CoV-2 Drives Development of COVID-19</strong><br />
Daniel Blanco-Melo, Benjamin E. Nilsson-Payant, Wen-Chun Liu, Skyler Uhl, Daisy Hoagland, Rasmus Møller, Tristan X. Jordan, Kohei Oishi, Maryline Panis, David Sachs, … Benjamin R. tenOever<br />
<em>Cell</em> (2020-05) <a href="https://doi.org/ggw5tq">https://doi.org/ggw5tq</a><br />
DOI: <a href="https://doi.org/10.1016/j.cell.2020.04.026">10.1016/j.cell.2020.04.026</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32416070">32416070</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7227586">PMC7227586</a></p>
</div>
<div id="ref-Gj8vlc0W">
<p>185. <strong>Bulk and single-cell gene expression profiling of SARS-CoV-2 infected human cell lines identifies molecular targets for therapeutic intervention</strong><br />
Wyler Emanuel, Mösbauer Kirstin, Franke Vedran, Diag Asija, Gottula Lina Theresa, Arsie Roberto, Klironomos Filippos, Koppstein David, Ayoub Salah, Buccitelli Christopher, … Landthaler Markus<br />
<em>bioRxiv</em> (2020-05-05) <a href="https://doi.org/ggxd2g">https://doi.org/ggxd2g</a><br />
DOI: <a href="https://doi.org/10.1101/2020.05.05.079194">10.1101/2020.05.05.079194</a></p>
</div>
<div id="ref-1228YjPRv">
<p>186. <strong>Type I and Type III Interferons – Induction, Signaling, Evasion, and Application to Combat COVID-19</strong><br />
Annsea Park, Akiko Iwasaki<br />
<em>Cell Host &amp; Microbe</em> (2020-06) <a href="https://doi.org/gg2ccp">https://doi.org/gg2ccp</a><br />
DOI: <a href="https://doi.org/10.1016/j.chom.2020.05.008">10.1016/j.chom.2020.05.008</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32464097">32464097</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7255347">PMC7255347</a></p>
</div>
<div id="ref-A4XFZlP">
<p>187. <strong>Cytokines and chemokines: At the crossroads of cell signalling and inflammatory disease</strong><br />
Mark D. Turner, Belinda Nedjai, Tara Hurst, Daniel J. Pennington<br />
<em>Biochimica et Biophysica Acta (BBA) - Molecular Cell Research</em> (2014-11) <a href="https://doi.org/f6jrj9">https://doi.org/f6jrj9</a><br />
DOI: <a href="https://doi.org/10.1016/j.bbamcr.2014.05.014">10.1016/j.bbamcr.2014.05.014</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24892271">24892271</a></p>
</div>
<div id="ref-PJgYPcKM">
<p>188. <strong>SARS-CoV-2 ORF3b is a potent interferon antagonist whose activity is further increased by a naturally occurring elongation variant</strong><br />
Yoriyuki Konno, Izumi Kimura, Keiya Uriu, Masaya Fukushi, Takashi Irie, Yoshio Koyanagi, So Nakagawa, Kei Sato<br />
<em>bioRxiv</em> (2020-05-11) <a href="https://doi.org/gg2qgs">https://doi.org/gg2qgs</a><br />
DOI: <a href="https://doi.org/10.1101/2020.05.11.088179">10.1101/2020.05.11.088179</a></p>
</div>
<div id="ref-16EFyBURq">
<p>189. <strong>Isolation and characterization of SARS-CoV-2 from the first US COVID-19 patient</strong><br />
Jennifer Harcourt, Azaibi Tamin, Xiaoyan Lu, Shifaq Kamili, Senthil Kumar. Sakthivel, Janna Murray, Krista Queen, Ying Tao, Clinton R. Paden, Jing Zhang, … Natalie J. Thornburg<br />
<em>bioRxiv</em> (2020-03-07) <a href="https://doi.org/gg2fkm">https://doi.org/gg2fkm</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.02.972935">10.1101/2020.03.02.972935</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32511316">32511316</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7239045">PMC7239045</a></p>
</div>
<div id="ref-1AHtg71LX">
<p>190. <strong>Detection of 2019 novel coronavirus (2019-nCoV) by real-time RT-PCR</strong><br />
Victor M Corman, Olfert Landt, Marco Kaiser, Richard Molenkamp, Adam Meijer, Daniel KW Chu, Tobias Bleicker, Sebastian Brünink, Julia Schneider, Marie Luisa Schmidt, … Christian Drosten<br />
<em>Eurosurveillance</em> (2020-01-23) <a href="https://doi.org/ggjs7g">https://doi.org/ggjs7g</a><br />
DOI: <a href="https://doi.org/10.2807/1560-7917.es.2020.25.3.2000045">10.2807/1560-7917.es.2020.25.3.2000045</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31992387">31992387</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6988269">PMC6988269</a></p>
</div>
<div id="ref-fCZXxoas">
<p>191. <strong>Temporal profiles of viral load in posterior oropharyngeal saliva samples and serum antibody responses during infection by SARS-CoV-2: an observational cohort study</strong><br />
Kelvin Kai-Wang To, Owen Tak-Yin Tsang, Wai-Shing Leung, Anthony Raymond Tam, Tak-Chiu Wu, David Christopher Lung, Cyril Chik-Yan Yip, Jian-Piao Cai, Jacky Man-Chun Chan, Thomas Shiu-Hong Chik, … Kwok-Yung Yuen<br />
<em>The Lancet Infectious Diseases</em> (2020-03) <a href="https://doi.org/ggp4qx">https://doi.org/ggp4qx</a><br />
DOI: <a href="https://doi.org/10.1016/s1473-3099(20)30196-1">10.1016/s1473-3099(20)30196-1</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32213337">32213337</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7158907">PMC7158907</a></p>
</div>
<div id="ref-oNFnjRBI">
<p>192. <strong>Fecal specimen diagnosis 2019 novel coronavirus–infected pneumonia</strong><br />
JingCheng Zhang, SaiBin Wang, YaDong Xue<br />
<em>Journal of Medical Virology</em> (2020-03-12) <a href="https://doi.org/ggpx6d">https://doi.org/ggpx6d</a><br />
DOI: <a href="https://doi.org/10.1002/jmv.25742">10.1002/jmv.25742</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32124995">32124995</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7228355">PMC7228355</a></p>
</div>
<div id="ref-jr3z6jGU">
<p>193. <strong>Library preparation for next generation sequencing: A review of automation strategies</strong><br />
J. F. Hess, T. A. Kohl, M. Kotrová, K. Rönsch, T. Paprotka, V. Mohr, T. Hutzenlaub, M. Brüggemann, R. Zengerle, S. Niemann, N. Paust<br />
<em>Biotechnology Advances</em> (2020-07) <a href="https://doi.org/ggth2v">https://doi.org/ggth2v</a><br />
DOI: <a href="https://doi.org/10.1016/j.biotechadv.2020.107537">10.1016/j.biotechadv.2020.107537</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32199980">32199980</a></p>
</div>
<div id="ref-43modeVu">
<p>194. <strong>Diagnosing COVID-19: The Disease and Tools for Detection</strong><br />
Buddhisha Udugama, Pranav Kadhiresan, Hannah N. Kozlowski, Ayden Malekjahani, Matthew Osborne, Vanessa Y. C. Li, Hongmin Chen, Samira Mubareka, Jonathan B. Gubbay, Warren C. W. Chan<br />
<em>ACS Nano</em> (2020-03-30) <a href="https://doi.org/ggq8ds">https://doi.org/ggq8ds</a><br />
DOI: <a href="https://doi.org/10.1021/acsnano.0c02624">10.1021/acsnano.0c02624</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32223179">32223179</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7144809">PMC7144809</a></p>
</div>
<div id="ref-COEfjKo5">
<p>195. <strong>Molecular Diagnosis of a Novel Coronavirus (2019-nCoV) Causing an Outbreak of Pneumonia</strong><br />
Daniel KW Chu, Yang Pan, Samuel MS Cheng, Kenrie PY Hui, Pavithra Krishnan, Yingzhi Liu, Daisy YM Ng, Carrie KC Wan, Peng Yang, Quanyi Wang, … Leo LM Poon<br />
<em>Clinical Chemistry</em> (2020-01-31) <a href="https://doi.org/ggnbpp">https://doi.org/ggnbpp</a><br />
DOI: <a href="https://doi.org/10.1093/clinchem/hvaa029">10.1093/clinchem/hvaa029</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32031583">32031583</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7108203">PMC7108203</a></p>
</div>
<div id="ref-Hl2pASeg">
<p>196. <strong>dPCR: A Technology Review</strong><br />
Phenix-Lan Quan, Martin Sauzade, Eric Brouzes<br />
<em>Sensors</em> (2018-04-20) <a href="https://doi.org/ggr39c">https://doi.org/ggr39c</a><br />
DOI: <a href="https://doi.org/10.3390/s18041271">10.3390/s18041271</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29677144">29677144</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5948698">PMC5948698</a></p>
</div>
<div id="ref-2WgqQLF8">
<p>197. <strong>ddPCR: a more accurate tool for SARS-CoV-2 detection in low viral load specimens</strong><br />
Tao Suo, Xinjin Liu, Jiangpeng Feng, Ming Guo, Wenjia Hu, Dong Guo, Hafiz Ullah, Yang Yang, Qiuhan Zhang, Xin Wang, … Yu Chen<br />
<em>Emerging Microbes &amp; Infections</em> (2020-06-07) <a href="https://doi.org/ggx2t2">https://doi.org/ggx2t2</a><br />
DOI: <a href="https://doi.org/10.1080/22221751.2020.1772678">10.1080/22221751.2020.1772678</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32438868">32438868</a></p>
</div>
<div id="ref-UYD5T2mp">
<p>198. <strong>Highly accurate and sensitive diagnostic detection of SARS-CoV-2 by digital PCR</strong><br />
Lianhua Dong, Junbo Zhou, Chunyan Niu, Quanyi Wang, Yang Pan, Sitong Sheng, Xia Wang, Yongzhuo Zhang, Jiayi Yang, Manqing Liu, … Xiang Fang<br />
<em>medRxiv</em> (2020-03-30) <a href="https://doi.org/ggqnqh">https://doi.org/ggqnqh</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.14.20036129">10.1101/2020.03.14.20036129</a></p>
</div>
<div id="ref-93NBANq2">
<p>199. <strong>Evaluation of COVID-19 RT-qPCR test in multi-sample pools</strong><br />
Idan Yelin, Noga Aharony, Einat Shaer-Tamar, Amir Argoetti, Esther Messer, Dina Berenbaum, Einat Shafran, Areen Kuzli, Nagam Gandali, Tamar Hashimshony, … Roy Kishony<br />
<em>medRxiv</em> (2020-03-27) <a href="https://doi.org/ggrn74">https://doi.org/ggrn74</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.26.20039438">10.1101/2020.03.26.20039438</a></p>
</div>
<div id="ref-IQ5kUQEO">
<p>200. <strong>Analytical Validation of a COVID-19 qRT-PCR Detection Assay Using a 384-well Format and Three Extraction Methods</strong><br />
Andrew C. Nelson, Benjamin Auch, Matthew Schomaker, Daryl M. Gohl, Patrick Grady, Darrell Johnson, Robyn Kincaid, Kylene E. Karnuth, Jerry Daniel, Jessica K. Fiege, … Sophia Yohe<br />
<em>bioRxiv</em> (2020-04-05) <a href="https://doi.org/ggs45d">https://doi.org/ggs45d</a><br />
DOI: <a href="https://doi.org/10.1101/2020.04.02.022186">10.1101/2020.04.02.022186</a></p>
</div>
<div id="ref-1272qWb3a">
<p>201. <strong>Development and Applications of CRISPR-Cas9 for Genome Engineering</strong><br />
Patrick D. Hsu, Eric S. Lander, Feng Zhang<br />
<em>Cell</em> (2014-06) <a href="https://doi.org/f6d3wg">https://doi.org/f6d3wg</a><br />
DOI: <a href="https://doi.org/10.1016/j.cell.2014.05.010">10.1016/j.cell.2014.05.010</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24906146">24906146</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4343198">PMC4343198</a></p>
</div>
<div id="ref-1dxrDub1">
<p>202. <strong>Rapid Detection of 2019 Novel Coronavirus SARS-CoV-2 Using a CRISPR-based DETECTR Lateral Flow Assay</strong><br />
James P Broughton, Xianding Deng, Guixia Yu, Clare L Fasching, Jasmeet Singh, Jessica Streithorst, Andrea Granados, Alicia Sotomayor-Gonzalez, Kelsey Zorn, Allan Gopez, … Charles Y Chiu<br />
<em>medRxiv</em> (2020-03-27) <a href="https://doi.org/ggrmwt">https://doi.org/ggrmwt</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.06.20032334">10.1101/2020.03.06.20032334</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32511449">32511449</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7239074">PMC7239074</a></p>
</div>
<div id="ref-NT5mrVGY">
<p>203. <strong>The standard coronavirus test, if available, works well—but can new diagnostics help in this pandemic?</strong><br />
Robert Service<br />
<em>Science</em> (2020-03-22) <a href="https://doi.org/ggq9wm">https://doi.org/ggq9wm</a><br />
DOI: <a href="https://doi.org/10.1126/science.abb8400">10.1126/science.abb8400</a></p>
</div>
<div id="ref-Fdc7seLs">
<p>204. <strong>Coronavirus and the race to distribute reliable diagnostics</strong><br />
Cormac Sheridan<br />
<em>Nature Biotechnology</em> (2020-02-19) <a href="https://doi.org/ggm4nt">https://doi.org/ggm4nt</a><br />
DOI: <a href="https://doi.org/10.1038/d41587-020-00002-2">10.1038/d41587-020-00002-2</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32265548">32265548</a></p>
</div>
<div id="ref-AvQI3GSm">
<p>205. <strong>Laboratory Diagnosis of COVID-19: Current Issues and Challenges</strong><br />
Yi-Wei Tang, Jonathan E. Schmitz, David H. Persing, Charles W. Stratton<br />
<em>Journal of Clinical Microbiology</em> (2020-05-26) <a href="https://doi.org/ggq7h8">https://doi.org/ggq7h8</a><br />
DOI: <a href="https://doi.org/10.1128/jcm.00512-20">10.1128/jcm.00512-20</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32245835">32245835</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7269383">PMC7269383</a></p>
</div>
<div id="ref-11vIuqmS2">
<p>206. <strong>Negative Nasopharyngeal and Oropharyngeal Swab Does Not Rule Out COVID-19</strong><br />
Poramed Winichakoon, Romanee Chaiwarith, Chalerm Liwsrisakun, Parichat Salee, Aree Goonna, Atikun Limsukon, Quanhathai Kaewpoowat<br />
<em>Journal of Clinical Microbiology</em> (2020-02-26) <a href="https://doi.org/ggpw9m">https://doi.org/ggpw9m</a><br />
DOI: <a href="https://doi.org/10.1128/jcm.00297-20">10.1128/jcm.00297-20</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32102856">32102856</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7180262">PMC7180262</a></p>
</div>
<div id="ref-EUrJt06r">
<p>207. <strong>A systematic review of antibody mediated immunity to coronaviruses: antibody kinetics, correlates of protection, and association of antibody responses with severity of disease</strong><br />
Angkana T. Huang, Bernardo Garcia-Carreras, Matt D. T. Hitchings, Bingyi Yang, Leah Katzelnick, Susan M Rattigan, Brooke Borgert, Carlos Moreno, Benjamin D. Solomon, Isabel Rodriguez-Barraquer, … Derek A. T. Cummings<br />
<em>medRxiv</em> (2020-04-17) <a href="https://doi.org/ggsfmz">https://doi.org/ggsfmz</a><br />
DOI: <a href="https://doi.org/10.1101/2020.04.14.20065771">10.1101/2020.04.14.20065771</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32511434">32511434</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7217088">PMC7217088</a></p>
</div>
<div id="ref-n6sKSpIk">
<p>208. <strong>Cellex qSARS-CoV-2 IgG/IgM Rapid Test</strong><br />
Cellex<br />
(2020-04-07) <a href="https://www.fda.gov/media/136625/download">https://www.fda.gov/media/136625/download</a></p>
</div>
<div id="ref-FFMfMDRN">
<p>209. <strong>Detection of antibodies against SARS-CoV-2 in patients with COVID-19</strong><br />
Zhe Du, Fengxue Zhu, Fuzheng Guo, Bo Yang, Tianbing Wang<br />
<em>Journal of Medical Virology</em> (2020-04-10) <a href="https://doi.org/ggq7m2">https://doi.org/ggq7m2</a><br />
DOI: <a href="https://doi.org/10.1002/jmv.25820">10.1002/jmv.25820</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32243608">32243608</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7228383">PMC7228383</a></p>
</div>
<div id="ref-yJW1RVQb">
<p>210. <strong>A serological assay to detect SARS-CoV-2 seroconversion in humans</strong><br />
Fatima Amanat, Daniel Stadlbauer, Shirin Strohmeier, Thi Nguyen, Veronika Chromikova, Meagan McMahon, Kaijun Jiang, Guha Asthagiri-Arunkumar, Denise Jurczyszak, Jose Polanco, … Thomas Moran<br />
<em>medRxiv</em> (2020-04-16) <a href="https://doi.org/ggpn83">https://doi.org/ggpn83</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.17.20037713">10.1101/2020.03.17.20037713</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32511441">32511441</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7239062">PMC7239062</a></p>
</div>
<div id="ref-Gk9WwhYB">
<p>211. <strong>Coronavirus testing is ramping up. Here are the new tests and how they work.</strong><br />
Stephanie Pappas-Live Science Contributor 31 March 2020<br />
<em>livescience.com</em> <a href="https://www.livescience.com/coronavirus-tests-available.html">https://www.livescience.com/coronavirus-tests-available.html</a></p>
</div>
<div id="ref-91dto3z3">
<p>212. <strong>Strong associations and moderate predictive value of early symptoms for SARS-CoV-2 test positivity among healthcare workers, the Netherlands, March 2020</strong><br />
Alma Tostmann, John Bradley, Teun Bousema, Wing-Kee Yiek, Minke Holwerda, Chantal Bleeker-Rovers, Jaap ten Oever, Corianne Meijer, Janette Rahamat-Langendoen, Joost Hopman, … Heiman Wertheim<br />
<em>Eurosurveillance</em> (2020-04-23) <a href="https://doi.org/ggthwx">https://doi.org/ggthwx</a><br />
DOI: <a href="https://doi.org/10.2807/1560-7917.es.2020.25.16.2000508">10.2807/1560-7917.es.2020.25.16.2000508</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32347200">32347200</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7189649">PMC7189649</a></p>
</div>
<div id="ref-itCGVb5b">
<p>213. <strong>Correlation of Chest CT and RT-PCR Testing in Coronavirus Disease 2019 (COVID-19) in China: A Report of 1014 Cases</strong><br />
Tao Ai, Zhenlu Yang, Hongyan Hou, Chenao Zhan, Chong Chen, Wenzhi Lv, Qian Tao, Ziyong Sun, Liming Xia<br />
<em>Radiology</em> (2020-02-26) <a href="https://doi.org/ggmw6p">https://doi.org/ggmw6p</a><br />
DOI: <a href="https://doi.org/10.1148/radiol.2020200642">10.1148/radiol.2020200642</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32101510">32101510</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7233399">PMC7233399</a></p>
</div>
<div id="ref-tkuUFNIX">
<p>214. <strong>Performance of radiologists in differentiating COVID-19 from viral pneumonia on chest CT</strong><br />
Harrison X. Bai, Ben Hsieh, Zeng Xiong, Kasey Halsey, Ji Whae Choi, Thi My Linh Tran, Ian Pan, Lin-Bo Shi, Dong-Cui Wang, Ji Mei, … Wei-Hua Liao<br />
<em>Radiology</em> (2020-03-10) <a href="https://doi.org/ggnqw4">https://doi.org/ggnqw4</a><br />
DOI: <a href="https://doi.org/10.1148/radiol.2020200823">10.1148/radiol.2020200823</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32155105">32155105</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7233414">PMC7233414</a></p>
</div>
<div id="ref-nN9zfzPe">
<p>215. <strong>Covid-19: automatic detection from X-ray images utilizing transfer learning with convolutional neural networks</strong><br />
Ioannis D. Apostolopoulos, Tzani A. Mpesiana<br />
<em>Physical and Engineering Sciences in Medicine</em> (2020-04-03) <a href="https://doi.org/ggs448">https://doi.org/ggs448</a><br />
DOI: <a href="https://doi.org/10.1007/s13246-020-00865-4">10.1007/s13246-020-00865-4</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32524445">32524445</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7118364">PMC7118364</a></p>
</div>
<div id="ref-OB8BJiwj">
<p>216. <strong>Coronavirus Disease 2019 (COVID-19)</strong><br />
CDC<br />
<em>Centers for Disease Control and Prevention</em> (2020-02-11) <a href="https://www.cdc.gov/coronavirus/2019-ncov/hcp/testing-overview.html">https://www.cdc.gov/coronavirus/2019-ncov/hcp/testing-overview.html</a></p>
</div>
<div id="ref-IldxlHwA">
<p>217. <strong>NY Forward: a guide to reopening New York &amp; building back better</strong>(2020-05-15) <a href="https://www.governor.ny.gov/sites/governor.ny.gov/files/atoms/files/NYForwardReopeningGuide.pdf">https://www.governor.ny.gov/sites/governor.ny.gov/files/atoms/files/NYForwardReopeningGuide.pdf</a></p>
</div>
<div id="ref-181QWa7HL">
<p>218. <strong>Neutralizing Monoclonal Antibodies as Promising Therapeutics against Middle East Respiratory Syndrome Coronavirus Infection</strong><br />
Hui-Ju Han, Jian-Wei Liu, Hao Yu, Xue-Jie Yu<br />
<em>Viruses</em> (2018-11-30) <a href="https://doi.org/ggp87v">https://doi.org/ggp87v</a><br />
DOI: <a href="https://doi.org/10.3390/v10120680">10.3390/v10120680</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30513619">30513619</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6315345">PMC6315345</a></p>
</div>
<div id="ref-eyBAUUCl">
<p>219. <strong>Nutraceuticals have potential for boosting the type 1 interferon response to RNA viruses including influenza and coronavirus</strong><br />
Mark F. McCarty, James J. DiNicolantonio<br />
<em>Progress in Cardiovascular Diseases</em> (2020-02) <a href="https://doi.org/ggpwx2">https://doi.org/ggpwx2</a><br />
DOI: <a href="https://doi.org/10.1016/j.pcad.2020.02.007">10.1016/j.pcad.2020.02.007</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32061635">32061635</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7130854">PMC7130854</a></p>
</div>
<div id="ref-GT7yDBjv">
<p>220. <strong>Reducing mortality from 2019-nCoV: host-directed therapies should be an option</strong><br />
Alimuddin Zumla, David S Hui, Esam I Azhar, Ziad A Memish, Markus Maeurer<br />
<em>The Lancet</em> (2020-02) <a href="https://doi.org/ggkd3b">https://doi.org/ggkd3b</a><br />
DOI: <a href="https://doi.org/10.1016/s0140-6736(20)30305-6">10.1016/s0140-6736(20)30305-6</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32035018">32035018</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7133595">PMC7133595</a></p>
</div>
<div id="ref-SSbnPnzT">
<p>221. <strong>Evidence-Based Medicine Data Lab COVID-19 TrialsTracker</strong><br />
Nick DeVito, Peter Inglesby<br />
<em>GitHub</em> (2020-03-29) <a href="https://github.com/ebmdatalab/covid_trials_tracker-covid">https://github.com/ebmdatalab/covid_trials_tracker-covid</a><br />
DOI: <a href="https://doi.org/10.5281/zenodo.3732709">10.5281/zenodo.3732709</a></p>
</div>
<div id="ref-4oZGCeCo">
<p>222. <strong>Digestive Symptoms in COVID-19 Patients With Mild Disease Severity</strong><br />
Chaoqun Han, Caihan Duan, Shengyan Zhang, Brennan Spiegel, Huiying Shi, Weijun Wang, Lei Zhang, Rong Lin, Jun Liu, Zhen Ding, Xiaohua Hou<br />
<em>American Journal of Gastroenterology</em> (2020-06) <a href="https://doi.org/ggtzm2">https://doi.org/ggtzm2</a><br />
DOI: <a href="https://doi.org/10.14309/ajg.0000000000000664">10.14309/ajg.0000000000000664</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32301761">32301761</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7172493">PMC7172493</a></p>
</div>
<div id="ref-vEFSIErd">
<p>223. <strong>Interim Clinical Guidance for Management of Patients with Confirmed Coronavirus Disease (COVID-19)</strong><br />
Centers for Disease Control and Prevention<br />
(2020-06-30) <a href="https://www.cdc.gov/coronavirus/2019-ncov/hcp/clinical-guidance-management-patients.html">https://www.cdc.gov/coronavirus/2019-ncov/hcp/clinical-guidance-management-patients.html</a></p>
</div>
<div id="ref-14gthSuhm">
<p>224. <strong>Introduction to modern virology</strong><br />
N. J. Dimmock, A. J. Easton, K. N. Leppard<br />
<em>Blackwell Pub</em> (2007)<br />
ISBN: <a href="https://worldcat.org/isbn/9781405136457">9781405136457</a></p>
</div>
<div id="ref-tn6j6IZ0">
<p>225. <strong>Coronaviruses</strong><br />
Helena Jane Maier, Erica Bickerton, Paul Britton (editors)<br />
<em>Methods in Molecular Biology</em> (2015) <a href="https://doi.org/ggqfqx">https://doi.org/ggqfqx</a><br />
DOI: <a href="https://doi.org/10.1007/978-1-4939-2438-7">10.1007/978-1-4939-2438-7</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25870870">25870870</a> · ISBN: <a href="https://worldcat.org/isbn/9781493924370">9781493924370</a></p>
</div>
<div id="ref-Sh1FwWps">
<p>226. <strong>The potential chemical structure of anti‐SARS‐CoV‐2 RNA‐dependent RNA polymerase</strong><br />
Jrhau Lung, Yu‐Shih Lin, Yao‐Hsu Yang, Yu‐Lun Chou, Li‐Hsin Shu, Yu‐Ching Cheng, Hung Te Liu, Ching‐Yuan Wu<br />
<em>Journal of Medical Virology</em> (2020-03-18) <a href="https://doi.org/ggp6fm">https://doi.org/ggp6fm</a><br />
DOI: <a href="https://doi.org/10.1002/jmv.25761">10.1002/jmv.25761</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32167173">32167173</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7228302">PMC7228302</a></p>
</div>
<div id="ref-IkAU62Pd">
<p>227. <strong>Favipiravir</strong><br />
DrugBank<br />
(2020-06-12) <a href="https://www.drugbank.ca/drugs/DB12466">https://www.drugbank.ca/drugs/DB12466</a></p>
</div>
<div id="ref-EgoJKtW4">
<p>228. <strong>In Vitro and In Vivo Activities of Anti-Influenza Virus Compound T-705</strong><br />
Y. Furuta, K. Takahashi, Y. Fukuda, M. Kuno, T. Kamiyama, K. Kozaki, N. Nomura, H. Egawa, S. Minami, Y. Watanabe, … K. Shiraki<br />
<em>Antimicrobial Agents and Chemotherapy</em> (2002-04-01) <a href="https://doi.org/cndw7n">https://doi.org/cndw7n</a><br />
DOI: <a href="https://doi.org/10.1128/aac.46.4.977-981.2002">10.1128/aac.46.4.977-981.2002</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/11897578">11897578</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC127093">PMC127093</a></p>
</div>
<div id="ref-rG5RCRdJ">
<p>229. <strong>Efficacy of Orally Administered T-705 on Lethal Avian Influenza A (H5N1) Virus Infections in Mice</strong><br />
R. W. Sidwell, D. L. Barnard, C. W. Day, D. F. Smee, K. W. Bailey, M.-H. Wong, J. D. Morrey, Y. Furuta<br />
<em>Antimicrobial Agents and Chemotherapy</em> (2006-12-28) <a href="https://doi.org/dm9xr2">https://doi.org/dm9xr2</a><br />
DOI: <a href="https://doi.org/10.1128/aac.01051-06">10.1128/aac.01051-06</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17194832">17194832</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1803113">PMC1803113</a></p>
</div>
<div id="ref-14p3btkRH">
<p>230. <strong>Favipiravir (T-705), a broad spectrum inhibitor of viral RNA polymerase</strong><br />
Yousuke FURUTA, Takashi KOMENO, Takaaki NAKAMURA<br />
<em>Proceedings of the Japan Academy, Series B</em> (2017) <a href="https://doi.org/gbxcxw">https://doi.org/gbxcxw</a><br />
DOI: <a href="https://doi.org/10.2183/pjab.93.027">10.2183/pjab.93.027</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28769016">28769016</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5713175">PMC5713175</a></p>
</div>
<div id="ref-DpdKJbki">
<p>231. <strong>Mechanism of Action of T-705 against Influenza Virus</strong><br />
Y. Furuta, K. Takahashi, M. Kuno-Maekawa, H. Sangawa, S. Uehara, K. Kozaki, N. Nomura, H. Egawa, K. Shiraki<br />
<em>Antimicrobial Agents and Chemotherapy</em> (2005-02-23) <a href="https://doi.org/dgbwdh">https://doi.org/dgbwdh</a><br />
DOI: <a href="https://doi.org/10.1128/aac.49.3.981-986.2005">10.1128/aac.49.3.981-986.2005</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15728892">15728892</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC549233">PMC549233</a></p>
</div>
<div id="ref-9j7NVPJS">
<p>232. <strong>Activity of T-705 in a Hamster Model of Yellow Fever Virus Infection in Comparison with That of a Chemically Related Compound, T-1106</strong><br />
Justin G. Julander, Kristiina Shafer, Donald F. Smee, John D. Morrey, Yousuke Furuta<br />
<em>Antimicrobial Agents and Chemotherapy</em> (2009-01) <a href="https://doi.org/brknds">https://doi.org/brknds</a><br />
DOI: <a href="https://doi.org/10.1128/aac.01074-08">10.1128/aac.01074-08</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/18955536">18955536</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2612161">PMC2612161</a></p>
</div>
<div id="ref-IYEoCGHy">
<p>233. <strong>In Vitro and In Vivo Activities of T-705 against Arenavirus and Bunyavirus Infections</strong><br />
Brian B. Gowen, Min-Hui Wong, Kie-Hoon Jung, Andrew B. Sanders, Michelle Mendenhall, Kevin W. Bailey, Yousuke Furuta, Robert W. Sidwell<br />
<em>Antimicrobial Agents and Chemotherapy</em> (2007-09) <a href="https://doi.org/d98c87">https://doi.org/d98c87</a><br />
DOI: <a href="https://doi.org/10.1128/aac.00356-07">10.1128/aac.00356-07</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17606691">17606691</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2043187">PMC2043187</a></p>
</div>
<div id="ref-7d1TnpE3">
<p>234. <strong>Favipiravir (T-705) inhibits in vitro norovirus replication</strong><br />
J. Rocha-Pereira, D. Jochmans, K. Dallmeier, P. Leyssen, M. S. J. Nascimento, J. Neyts<br />
<em>Biochemical and Biophysical Research Communications</em> (2012-08) <a href="https://doi.org/f369j7">https://doi.org/f369j7</a><br />
DOI: <a href="https://doi.org/10.1016/j.bbrc.2012.07.034">10.1016/j.bbrc.2012.07.034</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22809499">22809499</a></p>
</div>
<div id="ref-15kQ4QYbx">
<p>235. <strong>T-705 (Favipiravir) Inhibition of Arenavirus Replication in Cell Culture</strong><br />
Michelle Mendenhall, Andrew Russell, Terry Juelich, Emily L. Messina, Donald F. Smee, Alexander N. Freiberg, Michael R. Holbrook, Yousuke Furuta, Juan-Carlos de la Torre, Jack H. Nunberg, Brian B. Gowen<br />
<em>Antimicrobial Agents and Chemotherapy</em> (2011-02) <a href="https://doi.org/cppwsc">https://doi.org/cppwsc</a><br />
DOI: <a href="https://doi.org/10.1128/aac.01219-10">10.1128/aac.01219-10</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21115797">21115797</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3028760">PMC3028760</a></p>
</div>
<div id="ref-MPJdSF1p">
<p>236. <strong>The evolution of nucleoside analogue antivirals: A review for chemists and non-chemists. Part 1: Early structural modifications to the nucleoside scaffold</strong><br />
Katherine L. Seley-Radtke, Mary K. Yates<br />
<em>Antiviral Research</em> (2018-06) <a href="https://doi.org/gdpn35">https://doi.org/gdpn35</a><br />
DOI: <a href="https://doi.org/10.1016/j.antiviral.2018.04.004">10.1016/j.antiviral.2018.04.004</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29649496">29649496</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6396324">PMC6396324</a></p>
</div>
<div id="ref-ukZRKllk">
<p>237. <strong>The Ambiguous Base-Pairing and High Substrate Efficiency of T-705 (Favipiravir) Ribofuranosyl 5′-Triphosphate towards Influenza A Virus Polymerase</strong><br />
Zhinan Jin, Lucas K. Smith, Vivek K. Rajwanshi, Baek Kim, Jerome Deval<br />
<em>PLoS ONE</em> (2013-07-10) <a href="https://doi.org/f5br92">https://doi.org/f5br92</a><br />
DOI: <a href="https://doi.org/10.1371/journal.pone.0068347">10.1371/journal.pone.0068347</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23874596">23874596</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3707847">PMC3707847</a></p>
</div>
<div id="ref-13Oz7yRPx">
<p>238. <strong>TEMPORARY REMOVAL: Experimental Treatment with Favipiravir for COVID-19: An Open-Label Control Study</strong><br />
Qingxian Cai, Minghui Yang, Dongjing Liu, Jun Chen, Dan Shu, Junxia Xia, Xuejiao Liao, Yuanbo Gu, Qiue Cai, Yang Yang, … Lei Liu<br />
<em>Engineering</em> (2020-03) <a href="https://doi.org/ggpprd">https://doi.org/ggpprd</a><br />
DOI: <a href="https://doi.org/10.1016/j.eng.2020.03.007">10.1016/j.eng.2020.03.007</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32346491">32346491</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7185795">PMC7185795</a></p>
</div>
<div id="ref-1BYoe0y2H">
<p>239. <strong>Remdesivir EUA Letter of Authorization</strong><br />
Denise M Hinton<br />
(2020-05-01) <a href="https://www.fda.gov/media/137564/download">https://www.fda.gov/media/137564/download</a></p>
</div>
<div id="ref-yTCAmOyt">
<p>240. <strong>A Phase 3 Randomized Study to Evaluate the Safety and Antiviral Activity of Remdesivir (GS-5734™) in Participants With Severe COVID-19</strong><br />
Gilead Sciences<br />
<em>clinicaltrials.gov</em> (2020-07-23) <a href="https://clinicaltrials.gov/ct2/show/NCT04292899">https://clinicaltrials.gov/ct2/show/NCT04292899</a></p>
</div>
<div id="ref-1ATTPbCCO">
<p>241. <strong>A Multicenter, Adaptive, Randomized Blinded Controlled Trial of the Safety and Efficacy of Investigational Therapeutics for the Treatment of COVID-19 in Hospitalized Adults</strong><br />
National Institute of Allergy and Infectious Diseases (NIAID)<br />
<em>clinicaltrials.gov</em> (2020-07-20) <a href="https://clinicaltrials.gov/ct2/show/NCT04280705">https://clinicaltrials.gov/ct2/show/NCT04280705</a></p>
</div>
<div id="ref-DtpdBrgt">
<p>242. <strong>Remdesivir for the Treatment of Covid-19 — Preliminary Report</strong><br />
John H. Beigel, Kay M. Tomashek, Lori E. Dodd, Aneesh K. Mehta, Barry S. Zingman, Andre C. Kalil, Elizabeth Hohmann, Helen Y. Chu, Annie Luetkemeyer, Susan Kline, … H. Clifford Lane<br />
<em>New England Journal of Medicine</em> (2020-05-22) <a href="https://doi.org/dwkd">https://doi.org/dwkd</a><br />
DOI: <a href="https://doi.org/10.1056/nejmoa2007764">10.1056/nejmoa2007764</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32445440">32445440</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7262788">PMC7262788</a></p>
</div>
<div id="ref-16zLDIOyk">
<p>243. <strong>Compassionate Use of Remdesivir for Patients with Severe Covid-19</strong><br />
Jonathan Grein, Norio Ohmagari, Daniel Shin, George Diaz, Erika Asperges, Antonella Castagna, Torsten Feldt, Gary Green, Margaret L. Green, François-Xavier Lescure, … Timothy Flanigan<br />
<em>New England Journal of Medicine</em> (2020-04-10) <a href="https://doi.org/ggrm99">https://doi.org/ggrm99</a><br />
DOI: <a href="https://doi.org/10.1056/nejmoa2007016">10.1056/nejmoa2007016</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32275812">32275812</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7169476">PMC7169476</a></p>
</div>
<div id="ref-YaxKcqq8">
<p>244. <strong>The antiviral compound remdesivir potently inhibits RNA-dependent RNA polymerase from Middle East respiratory syndrome coronavirus</strong><br />
Calvin J Gordon, Egor P Tchesnokov, Joy Y. Feng, Danielle P Porter, Matthias Gotte<br />
<em>Journal of Biological Chemistry</em> (2020-02-24) <a href="https://doi.org/ggqm6x">https://doi.org/ggqm6x</a><br />
DOI: <a href="https://doi.org/10.1074/jbc.ac120.013056">10.1074/jbc.ac120.013056</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32094225">32094225</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7152756">PMC7152756</a></p>
</div>
<div id="ref-lIko9Imd">
<p>245. <strong>Coronavirus Susceptibility to the Antiviral Remdesivir (GS-5734) Is Mediated by the Viral Polymerase and the Proofreading Exoribonuclease</strong><br />
Maria L. Agostini, Erica L. Andres, Amy C. Sims, Rachel L. Graham, Timothy P. Sheahan, Xiaotao Lu, Everett Clinton Smith, James Brett Case, Joy Y. Feng, Robert Jordan, … Mark R. Denison<br />
<em>mBio</em> (2018-03-06) <a href="https://doi.org/gc45v6">https://doi.org/gc45v6</a><br />
DOI: <a href="https://doi.org/10.1128/mbio.00221-18">10.1128/mbio.00221-18</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29511076">29511076</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5844999">PMC5844999</a></p>
</div>
<div id="ref-IhdcvbzC">
<p>246. <strong>Remdesivir and chloroquine effectively inhibit the recently emerged novel coronavirus (2019-nCoV) in vitro</strong><br />
Manli Wang, Ruiyuan Cao, Leike Zhang, Xinglou Yang, Jia Liu, Mingyue Xu, Zhengli Shi, Zhihong Hu, Wu Zhong, Gengfu Xiao<br />
<em>Cell Research</em> (2020-02-04) <a href="https://doi.org/ggkbsg">https://doi.org/ggkbsg</a><br />
DOI: <a href="https://doi.org/10.1038/s41422-020-0282-0">10.1038/s41422-020-0282-0</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32020029">32020029</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7054408">PMC7054408</a></p>
</div>
<div id="ref-1H0NPTMw9">
<p>247. <strong>Broad-spectrum antiviral GS-5734 inhibits both epidemic and zoonotic coronaviruses</strong><br />
Timothy P. Sheahan, Amy C. Sims, Rachel L. Graham, Vineet D. Menachery, Lisa E. Gralinski, James B. Case, Sarah R. Leist, Krzysztof Pyrc, Joy Y. Feng, Iva Trantcheva, … Ralph S. Baric<br />
<em>Science Translational Medicine</em> (2017-06-28) <a href="https://doi.org/gc3grb">https://doi.org/gc3grb</a><br />
DOI: <a href="https://doi.org/10.1126/scitranslmed.aal3653">10.1126/scitranslmed.aal3653</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28659436">28659436</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5567817">PMC5567817</a></p>
</div>
<div id="ref-gnfviuGp">
<p>248. <strong>A Randomized, Controlled Trial of Ebola Virus Disease Therapeutics</strong><br />
Sabue Mulangu, Lori E. Dodd, Richard T. Davey, Olivier Tshiani Mbaya, Michael Proschan, Daniel Mukadi, Mariano Lusakibanza Manzo, Didier Nzolo, Antoine Tshomba Oloma, Augustin Ibanda, … the PALM Writing Group<br />
<em>New England Journal of Medicine</em> (2019-12-12) <a href="https://doi.org/ggqmx4">https://doi.org/ggqmx4</a><br />
DOI: <a href="https://doi.org/10.1056/nejmoa1910993">10.1056/nejmoa1910993</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31774950">31774950</a></p>
</div>
<div id="ref-D4HwvRMd">
<p>249. <strong>Did an experimental drug help a U.S. coronavirus patient?</strong><br />
Jon Cohen<br />
<em>Science</em> (2020-03-13) <a href="https://doi.org/ggqm62">https://doi.org/ggqm62</a><br />
DOI: <a href="https://doi.org/10.1126/science.abb7243">10.1126/science.abb7243</a></p>
</div>
<div id="ref-zoLViMNg">
<p>250. <strong>First 12 patients with coronavirus disease 2019 (COVID-19) in the United States</strong><br />
Stephanie A. Kujawski, Karen K Wong, Jennifer P. Collins, Lauren Epstein, Marie E. Killerby, Claire M. Midgley, Glen R. Abedi, N. Seema Ahmed, Olivia Almendares, Francisco N. Alvarez, … Jing Zhang<br />
<em>medRxiv</em> (2020-03-12) <a href="https://doi.org/ggqm6z">https://doi.org/ggqm6z</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.09.20032896">10.1101/2020.03.09.20032896</a></p>
</div>
<div id="ref-gleq16qf">
<p>251. <strong>First Case of 2019 Novel Coronavirus in the United States</strong><br />
Michelle L. Holshue, Chas DeBolt, Scott Lindquist, Kathy H. Lofy, John Wiesman, Hollianne Bruce, Christopher Spitters, Keith Ericson, Sara Wilkerson, Ahmet Tural, … Satish K. Pillai<br />
<em>New England Journal of Medicine</em> (2020-01-31) <a href="https://doi.org/ggjvr6">https://doi.org/ggjvr6</a><br />
DOI: <a href="https://doi.org/10.1056/nejmoa2001191">10.1056/nejmoa2001191</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32004427">32004427</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7092802">PMC7092802</a></p>
</div>
<div id="ref-w5y1LWUR">
<p>252. <strong>Remdesivir for 5 or 10 Days in Patients with Severe Covid-19</strong><br />
Jason D. Goldman, David C. B. Lye, David S. Hui, Kristen M. Marks, Raffaele Bruno, Rocio Montejano, Christoph D. Spinner, Massimo Galli, Mi-Young Ahn, Ronald G. Nahass, … Aruna Subramanian<br />
<em>New England Journal of Medicine</em> (2020-05-27) <a href="https://doi.org/ggz7qv">https://doi.org/ggz7qv</a><br />
DOI: <a href="https://doi.org/10.1056/nejmoa2015301">10.1056/nejmoa2015301</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32459919">32459919</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7377062">PMC7377062</a></p>
</div>
<div id="ref-oso9r21L">
<p>253. <strong>A Phase 3 Randomized Study to Evaluate the Safety and Antiviral Activity of Remdesivir (GS-5734™) in Participants With Moderate COVID-19 Compared to Standard of Care Treatment</strong><br />
Gilead Sciences<br />
<em>clinicaltrials.gov</em> (2020-07-23) <a href="https://clinicaltrials.gov/ct2/show/NCT04292730">https://clinicaltrials.gov/ct2/show/NCT04292730</a></p>
</div>
<div id="ref-1q4PmqTm">
<p>254. <strong>Multi-centre, adaptive, randomized trial of the safety and efficacy of treatments of COVID-19 in hospitalized adults</strong><br />
EU Clinical Trials Register<br />
(2020-03-09) <a href="https://www.clinicaltrialsregister.eu/ctr-search/trial/2020-000936-23/FR">https://www.clinicaltrialsregister.eu/ctr-search/trial/2020-000936-23/FR</a></p>
</div>
<div id="ref-rUTsIlZs">
<p>255. <strong>A Phase 3 Randomized, Double-blind, Placebo-controlled Multicenter Study to Evaluate the Efficacy and Safety of Remdesivir in Hospitalized Adult Patients With Mild and Moderate COVID-19.</strong><br />
Bin Cao<br />
<em>clinicaltrials.gov</em> (2020-04-13) <a href="https://clinicaltrials.gov/ct2/show/NCT04252664">https://clinicaltrials.gov/ct2/show/NCT04252664</a></p>
</div>
<div id="ref-JSoOOX98">
<p>256. <strong>A Phase 3 Randomized, Double-blind, Placebo-controlled, Multicenter Study to Evaluate the Efficacy and Safety of Remdesivir in Hospitalized Adult Patients With Severe COVID-19.</strong><br />
Bin Cao<br />
<em>clinicaltrials.gov</em> (2020-04-13) <a href="https://clinicaltrials.gov/ct2/show/NCT04257656">https://clinicaltrials.gov/ct2/show/NCT04257656</a></p>
</div>
<div id="ref-26dpptLw">
<p>257. <strong>Proteases Essential for Human Influenza Virus Entry into Cells and Their Inhibitors as Potential Therapeutic Agents</strong><br />
Hiroshi Kido, Yuushi Okumura, Hiroshi Yamada, Trong Quang Le, Mihiro Yano<br />
<em>Current Pharmaceutical Design</em> (2007-02-01) <a href="https://doi.org/bts3xp">https://doi.org/bts3xp</a><br />
DOI: <a href="https://doi.org/10.2174/138161207780162971">10.2174/138161207780162971</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17311557">17311557</a></p>
</div>
<div id="ref-f7csoyEu">
<p>258. <strong>Protease inhibitors targeting coronavirus and filovirus entry</strong><br />
Yanchen Zhou, Punitha Vedantham, Kai Lu, Juliet Agudelo, Ricardo Carrion, Jerritt W. Nunneley, Dale Barnard, Stefan Pöhlmann, James H. McKerrow, Adam R. Renslo, Graham Simmons<br />
<em>Antiviral Research</em> (2015-04) <a href="https://doi.org/ggr984">https://doi.org/ggr984</a><br />
DOI: <a href="https://doi.org/10.1016/j.antiviral.2015.01.011">10.1016/j.antiviral.2015.01.011</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25666761">25666761</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4774534">PMC4774534</a></p>
</div>
<div id="ref-15JYG0AIU">
<p>259. <strong>Design of Wide-Spectrum Inhibitors Targeting Coronavirus Main Proteases</strong><br />
Haitao Yang, Weiqing Xie, Xiaoyu Xue, Kailin Yang, Jing Ma, Wenxue Liang, Qi Zhao, Zhe Zhou, Duanqing Pei, John Ziebuhr, … Zihe Rao<br />
<em>PLoS Biology</em> (2005-09-06) <a href="https://doi.org/bcm9k7">https://doi.org/bcm9k7</a><br />
DOI: <a href="https://doi.org/10.1371/journal.pbio.0030324">10.1371/journal.pbio.0030324</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16128623">16128623</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1197287">PMC1197287</a></p>
</div>
<div id="ref-XfOTPQPJ">
<p>260. <strong>The newly emerged SARS-Like coronavirus HCoV-EMC also has an “Achilles’ heel”: current effective inhibitor targeting a 3C-like protease</strong><br />
Zhilin Ren, Liming Yan, Ning Zhang, Yu Guo, Cheng Yang, Zhiyong Lou, Zihe Rao<br />
<em>Protein &amp; Cell</em> (2013-04-03) <a href="https://doi.org/ggr7vh">https://doi.org/ggr7vh</a><br />
DOI: <a href="https://doi.org/10.1007/s13238-013-2841-3">10.1007/s13238-013-2841-3</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23549610">23549610</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4875521">PMC4875521</a></p>
</div>
<div id="ref-4Sja6dIz">
<p>261. <strong>Structure of Mpro from SARS-CoV-2 and discovery of its inhibitors</strong><br />
Zhenming Jin, Xiaoyu Du, Yechun Xu, Yongqiang Deng, Meiqin Liu, Yao Zhao, Bing Zhang, Xiaofeng Li, Leike Zhang, Chao Peng, … Haitao Yang<br />
<em>Nature</em> (2020-04-09) <a href="https://doi.org/ggrp42">https://doi.org/ggrp42</a><br />
DOI: <a href="https://doi.org/10.1038/s41586-020-2223-y">10.1038/s41586-020-2223-y</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32272481">32272481</a></p>
</div>
<div id="ref-17EErh01r">
<p>262. <strong>Ebselen, a promising antioxidant drug: mechanisms of action and targets of biological pathways</strong><br />
Gajendra Kumar Azad, Raghuvir S. Tomar<br />
<em>Molecular Biology Reports</em> (2014-05-28) <a href="https://doi.org/f6cnq3">https://doi.org/f6cnq3</a><br />
DOI: <a href="https://doi.org/10.1007/s11033-014-3417-x">10.1007/s11033-014-3417-x</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24867080">24867080</a></p>
</div>
<div id="ref-10NcfBSiT">
<p>263. <strong>Crystal structure of SARS-CoV-2 main protease provides a basis for design of improved α-ketoamide inhibitors</strong><br />
Linlin Zhang, Daizong Lin, Xinyuanyuan Sun, Ute Curth, Christian Drosten, Lucie Sauerhering, Stephan Becker, Katharina Rox, Rolf Hilgenfeld<br />
<em>Science</em> (2020-03-20) <a href="https://doi.org/ggp9sb">https://doi.org/ggp9sb</a><br />
DOI: <a href="https://doi.org/10.1126/science.abb3405">10.1126/science.abb3405</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32198291">32198291</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7164518">PMC7164518</a></p>
</div>
<div id="ref-vXFGEybh">
<p>264. <strong>Target discovery of ebselen with a biotinylated probe</strong><br />
Zhenzhen Chen, Zhongyao Jiang, Nan Chen, Qian Shi, Lili Tong, Fanpeng Kong, Xiufen Cheng, Hao Chen, Chu Wang, Bo Tang<br />
<em>Chemical Communications</em> (2018) <a href="https://doi.org/ggrtcm">https://doi.org/ggrtcm</a><br />
DOI: <a href="https://doi.org/10.1039/c8cc04258f">10.1039/c8cc04258f</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30091742">30091742</a></p>
</div>
<div id="ref-CIeWPFHG">
<p>265. <strong>Lisinopril - Drug Usage Statistics</strong><br />
ClinCalc DrugStats Database<br />
<a href="https://clincalc.com/DrugStats/Drugs/Lisinopril">https://clincalc.com/DrugStats/Drugs/Lisinopril</a></p>
</div>
<div id="ref-pbfxOvnK">
<p>266. <strong>Hypertension Hot Potato — Anatomy of the Angiotensin-Receptor Blocker Recalls</strong><br />
J. Brian Byrd, Glenn M. Chertow, Vivek Bhalla<br />
<em>New England Journal of Medicine</em> (2019-04-25) <a href="https://doi.org/ggvc7g">https://doi.org/ggvc7g</a><br />
DOI: <a href="https://doi.org/10.1056/nejmp1901657">10.1056/nejmp1901657</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30865819">30865819</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7066505">PMC7066505</a></p>
</div>
<div id="ref-17xTGqKp2">
<p>267. <strong>Hypertension, the renin–angiotensin system, and the risk of lower respiratory tract infections and lung injury: implications for COVID-19</strong><br />
Reinhold Kreutz, Engi Abd El-Hady Algharably, Michel Azizi, Piotr Dobrowolski, Tomasz Guzik, Andrzej Januszewicz, Alexandre Persu, Aleksander Prejbisz, Thomas Günther Riemer, Ji-Guang Wang, Michel Burnier<br />
<em>Cardiovascular Research</em> (2020-08-01) <a href="https://doi.org/ggtwpj">https://doi.org/ggtwpj</a><br />
DOI: <a href="https://doi.org/10.1093/cvr/cvaa097">10.1093/cvr/cvaa097</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32293003">32293003</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7184480">PMC7184480</a></p>
</div>
<div id="ref-ZhhV4ztO">
<p>268. <strong>Angiotensin converting enzyme 2 activity and human atrial fibrillation: increased plasma angiotensin converting enzyme 2 activity is associated with atrial fibrillation and more advanced left atrial structural remodelling</strong><br />
Tomos E. Walters, Jonathan M. Kalman, Sheila K. Patel, Megan Mearns, Elena Velkoska, Louise M. Burrell<br />
<em>Europace</em> (2016-10-12) <a href="https://doi.org/gbt2jw">https://doi.org/gbt2jw</a><br />
DOI: <a href="https://doi.org/10.1093/europace/euw246">10.1093/europace/euw246</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27738071">27738071</a></p>
</div>
<div id="ref-10q7vcaWw">
<p>269. <strong>Cardiovascular Disease, Drug Therapy, and Mortality in Covid-19</strong><br />
Mandeep R. Mehra, Sapan S. Desai, SreyRam Kuy, Timothy D. Henry, Amit N. Patel<br />
<em>New England Journal of Medicine</em> (2020-06-18) <a href="https://doi.org/ggtp6v">https://doi.org/ggtp6v</a><br />
DOI: <a href="https://doi.org/10.1056/nejmoa2007621">10.1056/nejmoa2007621</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32356626">32356626</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7206931">PMC7206931</a></p>
</div>
<div id="ref-19quOF2kK">
<p>270. <strong>Response by Cohen et al to Letter Regarding Article, “Association of Inpatient Use of Angiotensin-Converting Enzyme Inhibitors and Angiotensin II Receptor Blockers With Mortality Among Patients With Hypertension Hospitalized With COVID-19”</strong><br />
Jordana B. Cohen, Thomas C. Hanff, Andrew M. South, Matthew A. Sparks, Swapnil Hiremath, Adam P. Bress, J. Brian Byrd, Julio A. Chirinos<br />
<em>Circulation Research</em> (2020-06-05) <a href="https://doi.org/gg3xsg">https://doi.org/gg3xsg</a><br />
DOI: <a href="https://doi.org/10.1161/circresaha.120.317205">10.1161/circresaha.120.317205</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32496917">32496917</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7265880">PMC7265880</a></p>
</div>
<div id="ref-k5xEN3Ke">
<p>271. <strong>Sound Science before Quick Judgement Regarding RAS Blockade in COVID-19</strong><br />
Matthew A. Sparks, Andrew South, Paul Welling, J. Matt Luther, Jordana Cohen, James Brian Byrd, Louise M. Burrell, Daniel Batlle, Laurie Tomlinson, Vivek Bhalla, … Swapnil Hiremath<br />
<em>Clinical Journal of the American Society of Nephrology</em> (2020-05-07) <a href="https://doi.org/ggq8gn">https://doi.org/ggq8gn</a><br />
DOI: <a href="https://doi.org/10.2215/cjn.03530320">10.2215/cjn.03530320</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32220930">32220930</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7269218">PMC7269218</a></p>
</div>
<div id="ref-H0oEjZR8">
<p>272. <strong>The Randomized Elimination or Prolongation of Angiotensin Converting Enzyme Inhibitors and Angiotensin Receptor Blockers in Coronavirus Disease 2019</strong><br />
Julio A. Chirinos<br />
<em>clinicaltrials.gov</em> (2020-04-22) <a href="https://clinicaltrials.gov/ct2/show/NCT04338009">https://clinicaltrials.gov/ct2/show/NCT04338009</a></p>
</div>
<div id="ref-kuUcA1G9">
<p>273. <strong>Stopping ACE-inhibitors in COVID-19: A Randomized Controlled Trial</strong><br />
Medical University Innsbruck<br />
<em>clinicaltrials.gov</em> (2020-06-08) <a href="https://clinicaltrials.gov/ct2/show/NCT04353596">https://clinicaltrials.gov/ct2/show/NCT04353596</a></p>
</div>
<div id="ref-3ccJAyrG">
<p>274. <strong>Randomized Controlled Trial of Losartan for Patients With COVID-19 Not Requiring Hospitalization</strong><br />
University of Minnesota<br />
<em>clinicaltrials.gov</em> (2020-06-11) <a href="https://clinicaltrials.gov/ct2/show/NCT04311177">https://clinicaltrials.gov/ct2/show/NCT04311177</a></p>
</div>
<div id="ref-NCb9Wq2z">
<p>275. <strong>Randomized Controlled Trial of Losartan for Patients With COVID-19 Requiring Hospitalization</strong><br />
University of Minnesota<br />
<em>clinicaltrials.gov</em> (2020-07-17) <a href="https://clinicaltrials.gov/ct2/show/NCT04312009">https://clinicaltrials.gov/ct2/show/NCT04312009</a></p>
</div>
<div id="ref-5fkIarR7">
<p>276. <strong>The CORONAvirus Disease 2019 Angiotensin Converting Enzyme Inhibitor/Angiotensin Receptor Blocker InvestigatiON (CORONACION) Randomized Clinical Trial</strong><br />
Prof John William McEvoy<br />
<em>clinicaltrials.gov</em> (2020-06-26) <a href="https://clinicaltrials.gov/ct2/show/NCT04330300">https://clinicaltrials.gov/ct2/show/NCT04330300</a></p>
</div>
<div id="ref-c7V9rAFR">
<p>277. <strong>A Randomized, Double-blind, Placebo-Controlled Trial to Evaluate the Efficacy of Ramipril to Prevent ICU Admission, Need for Mechanical Ventilation or Death in Persons With COVID-19</strong><br />
Rohit Loomba<br />
<em>clinicaltrials.gov</em> (2020-07-07) <a href="https://clinicaltrials.gov/ct2/show/NCT04366050">https://clinicaltrials.gov/ct2/show/NCT04366050</a></p>
</div>
<div id="ref-1Hrsy8Hiq">
<p>278. <strong>The Coronavirus Conundrum: ACE2 and Hypertension Edition</strong><br />
Matthew Sparks, Swapnil Hiremath<br />
<em>NephJC</em> <a href="http://www.nephjc.com/news/covidace2">http://www.nephjc.com/news/covidace2</a></p>
</div>
<div id="ref-ggYRK4FU">
<p>279. <strong>Lysosomotropic agents as HCV entry inhibitors</strong><br />
Usman A Ashfaq, Tariq Javed, Sidra Rehman, Zafar Nawaz, Sheikh Riazuddin<br />
<em>Virology Journal</em> (2011-04-12) <a href="https://doi.org/dr5g4m">https://doi.org/dr5g4m</a><br />
DOI: <a href="https://doi.org/10.1186/1743-422x-8-163">10.1186/1743-422x-8-163</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21481279">21481279</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3090357">PMC3090357</a></p>
</div>
<div id="ref-16IRfGFKb">
<p>280. <strong>New concepts in antimalarial use and mode of action in dermatology</strong><br />
Sunil Kalia, Jan P Dutz<br />
<em>Dermatologic Therapy</em> (2007-07) <a href="https://doi.org/fv69cb">https://doi.org/fv69cb</a><br />
DOI: <a href="https://doi.org/10.1111/j.1529-8019.2007.00131.x">10.1111/j.1529-8019.2007.00131.x</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17970883">17970883</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7163426">PMC7163426</a></p>
</div>
<div id="ref-x6nInyCf">
<p>281. <strong>Chloroquine is a potent inhibitor of SARS coronavirus infection and spread</strong><br />
Martin J Vincent, Eric Bergeron, Suzanne Benjannet, Bobbie R Erickson, Pierre E Rollin, Thomas G Ksiazek, Nabil G Seidah, Stuart T Nichol<br />
<em>Virology Journal</em> (2005) <a href="https://doi.org/dvbds4">https://doi.org/dvbds4</a><br />
DOI: <a href="https://doi.org/10.1186/1743-422x-2-69">10.1186/1743-422x-2-69</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16115318">16115318</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1232869">PMC1232869</a></p>
</div>
<div id="ref-1GBsnkeLK">
<p>282. <strong>In Vitro Antiviral Activity and Projection of Optimized Dosing Design of Hydroxychloroquine for the Treatment of Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2)</strong><br />
Xueting Yao, Fei Ye, Miao Zhang, Cheng Cui, Baoying Huang, Peihua Niu, Xu Liu, Li Zhao, Erdan Dong, Chunli Song, … Dongyang Liu<br />
<em>Clinical Infectious Diseases</em> (2020-03-09) <a href="https://doi.org/ggpx7z">https://doi.org/ggpx7z</a><br />
DOI: <a href="https://doi.org/10.1093/cid/ciaa237">10.1093/cid/ciaa237</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32150618">32150618</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7108130">PMC7108130</a></p>
</div>
<div id="ref-nEaG534R">
<p>283. <strong>Mechanism of Action of Hydroxychloroquine in the Antiphospholipid Syndrome</strong><br />
Nadine Müller-Calleja, Davit Manukyan, Wolfram Ruf, Karl Lackner<br />
<em>Blood</em> (2016-12-02) <a href="https://doi.org/ggrm82">https://doi.org/ggrm82</a><br />
DOI: <a href="https://doi.org/10.1182/blood.v128.22.5023.5023">10.1182/blood.v128.22.5023.5023</a></p>
</div>
<div id="ref-rL8owM5Y">
<p>284. <strong>14th International Congress on Antiphospholipid Antibodies Task Force Report on Antiphospholipid Syndrome Treatment Trends</strong><br />
Doruk Erkan, Cassyanne L. Aguiar, Danieli Andrade, Hannah Cohen, Maria J. Cuadrado, Adriana Danowski, Roger A. Levy, Thomas L. Ortel, Anisur Rahman, Jane E. Salmon, … Michael D. Lockshin<br />
<em>Autoimmunity Reviews</em> (2014-06) <a href="https://doi.org/ggp8r8">https://doi.org/ggp8r8</a><br />
DOI: <a href="https://doi.org/10.1016/j.autrev.2014.01.053">10.1016/j.autrev.2014.01.053</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24468415">24468415</a></p>
</div>
<div id="ref-134vCyU3E">
<p>285. <strong>What is the role of hydroxychloroquine in reducing thrombotic risk in patients with antiphospholipid antibodies?</strong><br />
Tzu-Fei Wang, Wendy Lim<br />
<em>Hematology</em> (2016-12-02) <a href="https://doi.org/ggrn3k">https://doi.org/ggrn3k</a><br />
DOI: <a href="https://doi.org/10.1182/asheducation-2016.1.714">10.1182/asheducation-2016.1.714</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27913551">27913551</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6142483">PMC6142483</a></p>
</div>
<div id="ref-ovHLMvCi">
<p>286. <strong>Hydroxychloroquine and azithromycin as a treatment of COVID-19: results of an open-label non-randomized clinical trial</strong><br />
Philippe Gautret, Jean-Christophe Lagier, Philippe Parola, Van Thuan Hoang, Line Meddeb, Morgane Mailhe, Barbara Doudier, Johan Courjon, Valérie Giordanengo, Vera Esteves Vieira, … Didier Raoult<br />
<em>International Journal of Antimicrobial Agents</em> (2020-03) <a href="https://doi.org/dp7d">https://doi.org/dp7d</a><br />
DOI: <a href="https://doi.org/10.1016/j.ijantimicag.2020.105949">10.1016/j.ijantimicag.2020.105949</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32205204">32205204</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7102549">PMC7102549</a></p>
</div>
<div id="ref-10J3hBiMo">
<p>287. <strong>Official Statement from International Society of Antimicrobial Chemotherapy</strong><br />
Andreas Voss<br />
(2020-04-03) <a href="https://www.isac.world/news-and-publications/official-isac-statement">https://www.isac.world/news-and-publications/official-isac-statement</a></p>
</div>
<div id="ref-18KmvBOFr">
<p>288. <strong>No Evidence of Rapid Antiviral Clearance or Clinical Benefit with the Combination of Hydroxychloroquine and Azithromycin in Patients with Severe COVID-19 Infection</strong><br />
Jean Michel Molina, Constance Delaugerre, Jerome Le Goff, Breno Mela-Lima, Diane Ponscarme, Lauriane Goldwirt, Nathalie de Castro<br />
<em>Médecine et Maladies Infectieuses</em> (2020-03) <a href="https://doi.org/ggqzrb">https://doi.org/ggqzrb</a><br />
DOI: <a href="https://doi.org/10.1016/j.medmal.2020.03.006">10.1016/j.medmal.2020.03.006</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32240719">32240719</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7195369">PMC7195369</a></p>
</div>
<div id="ref-wzHb7Mj5">
<p>289. <strong>Efficacy of hydroxychloroquine in patients with COVID-19: results of a randomized clinical trial</strong><br />
Zhaowei Chen, Jijia Hu, Zongwei Zhang, Shan Jiang, Shoumeng Han, Dandan Yan, Ruhong Zhuang, Ben Hu, Zhan Zhang<br />
<em>medRxiv</em> (2020-03-31) <a href="https://doi.org/ggqm4v">https://doi.org/ggqm4v</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.22.20040758">10.1101/2020.03.22.20040758</a></p>
</div>
<div id="ref-o5gwQA7W">
<p>290. <strong>A pilot study of hydroxychloroquine in treatment of patients with common coronavirus disease-19 (COVID-19)</strong><br />
CHEN Jun, LIU Danping, LIU Li, LIU Ping, XU Qingnian, XIA Lu, LING Yun, HUANG Dan, SONG Shuli, ZHANG Dandan, … LU Hongzhou<br />
<em>Journal of Zhejiang University (Medical Sciences)</em> (2020-03) <a href="https://doi.org/10.3785/j.issn.1008-9292.2020.03.03">https://doi.org/10.3785/j.issn.1008-9292.2020.03.03</a><br />
DOI: <a href="https://doi.org/10.3785/j.issn.1008-9292.2020.03.03">10.3785/j.issn.1008-9292.2020.03.03</a></p>
</div>
<div id="ref-aiItPxSY">
<p>291. <strong>Breakthrough: Chloroquine phosphate has shown apparent efficacy in treatment of COVID-19 associated pneumonia in clinical studies</strong><br />
Jianjun Gao, Zhenxue Tian, Xu Yang<br />
<em>BioScience Trends</em> (2020) <a href="https://doi.org/ggm3mv">https://doi.org/ggm3mv</a><br />
DOI: <a href="https://doi.org/10.5582/bst.2020.01047">10.5582/bst.2020.01047</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32074550">32074550</a></p>
</div>
<div id="ref-1BdQKtXnY">
<p>292. <strong>Targeting the Endocytic Pathway and Autophagy Process as a Novel Therapeutic Strategy in COVID-19</strong><br />
Naidi Yang, Han-Ming Shen<br />
<em>International Journal of Biological Sciences</em> (2020) <a href="https://doi.org/ggqspm">https://doi.org/ggqspm</a><br />
DOI: <a href="https://doi.org/10.7150/ijbs.45498">10.7150/ijbs.45498</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32226290">32226290</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7098027">PMC7098027</a></p>
</div>
<div id="ref-z8XnLYWM">
<p>293. <strong>SARS-CoV-2: an Emerging Coronavirus that Causes a Global Threat</strong><br />
Jun Zheng<br />
<em>International Journal of Biological Sciences</em> (2020) <a href="https://doi.org/ggqspr">https://doi.org/ggqspr</a><br />
DOI: <a href="https://doi.org/10.7150/ijbs.45053">10.7150/ijbs.45053</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32226285">32226285</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7098030">PMC7098030</a></p>
</div>
<div id="ref-1BApBNHvd">
<p>294. <strong>Hydroxychloroquine or chloroquine with or without a macrolide for treatment of COVID-19: a multinational registry analysis</strong><br />
Mandeep R Mehra, Sapan S Desai, Frank Ruschitzka, Amit N Patel<br />
<em>The Lancet</em> (2020-05) <a href="https://doi.org/ggwzsb">https://doi.org/ggwzsb</a><br />
DOI: <a href="https://doi.org/10.1016/s0140-6736(20)31180-6">10.1016/s0140-6736(20)31180-6</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32450107">32450107</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7255293">PMC7255293</a></p>
</div>
<div id="ref-11FsxWalS">
<p>295. <strong>Retraction—Hydroxychloroquine or chloroquine with or without a macrolide for treatment of COVID-19: a multinational registry analysis</strong><br />
Mandeep R Mehra, Frank Ruschitzka, Amit N Patel<br />
<em>The Lancet</em> (2020-06) <a href="https://doi.org/ggzqng">https://doi.org/ggzqng</a><br />
DOI: <a href="https://doi.org/10.1016/s0140-6736(20)31324-6">10.1016/s0140-6736(20)31324-6</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32511943">32511943</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7274621">PMC7274621</a></p>
</div>
<div id="ref-dY9kXLAS">
<p>296. <strong>Life Threatening Severe QTc Prolongation in Patient with Systemic Lupus Erythematosus due to Hydroxychloroquine</strong><br />
John P. O’Laughlin, Parag H. Mehta, Brian C. Wong<br />
<em>Case Reports in Cardiology</em> (2016) <a href="https://doi.org/ggqzrc">https://doi.org/ggqzrc</a><br />
DOI: <a href="https://doi.org/10.1155/2016/4626279">10.1155/2016/4626279</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27478650">27478650</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4960328">PMC4960328</a></p>
</div>
<div id="ref-wy3S7Cpa">
<p>297. <strong>Keep the QT interval: It is a reliable predictor of ventricular arrhythmias</strong><br />
Dan M. Roden<br />
<em>Heart Rhythm</em> (2008-08) <a href="https://doi.org/d5rchx">https://doi.org/d5rchx</a><br />
DOI: <a href="https://doi.org/10.1016/j.hrthm.2008.05.008">10.1016/j.hrthm.2008.05.008</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/18675237">18675237</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3212752">PMC3212752</a></p>
</div>
<div id="ref-OQ0b31BA">
<p>298. <strong>Safety of hydroxychloroquine, alone and in combination with azithromycin, in light of rapid wide-spread use for COVID-19: a multinational, network cohort and self-controlled case series study</strong><br />
Jennifer C. E Lane, James Weaver, Kristin Kostka, Talita Duarte-Salles, Maria Tereza F. Abrahao, Heba Alghoul, Osaid Alser, Thamir M Alshammari, Patricia Biedermann, Edward Burn, … Daniel Prieto-Alhambra<br />
<em>medRxiv</em> (2020-05-31) <a href="https://doi.org/ggrn7s">https://doi.org/ggrn7s</a><br />
DOI: <a href="https://doi.org/10.1101/2020.04.08.20054551">10.1101/2020.04.08.20054551</a></p>
</div>
<div id="ref-1CgZR55GZ">
<p>299. <strong>Chloroquine diphosphate in two different dosages as adjunctive therapy of hospitalized patients with severe respiratory syndrome in the context of coronavirus (SARS-CoV-2) infection: Preliminary safety results of a randomized, double-blinded, phase IIb clinical trial (CloroCovid-19 Study)</strong><br />
Mayla Gabriela Silva Borba, Fernando de Almeida Val, Vanderson Sousa Sampaio, Marcia Almeida Ara&amp;uacutejo Alexandre, Gisely Cardoso Melo, Marcelo Brito, Maria Paula Gomes Mour&amp;atildeo, José Diego Brito Sousa, Djane Clarys Baia-da-Silva, Marcus Vinitius Farias Guerra, … CloroCovid-19 Team<br />
<em>medRxiv</em> (2020-04-16) <a href="https://doi.org/ggr3nj">https://doi.org/ggr3nj</a><br />
DOI: <a href="https://doi.org/10.1101/2020.04.07.20056424">10.1101/2020.04.07.20056424</a></p>
</div>
<div id="ref-1EcscrT32">
<p>300. <strong>Heart risk concerns mount around use of chloroquine and hydroxychloroquine for Covid-19 treatment</strong><br />
Jacqueline Howard, Elizabeth Cohen, Nadia Kounang, Per Nyberg<br />
<em>CNN</em> (2020-04-14) <a href="https://www.cnn.com/2020/04/13/health/chloroquine-risks-coronavirus-treatment-trials-study/index.html">https://www.cnn.com/2020/04/13/health/chloroquine-risks-coronavirus-treatment-trials-study/index.html</a></p>
</div>
<div id="ref-AMQ5402v">
<p>301. <strong>WHO Director-General’s opening remarks at the media briefing on COVID-19</strong><br />
World Health Organization<br />
(2020-05-25) <a href="https://www.who.int/dg/speeches/detail/who-director-general-s-opening-remarks-at-the-media-briefing-on-covid-19---25-may-2020">https://www.who.int/dg/speeches/detail/who-director-general-s-opening-remarks-at-the-media-briefing-on-covid-19---25-may-2020</a></p>
</div>
<div id="ref-RXr9BKKT">
<p>302. <strong>Hydroxychloroquine in patients mainly with mild to moderate COVID-19: an open-label, randomized, controlled trial</strong><br />
Wei Tang, Zhujun Cao, Mingfeng Han, Zhengyan Wang, Junwen Chen, Wenjin Sun, Yaojie Wu, Wei Xiao, Shengyong Liu, Erzhen Chen, … Qing Xie<br />
<em>medRxiv</em> (2020-05-07) <a href="https://doi.org/ggr68m">https://doi.org/ggr68m</a><br />
DOI: <a href="https://doi.org/10.1101/2020.04.10.20060558">10.1101/2020.04.10.20060558</a></p>
</div>
<div id="ref-MENbRfOl">
<p>303. <strong>Outcomes of hydroxychloroquine usage in United States veterans hospitalized with Covid-19</strong><br />
Joseph Magagnoli, Siddharth Narendran, Felipe Pereira, Tammy Cummings, James W Hardin, S Scott Sutton, Jayakrishna Ambati<br />
<em>medRxiv</em> (2020-04-23) <a href="https://doi.org/ggspt6">https://doi.org/ggspt6</a><br />
DOI: <a href="https://doi.org/10.1101/2020.04.16.20065920">10.1101/2020.04.16.20065920</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32511622">32511622</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7276049">PMC7276049</a></p>
</div>
<div id="ref-NiVdIN7Z">
<p>304. <strong>A Randomized Trial of Hydroxychloroquine as Postexposure Prophylaxis for Covid-19</strong><br />
David R. Boulware, Matthew F. Pullen, Ananta S. Bangdiwala, Katelyn A. Pastick, Sarah M. Lofgren, Elizabeth C. Okafor, Caleb P. Skipper, Alanna A. Nascene, Melanie R. Nicol, Mahsa Abassi, … Kathy H. Hullsiek<br />
<em>New England Journal of Medicine</em> (2020-06-03) <a href="https://doi.org/dxkv">https://doi.org/dxkv</a><br />
DOI: <a href="https://doi.org/10.1056/nejmoa2016638">10.1056/nejmoa2016638</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32492293">32492293</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7289276">PMC7289276</a></p>
</div>
<div id="ref-WFGcbZhe">
<p>305. <strong>COVID-19: a recommendation to examine the effect of hydroxychloroquine in preventing infection and progression</strong><br />
Dan Zhou, Sheng-Ming Dai, Qiang Tong<br />
<em>Journal of Antimicrobial Chemotherapy</em> (2020-03-20) <a href="https://doi.org/ggq84c">https://doi.org/ggq84c</a><br />
DOI: <a href="https://doi.org/10.1093/jac/dkaa114">10.1093/jac/dkaa114</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32196083">32196083</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7184499">PMC7184499</a></p>
</div>
<div id="ref-hFGjQQN8">
<p>306. <strong>Hydroxychloroquine treatment of patients with human immunodeficiency virus type 1</strong><br />
Kirk Sperber, Michael Louie, Thomas Kraus, Jacqueline Proner, Erica Sapira, Su Lin, Vera Stecher, Lloyd Mayer<br />
<em>Clinical Therapeutics</em> (1995-07) <a href="https://doi.org/cq2hx9">https://doi.org/cq2hx9</a><br />
DOI: <a href="https://doi.org/10.1016/0149-2918(95)80039-5">10.1016/0149-2918(95)80039-5</a></p>
</div>
<div id="ref-rQNzKxRd">
<p>307. <strong>Hydroxychloroquine augments early virological response to pegylated interferon plus ribavirin in genotype-4 chronic hepatitis C patients</strong><br />
Gouda Kamel Helal, Magdy Abdelmawgoud Gad, Mohamed Fahmy Abd-Ellah, Mahmoud Saied Eid<br />
<em>Journal of Medical Virology</em> (2016-12) <a href="https://doi.org/f889nt">https://doi.org/f889nt</a><br />
DOI: <a href="https://doi.org/10.1002/jmv.24575">10.1002/jmv.24575</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27183377">27183377</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7167065">PMC7167065</a></p>
</div>
<div id="ref-j3rGUH1f">
<p>308. <strong>Mechanisms of action of hydroxychloroquine and chloroquine: implications for rheumatology</strong><br />
Eva Schrezenmeier, Thomas Dörner<br />
<em>Nature Reviews Rheumatology</em> (2020-02-07) <a href="https://doi.org/ggzjnh">https://doi.org/ggzjnh</a><br />
DOI: <a href="https://doi.org/10.1038/s41584-020-0372-x">10.1038/s41584-020-0372-x</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32034323">32034323</a></p>
</div>
<div id="ref-xLPeszeO">
<p>309. <strong>Structural Design Principles for Delivery of Bioactive Components in Nutraceuticals and Functional Foods</strong><br />
David Julian McClements, Eric Andrew Decker, Yeonhwa Park, Jochen Weiss<br />
<em>Critical Reviews in Food Science and Nutrition</em> (2009-06-16) <a href="https://doi.org/dt68m4">https://doi.org/dt68m4</a><br />
DOI: <a href="https://doi.org/10.1080/10408390902841529">10.1080/10408390902841529</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19484636">19484636</a></p>
</div>
<div id="ref-2Us0gNMr">
<p>310. <strong>Nutraceutical therapies for atherosclerosis</strong><br />
Joe W. E. Moss, Dipak P. Ramji<br />
<em>Nature Reviews Cardiology</em> (2016-07-07) <a href="https://doi.org/f9g389">https://doi.org/f9g389</a><br />
DOI: <a href="https://doi.org/10.1038/nrcardio.2016.103">10.1038/nrcardio.2016.103</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27383080">27383080</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5228762">PMC5228762</a></p>
</div>
<div id="ref-X0jEogv4">
<p>311. <strong>Nutraceutical-definition and introduction</strong><br />
Ekta K. Kalra<br />
<em>AAPS PharmSci</em> (2015-07-10) <a href="https://doi.org/cg5wc9">https://doi.org/cg5wc9</a><br />
DOI: <a href="https://doi.org/10.1208/ps050325">10.1208/ps050325</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/14621960">14621960</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2750935">PMC2750935</a></p>
</div>
<div id="ref-10Tnnl6nF">
<p>312. <strong>Dietary Supplement Health and Education Act of 1994</strong><br />
National Institutes of Health Office of Dietary Supplements<br />
<a href="https://ods.od.nih.gov/About/DSHEA_Wording.aspx">https://ods.od.nih.gov/About/DSHEA_Wording.aspx</a></p>
</div>
<div id="ref-16eEQf0HM">
<p>313. <strong>Food and Drug Administration Modernization Act (FDAMA) of 1997</strong><br />
Office of the Commissioner<br />
<em>FDA</em> (2018-11-03) <a href="https://www.fda.gov/regulatory-information/selected-amendments-fdc-act/food-and-drug-administration-modernization-act-fdama-1997">https://www.fda.gov/regulatory-information/selected-amendments-fdc-act/food-and-drug-administration-modernization-act-fdama-1997</a></p>
</div>
<div id="ref-5TmtJWVQ">
<p>314. <strong>Nutraceuticals - shedding light on the grey area between pharmaceuticals and food</strong><br />
Antonello Santini, Ettore Novellino<br />
<em>Expert Review of Clinical Pharmacology</em> (2018-04-23) <a href="https://doi.org/ggwztk">https://doi.org/ggwztk</a><br />
DOI: <a href="https://doi.org/10.1080/17512433.2018.1464911">10.1080/17512433.2018.1464911</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29667442">29667442</a></p>
</div>
<div id="ref-dzOveIQY">
<p>315. <strong>Nutrition and Health Claims - European Commission</strong><a href="https://ec.europa.eu/food/safety/labelling_nutrition/claims/register/public/">https://ec.europa.eu/food/safety/labelling_nutrition/claims/register/public/</a></p>
</div>
<div id="ref-MhvDTmeu">
<p>316. <strong>Nutraceuticals: opening the debate for a regulatory framework</strong><br />
Antonello Santini, Silvia Miriam Cammarata, Giacomo Capone, Angela Ianaro, Gian Carlo Tenore, Luca Pani, Ettore Novellino<br />
<em>British Journal of Clinical Pharmacology</em> (2018-04) <a href="https://doi.org/ggwztm">https://doi.org/ggwztm</a><br />
DOI: <a href="https://doi.org/10.1111/bcp.13496">10.1111/bcp.13496</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29433155">29433155</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5867125">PMC5867125</a></p>
</div>
<div id="ref-2hMFPQJR">
<p>317. <strong>Potential interventions for novel coronavirus in China: A systematic review</strong><br />
Lei Zhang, Yunhui Liu<br />
<em>Journal of Medical Virology</em> (2020-03-03) <a href="https://doi.org/ggpx57">https://doi.org/ggpx57</a><br />
DOI: <a href="https://doi.org/10.1002/jmv.25707">10.1002/jmv.25707</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32052466">32052466</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7166986">PMC7166986</a></p>
</div>
<div id="ref-Ufvsmc3N">
<p>318. <strong>Inflammation and cardiovascular disease: are marine phospholipids the answer?</strong><br />
Ronan Lordan, Shane Redfern, Alexandros Tsoupras, Ioannis Zabetakis<br />
<em>Food &amp; Function</em> (2020) <a href="https://doi.org/gg29hg">https://doi.org/gg29hg</a><br />
DOI: <a href="https://doi.org/10.1039/c9fo01742a">10.1039/c9fo01742a</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32270798">32270798</a></p>
</div>
<div id="ref-JNyhLAfe">
<p>319. <strong>Omega-3 polyunsaturated fatty acids and inflammatory processes: nutrition or pharmacology?</strong><br />
Philip C. Calder<br />
<em>British Journal of Clinical Pharmacology</em> (2013-03) <a href="https://doi.org/ggqmgg">https://doi.org/ggqmgg</a><br />
DOI: <a href="https://doi.org/10.1111/j.1365-2125.2012.04374.x">10.1111/j.1365-2125.2012.04374.x</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22765297">22765297</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3575932">PMC3575932</a></p>
</div>
<div id="ref-BIVORG2X">
<p>320. <strong>Proresolving Lipid Mediators and Mechanisms in the Resolution of Acute Inflammation</strong><br />
Christopher D. Buckley, Derek W. Gilroy, Charles N. Serhan<br />
<em>Immunity</em> (2014-03) <a href="https://doi.org/f5wntr">https://doi.org/f5wntr</a><br />
DOI: <a href="https://doi.org/10.1016/j.immuni.2014.02.009">10.1016/j.immuni.2014.02.009</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24656045">24656045</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4004957">PMC4004957</a></p>
</div>
<div id="ref-17jegDTvF">
<p>321. <strong>COVID-19 and its implications for thrombosis and anticoagulation</strong><br />
Jean M. Connors, Jerrold H. Levy<br />
<em>Blood</em> (2020-06-04) <a href="https://doi.org/ggv35b">https://doi.org/ggv35b</a><br />
DOI: <a href="https://doi.org/10.1182/blood.2020006000">10.1182/blood.2020006000</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32339221">32339221</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7273827">PMC7273827</a></p>
</div>
<div id="ref-SxPCbOB1">
<p>322. <strong>COVID-19 and Thrombotic or Thromboembolic Disease: Implications for Prevention, Antithrombotic Therapy, and Follow-Up</strong><br />
Behnood Bikdeli, Mahesh V. Madhavan, David Jimenez, Taylor Chuich, Isaac Dreyfus, Elissa Driggin, Caroline Der Nigoghossian, Walter Ageno, Mohammad Madjid, Yutao Guo, … Gregory Y. H. Lip<br />
<em>Journal of the American College of Cardiology</em> (2020-06) <a href="https://doi.org/ggsppk">https://doi.org/ggsppk</a><br />
DOI: <a href="https://doi.org/10.1016/j.jacc.2020.04.031">10.1016/j.jacc.2020.04.031</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32311448">32311448</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7164881">PMC7164881</a></p>
</div>
<div id="ref-uPOTvvjR">
<p>323. <strong>Specialized pro-resolving mediators: endogenous regulators of infection and inflammation</strong><br />
Maria C. Basil, Bruce D. Levy<br />
<em>Nature Reviews Immunology</em> (2015-12-21) <a href="https://doi.org/f9fgtd">https://doi.org/f9fgtd</a><br />
DOI: <a href="https://doi.org/10.1038/nri.2015.4">10.1038/nri.2015.4</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26688348">26688348</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5242505">PMC5242505</a></p>
</div>
<div id="ref-fIo3ESPM">
<p>324. <strong>The Lipid Mediator Protectin D1 Inhibits Influenza Virus Replication and Improves Severe Influenza</strong><br />
Masayuki Morita, Keiji Kuba, Akihiko Ichikawa, Mizuho Nakayama, Jun Katahira, Ryo Iwamoto, Tokiko Watanebe, Saori Sakabe, Tomo Daidoji, Shota Nakamura, … Yumiko Imai<br />
<em>Cell</em> (2013-03) <a href="https://doi.org/f4rbgb">https://doi.org/f4rbgb</a><br />
DOI: <a href="https://doi.org/10.1016/j.cell.2013.02.027">10.1016/j.cell.2013.02.027</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23477864">23477864</a></p>
</div>
<div id="ref-OSEnHt6H">
<p>325. <strong>Pro-resolving lipid mediators are leads for resolution physiology</strong><br />
Charles N. Serhan<br />
<em>Nature</em> (2014-06-04) <a href="https://doi.org/ggv53d">https://doi.org/ggv53d</a><br />
DOI: <a href="https://doi.org/10.1038/nature13479">10.1038/nature13479</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24899309">24899309</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4263681">PMC4263681</a></p>
</div>
<div id="ref-MK0GQ55y">
<p>326. <strong>The Specialized Proresolving Mediator 17-HDHA Enhances the Antibody-Mediated Immune Response against Influenza Virus: A New Class of Adjuvant?</strong><br />
Sesquile Ramon, Steven F. Baker, Julie M. Sahler, Nina Kim, Eric A. Feldsott, Charles N. Serhan, Luis Martínez-Sobrido, David J. Topham, Richard P. Phipps<br />
<em>The Journal of Immunology</em> (2014-12-15) <a href="https://doi.org/f6spr8">https://doi.org/f6spr8</a><br />
DOI: <a href="https://doi.org/10.4049/jimmunol.1302795">10.4049/jimmunol.1302795</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25392529">25392529</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4258475">PMC4258475</a></p>
</div>
<div id="ref-14LjKv9ZH">
<p>327. <strong>Inflammation resolution: a dual-pronged approach to averting cytokine storms in COVID-19?</strong><br />
Dipak Panigrahy, Molly M. Gilligan, Sui Huang, Allison Gartung, Irene Cortés-Puch, Patricia J. Sime, Richard P. Phipps, Charles N. Serhan, Bruce D. Hammock<br />
<em>Cancer and Metastasis Reviews</em> (2020-05-08) <a href="https://doi.org/ggvv7w">https://doi.org/ggvv7w</a><br />
DOI: <a href="https://doi.org/10.1007/s10555-020-09889-4">10.1007/s10555-020-09889-4</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32385712">32385712</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7207990">PMC7207990</a></p>
</div>
<div id="ref-B8toe6Mr">
<p>328. <strong>Fish Oil-Fed Mice Have Impaired Resistance to Influenza Infection</strong><br />
Nicole M. J. Schwerbrock, Erik A. Karlsson, Qing Shi, Patricia A. Sheridan, Melinda A. Beck<br />
<em>The Journal of Nutrition</em> (2009-08) <a href="https://doi.org/dv45f4">https://doi.org/dv45f4</a><br />
DOI: <a href="https://doi.org/10.3945/jn.109.108027">10.3945/jn.109.108027</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19549756">19549756</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2709305">PMC2709305</a></p>
</div>
<div id="ref-3Xht24k3">
<p>329. <strong>Modulation of host defence against bacterial and viral infections by omega-3 polyunsaturated fatty acids</strong><br />
Marie-Odile Husson, Delphine Ley, Céline Portal, Madeleine Gottrand, Thomas Hueso, Jean-Luc Desseyn, Frédéric Gottrand<br />
<em>Journal of Infection</em> (2016-12) <a href="https://doi.org/f9pp2h">https://doi.org/f9pp2h</a><br />
DOI: <a href="https://doi.org/10.1016/j.jinf.2016.10.001">10.1016/j.jinf.2016.10.001</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27746159">27746159</a></p>
</div>
<div id="ref-Ge00xeR3">
<p>330. <strong>Bioactive products formed in humans from fish oils</strong><br />
Carsten Skarke, Naji Alamuddin, John A. Lawson, Xuanwen Li, Jane F. Ferguson, Muredach P. Reilly, Garret A. FitzGerald<br />
<em>Journal of Lipid Research</em> (2015-09) <a href="https://doi.org/f7pm5g">https://doi.org/f7pm5g</a><br />
DOI: <a href="https://doi.org/10.1194/jlr.m060392">10.1194/jlr.m060392</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26180051">26180051</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4548785">PMC4548785</a></p>
</div>
<div id="ref-umzPtQG7">
<p>331. <strong>Regulation of platelet function and thrombosis by omega-3 and omega-6 polyunsaturated fatty acids</strong><br />
Reheman Adili, Megan Hawley, Michael Holinstat<br />
<em>Prostaglandins &amp; Other Lipid Mediators</em> (2018-11) <a href="https://doi.org/ggvv73">https://doi.org/ggvv73</a><br />
DOI: <a href="https://doi.org/10.1016/j.prostaglandins.2018.09.005">10.1016/j.prostaglandins.2018.09.005</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30266534">30266534</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6242736">PMC6242736</a></p>
</div>
<div id="ref-V9MDMfpj">
<p>332. <strong>Platelet activation and prothrombotic mediators at the nexus of inflammation and atherosclerosis: Potential role of antiplatelet agents</strong><br />
Ronan Lordan, Alexandros Tsoupras, Ioannis Zabetakis<br />
<em>Blood Reviews</em> (2020-04) <a href="https://doi.org/ggvv7x">https://doi.org/ggvv7x</a><br />
DOI: <a href="https://doi.org/10.1016/j.blre.2020.100694">10.1016/j.blre.2020.100694</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32340775">32340775</a></p>
</div>
<div id="ref-QKHdIx2w">
<p>333. <strong>An Investigation on the Effects of Icosapent Ethyl (VascepaTM) on Inflammatory Biomarkers in Individuals With COVID-19 (VASCEPA-COVID-19)</strong><br />
Canadian Medical and Surgical Knowledge Translation Research Group<br />
<em>clinicaltrials.gov</em> (2020-06-03) <a href="https://clinicaltrials.gov/ct2/show/NCT04412018">https://clinicaltrials.gov/ct2/show/NCT04412018</a></p>
</div>
<div id="ref-12Q4bY0vF">
<p>334. <strong>Cardiovascular Risk Reduction with Icosapent Ethyl for Hypertriglyceridemia</strong><br />
Deepak L. Bhatt, P. Gabriel Steg, Michael Miller, Eliot A. Brinton, Terry A. Jacobson, Steven B. Ketchum, Ralph T. Doyle, Rebecca A. Juliano, Lixia Jiao, Craig Granowitz, … Christie M. Ballantyne<br />
<em>New England Journal of Medicine</em> (2019-01-03) <a href="https://doi.org/gfj3w9">https://doi.org/gfj3w9</a><br />
DOI: <a href="https://doi.org/10.1056/nejmoa1812792">10.1056/nejmoa1812792</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30415628">30415628</a></p>
</div>
<div id="ref-8lPjdnE7">
<p>335. <strong>A Randomized Controlled Study of Eicosapentaenoic Acid (EPA-FFA) Gastro-resistant Capsules to Treat Hospitalized Subjects With Confirmed SARS-CoV-2</strong><br />
S. L. A. Pharma AG<br />
<em>clinicaltrials.gov</em> (2020-04-23) <a href="https://clinicaltrials.gov/ct2/show/NCT04335032">https://clinicaltrials.gov/ct2/show/NCT04335032</a></p>
</div>
<div id="ref-1AGVAbQuf">
<p>336. <strong>Anti-inflammatory/Antioxidant Oral Nutrition Supplementation on the Cytokine Storm and Progression of COVID-19: A Randomized Controlled Trial</strong><br />
Mahmoud Abulmeaty FACN M. D.<br />
<em>clinicaltrials.gov</em> (2020-05-28) <a href="https://clinicaltrials.gov/ct2/show/NCT04323228">https://clinicaltrials.gov/ct2/show/NCT04323228</a></p>
</div>
<div id="ref-XHtzuR5c">
<p>337. <strong>Functional Role of Dietary Intervention to Improve the Outcome of COVID-19: A Hypothesis of Work</strong><br />
Giovanni Messina, Rita Polito, Vincenzo Monda, Luigi Cipolloni, Nunzio Di Nunno, Giulio Di Mizio, Paolo Murabito, Marco Carotenuto, Antonietta Messina, Daniela Pisanelli, … Francesco Sessa<br />
<em>International Journal of Molecular Sciences</em> (2020-04-28) <a href="https://doi.org/ggvb88">https://doi.org/ggvb88</a><br />
DOI: <a href="https://doi.org/10.3390/ijms21093104">10.3390/ijms21093104</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32354030">32354030</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7247152">PMC7247152</a></p>
</div>
<div id="ref-VMI1lOFR">
<p>338. <strong>Zinc and immunity: An essential interrelation</strong><br />
Maria Maares, Hajo Haase<br />
<em>Archives of Biochemistry and Biophysics</em> (2016-12) <a href="https://doi.org/f9c9b5">https://doi.org/f9c9b5</a><br />
DOI: <a href="https://doi.org/10.1016/j.abb.2016.03.022">10.1016/j.abb.2016.03.022</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27021581">27021581</a></p>
</div>
<div id="ref-Qif5U0Ig">
<p>339. <strong>Zinc-Dependent Suppression of TNF-α Production Is Mediated by Protein Kinase A-Induced Inhibition of Raf-1, IκB Kinase β, and NF-κB</strong><br />
Verena von Bülow, Svenja Dubben, Gabriela Engelhardt, Silke Hebel, Birgit Plümäkers, Holger Heine, Lothar Rink, Hajo Haase<br />
<em>The Journal of Immunology</em> (2007-09-15) <a href="https://doi.org/f3vs45">https://doi.org/f3vs45</a><br />
DOI: <a href="https://doi.org/10.4049/jimmunol.179.6.4180">10.4049/jimmunol.179.6.4180</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17785857">17785857</a></p>
</div>
<div id="ref-1CEzU1G34">
<p>340. <strong>Zinc activates NF-κB in HUT-78 cells</strong><br />
Ananda S. Prasad, Bin Bao, Frances W. J. Beck, Fazlul H. Sarkar<br />
<em>Journal of Laboratory and Clinical Medicine</em> (2001-10) <a href="https://doi.org/cnc6fr">https://doi.org/cnc6fr</a><br />
DOI: <a href="https://doi.org/10.1067/mlc.2001.118108">10.1067/mlc.2001.118108</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/11574819">11574819</a></p>
</div>
<div id="ref-7ElRN12v">
<p>341. <strong>Innate or Adaptive Immunity? The Example of Natural Killer Cells</strong><br />
E. Vivier, D. H. Raulet, A. Moretta, M. A. Caligiuri, L. Zitvogel, L. L. Lanier, W. M. Yokoyama, S. Ugolini<br />
<em>Science</em> (2011-01-06) <a href="https://doi.org/ckzg9g">https://doi.org/ckzg9g</a><br />
DOI: <a href="https://doi.org/10.1126/science.1198687">10.1126/science.1198687</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21212348">21212348</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3089969">PMC3089969</a></p>
</div>
<div id="ref-kxuSwrCI">
<p>342. <strong>Zinc supplementation decreases incidence of infections in the elderly: effect of zinc on generation of cytokines and oxidative stress</strong><br />
Ananda S Prasad, Frances WJ Beck, Bin Bao, James T Fitzgerald, Diane C Snell, Joel D Steinberg, Lavoisier J Cardozo<br />
<em>The American Journal of Clinical Nutrition</em> (2007-03) <a href="https://doi.org/ggqmgs">https://doi.org/ggqmgs</a><br />
DOI: <a href="https://doi.org/10.1093/ajcn/85.3.837">10.1093/ajcn/85.3.837</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17344507">17344507</a></p>
</div>
<div id="ref-IAOnO0Gj">
<p>343. <strong>The Role of Zinc in Antiviral Immunity</strong><br />
Scott A Read, Stephanie Obeid, Chantelle Ahlenstiel, Golo Ahlenstiel<br />
<em>Advances in Nutrition</em> (2019-07) <a href="https://doi.org/ggqmgr">https://doi.org/ggqmgr</a><br />
DOI: <a href="https://doi.org/10.1093/advances/nmz013">10.1093/advances/nmz013</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31305906">31305906</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6628855">PMC6628855</a></p>
</div>
<div id="ref-sgr0mlJK">
<p>344. <strong>Efficacy of Zinc Against Common Cold Viruses: An Overview</strong><br />
Darrell Hulisz<br />
<em>Journal of the American Pharmacists Association</em> (2004-09) <a href="https://doi.org/cf6pmt">https://doi.org/cf6pmt</a><br />
DOI: <a href="https://doi.org/10.1331/1544-3191.44.5.594.hulisz">10.1331/1544-3191.44.5.594.hulisz</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15496046">15496046</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7185598">PMC7185598</a></p>
</div>
<div id="ref-lPexOhy6">
<p>345. <strong>Zinc Lozenges May Shorten the Duration of Colds: A Systematic Review</strong><br />
Harri Hemilä<br />
<em>The Open Respiratory Medicine Journal</em> (2011-06-23) <a href="https://doi.org/bndmfq">https://doi.org/bndmfq</a><br />
DOI: <a href="https://doi.org/10.2174/1874306401105010051">10.2174/1874306401105010051</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21769305">21769305</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3136969">PMC3136969</a></p>
</div>
<div id="ref-6aOUJdHa">
<p>346. <strong>Zn2+ Inhibits Coronavirus and Arterivirus RNA Polymerase Activity In Vitro and Zinc Ionophores Block the Replication of These Viruses in Cell Culture</strong><br />
Aartjan J. W. te Velthuis, Sjoerd H. E. van den Worm, Amy C. Sims, Ralph S. Baric, Eric J. Snijder, Martijn J. van Hemert<br />
<em>PLoS Pathogens</em> (2010-11-04) <a href="https://doi.org/d95x4g">https://doi.org/d95x4g</a><br />
DOI: <a href="https://doi.org/10.1371/journal.ppat.1001176">10.1371/journal.ppat.1001176</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21079686">21079686</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2973827">PMC2973827</a></p>
</div>
<div id="ref-NB1udFHF">
<p>347. <strong>The SARS-coronavirus papain-like protease: Structure, function and inhibition by designed antiviral compounds</strong><br />
Yahira M. Báez-Santos, Sarah E. St. John, Andrew D. Mesecar<br />
<em>Antiviral Research</em> (2015-03) <a href="https://doi.org/f63hjp">https://doi.org/f63hjp</a><br />
DOI: <a href="https://doi.org/10.1016/j.antiviral.2014.12.015">10.1016/j.antiviral.2014.12.015</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25554382">25554382</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5896749">PMC5896749</a></p>
</div>
<div id="ref-XJrj3MiZ">
<p>348. <strong>A Randomized Study Evaluating the Safety and Efficacy of Hydroxychloroquine and Zinc in Combination With Either Azithromycin or Doxycycline for the Treatment of COVID-19 in the Outpatient Setting</strong><br />
Avni Thakore MD<br />
<em>clinicaltrials.gov</em> (2020-05-14) <a href="https://clinicaltrials.gov/ct2/show/NCT04370782">https://clinicaltrials.gov/ct2/show/NCT04370782</a></p>
</div>
<div id="ref-1EqFW3G4X">
<p>349. <strong>A Study of Hydroxychloroquine and Zinc in the Prevention of COVID-19 Infection in Military Healthcare Workers (COVID-Milit)</strong><br />
Military Hospital of Tunis<br />
<em>clinicaltrials.gov</em> (2020-05-04) <a href="https://clinicaltrials.gov/ct2/show/NCT04377646">https://clinicaltrials.gov/ct2/show/NCT04377646</a></p>
</div>
<div id="ref-Hc9mjVB2">
<p>350. <strong>Therapies to Prevent Progression of COVID-19, Including Hydroxychloroquine, Azithromycin, Zinc, Vitamin D, Vitamin B12 With or Without Vitamin C, a Multi-centre, International, Randomized Trial: The International ALLIANCE Study</strong><br />
National Institute of Integrative Medicine, Australia<br />
<em>clinicaltrials.gov</em> (2020-05-19) <a href="https://clinicaltrials.gov/ct2/show/NCT04395768">https://clinicaltrials.gov/ct2/show/NCT04395768</a></p>
</div>
<div id="ref-T0aXNTIp">
<p>351. <strong>A Randomised Controlled Trial of Early Intervention in Patients HospItalised With COVID-19: Favipiravir and StaNdard Care vErsEs Standard CaRe</strong><br />
Chelsea and Westminster NHS Foundation Trust<br />
<em>clinicaltrials.gov</em> (2020-07-02) <a href="https://clinicaltrials.gov/ct2/show/NCT04373733">https://clinicaltrials.gov/ct2/show/NCT04373733</a></p>
</div>
<div id="ref-1GGbb9IZt">
<p>352. <strong>Coronavirus Disease 2019- Using Ascorbic Acid and Zinc Supplementation (COVIDAtoZ) Research Study A Randomized, Open Label Single Center Study</strong><br />
Milind Desai<br />
<em>clinicaltrials.gov</em> (2020-05-12) <a href="https://clinicaltrials.gov/ct2/show/NCT04342728">https://clinicaltrials.gov/ct2/show/NCT04342728</a></p>
</div>
<div id="ref-3RCeQRRc">
<p>353. <strong>Vitamin B12 May Inhibit RNA-Dependent-RNA Polymerase Activity of nsp12 from the COVID-19 Virus</strong><br />
Naveen Narayanan, Deepak T. Nair<br />
<em>Preprints</em> (2020-03-22) <a href="https://doi.org/ggqmjc">https://doi.org/ggqmjc</a><br />
DOI: <a href="https://doi.org/10.20944/preprints202003.0347.v1">10.20944/preprints202003.0347.v1</a></p>
</div>
<div id="ref-9czQt5cR">
<p>354. <strong>Vitamin C Mitigates Oxidative Stress and Tumor Necrosis Factor-Alpha in Severe Community-Acquired Pneumonia and LPS-Induced Macrophages</strong><br />
Yuanyuan Chen, Guangyan Luo, Jiao Yuan, Yuanyuan Wang, Xiaoqiong Yang, Xiaoyun Wang, Guoping Li, Zhiguang Liu, Nanshan Zhong<br />
<em>Mediators of Inflammation</em> (2014) <a href="https://doi.org/f6nb5f">https://doi.org/f6nb5f</a><br />
DOI: <a href="https://doi.org/10.1155/2014/426740">10.1155/2014/426740</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25253919">25253919</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4165740">PMC4165740</a></p>
</div>
<div id="ref-RjFxOURM">
<p>355. <strong>Intravenous infusion of ascorbic acid decreases serum histamine concentrations in patients with allergic and non-allergic diseases</strong><br />
Alexander F. Hagel, Christian M. Layritz, Wolfgang H. Hagel, Hans-Jürgen Hagel, Edith Hagel, Wolfgang Dauth, Jürgen Kressel, Tanja Regnet, Andreas Rosenberg, Markus F. Neurath, … Martin Raithel<br />
<em>Naunyn-Schmiedeberg’s Archives of Pharmacology</em> (2013-05-11) <a href="https://doi.org/f48jsb">https://doi.org/f48jsb</a><br />
DOI: <a href="https://doi.org/10.1007/s00210-013-0880-1">10.1007/s00210-013-0880-1</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23666445">23666445</a></p>
</div>
<div id="ref-13M3VAfY9">
<p>356. <strong>Vitamin C and Immune Function</strong><br />
Anitra Carr, Silvia Maggini<br />
<em>Nutrients</em> (2017-11-03) <a href="https://doi.org/gfzrjs">https://doi.org/gfzrjs</a><br />
DOI: <a href="https://doi.org/10.3390/nu9111211">10.3390/nu9111211</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29099763">29099763</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5707683">PMC5707683</a></p>
</div>
<div id="ref-uQFCDDIy">
<p>357. <strong>Changes in Leucocyte Ascorbic Acid during the Common Cold</strong><br />
R. Hume, Elspeth Weyers<br />
<em>Scottish Medical Journal</em> (2016-06-25) <a href="https://doi.org/ggqrfj">https://doi.org/ggqrfj</a><br />
DOI: <a href="https://doi.org/10.1177/003693307301800102">10.1177/003693307301800102</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/4717661">4717661</a></p>
</div>
<div id="ref-10nBT1HbX">
<p>358. <strong>ASCORBIC ACID FUNCTION AND METABOLISM DURING COLDS</strong><br />
C. W. M. Wilson<br />
<em>Annals of the New York Academy of Sciences</em> (1975-09) <a href="https://doi.org/bjfdtb">https://doi.org/bjfdtb</a><br />
DOI: <a href="https://doi.org/10.1111/j.1749-6632.1975.tb29312.x">10.1111/j.1749-6632.1975.tb29312.x</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/1106304">1106304</a></p>
</div>
<div id="ref-1Bj8VMlSk">
<p>359. <strong>Metabolism of ascorbic acid (vitamin C) in subjects infected with common cold viruses</strong><br />
J. E. W. Davies, R. E. Hughes, Eleri Jones, Sylvia E. Reed, J. W. Craig, D. A. J. Tyrrell<br />
<em>Biochemical Medicine</em> (1979-02) <a href="https://doi.org/fd22sv">https://doi.org/fd22sv</a><br />
DOI: <a href="https://doi.org/10.1016/0006-2944(79)90058-9">10.1016/0006-2944(79)90058-9</a></p>
</div>
<div id="ref-aEEZ8Pq9">
<p>360. <strong>Vitamin C and Infections</strong><br />
Harri Hemilä<br />
<em>Nutrients</em> (2017-03-29) <a href="https://doi.org/gfkb9n">https://doi.org/gfkb9n</a><br />
DOI: <a href="https://doi.org/10.3390/nu9040339">10.3390/nu9040339</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28353648">28353648</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5409678">PMC5409678</a></p>
</div>
<div id="ref-gZRPoTM9">
<p>361. <strong>Vitamin C and the common cold</strong><br />
Harri Hemilä<br />
<em>British Journal of Nutrition</em> (2007-03-09) <a href="https://doi.org/fszhc6">https://doi.org/fszhc6</a><br />
DOI: <a href="https://doi.org/10.1079/bjn19920004">10.1079/bjn19920004</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/1547201">1547201</a></p>
</div>
<div id="ref-Pb5PCH7y">
<p>362. <strong>Vitamin C for preventing and treating the common cold</strong><br />
Harri Hemilä, Elizabeth Chalker<br />
<em>Cochrane Database of Systematic Reviews</em> (2013-01-31) <a href="https://doi.org/xz5">https://doi.org/xz5</a><br />
DOI: <a href="https://doi.org/10.1002/14651858.cd000980.pub4">10.1002/14651858.cd000980.pub4</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23440782">23440782</a></p>
</div>
<div id="ref-JARvpqRw">
<p>363. <strong>Vitamin C intake and susceptibility to pneumonia</strong><br />
HARRI HEMILÄ<br />
<em>The Pediatric Infectious Disease Journal</em> (1997-09) <a href="https://doi.org/fkvs9d">https://doi.org/fkvs9d</a><br />
DOI: <a href="https://doi.org/10.1097/00006454-199709000-00003">10.1097/00006454-199709000-00003</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/9306475">9306475</a></p>
</div>
<div id="ref-DPhRTQw0">
<p>364. <strong>Vitamin C Can Shorten the Length of Stay in the ICU: A Meta-Analysis</strong><br />
Harri Hemilä, Elizabeth Chalker<br />
<em>Nutrients</em> (2019-03-27) <a href="https://doi.org/gfzscg">https://doi.org/gfzscg</a><br />
DOI: <a href="https://doi.org/10.3390/nu11040708">10.3390/nu11040708</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30934660">30934660</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6521194">PMC6521194</a></p>
</div>
<div id="ref-Rt5Aik4p">
<p>365. <strong>Effect of Vitamin C Infusion on Organ Failure and Biomarkers of Inflammation and Vascular Injury in Patients With Sepsis and Severe Acute Respiratory Failure</strong><br />
Alpha A. Fowler, Jonathon D. Truwit, R. Duncan Hite, Peter E. Morris, Christine DeWilde, Anna Priday, Bernard Fisher, Leroy R. Thacker, Ramesh Natarajan, Donald F. Brophy, … Matthew Halquist<br />
<em>JAMA</em> (2019-10-01) <a href="https://doi.org/ggqmh8">https://doi.org/ggqmh8</a><br />
DOI: <a href="https://doi.org/10.1001/jama.2019.11825">10.1001/jama.2019.11825</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31573637">31573637</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6777268">PMC6777268</a></p>
</div>
<div id="ref-wBwLIQkU">
<p>366. <strong>Vitamin C Infusion for the Treatment of Severe 2019-nCoV Infected Pneumonia: a Prospective Randomized Clinical Trial</strong><br />
ZhiYong Peng<br />
<em>clinicaltrials.gov</em> (2020-03-06) <a href="https://clinicaltrials.gov/ct2/show/NCT04264533">https://clinicaltrials.gov/ct2/show/NCT04264533</a></p>
</div>
<div id="ref-rSyBZFhE">
<p>367. <strong>A new clinical trial to test high-dose vitamin C in patients with COVID-19</strong><br />
Anitra C. Carr<br />
<em>Critical Care</em> (2020-04-07) <a href="https://doi.org/ggsbmg">https://doi.org/ggsbmg</a><br />
DOI: <a href="https://doi.org/10.1186/s13054-020-02851-4">10.1186/s13054-020-02851-4</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32264963">32264963</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7137406">PMC7137406</a></p>
</div>
<div id="ref-16435N02u">
<p>368. <strong>Use of Ascorbic Acid in Patients With COVID 19</strong><br />
Salvatore Corrao MD<br />
<em>clinicaltrials.gov</em> (2020-03-24) <a href="https://clinicaltrials.gov/ct2/show/NCT04323514">https://clinicaltrials.gov/ct2/show/NCT04323514</a></p>
</div>
<div id="ref-n3v9Thlz">
<p>369. <strong>Vitamin D and the Immune System</strong><br />
Cynthia Aranow<br />
<em>Journal of Investigative Medicine</em> (2015-12-15) <a href="https://doi.org/f3wh87">https://doi.org/f3wh87</a><br />
DOI: <a href="https://doi.org/10.2310/jim.0b013e31821b8755">10.2310/jim.0b013e31821b8755</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21527855">21527855</a></p>
</div>
<div id="ref-iWLsflrG">
<p>370. <strong>Vitamin D in the prevention of acute respiratory infection: Systematic review of clinical studies</strong><br />
David A. Jolliffe, Christopher J. Griffiths, Adrian R. Martineau<br />
<em>The Journal of Steroid Biochemistry and Molecular Biology</em> (2013-07) <a href="https://doi.org/ggqmh9">https://doi.org/ggqmh9</a><br />
DOI: <a href="https://doi.org/10.1016/j.jsbmb.2012.11.017">10.1016/j.jsbmb.2012.11.017</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23220552">23220552</a></p>
</div>
<div id="ref-nwEQBjV4">
<p>371. <strong>Vitamin D: modulator of the immune system</strong><br />
Femke Baeke, Tatiana Takiishi, Hannelie Korf, Conny Gysemans, Chantal Mathieu<br />
<em>Current Opinion in Pharmacology</em> (2010-08) <a href="https://doi.org/d43qtf">https://doi.org/d43qtf</a><br />
DOI: <a href="https://doi.org/10.1016/j.coph.2010.04.001">10.1016/j.coph.2010.04.001</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20427238">20427238</a></p>
</div>
<div id="ref-9ac0GYir">
<p>372. <strong>Evidence that Vitamin D Supplementation Could Reduce Risk of Influenza and COVID-19 Infections and Deaths</strong><br />
William B. Grant, Henry Lahore, Sharon L. McDonnell, Carole A. Baggerly, Christine B. French, Jennifer L. Aliano, Harjit P. Bhattoa<br />
<em>Nutrients</em> (2020-04-02) <a href="https://doi.org/ggr2v5">https://doi.org/ggr2v5</a><br />
DOI: <a href="https://doi.org/10.3390/nu12040988">10.3390/nu12040988</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32252338">32252338</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7231123">PMC7231123</a></p>
</div>
<div id="ref-1Gy7Svl72">
<p>373. <strong>25-Hydroxyvitamin D Concentrations Are Lower in Patients with Positive PCR for SARS-CoV-2</strong><br />
Antonio D’Avolio, Valeria Avataneo, Alessandra Manca, Jessica Cusato, Amedeo De Nicolò, Renzo Lucchini, Franco Keller, Marco Cantù<br />
<em>Nutrients</em> (2020-05-09) <a href="https://doi.org/ggvv76">https://doi.org/ggvv76</a><br />
DOI: <a href="https://doi.org/10.3390/nu12051359">10.3390/nu12051359</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32397511">32397511</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7285131">PMC7285131</a></p>
</div>
<div id="ref-GMAEaWD1">
<p>374. <strong>Vitamin D deficiency as risk factor for severe COVID-19: a convergence of two pandemics</strong><br />
Dieter De Smet, Kristof De Smet, Pauline Herroelen, Stefaan Gryspeerdt, Geert Antoine Martens<br />
<em>medRxiv</em> (2020-05-05) <a href="https://doi.org/ggvv75">https://doi.org/ggvv75</a><br />
DOI: <a href="https://doi.org/10.1101/2020.05.01.20079376">10.1101/2020.05.01.20079376</a></p>
</div>
<div id="ref-FZ8yXmxQ">
<p>375. <strong>Vitamin D concentrations and COVID-19 infection in UK Biobank</strong><br />
Claire E. Hastie, Daniel F. Mackay, Frederick Ho, Carlos A. Celis-Morales, Srinivasa Vittal Katikireddi, Claire L. Niedzwiedz, Bhautesh D. Jani, Paul Welsh, Frances S. Mair, Stuart R. Gray, … Jill P. Pell<br />
<em>Diabetes &amp; Metabolic Syndrome: Clinical Research &amp; Reviews</em> (2020-07) <a href="https://doi.org/ggvv72">https://doi.org/ggvv72</a><br />
DOI: <a href="https://doi.org/10.1016/j.dsx.2020.04.050">10.1016/j.dsx.2020.04.050</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32413819">32413819</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7204679">PMC7204679</a></p>
</div>
<div id="ref-162t7zdGv">
<p>376. <strong>Effect of Vitamin D Administration on Prevention and Treatment of Mild Forms of Suspected Covid-19</strong><br />
Manuel Castillo Garzón<br />
<em>clinicaltrials.gov</em> (2020-04-03) <a href="https://clinicaltrials.gov/ct2/show/NCT04334005">https://clinicaltrials.gov/ct2/show/NCT04334005</a></p>
</div>
<div id="ref-2K0jV5Bn">
<p>377. <strong>Improving Vitamin D Status in the Management of COVID-19</strong><br />
Aldo Montano-Loza<br />
<em>clinicaltrials.gov</em> (2020-06-03) <a href="https://clinicaltrials.gov/ct2/show/NCT04385940">https://clinicaltrials.gov/ct2/show/NCT04385940</a></p>
</div>
<div id="ref-rBiojRHk">
<p>378. <strong>Randomized Controlled Trial of High Dose of Vitamin D as Compared With Placebo to Prevent Complications Among COVID-19 Patients</strong><br />
Vitamin D Study Group<br />
<em>clinicaltrials.gov</em> (2020-06-03) <a href="https://clinicaltrials.gov/ct2/show/NCT04411446">https://clinicaltrials.gov/ct2/show/NCT04411446</a></p>
</div>
<div id="ref-6kLoX5qT">
<p>379. <strong>COvid-19 and Vitamin D Supplementation: a Multicenter Randomized Controlled Trial of High Dose Versus Standard Dose Vitamin D3 in High-risk COVID-19 Patients (CoVitTrial)</strong><br />
University Hospital, Angers<br />
<em>clinicaltrials.gov</em> (2020-07-22) <a href="https://clinicaltrials.gov/ct2/show/NCT04344041">https://clinicaltrials.gov/ct2/show/NCT04344041</a></p>
</div>
<div id="ref-32C2erIe">
<p>380. <strong>The LEAD COVID-19 Trial: Low-risk, Early Aspirin and Vitamin D to Reduce COVID-19 Hospitalizations</strong><br />
Louisiana State University Health Sciences Center in New Orleans<br />
<em>clinicaltrials.gov</em> (2020-04-24) <a href="https://clinicaltrials.gov/ct2/show/NCT04363840">https://clinicaltrials.gov/ct2/show/NCT04363840</a></p>
</div>
<div id="ref-tLVliODQ">
<p>381. <strong>Randomized Double-Blind Placebo-Controlled Proof-of-Concept Trial of Resveratrol for the Outpatient Treatment of Mild Coronavirus Disease (COVID-19)</strong><br />
Marvin McCreary MD<br />
<em>clinicaltrials.gov</em> (2020-05-22) <a href="https://clinicaltrials.gov/ct2/show/NCT04400890">https://clinicaltrials.gov/ct2/show/NCT04400890</a></p>
</div>
<div id="ref-1FyHVZ9yx">
<p>382. <strong>The International Scientific Association for Probiotics and Prebiotics consensus statement on the scope and appropriate use of the term probiotic</strong><br />
Colin Hill, Francisco Guarner, Gregor Reid, Glenn R. Gibson, Daniel J. Merenstein, Bruno Pot, Lorenzo Morelli, Roberto Berni Canani, Harry J. Flint, Seppo Salminen, … Mary Ellen Sanders<br />
<em>Nature Reviews Gastroenterology &amp; Hepatology</em> (2014-06-10) <a href="https://doi.org/f6ndv7">https://doi.org/f6ndv7</a><br />
DOI: <a href="https://doi.org/10.1038/nrgastro.2014.66">10.1038/nrgastro.2014.66</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24912386">24912386</a></p>
</div>
<div id="ref-pvZU55k9">
<p>383. <strong>The Effect of Probiotics on Prevention of Common Cold: A Meta-Analysis of Randomized Controlled Trial Studies</strong><br />
En-Jin Kang, Soo Young Kim, In-Hong Hwang, Yun-Jeong Ji<br />
<em>Korean Journal of Family Medicine</em> (2013) <a href="https://doi.org/gg3knf">https://doi.org/gg3knf</a><br />
DOI: <a href="https://doi.org/10.4082/kjfm.2013.34.1.2">10.4082/kjfm.2013.34.1.2</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23372900">23372900</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3560336">PMC3560336</a></p>
</div>
<div id="ref-HKeroTXj">
<p>384. <strong>Probiotics and Paraprobiotics in Viral Infection: Clinical Application and Effects on the Innate and Acquired Immune Systems</strong><br />
Osamu Kanauchi, Akira Andoh, Sazaly AbuBakar, Naoki Yamamoto<br />
<em>Current Pharmaceutical Design</em> (2018-05-10) <a href="https://doi.org/gdjnpk">https://doi.org/gdjnpk</a><br />
DOI: <a href="https://doi.org/10.2174/1381612824666180116163411">10.2174/1381612824666180116163411</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29345577">29345577</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6006794">PMC6006794</a></p>
</div>
<div id="ref-10e8cHDs8">
<p>385. <strong>Using Probiotics to Flatten the Curve of Coronavirus Disease COVID-2019 Pandemic</strong><br />
David Baud, Varvara Dimopoulou Agri, Glenn R. Gibson, Gregor Reid, Eric Giannoni<br />
<em>Frontiers in Public Health</em> (2020-05-08) <a href="https://doi.org/gg3knd">https://doi.org/gg3knd</a><br />
DOI: <a href="https://doi.org/10.3389/fpubh.2020.00186">10.3389/fpubh.2020.00186</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32574290">32574290</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7227397">PMC7227397</a></p>
</div>
<div id="ref-5fU7dexB">
<p>386. <strong>Next-generation probiotics: the spectrum from probiotics to live biotherapeutics</strong><br />
Paul W. O’Toole, Julian R. Marchesi, Colin Hill<br />
<em>Nature Microbiology</em> (2017-04-25) <a href="https://doi.org/ggzggv">https://doi.org/ggzggv</a><br />
DOI: <a href="https://doi.org/10.1038/nmicrobiol.2017.57">10.1038/nmicrobiol.2017.57</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28440276">28440276</a></p>
</div>
<div id="ref-1GCrGCp5P">
<p>387. <strong>Mechanisms of Action of Probiotics</strong><br />
Julio Plaza-Diaz, Francisco Javier Ruiz-Ojeda, Mercedes Gil-Campos, Angel Gil<br />
<em>Advances in Nutrition</em> (2019-01) <a href="https://doi.org/gft8sh">https://doi.org/gft8sh</a><br />
DOI: <a href="https://doi.org/10.1093/advances/nmy063">10.1093/advances/nmy063</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30721959">30721959</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6363529">PMC6363529</a></p>
</div>
<div id="ref-B2GeSSQo">
<p>388. <strong>Probiotic mechanisms of action</strong><br />
Katrina Halloran, Mark A. Underwood<br />
<em>Early Human Development</em> (2019-08) <a href="https://doi.org/gg3jc4">https://doi.org/gg3jc4</a><br />
DOI: <a href="https://doi.org/10.1016/j.earlhumdev.2019.05.010">10.1016/j.earlhumdev.2019.05.010</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31174927">31174927</a></p>
</div>
<div id="ref-nmXF8vkS">
<p>389. <strong>Probiotic Mechanisms of Action</strong><br />
Miriam Bermudez-Brito, Julio Plaza-Díaz, Sergio Muñoz-Quezada, Carolina Gómez-Llorente, Angel Gil<br />
<em>Annals of Nutrition and Metabolism</em> (2012) <a href="https://doi.org/gg3knb">https://doi.org/gg3knb</a><br />
DOI: <a href="https://doi.org/10.1159/000342079">10.1159/000342079</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23037511">23037511</a></p>
</div>
<div id="ref-13AWoYvqB">
<p>390. <strong>Pulmonary-intestinal cross-talk in mucosal inflammatory disease</strong><br />
S Keely, NJ Talley, PM Hansbro<br />
<em>Mucosal Immunology</em> (2011-11-16) <a href="https://doi.org/b5knk2">https://doi.org/b5knk2</a><br />
DOI: <a href="https://doi.org/10.1038/mi.2011.55">10.1038/mi.2011.55</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22089028">22089028</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3243663">PMC3243663</a></p>
</div>
<div id="ref-Smqu1EYN">
<p>391. <strong>The role of the lung microbiota and the gut-lung axis in respiratory infectious diseases</strong><br />
Alexia Dumas, Lucie Bernard, Yannick Poquet, Geanncarlo Lugo-Villarino, Olivier Neyrolles<br />
<em>Cellular Microbiology</em> (2018-12) <a href="https://doi.org/gfjds9">https://doi.org/gfjds9</a><br />
DOI: <a href="https://doi.org/10.1111/cmi.12966">10.1111/cmi.12966</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30329198">30329198</a></p>
</div>
<div id="ref-d27mQ4kO">
<p>392. <strong>Gut microbiota and Covid-19- possible link and implications</strong><br />
Debojyoti Dhar, Abhishek Mohanty<br />
<em>Virus Research</em> (2020-08) <a href="https://doi.org/gg3jc5">https://doi.org/gg3jc5</a><br />
DOI: <a href="https://doi.org/10.1016/j.virusres.2020.198018">10.1016/j.virusres.2020.198018</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32430279">32430279</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7217790">PMC7217790</a></p>
</div>
<div id="ref-1MIGrWIP">
<p>393. <strong>Probiotics in respiratory virus infections</strong><br />
L. Lehtoranta, A. Pitkäranta, R. Korpela<br />
<em>European Journal of Clinical Microbiology &amp; Infectious Diseases</em> (2014-03-18) <a href="https://doi.org/f583jr">https://doi.org/f583jr</a><br />
DOI: <a href="https://doi.org/10.1007/s10096-014-2086-y">10.1007/s10096-014-2086-y</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24638909">24638909</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7088122">PMC7088122</a></p>
</div>
<div id="ref-58eXvYEV">
<p>394. <strong>Probiotics for preventing acute upper respiratory tract infections</strong><br />
Qiukui Hao, Bi Rong Dong, Taixiang Wu<br />
<em>Cochrane Database of Systematic Reviews</em> (2015-02-03) <a href="https://doi.org/gg3jc3">https://doi.org/gg3jc3</a><br />
DOI: <a href="https://doi.org/10.1002/14651858.cd006895.pub3">10.1002/14651858.cd006895.pub3</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25927096">25927096</a></p>
</div>
<div id="ref-kjavvCzp">
<p>395. <strong>Probiotics for the prevention of respiratory tract infections: a systematic review</strong><br />
Evridiki K. Vouloumanou, Gregory C. Makris, Drosos E. Karageorgopoulos, Matthew E. Falagas<br />
<em>International Journal of Antimicrobial Agents</em> (2009-09) <a href="https://doi.org/dn8kw8">https://doi.org/dn8kw8</a><br />
DOI: <a href="https://doi.org/10.1016/j.ijantimicag.2008.11.005">10.1016/j.ijantimicag.2008.11.005</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19179052">19179052</a></p>
</div>
<div id="ref-dJIGDt4k">
<p>396. <strong>Effectiveness of probiotics on the duration of illness in healthy children and adults who develop common acute respiratory infectious conditions: a systematic review and meta-analysis</strong><br />
Sarah King, Julie Glanville, Mary Ellen Sanders, Anita Fitzgerald, Danielle Varley<br />
<em>British Journal of Nutrition</em> (2014-04-29) <a href="https://doi.org/f57hq5">https://doi.org/f57hq5</a><br />
DOI: <a href="https://doi.org/10.1017/s0007114514000075">10.1017/s0007114514000075</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24780623">24780623</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4054664">PMC4054664</a></p>
</div>
<div id="ref-AJav6c61">
<p>397. <strong>Effect of probiotics on the incidence of ventilator-associated pneumonia in critically ill patients: a randomized controlled multicenter trial</strong><br />
Juan Zeng, Chun-Ting Wang, Fu-Shen Zhang, Feng Qi, Shi-Fu Wang, Shuang Ma, Tie-Jun Wu, Hui Tian, Zhao-Tao Tian, Shu-Liu Zhang, … Yu-Ping Wang<br />
<em>Intensive Care Medicine</em> (2016-04-04) <a href="https://doi.org/f8jnrt">https://doi.org/f8jnrt</a><br />
DOI: <a href="https://doi.org/10.1007/s00134-016-4303-x">10.1007/s00134-016-4303-x</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27043237">27043237</a></p>
</div>
<div id="ref-1E2aDJp4T">
<p>398. <strong>Probiotic Prophylaxis of Ventilator-associated Pneumonia</strong><br />
Lee E. Morrow, Marin H. Kollef, Thomas B. Casale<br />
<em>American Journal of Respiratory and Critical Care Medicine</em> (2010-10-15) <a href="https://doi.org/d5hh4t">https://doi.org/d5hh4t</a><br />
DOI: <a href="https://doi.org/10.1164/rccm.200912-1853oc">10.1164/rccm.200912-1853oc</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20522788">20522788</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2970846">PMC2970846</a></p>
</div>
<div id="ref-zccFgajh">
<p>399. <strong>Synbiotics modulate gut microbiota and reduce enteritis and ventilator-associated pneumonia in patients with sepsis: a randomized controlled trial</strong><br />
Kentaro Shimizu, Tomoki Yamada, Hiroshi Ogura, Tomoyoshi Mohri, Takeyuki Kiguchi, Satoshi Fujimi, Takashi Asahara, Tomomi Yamada, Masahiro Ojima, Mitsunori Ikeda, Takeshi Shimazu<br />
<em>Critical Care</em> (2018-09-27) <a href="https://doi.org/gfdggj">https://doi.org/gfdggj</a><br />
DOI: <a href="https://doi.org/10.1186/s13054-018-2167-x">10.1186/s13054-018-2167-x</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30261905">30261905</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6161427">PMC6161427</a></p>
</div>
<div id="ref-1DI06gFPc">
<p>400. <strong>Probiotics for the Prevention of Ventilator-Associated Pneumonia: A Meta-Analysis of Randomized Controlled Trials</strong><br />
Minmin Su, Ying Jia, Yan Li, Dianyou Zhou, Jinsheng Jia<br />
<em>Respiratory Care</em> (2020-03-03) <a href="https://doi.org/gg3kng">https://doi.org/gg3kng</a><br />
DOI: <a href="https://doi.org/10.4187/respcare.07097">10.4187/respcare.07097</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32127415">32127415</a></p>
</div>
<div id="ref-164mVBFZx">
<p>401. <strong>COVID-19: An Alert to Ventilator-Associated Bacterial Pneumonia</strong><br />
Helvécio Cardoso Corrêa Póvoa, Gabriela Ceccon Chianca, Natalia Lopes Pontes Póvoa Iorio<br />
<em>Adis Journals</em> (2020) <a href="https://doi.org/gg3knh">https://doi.org/gg3knh</a><br />
DOI: <a href="https://doi.org/10.6084/m9.figshare.12340496">10.6084/m9.figshare.12340496</a></p>
</div>
<div id="ref-w2VFo0h1">
<p>402. <strong>The challenge of ventilator-associated pneumonia diagnosis in COVID-19 patients</strong><br />
Bruno François, Pierre-François Laterre, Charles-Edouard Luyt, Jean Chastre<br />
<em>Critical Care</em> (2020-06-05) <a href="https://doi.org/gg3knc">https://doi.org/gg3knc</a><br />
DOI: <a href="https://doi.org/10.1186/s13054-020-03013-2">10.1186/s13054-020-03013-2</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32503590">32503590</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7273812">PMC7273812</a></p>
</div>
<div id="ref-gcyQ73v7">
<p>403. <strong>Prophylactic use of probiotics for gastrointestinal disorders in children</strong><br />
Celine Perceval, Hania Szajewska, Flavia Indrio, Zvi Weizman, Yvan Vandenplas<br />
<em>The Lancet Child &amp; Adolescent Health</em> (2019-09) <a href="https://doi.org/d2qp">https://doi.org/d2qp</a><br />
DOI: <a href="https://doi.org/10.1016/s2352-4642(19)30182-8">10.1016/s2352-4642(19)30182-8</a></p>
</div>
<div id="ref-67ihxDiV">
<p>404. <strong>Effect of gastrointestinal symptoms on patients infected with COVID-19</strong><br />
Zili Zhou, Ning Zhao, Yan Shu, Shengbo Han, Bin Chen, Xiaogang Shu<br />
<em>Gastroenterology</em> (2020-03) <a href="https://doi.org/ggq8x8">https://doi.org/ggq8x8</a><br />
DOI: <a href="https://doi.org/10.1053/j.gastro.2020.03.020">10.1053/j.gastro.2020.03.020</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32199880">32199880</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7270807">PMC7270807</a></p>
</div>
<div id="ref-8Hvgbl3n">
<p>405. <strong>The digestive system is a potential route of 2019-nCov infection: a bioinformatics analysis based on single-cell transcriptomes</strong><br />
Hao Zhang, Zijian Kang, Haiyi Gong, Da Xu, Jing Wang, Zifu Li, Xingang Cui, Jianru Xiao, Tong Meng, Wang Zhou, … Huji Xu<br />
<em>bioRxiv</em> (2020-01-31) <a href="https://doi.org/ggjvx2">https://doi.org/ggjvx2</a><br />
DOI: <a href="https://doi.org/10.1101/2020.01.30.927806">10.1101/2020.01.30.927806</a></p>
</div>
<div id="ref-Nhxoefru">
<p>406. <strong>Cryo-electron microscopy structures of the SARS-CoV spike glycoprotein reveal a prerequisite conformational state for receptor binding</strong><br />
Miao Gui, Wenfei Song, Haixia Zhou, Jingwei Xu, Silian Chen, Ye Xiang, Xinquan Wang<br />
<em>Cell Research</em> (2016-12-23) <a href="https://doi.org/f9m247">https://doi.org/f9m247</a><br />
DOI: <a href="https://doi.org/10.1038/cr.2016.152">10.1038/cr.2016.152</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28008928">28008928</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5223232">PMC5223232</a></p>
</div>
<div id="ref-TPWaxApy">
<p>407. <strong>Prolonged presence of SARS-CoV-2 viral RNA in faecal samples</strong><br />
Yongjian Wu, Cheng Guo, Lantian Tang, Zhongsi Hong, Jianhui Zhou, Xin Dong, Huan Yin, Qiang Xiao, Yanping Tang, Xiujuan Qu, … Xi Huang<br />
<em>The Lancet Gastroenterology &amp; Hepatology</em> (2020-05) <a href="https://doi.org/ggq8zp">https://doi.org/ggq8zp</a><br />
DOI: <a href="https://doi.org/10.1016/s2468-1253(20)30083-2">10.1016/s2468-1253(20)30083-2</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32199469">32199469</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7158584">PMC7158584</a></p>
</div>
<div id="ref-r9HNMqTS">
<p>408. <strong>Enteric involvement of coronaviruses: is faecal–oral transmission of SARS-CoV-2 possible?</strong><br />
Charleen Yeo, Sanghvi Kaushal, Danson Yeo<br />
<em>The Lancet Gastroenterology &amp; Hepatology</em> (2020-02) <a href="https://doi.org/ggpx7s">https://doi.org/ggpx7s</a><br />
DOI: <a href="https://doi.org/10.1016/s2468-1253(20)30048-0">10.1016/s2468-1253(20)30048-0</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32087098">32087098</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7130008">PMC7130008</a></p>
</div>
<div id="ref-eOcc1BUx">
<p>409. <strong>Characteristics of pediatric SARS-CoV-2 infection and potential evidence for persistent fecal viral shedding</strong><br />
Yi Xu, Xufang Li, Bing Zhu, Huiying Liang, Chunxiao Fang, Yu Gong, Qiaozhi Guo, Xin Sun, Danyang Zhao, Jun Shen, … Sitang Gong<br />
<em>Nature Medicine</em> (2020-03-13) <a href="https://doi.org/ggpwx5">https://doi.org/ggpwx5</a><br />
DOI: <a href="https://doi.org/10.1038/s41591-020-0817-4">10.1038/s41591-020-0817-4</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32284613">32284613</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7095102">PMC7095102</a></p>
</div>
<div id="ref-VwStn9Ps">
<p>410. <strong>Modulation of rotavirus severe gastroenteritis by the combination of probiotics and prebiotics</strong><br />
Guadalupe Gonzalez-Ochoa, Lilian K. Flores-Mendoza, Ramona Icedo-Garcia, Ricardo Gomez-Flores, Patricia Tamez-Guerra<br />
<em>Archives of Microbiology</em> (2017-06-20) <a href="https://doi.org/gbsb4d">https://doi.org/gbsb4d</a><br />
DOI: <a href="https://doi.org/10.1007/s00203-017-1400-3">10.1007/s00203-017-1400-3</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28634691">28634691</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5548957">PMC5548957</a></p>
</div>
<div id="ref-UtAKolza">
<p>411. <strong>Multicenter Trial of a Combination Probiotic for Children with Gastroenteritis</strong><br />
Stephen B. Freedman, Sarah Williamson-Urquhart, Ken J. Farion, Serge Gouin, Andrew R. Willan, Naveen Poonai, Katrina Hurley, Philip M. Sherman, Yaron Finkelstein, Bonita E. Lee, … Suzanne Schuh<br />
<em>New England Journal of Medicine</em> (2018-11-22) <a href="https://doi.org/gfkbsf">https://doi.org/gfkbsf</a><br />
DOI: <a href="https://doi.org/10.1056/nejmoa1802597">10.1056/nejmoa1802597</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30462939">30462939</a></p>
</div>
<div id="ref-1AqF1ESKq">
<p>412. <strong>Synbiotic Therapy of Gastrointestinal Symptoms During Covid-19 Infection: A Randomized, Double-blind, Placebo Controlled, Telemedicine Study (SynCov Study)</strong><br />
Medical University of Graz<br />
<em>clinicaltrials.gov</em> (2020-06-05) <a href="https://clinicaltrials.gov/ct2/show/NCT04420676">https://clinicaltrials.gov/ct2/show/NCT04420676</a></p>
</div>
<div id="ref-9UYsmETk">
<p>413. <strong>Multicentric Study to Assess the Effect of Consumption of Lactobacillus Coryniformis K8 on Healthcare Personnel Exposed to COVID-19</strong><br />
Biosearch S. A.<br />
<em>clinicaltrials.gov</em> (2020-04-28) <a href="https://clinicaltrials.gov/ct2/show/NCT04366180">https://clinicaltrials.gov/ct2/show/NCT04366180</a></p>
</div>
<div id="ref-15pDlYtom">
<p>414. <strong>The Intestinal Microbiota as a Therapeutic Target in Hospitalized Patients With COVID-19 Infection</strong><br />
Bioithas SL<br />
<em>clinicaltrials.gov</em> (2020-05-14) <a href="https://clinicaltrials.gov/ct2/show/NCT04390477">https://clinicaltrials.gov/ct2/show/NCT04390477</a></p>
</div>
<div id="ref-m3SexO9F">
<p>415. <strong>Probiotics: definition, scope and mechanisms of action</strong><br />
Gregor Reid<br />
<em>Best Practice &amp; Research Clinical Gastroenterology</em> (2016-02) <a href="https://doi.org/f8m79k">https://doi.org/f8m79k</a><br />
DOI: <a href="https://doi.org/10.1016/j.bpg.2015.12.001">10.1016/j.bpg.2015.12.001</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27048893">27048893</a></p>
</div>
<div id="ref-5FTOk7ol">
<p>416. <strong>Health benefits and health claims of probiotics: bridging science and marketing</strong><br />
Ger T. Rijkers, Willem M. de Vos, Robert-Jan Brummer, Lorenzo Morelli, Gerard Corthier, Philippe Marteau<br />
<em>British Journal of Nutrition</em> (2011-08-24) <a href="https://doi.org/cb78rx">https://doi.org/cb78rx</a><br />
DOI: <a href="https://doi.org/10.1017/s000711451100287x">10.1017/s000711451100287x</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21861940">21861940</a></p>
</div>
<div id="ref-kv94VRLF">
<p>417. <strong>Probiotics and COVID-19: one size does not fit all</strong><br />
Joyce WY Mak, Francis KL Chan, Siew C Ng<br />
<em>The Lancet Gastroenterology &amp; Hepatology</em> (2020-07) <a href="https://doi.org/d2qq">https://doi.org/d2qq</a><br />
DOI: <a href="https://doi.org/10.1016/s2468-1253(20)30122-9">10.1016/s2468-1253(20)30122-9</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32339473">32339473</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7182525">PMC7182525</a></p>
</div>
<div id="ref-5tHMedSi">
<p>418. <strong>Optimal Nutritional Status for a Well-Functioning Immune System Is an Important Factor to Protect against Viral Infections</strong><br />
Philip C. Calder, Anitra C. Carr, Adrian F. Gombart, Manfred Eggersdorfer<br />
<em>Nutrients</em> (2020-04-23) <a href="https://doi.org/gg29hh">https://doi.org/gg29hh</a><br />
DOI: <a href="https://doi.org/10.3390/nu12041181">10.3390/nu12041181</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32340216">32340216</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7230749">PMC7230749</a></p>
</div>
<div id="ref-Lyd8s4Mk">
<p>419. <strong>COVID-19: The Inflammation Link and the Role of Nutrition in Potential Mitigation</strong><br />
Ioannis Zabetakis, Ronan Lordan, Catherine Norton, Alexandros Tsoupras<br />
<em>Nutrients</em> (2020-05-19) <a href="https://doi.org/ggxdq3">https://doi.org/ggxdq3</a><br />
DOI: <a href="https://doi.org/10.3390/nu12051466">10.3390/nu12051466</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32438620">32438620</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7284818">PMC7284818</a></p>
</div>
<div id="ref-1DmkRczqw">
<p>420. <strong>Nutritional management of COVID-19 patients in a rehabilitation unit</strong><br />
Luigia Brugliera, Alfio Spina, Paola Castellazzi, Paolo Cimino, Pietro Arcuri, Alessandra Negro, Elise Houdayer, Federica Alemanno, Alessandra Giordani, Pietro Mortini, Sandro Iannaccone<br />
<em>European Journal of Clinical Nutrition</em> (2020-05-20) <a href="https://doi.org/gg29hf">https://doi.org/gg29hf</a><br />
DOI: <a href="https://doi.org/10.1038/s41430-020-0664-x">10.1038/s41430-020-0664-x</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32433599">32433599</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7237874">PMC7237874</a></p>
</div>
<div id="ref-4vqbVndZ">
<p>421. <strong>Proflaxis for Healthcare Professionals Using Hydroxychloroquine Plus Vitamin Combining Vitamins C, D and Zinc During COVID-19 Pandemia: An Observational Study</strong><br />
Mehmet Mahir Ozmen<br />
<em>clinicaltrials.gov</em> (2020-07-27) <a href="https://clinicaltrials.gov/ct2/show/NCT04326725">https://clinicaltrials.gov/ct2/show/NCT04326725</a></p>
</div>
<div id="ref-lMhFSIiF">
<p>422. <strong>The frontier between nutrition and pharma: The international regulatory framework of functional foods, food supplements and nutraceuticals</strong><br />
Laura Domínguez Díaz, Virginia Fernández-Ruiz, Montaña Cámara<br />
<em>Critical Reviews in Food Science and Nutrition</em> (2019-03-29) <a href="https://doi.org/ggqs3w">https://doi.org/ggqs3w</a><br />
DOI: <a href="https://doi.org/10.1080/10408398.2019.1592107">10.1080/10408398.2019.1592107</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30924346">30924346</a></p>
</div>
<div id="ref-fez8aZAu">
<p>423. <strong>Coronavirus Update: FDA and FTC Warn Seven Companies Selling Fraudulent Products that Claim to Treat or Prevent COVID-19</strong><br />
Office of the Commissioner<br />
<em>FDA</em> (2020-03-27) <a href="https://www.fda.gov/news-events/press-announcements/coronavirus-update-fda-and-ftc-warn-seven-companies-selling-fraudulent-products-claim-treat-or">https://www.fda.gov/news-events/press-announcements/coronavirus-update-fda-and-ftc-warn-seven-companies-selling-fraudulent-products-claim-treat-or</a></p>
</div>
<div id="ref-2Xu90WtD">
<p>424. <strong>Coronavirus Disease 2019 (COVID-19) – Prevention &amp; Treatment</strong><br />
CDC<br />
<em>Centers for Disease Control and Prevention</em> (2020-04-24) <a href="https://www.cdc.gov/coronavirus/2019-ncov/prevent-getting-sick/prevention.html">https://www.cdc.gov/coronavirus/2019-ncov/prevent-getting-sick/prevention.html</a></p>
</div>
<div id="ref-EhbZIMo0">
<p>425. <strong>Potential roles of social distancing in mitigating the spread of coronavirus disease 2019 (COVID-19) in South Korea</strong><br />
Sang Woo Park, Kaiyuan Sun, Cécile Viboud, Bryan T Grenfell, Jonathan Dushoff<br />
<em>medRxiv</em> (2020-03-30) <a href="https://doi.org/gg3mhg">https://doi.org/gg3mhg</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.27.20045815">10.1101/2020.03.27.20045815</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32511429">32511429</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7217070">PMC7217070</a></p>
</div>
<div id="ref-12Met0WvX">
<p>426. <strong>Evaluating the Effectiveness of Social Distancing Interventions to Delay or Flatten the Epidemic Curve of Coronavirus Disease</strong><br />
Laura Matrajt, Tiffany Leung<br />
<em>Emerging Infectious Diseases</em> (2020-08) <a href="https://doi.org/ggtx3k">https://doi.org/ggtx3k</a><br />
DOI: <a href="https://doi.org/10.3201/eid2608.201093">10.3201/eid2608.201093</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32343222">32343222</a></p>
</div>
<div id="ref-bZMKqj6e">
<p>427. <strong>Into the Eye of the Cytokine Storm</strong><br />
J. R. Tisoncik, M. J. Korth, C. P. Simmons, J. Farrar, T. R. Martin, M. G. Katze<br />
<em>Microbiology and Molecular Biology Reviews</em> (2012-03-05) <a href="https://doi.org/f4n9h2">https://doi.org/f4n9h2</a><br />
DOI: <a href="https://doi.org/10.1128/mmbr.05015-11">10.1128/mmbr.05015-11</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22390970">22390970</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3294426">PMC3294426</a></p>
</div>
<div id="ref-cVGAh42z">
<p>428. <strong>Induction of cell cycle arrest and B cell terminal differentiation by CDK inhibitor p18(INK4c) and IL-6.</strong><br />
L Morse, D Chen, D Franklin, Y Xiong, S Chen-Kiang<br />
<em>Immunity</em> (1997-01) <a href="https://www.ncbi.nlm.nih.gov/pubmed/9052836">https://www.ncbi.nlm.nih.gov/pubmed/9052836</a><br />
DOI: <a href="https://doi.org/10.1016/s1074-7613(00)80241-1">10.1016/s1074-7613(00)80241-1</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/9052836">9052836</a></p>
</div>
<div id="ref-1HPUdhrBf">
<p>429. <strong>The effects of hybridoma growth factor in conditioned media upon the growth, cloning, and antibody production of heterohybridoma cell lines.</strong><br />
Y Zhu, B Jin, C Sun, C Huang, X Liu<br />
<em>Human antibodies and hybridomas</em> (1993-01) <a href="https://www.ncbi.nlm.nih.gov/pubmed/8431556">https://www.ncbi.nlm.nih.gov/pubmed/8431556</a><br />
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/8431556">8431556</a></p>
</div>
<div id="ref-g6GOVWD3">
<p>430. <strong>Interleukin-6 is the central tumor growth factor in vitro and in vivo in multiple myeloma.</strong><br />
B Klein, XG Zhang, M Jourdan, JM Boiron, M Portier, ZY Lu, J Wijdenes, J Brochier, R Bataille<br />
<em>European cytokine network</em> <a href="https://www.ncbi.nlm.nih.gov/pubmed/2104241">https://www.ncbi.nlm.nih.gov/pubmed/2104241</a><br />
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/2104241">2104241</a></p>
</div>
<div id="ref-eW4MC6Do">
<p>431. <strong>The Biology and Medical Implications of Interleukin-6</strong><br />
T. Tanaka, T. Kishimoto<br />
<em>Cancer Immunology Research</em> (2014-04-03) <a href="https://doi.org/f56dzb">https://doi.org/f56dzb</a><br />
DOI: <a href="https://doi.org/10.1158/2326-6066.cir-14-0022">10.1158/2326-6066.cir-14-0022</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24764575">24764575</a></p>
</div>
<div id="ref-QkTGQUcj">
<p>432. <strong>Up-regulation of IL-6 and TNF-α induced by SARS-coronavirus spike protein in murine macrophages via NF-κB pathway</strong><br />
Wei Wang, Linbai Ye, Li Ye, Baozong Li, Bo Gao, Yingchun Zeng, Lingbao Kong, Xiaonan Fang, Hong Zheng, Zhenghui Wu, Yinglong She<br />
<em>Virus Research</em> (2007-09) <a href="https://doi.org/bm7m55">https://doi.org/bm7m55</a><br />
DOI: <a href="https://doi.org/10.1016/j.virusres.2007.02.007">10.1016/j.virusres.2007.02.007</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17532082">17532082</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7114322">PMC7114322</a></p>
</div>
<div id="ref-NNFGje9g">
<p>433. <strong>Expression of elevated levels of pro-inflammatory cytokines in SARS-CoV-infected ACE2 <sup>+</sup> cells in SARS patients: relation to the acute lung injury and pathogenesis of SARS</strong><br />
L He, Y Ding, Q Zhang, X Che, Y He, H Shen, H Wang, Z Li, L Zhao, J Geng, … S Jiang<br />
<em>The Journal of Pathology</em> (2006-11) <a href="https://doi.org/bwb8ns">https://doi.org/bwb8ns</a><br />
DOI: <a href="https://doi.org/10.1002/path.2067">10.1002/path.2067</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17031779">17031779</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7167655">PMC7167655</a></p>
</div>
<div id="ref-f61jsRKY">
<p>434. <strong>Cytokine Balance in the Lungs of Patients with Acute Respiratory Distress Syndrome</strong><br />
WILLIAM Y. PARK, RICHARD B. GOODMAN, KENNETH P. STEINBERG, JOHN T. RUZINSKI, FRANK RADELLA, DAVID R. PARK, JEROME PUGIN, SHAWN J. SKERRETT, LEONARD D. HUDSON, THOMAS R. MARTIN<br />
<em>American Journal of Respiratory and Critical Care Medicine</em> (2001-11-15) <a href="https://doi.org/ggqfq7">https://doi.org/ggqfq7</a><br />
DOI: <a href="https://doi.org/10.1164/ajrccm.164.10.2104013">10.1164/ajrccm.164.10.2104013</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/11734443">11734443</a></p>
</div>
<div id="ref-suIFBW">
<p>435. <strong>Pathogenic human coronavirus infections: causes and consequences of cytokine storm and immunopathology</strong><br />
Rudragouda Channappanavar, Stanley Perlman<br />
<em>Seminars in Immunopathology</em> (2017-05-02) <a href="https://doi.org/ggqf2w">https://doi.org/ggqf2w</a><br />
DOI: <a href="https://doi.org/10.1007/s00281-017-0629-x">10.1007/s00281-017-0629-x</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28466096">28466096</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7079893">PMC7079893</a></p>
</div>
<div id="ref-1BxkPZYnj">
<p>436. <strong>Plasticity and cross-talk of Interleukin 6-type cytokines</strong><br />
Christoph Garbers, Heike M. Hermanns, Fred Schaper, Gerhard Müller-Newen, Joachim Grötzinger, Stefan Rose-John, Jürgen Scheller<br />
<em>Cytokine &amp; Growth Factor Reviews</em> (2012-06) <a href="https://doi.org/f3z743">https://doi.org/f3z743</a><br />
DOI: <a href="https://doi.org/10.1016/j.cytogfr.2012.04.001">10.1016/j.cytogfr.2012.04.001</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22595692">22595692</a></p>
</div>
<div id="ref-1EEloBjzn">
<p>437. <strong>Soluble receptors for cytokines and growth factors: generation and biological function</strong><br />
S Rose-John, PC Heinrich<br />
<em>Biochemical Journal</em> (1994-06-01) <a href="https://doi.org/ggqmgd">https://doi.org/ggqmgd</a><br />
DOI: <a href="https://doi.org/10.1042/bj3000281">10.1042/bj3000281</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/8002928">8002928</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1138158">PMC1138158</a></p>
</div>
<div id="ref-VXDsRwHC">
<p>438. <strong>Hall of Fame among Pro-inflammatory Cytokines: Interleukin-6 Gene and Its Transcriptional Regulation Mechanisms</strong><br />
Yang Luo, Song Guo Zheng<br />
<em>Frontiers in Immunology</em> (2016-12-19) <a href="https://doi.org/ggqmgv">https://doi.org/ggqmgv</a><br />
DOI: <a href="https://doi.org/10.3389/fimmu.2016.00604">10.3389/fimmu.2016.00604</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28066415">28066415</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5165036">PMC5165036</a></p>
</div>
<div id="ref-1H8o8TFtM">
<p>439. <strong>Interleukin-6; pathogenesis and treatment of autoimmune inflammatory diseases</strong><br />
Toshio Tanaka, Masashi Narazaki, Kazuya Masuda, Tadamitsu Kishimoto<br />
<em>Inflammation and Regeneration</em> (2013) <a href="https://doi.org/ggqmgt">https://doi.org/ggqmgt</a><br />
DOI: <a href="https://doi.org/10.2492/inflammregen.33.054">10.2492/inflammregen.33.054</a></p>
</div>
<div id="ref-JgXCBFEo">
<p>440. <strong>Genentech Launches Phase III Trial of Actemra as Coronavirus Treatment</strong><br />
GEN - Genetic Engineering and Biotechnology News<br />
(2020-03-19) <a href="https://www.genengnews.com/virology/coronavirus/genentech-launches-phase-iii-trial-of-actemra-as-coronavirus-treatment/">https://www.genengnews.com/virology/coronavirus/genentech-launches-phase-iii-trial-of-actemra-as-coronavirus-treatment/</a></p>
</div>
<div id="ref-123fXWICW">
<p>441. <strong>COVID-19 – Italy launches an independent trial on tocilizumab</strong><br />
COVID-19 – Italy launches an independent trial on tocilizumab | Univadis<br />
<a href="https://www.univadis.co.uk/viewarticle/covid-19-italy-launches-an-independent-trial-on-tocilizumab-715741">https://www.univadis.co.uk/viewarticle/covid-19-italy-launches-an-independent-trial-on-tocilizumab-715741</a></p>
</div>
<div id="ref-16AWSEc1x">
<p>442. <strong>Roche’s treatment for coronavirus enters Phase III</strong><br />
biopharma-reporter.com<br />
<em>biopharma-reporter.com</em> <a href="https://www.biopharma-reporter.com/Article/2020/03/19/Roche-enters-Phase-III-for-COVID-19-treatment">https://www.biopharma-reporter.com/Article/2020/03/19/Roche-enters-Phase-III-for-COVID-19-treatment</a></p>
</div>
<div id="ref-TgV8gb30">
<p>443. <strong>Risk of infections in rheumatoid arthritis patients treated with tocilizumab</strong><br />
Veronika R. Lang, Matthias Englbrecht, Jürgen Rech, Hubert Nüsslein, Karin Manger, Florian Schuch, Hans-Peter Tony, Martin Fleck, Bernhard Manger, Georg Schett, Jochen Zwerina<br />
<em>Rheumatology</em> (2012-05) <a href="https://doi.org/d3b3rh">https://doi.org/d3b3rh</a><br />
DOI: <a href="https://doi.org/10.1093/rheumatology/ker223">10.1093/rheumatology/ker223</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21865281">21865281</a></p>
</div>
<div id="ref-IJzmtJOr">
<p>444. <strong>Short-course tocilizumab increases risk of hepatitis B virus reactivation in patients with rheumatoid arthritis: a prospective clinical observation</strong><br />
Le-Feng Chen, Ying-Qian Mo, Jun Jing, Jian-Da Ma, Dong-Hui Zheng, Lie Dai<br />
<em>International Journal of Rheumatic Diseases</em> (2017-07) <a href="https://doi.org/f9pbc5">https://doi.org/f9pbc5</a><br />
DOI: <a href="https://doi.org/10.1111/1756-185x.13010">10.1111/1756-185x.13010</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28160426">28160426</a></p>
</div>
<div id="ref-JMM4eCZ3">
<p>445. <strong>Arthritis Drug Presents Promise as Treatment for COVID-19 Pneumonia</strong><br />
Scott LaFee<br />
<em>UC San Diego News Center</em> (2020-04-29) <a href="https://ucsdnews.ucsd.edu/pressrelease/arthritis-drug-presents-promise-as-treatment-for-covid-19-pneumonia">https://ucsdnews.ucsd.edu/pressrelease/arthritis-drug-presents-promise-as-treatment-for-covid-19-pneumonia</a></p>
</div>
<div id="ref-SiOKIbl3">
<p>446. <strong>Why tocilizumab could be an effective treatment for severe COVID-19?</strong><br />
Binqing Fu, Xiaoling Xu, Haiming Wei<br />
<em>Journal of Translational Medicine</em> (2020-04-14) <a href="https://doi.org/ggv5c8">https://doi.org/ggv5c8</a><br />
DOI: <a href="https://doi.org/10.1186/s12967-020-02339-3">10.1186/s12967-020-02339-3</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32290839">32290839</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7154566">PMC7154566</a></p>
</div>
<div id="ref-bj2feMy4">
<p>447. <strong>Effective treatment of severe COVID-19 patients with tocilizumab</strong><br />
Xiaoling Xu, Mingfeng Han, Tiantian Li, Wei Sun, Dongsheng Wang, Binqing Fu, Yonggang Zhou, Xiaohu Zheng, Yun Yang, Xiuyong Li, … Haiming Wei<br />
<em>Proceedings of the National Academy of Sciences</em> (2020-05-19) <a href="https://doi.org/ggv3r3">https://doi.org/ggv3r3</a><br />
DOI: <a href="https://doi.org/10.1073/pnas.2005615117">10.1073/pnas.2005615117</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32350134">32350134</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7245089">PMC7245089</a></p>
</div>
<div id="ref-6xLhs9st">
<p>448. <strong>Tocilizumab during pregnancy and lactation: drug levels in maternal serum, cord blood, breast milk and infant serum</strong><br />
Jumpei Saito, Naho Yakuwa, Kayoko Kaneko, Chinatsu Takai, Mikako Goto, Ken Nakajima, Akimasa Yamatani, Atsuko Murashima<br />
<em>Rheumatology</em> (2019-08) <a href="https://doi.org/ggzhks">https://doi.org/ggzhks</a><br />
DOI: <a href="https://doi.org/10.1093/rheumatology/kez100">10.1093/rheumatology/kez100</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30945743">30945743</a></p>
</div>
<div id="ref-1AKi0FYUB">
<p>449. <strong>Development of therapeutic antibodies for the treatment of diseases</strong><br />
Ruei-Min Lu, Yu-Chyi Hwang, I-Ju Liu, Chi-Chiu Lee, Han-Zen Tsai, Hsin-Jung Li, Han-Chung Wu<br />
<em>Journal of Biomedical Science</em> (2020-01-02) <a href="https://doi.org/ggqbpx">https://doi.org/ggqbpx</a><br />
DOI: <a href="https://doi.org/10.1186/s12929-019-0592-z">10.1186/s12929-019-0592-z</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31894001">31894001</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6939334">PMC6939334</a></p>
</div>
<div id="ref-yumBaJ6U">
<p>450. <strong>Broadly Neutralizing Antiviral Antibodies</strong><br />
Davide Corti, Antonio Lanzavecchia<br />
<em>Annual Review of Immunology</em> (2013-03-21) <a href="https://doi.org/gf25g8">https://doi.org/gf25g8</a><br />
DOI: <a href="https://doi.org/10.1146/annurev-immunol-032712-095916">10.1146/annurev-immunol-032712-095916</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23330954">23330954</a></p>
</div>
<div id="ref-dUnB3gD6">
<p>451. <strong>Ibalizumab Targeting CD4 Receptors, An Emerging Molecule in HIV Therapy</strong><br />
Simona A. Iacob, Diana G. Iacob<br />
<em>Frontiers in Microbiology</em> (2017-11-27) <a href="https://doi.org/gcn3kh">https://doi.org/gcn3kh</a><br />
DOI: <a href="https://doi.org/10.3389/fmicb.2017.02323">10.3389/fmicb.2017.02323</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29230203">29230203</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5711820">PMC5711820</a></p>
</div>
<div id="ref-cwbHiM1o">
<p>452. <strong>Product review on the monoclonal antibody palivizumab for prevention of respiratory syncytial virus infection</strong><br />
Bernhard Resch<br />
<em>Human Vaccines &amp; Immunotherapeutics</em> (2017-06-12) <a href="https://doi.org/ggqbps">https://doi.org/ggqbps</a><br />
DOI: <a href="https://doi.org/10.1080/21645515.2017.1337614">10.1080/21645515.2017.1337614</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28605249">28605249</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5612471">PMC5612471</a></p>
</div>
<div id="ref-3gp7t98X">
<p>453. <strong>Chronological evolution of IgM, IgA, IgG and neutralisation antibodies after infection with SARS-associated coronavirus</strong><br />
P.-R. Hsueh, L.-M. Huang, P.-J. Chen, C.-L. Kao, P.-C. Yang<br />
<em>Clinical Microbiology and Infection</em> (2004-12) <a href="https://doi.org/cwwg87">https://doi.org/cwwg87</a><br />
DOI: <a href="https://doi.org/10.1111/j.1469-0691.2004.01009.x">10.1111/j.1469-0691.2004.01009.x</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15606632">15606632</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7129952">PMC7129952</a></p>
</div>
<div id="ref-KULxo48U">
<p>454. <strong>Neutralizing Antibodies in Patients with Severe Acute Respiratory Syndrome-Associated Coronavirus Infection</strong><br />
Nie Yuchun, Wang Guangwen, Shi Xuanling, Zhang Hong, Qiu Yan, He Zhongping, Wang Wei, Lian Gewei, Yin Xiaolei, Du Liying, … Ding Mingxiao<br />
<em>The Journal of Infectious Diseases</em> (2004-09) <a href="https://doi.org/cgqj5b">https://doi.org/cgqj5b</a><br />
DOI: <a href="https://doi.org/10.1086/423286">10.1086/423286</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15319862">15319862</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7199490">PMC7199490</a></p>
</div>
<div id="ref-MTcTiS7i">
<p>455. <strong>Potent human monoclonal antibodies against SARS CoV, Nipah and Hendra viruses</strong><br />
Ponraj Prabakaran, Zhongyu Zhu, Xiaodong Xiao, Arya Biragyn, Antony S Dimitrov, Christopher C Broder, Dimiter S Dimitrov<br />
<em>Expert Opinion on Biological Therapy</em> (2009-04-08) <a href="https://doi.org/b88kw8">https://doi.org/b88kw8</a><br />
DOI: <a href="https://doi.org/10.1517/14712590902763755">10.1517/14712590902763755</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19216624">19216624</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2705284">PMC2705284</a></p>
</div>
<div id="ref-rXzkS3gA">
<p>456. <strong>Prior Infection and Passive Transfer of Neutralizing Antibody Prevent Replication of Severe Acute Respiratory Syndrome Coronavirus in the Respiratory Tract of Mice</strong><br />
Kanta Subbarao, Josephine McAuliffe, Leatrice Vogel, Gary Fahle, Steven Fischer, Kathleen Tatti, Michelle Packard, Wun-Ju Shieh, Sherif Zaki, Brian Murphy<br />
<em>Journal of Virology</em> (2004-04-01) <a href="https://doi.org/b8wr7c">https://doi.org/b8wr7c</a><br />
DOI: <a href="https://doi.org/10.1128/jvi.78.7.3572-3577.2004">10.1128/jvi.78.7.3572-3577.2004</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15016880">15016880</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC371090">PMC371090</a></p>
</div>
<div id="ref-RRx6YovP">
<p>457. <strong>The Effectiveness of Convalescent Plasma and Hyperimmune Immunoglobulin for the Treatment of Severe Acute Respiratory Infections of Viral Etiology: A Systematic Review and Exploratory Meta-analysis</strong><br />
John Mair-Jenkins, Maria Saavedra-Campos, J. Kenneth Baillie, Paul Cleary, Fu-Meng Khaw, Wei Shen Lim, Sophia Makki, Kevin D. Rooney, Jonathan S. Nguyen-Van-Tam, Charles R. Beck, Convalescent Plasma Study Group<br />
<em>Journal of Infectious Diseases</em> (2015-01-01) <a href="https://doi.org/f632n7">https://doi.org/f632n7</a><br />
DOI: <a href="https://doi.org/10.1093/infdis/jiu396">10.1093/infdis/jiu396</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25030060">25030060</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4264590">PMC4264590</a></p>
</div>
<div id="ref-Ou3P9pGV">
<p>458. <strong>Identification of human neutralizing antibodies against MERS-CoV and their role in virus adaptive evolution</strong><br />
X.-C. Tang, S. S. Agnihothram, Y. Jiao, J. Stanhope, R. L. Graham, E. C. Peterson, Y. Avnir, A. S. C. Tallarico, J. Sheehan, Q. Zhu, … W. A. Marasco<br />
<em>Proceedings of the National Academy of Sciences</em> (2014-04-28) <a href="https://doi.org/smr">https://doi.org/smr</a><br />
DOI: <a href="https://doi.org/10.1073/pnas.1402074111">10.1073/pnas.1402074111</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24778221">24778221</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4024880">PMC4024880</a></p>
</div>
<div id="ref-1GLr0EJU">
<p>459. <strong>The Role of ACE2 in Cardiovascular Physiology</strong><br />
Gavin Y. Oudit, Michael A. Crackower, Peter H. Backx, Josef M. Penninger<br />
<em>Trends in Cardiovascular Medicine</em> (2003-04) <a href="https://doi.org/bsbp49">https://doi.org/bsbp49</a><br />
DOI: <a href="https://doi.org/10.1016/s1050-1738(02)00233-5">10.1016/s1050-1738(02)00233-5</a></p>
</div>
<div id="ref-XcuzhxrJ">
<p>460. <strong>A human monoclonal antibody blocking SARS-CoV-2 infection</strong><br />
Chunyan Wang, Wentao Li, Dubravka Drabek, Nisreen M. A. Okba, Rien van Haperen, Albert D. M. E. Osterhaus, Frank J. M. van Kuppeveld, Bart L. Haagmans, Frank Grosveld, Berend-Jan Bosch<br />
<em>bioRxiv</em> (2020-03-12) <a href="https://doi.org/ggnw4t">https://doi.org/ggnw4t</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.11.987958">10.1101/2020.03.11.987958</a></p>
</div>
<div id="ref-Ftbm1M9p">
<p>461. <strong>Unexpected Receptor Functional Mimicry Elucidates Activation of Coronavirus Fusion</strong><br />
Alexandra C. Walls, Xiaoli Xiong, Young-Jun Park, M. Alejandra Tortorici, Joost Snijder, Joel Quispe, Elisabetta Cameroni, Robin Gopal, Mian Dai, Antonio Lanzavecchia, … David Veesler<br />
<em>Cell</em> (2019-02) <a href="https://doi.org/gft3jg">https://doi.org/gft3jg</a><br />
DOI: <a href="https://doi.org/10.1016/j.cell.2018.12.028">10.1016/j.cell.2018.12.028</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30712865">30712865</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6751136">PMC6751136</a></p>
</div>
<div id="ref-O1whWg6Q">
<p>462. <strong><em>In Vitro</em> Neutralization Is Not Predictive of Prophylactic Efficacy of Broadly Neutralizing Monoclonal Antibodies CR6261 and CR9114 against Lethal H2 Influenza Virus Challenge in Mice</strong><br />
Troy C. Sutton, Elaine W. Lamirande, Kevin W. Bock, Ian N. Moore, Wouter Koudstaal, Muniza Rehman, Gerrit Jan Weverling, Jaap Goudsmit, Kanta Subbarao<br />
<em>Journal of Virology</em> (2017-12-15) <a href="https://doi.org/ggqbpt">https://doi.org/ggqbpt</a><br />
DOI: <a href="https://doi.org/10.1128/jvi.01603-17">10.1128/jvi.01603-17</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29046448">29046448</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5709608">PMC5709608</a></p>
</div>
<div id="ref-pUnzB8wV">
<p>463. <strong>Human neutralizing antibodies against MERS coronavirus: implications for future immunotherapy</strong><br />
Xian-Chun Tang, Wayne A Marasco<br />
<em>Immunotherapy</em> (2015-07) <a href="https://doi.org/ggqbpz">https://doi.org/ggqbpz</a><br />
DOI: <a href="https://doi.org/10.2217/imt.15.33">10.2217/imt.15.33</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26098703">26098703</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5068219">PMC5068219</a></p>
</div>
<div id="ref-14motyCOm">
<p>464. <strong>A Potent and Broad Neutralizing Antibody Recognizes and Penetrates the HIV Glycan Shield</strong><br />
R. Pejchal, K. J. Doores, L. M. Walker, R. Khayat, P.-S. Huang, S.-K. Wang, R. L. Stanfield, J.-P. Julien, A. Ramos, M. Crispin, … I. A. Wilson<br />
<em>Science</em> (2011-10-13) <a href="https://doi.org/bzqv8c">https://doi.org/bzqv8c</a><br />
DOI: <a href="https://doi.org/10.1126/science.1213256">10.1126/science.1213256</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21998254">21998254</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3280215">PMC3280215</a></p>
</div>
<div id="ref-oGHFF5Cm">
<p>465. <strong>Broadly Neutralizing Antibodies against HIV: Back to Blood</strong><br />
Amir Dashti, Anthony L. DeVico, George K. Lewis, Mohammad M. Sajadi<br />
<em>Trends in Molecular Medicine</em> (2019-03) <a href="https://doi.org/ggqbpr">https://doi.org/ggqbpr</a><br />
DOI: <a href="https://doi.org/10.1016/j.molmed.2019.01.007">10.1016/j.molmed.2019.01.007</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30792120">30792120</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6401214">PMC6401214</a></p>
</div>
<div id="ref-hvjXMm9H">
<p>466. <strong>Importance of Neutralizing Monoclonal Antibodies Targeting Multiple Antigenic Sites on the Middle East Respiratory Syndrome Coronavirus Spike Glycoprotein To Avoid Neutralization Escape</strong><br />
Lingshu Wang, Wei Shi, James D. Chappell, M. Gordon Joyce, Yi Zhang, Masaru Kanekiyo, Michelle M. Becker, Neeltje van Doremalen, Robert Fischer, Nianshuang Wang, … Barney S. Graham<br />
<em>Journal of Virology</em> (2018-04-27) <a href="https://doi.org/ggqbpv">https://doi.org/ggqbpv</a><br />
DOI: <a href="https://doi.org/10.1128/jvi.02002-17">10.1128/jvi.02002-17</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29514901">29514901</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5923077">PMC5923077</a></p>
</div>
<div id="ref-tC5vJmwj">
<p>467. <strong>Anti–spike IgG causes severe acute lung injury by skewing macrophage responses during acute SARS-CoV infection</strong><br />
Li Liu, Qiang Wei, Qingqing Lin, Jun Fang, Haibo Wang, Hauyee Kwok, Hangying Tang, Kenji Nishiura, Jie Peng, Zhiwu Tan, … Zhiwei Chen<br />
<em>JCI Insight</em> (2019-02-21) <a href="https://doi.org/ggqbpw">https://doi.org/ggqbpw</a><br />
DOI: <a href="https://doi.org/10.1172/jci.insight.123158">10.1172/jci.insight.123158</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30830861">30830861</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6478436">PMC6478436</a></p>
</div>
<div id="ref-LtFtUo2P">
<p>468. <strong>The antiviral effect of interferon-beta against SARS-Coronavirus is not mediated by MxA protein</strong><br />
Martin Spiegel, Andreas Pichlmair, Elke Mühlberger, Otto Haller, Friedemann Weber<br />
<em>Journal of Clinical Virology</em> (2004-07) <a href="https://doi.org/cmc3ds">https://doi.org/cmc3ds</a><br />
DOI: <a href="https://doi.org/10.1016/j.jcv.2003.11.013">10.1016/j.jcv.2003.11.013</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15135736">15135736</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7128634">PMC7128634</a></p>
</div>
<div id="ref-Nfu9kiae">
<p>469. <strong>Coronavirus virulence genes with main focus on SARS-CoV envelope gene</strong><br />
Marta L. DeDiego, Jose L. Nieto-Torres, Jose M. Jimenez-Guardeño, Jose A. Regla-Nava, Carlos Castaño-Rodriguez, Raul Fernandez-Delgado, Fernando Usera, Luis Enjuanes<br />
<em>Virus Research</em> (2014-12) <a href="https://doi.org/f6wm24">https://doi.org/f6wm24</a><br />
DOI: <a href="https://doi.org/10.1016/j.virusres.2014.07.024">10.1016/j.virusres.2014.07.024</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25093995">25093995</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4261026">PMC4261026</a></p>
</div>
<div id="ref-1GA95MF2m">
<p>470. <strong>The COVID-19 vaccine development landscape</strong><br />
Tung Thanh Le, Zacharias Andreadakis, Arun Kumar, Raúl Gómez Román, Stig Tollefsen, Melanie Saville, Stephen Mayhew<br />
<em>Nature Reviews Drug Discovery</em> (2020-04-09) <a href="https://doi.org/ggrnbr">https://doi.org/ggrnbr</a><br />
DOI: <a href="https://doi.org/10.1038/d41573-020-00073-5">10.1038/d41573-020-00073-5</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32273591">32273591</a></p>
</div>
<div id="ref-HyYY2agc">
<p>471. <strong>Developing Covid-19 Vaccines at Pandemic Speed</strong><br />
Nicole Lurie, Melanie Saville, Richard Hatchett, Jane Halton<br />
<em>New England Journal of Medicine</em> (2020-03-30) <a href="https://doi.org/ggq8bc">https://doi.org/ggq8bc</a><br />
DOI: <a href="https://doi.org/10.1056/nejmp2005630">10.1056/nejmp2005630</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32227757">32227757</a></p>
</div>
<div id="ref-wPl93ATP">
<p>472. <strong>Newer Vaccine Technologies Deployed to Develop COVID-19 Shot</strong><br />
Abby Olena<br />
<em>The Scientist Magazine</em> (2020-02-21) <a href="https://www.the-scientist.com/news-opinion/newer-vaccine-technologies-deployed-to-develop-covid-19-shot-67152">https://www.the-scientist.com/news-opinion/newer-vaccine-technologies-deployed-to-develop-covid-19-shot-67152</a></p>
</div>
<div id="ref-plfPrQP7">
<p>473. <strong>A strategic approach to COVID-19 vaccine R&amp;D</strong><br />
Lawrence Corey, John R. Mascola, Anthony S. Fauci, Francis S. Collins<br />
<em>Science</em> (2020-05-29) <a href="https://doi.org/ggwfck">https://doi.org/ggwfck</a><br />
DOI: <a href="https://doi.org/10.1126/science.abc5312">10.1126/science.abc5312</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32393526">32393526</a></p>
</div>
<div id="ref-XnrBoKVk">
<p>474. <strong>WHO | DNA vaccines</strong><br />
WHO<br />
<a href="https://www.who.int/biologicals/areas/vaccines/dna/en/">https://www.who.int/biologicals/areas/vaccines/dna/en/</a></p>
</div>
<div id="ref-xuzLfS0y">
<p>475. <strong>Phase 1 Open-label Study to Evaluate the Safety, Tolerability and Immunogenicity of INO-4800, a Prophylactic Vaccine Against SARS-CoV-2, Administered Intradermally Followed by Electroporation in Healthy Volunteers</strong><br />
Inovio Pharmaceuticals<br />
<em>clinicaltrials.gov</em> (2020-07-02) <a href="https://clinicaltrials.gov/ct2/show/NCT04336410">https://clinicaltrials.gov/ct2/show/NCT04336410</a></p>
</div>
<div id="ref-H6tWVs5R">
<p>476. <strong>Electroporation delivery of DNA vaccines: prospects for success</strong><br />
Niranjan Y Sardesai, David B Weiner<br />
<em>Current Opinion in Immunology</em> (2011-06) <a href="https://doi.org/cq8b4p">https://doi.org/cq8b4p</a><br />
DOI: <a href="https://doi.org/10.1016/j.coi.2011.03.008">10.1016/j.coi.2011.03.008</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21530212">21530212</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3109217">PMC3109217</a></p>
</div>
<div id="ref-1Hsm2J1sc">
<p>477. <strong>Tolerability of intramuscular and intradermal delivery by CELLECTRA <sup>®</sup> adaptive constant current electroporation device in healthy volunteers</strong><br />
Malissa C Diehl, Jessica C Lee, Stephen E Daniels, Pablo Tebas, Amir S Khan, Mary Giffear, Niranjan Y Sardesai, Mark L Bagarazzi<br />
<em>Human Vaccines &amp; Immunotherapeutics</em> (2014-10-27) <a href="https://doi.org/ggrj7h">https://doi.org/ggrj7h</a><br />
DOI: <a href="https://doi.org/10.4161/hv.24702">10.4161/hv.24702</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24051434">24051434</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3906411">PMC3906411</a></p>
</div>
<div id="ref-K0Ltu31S">
<p>478. <strong>Advances in mRNA Vaccines for Infectious Diseases</strong><br />
Cuiling Zhang, Giulietta Maruggi, Hu Shan, Junwei Li<br />
<em>Frontiers in Immunology</em> (2019-03-27) <a href="https://doi.org/ggsnm7">https://doi.org/ggsnm7</a><br />
DOI: <a href="https://doi.org/10.3389/fimmu.2019.00594">10.3389/fimmu.2019.00594</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30972078">30972078</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6446947">PMC6446947</a></p>
</div>
<div id="ref-zNKWlCwE">
<p>479. <strong>mRNA vaccine delivery using lipid nanoparticles</strong><br />
Andreas M Reichmuth, Matthias A Oberli, Ana Jaklenec, Robert Langer, Daniel Blankschtein<br />
<em>Therapeutic Delivery</em> (2016-05) <a href="https://doi.org/f8xfzc">https://doi.org/f8xfzc</a><br />
DOI: <a href="https://doi.org/10.4155/tde-2016-0006">10.4155/tde-2016-0006</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27075952">27075952</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5439223">PMC5439223</a></p>
</div>
<div id="ref-pRoqjur8">
<p>480. <strong>Mechanism of action of mRNA-based vaccines</strong><br />
Carlo Iavarone, Derek T. O’hagan, Dong Yu, Nicolas F. Delahaye, Jeffrey B. Ulmer<br />
<em>Expert Review of Vaccines</em> (2017-07-28) <a href="https://doi.org/ggsnm6">https://doi.org/ggsnm6</a><br />
DOI: <a href="https://doi.org/10.1080/14760584.2017.1355245">10.1080/14760584.2017.1355245</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28701102">28701102</a></p>
</div>
<div id="ref-wYZ6qJMu">
<p>481. <strong>SARS-CoV-2 Vaccines: Status Report</strong><br />
Fatima Amanat, Florian Krammer<br />
<em>Immunity</em> (2020-04) <a href="https://doi.org/ggrdj4">https://doi.org/ggrdj4</a><br />
DOI: <a href="https://doi.org/10.1016/j.immuni.2020.03.007">10.1016/j.immuni.2020.03.007</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32259480">32259480</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7136867">PMC7136867</a></p>
</div>
<div id="ref-3EUiWZdN">
<p>482. <strong>Evaluation of the Kinetics of mRNA Expression After Two Doses of GSK Biologicals’ Candidate Tuberculosis (Tuberculosis) Vaccine GSK 692342 in Healthy Adults</strong><br />
GlaxoSmithKline<br />
<em>clinicaltrials.gov</em> (2019-03-20) <a href="https://clinicaltrials.gov/ct2/show/NCT01669096">https://clinicaltrials.gov/ct2/show/NCT01669096</a></p>
</div>
<div id="ref-6wZy2mn8">
<p>483. <strong>Nucleoside-modified mRNA immunization elicits influenza virus hemagglutinin stalk-specific antibodies</strong><br />
Norbert Pardi, Kaela Parkhouse, Ericka Kirkpatrick, Meagan McMahon, Seth J. Zost, Barbara L. Mui, Ying K. Tam, Katalin Karikó, Christopher J. Barbosa, Thomas D. Madden, … Drew Weissman<br />
<em>Nature Communications</em> (2018-08-22) <a href="https://doi.org/gd49qt">https://doi.org/gd49qt</a><br />
DOI: <a href="https://doi.org/10.1038/s41467-018-05482-0">10.1038/s41467-018-05482-0</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30135514">30135514</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6105651">PMC6105651</a></p>
</div>
<div id="ref-1EM5nGaYd">
<p>484. <strong>RNA vaccines: an introduction</strong><br />
PHG Foundation<br />
<a href="https://www.phgfoundation.org/briefing/rna-vaccines">https://www.phgfoundation.org/briefing/rna-vaccines</a></p>
</div>
<div id="ref-3LMMW7F0">
<p>485. <strong>T Follicular Helper Cell Differentiation, Function, and Roles in Disease</strong><br />
Shane Crotty<br />
<em>Immunity</em> (2014-10) <a href="https://doi.org/ggsp64">https://doi.org/ggsp64</a><br />
DOI: <a href="https://doi.org/10.1016/j.immuni.2014.10.004">10.1016/j.immuni.2014.10.004</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25367570">25367570</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4223692">PMC4223692</a></p>
</div>
<div id="ref-ENBWnhAh">
<p>486. <strong>mRNA vaccines — a new era in vaccinology</strong><br />
Norbert Pardi, Michael J. Hogan, Frederick W. Porter, Drew Weissman<br />
<em>Nature Reviews Drug Discovery</em> (2018-01-12) <a href="https://doi.org/gcsmgr">https://doi.org/gcsmgr</a><br />
DOI: <a href="https://doi.org/10.1038/nrd.2017.243">10.1038/nrd.2017.243</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29326426">29326426</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5906799">PMC5906799</a></p>
</div>
<div id="ref-Biu1CQeQ">
<p>487. <strong>Phase I, Open-Label, Dose-Ranging Study of the Safety and Immunogenicity of 2019-nCoV Vaccine (mRNA-1273) in Healthy Adults</strong><br />
National Institute of Allergy and Infectious Diseases (NIAID)<br />
<em>clinicaltrials.gov</em> (2020-07-16) <a href="https://clinicaltrials.gov/ct2/show/NCT04283461">https://clinicaltrials.gov/ct2/show/NCT04283461</a></p>
</div>
<div id="ref-ouOXy0wH">
<p>488. <strong>Progress and Prospects on Vaccine Development against SARS-CoV-2</strong><br />
Jinyong Zhang, Hao Zeng, Jiang Gu, Haibo Li, Lixin Zheng, Quanming Zou<br />
<em>Vaccines</em> (2020-03-29) <a href="https://doi.org/ggq726">https://doi.org/ggq726</a><br />
DOI: <a href="https://doi.org/10.3390/vaccines8020153">10.3390/vaccines8020153</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32235387">32235387</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7349596">PMC7349596</a></p>
</div>
<div id="ref-13bVbfc5h">
<p>489. <strong>Towards an understanding of the adjuvant action of aluminium</strong><br />
Philippa Marrack, Amy S. McKee, Michael W. Munks<br />
<em>Nature Reviews Immunology</em> (2009-04) <a href="https://doi.org/drcwvf">https://doi.org/drcwvf</a><br />
DOI: <a href="https://doi.org/10.1038/nri2510">10.1038/nri2510</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19247370">19247370</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3147301">PMC3147301</a></p>
</div>
<div id="ref-122h6fIxE">
<p>490. <strong>DAMP-Inducing Adjuvant and PAMP Adjuvants Parallelly Enhance Protective Type-2 and Type-1 Immune Responses to Influenza Split Vaccination</strong><br />
Tomoya Hayashi, Masatoshi Momota, Etsushi Kuroda, Takato Kusakabe, Shingo Kobari, Kotaro Makisaka, Yoshitaka Ohno, Yusuke Suzuki, Fumika Nakagawa, Michelle S. J. Lee, … Hidetoshi Arima<br />
<em>Frontiers in Immunology</em> (2018-11-20) <a href="https://doi.org/gfqq89">https://doi.org/gfqq89</a><br />
DOI: <a href="https://doi.org/10.3389/fimmu.2018.02619">10.3389/fimmu.2018.02619</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30515151">30515151</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6255964">PMC6255964</a></p>
</div>
<div id="ref-uO0uqhxc">
<p>491. <strong>Better Adjuvants for Better Vaccines: Progress in Adjuvant Delivery Systems, Modifications, and Adjuvant–Antigen Codelivery</strong><br />
Zhi-Biao Wang, Jing Xu<br />
<em>Vaccines</em> (2020-03-13) <a href="https://doi.org/gg35vj">https://doi.org/gg35vj</a><br />
DOI: <a href="https://doi.org/10.3390/vaccines8010128">10.3390/vaccines8010128</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32183209">32183209</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7157724">PMC7157724</a></p>
</div>
<div id="ref-Cd5uMaAr">
<p>492. <strong>Design and Synthesis of Hydroxyferroquine Derivatives with Antimalarial and Antiviral Activities</strong><br />
Christophe Biot, Wassim Daher, Natascha Chavain, Thierry Fandeur, Jamal Khalife, Daniel Dive, Erik De Clercq<br />
<em>Journal of Medicinal Chemistry</em> (2006-05) <a href="https://doi.org/db4n83">https://doi.org/db4n83</a><br />
DOI: <a href="https://doi.org/10.1021/jm0601856">10.1021/jm0601856</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16640347">16640347</a></p>
</div>
<div id="ref-4DAIEbQF">
<p>493. <strong>Sanofi and Regeneron begin global Kevzara® (sarilumab) clinical trial program in patients with severe COVID-19</strong><br />
Sanofi<br />
(2020-03-16) <a href="http://www.news.sanofi.us/2020-03-16-Sanofi-and-Regeneron-begin-global-Kevzara-R-sarilumab-clinical-trial-program-in-patients-with-severe-COVID-19">http://www.news.sanofi.us/2020-03-16-Sanofi-and-Regeneron-begin-global-Kevzara-R-sarilumab-clinical-trial-program-in-patients-with-severe-COVID-19</a></p>
</div>
<div id="ref-amy3rDsj">
<p>494. <strong>An Adaptive Phase 3, Randomized, Double-blind, Placebo-controlled Study Assessing Efficacy and Safety of Sarilumab for Hospitalized Patients With COVID19</strong><br />
Sanofi<br />
<em>clinicaltrials.gov</em> (2020-07-15) <a href="https://clinicaltrials.gov/ct2/show/NCT04327388">https://clinicaltrials.gov/ct2/show/NCT04327388</a></p>
</div>
<div id="ref-nbiwprc">
<p>495. <strong>Prevention of infection caused by immunosuppressive drugs in gastroenterology</strong><br />
Katarzyna Orlicka, Eleanor Barnes, Emma L. Culver<br />
<em>Therapeutic Advances in Chronic Disease</em> (2013-04-22) <a href="https://doi.org/ggrqd3">https://doi.org/ggrqd3</a><br />
DOI: <a href="https://doi.org/10.1177/2040622313485275">10.1177/2040622313485275</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23819020">23819020</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3697844">PMC3697844</a></p>
</div>
<div id="ref-suHq6Qnm">
<p>496. <strong>Immunosuppression for hyperinflammation in COVID-19: a double-edged sword?</strong><br />
Andrew I Ritchie, Aran Singanayagam<br />
<em>The Lancet</em> (2020-04) <a href="https://doi.org/ggq8hs">https://doi.org/ggq8hs</a><br />
DOI: <a href="https://doi.org/10.1016/s0140-6736(20)30691-7">10.1016/s0140-6736(20)30691-7</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32220278">32220278</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7138169">PMC7138169</a></p>
</div>
<div id="ref-LDQlolY3">
<p>497. <strong>TNF-α inhibition for potential therapeutic modulation of SARS coronavirus infection</strong><br />
Edward Tobinick<br />
<em>Current Medical Research and Opinion</em> (2008-09-22) <a href="https://doi.org/bq4cx2">https://doi.org/bq4cx2</a><br />
DOI: <a href="https://doi.org/10.1185/030079903125002757">10.1185/030079903125002757</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/14741070">14741070</a></p>
</div>
<div id="ref-WF7ymA4m">
<p>498. <strong>COVID-19: combining antiviral and anti-inflammatory treatments</strong><br />
Justin Stebbing, Anne Phelan, Ivan Griffin, Catherine Tucker, Olly Oechsle, Dan Smith, Peter Richardson<br />
<em>The Lancet Infectious Diseases</em> (2020-04) <a href="https://doi.org/dph5">https://doi.org/dph5</a><br />
DOI: <a href="https://doi.org/10.1016/s1473-3099(20)30132-8">10.1016/s1473-3099(20)30132-8</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32113509">32113509</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7158903">PMC7158903</a></p>
</div>
<div id="ref-uCns3aFw">
<p>499. <strong>Baricitinib as potential treatment for 2019-nCoV acute respiratory disease</strong><br />
Peter Richardson, Ivan Griffin, Catherine Tucker, Dan Smith, Olly Oechsle, Anne Phelan, Justin Stebbing<br />
<em>The Lancet</em> (2020-02) <a href="https://doi.org/ggnrsx">https://doi.org/ggnrsx</a><br />
DOI: <a href="https://doi.org/10.1016/s0140-6736(20)30304-4">10.1016/s0140-6736(20)30304-4</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32032529">32032529</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7137985">PMC7137985</a></p>
</div>
<div id="ref-jKRGxazA">
<p>500. <strong>Lilly Begins Clinical Testing of Therapies for COVID-19</strong><br />
Eli Lilly and Company<br />
<a href="https://investor.lilly.com/news-releases/news-release-details/lilly-begins-clinical-testing-therapies-covid-19">https://investor.lilly.com/news-releases/news-release-details/lilly-begins-clinical-testing-therapies-covid-19</a></p>
</div>
<div id="ref-JCMhG5r9">
<p>501. <strong>Baricitinib Combined With Antiviral Therapy in Symptomatic Patients Infected by COVID-19: an Open-label, Pilot Study</strong><br />
Fabrizio Cantini<br />
<em>clinicaltrials.gov</em> (2020-04-19) <a href="https://clinicaltrials.gov/ct2/show/NCT04320277">https://clinicaltrials.gov/ct2/show/NCT04320277</a></p>
</div>
<div id="ref-7fZ9eyMk">
<p>502. <strong>Table 1, Cost-Comparison Table for Biologic Disease-Modifying Drugs for Rheumatoid Arthritis</strong><br />
National Center for Biotechnology Information, U. S. National Library of Medicine 8600 Rockville Pike, Bethesda MD, 20894 Usa<br />
(2015-08) <a href="https://www.ncbi.nlm.nih.gov/books/NBK349513/table/T43/">https://www.ncbi.nlm.nih.gov/books/NBK349513/table/T43/</a></p>
</div>
<div id="ref-trWLoLFz">
<p>503. <strong>A Cost Comparison of Treatments of Moderate to Severe Psoriasis</strong><br />
Cheryl Hankin, Steven Feldman, Andy Szczotka, Randolph Stinger, Leslie Fish, David Hankin<br />
<em>Drug Benefit Trends</em> (2005-05) <a href="https://escholarship.umassmed.edu/meyers_pp/385">https://escholarship.umassmed.edu/meyers_pp/385</a></p>
</div>
<div id="ref-TZleuwHX">
<p>504. <strong>An orally bioavailable broad-spectrum antiviral inhibits SARS-CoV-2 in human airway epithelial cell cultures and multiple coronaviruses in mice</strong><br />
Timothy P. Sheahan, Amy C. Sims, Shuntai Zhou, Rachel L. Graham, Andrea J. Pruijssers, Maria L. Agostini, Sarah R. Leist, Alexandra Schäfer, Kenneth H. Dinnon, Laura J. Stevens, … Ralph S. Baric<br />
<em>Science Translational Medicine</em> (2020-04-29) <a href="https://doi.org/ggrqd2">https://doi.org/ggrqd2</a><br />
DOI: <a href="https://doi.org/10.1126/scitranslmed.abb5883">10.1126/scitranslmed.abb5883</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32253226">32253226</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7164393">PMC7164393</a></p>
</div>
<div id="ref-1FTnvyZlf">
<p>505. <strong>Antiviral Monoclonal Antibodies: Can They Be More Than Simple Neutralizing Agents?</strong><br />
Mireia Pelegrin, Mar Naranjo-Gomez, Marc Piechaczyk<br />
<em>Trends in Microbiology</em> (2015-10) <a href="https://doi.org/f7vzrf">https://doi.org/f7vzrf</a><br />
DOI: <a href="https://doi.org/10.1016/j.tim.2015.07.005">10.1016/j.tim.2015.07.005</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26433697">26433697</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7127033">PMC7127033</a></p>
</div>
<div id="ref-o6BQnEt7">
<p>506. <strong>Molecular biology of the cell</strong><br />
Bruce Alberts (editor)<br />
<em>Garland Science</em> (2002)<br />
ISBN: <a href="https://worldcat.org/isbn/9780815332183">9780815332183</a></p>
</div>
<div id="ref-Ull2rQ5L">
<p>507. <strong>Intranasal Treatment with Poly(I{middle dot}C) Protects Aged Mice from Lethal Respiratory Virus Infections</strong><br />
J. Zhao, C. Wohlford-Lenane, J. Zhao, E. Fleming, T. E. Lane, P. B. McCray, S. Perlman<br />
<em>Journal of Virology</em> (2012-08-22) <a href="https://doi.org/f4bzfp">https://doi.org/f4bzfp</a><br />
DOI: <a href="https://doi.org/10.1128/jvi.01410-12">10.1128/jvi.01410-12</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22915814">22915814</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3486278">PMC3486278</a></p>
</div>
<div id="ref-Vu1VILWK">
<p>508. <strong>Trained Immunity: a Tool for Reducing Susceptibility to and the Severity of SARS-CoV-2 Infection</strong><br />
Mihai G. Netea, Evangelos J. Giamarellos-Bourboulis, Jorge Domínguez-Andrés, Nigel Curtis, Reinout van Crevel, Frank L. van de Veerdonk, Marc Bonten<br />
<em>Cell</em> (2020-05) <a href="https://doi.org/gg2584">https://doi.org/gg2584</a><br />
DOI: <a href="https://doi.org/10.1016/j.cell.2020.04.042">10.1016/j.cell.2020.04.042</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32437659">32437659</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7196902">PMC7196902</a></p>
</div>
<div id="ref-103fS7Kz2">
<p>509. <strong>Defining trained immunity and its role in health and disease</strong><br />
Mihai G. Netea, Jorge Domínguez-Andrés, Luis B. Barreiro, Triantafyllos Chavakis, Maziar Divangahi, Elaine Fuchs, Leo A. B. Joosten, Jos W. M. van der Meer, Musa M. Mhlanga, Willem J. M. Mulder, … Eicke Latz<br />
<em>Nature Reviews Immunology</em> (2020-03-04) <a href="https://doi.org/gg28pr">https://doi.org/gg28pr</a><br />
DOI: <a href="https://doi.org/10.1038/s41577-020-0285-6">10.1038/s41577-020-0285-6</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32132681">32132681</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7186935">PMC7186935</a></p>
</div>
<div id="ref-9m3rP633">
<p>510. <strong>BCG Vaccination to Reduce the Impact of COVID-19 in Healthcare Workers Following Coronavirus Exposure (BRACE) Trial</strong><br />
Murdoch Childrens Research Institute<br />
<em>clinicaltrials.gov</em> (2020-07-06) <a href="https://clinicaltrials.gov/ct2/show/NCT04327206">https://clinicaltrials.gov/ct2/show/NCT04327206</a></p>
</div>
<div id="ref-y9IYdfM3">
<p>511. <strong>Reducing Health Care Workers Absenteeism in COVID-19 Pandemic by Enhanced Trained Immune Responses Through Bacillus Calmette-Guérin Vaccination, a Randomized Controlled Trial.</strong><br />
M. J. M. Bonten<br />
<em>clinicaltrials.gov</em> (2020-04-27) <a href="https://clinicaltrials.gov/ct2/show/NCT04328441">https://clinicaltrials.gov/ct2/show/NCT04328441</a></p>
</div>
<div id="ref-xdqxBruc">
<p>512. <strong>Bacillus Calmette-Guerin Vaccination as Defense Against SARS-CoV-2: A Randomized Controlled Trial to Protect Health Care Workers by Enhanced Trained Immune Responses</strong><br />
Texas A&amp;M University<br />
<em>clinicaltrials.gov</em> (2020-05-24) <a href="https://clinicaltrials.gov/ct2/show/NCT04348370">https://clinicaltrials.gov/ct2/show/NCT04348370</a></p>
</div>
<div id="ref-962rELVS">
<p>513. <strong>Application of BCG Vaccine for Immune-prophylaxis Among Egyptian Healthcare Workers During the Pandemic of COVID-19</strong><br />
Adel Khattab<br />
<em>clinicaltrials.gov</em> (2020-04-17) <a href="https://clinicaltrials.gov/ct2/show/NCT04350931">https://clinicaltrials.gov/ct2/show/NCT04350931</a></p>
</div>
<div id="ref-EuwTWcPi">
<p>514. <strong>Performance Evaluation of BCG Vaccination in Healthcare Personnel to Reduce the Severity of SARS-COV-2 Infection in Medellín, Colombia, 2020</strong><br />
Universidad de Antioquia<br />
<em>clinicaltrials.gov</em> (2020-04-24) <a href="https://clinicaltrials.gov/ct2/show/NCT04362124">https://clinicaltrials.gov/ct2/show/NCT04362124</a></p>
</div>
<div id="ref-dQtUeruv">
<p>515. <strong>COVID-19: BCG As Therapeutic Vaccine, Transmission Limitation, and Immunoglobulin Enhancement</strong><br />
Leonardo Oliveira Reis<br />
<em>clinicaltrials.gov</em> (2020-04-28) <a href="https://clinicaltrials.gov/ct2/show/NCT04369794">https://clinicaltrials.gov/ct2/show/NCT04369794</a></p>
</div>
<div id="ref-DjXsPR8O">
<p>516. <strong>Using BCG Vaccine to Enhance Non-specific Protection of Health Care Workers During the COVID-19 Pandemic. A Randomized Controlled Multi-center Trial</strong><br />
Bandim Health Project<br />
<em>clinicaltrials.gov</em> (2020-05-03) <a href="https://clinicaltrials.gov/ct2/show/NCT04373291">https://clinicaltrials.gov/ct2/show/NCT04373291</a></p>
</div>
<div id="ref-10OE6y3Pv">
<p>517. <strong>Reducing Morbidity and Mortality in Health Care Workers Exposed to SARS-CoV-2 by Enhancing Non-specific Immune Responses Through Bacillus Calmette-Guérin Vaccination, a Randomized Controlled Trial</strong><br />
TASK Applied Science<br />
<em>clinicaltrials.gov</em> (2020-05-06) <a href="https://clinicaltrials.gov/ct2/show/NCT04379336">https://clinicaltrials.gov/ct2/show/NCT04379336</a></p>
</div>
<div id="ref-86OjIybR">
<p>518. <strong>Randomized Controlled Trial Evaluating the Efficacy of Vaccination With Bacillus Calmette and Guérin (BCG) in the Prevention of COVID-19 Via the Strengthening of Innate Immunity in Health Care Workers</strong><br />
Assistance Publique - Hôpitaux de Paris<br />
<em>clinicaltrials.gov</em> (2020-05-09) <a href="https://clinicaltrials.gov/ct2/show/NCT04384549">https://clinicaltrials.gov/ct2/show/NCT04384549</a></p>
</div>
<div id="ref-ITO15LIz">
<p>519. <strong>A Phase III, Double-blind, Randomized, Placebo-controlled Multicentre Clinical Trial to Assess the Efficacy and Safety of VPM1002 in Reducing Healthcare Professionals’ Absenteeism in the SARS-CoV-2 Pandemic by Modulating the Immune System</strong><br />
Vakzine Projekt Management GmbH<br />
<em>clinicaltrials.gov</em> (2020-06-12) <a href="https://clinicaltrials.gov/ct2/show/NCT04387409">https://clinicaltrials.gov/ct2/show/NCT04387409</a></p>
</div>
<div id="ref-VkZGZxLn">
<p>520. <strong>A Randomized Clinical Trial for Enhanced Trained Immune Responses Through Bacillus Calmette-Guérin Vaccination to Prevent Infections by COVID-19: The ACTIVATE II Trial</strong><br />
Hellenic Institute for the Study of Sepsis<br />
<em>clinicaltrials.gov</em> (2020-07-10) <a href="https://clinicaltrials.gov/ct2/show/NCT04414267">https://clinicaltrials.gov/ct2/show/NCT04414267</a></p>
</div>
<div id="ref-13JVjMfQI">
<p>521. <strong>Reducing Hospital Admission of Elderly in SARS-CoV-2 Pandemic Via the Induction of Trained Immunity by Bacillus Calmette-Guérin Vaccination, a Randomized Controlled Trial</strong><br />
Radboud University<br />
<em>clinicaltrials.gov</em> (2020-06-03) <a href="https://clinicaltrials.gov/ct2/show/NCT04417335">https://clinicaltrials.gov/ct2/show/NCT04417335</a></p>
</div>
<div id="ref-nk2MVsld">
<p>522. <strong>A Phase III, Randomized, Double-blind, Placebo-controlled, Multicentre, Clinical Trial to Assess the Efficacy and Safety of VPM1002 in Reducing Hospital Admissions and/or Severe Respiratory Infectious Diseases in Elderly in the SARS-CoV-2 Pandemic by Modulating the Immune System</strong><br />
Vakzine Projekt Management GmbH<br />
<em>clinicaltrials.gov</em> (2020-07-29) <a href="https://clinicaltrials.gov/ct2/show/NCT04435379">https://clinicaltrials.gov/ct2/show/NCT04435379</a></p>
</div>
<div id="ref-1E2t9tr8h">
<p>523. <strong>A Randomized, Double-blind, Placebo-controlled Phase 3 Study: Efficacy and Safety of VPM1002 in Reducing SARS-CoV-2 Infection Rate and COVID-19 Severity</strong><br />
University Health Network, Toronto<br />
<em>clinicaltrials.gov</em> (2020-06-18) <a href="https://clinicaltrials.gov/ct2/show/NCT04439045">https://clinicaltrials.gov/ct2/show/NCT04439045</a></p>
</div>
<div id="ref-i5k18bpX">
<p>524. <strong>Complex Immune Dysregulation in COVID-19 Patients with Severe Respiratory Failure</strong><br />
Evangelos J. Giamarellos-Bourboulis, Mihai G. Netea, Nikoletta Rovina, Karolina Akinosoglou, Anastasia Antoniadou, Nikolaos Antonakos, Georgia Damoraki, Theologia Gkavogianni, Maria-Evangelia Adami, Paraskevi Katsaounou, … Antonia Koutsoukou<br />
<em>Cell Host &amp; Microbe</em> (2020-06) <a href="https://doi.org/ggthxs">https://doi.org/ggthxs</a><br />
DOI: <a href="https://doi.org/10.1016/j.chom.2020.04.009">10.1016/j.chom.2020.04.009</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32320677">32320677</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7172841">PMC7172841</a></p>
</div>
<div id="ref-1GnFL9zeN">
<p>525. <strong>The Innate Immune System: Fighting on the Front Lines or Fanning the Flames of COVID-19?</strong><br />
Julia L. McKechnie, Catherine A. Blish<br />
<em>Cell Host &amp; Microbe</em> (2020-06) <a href="https://doi.org/gg28pq">https://doi.org/gg28pq</a><br />
DOI: <a href="https://doi.org/10.1016/j.chom.2020.05.009">10.1016/j.chom.2020.05.009</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32464098">32464098</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7237895">PMC7237895</a></p>
</div>
<div id="ref-v08gazBj">
<p>526. <strong>Structure of M <sup>pro</sup> from COVID-19 virus and discovery of its inhibitors</strong><br />
Zhenming Jin, Xiaoyu Du, Yechun Xu, Yongqiang Deng, Meiqin Liu, Yao Zhao, Bing Zhang, Xiaofeng Li, Leike Zhang, Chao Peng, … Haitao Yang<br />
<em>bioRxiv</em> (2020-03-29) <a href="https://doi.org/ggqs5x">https://doi.org/ggqs5x</a><br />
DOI: <a href="https://doi.org/10.1101/2020.02.26.964882">10.1101/2020.02.26.964882</a></p>
</div>
<div id="ref-AJLaaguT">
<p>527. <strong>Main protease structure and XChem fragment screen</strong><br />
Diamond<br />
(2020-05-05) <a href="https://www.diamond.ac.uk/covid-19/for-scientists/Main-protease-structure-and-XChem.html">https://www.diamond.ac.uk/covid-19/for-scientists/Main-protease-structure-and-XChem.html</a></p>
</div>
<div id="ref-17OQtAY4l">
<p>528. <strong>Using the MAARIE Framework To Read the Research Literature</strong><br />
M. Corcoran<br />
<em>American Journal of Occupational Therapy</em> (2006-07-01) <a href="https://doi.org/bqh97x">https://doi.org/bqh97x</a><br />
DOI: <a href="https://doi.org/10.5014/ajot.60.4.367">10.5014/ajot.60.4.367</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16915865">16915865</a></p>
</div>
<div id="ref-YuJbg3zO">
<p>529. <strong>Open collaborative writing with Manubot</strong><br />
Daniel S. Himmelstein, Vincent Rubinetti, David R. Slochower, Dongbo Hu, Venkat S. Malladi, Casey S. Greene, Anthony Gitter<br />
<em>PLOS Computational Biology</em> (2019-06-24) <a href="https://doi.org/c7np">https://doi.org/c7np</a><br />
DOI: <a href="https://doi.org/10.1371/journal.pcbi.1007128">10.1371/journal.pcbi.1007128</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31233491">31233491</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6611653">PMC6611653</a></p>
</div>
<div id="ref-1HLSeqnY8">
<p>530. <strong>Potent binding of 2019 novel coronavirus spike protein by a SARS coronavirus-specific human monoclonal antibody</strong><br />
Xiaolong Tian, Cheng Li, Ailing Huang, Shuai Xia, Sicong Lu, Zhengli Shi, Lu Lu, Shibo Jiang, Zhenlin Yang, Yanling Wu, Tianlei Ying<br />
<em>bioRxiv</em> (2020-01-28) <a href="https://doi.org/ggjqfd">https://doi.org/ggjqfd</a><br />
DOI: <a href="https://doi.org/10.1101/2020.01.28.923011">10.1101/2020.01.28.923011</a></p>
</div>
<div id="ref-lOQPScZt">
<p>531. <strong>Integrative Bioinformatics Analysis Provides Insight into the Molecular Mechanisms of 2019-nCoV</strong><br />
Xiang He, Lei Zhang, Qin Ran, Anying Xiong, Junyi Wang, Dehong Wu, Feng Chen, Guoping Li<br />
<em>medRxiv</em> (2020-02-05) <a href="https://doi.org/ggrbd8">https://doi.org/ggrbd8</a><br />
DOI: <a href="https://doi.org/10.1101/2020.02.03.20020206">10.1101/2020.02.03.20020206</a></p>
</div>
<div id="ref-rL9IRhVy">
<p>532. <strong>Diarrhea may be underestimated: a missing link in 2019 novel coronavirus</strong><br />
Weicheng Liang, Zhijie Feng, Shitao Rao, Cuicui Xiao, Zexiao Lin, Qi Zhang, Wei Qi<br />
<em>medRxiv</em> (2020-02-17) <a href="https://doi.org/ggrbdw">https://doi.org/ggrbdw</a><br />
DOI: <a href="https://doi.org/10.1101/2020.02.03.20020289">10.1101/2020.02.03.20020289</a></p>
</div>
<div id="ref-BHQvMYPG">
<p>533. <strong>Specific ACE2 Expression in Cholangiocytes May Cause Liver Damage After 2019-nCoV Infection</strong><br />
Xiaoqiang Chai, Longfei Hu, Yan Zhang, Weiyu Han, Zhou Lu, Aiwu Ke, Jian Zhou, Guoming Shi, Nan Fang, Jia Fan, … Fei Lan<br />
<em>bioRxiv</em> (2020-02-04) <a href="https://doi.org/ggq626">https://doi.org/ggq626</a><br />
DOI: <a href="https://doi.org/10.1101/2020.02.03.931766">10.1101/2020.02.03.931766</a></p>
</div>
<div id="ref-OuDMtCRl">
<p>534. <strong>Recapitulation of SARS-CoV-2 Infection and Cholangiocyte Damage with Human Liver Organoids</strong><br />
Bing Zhao, Chao Ni, Ran Gao, Yuyan Wang, Li Yang, Jinsong Wei, Ting Lv, Jianqing Liang, Qisheng Zhang, Wei Xu, … Xinhua Lin<br />
<em>bioRxiv</em> (2020-03-17) <a href="https://doi.org/ggq648">https://doi.org/ggq648</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.16.990317">10.1101/2020.03.16.990317</a></p>
</div>
<div id="ref-ViWCb8MJ">
<p>535. <strong>ACE2 expression by colonic epithelial cells is associated with viral infection, immunity and energy metabolism</strong><br />
Jun Wang, Shanmeizi Zhao, Ming Liu, Zhiyao Zhao, Yiping Xu, Ping Wang, Meng Lin, Yanhui Xu, Bing Huang, Xiaoyu Zuo, … Yuxia Zhang<br />
<em>medRxiv</em> (2020-02-07) <a href="https://doi.org/ggrfbx">https://doi.org/ggrfbx</a><br />
DOI: <a href="https://doi.org/10.1101/2020.02.05.20020545">10.1101/2020.02.05.20020545</a></p>
</div>
<div id="ref-MgLBA5m">
<p>536. <strong>The Pathogenicity of SARS-CoV-2 in hACE2 Transgenic Mice</strong><br />
Linlin Bao, Wei Deng, Baoying Huang, Hong Gao, Jiangning Liu, Lili Ren, Qiang Wei, Pin Yu, Yanfeng Xu, Feifei Qi, … Chuan Qin<br />
<em>bioRxiv</em> (2020-02-28) <a href="https://doi.org/dph2">https://doi.org/dph2</a><br />
DOI: <a href="https://doi.org/10.1101/2020.02.07.939389">10.1101/2020.02.07.939389</a></p>
</div>
<div id="ref-1Acv5UMSt">
<p>537. <strong>Caution on Kidney Dysfunctions of COVID-19 Patients</strong><br />
Zhen Li, Ming Wu, Jiwei Yao, Jie Guo, Xiang Liao, Siji Song, Jiali Li, Guangjie Duan, Yuanxiu Zhou, Xiaojun Wu, … Anti-2019-nCoV Volunteers<br />
<em>medRxiv</em> (2020-03-27) <a href="https://doi.org/ggq627">https://doi.org/ggq627</a><br />
DOI: <a href="https://doi.org/10.1101/2020.02.08.20021212">10.1101/2020.02.08.20021212</a></p>
</div>
<div id="ref-DMaiZBE1">
<p>538. <strong>Acute renal impairment in coronavirus-associated severe acute respiratory syndrome</strong><br />
Kwok Hong Chu, Wai Kay Tsang, Colin S. Tang, Man Fai Lam, Fernand M. Lai, Ka Fai To, Ka Shun Fung, Hon Lok Tang, Wing Wa Yan, Hilda W. H. Chan, … Kar Neng Lai<br />
<em>Kidney International</em> (2005-02) <a href="https://doi.org/b7tgtx">https://doi.org/b7tgtx</a><br />
DOI: <a href="https://doi.org/10.1111/j.1523-1755.2005.67130.x">10.1111/j.1523-1755.2005.67130.x</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15673319">15673319</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7112337">PMC7112337</a></p>
</div>
<div id="ref-16Xj27QYj">
<p>539. <strong>Single-cell Analysis of ACE2 Expression in Human Kidneys and Bladders Reveals a Potential Route of 2019-nCoV Infection</strong><br />
Wei Lin, Longfei Hu, Yan Zhang, Joshua D. Ooi, Ting Meng, Peng Jin, Xiang Ding, Longkai Peng, Lei Song, Zhou Xiao, … Yong Zhong<br />
<em>bioRxiv</em> (2020-02-18) <a href="https://doi.org/ggq629">https://doi.org/ggq629</a><br />
DOI: <a href="https://doi.org/10.1101/2020.02.08.939892">10.1101/2020.02.08.939892</a></p>
</div>
<div id="ref-QhE5mVCc">
<p>540. <strong>The immune vulnerability landscape of the 2019 Novel Coronavirus, SARS-CoV-2</strong><br />
James Zhu, Jiwoong Kim, Xue Xiao, Yunguan Wang, Danni Luo, Ran Chen, Lin Xu, He Zhang, Guanghua Xiao, John W. Schoggins, … Yang Xie<br />
<em>bioRxiv</em> (2020-03-23) <a href="https://doi.org/ggq628">https://doi.org/ggq628</a><br />
DOI: <a href="https://doi.org/10.1101/2020.02.08.939553">10.1101/2020.02.08.939553</a></p>
</div>
<div id="ref-1FE1YGsaV">
<p>541. <strong>Tissue distribution of ACE2 protein, the functional receptor for SARS coronavirus. A first step in understanding SARS pathogenesis</strong><br />
I Hamming, W Timens, MLC Bulthuis, AT Lely, GJ Navis, H van Goor<br />
<em>The Journal of Pathology</em> (2004-06) <a href="https://doi.org/bhpzc3">https://doi.org/bhpzc3</a><br />
DOI: <a href="https://doi.org/10.1002/path.1570">10.1002/path.1570</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15141377">15141377</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7167720">PMC7167720</a></p>
</div>
<div id="ref-iObQU2el">
<p>542. <strong>Clinical Course and Outcomes of Critically Ill Patients With Middle East Respiratory Syndrome Coronavirus Infection</strong><br />
Yaseen M. Arabi, Ahmed A. Arifi, Hanan H. Balkhy, Hani Najm, Abdulaziz S. Aldawood, Alaa Ghabashi, Hassan Hawa, Adel Alothman, Abdulaziz Khaldi, Basel Al Raiy<br />
<em>Annals of Internal Medicine</em> (2014-03-18) <a href="https://doi.org/ggptxw">https://doi.org/ggptxw</a><br />
DOI: <a href="https://doi.org/10.7326/m13-2486">10.7326/m13-2486</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24474051">24474051</a></p>
</div>
<div id="ref-gK5e7Yhl">
<p>543. <strong>Neutrophil-to-Lymphocyte Ratio Predicts Severe Illness Patients with 2019 Novel Coronavirus in the Early Stage</strong><br />
Jingyuan Liu, Yao Liu, Pan Xiang, Lin Pu, Haofeng Xiong, Chuansheng Li, Ming Zhang, Jianbo Tan, Yanli Xu, Rui Song, … Xianbo Wang<br />
<em>medRxiv</em> (2020-02-12) <a href="https://doi.org/ggrbdx">https://doi.org/ggrbdx</a><br />
DOI: <a href="https://doi.org/10.1101/2020.02.10.20021584">10.1101/2020.02.10.20021584</a></p>
</div>
<div id="ref-6Y81C4P3">
<p>544. <strong>Dysregulation of immune response in patients with COVID-19 in Wuhan, China</strong><br />
Chuan Qin, Luoqi Zhou, Ziwei Hu, Shuoqi Zhang, Sheng Yang, Yu Tao, Cuihong Xie, Ke Ma, Ke Shang, Wei Wang, Dai-Shi Tian<br />
<em>Clinical Infectious Diseases</em> (2020-03-12) <a href="https://doi.org/ggpxcf">https://doi.org/ggpxcf</a><br />
DOI: <a href="https://doi.org/10.1093/cid/ciaa248">10.1093/cid/ciaa248</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32161940">32161940</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7108125">PMC7108125</a></p>
</div>
<div id="ref-P4assQHj">
<p>545. <strong>Characteristics of lymphocyte subsets and cytokines in peripheral blood of 123 hospitalized patients with 2019 novel coronavirus pneumonia (NCP)</strong><br />
Suxin Wan, Qingjie Yi, Shibing Fan, Jinglong Lv, Xianxiang Zhang, Lian Guo, Chunhui Lang, Qing Xiao, Kaihu Xiao, Zhengjun Yi, … Yongping Chen<br />
<em>medRxiv</em> (2020-02-12) <a href="https://doi.org/ggq63b">https://doi.org/ggq63b</a><br />
DOI: <a href="https://doi.org/10.1101/2020.02.10.20021832">10.1101/2020.02.10.20021832</a></p>
</div>
<div id="ref-zskgSnle">
<p>546. <strong>Longitudinal Characteristics of Lymphocyte Responses and Cytokine Profiles in the Peripheral Blood of SARS-CoV-2 Infected Patients</strong><br />
Jing Liu, Sumeng Li, Jia Liu, Boyun Liang, Xiaobei Wang, Wei Li, Hua Wang, Qiaoxia Tong, Jianhua Yi, Lei Zhao, … Xin Zheng<br />
<em>SSRN Electronic Journal</em> (2020) <a href="https://doi.org/ggq655">https://doi.org/ggq655</a><br />
DOI: <a href="https://doi.org/10.2139/ssrn.3539682">10.2139/ssrn.3539682</a></p>
</div>
<div id="ref-TXlNzmXd">
<p>547. <strong>Epidemiological and Clinical Characteristics of 17 Hospitalized Patients with 2019 Novel Coronavirus Infections Outside Wuhan, China</strong><br />
Jie Li, Shilin Li, Yurui Cai, Qin Liu, Xue Li, Zhaoping Zeng, Yanpeng Chu, Fangcheng Zhu, Fanxin Zeng<br />
<em>medRxiv</em> (2020-02-12) <a href="https://doi.org/ggq63c">https://doi.org/ggq63c</a><br />
DOI: <a href="https://doi.org/10.1101/2020.02.11.20022053">10.1101/2020.02.11.20022053</a></p>
</div>
<div id="ref-15K5Ovntj">
<p>548. <strong>ACE2 Expression in Kidney and Testis May Cause Kidney and Testis Damage After 2019-nCoV Infection</strong><br />
Caibin Fan, Kai Li, Yanhong Ding, Wei Lu Lu, Jianqing Wang<br />
<em>medRxiv</em> (2020-02-13) <a href="https://doi.org/ggq63d">https://doi.org/ggq63d</a><br />
DOI: <a href="https://doi.org/10.1101/2020.02.12.20022418">10.1101/2020.02.12.20022418</a></p>
</div>
<div id="ref-Bgo7Cmj3">
<p>549. <strong>Aberrant pathogenic GM-CSF <sup>+</sup> T cells and inflammatory CD14 <sup>+</sup> CD16 <sup>+</sup> monocytes in severe pulmonary syndrome patients of a new coronavirus</strong><br />
Yonggang Zhou, Binqing Fu, Xiaohu Zheng, Dongsheng Wang, Changcheng Zhao, Yingjie qi, Rui Sun, Zhigang Tian, Xiaoling Xu, Haiming Wei<br />
<em>bioRxiv</em> (2020-02-20) <a href="https://doi.org/ggq63f">https://doi.org/ggq63f</a><br />
DOI: <a href="https://doi.org/10.1101/2020.02.12.945576">10.1101/2020.02.12.945576</a></p>
</div>
<div id="ref-4Z1pEioX">
<p>550. <strong>Clinical Characteristics of 2019 Novel Infected Coronavirus Pneumonia：A Systemic Review and Meta-analysis</strong><br />
Kai Qian, Yi Deng, Yonghang Tai, Jun Peng, Hao Peng, Lihong Jiang<br />
<em>medRxiv</em> (2020-02-17) <a href="https://doi.org/ggrgbq">https://doi.org/ggrgbq</a><br />
DOI: <a href="https://doi.org/10.1101/2020.02.14.20021535">10.1101/2020.02.14.20021535</a></p>
</div>
<div id="ref-zTGzuiTI">
<p>551. <strong>Longitudinal characteristics of lymphocyte responses and cytokine profiles in the peripheral blood of SARS-CoV-2 infected patients</strong><br />
Jing Liu, Sumeng Li, Jia Liu, Boyun Liang, Xiaobei Wang, Hua Wang, Wei Li, Qiaoxia Tong, Jianhua Yi, Lei Zhao, … Xin Zheng<br />
<em>medRxiv</em> (2020-02-22) <a href="https://doi.org/ggq63g">https://doi.org/ggq63g</a><br />
DOI: <a href="https://doi.org/10.1101/2020.02.16.20023671">10.1101/2020.02.16.20023671</a></p>
</div>
<div id="ref-1EIEx8Qe4">
<p>552. <strong>Clinical and immunologic features in severe and moderate forms of Coronavirus Disease 2019</strong><br />
Guang Chen, Di Wu, Wei Guo, Yong Cao, Da Huang, Hongwu Wang, Tao Wang, Xiaoyun Zhang, Huilong Chen, Haijing Yu, … Qin Ning<br />
<em>medRxiv</em> (2020-02-19) <a href="https://doi.org/ggq63h">https://doi.org/ggq63h</a><br />
DOI: <a href="https://doi.org/10.1101/2020.02.16.20023903">10.1101/2020.02.16.20023903</a></p>
</div>
<div id="ref-41WAtZmB">
<p>553. <strong>SARS-CoV-2 and SARS-CoV Spike-RBD Structure and Receptor Binding Comparison and Potential Implications on Neutralizing Antibody and Vaccine Development</strong><br />
Chunyun Sun, Long Chen, Ji Yang, Chunxia Luo, Yanjing Zhang, Jing Li, Jiahui Yang, Jie Zhang, Liangzhi Xie<br />
<em>bioRxiv</em> (2020-02-20) <a href="https://doi.org/ggq63j">https://doi.org/ggq63j</a><br />
DOI: <a href="https://doi.org/10.1101/2020.02.16.951723">10.1101/2020.02.16.951723</a></p>
</div>
<div id="ref-NPxs7bbq">
<p>554. <strong>Protection of Rhesus Macaque from SARS-Coronavirus challenge by recombinant adenovirus vaccine</strong><br />
Yiyou Chen, Qiang Wei, Ruobing Li, Hong Gao, Hua Zhu, Wei Deng, Linlin Bao, Wei Tong, Zhe Cong, Hong Jiang, Chuan Qin<br />
<em>bioRxiv</em> (2020-02-21) <a href="https://doi.org/ggq63k">https://doi.org/ggq63k</a><br />
DOI: <a href="https://doi.org/10.1101/2020.02.17.951939">10.1101/2020.02.17.951939</a></p>
</div>
<div id="ref-f50ahNxF">
<p>555. <strong>Reduction and Functional Exhaustion of T Cells in Patients with Coronavirus Disease 2019 (COVID-19)</strong><br />
Bo Diao, Chenhui Wang, Yingjun Tan, Xiewan Chen, Ying Liu, Lifeng Ning, Li Chen, Min Li, Yueping Liu, Gang Wang, … Yongwen Chen<br />
<em>medRxiv</em> (2020-02-20) <a href="https://doi.org/ggq63m">https://doi.org/ggq63m</a><br />
DOI: <a href="https://doi.org/10.1101/2020.02.18.20024364">10.1101/2020.02.18.20024364</a></p>
</div>
<div id="ref-EwNi3WJq">
<p>556. <strong>Clinical characteristics of 25 death cases infected with COVID-19 pneumonia: a retrospective review of medical records in a single medical center, Wuhan, China</strong><br />
Xun Li, Luwen Wang, Shaonan Yan, Fan Yang, Longkui Xiang, Jiling Zhu, Bo Shen, Zuojiong Gong<br />
<em>medRxiv</em> (2020-02-25) <a href="https://doi.org/ggq63n">https://doi.org/ggq63n</a><br />
DOI: <a href="https://doi.org/10.1101/2020.02.19.20025239">10.1101/2020.02.19.20025239</a></p>
</div>
<div id="ref-KG5Lg65z">
<p>557. <strong>SARS-CoV-2 infection does not significantly cause acute renal injury: an analysis of 116 hospitalized patients with COVID-19 in a single hospital, Wuhan, China</strong><br />
Lunwen Wang, Xun Li, Hui Chen, Shaonan Yan, Yan Li, Dong Li, Zuojiong Gong<br />
<em>medRxiv</em> (2020-02-23) <a href="https://doi.org/ggq63p">https://doi.org/ggq63p</a><br />
DOI: <a href="https://doi.org/10.1101/2020.02.19.20025288">10.1101/2020.02.19.20025288</a></p>
</div>
<div id="ref-VRoblZFJ">
<p>558. <strong>Clinical Characteristics of 138 Hospitalized Patients With 2019 Novel Coronavirus–Infected Pneumonia in Wuhan, China</strong><br />
Dawei Wang, Bo Hu, Chang Hu, Fangfang Zhu, Xing Liu, Jing Zhang, Binbin Wang, Hui Xiang, Zhenshun Cheng, Yong Xiong, … Zhiyong Peng<br />
<em>JAMA</em> (2020-02-07) <a href="https://doi.org/ggkh48">https://doi.org/ggkh48</a><br />
DOI: <a href="https://doi.org/10.1001/jama.2020.1585">10.1001/jama.2020.1585</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32031570">32031570</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7042881">PMC7042881</a></p>
</div>
<div id="ref-bgrzawWp">
<p>559. <strong>Clinical characteristics of 2019 novel coronavirus infection in China</strong><br />
Wei-jie Guan, Zheng-yi Ni, Yu Hu, Wen-hua Liang, Chun-quan Ou, Jian-xing He, Lei Liu, Hong Shan, Chun-liang Lei, David SC Hui, … Nan-shan Zhong<br />
<em>medRxiv</em> (2020-02-09) <a href="https://doi.org/ggkj9s">https://doi.org/ggkj9s</a><br />
DOI: <a href="https://doi.org/10.1101/2020.02.06.20020974">10.1101/2020.02.06.20020974</a></p>
</div>
<div id="ref-hdSafq1o">
<p>560. <strong>Potential T-cell and B-cell Epitopes of 2019-nCoV</strong><br />
Ethan Fast, Russ B. Altman, Binbin Chen<br />
<em>bioRxiv</em> (2020-03-18) <a href="https://doi.org/ggq63q">https://doi.org/ggq63q</a><br />
DOI: <a href="https://doi.org/10.1101/2020.02.19.955484">10.1101/2020.02.19.955484</a></p>
</div>
<div id="ref-8xnxgpag">
<p>561. <strong>Structure, function and antigenicity of the SARS-CoV-2 spike glycoprotein</strong><br />
Alexandra C. Walls, Young-Jun Park, M. Alexandra Tortorici, Abigail Wall, Andrew T. McGuire, David Veesler<br />
<em>bioRxiv</em> (2020-02-20) <a href="https://doi.org/ggrgbr">https://doi.org/ggrgbr</a><br />
DOI: <a href="https://doi.org/10.1101/2020.02.19.956581">10.1101/2020.02.19.956581</a></p>
</div>
<div id="ref-QdimT8mN">
<p>562. <strong>Breadth of concomitant immune responses underpinning viral clearance and patient recovery in a non-severe case of COVID-19</strong><br />
Irani Thevarajan, Thi HO Nguyen, Marios Koutsakos, Julian Druce, Leon Caly, Carolien E van de Sandt, Xiaoxiao Jia, Suellen Nicholson, Mike Catton, Benjamin Cowie, … Katherine Kedzierska<br />
<em>medRxiv</em> (2020-02-23) <a href="https://doi.org/ggq63r">https://doi.org/ggq63r</a><br />
DOI: <a href="https://doi.org/10.1101/2020.02.20.20025841">10.1101/2020.02.20.20025841</a></p>
</div>
<div id="ref-tVkKuIJt">
<p>563. <strong>The landscape of lung bronchoalveolar immune cells in COVID-19 revealed by single-cell RNA sequencing</strong><br />
Minfeng Liao, Yang Liu, Jin Yuan, Yanling Wen, Gang Xu, Juanjuan Zhao, Lin Chen, Jinxiu Li, Xin Wang, Fuxiang Wang, … Zheng Zhang<br />
<em>medRxiv</em> (2020-02-26) <a href="https://doi.org/ggq63s">https://doi.org/ggq63s</a><br />
DOI: <a href="https://doi.org/10.1101/2020.02.23.20026690">10.1101/2020.02.23.20026690</a></p>
</div>
<div id="ref-87CJN2CG">
<p>564. <strong>Influenza A Virus Infection Induces Hyperresponsiveness in Human Lung Tissue-Resident and Peripheral Blood NK Cells</strong><br />
Marlena Scharenberg, Sindhu Vangeti, Eliisa Kekäläinen, Per Bergman, Mamdoh Al-Ameri, Niclas Johansson, Klara Sondén, Sara Falck-Jones, Anna Färnert, Hans-Gustaf Ljunggren, … Nicole Marquardt<br />
<em>Frontiers in Immunology</em> (2019-05-17) <a href="https://doi.org/ggq656">https://doi.org/ggq656</a><br />
DOI: <a href="https://doi.org/10.3389/fimmu.2019.01116">10.3389/fimmu.2019.01116</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31156653">31156653</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6534051">PMC6534051</a></p>
</div>
<div id="ref-vasHgjGH">
<p>565. <strong>Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China</strong><br />
Chaolin Huang, Yeming Wang, Xingwang Li, Lili Ren, Jianping Zhao, Yi Hu, Li Zhang, Guohui Fan, Jiuyang Xu, Xiaoying Gu, … Bin Cao<br />
<em>The Lancet</em> (2020-02) <a href="https://doi.org/ggjfnn">https://doi.org/ggjfnn</a><br />
DOI: <a href="https://doi.org/10.1016/s0140-6736(20)30183-5">10.1016/s0140-6736(20)30183-5</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31986264">31986264</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7159299">PMC7159299</a></p>
</div>
<div id="ref-9GEyqic3">
<p>566. <strong>Alveolar Macrophages in the Resolution of Inflammation, Tissue Repair, and Tolerance to Infection</strong><br />
Benoit Allard, Alice Panariti, James G. Martin<br />
<em>Frontiers in Immunology</em> (2018-07-31) <a href="https://doi.org/gd3bnz">https://doi.org/gd3bnz</a><br />
DOI: <a href="https://doi.org/10.3389/fimmu.2018.01777">10.3389/fimmu.2018.01777</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30108592">30108592</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6079255">PMC6079255</a></p>
</div>
<div id="ref-WBN5f37b">
<p>567. <strong>PPAR-γ in Macrophages Limits Pulmonary Inflammation and Promotes Host Recovery following Respiratory Viral Infection</strong><br />
Su Huang, Bibo Zhu, In Su Cheon, Nick P. Goplen, Li Jiang, Ruixuan Zhang, R. Stokes Peebles, Matthias Mack, Mark H. Kaplan, Andrew H. Limper, Jie Sun<br />
<em>Journal of Virology</em> (2019-04-17) <a href="https://doi.org/ggq652">https://doi.org/ggq652</a><br />
DOI: <a href="https://doi.org/10.1128/jvi.00030-19">10.1128/jvi.00030-19</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30787149">30787149</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6475778">PMC6475778</a></p>
</div>
<div id="ref-FOVDaThy">
<p>568. <strong>Can routine laboratory tests discriminate 2019 novel coronavirus infected pneumonia from other community-acquired pneumonia?</strong><br />
Yunbao Pan, Guangming Ye, Xiantao Zeng, Guohong Liu, Xiaojiao Zeng, Xianghu Jiang, Jin Zhao, Liangjun Chen, Shuang Guo, Qiaoling Deng, … Xinghuan Wang<br />
<em>medRxiv</em> (2020-02-25) <a href="https://doi.org/ggq63t">https://doi.org/ggq63t</a><br />
DOI: <a href="https://doi.org/10.1101/2020.02.25.20024711">10.1101/2020.02.25.20024711</a></p>
</div>
<div id="ref-1Du40IOOH">
<p>569. <strong>Correlation Analysis Between Disease Severity and Inflammation-related Parameters in Patients with COVID-19 Pneumonia</strong><br />
Jing Gong, Hui Dong, Song Qing Xia, Yi Zhao Huang, Dingkun Wang, Yan Zhao, Wenhua Liu, Shenghao Tu, Mingmin Zhang, Qi Wang, Fuer Lu<br />
<em>medRxiv</em> (2020-02-26) <a href="https://doi.org/ggq63v">https://doi.org/ggq63v</a><br />
DOI: <a href="https://doi.org/10.1101/2020.02.25.20025643">10.1101/2020.02.25.20025643</a></p>
</div>
<div id="ref-1DTLMIDoM">
<p>570. <strong>An Effective CTL Peptide Vaccine for Ebola Zaire Based on Survivors’ CD8+ Targeting of a Particular Nucleocapsid Protein Epitope with Potential Implications for COVID-19 Vaccine Design</strong><br />
CV Herst, S Burkholz, J Sidney, A Sette, PE Harris, S Massey, T Brasel, E Cunha-Neto, DS Rosa, WCH Chao, … R Rubsamen<br />
<em>bioRxiv</em> (2020-04-06) <a href="https://doi.org/ggq63x">https://doi.org/ggq63x</a><br />
DOI: <a href="https://doi.org/10.1101/2020.02.25.963546">10.1101/2020.02.25.963546</a></p>
</div>
<div id="ref-9spK49tt">
<p>571. <strong>Epitope-based peptide vaccine design and target site characterization against novel coronavirus disease caused by SARS-CoV-2</strong><br />
Lin Li, Ting Sun, Yufei He, Wendong Li, Yubo Fan, Jing Zhang<br />
<em>bioRxiv</em> (2020-02-27) <a href="https://doi.org/ggnqwt">https://doi.org/ggnqwt</a><br />
DOI: <a href="https://doi.org/10.1101/2020.02.25.965434">10.1101/2020.02.25.965434</a></p>
</div>
<div id="ref-qg1bmQng">
<p>572. <strong>The definition and risks of Cytokine Release Syndrome-Like in 11 COVID-19-Infected Pneumonia critically ill patients: Disease Characteristics and Retrospective Analysis</strong><br />
Wenjun Wang, Jianxing He, puyi Lie, liyan Huang, Sipei Wu, yongping lin, xiaoqing liu<br />
<em>medRxiv</em> (2020-02-27) <a href="https://doi.org/ggrgbs">https://doi.org/ggrgbs</a><br />
DOI: <a href="https://doi.org/10.1101/2020.02.26.20026989">10.1101/2020.02.26.20026989</a></p>
</div>
<div id="ref-YODgfrtM">
<p>573. <strong>Clinical characteristics of 36 non-survivors with COVID-19 in Wuhan, China</strong><br />
Ying Huang, Rui Yang, Ying Xu, Ping Gong<br />
<em>medRxiv</em> (2020-03-05) <a href="https://doi.org/ggq63z">https://doi.org/ggq63z</a><br />
DOI: <a href="https://doi.org/10.1101/2020.02.27.20029009">10.1101/2020.02.27.20029009</a></p>
</div>
<div id="ref-5CGbn9Jj">
<p>574. <strong>Risk factors related to hepatic injury in patients with corona virus disease 2019</strong><br />
Lu Li, Shuang Li, Manman Xu, Pengfei Yu, Sujun Zheng, Zhongping Duan, Jing Liu, Yu Chen, Junfeng Li<br />
<em>medRxiv</em> (2020-03-10) <a href="https://doi.org/ggq632">https://doi.org/ggq632</a><br />
DOI: <a href="https://doi.org/10.1101/2020.02.28.20028514">10.1101/2020.02.28.20028514</a></p>
</div>
<div id="ref-T0BDaXkw">
<p>575. <strong>Detectable serum SARS-CoV-2 viral load (RNAaemia) is closely associated with drastically elevated interleukin 6 (IL-6) level in critically ill COVID-19 patients</strong><br />
Xiaohua Chen, Binghong Zhao, Yueming Qu, Yurou Chen, Jie Xiong, Yong Feng, Dong Men, Qianchuan Huang, Ying Liu, Bo Yang, … Feng Li<br />
<em>medRxiv</em> (2020-03-03) <a href="https://doi.org/ggq633">https://doi.org/ggq633</a><br />
DOI: <a href="https://doi.org/10.1101/2020.02.29.20029520">10.1101/2020.02.29.20029520</a></p>
</div>
<div id="ref-l3CzJ5ni">
<p>576. <strong>Prognostic factors in the acute respiratory distress syndrome</strong><br />
Wei Chen, Lorraine B Ware<br />
<em>Clinical and Translational Medicine</em> (2015-07-02) <a href="https://doi.org/ggq653">https://doi.org/ggq653</a><br />
DOI: <a href="https://doi.org/10.1186/s40169-015-0065-2">10.1186/s40169-015-0065-2</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26162279">26162279</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4534483">PMC4534483</a></p>
</div>
<div id="ref-uIwTzZkJ">
<p>577. <strong>Lymphopenia predicts disease severity of COVID-19: a descriptive and predictive study</strong><br />
Li Tan, Qi Wang, Duanyang Zhang, Jinya Ding, Qianchuan Huang, Yi-Quan Tang, Qiongshu Wang, Hongming Miao<br />
<em>medRxiv</em> (2020-03-03) <a href="https://doi.org/ggq634">https://doi.org/ggq634</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.01.20029074">10.1101/2020.03.01.20029074</a></p>
</div>
<div id="ref-1GMxHWnxN">
<p>578. <strong>The potential role of IL-6 in monitoring severe case of coronavirus disease 2019</strong><br />
Tao Liu, Jieying Zhang, Yuhui Yang, Hong Ma, Zhengyu Li, Jiaoyue Zhang, Ji Cheng, Xiaoyun Zhang, Yanxia Zhao, Zihan Xia, … Jianhua Yi<br />
<em>medRxiv</em> (2020-03-10) <a href="https://doi.org/ggq635">https://doi.org/ggq635</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.01.20029769">10.1101/2020.03.01.20029769</a></p>
</div>
<div id="ref-1B0vBwv1q">
<p>579. <strong>Clinical and Laboratory Profiles of 75 Hospitalized Patients with Novel Coronavirus Disease 2019 in Hefei, China</strong><br />
Zonghao Zhao, Jiajia Xie, Ming Yin, Yun Yang, Hongliang He, Tengchuan Jin, Wenting Li, Xiaowu Zhu, Jing Xu, Changcheng Zhao, … Xiaoling Ma<br />
<em>medRxiv</em> (2020-03-06) <a href="https://doi.org/ggq636">https://doi.org/ggq636</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.01.20029785">10.1101/2020.03.01.20029785</a></p>
</div>
<div id="ref-RyXNw87v">
<p>580. <strong>Exuberant elevation of IP-10, MCP-3 and IL-1ra during SARS-CoV-2 infection is associated with disease severity and fatal outcome</strong><br />
Yang Yang, Chenguang Shen, Jinxiu Li, Jing Yuan, Minghui Yang, Fuxiang Wang, Guobao Li, Yanjie Li, Li Xing, Ling Peng, … Yingxia Liu<br />
<em>medRxiv</em> (2020-03-06) <a href="https://doi.org/ggq637">https://doi.org/ggq637</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.02.20029975">10.1101/2020.03.02.20029975</a></p>
</div>
<div id="ref-5HXuCa5t">
<p>581. <strong>Antibody responses to SARS-CoV-2 in patients of novel coronavirus disease 2019</strong><br />
Juanjuan Zhao, Quan Yuan, Haiyan Wang, Wei Liu, Xuejiao Liao, Yingying Su, Xin Wang, Jing Yuan, Tingdong Li, Jinxiu Li, … Zheng Zhang<br />
<em>medRxiv</em> (2020-03-02) <a href="https://doi.org/ggrbj6">https://doi.org/ggrbj6</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.02.20030189">10.1101/2020.03.02.20030189</a></p>
</div>
<div id="ref-uCyoHtV3">
<p>582. <strong>Restoration of leukomonocyte counts is associated with viral clearance in COVID-19 hospitalized patients</strong><br />
Xiaoping Chen, Jiaxin Ling, Pingzheng Mo, Yongxi Zhang, Qunqun Jiang, Zhiyong Ma, Qian Cao, Wenjia Hu, Shi Zou, Liangjun Chen, … Yong Xiong<br />
<em>medRxiv</em> (2020-03-06) <a href="https://doi.org/ggq639">https://doi.org/ggq639</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.03.20030437">10.1101/2020.03.03.20030437</a></p>
</div>
<div id="ref-L6kwccjn">
<p>583. <strong>Effects of Systemically Administered Hydrocortisone on the Human Immunome</strong><br />
Matthew J. Olnes, Yuri Kotliarov, Angélique Biancotto, Foo Cheung, Jinguo Chen, Rongye Shi, Huizhi Zhou, Ena Wang, John S. Tsang, Robert Nussenblatt, The CHI Consortium<br />
<em>Scientific Reports</em> (2016-03-14) <a href="https://doi.org/f8dmvw">https://doi.org/f8dmvw</a><br />
DOI: <a href="https://doi.org/10.1038/srep23002">10.1038/srep23002</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26972611">26972611</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4789739">PMC4789739</a></p>
</div>
<div id="ref-K9G9QT8y">
<p>584. <strong>Procalcitonin in patients with severe coronavirus disease 2019 (COVID-19): A meta-analysis</strong><br />
Giuseppe Lippi, Mario Plebani<br />
<em>Clinica Chimica Acta</em> (2020-06) <a href="https://doi.org/ggpxp7">https://doi.org/ggpxp7</a><br />
DOI: <a href="https://doi.org/10.1016/j.cca.2020.03.004">10.1016/j.cca.2020.03.004</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32145275">32145275</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7094472">PMC7094472</a></p>
</div>
<div id="ref-USOnp7c0">
<p>585. <strong>Clinical findings in critical ill patients infected with SARS-Cov-2 in Guangdong Province, China: a multi-center, retrospective, observational study</strong><br />
Yonghao Xu, Zhiheng Xu, Xuesong Liu, Lihua Cai, Haichong Zheng, Yongbo Huang, Lixin Zhou, Linxi Huang, Yun Lin, Liehua Deng, … Yimin Li<br />
<em>medRxiv</em> (2020-03-06) <a href="https://doi.org/ggq64b">https://doi.org/ggq64b</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.03.20030668">10.1101/2020.03.03.20030668</a></p>
</div>
<div id="ref-14OLQIYK0">
<p>586. <strong>Multi-epitope vaccine design using an immunoinformatics approach for 2019 novel coronavirus (SARS-CoV-2)</strong><br />
Ye Feng, Min Qiu, Liang Liu, Shengmei Zou, Yun Li, Kai Luo, Qianpeng Guo, Ning Han, Yingqiang Sun, Kui Wang, … Fan Mo<br />
<em>bioRxiv</em> (2020-06-30) <a href="https://doi.org/ggq64c">https://doi.org/ggq64c</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.03.962332">10.1101/2020.03.03.962332</a></p>
</div>
<div id="ref-12ilBRq6I">
<p>587. <strong>Clinical Features of Patients Infected with the 2019 Novel Coronavirus (COVID-19) in Shanghai, China</strong><br />
Min Cao, Dandan Zhang, Youhua Wang, Yunfei Lu, Xiangdong Zhu, Ying Li, Honghao Xue, Yunxiao Lin, Min Zhang, Yiguo Sun, … Longping Peng<br />
<em>medRxiv</em> (2020-03-06) <a href="https://doi.org/ggq64d">https://doi.org/ggq64d</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.04.20030395">10.1101/2020.03.04.20030395</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32511465">32511465</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7255784">PMC7255784</a></p>
</div>
<div id="ref-IPJURDxi">
<p>588. <strong>Serological detection of 2019-nCoV respond to the epidemic: A useful complement to nucleic acid testing</strong><br />
Jin Zhang, Jianhua Liu, Na Li, Yong Liu, Rui Ye, Xiaosong Qin, Rui Zheng<br />
<em>medRxiv</em> (2020-03-10) <a href="https://doi.org/ggq64f">https://doi.org/ggq64f</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.04.20030916">10.1101/2020.03.04.20030916</a></p>
</div>
<div id="ref-H0nvocJz">
<p>589. <strong>Human Kidney is a Target for Novel Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) Infection</strong><br />
Bo Diao, Chenhui Wang, Rongshuai Wang, Zeqing Feng, Yingjun Tan, Huiming Wang, Changsong Wang, Liang Liu, Ying Liu, Yueping Liu, … Yongwen Chen<br />
<em>medRxiv</em> (2020-04-10) <a href="https://doi.org/ggq64g">https://doi.org/ggq64g</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.04.20031120">10.1101/2020.03.04.20031120</a></p>
</div>
<div id="ref-KHNnjP7M">
<p>590. <strong>COVID-19 early warning score: a multi-parameter screening tool to identify highly suspected patients</strong><br />
Cong-Ying Song, Jia Xu, Jian-Qin He, Yuan-Qiang Lu<br />
<em>medRxiv</em> (2020-03-08) <a href="https://doi.org/ggq64h">https://doi.org/ggq64h</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.05.20031906">10.1101/2020.03.05.20031906</a></p>
</div>
<div id="ref-19xwRoatu">
<p>591. <strong>LY6E impairs coronavirus fusion and confers immune control of viral disease</strong><br />
Stephanie Pfaender, Katrina B. Mar, Eleftherios Michailidis, Annika Kratzel, Dagny Hirt, Philip V’kovski, Wenchun Fan, Nadine Ebert, Hanspeter Stalder, Hannah Kleine-Weber, … Volker Thiel<br />
<em>bioRxiv</em> (2020-03-07) <a href="https://doi.org/dpvn">https://doi.org/dpvn</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.05.979260">10.1101/2020.03.05.979260</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32511345">32511345</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7255780">PMC7255780</a></p>
</div>
<div id="ref-574aSa8C">
<p>592. <strong>A preliminary study on serological assay for severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) in 238 admitted hospital patients</strong><br />
Lei Liu, Wanbing Liu, Shengdian Wang, Shangen Zheng<br />
<em>medRxiv</em> (2020-03-08) <a href="https://doi.org/ggq64j">https://doi.org/ggq64j</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.06.20031856">10.1101/2020.03.06.20031856</a></p>
</div>
<div id="ref-NT3T9a8H">
<p>593. <strong>Monoclonal antibodies for the S2 subunit of spike of SARS-CoV cross-react with the newly-emerged SARS-CoV-2</strong><br />
Zhiqiang Zheng, Vanessa M. Monteil, Sebastian Maurer-Stroh, Chow Wenn Yew, Carol Leong, Nur Khairiah Mohd-Ismail, Suganya Cheyyatraivendran Arularasu, Vincent Tak Kwong Chow, Raymond Lin Tzer Pin, Ali Mirazimi, … Yee-Joo Tan<br />
<em>bioRxiv</em> (2020-03-07) <a href="https://doi.org/ggrbj7">https://doi.org/ggrbj7</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.06.980037">10.1101/2020.03.06.980037</a></p>
</div>
<div id="ref-1GEbBtBgh">
<p>594. <strong>Mortality of COVID-19 is Associated with Cellular Immune Function Compared to Immune Function in Chinese Han Population</strong><br />
Qiang Zeng, Yong-zhe Li, Gang Huang, Wei Wu, Sheng-yong Dong, Yang Xu<br />
<em>medRxiv</em> (2020-03-13) <a href="https://doi.org/ggq64k">https://doi.org/ggq64k</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.08.20031229">10.1101/2020.03.08.20031229</a></p>
</div>
<div id="ref-WuHcOc9T">
<p>595. <strong>Retrospective Analysis of Clinical Features in 101 Death Cases with COVID-19</strong><br />
JIan Chen, Hua Fan, Lin Zhang, Bin Huang, Muxin Zhu, Yong Zhou, WenHu Yu, Liping Zhu, Shaohui Cheng, Xiaogen Tao, Huan Zhang<br />
<em>medRxiv</em> (2020-03-17) <a href="https://doi.org/ggq64n">https://doi.org/ggq64n</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.09.20033068">10.1101/2020.03.09.20033068</a></p>
</div>
<div id="ref-1DbV444Pj">
<p>596. <strong>Relationship between the ABO Blood Group and the COVID-19 Susceptibility</strong><br />
Jiao Zhao, Yan Yang, Hanping Huang, Dong Li, Dongfeng Gu, Xiangfeng Lu, Zheng Zhang, Lei Liu, Ting Liu, Yukun Liu, … Peng George Wang<br />
<em>medRxiv</em> (2020-03-27) <a href="https://doi.org/ggpn3d">https://doi.org/ggpn3d</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.11.20031096">10.1101/2020.03.11.20031096</a></p>
</div>
<div id="ref-JwL9G6bd">
<p>597. <strong>The inhaled corticosteroid ciclesonide blocks coronavirus RNA replication by targeting viral NSP15</strong><br />
Shutoku Matsuyama, Miyuki Kawase, Naganori Nao, Kazuya Shirato, Makoto Ujike, Wataru Kamitani, Masayuki Shimojima, Shuetsu Fukushi<br />
<em>bioRxiv</em> (2020-03-12) <a href="https://doi.org/ggq64p">https://doi.org/ggq64p</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.11.987016">10.1101/2020.03.11.987016</a></p>
</div>
<div id="ref-1CwCTXPqj">
<p>598. <strong>Immune phenotyping based on neutrophil-to-lymphocyte ratio and IgG predicts disease severity and outcome for patients with COVID-19</strong><br />
Bicheng Zhang, Xiaoyang Zhou, Chengliang Zhu, Fan Feng, Yanru Qiu, Jia Feng, Qingzhu Jia, Qibin Song, Bo Zhu, Jun Wang<br />
<em>medRxiv</em> (2020-03-16) <a href="https://doi.org/ggq64q">https://doi.org/ggq64q</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.12.20035048">10.1101/2020.03.12.20035048</a></p>
</div>
<div id="ref-3XqFhi8l">
<p>599. <strong>Reinfection could not occur in SARS-CoV-2 infected rhesus macaques</strong><br />
Linlin Bao, Wei Deng, Hong Gao, Chong Xiao, Jiayi Liu, Jing Xue, Qi Lv, Jiangning Liu, Pin Yu, Yanfeng Xu, … Chuan Qin<br />
<em>bioRxiv</em> (2020-03-14) <a href="https://doi.org/ggn8r8">https://doi.org/ggn8r8</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.13.990226">10.1101/2020.03.13.990226</a></p>
</div>
<div id="ref-6gQxRkWM">
<p>600. <strong>A highly conserved cryptic epitope in the receptor-binding domains of SARS-CoV-2 and SARS-CoV</strong><br />
Meng Yuan, Nicholas C. Wu, Xueyong Zhu, Chang-Chun D. Lee, Ray T. Y. So, Huibin Lv, Chris K. P. Mok, Ian A. Wilson<br />
<em>bioRxiv</em> (2020-03-14) <a href="https://doi.org/ggq64s">https://doi.org/ggq64s</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.13.991570">10.1101/2020.03.13.991570</a></p>
</div>
<div id="ref-14y8UvAbH">
<p>601. <strong>SARS-CoV-2 invades host cells via a novel route: CD147-spike protein</strong><br />
Ke Wang, Wei Chen, Yu-Sen Zhou, Jian-Qi Lian, Zheng Zhang, Peng Du, Li Gong, Yang Zhang, Hong-Yong Cui, Jie-Jie Geng, … Zhi-Nan Chen<br />
<em>bioRxiv</em> (2020-03-14) <a href="https://doi.org/ggq64t">https://doi.org/ggq64t</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.14.988345">10.1101/2020.03.14.988345</a></p>
</div>
<div id="ref-BcOf65BE">
<p>602. <strong>CD147 (EMMPRIN/Basigin) in kidney diseases: from an inflammation and immune system viewpoint</strong><br />
Tomoki Kosugi, Kayaho Maeda, Waichi Sato, Shoichi Maruyama, Kenji Kadomatsu<br />
<em>Nephrology Dialysis Transplantation</em> (2015-07) <a href="https://doi.org/ggq624">https://doi.org/ggq624</a><br />
DOI: <a href="https://doi.org/10.1093/ndt/gfu302">10.1093/ndt/gfu302</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25248362">25248362</a></p>
</div>
<div id="ref-34nQJalJ">
<p>603. <strong>The roles of CyPA and CD147 in cardiac remodelling</strong><br />
Hongyan Su, Yi Yang<br />
<em>Experimental and Molecular Pathology</em> (2018-06) <a href="https://doi.org/ggq622">https://doi.org/ggq622</a><br />
DOI: <a href="https://doi.org/10.1016/j.yexmp.2018.05.001">10.1016/j.yexmp.2018.05.001</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29772453">29772453</a></p>
</div>
<div id="ref-KcThTnsm">
<p>604. <strong>Cancer-related issues of CD147.</strong><br />
Ulrich H Weidle, Werner Scheuer, Daniela Eggle, Stefan Klostermann, Hannes Stockinger<br />
<em>Cancer genomics &amp; proteomics</em> <a href="https://www.ncbi.nlm.nih.gov/pubmed/20551248">https://www.ncbi.nlm.nih.gov/pubmed/20551248</a><br />
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20551248">20551248</a></p>
</div>
<div id="ref-18l4l0nLX">
<p>605. <strong>Blood single cell immune profiling reveals the interferon-MAPK pathway mediated adaptive immune response for COVID-19</strong><br />
Lulin Huang, Yi Shi, Bo Gong, Li Jiang, Xiaoqi Liu, Jialiang Yang, Juan Tang, Chunfang You, Qi Jiang, Bo Long, … Zhenglin Yang<br />
<em>medRxiv</em> (2020-03-17) <a href="https://doi.org/ggq64v">https://doi.org/ggq64v</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.15.20033472">10.1101/2020.03.15.20033472</a></p>
</div>
<div id="ref-dM1rOHmH">
<p>606. <strong>Cross-reactive antibody response between SARS-CoV-2 and SARS-CoV infections</strong><br />
Huibin Lv, Nicholas C. Wu, Owen Tak-Yin Tsang, Meng Yuan, Ranawaka A. P. M. Perera, Wai Shing Leung, Ray T. Y. So, Jacky Man Chun Chan, Garrick K. Yip, Thomas Shiu Hong Chik, … Chris K. P. Mok<br />
<em>bioRxiv</em> (2020-03-17) <a href="https://doi.org/ggq64w">https://doi.org/ggq64w</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.15.993097">10.1101/2020.03.15.993097</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32511317">32511317</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7239046">PMC7239046</a></p>
</div>
<div id="ref-UENoLkhz">
<p>607. <strong>The feasibility of convalescent plasma therapy in severe COVID-19 patients: a pilot study</strong><br />
Kai Duan, Bende Liu, Cesheng Li, Huajun Zhang, Ting Yu, Jieming Qu, Min Zhou, Li Chen, Shengli Meng, Yong Hu, … Xiaoming Yang<br />
<em>medRxiv</em> (2020-03-23) <a href="https://doi.org/dqrs">https://doi.org/dqrs</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.16.20036145">10.1101/2020.03.16.20036145</a></p>
</div>
<div id="ref-Xbkby2Jr">
<p>608. <strong>Hydroxychloroquine and Azithromycin as a treatment of COVID-19: preliminary results of an open-label non-randomized clinical trial</strong><br />
Philippe GAUTRET, Jean Christophe LAGIER, Philippe PAROLA, Van Thuan HOANG, Line MEDDED, Morgan MAILHE, Barbara DOUDIER, Johan COURJON, Valerie GIORDANENGO, Vera ESTEVES VIEIRA, … Didier RAOULT<br />
<em>medRxiv</em> (2020-03-20) <a href="https://doi.org/dqbv">https://doi.org/dqbv</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.16.20037135">10.1101/2020.03.16.20037135</a></p>
</div>
<div id="ref-BIlqPh6M">
<p>609. <strong>Chloroquine: Modes of action of an undervalued drug</strong><br />
Rodolfo Thomé, Stefanie Costa Pinto Lopes, Fabio Trindade Maranhão Costa, Liana Verinaud<br />
<em>Immunology Letters</em> (2013-06) <a href="https://doi.org/f5b5cr">https://doi.org/f5b5cr</a><br />
DOI: <a href="https://doi.org/10.1016/j.imlet.2013.07.004">10.1016/j.imlet.2013.07.004</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23891850">23891850</a></p>
</div>
<div id="ref-Q9tXjG1R">
<p>610. <strong>Patients with LRBA deficiency show CTLA4 loss and immune dysregulation responsive to abatacept therapy</strong><br />
B. Lo, K. Zhang, W. Lu, L. Zheng, Q. Zhang, C. Kanellopoulou, Y. Zhang, Z. Liu, J. M. Fritz, R. Marsh, … M. B. Jordan<br />
<em>Science</em> (2015-07-23) <a href="https://doi.org/f7kc8d">https://doi.org/f7kc8d</a><br />
DOI: <a href="https://doi.org/10.1126/science.aaa1663">10.1126/science.aaa1663</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26206937">26206937</a></p>
</div>
<div id="ref-3VPi509z">
<p>611. <strong>The sequence of human ACE2 is suboptimal for binding the S spike protein of SARS coronavirus 2</strong><br />
Erik Procko<br />
<em>bioRxiv</em> (2020-05-11) <a href="https://doi.org/ggrbj8">https://doi.org/ggrbj8</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.16.994236">10.1101/2020.03.16.994236</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32511321">32511321</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7239051">PMC7239051</a></p>
</div>
<div id="ref-1EcOLnVXk">
<p>612. <strong>Comparative Pathogenesis Of COVID-19, MERS And SARS In A Non-Human Primate Model</strong><br />
Barry Rockx, Thijs Kuiken, Sander Herfst, Theo Bestebroer, Mart M. Lamers, Dennis de Meulder, Geert van Amerongen, Judith van den Brand, Nisreen M. A. Okba, Debby Schipper, … Bart L. Haagmans<br />
<em>bioRxiv</em> (2020-03-17) <a href="https://doi.org/ggq649">https://doi.org/ggq649</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.17.995639">10.1101/2020.03.17.995639</a></p>
</div>
<div id="ref-NzFgXYvQ">
<p>613. <strong>Lethal Infection of K18-hACE2 Mice Infected with Severe Acute Respiratory Syndrome Coronavirus</strong><br />
P. B. McCray, L. Pewe, C. Wohlford-Lenane, M. Hickey, L. Manzel, L. Shi, J. Netland, H. P. Jia, C. Halabi, C. D. Sigmund, … S. Perlman<br />
<em>Journal of Virology</em> (2006-11-01) <a href="https://doi.org/b2dr3s">https://doi.org/b2dr3s</a><br />
DOI: <a href="https://doi.org/10.1128/jvi.02012-06">10.1128/jvi.02012-06</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17079315">17079315</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1797474">PMC1797474</a></p>
</div>
<div id="ref-OvFueDLZ">
<p>614. <strong>Investigating the Impact of Asymptomatic Carriers on COVID-19 Transmission</strong><br />
Jacob B Aguilar, Jeremy Samuel Faust, Lauren M. Westafer, Juan B. Gutierrez<br />
<em>medRxiv</em> (2020-03-31) <a href="https://doi.org/ggqnvp">https://doi.org/ggqnvp</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.18.20037994">10.1101/2020.03.18.20037994</a></p>
</div>
<div id="ref-Y9g1sQ1Y">
<p>615. <strong>Antibody responses to SARS-CoV-2 in COVID-19 patients: the perspective application of serological tests in clinical practice</strong><br />
Quan-xin Long, Hai-jun Deng, Juan Chen, Jieli Hu, Bei-zhong Liu, Pu Liao, Yong Lin, Li-hua Yu, Zhan Mo, Yin-yin Xu, … Ai-long Huang<br />
<em>medRxiv</em> (2020-03-20) <a href="https://doi.org/ggpvz3">https://doi.org/ggpvz3</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.18.20038018">10.1101/2020.03.18.20038018</a></p>
</div>
<div id="ref-p1c9Nilh">
<p>616. <strong>Heat inactivation of serum interferes with the immunoanalysis of antibodies to SARS-CoV-2</strong><br />
Xiumei Hu, Taixue An, Bo Situ, Yuhai Hu, Zihao Ou, Qiang Li, Xiaojing He, Ye Zhang, Peifu Tian, Dehua Sun, … Lei Zheng<br />
<em>medRxiv</em> (2020-03-16) <a href="https://doi.org/ggq646">https://doi.org/ggq646</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.12.20034231">10.1101/2020.03.12.20034231</a></p>
</div>
<div id="ref-1BLNvgEao">
<p>617. <strong>SARS-CoV-2 specific antibody responses in COVID-19 patients</strong><br />
NISREEN M. A. OKBA, Marcel A Muller, Wentao Li, Chunyan Wang, Corine H. GeurtsvanKessel, Victor M. Corman, Mart M. Lamers, Reina S. Sikkema, Erwin de Bruin, Felicity D. Chandler, … Bart L. Haagmans<br />
<em>medRxiv</em> (2020-03-20) <a href="https://doi.org/ggpvz2">https://doi.org/ggpvz2</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.18.20038059">10.1101/2020.03.18.20038059</a></p>
</div>
<div id="ref-K41YFM2N">
<p>618. <strong>A brief review of antiviral drugs evaluated in registered clinical trials for COVID-19</strong><br />
Drifa Belhadi, Nathan Peiffer-Smadja, François-Xavier Lescure, Yazdan Yazdanpanah, France Mentré, Cédric Laouénan<br />
<em>medRxiv</em> (2020-03-27) <a href="https://doi.org/ggq65b">https://doi.org/ggq65b</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.18.20038190">10.1101/2020.03.18.20038190</a></p>
</div>
<div id="ref-eTsOvBVf">
<p>619. <strong>ACE-2 Expression in the Small Airway Epithelia of Smokers and COPD Patients: Implications for COVID-19</strong><br />
Janice M Leung, Chen Xi Yang, Anthony Tam, Tawimas Shaipanich, Tillie L Hackett, Gurpreet K Singhera, Delbert R Dorscheid, Don D Sin<br />
<em>medRxiv</em> (2020-03-23) <a href="https://doi.org/dqx2">https://doi.org/dqx2</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.18.20038455">10.1101/2020.03.18.20038455</a></p>
</div>
<div id="ref-18KZGwJ0J">
<p>620. <strong>Dynamic profile of severe or critical COVID-19 cases</strong><br />
Yang Xu<br />
<em>medRxiv</em> (2020-03-20) <a href="https://doi.org/ggrbj9">https://doi.org/ggrbj9</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.18.20038513">10.1101/2020.03.18.20038513</a></p>
</div>
<div id="ref-1BJXDZ04c">
<p>621. <strong>Association between Clinical, Laboratory and CT Characteristics and RT-PCR Results in the Follow-up of COVID-19 patients</strong><br />
Hang Fu, Huayan Xu, Na Zhang, Hong Xu, Zhenlin Li, Huizhu Chen, Rong Xu, Ran Sun, Lingyi Wen, Linjun Xie, … Yingkun Guo<br />
<em>medRxiv</em> (2020-03-23) <a href="https://doi.org/ggq65c">https://doi.org/ggq65c</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.19.20038315">10.1101/2020.03.19.20038315</a></p>
</div>
<div id="ref-iMCz5Twj">
<p>622. <strong>An orally bioavailable broad-spectrum antiviral inhibits SARS-CoV-2 and multiple endemic, epidemic and bat coronavirus</strong><br />
Timothy P. Sheahan, Amy C. Sims, Shuntai Zhou, Rachel L. Graham, Collin S. Hill, Sarah R. Leist, Alexandra Schäfer, Kenneth H. Dinnon, Stephanie A. Montgomery, Maria L. Agostini, … Ralph S. Baric<br />
<em>bioRxiv</em> (2020-03-20) <a href="https://doi.org/ggrbkb">https://doi.org/ggrbkb</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.19.997890">10.1101/2020.03.19.997890</a></p>
</div>
<div id="ref-ZDP9vSpg">
<p>623. <strong>Identification of antiviral drug candidates against SARS-CoV-2 from FDA-approved drugs</strong><br />
Sangeun Jeon, Meehyun Ko, Jihye Lee, Inhee Choi, Soo Young Byun, Soonju Park, David Shum, Seungtaek Kim<br />
<em>bioRxiv</em> (2020-03-28) <a href="https://doi.org/ggq65h">https://doi.org/ggq65h</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.20.999730">10.1101/2020.03.20.999730</a></p>
</div>
<div id="ref-LYFzvlzE">
<p>624. <strong>Respiratory disease and virus shedding in rhesus macaques inoculated with SARS-CoV-2</strong><br />
Vincent J. Munster, Friederike Feldmann, Brandi N. Williamson, Neeltje van Doremalen, Lizzette Pérez-Pérez, Jonathan Schulz, Kimberly Meade-White, Atsushi Okumura, Julie Callison, Beniah Brumbaugh, … Emmie de Wit<br />
<em>bioRxiv</em> (2020-03-21) <a href="https://doi.org/ggq65j">https://doi.org/ggq65j</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.21.001628">10.1101/2020.03.21.001628</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32511299">32511299</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7217148">PMC7217148</a></p>
</div>
<div id="ref-unVgVIfU">
<p>625. <strong>Ocular conjunctival inoculation of SARS-CoV-2 can cause mild COVID-19 in Rhesus macaques</strong><br />
Wei Deng, Linlin Bao, Hong Gao, Zhiguang Xiang, Yajin Qu, Zhiqi Song, Shunran Gong, Jiayi Liu, Jiangning Liu, Pin Yu, … Chuan Qin<br />
<em>bioRxiv</em> (2020-03-30) <a href="https://doi.org/ggq64r">https://doi.org/ggq64r</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.13.990036">10.1101/2020.03.13.990036</a></p>
</div>
<div id="ref-wjtVy2v0">
<p>626. <strong>ACE2 Expression is Increased in the Lungs of Patients with Comorbidities Associated with Severe COVID-19</strong><br />
Bruna GG Pinto, Antonio ER Oliveira, Youvika Singh, Leandro Jimenez, Andre NA Goncalves, Rodrigo LT Ogava, Rachel Creighton, Jean PS Peron, Helder I Nakaya<br />
<em>medRxiv</em> (2020-03-27) <a href="https://doi.org/ggq65k">https://doi.org/ggq65k</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.21.20040261">10.1101/2020.03.21.20040261</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32511627">32511627</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7276054">PMC7276054</a></p>
</div>
<div id="ref-DFhY5nfb">
<p>627. <strong>Meplazumab treats COVID-19 pneumonia: an open-labelled, concurrent controlled add-on clinical trial</strong><br />
Huijie Bian, Zhao-Hui Zheng, Ding Wei, Zheng Zhang, Wen-Zhen Kang, Chun-Qiu Hao, Ke Dong, Wen Kang, Jie-Lai Xia, Jin-Lin Miao, … Ping Zhu<br />
<em>medRxiv</em> (2020-07-15) <a href="https://doi.org/ggq65m">https://doi.org/ggq65m</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.21.20040691">10.1101/2020.03.21.20040691</a></p>
</div>
<div id="ref-yP8TXdq2">
<p>628. <strong>CD147 facilitates HIV-1 infection by interacting with virus-associated cyclophilin A</strong><br />
T. Pushkarsky, G. Zybarth, L. Dubrovsky, V. Yurchenko, H. Tang, H. Guo, B. Toole, B. Sherry, M. Bukrinsky<br />
<em>Proceedings of the National Academy of Sciences</em> (2001-05-15) <a href="https://doi.org/cc4c7p">https://doi.org/cc4c7p</a><br />
DOI: <a href="https://doi.org/10.1073/pnas.111583198">10.1073/pnas.111583198</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/11353871">11353871</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC33473">PMC33473</a></p>
</div>
<div id="ref-X8l1n4zY">
<p>629. <strong>CD147/EMMPRIN Acts as a Functional Entry Receptor for Measles Virus on Epithelial Cells</strong><br />
Akira Watanabe, Misako Yoneda, Fusako Ikeda, Yuri Terao-Muto, Hiroki Sato, Chieko Kai<br />
<em>Journal of Virology</em> (2010-05-01) <a href="https://doi.org/dpcsqg">https://doi.org/dpcsqg</a><br />
DOI: <a href="https://doi.org/10.1128/jvi.02168-09">10.1128/jvi.02168-09</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20147391">20147391</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2863760">PMC2863760</a></p>
</div>
<div id="ref-YcyNaIVC">
<p>630. <strong>Basigin is a receptor essential for erythrocyte invasion by Plasmodium falciparum</strong><br />
Cécile Crosnier, Leyla Y. Bustamante, S. Josefin Bartholdson, Amy K. Bei, Michel Theron, Makoto Uchikawa, Souleymane Mboup, Omar Ndir, Dominic P. Kwiatkowski, Manoj T. Duraisingh, … Gavin J. Wright<br />
<em>Nature</em> (2011-11-09) <a href="https://doi.org/dm59hf">https://doi.org/dm59hf</a><br />
DOI: <a href="https://doi.org/10.1038/nature10606">10.1038/nature10606</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22080952">22080952</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3245779">PMC3245779</a></p>
</div>
<div id="ref-m9yLJ5Da">
<p>631. <strong>Function of HAb18G/CD147 in Invasion of Host Cells by Severe Acute Respiratory Syndrome Coronavirus</strong><br />
Zhinan Chen, Li Mi, Jing Xu, Jiyun Yu, Xianhui Wang, Jianli Jiang, Jinliang Xing, Peng Shang, Airong Qian, Yu Li, … Ping Zhu<br />
<em>The Journal of Infectious Diseases</em> (2005-03) <a href="https://doi.org/cd8snd">https://doi.org/cd8snd</a><br />
DOI: <a href="https://doi.org/10.1086/427811">10.1086/427811</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15688292">15688292</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7110046">PMC7110046</a></p>
</div>
<div id="ref-J0ibHE3l">
<p>632. <strong>CD147 mediates intrahepatic leukocyte aggregation and determines the extent of liver injury</strong><br />
Christine Yee, Nathan M. Main, Alexandra Terry, Igor Stevanovski, Annette Maczurek, Alison J. Morgan, Sarah Calabro, Alison J. Potter, Tina L. Iemma, David G. Bowen, … Nicholas A. Shackel<br />
<em>PLOS ONE</em> (2019-07-10) <a href="https://doi.org/ggq654">https://doi.org/ggq654</a><br />
DOI: <a href="https://doi.org/10.1371/journal.pone.0215557">10.1371/journal.pone.0215557</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31291257">31291257</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6619953">PMC6619953</a></p>
</div>
<div id="ref-bCYYaFKM">
<p>633. <strong>Characterisation of the transcriptome and proteome of SARS-CoV-2 using direct RNA sequencing and tandem mass spectrometry reveals evidence for a cell passage induced in-frame deletion in the spike glycoprotein that removes the furin-like cleavage site</strong><br />
Andrew D. Davidson, Maia Kavanagh Williamson, Sebastian Lewis, Deborah Shoemark, Miles W. Carroll, Kate Heesom, Maria Zambon, Joanna Ellis, Phillip A. Lewis, Julian A. Hiscox, David A. Matthews<br />
<em>bioRxiv</em> (2020-03-24) <a href="https://doi.org/ggq65n">https://doi.org/ggq65n</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.22.002204">10.1101/2020.03.22.002204</a></p>
</div>
<div id="ref-16tHVjVzu">
<p>634. <strong>Modifications to the Hemagglutinin Cleavage Site Control the Virulence of a Neurotropic H1N1 Influenza Virus</strong><br />
X. Sun, L. V. Tse, A. D. Ferguson, G. R. Whittaker<br />
<em>Journal of Virology</em> (2010-06-16) <a href="https://doi.org/drs2zt">https://doi.org/drs2zt</a><br />
DOI: <a href="https://doi.org/10.1128/jvi.00797-10">10.1128/jvi.00797-10</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20554779">20554779</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2919019">PMC2919019</a></p>
</div>
<div id="ref-hRVfFbS">
<p>635. <strong>The architecture of SARS-CoV-2 transcriptome</strong><br />
Dongwan Kim, Joo-Yeon Lee, Jeong-Sun Yang, Jun Won Kim, V. Narry Kim, Hyeshik Chang<br />
<em>bioRxiv</em> (2020-03-14) <a href="https://doi.org/ggpx9q">https://doi.org/ggpx9q</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.12.988865">10.1101/2020.03.12.988865</a></p>
</div>
<div id="ref-RTIk8DTo">
<p>636. <strong>First Clinical Study Using HCV Protease Inhibitor Danoprevir to Treat Naive and Experienced COVID-19 Patients</strong><br />
Hongyi Chen, Zhicheng Zhang, Li Wang, Zhihua Huang, Fanghua Gong, Xiaodong Li, Yahong Chen, Jinzi J. WU<br />
<em>medRxiv</em> (2020-03-24) <a href="https://doi.org/ggrgbt">https://doi.org/ggrgbt</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.22.20034041">10.1101/2020.03.22.20034041</a></p>
</div>
<div id="ref-YoCPtq6P">
<p>637. <strong>Preclinical Characteristics of the Hepatitis C Virus NS3/4A Protease Inhibitor ITMN-191 (R7227)</strong><br />
S. D. Seiwert, S. W. Andrews, Y. Jiang, V. Serebryany, H. Tan, K. Kossen, P. T. R. Rajagopalan, S. Misialek, S. K. Stevens, A. Stoycheva, … L. M. Blatt<br />
<em>Antimicrobial Agents and Chemotherapy</em> (2008-09-29) <a href="https://doi.org/btpg52">https://doi.org/btpg52</a><br />
DOI: <a href="https://doi.org/10.1128/aac.00699-08">10.1128/aac.00699-08</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/18824605">18824605</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2592891">PMC2592891</a></p>
</div>
<div id="ref-10GnhBctk">
<p>638. <strong>Efficacy and Safety of All-oral, 12-week Ravidasvir Plus Ritonavir-boosted Danoprevir and Ribavirin in Treatment-naïve Noncirrhotic HCV Genotype 1 Patients: Results from a Phase 2/3 Clinical Trial in China</strong><br />
Xiaoyuan Xu, Bo Feng, Yujuan Guan, Sujun Zheng, Jifang Sheng, Xingxiang Yang, Yuanji Ma, Yan Huang, Yi Kang, Xiaofeng Wen, … Lai Wei<br />
<em>Journal of Clinical and Translational Hepatology</em> (2019-09-30) <a href="https://doi.org/ggrbkd">https://doi.org/ggrbkd</a><br />
DOI: <a href="https://doi.org/10.14218/jcth.2019.00033">10.14218/jcth.2019.00033</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31608212">31608212</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6783683">PMC6783683</a></p>
</div>
<div id="ref-7zBwK1wn">
<p>639. <strong>Potentially highly potent drugs for 2019-nCoV</strong><br />
Duc Duy Nguyen, Kaifu Gao, Jiahui Chen, Rui Wang, Guo-Wei Wei<br />
<em>bioRxiv</em> (2020-02-13) <a href="https://doi.org/ggrbj5">https://doi.org/ggrbj5</a><br />
DOI: <a href="https://doi.org/10.1101/2020.02.05.936013">10.1101/2020.02.05.936013</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32511344">32511344</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7255774">PMC7255774</a></p>
</div>
<div id="ref-lnuNSHEW">
<p>640. <strong>Serology characteristics of SARS-CoV-2 infection since the exposure and post symptoms onset</strong><br />
Bin Lou, Tigndong Li, Shufa Zheng, Yingying Su, Zhiyong Li, Wei Liu, Fei Yu, Shengxiang Ge, Qianda Zou, Quan Yuan, … Yu Chen<br />
<em>medRxiv</em> (2020-03-27) <a href="https://doi.org/ggrbkc">https://doi.org/ggrbkc</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.23.20041707">10.1101/2020.03.23.20041707</a></p>
</div>
<div id="ref-aNSdARwe">
<p>641. <strong>SARS-CoV-2 launches a unique transcriptional signature from in vitro, ex vivo, and in vivo systems</strong><br />
Daniel Blanco-Melo, Benjamin E. Nilsson-Payant, Wen-Chun Liu, Rasmus Møller, Maryline Panis, David Sachs, Randy A. Albrecht, Benjamin R. tenOever<br />
<em>bioRxiv</em> (2020-03-24) <a href="https://doi.org/ggq65q">https://doi.org/ggq65q</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.24.004655">10.1101/2020.03.24.004655</a></p>
</div>
<div id="ref-EWZmuLvu">
<p>642. <strong>A New Predictor of Disease Severity in Patients with COVID-19 in Wuhan, China</strong><br />
Ying Zhou, Zhen Yang, Yanan Guo, Shuang Geng, Shan Gao, Shenglan Ye, Yi Hu, Yafei Wang<br />
<em>medRxiv</em> (2020-03-27) <a href="https://doi.org/ggq65r">https://doi.org/ggq65r</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.24.20042119">10.1101/2020.03.24.20042119</a></p>
</div>
<div id="ref-95sQSGzx">
<p>643. <strong>Metabolic disturbances and inflammatory dysfunction predict severity of coronavirus disease 2019 (COVID-19): a retrospective study</strong><br />
Shuke Nie, Xueqing Zhao, Kang Zhao, Zhaohui Zhang, Zhentao Zhang, Zhan Zhang<br />
<em>medRxiv</em> (2020-03-26) <a href="https://doi.org/ggq65s">https://doi.org/ggq65s</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.24.20042283">10.1101/2020.03.24.20042283</a></p>
</div>
<div id="ref-btVuoX6V">
<p>644. <strong>Viral Kinetics and Antibody Responses in Patients with COVID-19</strong><br />
Wenting Tan, Yanqiu Lu, Juan Zhang, Jing Wang, Yunjie Dan, Zhaoxia Tan, Xiaoqing He, Chunfang Qian, Qiangzhong Sun, Qingli Hu, … Guohong Deng<br />
<em>medRxiv</em> (2020-03-26) <a href="https://doi.org/ggq65t">https://doi.org/ggq65t</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.24.20042382">10.1101/2020.03.24.20042382</a></p>
</div>
<div id="ref-LIf5SSUs">
<p>645. <strong>Global profiling of SARS-CoV-2 specific IgG/ IgM responses of convalescents using a proteome microarray</strong><br />
He-wei Jiang, Yang Li, Hai-nan Zhang, Wei Wang, Dong Men, Xiao Yang, Huan Qi, Jie Zhou, Sheng-ce Tao<br />
<em>medRxiv</em> (2020-03-27) <a href="https://doi.org/ggq65g">https://doi.org/ggq65g</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.20.20039495">10.1101/2020.03.20.20039495</a></p>
</div>
<div id="ref-1FZgghazg">
<p>646. <strong>COVID-19 infection induces readily detectable morphological and inflammation-related phenotypic changes in peripheral blood monocytes, the severity of which correlate with patient outcome</strong><br />
Dan Zhang, Rui Guo, Lei Lei, Hongjuan Liu, Yawen Wang, Yili Wang, Tongxin Dai, Tianxiao Zhang, Yanjun Lai, Jingya Wang, … Jinsong Hu<br />
<em>medRxiv</em> (2020-03-26) <a href="https://doi.org/ggq65v">https://doi.org/ggq65v</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.24.20042655">10.1101/2020.03.24.20042655</a></p>
</div>
<div id="ref-15gdkEnfn">
<p>647. <strong>Correlation between universal BCG vaccination policy and reduced morbidity and mortality for COVID-19: an epidemiological study</strong><br />
Aaron Miller, Mac Josh Reandelar, Kimberly Fasciglione, Violeta Roumenova, Yan Li, Gonzalo H Otazu<br />
<em>medRxiv</em> (2020-03-28) <a href="https://doi.org/ggq65w">https://doi.org/ggq65w</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.24.20042937">10.1101/2020.03.24.20042937</a></p>
</div>
<div id="ref-16FOT89K5">
<p>648. <strong>Non-specific effects of BCG vaccine on viral infections</strong><br />
S. J. C. F. M. Moorlag, R. J. W. Arts, R. van Crevel, M. G. Netea<br />
<em>Clinical Microbiology and Infection</em> (2019-12) <a href="https://doi.org/ggq62z">https://doi.org/ggq62z</a><br />
DOI: <a href="https://doi.org/10.1016/j.cmi.2019.04.020">10.1016/j.cmi.2019.04.020</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31055165">31055165</a></p>
</div>
<div id="ref-MCuIRGFE">
<p>649. <strong>BCG vaccination to reduce the impact of COVID-19 in healthcare workers (The BRACE Trial)</strong><br />
Murdoch Children’s Research Institute<br />
<a href="https://www.mcri.edu.au/BRACE">https://www.mcri.edu.au/BRACE</a></p>
</div>
<div id="ref-12oDb7aki">
<p>650. <strong>Non-neural expression of SARS-CoV-2 entry genes in the olfactory epithelium suggests mechanisms underlying anosmia in COVID-19 patients</strong><br />
David H. Brann, Tatsuya Tsukahara, Caleb Weinreb, Darren W. Logan, Sandeep Robert Datta<br />
<em>bioRxiv</em> (2020-03-28) <a href="https://doi.org/ggqr4m">https://doi.org/ggqr4m</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.25.009084">10.1101/2020.03.25.009084</a></p>
</div>
<div id="ref-t4vofknR">
<p>651. <strong>Cigarette smoke triggers the expansion of a subpopulation of respiratory epithelial cells that express the SARS-CoV-2 receptor ACE2</strong><br />
Joan C Smith, Jason Meyer Sheltzer<br />
<em>bioRxiv</em> (2020-03-31) <a href="https://doi.org/ggq65x">https://doi.org/ggq65x</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.28.013672">10.1101/2020.03.28.013672</a></p>
</div>
<div id="ref-k1E3DuFN">
<p>652. <strong>The comparative superiority of IgM-IgG antibody test to real-time reverse transcriptase PCR detection for SARS-CoV-2 infection diagnosis</strong><br />
Rui Liu, Xinghui Liu, Huan Han, Muhammad Adnan Shereen, Zhili Niu, Dong Li, Fang Liu, Kailang Wu, Zhen Luo, Chengliang Zhu<br />
<em>medRxiv</em> (2020-03-30) <a href="https://doi.org/ggqtp5">https://doi.org/ggqtp5</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.28.20045765">10.1101/2020.03.28.20045765</a></p>
</div>
<div id="ref-1IPKmXjW">
<p>653. <strong>Prediction of SARS-CoV-2 epitopes across 9360 HLA class I alleles</strong><br />
Katie M. Campbell, Gabriela Steiner, Daniel K. Wells, Antoni Ribas, Anusha Kalbasi<br />
<em>bioRxiv</em> (2020-04-01) <a href="https://doi.org/ggqr4n">https://doi.org/ggqr4n</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.30.016931">10.1101/2020.03.30.016931</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32511325">32511325</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7239055">PMC7239055</a></p>
</div>
<div id="ref-5OE9jzTg">
<p>654. <strong>A single-cell atlas of the peripheral immune response to severe COVID-19</strong><br />
Aaron J Wilk, Arjun Rustagi, Nancy Q Zhao, Jonasel Roque, Giovanny J Martinez-Colon, Julia L McKechnie, Geoffrey T Ivison, Thanmayi Ranganath, Rosemary Vergara, Taylor Hollis, … Catherine A Blish<br />
<em>medRxiv</em> (2020-04-23) <a href="https://doi.org/ggvrxm">https://doi.org/ggvrxm</a><br />
DOI: <a href="https://doi.org/10.1101/2020.04.17.20069930">10.1101/2020.04.17.20069930</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32511639">32511639</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7276995">PMC7276995</a></p>
</div>
</div>
<!-- Define citation tags below -->
<h1 class="page_break_before" data-number="9" id="appendix-1"><span class="header-section-number">9</span> Appendix 1</h1>
<p>This appendix contains reviews produced by the Immunology Institute of the Icahn School of Medicine</p>
<!-- Appendix source: https://github.com/ismms-himc/covid-19_sinai_reviews/tree/05d387bf10b829dad6bedb4bf313686750b78cc9 -->
<h2 data-number="9.1" id="potent-binding-of-2019-novel-coronavirus-spike-protein-by-a-sars-coronavirus-specific-human-monoclonal-antibody"><span class="header-section-number">9.1</span> Potent binding of 2019 novel coronavirus spike protein by a SARS coronavirus-specific human monoclonal antibody</h2>
<p>Tian et al. <em>Emerg Microbes Infect 2020</em> <span class="citation" data-cites="1HLSeqnY8">[<a href="#ref-1HLSeqnY8" role="doc-biblioref">530</a>]</span></p>
<h3 data-number="9.1.1" id="keywords"><span class="header-section-number">9.1.1</span> Keywords</h3>
<ul>
<li>Monoclonal antibody</li>
<li>Cross-reactivity</li>
<li>receptor binding domain</li>
</ul>
<h3 data-number="9.1.2" id="summary-1"><span class="header-section-number">9.1.2</span> Summary</h3>
<p>Considering the relatively high identity of the receptor binding domain
(RBD) of the spike proteins from 2019-nCoV and SARS-CoV (73%), this
study aims to assess the cross-reactivity of several anti-SARS-CoV
monoclonal antibodies with 2019-nCoV. The results showed that the
SARS-CoV-specific antibody CR3022 can potently bind 2019-nCoV RBD.</p>
<h3 data-number="9.1.3" id="main-findings"><span class="header-section-number">9.1.3</span> Main Findings</h3>
<p>The structure of the 2019-nCoV spike RBD and its conformation in complex
with the receptor angiotensin-converting enzyme (ACE2) was modeled <em>in
silico</em> and compared with the SARS-CoV RBD structure. The models
predicted very similar RBD-ACE2 interactions for both viruses. The
binding capacity of representative SARS-CoV-RBD specific monoclonal
antibodies (m396, CR3014, and CR3022) to recombinant 2019-nCoV RBD was
then investigated by ELISA and their binding kinetics studied using
biolayer interferometry. The analysis showed that only CR3022 was able
to bind 2019-nCoV RBD with high affinity (KD of 6.3 nM), however it did
not interfere with ACE2 binding. Antibodies m396 and CR3014, which
target the ACE2 binding site of SARS-CoV failed to bind 2019-nCoV spike
protein.</p>
<h3 data-number="9.1.4" id="limitations"><span class="header-section-number">9.1.4</span> Limitations</h3>
<p>The 2019-nCoV RBD largely differ from the SARS-CoV at the C-terminus
residues, which drastically impact the cross-reactivity of antibodies
described for other B beta-coronaviruses, including SARS-CoV. This study
claims that CR3022 antibody could be a potential candidate for therapy.
However, none of the antibodies assayed in this work showed
cross-reactivity with the ACE2 binding site of 2019-nCoV, essential for
the replication of this virus. Furthermore, neutralization assays with
2019-nCoV virus or pseudovirus were not performed. Although the use of
neutralizing antibodies is an interesting approach, these results
suggest that it is critical the development of novel monoclonal
antibodies able to specifically bind 2019-nCoV spike protein.</p>
<h3 data-number="9.1.5" id="credit"><span class="header-section-number">9.1.5</span> Credit</h3>
<p><em>Review by D.L.O as part of a project by students, postdocs and faculty
at the Immunology Institute of the Icahn School of Medicine, Mount
Sinai.</em></p>
<h2 data-number="9.2" id="integrative-bioinformatics-analysis-provides-insight-into-the-molecular-mechanisms-of-2019-ncov"><span class="header-section-number">9.2</span> Integrative Bioinformatics Analysis Provides Insight into the Molecular Mechanisms of 2019-nCoV</h2>
<p>He et al. <em>medRxiv</em> <span class="citation" data-cites="lOQPScZt">[<a href="#ref-lOQPScZt" role="doc-biblioref">531</a>]</span></p>
<h3 data-number="9.2.1" id="keywords-1"><span class="header-section-number">9.2.1</span> Keywords</h3>
<ul>
<li>ACE2</li>
<li>lungs</li>
<li>smoking</li>
<li>COPD</li>
<li>asthma</li>
<li>SARS-Cov</li>
<li>IL-1</li>
<li>IL-10</li>
<li>IL-6</li>
<li>IL-8</li>
</ul>
<h3 data-number="9.2.2" id="main-findings-1"><span class="header-section-number">9.2.2</span> Main Findings</h3>
<p>The authors used bioinformatics tools to identify features
of ACE2 expression in the lungs of different patent groups: healthy,
smokers, patients with chronic airway disease (i.e., COPD) or asthma.
They used gene expression data publicly available from GEO that included
lung tissues, bronchoalveolar lavage, bronchial epithelial cells, small
airway epithelial cells, or SARS-Cov infected cells.</p>
<p>The authors describe no significant differences in ACE2 expression in
lung tissues of Healthy, COPD, and Asthma groups (p=0.85); or in BAL of
Healthy and COPD (p=0.48); or in epithelial brushings of Healthy and
Mild/Moderate/Severe Asthma (p=0.99). ACE2 was higher in the small
airway epithelium of long-term smokers vs non-smokers (p&lt;0.001).
Consistently, there was a trend of higher ACE2 expression in the
bronchial airway epithelial cells 24h post-acute smoking exposure
(p=0.073). Increasing ACE2 expression at 24h and 48h compared to 12h
post SARS-Cov infection (p=0.026; n=3 at each time point) was also
detected.</p>
<p>15 lung samples’ data from healthy participants were separated into high
and low ACE2 expression groups. “High” ACE2 expression was associated
with the following GO pathways: innate and adaptive immune responses, B
cell mediated immunity, cytokine secretion, and IL-1, IL-10, IL-6, IL-8
cytokines. The authors speculate that a high basal ACE2 expression will
increase susceptibility to SARS-CoV infection.</p>
<p>In 3 samples SARS-Cov infection was associated with IL-1, IL-10 and IL-6
cytokine production (GO pathways) at 24h. And later, at 48h, with T-cell
activation and T-cell cytokine production. It is unclear whether those
changes were statistically significant.</p>
<p>The authors describe a time course quantification of immune infiltrates
in epithelial cells infected with SARS-Cov infection. They state that in
healthy donors ACE2 expression did not correlate with the immune cell
infiltration. However, in SARS-Cov samples, at 48h they found that ACE2
correlated with neutrophils, NK-, Th17-, Th2-, Th1- cells, and DCs.
Again, while authors claim significance, the corresponding correlation
coefficients and p-values are not presented in the text or figures. In
addition, the source of the data for this analysis is not clear.</p>
<p>Using network analysis, proteins SRC, FN1, MAPK3, LYN, MBP, NLRC4, NLRP1
and PRKCD were found to be central (Hub proteins) in the regulating
network of cytokine secretion after coronavirus infection. Authors
conclude this indicates that these molecules were critically important
in ACE2-induced inflammatory response. Additionally, authors speculate
that the increased expression of ACE2 affected RPS3 and SRC, which were
the two hub genes involved in viral replication and inflammatory
response.</p>
<h3 data-number="9.2.3" id="limitations-1"><span class="header-section-number">9.2.3</span> Limitations</h3>
<p>The methods section is very limited and does not
describe any of the statistical analyses; and description of the
construction of the regulatory protein networks is also limited. For the
findings in Figures 2 authors claim significance, which is not supported
by p-values or coefficients. For the sample selection, would be useful
if sample sizes and some of the patients’ demographics (e.g. age) were
described.</p>
<p>For the analysis of high vs low ACE2 expression in healthy subjects, it
is not clear what was the cut off for ‘high’ expression and how it was
determined. Additionally, further laboratory studies are warranted to
confirm that high ACE2 gene expression would have high correlation with
the amount of ACE2 protein on cell surface. For the GO pathway analysis
significance was set at p&lt;0.05, but not adjusted for multiple
comparisons.</p>
<p>There were no samples with SARS-CoV-2 infection. While SARS-Cov and
SARS-CoV-2 both use ACE2 to enter the host cells, the analysis only
included data on SARS-Cov and any conclusions about SARS-CoV2 are
limited.</p>
<p>Upon checking GSE accession numbers of the datasets references, two
might not be cited correctly: GSE37758 (“A spergillus niger: Control
(fructose) vs. steam-exploded sugarcane induction (SEB)”" was used in
this paper as “lung tissue” data) and GSE14700 (“Steroid Pretreatment
of Organ Donors to Prevent Postischemic Renal Allograft Failure: A
Randomized, Controlled Trial” – was used as SARS-Cov infection data).</p>
<h3 data-number="9.2.4" id="credit-1"><span class="header-section-number">9.2.4</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.3" id="diarrhea-may-be-underestimated-a-missing-link-in-2019-novel-coronavirus"><span class="header-section-number">9.3</span> Diarrhea may be underestimated: a missing link in 2019 novel coronavirus</h2>
<p>Liang et al. <em>medRxiv</em> <span class="citation" data-cites="rL9IRhVy">[<a href="#ref-rL9IRhVy" role="doc-biblioref">532</a>]</span></p>
<h3 data-number="9.3.1" id="keywords-2"><span class="header-section-number">9.3.1</span> Keywords</h3>
<ul>
<li>SARS-CoV-2</li>
<li>diarrhea</li>
<li>ACE2</li>
<li>scRNA-seq</li>
</ul>
<h3 data-number="9.3.2" id="main-findings-2"><span class="header-section-number">9.3.2</span> Main Findings</h3>
<p>This study examined the incidence of diarrhea in
patients infected with SARS-CoV-2 across three recently published
cohorts and found that there are statistically significant differences
by Fisher’s exact test. They report that this could be due to subjective
diagnosis criterion for diarrhea or from patients first seeking medical
care from gastroenterologist. In order to minimize nosocomial infections
arising from unsuspected patients with diarrhea and gain comprehensive
understanding of transmission routes for this viral pathogen, they
compared the transcriptional levels of ACE2 of various human tissues
from NCBI public database as well as in small intestine tissue from
CD57BL/6 mice using single cell sequencing. They show that ACE2
expression is not only increased in the human small intestine, but
demonstrate a particular increase in mice enterocytes positioned on the
surface of the intestinal lining exposed to viral pathogens. Given that
ACE2 is the viral receptor for SARS-CoV-2 and also reported to regulate
diarrhea, their data suggests the small intestine as a potential
transmission route and diarrhea as a potentially underestimated symptom
in COVID19 patients that must be carefully monitored. Interestingly,
however, they show that ACE2 expression level is not elevated in human
lung tissue.</p>
<h3 data-number="9.3.3" id="limitations-2"><span class="header-section-number">9.3.3</span> Limitations</h3>
<p>Although this study demonstrates a
statistical difference in the incidence of diarrhea across three
separate COVID19 patient cohorts, their conclusions are limited by a
small sample size. Specifically, the p-value computed by Fisher’s exact
test is based on a single patient cohort of only six cases of which 33%
are reported to have diarrhea, while the remaining two larger cohorts
with 41 and 99 cases report 3% and 2% diarrhea incidence, respectively.
Despite showing significance, they would need to acquire larger sample
sizes and cohorts to minimize random variability and draw meaningful
conclusions. Furthermore, they do not address why ACE2 expression level
is not elevated in human lung tissue despite it being a major
established route of transmission for SARS-CoV-2. It could be helpful to
validate this result by looking at ACE2 expression in mouse lung tissue.
Finally, although this study is descriptive and shows elevated ACE2
expression in small intestinal epithelial cells, it does not establish a
mechanistic link to SARS-CoV-2 infection of the host. Overall, their
claim that infected patients exhibiting diarrhea pose an increased risk
to hospital staff needs to be further substantiated.</p>
<h3 data-number="9.3.4" id="significance"><span class="header-section-number">9.3.4</span> Significance</h3>
<p>This study provides a possible transmission route and a
potentially underappreciated clinical symptom for SARS-CoV-2 for better
clinical management and control of COVID19.</p>
<h3 data-number="9.3.5" id="credit-2"><span class="header-section-number">9.3.5</span> Credit</h3>
<p><em>Summary generated as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.4" id="specific-ace2-expression-in-cholangiocytes-may-cause-liver-damage-after-2019-ncov-infection"><span class="header-section-number">9.4</span> Specific ACE2 Expression in Cholangiocytes May Cause Liver Damage After 2019-nCoV Infection</h2>
<p>Chai et al. <em>bioRxiv</em> <span class="citation" data-cites="BHQvMYPG">[<a href="#ref-BHQvMYPG" role="doc-biblioref">533</a>]</span></p>
<h3 data-number="9.4.1" id="keywords-3"><span class="header-section-number">9.4.1</span> Keywords</h3>
<ul>
<li>ACE2</li>
<li>Cholangiocytes</li>
<li>COVID-associated Liver Damage</li>
</ul>
<h3 data-number="9.4.2" id="summary-2"><span class="header-section-number">9.4.2</span> Summary</h3>
<p>Using both publicly available scRNA-seq dataset of liver
samples from colorectal patients and scRNA-sequencing of four liver
samples from healthy volunteers, the authors show that ACE2 is
significantly enriched in the majority of cholangiocytes (59.7 %) but
not in hepatocytes (2.6%).</p>
<h3 data-number="9.4.3" id="main-findings-3"><span class="header-section-number">9.4.3</span> Main Findings</h3>
<p>Using bioinformatics approaches of
RNASeq analysis, this study reveals that ACE2 dominates in
cholangiocytes and is present at very low levels in hepatocytes.</p>
<h3 data-number="9.4.4" id="limitations-3"><span class="header-section-number">9.4.4</span> Limitations</h3>
<p>The
study does not provide mechanistic insights into how SARS-CoV-2 can
infect and replicate in cholangiocytes and the types of intrinsic
anti-viral responses induced by cholangiocytes when infected. In
addition, because the study relies on the assumption that SARS-CoV-2
infects cells only through ACE2, it cannot discount the possibility that
the virus can infect hepatocytes through mechanisms other than
ACE2-mediated entry. Furthermore, because the scRNA-seq analysis were
performed on healthy liver samples, one cannot draw any definitive
conclusions about gene expression states (including ACE2 expression in
liver cell types) in system-wide inflammatory contexts.</p>
<h3 data-number="9.4.5" id="significance-1"><span class="header-section-number">9.4.5</span> Significance</h3>
<p>This article with other studies on liver
damage in COVID patients suggests that liver damage observed in COVID
patients is more due to inflammatory cytokines than direct infection of
the liver. Even if cholangiocytes are infectable by SARS-CoV-2 (which
was demonstrated by human liver ductal organoid study
(<span class="citation" data-cites="OuDMtCRl">[<a href="#ref-OuDMtCRl" role="doc-biblioref">534</a>]</span>),
published clinical data show no significant increase in bile duct injury
related indexes (i.e. alkaline phosphatase, gamma-glutamyl
transpeptidase and total bilirubin). In sum, it underscores the
importance of future studies characterizing cellular responses of
extra-pulmonary organs in the context of COVID or at least in viral lung
infections..</p>
<h3 data-number="9.4.6" id="credit-3"><span class="header-section-number">9.4.6</span> Credit</h3>
<p><em>Summary generated by Chang Moon as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.5" id="ace2-expression-by-colonic-epithelial-cells-is-associated-with-viral-infection-immunity-and-energy-metabolism"><span class="header-section-number">9.5</span> ACE2 expression by colonic epithelial cells is associated with viral infection, immunity and energy metabolism</h2>
<p>Wang et al. <em>medRxiv</em>. <span class="citation" data-cites="ViWCb8MJ">[<a href="#ref-ViWCb8MJ" role="doc-biblioref">535</a>]</span></p>
<h3 data-number="9.5.1" id="keywords-4"><span class="header-section-number">9.5.1</span> Keywords</h3>
<ul>
<li>single cell RNA seq</li>
<li>ACE2 expression</li>
<li>human colonic biopsy</li>
</ul>
<h3 data-number="9.5.2" id="main-findings-4"><span class="header-section-number">9.5.2</span> Main Findings</h3>
<p>Colonic enterocytes primarily express ACE2. Cellular pathways associated with ACE2 expression include innate immune signaling, HLA up regulation, energy metabolism and apoptotic signaling.</p>
<h3 data-number="9.5.3" id="limitations-4"><span class="header-section-number">9.5.3</span> Limitations</h3>
<p>This is a study of colonic biopsies taken from 17 children with and without IBD and analyzed using scRNAseq to look at ACE2 expression and identify gene families correlated with ACE2 expression. The authors find ACE2 expression to be primarily in colonocytes. It is not clear why both healthy and IBD patients were combined for the analysis. Biopsies were all of children so extrapolation to adults is limited. The majority of genes found to be negatively correlated with ACE2 expression include immunoglobulin genes (IGs). IG expression will almost certainly be low in colonocytes irrespective of ACE2 expression.</p>
<h3 data-number="9.5.4" id="significance-2"><span class="header-section-number">9.5.4</span> Significance</h3>
<p>This study performs a retrospective analysis of ACE2 expression using an RNAseq dataset from intestinal biopsies of children with and without IBD. The implications for the CoV-19 epidemic are modest, but do provide support that ACE2 expression is specific to colonocytes in the intestines. The ontological pathway analysis provides some limited insights into gene expression associated with ACE2.</p>
<h3 data-number="9.5.5" id="credit-4"><span class="header-section-number">9.5.5</span> Credit</h3>
<p><em>Summary generated as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.6" id="the-pathogenicity-of-2019-novel-coronavirus-in-hace2-transgenic-mice"><span class="header-section-number">9.6</span> The Pathogenicity of 2019 Novel Coronavirus in hACE2 Transgenic Mice</h2>
<p>Bao et al. <em>bioRxiv</em> <span class="citation" data-cites="MgLBA5m">[<a href="#ref-MgLBA5m" role="doc-biblioref">536</a>]</span></p>
<h3 data-number="9.6.1" id="keywords-5"><span class="header-section-number">9.6.1</span> Keywords</h3>
<ul>
<li>Covid-19 mouse model</li>
<li>hACE2 mice</li>
<li>2019-nCoV model</li>
<li>ACE2</li>
<li>2019-nCoV </li>
</ul>
<h3 data-number="9.6.2" id="main-findings-5"><span class="header-section-number">9.6.2</span> Main Findings</h3>
<p>Using a transgenic human Angiotensin-converting enzyme 2
(hACE2) mouse that has previously been shown susceptible to infection by
SARS-CoV, Bao et al. create a model of pandemic 2019-nCoV strain
coronavirus. The model includes interstitial hyperplasia in lung tissue,
moderate inflammation in bronchioles and blood vessels, and histology
consistent with viral pneumonia at 3 days post infection. Wildtype did
not experience these symptoms. In addition, viral antigen and hACE2
receptor were found to co-localize the lung by immunofluorescence 3-10
days post infection only in the hACE2 infected mice.</p>
<h3 data-number="9.6.3" id="limitations-5"><span class="header-section-number">9.6.3</span> Limitations</h3>
<p>The characterization of the infection remains incomplete,
as well as lacking characterization of the immune response other than
the presence of a single antiviral antibody. Though they claim to
fulfill Koch’s postulates, they only isolate the virus and re-infect
Vero cells, rather than naive mice.</p>
<h3 data-number="9.6.4" id="significance-3"><span class="header-section-number">9.6.4</span> Significance</h3>
<p>This paper establishes a murine model for 2019-nCoV
infection with symptoms consistent with viral pneumonia. Though not
fully characterized, this model allows in vivo analysis of viral entry
and pathology that is important for the development of vaccines and
antiviral therapeutics.</p>
<h3 data-number="9.6.5" id="credit-5"><span class="header-section-number">9.6.5</span> Credit</h3>
<p><em>Review by Dan Fu Ruan, Evan Cody and Venu Pothula as part of a project by students, postdocs and faculty
at the Immunology Institute of the Icahn School of Medicine, Mount
Sinai.</em></p>
<h2 data-number="9.7" id="caution-on-kidney-dysfunctions-of-2019-ncov-patients"><span class="header-section-number">9.7</span> Caution on Kidney Dysfunctions of 2019-nCoV Patients</h2>
<p>Li et al. <em>medRxiv</em>. <span class="citation" data-cites="1Acv5UMSt">[<a href="#ref-1Acv5UMSt" role="doc-biblioref">537</a>]</span></p>
<h3 data-number="9.7.1" id="keywords-6"><span class="header-section-number">9.7.1</span> Keywords</h3>
<p>CoVID-19, 2019-nCoV, SARS-CoV-2, kidney, clinical, creatinine,
proteinuria, albuminuria, CT</p>
<h3 data-number="9.7.2" id="main-findings-6"><span class="header-section-number">9.7.2</span> Main Findings</h3>
<ul>
<li><p>Retrospective study of 59 patients assayed key function indicators
of the kidney–including urine protein, blood urea nitrogen (BUN),
plasma creatinine (Cre), and renal CT scan data.</p></li>
<li><p>Found that 34% of patients developed massive albuminuria on the
first day of admission, and 63% developed proteinuria during their
stay in hospital; and 19% of patients had high plasma creatinine,
especially the terminal cases.</p></li>
<li><p>CT analyses of 27 patients showed all patients to have abnormal
kidney damage; indicate that inflammation and edema of the renal
parenchyma very common.</p></li>
</ul>
<h3 data-number="9.7.3" id="limitations-6"><span class="header-section-number">9.7.3</span> Limitations</h3>
<ul>
<li><p>No analysis of immunity-dependent damage and cytokines in
blood/plasma/urine. Will be worth correlating disease progression
with cytokine production, immune activity and kidney function.</p></li>
<li><p>Extrapolating to earlier SARS-CoV studies provides the only
rationale for viral-damage in kidney and resultant pathologic immune
response <em>(understandable for this clinical study)</em>.</p></li>
</ul>
<h3 data-number="9.7.4" id="significance-4"><span class="header-section-number">9.7.4</span> Significance</h3>
<ul>
<li><p>Multiple lines of evidence along this study’s finding point to the
idea that renal impairment/injury is a key risk factor in 2019-nCoV
patients similar to what has been reported for SARS-CoV<span class="citation" data-cites="DMaiZBE1">[<a href="#ref-DMaiZBE1" role="doc-biblioref">538</a>]</span>; this
may be one of the major causes of virally-induced damage and
contribute to multiorgan failure.</p></li>
<li><p>ACE2 expression in kidney proximal tubule epithelia and bladder
epithelia <span class="citation" data-cites="16Xj27QYj">[<a href="#ref-16Xj27QYj" role="doc-biblioref">539</a>]</span> support
these clinical findings.</p></li>
<li><p>Study argues for closely monitoring kidney function, and applying
potential interventions including continuous renal replacement
therapies (CRRT) for protecting kidney functions as early as
possible, particularly for those with rising plasma creatinine.</p></li>
</ul>
<h3 data-number="9.7.5" id="credit-6"><span class="header-section-number">9.7.5</span> Credit</h3>
<p><em>Review by Samarth Hegde as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.8" id="profiling-the-immune-vulnerability-landscape-of-the-2019-novel-coronavirus"><span class="header-section-number">9.8</span> Profiling the immune vulnerability landscape of the 2019 Novel Coronavirus</h2>
<p>Zhu et al. <em>bioRxiv</em> <span class="citation" data-cites="QhE5mVCc">[<a href="#ref-QhE5mVCc" role="doc-biblioref">540</a>]</span></p>
<h3 data-number="9.8.1" id="keywords-7"><span class="header-section-number">9.8.1</span> Keywords</h3>
<ul>
<li>epitope prediction</li>
<li>vaccine development.</li>
</ul>
<h3 data-number="9.8.2" id="main-findings-7"><span class="header-section-number">9.8.2</span> Main Findings</h3>
<p>This study harnesses bioinformatic profiling to predict the potential of
COV2 viral proteins to be presented on MHC I and II and to form linear
B-cell epitopes. These estimates suggest a T-cell antigenic profile
distinct from SARS-CoV or MERS-CoV, identify focused regions of the
virus with a high density of predicted epitopes, and provide preliminary
evidence for adaptive immune pressure in the genetic evolution of the
virus.</p>
<h3 data-number="9.8.3" id="limitations-7"><span class="header-section-number">9.8.3</span> Limitations</h3>
<p>While the study performs a comprehensive analysis of potential epitopes
within the virus genome, the analysis relies solely on bioinformatic
prediction to examine MHC binding affinity and B-cell epitope potential
and does not capture the immunogenicity or recognition of
these epitopes. Future experimental validation in data from patients
infected with SARS-CoV-2 will be important to validate and refine these
findings. Thus some of the potential conclusions stated, including viral
evolution toward lower immunogenicity or a dominant role for CD4+
T-cells rather than CD8+ T-cells in viral clearance, require further
valiadtion.</p>
<h3 data-number="9.8.4" id="significance-5"><span class="header-section-number">9.8.4</span> Significance</h3>
<p>These findings may help direct peptide vaccine design toward relevant
epitopes and provide intriguing evidence of viral evolution in response
to immune pressure.</p>
<h3 data-number="9.8.5" id="credit-7"><span class="header-section-number">9.8.5</span> Credit</h3>
<p><em>Summary generated as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.9" id="single-cell-analysis-of-ace2-expression-in-human-kidneys-and-bladders-reveals-a-potential-route-of-2019-ncov-infection"><span class="header-section-number">9.9</span> Single-cell Analysis of ACE2 Expression in Human Kidneys and Bladders Reveals a Potential Route of 2019-nCoV Infection</h2>
<p>Lin et al. <em>bioRxiv</em> <span class="citation" data-cites="16Xj27QYj">[<a href="#ref-16Xj27QYj" role="doc-biblioref">539</a>]</span></p>
<h3 data-number="9.9.1" id="keywords-8"><span class="header-section-number">9.9.1</span> Keywords</h3>
<ul>
<li>CoVID-19</li>
<li>2019-nCoV</li>
<li>SARS-CoV-2</li>
<li>ACE2</li>
<li>scRNAseq</li>
<li>kidney</li>
<li>bladder</li>
<li>public dataset</li>
</ul>
<h3 data-number="9.9.2" id="main-findings-8"><span class="header-section-number">9.9.2</span> Main Findings</h3>
<ul>
<li><p>To investigate the possible cause of kidney damage in 2019-nCoV
patients, authors used published kidney and bladder cell atlas data
(GSE131685, GSE108097; 3 healthy donors each) as well as an
unpublished kidney single-cell RNA-Seq data (in-house from 2
transplant donors) to evaluate ACE2 gene expressions in all cell
types of healthy kidneys and bladders.</p></li>
<li><p>They find enriched expression of ACE2 transcript in all subtypes of
proximal tubule cells of kidney, with 5%-15% of both straight and
convoluted proximal tubule cells expressing ACE2.</p></li>
<li><p>They also find detectable levels of ACE2 in bladder epithelial
cells, noting expression from around 1.5% of cells in the outer
layer umbrella cells of the bladder epithelium and decreasing in the
basal cells.</p></li>
<li><p>Importantly endothelial or immune cells in kidney/bladder do not
express ACE2.</p></li>
</ul>
<h3 data-number="9.9.3" id="limitations-8"><span class="header-section-number">9.9.3</span> Limitations</h3>
<ul>
<li><p>This study primarily characterizes ACE2 expression (amongst other
genes) from a small healthy-donor dataset, and will benefit from
supporting data in (expired) patient samples to show functional
viral damage. ACE2 transcript does not necessarily translate to
viral permissiveness in kidney/bladder epithelia or cytokine
release.</p></li>
<li><p>This study focuses on only healthy tissue; it will be useful to
analyze kidney/bladder epithelial ACE2 expression under inflammatory
conditions or in patients with underlying kidney conditions.</p></li>
<li><p>Given what is known about protease TMPRSS2 expression during
SARS-CoV-2 infection, ACE2+TMPRSS2+ double-positive cell
identification would be useful in these datasets.</p></li>
</ul>
<h3 data-number="9.9.4" id="significance-6"><span class="header-section-number">9.9.4</span> Significance</h3>
<ul>
<li><p>ACE2 protein is spatially restricted to brush border of proximal
tubules and in bladder umbrella cells <span class="citation" data-cites="1FE1YGsaV">[<a href="#ref-1FE1YGsaV" role="doc-biblioref">541</a>]</span>, such cells in direct
contact with viral particles are likely to be highly sensitive to
viral-induced damage.</p></li>
<li><p>SARS-CoV and MERS-CoV have been shown to be detected in urine of
patients and associate with higher mortality <span class="citation" data-cites="DMaiZBE1 iObQU2el">[<a href="#ref-DMaiZBE1" role="doc-biblioref">538</a>,<a href="#ref-iObQU2el" role="doc-biblioref">542</a>]</span>, thus worth
understanding kidney damage and resultant immune response in
SARS-CoV-2 as well.</p></li>
<li><p>This study argues for a potential mode of viral infectivity and
resultant inflammatory responses in these tissue in addition to
reported infectivity in the lung and digestive system, which is
supported by clinical data showing acute and early kidney
complications in 2019-nCoV patients <span class="citation" data-cites="1Acv5UMSt">[<a href="#ref-1Acv5UMSt" role="doc-biblioref">537</a>]</span>.</p></li>
<li><p>Clinically, thus very important to track urinary CoVID-19 shedding
as well as study acute kidney injury-related co-morbidities.</p></li>
</ul>
<h3 data-number="9.9.5" id="credit-8"><span class="header-section-number">9.9.5</span> Credit</h3>
<p><em>Review by Samarth Hegde as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.10" id="neutrophil-to-lymphocyte-ratio-predicts-severe-illness-patients-with-2019-novel-coronavirus-in-the-early-stage"><span class="header-section-number">9.10</span> Neutrophil-to-Lymphocyte Ratio Predicts Severe Illness Patients with 2019 Novel Coronavirus in the Early Stage</h2>
<p>Liu et al. <em>medRxiv</em> <span class="citation" data-cites="gK5e7Yhl">[<a href="#ref-gK5e7Yhl" role="doc-biblioref">543</a>]</span></p>
<h3 data-number="9.10.1" id="keywords-9"><span class="header-section-number">9.10.1</span> Keywords</h3>
<ul>
<li>severe disease</li>
<li>pneumonia</li>
<li>lymphocytes</li>
<li>neutrophils</li>
</ul>
<h3 data-number="9.10.2" id="main-findings-9"><span class="header-section-number">9.10.2</span> Main Findings</h3>
<p>This study aimed to find prognostic biomarkers of COVID-19
pneumonia severity. Sixty-one (61) patients with COVID-19 treated in
January at a hospital in Beijing, China were included. On average,
patients were seen within 5 days from illness onset. Samples were
collected on admission; and then patients were monitored for the
development of severe illness with a median follow-up of 10 days].</p>
<p>Patients were grouped as “mild” (N=44) or “moderate/severe” (N=17)
according to symptoms on admission and compared for different
clinical/laboratory features. “Moderate/severe” patients were
significantly older (median of 56 years old, compared to 41 years old).
Whereas comorbidies rates were largely similar between the groups,
except for hypertension, which was more frequent in the severe group (p=
0.056). ‘Severe’ patients had higher counts of neutrophils, and serum
glucose levels; but lower lymphocyte counts, sodium and serum chlorine
levels. The ratio of neutrophils to lymphocytes (NLR) was also higher
for the ‘severe’ group. ‘Severe’ patients had a higher rate of bacterial
infections (and antibiotic treatment) and received more intensive
respiratory support and treatment.</p>
<p>26 clinical/laboratory variables were used to select NLR and age as the
best predictors of the severe disease. Predictive cutoffs for a severe
illness as NLR ≥ 3.13 or age ≥ 50 years.</p>
<h3 data-number="9.10.3" id="limitations-9"><span class="header-section-number">9.10.3</span> Limitations</h3>
<p>Identification of early biomarkers is important for
making clinical decisions, but large sample size and validation cohorts
are necessary to confirm findings. It is worth noting that patients
classified as “mild” showed pneumonia by imaging and fever, and in
accordance with current classifications this would be consistent with
“moderate” cases. Hence it would be more appropriate to refer to the
groups as “moderate” vs “severe/critical”. Furthermore, there are
several limitations that could impact the interpretation of the results:
e.g. classification of patients was based on symptoms presented on
admission and not based on disease progression, small sample size,
especially the number of ‘severe’ cases (with no deaths among these
patients). Given the small sample size, the proposed NLR and age cut
offs might not hold for a slightly different set of patients. For
example, in a study of &gt;400 patients, ‘non-severe’ and ‘severe’ NLR
were 3.2 and 5.5, respectively <span class="citation" data-cites="6Y81C4P3">[<a href="#ref-6Y81C4P3" role="doc-biblioref">544</a>]</span>.</p>
<h3 data-number="9.10.4" id="credit-9"><span class="header-section-number">9.10.4</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.11" id="characteristics-of-lymphocyte-subsets-and-cytokines-in-peripheral-blood-of-123-hospitalized-patients-with-2019-novel-coronavirus-pneumonia-ncp"><span class="header-section-number">9.11</span> Characteristics of lymphocyte subsets and cytokines in peripheral blood of 123 hospitalized patients with 2019 novel coronavirus pneumonia (NCP)</h2>
<p>Wan et. al. <em>medRxiv</em> <span class="citation" data-cites="P4assQHj">[<a href="#ref-P4assQHj" role="doc-biblioref">545</a>]</span></p>
<h3 data-number="9.11.1" id="keywords-10"><span class="header-section-number">9.11.1</span> Keywords</h3>
<ul>
<li>Cytokines</li>
<li>lymphocyte subsets</li>
<li>CD8 + T</li>
<li>B cells</li>
<li>NK cells,</li>
<li>PBMCs</li>
<li>IL-6</li>
<li>IL-10</li>
</ul>
<h3 data-number="9.11.2" id="main-findings-10"><span class="header-section-number">9.11.2</span> Main Findings</h3>
<p>The authors analyzed lymphocyte subsets and cytokines of 102 patients
with mild disease and 21 with severe disease. CD8+T cells and CD4+T
cells were significantly reduced in both cohort. particularly in severe
patients. The cytokines IL6 and IL10 were significantly elevated in
severe patients as compared to mild. No significant differences were
observed in frequency of B cells and NK cells.</p>
<p>The authors argue that the measurement of T cell frequencies and
cytokine levels of IL6 and IL10 can be used to predict progression of
disease from Mild to severe Cov-2 infection.</p>
<h3 data-number="9.11.3" id="limitations-10"><span class="header-section-number">9.11.3</span> Limitations</h3>
<p>The study demonstrates in a limited cohort
similar associations to several other reported studies. The authors
didn’t compare the changes in lymphocyte and cytokine with healthy
individual (Covid-19 Negative) rather used an internal standard value.
The recently preprint in LANCET shows The degree of lymphopenia and a
pro-inflammatory cytokine storm is higher in severe COVID-19 patients
than in mild cases, and is associated with the disease severity <span class="citation" data-cites="zskgSnle">[<a href="#ref-zskgSnle" role="doc-biblioref">546</a>]</span>.</p>
<h3 data-number="9.11.4" id="significance-7"><span class="header-section-number">9.11.4</span> Significance</h3>
<p>This translational data identifies key cytokines and lymphopenia
associated with disease severity although mechanism and key cellular
players are still unknown. Higher level IL-6 production in severe
patient suggests potential role of Tocilizumab (anti-IL6R) biologic
although clinical trial will be necessary.</p>
<h3 data-number="9.11.5" id="credit-10"><span class="header-section-number">9.11.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.12" id="epidemiological-and-clinical-characteristics-of-17-hospitalized-patients-with-2019-novel-coronavirus-infections-outside-wuhan-china"><span class="header-section-number">9.12</span> Epidemiological and Clinical Characteristics of 17 Hospitalized Patients with 2019 Novel Coronavirus Infections Outside Wuhan, China</h2>
<p>Li et al. <em>medRxiv</em> <span class="citation" data-cites="TXlNzmXd">[<a href="#ref-TXlNzmXd" role="doc-biblioref">547</a>]</span></p>
<h3 data-number="9.12.1" id="keywords-11"><span class="header-section-number">9.12.1</span> Keywords</h3>
<ul>
<li>epidemiology</li>
<li>clinical characteristics</li>
</ul>
<h3 data-number="9.12.2" id="major-findings"><span class="header-section-number">9.12.2</span> Major Findings</h3>
<p>These authors looked at 17 hospitalized patients with COVID-19
confirmed by RT-PCR in Dazhou, Sichuan. Patients were admitted between
January 22 and February 10 and the final data were collected on February
11. Of the 17 patients, 12 remained hospitalized while 5 were discharged
after meeting national standards. The authors observed no differences
based on the sex of the patients but found that the discharged patients
were younger in age (p = 0.026) and had higher lymphocyte counts (p =
0.005) and monocyte counts (p = 0.019) upon admission.</p>
<h3 data-number="9.12.3" id="limitations-11"><span class="header-section-number">9.12.3</span> Limitations</h3>
<p>This study is limited in the sample size of the study and the last
data collection point was only one day after some of the patients were
admitted.</p>
<h3 data-number="9.12.4" id="significance-8"><span class="header-section-number">9.12.4</span> Significance</h3>
<p>These findings have been somewhat supported by subsequent studies
that show that older age and an immunocompromised state are more likely
to result in a more severe clinical course with COVID-19. However, other
studies have been published that report on larger numbers of cases.</p>
<h3 data-number="9.12.5" id="credit-11"><span class="header-section-number">9.12.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.13" id="ace2-expression-in-kidney-and-testis-may-cause-kidney-and-testis-damage-after-2019-ncov-infection"><span class="header-section-number">9.13</span> ACE2 Expression in Kidney and Testis May Cause Kidney and Testis Damage After 2019-nCoV Infection</h2>
<p><span class="citation" data-cites="15K5Ovntj">[<a href="#ref-15K5Ovntj" role="doc-biblioref">548</a>]</span></p>
<h3 data-number="9.13.1" id="keywords-12"><span class="header-section-number">9.13.1</span> Keywords</h3>
<ul>
<li>CoVID-19</li>
<li>2019-nCoV</li>
<li>SARS-CoV-2</li>
<li>kidney</li>
<li>testis</li>
<li>ACE2</li>
<li>scRNAseq</li>
</ul>
<h3 data-number="9.13.2" id="main-findings-11"><span class="header-section-number">9.13.2</span> Main Findings</h3>
<ul>
<li><p>Study used online datasets (scRNAseq GSE131685, scRNAseq GSE107585,
Human Protein Atlas, GTEx portal, CCLE) to analyze ACE2 expression
in different human organs.</p></li>
<li><p>Study re-analyzed three clinical datasets (n=6, n=99, and n=41) to
show 3~10% of 2019-nCoV patients present with abnormal renal
function.</p></li>
<li><p>results indicate ACE2 highly expressed in renal tubular cells,
Leydig cells and seminiferous ductal cells of testis.</p></li>
</ul>
<h3 data-number="9.13.3" id="limitations-12"><span class="header-section-number">9.13.3</span> Limitations</h3>
<ul>
<li><p>Very preliminary transcript/protein dataset analysis in healthy
cohorts; does not necessarily translate to actual viral tropism and
permissiveness.</p></li>
<li><p>Clinically, would be important to determine with larger longitudinal
dataset if SARS-CoV-2 infection changes sperm quality or testicular
inflammation.</p></li>
<li><p>Similarly, would be important to determine if simultaneous HBV or
syphilis infection and orchitis impacts SARS-CoV-2 severity.</p></li>
<li><p>Examination and follow-up of renal function and viral orchitis/sperm
quality of CoVID-19 patients not done in this preliminary study.</p></li>
</ul>
<h3 data-number="9.13.4" id="significance-9"><span class="header-section-number">9.13.4</span> Significance</h3>
<ul>
<li><p>Kidney ACE2 result supports other concurrent sequencing studies
<span class="citation" data-cites="16Xj27QYj">[<a href="#ref-16Xj27QYj" role="doc-biblioref">539</a>]</span> and clinical reports
of abnormal renal function or even kidney damage in patients
infected with 2019-nCoV
<span class="citation" data-cites="1Acv5UMSt">[<a href="#ref-1Acv5UMSt" role="doc-biblioref">537</a>]</span>.</p></li>
<li><p>High ACE2 expression in testis suggests potential tropism of the
virus to testicular tissues and indicates potential risks for male
fertility. Viral orchitis reported for SARS-CoV previously [1],
but no clear evidence so far of infertility in SARS, MERS or
CoVID-19 patients.</p></li>
</ul>
<h3 data-number="9.13.5" id="credit-12"><span class="header-section-number">9.13.5</span> Credit</h3>
<p><em>Review by Samarth Hegde as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.14" id="aberrant-pathogenic-gm-csf-t-cells-and-inflammatory-cd14cd16-monocytes-in-severe-pulmonary-syndrome-patients-of-a-new-coronavirus"><span class="header-section-number">9.14</span> Aberrant pathogenic GM-CSF+ T cells and inflammatory CD14+CD16+ monocytes in severe pulmonary syndrome patients of a new coronavirus</h2>
<p><span class="citation" data-cites="Bgo7Cmj3">[<a href="#ref-Bgo7Cmj3" role="doc-biblioref">549</a>]</span></p>
<h3 data-number="9.14.1" id="keywords-13"><span class="header-section-number">9.14.1</span> Keywords</h3>
<ul>
<li>immunopathology</li>
<li>Th1</li>
<li>inflammatory monocytes</li>
<li>GM-CSF</li>
<li>IFN-γ</li>
<li>IL-6</li>
</ul>
<h3 data-number="9.14.2" id="main-findings-12"><span class="header-section-number">9.14.2</span> Main Findings</h3>
<p>The authors of this study sought to characterize the immune
mechanism causing severe pulmonary disease and mortality in 2019-nCoV
(COVID-19) patients. Peripheral blood was collected from hospitalized
ICU (n=12) and non-ICU (n=21) patients with confirmed 2019-nCoV and from
healthy controls (n=10) in The First Affiliated Hospital of University
of Science and Technology China (Hefei, Anhui). Immune analysis was
conducted by flow cytometry. 2019-nCoV patients had decreased
lymphocyte, monocyte, and CD4 T cell counts compared to healthy
controls. ICU patients had fewer lymphocytes than non-ICU patients. CD4
T cells of 2019-nCoV patients expressed higher levels of activation
markers (OX40, CD69, CD38, CD44) and exhaustion markers (PD-1 and Tim3)
than those of healthy controls. CD4 cells of ICU patients expressed
significantly higher levels of OX40, PD-1, and Tim3 than those of
non-ICU patients. 2019-nCoV patients had higher percentages of CD4 T
cells co-expressing GM-CSF and IL-6 compared to healthy controls, while
ICU patients had a markedly higher percentage of GM-CSF+ IFN-γ+ CD4 T
cells than non-ICU patients. The CD4 T cells of nCoV patients and
healthy controls showed no differences in TNF-α secretion.</p>
<p>The CD8 T cells of 2019-nCoV patients also showed higher expression of
activation markers CD69, CD38, and CD44, as well as exhaustion markers
PD-1 and Tim3, compared to healthy controls. CD8 T cells of ICU patients
expressed higher levels of GM-CSF than those of non-ICU patients and
healthy controls. No IL-6 or TNF-α was found in the CD8 T cells of any
group. There were no differences in numbers of NK cells or B cells in
2019-nCoV patients and healthy controls, nor was there any GM-CSF or
IL-6 secretion from these cells in either group.</p>
<p>Percentages of CD14+ CD16+ GM-CSF+ and CD14+ CD16+ IL-6+ inflammatory
monocytes were significantly increased in nCoV patients compared to
healthy controls; in particular, patients in the ICU had greater
percentages of CD14+ CD16+ IL-6+ monocytes than non-ICU patients. The
authors suggest that in 2019-nCoV patients, pathogenic Th1 cells produce
GM-CSF, recruiting CD14+ CD16+ inflammatory monocytes that secrete high
levels of IL-6. These may enter pulmonary circulation and damage lung
tissue while initiating the cytokine storm that causes mortality in
severe cases. This is consistent with the cytokine storm seen in similar
coronaviruses, as IL-6, IFN-γ, and GM-CSF are key inflammatory mediators
seen in patients with SARS-CoV-1 and MERS-CoV.</p>
<h3 data-number="9.14.3" id="limitations-13"><span class="header-section-number">9.14.3</span> Limitations</h3>
<p>Though the results of this study open
questions for further investigation, this is an early study on a small
cohort of patients, and as such there are a number of limitations. The
study included only 12 ICU patients and 21 non-ICU patients, and ideally
would be repeated with a much larger patient cohort. Though the authors
make claims about differences in lymphocyte and monocyte counts between
patients and healthy controls, they did not report baseline laboratory
findings for the control group. Additionally, severity of disease was
classified based on whether or not patients were in the ICU. It would be
interesting to contextualize the authors’ immunological findings with
more specific metrics of disease severity or time course. Noting
mortality, time from disease onset, pre-existing conditions, or severity
of lung pathology in post-mortem tissue samples would paint a fuller
picture of how to assess risk level and the relationship between
severity of disease and immunopathology. Another limitation is the
selection of cytokines and immune markers for analysis, as the selection
criteria were based on the cell subsets and cytokine storm typically
seen in SARS-CoV-1 and MERS-CoV patients. Unbiased cytokine screens and
immune profiling may reveal novel therapeutic targets that were not
included in this study.</p>
<h3 data-number="9.14.4" id="significance-10"><span class="header-section-number">9.14.4</span> Significance</h3>
<p>This study identifies potential therapeutic targets
that could prevent acute respiratory disease syndrome (ARDS) and
mortality in patients most severely affected by COVID-19. The authors
propose testing monoclonal antibodies against IL6-R or GM-CSF to block
recruitment of inflammatory monocytes and the subsequent cytokine storm
in these patients.</p>
<h3 data-number="9.14.5" id="credit-13"><span class="header-section-number">9.14.5</span> Credit</h3>
<p><em>Review by Gabrielle Lubitz as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.15" id="clinical-characteristics-of-2019-novel-infected-coronavirus-pneumoniaa-systemic-review-and-meta-analysis"><span class="header-section-number">9.15</span> Clinical Characteristics of 2019 Novel Infected Coronavirus Pneumonia：A Systemic Review and Meta-analysis</h2>
<p>Qian et al. <em>medRxiv</em>.
<span class="citation" data-cites="4Z1pEioX">[<a href="#ref-4Z1pEioX" role="doc-biblioref">550</a>]</span></p>
<h3 data-number="9.15.1" id="keywords-14"><span class="header-section-number">9.15.1</span> Keywords</h3>
<ul>
<li>White Blood Cells</li>
<li>Lymphocytes</li>
<li>Neutrophils</li>
</ul>
<h3 data-number="9.15.2" id="main-findings-13"><span class="header-section-number">9.15.2</span> Main Findings</h3>
<p>The authors performed a meta analysis of literature on clinical,
laboratory and radiologic characteristics of patients presenting with
pneumonia related to SARSCoV2 infection, published up to Feb 6 2020.
They found that symptoms that were mostly consistent among studies were
sore throat, headache, diarrhea and rhinorrhea. Fever, cough, malaise
and muscle pain were highly variable across studies. Leukopenia (mostly
lymphocytopenia) and increased white blood cells were highly variable
across studies. They identified three most common patterns seen on CT
scan, but there was high variability across studies. Consistently across
the studies examined, the authors found that about 75% of patients need
supplemental oxygen therapy, about 23% mechanical ventilation and about
5% extracorporeal membrane oxygenation (ECMO). The authors calculated a
staggering pooled mortality incidence of 78% for these patients.</p>
<h3 data-number="9.15.3" id="limitations-14"><span class="header-section-number">9.15.3</span> Limitations</h3>
<p>The authors mention that the total number of studies included in this
meta analysis is nine, however they also mentioned that only three
studies reported individual patient data. It is overall unclear how many
patients in total were included in their analysis. This is mostly
relevant as they reported an incredibly high mortality (78%) and mention
an absolute number of deaths of 26 cases overall. It is not clear from
their report how the mortality rate was calculated.</p>
<p>The data is based on reports from China and mostly from the Wuhan area,
which somewhat limits the overall generalizability and applicability of
these results.</p>
<h3 data-number="9.15.4" id="significance-11"><span class="header-section-number">9.15.4</span> Significance</h3>
<p>This meta analysis offers some important data for clinicians to refer to
when dealing with patients with COVID-19 and specifically with
pneumonia. It is very helpful to set expectations about the course of
the disease.</p>
<h3 data-number="9.15.5" id="credit-14"><span class="header-section-number">9.15.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.16" id="longitudinal-characteristics-of-lymphocyte-responses-and-cytokine-profiles-in-the-peripheral-blood-of-sars-cov-2-infected-patients"><span class="header-section-number">9.16</span> Longitudinal characteristics of lymphocyte responses and cytokine profiles in the peripheral blood of SARS-CoV-2 infected patients</h2>
<p>Liu et al. <em>medRxiv</em> <span class="citation" data-cites="zTGzuiTI">[<a href="#ref-zTGzuiTI" role="doc-biblioref">551</a>]</span></p>
<h3 data-number="9.16.1" id="keywords-15"><span class="header-section-number">9.16.1</span> Keywords</h3>
<ul>
<li>Lymphopenia</li>
<li>Neutrophil to CD8 T cell ratio (N8R)</li>
<li>inflammatory cytokines</li>
</ul>
<h3 data-number="9.16.2" id="main-findings-14"><span class="header-section-number">9.16.2</span> Main Findings</h3>
<p>Liu et al. enrolled a cohort of 40 patients from Wuhan
including 27 mild cases and 13 severe cases of COVID-19. They performed
a 16-day kinetic analysis of peripheral blood from time of disease
onset. Patients in the severe group were older (medium age of 59.7,
compared to 48.7 in mild group) and more likely to have hypertension as
a co-morbidity. Lymphopenia was observed in 44.4% of the mild patients
and 84.6% of the severe patients. Lymphopenia was due to low T cell
count, specially CD8 T cells. Severe patients showed higher neutrophil
counts and an increase of cytokines in the serum (IL2, IL6, IL10 and
IFNγ). The authors measured several other clinical laboratory parameters
were also higher in severe cases compared to mild, but concluded that
neutrophil to CD8 T cell ratio (N8R) as the best prognostic factor to
identify the severe cases compared to other receiver operating
characteristic (ROC).</p>
<h3 data-number="9.16.3" id="limitations-15"><span class="header-section-number">9.16.3</span> Limitations</h3>
<p>This was a small cohort (N=40), and two of the patients
initially included in the severe group (N=13) passed away and were
excluded from the analysis due to lack of longitudinal data. However, it
would be most important to be able to identify patients with severe
disease with higher odds of dying. It seems that the different time
points analyzed relate to hospital admission, which the authors describe
as disease onset. The time between first symptoms and first data points
is not described. It would have been important to analyze how the
different measured parameters change according to health condition, and
not just time (but that would require a larger cohort). The predictive
value of N8R compared to the more commonly used NLR needs to be assessed
in other independent and larger cohorts. Lastly, it is important to note
that pneumonia was detected in patients included in the “mild” group,
but according to the Chinese Clinical Guidance for COVID-19 Pneumonia
Diagnosis and Treatment (7th edition) this group should be considered
“moderate”.</p>
<h3 data-number="9.16.4" id="significance-12"><span class="header-section-number">9.16.4</span> Significance</h3>
<p>Lymphopenia and cytokine storm have been described to be
detrimental in many other infections including SARS-CoV1 and MERS-CoV.
However, it was necessary to confirm that this dramatic immune response
was also observed in the SARS-CoV2 infected patients. These results and
further validation of the N8R ratio as a predictor of disease severity
will contribute for the management of COVID19 patients and potential
development of therapies.</p>
<h3 data-number="9.16.5" id="credit-15"><span class="header-section-number">9.16.5</span> Credit</h3>
<p><em>Review by Pauline Hamon as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.17" id="clinical-and-immunologic-features-in-severe-and-moderate-forms-of-coronavirus-disease-2019"><span class="header-section-number">9.17</span> Clinical and immunologic features in severe and moderate forms of Coronavirus Disease 2019</h2>
<p>Chen et al. <em>medRxiv</em> <span class="citation" data-cites="1EIEx8Qe4">[<a href="#ref-1EIEx8Qe4" role="doc-biblioref">552</a>]</span></p>
<h3 data-number="9.17.1" id="keywords-16"><span class="header-section-number">9.17.1</span> Keywords</h3>
<ul>
<li>severe disease</li>
<li>lymphocytes</li>
<li>cytokines</li>
<li>IFNy</li>
<li>CD4 Tcells</li>
<li>HLA-DR CD8</li>
<li>Tcells</li>
</ul>
<h3 data-number="9.17.2" id="main-findings-15"><span class="header-section-number">9.17.2</span> Main Findings</h3>
<p>This study retrospectively evaluated clinical, laboratory,
hematological, biochemical and immunologic data from 21 subjects
admitted to the hospital in Wuhan, China (late December/January) with
confirmed SARS-CoV-2 infection. The aim of the study was to compare
‘severe’ (n=11, ~64 years old) and ‘moderate’ (n=10, ~51 years old)
COVID-19 cases. Disease severity was defined by patients’ blood oxygen
level and respiratory output. They were classified as ‘severe’ if SpO2
93% or respiratory rates 30 per min.</p>
<p>In terms of the clinical laboratory measures, ‘severe’ patients had
higher CRP and ferritin, alanine and aspartate aminotransferases, and
lactate dehydrogenase but lower albumin concentrations.</p>
<p>The authors then compared plasma cytokine levels (ELISA) and immune cell
populations (PBMCs, Flow Cytometry). ‘Severe’ cases had higher levels of
IL-2R, IL-10, TNFa, and IL-6 (marginally significant). For the immune
cell counts, ‘severe’ group had higher neutrophils, HLA-DR+ CD8 T cells
and total B cells; and lower total lymphocytes, CD4 and CD8 T cells
(except for HLA-DR+), CD45RA Tregs, and IFNy-expressing CD4 T cells. No
significant differences were observed for IL-8, counts of NK cells,
CD45+RO Tregs, IFNy-expressing CD8 T and NK cells.</p>
<h3 data-number="9.17.3" id="limitations-16"><span class="header-section-number">9.17.3</span> Limitations</h3>
<p>Several potential limitations should be noted: 1) Blood samples were
collected 2 days post hospital admission and no data on viral loads were
available; 2) Most patients were administered medications (e.g.
corticosteroids), which could have affected lymphocyte counts.
Medications are briefly mentioned in the text of the manuscript; authors
should include medications as part of Table 1. 3) ‘Severe’ cases were
significantly older and 4/11 ‘severe’ patients died within 20 days.
Authors should consider a sensitivity analysis of biomarkers with the
adjustment for patients’ age.</p>
<h3 data-number="9.17.4" id="significance-13"><span class="header-section-number">9.17.4</span> Significance</h3>
<p>Although the sample size was small, this paper presented a broad range
of clinical, biochemical, and immunologic data on patients with
COVID-19. One of the main findings is that SARS-CoV-2 may affect T
lymphocytes, primarily CD4+ T cells, resulting in decreased IFNy
production. Potentially, diminished T lymphocytes and elevated cytokines
can serve as biomarkers of severity of COVID-19.</p>
<h3 data-number="9.17.5" id="credit-16"><span class="header-section-number">9.17.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.18" id="sars-cov-2-and-sars-cov-spike-rbd-structure-and-receptor-binding-comparison-and-potential-implications-on-neutralizing-antibody-and-vaccine-development"><span class="header-section-number">9.18</span> SARS-CoV-2 and SARS-CoV Spike-RBD Structure and Receptor Binding Comparison and Potential Implications on Neutralizing Antibody and Vaccine Development</h2>
<p>Sun et al. <em>bioRxiv</em> <span class="citation" data-cites="41WAtZmB">[<a href="#ref-41WAtZmB" role="doc-biblioref">553</a>]</span></p>
<h3 data-number="9.18.1" id="keywords-17"><span class="header-section-number">9.18.1</span> Keywords</h3>
<ul>
<li>SARS-CoV</li>
<li>SARS-CoV-2</li>
<li>ACE2</li>
<li>Spike (S) protein</li>
<li>receptor binding domain (RBD)</li>
<li>receptor binding motif (RBM)</li>
<li>neutralizing antibody</li>
</ul>
<h3 data-number="9.18.2" id="main-findings-16"><span class="header-section-number">9.18.2</span> Main Findings</h3>
<p>This study compared the structure of SARS-CoV and SARS-CoV-2 Spike (S)
protein receptor binding domain (RBD) and interactions with ACE2 using
computational modeling, and interrogated cross-reactivity and
cross-neutralization of SARS-CoV-2 by antibodies against SARS-CoV. While
SARS-CoV and SARS-CoV-2 have over 70 % sequence homology and share the
same human receptor ACE2, the receptor binding motif (RBM) is only 50%
homologous.</p>
<p>Computational prediction of the SARS-CoV-2 and ACE2 interactions based
on the previous crystal structure data of SARS-CoV, and measurement of
binding affinities against human ACE2 using recombinant SARS-CoV and
SARS-CoV-2 S1 peptides, demonstrated similar binding of the two S1
peptides to ACE2, explaining the similar transmissibility of SARS-CoV
and SARS-CoV-2 and consistent with previous data (Wall et al Cell
2020).</p>
<p>The neutralization activity of SARS-CoV-specific rabbit polyclonal
antibodies were about two-order of magnitude less efficient to
neutralize SARS-CoV-2 than SARS-CoV, and four potently neutralizing
monoclonal antibodies against SARS-CoV had poor binding and neutralizing
activity against SARS-CoV-2. In contrast, 3 poor SARS-CoV-binding
monoclonal antibodies show some efficiency to bind and neutralize
SARS-CoV-2. The results suggest that that antibodies to more conserved
regions outside the RBM motif might possess better cross-protective
neutralizing activities between two strains.</p>
<h3 data-number="9.18.3" id="limitations-17"><span class="header-section-number">9.18.3</span> Limitations</h3>
<p>It would have been helpful to show the epitopes recognized by the
monoclonal antibodies tested on both SARS-CoV, SARS-CoV-2 to be able to
make predictions for induction of broadly neutralizing antibodies. The
data on monoclonal antibody competition with ACE2 for binding to
SARS-CoV RBD should have also included binding on SARS-CoV2, especially
for the three monoclonal antibodies that showed neutralization activity
for SARS-CoV2. Because of the less homology in RBM sequences between
viruses, it still may be possible that these antibodies would recognize
the ACE2 RBD in SARS-CoV-2.</p>
<h3 data-number="9.18.4" id="significance-14"><span class="header-section-number">9.18.4</span> Significance</h3>
<p>It is noteworthy that immunization to mice and rabbit with SARS-CoV S1
or RBD protein could induce monoclonal antibodies to cross-bind and
cross-neutralize SARS-CoV-2 even if they are not ACE2-blocking. If these
types of antibodies could be found in human survivors or in the
asymptomatic populations as well, it might suggest that exposure to
previous Coronavirus strains could have induced cross-neutralizing
antibodies and resulted in the protection from severe symptoms in some
cases of SARS-CoV2.</p>
<h3 data-number="9.18.5" id="credit-17"><span class="header-section-number">9.18.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.19" id="protection-of-rhesus-macaque-from-sars-coronavirus-challenge-by-recombinant-adenovirus-vaccine"><span class="header-section-number">9.19</span> Protection of Rhesus Macaque from SARS-Coronavirus challenge by recombinant adenovirus vaccine</h2>
<p>Chen et al. <em>bioRxiv</em> <span class="citation" data-cites="NPxs7bbq">[<a href="#ref-NPxs7bbq" role="doc-biblioref">554</a>]</span></p>
<h3 data-number="9.19.1" id="keywords-18"><span class="header-section-number">9.19.1</span> Keywords</h3>
<ul>
<li>SARS-CoV-1</li>
<li>rhesus macaque</li>
<li>recombinant adenovirus vaccine</li>
</ul>
<h3 data-number="9.19.2" id="main-findings-17"><span class="header-section-number">9.19.2</span> Main Findings</h3>
<p>Rhesus macaques were immunized intramuscularly twice (week 0 and
week 4) with SV8000 carrying the information to express a S1-orf8
fusion protein and the N protein from the BJ01 strain of SARS-CoV-1.
By week 8, immunized animals had signs of immunological protection
(IgG and neutralization titers) against SARS-CoV-1 and were
protected against challenge with the PUMC-1 strain, with fewer
detectable symptoms of respiratory distress, lower viral load,
shorter periods of viral persistence, and less pathology in the
lungs compared to non-immunized animals.</p>
<h3 data-number="9.19.3" id="limitations-18"><span class="header-section-number">9.19.3</span> Limitations</h3>
<p>The authors should write clearer descriptions of the methods used in
this article. They do not describe how the IgG titers or
neutralization titers were determined. There are some issues with
the presentation of data, for example, in Figure 1a, y-axis should
not be Vmax; forming cells and 1d would benefit from showing error
bars. Furthermore, although I inferred that the animals were
challenged at week 8, the authors did not explicitly detail when the
animals were challenged. The authors should explain the design of
their vaccine, including the choice of antigens and vector. The
authors also do not include a description of the ethical use of
animals in their study.</p>
<h3 data-number="9.19.4" id="significance-15"><span class="header-section-number">9.19.4</span> Significance</h3>
<p>The authors describe a vaccine for SARS-CoV-1 with no discussion of
possible implications for the current SARS-CoV-2 pandemic. Could a
similar vaccine be designed to protect against SARS-CoV-2 and would
the concerns regarding emerging viral mutations that the authors
describe as a limitation for SARS-CoV-1 also be true in the context
of SARS-CoV-2?</p>
<h3 data-number="9.19.5" id="credit-18"><span class="header-section-number">9.19.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.20" id="reduction-and-functional-exhaustion-of-t-cells-in-patients-with-coronavirus-disease-2019-covid-19"><span class="header-section-number">9.20</span> Reduction and Functional Exhaustion of T cells in Patients with Coronavirus Disease 2019 (COVID-19)</h2>
<p><span class="citation" data-cites="f50ahNxF">[<a href="#ref-f50ahNxF" role="doc-biblioref">555</a>]</span></p>
<h3 data-number="9.20.1" id="keywords-19"><span class="header-section-number">9.20.1</span> Keywords</h3>
<ul>
<li>T cell exhaustion</li>
<li>T cell lymphopenia</li>
<li>IL-6</li>
<li>IL-10</li>
<li>TNF-α</li>
</ul>
<h3 data-number="9.20.2" id="main-findings-18"><span class="header-section-number">9.20.2</span> Main Findings</h3>
<p>Based on a retrospective study of 522 COVID patients and 40
healthy controls from two hospitals in Wuhan, China, authors show both
age-dependent and clinical severity-dependent decrease in T cell numbers
with elderly patients and patients who are in ICU-care showing the most
dramatic decrease in T cell counts. Cytokine profiling of COVID patients
reveal that TNF-α, IL-6 and IL-10 are increased in infected patients
with patients in the ICU showing the highest levels. Interestingly,
these three cytokine levels were inversely correlated with T cell counts
and such inverse relationship was preserved throughout the disease
progression. Surface staining of exhaustion markers (PD-1 and Tim-3) and
flow cytometry of stained peripheral blood of 14 patients and 3 healthy
volunteers demonstrate that T cells of COVID patients have increased
expression of PD-1 with patients in ICU having the highest number of
CD8<sup>+</sup>PD-1<sup>+</sup> cells than their counterparts in non-ICU groups.</p>
<h3 data-number="9.20.3" id="limitations-19"><span class="header-section-number">9.20.3</span> Limitations</h3>
<p>Compared to the number of patients, number of control (n=
40) is small and is not controlled for age. Additional data linking
inflammatory cytokines and the quality of the adaptive response
including humoral and antigen specific T cell response is much needed. T
cell exhaustion study relies on marker-dependent labeling of T cell
functionality of a very limited sample size (n=17)—a
functional/mechanistic study of these T cells from PBMCs would have
bolstered their claims.</p>
<h3 data-number="9.20.4" id="significance-16"><span class="header-section-number">9.20.4</span> Significance</h3>
<p>Limited but contains interesting
implications. It is already known in literature that in the context of
acute respiratory viral infections CD8 T cells exhibit exhaustion-like
phenotypes which further underscores the importance of mechanistic
studies that can elucidate how COVID infection leads to lymphopenia and
T cell exhaustion-like phenotype.</p>
<p>However, as authors have noted, the data does point to an interesting
question: How these inflammatory cytokines (TNF-α, IL-6 and IL-10)
correlate with or affect effective viral immunity and what types of
cells produce these cytokines? Answering that question will help us
refine our targets for immune-modulatory therapies especially in
patients suffering from cytokine storms.</p>
<h3 data-number="9.20.5" id="credit-19"><span class="header-section-number">9.20.5</span> Credit</h3>
<p><em>This review by Chang Moon was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.21" id="clinical-characteristics-of-25-death-cases-infected-with-covid-19-pneumonia-a-retrospective-review-of-medical-records-in-a-single-medical-center-wuhan-china"><span class="header-section-number">9.21</span> Clinical Characteristics of 25 death cases infected with COVID-19 pneumonia: a retrospective review of medical records in a single medical center, Wuhan, China</h2>
<p><span class="citation" data-cites="EwNi3WJq">[<a href="#ref-EwNi3WJq" role="doc-biblioref">556</a>]</span></p>
<h3 data-number="9.21.1" id="keywords-20"><span class="header-section-number">9.21.1</span> Keywords</h3>
<ul>
<li>COVID-19</li>
<li>pneumonia</li>
<li>hypertension</li>
<li>diabetes</li>
<li>biomarker</li>
<li>neutrophilia</li>
<li>lymphopenia</li>
</ul>
<h3 data-number="9.21.2" id="main-findings-19"><span class="header-section-number">9.21.2</span> Main Findings</h3>
<p>Most common chronic conditions among 25 patients that died from
COVID-19 related respiratory failure were hypertension (64%) and
diabetes (40%). Disease progression was marked by progressive organ
failure, starting first with lung dysfunction, then heart (e.g.
increased cTnI and pro-BNP), followed by kidney (e.g. increased BUN,
Cr), and liver (e.g. ALT, AST). 72% of patients had neutrophilia and 88%
also had lymphopenia. General markers of inflammation were also
increased (e.g. PCT, D-Dimer, CRP, LDH, and SAA).</p>
<h3 data-number="9.21.3" id="limitations-20"><span class="header-section-number">9.21.3</span> Limitations</h3>
<p>The limitations of this study include small sample size and
lack of measurements for some tests for several patients. This study
would also have been stronger with comparison of the same measurements
to patients suffering from less severe disease to further validate and
correlate proposed biomarkers with disease severity.</p>
<h3 data-number="9.21.4" id="significance-17"><span class="header-section-number">9.21.4</span> Significance</h3>
<p>This study identifies chronic conditions (i.e. hypertension
and diabetes) that strongly correlates with disease severity. In
addition to general markers of inflammation, the authors also identify
concomitant neutrophilia and lymphopenia among their cohort of patients.
This is a potentially interesting immunological finding because we would
typically expect increased lymphocytes during a viral infection.
Neutrophilia may also be contributing to cytokine storm. In addition,
PCT was elevated in 90.5% of patients, suggesting a role for sepsis or
secondary bacterial infection in COVID-19 related respiratory failure.</p>
<h3 data-number="9.21.5" id="credit-20"><span class="header-section-number">9.21.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.22" id="sars-cov-2-infection-does-not-significantly-cause-acute-renal-injury-an-analysis-of-116-hospitalized-patients-with-covid-19-in-a-single-hospital-wuhan-china"><span class="header-section-number">9.22</span> SARS-CoV-2 infection does not significantly cause acute renal injury: an analysis of 116 hospitalized patients with COVID-19 in a single hospital, Wuhan, China</h2>
<p><span class="citation" data-cites="KG5Lg65z">[<a href="#ref-KG5Lg65z" role="doc-biblioref">557</a>]</span></p>
<h3 data-number="9.22.1" id="keywords-21"><span class="header-section-number">9.22.1</span> Keywords</h3>
<ul>
<li>CoVID-19</li>
<li>2019-nCoV</li>
<li>SARS-CoV-2</li>
<li>kidney</li>
<li>clinical</li>
<li>longitudinal</li>
</ul>
<h3 data-number="9.22.2" id="main-findings-20"><span class="header-section-number">9.22.2</span> Main Findings</h3>
<ul>
<li><p>Clinical data from 116 hospitalized CoVID-19 patients analyzed over
4 weeks for correlation with renal injury. Comorbidities included
chronic renal failure (CRF) in 5 patients (4.3%).</p></li>
<li><p>10.8% of patients with no prior kidney disease showed elevations in
blood urea or creatinine, and 7.2% of patients with no prior kidney
disease showed albuminuria.</p></li>
<li><p>Patients with pre-existing CRF underwent continuous renal
replacement therapy (CRRT) alongside CoVID-19 treatment. Renal
functions remained stable in these patients.</p></li>
<li><p>All 5 patients with CRF survived CoVID-19 therapy without
progression to ARDS or worsening of CRF.</p></li>
</ul>
<h3 data-number="9.22.3" id="limitations-21"><span class="header-section-number">9.22.3</span> Limitations</h3>
<ul>
<li><p>Renal injury biomarkers in patients with incipient kidney
abnormalities not tabulated separately, making overall data hard to
interpret. It will be critical to separately examine kidney function
(BUN, urine creatinine and eGFR) in patients that developed any
kidney abnormalities (7.2~10.8% of cohort).</p></li>
<li><p>No information on type of CoVID-19 therapy used across cohort; will
be useful to correlate how treatment modality influences kidney
function (and other parameters).</p></li>
<li><p>Invokes previous clinical-correlation studies that indicate low
instances of kidney damage<span class="citation" data-cites="VRoblZFJ bgrzawWp">[<a href="#ref-VRoblZFJ" role="doc-biblioref">558</a>,<a href="#ref-bgrzawWp" role="doc-biblioref">559</a>]</span>, but those studies did not track
longitudinal urine samples for acute renal injury markers and viral
shedding.</p></li>
<li><p>CRRT in patients with CRF is standard therapy irrespective of
CoVID-19 status; it will be important to compare clinical parameters
of these patients (n=5) with virus-naïve CRF patients (none in this
study) to make any meaningful conclusions.</p></li>
</ul>
<h3 data-number="9.22.4" id="significance-18"><span class="header-section-number">9.22.4</span> Significance</h3>
<ul>
<li><p>This study argues that renal impairment is uncommon in CoVID-19 and
not associated with high mortaility, in stark contrast with a
concurrent study <span class="citation" data-cites="1Acv5UMSt">[<a href="#ref-1Acv5UMSt" role="doc-biblioref">537</a>]</span>.
If supported by further studies, this argues kidney impairment is
secondary to cytokine storm/inflammation-induced organ failure, and
not due to direct viral replication.</p></li>
<li><p>Will be important to comprehensively characterize large-datasets of
CoVID-19 patients to conclude if kidney function actively disrupted
due to viral infection.</p></li>
</ul>
<h3 data-number="9.22.5" id="credit-21"><span class="header-section-number">9.22.5</span> Credit</h3>
<p><em>Review by Samarth Hegde as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.23" id="potential-t-cell-and-b-cell-epitopes-of-2019-ncov"><span class="header-section-number">9.23</span> Potential T-cell and B-Cell Epitopes of 2019-nCoV</h2>
<p><span class="citation" data-cites="hdSafq1o">[<a href="#ref-hdSafq1o" role="doc-biblioref">560</a>]</span></p>
<h3 data-number="9.23.1" id="keywords-22"><span class="header-section-number">9.23.1</span> Keywords</h3>
<ul>
<li>COVID-19</li>
<li>vaccine</li>
<li>epitopes</li>
<li>spike protein</li>
<li>MHC-I</li>
<li>MHC-II</li>
<li>neutralizing antibodies</li>
<li>ACE2</li>
</ul>
<h3 data-number="9.23.2" id="main-findings-21"><span class="header-section-number">9.23.2</span> Main Findings</h3>
<p>The authors use 2 neural network algorithms, NetMHCpan4 and
MARIA, to identify regions within the COVID-19 genome that are
presentable by HLA. They identify 405 viral epitopes that are
presentable on MHC-I and MHC-II and validate using known epitopes from
SARS-CoV. To determine whether immune surveillance drives viral
mutations to evade MHC presentation, the authors analyzed 68 viral
genomes from 4 continents. They identified 93 point mutations that
occurred preferentially in regions predicted to be presented by MHC-I
(p=0.02) suggesting viral evolution to evade CD8 T-cell mediated
killing. 2 nonsense mutations were also identified that resulted in loss
of presentation of an associated antigen (FGDSVEEVL) predicted to be
good antigen for presentation across multiple HLA alleles.</p>
<p>To identify potential sites of neutralizing antibody binding, the
authors used homology modeling to the SARS-CoV’s spike protein (S
protein) to determine the putative structure of the CoV2 spike protein.
They used Discotope2 to identify antibody binding sites on the protein
surface in both the down and up conformations of the S protein. The
authors validate this approach by first identifying antibody binding
site in SARS-CoV S protein. In both the down and up conformation of the
CoV2 S protein, the authors identified a potential antibody binding site
on the S protein receptor binding domain (RBD) of the ACE2 receptor
(residues 440-460, 494-506). While RBDs in both SARS-CoV and CoV2 spike
proteins may be important for antibody binding, the authors note that
SARS-CoV has larger attack surfaces than CoV2. These results were later
validated on published crystal structures of the CoV2 S protein RBD and
human ACE2. Furthermore, analysis of 68 viral genomes did not identify
any mutations in this potential antibody binding site in CoV2.</p>
<p>Finally, the authors compile a list of potential peptide vaccine
candidates across the viral genome that can be presented by multiple HLA
alleles. Several of the peptides showed homology to SARS-CoV T-cell and
B-cell epitopes.</p>
<h3 data-number="9.23.3" id="limitations-22"><span class="header-section-number">9.23.3</span> Limitations</h3>
<p>While the authors used computational methods of validation,
primarily through multiple comparisons to published SARS-CoV structures
and epitopes, future work should include experimental validation of
putative T-cell and B-cell epitopes.</p>
<h3 data-number="9.23.4" id="significance-19"><span class="header-section-number">9.23.4</span> Significance</h3>
<p>The authors identified potential T-cell and B-cell epitopes
that may be good candidates for peptide based vaccines against CoV2.
They also made interesting observations in comparing SARS-CoV and CoV2
potential antibody binding sites, noting that SARS-CoV had larger attack
surfaces for potential neutralizing antibody binding. One of the
highlights of this paper was the authors’ mutation analysis of 68 viral
genomes from 4 continents. This analysis not only validated their
computational method for identifying T-cell epitopes, but showed that
immune surveillance likely drives viral mutation in MHC-I binding
peptides. The smaller attack surface may point to potential mechanisms
of immune evasion by CoV2. However, absence of mutations in the RBD of
CoV2 and the small number of mutations in peptides presentable to T
cells suggests that vaccines against multiple epitopes could still
elicit robust immunity against CoV2.</p>
<h3 data-number="9.23.5" id="credit-22"><span class="header-section-number">9.23.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.24" id="structure-function-and-antigenicity-of-the-sarscov-2-spike-glycoprotein"><span class="header-section-number">9.24</span> Structure, Function, and Antigenicity of the SARSCoV-2 Spike Glycoprotein</h2>
<p>Walls et al. <em>bioRxiv</em>. <span class="citation" data-cites="8xnxgpag">[<a href="#ref-8xnxgpag" role="doc-biblioref">561</a>]</span> now <span class="citation" data-cites="qcVbT0w4">[<a href="#ref-qcVbT0w4" role="doc-biblioref">38</a>]</span></p>
<h3 data-number="9.24.1" id="keywords-23"><span class="header-section-number">9.24.1</span> Keywords</h3>
<ul>
<li>binding affinity</li>
<li>antigenicity</li>
<li>neutralizing antibody</li>
</ul>
<h3 data-number="9.24.2" id="main-findings-22"><span class="header-section-number">9.24.2</span> Main Findings</h3>
<p>The authors highlight a human angiotensin-converting enzyme 2
(hACE2), as a potential receptor used by the current Severe Acute
respiratory syndrome coronavirus-2 (SARS-CoV-2) as a host factor that
allows the virus target human cells. This virus-host interaction
facilitates the infection of human cells with a high affinity comparable
with SARS-CoV. The authors propose this mechanism as a probable
explanation of the efficient transmission of SARS-CoV-2 between humans.
Besides, Walls and colleagues described SARS-CoV-2 S glycoprotein S by
Cryo-EM along with neutralizing polyclonal response against SAR-CoV-2 S
from mice immunized with SAR-CoV and blocking SAR-CoV-2 S-mediated entry
into VeroE6 infected cells.**</p>
<h3 data-number="9.24.3" id="limitations-23"><span class="header-section-number">9.24.3</span> Limitations</h3>
<p>The SARS-CoV-2 depends on the cell factors ACE2 and TMPRSS2, this
last, according to a recent manuscript by Markus Hoffman et al., <em>Cell</em>,
2020. The authors used green monkey (VeroE6) and hamster (BHK) cell
lines in the experiments to drive its conclusions to humans; however, it
is well known the caucasian colon adenocarcinoma human cell line
(CaCo-2), highly express the hACE2 receptor as the TMPRSS2 protease as
well. In humans, ACE2 protein is highly expressed in the
gastrointestinal tract, which again, makes the CaCo-2 cell line suitable
for the following SARS-CoV-2 studies.</p>
<h3 data-number="9.24.4" id="significance-20"><span class="header-section-number">9.24.4</span> Significance</h3>
<p>The results propose a functional receptor used by SARS-CoV-2 to
infect humans worldwide and defining two distinct conformations of spike
(S) glycoprotein by cryogenic electron microscopy (Cryo-EM). This study
might help establish a precedent for initial drug design and treatment
of the current global human coronavirus epidemic.</p>
<h3 data-number="9.24.5" id="credit-23"><span class="header-section-number">9.24.5</span> Credit</h3>
<p><em>Review by postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.25" id="breadth-of-concomitant-immune-responses-underpinning-viral-clearance-and-patient-recovery-in-a-non-severe-case-of-covid-19"><span class="header-section-number">9.25</span> Breadth of concomitant immune responses underpinning viral clearance and patient recovery in a non-severe case of COVID-19</h2>
<p>Thevarajan et al. <em>medRxiv</em> <span class="citation" data-cites="QdimT8mN">[<a href="#ref-QdimT8mN" role="doc-biblioref">562</a>]</span></p>
<h3 data-number="9.25.1" id="keywords-24"><span class="header-section-number">9.25.1</span> Keywords</h3>
<ul>
<li>IgG</li>
<li>IgM</li>
<li>TfH cells</li>
<li>NK cells</li>
<li>SNP</li>
</ul>
<h3 data-number="9.25.2" id="main-findings-23"><span class="header-section-number">9.25.2</span> Main Findings</h3>
<p>The authors characterized the immune response in peripheral blood of a
47-year old COVID-19 patient.</p>
<p>SARS-CoV2 was detected in nasopharyngeal swab, sputum and faeces
samples, but not in urine, rectal swab, whole blood or throat swab. 7
days after symptom onset, the nasopharyngeal swab test turned negative,
at day 10 the radiography infiltrates were cleared and at day 13 the
patient became asymptomatic.</p>
<p>Immunofluorescence staining shows from day 7 the presence of
<strong>COVID-19-binding IgG and IgM</strong> antibodies in plasma, that increase
until day 20.</p>
<p>Flow cytometry on whole blood reveals a plasmablast peak at day 8, a
gradual increase in T follicular helper cells, stable HLA-DR<sup>+</sup> NK
frequencies and decreased monocyte frequencies compared to healthy
counterparts. The expression of CD38 and HLA-DR peaked on T cells at D9
and was associated with higher production of cytotoxic mediators by
CD8<sup>+</sup> T cells.</p>
<p>IL-6 and IL-8 were undetectable in plasma.</p>
<p>The authors further highlight the presence of the <strong>IFITM3
SNP-rs12252-C/C variant</strong> in this patient, which is associated with
higher susceptibility to influenza virus.</p>
<h3 data-number="9.25.3" id="limitations-24"><span class="header-section-number">9.25.3</span> Limitations</h3>
<p>These results need to be confirmed in additional patients.</p>
<p>COVID-19 patients have increased infiltration of macrophages in their
lungs <span class="citation" data-cites="tVkKuIJt">[<a href="#ref-tVkKuIJt" role="doc-biblioref">563</a>]</span>. Monitoring monocyte proportions in blood earlier in the
disease might help to evaluate their eventual migration to the lungs.</p>
<p>The stable concentration of HLA-DR<sup>+</sup> NK cells in blood from day 7 is
not sufficient to rule out NK cell activation upon SARS-CoV2 infection.
In response to influenza A virus, NK cells express higher levels of
activation markers CD69 and CD38, proliferate better and display higher
cytotoxicity <span class="citation" data-cites="87CJN2CG">[<a href="#ref-87CJN2CG" role="doc-biblioref">564</a>]</span>. Assessing these parameters in COVID-19 patients is
required to better understand NK cell role in clearing this infection.</p>
<p>Neutralization potential of the COVID-19-binding IgG and IgM antibodies
should be assessed in future studies.</p>
<p>This patient was able to clear the virus, while presenting a SNP
associated with severe outcome following influenza infection. The
association between this SNP and outcome upon SARS-CoV2 infection should
be further investigated.</p>
<h3 data-number="9.25.4" id="significance-21"><span class="header-section-number">9.25.4</span> Significance</h3>
<p>This study is among the first to describe the appearance of
COVID-19-binding IgG and IgM antibodies upon infection. The emergence of
new serological assays might contribute to monitor more precisely the
seroconversion kinetics of COVID-19 patients <span class="citation" data-cites="yJW1RVQb">[<a href="#ref-yJW1RVQb" role="doc-biblioref">210</a>]</span>. Further association
studies between IFITM3 SNP-rs12252-C/C variant and clinical data might
help to refine the COVID-19 outcome prediction tools.</p>
<h3 data-number="9.25.5" id="credit-24"><span class="header-section-number">9.25.5</span> Credit</h3>
<p><em>Review by Bérengère Salomé as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.26" id="the-landscape-of-lung-bronchoalveolar-immune-cells-in-covid-19-revealed-by-single-cell-rna-sequencing"><span class="header-section-number">9.26</span> The landscape of lung bronchoalveolar immune cells in COVID-19 revealed by single-cell RNA sequencing</h2>
<p>Liao et al. <em>medRxiv</em> <span class="citation" data-cites="tVkKuIJt">[<a href="#ref-tVkKuIJt" role="doc-biblioref">563</a>]</span></p>
<h3 data-number="9.26.1" id="keywords-25"><span class="header-section-number">9.26.1</span> Keywords</h3>
<ul>
<li>COVID-19</li>
<li>SARS-CoV-2</li>
<li>Broncho-alveolar lavage</li>
<li>macrophages</li>
<li>NK cells</li>
<li>T cells</li>
<li>cytokine storm</li>
<li>scRNAseq</li>
</ul>
<h3 data-number="9.26.2" id="main-findings-24"><span class="header-section-number">9.26.2</span> Main Findings</h3>
<p>The authors performed single-cell RNA sequencing (scRNAseq) on
bronchoalveolar lavage fluid (BAL) from 6 COVID-19 patients (n=3 mild
cases, n=3 severe cases). Data was compared to previously generated
scRNAseq data from healthy donor lung tissue (n=8).</p>
<p>Clustering analysis of the 6 patients revealed distinct immune cell
organization between mild and severe disease. Specifically, they found
that transcriptional clusters annotated as tissue resident alveolar
macrophages were strongly reduced while monocytes-derived FCN1<sup>+</sup>SPP1<sup>+</sup>
inflammatory macrophages dominated the BAL of patients with severe
COVID19 diseases. They show that inflammatory macrophages upregulated
interferon-signaling genes, monocytes recruiting chemokines including
CCL2, CCL3, CCL4 as well as IL-6, TNF, IL-8 and profibrotic cytokine
TGF-β, while alveolar macrophages expressed lipid metabolism genes, such
as PPARG.</p>
<p>The lymphoid compartment was overall enriched in lungs from patients.
Clonally expanded CD8 T cells were enriched in mild cases suggesting
that CD8 T cells contribute to viral clearance as in Flu infection,
whereas proliferating T cells were enriched in severe cases.</p>
<p>SARS-CoV-2 viral transcripts were detected in severe patients, but
considered here as ambient contaminations.</p>
<h3 data-number="9.26.3" id="limitations-25"><span class="header-section-number">9.26.3</span> Limitations</h3>
<p>These results are based on samples from 6 patients and should therefore
be confirmed in the future in additional patients. Longitudinal
monitoring of BAL during disease progression or resolution would have
been most useful.</p>
<p>The mechanisms underlying the skewing of the macrophage compartment in
patients towards inflammatory macrophages should be investigated in
future studies.</p>
<p>Deeper characterization of the lymphoid subsets is required. The
composition of the “proliferating” cluster and how these cells differ
from conventional T cell clusters should be assessed. NK and CD8 T cell
transcriptomic profile, in particular the expression of cytotoxic
mediator and immune checkpoint transcripts, should be compared between
healthy and diseased lesions.</p>
<h3 data-number="9.26.4" id="significance-22"><span class="header-section-number">9.26.4</span> Significance</h3>
<p>COVID-19 induces a robust inflammatory cytokine storm in patients that
contributes to severe lung tissue damage and ARDS <span class="citation" data-cites="vasHgjGH">[<a href="#ref-vasHgjGH" role="doc-biblioref">565</a>]</span>. Accumulation of
monocyte-derived inflammatory macrophages at the expense of Alveolar
macrophages known to play an anti-inflammatory role following
respiratory viral infection, in part through the PPARγ pathway <span class="citation" data-cites="9GEyqic3 WBN5f37b">[<a href="#ref-9GEyqic3" role="doc-biblioref">566</a>,<a href="#ref-WBN5f37b" role="doc-biblioref">567</a>]</span> are
likely contributing to lung tissue injuries. These data suggest that
reduction of monocyte accumulation in the lung tissues could help
modulate COVID-19-induced inflammation. Further analysis of lymphoid
subsets is required to understand the contribution of adaptive immunity
to disease outcome.</p>
<h3 data-number="9.26.5" id="credit-25"><span class="header-section-number">9.26.5</span> Credit</h3>
<p><em>Review by Bérengère Salomé and Assaf Magen as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.27" id="can-routine-laboratory-tests-discriminate-2019-novel-coronavirus-infected-pneumonia-from-other-community-acquired-pneumonia"><span class="header-section-number">9.27</span> Can routine laboratory tests discriminate 2019 novel coronavirus infected pneumonia from other community-acquired pneumonia?</h2>
<p>Pan et al. <em>medRxiv</em> <span class="citation" data-cites="FOVDaThy">[<a href="#ref-FOVDaThy" role="doc-biblioref">568</a>]</span></p>
<h3 data-number="9.27.1" id="keywords-26"><span class="header-section-number">9.27.1</span> Keywords</h3>
<ul>
<li>Routine laboratory testing</li>
</ul>
<h3 data-number="9.27.2" id="main-findings-25"><span class="header-section-number">9.27.2</span> Main Findings</h3>
<p>In an attempt to use standard laboratory testing for the discrimination
between “Novel Coronavirus Infected Pneumonia” (NCIP) and a usual
community acquired pneumonia (CAP), the authors compared laboratory
testing results of 84 NCIP patients with those of a historical group of
316 CAP patients from 2018 naturally COVID-19 negative. The authors
describe significantly lower white blood- as well as red blood- and
platelet counts in NCIP patients. When analyzing differential blood
counts, lower absolute counts were measured in all subsets of NCIP
patients. With regard to clinical chemistry parameters, they found
increased AST and bilirubin in NCIP patients as compared to CAP
patients.</p>
<h3 data-number="9.27.3" id="limitations-26"><span class="header-section-number">9.27.3</span> Limitations</h3>
<p>The authors claim to describe a simple method to rapidly assess a
pre-test probability for NCIP. However, the study has substantial
weakpoints. The deviation in clinical laboratory values in NCIP patients
described here can usually be observed in severely ill patients. The
authors do not comment on how severely ill the patients tested here were
in comparison to the historical control. Thus, the conclusion that the
tests discriminate between CAP and NCIP lacks justification.</p>
<h3 data-number="9.27.4" id="significance-23"><span class="header-section-number">9.27.4</span> Significance</h3>
<p>The article strives to compare initial laboratory testing results in
patients with COVID-19 pneumonia as compared to patients with a usual
community acquired pneumonia. The implications of this study for the
current clinical situation seem restricted due to a lack in clinical
information and the use of a control group that might not be
appropriate.</p>
<h3 data-number="9.27.5" id="credit-26"><span class="header-section-number">9.27.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.28" id="correlation-analysis-between-disease-severity-and-inflammation-related-parameters-in-patients-with-covid-19-pneumonia"><span class="header-section-number">9.28</span> Correlation Analysis Between Disease Severity and Inflammation-related Parameters in Patients with COVID-19 Pneumonia</h2>
<p><span class="citation" data-cites="1Du40IOOH">[<a href="#ref-1Du40IOOH" role="doc-biblioref">569</a>]</span></p>
<h3 data-number="9.28.1" id="keywords-27"><span class="header-section-number">9.28.1</span> Keywords</h3>
<ul>
<li>cytokine</li>
<li>COVID-19 pneumonia</li>
<li>severity</li>
<li>disease progression</li>
</ul>
<h3 data-number="9.28.2" id="main-findings-26"><span class="header-section-number">9.28.2</span> Main Findings</h3>
<p>This study is a cross-sectional analysis of 100 patients with
COVID-19 pneumonia, divided into mild (n = 34), severe (n = 34), and
critical (n = 32) disease status based on clinical definitions.</p>
<p>The criteria used to define disease severity are as follows:</p>
<ol type="1">
<li><p><em>Severe</em> – any of the following: respiratory distress or respiratory
rate ≥ 30 respirations/minute; oxygen saturation ≤ 93% at rest;
oxygen partial pressure (PaO2)/oxygen concentration (FiO2) in
arterial blood ≤ 300mmHg, progression of disease on imaging
to &gt;50% lung involvement in the short term.</p></li>
<li><p><em>Critical</em> – any of the following: respiratory failure that requires
mechanical ventilation; shock; other organ failure that requires
treatment in the ICU.</p></li>
<li><p>Patients with pneumonia who test positive for COVID-19 who do not
have the symptoms delineated above are considered <em>mild</em>.</p></li>
</ol>
<p>Peripheral blood inflammatory markers were correlated to disease status.
Disease severity was significantly associated with levels of IL-2R,
IL-6, IL-8, IL-10, TNF-α, CRP, ferroprotein, and procalcitonin. Total
WBC count, lymphocyte count, neutrophil count, and eosinophil count were
also significantly correlated with disease status. Since this is a
retrospective, cross-sectional study of clinical laboratory values,
these data may be extrapolated for clinical decision making, but without
studies of underlying cellular causes of these changes this study does
not contribute to a deeper understanding of SARS-CoV-2 interactions with
the immune system.</p>
<p>It is also notable that the mean age of patients in the mild group was
significantly different from the mean ages of patients designated as
severe or critical (p &lt; 0.001). The mean patient age was not
significantly different between the severe and critical groups. However,
IL-6, IL-8, procalcitonin (Table 2), CRP, ferroprotein (Figure 3A, 3B),
WBC count, and neutrophil count (Figure 4A, 4B) were all significantly
elevated in the critical group compared to severe. These data suggest
underlying differences in COVID-19 progression that is unrelated to age.</p>
<h3 data-number="9.28.3" id="significance-24"><span class="header-section-number">9.28.3</span> Significance</h3>
<p>Given the inflammatory profile outlined in this study,
patients who have mild or severe COVID-19 pneumonia, who <em>also</em> have any
elevations in the inflammatory biomarkers listed above, should be
closely monitored for potential progression to critical status.</p>
<h3 data-number="9.28.4" id="credit-27"><span class="header-section-number">9.28.4</span> Credit</h3>
<p><em>This review by JJF was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.29" id="an-effective-ctl-peptide-vaccine-for-ebola-zaire-based-on-survivors-cd8-targeting-of-a-particular-nucleocapsid-protein-epitope-with-potential-implications-for-covid-19-vaccine-design"><span class="header-section-number">9.29</span> An Effective CTL Peptide Vaccine for Ebola Zaire Based on Survivors’ CD8+ Targeting of a Particular Nucleocapsid Protein Epitope with Potential Implications for COVID-19 Vaccine Design</h2>
<p>Herst et al. <em>bioRxiv</em> <span class="citation" data-cites="1DTLMIDoM">[<a href="#ref-1DTLMIDoM" role="doc-biblioref">570</a>]</span></p>
<h3 data-number="9.29.1" id="keywords-28"><span class="header-section-number">9.29.1</span> Keywords</h3>
<ul>
<li>Peptide vaccine</li>
<li>Ebolavirus</li>
<li>nucleocapsid</li>
<li>epitope</li>
<li>vaccine design</li>
<li>microsphere</li>
</ul>
<h3 data-number="9.29.2" id="main-findings-27"><span class="header-section-number">9.29.2</span> Main Findings</h3>
<p>Vaccination of mice with a single dose of a 9-amino-acid peptide NP44-52
located in a conserved region of ebolavirus (EBOV) nucleocapsid protein
(NP) confers CD8+ T-cell-mediated immunity against mouse adapted EBOV
(maEBOV). Bioinformatic analyses predict multiple conserved CD8+ T cell
epitopes in the SARS-CoV-2 NP, suggesting that a similar approach may be
feasible for vaccine design against SARS-CoV-2.</p>
<p>The authors focus on a site within a 20-peptide region of EBOV NP which
was commonly targeted by CD8+ T cells in a group of EBOV survivors
carrying the HLA-A*30:01:01 allele. To justify the testing of specific
vaccine epitopes in a mouse challenge setting, the authors cite known
examples of human pathogen-derived peptide antigens that are also
recognized by C57BL/6 mice, as well as existing data surrounding known
mouse immunogenicity of peptides related to this EBOV NP region. Testing
3 distinct 9mer peptides over an 11 amino-acid window and comparing to
vaccination with the 11mer with a T-cell reactivity readout demonstrated
that optimizing peptide length and position for immunogenicity may be
crucial, likely due to suboptimal peptide processing and MHC-class-I
loading.</p>
<p>Vaccines for maEBOV challenge studies were constructed by packaging
NP44-52 in d,l poly(lactic-co-glycolic) acid microspheres. CpG was also
packaged within the microspheres, while Monophosphoryl Lipid A (a TLR4
ligand) was added to the injectate solution. A second peptide consisting
of a predicted MHC-II epitope from the EBOV VG19 protein was added using
a separate population of microspheres, and the formulation was injected
by intraperitoneal administration. The vaccine was protective against a
range of maEBOV doses up to at least 10,000 PFU. Survival was
anticorrelated with levels of IL6, MCP-1 (CCL2), IL9, and GM-CSF, which
recapitulated trends seen in human EBOV infection.</p>
<p>While HLA-A*30:01:01 is only present in a minority of humans, the
authors state that MHC binding algorithms predict NP44-52 to be a strong
binder of a set of more common HLA-A*02 alleles. The authors predict
that a peptide vaccine based on the proposed formulation could elicit
responses in up to 50% of people in Sudan or 30% of people in North
America.</p>
<p>SARS-CoV-2 NP, meanwhile, has conserved regions which may provide
peptide-vaccine candidates. Scanning the SARS-CoV-2 NP sequence for
HLA-binding 9mers identified 53 peptides with predicted binding affinity
&lt; 500nM, including peptides that are predicted to bind to HLA-class-I
alleles of 97% of humans, 7 of which have previously been tested
<em>in-vitro</em>.</p>
<p>The results support previously appreciated correlations between certain
cytokines and disease severity, specifically IL6 which relates to
multiple trial therapies. Prediction of HLA-class-I binding of
SARS-CoV-2 NP peptides suggests the plausibility of a peptide vaccine
targeting conserved regions of SARS-CoV-2 NP although further validation
in previously infected patient samples will be essential.</p>
<h3 data-number="9.29.3" id="credit-28"><span class="header-section-number">9.29.3</span> Credit</h3>
<p><em>Review by Andrew M. Leader as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.30" id="epitope-based-peptide-vaccines-predicted-against-novel-coronavirus-disease-caused-by-sars-cov-2"><span class="header-section-number">9.30</span> Epitope-based peptide vaccines predicted against novel coronavirus disease caused by SARS-CoV-2</h2>
<p>Li et al. <em>bioRxiv</em>. <span class="citation" data-cites="9spK49tt">[<a href="#ref-9spK49tt" role="doc-biblioref">571</a>]</span></p>
<h3 data-number="9.30.1" id="keywords-29"><span class="header-section-number">9.30.1</span> Keywords</h3>
<ul>
<li>SARS-CoV-2</li>
<li>immune-informatics</li>
<li>vaccine design</li>
<li>T cell epitope</li>
<li>B cell epitope</li>
</ul>
<h3 data-number="9.30.2" id="main-findings-28"><span class="header-section-number">9.30.2</span> Main Findings</h3>
<p>This study employs a series of bioinformatic
pipelines to identify T and B cell epitopes on spike (S) protein of
SARS-CoV-2 and assess their properties for vaccine potential. To
identify B cell epitopes, they assessed structural accessibility,
hydrophilicity, and beta-turn and flexibility which are all factors that
promote their targeting by antibodies. To identify T cell epitopes, they
filtered for peptides with high antigenicity score and capacity to bind
3 or more MHC alleles. Using the protein digest server, they also
demonstrated that their identified T and B cell epitopes are stable,
having multiple non-digesting enzymes per epitope. Epitopes were also
determined to be non-allergenic and non-toxin as assessed by Allergen FP
1.0 and ToxinPred, respectively. For T cell epitopes, they assessed the
strength of epitope-HLA interaction via PepSite. Overall, they predict
four B cell and eleven T cell epitopes (two MHC I and nine MHC II
binding) to pass stringent computational thresholds as candidates for
vaccine development. Furthermore, they performed sequence alignment
between all identified SARS-CoV-2 S protein mutations and predicted
epitopes, and showed that the epitopes are conserved across 134 isolates
from 38 locations worldwide. However, they report that these conserved
epitopes may soon become obsolete given the known mutation rate of
related SARS-CoV is estimated to be 4x10-4/site/year, underscoring the
urgency of anti-viral vaccine development.</p>
<h3 data-number="9.30.3" id="limitations-27"><span class="header-section-number">9.30.3</span> Limitations</h3>
<p>While spike (S) protein may have a
critical role in viral entry into host cells and their epitope
prediction criterion were comprehensive, this study did not examine
other candidate SARS-CoV-2 proteins. This point is particularly
important given that a single epitope may not be sufficient to induce
robust immune memory, and recent approaches involve multi-epitope
vaccine design. Furthermore, their study only included a direct
implementation of various published methods, but did not validate
individual bioinformatic tools with controls to demonstrate robustness.
Finally, it is critical that these predicted epitopes are experimentally
validated before any conclusions can be drawn about their potential as
vaccine candidates or their clinical efficacy.</p>
<h3 data-number="9.30.4" id="significance-25"><span class="header-section-number">9.30.4</span> Significance</h3>
<p>This study provides a computational framework to
rapidly identify epitopes that may serve as potential vaccine candidates
for treating SARS-CoV-2.</p>
<h3 data-number="9.30.5" id="credit-29"><span class="header-section-number">9.30.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.31" id="the-definition-and-risks-of-cytokine-release-syndrome-like-in-11-covid-19-infected-pneumonia-critically-ill-patients-disease-characteristics-and-retrospective-analysis"><span class="header-section-number">9.31</span> The definition and risks of Cytokine Release Syndrome-Like in 11 COVID-19-Infected Pneumonia critically ill patients: Disease Characteristics and Retrospective Analysis</h2>
<p>Wang Jr. et al. <em>medRxiv</em>.
<span class="citation" data-cites="qg1bmQng">[<a href="#ref-qg1bmQng" role="doc-biblioref">572</a>]</span></p>
<h3 data-number="9.31.1" id="keywords-30"><span class="header-section-number">9.31.1</span> Keywords</h3>
<ul>
<li>Cytokine release syndrome (CRS)</li>
<li>biomarkers</li>
<li>ARDS</li>
<li>IL-6</li>
<li>lymphopenia</li>
</ul>
<h3 data-number="9.31.2" id="main-findings-29"><span class="header-section-number">9.31.2</span> Main Findings</h3>
<p>This study describes the occurrence of a cytokine release syndrome-like
(CRSL) toxicity in ICU patients with COVID-19 pneumonia. The median time
from first symptom to acute respiratory distress syndrome (ARDS) was 10
days. All patients had decreased CD3, CD4 and CD8 cells, and a
significant increase of serum IL-6. Furthermore, 91% had decreased NK
cells. The changes in IL-6 levels preceded those in CD4 and CD8 cell
counts. All of these parameters correlated with the area of pulmonary
inflammation in CT scan images. Mechanical ventilation increased the
numbers of CD4 and CD8 cells, while decreasing the levels of IL-6, and
improving the immunological parameters.</p>
<h3 data-number="9.31.3" id="limitations-28"><span class="header-section-number">9.31.3</span> Limitations</h3>
<p>The number of patients included in this retrospective single center
study is small (n=11), and the follow-up period very short (25 days).
Eight of the eleven patients were described as having CRSL, and were
treated by intubation (7) or ECMO (2). Nine patients were still in the
intensive care unit at the time of publication of this article, so their
disease outcome is unknown.</p>
<h3 data-number="9.31.4" id="significance-26"><span class="header-section-number">9.31.4</span> Significance</h3>
<p>The authors define a cytokine release syndrome-like toxicity in patients
with COVID-19 with clinical radiological and immunological criteria: 1)
decrease of circulating CD4, CD8 and NK cells; 2) substantial increase
of IL-6 in peripheral blood; 3) continuous fever; 4) organ and tissue
damage. This event seems to occur very often in critically ill patients
with COVID-19 pneumonia. Interestingly, the increase of IL-6 in the
peripheral blood preceded other laboratory alterations, thus, IL-6 might
be an early biomarker for the severity of COVID-19 pneumonia. The
manuscript will require considerable editing for organization and
clarity.</p>
<h3 data-number="9.31.5" id="credit-30"><span class="header-section-number">9.31.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.32" id="clinical-characteristics-of-36-non-survivors-with-covid-19-in-wuhan-china"><span class="header-section-number">9.32</span> Clinical characteristics of 36 non-survivors with COVID-19 in Wuhan, China</h2>
<p>Huang et al. <em>medRxiv</em>. <span class="citation" data-cites="YODgfrtM">[<a href="#ref-YODgfrtM" role="doc-biblioref">573</a>]</span></p>
<h3 data-number="9.32.1" id="keywords-31"><span class="header-section-number">9.32.1</span> Keywords</h3>
<ul>
<li>Clinical Characteristics</li>
<li>Non Survivors</li>
<li>retrospective study</li>
</ul>
<h3 data-number="9.32.2" id="main-findings-30"><span class="header-section-number">9.32.2</span> Main Findings</h3>
<p>This is a simple study reporting clinical
characteristics of patients who did not survive COVID-19. All patients
(mean age=69.22 years) had acute respiratory distress syndrome (ARDS)
and their median time from onset to ARDS was 11 days. The median time
from onset to death was 17 days. Most patients were older male (70%
male) with co-morbidities and only 11 % were smokers. 75% patients
showed bilateral pneumonia. Many patients had chronic diseases,
including hypertension (58.33%). cardiovascular disease (22.22%) and
diabetes (19.44%). Typical clinical feature measured in these patients
includes lymphopenia and elevated markers of inflammation.</p>
<h3 data-number="9.32.3" id="limitations-29"><span class="header-section-number">9.32.3</span> Limitations</h3>
<p>As noted by the authors, the conclusions of this study are
very limited because this is single-centered study focusing on a small
cohort of patients who did not survive. Many clinical parameters
observed by the authors (such* as increase levels of serum CRP, PCT,
IL-6) have also been described in other COVID19 patients who survived
the infection</p>
<h3 data-number="9.32.4" id="significance-27"><span class="header-section-number">9.32.4</span> Significance</h3>
<p>This study is essentially descriptive and may be useful for
clinical teams monitoring COVID19 patients.</p>
<h3 data-number="9.32.5" id="credit-31"><span class="header-section-number">9.32.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.33" id="risk-factors-related-to-hepatic-injury-in-patients-with-corona-virus-disease-2019"><span class="header-section-number">9.33</span> Risk Factors Related to Hepatic Injury in Patients with Corona Virus Disease 2019</h2>
<p><span class="citation" data-cites="5CGbn9Jj">[<a href="#ref-5CGbn9Jj" role="doc-biblioref">574</a>]</span></p>
<h3 data-number="9.33.1" id="keywords-32"><span class="header-section-number">9.33.1</span> Keywords</h3>
<ul>
<li>COVID-related Hepatic Injury</li>
</ul>
<h3 data-number="9.33.2" id="main-findings-31"><span class="header-section-number">9.33.2</span> Main Findings</h3>
<p>Based on a retrospective study of 85 hospitalized COVID
patients in a Beijing hospital, authors showed that patients with
elevated ALT levels (n = 33) were characterized by significantly higher
levels of lactic acid and CRP as well as lymphopenia and hypoalbuminemia
compared to their counterparts with normal ALT levels. Proportion of
severe and critical patients in the ALT elevation group was
significantly higher than that of normal ALT group. Multivariate
logistic regression performed on clinical factors related to ALT
elevation showed that CRP <span class="math inline">\(\geq\)</span> 20mg/L and low lymphocyte count
(&lt;1.1*10^9 cells/L) were independently related to ALT elevation—a
finding that led the authors to suggest cytokine storm as a major
mechanism of liver damage.</p>
<h3 data-number="9.33.3" id="limitations-30"><span class="header-section-number">9.33.3</span> Limitations</h3>
<p>The article’s most attractive claim that liver damage seen in COVID
patients is caused by cytokine storm (rather than direct infection of
the liver) hinges solely on their multivariate regression analysis.
Without further mechanistic studies a) demonstrating how high levels of
inflammatory cytokines can induce liver damage and b) contrasting types
of liver damage incurred by direct infection of the liver vs.
system-wide elevation of inflammatory cytokines, their claim remains
thin. It is also worth noting that six of their elevated ALT group
(n=33) had a history of liver disease (i.e. HBV infection, alcoholic
liver disease, fatty liver) which can confound their effort to pin down
the cause of hepatic injury to COVID.</p>
<h3 data-number="9.33.4" id="significance-28"><span class="header-section-number">9.33.4</span> Significance</h3>
<p>Limited. This article confirms a rich body
of literature describing liver damage and lymphopenia in COVID patients.</p>
<h3 data-number="9.33.5" id="credit-32"><span class="header-section-number">9.33.5</span> Credit</h3>
<p><em>Review by Chang Moon as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.34" id="detectable-serum-sars-cov-2-viral-load-rnaaemia-is-closely-associated-with-drastically-elevated-interleukin-6-il-6-level-in-critically-ill-covid-19-patients"><span class="header-section-number">9.34</span> Detectable serum SARS-CoV-2 viral load (RNAaemia) is closely associated with drastically elevated interleukin 6 (IL-6) level in critically ill COVID-19 patients</h2>
<p><span class="citation" data-cites="T0BDaXkw">[<a href="#ref-T0BDaXkw" role="doc-biblioref">575</a>]</span></p>
<h3 data-number="9.34.1" id="keywords-33"><span class="header-section-number">9.34.1</span> Keywords</h3>
<ul>
<li>ARDS</li>
<li>interleukin-6 (IL-6)</li>
<li>procalcitonin (PCT)</li>
<li>pro-inflammatory cytokines</li>
<li>SARS-CoV-2 RNAaemia</li>
</ul>
<h3 data-number="9.34.2" id="main-findings-32"><span class="header-section-number">9.34.2</span> Main Findings</h3>
<p>48 adult patients diagnosed with Covid19 according to Chinese guidelines
for Covid19 diagnosis and treatment version 6 were included in this
study. Patients were further sub-divided into three groups based on
clinical symptoms and disease severity: (1) mild, positive Covid19 qPCR
with no or mild clinical symptoms (fever; respiratory; radiological
abnormalities); (2) severe, at least one of the following: shortness of
breath/respiratory rate &gt;30/min, oxygen saturation SaO<sub>2</sub>&lt;93%,
Horowitz index paO2/FiO<sub>2</sub> &lt; 300 mmHg (indicating moderate pulmonary
damage); and (3) critically ill, at least one additional complicating
factor: respiratory failure with need for mechanical ventilation;
systemic shock; multi-organ failure and transfer to ICU. Serum samples
and throat-swaps were collected from all 48 patients enrolled.
SARS-CoV-2 RNA was assessed by qPCR with positive results being defined
as Ct values &lt; 40, and serum interleukin-6 (IL-6) was quantified
using a commercially available detection kit. Briefly, patient
characteristics in this study confirm previous reports suggesting that
higher age and comorbidities are significant risk factors of clinical
severity. Of note, 5 out of 48 of patients (10.41%), all in the
critically ill category, were found to have detectable serum SARS-CoV-2
RNA levels, so-called RNAaemia. Moreover, serum IL-6 levels in these
patients were found to be substantially higher and this correlated with
the presence of detectable SARS-CoV-2 RNA levels. The authors
hypothesize that viral RNA might be released from acutely damages
tissues in moribund patients during the course of Covid19 and that
RNaemia along with IL-6 could potentially be used as a prognostic
marker.</p>
<h3 data-number="9.34.3" id="limitations-31"><span class="header-section-number">9.34.3</span> Limitations</h3>
<p>While this group’s report generally confirms some of the major findings
of a more extensive study, published in early February 2020, <span class="citation" data-cites="vasHgjGH">[<a href="#ref-vasHgjGH" role="doc-biblioref">565</a>]</span>,
there are limitations that should be taken into account. First, the
number of patients enrolled is relatively small; second, interpretation
of these data would benefit from inclusion of information about study
specifics as well as providing relevant data on the clinical course of
these patients other than the fact that some were admitted to ICU (i.e.
demographics on how many patients needed respiratory support, dialysis,
APACHE Ii/III or other standard ICU scores as robust prognostic markers
for mortality etc). It also remains unclear at which time point the
serum samples were taken, i.e. whether at admission, when the diagnosis
was made or during the course of the hospital stay (and potentially
after onset of therapy, which could have affected both IL-6 and RNA
levels). The methods section lacks important information on the qPCR
protocol employed, including primers and cycling conditions used. From a
technical point of view, Ct values &gt;35 seem somewhat non-specific
(although Ct &lt;40 was defined as the CDC cutoff as well) indicating
that serum RNA levels are probably very low, therefore stressing the
need for highly specific primers and high qPCR efficiency. In addition,
the statistical tests used (t-tests, according to the methods section)
do not seem appropriate as the organ-specific data such as BUN and
troponin T values seem to be not normally distributed across groups (n=
5 RNAaemia+ vs. n= 43 RNAaemia-). Given the range of standard deviations
and the differences in patient sample size, it is difficult to believe
that these data are statistically significantly different.</p>
<h3 data-number="9.34.4" id="significance-29"><span class="header-section-number">9.34.4</span> Significance</h3>
<p>This study is very rudimentary and lacks a lot of relevant clinical
details. However, it corroborates some previously published observations
regarding RNAemia and IL-6 by another group. Generally, regarding future
studies, it would be important to address the question of IL-6 and other
inflammatory cytokine dynamics in relation to Covid19 disease kinetics
(high levels of IL-6, IL-8 and plasma leukotriene were shown to have
prognostic value at the onset of ARDS ; serum IL-2 and IL-15 have been
associated with mortality; reviewed by Chen W &amp; Ware L, Clin Transl Med.
2015 <span class="citation" data-cites="l3CzJ5ni">[<a href="#ref-l3CzJ5ni" role="doc-biblioref">576</a>]</span>).</p>
<h3 data-number="9.34.5" id="credit-33"><span class="header-section-number">9.34.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.35" id="lymphopenia-predicts-disease-severity-of-covid-19-a-descriptive-and-predictive-study"><span class="header-section-number">9.35</span> Lymphopenia predicts disease severity of COVID-19: a descriptive and predictive study</h2>
<p><span class="citation" data-cites="uIwTzZkJ">[<a href="#ref-uIwTzZkJ" role="doc-biblioref">577</a>]</span></p>
<h3 data-number="9.35.1" id="keywords-34"><span class="header-section-number">9.35.1</span> Keywords</h3>
<ul>
<li>Lymphopenia</li>
</ul>
<h3 data-number="9.35.2" id="main-findings-33"><span class="header-section-number">9.35.2</span> Main Findings</h3>
<p>Based on a retrospective study of 162 COVID patients from a
local hospital in Wuhan, China, the authors show an inverse correlation
between lymphocyte % (LYM%) of patients and their disease severity. The
authors have also tracked LYM% of 70 cases (15 deaths; 15 severe; 40
moderate) throughout the disease progression with fatal cases showing no
recovery of lymphocytes ( &lt;5%) even after 17-19 days post-onset. The
temporal data of LYM % in COVID patients was used to construct a
Time-Lymphocyte% model which is used to categorize and predict patients’
disease severity and progression. The model was validated using 92
hospitalized cases and kappa statistic test was used to assess agreement
between predicted disease severity and the assigned clinical severity (k
= 0.49).</p>
<h3 data-number="9.35.3" id="limitations-32"><span class="header-section-number">9.35.3</span> Limitations</h3>
<p>Time-Lymphocyte % Model (TLM) that authors have proposed as
a predictive model for clinical severity is very simple in its
construction and derives from correlative data of 162 patients. In order
for the model to be of use, it needs validation using a far more robust
data set and possibly a mechanistic study on how COVID leads to
lymphopenia in the first place. In addition, it should be noted that no
statistical test assessing significance of LYM % values between disease
severities was performed.</p>
<h3 data-number="9.35.4" id="significance-30"><span class="header-section-number">9.35.4</span> Significance</h3>
<p>This article is of limited significance as
it simply reports similar descriptions of COVID patients made in
previous literature that severe cases are characterized by lymphopenia.</p>
<h3 data-number="9.35.5" id="credit-34"><span class="header-section-number">9.35.5</span> Credit</h3>
<p><em>Review by Chang Moon as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.36" id="the-potential-role-of-il-6-in-monitoring-severe-case-of-coronavirus-disease-2019"><span class="header-section-number">9.36</span> The potential role of IL-6 in monitoring severe case of coronavirus disease 2019</h2>
<p>Liu et al. <em>medRxiv</em>. <span class="citation" data-cites="1GMxHWnxN">[<a href="#ref-1GMxHWnxN" role="doc-biblioref">578</a>]</span></p>
<h3 data-number="9.36.1" id="keywords-35"><span class="header-section-number">9.36.1</span> Keywords</h3>
<ul>
<li>Cytokine Release Syndrome</li>
<li>lymphocytopenia</li>
<li>IL-6</li>
<li>CRP</li>
<li>COVID19</li>
<li>pneumonia</li>
</ul>
<h3 data-number="9.36.2" id="main-findings-34"><span class="header-section-number">9.36.2</span> Main Findings</h3>
<p>Study on blood biomarkers on 80 COVID19 patients (69 severe
and 11 non-severe). Patients with severe symptoms at admission
(baseline) showed obvious lymphocytopenia and significantly increased
interleukin-6 (IL-6) and CRP, which was positively correlated with
symptoms severity. IL-6 at baseline positively correlates with CRP, LDH,
ferritin and D-Dimer abundance in blood.</p>
<p>Longitudinal analysis of 30 patients (before and after treatment) showed
significant reduction of IL-6 in remission cases.</p>
<h3 data-number="9.36.3" id="limitations-33"><span class="header-section-number">9.36.3</span> Limitations</h3>
<p>Limited sample size at baseline, especially for the
non-severe leads to question on representativeness. The longitudinal
study method is not described in detail and suffers from
non-standardized treatment. Limited panel of pro-inflammatory cytokine
was analyzed. Patients with severe disease show a wide range of altered
blood composition and biomarkers of inflammation, as well as differences
in disease course (53.6% were cured, about 10% developed acute
respiratory distress syndrome). The authors comment on associations
between IL-6 levels and outcomes, but these were not statistically
significant (maybe due to the number of patients, non-standardized
treatments, etc.) and data is not shown. Prognostic biomarkers could
have been better explored. Study lacks multivariate analysis.</p>
<h3 data-number="9.36.4" id="significance-31"><span class="header-section-number">9.36.4</span> Significance</h3>
<p>IL-6 could be used as a pharmacodynamic
marker of disease severity. Cytokine Release Syndrome (CRS) is a
well-known side effect for CAR-T cancer therapy and there are several
effective drugs to manage CRS. Drugs used to manage CRS could be tested
to treat the most severe cases of COVID19.</p>
<h3 data-number="9.36.5" id="credit-35"><span class="header-section-number">9.36.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.37" id="clinical-and-laboratory-profiles-of-75-hospitalized-patients-with-novel-coronavirus-disease-2019-in-hefei-china"><span class="header-section-number">9.37</span> Clinical and Laboratory Profiles of 75 Hospitalized Patients with Novel Coronavirus Disease 2019 in Hefei, China</h2>
<p>Zhao et al. <em>medRxiv</em>. <span class="citation" data-cites="1B0vBwv1q">[<a href="#ref-1B0vBwv1q" role="doc-biblioref">579</a>]</span></p>
<h3 data-number="9.37.1" id="keywords-36"><span class="header-section-number">9.37.1</span> Keywords</h3>
<ul>
<li>Routine laboratory testing</li>
</ul>
<h3 data-number="9.37.2" id="main-findings-35"><span class="header-section-number">9.37.2</span> Main Findings</h3>
<p>The authors of this study provide a comprehensive analysis of clinical
laboratory assessments in 75 patients (median age 47 year old)
hospitalized for Corona virus infection in China measuring differential
blood counts including T-cell subsets (CD4, CD8), coagulation function,
basic blood chemistry, of infection-related biomarkers including CRP,
Procalcitonin (PCT) (Precursor of calcitonin that increases during
bacterial infection or tissue injury), IL-6 and erythrocyte
sedimentation rate as well as clinical parameters. Among the most common
hematological changes they found increased neutrophils, reduced CD4 and
CD8 lymphocytes, increased LDH, CRP and PCT</p>
<p>When looking at patients with elevated IL-6, the authors describe
significantly reduced CD4 and CD8 lymphocyte counts and elevated CRP and
PCT levels were significantly increased in infected patients suggesting
that increased IL-6 may correlate well with disease severity in COVID-19
infections</p>
<h3 data-number="9.37.3" id="limitations-34"><span class="header-section-number">9.37.3</span> Limitations</h3>
<p>The authors performed an early assessment of clinical standard
parameters in patients infected with COVID-19. Overall, the number of
cases (75) is rather low and the snapshot approach does not inform about
dynamics and thus potential relevance in the assessment of treatment
options in this group of patients.</p>
<h3 data-number="9.37.4" id="significance-32"><span class="header-section-number">9.37.4</span> Significance</h3>
<p>The article summarizes provides a good summary of some of the common
changes in immune cells inflammatory cytokines in patients with a
COVID-19 infection and. Understanding how these changes can help predict
severity of disease and guide therapy including IL-6 cytokine receptor
blockade using Tocilizumab or Sarilumab will be important to explore.</p>
<h3 data-number="9.37.5" id="credit-36"><span class="header-section-number">9.37.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.38" id="exuberant-elevation-of-ip-10-mcp-3-and-il-1ra-during-sars-cov-2-infection-is-associated-with-disease-severity-and-fatal-outcome"><span class="header-section-number">9.38</span> Exuberant elevation of IP-10, MCP-3 and IL-1ra during SARS-CoV-2 infection is associated with disease severity and fatal outcome</h2>
<p>Yang et al. <em>medRxiv</em> <span class="citation" data-cites="RyXNw87v">[<a href="#ref-RyXNw87v" role="doc-biblioref">580</a>]</span></p>
<h3 data-number="9.38.1" id="keywords-37"><span class="header-section-number">9.38.1</span> Keywords</h3>
<ul>
<li>cytokine</li>
<li>IP-10</li>
<li>MCP-3</li>
<li>IL-1Ra</li>
<li>lymphocyte</li>
<li>neutrophil</li>
<li>stratification</li>
<li>disease severity</li>
<li>viral load</li>
<li>lung function</li>
<li>complications</li>
<li>clinical data</li>
</ul>
<h3 data-number="9.38.2" id="summary-3"><span class="header-section-number">9.38.2</span> Summary</h3>
<p>Plasma cytokine analysis (48 cytokines) was
performed on COVID-19 patient plasma samples, who were sub-stratified as
severe (N=34), moderate (N=19), and compared to healthy controls (N=8).
Patients were monitored for up to 24 days after illness onset: viral
load (qRT-PCR), cytokine (multiplex on subset of patients), lab tests,
and epidemiological/clinical characteristics of patients were reported.</p>
<h3 data-number="9.38.3" id="main-findings-36"><span class="header-section-number">9.38.3</span> Main Findings</h3>
<ul>
<li><p>Many elevated cytokines with COVID-19 onset compared to healthy
controls (IFNy, IL-1Ra, IL-2Ra, IL-6, IL-10, IL-18, HGF, MCP-3, MIG,
M-CSF, G-CSF, MIG-1a, and IP-10).</p></li>
<li><p>IP-10, IL-1Ra, and MCP-3 (esp. together) were associated with
disease severity and fatal outcome.</p></li>
<li><p>IP-10 was correlated to patient viral load (r=0.3006, p=0.0075).</p></li>
<li><p>IP-10, IL-1Ra, and MCP-3 were correlated to loss of lung function
(PaO<sub>2</sub>/FaO<sub>2</sub> (arterial/atmospheric O2) and Murray Score (lung
injury) with MCP-3 being the most correlated (r=0.4104 p&lt;0.0001
and r=0.5107 p&lt;0.0001 respectively).</p></li>
<li><p>Viral load (Lower Ct Value from qRT-PCR) was associated with
upregulated IP-10 only (not IL-1Ra or MCP-3) and was mildly
correlated with decreased lung function: PaO<sub>2</sub>/FaO<sub>2</sub>
(arterial/atmospheric O2) and Murray Score (lung injury).</p></li>
<li><p>Lymphopenia (decreased CD4 and CD8 T cells) and increased neutrophil
correlated w/ severe patients.</p></li>
<li><p>Complications were associated with COVID severity (ARDS, hepatic
insufficiency, renal insufficiency).</p></li>
</ul>
<h3 data-number="9.38.4" id="limitations-35"><span class="header-section-number">9.38.4</span> Limitations</h3>
<p>Collection time of clinical data and lab results
not reported directly (likely 4 days (2,6) after illness onset), making
it very difficult to determine if cytokines were predictive of patient
outcome or reflective of patient compensatory immune response (likely
the latter). Small N for cytokine analysis (N=2 fatal and N=5
severe/critical, and N=7 moderate or discharged). Viral treatment
strategy not clearly outlined.</p>
<h3 data-number="9.38.5" id="expanded-results"><span class="header-section-number">9.38.5</span> Expanded Results</h3>
<p><em>NOTE</em>: Moderate COVID-19 was classified by fever, respiratory
manifestations, and radiological findings consistent with pneumonia
while severe patients had one or more of the following: 1) respiratory
distraction, resting O2 saturation, or 3) arterial PaO2/FiO2 <em>&lt;</em> 300
mmHg.</p>
<p>Cytokine Results (Human Cytokine Screening Panel, Bio-Rad):</p>
<ul>
<li><p><strong>Significant elevation of cytokines observed in COVID patients
compared to healthy controls: IFNy, IL-1Ra, IL-2Ra, IL-6, IL-10,
IL-18, HGF, MCP-3, MIG, M-CSF, G-CSF, MIG-1a, and IP-10.</strong></p></li>
<li><p>Severity was correlated <strong>with increase in measured IP-10, MCP-3,
and IL-Ra</strong> as measure by area under the curve analysis during
sample timecourse (2-24 days after illness onset).</p></li>
<li><p>IL-1Ra incr. significant 0-7 days after onset, MCP-3 signif.
upregulated throughout observation timecourse, and IP-10 increased
and upregulated throughout (trending downwards over time).</p></li>
<li><p><strong>The three cytokines together (IP-10, IL-1Ra, and MCP-3 AUC) served
as the best predictors of disease deterioration and fatal outcome.</strong></p></li>
<li><p>No significant differences between moderate/severe observed between
groups in IL-2Ra, IL-6, IL-10, IL-18, CTACK, G-CSF, HGF, M-CSF,
MIP-1a, MIG, and IFNy at any timepoints.</p></li>
<li><p><strong>Viral load (Lower Ct Value from qRT-PCR) was associated with
upregulated IP-10 only (not IL-1Ra or MCP-3) and was highly
correlated with decreased lung function: PaO<sub>2</sub>/FaO<sub>2</sub>
(arterial/atmospheric O2) and Murray Score (lung injury).</strong></p></li>
<li><p><strong>Antibodies against these cytokines (esp. anti-IP-10) may serve as
a potential treatment for amelioration of COVID-19 (and associated
ARDS).</strong></p></li>
</ul>
<p>Lab results:</p>
<ul>
<li><p><strong>Decreased lymphocytes (%) in all patients – lymphopenia corr. w/
severe patients</strong></p>
<ul>
<li><strong>Decreased CD4 and CD8 T cells</strong> – no monocyte or
eosino/basophil % measured</li>
</ul></li>
<li><p><strong>Increased neutrophils (%)</strong></p></li>
<li><p>Increased BUN (mmol/L) – other kidney markers, liver markers, and
LDH were not significantly different between groups and were not
compared to healthy controls.</p></li>
</ul>
<p>Clinical features (between moderate vs. severe patient groups):</p>
<ul>
<li><p><strong>Complications were associated with severity (ARDS, hepatic
insufficiency, renal insufficiency).</strong></p></li>
<li><p>Coexisting conditions between groups were not significantly
different (chronic heart/lung/renal/liver disease, diabetes, or
cancer) and patient time courses (onset to admission and onset to
viral tx) also not significantly different – 4 days (2, 6) on
average for admission and 4 (3,7) for antiviral.</p></li>
<li><p>Increased corticosteroids and mechanical/ invasive mechanical
ventilation in severe patients.</p></li>
<li><p>Increased median age in severe group (Median (Range = 63.5 (42-74)
vs. 51 (22-78)) and patients <em>&gt;</em> 60 yrs had higher ratio of
severe patients as compared patients 16-59 yrs.</p></li>
<li><p>Higher incidence of fever is severe patients (91.2 vs. 68.4%),
myalgia (57.7 vs. 48.1%), and chill (17.6% vs. 0%).</p></li>
<li><p>No differences in cough, headache, nausea/vomiting, or diarrhea.</p></li>
</ul>
<h3 data-number="9.38.6" id="significance-33"><span class="header-section-number">9.38.6</span> Significance</h3>
<p>Outline of pathological time course (implicating innate
immunity esp.) and identification key cytokines associated with disease
severity and prognosis (+ comorbidities). Anti-IP-10 as a possible
therapeutic intervention (ex: Eldelumab).</p>
<h3 data-number="9.38.7" id="credit-37"><span class="header-section-number">9.38.7</span> Credit</h3>
<p><em>Review by Natalie Vaninov as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.39" id="antibody-responses-to-sars-cov-2-in-patients-of-novel-coronavirus-disease-2019"><span class="header-section-number">9.39</span> Antibody responses to SARS-CoV-2 in patients of novel coronavirus disease 2019</h2>
<p>Zhao Jr. et al. <em>medRxiv</em>. <span class="citation" data-cites="5HXuCa5t">[<a href="#ref-5HXuCa5t" role="doc-biblioref">581</a>]</span></p>
<h3 data-number="9.39.1" id="keywords-38"><span class="header-section-number">9.39.1</span> Keywords</h3>
<ul>
<li>SARS-CoV-2 IgG</li>
<li>seroconversion rate</li>
<li>total Ab</li>
<li>Ig</li>
<li>IgM</li>
</ul>
<h3 data-number="9.39.2" id="main-findings-37"><span class="header-section-number">9.39.2</span> Main Findings</h3>
<p>This study examined antibody responses in
the blood of COVID-19 patients during the early SARS CoV2 outbreak in
China. Total 535 plasma samples were collected from 173 patients (51.4%
female) and were tested for seroconversion rate using ELISA. Authors
also compared the sensitivity of RNA and antibody tests over the course
of the disease . The key findings are:</p>
<ul>
<li><p>Among 173 patients, the seroconversion rates for total antibody
(Ab), IgM and IgG were 93.1% (161/173), 82.7% (143/173) and 64.7%
(112/173), respectively.</p></li>
<li><p>The seroconversion sequentially appeared for Ab, IgM and then IgG,
with a median time of 11, 12 and 14 days, respectively. Overall, the
seroconversion of Ab was significantly quicker than that of IgM (p =
0.012) and IgG (p &lt; 0.001). Comparisons of seroconversion rates
between critical and non-critical patients did not reveal any
significant differences.</p></li>
<li><p>RNA tests had higher sensitivity in early phase and within 7 days of
disease onset than antibody assays (66.7% Vs 38.3% respectively).</p></li>
<li><p>The sensitivity of the Ab assays was higher 8 days after disease
onset, reached 90% at day 13 and 100% at later time points (15-39
days). In contrast, RNA was only detectable in 45.5% of samples at
days 15-39.</p></li>
<li><p>In patients with undetectable RNA in nasal samples collected during
day 1-3, day 4-7, day 8-14 and day 15-39 since disease onset, 28.6%
(2/7), 53.6% (15/28), 98.2% (56/57) and 100% (30/30) had detectable
total Ab titers respectively Combining RNA and antibody tests
significantly raised the sensitivity for detecting COVID-19 patients
in different stages of the disease (p &lt; 0.001).</p></li>
<li><p>There was a strong positive correlation between clinical severity
and antibody titer 2-weeks after illness onset.</p></li>
<li><p>Dynamic profiling of viral RNA and antibodies in representative
COVID-19 patients (n=9) since onset of disease revealed that
antibodies may not be sufficient to clear the virus. It should be
noted that increases in of antibody titers were not always
accompanied by RNA clearance.</p></li>
</ul>
<h3 data-number="9.39.3" id="limitations-36"><span class="header-section-number">9.39.3</span> Limitations</h3>
<p>Because different types of ELISA assays were used for
determining antibody concentrations at different time points after
disease onset, sequential seroconversion of total Ab, IgM and IgG may
not represent actual temporal differences but rather differences in the
affinities of the assays used. Also, due to the lack of blood samples
collected from patients in the later stage of illness, how long the
antibodies could last remain unknown. For investigative dynamics of
antibodies, more samples were required.</p>
<h3 data-number="9.39.4" id="significance-34"><span class="header-section-number">9.39.4</span> Significance</h3>
<p>Total and IgG antibody titers could be used to understand
the epidemiology of SARS CoV-2 infection and to assist in determining
the level of humoral immune response in patients.</p>
<p>The findings provide strong clinical evidence for routine serological
and RNA testing in the diagnosis and clinical management of COVID-19
patients. The understanding of antibody responses and their half-life
during and after SARS CoV2 infection is important and warrants further
investigations.</p>
<h3 data-number="9.39.5" id="credit-38"><span class="header-section-number">9.39.5</span> Credit</h3>
<p><em>This review was undertaken by Zafar Mahmood and edited by K Alexandropoulos as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.40" id="restoration-of-leukomonocyte-counts-is-associated-with-viral-clearance-in-covid-19-hospitalized-patients"><span class="header-section-number">9.40</span> Restoration of leukomonocyte counts is associated with viral clearance in COVID-19 hospitalized patients</h2>
<p>Chen et al. <em>medRxiv</em> <span class="citation" data-cites="uCyoHtV3">[<a href="#ref-uCyoHtV3" role="doc-biblioref">582</a>]</span></p>
<h3 data-number="9.40.1" id="keywords-39"><span class="header-section-number">9.40.1</span> Keywords</h3>
<ul>
<li>COVID-19</li>
<li>T cell</li>
<li>B cell</li>
<li>NK cell</li>
<li>IL-6</li>
<li>pro-calcitonin</li>
<li>cytokine storm</li>
</ul>
<h3 data-number="9.40.2" id="main-findings-38"><span class="header-section-number">9.40.2</span> Main Findings</h3>
<p>The authors collected data on 25 COVID-19 patients (n=11 men, n=14
women) using standard laboratory tests and flow cytometry. All patients
were treated with antibiotics. Twenty-four of the 25 patients were also
treated with anti-viral Umefinovir and 14 of the patients were treated
with corticosteroids. 14 patients became negative for the virus after
8-14 days of treatment. The same treatment course was extended to 15-23
days for patients who were still positive for the virus at day 14.</p>
<p>The authors found a negative association between age and resolution of
infection. Patients with hypertension, diabetes, malignancy or chronic
liver disease were all unable to clear the virus at day 14, though not
statistically significant.</p>
<p>Elevated procalcitonin and a trend for increased IL-6 were also found in
peripheral blood prior to the treatment.</p>
<p>A trend for lower NK cell, T cell and B cell counts in patients was also
reported. B cell, CD4 and CD8 T cell counts were only increased upon
treatment in patients who cleared the virus. NK cell frequencies
remained unchanged after treatment in all the patients.</p>
<h3 data-number="9.40.3" id="limitations-37"><span class="header-section-number">9.40.3</span> Limitations</h3>
<p>73% of the patients who remained positive for SARS-CoV2 after the 1<sup>st</sup>
treatment, and 43% of all patients who cleared the virus were treated
with corticosteroids. Corticosteroids have strong effects on the immune
compartment in blood <span class="citation" data-cites="L6kwccjn">[<a href="#ref-L6kwccjn" role="doc-biblioref">583</a>]</span>. The authors should have accounted for
corticosteroid treatment when considering changes in T, NK and B cell
frequencies.</p>
<p>Assessing if IL-6 concentrations were back to baseline levels following
treatment would have provided insights into the COVID-19 cytokine storm
biology. Patients with higher baseline levels of IL-6 have been reported
to have lower CD8 and CD4 T cell frequencies <span class="citation" data-cites="1B0vBwv1q">[<a href="#ref-1B0vBwv1q" role="doc-biblioref">579</a>]</span>. Correlating IL-6 with
cell counts before and after treatment would thus have also been of
interest. The report of the laboratory measures in table 2 is incomplete
and should include the frequencies of patients with increased/decreased
levels for each parameter.</p>
<p>Correction is needed for the 1<sup>st</sup> paragraph of the discussion as data
does not support NK cell restoration upon treatment in patients who
cleared the virus. NK cells remain unchanged after the 1<sup>st</sup> treatment
course and only seem to increase in 2 out of 6 donors after the 2<sup>nd</sup>
treatment course in those patients.</p>
<h3 data-number="9.40.4" id="significance-35"><span class="header-section-number">9.40.4</span> Significance</h3>
<p>Previous reports suggest an association between disease severity and
elevated IL-6 or pro-calcitonin concentrations in COVID-19
patients <span class="citation" data-cites="T0BDaXkw K9G9QT8y">[<a href="#ref-T0BDaXkw" role="doc-biblioref">575</a>,<a href="#ref-K9G9QT8y" role="doc-biblioref">584</a>]</span>. IL-6 receptor blockade is also being administered to
patients enrolled in clinical trials (NCT04317092). This report thus
contributes to highlight elevated concentrations of these analytes in
COVID-19 patients. Mechanisms underlying the association between viral
clearance and restoration of the T cell and B cell frequencies suggests
viral-driven immune dysregulation, which needs to be investigated in
further studies.</p>
<h3 data-number="9.40.5" id="credit-39"><span class="header-section-number">9.40.5</span> Credit</h3>
<p><em>Review by Bérengère Salomé as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.41" id="clinical-findings-in-critically-ill-patients-infected-with-sars-cov-2-in-guangdong-province-china-a-multi-center-retrospective-observational-study"><span class="header-section-number">9.41</span> Clinical findings in critically ill patients infected with SARS-CoV-2 in Guangdong Province, China: a multi-center, retrospective, observational study</h2>
<p>Xu et al. <em>medRxiv</em>. <span class="citation" data-cites="USOnp7c0">[<a href="#ref-USOnp7c0" role="doc-biblioref">585</a>]</span></p>
<h3 data-number="9.41.1" id="keywords-40"><span class="header-section-number">9.41.1</span> Keywords</h3>
<ul>
<li>clinical outcomes</li>
<li>prognosis</li>
<li>critically ill patients</li>
<li>ICU</li>
<li>lymphopenia</li>
<li>LDH</li>
</ul>
<h3 data-number="9.41.2" id="main-findings-39"><span class="header-section-number">9.41.2</span> Main Findings:</h3>
<p>This work analyses laboratory and clinical data from 45 patients treated
in the in ICU in a single province in China. Overall, 44% of the
patients were intubated within 3 days of ICU admission with only 1
death.</p>
<p>Lymphopenia was noted in 91% of patient with an inverse correlation with
LDH.</p>
<p>Lymphocyte levels are negatively correlated with Sequential Organ
Failure Assessment (SOFA) score (clinical score, the higher the more
critical state), LDH levels are positively correlated to SOFA score.
Overall, older patients (&gt;60yo), with high SOFA score, high LDH
levels and low lymphocytes levels at ICU admission are at higher risk of
intubation.</p>
<p>Of note, convalescent plasma was administered to 6 patients but due to
limited sample size no conclusion can be made.</p>
<h3 data-number="9.41.3" id="limitations-38"><span class="header-section-number">9.41.3</span> Limitations</h3>
<p>While the study offers important insights into disease course and
clinical lab correlates of outcome, the cohort is relatively small and
is likely skewed towards a less-severe population compared to other ICU
reports given the outcomes observed. Analysis of laboratory values and
predictors of outcomes in larger cohorts will be important to make
triage and treatment decisions. As with many retrospective analyses,
pre-infection data is limited and thus it is not possible to understand
whether lymphopenia was secondary to underlying comorbidities or
infection.</p>
<p>Well-designed studies are necessary to evaluate the effect of
convalescent plasma administration.</p>
<h3 data-number="9.41.4" id="significance-36"><span class="header-section-number">9.41.4</span> Significance</h3>
<p>This clinical data enables the identification of at-risk
patients and gives guidance for research for treatment options. Indeed,
further work is needed to better understand the causes of the
lymphopenia and its correlation with outcome.</p>
<h3 data-number="9.41.5" id="credit-40"><span class="header-section-number">9.41.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.42" id="immune-multi-epitope-vaccine-design-using-an-immunoinformatics-approach-for-2019-novel-coronavirus-in-china-sars-cov-2"><span class="header-section-number">9.42</span> Immune Multi-epitope vaccine design using an immunoinformatics approach for 2019 novel coronavirus in China (SARS-CoV-2)</h2>
<p><span class="citation" data-cites="14OLQIYK0">[<a href="#ref-14OLQIYK0" role="doc-biblioref">586</a>]</span></p>
<h3 data-number="9.42.1" id="keywords-41"><span class="header-section-number">9.42.1</span> Keywords</h3>
<ul>
<li>Vaccine</li>
<li>in silico</li>
<li>B cell epitopes</li>
<li>T cell epitopes</li>
</ul>
<h3 data-number="9.42.2" id="main-findings-40"><span class="header-section-number">9.42.2</span> Main Findings</h3>
<p>Using in silico bioinformatic tools, this study identified putative
antigenic B-cell epitopes and HLA restricted T-cell epitopes from the
spike, envelope and membrane proteins of SARS-CoV-2, based on the genome
sequence available on the NCBI database. T cell epitopes were selected
based on predicted affinity for the more common HLA-I alleles in the
Chinese population. Subsequently, the authors designed vaccine peptides
by bridging selected B-cell epitopes and adjacent T-cell epitopes.
Vaccine peptides containing only T-cell epitopes were also generated.</p>
<p>From 61 predicted B-cell epitopes, only 19 were exposed on the surface
of the virion and had a high antigenicity score. A total of 499 T-cell
epitopes were predicted. Based on the 19 B-cell epitopes and their 121
adjacent T-cell epitopes, 17 candidate vaccine peptides were designed.
Additionally, another 102 vaccine peptides containing T-cell epitopes
only were generated. Based on the epitope counts and HLA score, 13 of
those were selected. Thus, a total of 30 peptide vaccine candidates were
designed.</p>
<h3 data-number="9.42.3" id="limitations-39"><span class="header-section-number">9.42.3</span> Limitations</h3>
<p>While this study provides candidates for the development of vaccines
against SARS-CoV-2, in vitro and in vivo trials are required to validate
the immunogenicity of the selected B and T cell epitopes. This could be
done using serum and cells from CoV-2-exposed individuals, and in
preclinical studies. The implication of this study for the current
epidemic are thus limited. Nevertheless, further research on this field
is greatly needed.</p>
<h3 data-number="9.42.4" id="credit-41"><span class="header-section-number">9.42.4</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.43" id="clinical-features-of-patients-infected-with-the-2019-novel-coronavirus-covid-19-in-shanghai-china"><span class="header-section-number">9.43</span> Clinical Features of Patients Infected with the 2019 Novel Coronavirus (COVID-19) in Shanghai, China</h2>
<p>Cao et al. <em>medRxiv</em> <span class="citation" data-cites="12ilBRq6I">[<a href="#ref-12ilBRq6I" role="doc-biblioref">587</a>]</span></p>
<h3 data-number="9.43.1" id="keywords-42"><span class="header-section-number">9.43.1</span> Keywords</h3>
<ul>
<li>Disease severity</li>
<li>clinical features</li>
<li>laboratory abnormalities</li>
</ul>
<h3 data-number="9.43.2" id="main-findings-41"><span class="header-section-number">9.43.2</span> Main Findings</h3>
<p>This single-center cohort study analyzes the clinical and
laboratory features of 198 patients with confirmed COVID-19 infection in
Shanghai, China and correlated these parameters with clinical disease
severity, including subsequent intensive care unit (ICU) admission. 19
cases (9.5%) required ICU admission after developing respiratory failure
or organ dysfunction. Age, male sex, underlying cardiovascular disease,
and high symptom severity (high fever, dyspnea) were all significantly
correlated with ICU admission. Additionally, ICU admission was more
common in patients who presented with lymphopenia and elevated
neutrophil counts, among other laboratory abnormalities. Flow cytometric
analysis revealed that patients admitted to the ICU had significantly
reduced circulating CD3+ T cell, CD4+ T cell, CD8+ T cell, and CD45+
leukocyte populations compared to the cohort of patients not requiring
ICU admission.</p>
<h3 data-number="9.43.3" id="limitations-40"><span class="header-section-number">9.43.3</span> Limitations</h3>
<p>The limitations of this study include the relatively
small sample size and lack of longitudinal testing. The authors also did
not assess whether respiratory comorbidity – such as asthma or chronic
obstructive lung disease – in addition to immunosuppression affected ICU
admission likelihood.</p>
<h3 data-number="9.43.4" id="significance-37"><span class="header-section-number">9.43.4</span> Significance</h3>
<p>COVID-19 has already sickened thousands across the globe,
though the severity of these infections is markedly diverse, ranging
from mild symptoms to respiratory failure requiring maximal
intervention. Understanding what clinical, laboratory, and immunologic
factors predict the clinical course of COVID-19 infection permits
frontline providers to distribute limited medical resources more
effectively.</p>
<h3 data-number="9.43.5" id="credit-42"><span class="header-section-number">9.43.5</span> Credit</h3>
<p><em>Review by Andrew Charap as part of a project by students, postdocs and
faculty at the Immunology Institute of the Icahn School of Medicine at
Mount Sinai.</em></p>
<h2 data-number="9.44" id="serological-detection-of-2019-ncov-respond-to-the-epidemic-a-useful-complement-to-nucleic-acid-testing"><span class="header-section-number">9.44</span> Serological detection of 2019-nCoV respond to the epidemic: A useful complement to nucleic acid testing</h2>
<p>Zhang et al. <em>medRxiv</em>. <span class="citation" data-cites="IPJURDxi">[<a href="#ref-IPJURDxi" role="doc-biblioref">588</a>]</span></p>
<h3 data-number="9.44.1" id="keywords-43"><span class="header-section-number">9.44.1</span> Keywords</h3>
<ul>
<li>Immunoassay</li>
<li>serum IgM and IgG</li>
<li>specific antibodies</li>
</ul>
<h3 data-number="9.44.2" id="main-finding"><span class="header-section-number">9.44.2</span> Main Finding</h3>
<p>This study showed that both anti-2019-nCov IgM and IgG were detected by
automated chemiluminescent immunoassay in the patients who had been
already confirmed as positive by nucleic acid detection, while single
positivity of IgM or IgG were detected in a very few cases in the other
population including 225 non-COVID-19 cases. In addition to the increase
of anti-2019-nCov IgM 7-12 days after morbidity, the increase of IgG was
detected in three patients with COVID-19 within a very short of time
(0-1 day).</p>
<h3 data-number="9.44.3" id="limitations-41"><span class="header-section-number">9.44.3</span> Limitations</h3>
<p>The limitation of this study is only
3 confirmed COVID-19 cases were included, so that the relationship
between anti-2019-nCov antibodies and disease progression might not be
clearly defined. Another limitation is that they did not show the course
of 2019-nCov specific antibodies in the cases with positive for COVID-19
but without clinical symptoms.</p>
<h3 data-number="9.44.4" id="significance-38"><span class="header-section-number">9.44.4</span> Significance</h3>
<p>The detection of anti-2019-nCov antibodies can be an alternative method
to diagnose and treat COVID-19 more comprehensively by distinguish non
COVID-19 patients. It may be helpful to understand the course of
individual cases with COVID-19 to predict the prognosis if more cases
will be evaluated.</p>
<h3 data-number="9.44.5" id="credit-43"><span class="header-section-number">9.44.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.45" id="human-kidney-is-a-target-for-novel-severe-acute-respiratory-syndrome-coronavirus-2-sars-cov-2-infection"><span class="header-section-number">9.45</span> Human Kidney is a Target for Novel Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) Infection</h2>
<p><span class="citation" data-cites="H0nvocJz">[<a href="#ref-H0nvocJz" role="doc-biblioref">589</a>]</span></p>
<h3 data-number="9.45.1" id="keywords-44"><span class="header-section-number">9.45.1</span> Keywords</h3>
<ul>
<li>Kidney/Renal Failure</li>
<li>Macrophage Infiltration</li>
<li>Complement Activation</li>
</ul>
<h3 data-number="9.45.2" id="main-finding-1"><span class="header-section-number">9.45.2</span> Main Finding</h3>
<p>Analyzing the eGFR (effective glomerular flow rate) of 85 Covid-19
patients and characterizing tissue damage and viral presence in
post-mortem kidney samples from 6 Covid-19 patients, the authors
conclude that significant damage occurs to the kidney, following
Covid-19 infection. This is in contrast to the SARS infection from the
2003 outbreak. They determine this damage to be more prevalent in
patients older than 60 years old, as determined by analysis of eGFR. H&amp;E
and IHC analysis in 6 Covid-19 patients revealed that damage was in the
tubules, not the glomeruli of the kidneys and suggested that macrophage
accumulation and C5b-9 deposition are key to this process. </p>
<h3 data-number="9.45.3" id="limitations-42"><span class="header-section-number">9.45.3</span> Limitations</h3>
<p>Severe limitations include that the H&amp;E and IHC samples were performed
on post-mortem samples of unknown age, thus we cannot assess how/if age
correlates with kidney damage, upon Covid-19 infection. Additionally,
eGFR was the only <em>in-vivo</em> measurement. Blood urea nitrogen and
proteinuria are amongst other measurements that could have been obtained
from patient records. An immune panel of the blood was not performed to
assess immune system activation. Additionally, patients are only from
one hospital. </p>
<h3 data-number="9.45.4" id="significance-39"><span class="header-section-number">9.45.4</span> Significance</h3>
<p>This report makes clear that kidney damage is prevalent
in Covid-19 patients and should be accounted for. </p>
<h3 data-number="9.45.5" id="credit-44"><span class="header-section-number">9.45.5</span> Credit</h3>
<p><em>Review by Dan Fu Ruan, Evan Cody and Venu Pothula as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine at Mount Sinai.</em></p>
<h2 data-number="9.46" id="covid-19-early-warning-score-a-multi-parameter-screening-tool-to-identify-highly-suspected-patients"><span class="header-section-number">9.46</span> COVID-19 early warning score: a multi-parameter screening tool to identify highly suspected patients</h2>
<p>Song et al. <em>medRxiv</em>. <span class="citation" data-cites="KHNnjP7M">[<a href="#ref-KHNnjP7M" role="doc-biblioref">590</a>]</span></p>
<h3 data-number="9.46.1" id="keywords-45"><span class="header-section-number">9.46.1</span> Keywords</h3>
<ul>
<li>retrospective</li>
<li>electronic health records</li>
<li>blood counts</li>
<li>diagnostic</li>
<li>prognostic</li>
<li>modeling</li>
</ul>
<h3 data-number="9.46.2" id="main-findings-42"><span class="header-section-number">9.46.2</span> Main Findings</h3>
<p>The aim of this study was to identify diagnostic or prognostic criteria
which could identify patients with COVID-19 and predict patients who
would go on to develop severe respiratory disease. The authors use EMR
data from individuals taking a COVID-19 test at Zhejiang hospital, China
in late January/Early February. A large number of clinical parameters
were different between individuals with COVID-19 and also between
‘severe’ and ‘non-severe’ infections and the authors combine these into
a multivariate linear model to derive a weighted score, presumably
intended for clinical use.</p>
<h3 data-number="9.46.3" id="limitations-43"><span class="header-section-number">9.46.3</span> Limitations</h3>
<p>Unfortunately, the paper is lacking a lot of crucial information, making
it impossible to determine the importance or relevance of the findings.
Most importantly, the timings of the clinical measurements are not
described relative to the disease course, so it is unclear if the
differences between ‘severe’ and ‘non-severe’ infections are occurring
before progression to severe disease (which would make them useful
prognostic markers), or after (which would not).</p>
<h3 data-number="9.46.4" id="significance-40"><span class="header-section-number">9.46.4</span> Significance</h3>
<p>This paper is one of many retrospective studies coming from hospitals in
China studying individuals with COVID-19. Because of the sparse
description of the study design, this paper offers little new
information. However, studies like this could be very valuable and we
would strongly encourage the authors to revise this manuscript to
include more information about the timeline of clinical measurements in
relation to disease onset and more details of patient outcomes.</p>
<h3 data-number="9.46.5" id="credit-45"><span class="header-section-number">9.46.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.47" id="ly6e-impairs-coronavirus-fusion-and-confers-immune-control-of-viral-disease"><span class="header-section-number">9.47</span> LY6E impairs coronavirus fusion and confers immune control of viral disease</h2>
<p>Pfaender et al. <em>bioRxiv</em>. <span class="citation" data-cites="19xwRoatu">[<a href="#ref-19xwRoatu" role="doc-biblioref">591</a>]</span></p>
<h3 data-number="9.47.1" id="keywords-46"><span class="header-section-number">9.47.1</span> Keywords</h3>
<ul>
<li>interferon-stimulated genes</li>
<li>antiviral interferons</li>
<li>human coronaviruses (CoV)</li>
<li>murine hepatitis virus (MHV)</li>
</ul>
<h3 data-number="9.47.2" id="main-findings-43"><span class="header-section-number">9.47.2</span> Main Findings</h3>
<p>Screening a cDNA library of &gt;350 human interferon-stimulated genes
for antiviral activity against endemic human coronavirus HCoV-229E
(associated with the common cold), Pfaender S &amp; Mar K <em>et al.</em> identify
lymphocyte antigen 6 complex, locus E (Ly6E) as an inhibitor of cellular
infection of Huh7 cells, a human hepatoma cell line susceptible to
HCoV-229E and other coronaviruses. In a series of consecutive <em>in vitro</em>
experiments including both stable Ly6E overexpression and
CRISPR-Cas9-mediated knockout the authors further demonstrate that Ly6E
reduces cellular infection by various other coronaviruses including
human SARS-CoV and SARS-CoV-2 as well as murine CoV mouse hepatitis
virus (MHV). Their experiments suggest that this effect is dependent on
Ly6E inhibition of CoV strain-specific spike protein-mediated membrane
fusion required for viral cell entry.</p>
<p>To address the function of Ly6E <em>in vivo</em>, hematopoietic stem
cell-specific Ly6E knock-out mice were generated by breeding Ly6E<sup>fl/fl</sup>
mice (referred to as functional wild-type mice) with transgenic
<em>Vav-iCre</em> mice (offspring referred to as Ly6E HSC ko mice); wild-type
and Ly6E HSC ko mice of both sexes were infected intraperitoneally with
varying doses of the natural murine coronavirus MHV, generally causing a
wide range of diseases in mice including hepatitis, enteritis and
encephalomyelitis. Briefly, compared to wild-type controls, mice lacking
hematopoietic cell-expressed Ly6E were found to present with a more
severe disease phenotype as based on serum ALT levels (prognostic of
liver damage), liver histopathology, and viral titers in the spleen.
Moreover, bulk RNAseq analysis of infected liver and spleen tissues
indicated changes in gene expression pathways related to tissue damage
and antiviral immune responses as well as a reduction of genes
associated with type I IFN response and inflammation. Finally, the
authors report substantial differences in the numbers of hepatic and
splenic APC subsets between wild-type and knockout mice following MHV
infection and show that Ly6E-deficient B cells and to a lesser extent
also DCs are particularly susceptible to MHV infection <em>in vitro</em>.</p>
<h3 data-number="9.47.3" id="limitations-44"><span class="header-section-number">9.47.3</span> Limitations</h3>
<p>Experiments and data in this study are presented in an overall logical
and coherent fashion; however, some observations and the conclusions
drawn are problematic and should be further addressed &amp; discussed by the
authors. Methodological &amp; formal limitations include relatively low
replicate numbers as well as missing technical replicates for some <em>in
vitro</em> experiments (<em>cf</em>. Fig. legend 1; Fig. legend 2e); the omission
of “outliers” in Fig. legend 2 without an apparent rationale as to why
this approach was chosen; the lack of detection of actual Ly6E protein
levels in Ly6E HSC ko or wild-type mice; and most importantly, missing
information on RNAseq data collection &amp; analysis in the method section
and throughout the paper. A more relevant concern though is that the
interpretation of the experimental data presented and the language used
tend to overrate and at times overgeneralize findings: for example,
while the authors demonstrate statistically significant, Ly6E-mediated
reduction of coronavirus titers in stable cells lines <em>in vitro</em>, it
remains unclear whether a viral titer reduction by one log decade would
be of actual biological relevance in face of high viral titers <em>in
vivo</em>. After high-dose intraperitoneal MHV infection <em>in vivo</em>, early
viral titers in Ly6E HSC knockout vs. wt mice only showed an elevation
in the spleen (~1.5 log decades) but not liver of the ko mice (other
tissue not evaluated), and while ko mice presented with only modestly
increased liver pathology, both male and female ko mice exhibited
significantly higher mortality. Thus, the manuscript tile statement that
“Ly6E … confers immune control of viral disease” is supported by only
limited <em>in vivo</em> data, and gain-of-function experiments (eg. Ly6E
overexpression) were not performed. Of additional note here, tissue
tropism and virulence differ greatly among various MHV strains and
isolates whereas dose, route of infection, age, genetic background and
sex of the mice used may additionally affect disease outcome and
phenotype (<em>cf</em>. Taguchi F &amp; Hirai-Yuki A,
<a href="https://doi.org/10.3389/fmicb.2012.00068" class="uri">https://doi.org/10.3389/fmicb.2012.00068</a>; Kanolkhar A et al,
<a href="https://jvi.asm.org/content/%2083/18/9258">https://jvi.asm.org/content/
83/18/9258</a>). Observations
attributed to hematopoietic stem cell-specific Ly6E deletion could
therefore be influenced by the different genetic backgrounds of floxed
and cre mice used, and although it appears that littermates wt and ko
littermates were used in the experiments, the potentially decisive
impact of strain differences should at least have been discussed. Along
these lines, it should also be taken into account that the majority of
human coronaviruses cause respiratory symptoms, which follow a different
clinical course engaging other primary cellular mediators than the
hepatotropic murine MHV disease studied here. It therefore remains
highly speculative how the findings reported in this study will
translate to human disease and it would therefore be important to test
other routes of MHV infection and doses that have been described to
produce a more comparable phenotype to human coronavirus disease (<em>cf.</em>
Kanolkhar A et al, <a href="https://jvi.asm.org/content/%2083/18/9258">https://jvi.asm.org/content/
83/18/9258</a>). Another
important shortcoming of this study is the lack of any information on
functional deficits or changes in Ly6E-deficient immune cells and how
this might relate to the phenotype observed. Overall, the <em>in vitro</em>
experiments are more convincing than the <em>in vivo</em> studies which appear
somewhat limited.</p>
<h3 data-number="9.47.4" id="significance-41"><span class="header-section-number">9.47.4</span> Significance</h3>
<p>Despite some shortcomings, the experiments performed in this study
suggest a novel and somewhat unexpected role of Ly6E in the protection
against coronaviruses across species. These findings are of relevance
and should be further explored in ongoing research on potential
coronavirus therapies. Yet an important caveat pertains to the authors’
suggestion that “therapeutic mimicking of Ly6E action” may constitute a
first line of defense against novel coronaviruses since their own prior
work demonstrated that Ly6E can enhance rather than curtail infection
with influenza A and other viruses.</p>
<h3 data-number="9.47.5" id="credit-46"><span class="header-section-number">9.47.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.48" id="a-preliminary-study-on-serological-assay-for-severe-acute-respiratory-syndrome-coronavirus-2-sars-cov-2-in-238-admitted-hospital-patients"><span class="header-section-number">9.48</span> A preliminary study on serological assay for severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) in 238 admitted hospital patients</h2>
<p>Liu et al. <em>medRxiv</em>. <span class="citation" data-cites="574aSa8C">[<a href="#ref-574aSa8C" role="doc-biblioref">592</a>]</span></p>
<h3 data-number="9.48.1" id="keywords-47"><span class="header-section-number">9.48.1</span> Keywords</h3>
<ul>
<li>diagnosis</li>
<li>serological assay</li>
<li>ELISA</li>
<li>RT-PCR</li>
</ul>
<h3 data-number="9.48.2" id="main-findings-44"><span class="header-section-number">9.48.2</span> Main Findings</h3>
<p>While RT-PCR is being used currently to routinely diagnose infection
with SARS-CoV-2, there are significant limitations to the use of a
nucleic acid test that lead to a high false-negative rate. This
article describes ELISAs that can measure IgM and IgG antibodies
against the N protein of SARS-CoV-2 to test samples from 238
patients (153 positive by RT-PCR and 85 negative by RT-PCR) at
different times after symptom onset. The positivity rate of the IgM
and/or IgG ELISAs was greater than that of the RT-PCR (81.5%
compared to 64.3%) with similar positive rates in the confirmed and
suspected cases (83% and 78.8%, respectively), suggesting that many
of the suspected but RT-PCR-negative cases were also infected. The
authors also found that the ELISAs have higher positive rates later
after symptom onset while RT-PCR is more effective as a diagnostic
test early during the infection.</p>
<h3 data-number="9.48.3" id="limitations-45"><span class="header-section-number">9.48.3</span> Limitations</h3>
<p>I cannot identify any limitations to this study.</p>
<h3 data-number="9.48.4" id="significance-42"><span class="header-section-number">9.48.4</span> Significance</h3>
<p>The authors make a strong case for using a combination of ELISA and
RT-PCR for diagnosis of infection with SARS-CoV-2, especially
considering the dynamics of positivity rates of RT-PCR and ELISA.
Fewer false-negative diagnoses would improve infection control and
patient management.</p>
<h3 data-number="9.48.5" id="credit-47"><span class="header-section-number">9.48.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.49" id="monoclonal-antibodies-for-the-s2-subunit-of-spike-of-sars-cov-cross-react-with-the-newly-emerged-sars-cov-2"><span class="header-section-number">9.49</span> Monoclonal antibodies for the S2 subunit of spike of SARS-CoV cross-react with the newly-emerged SARS-CoV-2</h2>
<p><span class="citation" data-cites="NT3T9a8H">[<a href="#ref-NT3T9a8H" role="doc-biblioref">593</a>]</span></p>
<h3 data-number="9.49.1" id="keywords-48"><span class="header-section-number">9.49.1</span> Keywords</h3>
<ul>
<li>Coronavirus Disease 2019 (COVID-19)</li>
<li>SARS-CoV-2</li>
<li>Spike protein</li>
<li>Cross- reactive antibodies</li>
</ul>
<h3 data-number="9.49.2" id="main-findings-45"><span class="header-section-number">9.49.2</span> Main Findings</h3>
<p>Whole genome sequencing-based comparisons of the 2003
Severe Acute Respiratory Syndrome Coronavirus (SARS-CoV) and the 2019
SARS-CoV-2 revealed conserved receptor binding domain (RBD) and host
cell receptor, angiotensin-converting enzyme 2 (ACE2). In line with
this, the authors tested cross-reactivity of murine monoclonal
antibodies (mAbs) previously generated against the SARS-CoV spike (S)
glycoprotein involved in viral entry. One of the screened mAb, 1A9, was
able to bind and cross-neutralize multiple strains of SARS-CoV, as well
as, detect the S protein in SARS-CoV-2-infected cells. mAb 1A9 was
generated using an immunogenic fragment in the S2 subunit of SARS-CoV
and binds through a novel epitope within the S2 subunit at amino acids
1111-1130. It is important to note that CD8+ T lymphocyte epitopes
overlap with these residues, suggesting that S2 subunit could be
involved in inducing both, humoral and cell-mediated immunity.</p>
<h3 data-number="9.49.3" id="limitations-46"><span class="header-section-number">9.49.3</span> Limitations</h3>
<p>The authors used previously generated mouse mAbs
against the S protein in SARS-CoV expressed in mammalian cell line.
Future experimental validation using COVID-19 patient samples is needed
to validate these findings. In addition, the results of these studies
are predominantly based on in vitro experiments and so, evaluating the
effects of the mAb 1A9 in an animal model infected with this virus will
help us better understand the host immune responses in COVID-19 and
potential therapeutic vaccines.</p>
<h3 data-number="9.49.4" id="significance-43"><span class="header-section-number">9.49.4</span> Significance</h3>
<p>This study identified mAbs that recognize the new
coronavirus, SARS-Cov-2. These cross-reactive mAbs will help in
developing diagnostic assays for COVID-19.</p>
<h3 data-number="9.49.5" id="credit-48"><span class="header-section-number">9.49.5</span> Credit</h3>
<p><em>This review was undertaken by Tamar Plitt and Katherine Lindblad as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.50" id="mortality-of-covid-19-is-associated-with-cellular-immune-function-compared-to-immune-function-in-chinese-han-population"><span class="header-section-number">9.50</span> Mortality of COVID-19 is Associated with Cellular Immune Function Compared to Immune Function in Chinese Han Population</h2>
<p>Zeng et al. <em>medRxiv</em>. <span class="citation" data-cites="1GEbBtBgh">[<a href="#ref-1GEbBtBgh" role="doc-biblioref">594</a>]</span></p>
<h3 data-number="9.50.1" id="keywords-49"><span class="header-section-number">9.50.1</span> Keywords</h3>
<ul>
<li>WBC</li>
<li>peripheral blood</li>
<li>CD4</li>
<li>CD8 T cells</li>
</ul>
<h3 data-number="9.50.2" id="main-findings-46"><span class="header-section-number">9.50.2</span> Main Findings</h3>
<p>Retrospective study of the clinical characteristics of 752
patients infected with COVID-19 at Chinese PLA General Hospital, Peking
Union Medical College Hospital, and affiliated hospitals at Shanghai
University of medicine &amp; Health Sciences. This study is the first one
that compares PB from healthy controls from the same regions in Shanghai
and Beijing, and infected COVID-19 patients to standardize a reference
range of WBCs of people at high risk.</p>
<h3 data-number="9.50.3" id="limitations-47"><span class="header-section-number">9.50.3</span> Limitations</h3>
<p>Lower levels of leukocyte counts -B cells, CD4 and CD8 T
cells- correlated with mortality (WBCs are significantly lower in severe
or critical UCI patients vs mild ones). Based on 14,117 normal controls
in Chinese Han population (ranging in age from 18-86) it is recommended
that reference ranges of people at high risk of COVID-19 infection are
CD3+ lymphocytes below 900 cells/mm3, CD4+ lymphocytes below 500
cells/mm3, and CD8+ lymphocytes below 300 cells/mm3. Importantly, this
study also reported that the levels of D-dimer, C-reactive protein and
IL-6 were elevated in COVID-19 pts., indicating clot formation, severe
inflammation and cytokine storm.</p>
<h3 data-number="9.50.4" id="significance-44"><span class="header-section-number">9.50.4</span> Significance</h3>
<p>This study sets a threshold to identify patients at risk
by analyzing their levels of leukocytes, which is an easy and fast
approach to stratify individuals that require hospitalization. Although
the study is limited (only counts of WBC are analyzed and not its
profile) the data is solid and statistically robust to correlate levels
of lymphopenia with mortality.</p>
<h3 data-number="9.50.5" id="credit-49"><span class="header-section-number">9.50.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.51" id="retrospective-analysis-of-clinical-features-in-101-death-cases-with-covid-19"><span class="header-section-number">9.51</span> Retrospective Analysis of Clinical Features in 101 Death Cases with COVID-19</h2>
<p>Chen et al. <em>medRxiv</em>. <span class="citation" data-cites="WuHcOc9T">[<a href="#ref-WuHcOc9T" role="doc-biblioref">595</a>]</span></p>
<h3 data-number="9.51.1" id="keywords-50"><span class="header-section-number">9.51.1</span> Keywords</h3>
<ul>
<li>death biomarkers</li>
<li>cardiac damage</li>
<li>Troponin</li>
<li>Blood type</li>
<li>respiratory failure</li>
<li>hypertension</li>
</ul>
<h3 data-number="9.51.2" id="main-findings-47"><span class="header-section-number">9.51.2</span> Main Findings</h3>
<p>This is a retrospective study involving 101 death cases with COVID-19 in
Wuhan Jinyintan Hospital. The aim was to describe clinical,
epidemiological and laboratory features of fatal cases in order to
identify the possible primary mortality causes related to COVID-19.</p>
<p>Among 101 death cases, 56.44% were confirmed by RT-PCR and 43.6% by
clinical diagnostics. Males dominated the number of deaths and the
average age was 65.46 years. All patients died of respiratory failure
and multiple organs failure, except one (acute coronary syndrome). The
predominant comorbidities were hypertension (42.57%) and diabetes
(22.77%). 25.74% of the patients presented more than two underlying
diseases. 82% of patients presented myocardial enzymes abnormalities at
admission and further increase in myocardial damage indicators with
disease progression: patients with elevated Troponin I progressed faster
to death. Alterations in coagulation were also detected. Indicators of
liver and kidney damage increased 48 hours before death. The authors
studied the deceased patients’ blood type and presented the following
results: type A (44.44%), type B (29.29%), type AB (8.08%) and type O
(18.19%), which is inconsistent with the distribution in Han population
in Wuhan.</p>
<p>Clinical analysis showed that the most common symptom was fever (91.9%),
followed by cough and dyspnea. The medium time from onset of symptoms to
acute respiratory distress syndrome (ARDS) development was 12 days.
Unlike SARS, only 2 patients with COVID-19 had diarrhea. 98% presented
abnormal lung imaging at admission and most had double-lung
abnormalities. Related to the laboratorial findings some inflammatory
indicators gradually increased during the disease progression, such as
IL-6 secretion in the circulation, procalcitonin (PCT) and C-reactive
protein (CRP), while platelets numbers decreased. The authors also
reported an initial lymphopenia that was followed by an increase in the
lymphocytes numbers. Neutrophil count increased with disease
progression.</p>
<p>The patients received different treatments such as antiviral drugs
(60.40%), glucocorticoids, thymosin and immunoglobulins. All patients
received antibiotic treatment and some received antifungal drugs. All
patients received oxygen therapy (invasive or non-invasive ones).</p>
<h3 data-number="9.51.3" id="limitations-48"><span class="header-section-number">9.51.3</span> Limitations</h3>
<p>This study involves just fatal patients, lacking comparisons with other
groups of patients e.g. patients that recovered from COVID-19. The
authors didn’t discuss the different approaches used for treatments and
how these may affect the several parameters measured. The possible
relationship between the increase of inflammatory indicators and
morbidities of COVID-19 are not discussed.</p>
<h3 data-number="9.51.4" id="significance-45"><span class="header-section-number">9.51.4</span> Significance</h3>
<p>This study has the largest cohort of fatal cases reported so far. The
authors show that COVID-19 causes fatal respiratory distress syndrome
and multiple organ failure. This study highlights prevalent myocardial
damage and indicates that cardiac function of COVID-19 patients should
be carefully monitored. The data suggest that Troponin I should be
further investigated as an early indicator of patients with high risk of
accelerated health deterioration. Secondary bacterial and fungal
infections were frequent in critically ill patients and these need to be
carefully monitored in severe COVID-19 patients. Differences in blood
type distribution were observed, suggesting that type A is detrimental
while type O is protective – but further studies are needed to confirm
these findings and elucidate if blood type influences infection or
disease severity. Several inflammatory indicators (neutrophils, PCT, CRP
and IL-6, D-dimer) increased according to disease severity and should be
assessed as biomarkers and to better understand the biology of
progression to severe disease.</p>
<h3 data-number="9.51.5" id="credit-50"><span class="header-section-number">9.51.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.52" id="relationship-between-the-abo-blood-group-and-the-covid-19-susceptibility"><span class="header-section-number">9.52</span> Relationship between the ABO Blood Group and the COVID-19 Susceptibility</h2>
<p>Zhao et al. <em>medRxiv</em>. <span class="citation" data-cites="1DbV444Pj">[<a href="#ref-1DbV444Pj" role="doc-biblioref">596</a>]</span></p>
<h3 data-number="9.52.1" id="keywords-51"><span class="header-section-number">9.52.1</span> Keywords</h3>
<ul>
<li>ABO blood group</li>
<li>COVID-19 susceptibility</li>
</ul>
<h3 data-number="9.52.2" id="main-findings-48"><span class="header-section-number">9.52.2</span> Main Findings</h3>
<p>These authors compared the ABO blood group of 2,173 patients with
RT-PCR-confirmed COVID-19 from hospitals in Wuhan and Shenzhen with the
ABO blood group distribution in unaffected people in the same cities
from previous studies (2015 and 2010 for Wuhan and Shenzhen,
respectively). They found that people with blood group A are
statistically over-represented in the number of those infected and who
succumb to death while those with blood group O are statistically
underrepresented with no influence of age or sex.</p>
<h3 data-number="9.52.3" id="limitations-49"><span class="header-section-number">9.52.3</span> Limitations</h3>
<p>This study compares patients with COVID-19 to the general population
but relies on data published 5 and 10 years ago for the control. The
mechanisms that the authors propose may underlie the differences they
observed require further study.</p>
<h3 data-number="9.52.4" id="significance-46"><span class="header-section-number">9.52.4</span> Significance</h3>
<p>Risk stratification based on blood group may be beneficial for
patients and also healthcare workers in infection control. Additionally,
investigating the mechanism behind these findings could lead to better
developing prophylactic and therapeutic targets for COVID-19.</p>
<h3 data-number="9.52.5" id="credit-51"><span class="header-section-number">9.52.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.53" id="the-inhaled-corticosteroid-ciclesonide-blocks-coronavirus-rna-replication-by-targeting-viral-nsp15"><span class="header-section-number">9.53</span> The inhaled corticosteroid ciclesonide blocks coronavirus RNA replication by targeting viral NSP15</h2>
<p>Matsuyama et al. <em>bioRxiv</em> <span class="citation" data-cites="JwL9G6bd">[<a href="#ref-JwL9G6bd" role="doc-biblioref">597</a>]</span></p>
<h3 data-number="9.53.1" id="keywords-52"><span class="header-section-number">9.53.1</span> Keywords</h3>
<ul>
<li>Corticosteroids</li>
<li>ciclesonide</li>
<li>mometasone</li>
<li>NSP15</li>
<li>MERS-CoV</li>
</ul>
<h3 data-number="9.53.2" id="main-findings-49"><span class="header-section-number">9.53.2</span> Main Findings</h3>
<p>This study reconsiders the use of inhaled corticosteroids in the
treatment of pneumonia by coronavirus. Corticosteroids were associated
with increased mortality for SARS in 2003 and for MERS in 2013, probably
due to that fact that systemic corticosteroids suppress the innate
immune system, resulting in increased viral replication. However, some
steroid compounds might block coronavirus replication. The authors
screened steroids from a chemical library and assessed the viral growth
suppression and drug cytotoxicity. Ciclesonide demonstrated low
cytotoxicity and potent suppression of MERS-CoV viral growth. The
commonly used systemic steroids cortisone, prednisolone and
dexamethasone did not suppress viral growth, nor did the commonly used
inhaled steroid fluticasone. To identify the drug target of virus
replication, the authors conducted 11 consecutive MERS-CoV passages in
the presence of ciclesonide or mometasone, and they could generate a
mutant virus that developed resistance to ciclesonide, but not to
mometasone. Afterwards, they performed next-generation sequencing and
identified an amino acid substitution in nonstructural protein 15
(NSP15) as the predicted mechanism for viral resistance to ciclesonide.
The authors were able to successfully generate a recombinant virus
carrying that amino acid substitution, which overcome the antiviral
effect of ciclesonide, suggesting that ciclosenide interacts with NSP15.
The mutant virus was inhibited by mometasone, suggesting that the
antiviral target of mometasone is different from that of ciclesonide.
Lastly, the effects of ciclesonide and mometason on suppressing the
replication of SARS-CoV-2 were evaluated. Both compounds were found to
suppress viral replication with a similar efficacy to lopinavir.</p>
<h3 data-number="9.53.3" id="limitations-50"><span class="header-section-number">9.53.3</span> Limitations</h3>
<p>Most of the experiments, including the identification of the mutation in
NSP15 were conducted with MERS-CoV. This is not the closest related
virus to SARS-CoV-2, as that would be SARS-CoV. Thus, to repeat the
initial experiments with SARS-CoV, or preferably SARS-CoV-2, is
essential. The manuscript should address this and, therefore, it will
require considerable editing for organization and clarity. Also, in
terms of cell immunogenic epitopes, while SARS-CoV-2 spike protein
contains several predicted B and T cell immunogenic epitopes that are
shared with other coronaviruses, some studies have shown critical
differences between MERS-CoV, SARS-CoV and SARS-CoV-2. A main criticism
is that the authors only used VeroE6/TMPRSS2 cells to gauge the direct
cytotoxic effects of viral replication. To evaluate this in other cell
lines, including human airway epithelial cells, is crucial, as the
infectivity of coronavirus strains greatly varies in different cell
lines,</p>
<h3 data-number="9.53.4" id="significance-47"><span class="header-section-number">9.53.4</span> Significance</h3>
<p>Nevertheless, these findings encourage evaluating ciclesonide and
mometasone as better options for patients with COVID-19 in need of
inhaled steroids, especially as an alternative to other corticosteroids
that have been shown to increase viral replication in vitro. This should
be evaluated in future clinical studies.</p>
<h3 data-number="9.53.5" id="credit-52"><span class="header-section-number">9.53.5</span> Credit</h3>
<p><em>This review was undertaken by Alvaro Moreira, MD as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.54" id="a-human-monoclonal-antibody-blocking-sars-cov-2-infection"><span class="header-section-number">9.54</span> A human monoclonal antibody blocking SARS-CoV-2 infection **</h2>
<p>Wang et al. <em>bioRxiv</em>. <span class="citation" data-cites="XcuzhxrJ">[<a href="#ref-XcuzhxrJ" role="doc-biblioref">460</a>]</span></p>
<h3 data-number="9.54.1" id="keywords-53"><span class="header-section-number">9.54.1</span> Keywords</h3>
<ul>
<li>Monoclonal antibodies</li>
<li>SARS-CoV2</li>
<li>cross-neutralization</li>
<li>potential treatment</li>
<li>spike receptor</li>
</ul>
<h3 data-number="9.54.2" id="main-findings-50"><span class="header-section-number">9.54.2</span> Main Findings</h3>
<p>The authors reported a human monoclonal
antibody that neutralizes SARS-CoV-2 and SARS-Cov which belong to same
family of corona viruses. For identifying mAbs, supernatants of a
collection of 51 hybridomas raised against the spike protein of SARS-CoV
(SARS-S) were screened by ELISA for cross-reactivity against the spike
protein of SARS-CoN2 (SARS2-S). Hybridomas were derived from immunized
transgenic H2L2 mice (chimeric for fully human VH-VL and rat constant
region). Four SARS-S hybridomas displayed cross-reactivity with SARS2-S,
one of which (47D11) exhibited cross-neutralizing activity for SARS-S
and SARS2-S pseudotyped VSV infection. A recombinant, fully human IgG1
isotype antibody was generated and used for further characterization.</p>
<p>The humanized 47D11 antibody inhibited infection of VeroE6 cells with
SARS-CoV and SARS-CoV-2 with IC50 values of 0.19 and 0.57 μg/ml
respectively. 47D11 mAb bound a conserved epitope on the spike receptor
binding domain (RBD) explaining its ability to cross-neutralize SARS-CoV
and SARS-CoV-2. 47D11 was shown to target the S1B RBD of SARS-S and
SARS2-S with similar affinities. Interestingly, binding of 47D11 to
SARS-S1B and SARS2-S1B did not interfere with S1B binding to ACE2
receptor-expressing cells assayed by flow cytometry.</p>
<h3 data-number="9.54.3" id="limitations-51"><span class="header-section-number">9.54.3</span> Limitations</h3>
<p>These results show that the human 47D11 antibody
neutralizes SARS-CoV and SARS-Cov2 infectivity via an as yet unknown
mechanism that is different from receptor binding interference.
Alternative mechanisms were proposed but these as yet remain to be
tested in the context of SARS-CoV2. From a therapeutic standpoint and in
the absence of in vivo data, it is unclear whether the 47D11 ab can
alter the course of infection in an infected host through virus
clearance or protect an uninfected host that is exposed to the virus.
There is a precedent for the latter possibility as it relates to
SARS-CoV that was cited by the authors and could turn out to be true for
SARS-CoV2.</p>
<h3 data-number="9.54.4" id="significance-48"><span class="header-section-number">9.54.4</span> Significance</h3>
<p>This study enabled the identification of novel neutralizing
antibody against COV-that could potentially be used as first line of
treatment in the near future to reduce the viral load and adverse
effects in infected patients. In addition, neutralizing antibodies such
as 47D11 represent promising reagents for developing
antigen-antibody-based detection test kits and assays.</p>
<h3 data-number="9.54.5" id="credit-53"><span class="header-section-number">9.54.5</span> Credit</h3>
<p><em>This review was edited by K. Alexandropoulos as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<p><strong>Heat inactivation of serum interferes with the immunoanalysis of
antibodies to SARS-CoV-2</strong></p>
<p>Heat inactivation, immunochromatography, diagnosis, serum antibodies,
IgM, IgG</p>
<p><em>Summary</em></p>
<p>The use of heat inactivation to neutralize pathogens in serum samples
collected from suspected COVID-19 patients reduces the sensitivity of a
fluorescent immunochromatographic assay to detect anti-SARS-CoV-2 IgM
and IgG.</p>
<p><em>Major findings</em></p>
<p>Coronaviruses can be killed by heat inactivation, and this is an
important safety precaution in laboratory manipulation of clinical
samples. However, the effect of this step on downstream
SARS-CoV-2-specific serum antibody assays has not been examined. The
authors tested the effect of heat inactivation (56 deg C for 30 minutes)
versus no heat inactivation on a fluorescence immunochromatography
assay. Heat inactivation reduced all IgM measurements by an average of
54% and most IgG measurements (22/36 samples, average reduction of 50%),
consistent with the lower thermal stability of IgM than that of IgG.
Heat inactivation caused a subset of IgM but not IgG readings to fall
below a specified positivity threshold.</p>
<p><em>Limitations</em></p>
<p>Limitations included the use of only one type of assay for testing heat
inactivated vs non-inactivated sera, and the use of the same baseline
for heat inactivated and non-inactivated sera. The results indicate that
heat inactivation affects the quantification of SARS-CoV-2-antibody
response, specially IgM, but still allows to distinguish positive
specific IgG. Therefore, the effect of heat inactivation should be
studied when designing assays that quantitatively associate
immunoglobulin levels (especially IgM) to immune state.</p>
<p><em>Review by Andrew M. Leader as part of a project by students, postdocs
and faculty at the Immunology Institute of the Icahn school of medicine,
Mount Sinai.</em></p>
<h2 data-number="9.55" id="immune-phenotyping-based-on-neutrophil-to-lymphocyte-ratio-and-igg-predicts-disease-severity-and-outcome-for-patients-with-covid-19"><span class="header-section-number">9.55</span> Immune phenotyping based on neutrophil-to-lymphocyte ratio and IgG predicts disease severity and outcome for patients with COVID-19</h2>
<p>Zhang et al. <em>medRxiv</em> <span class="citation" data-cites="1CwCTXPqj">[<a href="#ref-1CwCTXPqj" role="doc-biblioref">598</a>]</span></p>
<h3 data-number="9.55.1" id="keywords-54"><span class="header-section-number">9.55.1</span> Keywords</h3>
<ul>
<li>Biomarkers</li>
<li>cytokines</li>
<li>IgG</li>
<li>immune cells</li>
</ul>
<h3 data-number="9.55.2" id="main-findings-51"><span class="header-section-number">9.55.2</span> Main Findings</h3>
<p>In a cohort of 222 patients, anti-SARS-CoV-2 IgM and IgG levels were
analyzed during acute and convalescent phases (up to day 35) and
correlated to the diseases’ severity. The same was done with
neutrophil-to-lymphocyte ratio. High IgG levels and high
neutrophil-to-lymphocyte ratio in convalescence were both independently
associated to the severity of the disease. The simultaneous occurrence
of both of these laboratory findings correlated even stronger to the
diseases’ severity.</p>
<p>Severe cases with high neutrophil-to-lymphocyte ratios had clearly
higher levels of IL-6. The authors propose that a robust IgG response
leads to immune-mediated tissue damage, thus explaining the worse
outcome in patients with overexuberant antibody response.</p>
<h3 data-number="9.55.3" id="limitations-52"><span class="header-section-number">9.55.3</span> Limitations</h3>
<p>A main criticism is that the criteria for stratifying patients in severe
vs. non-severe are not described. The only reference related to this is
the difference between the percentage of patients who needed mechanical
ventilation, which was greater in patients with both high IgG levels and
high neutrophil-to-lymphocyte ratio. No patient with both low IgG levels
and low neutrophil-to-lymphocyte ratio was treated with mechanical
ventilation.</p>
<p>The proposed correlation of severity with IL-2 and IL-10 levels is not
very strong.</p>
<p>Furthermore, although mostly ignored in the paper’s discussion, one of
the most interesting findings is that an early increase in
anti-SARS-CoV-2 IgM levels also seems to correlate with severe disease.
However, as only median values are shown for antibody kinetics curves,
the extent of variation in acute phase cannot be assessed.</p>
<h3 data-number="9.55.4" id="significance-49"><span class="header-section-number">9.55.4</span> Significance</h3>
<p>Anti-SARS-CoV-2 IgG levels and with neutrophil-to-lymphocyte ratio
predict severity of COVID-19 independently of each other. An additive
predictive value of both variables is noticeable. Importantly, an
early-on increase in anti-SARS-CoV-2 IgM levels also seem to predict
outcome.</p>
<h3 data-number="9.55.5" id="credit-54"><span class="header-section-number">9.55.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.56" id="reinfection-could-not-occur-in-sars-2-cov-2-infected-rhesus-macaques"><span class="header-section-number">9.56</span> Reinfection could not occur in SARS-2 CoV-2 infected rhesus macaques</h2>
<p>Bao et al. <em>bioRxiv</em> <span class="citation" data-cites="3XqFhi8l">[<a href="#ref-3XqFhi8l" role="doc-biblioref">599</a>]</span></p>
<h3 data-number="9.56.1" id="keywords-55"><span class="header-section-number">9.56.1</span> Keywords</h3>
<ul>
<li>SARS-CoV-2</li>
<li>viral load</li>
<li>reinfection</li>
<li>relapse</li>
<li>non-human primate model</li>
</ul>
<h3 data-number="9.56.2" id="main-findings-52"><span class="header-section-number">9.56.2</span> Main Findings</h3>
<p>This study addresses the issue or acquired immunity after a primary
COVID-19 infection in rhesus monkeys. Four Chinese rhesus macaques were
intratracheally infected with SARS-CoV-2 and two out of the four were
re-infected at 28 days post initial infection (dpi) with the same viral
dose after confirming the recovery by the absence of clinical symptoms,
radiological abnormalities and viral detection (2 negative RT-PCR
tests). While the initial infection led the viral loads in nasal and
pharyngeal swabs that reach approximately 6.5 log<sub>10</sub> RNA copies/ml at 3
dpi in all four monkeys, viral loads in the swabs tested negative after
reinfection in the two reinfected monkeys. In addition, the necropsies
from a monkey (M1) at 7 days after primary infection, and another monkey
(M3) at 5 days post reinfection, revealed the histopathological damages
and viral replication in the examined tissues from M1, while no viral
replication as well as no histological damages were detected in the
tissues from M3. Furthermore, sera from three monkeys at 21 and 28 dpi
exhibited neutralizing activity against SARS-CoV-2 in vitro, suggesting
the production of protective neutralizing antibodies in these monkeys.
Overall, this study indicates that primary infection with SARS-CoV-2 may
protect from subsequent exposure to the same virus.</p>
<h3 data-number="9.56.3" id="limitations-53"><span class="header-section-number">9.56.3</span> Limitations</h3>
<p>In human, virus has been detected by nasopharyngeal swabs until 9 to 15
days after the onset of symptoms. In the infected monkeys in this study,
virus were detected from day 1 after the infection, declining to
undetectable level by day 15 post infection. It may suggest that there
is a faster viral clearance mechanism in monkeys, therefore the
conclusions of reinfection protection for humans need to be carefully
considered. In addition, only two monkeys were re-infected in this study
and the clinical signs of these monkeys were not similar: M3 did not
show weight loss and M4 showed relatively higher fever on the day of
infection and the day of re-challenge.</p>
<h3 data-number="9.56.4" id="significance-50"><span class="header-section-number">9.56.4</span> Significance</h3>
<p>This study showed clear viral clearance and no indications of relapse or
viremia after a secondary infection with SARS-CoV-2 in a Chinese rhesus
macaque model. These results support the idea that patients with full
recovery (two negative RT-PCR results) may also be protected from
secondary SARS-CoV-2 infection. Recovered patients may be able to
reintegrate to normal public life and provide protective serum perhaps
even if having had a mild infection. The results are also encouraging
for successful vaccine development against SARS-CoV-2.</p>
<h3 data-number="9.56.5" id="credit-55"><span class="header-section-number">9.56.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.57" id="a-highly-conserved-cryptic-epitope-in-the-receptor-binding-domains-of-sars-cov-2-and-sars-cov"><span class="header-section-number">9.57</span> A highly conserved cryptic epitope in the receptor-binding domains of SARS-CoV-2 and SARS-CoV</h2>
<p><span class="citation" data-cites="6gQxRkWM">[<a href="#ref-6gQxRkWM" role="doc-biblioref">600</a>]</span></p>
<h3 data-number="9.57.1" id="keywords-56"><span class="header-section-number">9.57.1</span> Keywords</h3>
<ul>
<li>neutralizing antibody</li>
<li>cross-reactivity</li>
</ul>
<h3 data-number="9.57.2" id="main-findings-53"><span class="header-section-number">9.57.2</span> Main Findings</h3>
<p>Given the sequence similarity of the surface spike glycoprotein (S) of
SARS-CoV-2 and SAR-CoV, Yuan et al. (2020) propose that neutralizing
antibodies isolated from convalescent SARS-CoV patients may offer
insight into cross-reactive antibodies targeting SARS-CoV-2. In
particular, they find that the receptor-binding domain (RBD) of
SARS-CoV-2 S protein shares 86% sequence similarity with the RBD of
SARS-CoV S protein that binds to the CR3022 neutralizing antibody.
CR3022 also displays increased affinity for the “up” conformation of the
SARS-CoV-2 S protein compared to the “down” conformation as it does for
the SARS-CoV S protein. Therefore, the authors propose that this
cross-reactive antibody may confer some degree of protection <em>in vivo</em>
even if it fails to neutralize <em>in vitro</em>.</p>
<h3 data-number="9.57.3" id="limitations-54"><span class="header-section-number">9.57.3</span> Limitations</h3>
<p>Although the authors offer a logical rationale for
identifying cross-reactive neutralizing antibodies derived from
SARS-CoV, their study using only CR3022 failed to demonstrate whether
this approach will be successful. After all, CR3022 failed to neutralize
<em>in vitro</em> despite the binding affinity to a similar epitope on
SARS-CoV-2. They would benefit from testing more candidates and using an
<em>in vivo</em> model to demonstrate their claim that protection may be
possible in the absence neutralization if combinations are used <em>in
vivo.</em></p>
<h3 data-number="9.57.4" id="significance-51"><span class="header-section-number">9.57.4</span> Significance</h3>
<p>The ability to make use of previously characterized
neutralizing antibodies for conserved epitopes can expedite drug design
and treatment options.</p>
<h3 data-number="9.57.5" id="credit-56"><span class="header-section-number">9.57.5</span> Credit</h3>
<p><em>This review was undertaken by Dan Fu Ruan, Evan Cody and Venu Pothula as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.58" id="highly-accurate-and-sensitive-diagnostic-detection-of-sars-cov-2-by-digital-pcr"><span class="header-section-number">9.58</span> Highly accurate and sensitive diagnostic detection of SARS-CoV-2 by digital PCR</h2>
<p>Dong et al. <em>medRxiv</em> <span class="citation" data-cites="UYD5T2mp">[<a href="#ref-UYD5T2mp" role="doc-biblioref">198</a>]</span></p>
<h3 data-number="9.58.1" id="keywords-57"><span class="header-section-number">9.58.1</span> Keywords</h3>
<ul>
<li>Diagnosis</li>
<li>digital PCR</li>
</ul>
<h3 data-number="9.58.2" id="main-findings-54"><span class="header-section-number">9.58.2</span> Main Findings</h3>
<p>The authors present a digital PCR (dPCR) diagnostic test for
SARS-CoV-2 infection. In 103 individuals that were confirmed in a
follow-up to be infected, the standard qPCR test had a positivity
rate of 28.2% while the dPCR test detected 87.4% of the infections
by detecting an additional 61 positive cases. The authors also
tested samples from close contacts (early in infection stage) and
convalescing individuals (late in infection stage) and were able to
detect SARS-CoV-2 nucleic acid in many more samples using dPCR
compared to qPCR.</p>
<h3 data-number="9.58.3" id="limitations-55"><span class="header-section-number">9.58.3</span> Limitations</h3>
<p>I did not detect limitations.</p>
<h3 data-number="9.58.4" id="significance-52"><span class="header-section-number">9.58.4</span> Significance</h3>
<p>The authors make a strong case for the need for a highly sensitive
and accurate confirmatory method for diagnosing COVID-19 during this
outbreak and present a potential addition to the diagnostic arsenal.
They propose a dPCR test that they present has a dramatically lower
false negative rate than the standard RT-qPCR tests and can be
especially beneficial in people with low viral load, whether they
are in the earlier or later stages of infection.</p>
<h3 data-number="9.58.5" id="credit-57"><span class="header-section-number">9.58.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.59" id="sars-cov-2-invades-host-cells-via-a-novel-route-cd147-spike-protein"><span class="header-section-number">9.59</span> SARS-CoV-2 invades host cells via a novel route: CD147-spike protein</h2>
<p>Wang et al. <em>bioRxiv</em> <span class="citation" data-cites="14y8UvAbH">[<a href="#ref-14y8UvAbH" role="doc-biblioref">601</a>]</span></p>
<h3 data-number="9.59.1" id="keywords-58"><span class="header-section-number">9.59.1</span> Keywords</h3>
<ul>
<li>spike protein</li>
<li>viral entry</li>
<li>CD147</li>
<li>SARS-CoV-2</li>
</ul>
<h3 data-number="9.59.2" id="main-findings-55"><span class="header-section-number">9.59.2</span> Main Findings</h3>
<p>The authors propose a novel mechanism of SARS-CoV-2 viral entry through
the interaction of the viral spike protein (SP) and the immunoglobulin
superfamily protein CD147 (also known as Basigin). Using an in-house
developed humanized antibody against CD147 (maplazumab), they show that
blocking CD147 decreases viral replication in Vero E6 cells. Using
surface plasmon resonance (SPR), ELISA, and Co-IP assays, they show that
the spike protein of SARS-CoV-2 directly interacts with CD147. Lastly,
they utilize immune-election microscopy to show spike protein and CD147
localize to viral inclusion bodies of Vero E6 cells.</p>
<h3 data-number="9.59.3" id="limitations-56"><span class="header-section-number">9.59.3</span> Limitations</h3>
<p>The authors claim that an anti-CD147 antibody (Meplazumab) inhibits
SARS-CoV-2 replication by testing cell growth and viral load in cells
infected with SARS-CoV-2, however there are key pieces of this
experiment that are missing. First, the authors fail to use a
non-specific antibody control. Second, the authors claim that viral
replication is inhibited, and that they test this by qPCR, however this
data is <strong>not shown.</strong> To further prove specificity, the authors should
introduce CD147 to non-susceptible cells and show that they become
permissive.</p>
<p>The authors claim that there is a direct interaction between CD147 and
SP through SPR, ELISA, and Co-IP, and this data seems generally
convincing. The electron microscopy provides further correlative
evidence that SARS-CoV-2 may interact with CD147 as they are both found
in the same viral inclusion body. A quantification of this data would
make the findings more robust.</p>
<p>Finally, the data in this paper lacks replicates, error bars, and
statistics to show that the data are reproducible and statistically
significant.</p>
<h3 data-number="9.59.4" id="significance-53"><span class="header-section-number">9.59.4</span> Significance</h3>
<p>It has been shown in various studies that SARS-CoV-2 binds to the cell
surface protein ACE2 for cell entry, yet ACE2 is highly expressed in
heart, kidney, and intestinal cells, raising the concern that blocking
ACE2 would result in harmful side effects <span class="citation" data-cites="BcOf65BE">[<a href="#ref-BcOf65BE" role="doc-biblioref">602</a>]</span> CD147 on the other hand
is highly expressed in various tumor types, inflamed tissues, and
pathogen infected cells, suggesting that the inhibition of CD147 would
not result in major side effects <span class="citation" data-cites="34nQJalJ KcThTnsm">[<a href="#ref-34nQJalJ" role="doc-biblioref">603</a>,<a href="#ref-KcThTnsm" role="doc-biblioref">604</a>]</span> The research in this paper has
resulted in an ongoing clinical trail in China to test the safety and
efficacy of anti-CD147 Meplazumab to treat COVID-19. (ClinicalTrails.gov
identifier NCT04275245).</p>
<h3 data-number="9.59.5" id="credit-58"><span class="header-section-number">9.59.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.60" id="blood-single-cell-immune-profiling-reveals-that-interferon-mapk-pathway-mediated-adaptive-immune-response-for-covid-19"><span class="header-section-number">9.60</span> Blood single cell immune profiling reveals that interferon-MAPK pathway mediated adaptive immune response for COVID-19</h2>
<p>Huang et al. <em>medRxiv</em> <span class="citation" data-cites="18l4l0nLX">[<a href="#ref-18l4l0nLX" role="doc-biblioref">605</a>]</span></p>
<h3 data-number="9.60.1" id="keywords-59"><span class="header-section-number">9.60.1</span> Keywords</h3>
<ul>
<li>COVID-19</li>
<li>SARS-CoV-2</li>
<li>PBMC</li>
<li>single cell</li>
<li>MAPK</li>
</ul>
<h3 data-number="9.60.2" id="main-findings-56"><span class="header-section-number">9.60.2</span> Main Findings</h3>
<p>The authors performed single-cell RNA sequencing (scRNAseq) of
peripheral blood mononuclear cells isolated from whole blood samples of
COVID-19 patients (n=10). Data was compared to scRNAseq of samples
collected from patients with influenza A (n=1), acute pharyngitis (n=1),
and cerebral infarction (n=1), as well as, three healthy controls.
COVID-19 patients were categorized into those with moderate (n=6),
severe (n=1), critical (n=1), and cured (n=2) disease. Analysis across
all COVID-19 disease levels revealed 56 different cellular subtypes,
among 17 immune cell types; comparisons between each category to the
normal controls revealed <strong>increased proportions of CD1c<sup>+</sup> dendritic
cells, CD8<sup>+</sup> CTLs, and plasmacytoid dendritic cells and a decrease in
proportions of B cells and CD4<sup>+</sup> T cells</strong>.</p>
<p><strong>TCR sequencing revealed that greater clonality is associated with
milder COVID-19 disease</strong>; BCR sequencing revealed that COVID-19
patients have circulating antibodies against known viral antigens,
including EBV, HIV, influenza A, and other RNA viruses. This may suggest
that the immune response to SARS-CoV-2 infection elicits production of
antibodies against known RNA viruses.</p>
<p>Excluding enriched pathways shared by COVID-19 patients and patients
with other conditions (influenza A, acute pharyngitis, and cerebral
infarction), the authors identified the <strong>interferon-MAPK signaling
pathway as a major response to SARS-CoV-2 infection</strong>. The authors
performed quantitative real-time reverse transcriptase polymerase chain
reaction (RT-PCR) for interferon-MAPK signaling genes: <em>IRF27, BST2,</em>
and <em>FOS</em>. These samples were collected from a separate cohort of
COVID-19 patients (critical, n=3; severe, n=3; moderate, n=19; mild,
n=3; and cured, n=10; and healthy controls, n=5). Notably, consistent
with the original scRNAseq data, <em>FOS</em> showed up-regulation in COVID-19
patients and down-regulation in cured patients. <strong>The authors propose
that <em>FOS</em> may be a candidate marker gene for curative COVID-19
disease</strong>.</p>
<h3 data-number="9.60.3" id="limitations-57"><span class="header-section-number">9.60.3</span> Limitations</h3>
<p>The sample size of this study is limited. To further delineate
differences in the immune profile of peripheral blood of COVID-19
patients, a greater sample size is needed, and longitudinal samples are
needed, as well. A better understanding of the immunological
interactions in cured patients, for example, would require a profile
before and after improvement.</p>
<p>Moreover, the conclusions drawn from this scRNAseq study point to
potential autoimmunity and immune deficiency to distinguish different
severities of COVID-19 disease. However, this requires an expanded
number of samples and a more robust organization of specific immune cell
subtypes that can be compared across different patients. Importantly,
this criterion is likely needed to ensure greater specificity in
identifying markers for COVID-19 infection and subsequent immune
response.</p>
<h3 data-number="9.60.4" id="significance-54"><span class="header-section-number">9.60.4</span> Significance</h3>
<p>At the single-cell level, COVID-19 disease has been characterized in the
lung, but a greater understanding of systemic immunological responses is
furthered in this study. Type I interferon is an important signaling
molecule for the anti-viral response. The identification of the
interferon-MAPK signaling pathway and the differential expression of
MAPK regulators between patients of differing COVID-19 severity and
compared to cured patients may underscore the importance of either
immune deficiency or autoimmunity in COVID-19 disease.</p>
<h3 data-number="9.60.5" id="credit-59"><span class="header-section-number">9.60.5</span> Credit</h3>
<p><em>This review was undertaken by Matthew D. Park as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.61" id="cross-reactive-antibody-response-between-sars-cov-2-and-sars-cov-infection."><span class="header-section-number">9.61</span> Cross-reactive antibody response between SARS-CoV-2 and SARS-CoV infection.</h2>
<p>Lv et al. <em>bioRxiv</em> <span class="citation" data-cites="dM1rOHmH">[<a href="#ref-dM1rOHmH" role="doc-biblioref">606</a>]</span></p>
<h3 data-number="9.61.1" id="keywords-60"><span class="header-section-number">9.61.1</span> Keywords</h3>
<p>SARS-CoV-2, SARS-CoV, spike protein, RBD, cross-reactivity,
cross-neutralization, antibody, human patients, mouse</p>
<h3 data-number="9.61.2" id="main-findings-57"><span class="header-section-number">9.61.2</span> Main Findings</h3>
<p>The authors explore the antigenic differences between SARS-CoV-2 and
SARS-CoV by analyzing plasma samples from SARS-CoV-2 (n = 15) and
SARS-CoV (n = 7) patients. Cross-reactivity in antibody binding to the
spike protein between SARS-CoV-2 and SARS-CoV was found to be common,
mostly targeting non-RBD regions in plasma from SARS-CoV-2 patients.
Only one SARS-CoV-2 plasma sample was able to cross-neutralize SARS-CoV,
with low neutralization activity. No cross-neutralization response was
detected in plasma from SARS-CoV patients.</p>
<p>To further investigate the cross-reactivity of antibody responses to
SARS-CoV-2 and SARS-CoV, the authors analyzed the antibody response of
plasma collected from mice infected or immunized with SARS-CoV-2 or
SARS-CoV (n = 5 or 6 per group). Plasma from mice immunized with
SARS-CoV-2 displayed cross-reactive responses to SARS-CoV S ectodomain
and, to a lesser extent, SARS-CoV RBD. Similarly, plasma from mice
immunized with SARS-CoV displayed cross-reactive responses to SARS-CoV-2
S ectodomain. Cross-neutralization activity was not detected in any of
the mouse plasma samples.</p>
<h3 data-number="9.61.3" id="limitations-58"><span class="header-section-number">9.61.3</span> Limitations</h3>
<p>The size of each patient cohort is insufficient to accurately determine
the frequency of cross-reactivity and cross-neutralization in the
current SARS-CoV-2 pandemic. Recruitment of additional patients from a
larger range of geographical regions and time points would also enable
exploration into the effect of the genetic diversity and evolution of
the SARS-CoV-2 virus on cross-reactivity. This work would also benefit
from the mapping of specific epitopes for each sample. Future studies
may determine whether the non-neutralizing antibody responses can confer
<em>in vitro</em> protection or lead to antibody-dependent disease enhancement.</p>
<h3 data-number="9.61.4" id="significance-55"><span class="header-section-number">9.61.4</span> Significance</h3>
<p>The cross-reactive antibody responses to S protein in the majority of
SARS-CoV-2 patients is an important consideration for development of
serological assays and vaccine development during the current outbreak.
The limited extent of cross-neutralization demonstrated in this study
indicates that vaccinating to cross-reactive conserved epitopes may have
limited efficacy, presenting a key concern for the development of a more
universal coronavirus vaccine to address the global health risk of novel
coronavirus outbreaks.</p>
<h3 data-number="9.61.5" id="credit-60"><span class="header-section-number">9.61.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.62" id="the-feasibility-of-convalescent-plasma-therapy-in-severe-covid-19-patients-a-pilot-study"><span class="header-section-number">9.62</span> The feasibility of convalescent plasma therapy in severe COVID-19 patients: a pilot study</h2>
<p>Duan et al. <em>medRxiv</em> <span class="citation" data-cites="UENoLkhz">[<a href="#ref-UENoLkhz" role="doc-biblioref">607</a>]</span></p>
<h3 data-number="9.62.1" id="keywords-61"><span class="header-section-number">9.62.1</span> Keywords</h3>
<ul>
<li>COVID-19</li>
<li>SARS-CoV-2</li>
<li>convalescent plasma</li>
<li>treatment outcome</li>
<li>pilot</li>
<li>therapy</li>
<li>transfusion</li>
</ul>
<h3 data-number="9.62.2" id="main-findings-58"><span class="header-section-number">9.62.2</span> Main Findings</h3>
<p>This is the first report to date of convalescent plasma therapy as a
therapeutic against COVID-19 disease. This is a feasibility pilot study.
The authors report the administration and clinical benefit of 200 mL of
convalescent plasma (CP) (1:640 titer) derived from recently cured
donors (CP selected among 40 donors based on high neutralizing titer and
ABO compatibility) to 10 severe COVID-19 patients with confirmed
viremia. The primary endpoint was the safety of CP transfusion. The
secondary endpoint were clinical signs of improvement based on symptoms
and laboratory parameters.</p>
<p>The authors reported use of methylene blue photochemistry to inactivate
any potential residual virus in the plasma samples, without compromising
neutralizing antibodies, and no virus was detected before transfusion.</p>
<p>The authors report the following:</p>
<ul>
<li><p>No adverse events were observed in all patients, except 1 patient
who exhibited transient facial red spotting.</p></li>
<li><p>All patients showed significant improvement in or complete
disappearance of clinical symptoms, including fever, cough,
shortness of breath, and chest pain after 3 days of CP therapy.</p></li>
<li><p>Reduction of pulmonary lesions revealed by chest CT.</p></li>
<li><p>Elevation of lymphocyte counts in patients with lymphocytopenia.</p></li>
<li><p>Increase in SaO<sub>2</sub> in all patients, indicative of recuperating lung
function.</p></li>
<li><p>Resolution of SARS-CoV-2 viremia in 7 patients and increase in
neutralizing antibody titers in 5 patients. Persistence of
neutralizing antibody levels in 4 patients.</p></li>
</ul>
<h3 data-number="9.62.3" id="limitations-59"><span class="header-section-number">9.62.3</span> Limitations</h3>
<p>It is important to note that most recipients had high neutralization
titers of antibodies before plasma transfusion and even without
transfusion it would be expected to see an increase in neutralizing
antibodies over time. In addition to the small sample set number (n=10),
there are additional limitations to this pilot study:</p>
<ol type="1">
<li><p>All patients received concurrent therapy, in addition to the CP
transfusion. Therefore, it is unclear whether a combinatorial or
synergistic effect between these standards of care and CP
transfusion contributed to the clearance of viremia and improvement
of symptoms in these COVID-19 patients.</p></li>
<li><p>The kinetics of viral clearance was not investigated, with respect
to the administration of CP transfusion. So, the definitive impact
of CP transfusion on immune dynamics and subsequent viral load is
not well defined.</p></li>
<li><p>Comparison with a small historical control group is not ideal.</p></li>
</ol>
<h3 data-number="9.62.4" id="significance-56"><span class="header-section-number">9.62.4</span> Significance</h3>
<p>For the first time, a pilot study provides promising results involving
the use of convalescent plasma from cured COVID-19 patients to treat
others with more severe disease. The authors report that the
administration of a single, high-dose of neutralizing antibodies is
safe. In addition, there were encouraging results with regards to the
reduction of viral load and improvement of clinical outcomes. It is,
therefore, necessary to expand this type of study with more
participants, in order to determine optimal dose and treatment kinetics.
It is important to note that CP has been studied to treat H1N1
influenza, SARS-CoV-1, and MERS-CoV, although it has not been proven to
be effective in treating these infections.</p>
<h3 data-number="9.62.5" id="credit-61"><span class="header-section-number">9.62.5</span> Credit</h3>
<p><em>Review by Matthew D. Park and revised by Alice O. Kamphorst and Maria A. Curotto de Lafaille as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.63" id="hydroxychloroquine-and-azithromycin-as-a-treatment-of-covid-19-results-of-an-open-label-non-randomized-clinical-trial"><span class="header-section-number">9.63</span> Hydroxychloroquine and azithromycin as a treatment of COVID-19: results of an open label non-randomized clinical trial</h2>
<p><span class="citation" data-cites="Xbkby2Jr">[<a href="#ref-Xbkby2Jr" role="doc-biblioref">608</a>]</span></p>
<h3 data-number="9.63.1" id="keywords-62"><span class="header-section-number">9.63.1</span> Keywords</h3>
<ul>
<li>hydroxychloroquine</li>
<li>clearance</li>
<li>viral load</li>
<li>clinical trial</li>
</ul>
<h3 data-number="9.63.2" id="main-findings-59"><span class="header-section-number">9.63.2</span> Main Findings</h3>
<p>This study was a single-arm, open label clinical trial with
600 mg hydroxychloroquine (HCQ) in the treatment arm (n = 20). Patients
who refused participation or patients from another center not treated
with HCQ were included as negative controls (n = 16). Among the patients
in the treatment arm, 6 received concomitant azithromycin to prevent
superimposed bacterial infection. The primary endpoint was respiratory
viral loads on day 6 post enrollment, measured by nasopharyngeal swab
followed by real-time reverse transcription-PCR.</p>
<p>HCQ alone was able to significantly reduce viral loads by day 6 (n =
8/14, 57.1% complete clearance, p &lt; 0.001); azithromycin appears to
be synergistic with HCQ, as 6/6 patients receiving combined treatment
had complete viral clearance (p &lt; 0.001).</p>
<h3 data-number="9.63.3" id="limitations-60"><span class="header-section-number">9.63.3</span> Limitations</h3>
<p>Despite what is outlined above, this study has a number
of limitations that must be considered. First, there were originally n =
26 patients in the treatment arm, with 6 lost to follow up for the
following reasons: 3 transferred to ICU, 1 discharge, 1
self-discontinued treatment d/t side effects, and 1 patient expired.
Total length of clinical follow up was 14 days, but the data beyond day
6 post-inclusion are not shown.</p>
<p>Strikingly, in supplementary table 1, results of the real-time RT-PCR
are listed for the control and treatment arms from D0 – D6. However, the
data are not reported in a standard way, with a mix of broadly positive
or negative result delineation with Ct (cycle threshold) values, the
standard output of real time PCR. It is impossible to compare what is
defined as a positive value between the patients in the control and
treatment arms without a standardized threshold for a positive test.
Further, the starting viral loads reported at D0 in the groups receiving
HCQ or HCQ + azithromycin were significantly different (ct of 25.3 vs
26.8 respectively), which could explain in part the differences observed
in the response to treatment between 2 groups. Finally, patients in the
control arm from outside the primary medical center in this study
(Marseille) did not actually have samples tested by PCR daily. Instead,
positive test results from every other day were extrapolated to mean
positive results on the day before and after testing as well (Table 2,
footnote <sup>a</sup>).</p>
<p>Taken together, the results of this study suggest that HCQ represents a
promising treatment avenue for COVID-19 patients. However, the limited
size of the trial, and the way in which the results were reported does
not allow for other medical centers to extrapolate a positive or
negative result in the treatment of their own patients with HCQ +/-
azithromycin. Further larger randomized clinical trials will be required
to ascertain the efficacy of HCQ +/- azithromycin in the treatment of
COVID-19.</p>
<h3 data-number="9.63.4" id="significance-57"><span class="header-section-number">9.63.4</span> Significance</h3>
<p>Chloroquine is thought to inhibit viral infection,
including SARS-Cov-2, by increasing pH within endosomes and lysosomes,
altering the biochemical conditions required for viral fusion <span class="citation" data-cites="IhdcvbzC BIlqPh6M">[<a href="#ref-IhdcvbzC" role="doc-biblioref">246</a>,<a href="#ref-BIlqPh6M" role="doc-biblioref">609</a>]</span>.
However, chloroquine also has immuno-modulatory effects that I think may
play a role. Chloroquine has been shown to increase CTLA-4 expression at
the cell surface by decreasing its degradation in the endo-lysosome
pathway; AP-1 traffics the cytoplasmic tail of CTLA-4 to lysosomes, but
in conditions of increased pH, the protein machinery required for
degradation is less functional <span class="citation" data-cites="Q9tXjG1R">[<a href="#ref-Q9tXjG1R" role="doc-biblioref">610</a>]</span>. As such, more CTLA-4 remains in
endosomes and is trafficked back to the cell surface. It is possible
that this may also contribute to patient recovery via reduction of
cytokine storm, in addition to the direct anti-viral effects of HCQ.</p>
<h3 data-number="9.63.5" id="credit-62"><span class="header-section-number">9.63.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.64" id="recapitulation-of-sars-cov-2-infection-and-cholangiocyte-damage-with-human-liver-organoids"><span class="header-section-number">9.64</span> Recapitulation of SARS-CoV-2 Infection and Cholangiocyte Damage with Human Liver Organoids</h2>
<p><span class="citation" data-cites="OuDMtCRl">[<a href="#ref-OuDMtCRl" role="doc-biblioref">534</a>]</span></p>
<h3 data-number="9.64.1" id="keywords-63"><span class="header-section-number">9.64.1</span> Keywords</h3>
<ul>
<li>CoVID-19</li>
<li>2019-nCoV</li>
<li>SARS-CoV-2</li>
<li>liver</li>
<li>organoids</li>
<li>Cholangiocyte</li>
</ul>
<h3 data-number="9.64.2" id="main-findings-60"><span class="header-section-number">9.64.2</span> Main Findings</h3>
<ul>
<li><p>Used human liver ductal organoids to determine ACE2+ cholangiocytes
in healthy liver (2.92% of all cells) are infectable and support
SARS-CoV-2 viral replication.</p></li>
<li><p>Plaque-purified SARS-CoV-2 viral infection disrupted organoid
barrier and bile transporting functions of cholangiocytes through
dysregulation of genes involved in tight junction formation (CLDN1)
and bile acid transportation (ASBT and CFTR).</p></li>
</ul>
<h3 data-number="9.64.3" id="limitations-61"><span class="header-section-number">9.64.3</span> Limitations:</h3>
<ul>
<li><p>Unclear if liver damage observed in patients due to direct
cholangiocyte infection or due to secondary immune/cytokine effects.
This study argues for direct damage as it lacks immune contexture;
but further studies needed with autopsy samples or organoid-immune
cell co-culture to conclude strongly.</p></li>
<li><p>Would be important to measure cholangiocyte-intrinsic anti-viral
response and alarmins secreted upon infection, and furthermore study
tropism of various immune cells to conditioned media from organoids
infected with SARS-CoV-2.</p></li>
<li><p>Does not address how cirrhotic liver or
alcohol/smoking/obesity-associated liver organoids respond to
SARS-CoV-2 infectivity and replication, worth pursuing to
experimentally address clinical data indicating co-morbidities.</p></li>
</ul>
<h3 data-number="9.64.4" id="significance-58"><span class="header-section-number">9.64.4</span> Significance</h3>
<ul>
<li><p>Useful model to rapidly study drug activity against SARS-CoV-2
infection in liver, while monitoring baseline liver damage.</p></li>
<li><p>Liver abnormality observed in &gt;50% of CoVID-19 patients; the
results from this study could explain the bile acid accumulation and
consequent liver damage observed.</p></li>
</ul>
<h3 data-number="9.64.5" id="credit-63"><span class="header-section-number">9.64.5</span> Credit</h3>
<p><em>Review by Samarth Hegde as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.65" id="the-sequence-of-human-ace2-is-suboptimal-for-binding-the-s-spike-protein-of-sars-coronavirus-2"><span class="header-section-number">9.65</span> The sequence of human ACE2 is suboptimal for binding the S spike protein of SARS coronavirus 2</h2>
<p><span class="citation" data-cites="3VPi509z">[<a href="#ref-3VPi509z" role="doc-biblioref">611</a>]</span></p>
<h3 data-number="9.65.1" id="keywords-64"><span class="header-section-number">9.65.1</span> Keywords</h3>
<ul>
<li>Coronavirus Disease 2019 (COVID-19)</li>
<li>SARS-CoV-2</li>
<li>Spike protein S</li>
<li>ACE2</li>
</ul>
<h3 data-number="9.65.2" id="main-findings-61"><span class="header-section-number">9.65.2</span> Main Findings</h3>
<p>Severe Acute Respiratory Syndrome Coronavirus 2
(SARS-CoV-2) infects cells through S spike glycoprotein binding
angiotensin-converting enzyme (ACE2) on host cells. S protein can bind
both membrane-bound ACE2 and soluble ACE2 (sACE2), which can serve as a
decoy that neutralizes infection. Recombinant sACE2 is now being tested
in clinical trials for COVID-19. To determine if a therapeutic sACE2
with higher affinity for S protein could be designed, authors generated
a library containing every amino acid substitution possible at the 117
sites spanning the binding interface with S protein. The ACE2 library
was expressed in human Expi293F cells and cells were incubated with
medium containing the receptor binding domain (RBD) of SARS-CoV-2 fused
to GFP. Cells with high or low affinity mutant ACE2 receptor compared to
affinity of wild type ACE2 for the RBD were FACS sorted and transcripts
from these sorted populations were deep sequenced. Deep mutagenesis
identified numerous mutations in ACE2 that enhance RBD binding. This
work serves to identify putative high affinity ACE2 therapeutics for the
treatment of CoV-2.</p>
<h3 data-number="9.65.3" id="limitations-62"><span class="header-section-number">9.65.3</span> Limitations</h3>
<p>The authors generated a large library of mutated
ACE2, expressed them in human Expi293F cells, and performed deep
mutagenesis to identify enhanced binders for the RBD of SARS-CoV-2 S
protein. While these data serve as a useful resource, the ability of the
high affinity ACE2 mutants identified to serve as therapeutics needs
further validation in terms of conformational stability when purified as
well as efficacy/safety both <em>in vitro</em> and <em>in vivo</em>. Additionally,
authors mentioned fusing the therapeutic ACE2 to Fc receptors to elicit
beneficial host immune responses, which would need further design and
validation.</p>
<h3 data-number="9.65.4" id="significance-59"><span class="header-section-number">9.65.4</span> Significance</h3>
<p>This study identified structural ACE2 mutants that
have potential to serve as therapeutics in the treatment of SARS-CoV-2
upon further testing and validation.</p>
<h3 data-number="9.65.5" id="credit-64"><span class="header-section-number">9.65.5</span> Credit</h3>
<p><em>This review was undertaken by Katherine Lindblad and Tamar Plitt as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<p><strong><em>Title</em></strong>:  A serological assay to detect SARS-Cov-2 seroconversion in
humans</p>
<p><strong><em>Immunology keywords</em></strong>:  specific serological assay - ELISA -
seroconversion - antibody titers</p>
<p><em>Note: the authors of this review work in the same institution as the
authors of the study</em></p>
<p><strong><em>Main findings</em></strong>:  </p>
<p>Production of recombinant whole Spike (S) protein and the smaller
Receptor Binding Domain (RBD) based on the sequence of Wuhan-Hu-1
SARS-CoV-2 isolate. The S protein was modified to allow trimerization
and increase stability. The authors compared the antibody reactivity of
59 banked human serum samples (non-exposed) and 3 serum samples from
confirmed SARS-CoV-2 infected patients. All Covid-19 patient sera
reacted to the S protein and RBD domain compared to the control sera.</p>
<p>The authors also characterized the antibody isotypes from the Covid-19
patients, and observed stronger IgG3 response than IgG1. IgM and IgA
responses were also prevalent.</p>
<p><strong><em>Limitations of the study</em></strong>:  </p>
<p>The authors analyzed a total of 59 control human serum samples, and
samples from only three different patients to test for reactivity
against the RBD domain and full-length spike protein. It will be
important to follow up with a larger number of patient samples to
confirm the data obtained. Furthermore, it would be interesting to
assess people at different age groups and determine whether unexposed
control kids have a higher “background”.</p>
<p>Applications of the assay described in this study in diagnosis are
limited, since antibody response should start to be detectable only one
to two weeks after infection. Future studies will be required to assess
how long after infection this assay allow to detect anti-CoV2
antibodies. Finally, while likely, the association of seroconversion
with protective immunity against SARS-Cov-2 infection still needs to be
fully established.</p>
<p><strong><em>Relevance</em></strong>:</p>
<p>This study has strong implications in the research against SARS-Cov-2.
First, it is now possible to perform serosurveys and determine who has
been infected, allowing a more accurate estimate of infection prevalence
and death rate. Second, if it is confirmed that re-infection does not
happen (or is rare), this assay can be used as a tool to screen
healthcare workers and prioritize immune ones to work with infected
patients. Third, potential convalescent plasma donors can now be
screened to help treating currently infected patients. Of note, this
assay does not involve live virus handling. experimentally, this is an
advantage as the assay does not require the precautions required by
manipulation of live virus. Finally, the recombinant proteins described
in this study represent new tools that can be used for further
applications, including vaccine development.</p>
<h2 data-number="9.66" id="comparative-pathogenesis-of-covid-19-mers-and-sars-in-a-non-human-primate-model"><span class="header-section-number">9.66</span> COMPARATIVE PATHOGENESIS OF COVID-19, MERS AND SARS IN A NON-HUMAN PRIMATE MODEL</h2>
<p><span class="citation" data-cites="1EcOLnVXk">[<a href="#ref-1EcOLnVXk" role="doc-biblioref">612</a>]</span></p>
<h3 data-number="9.66.1" id="keywords-65"><span class="header-section-number">9.66.1</span> Keywords</h3>
<ul>
<li>SARS-CoV2</li>
<li>cynomolgus macaque</li>
<li>SARS-CoV</li>
</ul>
<h3 data-number="9.66.2" id="main-findings-62"><span class="header-section-number">9.66.2</span> Main Findings</h3>
<p>This work assesses SARS-CoV-2 infection in young adult and aged
cynomolgus macaques. 4 macaques per age group were infected with
low-passage clinical sample of SARS-CoV-2 by intranasal and
intratracheal administration. Viral presence was assessed in nose,
throat and rectum through RT-PCR and viral culture. SARS-CoV-2
replication was confirmed in the respiratory track (including nasal
samples), and it was also detected in ileum. Viral nucleocapsid
detection by IHC showed infection of type I and II pneumocytes and
epithelia. Virus was found to peak between 2 and 4 days after
administration and reached higher levels in aged vs. young animals. The
early peak is consistent with data in patients and contrasts to SARS-CoV
replication. SARS-CoV-2 reached levels below detection between 8 and 21
days after inoculation and macaques established antibody immunity
against the virus by day 14. There were histopathological alteration in
lung, but no overt clinical signs. At day 4 post inoculation of
SARS-CoV-2, two of four animals presented foci of pulmonary
consolidation, with limited areas of alveolar edema and pneumonia, as
well as immune cell infiltration. In sum, cynomolgus macaques are
permissive to SARS- CoV-2 and develop lung pathology (less severe than
SARC-CoV, but more severe than MERS-CoV).</p>
<h3 data-number="9.66.3" id="limitations-63"><span class="header-section-number">9.66.3</span> Limitations</h3>
<p>Even though cynomolgus macaques were permissive to SARS-CoV-2
replication, it is unclear if the viral load reaches levels comparable
to humans and there wasn’t overt clinical pathology.</p>
<h3 data-number="9.66.4" id="significance-60"><span class="header-section-number">9.66.4</span> Significance</h3>
<p>The development of platforms in which to carry out relevant
experimentation on SARS-CoV-2 pathophysiology is of great urgency.
Cynomolgus macaques offer an environment in which viral replication can
happen, albeit in a limited way and without translating into clinically
relevant symptoms. Other groups are contributing to SARS-CoV2 literature
using this animal model <span class="citation" data-cites="3XqFhi8l">[<a href="#ref-3XqFhi8l" role="doc-biblioref">599</a>]</span>, potentially showing protection against
reinfection in cured macaques. Therefore, this platform could be used to
examine SARS-CoV2 pathophysiology while studies in other animal models
are also underway <span class="citation" data-cites="NzFgXYvQ MgLBA5m">[<a href="#ref-MgLBA5m" role="doc-biblioref">536</a>,<a href="#ref-NzFgXYvQ" role="doc-biblioref">613</a>]</span>.</p>
<h3 data-number="9.66.5" id="credit-65"><span class="header-section-number">9.66.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.67" id="investigating-the-impact-of-asymptomatic-carriers-on-covid-19-transmission"><span class="header-section-number">9.67</span> Investigating the Impact of Asymptomatic Carriers on COVID-19 Transmission</h2>
<p><span class="citation" data-cites="OvFueDLZ">[<a href="#ref-OvFueDLZ" role="doc-biblioref">614</a>]</span></p>
<h3 data-number="9.67.1" id="keywords-66"><span class="header-section-number">9.67.1</span> Keywords</h3>
<ul>
<li>Coronavirus Disease 2019 (COVID-19)</li>
<li>SARS-CoV-2</li>
<li>asymptomatic carriers</li>
<li>mathematical model</li>
<li>transmission</li>
</ul>
<h3 data-number="9.67.2" id="main-findings-63"><span class="header-section-number">9.67.2</span> Main Findings</h3>
<p>Multiple studies reported the same level of infectiousness
between symptomatic and asymptomatic carriers of SARS-CoV-2. Given that
asymptomatic and undocumented carriers escape public health surveillance
systems, a better mathematical model of transmission is needed to
determine a more accurate estimate of the basic reproductive number
(R<sub>0</sub>) of the virus to assess the contagiousness of virus. The authors
developed a SEYAR dynamical model for transmission of the new
coronavirus that takes into account asymptomatic and undocumented
carriers. The model was validated using data reported from thirteen
countries during the first three weeks of community transmission. While
current studies estimate R<sub>0</sub> to be around 3, this model indicates that
the value could range between 5.5 to 25.4.</p>
<h3 data-number="9.67.3" id="limitations-64"><span class="header-section-number">9.67.3</span> Limitations</h3>
<p>The SEYAR model realistically depicts
transmission of the virus only during the initial stages of the disease.
More data is necessary to better fit the model with current trends. In
addition, multiple factors (e.g. behavioral patterns, surveillance
capabilities, environmental and socioeconomic factors) affect
transmission of the virus and so, these factors must be taken into
consideration when estimating the R<sub>0</sub>.</p>
<h3 data-number="9.67.4" id="significance-61"><span class="header-section-number">9.67.4</span> Significance</h3>
<p>Public health authorities use the basic reproductive
number to determine the severity of disease. An accurate estimate of
R<sub>0</sub> will inform intervention strategies. This model can be applied to
different locations to assess the potential impact of COVID-19.</p>
<h3 data-number="9.67.5" id="credit-66"><span class="header-section-number">9.67.5</span> Credit</h3>
<p><em>This review was undertaken by Tamar Plitt and Katherine Lindblad as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.68" id="antibody-responses-to-sars-cov-2-in-covid-19-patients-the-perspective-application-of-serological-tests-in-clinical-practice"><span class="header-section-number">9.68</span> Antibody responses to SARS-CoV-2 in COVID-19 patients: the perspective application of serological tests in clinical practice</h2>
<p>Long et al. <em>medRxiv</em> <span class="citation" data-cites="Y9g1sQ1Y">[<a href="#ref-Y9g1sQ1Y" role="doc-biblioref">615</a>]</span></p>
<h3 data-number="9.68.1" id="keywords-67"><span class="header-section-number">9.68.1</span> Keywords</h3>
<ul>
<li>Serum antibodies</li>
<li>IgM</li>
<li>IgG</li>
<li>immunoassay</li>
<li>diagnosis</li>
<li>seroconversion</li>
</ul>
<h3 data-number="9.68.2" id="main-findings-64"><span class="header-section-number">9.68.2</span> Main Findings</h3>
<p>This study investigated the profile of the acute antibody response
against SARS-CoV-2 and provided proposals for serologic tests in
clinical practice. Magnetic Chemiluminescence Enzyme Immunoassay was
used to evaluate IgM and IgG seroconversion in 285 hospital admitted
patients who tested positive for SARS-CoV-2 by RT-PCR and in 52 COVID-19
suspected patients that tested negative by RT-PCR. A follow up study
with 63 patients was performed to investigate longitudinal effects. In
addition, IgG and IgM titers were evaluated in a cohort of close
contacts (164 persons) of an infected couple.</p>
<p>The median day of seroconversion for both IgG and IgM was 13 days after
symptom onset. Patients varied in the order of IgM/ IgG seroconversion
and there was no apparent correlation of order with age, severity, or
hospitalization time. This led the authors to conclude that for
diagnosis IgM and IgG should be detected simultaneously at the early
phase of infection.</p>
<p>IgG titers, but not IgM titers were higher in severe patients compared
to non-severe patients after controlling for days post-symptom onset.
Importantly, 12% of COVID-19 patients (RT-PCR confirmed) did not meet
the WHO serological diagnosis criterion of either seroconversion or
<em>&gt;</em> 4-fold increase in IgG titer in sequential samples. This suggests
the current serological criteria may be too stringent for COVID-19
diagnosis.</p>
<p>Of note, 4 patients from a group of 52 suspects (negative RT-PCR test)
had anti-SARS-Cov-2 IgM and IgG. Similarly, 4.3% (7/162) of “close
contacts” who had negative RT-PCR tests were positive for IgG and/or
IgM. This highlights the usefulness of a serological assay to identify
asymptomatic infections and/or infections that are missed by RT-PCR.</p>
<h3 data-number="9.68.3" id="limitations-65"><span class="header-section-number">9.68.3</span> Limitations</h3>
<p>This group’s report generally confirms the findings of others that have
evaluated the acute antibody response to SARS-Cov-2. However, these data
would benefit from inclusion of data on whether the participants had a
documented history of viral infection. Moreover, serum samples that were
collected prior to SARS-Cov-2 outbreak from patients with other viral
infections would serve as a useful negative control for their assay.
Methodological limitations include that only one serum sample per case
was tested as well as the heat inactivation of serum samples prior to
testing. It has previously been reported that heat inactivation
interferes with the level of antibodies to SARS-Cov-2 and their protocol
may have resulted in diminished quantification of IgM, specifically <span class="citation" data-cites="p1c9Nilh">[<a href="#ref-p1c9Nilh" role="doc-biblioref">616</a>]</span>.</p>
<h3 data-number="9.68.4" id="significance-62"><span class="header-section-number">9.68.4</span> Significance</h3>
<p>Understanding the features of the antibody responses against SARS-CoV is
useful in the development of a serological test for the diagnosis of
COVID-19. This paper addresses the need for additional screening methods
that can detect the presence of infection despite lower viral titers.
Detecting the production of antibodies, especially IgM, which are
produced rapidly after infection can be combined with PCR to enhance
detection sensitivity and accuracy and map the full spread of infection
in communities, Moreover, serologic assays would be useful to screen
health care workers in order to identify those with immunity to care for
patients with COVID19.</p>
<h3 data-number="9.68.5" id="credit-67"><span class="header-section-number">9.68.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.69" id="sars-cov-2-specific-antibody-responses-in-covid-19-patients"><span class="header-section-number">9.69</span> SARS-CoV-2 specific antibody responses in COVID-19 patients</h2>
<p><span class="citation" data-cites="1BLNvgEao">[<a href="#ref-1BLNvgEao" role="doc-biblioref">617</a>]</span></p>
<h3 data-number="9.69.1" id="keywords-68"><span class="header-section-number">9.69.1</span> Keywords</h3>
<ul>
<li>immunoassay</li>
<li>antibody specificity</li>
<li>serology</li>
<li>cross-reactivity</li>
</ul>
<h3 data-number="9.69.2" id="main-findings-65"><span class="header-section-number">9.69.2</span> Main findings</h3>
<p>Antibodies specific to SARS-CoV-2 S protein, the S1 subunit and the RBD (receptor-binding domain) were detected in all SARS-CoV-2 patient sera by 13 to 21 days post onset of disease. Antibodies specific to SARS-CoV N protein (90% similarity to SARS-CoV-2) were able to neutralize SARS-CoV-2 by PRNT (plaque reduction neutralizing test). SARS-CoV-2 serum cross-reacted with SARS-CoV S and S1 proteins, and to a lower extent with MERS-CoV S protein, but not with the MERS-CoV S1 protein, consistent with an analysis of genetic similarity. No reactivity to SARS-CoV-2 antigens was observed in serum from patients with ubiquitous human CoV infections (common cold) or to non-CoV viral respiratory infections.</p>
<h3 data-number="9.69.3" id="limitations-66"><span class="header-section-number">9.69.3</span> Limitations</h3>
<p>Authors describe development of a serological ELISA based assay for the detection of neutralizing antibodies towards regions of the spike and nucleocapsid domains of the SARS-CoV-2 virus. Serum samples were obtained from PCR-confirmed COVID-19 patients. Negative control samples include a cohort of patients with confirmed recent exposure to non-CoV infections (i.e. adenovirus, bocavirus, enterovirus, influenza, RSV, CMV, EBV) as well as a cohort of patients with confirmed infections with ubiquitous human CoV infections known to cause the common cold. The study also included serum from patients with previous MERS-CoV and SARS-CoV zoonotic infections. This impressive patient cohort allowed the authors to determine the sensitivity and specificity of the development of their in-house ELISA assay. Of note, seroconversion was observed as early as 13 days following COVID-19 onset but the authors were not clear how disease onset was determined.</p>
<h3 data-number="9.69.4" id="significance-63"><span class="header-section-number">9.69.4</span> Significance</h3>
<p>Validated serological tests are urgently needed to map the full spread of SARS-CoV-2 in the population and to determine the kinetics of the antibody response to SARS-CoV-2. Furthermore, clinical trials are ongoing using plasma from patients who have recovered from SARS-CoV-2 as a therapeutic option. An assay such as the one described in this study could be used to screen for strong antibody responses in recovered patients. Furthermore, the assay could be used to screen health care workers for antibody responses to SARS-CoV-2 as personal protective equipment continues to dwindle. The challenge going forward will be to standardize and scale-up the various in-house ELISA’s being developed in independent laboratories across the world.</p>
<h2 data-number="9.70" id="a-brief-review-of-antiviral-drugs-evaluated-in-registered-clinical-trials-for-covid-19"><span class="header-section-number">9.70</span> A brief review of antiviral drugs evaluated in registered clinical trials for COVID-19</h2>
<p>Belhadi et al. <span class="citation" data-cites="K41YFM2N">[<a href="#ref-K41YFM2N" role="doc-biblioref">618</a>]</span></p>
<h3 data-number="9.70.1" id="keywords-69"><span class="header-section-number">9.70.1</span> Keywords</h3>
<ul>
<li>Clinical trials</li>
<li>COVID-19</li>
<li>SARS CoV-2</li>
<li>2019-nCoV</li>
<li>SARS Cov-2</li>
<li>Hcov-19</li>
<li>novel coronal virus</li>
<li>new corona virus</li>
<li>antiviral drugs</li>
</ul>
<h3 data-number="9.70.2" id="main-findings-66"><span class="header-section-number">9.70.2</span> Main Findings</h3>
<p>Summary of clinical trials registered as
of March7, 2020 from U.S, Chinese, Korean, Iranian and European
registries. Out of the 353 studies identified, 115 were selected for
data extraction. 80% of the trials were randomized with parallel
assignment and the median number of planned inclusions was 63 (IRQ,
36-120). Most frequent therapies in the trials included; 1) antiviral
drugs [lopinavir/ritonavir (n-15); umifenovir (n=9); favipiravir (n=7);
redmesivir (n=5)]; 2) anti-malaria drugs [chloroquine (n-11);
hydroxychloroquine (n=7)}; immunosuppressant drugs [methylprednisolone
(n=5)]; and stem cell therapies (n=23). Medians of the total number of
planned inclusions per trial for these therapies were also included.
Stem cells and lopunavir/ritonavir were the most frequently evaluated
candidate therapies (23 and 15 trials respectively), whereas remdesivir
was only tested in 5 trials but these trials had the highest median
number of planned inclusions per trial (400, IQR 394-453). Most of the
agents used in the different trials were chosen based on preclinical
assessments of antiviral activity against SARS CoV and MERS Cov corona
viruses.</p>
<p>The primary outcomes of the studies were clinical (66%); virological
(23%); radiological (8%); or immunological (3%). The trials were
classified as those that included patients with severe disease only;
trials that included patients with moderate disease; and trials that
included patients with severe or moderate disease.</p>
<h3 data-number="9.70.3" id="limitations-67"><span class="header-section-number">9.70.3</span> Limitations</h3>
<p>The trials evaluated provided incomplete information:
23% of these were phase IV trials but the bulk of the trials (54%) did
not describe the phase of the study. Only 52% of the trials (n=60)
reported treatment dose and only 34% (n=39) reported the duration. A lot
of the trials included a small number of patients and the trials are
still ongoing, therefore no insight was provided on the outcome of the
trials.</p>
<h3 data-number="9.70.4" id="significance-64"><span class="header-section-number">9.70.4</span> Significance</h3>
<p>Nonetheless, this review serves as framework for
identifying COVID-19 related trials, which can be expanded upon as new
trials begin at an accelerated rate as the disease spreads around the
world.</p>
<h3 data-number="9.70.5" id="credit-68"><span class="header-section-number">9.70.5</span> Credit</h3>
<p><em>This review was undertaken by K Alexandropoulos as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.71" id="ace-2-expression-in-the-small-airway-epithelia-of-smokers-and-copd-patients-implications-for-covid-19"><span class="header-section-number">9.71</span> ACE-2 Expression in the Small Airway Epithelia of Smokers and COPD Patients: Implications for COVID-19</h2>
<p>Leung et al. <em>medRxiv</em>. <span class="citation" data-cites="eTsOvBVf">[<a href="#ref-eTsOvBVf" role="doc-biblioref">619</a>]</span></p>
<h3 data-number="9.71.1" id="keywords-70"><span class="header-section-number">9.71.1</span> Keywords</h3>
<ul>
<li>chronic obstructive pulmonary disease</li>
<li>COPD</li>
<li>smokingE-2</li>
<li>risk factors</li>
</ul>
<h3 data-number="9.71.2" id="main-findings-67"><span class="header-section-number">9.71.2</span> Main Findings</h3>
<p>In bronchial epithelial samples from 3 different cohorts of
individuals, ACE-2 gene expression was found to be significantly
increased in both COPD patients and smokers relative to healthy
controls. Across all test subjects, ACE-2 gene expression was also
highly correlated with decreased forced expiratory volume in 1 second
(FEV1), which may explain the increased COVID-19 disease severity in
COPD patients. Former smokers were also found to show decreased ACE2
expression relative to current smokers and had no significant difference
when compared to non-smokers.</p>
<h3 data-number="9.71.3" id="limitations-68"><span class="header-section-number">9.71.3</span> Limitations</h3>
<p>While the upregulation of ACE-2 is an interesting
hypothesis for COVID-19 disease severity in COPD patients, this study
leaves many more unanswered questions than it addresses. Further studies
are required to show whether the specific cell type isolated in these
studies is relevant to the pathophysiology of COVID-19. Furthermore,
there is no attempt to show whether that increased ACE-2 expression
contributes to greater disease severity. Does the increased ACE-2
expression lead to greater infectivity with SARS-CoV-2? There is no
mechanistic explanation for why ACE-2 levels are increased in COPD
patients. The authors could also have considered the impact of
co-morbidities and interventions such as corticosteroids or
bronchodilators on ACE-2 expression. Finally, given the extensive
sequencing performed, the authors could have conducted significantly
more in-depth analyses into gene signature differences.</p>
<h3 data-number="9.71.4" id="significance-65"><span class="header-section-number">9.71.4</span> Significance</h3>
<p>This study attempts to address an
important clinical finding that both smokers and COPD patients show
increased mortality from COVID-19. The novel finding that ACE-2
expression is induced in smokers and COPD patients suggests not only a
mechanism for the clinical observation, but also highlights the
potential benefit of smoking cessation in reducing the risk of severe
COVID-19 disease.</p>
<h3 data-number="9.71.5" id="credit-69"><span class="header-section-number">9.71.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.72" id="dynamic-profile-of-severe-or-critical-covid-19-cases"><span class="header-section-number">9.72</span> Dynamic profile of severe or critical COVID-19 cases</h2>
<p><span class="citation" data-cites="18KZGwJ0J">[<a href="#ref-18KZGwJ0J" role="doc-biblioref">620</a>]</span></p>
<h3 data-number="9.72.1" id="keywords-71"><span class="header-section-number">9.72.1</span> Keywords</h3>
<ul>
<li>Coronavirus Disease 2019 (COVID-19)</li>
<li>SARS-CoV-2</li>
<li>progressive lymphopenia (PLD)</li>
<li>T-lymphocytes</li>
<li>clinical data</li>
<li>co-infection</li>
<li>influenza A</li>
</ul>
<h3 data-number="9.72.2" id="main-findings-68"><span class="header-section-number">9.72.2</span> Main Findings</h3>
<p>Authors evaluate clinical correlates of 10 patients (6 male
and 4 female) hospitalized for severe Acute Respiratory Syndrome
Coronavirus 2 (SARS-CoV-2). All patients required oxygen support and
received broad spectrum antibiotics and 6 patients received anti-viral
drugs. Additionally, 40% of patients were co-infected with influenza A.
All 10 patients developed lymphopenia, two of which developed
progressive lymphopenia (PLD) and died. Peripheral blood (PB)
lymphocytes were analyzed – low CD4 and CD8 counts were noted in most
patients, though CD4:CD8 ratio remained normal.</p>
<h3 data-number="9.72.3" id="limitations-69"><span class="header-section-number">9.72.3</span> Limitations</h3>
<p>The authors evaluated a small cohort of severe
SARS-CoV-2 cases and found an association between T cell lymphopenia and
adverse outcomes. However, this is an extremely small and diverse cohort
(40% of patients were co-infected with influenza A). These findings need
to be validated in a larger cohort. Additionally, the value of this data
would be greatly increased by adding individual data points for each
patient as well as by adding error bars to each of the figures.</p>
<h3 data-number="9.72.4" id="significance-66"><span class="header-section-number">9.72.4</span> Significance</h3>
<p>This study provides a collection of clinical data and
tracks evolution of T lymphocyte in 10 patients hospitalized for
SARS-CoV-2, of which 4 patients were co-infected with influenza A.</p>
<h3 data-number="9.72.5" id="credit-70"><span class="header-section-number">9.72.5</span> Credit</h3>
<p><em>This review was undertaken by Katherine Lindblad and Tamar Plitt as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.73" id="association-between-clinical-laboratory-and-ct-characteristics-and-rt-pcr-results-in-the-follow-up-of-covid-19-patients"><span class="header-section-number">9.73</span> Association between Clinical, Laboratory and CT Characteristics and RT-PCR Results in the Follow-up of COVID-19 patients</h2>
<p>Fu et al. <em>medRxiv</em>. <span class="citation" data-cites="1BJXDZ04c">[<a href="#ref-1BJXDZ04c" role="doc-biblioref">621</a>]</span></p>
<h3 data-number="9.73.1" id="keywords-72"><span class="header-section-number">9.73.1</span> Keywords</h3>
<ul>
<li>COVID-19</li>
<li>clinical</li>
<li>lymphocyte</li>
<li>CRP</li>
<li>LDH</li>
<li>HSST TNT</li>
<li>PCR test</li>
<li>readmission</li>
<li>CT</li>
<li>GGO</li>
<li>disease progression</li>
</ul>
<h3 data-number="9.73.2" id="study-description"><span class="header-section-number">9.73.2</span> Study description</h3>
<p>Data analyzed from 52 COVID-19 patients admitted
and then discharged with COVID-19. Clinical, laboratory, and
radiological data were longitudinally recorded with illness timecourse
(PCR + to PCR-) and 7 patients (13.5%) were readmitted with a follow up
positive test (PCR+) within two weeks of discharge.</p>
<h3 data-number="9.73.3" id="main-findings-69"><span class="header-section-number">9.73.3</span> Main Findings</h3>
<ul>
<li><p>At admission:</p>
<ul>
<li><p>The majority of patients had increased CRP at admission (63.5%).</p></li>
<li><p>LDH, and HSST TNT were significantly increased at admission.</p></li>
<li><p>Radiographic signs via chest CT showed increased involvement in
lower lobes: right lower lobe (47 cases, 90.4%), left lower lobe
(37 cases, 71.2%).</p></li>
<li><p>GGO (90.4%), interlobular septal thickening (42.3%), vascular
enlargement (42.3%), and reticulation (11.5%) were most commonly
observed.</p></li>
</ul></li>
<li><p>After negative PCR test (discharge):</p>
<ul>
<li><p>CRP levels decreased lymphocyte counts (#/L) increased
significantly (CD3+, CD3+/8+ and CD3+/4+) after negative PCR.</p></li>
<li><p>Consolidation and mixed GGO observed in longitudinal CT imaging
w different extents of inflammatory exudation in lungs, with
overall tendency for improvement (except 2/7 patients that were
readmitted after discharge with re-positive test) after negative
PCR.</p></li>
</ul></li>
<li><p>Seven patients repeated positive RT-PCR test and were readmitted to
the hospital (9 to 17 day after initial discharge).</p>
<ul>
<li><p>Follow up CT necessary to monitor improvement during recovery
and patients with lesion progression should be given more
attention.</p></li>
<li><p>Dynamic CT in addition to negative test essential in clinical
diagnosis due to nasal swab PCR sampling bias (false-negatives).</p></li>
<li><p>Increase in CRP occurred in 2 readmitted patients (and decr. in
lymphocytes in one patient), but was not correlated with new
lesions or disease progression vs. improvement (very low N).</p></li>
<li><p>Patients readmitted attributed to false-negative PCR vs.
re-exposure.</p></li>
</ul></li>
</ul>
<h3 data-number="9.73.4" id="limitations-70"><span class="header-section-number">9.73.4</span> Limitations</h3>
<p>Patients sampled in this study were generally
younger (65.4% &lt; 50 yrs) and less critically ill/all discharged.
Small number of recovered patients (N=18). Time of follow up was
relatively short.. Limited clinical information available about patients
with re-positive test (except CRP and lymph tracking).</p>
<h3 data-number="9.73.5" id="extended-results"><span class="header-section-number">9.73.5</span> Extended Results</h3>
<p><em>NOTE</em>: Patients sampled in this study were generally younger (65.4%
&lt; 50 yrs) and less critically ill/all discharged. After two
consecutive negative PCR tests, patients were discharged.</p>
<p>Clinical Results at Admission</p>
<ul>
<li><p>Median interval disease onset to admission (5 days, IQR: 3-7)</p></li>
<li><p>Most common symptoms included fever, fatigue, dry cough, and
expectoration.</p></li>
<li><p>Fifteen patients had reduced lymphocyte counts (28.8%).</p></li>
<li><p>No change in WBC or Neutrophil counts.</p></li>
<li><p><strong>The majority of patients had increased CRP at admission (63.5%).</strong></p></li>
<li><p><strong>LDH, and HSST TNT were significantly increased at admission.</strong></p></li>
<li><p>Fibrinogen was trending high though not significant.</p></li>
<li><p>No major changes in liver function observed.</p></li>
<li><p><strong>Radiographic signs via chest CT showed increased involvement in
lower lobes: right lower lobe (47 cases, 90.4%), left lower lobe (37
cases, 71.2%).</strong></p></li>
<li><p><strong>GGO (90.4%), interlobular septal thickening (42.3%), vascular
enlargement (42.3%)</strong>, and reticulation (11.5%) were most commonly
observed.</p></li>
</ul>
<p>Change in Clinical Results following Negative Test</p>
<ul>
<li><p><strong>CRP levels decreased after negative PCR.</strong></p></li>
<li><p><strong>Lymphocyte counts (#/L) increased significantly (CD3+, CD3+/8+
and CD3+/4+).</strong></p></li>
<li><p>No significant change to CD4/8 ratio.</p></li>
<li><p>LDH, HSST TNT, and Fibronegin remained high throughout, though range
observed decreased over time.</p></li>
<li><p><strong>Consolidation and mixed GGO observed in longitudinal CT imaging.</strong></p></li>
<li><p><strong>Patients showed different extents of inflammatory exudation in
lungs, with overall tendency for improvement (except 2/7 patients
that were readmitted after discharge with re-positive test).</strong></p></li>
</ul>
<p>Patients Readmitted with PCR+ test</p>
<ul>
<li><p><strong>Seven patients repeated positive RT-PCR test and were readmitted
to the hospital (9 to 17 day after initial discharge).</strong></p></li>
<li><p>Improvement during readmission in 4 patients and observation of
segmental progression CT in 2 patients (2/18 or 11.1% - re-positive
9 and 10 days post-discharge).</p></li>
<li><p>Two patients showed new GGO, while others improved greatly.</p></li>
<li><p><strong>Follow up CT necessary to monitor improvement during recovery and
patients with lesion progression should be given more attention.</strong></p></li>
<li><p><strong>Dynamic CT in addition to negative test essential in clinical
diagnosis due to nasal swab PCR sampling bias (false-negatives).</strong></p></li>
<li><p><strong>Increase in CRP occurred in 2 readmitted patients (and decr. in
lymphocytes in one patient), but was not correlated with new lesions
or disease progression vs. improvement (very low N).</strong></p></li>
</ul>
<h3 data-number="9.73.6" id="significance-67"><span class="header-section-number">9.73.6</span> Significance</h3>
<p>Study tracked key clinical features associated with
disease progression, recovery, and determinants of clinical
diagnosis/management of COVID-19 patients.</p>
<h3 data-number="9.73.7" id="credit-71"><span class="header-section-number">9.73.7</span> Credit</h3>
<p><em>This review was undertaken by Natalie Vaninov as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.74" id="an-orally-bioavailable-broad-spectrum-antiviral-inhibits-sars-cov-2-and-multiple-2-endemic-epidemic-and-bat-coronavirus"><span class="header-section-number">9.74</span> An orally bioavailable broad-spectrum antiviral inhibits SARS-CoV-2 and multiple 2 endemic, epidemic and bat coronavirus</h2>
<p>Sheahan et al. <em>bioRxiv</em>. <span class="citation" data-cites="iMCz5Twj">[<a href="#ref-iMCz5Twj" role="doc-biblioref">622</a>]</span></p>
<h3 data-number="9.74.1" id="keywords-73"><span class="header-section-number">9.74.1</span> Keywords</h3>
<ul>
<li>Treatment</li>
<li>Antiviral</li>
<li>Broad spectrum antiviral</li>
<li>ribonucleoside analog β-D-N4 30 hydroxycytidine (NHC)</li>
<li>Remdesivir</li>
</ul>
<h3 data-number="9.74.2" id="main-findings-70"><span class="header-section-number">9.74.2</span> Main Findings</h3>
<p>β-D-N4 30 –hydroxycytidine (NHC, EIDD-1931) is an
orally bioavailable ribonucleoside with antiviral activity against
various RNA viruses including Ebola, Influenza and CoV. NHC activity
introduce<em>d</em><del>s</del> mutations in the viral (but not cellular) RNA in a
dose dependent <em>manner</em> that directly correlated with a decrease in
viral titers. Authors show that NHC inhibited multiple genetically
distinct Bat-CoV viruses in human primary epithelial cells <em>without
affecting</em> <em>cell viability even at high concentrations (100</em> <em>µM)</em>.
Prophylactic oral administration of NHC in C57BL/6 mice reduce lung
titers of SARS-CoV and prevented weight loss and hemorrhage. Therapeutic
administration of NHC in C57BL/6 mice 12 hours post infected with
SARS-CoV reduced acute lung injury, viral titer, and lung hemorrhage.
The degree of clinical benefit was dependent on the time of treatment
initiation post infection. The authors also demonstrate that NHC reduces
MERS-CoV infection titers, pathogenesis, and viral RNA in prophylactic
and therapeutic settings.</p>
<h3 data-number="9.74.3" id="limitations-71"><span class="header-section-number">9.74.3</span> Limitations</h3>
<p>Most of the experiments were conducted using MERS-CoV, and
SARS-CoV and a few experiments were conducted using other strains of CoV
as opposed to SARS-CoV-2. The authors note the core residues that make
up the RNA interaction sites (which constitutes the NHC interaction
sites) are highly conserved among CoV and because of this conservation
their understanding is that NHC can inhibit a broad-spectrum of CoV
including SARS-CoV-2.</p>
<p>The increased viral mutation rates associated with NHC activity may have
adverse effects if mutations cause the virus to become drug resistant,
more infectious or speed-up immune evasion. <em>In addition, the temporal
diminishing effectiveness of NHC on clinical outcome when NHC was used
therapeutically is concerning. However, the longer window (7-10 days)
for clinical disease onset in human patients from the time of infection
compared to that of mice (24-48 hours), may associate with increased NHC
effectiveness in the clinic.</em></p>
<h3 data-number="9.74.4" id="significance-68"><span class="header-section-number">9.74.4</span> Significance</h3>
<p>Prophylactic or therapeutic oral administration of NHC
reduces lung titers and prevents acute lung failure in C57B\6 mice
infected with CoV. Given its <em>broad-spectrum antiviral activity, NHC
could turn out to be a useful drug for treating current, emerging and
future corona virus outbreaks.</em>
 
#### Credit</p>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.75" id="identification-of-antiviral-drug-candidates-against-sars-cov-2-from-fda-approved-drugs"><span class="header-section-number">9.75</span> Identification of antiviral drug candidates against SARS-CoV-2 from FDA-approved drugs</h2>
<p>Sangeun Jeon et al. <span class="citation" data-cites="ZDP9vSpg">[<a href="#ref-ZDP9vSpg" role="doc-biblioref">623</a>]</span></p>
<h3 data-number="9.75.1" id="keywords-74"><span class="header-section-number">9.75.1</span> Keywords</h3>
<ul>
<li>COVID-19</li>
<li>SARS CoV-2</li>
<li>antiviral drugs</li>
<li>niclosamide</li>
<li>ciclesonide</li>
</ul>
<h3 data-number="9.75.2" id="main-findings-71"><span class="header-section-number">9.75.2</span> Main Findings</h3>
<p>A panel of ~3,000 FDA- and IND-approved
antiviral drugs were previously screened for inhibitory efficacy against
SARS CoV, a coronavirus related to the novel coronavirus SARS CoV-2
(79.5%) homology. 35 of these drugs along with another 15 (suggested by
infectious disease specialists) were tested in vitro for their ability
to inhibit SARS CoV-2 infectivity of Vero cells while preserving cell
viability. The infected cells were scored by immunofluorescence analysis
using an antibody against the N protein of SARS CoV-2. Chloroquine,
lopinavir and remdesivir were used as reference drugs.</p>
<p>Twenty four out of 50 drugs exhibited antiviral activity with IC50
values ranging from 0.1-10µM. Among these, two stood ou: 1) the-anti
helminthic drug niclosamide which exhibited potent antiviral activity
against SARS CoV-2 (IC50=0.28 µM). The broad-spectrum antiviral effect
of niclosamide against SARS and MERS-CoV have been previously documented
and recent evidence suggests that in may inhibit autophagy and reduce
MERS C0V replication. 2) Ciclesonide, a corticosteroid used to treat
asthma and allergic rhinitis, also exhibited antiviral efficacy but with
a lower IC50 (4.33µM) compared to niclosamide. The antiviral effects of
ciclesonide were directed against NSP15, a viral riboendonuclease which
is the molecular target of this drug.</p>
<h3 data-number="9.75.3" id="limitations-72"><span class="header-section-number">9.75.3</span> Limitations</h3>
<p>The drugs were tested against SARS CoV-2 infectivity in
vitro only, therefore preclinical studies in animals and clinical trials
in patients will be need for validation of these drugs as therapeutic
agents for COVID-19. In addition, niclosamide exhibits low adsorption
pharmatokinetically which could be alleviated with further development
of drug formulation to increase effective delivery of this drug to
target tissues. Nonetheless, niclosamide and ciclesonide represent
promising therapeutic agents against SARS CoV-2 given that other
compounds tested in the same study including favipiravir (currently used
in clinical trials) and atazanavir (predicted as the most potent
antiviral drug by AI-inference modeling) did not exhibit antiviral
activity in the current study.</p>
<h3 data-number="9.75.4" id="credit-72"><span class="header-section-number">9.75.4</span> Credit</h3>
<p><em>This review was undertaken by K Alexandropoulos as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.76" id="respiratory-disease-and-virus-shedding-in-rhesus-macaques-inoculated-with-sars-cov-2"><span class="header-section-number">9.76</span> Respiratory disease and virus shedding in rhesus macaques inoculated with SARS-CoV-2</h2>
<p>Munster et al. <em>bioRxiv</em>. <span class="citation" data-cites="LYFzvlzE">[<a href="#ref-LYFzvlzE" role="doc-biblioref">624</a>]</span></p>
<h3 data-number="9.76.1" id="keywords-75"><span class="header-section-number">9.76.1</span> Keywords</h3>
<p>animal model, pulmonary infiltrates, dynamic
of antibody response, cytokine</p>
<h3 data-number="9.76.2" id="main-findings-72"><span class="header-section-number">9.76.2</span> Main Findings</h3>
<p>Inoculation of 8 Resus macaques with SARS-Cov-2 , which
all showed clinical signs of infection (respiratory pattern, reduced
appetite, weight loss, elevated body temperature) resulting in moderate,
transient disease. Four animals were euthanized at 3dpi, the 4 others at
21 dpi. Study of viral loads in different organs showed that nose swab
and throat swabs were the most sensitive, with bronchio-alveolar lavage.
Interstitial pneumonia was visible in radiographies and at the
histological scale too. Clinically, the macaques had similar symptoms as
described in human patients with moderate disease.</p>
<p>Viral shedding was consistently detected in nose swabs and throat swabs
immediately after infection but less consistent thereafter which could
reflect virus administration route (intranasal, oral). Bronchoalveolar
lavages performed as a measure of virus replication in the lower
respiratory tract on animals maintained for 21 days, contained high
viral loads in 1 and 3dpi. The majority of the animals exhibited
pulmonary edema and mild to moderate interstitial pneumonia on terminal
bronchioles. In addition to the lung, viral RNA could also be detected
throughout the respiratory track where viral replication mainly
occurred.</p>
<p>Immunologic responses included leukocytosis, neutrophilia, monocytosis
and lymphopenia in the majority of the animals at 1dpi. Lymphocytes and
monocytes re-normalized at 2dpi. Neutrophils declined after 3dpi and
through 10dpi after which they started to recover. After infection,
serum analysis revealed significant increases in <strong>IL1ra, IL6, IL10,
IL15, MCP-1, MIP-1b, but quick normalization</strong> (3dpi). <strong>Antibody
response started around 7dpi, and the antibody titers stayed elevated
until 21dpi</strong> (day of animal euthanasia).</p>
<h3 data-number="9.76.3" id="limitations-73"><span class="header-section-number">9.76.3</span> Limitations</h3>
<p>The macaques were inoculated via a combination
of intratracheal, intranasal, ocular and oral routes, which might not
reproduce how humans get infected. Maybe this can lead to different
dynamics in the host immune response. Also, the authors noted that the
seroconversion was not directly followed by a decline in viral loads, as
observed in covid19 patients.</p>
<h3 data-number="9.76.4" id="significance-69"><span class="header-section-number">9.76.4</span> Significance</h3>
<p>This work confirms that rhesus
macaques can be a good model to study Covid-19, as it has been shown by
other groups <span class="citation" data-cites="3XqFhi8l unVgVIfU 1EcOLnVXk">[<a href="#ref-3XqFhi8l" role="doc-biblioref">599</a>,<a href="#ref-1EcOLnVXk" role="doc-biblioref">612</a>,<a href="#ref-unVgVIfU" role="doc-biblioref">625</a>]</span>. While these experiments recapitulate
moderate COVID-19 in humans, the mode of inoculation via a combination
of intratracheal, intranasal, ocular and oral routes, might not
reproduce how humans get infected and may lead to different dynamics in
the host immune response. For example, the authors noted that the
seroconversion was not directly followed by a decline in viral loads, as
observed in COVID-19 patients. Therefore, it will be interesting to
follow their antibody titers longer and further assess the
possibility/effect of reinfection in these macaques. It is essential to
be able to understand the dynamic of the disease and associated immune
responses, and to work on vaccine development and antiviral drug
testing.</p>
<h3 data-number="9.76.5" id="credit-73"><span class="header-section-number">9.76.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.77" id="ace2-expression-is-increased-in-the-lungs-of-patients-with-comorbidities-associated-with-severe-covid-19"><span class="header-section-number">9.77</span> ACE2 Expression is Increased in the Lungs of Patients with Comorbidities Associated with Severe COVID-19</h2>
<p><span class="citation" data-cites="wjtVy2v0">[<a href="#ref-wjtVy2v0" role="doc-biblioref">626</a>]</span></p>
<h3 data-number="9.77.1" id="keywords-76"><span class="header-section-number">9.77.1</span> Keywords</h3>
<ul>
<li>CoVID-19</li>
<li>2019-nCoV</li>
<li>SARS-CoV-2</li>
<li>lung</li>
<li>comorbidities</li>
<li>histone</li>
<li>epigenetics</li>
</ul>
<h3 data-number="9.77.2" id="main-findings-73"><span class="header-section-number">9.77.2</span> Main Findings</h3>
<ul>
<li><p>Transcriptomic analysis using systems-level meta-analysis and
network analysis of existing literature to determine ACE2 regulation
in patients who have frequent COVID-19 comorbidities [eg-
cardiovascular diseases, familial pulmonary hypertension, cancer].</p></li>
<li><p>Enrichment analyses indicated pathways associated with inflammation,
metabolism, macrophage autophagy, and ER stress.</p></li>
<li><p>ACE2 higher in adenocarcinoma compared to adjacent normal lung; ACE2
higher in COPD patients compared to normal.</p></li>
<li><p>Co-expression analysis identified genes important to viral entry
such as RAB1A, ADAM10, HMGBs, and TLR3 to be associated with ACE2 in
diseased lungs.</p></li>
<li><p>ACE2 expression could be potentially regulated by enzymes that
modify histones, including HAT1, HDAC2, and KDM5B.</p></li>
</ul>
<h3 data-number="9.77.3" id="limitations-74"><span class="header-section-number">9.77.3</span> Limitations:</h3>
<ul>
<li><p>Not actual CoVID-19 patients with co-morbidities, so interpretations
in this study need to be confirmed by analyzing upcoming
transcriptomics from CoVID-19 patients having co-morbidity metadata.</p></li>
<li><p>As mentioned by authors, study does not look at diabetes and
autoimmunity as risk factors in CoVID-19 patients due to lack of
data; would be useful to extend such analyses to those datasets when
available.</p></li>
<li><p>Co-expression analysis is perfunctory and needs
validation-experiments especially in CoVID-19 lung samples to mean
anything.</p></li>
<li><p>Epigenomic analyses are intriguing but incomplete, as existence of
histone marks does not necessarily mean occupancy. Would be
pertinent to check cell-line data (CCLE) or actual CoVID-19 patient
samples to confirm ACE2 epigenetic control.</p></li>
</ul>
<h3 data-number="9.77.4" id="significance-70"><span class="header-section-number">9.77.4</span> Significance</h3>
<ul>
<li><p>Study implies vulnerable populations have ACE2 upregulation that
could promote CoVID-19 severity. Shows important data-mining
strategy to find gene-networks associated with ACE2 upregulation in
co-morbid patients.</p></li>
<li><p>Several of the genes co-upregulated with ACE2 in diseased lung might
play an important role in CoVID-19 and can be preliminary targets
for therapeutics</p></li>
<li><p>If in silico findings hold true, epigenetic control of ACE2
expression could be a new target for CoVID-19 therapy with
strategies such as KDM5 demethylases.</p></li>
</ul>
<h3 data-number="9.77.5" id="credit-74"><span class="header-section-number">9.77.5</span> Credit</h3>
<p><em>Review by Samarth Hegde as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.78" id="meplazumab-treats-covid-19-pneumonia-an-open-labelled-concurrent-controlled-add-on-clinical-trial"><span class="header-section-number">9.78</span> Meplazumab treats COVID-19 pneumonia: an open-labelled, concurrent controlled add-on clinical trial</h2>
<p>Bian et al. <em>medRxiv</em>. <span class="citation" data-cites="DFhY5nfb">[<a href="#ref-DFhY5nfb" role="doc-biblioref">627</a>]</span></p>
<h3 data-number="9.78.1" id="keywords-77"><span class="header-section-number">9.78.1</span> Keywords</h3>
<ul>
<li>Meplazumab</li>
<li>CD147</li>
<li>humanized antibody</li>
<li>clinical trial</li>
</ul>
<h3 data-number="9.78.2" id="main-findings-74"><span class="header-section-number">9.78.2</span> Main Findings</h3>
<p>This work is based on previous work by the same group
that demonstrated that SARS-CoV-2can also enter host cells via CD147
(also called Basigin, part of the immunoglobulin superfamily, is
expressed by many cell types) consistent with their previous work with
SARS-CoV-1. <sup>1</sup> A prospective clinical trial was conducted with 17
patients receiving Meplazumab, a humanized anti-CD147 antibody, in
addition to all other treatments. 11 patients were included as a control
group (non-randomized).</p>
<p>They observed a faster overall improvement rate in the Meplazumab group
(e.g. at day 14 47% vs 17% improvement rate) compared to the control
patients. Also, virological clearance was more rapid with median of 3
days in the Meplazumab group vs 13 days in control group. In laboratory
values, a faster normalization of lymphocyte counts in the Meplazumab
group was observed, but no clear difference was observed for CRP levels.</p>
<h3 data-number="9.78.3" id="limitations-75"><span class="header-section-number">9.78.3</span> Limitations</h3>
<p>While the results from the study are encouraging, this
study was non-randomized, open-label and on a small number of patients,
all from the same hospital. It offers evidence to perform a larger scale
study. Selection bias as well as differences between treatment groups
(e.g. age 51yo vs 64yo) may have contributed to results. The authors
mention that there was no toxic effect to Meplazumab injection but more
patient and longer-term studies are necessary to assess this.</p>
<h3 data-number="9.78.4" id="significance-71"><span class="header-section-number">9.78.4</span> Significance</h3>
<p>These results seem promising as for now there are
limited treatments for Covid-19 patients, but a larger cohort of patient
is needed. CD147 has already been described to facilitate HIV <span class="citation" data-cites="yP8TXdq2">[<a href="#ref-yP8TXdq2" role="doc-biblioref">628</a>]</span>,
measles virus <span class="citation" data-cites="X8l1n4zY">[<a href="#ref-X8l1n4zY" role="doc-biblioref">629</a>]</span>, and malaria <span class="citation" data-cites="YcyNaIVC">[<a href="#ref-YcyNaIVC" role="doc-biblioref">630</a>]</span> entry into host cells. This group was
the first to describe the CD147-spike route of SARS-Cov-2 entry in host
cells <span class="citation" data-cites="14y8UvAbH">[<a href="#ref-14y8UvAbH" role="doc-biblioref">601</a>]</span> p147. Indeed, they had previously shown in 2005 that SARS-Cov
could enter host cells via this transmembrane protein <span class="citation" data-cites="m9yLJ5Da">[<a href="#ref-m9yLJ5Da" role="doc-biblioref">631</a>]</span>. Further
biological understanding of how SARS-CoV-2 can enter host cells and how
this integrates with ACE2R route of entry is needed. Also, the specific
cellular targets of the anti-CD147 antibody need to be assessed, as this
protein can be expressed by many cell types and has been shown to
involved in leukocytes aggregation <span class="citation" data-cites="J0ibHE3l">[<a href="#ref-J0ibHE3l" role="doc-biblioref">632</a>]</span>. Lastly, Meplazumab is not a
commercially-available drug and requires significant health resources to
generate and administer which might prevent rapid development and use.</p>
<h3 data-number="9.78.5" id="credit-75"><span class="header-section-number">9.78.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.79" id="potent-human-neutralizing-antibodies-elicited-1-by-sars-cov-2-infection"><span class="header-section-number">9.79</span> Potent human neutralizing antibodies elicited 1 by SARS-CoV-2 infection</h2>
<p>Ju et al. <em>bioRxiv</em>. <span class="citation" data-cites="GhJYjnft">[<a href="#ref-GhJYjnft" role="doc-biblioref">178</a>]</span></p>
<h3 data-number="9.79.1" id="keywords-78"><span class="header-section-number">9.79.1</span> Keywords</h3>
<ul>
<li>monoclonal antibodies</li>
<li>neutralization</li>
<li>antibody cross-reactivity</li>
<li>Receptor Binding Domain</li>
</ul>
<h3 data-number="9.79.2" id="main-findings-75"><span class="header-section-number">9.79.2</span> Main Findings</h3>
<p>In this study the authors report the affinity, cross reactivity (with
SARS-CoV and MERS-CoV virus) and viral neutralization capacity of 206
monoclonal antibodies engineered from isolated IgG memory B cells of
patients suffering from SARS-CoV-2 infection in Wuhan, China. All
patients but one recovered from disease. Interestingly, the patient that
did not recover had less SARS-CoV-2 specific B cells circulating
compared to other patients.</p>
<p>Plasma from all patients reacted to trimeric Spike proteins from
SARS-CoV-2, SARS-CoV and MERS-CoV but no HIV BG505 trimer. Furthermore,
plasma from patients recognized the receptor binding domain (RBD) from
SARS-CoV-2 but had little to no cross-reactivity against the RBD of
related viruses SARS-CoV and MERS-CoV, suggesting significant
differences between the RBDs of the different viruses. Negligible levels
of cross-neutralization using pseudoviruses bearing Spike proteins of
SARS-CoV-2, SARS-CoV or MERS-CoV, were observed, corroborating the ELISA
cross-reactivity assays on the RBDs.</p>
<p>SARS-CoV-2 RBD specific B cells constituted 0.005-0.065% of the total B
cell population and 0.023-0.329% of the memory subpopulation. SARS-CoV
specific IgG memory B cells were single cell sorted to sequence the
antibody genes that were subsequently expressed as recombinant IgG1
antibodies. From this library, 206 antibodies with different binding
capacities were obtained. No discernible patterns of VH usage were found
in the 206 antibodies suggesting immunologically distinct responses to
the infection. Nevertheless, most high-binding antibodies were derived
by clonal expansion. Further analyses in one of the patient derived
clones, showed that the antibodies from three different timepoints did
not group together in phylogenetic analysis, suggesting selection during
early infection.</p>
<p>Using surface plasmon resonance (SPR) 13 antibodies were found to have
10<sup>-8</sup> tp 10<sup>-9</sup> dissociation constants (Kd). Of the 13 antibodies, two
showed 98-99% blocking of SARS-CoV-2 RBD-ACE2 receptor binding in
competition assays. Thus, low Kd values alone did not predict ACE2
competing capacities. Consistent with competition assays the two
antibodies that show high ACE2 blocking (P2C-2F6 and P2C-1F11) were the
most capable of neutralizing pseudoviruses bearing SARS-CoV-2 spike
protein (IC<sub>50</sub> of 0.06 and 0.03 µg/mL, respectively). Finally, using
SPR the neutralizing antibodies were found to recognize both overlapping
and distinct epitopes of the RBD of SARS-CoV-2.</p>
<h3 data-number="9.79.3" id="limitations-76"><span class="header-section-number">9.79.3</span> Limitations</h3>
<ol type="1">
<li><p>Relatively low number of patients</p>
<ol type="a">
<li>No significant conclusion can be drawn about the possible
&gt; correlation between humoral response and disease severity</li>
</ol></li>
<li><p><em>In vitro</em> Cytopathic Effect Assay (CPE) for neutralization activity</p>
<ol type="a">
<li><p>Huh7 cells were used in neutralization assays with
&gt; pseudoviruses, and they may not entirely mimic what happens in
&gt; the upper respiratory tract</p></li>
<li><p>CPE assay is not quantitative</p></li>
</ol></li>
<li><p>Duplicated panel in Figure 4C reported (has been fixed in version 2)</p></li>
</ol>
<h3 data-number="9.79.4" id="significance-72"><span class="header-section-number">9.79.4</span> Significance</h3>
<p>This paper offers an explanation as to why
previously isolated antibodies against SARS-CoV do not effectively block
SARS-CoV-2. Also, it offers important insight into the development of
humoral responses at various time points during the first weeks of the
disease in small but clinically diverse group of patients. Furthermore,
it provides valuable information and well characterized antibody
candidates for the development of a recombinant antibody treatment for
SARS-CoV-2. Nevertheless, it also shows that plasmapheresis might have
variability in its effectiveness, depending on the donor’s antibody
repertoire at the time of donation.</p>
<h3 data-number="9.79.5" id="credit-76"><span class="header-section-number">9.79.5</span> Credit</h3>
<p><em>Review by Jovani Catalan-Dibene as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.80" id="characterisation-of-the-transcriptome-and-proteome-of-sars-cov-2-using-direct-rna-sequencing-and-tandem-mass-spectrometry-reveals-evidence-for-a-cell-passage-induced-in-frame-deletion-in-the-spike-glycoprotein-that-removes-the-furin-like-cleavage-site."><span class="header-section-number">9.80</span> Characterisation of the transcriptome and proteome of SARS-CoV-2 using direct RNA sequencing and tandem mass spectrometry reveals evidence for a cell passage induced in-frame deletion in the spike glycoprotein that removes the furin-like cleavage site.</h2>
<p>Davidson et al. <span class="citation" data-cites="bCYYaFKM">[<a href="#ref-bCYYaFKM" role="doc-biblioref">633</a>]</span></p>
<h3 data-number="9.80.1" id="keywords-79"><span class="header-section-number">9.80.1</span> Keywords</h3>
<ul>
<li>Transcription</li>
<li>RNA-seq</li>
<li>proteomics</li>
<li>mass spec</li>
<li>furin cleavage site</li>
<li>mutation</li>
<li>pathogenicity</li>
</ul>
<h3 data-number="9.80.2" id="main-findings-76"><span class="header-section-number">9.80.2</span> Main Findings</h3>
<p>The authors performed long read RNA sequencing using an Oxford Nanopore
MinION as well as tandem mass spec (MS) on Vero cells (a cell line
derived from kidney cells of the African green monkey that is deficient
in interferon) infected with SARS-CoV-2.</p>
<p>The authors found that passage of the virus in Vero cells gave rise to a
spontaneous 9 amino acid deletion (679-NSPRRARSV-687 to I) in the spike
(S) protein. The deleted sequence overlaps a predicted furin cleavage
site at the S1 / S2 domain boundary that is present in SARS-CoV-2 but
not SARS-CoV or the closely related bat coronavirus RaTG13, which are
cleaved at S1 / S2 by other proteases <span class="citation" data-cites="CqQVkaqj">[<a href="#ref-CqQVkaqj" role="doc-biblioref">42</a>]</span>. Furin cleavage sites at
similar positions in other viruses have been linked to increased
pathogenicity and greater cell tropism <span class="citation" data-cites="16tHVjVzu">[<a href="#ref-16tHVjVzu" role="doc-biblioref">634</a>]</span>. Loss of this site in
SARS-CoV-2 has also already been shown to increase viral entry into Vero
but not BHK cells (which are also interferon deficient) <span class="citation" data-cites="qcVbT0w4">[<a href="#ref-qcVbT0w4" role="doc-biblioref">38</a>]</span>. The
authors therefore make an important contribution in demonstrating that
passage in Vero cells may lead to spontaneous loss of a key
pathogenicity-conferring element in SARS-CoV-2.</p>
<h3 data-number="9.80.3" id="limitations-77"><span class="header-section-number">9.80.3</span> Limitations</h3>
<p>As the authors note, a similar study posted earlier by Kim et al., which
also passaged SARS-CoV-2 in Vero cells, did not identify any loss in the
S protein furin cleavage site <span class="citation" data-cites="hRVfFbS">[<a href="#ref-hRVfFbS" role="doc-biblioref">635</a>]</span>. It therefore remains to be
determined how likely it is that this mutation spontaneously arises. A
quantitative investigation using multiple experimental replicas to
understand the spontaneous viral mutation rate at this site and
elsewhere would be informative. Also, the mechanistic basis for the
higher viral fitness conferred by loss of the furin cleavage site in
Vero cells – but, evidently, not in vivo in humans, as this site is
maintained in all currently sequenced circulating isolates - remains to
be understood.</p>
<p>Due to the high base-call error rate of MinION sequencing, the authors’
bioinformatic pipeline required aligning transcripts to a reference to
correct sequencing artifacts. This presumably made it difficult or
impossible to identify other kinds of mutations, such as single
nucleotide substitutions, which may occur even more frequently than the
deletions identified in this work. Pairing long read sequencing with
higher-accuracy short-read sequencing may be one approach to overcome
this issue.</p>
<h3 data-number="9.80.4" id="significance-73"><span class="header-section-number">9.80.4</span> Significance</h3>
<p>As the authors suggest, animal studies using live virus challenge may
need to periodically verify the genomic integrity of the virus, or
potentially risk unknowingly using a likely less-pathogenic variant of
the virus.</p>
<p>More broadly, the results emphasize the complexity and plasticity of the
SARS-CoV-2 viral transcriptome and proteome. For example, the authors
found multiple versions of transcripts encoding the nucleocapsid (N)
protein, each with different small internal deletions, some of which
were verified for translation by MS. A number of peptides arising from
translation of unexpected rearrangements of transcripts were also
detected. Additionally, the authors identified phosphorylation of a
number of viral proteins (N, M, ORF 3a, nsp3, nsp9, nsp12 and S). For
any cases where these have functional consequences, targeting the
kinases responsible could be an avenue for drug development.
Understanding the functional consequences of the mutations, transcript
variations, and post translational modifications identified in this
study will be important future work.</p>
<h3 data-number="9.80.5" id="credit-77"><span class="header-section-number">9.80.5</span> Credit</h3>
<p><em>This review was undertaken by Tim O’Donnell, Maria Kuksin as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.81" id="a-sars-cov-2-human-protein-protein-interaction-map-reveals-drug-targets-and-potential-drug--repurposing"><span class="header-section-number">9.81</span> A SARS-CoV-2-Human Protein-Protein Interaction Map Reveals Drug Targets and Potential Drug- Repurposing</h2>
<p>Gordon et al. <em>bioRxiv</em> <span class="citation" data-cites="phJM8g2Y">[<a href="#ref-phJM8g2Y" role="doc-biblioref">181</a>]</span></p>
<h3 data-number="9.81.1" id="keywords-80"><span class="header-section-number">9.81.1</span> Keywords</h3>
<ul>
<li>protein-protein interactions</li>
<li>mass spectrometry</li>
<li>drug targets</li>
</ul>
<h3 data-number="9.81.2" id="main-findings-77"><span class="header-section-number">9.81.2</span> Main Findings</h3>
<p>Gordon et al cloned, tagged and expressed 26 of the 29
SARS-CoV-2 proteins individually in HEK293T cells and used mass
spectrometry to identify protein-protein interactions. They identified
332 viral-host protein-protein interactions. Furthermore, they used
these interactions to identify 66 existing drugs known to target host
proteins or host pathways (eg SARS-CoV-2 N and Orf8 proteins interact
with proteins regulated by the mTOR pathway, so mTOR inhibitors
Silmitasertib and Rapamycin are possible drug candidates).</p>
<h3 data-number="9.81.3" id="limitations-78"><span class="header-section-number">9.81.3</span> Limitations</h3>
<p>The main limitation of the study stems from
the reductionist model: overexpression of plasmids encoding individual
viral proteins in HEK293T cells. This precludes any interactions between
the viral proteins, or the combined effects of multiple proteins on the
host, as they are expressed individually. Moreover, HEK293T cells come
from primary embryonic kidney and therefore might not reflect how
SARS-CoV-2 interacts with its primary target, the lung. However, the
authors found that the proteins found to interact with viral proteins in
their experiments are enriched in lung tissue compared to HEK293Ts.</p>
<h3 data-number="9.81.4" id="significance-74"><span class="header-section-number">9.81.4</span> Significance</h3>
<p>The authors provide a “SARS-CoV-2 interaction map,”
which may provide potential hypotheses as to how the virus interacts
with the host. Further, they identified existing drugs that could
disrupt these host-viral interactions and curb SARS-CoV-2 infection.
Although these interactions have not been validated, this paper acts as
a valuable resource.</p>
<h3 data-number="9.81.5" id="credit-78"><span class="header-section-number">9.81.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.82" id="first-clinical-study-using-hcv-protease-inhibitor-danoprevir-to-treat-naïve-and-experienced-covid-19-patients"><span class="header-section-number">9.82</span> First Clinical Study Using HCV Protease Inhibitor Danoprevir to Treat Naïve and Experienced COVID-19 Patients</h2>
<p>Chen et al. <em>medRxiv</em>.
<span class="citation" data-cites="RTIk8DTo">[<a href="#ref-RTIk8DTo" role="doc-biblioref">636</a>]</span></p>
<h3 data-number="9.82.1" id="keywords-81"><span class="header-section-number">9.82.1</span> Keywords</h3>
<ul>
<li>Clinical study</li>
<li>HCV protease inhibitor</li>
<li>Danoprevir</li>
<li>Ritonavir</li>
<li>Covid19 treatment</li>
</ul>
<h3 data-number="9.82.2" id="main-findings-78"><span class="header-section-number">9.82.2</span> Main Findings</h3>
<p>The authors treated 11 Covid-19 patients with Danoprevir, a
commercialized HCV protease inhibitor <span class="citation" data-cites="YoCPtq6P">[<a href="#ref-YoCPtq6P" role="doc-biblioref">637</a>]</span><sup>(p4)</sup>, boosted by ritonavir <span class="citation" data-cites="10GnhBctk">[<a href="#ref-10GnhBctk" role="doc-biblioref">638</a>]</span>, a
CYP3A4 inhibitor (which enhances the plasma concentration and
bioavailabilty of Danoprevir). Two patients had never received
anti-viral therapy before (=naïve), whereas nine patients were on
Lopinavir/Ritonavir treatment before switching to Danoprevir/Ritonavir
(=experienced). The age ranged from 18 to 66yo.</p>
<p>Naïve patients that received Danoprevir/Ritonavir treatment had a
decreased hospitalization time. Patients treated with
Lopinavir/Ritonavir did not have a negative PCR test, while after
switching to Danoprevir/Ritonavir treatment, the first negative PCR test
occurred at a median of two days.</p>
<h3 data-number="9.82.3" id="limitations-79"><span class="header-section-number">9.82.3</span> Limitations</h3>
<p>The results of the study are very hard to interpret as there is no
control group not receiving Danoprevir/Ritonavir treatment. This was
especially true in naïve patients who seemed to have more mild symptoms
before the start of the study and were younger (18 and 44yo) compared to
the experienced patients (18 to 66yo). The possibility that the patients
would have recovered without Danoprevir/Ritonavir treatment cannot be
excluded.</p>
<h3 data-number="9.82.4" id="significance-75"><span class="header-section-number">9.82.4</span> Significance</h3>
<p>The authors of the study treated patients with Danoprevir, with the
rational to that this is an approved and well tolerated drug for HCV
patients <span class="citation" data-cites="10GnhBctk">[<a href="#ref-10GnhBctk" role="doc-biblioref">638</a>]</span>, and that it could also target the protease from SARS-CoV-2
(essential for viral replication and transcription). Indeed, homology
modelling data indicated that HCV protease inhibitors have the highest
binding affinity to Sars-Cov2 protease among other approved drugs <span class="citation" data-cites="7zBwK1wn">[<a href="#ref-7zBwK1wn" role="doc-biblioref">639</a>]</span>.</p>
<p>While this study shows that the combination of Danoprevir and Ritonavir
might be beneficial for Covid-19 patients, additional clinical trials
with more patients and with better methodology (randomization and
control group) are needed to make further conclusions.</p>
<h3 data-number="9.82.5" id="credit-79"><span class="header-section-number">9.82.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.83" id="efficacy-of-hydroxychloroquine-in-patients-with-covid-19-results-of-a-randomized-clinical-trial"><span class="header-section-number">9.83</span> Efficacy of hydroxychloroquine in patients with COVID-19: results of a randomized clinical trial</h2>
<p><span class="citation" data-cites="wzHb7Mj5">[<a href="#ref-wzHb7Mj5" role="doc-biblioref">289</a>]</span></p>
<h3 data-number="9.83.1" id="keywords-82"><span class="header-section-number">9.83.1</span> Keywords</h3>
<ul>
<li>hydroxychloroquine</li>
</ul>
<h3 data-number="9.83.2" id="study-description-1"><span class="header-section-number">9.83.2</span> Study Description</h3>
<p>This is a randomized clinical trial of hydroxychloroquine (HCQ) efficacy
in the treatment of COVID-19. From February 4 – February 28, 2020 142
COVID-19 positive patients were admitted to Renmin Hospital of Wuhan
University. 62 patients met inclusion criteria and were enrolled in a
double blind, randomized control trial, with 31 patients in each arm.</p>
<p>Inclusion criteria:</p>
<ol type="1">
<li><p>Age ≥ 18 years</p></li>
<li><p>Positive diagnosis COVID-19 by detection of SARS-CoV-2 by RT-PCR</p></li>
<li><p>Diagnosis of pneumonia on chest CT</p></li>
<li><p>Mild respiratory illness, defined by SaO<sub>2</sub>/SPO<sub>2</sub> ratio &gt; 93% or
PaO<sub>2</sub>/FIO<sub>2</sub> ratio &gt; 300 mmHg in hospital room conditions (Note:
relevant clinical references described below.)</p>
<ol type="a">
<li><p>Hypoxia is defined as an SpO2 of 85-94%; severe hypoxia &lt;
85%.</p></li>
<li><p>The PaO<sub>2</sub>/FIO<sub>2</sub> (ratio of arterial oxygen tension to fraction
of inspired oxygen) is used to classify the severity of acute
respiratory distress syndrome (ARDS). Mild ARDS has a
PaO<sub>2</sub>/FIO<sub>2</sub> of 200-300 mmHg, moderate is 100-200, and severe
&lt; 100.</p></li>
</ol></li>
<li><p>Willing to receive a random assignment to any designated treatment
group; not participating in another study at the same time</p></li>
</ol>
<p>Exclusion criteria:</p>
<ol type="1">
<li><p>Severe or critical respiratory illness (not explicitly defined,
presumed to be respiratory function worse than outlined in inclusion
criteria); or participation in trial does not meet patient’s maximum
benefit or safe follow up criteria</p></li>
<li><p>Retinopathy or other retinal diseases</p></li>
<li><p>Conduction block or other arrhythmias</p></li>
<li><p>Severe liver disease, defined by Child-Pugh score ≥ C or AST &gt;
twice the upper limit</p></li>
<li><p>Pregnant or breastfeeding</p></li>
<li><p>Severe renal failure, defined by eGFR ≤ 30 mL/min/1.73m<sup>2</sup>, or on
dialysis</p></li>
<li><p>Potential transfer to another hospital within 72h of enrollment</p></li>
<li><p>Received any trial treatment for COVID-19 within 30 days before the
current study</p></li>
</ol>
<p>All patients received the standard of care: oxygen therapy, antiviral
agents, antibacterial agents, and immunoglobulin, with or without
corticosteroids. Patients in the HCQ treatment group received additional
oral HCQ 400 mg/day, given as 200 mg 2x/day. HCQ was administered from
days 1-5 of the trial. The primary endpoint was 5 days post enrollment
or a severe adverse reaction to HCQ. The primary outcome evaluated was
time to clinical recovery (TTCR), defined as return to normal body
temperature and cough cessation for &gt; 72h. Chest CT were imaged on
days 0 and 6 of the trial for both groups; body temperature and patient
reports of cough were collected 3x/day from day 0 – 6. The mean age and
sex distribution between the HCQ and control arms were comparable.</p>
<h3 data-number="9.83.3" id="main-findings-79"><span class="header-section-number">9.83.3</span> Main Findings</h3>
<p>There were 2 patients showing mild secondary effects of HCQ treatment.
More importantly, while 4 patients in the control group progressed to
severe disease, none progressed in the treatment group.</p>
<p>TTCR was significantly decreased in the HCQ treatment arm; recovery from
fever was shortened by one day (3.2 days control vs. 2.2 days HCQ, p =
0.0008); time to cessation of cough was similarly reduced (3.1 days
control vs. 2.0 days HCQ, p = 0.0016).</p>
<p>Overall, it appears that HCQ treatment of patients with mild COVID-19
has a modest effect on clinical recovery (symptom relief on average 1
day earlier) but may be more potent in reducing the progression from
mild to severe disease.</p>
<h3 data-number="9.83.4" id="limitations-80"><span class="header-section-number">9.83.4</span> Limitations</h3>
<p>This study is limited in its inclusion of only patients with mild
disease, and exclusion of those on any treatment other than the standard
of care. It would also have been important to include the laboratory
values of positive RT-PCR detection of SARS-CoV-2 to compare the
baseline and evolution of the patients’ viral load.</p>
<h3 data-number="9.83.5" id="limitations-81"><span class="header-section-number">9.83.5</span> Limitations</h3>
<p>Despite its limitations, the study design has good rigor as a double
blind RCT and consistent symptom checks on each day of the trail. Now
that the FDA has approved HCQ for treatment of COVID-19 in the USA, this
study supports the efficacy of HCQ use early in treatment of patients
showing mild symptoms, to improve time to clinical recovery, and
possibly reduce disease progression. However, most of the current
applications of HCQ have been in patients with severe disease and for
compassionate use, which are out of the scope of the findings presented
in this trial. Several additional clinical trials to examine
<a href="https://twitter.com/hashtag/hydroxychloroquine?src=hashtag_click">hydroxychloroquine</a>
are now undergoing; their results will be critical to further validate
these findings.</p>
<h3 data-number="9.83.6" id="credit-80"><span class="header-section-number">9.83.6</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<p><strong>Structure-based modeling of SARS-CoV-2 peptide/HLA-A02 antigens</strong></p>
<p><a href="https://doi.org/10.1101/2020.03.23.004176" class="uri">https://doi.org/10.1101/2020.03.23.004176</a></p>
<p><em>Immunology keywords</em>:</p>
<p>CoVID-19, 2019-nCoV, SARS-CoV-2, comparative, homology, peptide,
modeling, simulation, HLA-A, antigen</p>
<p><em>Summary of Findings</em>:</p>
<ul>
<li><p>The authors utilize homology modeling to identify peptides from the
SARS-CoV-2 proteome that potentially bind HLA-A*02:01.</p></li>
<li><p>They utilize high-resolution X-ray structures of peptide/MHC
complexes on Protein Data Bank, substitute homologous peptides with
SARS-CoV-2 peptides, and calculate MHC/SARS-CoV-2 peptide Rosetta
binding energy.</p></li>
<li><p>They select MHC/SARS-CoV-2 complex models with highest binding
energy for further study and publish models in an online database
(https://rosettamhc.chemistry.ucsc.edu).</p></li>
</ul>
<p><em>Limitations</em>:</p>
<ul>
<li><p>The authors only utilize computational methods and predicted
SARS-CoV-2 peptides must be validated experimentally for
immunogenicity and clinical response.</p></li>
<li><p>Due to computational burden and limited availability of high
resolution X-ray structures on PDB, authors only simulate 9-mer and
10-mer peptide binding to HLA-A*02:01.</p></li>
<li><p>Since the authors compare select existing X-ray structures as a
starting point, backbone conformations that deviate significantly
between test and template peptides are not captured. Furthermore,
Rosetta modeling protocols do not capture all possible structures
and binding energy scoring does not fully recapitulate fundamental
forces.<sup>1,2</sup></p></li>
</ul>
<p><em>Importance/Relevance:</em></p>
<ul>
<li><p>The authors identify and publish high-scoring SARS-CoV-2 peptides
that may direct a targeted, experimental validation approach toward
a COVID-19 vaccine.</p></li>
<li><p>The authors utilize Rosetta simulation to further filter results
from NetMHCpan 4.0, supporting machine learning prediction with
structural analysis.</p></li>
<li><p>The authors develop RosettaMHC, a computationally efficient method
of leveraging existing X-ray structures for identification of
immunogenic peptides.</p></li>
</ul>
<p><em>References:</em></p>
<ol type="1">
<li><p>Bender, B. J., Cisneros, A., 3rd, Duran, A. M., Finn, J. A., Fu, D.,
Lokits, A. D., . . . Moretti, R. (2016). Protocols for Molecular
Modeling with Rosetta3 and RosettaScripts. Biochemistry, 55(34),
4748-4763. doi:10.1021/acs.biochem.6b00444</p></li>
<li><p>Alford, R. F., Leaver-Fay, A., Jeliazkov, J. R., O’Meara, M. J.,
DiMaio, F. P., Park, H., . . . Gray, J. J. (2017). The Rosetta
All-Atom Energy Function for Macromolecular Modeling and Design. J
Chem Theory Comput, 13(6), 3031-3048. doi:10.1021/acs.jctc.7b00125</p></li>
</ol>
<p><em>Review by Jonathan Chung as part of a project by students, postdocs and
faculty at the Immunology Institute of the Icahn school of medicine,
Mount Sinai.</em></p>
<h2 data-number="9.84" id="serology-characteristics-of-sars-cov-2-infection-since-the-exposure-and-post-symptoms-onset"><span class="header-section-number">9.84</span> Serology characteristics of SARS-CoV-2 infection since the exposure and post symptoms onset</h2>
<p>Lou et al. <em>medRxiv</em>. <span class="citation" data-cites="lnuNSHEW">[<a href="#ref-lnuNSHEW" role="doc-biblioref">640</a>]</span></p>
<h3 data-number="9.84.1" id="keywords-83"><span class="header-section-number">9.84.1</span> Keywords</h3>
<ul>
<li>Seroconversion rate</li>
<li>Total Antibody</li>
<li>Ab</li>
<li>IgG and IgM</li>
<li>antibody</li>
</ul>
<h3 data-number="9.84.2" id="main-findings-80"><span class="header-section-number">9.84.2</span> Main Findings</h3>
<p>Currently, the diagnosis of SARS-CoV-2 infection
entirely depends on the detection of viral RNA using polymerase chain
reaction (PCR) assays. False negative results are common, particularly
when the samples are collected from upper respiratory. Serological
detection may be useful as an additional testing strategy. In this study
the authors reported that a typical acute antibody response was induced
during the SARS-CoV-2 infection, which was discuss earlier<sup>1</sup>. The
seroconversion rate for Ab, IgM and IgG in COVID-19 patients was 98.8%
(79/80), 93.8% (75/80) and 93.8% (75/80), respectively. The first
detectible serology marker was total antibody followed by IgM and IgG,
with a median seroconversion time of 15, 18 and 20 days-post exposure
(d.p.e) or 9, 10- and 12-days post-onset (d.p.o). Seroconversion was
first detected at day 7d.p.e in 98.9% of the patients. Interestingly
they found that viral load declined as antibody levels increased. This
was in contrast to a previous study <span class="citation" data-cites="5HXuCa5t">[<a href="#ref-5HXuCa5t" role="doc-biblioref">581</a>]</span>, showing that increased antibody
titers did not always correlate with RNA clearance (low number of
patient sample).</p>
<h3 data-number="9.84.3" id="limitations-82"><span class="header-section-number">9.84.3</span> Limitations</h3>
<p>Current knowledge of the antibody response to SAR-CoV-2
infection and its mechanism is not yet well elucidated. Similar to the
RNA test, the absence of antibody titers in the early stage of illness
could not exclude the possibility of infection. A diagnostic test, which
is the aim of the authors, would not be useful at the early time points
of infection but it could be used to screen asymptomatic patients or
patients with mild disease at later times after exposure.</p>
<h3 data-number="9.84.4" id="significance-76"><span class="header-section-number">9.84.4</span> Significance</h3>
<p>Understanding the antibody responses against SARS-CoV2 is
useful in the development of a serological test for the diagnosis of
COVID-19. This manuscript discussed acute antibody responses which can
be deducted in plasma for diagnostic as well as prognostic purposes.
Thus, patient-derived plasma with known antibody titers may be used
therapeutically for treating COVID-19 patients with severe illness.
<span id="OLE_LINK3" class="anchor"><span id="OLE_LINK4" class="anchor"></span></span></p>
<h3 data-number="9.84.5" id="credit-81"><span class="header-section-number">9.84.5</span> Credit</h3>
<p><em>This review was undertaken and edited by Konstantina A as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.85" id="sars-cov-2-launches-a-unique-transcriptional-signature-from-in-vitro-ex-vivo-and-in-vivo-systems"><span class="header-section-number">9.85</span> SARS-CoV-2 launches a unique transcriptional signature from in vitro, ex vivo, and in vivo systems</h2>
<p>Blanco-Melo et al. <em>bioRxiv</em>. <span class="citation" data-cites="aNSdARwe">[<a href="#ref-aNSdARwe" role="doc-biblioref">641</a>]</span></p>
<h3 data-number="9.85.1" id="keywords-84"><span class="header-section-number">9.85.1</span> Keywords</h3>
<ul>
<li>host cellular response</li>
<li>host-pathogen interaction</li>
<li>type I interferon</li>
<li>type III interferon</li>
<li>inflammation</li>
<li>RNA-seq</li>
<li>comparative analysis</li>
</ul>
<h3 data-number="9.85.2" id="main-findings-81"><span class="header-section-number">9.85.2</span> Main Findings</h3>
<p>Given the high mortality rate of SARS-CoV-2 relative to
other respiratory viruses such seasonal IAV and RSV, there may be
underlying host-pathogen interactions specific to SARS-CoV-2 that
predispose to a worse clinical outcome. Using <em>in vivo</em>, <em>ex vivo</em>, and
<em>in vitro</em> systems, the authors profiled host cell transcriptional
responses to SARS-CoV-2 and to other common respiratory viruses
(seasonal IAV and RSV). SARS-CoV-2 infection <em>in vitro</em> led to an
induction of type I interferon response signaling and the upregulation
of cytokine/chemokines transcripts. In comparison with IAV and RSV
infection, SARS-CoV-2 <em>in vitro</em> appears to uniquely induce less type I
and type III interferon expression and higher levels of two cytokines
previously implicated in respiratory inflammation. Lastly, <em>in vivo</em>
data from ferrets showed a reduced induction of cytokines and chemokines
by SARS-CoV-2 infection relative to IAV infection.</p>
<h3 data-number="9.85.3" id="limitations-83"><span class="header-section-number">9.85.3</span> Limitations</h3>
<p>While these results are promising, there are several
key weaknesses of this paper. 1) As the authors point out, there is an
undetectable level of SARS-CoV-2 putative receptor (ACE2) and protease
(TMPRSS2) expression in the lung epithelial cell line used for the <em>in
vitro</em> studies. This raises the important question of whether viral
replication actually occurs in any of the models used, which may explain
the lack of interferon production observed <em>in vitro</em> in SARS-CoV-2
treated cells. Further studies characterizing viral titers across
timepoints are needed. 2) Furthermore, these studies only characterize
the host response at a single dose and timepoint per virus, and it is
unclear why these doses/timepoints were chosen. This leaves open the
possibility that the observed differences between viruses could be due
to differences in dose, timing, host response, or a combination of all
of these. 3) It is unclear whether ferrets are productively infected,
which cell types are infected, and the extent/timing of the clinical
course of infection. Moreover, the <em>in vitro</em> and <em>in vivo</em> data do not
strongly correlate and the reasons for this are unclear.</p>
<h3 data-number="9.85.4" id="significance-77"><span class="header-section-number">9.85.4</span> Significance</h3>
<p>This paper describes potentially unique
transcriptional signatures of host cells exposed to SARS-CoV-2. If
validated, these findings may help explain clinical outcomes and could
be targeted in future therapeutic interventions.</p>
<h3 data-number="9.85.5" id="potential-conflicts-of-interest-disclosure"><span class="header-section-number">9.85.5</span> Potential Conflicts of Interest Disclosure</h3>
<p>The reviewers are also researchers at the Icahn School
of Medicine at Mount Sinai.</p>
<h3 data-number="9.85.6" id="credit-82"><span class="header-section-number">9.85.6</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.86" id="a-new-predictor-of-disease-severity-in-patients-with-covid-19-in-wuhan-china"><span class="header-section-number">9.86</span> A New Predictor of Disease Severity in Patients with COVID-19 in Wuhan, China</h2>
<p>Zhou et al. <em>bioRxiv</em>. <span class="citation" data-cites="EWZmuLvu">[<a href="#ref-EWZmuLvu" role="doc-biblioref">642</a>]</span></p>
<h3 data-number="9.86.1" id="keywords-85"><span class="header-section-number">9.86.1</span> Keywords</h3>
<ul>
<li>disease severity</li>
<li>clinical data</li>
<li>Neutrophils/lymphocytes ratio</li>
<li>CRP</li>
<li>D-dimer</li>
</ul>
<h3 data-number="9.86.2" id="main-findings-82"><span class="header-section-number">9.86.2</span> Main Findings</h3>
<p>377 hospitalized patients were divided into two groups: severe and
non-severe pneumonia. The laboratory results of their first day of
admission were retrospectively analyzed to identify predictors of
disease severity.</p>
<p>After adjusting for confounding factors from chronic comorbidities (such
as high blood pressure, type 2 diabetes, coronary heart disease, and
chronic obstructive pulmonary disease), the independent risk factors
identified for severe pneumonia were <strong>age</strong>, the <strong>ratio of
neutrophil/lymphocytes counts</strong>, <strong>CRP</strong> and <strong>D-dimer</strong> levels.</p>
<p>To further increase the specificity and sensibility of these markers,
they showed that their multiplication <strong>[(Neutrophil/lymphocyte count)
* CRP * D-dimer]</strong> was a better predictor of disease severity, with
higher sensitivity (95.7%) and specificity (63.3%), with a cutoff value
of 2.68.</p>
<h3 data-number="9.86.3" id="limitations-84"><span class="header-section-number">9.86.3</span> Limitations</h3>
<p>This study included 377 hospitalized patients. Among them, 45.6%
patients tested positive for SARS-Cov-2 nucleic acid test results, and
others were included in the study based on clinically diagnosis even if
the molecular diagnosis was negative. Thus, additional studies are
needed to verify this on a larger number of covid-19 certified patients
and the cutoff value might be adjusted. Also, all the patients that did
not have the clinical characteristics of severe pneumonia were included
in the non-severe pneumonia group, but usually patients are also divided
into moderate and mild disease.</p>
<p>Also, studying different subset of lymphocytes could lead to a more
specific predictor. Another study showed that the neutrophils to CD8+ T
cells ratio was a strong predictor of disease severity <span class="citation" data-cites="zTGzuiTI">[<a href="#ref-zTGzuiTI" role="doc-biblioref">551</a>]</span>. Another more
precise study showed that the percentage of helper T cells and
regulatory T cells decrease but the percentage of naïve helper T cells
increases in severe cases <span class="citation" data-cites="6Y81C4P3">[<a href="#ref-6Y81C4P3" role="doc-biblioref">544</a>]</span>. Taking these subpopulations into account
might make the predictor more powerful.</p>
<p>Other studies also noted an inverse correlation between disease severity
and LDH <span class="citation" data-cites="USOnp7c0">[<a href="#ref-USOnp7c0" role="doc-biblioref">585</a>]</span> or IL6 <span class="citation" data-cites="1GEbBtBgh">[<a href="#ref-1GEbBtBgh" role="doc-biblioref">594</a>]</span> levels, but the authors here do not discuss LDH nor
IL6 levels, although this could help to strengthen the predictor.</p>
<p>The study is based on the results obtained on the first day of
admission, studying the dynamic of the changes in patients might also be
interesting to better predict disease severity.</p>
<h3 data-number="9.86.4" id="significance-78"><span class="header-section-number">9.86.4</span> Significance</h3>
<p>This study confirms that the neutrophil to lymphocyte ratio can be a
predictor of disease severity as shown by many others <span class="citation" data-cites="6Y81C4P3 VRoblZFJ gK5e7Yhl">[<a href="#ref-gK5e7Yhl" role="doc-biblioref">543</a>,<a href="#ref-6Y81C4P3" role="doc-biblioref">544</a>,<a href="#ref-VRoblZFJ" role="doc-biblioref">558</a>]</span>. The
novelty here is that they show that a combination with other markers can
enhance the specificity and sensibility of the predictor, although the
study could be improved by taking into account sub-populations of
lymphocytes and more biological factors from patients such as LDH and
IL6.</p>
<h3 data-number="9.86.5" id="credit-83"><span class="header-section-number">9.86.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.87" id="metabolic-disturbances-and-inflammatory-dysfunction-predict-severity-of-coronavirus-disease-2019-covid-19-a-retrospective-study"><span class="header-section-number">9.87</span> Metabolic disturbances and inflammatory dysfunction predict severity of coronavirus disease 2019 (COVID-19): a retrospective study</h2>
<p>Shuke Nie et al. <em>medRxiv</em>. <span class="citation" data-cites="95sQSGzx">[<a href="#ref-95sQSGzx" role="doc-biblioref">643</a>]</span></p>
<h3 data-number="9.87.1" id="keywords-86"><span class="header-section-number">9.87.1</span> Keywords</h3>
<ul>
<li>metabolism</li>
<li>fasting blood glucose</li>
<li>serum total protein</li>
<li>albumin</li>
<li>blood lipid</li>
<li>HDL-C</li>
<li>APOA1</li>
<li>lymphocytopenia</li>
<li>IL-6</li>
<li>CRP</li>
<li>severity prediction of COVID19</li>
</ul>
<h3 data-number="9.87.2" id="main-findings-83"><span class="header-section-number">9.87.2</span> Main Findings</h3>
<p>Retrospective Study on 97 COVID-19 hospitalized patients
(25 severe and 72 non-severe) analyzing clinical and laboratory
parameter to predict transition from mild to severe disease based on
more accessible indicators (such as fasting blood glucose, serum protein
or blood lipid) than inflammatory indicators. In accordance with other
studies, age and hypertension were risk factors for disease severity,
and lymphopenia and increased IL-6 was observed in severe patients. The
authors show that fasting blood glucose (FBG) was altered and patients
with severe disease were often hyperglycemic. Data presented support
that hypoproteinaemia, hypoalbuminemia, and reduction in
high-densitylipoprotein (HDL-C) and ApoA1 were associated with disease
severity.</p>
<h3 data-number="9.87.3" id="limitations-85"><span class="header-section-number">9.87.3</span> Limitations</h3>
<p>In this study non-severe patients were divided in two groups based on
average course of the disease: mild group1 (14 days, n=28) and mild
group 2 (30 days, n=44). However mild patients with a longer disease
course did not show an intermediate phenotype (between mild patients
with shorter disease course and severe patients), hence it is unclear
whether this was a useful and how it impacted the analysis. Furthermore,
the non-exclusion of co-morbidity factors in the analysis may bias the
results (e.g. diabetic patients and glucose tests) It is not clear at
what point in time the laboratory parameters are sampled. In table 3, it
would have been interesting to explore a multivariate multiple
regression. The correlation lacks of positive control to assess the
specificity of the correlation to the disease vs. correlation in any
inflammatory case. The dynamic study assessing the predictability of the
laboratory parameter is limited to 2 patients. Hence there are several
associations with disease severity, but larger studies are necessary to
test the independent predictive value of these potential biomarkers.</p>
<h3 data-number="9.87.4" id="significance-79"><span class="header-section-number">9.87.4</span> Significance</h3>
<p>As hospital are getting overwhelmed a set of
easily accessible laboratory indicators (such as serum total protein)
would potentially provide a triage methodology between potentially
severe cases and mild ones. This paper also opens the question regarding
metabolic deregulation and COVID-19 severity.</p>
<h3 data-number="9.87.5" id="credit-84"><span class="header-section-number">9.87.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.88" id="viral-kinetics-and-antibody-responses-in-patients-with-covid-19"><span class="header-section-number">9.88</span> Viral Kinetics and Antibody Responses in Patients with COVID-19</h2>
<p><span class="citation" data-cites="btVuoX6V">[<a href="#ref-btVuoX6V" role="doc-biblioref">644</a>]</span></p>
<h3 data-number="9.88.1" id="keywords-87"><span class="header-section-number">9.88.1</span> Keywords</h3>
<ul>
<li>CoVID-19</li>
<li>2019-nCoV</li>
<li>SARS-CoV-2</li>
<li>IgG</li>
<li>IgM</li>
<li>clinical</li>
<li>kinetics</li>
<li>antibodies</li>
</ul>
<h3 data-number="9.88.2" id="main-findings-84"><span class="header-section-number">9.88.2</span> Main Findings</h3>
<ul>
<li><p>Prospective cohort of 67 patients, clinical specimens taken and
follow-up conducted.</p></li>
<li><p>Viral shedding, serum IgM, IgG antibody against NP evaluated and
correlated to disease severity and clinical outcome</p></li>
<li><p>Viral RNA levels peaked at 1 week from febrile/cough symptom onset
in sputum, nasal swabs, and stool samples. Shedding ranged from
12-19 days (median ranges) and was longer in severe patients.</p></li>
<li><p>IgM and IgG titers stratified patients into three archetypes as
‘strong vs weak vs non-responders’. Strong responders (with higher
IgM/IgG titers) were significantly higher in severe patients.</p></li>
</ul>
<h3 data-number="9.88.3" id="limitations-86"><span class="header-section-number">9.88.3</span> Limitations</h3>
<p>Specific for immune monitoring.</p>
<ul>
<li><p>Not clear if stool RNA captured from live infection in
intestine/liver or from swallowed sputum. Transmission electron
microscopy (TEM) carried out on sputum samples as proof of concept,
but not stools. TEM unreasonable for actual clinical diagnosis.</p></li>
<li><p>Several patients had co-morbidities (such as pulmonary and liver
disease) that were not accounted for when tracking antibody
responses. Viral kinetics and IgM/IgG titers in subsets of patients
with underlying conditions/undergoing certain medication would be
informative.</p></li>
</ul>
<h3 data-number="9.88.4" id="significance-80"><span class="header-section-number">9.88.4</span> Significance</h3>
<ul>
<li><p>Three archetypes of antibody response to SARS-CoV-2 with different
disease progression and kinetics is useful to stratify patients, and
for future serological tests.</p></li>
<li><p>Strong spike-IgG levels often correlate with lymphopenia and
CoVID-19 disease severity <span class="citation" data-cites="LIf5SSUs">[<a href="#ref-LIf5SSUs" role="doc-biblioref">645</a>]</span>, similar to macaque
studies in SARS <span class="citation" data-cites="tC5vJmwj">[<a href="#ref-tC5vJmwj" role="doc-biblioref">467</a>]</span>. It would be critical to see if anti-NP or
anti-Spike IgG antibodies for SARS-CoV-2 also elicit similar
detrimental effects before clinical use.</p></li>
</ul>
<h3 data-number="9.88.5" id="credit-85"><span class="header-section-number">9.88.5</span> Credit</h3>
<p><em>Review by Samarth Hegde as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.89" id="covid-19-infection-induces-readily-detectable-morphological-and-inflammation-related-phenotypic-changes-in-peripheral-blood-monocytes-the-severity-of-which-correlate-with-patient-outcome"><span class="header-section-number">9.89</span> COVID-19 infection induces readily detectable morphological and inflammation-related phenotypic changes in peripheral blood monocytes, the severity of which correlate with patient outcome</h2>
<p><span class="citation" data-cites="1FZgghazg">[<a href="#ref-1FZgghazg" role="doc-biblioref">646</a>]</span></p>
<h3 data-number="9.89.1" id="keywords-88"><span class="header-section-number">9.89.1</span> Keywords</h3>
<ul>
<li>Monocytes</li>
<li>FSC-high</li>
<li>PBMC</li>
<li>ACE2</li>
<li>inflammatory cytokines</li>
</ul>
<h3 data-number="9.89.2" id="main-findings-85"><span class="header-section-number">9.89.2</span> Main Findings</h3>
<p>This study is based on flow cytometry immunophenotyping of
PBMCs from 28 patients diagnosed positive for SARS-Cov2 (COVID19). The
authors identify a population of abnormally large (FSC-hi) monocytes,
present in COVID19 patients, but absent in PBMCs of healthy volunteers
(n=16) or patients with different infections (AIDS, malaria, TB). This
FSC-hi monocytic population contains classical, intermediate and
non-classical (monocytes (based on CD14 and CD16 expression) that
produce inflammatory cytokines (IL-6, TNF and IL-10). The authors
suggest an association of FSC-hi monocytes with poor outcome and
correlate a high percentage of FSC-low monocytes, or higher ratio of
FSC-low/hi monocytes, with faster hospital discharge.</p>
<h3 data-number="9.89.3" id="limitations-87"><span class="header-section-number">9.89.3</span> Limitations</h3>
<p>While identification of the monocytic population based
on FSC is rather robust, the characterization of these cells remains
weak. A comprehensive comparison of FSC-hi monocytes with FSC-low
monocytes from patients and healthy controls would be of high value. It
is unclear if percentages in blood are among CD45+ cells. Furthermore,
it would have been important to include absolute numbers of different
monocytic populations (in table 1 there are not enough samples and it is
unclear what the authors show).</p>
<p>The authors show expression of the ACE2 receptor on the surface of the
monocytes, and highlight these cells as potential targets of SARS-Cov2.
However, appropriate controls are needed. CD16 has high affinity to
rabbit IgG and it is unclear whether the authors considered unspecific
binding of rabbit anti-ACE2 to Fc receptors. Gene expression of ACE-2 on
monocytes needs to be assessed. Furthermore, it would be important to
confirm infection of monocytes by presence of viral proteins or viral
particles by microscopy.</p>
<p>Considering the predictive role of FSC-hi monocytes on the development
of the disease and its severity, some data expected at this level are
neither present nor addressed. Although the cohort is small, it does
include 3 ICU patients. What about their ratio of FSC-low vs FSC-hi
monocytes in comparison to other patients? Was this apparent early in
the disease course? Does this population of FSC-hi monocytes differ
between ICU patients and others in terms of frequency, phenotype or
cytokine secretion?</p>
<p>In general, figures need to revised to make the data clear. For example,
in Fig. 5, according to the legend it seems that patients with FSC-high
monocytes are discharged faster from the hospital. However according to
description in the text, patients were grouped in high or low levels of
FSC-low monocytes.</p>
<h3 data-number="9.89.4" id="significance-81"><span class="header-section-number">9.89.4</span> Significance</h3>
<p>Despite the limitations of this study, the discovery of
a FSC-high monocyte population in COVID-19 patients is of great
interest. With similar implication, a the recent study by Zhou et al. <span class="citation" data-cites="Bgo7Cmj3">[<a href="#ref-Bgo7Cmj3" role="doc-biblioref">549</a>]</span>
identified a connection between an inflammatory CD14+CD16+ monocyte
population and pulmonary immunopathology leading to deleterious clinical
manifestations and even acute mortality after SARS-CoV-2 infections.
Although the presence of these monocytes in the lungs has yet to be
demonstrated, such results support the importance of monocytes in the
critical inflammation observed in some COVID19 patients.</p>
<h3 data-number="9.89.5" id="credit-86"><span class="header-section-number">9.89.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.90" id="correlation-between-universal-bcg-vaccination-policy-and-reduced-morbidity-and-mortality-for-covid-19-an-epidemiological-study"><span class="header-section-number">9.90</span> Correlation between universal BCG vaccination policy and reduced morbidity and mortality for COVID-19: an epidemiological study</h2>
<p>Miller et al. <em>medRxiv</em>. <span class="citation" data-cites="15gdkEnfn">[<a href="#ref-15gdkEnfn" role="doc-biblioref">647</a>]</span></p>
<h3 data-number="9.90.1" id="keywords-89"><span class="header-section-number">9.90.1</span> Keywords</h3>
<ul>
<li>BCG vaccine</li>
<li>epidemiology</li>
<li>vaccination policy</li>
</ul>
<h3 data-number="9.90.2" id="main-findings-86"><span class="header-section-number">9.90.2</span> Main Findings</h3>
<p>The authors compared middle and high income countries that never had a
universal BCG vaccination policy (Italy, Lebanon, Nederland, Belgium)
and countries with a current policy (low income countries were excluded
from the analysis as their number of cases and deaths might be
underreported for the moment). <strong>Countries that never implement BCG
vaccination have a higher mortality rate than countries which have a BCG
vaccination policy</strong> (16.38 deaths per million people vs 0.78). Next,
<strong>the authors show that an earlier start of vaccination correlates with
a lower number of deaths per million inhabitants</strong>. They interpret this
as the vaccine protecting a larger fraction of elderly people, which are
usually more affected by COVID-19. Moreover, higher number of COVID-19
<strong>cases</strong> were presented in countries that never implemented a universal
BCG vaccination policy.</p>
<h3 data-number="9.90.3" id="limitations-88"><span class="header-section-number">9.90.3</span> Limitations</h3>
<p>While this study aims to test an intriguing hypothesis unfortunately,
the data is not sufficient at this time to accurately make any
determinations. Several caveats must be noted including: not all
countries are in the same stage of the pandemic, the number of
cases/deaths is still changing very rapidly in a lot of countries and
thus the association may only reflect exposure to the virus. This
analysis would need to be re-evaluated when all the countries are passed
the pandemic and more accurate numbers are available. Additionally, very
few middle and high-income countries ever implemented universal BCG
vaccination, which can be a source of bias (5 countries, vs 55 that have
a BCG vaccine policy). Effective screening and social isolation policies
also varied considerable across the countries tested and may reflect
another important confounder. The authors could consider analyzing the
Case Fatality Rate (CFR, % of patients with COVID-19 that die), to more
correct for exposure although testing availability will still bias this
result. Variability in mortality within countries or cities with
variable vaccination and similar exposure could also be appropriate
although confounders will still be present.</p>
<h3 data-number="9.90.4" id="significance-82"><span class="header-section-number">9.90.4</span> Significance</h3>
<p>BCG vaccine is a live attenuated strain derived from <em>Mycobacterium
bovis</em> and used for a vaccine for tuberculosis (TB). This vaccine has
been proven to be efficient in preventing childhood meningitis TB, but
doesn’t prevent adult TB as efficiently. For this reason, several
countries are now only recommending this vaccine for at-risk population
only.</p>
<p><strong>This study shows that there is a correlation between BCG vaccination
policy and reduced mortality for Covid-19</strong>. Indeed, BCG vaccine has
been shown to protect against several viruses and enhance innate
immunity <span class="citation" data-cites="16FOT89K5">[<a href="#ref-16FOT89K5" role="doc-biblioref">648</a>]</span>, which could explain why it could protect against SARS-CoV-2
infection, but the exact mechanism is still unknown. <strong>Moreover, the
efficiency of adult/older people vaccination and protection against
Covid-19 still needs to be assessed.</strong> Regarding this, Australian
researchers are starting a clinical trial of BCG vaccine for healthcare
workers <span class="citation" data-cites="MCuIRGFE">[<a href="#ref-MCuIRGFE" role="doc-biblioref">649</a>]</span>, to assess if it can protect them against Covid-19.</p>
<h3 data-number="9.90.5" id="credit-87"><span class="header-section-number">9.90.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.91" id="non-neural-expression-of-sars-cov-2-entry-genes-in-the-olfactory-epithelium"><span class="header-section-number">9.91</span> Non-neural expression of SARS-CoV-2 entry genes in the olfactory epithelium</h2>
<p><span class="citation" data-cites="12oDb7aki">[<a href="#ref-12oDb7aki" role="doc-biblioref">650</a>]</span></p>
<h3 data-number="9.91.1" id="keywords-90"><span class="header-section-number">9.91.1</span> Keywords</h3>
<ul>
<li>CoVID-19</li>
<li>2019-nCoV</li>
<li>SARS-CoV-2</li>
<li>anosmia</li>
<li>olfaction</li>
<li>scRNAseq</li>
</ul>
<h3 data-number="9.91.2" id="main-findings-87"><span class="header-section-number">9.91.2</span> Main Findings</h3>
<ul>
<li><p>Study analyzed bulk and scRNAseq data of olfactory cell types from
publicly-available mouse, nonhuman primate and human datasets.</p></li>
<li><p>show that <em>ACE2</em> and <em>TMPRSS2</em> (genes involved in SARS-CoV-2 entry)
are expressed in olfactory epithelial (OE) cells, basal stem cells
and respiratory epithelium (RE), but not sensory neurons.</p></li>
<li><p>Comparison of human RE and OE datasets (Deprez et al. 2019; Durante
et al. 2020) revealed that <em>ACE2</em> and <em>TMPRSS2</em> expression in OE
sustentacular cells was similar to expression in the remainder of
the non-nasal respiratory tract.</p></li>
</ul>
<h3 data-number="9.91.3" id="limitations-89"><span class="header-section-number">9.91.3</span> Limitations</h3>
<ul>
<li><p>Transcript data alone from healthy respiratory/olfactory cells is
not sufficient to confirm infectivity of nasal passage, or to
indicate damage to epithelia.</p></li>
<li><p>No mechanism defined for anosmia; it is not clear if epithelial
injury leads to reduced sensitivity or increased inflammation and
altered immune contexture drives neural/epithelial dysfunction. Will
be critical to test this in CoVID-19 patient samples or mouse
models.</p></li>
</ul>
<h3 data-number="9.91.4" id="significance-83"><span class="header-section-number">9.91.4</span> Significance</h3>
<ul>
<li><p>Study provides possible rationale for anosmia observed in several
CoVID-19 patients.</p></li>
<li><p>Raises possibility that nasal respiratory goblet, ciliated cells,
and olfactory epithelia may serve as a viral reservoir after initial
SARS-CoV-2 infection.</p></li>
<li><p>Human olfactory sensory neurons express several other molecules
important to CoV (not CoV-19) entry such as <em>FURIN, ST6GAL1,
ST3GAL4</em>; this suggests wider mechanism of neuronal infectivity in
other coronaviruses.</p></li>
</ul>
<h3 data-number="9.91.5" id="credit-88"><span class="header-section-number">9.91.5</span> Credit</h3>
<p><em>Review by Samarth Hegde as part of a project by students, postdocs and
faculty at the Immunology Institute of the Icahn School of Medicine,
Mount Sinai.</em></p>
<p><strong>Title: </strong></p>
<p><strong>SARS-CoV-2 proteome microarray for mapping COVID-19 antibody
interactions at amino acid resolution</strong></p>
<p><strong>Immunology keywords:</strong> SARS-CoV-2, COVID-19, high throughput, peptide
microarray, antibody epitope screening</p>
<p><strong>The main finding of the article: </strong></p>
<p>This study screened the viral protein epitopes recognized by antibodies
in the serum of 10 COVID-19 patients using a new SARS-CoV-2 proteome
peptide microarray. The peptide library was constructed with 966 linear
peptides, each 15 amino acids long with a 5 amino acid overlap, based on
the protein sequences encoded by the genome of the Wuhan-Hu-1 strain.</p>
<p>To investigate crossreactivity between SARS-CoV-1 and SARS-CoV-2, they
tested rabbit monoclonal and polyclonal antibodies against SARS-CoV-1
nucleocapsid (N) in the microarray. Antibodies against SARS-CoV-1 N
displayed binding to the SARS-CoV-2 nucleocapsid (N) peptides.
Polyclonal antibodies showed some crossreactivity to other epitopes from
membrane (M), spike (S), ORF1ab and ORF8. This suggests that previous
exposure to SARS-CoV-1 may induced antibodies recognizing both viruses.</p>
<p>Screening of IgM and IgG antibodies from 10 COVID-19 patients showed
that many antibodies targeted peptides on M, N, S, Orf1ab, Orf3a, Orf7a,
and Orf8 from SARS-CoV-2, while immunodominant epitopes with antibodies
in more than 80 % COVID-19 patients were present in N, S and Orf3. It is
shown that the receptor binding domain (RBD) resides on S protein and
RBD is important for SARS-CoV-2 to enter the host cells via ACE2. Among
six epitopes on S protein, structural analysis predicted that three
epitopes were located at the surface and three epitopes were located
inside of the protein. Furthermore, some IgM antibodies from 1 patient
and IgG antibodies from 2 patients bound to the same epitope (residue
456-460, FRKSN) which resided within the RBD, and structural analysis
determined that this epitope was located in the region of the RBD loop
that engages with ACE2.</p>
<p><strong>Critical analysis of the study:</strong></p>
<p>In addition to the limitations mentioned in the manuscript, it would
have been informative to do the analysis over the course of the disease.
The pattern of antibody recognition, especially on S protein, and the
course of antibodies of different isotypes recognizing the same peptide
might correlate to the clinical course in these patients. It would alos
have been informative to analyze the presence of cross-reactive
antibodies from pateints previously exposed to SARS-CoV-1.</p>
<p><strong>The importance and implications for the current epidemics:</strong></p>
<p>This study identified linear immunodominant epitopes on SARS-CoV-2,
Wuhan-Hu-1 strain. This is a valuable information to design vaccines
that will elicit desirable immune responses.</p>
<p><strong>The Novel Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2)
Directly Decimates Human Spleens and Lymph Nodes</strong></p>
<p>Review by Matthew D. Park</p>
<p>Revised by Miriam Merad</p>
<p>Keywords: COVID-19, SARS-CoV-2, spleen, lymph node, ACE2, macrophage</p>
<p><em>Main findings</em></p>
<p>It has been previously reported that COVID-19 patients exhibit severe
lymphocytopenia, but the mechanism through which this depletion occurs
has not been described. In order to characterize the cause and process
of lymphocyte depletion in COVID-19 patients, the authors performed
gross anatomical and <em>in situ</em> immune-histochemical analyses of spleens
and lymph nodes (hilar and subscapular) obtained from post-mortem
autopsies of 6 patients with confirmed positive viremia and 3 healthy
controls (deceased due to vehicle accidents).</p>
<p>Primary gross observations noted significant splenic and LN atrophy,
hemorrhaging, and necrosis with congestion of interstitial blood vessels
and large accumulation of mononuclear cells and massive lymphocyte
death. They found that CD68<sup>+</sup> CD169<sup>+</sup> cells in the spleens, hilar and
subscapular LN, and capillaries of these secondary lymphoid organs
expressed the ACE2 receptor and stain positive for the SARS-CoV-2
nucleoprotein (NP) antigen, while CD3<sup>+</sup> T cells and B220<sup>+</sup> B cells
lacked both the ACE2 receptor and SARS-CoV-2 NP antigen. ACE2<sup>+</sup> NP<sup>+</sup>
CD169<sup>+</sup> macrophages were positioned in the splenic marginal zone (MZ)
and in the marginal sinuses of LN, which suggests that these macrophages
were positioned to encounter invading pathogens first and may contribute
to virus dissemination.</p>
<p>Since SARS-CoV-2 does not directly infect lymphocytes, the authors
hypothesized that the NP<sup>+</sup> CD169<sup>+</sup> macrophages are responsible for
persistent activation of lymphocytes via Fas::FasL interactions that
would mediate activation-induced cell death (AICD). Indeed, the
expression of Fas was significantly higher in virus-infected tissue than
that of healthy controls, and TUNEL staining showed significant
lymphocytic apoptosis. Since pro-inflammatory cytokines like IL-6 and
TNF-α can also engage cellular apoptosis and necrosis, the authors
interrogated the cytokine expression of the secondary lymphoid organs
from COVID-19 patients; IL-6, not TNF-α, was elevated in virus-infected
splenic and lymph node tissues, compared to those of healthy controls,
and immunofluorescent staining showed that IL-6 is primarily produced by
the infected macrophages. <em>In vitro</em> infection of THP1 cells with
SARS-CoV-2 spike protein resulted in selectively increased <em>Il6</em>
expression, as opposed to <em>Il1b</em> and <em>Tnfa</em> transcription. Collectively,
the authors concluded that a combination of Fas up-regulation and IL-6
production by NP<sup>+</sup> CD169<sup>+</sup> macrophages induce AICD in lymphocytes in
secondary lymphoid organs, resulting in lymphocytopenia.</p>
<p>In summary, this study reports that CD169<sup>+</sup> macrophages in the splenic
MZ, subscapular LN, and the lining capillaries of the secondary lymphoid
tissues express ACE2 and are susceptible to SARS-CoV-2 infection. The
findings point to the potential role of these macrophages in viral
dissemination, immunopathology of these secondary lymphoid organs,
hyperinflammation and lymphopenia.</p>
<p><em>Limitations</em></p>
<p><em>Technical</em></p>
<p>A notable technical limitation is the small number of samples (n=6);
moreover, the analysis of these samples using multiplexed
immunohistochemistry and immunofluorescence do not necessarily provide
the depth of unbiased interrogation needed to better identify the cell
types involved.</p>
<p><em>Biological</em></p>
<p>The available literature and ongoing unpublished studies, including
single-cell experiments of spleen and LN from organ donors, do not
indicate that ACE2 is expressed by macrophages; however, it remains
possible that ACE2 expression may be triggered by type I IFN in COVID-19
patients. Importantly, the SARS-CoV-2 NP staining of the macrophages
does not necessarily reflect direct infection of these macrophages;
instead, positive staining only indicates that these macrophages carry
SARS-CoV-2 NP as antigen cargo, which may have been phagocytosed. Direct
viral culture of macrophages isolated from the secondary lymphoid organs
with SARS-CoV-2 is required to confirm the potential for direct
infection of macrophages by SARS-CoV-2. Additionally, it is important to
note that the low to negligible viremia reported in COVID-19 patients
to-date does not favor a dissemination route via the blood, as suggested
by this study, which would be necessary to explain the presence of
virally infected cells in the spleen.</p>
<p><em>Relevance</em></p>
<p>Excess inflammation in response to SARS-CoV-2 infection is characterized
by cytokine storm in many COVID-19 patients. The contribution of this
pathology to the overall fatality rate due to COVID-19, not even
necessarily directly due to SARS-CoV-2 infection, is significant. A
better understanding of the full effect and source of some of these
major cytokines, like IL-6, as well as the deficient immune responses,
like lymphocytopenia, is urgently needed. In this study, the authors
report severe tissue damage in spleens and lymph nodes of COVID-19
patients and identify the role that CD169<sup>+</sup> macrophages may play in the
hyperinflammation and lymphocytopenia that are both characteristic of
the disease. It may, therefore, be important to note the effects that
IL-6 inhibitors like Tocilizumab and Sarilumab may specifically have on
splenic and LN function. It is important to note that similar
observations of severe splenic and LN necrosis and inflammation in
patients infected with SARS-CoV-1 further support the potential
importance and relevance of this study.</p>
<h2 data-number="9.92" id="cigarette-smoke-triggers-the-expansion-of-a-subpopulation-of-respiratory-epithelial-cells-that-express-the-sars-cov-2-receptor-ace2"><span class="header-section-number">9.92</span> Cigarette smoke triggers the expansion of a subpopulation of respiratory epithelial cells that express the SARS-CoV-2 receptor ACE2</h2>
<p><span class="citation" data-cites="t4vofknR">[<a href="#ref-t4vofknR" role="doc-biblioref">651</a>]</span></p>
<h3 data-number="9.92.1" id="keywords-91"><span class="header-section-number">9.92.1</span> Keywords</h3>
<ul>
<li>CoVID-19</li>
<li>2019-nCoV</li>
<li>SARS-CoV-2</li>
<li>respiration</li>
<li>cigarette</li>
<li>ACE2</li>
<li>lung</li>
</ul>
<h3 data-number="9.92.2" id="main-findings-88"><span class="header-section-number">9.92.2</span> Main Findings</h3>
<ul>
<li><p>Study uses scRNAseq, bulk seq data and air-liquid interface culture
experiments to show that cigarette smoke causes a dose-dependent
upregulation of ACE2 in mouse and human lungs (transplantation,
tumor resection, or IPF datasets).</p></li>
<li><p>ACE2 was not up-regulated in patients with asthma or
lung-sarcoidosis or in mouse models of cystic fibrosis or carcinogen
exposure.</p></li>
<li><p>Cathepsin B (alternate protease involved in viral entry) is
increased in smoke-exposed mouse or human lungs.</p></li>
<li><p>Smoke triggers a protective expansion of mucus-secreting MUC5AC+
goblet and SCGB1A1+ club cells; ACE2 presence in these cells is
increased upon smoke exposure.</p></li>
</ul>
<h3 data-number="9.92.3" id="limitations-90"><span class="header-section-number">9.92.3</span> Limitations:</h3>
<ul>
<li><p>Long-term smokers usually have several co-morbidities including
immune dysfunction, which can affect interpretation of CoV-2
susceptibility in these datasets. Ideally, analyses can control for
major co-morbidities across smokers and non-smokers (immune
suppression, cardiovascular disease and atherosclerosis).</p></li>
<li><p>Hyperplasia of ACE2+ goblet cells upon smoking needs to be separated
from ACE2 upregulation in existing goblet cells.</p></li>
<li><p>ACE2 expression increase alone does not confirm increased viral
entry into goblet cells; future studies with air-liquid interface
cultures testing CoV-2 infectivity in <em>ex vivo</em> epithelial cells
from human epithelial lines, ex vivo samples or hACE2 mice will be
very informative.</p></li>
</ul>
<h3 data-number="9.92.4" id="significance-84"><span class="header-section-number">9.92.4</span> Significance</h3>
<ul>
<li><p>This study may partially explain why smokers are more likely to
develop severe SARS-CoV-2 infections. Also, the reversibility of
ACE2 expression upon smoking cessation suggests that quitting
smoking could lessen CoV-2 susceptibility.</p></li>
<li><p>Absence of ACE2 upregulation in other lung inflammation pathologies
implies CoV-2 susceptibility might be smoking-specific, and not
fibrosis-specific.</p></li>
<li><p>Another preprint showed ACE2 expression increases in lung of
patients with CoV-2 co-morbidities such as hypertension <span class="citation" data-cites="wjtVy2v0">[<a href="#ref-wjtVy2v0" role="doc-biblioref">626</a>]</span>; these studies
collectively paint a better picture of CoV-2 susceptibility before
actual experiments can be carried out.</p></li>
</ul>
<h3 data-number="9.92.5" id="credit-89"><span class="header-section-number">9.92.5</span> Credit</h3>
<p><em>Review by Samarth Hegde as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.93" id="the-comparative-superiority-of-igm-igg-antibody-test-to-real-time-reverse-transcriptase-pcr-detection-for-sars-cov-2-infection-diagnosis"><span class="header-section-number">9.93</span> The comparative superiority of IgM-IgG antibody test to real-time reverse transcriptase PCR detection for SARS-CoV-2 infection diagnosis</h2>
<p>Liu et al. <em>medRxiv</em>.
<span class="citation" data-cites="k1E3DuFN">[<a href="#ref-k1E3DuFN" role="doc-biblioref">652</a>]</span></p>
<h3 data-number="9.93.1" id="keywords-92"><span class="header-section-number">9.93.1</span> Keywords</h3>
<ul>
<li>IgM/IgG antibody test</li>
<li>nucleic acid test</li>
<li>SARS-CoV-2 detection</li>
</ul>
<h3 data-number="9.93.2" id="main-findings-89"><span class="header-section-number">9.93.2</span> Main Findings</h3>
<p>The study compares IgM and IgG antibody testing to RT-PCR detection of
SARS-CoV-2 infection. 133 patients diagnosed with SARS-CoV-2 in Renmin
Hospital (Wuhan University, China) were analyzed. The positive ratio was
78.95% (105/133) in IgM antibody test (SARS-CoV-2 antibody detection kit
from YHLO Biotech) and 68.42% (91/133) in RT-PCR (SARS-CoV-2 ORF1ab/N
qPCR detection kit). There were no differences in the sensitivity of
SARS-CO-V2 diagnosis in patients grouped according to disease severity.
For example, IgG responses were detected in 93.18% of moderate cases,
100% of severe cases and 97.3% of critical cases. In sum, positive
ratios were higher in antibody testing compared to RT-PCR detection,
demonstrating a higher detection sensitivity of IgM-IgG testing for
patients hospitalized with COVID-19 symptoms.</p>
<h3 data-number="9.93.3" id="limitations-91"><span class="header-section-number">9.93.3</span> Limitations</h3>
<p>This analysis only included one-time point of 133 hospitalized patients,
and the time from symptom onset was not described. There was no
discussion about specificity of the tests and no healthy controls were
included. It would be important to perform similar studies with more
patients, including younger age groups and patients with mild symptoms
as well as asymptomatic individuals. It is critical to determine how
early after infection/symptom onset antibodies can be detected and the
duration of this immune response.</p>
<h3 data-number="9.93.4" id="significance-85"><span class="header-section-number">9.93.4</span> Significance</h3>
<p>The IgM-IgG combined testing is important to improve clinical
sensitivity and diagnose COVID-19 patients. The combined antibody test
shows higher sensitivity than individual IgM and IgG tests or nucleic
acid-based methods, at least in patients hospitalized with symptoms.</p>
<h3 data-number="9.93.5" id="credit-90"><span class="header-section-number">9.93.5</span> Credit</h3>
<p><em>Review by Erica Dalla as part of a project by students, postdocs and
faculty at the Immunology Institute of the Icahn School of Medicine,
Mount Sinai.</em></p>
<p><strong>Title:</strong> Lectin-like Intestinal Defensin Inhibits 2019-nCoV Spike
binding to ACE2</p>
<p><strong>Immunology keywords:</strong> defensins, spike protein, intestinal Paneth
cells, ACE2 binding</p>
<p><strong>Main Findings:</strong></p>
<p>Human ACE2 was previously identified as the host receptor for
SARS-CoV-2. Despite ACE2 being expressed in both lung alveolar
epithelial cells and small intestine enterocytes, respiratory problems
are the most common symptom after viral infection while intestinal
symptoms are much less frequent. Thus, the authors here investigate the
biology behind the observed protection of the intestinal epithelium from
SARS-CoV-2. Human defensin 5 (HD5), produced by Paneth cells in the
small intestine, was shown to interact with human ACE2, with a binding
affinity of 39.3 nM by biolayer interferometry (BLI). A blocking
experiment using different doses of HD5 coating ACE2 showed that HD5
lowered viral spike protein S1 binding to ACE2. Further, a molecular
dynamic simulation demonstrated a strong intermolecular interaction
between HD5 and the ACE2 ligand binding domain. To test HD5 inhibitory
effect on S1 binding to ACE2, human intestinal epithelium Caco-2 cells
were preincubated with HD5. Preincubation strongly reduced adherence of
S1 to surface of cells. HD5 was effective at a concentration as low as
10 µg/mL, comparable to the concentration found in the intestinal fluid.</p>
<p><strong>Limitations:</strong></p>
<p>The study focuses exclusively on intestinal cells. However, HD5 could
have been tested to block ACE2-S1 binding in human lung epithelial cells
as a potential treatment strategy. It would be useful to know whether
HD5 could also prevent viral entry in lung cells.</p>
<p><strong>Relevance:</strong></p>
<p>This work provides the first understanding of the different efficiency
of viral entry and infection among ACE2-expressing cells and tissues.
Specifically, the authors show that human defensin 5 produced in the
small intestine is able to block binding between S1 and ACE2 necessary
for viral entry into cells. The study provides a plausible explanation
on why few patients show intestinal symptoms and suggests that patients
with intestinal disease that decrease defensins’ production may be more
susceptible to SARS-CoV-2. It also indicates that HD5 could be used as a
molecule to be exogenously administered to patients to prevent viral
infection in lung epithelial cells.</p>
<p><strong>Title: </strong></p>
<p><strong>Susceptibility of ferrets, cats, dogs and different domestic animals
to SARS-coronavirus-2</strong></p>
<p><strong>Immunology keywords:</strong> SARS-CoV-2, ferret, cat, laboratory animal,
domestic animals</p>
<p><strong>The main finding of the article: </strong></p>
<p>This study evaluated the susceptibility of different model laboratory
animals (ferrets), as well as companion (cats and dogs), and domestic
animals (pigs, chickens and ducks) to SARS-CoV-2. They tested infection
with two SARS-CoV2 isolates, one from an environmental sample collected
in the Huanan Seafood Market in Wuhan (F13-E) and the other from a human
patient in Wuhan (CTan-H).</p>
<p><em>Ferrets</em> were inoculated with either of the two viruses by intranasal
route with 10<sup>5</sup> pfu, and the viral replication was evaluated. Two
ferrets from each group were euthanized on day 4 post infection (p.i.).
AT day 4 p.i., viral RNA and infectious viruses were detected only in
upper respiratory tract (nasal turbinate, upper palate, tonisls, but not
in the trachea, lungs or other tissues. Viral RNA and virus titer in the
remaining ferrets were monitored in nasal washes and rectal swabs on
days 2, 4, 6, 8 and 10 p.i. Viral RNA and infectious viruses were
detected in nasal washes until day 8 p.i. One ferret in each group
developed fever and loss of appetite on days 10 and 12 p.i., however,
viral RNA was practically undetactable. These two ferrets showed severe
lymphoplasmacytic perivasculitis and vasculitis in the lungs and lower
antibody titers compare to other 4 ferrets.</p>
<p><em>Cats</em>. Five <em>s</em>ubadult 8-month-old domestic cats were inoculated with
CTan-h virus and three uninfected cats were placed in a cage adjacent to
each of the infected cats to monitor respiratory droplet transmission.
Viral RNA was detected in the upper respiratory organs from all infected
cats and in one out of three exposed cats. All infected (inoculated and
exposed) cats developed elevated antibodies against SARS-CoV2. Viral
replication studies with juvenile cats (70-100 days) revealed massive
lesions in the nasal and tracheal mucosa epithelium and lungs of two
inoculated cats which died or were euthanized on day 3 p.i., and
infection in one out of three exposed cats. These results indicated
SARS-CoV2 could replicate in cats, that juvenile cats were more
susceptible that adults, and theat SARS-CoV2 could be transmit via
respiratory droplets between cats.</p>
<p><em>Dogs and others.</em> Five 3-month-old beagle dogs were inoculated and
housed with two uninoculated beagles in a room. Two virus inoculated
dogs seroconverted, but others including two contact dogs were all
seronegative for SARS-CoV2 and infectious virus was not detected in any
swabs collected. Viral RNA was not detected in swabs from pigs,
chickens, and ducks inoculated or contacted. These results indicated
that dogs, pigs, chickens, and ducks might have low or no susceptibility
to SARS-CoV2.</p>
<p><strong>Critical analysis of the study:</strong></p>
<p>This manuscript describes the viral replication and clinical symptoms of
SARS-CoV2 infection in ferrets, and the SARS-CoV2 infection and
transmission in cats. Clinical and pathological analysis was not
performed in cats, therefore the correlation of virus titer with
symptoms severity in the adult and juvenile cats could not be
determined.</p>
<p><strong>The importance and implications for the current epidemics:</strong></p>
<p>SARS-CoV-2 transmission to tigers, cats and dogs has been previously
reported. It should be noted that this manuscript did not evaluate the
transmission from cats to human. Nevertheless, it clearly showed higher
susceptibility of ferrets and domestic cats to SARS-CoV-2. This data
strongly indicates the need for surveillance of possible infection and
transmission of SARS-CoV-2 by domestic cats.</p>
<h2 data-number="9.94" id="virus-host-interactome-and-proteomic-survey-of-pmbcs-from-covid-19-patients-reveal-potential-virulence-factors-influencing-sars-cov-2-pathogenesis"><span class="header-section-number">9.94</span> Virus-host interactome and proteomic survey of PMBCs from COVID-19 patients reveal potential virulence factors influencing SARS-CoV-2 pathogenesis</h2>
<p>Li et al. <em>bioRxiv</em>.
<span class="citation" data-cites="NLoN4aYj">[<a href="#ref-NLoN4aYj" role="doc-biblioref">182</a>]</span></p>
<h3 data-number="9.94.1" id="keywords-93"><span class="header-section-number">9.94.1</span> Keywords</h3>
<ul>
<li>PBMC</li>
<li>virulence factors
– interaction network
– nsp9</li>
<li>nsp10
– NKRF</li>
</ul>
<h3 data-number="9.94.2" id="main-findings-90"><span class="header-section-number">9.94.2</span> Main findings</h3>
<p>The authors identified <strong>intra-viral protein-protein interactions</strong>
(PPI) with two different approaches: genome wide yeast-two hybrid (Y2H)
and co-immunoprecipitation (co-IP). A total of 58 distinct PPI were
characterized. A screen of <strong>viral-host PPI</strong> was also established by
overexpressing all the SARS-CoV-2 genes with a Flag epitope into HEK293
cells and purifying each protein complex. Interacting host proteins were
then identified by liquid chromatography and tandem mass spectrometry.
251 cellular proteins were identified, such as subunits of ATPase, 40S
ribosomal proteins, T complex proteins and proteasome related proteins,
for a total of 631 viral-host PPI. Several interactions suggesting
protein-mediated modulation of the immune response were identified,
highlighting the multiple ways SARS-CoV-2 might reprogram infected
cells.</p>
<p>Subsequently, the authors compared global proteome profiles of PBMCs
from healthy donors (n=6) with PBMC from COVID-19 patients with mild
(n=22) or severe (n=13) symptoms. 220 proteins were found to be
differentially expressed between <em>healthy donors and mild COVID-19
patients</em>, and a pathway analysis showed <strong>a general activation of the
innate immune response.</strong> 553 proteins were differentially expressed
between the PBMC of <em>mild and severe COVID-19 patients</em>, most of them
(95%) being downregulated in severe patients<em>.</em> Functional pathway
analysis indicated a defect of T cell activation and function in severe
COVID-19. There was also evidence suggesting reduced antibody secretion
by B cells. Together, these results suggest a <strong>functional decline of
adaptive immunity</strong>. A FACS analysis of PBMC from severe patients
indicated higher levels of IL6 and IL8 but not IL17 compared to mild
patients.</p>
<p>Finally, the authors focused on NKRF, an endogenous repressor of IL8/IL6
synthesis that was previously identified as interacting with SARS-Cov-2
nsp9,10,12,13 and 15. Individually expressed nsp9 and nsp10 (but not
nsp12, nsp13, nsp15) induced both IL6 and IL8 in lung epithelial A459
cells, indicating that nsp9 and nsp10 may be directly involved in the
induction of these pro-inflammatory cytokines. The authors finally argue
that nsp9 and nsp10 represent potential drug targets to prevent
over-production of IL6 and IL8 in infected cells, and reducing the
over-activation of neutrophils.</p>
<h3 data-number="9.94.3" id="limitations-92"><span class="header-section-number">9.94.3</span> Limitations</h3>
<p>First, the authors seem to have forgotten to include the extended data
in the manuscript, and their proteomic data does not seem to be publicly
available for the moment, which limits greatly our analysis of their
results.</p>
<p>While this work provides important data on host and viral PPI, only 19
interactions were identified by Y2H system but 52 with co-IP. The
authors do not comment about what could lead to such differences between
the two techniques and they don’t specify whether they detected the same
interactions using the two techniques.</p>
<p>Moreover, the PBMC protein quantification was performed comparing bulk
PBMC. Consequently, protein differences likely reflect differences in
cell populations rather than cell-intrinsic differences in protein
expression. While this analysis is still interesting, a similar
experiment performed on pre-sorted specific cell populations would allow
measuring proteome dynamics at a higher resolution.</p>
<p>Finally, the authors did not discussed their results in regards to
another SARS-CoV-2 interactome of host-viral PPI that had been published
previously<sup>1</sup>. This study reported 332 host-virus PPI, but no
interaction of viral proteins with NKRF was found. Some interactions
were found in both studies (eg. N and G3BP1, Orf6 and RAE1). However,
the time point used to lyse the cells were different (40h previously vs
72h here), which could explain some of the differences.</p>
<h3 data-number="9.94.4" id="relevance"><span class="header-section-number">9.94.4</span> Relevance</h3>
<p>The identification of many interactions between intra-viral and
host-virus PPI provides an overview of host protein and pathways that
are modulated by SARS-CoV-2, which can lead to the identification of
potential targets for drug development.</p>
<p>In the model proposed by the authors, nsp9 and nsp10 from SARS-Cov-2
induce an over-expression of IL6 and IL8 by lung epithelial cells, which
recruits neutrophils and could lead to an excess in lung infiltration.
Nsp9 has been shown to be essential for viral replication for
SARS-Cov-1<sup>2</sup>, and shares a 97% homology with nsp9 from SARS-Cov-2<sup>3</sup>.
Further, nsp9 crystal structure was recently solved<sup>3</sup>, which can help
to develop drug inhibitors if this protein is further confirmed as being
important for the virulence of SARS-Cov-2.</p>
<p>1. Gordon DE, Jang GM, Bouhaddou M, et al. A SARS-CoV-2-Human
Protein-Protein Interaction Map Reveals Drug Targets and Potential
Drug-Repurposing. <em>bioRxiv</em>. March 2020:2020.03.22.002386.
doi:10.1101/2020.03.22.002386</p>
<p>2. Miknis ZJ, Donaldson EF, Umland TC, Rimmer RA, Baric RS, Schultz LW.
Severe acute respiratory syndrome coronavirus nsp9 dimerization is
essential for efficient viral growth. <em>J Virol</em>. 2009;83(7):3007-3018.
doi:10.1128/JVI.01505-08</p>
<p>3. Littler DR, Gully BS, Colson RN, Rossjohn J. <em>Crystal Structure of
the SARS-CoV-2 Non-Structural Protein 9, Nsp9</em>. Molecular Biology; 2020.
doi:10.1101/2020.03.28.013920</p>
<p><strong><em>Title:</em></strong> Prediction and Evolution of B Cell Epitopes of Surface
Protein in SARS-CoV-2</p>
<p><strong><em>Keywords:</em></strong> SARS-CoV-2; Epitopes; Bioinformatics; Evolution</p>
<p><strong><em>Summary/Main findings: </em></strong></p>
<p>Lon et al. used a bioinformatic analysis of the published SARS-CoV-2
genomes in order to identify conserved linear and conformational B cell
epitopes found on the spike (S), envelope (E), and membrane (M)
proteins. The characterization of the surface proteins in this study
began with an assessment of the peptide sequences in order to identify
hydrophilicity indices and protein instability indices using the
Port-Param tool in ExPASy. All three surface proteins were calculated to
have an instability score under 40 indicating that they were stable.
Linear epitopes were identified on the basis of surface probability and
antigenicity, excluding regions of glycosylation. Using BepiPred 2.0
(with a cutoff value of 0.35) and ABCpred (with a cutoff value of 0.51),
4 linear B cell epitopes were predicted for the S protein, 1 epitope for
the E protein, and 1 epitope for the M protein. For structural analysis,
SARS-CoV assemblies published in the Protein Data Bank (PDB) acting as
scaffolds for the SARS-CoV-2 S and E amino acid sequences were used for
input into the SWISS-MODEL server in order to generate three-dimensional
structural models for the assessment of conformational epitopes. Using
Ellipro (cutoff value of 0.063) and SEPPA (cutoff value of 0.5), 1
conformational epitope was identified for the S protein and 1 epitope
was identified for the E protein, both of which are accessible on the
surface of the virus. Finally, the Consurf Server was used to assess the
conservation of these epitopes. All epitopes were conserved across the
published SARS-CoV-2 genomes and one epitope of the spike protein was
predicted to be the most stable across coronavirus phylogeny.</p>
<p><strong><em>Critical Analysis/Limitations:</em></strong></p>
<p>While this study provides a preliminary identification of potential
linear and conformational B cell epitopes, the translational value of
the epitopes described still needs extensive experimental validation to
ascertain whether these elicit a humoral immune response. The
conformational epitope analyses are also limited by the fact that they
are based off of predicted 3D structure from homology comparisons and
not direct crystal structures of the proteins themselves. Additionally,
since there was not a published M protein with a high homology to
SARS-CoV-2, no conformational epitopes were assessed for this protein.
Finally, while evolutionary conservation is an important consideration
in understanding the biology of the virus, conservation does not
necessarily imply that these sites neutralize the virus or aid in
non-neutralizing <em>in vivo</em> protection.</p>
<p><strong><em>Relevance/Implications:</em></strong></p>
<p>With further experimental validation that confirms that these epitopes
induce effective antibody responses to the virus, the epitopes described
can be used for the development of treatments and vaccines as well as
better characterize the viral structure to more deeply understand
pathogenesis.</p>
<h2 data-number="9.95" id="potent-binding-of-2019-novel-coronavirus-spike-protein-by-a-sars-coronavirus-specific-human-monoclonal-antibody-1"><span class="header-section-number">9.95</span> Potent binding of 2019 novel coronavirus spike protein by a SARS coronavirus-specific human monoclonal antibody</h2>
<p>Tian et al. <em>Emerg Microbes Infect 2020</em> <span class="citation" data-cites="1HLSeqnY8">[<a href="#ref-1HLSeqnY8" role="doc-biblioref">530</a>]</span></p>
<h3 data-number="9.95.1" id="keywords-94"><span class="header-section-number">9.95.1</span> Keywords</h3>
<ul>
<li>Monoclonal antibody</li>
<li>Cross-reactivity</li>
<li>receptor binding domain</li>
</ul>
<h3 data-number="9.95.2" id="summary-4"><span class="header-section-number">9.95.2</span> Summary</h3>
<p>Considering the relatively high identity of the receptor binding domain
(RBD) of the spike proteins from 2019-nCoV and SARS-CoV (73%), this
study aims to assess the cross-reactivity of several anti-SARS-CoV
monoclonal antibodies with 2019-nCoV. The results showed that the
SARS-CoV-specific antibody CR3022 can potently bind 2019-nCoV RBD.</p>
<h3 data-number="9.95.3" id="main-findings-91"><span class="header-section-number">9.95.3</span> Main Findings</h3>
<p>The structure of the 2019-nCoV spike RBD and its conformation in complex
with the receptor angiotensin-converting enzyme (ACE2) was modeled <em>in
silico</em> and compared with the SARS-CoV RBD structure. The models
predicted very similar RBD-ACE2 interactions for both viruses. The
binding capacity of representative SARS-CoV-RBD specific monoclonal
antibodies (m396, CR3014, and CR3022) to recombinant 2019-nCoV RBD was
then investigated by ELISA and their binding kinetics studied using
biolayer interferometry. The analysis showed that only CR3022 was able
to bind 2019-nCoV RBD with high affinity (KD of 6.3 nM), however it did
not interfere with ACE2 binding. Antibodies m396 and CR3014, which
target the ACE2 binding site of SARS-CoV failed to bind 2019-nCoV spike
protein.</p>
<h3 data-number="9.95.4" id="limitations-93"><span class="header-section-number">9.95.4</span> Limitations</h3>
<p>The 2019-nCoV RBD largely differ from the SARS-CoV at the C-terminus
residues, which drastically impact the cross-reactivity of antibodies
described for other B beta-coronaviruses, including SARS-CoV. This study
claims that CR3022 antibody could be a potential candidate for therapy.
However, none of the antibodies assayed in this work showed
cross-reactivity with the ACE2 binding site of 2019-nCoV, essential for
the replication of this virus. Furthermore, neutralization assays with
2019-nCoV virus or pseudovirus were not performed. Although the use of
neutralizing antibodies is an interesting approach, these results
suggest that it is critical the development of novel monoclonal
antibodies able to specifically bind 2019-nCoV spike protein.</p>
<h3 data-number="9.95.5" id="credit-91"><span class="header-section-number">9.95.5</span> Credit</h3>
<p><em>Review by D.L.O as part of a project by students, postdocs and faculty
at the Immunology Institute of the Icahn School of Medicine, Mount
Sinai.</em></p>
<p><strong>Title</strong>: Potent neutralization of 2019 novel coronavirus by
recombinant ACE2-Ig</p>
<p><strong>Keywords</strong>: ACE2, Ig-like protein, SARS-CoV neutralization</p>
<p><strong>Summary</strong>:</p>
<p>Angiotensin-converting enzyme 2 ACE2 is a negative regulator of the
renin-angiotensin system. In the lung tissues, ACE2 is expressed on lung
epithelial cells (AT2 cells) and has been identified as a receptor for
SARS-CoV-1 and SARS-CoV-2[1]⁠.</p>
<p>Administration of recombinant human ACE2 has been shown to protect mice
from severe acute lung injury induced by acid aspiration or sepsis and
lethal avian influenza H5N1[2,3]⁠⁠.</p>
<p>Human recombinant ACE has already been shown in animal models and humans
to have a fast clearance rate with a half-life of only a few
hours[4[]{#ADDIN_CSL_CITATION_%7B%22citationItems%2 .anchor}]⁠,
thereby limiting its therapeutic potential.</p>
<p>To address this protein stability limitation, the authors generated a
fusion protein that links the extracellular domain of human ACE2 to the
Fc domain of human IgG1. This fusion protein was shown to have a
prolonged half-life and neutralize viruses pseudotyped with the S
glycoprotein of both of SARS-CoV and 2019-nCoV in vitro, thus providing
a potential therapeutic for COVID-19.</p>
<p><strong>Critical analysis</strong>: ACE2-Ig fusion proteins are promising but their
ability to neutralize the virus and reduce viral load remains be tested
with intact Coronaviruses in vitro and in animals before being tested in
clinical trials.</p>
<p><strong>Implications for current epidemic</strong>: If neutralizing capacity can be
validated with 2019-nCov viruses, this novel drug target provides a
promising therapeutic strategy for the treatment of COVID-19 patients.
Nonetheless, the authors mention potential cardiovascular side-effects
stemming from the role of ACE2 in the renin-engiotensin system. These
will need to be examined prior to the initiation of a phase I clinical
trial.</p>
<p><strong>References</strong>:</p>
<p>1. Kuba K, Imai Y, Rao S, Jiang C, Penninger JM: <strong>Lessons from SARS:
Control of acute lung failure by the SARS receptor ACE2</strong>. <em>J Mol Med</em>
2006, <strong>84</strong>:814–820.</p>
<p>2. Imai Y, Kuba K, Rao S, Huan Y, Guo F, Guan B, Yang P, Sarao R, Wada
T, Leong-Poi H, et al.: <strong>Angiotensin-converting enzyme 2 protects from
severe acute lung failure</strong>. <em>Nature</em> 2005, <strong>436</strong>:112–116.</p>
<p>3. Zou Z, Yan Y, Shu Y, Gao R, Sun Y, Li X, Ju X, Liang Z, Liu Q, Zhao
Y, et al.: <strong>Angiotensin-converting enzyme 2 protects from lethal avian
influenza A H5N1 infections</strong>. <em>Nat Commun</em> 2014, <strong>5</strong>:3594.</p>
<p>4. Haschke M, Schuster M, Poglitsch M, Loibner H, Salzberg M, Bruggisser
M, Penninger J, Krähenbühl S: <strong>Pharmacokinetics and pharmacodynamics of
recombinant human angiotensin-converting enzyme 2 in healthy human
subjects</strong>. <em>Clin Pharmacokinet</em> 2013, <strong>52</strong>:783–792.</p>
<p><strong>Title</strong>: Potent neutralization of 2019 novel coronavirus by
recombinant ACE2-Ig</p>
<p><strong>Keywords</strong>: ACE2, Ig-like protein, SARS-CoV neutralization</p>
<p><strong>Summary</strong>:</p>
<p>Angiotensin-converting enzyme 2 ACE2 is a negative regulator of the
renin-angiotensin system. In the lung tissues, ACE2 is expressed on lung
epithelial cells (AT2 cells) and has been identified as a receptor for
SARS-CoV-1 and SARS-CoV-2[1]⁠.</p>
<p>Administration of recombinant human ACE2 has been shown to protect mice
from severe acute lung injury induced by acid aspiration or sepsis and
lethal avian influenza H5N1[2,3]⁠⁠.</p>
<p>Human recombinant ACE has already been shown in animal models and humans
to have a fast clearance rate with a half-life of only a few
hours[4[]{#ADDIN_CSL_CITATION_%7B%22citationItems%2 .anchor}]⁠,
thereby limiting its therapeutic potential.</p>
<p>To address this protein stability limitation, the authors generated a
fusion protein that links the extracellular domain of human ACE2 to the
Fc domain of human IgG1. This fusion protein was shown to have a
prolonged half-life and neutralize viruses pseudotyped with the S
glycoprotein of both of SARS-CoV and 2019-nCoV in vitro, thus providing
a potential therapeutic for COVID-19.</p>
<p><strong>Critical analysis</strong>: ACE2-Ig fusion proteins are promising but their
ability to neutralize the virus and reduce viral load remains be tested
with intact Coronaviruses in vitro and in animals before being tested in
clinical trials.</p>
<p><strong>Implications for current epidemic</strong>: If neutralizing capacity can be
validated with 2019-nCov viruses, this novel drug target provides a
promising therapeutic strategy for the treatment of COVID-19 patients.
Nonetheless, the authors mention potential cardiovascular side-effects
stemming from the role of ACE2 in the renin-engiotensin system. These
will need to be examined prior to the initiation of a phase I clinical
trial.</p>
<p><strong>References</strong>:</p>
<p>1. Kuba K, Imai Y, Rao S, Jiang C, Penninger JM: <strong>Lessons from SARS:
Control of acute lung failure by the SARS receptor ACE2</strong>. <em>J Mol Med</em>
2006, <strong>84</strong>:814–820.</p>
<p>2. Imai Y, Kuba K, Rao S, Huan Y, Guo F, Guan B, Yang P, Sarao R, Wada
T, Leong-Poi H, et al.: <strong>Angiotensin-converting enzyme 2 protects from
severe acute lung failure</strong>. <em>Nature</em> 2005, <strong>436</strong>:112–116.</p>
<p>3. Zou Z, Yan Y, Shu Y, Gao R, Sun Y, Li X, Ju X, Liang Z, Liu Q, Zhao
Y, et al.: <strong>Angiotensin-converting enzyme 2 protects from lethal avian
influenza A H5N1 infections</strong>. <em>Nat Commun</em> 2014, <strong>5</strong>:3594.</p>
<p>4. Haschke M, Schuster M, Poglitsch M, Loibner H, Salzberg M, Bruggisser
M, Penninger J, Krähenbühl S: <strong>Pharmacokinetics and pharmacodynamics of
recombinant human angiotensin-converting enzyme 2 in healthy human
subjects</strong>. <em>Clin Pharmacokinet</em> 2013, <strong>52</strong>:783–792.</p>
<h2 data-number="9.96" id="integrative-bioinformatics-analysis-provides-insight-into-the-molecular-mechanisms-of-2019-ncov-1"><span class="header-section-number">9.96</span> Integrative Bioinformatics Analysis Provides Insight into the Molecular Mechanisms of 2019-nCoV</h2>
<p>He et al. <em>medRxiv</em> <span class="citation" data-cites="lOQPScZt">[<a href="#ref-lOQPScZt" role="doc-biblioref">531</a>]</span></p>
<h3 data-number="9.96.1" id="keywords-95"><span class="header-section-number">9.96.1</span> Keywords</h3>
<ul>
<li>ACE2</li>
<li>lungs</li>
<li>smoking</li>
<li>COPD</li>
<li>asthma</li>
<li>SARS-Cov</li>
<li>IL-1</li>
<li>IL-10</li>
<li>IL-6</li>
<li>IL-8</li>
</ul>
<h3 data-number="9.96.2" id="main-findings-92"><span class="header-section-number">9.96.2</span> Main Findings</h3>
<p>The authors used bioinformatics tools to identify features
of ACE2 expression in the lungs of different patent groups: healthy,
smokers, patients with chronic airway disease (i.e., COPD) or asthma.
They used gene expression data publicly available from GEO that included
lung tissues, bronchoalveolar lavage, bronchial epithelial cells, small
airway epithelial cells, or SARS-Cov infected cells.</p>
<p>The authors describe no significant differences in ACE2 expression in
lung tissues of Healthy, COPD, and Asthma groups (p=0.85); or in BAL of
Healthy and COPD (p=0.48); or in epithelial brushings of Healthy and
Mild/Moderate/Severe Asthma (p=0.99). ACE2 was higher in the small
airway epithelium of long-term smokers vs non-smokers (p&lt;0.001).
Consistently, there was a trend of higher ACE2 expression in the
bronchial airway epithelial cells 24h post-acute smoking exposure
(p=0.073). Increasing ACE2 expression at 24h and 48h compared to 12h
post SARS-Cov infection (p=0.026; n=3 at each time point) was also
detected.</p>
<p>15 lung samples’ data from healthy participants were separated into high
and low ACE2 expression groups. “High” ACE2 expression was associated
with the following GO pathways: innate and adaptive immune responses, B
cell mediated immunity, cytokine secretion, and IL-1, IL-10, IL-6, IL-8
cytokines. The authors speculate that a high basal ACE2 expression will
increase susceptibility to SARS-CoV infection.</p>
<p>In 3 samples SARS-Cov infection was associated with IL-1, IL-10 and IL-6
cytokine production (GO pathways) at 24h. And later, at 48h, with T-cell
activation and T-cell cytokine production. It is unclear whether those
changes were statistically significant.</p>
<p>The authors describe a time course quantification of immune infiltrates
in epithelial cells infected with SARS-Cov infection. They state that in
healthy donors ACE2 expression did not correlate with the immune cell
infiltration. However, in SARS-Cov samples, at 48h they found that ACE2
correlated with neutrophils, NK-, Th17-, Th2-, Th1- cells, and DCs.
Again, while authors claim significance, the corresponding correlation
coefficients and p-values are not presented in the text or figures. In
addition, the source of the data for this analysis is not clear.</p>
<p>Using network analysis, proteins SRC, FN1, MAPK3, LYN, MBP, NLRC4, NLRP1
and PRKCD were found to be central (Hub proteins) in the regulating
network of cytokine secretion after coronavirus infection. Authors
conclude this indicates that these molecules were critically important
in ACE2-induced inflammatory response. Additionally, authors speculate
that the increased expression of ACE2 affected RPS3 and SRC, which were
the two hub genes involved in viral replication and inflammatory
response.</p>
<h3 data-number="9.96.3" id="limitations-94"><span class="header-section-number">9.96.3</span> Limitations</h3>
<p>The methods section is very limited and does not
describe any of the statistical analyses; and description of the
construction of the regulatory protein networks is also limited. For the
findings in Figures 2 authors claim significance, which is not supported
by p-values or coefficients. For the sample selection, would be useful
if sample sizes and some of the patients’ demographics (e.g. age) were
described.</p>
<p>For the analysis of high vs low ACE2 expression in healthy subjects, it
is not clear what was the cut off for ‘high’ expression and how it was
determined. Additionally, further laboratory studies are warranted to
confirm that high ACE2 gene expression would have high correlation with
the amount of ACE2 protein on cell surface. For the GO pathway analysis
significance was set at p&lt;0.05, but not adjusted for multiple
comparisons.</p>
<p>There were no samples with SARS-CoV-2 infection. While SARS-Cov and
SARS-CoV-2 both use ACE2 to enter the host cells, the analysis only
included data on SARS-Cov and any conclusions about SARS-CoV2 are
limited.</p>
<p>Upon checking GSE accession numbers of the datasets references, two
might not be cited correctly: GSE37758 (“A spergillus niger: Control
(fructose) vs. steam-exploded sugarcane induction (SEB)”" was used in
this paper as “lung tissue” data) and GSE14700 (“Steroid Pretreatment
of Organ Donors to Prevent Postischemic Renal Allograft Failure: A
Randomized, Controlled Trial” – was used as SARS-Cov infection data).</p>
<h3 data-number="9.96.4" id="credit-92"><span class="header-section-number">9.96.4</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.97" id="diarrhea-may-be-underestimated-a-missing-link-in-2019-novel-coronavirus-1"><span class="header-section-number">9.97</span> Diarrhea may be underestimated: a missing link in 2019 novel coronavirus</h2>
<p>Liang et al. <em>medRxiv</em> <span class="citation" data-cites="rL9IRhVy">[<a href="#ref-rL9IRhVy" role="doc-biblioref">532</a>]</span></p>
<h3 data-number="9.97.1" id="keywords-96"><span class="header-section-number">9.97.1</span> Keywords</h3>
<ul>
<li>SARS-CoV-2</li>
<li>diarrhea</li>
<li>ACE2</li>
<li>scRNA-seq</li>
</ul>
<h3 data-number="9.97.2" id="main-findings-93"><span class="header-section-number">9.97.2</span> Main Findings</h3>
<p>This study examined the incidence of diarrhea in
patients infected with SARS-CoV-2 across three recently published
cohorts and found that there are statistically significant differences
by Fisher’s exact test. They report that this could be due to subjective
diagnosis criterion for diarrhea or from patients first seeking medical
care from gastroenterologist. In order to minimize nosocomial infections
arising from unsuspected patients with diarrhea and gain comprehensive
understanding of transmission routes for this viral pathogen, they
compared the transcriptional levels of ACE2 of various human tissues
from NCBI public database as well as in small intestine tissue from
CD57BL/6 mice using single cell sequencing. They show that ACE2
expression is not only increased in the human small intestine, but
demonstrate a particular increase in mice enterocytes positioned on the
surface of the intestinal lining exposed to viral pathogens. Given that
ACE2 is the viral receptor for SARS-CoV-2 and also reported to regulate
diarrhea, their data suggests the small intestine as a potential
transmission route and diarrhea as a potentially underestimated symptom
in COVID19 patients that must be carefully monitored. Interestingly,
however, they show that ACE2 expression level is not elevated in human
lung tissue.</p>
<h3 data-number="9.97.3" id="limitations-95"><span class="header-section-number">9.97.3</span> Limitations</h3>
<p>Although this study demonstrates a
statistical difference in the incidence of diarrhea across three
separate COVID19 patient cohorts, their conclusions are limited by a
small sample size. Specifically, the p-value computed by Fisher’s exact
test is based on a single patient cohort of only six cases of which 33%
are reported to have diarrhea, while the remaining two larger cohorts
with 41 and 99 cases report 3% and 2% diarrhea incidence, respectively.
Despite showing significance, they would need to acquire larger sample
sizes and cohorts to minimize random variability and draw meaningful
conclusions. Furthermore, they do not address why ACE2 expression level
is not elevated in human lung tissue despite it being a major
established route of transmission for SARS-CoV-2. It could be helpful to
validate this result by looking at ACE2 expression in mouse lung tissue.
Finally, although this study is descriptive and shows elevated ACE2
expression in small intestinal epithelial cells, it does not establish a
mechanistic link to SARS-CoV-2 infection of the host. Overall, their
claim that infected patients exhibiting diarrhea pose an increased risk
to hospital staff needs to be further substantiated.</p>
<h3 data-number="9.97.4" id="significance-86"><span class="header-section-number">9.97.4</span> Significance</h3>
<p>This study provides a possible transmission route and a
potentially underappreciated clinical symptom for SARS-CoV-2 for better
clinical management and control of COVID19.</p>
<h3 data-number="9.97.5" id="credit-93"><span class="header-section-number">9.97.5</span> Credit</h3>
<p><em>Summary generated as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.98" id="specific-ace2-expression-in-cholangiocytes-may-cause-liver-damage-after-2019-ncov-infection-1"><span class="header-section-number">9.98</span> Specific ACE2 Expression in Cholangiocytes May Cause Liver Damage After 2019-nCoV Infection</h2>
<p>Chai et al. <em>bioRxiv</em> <span class="citation" data-cites="BHQvMYPG">[<a href="#ref-BHQvMYPG" role="doc-biblioref">533</a>]</span></p>
<h3 data-number="9.98.1" id="keywords-97"><span class="header-section-number">9.98.1</span> Keywords</h3>
<ul>
<li>ACE2</li>
<li>Cholangiocytes</li>
<li>COVID-associated Liver Damage</li>
</ul>
<h3 data-number="9.98.2" id="summary-5"><span class="header-section-number">9.98.2</span> Summary</h3>
<p>Using both publicly available scRNA-seq dataset of liver
samples from colorectal patients and scRNA-sequencing of four liver
samples from healthy volunteers, the authors show that ACE2 is
significantly enriched in the majority of cholangiocytes (59.7 %) but
not in hepatocytes (2.6%).</p>
<h3 data-number="9.98.3" id="main-findings-94"><span class="header-section-number">9.98.3</span> Main Findings</h3>
<p>Using bioinformatics approaches of
RNASeq analysis, this study reveals that ACE2 dominates in
cholangiocytes and is present at very low levels in hepatocytes.</p>
<h3 data-number="9.98.4" id="limitations-96"><span class="header-section-number">9.98.4</span> Limitations</h3>
<p>The
study does not provide mechanistic insights into how SARS-CoV-2 can
infect and replicate in cholangiocytes and the types of intrinsic
anti-viral responses induced by cholangiocytes when infected. In
addition, because the study relies on the assumption that SARS-CoV-2
infects cells only through ACE2, it cannot discount the possibility that
the virus can infect hepatocytes through mechanisms other than
ACE2-mediated entry. Furthermore, because the scRNA-seq analysis were
performed on healthy liver samples, one cannot draw any definitive
conclusions about gene expression states (including ACE2 expression in
liver cell types) in system-wide inflammatory contexts.</p>
<h3 data-number="9.98.5" id="significance-87"><span class="header-section-number">9.98.5</span> Significance</h3>
<p>This article with other studies on liver
damage in COVID patients suggests that liver damage observed in COVID
patients is more due to inflammatory cytokines than direct infection of
the liver. Even if cholangiocytes are infectable by SARS-CoV-2 (which
was demonstrated by human liver ductal organoid study
(<span class="citation" data-cites="OuDMtCRl">[<a href="#ref-OuDMtCRl" role="doc-biblioref">534</a>]</span>),
published clinical data show no significant increase in bile duct injury
related indexes (i.e. alkaline phosphatase, gamma-glutamyl
transpeptidase and total bilirubin). In sum, it underscores the
importance of future studies characterizing cellular responses of
extra-pulmonary organs in the context of COVID or at least in viral lung
infections..</p>
<h3 data-number="9.98.6" id="credit-94"><span class="header-section-number">9.98.6</span> Credit</h3>
<p><em>Summary generated by Chang Moon as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.99" id="ace2-expression-by-colonic-epithelial-cells-is-associated-with-viral-infection-immunity-and-energy-metabolism-1"><span class="header-section-number">9.99</span> ACE2 expression by colonic epithelial cells is associated with viral infection, immunity and energy metabolism</h2>
<p>Wang et al. <em>medRxiv</em>. <span class="citation" data-cites="ViWCb8MJ">[<a href="#ref-ViWCb8MJ" role="doc-biblioref">535</a>]</span></p>
<h3 data-number="9.99.1" id="keywords-98"><span class="header-section-number">9.99.1</span> Keywords</h3>
<ul>
<li>single cell RNA seq</li>
<li>ACE2 expression</li>
<li>human colonic biopsy</li>
</ul>
<h3 data-number="9.99.2" id="main-findings-95"><span class="header-section-number">9.99.2</span> Main Findings</h3>
<p>Colonic enterocytes primarily express ACE2. Cellular pathways associated with ACE2 expression include innate immune signaling, HLA up regulation, energy metabolism and apoptotic signaling.</p>
<h3 data-number="9.99.3" id="limitations-97"><span class="header-section-number">9.99.3</span> Limitations</h3>
<p>This is a study of colonic biopsies taken from 17 children with and without IBD and analyzed using scRNAseq to look at ACE2 expression and identify gene families correlated with ACE2 expression. The authors find ACE2 expression to be primarily in colonocytes. It is not clear why both healthy and IBD patients were combined for the analysis. Biopsies were all of children so extrapolation to adults is limited. The majority of genes found to be negatively correlated with ACE2 expression include immunoglobulin genes (IGs). IG expression will almost certainly be low in colonocytes irrespective of ACE2 expression.</p>
<h3 data-number="9.99.4" id="significance-88"><span class="header-section-number">9.99.4</span> Significance</h3>
<p>This study performs a retrospective analysis of ACE2 expression using an RNAseq dataset from intestinal biopsies of children with and without IBD. The implications for the CoV-19 epidemic are modest, but do provide support that ACE2 expression is specific to colonocytes in the intestines. The ontological pathway analysis provides some limited insights into gene expression associated with ACE2.</p>
<h3 data-number="9.99.5" id="credit-95"><span class="header-section-number">9.99.5</span> Credit</h3>
<p><em>Summary generated as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.100" id="the-pathogenicity-of-2019-novel-coronavirus-in-hace2-transgenic-mice-1"><span class="header-section-number">9.100</span> The Pathogenicity of 2019 Novel Coronavirus in hACE2 Transgenic Mice</h2>
<p>Bao et al. <em>bioRxiv</em> <span class="citation" data-cites="MgLBA5m">[<a href="#ref-MgLBA5m" role="doc-biblioref">536</a>]</span></p>
<h3 data-number="9.100.1" id="keywords-99"><span class="header-section-number">9.100.1</span> Keywords</h3>
<ul>
<li>Covid-19 mouse model</li>
<li>hACE2 mice</li>
<li>2019-nCoV model</li>
<li>ACE2</li>
<li>2019-nCoV </li>
</ul>
<h3 data-number="9.100.2" id="main-findings-96"><span class="header-section-number">9.100.2</span> Main Findings</h3>
<p>Using a transgenic human Angiotensin-converting enzyme 2
(hACE2) mouse that has previously been shown susceptible to infection by
SARS-CoV, Bao et al. create a model of pandemic 2019-nCoV strain
coronavirus. The model includes interstitial hyperplasia in lung tissue,
moderate inflammation in bronchioles and blood vessels, and histology
consistent with viral pneumonia at 3 days post infection. Wildtype did
not experience these symptoms. In addition, viral antigen and hACE2
receptor were found to co-localize the lung by immunofluorescence 3-10
days post infection only in the hACE2 infected mice.</p>
<h3 data-number="9.100.3" id="limitations-98"><span class="header-section-number">9.100.3</span> Limitations</h3>
<p>The characterization of the infection remains incomplete,
as well as lacking characterization of the immune response other than
the presence of a single antiviral antibody. Though they claim to
fulfill Koch’s postulates, they only isolate the virus and re-infect
Vero cells, rather than naive mice.</p>
<h3 data-number="9.100.4" id="significance-89"><span class="header-section-number">9.100.4</span> Significance</h3>
<p>This paper establishes a murine model for 2019-nCoV
infection with symptoms consistent with viral pneumonia. Though not
fully characterized, this model allows in vivo analysis of viral entry
and pathology that is important for the development of vaccines and
antiviral therapeutics.</p>
<h3 data-number="9.100.5" id="credit-96"><span class="header-section-number">9.100.5</span> Credit</h3>
<p><em>Review by Dan Fu Ruan, Evan Cody and Venu Pothula as part of a project by students, postdocs and faculty
at the Immunology Institute of the Icahn School of Medicine, Mount
Sinai.</em></p>
<h2 data-number="9.101" id="caution-on-kidney-dysfunctions-of-2019-ncov-patients-1"><span class="header-section-number">9.101</span> Caution on Kidney Dysfunctions of 2019-nCoV Patients</h2>
<p>Li et al. <em>medRxiv</em>. <span class="citation" data-cites="1Acv5UMSt">[<a href="#ref-1Acv5UMSt" role="doc-biblioref">537</a>]</span></p>
<h3 data-number="9.101.1" id="keywords-100"><span class="header-section-number">9.101.1</span> Keywords</h3>
<p>CoVID-19, 2019-nCoV, SARS-CoV-2, kidney, clinical, creatinine,
proteinuria, albuminuria, CT</p>
<h3 data-number="9.101.2" id="main-findings-97"><span class="header-section-number">9.101.2</span> Main Findings</h3>
<ul>
<li><p>Retrospective study of 59 patients assayed key function indicators
of the kidney–including urine protein, blood urea nitrogen (BUN),
plasma creatinine (Cre), and renal CT scan data.</p></li>
<li><p>Found that 34% of patients developed massive albuminuria on the
first day of admission, and 63% developed proteinuria during their
stay in hospital; and 19% of patients had high plasma creatinine,
especially the terminal cases.</p></li>
<li><p>CT analyses of 27 patients showed all patients to have abnormal
kidney damage; indicate that inflammation and edema of the renal
parenchyma very common.</p></li>
</ul>
<h3 data-number="9.101.3" id="limitations-99"><span class="header-section-number">9.101.3</span> Limitations</h3>
<ul>
<li><p>No analysis of immunity-dependent damage and cytokines in
blood/plasma/urine. Will be worth correlating disease progression
with cytokine production, immune activity and kidney function.</p></li>
<li><p>Extrapolating to earlier SARS-CoV studies provides the only
rationale for viral-damage in kidney and resultant pathologic immune
response <em>(understandable for this clinical study)</em>.</p></li>
</ul>
<h3 data-number="9.101.4" id="significance-90"><span class="header-section-number">9.101.4</span> Significance</h3>
<ul>
<li><p>Multiple lines of evidence along this study’s finding point to the
idea that renal impairment/injury is a key risk factor in 2019-nCoV
patients similar to what has been reported for SARS-CoV<span class="citation" data-cites="DMaiZBE1">[<a href="#ref-DMaiZBE1" role="doc-biblioref">538</a>]</span>; this
may be one of the major causes of virally-induced damage and
contribute to multiorgan failure.</p></li>
<li><p>ACE2 expression in kidney proximal tubule epithelia and bladder
epithelia <span class="citation" data-cites="16Xj27QYj">[<a href="#ref-16Xj27QYj" role="doc-biblioref">539</a>]</span> support
these clinical findings.</p></li>
<li><p>Study argues for closely monitoring kidney function, and applying
potential interventions including continuous renal replacement
therapies (CRRT) for protecting kidney functions as early as
possible, particularly for those with rising plasma creatinine.</p></li>
</ul>
<h3 data-number="9.101.5" id="credit-97"><span class="header-section-number">9.101.5</span> Credit</h3>
<p><em>Review by Samarth Hegde as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.102" id="profiling-the-immune-vulnerability-landscape-of-the-2019-novel-coronavirus-1"><span class="header-section-number">9.102</span> Profiling the immune vulnerability landscape of the 2019 Novel Coronavirus</h2>
<p>Zhu et al. <em>bioRxiv</em> <span class="citation" data-cites="QhE5mVCc">[<a href="#ref-QhE5mVCc" role="doc-biblioref">540</a>]</span></p>
<h3 data-number="9.102.1" id="keywords-101"><span class="header-section-number">9.102.1</span> Keywords</h3>
<ul>
<li>epitope prediction</li>
<li>vaccine development.</li>
</ul>
<h3 data-number="9.102.2" id="main-findings-98"><span class="header-section-number">9.102.2</span> Main Findings</h3>
<p>This study harnesses bioinformatic profiling to predict the potential of
COV2 viral proteins to be presented on MHC I and II and to form linear
B-cell epitopes. These estimates suggest a T-cell antigenic profile
distinct from SARS-CoV or MERS-CoV, identify focused regions of the
virus with a high density of predicted epitopes, and provide preliminary
evidence for adaptive immune pressure in the genetic evolution of the
virus.</p>
<h3 data-number="9.102.3" id="limitations-100"><span class="header-section-number">9.102.3</span> Limitations</h3>
<p>While the study performs a comprehensive analysis of potential epitopes
within the virus genome, the analysis relies solely on bioinformatic
prediction to examine MHC binding affinity and B-cell epitope potential
and does not capture the immunogenicity or recognition of
these epitopes. Future experimental validation in data from patients
infected with SARS-CoV-2 will be important to validate and refine these
findings. Thus some of the potential conclusions stated, including viral
evolution toward lower immunogenicity or a dominant role for CD4+
T-cells rather than CD8+ T-cells in viral clearance, require further
valiadtion.</p>
<h3 data-number="9.102.4" id="significance-91"><span class="header-section-number">9.102.4</span> Significance</h3>
<p>These findings may help direct peptide vaccine design toward relevant
epitopes and provide intriguing evidence of viral evolution in response
to immune pressure.</p>
<h3 data-number="9.102.5" id="credit-98"><span class="header-section-number">9.102.5</span> Credit</h3>
<p><em>Summary generated as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.103" id="single-cell-analysis-of-ace2-expression-in-human-kidneys-and-bladders-reveals-a-potential-route-of-2019-ncov-infection-1"><span class="header-section-number">9.103</span> Single-cell Analysis of ACE2 Expression in Human Kidneys and Bladders Reveals a Potential Route of 2019-nCoV Infection</h2>
<p>Lin et al. <em>bioRxiv</em> <span class="citation" data-cites="16Xj27QYj">[<a href="#ref-16Xj27QYj" role="doc-biblioref">539</a>]</span></p>
<h3 data-number="9.103.1" id="keywords-102"><span class="header-section-number">9.103.1</span> Keywords</h3>
<ul>
<li>CoVID-19</li>
<li>2019-nCoV</li>
<li>SARS-CoV-2</li>
<li>ACE2</li>
<li>scRNAseq</li>
<li>kidney</li>
<li>bladder</li>
<li>public dataset</li>
</ul>
<h3 data-number="9.103.2" id="main-findings-99"><span class="header-section-number">9.103.2</span> Main Findings</h3>
<ul>
<li><p>To investigate the possible cause of kidney damage in 2019-nCoV
patients, authors used published kidney and bladder cell atlas data
(GSE131685, GSE108097; 3 healthy donors each) as well as an
unpublished kidney single-cell RNA-Seq data (in-house from 2
transplant donors) to evaluate ACE2 gene expressions in all cell
types of healthy kidneys and bladders.</p></li>
<li><p>They find enriched expression of ACE2 transcript in all subtypes of
proximal tubule cells of kidney, with 5%-15% of both straight and
convoluted proximal tubule cells expressing ACE2.</p></li>
<li><p>They also find detectable levels of ACE2 in bladder epithelial
cells, noting expression from around 1.5% of cells in the outer
layer umbrella cells of the bladder epithelium and decreasing in the
basal cells.</p></li>
<li><p>Importantly endothelial or immune cells in kidney/bladder do not
express ACE2.</p></li>
</ul>
<h3 data-number="9.103.3" id="limitations-101"><span class="header-section-number">9.103.3</span> Limitations</h3>
<ul>
<li><p>This study primarily characterizes ACE2 expression (amongst other
genes) from a small healthy-donor dataset, and will benefit from
supporting data in (expired) patient samples to show functional
viral damage. ACE2 transcript does not necessarily translate to
viral permissiveness in kidney/bladder epithelia or cytokine
release.</p></li>
<li><p>This study focuses on only healthy tissue; it will be useful to
analyze kidney/bladder epithelial ACE2 expression under inflammatory
conditions or in patients with underlying kidney conditions.</p></li>
<li><p>Given what is known about protease TMPRSS2 expression during
SARS-CoV-2 infection, ACE2+TMPRSS2+ double-positive cell
identification would be useful in these datasets.</p></li>
</ul>
<h3 data-number="9.103.4" id="significance-92"><span class="header-section-number">9.103.4</span> Significance</h3>
<ul>
<li><p>ACE2 protein is spatially restricted to brush border of proximal
tubules and in bladder umbrella cells <span class="citation" data-cites="1FE1YGsaV">[<a href="#ref-1FE1YGsaV" role="doc-biblioref">541</a>]</span>, such cells in direct
contact with viral particles are likely to be highly sensitive to
viral-induced damage.</p></li>
<li><p>SARS-CoV and MERS-CoV have been shown to be detected in urine of
patients and associate with higher mortality <span class="citation" data-cites="DMaiZBE1 iObQU2el">[<a href="#ref-DMaiZBE1" role="doc-biblioref">538</a>,<a href="#ref-iObQU2el" role="doc-biblioref">542</a>]</span>, thus worth
understanding kidney damage and resultant immune response in
SARS-CoV-2 as well.</p></li>
<li><p>This study argues for a potential mode of viral infectivity and
resultant inflammatory responses in these tissue in addition to
reported infectivity in the lung and digestive system, which is
supported by clinical data showing acute and early kidney
complications in 2019-nCoV patients <span class="citation" data-cites="1Acv5UMSt">[<a href="#ref-1Acv5UMSt" role="doc-biblioref">537</a>]</span>.</p></li>
<li><p>Clinically, thus very important to track urinary CoVID-19 shedding
as well as study acute kidney injury-related co-morbidities.</p></li>
</ul>
<h3 data-number="9.103.5" id="credit-99"><span class="header-section-number">9.103.5</span> Credit</h3>
<p><em>Review by Samarth Hegde as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.104" id="neutrophil-to-lymphocyte-ratio-predicts-severe-illness-patients-with-2019-novel-coronavirus-in-the-early-stage-1"><span class="header-section-number">9.104</span> Neutrophil-to-Lymphocyte Ratio Predicts Severe Illness Patients with 2019 Novel Coronavirus in the Early Stage</h2>
<p>Liu et al. <em>medRxiv</em> <span class="citation" data-cites="gK5e7Yhl">[<a href="#ref-gK5e7Yhl" role="doc-biblioref">543</a>]</span></p>
<h3 data-number="9.104.1" id="keywords-103"><span class="header-section-number">9.104.1</span> Keywords</h3>
<ul>
<li>severe disease</li>
<li>pneumonia</li>
<li>lymphocytes</li>
<li>neutrophils</li>
</ul>
<h3 data-number="9.104.2" id="main-findings-100"><span class="header-section-number">9.104.2</span> Main Findings</h3>
<p>This study aimed to find prognostic biomarkers of COVID-19
pneumonia severity. Sixty-one (61) patients with COVID-19 treated in
January at a hospital in Beijing, China were included. On average,
patients were seen within 5 days from illness onset. Samples were
collected on admission; and then patients were monitored for the
development of severe illness with a median follow-up of 10 days].</p>
<p>Patients were grouped as “mild” (N=44) or “moderate/severe” (N=17)
according to symptoms on admission and compared for different
clinical/laboratory features. “Moderate/severe” patients were
significantly older (median of 56 years old, compared to 41 years old).
Whereas comorbidies rates were largely similar between the groups,
except for hypertension, which was more frequent in the severe group (p=
0.056). ‘Severe’ patients had higher counts of neutrophils, and serum
glucose levels; but lower lymphocyte counts, sodium and serum chlorine
levels. The ratio of neutrophils to lymphocytes (NLR) was also higher
for the ‘severe’ group. ‘Severe’ patients had a higher rate of bacterial
infections (and antibiotic treatment) and received more intensive
respiratory support and treatment.</p>
<p>26 clinical/laboratory variables were used to select NLR and age as the
best predictors of the severe disease. Predictive cutoffs for a severe
illness as NLR ≥ 3.13 or age ≥ 50 years.</p>
<h3 data-number="9.104.3" id="limitations-102"><span class="header-section-number">9.104.3</span> Limitations</h3>
<p>Identification of early biomarkers is important for
making clinical decisions, but large sample size and validation cohorts
are necessary to confirm findings. It is worth noting that patients
classified as “mild” showed pneumonia by imaging and fever, and in
accordance with current classifications this would be consistent with
“moderate” cases. Hence it would be more appropriate to refer to the
groups as “moderate” vs “severe/critical”. Furthermore, there are
several limitations that could impact the interpretation of the results:
e.g. classification of patients was based on symptoms presented on
admission and not based on disease progression, small sample size,
especially the number of ‘severe’ cases (with no deaths among these
patients). Given the small sample size, the proposed NLR and age cut
offs might not hold for a slightly different set of patients. For
example, in a study of &gt;400 patients, ‘non-severe’ and ‘severe’ NLR
were 3.2 and 5.5, respectively <span class="citation" data-cites="6Y81C4P3">[<a href="#ref-6Y81C4P3" role="doc-biblioref">544</a>]</span>.</p>
<h3 data-number="9.104.4" id="credit-100"><span class="header-section-number">9.104.4</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.105" id="characteristics-of-lymphocyte-subsets-and-cytokines-in-peripheral-blood-of-123-hospitalized-patients-with-2019-novel-coronavirus-pneumonia-ncp-1"><span class="header-section-number">9.105</span> Characteristics of lymphocyte subsets and cytokines in peripheral blood of 123 hospitalized patients with 2019 novel coronavirus pneumonia (NCP)</h2>
<p>Wan et. al. <em>medRxiv</em> <span class="citation" data-cites="P4assQHj">[<a href="#ref-P4assQHj" role="doc-biblioref">545</a>]</span></p>
<h3 data-number="9.105.1" id="keywords-104"><span class="header-section-number">9.105.1</span> Keywords</h3>
<ul>
<li>Cytokines</li>
<li>lymphocyte subsets</li>
<li>CD8 + T</li>
<li>B cells</li>
<li>NK cells,</li>
<li>PBMCs</li>
<li>IL-6</li>
<li>IL-10</li>
</ul>
<h3 data-number="9.105.2" id="main-findings-101"><span class="header-section-number">9.105.2</span> Main Findings</h3>
<p>The authors analyzed lymphocyte subsets and cytokines of 102 patients
with mild disease and 21 with severe disease. CD8+T cells and CD4+T
cells were significantly reduced in both cohort. particularly in severe
patients. The cytokines IL6 and IL10 were significantly elevated in
severe patients as compared to mild. No significant differences were
observed in frequency of B cells and NK cells.</p>
<p>The authors argue that the measurement of T cell frequencies and
cytokine levels of IL6 and IL10 can be used to predict progression of
disease from Mild to severe Cov-2 infection.</p>
<h3 data-number="9.105.3" id="limitations-103"><span class="header-section-number">9.105.3</span> Limitations</h3>
<p>The study demonstrates in a limited cohort
similar associations to several other reported studies. The authors
didn’t compare the changes in lymphocyte and cytokine with healthy
individual (Covid-19 Negative) rather used an internal standard value.
The recently preprint in LANCET shows The degree of lymphopenia and a
pro-inflammatory cytokine storm is higher in severe COVID-19 patients
than in mild cases, and is associated with the disease severity <span class="citation" data-cites="zskgSnle">[<a href="#ref-zskgSnle" role="doc-biblioref">546</a>]</span>.</p>
<h3 data-number="9.105.4" id="significance-93"><span class="header-section-number">9.105.4</span> Significance</h3>
<p>This translational data identifies key cytokines and lymphopenia
associated with disease severity although mechanism and key cellular
players are still unknown. Higher level IL-6 production in severe
patient suggests potential role of Tocilizumab (anti-IL6R) biologic
although clinical trial will be necessary.</p>
<h3 data-number="9.105.5" id="credit-101"><span class="header-section-number">9.105.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.106" id="epidemiological-and-clinical-characteristics-of-17-hospitalized-patients-with-2019-novel-coronavirus-infections-outside-wuhan-china-1"><span class="header-section-number">9.106</span> Epidemiological and Clinical Characteristics of 17 Hospitalized Patients with 2019 Novel Coronavirus Infections Outside Wuhan, China</h2>
<p>Li et al. <em>medRxiv</em> <span class="citation" data-cites="TXlNzmXd">[<a href="#ref-TXlNzmXd" role="doc-biblioref">547</a>]</span></p>
<h3 data-number="9.106.1" id="keywords-105"><span class="header-section-number">9.106.1</span> Keywords</h3>
<ul>
<li>epidemiology</li>
<li>clinical characteristics</li>
</ul>
<h3 data-number="9.106.2" id="major-findings-1"><span class="header-section-number">9.106.2</span> Major Findings</h3>
<p>These authors looked at 17 hospitalized patients with COVID-19
confirmed by RT-PCR in Dazhou, Sichuan. Patients were admitted between
January 22 and February 10 and the final data were collected on February
11. Of the 17 patients, 12 remained hospitalized while 5 were discharged
after meeting national standards. The authors observed no differences
based on the sex of the patients but found that the discharged patients
were younger in age (p = 0.026) and had higher lymphocyte counts (p =
0.005) and monocyte counts (p = 0.019) upon admission.</p>
<h3 data-number="9.106.3" id="limitations-104"><span class="header-section-number">9.106.3</span> Limitations</h3>
<p>This study is limited in the sample size of the study and the last
data collection point was only one day after some of the patients were
admitted.</p>
<h3 data-number="9.106.4" id="significance-94"><span class="header-section-number">9.106.4</span> Significance</h3>
<p>These findings have been somewhat supported by subsequent studies
that show that older age and an immunocompromised state are more likely
to result in a more severe clinical course with COVID-19. However, other
studies have been published that report on larger numbers of cases.</p>
<h3 data-number="9.106.5" id="credit-102"><span class="header-section-number">9.106.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.107" id="ace2-expression-in-kidney-and-testis-may-cause-kidney-and-testis-damage-after-2019-ncov-infection-1"><span class="header-section-number">9.107</span> ACE2 Expression in Kidney and Testis May Cause Kidney and Testis Damage After 2019-nCoV Infection</h2>
<p><span class="citation" data-cites="15K5Ovntj">[<a href="#ref-15K5Ovntj" role="doc-biblioref">548</a>]</span></p>
<h3 data-number="9.107.1" id="keywords-106"><span class="header-section-number">9.107.1</span> Keywords</h3>
<ul>
<li>CoVID-19</li>
<li>2019-nCoV</li>
<li>SARS-CoV-2</li>
<li>kidney</li>
<li>testis</li>
<li>ACE2</li>
<li>scRNAseq</li>
</ul>
<h3 data-number="9.107.2" id="main-findings-102"><span class="header-section-number">9.107.2</span> Main Findings</h3>
<ul>
<li><p>Study used online datasets (scRNAseq GSE131685, scRNAseq GSE107585,
Human Protein Atlas, GTEx portal, CCLE) to analyze ACE2 expression
in different human organs.</p></li>
<li><p>Study re-analyzed three clinical datasets (n=6, n=99, and n=41) to
show 3~10% of 2019-nCoV patients present with abnormal renal
function.</p></li>
<li><p>results indicate ACE2 highly expressed in renal tubular cells,
Leydig cells and seminiferous ductal cells of testis.</p></li>
</ul>
<h3 data-number="9.107.3" id="limitations-105"><span class="header-section-number">9.107.3</span> Limitations</h3>
<ul>
<li><p>Very preliminary transcript/protein dataset analysis in healthy
cohorts; does not necessarily translate to actual viral tropism and
permissiveness.</p></li>
<li><p>Clinically, would be important to determine with larger longitudinal
dataset if SARS-CoV-2 infection changes sperm quality or testicular
inflammation.</p></li>
<li><p>Similarly, would be important to determine if simultaneous HBV or
syphilis infection and orchitis impacts SARS-CoV-2 severity.</p></li>
<li><p>Examination and follow-up of renal function and viral orchitis/sperm
quality of CoVID-19 patients not done in this preliminary study.</p></li>
</ul>
<h3 data-number="9.107.4" id="significance-95"><span class="header-section-number">9.107.4</span> Significance</h3>
<ul>
<li><p>Kidney ACE2 result supports other concurrent sequencing studies
<span class="citation" data-cites="16Xj27QYj">[<a href="#ref-16Xj27QYj" role="doc-biblioref">539</a>]</span> and clinical reports
of abnormal renal function or even kidney damage in patients
infected with 2019-nCoV
<span class="citation" data-cites="1Acv5UMSt">[<a href="#ref-1Acv5UMSt" role="doc-biblioref">537</a>]</span>.</p></li>
<li><p>High ACE2 expression in testis suggests potential tropism of the
virus to testicular tissues and indicates potential risks for male
fertility. Viral orchitis reported for SARS-CoV previously [1],
but no clear evidence so far of infertility in SARS, MERS or
CoVID-19 patients.</p></li>
</ul>
<h3 data-number="9.107.5" id="credit-103"><span class="header-section-number">9.107.5</span> Credit</h3>
<p><em>Review by Samarth Hegde as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.108" id="aberrant-pathogenic-gm-csf-t-cells-and-inflammatory-cd14cd16-monocytes-in-severe-pulmonary-syndrome-patients-of-a-new-coronavirus-1"><span class="header-section-number">9.108</span> Aberrant pathogenic GM-CSF+ T cells and inflammatory CD14+CD16+ monocytes in severe pulmonary syndrome patients of a new coronavirus</h2>
<p><span class="citation" data-cites="Bgo7Cmj3">[<a href="#ref-Bgo7Cmj3" role="doc-biblioref">549</a>]</span></p>
<h3 data-number="9.108.1" id="keywords-107"><span class="header-section-number">9.108.1</span> Keywords</h3>
<ul>
<li>immunopathology</li>
<li>Th1</li>
<li>inflammatory monocytes</li>
<li>GM-CSF</li>
<li>IFN-γ</li>
<li>IL-6</li>
</ul>
<h3 data-number="9.108.2" id="main-findings-103"><span class="header-section-number">9.108.2</span> Main Findings</h3>
<p>The authors of this study sought to characterize the immune
mechanism causing severe pulmonary disease and mortality in 2019-nCoV
(COVID-19) patients. Peripheral blood was collected from hospitalized
ICU (n=12) and non-ICU (n=21) patients with confirmed 2019-nCoV and from
healthy controls (n=10) in The First Affiliated Hospital of University
of Science and Technology China (Hefei, Anhui). Immune analysis was
conducted by flow cytometry. 2019-nCoV patients had decreased
lymphocyte, monocyte, and CD4 T cell counts compared to healthy
controls. ICU patients had fewer lymphocytes than non-ICU patients. CD4
T cells of 2019-nCoV patients expressed higher levels of activation
markers (OX40, CD69, CD38, CD44) and exhaustion markers (PD-1 and Tim3)
than those of healthy controls. CD4 cells of ICU patients expressed
significantly higher levels of OX40, PD-1, and Tim3 than those of
non-ICU patients. 2019-nCoV patients had higher percentages of CD4 T
cells co-expressing GM-CSF and IL-6 compared to healthy controls, while
ICU patients had a markedly higher percentage of GM-CSF+ IFN-γ+ CD4 T
cells than non-ICU patients. The CD4 T cells of nCoV patients and
healthy controls showed no differences in TNF-α secretion.</p>
<p>The CD8 T cells of 2019-nCoV patients also showed higher expression of
activation markers CD69, CD38, and CD44, as well as exhaustion markers
PD-1 and Tim3, compared to healthy controls. CD8 T cells of ICU patients
expressed higher levels of GM-CSF than those of non-ICU patients and
healthy controls. No IL-6 or TNF-α was found in the CD8 T cells of any
group. There were no differences in numbers of NK cells or B cells in
2019-nCoV patients and healthy controls, nor was there any GM-CSF or
IL-6 secretion from these cells in either group.</p>
<p>Percentages of CD14+ CD16+ GM-CSF+ and CD14+ CD16+ IL-6+ inflammatory
monocytes were significantly increased in nCoV patients compared to
healthy controls; in particular, patients in the ICU had greater
percentages of CD14+ CD16+ IL-6+ monocytes than non-ICU patients. The
authors suggest that in 2019-nCoV patients, pathogenic Th1 cells produce
GM-CSF, recruiting CD14+ CD16+ inflammatory monocytes that secrete high
levels of IL-6. These may enter pulmonary circulation and damage lung
tissue while initiating the cytokine storm that causes mortality in
severe cases. This is consistent with the cytokine storm seen in similar
coronaviruses, as IL-6, IFN-γ, and GM-CSF are key inflammatory mediators
seen in patients with SARS-CoV-1 and MERS-CoV.</p>
<h3 data-number="9.108.3" id="limitations-106"><span class="header-section-number">9.108.3</span> Limitations</h3>
<p>Though the results of this study open
questions for further investigation, this is an early study on a small
cohort of patients, and as such there are a number of limitations. The
study included only 12 ICU patients and 21 non-ICU patients, and ideally
would be repeated with a much larger patient cohort. Though the authors
make claims about differences in lymphocyte and monocyte counts between
patients and healthy controls, they did not report baseline laboratory
findings for the control group. Additionally, severity of disease was
classified based on whether or not patients were in the ICU. It would be
interesting to contextualize the authors’ immunological findings with
more specific metrics of disease severity or time course. Noting
mortality, time from disease onset, pre-existing conditions, or severity
of lung pathology in post-mortem tissue samples would paint a fuller
picture of how to assess risk level and the relationship between
severity of disease and immunopathology. Another limitation is the
selection of cytokines and immune markers for analysis, as the selection
criteria were based on the cell subsets and cytokine storm typically
seen in SARS-CoV-1 and MERS-CoV patients. Unbiased cytokine screens and
immune profiling may reveal novel therapeutic targets that were not
included in this study.</p>
<h3 data-number="9.108.4" id="significance-96"><span class="header-section-number">9.108.4</span> Significance</h3>
<p>This study identifies potential therapeutic targets
that could prevent acute respiratory disease syndrome (ARDS) and
mortality in patients most severely affected by COVID-19. The authors
propose testing monoclonal antibodies against IL6-R or GM-CSF to block
recruitment of inflammatory monocytes and the subsequent cytokine storm
in these patients.</p>
<h3 data-number="9.108.5" id="credit-104"><span class="header-section-number">9.108.5</span> Credit</h3>
<p><em>Review by Gabrielle Lubitz as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.109" id="clinical-characteristics-of-2019-novel-infected-coronavirus-pneumoniaa-systemic-review-and-meta-analysis-1"><span class="header-section-number">9.109</span> Clinical Characteristics of 2019 Novel Infected Coronavirus Pneumonia：A Systemic Review and Meta-analysis</h2>
<p>Qian et al. <em>medRxiv</em>.
<span class="citation" data-cites="4Z1pEioX">[<a href="#ref-4Z1pEioX" role="doc-biblioref">550</a>]</span></p>
<h3 data-number="9.109.1" id="keywords-108"><span class="header-section-number">9.109.1</span> Keywords</h3>
<ul>
<li>White Blood Cells</li>
<li>Lymphocytes</li>
<li>Neutrophils</li>
</ul>
<h3 data-number="9.109.2" id="main-findings-104"><span class="header-section-number">9.109.2</span> Main Findings</h3>
<p>The authors performed a meta analysis of literature on clinical,
laboratory and radiologic characteristics of patients presenting with
pneumonia related to SARSCoV2 infection, published up to Feb 6 2020.
They found that symptoms that were mostly consistent among studies were
sore throat, headache, diarrhea and rhinorrhea. Fever, cough, malaise
and muscle pain were highly variable across studies. Leukopenia (mostly
lymphocytopenia) and increased white blood cells were highly variable
across studies. They identified three most common patterns seen on CT
scan, but there was high variability across studies. Consistently across
the studies examined, the authors found that about 75% of patients need
supplemental oxygen therapy, about 23% mechanical ventilation and about
5% extracorporeal membrane oxygenation (ECMO). The authors calculated a
staggering pooled mortality incidence of 78% for these patients.</p>
<h3 data-number="9.109.3" id="limitations-107"><span class="header-section-number">9.109.3</span> Limitations</h3>
<p>The authors mention that the total number of studies included in this
meta analysis is nine, however they also mentioned that only three
studies reported individual patient data. It is overall unclear how many
patients in total were included in their analysis. This is mostly
relevant as they reported an incredibly high mortality (78%) and mention
an absolute number of deaths of 26 cases overall. It is not clear from
their report how the mortality rate was calculated.</p>
<p>The data is based on reports from China and mostly from the Wuhan area,
which somewhat limits the overall generalizability and applicability of
these results.</p>
<h3 data-number="9.109.4" id="significance-97"><span class="header-section-number">9.109.4</span> Significance</h3>
<p>This meta analysis offers some important data for clinicians to refer to
when dealing with patients with COVID-19 and specifically with
pneumonia. It is very helpful to set expectations about the course of
the disease.</p>
<h3 data-number="9.109.5" id="credit-105"><span class="header-section-number">9.109.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.110" id="longitudinal-characteristics-of-lymphocyte-responses-and-cytokine-profiles-in-the-peripheral-blood-of-sars-cov-2-infected-patients-1"><span class="header-section-number">9.110</span> Longitudinal characteristics of lymphocyte responses and cytokine profiles in the peripheral blood of SARS-CoV-2 infected patients</h2>
<p>Liu et al. <em>medRxiv</em> <span class="citation" data-cites="zTGzuiTI">[<a href="#ref-zTGzuiTI" role="doc-biblioref">551</a>]</span></p>
<h3 data-number="9.110.1" id="keywords-109"><span class="header-section-number">9.110.1</span> Keywords</h3>
<ul>
<li>Lymphopenia</li>
<li>Neutrophil to CD8 T cell ratio (N8R)</li>
<li>inflammatory cytokines</li>
</ul>
<h3 data-number="9.110.2" id="main-findings-105"><span class="header-section-number">9.110.2</span> Main Findings</h3>
<p>Liu et al. enrolled a cohort of 40 patients from Wuhan
including 27 mild cases and 13 severe cases of COVID-19. They performed
a 16-day kinetic analysis of peripheral blood from time of disease
onset. Patients in the severe group were older (medium age of 59.7,
compared to 48.7 in mild group) and more likely to have hypertension as
a co-morbidity. Lymphopenia was observed in 44.4% of the mild patients
and 84.6% of the severe patients. Lymphopenia was due to low T cell
count, specially CD8 T cells. Severe patients showed higher neutrophil
counts and an increase of cytokines in the serum (IL2, IL6, IL10 and
IFNγ). The authors measured several other clinical laboratory parameters
were also higher in severe cases compared to mild, but concluded that
neutrophil to CD8 T cell ratio (N8R) as the best prognostic factor to
identify the severe cases compared to other receiver operating
characteristic (ROC).</p>
<h3 data-number="9.110.3" id="limitations-108"><span class="header-section-number">9.110.3</span> Limitations</h3>
<p>This was a small cohort (N=40), and two of the patients
initially included in the severe group (N=13) passed away and were
excluded from the analysis due to lack of longitudinal data. However, it
would be most important to be able to identify patients with severe
disease with higher odds of dying. It seems that the different time
points analyzed relate to hospital admission, which the authors describe
as disease onset. The time between first symptoms and first data points
is not described. It would have been important to analyze how the
different measured parameters change according to health condition, and
not just time (but that would require a larger cohort). The predictive
value of N8R compared to the more commonly used NLR needs to be assessed
in other independent and larger cohorts. Lastly, it is important to note
that pneumonia was detected in patients included in the “mild” group,
but according to the Chinese Clinical Guidance for COVID-19 Pneumonia
Diagnosis and Treatment (7th edition) this group should be considered
“moderate”.</p>
<h3 data-number="9.110.4" id="significance-98"><span class="header-section-number">9.110.4</span> Significance</h3>
<p>Lymphopenia and cytokine storm have been described to be
detrimental in many other infections including SARS-CoV1 and MERS-CoV.
However, it was necessary to confirm that this dramatic immune response
was also observed in the SARS-CoV2 infected patients. These results and
further validation of the N8R ratio as a predictor of disease severity
will contribute for the management of COVID19 patients and potential
development of therapies.</p>
<h3 data-number="9.110.5" id="credit-106"><span class="header-section-number">9.110.5</span> Credit</h3>
<p><em>Review by Pauline Hamon as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.111" id="clinical-and-immunologic-features-in-severe-and-moderate-forms-of-coronavirus-disease-2019-1"><span class="header-section-number">9.111</span> Clinical and immunologic features in severe and moderate forms of Coronavirus Disease 2019</h2>
<p>Chen et al. <em>medRxiv</em> <span class="citation" data-cites="1EIEx8Qe4">[<a href="#ref-1EIEx8Qe4" role="doc-biblioref">552</a>]</span></p>
<h3 data-number="9.111.1" id="keywords-110"><span class="header-section-number">9.111.1</span> Keywords</h3>
<ul>
<li>severe disease</li>
<li>lymphocytes</li>
<li>cytokines</li>
<li>IFNy</li>
<li>CD4 Tcells</li>
<li>HLA-DR CD8</li>
<li>Tcells</li>
</ul>
<h3 data-number="9.111.2" id="main-findings-106"><span class="header-section-number">9.111.2</span> Main Findings</h3>
<p>This study retrospectively evaluated clinical, laboratory,
hematological, biochemical and immunologic data from 21 subjects
admitted to the hospital in Wuhan, China (late December/January) with
confirmed SARS-CoV-2 infection. The aim of the study was to compare
‘severe’ (n=11, ~64 years old) and ‘moderate’ (n=10, ~51 years old)
COVID-19 cases. Disease severity was defined by patients’ blood oxygen
level and respiratory output. They were classified as ‘severe’ if SpO2
93% or respiratory rates 30 per min.</p>
<p>In terms of the clinical laboratory measures, ‘severe’ patients had
higher CRP and ferritin, alanine and aspartate aminotransferases, and
lactate dehydrogenase but lower albumin concentrations.</p>
<p>The authors then compared plasma cytokine levels (ELISA) and immune cell
populations (PBMCs, Flow Cytometry). ‘Severe’ cases had higher levels of
IL-2R, IL-10, TNFa, and IL-6 (marginally significant). For the immune
cell counts, ‘severe’ group had higher neutrophils, HLA-DR+ CD8 T cells
and total B cells; and lower total lymphocytes, CD4 and CD8 T cells
(except for HLA-DR+), CD45RA Tregs, and IFNy-expressing CD4 T cells. No
significant differences were observed for IL-8, counts of NK cells,
CD45+RO Tregs, IFNy-expressing CD8 T and NK cells.</p>
<h3 data-number="9.111.3" id="limitations-109"><span class="header-section-number">9.111.3</span> Limitations</h3>
<p>Several potential limitations should be noted: 1) Blood samples were
collected 2 days post hospital admission and no data on viral loads were
available; 2) Most patients were administered medications (e.g.
corticosteroids), which could have affected lymphocyte counts.
Medications are briefly mentioned in the text of the manuscript; authors
should include medications as part of Table 1. 3) ‘Severe’ cases were
significantly older and 4/11 ‘severe’ patients died within 20 days.
Authors should consider a sensitivity analysis of biomarkers with the
adjustment for patients’ age.</p>
<h3 data-number="9.111.4" id="significance-99"><span class="header-section-number">9.111.4</span> Significance</h3>
<p>Although the sample size was small, this paper presented a broad range
of clinical, biochemical, and immunologic data on patients with
COVID-19. One of the main findings is that SARS-CoV-2 may affect T
lymphocytes, primarily CD4+ T cells, resulting in decreased IFNy
production. Potentially, diminished T lymphocytes and elevated cytokines
can serve as biomarkers of severity of COVID-19.</p>
<h3 data-number="9.111.5" id="credit-107"><span class="header-section-number">9.111.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.112" id="sars-cov-2-and-sars-cov-spike-rbd-structure-and-receptor-binding-comparison-and-potential-implications-on-neutralizing-antibody-and-vaccine-development-1"><span class="header-section-number">9.112</span> SARS-CoV-2 and SARS-CoV Spike-RBD Structure and Receptor Binding Comparison and Potential Implications on Neutralizing Antibody and Vaccine Development</h2>
<p>Sun et al. <em>bioRxiv</em> <span class="citation" data-cites="41WAtZmB">[<a href="#ref-41WAtZmB" role="doc-biblioref">553</a>]</span></p>
<h3 data-number="9.112.1" id="keywords-111"><span class="header-section-number">9.112.1</span> Keywords</h3>
<ul>
<li>SARS-CoV</li>
<li>SARS-CoV-2</li>
<li>ACE2</li>
<li>Spike (S) protein</li>
<li>receptor binding domain (RBD)</li>
<li>receptor binding motif (RBM)</li>
<li>neutralizing antibody</li>
</ul>
<h3 data-number="9.112.2" id="main-findings-107"><span class="header-section-number">9.112.2</span> Main Findings</h3>
<p>This study compared the structure of SARS-CoV and SARS-CoV-2 Spike (S)
protein receptor binding domain (RBD) and interactions with ACE2 using
computational modeling, and interrogated cross-reactivity and
cross-neutralization of SARS-CoV-2 by antibodies against SARS-CoV. While
SARS-CoV and SARS-CoV-2 have over 70 % sequence homology and share the
same human receptor ACE2, the receptor binding motif (RBM) is only 50%
homologous.</p>
<p>Computational prediction of the SARS-CoV-2 and ACE2 interactions based
on the previous crystal structure data of SARS-CoV, and measurement of
binding affinities against human ACE2 using recombinant SARS-CoV and
SARS-CoV-2 S1 peptides, demonstrated similar binding of the two S1
peptides to ACE2, explaining the similar transmissibility of SARS-CoV
and SARS-CoV-2 and consistent with previous data (Wall et al Cell
2020).</p>
<p>The neutralization activity of SARS-CoV-specific rabbit polyclonal
antibodies were about two-order of magnitude less efficient to
neutralize SARS-CoV-2 than SARS-CoV, and four potently neutralizing
monoclonal antibodies against SARS-CoV had poor binding and neutralizing
activity against SARS-CoV-2. In contrast, 3 poor SARS-CoV-binding
monoclonal antibodies show some efficiency to bind and neutralize
SARS-CoV-2. The results suggest that that antibodies to more conserved
regions outside the RBM motif might possess better cross-protective
neutralizing activities between two strains.</p>
<h3 data-number="9.112.3" id="limitations-110"><span class="header-section-number">9.112.3</span> Limitations</h3>
<p>It would have been helpful to show the epitopes recognized by the
monoclonal antibodies tested on both SARS-CoV, SARS-CoV-2 to be able to
make predictions for induction of broadly neutralizing antibodies. The
data on monoclonal antibody competition with ACE2 for binding to
SARS-CoV RBD should have also included binding on SARS-CoV2, especially
for the three monoclonal antibodies that showed neutralization activity
for SARS-CoV2. Because of the less homology in RBM sequences between
viruses, it still may be possible that these antibodies would recognize
the ACE2 RBD in SARS-CoV-2.</p>
<h3 data-number="9.112.4" id="significance-100"><span class="header-section-number">9.112.4</span> Significance</h3>
<p>It is noteworthy that immunization to mice and rabbit with SARS-CoV S1
or RBD protein could induce monoclonal antibodies to cross-bind and
cross-neutralize SARS-CoV-2 even if they are not ACE2-blocking. If these
types of antibodies could be found in human survivors or in the
asymptomatic populations as well, it might suggest that exposure to
previous Coronavirus strains could have induced cross-neutralizing
antibodies and resulted in the protection from severe symptoms in some
cases of SARS-CoV2.</p>
<h3 data-number="9.112.5" id="credit-108"><span class="header-section-number">9.112.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.113" id="protection-of-rhesus-macaque-from-sars-coronavirus-challenge-by-recombinant-adenovirus-vaccine-1"><span class="header-section-number">9.113</span> Protection of Rhesus Macaque from SARS-Coronavirus challenge by recombinant adenovirus vaccine</h2>
<p>Chen et al. <em>bioRxiv</em> <span class="citation" data-cites="NPxs7bbq">[<a href="#ref-NPxs7bbq" role="doc-biblioref">554</a>]</span></p>
<h3 data-number="9.113.1" id="keywords-112"><span class="header-section-number">9.113.1</span> Keywords</h3>
<ul>
<li>SARS-CoV-1</li>
<li>rhesus macaque</li>
<li>recombinant adenovirus vaccine</li>
</ul>
<h3 data-number="9.113.2" id="main-findings-108"><span class="header-section-number">9.113.2</span> Main Findings</h3>
<p>Rhesus macaques were immunized intramuscularly twice (week 0 and
week 4) with SV8000 carrying the information to express a S1-orf8
fusion protein and the N protein from the BJ01 strain of SARS-CoV-1.
By week 8, immunized animals had signs of immunological protection
(IgG and neutralization titers) against SARS-CoV-1 and were
protected against challenge with the PUMC-1 strain, with fewer
detectable symptoms of respiratory distress, lower viral load,
shorter periods of viral persistence, and less pathology in the
lungs compared to non-immunized animals.</p>
<h3 data-number="9.113.3" id="limitations-111"><span class="header-section-number">9.113.3</span> Limitations</h3>
<p>The authors should write clearer descriptions of the methods used in
this article. They do not describe how the IgG titers or
neutralization titers were determined. There are some issues with
the presentation of data, for example, in Figure 1a, y-axis should
not be Vmax; forming cells and 1d would benefit from showing error
bars. Furthermore, although I inferred that the animals were
challenged at week 8, the authors did not explicitly detail when the
animals were challenged. The authors should explain the design of
their vaccine, including the choice of antigens and vector. The
authors also do not include a description of the ethical use of
animals in their study.</p>
<h3 data-number="9.113.4" id="significance-101"><span class="header-section-number">9.113.4</span> Significance</h3>
<p>The authors describe a vaccine for SARS-CoV-1 with no discussion of
possible implications for the current SARS-CoV-2 pandemic. Could a
similar vaccine be designed to protect against SARS-CoV-2 and would
the concerns regarding emerging viral mutations that the authors
describe as a limitation for SARS-CoV-1 also be true in the context
of SARS-CoV-2?</p>
<h3 data-number="9.113.5" id="credit-109"><span class="header-section-number">9.113.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.114" id="reduction-and-functional-exhaustion-of-t-cells-in-patients-with-coronavirus-disease-2019-covid-19-1"><span class="header-section-number">9.114</span> Reduction and Functional Exhaustion of T cells in Patients with Coronavirus Disease 2019 (COVID-19)</h2>
<p><span class="citation" data-cites="f50ahNxF">[<a href="#ref-f50ahNxF" role="doc-biblioref">555</a>]</span></p>
<h3 data-number="9.114.1" id="keywords-113"><span class="header-section-number">9.114.1</span> Keywords</h3>
<ul>
<li>T cell exhaustion</li>
<li>T cell lymphopenia</li>
<li>IL-6</li>
<li>IL-10</li>
<li>TNF-α</li>
</ul>
<h3 data-number="9.114.2" id="main-findings-109"><span class="header-section-number">9.114.2</span> Main Findings</h3>
<p>Based on a retrospective study of 522 COVID patients and 40
healthy controls from two hospitals in Wuhan, China, authors show both
age-dependent and clinical severity-dependent decrease in T cell numbers
with elderly patients and patients who are in ICU-care showing the most
dramatic decrease in T cell counts. Cytokine profiling of COVID patients
reveal that TNF-α, IL-6 and IL-10 are increased in infected patients
with patients in the ICU showing the highest levels. Interestingly,
these three cytokine levels were inversely correlated with T cell counts
and such inverse relationship was preserved throughout the disease
progression. Surface staining of exhaustion markers (PD-1 and Tim-3) and
flow cytometry of stained peripheral blood of 14 patients and 3 healthy
volunteers demonstrate that T cells of COVID patients have increased
expression of PD-1 with patients in ICU having the highest number of
CD8<sup>+</sup>PD-1<sup>+</sup> cells than their counterparts in non-ICU groups.</p>
<h3 data-number="9.114.3" id="limitations-112"><span class="header-section-number">9.114.3</span> Limitations</h3>
<p>Compared to the number of patients, number of control (n=
40) is small and is not controlled for age. Additional data linking
inflammatory cytokines and the quality of the adaptive response
including humoral and antigen specific T cell response is much needed. T
cell exhaustion study relies on marker-dependent labeling of T cell
functionality of a very limited sample size (n=17)—a
functional/mechanistic study of these T cells from PBMCs would have
bolstered their claims.</p>
<h3 data-number="9.114.4" id="significance-102"><span class="header-section-number">9.114.4</span> Significance</h3>
<p>Limited but contains interesting
implications. It is already known in literature that in the context of
acute respiratory viral infections CD8 T cells exhibit exhaustion-like
phenotypes which further underscores the importance of mechanistic
studies that can elucidate how COVID infection leads to lymphopenia and
T cell exhaustion-like phenotype.</p>
<p>However, as authors have noted, the data does point to an interesting
question: How these inflammatory cytokines (TNF-α, IL-6 and IL-10)
correlate with or affect effective viral immunity and what types of
cells produce these cytokines? Answering that question will help us
refine our targets for immune-modulatory therapies especially in
patients suffering from cytokine storms.</p>
<h3 data-number="9.114.5" id="credit-110"><span class="header-section-number">9.114.5</span> Credit</h3>
<p><em>This review by Chang Moon was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.115" id="clinical-characteristics-of-25-death-cases-infected-with-covid-19-pneumonia-a-retrospective-review-of-medical-records-in-a-single-medical-center-wuhan-china-1"><span class="header-section-number">9.115</span> Clinical Characteristics of 25 death cases infected with COVID-19 pneumonia: a retrospective review of medical records in a single medical center, Wuhan, China</h2>
<p><span class="citation" data-cites="EwNi3WJq">[<a href="#ref-EwNi3WJq" role="doc-biblioref">556</a>]</span></p>
<h3 data-number="9.115.1" id="keywords-114"><span class="header-section-number">9.115.1</span> Keywords</h3>
<ul>
<li>COVID-19</li>
<li>pneumonia</li>
<li>hypertension</li>
<li>diabetes</li>
<li>biomarker</li>
<li>neutrophilia</li>
<li>lymphopenia</li>
</ul>
<h3 data-number="9.115.2" id="main-findings-110"><span class="header-section-number">9.115.2</span> Main Findings</h3>
<p>Most common chronic conditions among 25 patients that died from
COVID-19 related respiratory failure were hypertension (64%) and
diabetes (40%). Disease progression was marked by progressive organ
failure, starting first with lung dysfunction, then heart (e.g.
increased cTnI and pro-BNP), followed by kidney (e.g. increased BUN,
Cr), and liver (e.g. ALT, AST). 72% of patients had neutrophilia and 88%
also had lymphopenia. General markers of inflammation were also
increased (e.g. PCT, D-Dimer, CRP, LDH, and SAA).</p>
<h3 data-number="9.115.3" id="limitations-113"><span class="header-section-number">9.115.3</span> Limitations</h3>
<p>The limitations of this study include small sample size and
lack of measurements for some tests for several patients. This study
would also have been stronger with comparison of the same measurements
to patients suffering from less severe disease to further validate and
correlate proposed biomarkers with disease severity.</p>
<h3 data-number="9.115.4" id="significance-103"><span class="header-section-number">9.115.4</span> Significance</h3>
<p>This study identifies chronic conditions (i.e. hypertension
and diabetes) that strongly correlates with disease severity. In
addition to general markers of inflammation, the authors also identify
concomitant neutrophilia and lymphopenia among their cohort of patients.
This is a potentially interesting immunological finding because we would
typically expect increased lymphocytes during a viral infection.
Neutrophilia may also be contributing to cytokine storm. In addition,
PCT was elevated in 90.5% of patients, suggesting a role for sepsis or
secondary bacterial infection in COVID-19 related respiratory failure.</p>
<h3 data-number="9.115.5" id="credit-111"><span class="header-section-number">9.115.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.116" id="sars-cov-2-infection-does-not-significantly-cause-acute-renal-injury-an-analysis-of-116-hospitalized-patients-with-covid-19-in-a-single-hospital-wuhan-china-1"><span class="header-section-number">9.116</span> SARS-CoV-2 infection does not significantly cause acute renal injury: an analysis of 116 hospitalized patients with COVID-19 in a single hospital, Wuhan, China</h2>
<p><span class="citation" data-cites="KG5Lg65z">[<a href="#ref-KG5Lg65z" role="doc-biblioref">557</a>]</span></p>
<h3 data-number="9.116.1" id="keywords-115"><span class="header-section-number">9.116.1</span> Keywords</h3>
<ul>
<li>CoVID-19</li>
<li>2019-nCoV</li>
<li>SARS-CoV-2</li>
<li>kidney</li>
<li>clinical</li>
<li>longitudinal</li>
</ul>
<h3 data-number="9.116.2" id="main-findings-111"><span class="header-section-number">9.116.2</span> Main Findings</h3>
<ul>
<li><p>Clinical data from 116 hospitalized CoVID-19 patients analyzed over
4 weeks for correlation with renal injury. Comorbidities included
chronic renal failure (CRF) in 5 patients (4.3%).</p></li>
<li><p>10.8% of patients with no prior kidney disease showed elevations in
blood urea or creatinine, and 7.2% of patients with no prior kidney
disease showed albuminuria.</p></li>
<li><p>Patients with pre-existing CRF underwent continuous renal
replacement therapy (CRRT) alongside CoVID-19 treatment. Renal
functions remained stable in these patients.</p></li>
<li><p>All 5 patients with CRF survived CoVID-19 therapy without
progression to ARDS or worsening of CRF.</p></li>
</ul>
<h3 data-number="9.116.3" id="limitations-114"><span class="header-section-number">9.116.3</span> Limitations</h3>
<ul>
<li><p>Renal injury biomarkers in patients with incipient kidney
abnormalities not tabulated separately, making overall data hard to
interpret. It will be critical to separately examine kidney function
(BUN, urine creatinine and eGFR) in patients that developed any
kidney abnormalities (7.2~10.8% of cohort).</p></li>
<li><p>No information on type of CoVID-19 therapy used across cohort; will
be useful to correlate how treatment modality influences kidney
function (and other parameters).</p></li>
<li><p>Invokes previous clinical-correlation studies that indicate low
instances of kidney damage<span class="citation" data-cites="VRoblZFJ bgrzawWp">[<a href="#ref-VRoblZFJ" role="doc-biblioref">558</a>,<a href="#ref-bgrzawWp" role="doc-biblioref">559</a>]</span>, but those studies did not track
longitudinal urine samples for acute renal injury markers and viral
shedding.</p></li>
<li><p>CRRT in patients with CRF is standard therapy irrespective of
CoVID-19 status; it will be important to compare clinical parameters
of these patients (n=5) with virus-naïve CRF patients (none in this
study) to make any meaningful conclusions.</p></li>
</ul>
<h3 data-number="9.116.4" id="significance-104"><span class="header-section-number">9.116.4</span> Significance</h3>
<ul>
<li><p>This study argues that renal impairment is uncommon in CoVID-19 and
not associated with high mortaility, in stark contrast with a
concurrent study <span class="citation" data-cites="1Acv5UMSt">[<a href="#ref-1Acv5UMSt" role="doc-biblioref">537</a>]</span>.
If supported by further studies, this argues kidney impairment is
secondary to cytokine storm/inflammation-induced organ failure, and
not due to direct viral replication.</p></li>
<li><p>Will be important to comprehensively characterize large-datasets of
CoVID-19 patients to conclude if kidney function actively disrupted
due to viral infection.</p></li>
</ul>
<h3 data-number="9.116.5" id="credit-112"><span class="header-section-number">9.116.5</span> Credit</h3>
<p><em>Review by Samarth Hegde as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.117" id="potential-t-cell-and-b-cell-epitopes-of-2019-ncov-1"><span class="header-section-number">9.117</span> Potential T-cell and B-Cell Epitopes of 2019-nCoV</h2>
<p><span class="citation" data-cites="hdSafq1o">[<a href="#ref-hdSafq1o" role="doc-biblioref">560</a>]</span></p>
<h3 data-number="9.117.1" id="keywords-116"><span class="header-section-number">9.117.1</span> Keywords</h3>
<ul>
<li>COVID-19</li>
<li>vaccine</li>
<li>epitopes</li>
<li>spike protein</li>
<li>MHC-I</li>
<li>MHC-II</li>
<li>neutralizing antibodies</li>
<li>ACE2</li>
</ul>
<h3 data-number="9.117.2" id="main-findings-112"><span class="header-section-number">9.117.2</span> Main Findings</h3>
<p>The authors use 2 neural network algorithms, NetMHCpan4 and
MARIA, to identify regions within the COVID-19 genome that are
presentable by HLA. They identify 405 viral epitopes that are
presentable on MHC-I and MHC-II and validate using known epitopes from
SARS-CoV. To determine whether immune surveillance drives viral
mutations to evade MHC presentation, the authors analyzed 68 viral
genomes from 4 continents. They identified 93 point mutations that
occurred preferentially in regions predicted to be presented by MHC-I
(p=0.02) suggesting viral evolution to evade CD8 T-cell mediated
killing. 2 nonsense mutations were also identified that resulted in loss
of presentation of an associated antigen (FGDSVEEVL) predicted to be
good antigen for presentation across multiple HLA alleles.</p>
<p>To identify potential sites of neutralizing antibody binding, the
authors used homology modeling to the SARS-CoV’s spike protein (S
protein) to determine the putative structure of the CoV2 spike protein.
They used Discotope2 to identify antibody binding sites on the protein
surface in both the down and up conformations of the S protein. The
authors validate this approach by first identifying antibody binding
site in SARS-CoV S protein. In both the down and up conformation of the
CoV2 S protein, the authors identified a potential antibody binding site
on the S protein receptor binding domain (RBD) of the ACE2 receptor
(residues 440-460, 494-506). While RBDs in both SARS-CoV and CoV2 spike
proteins may be important for antibody binding, the authors note that
SARS-CoV has larger attack surfaces than CoV2. These results were later
validated on published crystal structures of the CoV2 S protein RBD and
human ACE2. Furthermore, analysis of 68 viral genomes did not identify
any mutations in this potential antibody binding site in CoV2.</p>
<p>Finally, the authors compile a list of potential peptide vaccine
candidates across the viral genome that can be presented by multiple HLA
alleles. Several of the peptides showed homology to SARS-CoV T-cell and
B-cell epitopes.</p>
<h3 data-number="9.117.3" id="limitations-115"><span class="header-section-number">9.117.3</span> Limitations</h3>
<p>While the authors used computational methods of validation,
primarily through multiple comparisons to published SARS-CoV structures
and epitopes, future work should include experimental validation of
putative T-cell and B-cell epitopes.</p>
<h3 data-number="9.117.4" id="significance-105"><span class="header-section-number">9.117.4</span> Significance</h3>
<p>The authors identified potential T-cell and B-cell epitopes
that may be good candidates for peptide based vaccines against CoV2.
They also made interesting observations in comparing SARS-CoV and CoV2
potential antibody binding sites, noting that SARS-CoV had larger attack
surfaces for potential neutralizing antibody binding. One of the
highlights of this paper was the authors’ mutation analysis of 68 viral
genomes from 4 continents. This analysis not only validated their
computational method for identifying T-cell epitopes, but showed that
immune surveillance likely drives viral mutation in MHC-I binding
peptides. The smaller attack surface may point to potential mechanisms
of immune evasion by CoV2. However, absence of mutations in the RBD of
CoV2 and the small number of mutations in peptides presentable to T
cells suggests that vaccines against multiple epitopes could still
elicit robust immunity against CoV2.</p>
<h3 data-number="9.117.5" id="credit-113"><span class="header-section-number">9.117.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.118" id="structure-function-and-antigenicity-of-the-sarscov-2-spike-glycoprotein-1"><span class="header-section-number">9.118</span> Structure, Function, and Antigenicity of the SARSCoV-2 Spike Glycoprotein</h2>
<p>Walls et al. <em>bioRxiv</em>. <span class="citation" data-cites="8xnxgpag">[<a href="#ref-8xnxgpag" role="doc-biblioref">561</a>]</span> now <span class="citation" data-cites="qcVbT0w4">[<a href="#ref-qcVbT0w4" role="doc-biblioref">38</a>]</span></p>
<h3 data-number="9.118.1" id="keywords-117"><span class="header-section-number">9.118.1</span> Keywords</h3>
<ul>
<li>binding affinity</li>
<li>antigenicity</li>
<li>neutralizing antibody</li>
</ul>
<h3 data-number="9.118.2" id="main-findings-113"><span class="header-section-number">9.118.2</span> Main Findings</h3>
<p>The authors highlight a human angiotensin-converting enzyme 2
(hACE2), as a potential receptor used by the current Severe Acute
respiratory syndrome coronavirus-2 (SARS-CoV-2) as a host factor that
allows the virus target human cells. This virus-host interaction
facilitates the infection of human cells with a high affinity comparable
with SARS-CoV. The authors propose this mechanism as a probable
explanation of the efficient transmission of SARS-CoV-2 between humans.
Besides, Walls and colleagues described SARS-CoV-2 S glycoprotein S by
Cryo-EM along with neutralizing polyclonal response against SAR-CoV-2 S
from mice immunized with SAR-CoV and blocking SAR-CoV-2 S-mediated entry
into VeroE6 infected cells.**</p>
<h3 data-number="9.118.3" id="limitations-116"><span class="header-section-number">9.118.3</span> Limitations</h3>
<p>The SARS-CoV-2 depends on the cell factors ACE2 and TMPRSS2, this
last, according to a recent manuscript by Markus Hoffman et al., <em>Cell</em>,
2020. The authors used green monkey (VeroE6) and hamster (BHK) cell
lines in the experiments to drive its conclusions to humans; however, it
is well known the caucasian colon adenocarcinoma human cell line
(CaCo-2), highly express the hACE2 receptor as the TMPRSS2 protease as
well. In humans, ACE2 protein is highly expressed in the
gastrointestinal tract, which again, makes the CaCo-2 cell line suitable
for the following SARS-CoV-2 studies.</p>
<h3 data-number="9.118.4" id="significance-106"><span class="header-section-number">9.118.4</span> Significance</h3>
<p>The results propose a functional receptor used by SARS-CoV-2 to
infect humans worldwide and defining two distinct conformations of spike
(S) glycoprotein by cryogenic electron microscopy (Cryo-EM). This study
might help establish a precedent for initial drug design and treatment
of the current global human coronavirus epidemic.</p>
<h3 data-number="9.118.5" id="credit-114"><span class="header-section-number">9.118.5</span> Credit</h3>
<p><em>Review by postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.119" id="breadth-of-concomitant-immune-responses-underpinning-viral-clearance-and-patient-recovery-in-a-non-severe-case-of-covid-19-1"><span class="header-section-number">9.119</span> Breadth of concomitant immune responses underpinning viral clearance and patient recovery in a non-severe case of COVID-19</h2>
<p>Thevarajan et al. <em>medRxiv</em> <span class="citation" data-cites="QdimT8mN">[<a href="#ref-QdimT8mN" role="doc-biblioref">562</a>]</span></p>
<h3 data-number="9.119.1" id="keywords-118"><span class="header-section-number">9.119.1</span> Keywords</h3>
<ul>
<li>IgG</li>
<li>IgM</li>
<li>TfH cells</li>
<li>NK cells</li>
<li>SNP</li>
</ul>
<h3 data-number="9.119.2" id="main-findings-114"><span class="header-section-number">9.119.2</span> Main Findings</h3>
<p>The authors characterized the immune response in peripheral blood of a
47-year old COVID-19 patient.</p>
<p>SARS-CoV2 was detected in nasopharyngeal swab, sputum and faeces
samples, but not in urine, rectal swab, whole blood or throat swab. 7
days after symptom onset, the nasopharyngeal swab test turned negative,
at day 10 the radiography infiltrates were cleared and at day 13 the
patient became asymptomatic.</p>
<p>Immunofluorescence staining shows from day 7 the presence of
<strong>COVID-19-binding IgG and IgM</strong> antibodies in plasma, that increase
until day 20.</p>
<p>Flow cytometry on whole blood reveals a plasmablast peak at day 8, a
gradual increase in T follicular helper cells, stable HLA-DR<sup>+</sup> NK
frequencies and decreased monocyte frequencies compared to healthy
counterparts. The expression of CD38 and HLA-DR peaked on T cells at D9
and was associated with higher production of cytotoxic mediators by
CD8<sup>+</sup> T cells.</p>
<p>IL-6 and IL-8 were undetectable in plasma.</p>
<p>The authors further highlight the presence of the <strong>IFITM3
SNP-rs12252-C/C variant</strong> in this patient, which is associated with
higher susceptibility to influenza virus.</p>
<h3 data-number="9.119.3" id="limitations-117"><span class="header-section-number">9.119.3</span> Limitations</h3>
<p>These results need to be confirmed in additional patients.</p>
<p>COVID-19 patients have increased infiltration of macrophages in their
lungs <span class="citation" data-cites="tVkKuIJt">[<a href="#ref-tVkKuIJt" role="doc-biblioref">563</a>]</span>. Monitoring monocyte proportions in blood earlier in the
disease might help to evaluate their eventual migration to the lungs.</p>
<p>The stable concentration of HLA-DR<sup>+</sup> NK cells in blood from day 7 is
not sufficient to rule out NK cell activation upon SARS-CoV2 infection.
In response to influenza A virus, NK cells express higher levels of
activation markers CD69 and CD38, proliferate better and display higher
cytotoxicity <span class="citation" data-cites="87CJN2CG">[<a href="#ref-87CJN2CG" role="doc-biblioref">564</a>]</span>. Assessing these parameters in COVID-19 patients is
required to better understand NK cell role in clearing this infection.</p>
<p>Neutralization potential of the COVID-19-binding IgG and IgM antibodies
should be assessed in future studies.</p>
<p>This patient was able to clear the virus, while presenting a SNP
associated with severe outcome following influenza infection. The
association between this SNP and outcome upon SARS-CoV2 infection should
be further investigated.</p>
<h3 data-number="9.119.4" id="significance-107"><span class="header-section-number">9.119.4</span> Significance</h3>
<p>This study is among the first to describe the appearance of
COVID-19-binding IgG and IgM antibodies upon infection. The emergence of
new serological assays might contribute to monitor more precisely the
seroconversion kinetics of COVID-19 patients <span class="citation" data-cites="yJW1RVQb">[<a href="#ref-yJW1RVQb" role="doc-biblioref">210</a>]</span>. Further association
studies between IFITM3 SNP-rs12252-C/C variant and clinical data might
help to refine the COVID-19 outcome prediction tools.</p>
<h3 data-number="9.119.5" id="credit-115"><span class="header-section-number">9.119.5</span> Credit</h3>
<p><em>Review by Bérengère Salomé as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.120" id="the-landscape-of-lung-bronchoalveolar-immune-cells-in-covid-19-revealed-by-single-cell-rna-sequencing-1"><span class="header-section-number">9.120</span> The landscape of lung bronchoalveolar immune cells in COVID-19 revealed by single-cell RNA sequencing</h2>
<p>Liao et al. <em>medRxiv</em> <span class="citation" data-cites="tVkKuIJt">[<a href="#ref-tVkKuIJt" role="doc-biblioref">563</a>]</span></p>
<h3 data-number="9.120.1" id="keywords-119"><span class="header-section-number">9.120.1</span> Keywords</h3>
<ul>
<li>COVID-19</li>
<li>SARS-CoV-2</li>
<li>Broncho-alveolar lavage</li>
<li>macrophages</li>
<li>NK cells</li>
<li>T cells</li>
<li>cytokine storm</li>
<li>scRNAseq</li>
</ul>
<h3 data-number="9.120.2" id="main-findings-115"><span class="header-section-number">9.120.2</span> Main Findings</h3>
<p>The authors performed single-cell RNA sequencing (scRNAseq) on
bronchoalveolar lavage fluid (BAL) from 6 COVID-19 patients (n=3 mild
cases, n=3 severe cases). Data was compared to previously generated
scRNAseq data from healthy donor lung tissue (n=8).</p>
<p>Clustering analysis of the 6 patients revealed distinct immune cell
organization between mild and severe disease. Specifically, they found
that transcriptional clusters annotated as tissue resident alveolar
macrophages were strongly reduced while monocytes-derived FCN1<sup>+</sup>SPP1<sup>+</sup>
inflammatory macrophages dominated the BAL of patients with severe
COVID19 diseases. They show that inflammatory macrophages upregulated
interferon-signaling genes, monocytes recruiting chemokines including
CCL2, CCL3, CCL4 as well as IL-6, TNF, IL-8 and profibrotic cytokine
TGF-β, while alveolar macrophages expressed lipid metabolism genes, such
as PPARG.</p>
<p>The lymphoid compartment was overall enriched in lungs from patients.
Clonally expanded CD8 T cells were enriched in mild cases suggesting
that CD8 T cells contribute to viral clearance as in Flu infection,
whereas proliferating T cells were enriched in severe cases.</p>
<p>SARS-CoV-2 viral transcripts were detected in severe patients, but
considered here as ambient contaminations.</p>
<h3 data-number="9.120.3" id="limitations-118"><span class="header-section-number">9.120.3</span> Limitations</h3>
<p>These results are based on samples from 6 patients and should therefore
be confirmed in the future in additional patients. Longitudinal
monitoring of BAL during disease progression or resolution would have
been most useful.</p>
<p>The mechanisms underlying the skewing of the macrophage compartment in
patients towards inflammatory macrophages should be investigated in
future studies.</p>
<p>Deeper characterization of the lymphoid subsets is required. The
composition of the “proliferating” cluster and how these cells differ
from conventional T cell clusters should be assessed. NK and CD8 T cell
transcriptomic profile, in particular the expression of cytotoxic
mediator and immune checkpoint transcripts, should be compared between
healthy and diseased lesions.</p>
<h3 data-number="9.120.4" id="significance-108"><span class="header-section-number">9.120.4</span> Significance</h3>
<p>COVID-19 induces a robust inflammatory cytokine storm in patients that
contributes to severe lung tissue damage and ARDS <span class="citation" data-cites="vasHgjGH">[<a href="#ref-vasHgjGH" role="doc-biblioref">565</a>]</span>. Accumulation of
monocyte-derived inflammatory macrophages at the expense of Alveolar
macrophages known to play an anti-inflammatory role following
respiratory viral infection, in part through the PPARγ pathway <span class="citation" data-cites="9GEyqic3 WBN5f37b">[<a href="#ref-9GEyqic3" role="doc-biblioref">566</a>,<a href="#ref-WBN5f37b" role="doc-biblioref">567</a>]</span> are
likely contributing to lung tissue injuries. These data suggest that
reduction of monocyte accumulation in the lung tissues could help
modulate COVID-19-induced inflammation. Further analysis of lymphoid
subsets is required to understand the contribution of adaptive immunity
to disease outcome.</p>
<h3 data-number="9.120.5" id="credit-116"><span class="header-section-number">9.120.5</span> Credit</h3>
<p><em>Review by Bérengère Salomé and Assaf Magen as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.121" id="can-routine-laboratory-tests-discriminate-2019-novel-coronavirus-infected-pneumonia-from-other-community-acquired-pneumonia-1"><span class="header-section-number">9.121</span> Can routine laboratory tests discriminate 2019 novel coronavirus infected pneumonia from other community-acquired pneumonia?</h2>
<p>Pan et al. <em>medRxiv</em> <span class="citation" data-cites="FOVDaThy">[<a href="#ref-FOVDaThy" role="doc-biblioref">568</a>]</span></p>
<h3 data-number="9.121.1" id="keywords-120"><span class="header-section-number">9.121.1</span> Keywords</h3>
<ul>
<li>Routine laboratory testing</li>
</ul>
<h3 data-number="9.121.2" id="main-findings-116"><span class="header-section-number">9.121.2</span> Main Findings</h3>
<p>In an attempt to use standard laboratory testing for the discrimination
between “Novel Coronavirus Infected Pneumonia” (NCIP) and a usual
community acquired pneumonia (CAP), the authors compared laboratory
testing results of 84 NCIP patients with those of a historical group of
316 CAP patients from 2018 naturally COVID-19 negative. The authors
describe significantly lower white blood- as well as red blood- and
platelet counts in NCIP patients. When analyzing differential blood
counts, lower absolute counts were measured in all subsets of NCIP
patients. With regard to clinical chemistry parameters, they found
increased AST and bilirubin in NCIP patients as compared to CAP
patients.</p>
<h3 data-number="9.121.3" id="limitations-119"><span class="header-section-number">9.121.3</span> Limitations</h3>
<p>The authors claim to describe a simple method to rapidly assess a
pre-test probability for NCIP. However, the study has substantial
weakpoints. The deviation in clinical laboratory values in NCIP patients
described here can usually be observed in severely ill patients. The
authors do not comment on how severely ill the patients tested here were
in comparison to the historical control. Thus, the conclusion that the
tests discriminate between CAP and NCIP lacks justification.</p>
<h3 data-number="9.121.4" id="significance-109"><span class="header-section-number">9.121.4</span> Significance</h3>
<p>The article strives to compare initial laboratory testing results in
patients with COVID-19 pneumonia as compared to patients with a usual
community acquired pneumonia. The implications of this study for the
current clinical situation seem restricted due to a lack in clinical
information and the use of a control group that might not be
appropriate.</p>
<h3 data-number="9.121.5" id="credit-117"><span class="header-section-number">9.121.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.122" id="correlation-analysis-between-disease-severity-and-inflammation-related-parameters-in-patients-with-covid-19-pneumonia-1"><span class="header-section-number">9.122</span> Correlation Analysis Between Disease Severity and Inflammation-related Parameters in Patients with COVID-19 Pneumonia</h2>
<p><span class="citation" data-cites="1Du40IOOH">[<a href="#ref-1Du40IOOH" role="doc-biblioref">569</a>]</span></p>
<h3 data-number="9.122.1" id="keywords-121"><span class="header-section-number">9.122.1</span> Keywords</h3>
<ul>
<li>cytokine</li>
<li>COVID-19 pneumonia</li>
<li>severity</li>
<li>disease progression</li>
</ul>
<h3 data-number="9.122.2" id="main-findings-117"><span class="header-section-number">9.122.2</span> Main Findings</h3>
<p>This study is a cross-sectional analysis of 100 patients with
COVID-19 pneumonia, divided into mild (n = 34), severe (n = 34), and
critical (n = 32) disease status based on clinical definitions.</p>
<p>The criteria used to define disease severity are as follows:</p>
<ol type="1">
<li><p><em>Severe</em> – any of the following: respiratory distress or respiratory
rate ≥ 30 respirations/minute; oxygen saturation ≤ 93% at rest;
oxygen partial pressure (PaO2)/oxygen concentration (FiO2) in
arterial blood ≤ 300mmHg, progression of disease on imaging
to &gt;50% lung involvement in the short term.</p></li>
<li><p><em>Critical</em> – any of the following: respiratory failure that requires
mechanical ventilation; shock; other organ failure that requires
treatment in the ICU.</p></li>
<li><p>Patients with pneumonia who test positive for COVID-19 who do not
have the symptoms delineated above are considered <em>mild</em>.</p></li>
</ol>
<p>Peripheral blood inflammatory markers were correlated to disease status.
Disease severity was significantly associated with levels of IL-2R,
IL-6, IL-8, IL-10, TNF-α, CRP, ferroprotein, and procalcitonin. Total
WBC count, lymphocyte count, neutrophil count, and eosinophil count were
also significantly correlated with disease status. Since this is a
retrospective, cross-sectional study of clinical laboratory values,
these data may be extrapolated for clinical decision making, but without
studies of underlying cellular causes of these changes this study does
not contribute to a deeper understanding of SARS-CoV-2 interactions with
the immune system.</p>
<p>It is also notable that the mean age of patients in the mild group was
significantly different from the mean ages of patients designated as
severe or critical (p &lt; 0.001). The mean patient age was not
significantly different between the severe and critical groups. However,
IL-6, IL-8, procalcitonin (Table 2), CRP, ferroprotein (Figure 3A, 3B),
WBC count, and neutrophil count (Figure 4A, 4B) were all significantly
elevated in the critical group compared to severe. These data suggest
underlying differences in COVID-19 progression that is unrelated to age.</p>
<h3 data-number="9.122.3" id="significance-110"><span class="header-section-number">9.122.3</span> Significance</h3>
<p>Given the inflammatory profile outlined in this study,
patients who have mild or severe COVID-19 pneumonia, who <em>also</em> have any
elevations in the inflammatory biomarkers listed above, should be
closely monitored for potential progression to critical status.</p>
<h3 data-number="9.122.4" id="credit-118"><span class="header-section-number">9.122.4</span> Credit</h3>
<p><em>This review by JJF was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.123" id="an-effective-ctl-peptide-vaccine-for-ebola-zaire-based-on-survivors-cd8-targeting-of-a-particular-nucleocapsid-protein-epitope-with-potential-implications-for-covid-19-vaccine-design-1"><span class="header-section-number">9.123</span> An Effective CTL Peptide Vaccine for Ebola Zaire Based on Survivors’ CD8+ Targeting of a Particular Nucleocapsid Protein Epitope with Potential Implications for COVID-19 Vaccine Design</h2>
<p>Herst et al. <em>bioRxiv</em> <span class="citation" data-cites="1DTLMIDoM">[<a href="#ref-1DTLMIDoM" role="doc-biblioref">570</a>]</span></p>
<h3 data-number="9.123.1" id="keywords-122"><span class="header-section-number">9.123.1</span> Keywords</h3>
<ul>
<li>Peptide vaccine</li>
<li>Ebolavirus</li>
<li>nucleocapsid</li>
<li>epitope</li>
<li>vaccine design</li>
<li>microsphere</li>
</ul>
<h3 data-number="9.123.2" id="main-findings-118"><span class="header-section-number">9.123.2</span> Main Findings</h3>
<p>Vaccination of mice with a single dose of a 9-amino-acid peptide NP44-52
located in a conserved region of ebolavirus (EBOV) nucleocapsid protein
(NP) confers CD8+ T-cell-mediated immunity against mouse adapted EBOV
(maEBOV). Bioinformatic analyses predict multiple conserved CD8+ T cell
epitopes in the SARS-CoV-2 NP, suggesting that a similar approach may be
feasible for vaccine design against SARS-CoV-2.</p>
<p>The authors focus on a site within a 20-peptide region of EBOV NP which
was commonly targeted by CD8+ T cells in a group of EBOV survivors
carrying the HLA-A*30:01:01 allele. To justify the testing of specific
vaccine epitopes in a mouse challenge setting, the authors cite known
examples of human pathogen-derived peptide antigens that are also
recognized by C57BL/6 mice, as well as existing data surrounding known
mouse immunogenicity of peptides related to this EBOV NP region. Testing
3 distinct 9mer peptides over an 11 amino-acid window and comparing to
vaccination with the 11mer with a T-cell reactivity readout demonstrated
that optimizing peptide length and position for immunogenicity may be
crucial, likely due to suboptimal peptide processing and MHC-class-I
loading.</p>
<p>Vaccines for maEBOV challenge studies were constructed by packaging
NP44-52 in d,l poly(lactic-co-glycolic) acid microspheres. CpG was also
packaged within the microspheres, while Monophosphoryl Lipid A (a TLR4
ligand) was added to the injectate solution. A second peptide consisting
of a predicted MHC-II epitope from the EBOV VG19 protein was added using
a separate population of microspheres, and the formulation was injected
by intraperitoneal administration. The vaccine was protective against a
range of maEBOV doses up to at least 10,000 PFU. Survival was
anticorrelated with levels of IL6, MCP-1 (CCL2), IL9, and GM-CSF, which
recapitulated trends seen in human EBOV infection.</p>
<p>While HLA-A*30:01:01 is only present in a minority of humans, the
authors state that MHC binding algorithms predict NP44-52 to be a strong
binder of a set of more common HLA-A*02 alleles. The authors predict
that a peptide vaccine based on the proposed formulation could elicit
responses in up to 50% of people in Sudan or 30% of people in North
America.</p>
<p>SARS-CoV-2 NP, meanwhile, has conserved regions which may provide
peptide-vaccine candidates. Scanning the SARS-CoV-2 NP sequence for
HLA-binding 9mers identified 53 peptides with predicted binding affinity
&lt; 500nM, including peptides that are predicted to bind to HLA-class-I
alleles of 97% of humans, 7 of which have previously been tested
<em>in-vitro</em>.</p>
<p>The results support previously appreciated correlations between certain
cytokines and disease severity, specifically IL6 which relates to
multiple trial therapies. Prediction of HLA-class-I binding of
SARS-CoV-2 NP peptides suggests the plausibility of a peptide vaccine
targeting conserved regions of SARS-CoV-2 NP although further validation
in previously infected patient samples will be essential.</p>
<h3 data-number="9.123.3" id="credit-119"><span class="header-section-number">9.123.3</span> Credit</h3>
<p><em>Review by Andrew M. Leader as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.124" id="epitope-based-peptide-vaccines-predicted-against-novel-coronavirus-disease-caused-by-sars-cov-2-1"><span class="header-section-number">9.124</span> Epitope-based peptide vaccines predicted against novel coronavirus disease caused by SARS-CoV-2</h2>
<p>Li et al. <em>bioRxiv</em>. <span class="citation" data-cites="9spK49tt">[<a href="#ref-9spK49tt" role="doc-biblioref">571</a>]</span></p>
<h3 data-number="9.124.1" id="keywords-123"><span class="header-section-number">9.124.1</span> Keywords</h3>
<ul>
<li>SARS-CoV-2</li>
<li>immune-informatics</li>
<li>vaccine design</li>
<li>T cell epitope</li>
<li>B cell epitope</li>
</ul>
<h3 data-number="9.124.2" id="main-findings-119"><span class="header-section-number">9.124.2</span> Main Findings</h3>
<p>This study employs a series of bioinformatic
pipelines to identify T and B cell epitopes on spike (S) protein of
SARS-CoV-2 and assess their properties for vaccine potential. To
identify B cell epitopes, they assessed structural accessibility,
hydrophilicity, and beta-turn and flexibility which are all factors that
promote their targeting by antibodies. To identify T cell epitopes, they
filtered for peptides with high antigenicity score and capacity to bind
3 or more MHC alleles. Using the protein digest server, they also
demonstrated that their identified T and B cell epitopes are stable,
having multiple non-digesting enzymes per epitope. Epitopes were also
determined to be non-allergenic and non-toxin as assessed by Allergen FP
1.0 and ToxinPred, respectively. For T cell epitopes, they assessed the
strength of epitope-HLA interaction via PepSite. Overall, they predict
four B cell and eleven T cell epitopes (two MHC I and nine MHC II
binding) to pass stringent computational thresholds as candidates for
vaccine development. Furthermore, they performed sequence alignment
between all identified SARS-CoV-2 S protein mutations and predicted
epitopes, and showed that the epitopes are conserved across 134 isolates
from 38 locations worldwide. However, they report that these conserved
epitopes may soon become obsolete given the known mutation rate of
related SARS-CoV is estimated to be 4x10-4/site/year, underscoring the
urgency of anti-viral vaccine development.</p>
<h3 data-number="9.124.3" id="limitations-120"><span class="header-section-number">9.124.3</span> Limitations</h3>
<p>While spike (S) protein may have a
critical role in viral entry into host cells and their epitope
prediction criterion were comprehensive, this study did not examine
other candidate SARS-CoV-2 proteins. This point is particularly
important given that a single epitope may not be sufficient to induce
robust immune memory, and recent approaches involve multi-epitope
vaccine design. Furthermore, their study only included a direct
implementation of various published methods, but did not validate
individual bioinformatic tools with controls to demonstrate robustness.
Finally, it is critical that these predicted epitopes are experimentally
validated before any conclusions can be drawn about their potential as
vaccine candidates or their clinical efficacy.</p>
<h3 data-number="9.124.4" id="significance-111"><span class="header-section-number">9.124.4</span> Significance</h3>
<p>This study provides a computational framework to
rapidly identify epitopes that may serve as potential vaccine candidates
for treating SARS-CoV-2.</p>
<h3 data-number="9.124.5" id="credit-120"><span class="header-section-number">9.124.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.125" id="the-definition-and-risks-of-cytokine-release-syndrome-like-in-11-covid-19-infected-pneumonia-critically-ill-patients-disease-characteristics-and-retrospective-analysis-1"><span class="header-section-number">9.125</span> The definition and risks of Cytokine Release Syndrome-Like in 11 COVID-19-Infected Pneumonia critically ill patients: Disease Characteristics and Retrospective Analysis</h2>
<p>Wang Jr. et al. <em>medRxiv</em>.
<span class="citation" data-cites="qg1bmQng">[<a href="#ref-qg1bmQng" role="doc-biblioref">572</a>]</span></p>
<h3 data-number="9.125.1" id="keywords-124"><span class="header-section-number">9.125.1</span> Keywords</h3>
<ul>
<li>Cytokine release syndrome (CRS)</li>
<li>biomarkers</li>
<li>ARDS</li>
<li>IL-6</li>
<li>lymphopenia</li>
</ul>
<h3 data-number="9.125.2" id="main-findings-120"><span class="header-section-number">9.125.2</span> Main Findings</h3>
<p>This study describes the occurrence of a cytokine release syndrome-like
(CRSL) toxicity in ICU patients with COVID-19 pneumonia. The median time
from first symptom to acute respiratory distress syndrome (ARDS) was 10
days. All patients had decreased CD3, CD4 and CD8 cells, and a
significant increase of serum IL-6. Furthermore, 91% had decreased NK
cells. The changes in IL-6 levels preceded those in CD4 and CD8 cell
counts. All of these parameters correlated with the area of pulmonary
inflammation in CT scan images. Mechanical ventilation increased the
numbers of CD4 and CD8 cells, while decreasing the levels of IL-6, and
improving the immunological parameters.</p>
<h3 data-number="9.125.3" id="limitations-121"><span class="header-section-number">9.125.3</span> Limitations</h3>
<p>The number of patients included in this retrospective single center
study is small (n=11), and the follow-up period very short (25 days).
Eight of the eleven patients were described as having CRSL, and were
treated by intubation (7) or ECMO (2). Nine patients were still in the
intensive care unit at the time of publication of this article, so their
disease outcome is unknown.</p>
<h3 data-number="9.125.4" id="significance-112"><span class="header-section-number">9.125.4</span> Significance</h3>
<p>The authors define a cytokine release syndrome-like toxicity in patients
with COVID-19 with clinical radiological and immunological criteria: 1)
decrease of circulating CD4, CD8 and NK cells; 2) substantial increase
of IL-6 in peripheral blood; 3) continuous fever; 4) organ and tissue
damage. This event seems to occur very often in critically ill patients
with COVID-19 pneumonia. Interestingly, the increase of IL-6 in the
peripheral blood preceded other laboratory alterations, thus, IL-6 might
be an early biomarker for the severity of COVID-19 pneumonia. The
manuscript will require considerable editing for organization and
clarity.</p>
<h3 data-number="9.125.5" id="credit-121"><span class="header-section-number">9.125.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.126" id="clinical-characteristics-of-36-non-survivors-with-covid-19-in-wuhan-china-1"><span class="header-section-number">9.126</span> Clinical characteristics of 36 non-survivors with COVID-19 in Wuhan, China</h2>
<p>Huang et al. <em>medRxiv</em>. <span class="citation" data-cites="YODgfrtM">[<a href="#ref-YODgfrtM" role="doc-biblioref">573</a>]</span></p>
<h3 data-number="9.126.1" id="keywords-125"><span class="header-section-number">9.126.1</span> Keywords</h3>
<ul>
<li>Clinical Characteristics</li>
<li>Non Survivors</li>
<li>retrospective study</li>
</ul>
<h3 data-number="9.126.2" id="main-findings-121"><span class="header-section-number">9.126.2</span> Main Findings</h3>
<p>This is a simple study reporting clinical
characteristics of patients who did not survive COVID-19. All patients
(mean age=69.22 years) had acute respiratory distress syndrome (ARDS)
and their median time from onset to ARDS was 11 days. The median time
from onset to death was 17 days. Most patients were older male (70%
male) with co-morbidities and only 11 % were smokers. 75% patients
showed bilateral pneumonia. Many patients had chronic diseases,
including hypertension (58.33%). cardiovascular disease (22.22%) and
diabetes (19.44%). Typical clinical feature measured in these patients
includes lymphopenia and elevated markers of inflammation.</p>
<h3 data-number="9.126.3" id="limitations-122"><span class="header-section-number">9.126.3</span> Limitations</h3>
<p>As noted by the authors, the conclusions of this study are
very limited because this is single-centered study focusing on a small
cohort of patients who did not survive. Many clinical parameters
observed by the authors (such* as increase levels of serum CRP, PCT,
IL-6) have also been described in other COVID19 patients who survived
the infection</p>
<h3 data-number="9.126.4" id="significance-113"><span class="header-section-number">9.126.4</span> Significance</h3>
<p>This study is essentially descriptive and may be useful for
clinical teams monitoring COVID19 patients.</p>
<h3 data-number="9.126.5" id="credit-122"><span class="header-section-number">9.126.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.127" id="risk-factors-related-to-hepatic-injury-in-patients-with-corona-virus-disease-2019-1"><span class="header-section-number">9.127</span> Risk Factors Related to Hepatic Injury in Patients with Corona Virus Disease 2019</h2>
<p><span class="citation" data-cites="5CGbn9Jj">[<a href="#ref-5CGbn9Jj" role="doc-biblioref">574</a>]</span></p>
<h3 data-number="9.127.1" id="keywords-126"><span class="header-section-number">9.127.1</span> Keywords</h3>
<ul>
<li>COVID-related Hepatic Injury</li>
</ul>
<h3 data-number="9.127.2" id="main-findings-122"><span class="header-section-number">9.127.2</span> Main Findings</h3>
<p>Based on a retrospective study of 85 hospitalized COVID
patients in a Beijing hospital, authors showed that patients with
elevated ALT levels (n = 33) were characterized by significantly higher
levels of lactic acid and CRP as well as lymphopenia and hypoalbuminemia
compared to their counterparts with normal ALT levels. Proportion of
severe and critical patients in the ALT elevation group was
significantly higher than that of normal ALT group. Multivariate
logistic regression performed on clinical factors related to ALT
elevation showed that CRP <span class="math inline">\(\geq\)</span> 20mg/L and low lymphocyte count
(&lt;1.1*10^9 cells/L) were independently related to ALT elevation—a
finding that led the authors to suggest cytokine storm as a major
mechanism of liver damage.</p>
<h3 data-number="9.127.3" id="limitations-123"><span class="header-section-number">9.127.3</span> Limitations</h3>
<p>The article’s most attractive claim that liver damage seen in COVID
patients is caused by cytokine storm (rather than direct infection of
the liver) hinges solely on their multivariate regression analysis.
Without further mechanistic studies a) demonstrating how high levels of
inflammatory cytokines can induce liver damage and b) contrasting types
of liver damage incurred by direct infection of the liver vs.
system-wide elevation of inflammatory cytokines, their claim remains
thin. It is also worth noting that six of their elevated ALT group
(n=33) had a history of liver disease (i.e. HBV infection, alcoholic
liver disease, fatty liver) which can confound their effort to pin down
the cause of hepatic injury to COVID.</p>
<h3 data-number="9.127.4" id="significance-114"><span class="header-section-number">9.127.4</span> Significance</h3>
<p>Limited. This article confirms a rich body
of literature describing liver damage and lymphopenia in COVID patients.</p>
<h3 data-number="9.127.5" id="credit-123"><span class="header-section-number">9.127.5</span> Credit</h3>
<p><em>Review by Chang Moon as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.128" id="detectable-serum-sars-cov-2-viral-load-rnaaemia-is-closely-associated-with-drastically-elevated-interleukin-6-il-6-level-in-critically-ill-covid-19-patients-1"><span class="header-section-number">9.128</span> Detectable serum SARS-CoV-2 viral load (RNAaemia) is closely associated with drastically elevated interleukin 6 (IL-6) level in critically ill COVID-19 patients</h2>
<p><span class="citation" data-cites="T0BDaXkw">[<a href="#ref-T0BDaXkw" role="doc-biblioref">575</a>]</span></p>
<h3 data-number="9.128.1" id="keywords-127"><span class="header-section-number">9.128.1</span> Keywords</h3>
<ul>
<li>ARDS</li>
<li>interleukin-6 (IL-6)</li>
<li>procalcitonin (PCT)</li>
<li>pro-inflammatory cytokines</li>
<li>SARS-CoV-2 RNAaemia</li>
</ul>
<h3 data-number="9.128.2" id="main-findings-123"><span class="header-section-number">9.128.2</span> Main Findings</h3>
<p>48 adult patients diagnosed with Covid19 according to Chinese guidelines
for Covid19 diagnosis and treatment version 6 were included in this
study. Patients were further sub-divided into three groups based on
clinical symptoms and disease severity: (1) mild, positive Covid19 qPCR
with no or mild clinical symptoms (fever; respiratory; radiological
abnormalities); (2) severe, at least one of the following: shortness of
breath/respiratory rate &gt;30/min, oxygen saturation SaO<sub>2</sub>&lt;93%,
Horowitz index paO2/FiO<sub>2</sub> &lt; 300 mmHg (indicating moderate pulmonary
damage); and (3) critically ill, at least one additional complicating
factor: respiratory failure with need for mechanical ventilation;
systemic shock; multi-organ failure and transfer to ICU. Serum samples
and throat-swaps were collected from all 48 patients enrolled.
SARS-CoV-2 RNA was assessed by qPCR with positive results being defined
as Ct values &lt; 40, and serum interleukin-6 (IL-6) was quantified
using a commercially available detection kit. Briefly, patient
characteristics in this study confirm previous reports suggesting that
higher age and comorbidities are significant risk factors of clinical
severity. Of note, 5 out of 48 of patients (10.41%), all in the
critically ill category, were found to have detectable serum SARS-CoV-2
RNA levels, so-called RNAaemia. Moreover, serum IL-6 levels in these
patients were found to be substantially higher and this correlated with
the presence of detectable SARS-CoV-2 RNA levels. The authors
hypothesize that viral RNA might be released from acutely damages
tissues in moribund patients during the course of Covid19 and that
RNaemia along with IL-6 could potentially be used as a prognostic
marker.</p>
<h3 data-number="9.128.3" id="limitations-124"><span class="header-section-number">9.128.3</span> Limitations</h3>
<p>While this group’s report generally confirms some of the major findings
of a more extensive study, published in early February 2020, <span class="citation" data-cites="vasHgjGH">[<a href="#ref-vasHgjGH" role="doc-biblioref">565</a>]</span>,
there are limitations that should be taken into account. First, the
number of patients enrolled is relatively small; second, interpretation
of these data would benefit from inclusion of information about study
specifics as well as providing relevant data on the clinical course of
these patients other than the fact that some were admitted to ICU (i.e.
demographics on how many patients needed respiratory support, dialysis,
APACHE Ii/III or other standard ICU scores as robust prognostic markers
for mortality etc). It also remains unclear at which time point the
serum samples were taken, i.e. whether at admission, when the diagnosis
was made or during the course of the hospital stay (and potentially
after onset of therapy, which could have affected both IL-6 and RNA
levels). The methods section lacks important information on the qPCR
protocol employed, including primers and cycling conditions used. From a
technical point of view, Ct values &gt;35 seem somewhat non-specific
(although Ct &lt;40 was defined as the CDC cutoff as well) indicating
that serum RNA levels are probably very low, therefore stressing the
need for highly specific primers and high qPCR efficiency. In addition,
the statistical tests used (t-tests, according to the methods section)
do not seem appropriate as the organ-specific data such as BUN and
troponin T values seem to be not normally distributed across groups (n=
5 RNAaemia+ vs. n= 43 RNAaemia-). Given the range of standard deviations
and the differences in patient sample size, it is difficult to believe
that these data are statistically significantly different.</p>
<h3 data-number="9.128.4" id="significance-115"><span class="header-section-number">9.128.4</span> Significance</h3>
<p>This study is very rudimentary and lacks a lot of relevant clinical
details. However, it corroborates some previously published observations
regarding RNAemia and IL-6 by another group. Generally, regarding future
studies, it would be important to address the question of IL-6 and other
inflammatory cytokine dynamics in relation to Covid19 disease kinetics
(high levels of IL-6, IL-8 and plasma leukotriene were shown to have
prognostic value at the onset of ARDS ; serum IL-2 and IL-15 have been
associated with mortality; reviewed by Chen W &amp; Ware L, Clin Transl Med.
2015 <span class="citation" data-cites="l3CzJ5ni">[<a href="#ref-l3CzJ5ni" role="doc-biblioref">576</a>]</span>).</p>
<h3 data-number="9.128.5" id="credit-124"><span class="header-section-number">9.128.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.129" id="lymphopenia-predicts-disease-severity-of-covid-19-a-descriptive-and-predictive-study-1"><span class="header-section-number">9.129</span> Lymphopenia predicts disease severity of COVID-19: a descriptive and predictive study</h2>
<p><span class="citation" data-cites="uIwTzZkJ">[<a href="#ref-uIwTzZkJ" role="doc-biblioref">577</a>]</span></p>
<h3 data-number="9.129.1" id="keywords-128"><span class="header-section-number">9.129.1</span> Keywords</h3>
<ul>
<li>Lymphopenia</li>
</ul>
<h3 data-number="9.129.2" id="main-findings-124"><span class="header-section-number">9.129.2</span> Main Findings</h3>
<p>Based on a retrospective study of 162 COVID patients from a
local hospital in Wuhan, China, the authors show an inverse correlation
between lymphocyte % (LYM%) of patients and their disease severity. The
authors have also tracked LYM% of 70 cases (15 deaths; 15 severe; 40
moderate) throughout the disease progression with fatal cases showing no
recovery of lymphocytes ( &lt;5%) even after 17-19 days post-onset. The
temporal data of LYM % in COVID patients was used to construct a
Time-Lymphocyte% model which is used to categorize and predict patients’
disease severity and progression. The model was validated using 92
hospitalized cases and kappa statistic test was used to assess agreement
between predicted disease severity and the assigned clinical severity (k
= 0.49).</p>
<h3 data-number="9.129.3" id="limitations-125"><span class="header-section-number">9.129.3</span> Limitations</h3>
<p>Time-Lymphocyte % Model (TLM) that authors have proposed as
a predictive model for clinical severity is very simple in its
construction and derives from correlative data of 162 patients. In order
for the model to be of use, it needs validation using a far more robust
data set and possibly a mechanistic study on how COVID leads to
lymphopenia in the first place. In addition, it should be noted that no
statistical test assessing significance of LYM % values between disease
severities was performed.</p>
<h3 data-number="9.129.4" id="significance-116"><span class="header-section-number">9.129.4</span> Significance</h3>
<p>This article is of limited significance as
it simply reports similar descriptions of COVID patients made in
previous literature that severe cases are characterized by lymphopenia.</p>
<h3 data-number="9.129.5" id="credit-125"><span class="header-section-number">9.129.5</span> Credit</h3>
<p><em>Review by Chang Moon as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.130" id="the-potential-role-of-il-6-in-monitoring-severe-case-of-coronavirus-disease-2019-1"><span class="header-section-number">9.130</span> The potential role of IL-6 in monitoring severe case of coronavirus disease 2019</h2>
<p>Liu et al. <em>medRxiv</em>. <span class="citation" data-cites="1GMxHWnxN">[<a href="#ref-1GMxHWnxN" role="doc-biblioref">578</a>]</span></p>
<h3 data-number="9.130.1" id="keywords-129"><span class="header-section-number">9.130.1</span> Keywords</h3>
<ul>
<li>Cytokine Release Syndrome</li>
<li>lymphocytopenia</li>
<li>IL-6</li>
<li>CRP</li>
<li>COVID19</li>
<li>pneumonia</li>
</ul>
<h3 data-number="9.130.2" id="main-findings-125"><span class="header-section-number">9.130.2</span> Main Findings</h3>
<p>Study on blood biomarkers on 80 COVID19 patients (69 severe
and 11 non-severe). Patients with severe symptoms at admission
(baseline) showed obvious lymphocytopenia and significantly increased
interleukin-6 (IL-6) and CRP, which was positively correlated with
symptoms severity. IL-6 at baseline positively correlates with CRP, LDH,
ferritin and D-Dimer abundance in blood.</p>
<p>Longitudinal analysis of 30 patients (before and after treatment) showed
significant reduction of IL-6 in remission cases.</p>
<h3 data-number="9.130.3" id="limitations-126"><span class="header-section-number">9.130.3</span> Limitations</h3>
<p>Limited sample size at baseline, especially for the
non-severe leads to question on representativeness. The longitudinal
study method is not described in detail and suffers from
non-standardized treatment. Limited panel of pro-inflammatory cytokine
was analyzed. Patients with severe disease show a wide range of altered
blood composition and biomarkers of inflammation, as well as differences
in disease course (53.6% were cured, about 10% developed acute
respiratory distress syndrome). The authors comment on associations
between IL-6 levels and outcomes, but these were not statistically
significant (maybe due to the number of patients, non-standardized
treatments, etc.) and data is not shown. Prognostic biomarkers could
have been better explored. Study lacks multivariate analysis.</p>
<h3 data-number="9.130.4" id="significance-117"><span class="header-section-number">9.130.4</span> Significance</h3>
<p>IL-6 could be used as a pharmacodynamic
marker of disease severity. Cytokine Release Syndrome (CRS) is a
well-known side effect for CAR-T cancer therapy and there are several
effective drugs to manage CRS. Drugs used to manage CRS could be tested
to treat the most severe cases of COVID19.</p>
<h3 data-number="9.130.5" id="credit-126"><span class="header-section-number">9.130.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.131" id="clinical-and-laboratory-profiles-of-75-hospitalized-patients-with-novel-coronavirus-disease-2019-in-hefei-china-1"><span class="header-section-number">9.131</span> Clinical and Laboratory Profiles of 75 Hospitalized Patients with Novel Coronavirus Disease 2019 in Hefei, China</h2>
<p>Zhao et al. <em>medRxiv</em>. <span class="citation" data-cites="1B0vBwv1q">[<a href="#ref-1B0vBwv1q" role="doc-biblioref">579</a>]</span></p>
<h3 data-number="9.131.1" id="keywords-130"><span class="header-section-number">9.131.1</span> Keywords</h3>
<ul>
<li>Routine laboratory testing</li>
</ul>
<h3 data-number="9.131.2" id="main-findings-126"><span class="header-section-number">9.131.2</span> Main Findings</h3>
<p>The authors of this study provide a comprehensive analysis of clinical
laboratory assessments in 75 patients (median age 47 year old)
hospitalized for Corona virus infection in China measuring differential
blood counts including T-cell subsets (CD4, CD8), coagulation function,
basic blood chemistry, of infection-related biomarkers including CRP,
Procalcitonin (PCT) (Precursor of calcitonin that increases during
bacterial infection or tissue injury), IL-6 and erythrocyte
sedimentation rate as well as clinical parameters. Among the most common
hematological changes they found increased neutrophils, reduced CD4 and
CD8 lymphocytes, increased LDH, CRP and PCT</p>
<p>When looking at patients with elevated IL-6, the authors describe
significantly reduced CD4 and CD8 lymphocyte counts and elevated CRP and
PCT levels were significantly increased in infected patients suggesting
that increased IL-6 may correlate well with disease severity in COVID-19
infections</p>
<h3 data-number="9.131.3" id="limitations-127"><span class="header-section-number">9.131.3</span> Limitations</h3>
<p>The authors performed an early assessment of clinical standard
parameters in patients infected with COVID-19. Overall, the number of
cases (75) is rather low and the snapshot approach does not inform about
dynamics and thus potential relevance in the assessment of treatment
options in this group of patients.</p>
<h3 data-number="9.131.4" id="significance-118"><span class="header-section-number">9.131.4</span> Significance</h3>
<p>The article summarizes provides a good summary of some of the common
changes in immune cells inflammatory cytokines in patients with a
COVID-19 infection and. Understanding how these changes can help predict
severity of disease and guide therapy including IL-6 cytokine receptor
blockade using Tocilizumab or Sarilumab will be important to explore.</p>
<h3 data-number="9.131.5" id="credit-127"><span class="header-section-number">9.131.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.132" id="exuberant-elevation-of-ip-10-mcp-3-and-il-1ra-during-sars-cov-2-infection-is-associated-with-disease-severity-and-fatal-outcome-1"><span class="header-section-number">9.132</span> Exuberant elevation of IP-10, MCP-3 and IL-1ra during SARS-CoV-2 infection is associated with disease severity and fatal outcome</h2>
<p>Yang et al. <em>medRxiv</em> <span class="citation" data-cites="RyXNw87v">[<a href="#ref-RyXNw87v" role="doc-biblioref">580</a>]</span></p>
<h3 data-number="9.132.1" id="keywords-131"><span class="header-section-number">9.132.1</span> Keywords</h3>
<ul>
<li>cytokine</li>
<li>IP-10</li>
<li>MCP-3</li>
<li>IL-1Ra</li>
<li>lymphocyte</li>
<li>neutrophil</li>
<li>stratification</li>
<li>disease severity</li>
<li>viral load</li>
<li>lung function</li>
<li>complications</li>
<li>clinical data</li>
</ul>
<h3 data-number="9.132.2" id="summary-6"><span class="header-section-number">9.132.2</span> Summary</h3>
<p>Plasma cytokine analysis (48 cytokines) was
performed on COVID-19 patient plasma samples, who were sub-stratified as
severe (N=34), moderate (N=19), and compared to healthy controls (N=8).
Patients were monitored for up to 24 days after illness onset: viral
load (qRT-PCR), cytokine (multiplex on subset of patients), lab tests,
and epidemiological/clinical characteristics of patients were reported.</p>
<h3 data-number="9.132.3" id="main-findings-127"><span class="header-section-number">9.132.3</span> Main Findings</h3>
<ul>
<li><p>Many elevated cytokines with COVID-19 onset compared to healthy
controls (IFNy, IL-1Ra, IL-2Ra, IL-6, IL-10, IL-18, HGF, MCP-3, MIG,
M-CSF, G-CSF, MIG-1a, and IP-10).</p></li>
<li><p>IP-10, IL-1Ra, and MCP-3 (esp. together) were associated with
disease severity and fatal outcome.</p></li>
<li><p>IP-10 was correlated to patient viral load (r=0.3006, p=0.0075).</p></li>
<li><p>IP-10, IL-1Ra, and MCP-3 were correlated to loss of lung function
(PaO<sub>2</sub>/FaO<sub>2</sub> (arterial/atmospheric O2) and Murray Score (lung
injury) with MCP-3 being the most correlated (r=0.4104 p&lt;0.0001
and r=0.5107 p&lt;0.0001 respectively).</p></li>
<li><p>Viral load (Lower Ct Value from qRT-PCR) was associated with
upregulated IP-10 only (not IL-1Ra or MCP-3) and was mildly
correlated with decreased lung function: PaO<sub>2</sub>/FaO<sub>2</sub>
(arterial/atmospheric O2) and Murray Score (lung injury).</p></li>
<li><p>Lymphopenia (decreased CD4 and CD8 T cells) and increased neutrophil
correlated w/ severe patients.</p></li>
<li><p>Complications were associated with COVID severity (ARDS, hepatic
insufficiency, renal insufficiency).</p></li>
</ul>
<h3 data-number="9.132.4" id="limitations-128"><span class="header-section-number">9.132.4</span> Limitations</h3>
<p>Collection time of clinical data and lab results
not reported directly (likely 4 days (2,6) after illness onset), making
it very difficult to determine if cytokines were predictive of patient
outcome or reflective of patient compensatory immune response (likely
the latter). Small N for cytokine analysis (N=2 fatal and N=5
severe/critical, and N=7 moderate or discharged). Viral treatment
strategy not clearly outlined.</p>
<h3 data-number="9.132.5" id="expanded-results-1"><span class="header-section-number">9.132.5</span> Expanded Results</h3>
<p><em>NOTE</em>: Moderate COVID-19 was classified by fever, respiratory
manifestations, and radiological findings consistent with pneumonia
while severe patients had one or more of the following: 1) respiratory
distraction, resting O2 saturation, or 3) arterial PaO2/FiO2 <em>&lt;</em> 300
mmHg.</p>
<p>Cytokine Results (Human Cytokine Screening Panel, Bio-Rad):</p>
<ul>
<li><p><strong>Significant elevation of cytokines observed in COVID patients
compared to healthy controls: IFNy, IL-1Ra, IL-2Ra, IL-6, IL-10,
IL-18, HGF, MCP-3, MIG, M-CSF, G-CSF, MIG-1a, and IP-10.</strong></p></li>
<li><p>Severity was correlated <strong>with increase in measured IP-10, MCP-3,
and IL-Ra</strong> as measure by area under the curve analysis during
sample timecourse (2-24 days after illness onset).</p></li>
<li><p>IL-1Ra incr. significant 0-7 days after onset, MCP-3 signif.
upregulated throughout observation timecourse, and IP-10 increased
and upregulated throughout (trending downwards over time).</p></li>
<li><p><strong>The three cytokines together (IP-10, IL-1Ra, and MCP-3 AUC) served
as the best predictors of disease deterioration and fatal outcome.</strong></p></li>
<li><p>No significant differences between moderate/severe observed between
groups in IL-2Ra, IL-6, IL-10, IL-18, CTACK, G-CSF, HGF, M-CSF,
MIP-1a, MIG, and IFNy at any timepoints.</p></li>
<li><p><strong>Viral load (Lower Ct Value from qRT-PCR) was associated with
upregulated IP-10 only (not IL-1Ra or MCP-3) and was highly
correlated with decreased lung function: PaO<sub>2</sub>/FaO<sub>2</sub>
(arterial/atmospheric O2) and Murray Score (lung injury).</strong></p></li>
<li><p><strong>Antibodies against these cytokines (esp. anti-IP-10) may serve as
a potential treatment for amelioration of COVID-19 (and associated
ARDS).</strong></p></li>
</ul>
<p>Lab results:</p>
<ul>
<li><p><strong>Decreased lymphocytes (%) in all patients – lymphopenia corr. w/
severe patients</strong></p>
<ul>
<li><strong>Decreased CD4 and CD8 T cells</strong> – no monocyte or
eosino/basophil % measured</li>
</ul></li>
<li><p><strong>Increased neutrophils (%)</strong></p></li>
<li><p>Increased BUN (mmol/L) – other kidney markers, liver markers, and
LDH were not significantly different between groups and were not
compared to healthy controls.</p></li>
</ul>
<p>Clinical features (between moderate vs. severe patient groups):</p>
<ul>
<li><p><strong>Complications were associated with severity (ARDS, hepatic
insufficiency, renal insufficiency).</strong></p></li>
<li><p>Coexisting conditions between groups were not significantly
different (chronic heart/lung/renal/liver disease, diabetes, or
cancer) and patient time courses (onset to admission and onset to
viral tx) also not significantly different – 4 days (2, 6) on
average for admission and 4 (3,7) for antiviral.</p></li>
<li><p>Increased corticosteroids and mechanical/ invasive mechanical
ventilation in severe patients.</p></li>
<li><p>Increased median age in severe group (Median (Range = 63.5 (42-74)
vs. 51 (22-78)) and patients <em>&gt;</em> 60 yrs had higher ratio of
severe patients as compared patients 16-59 yrs.</p></li>
<li><p>Higher incidence of fever is severe patients (91.2 vs. 68.4%),
myalgia (57.7 vs. 48.1%), and chill (17.6% vs. 0%).</p></li>
<li><p>No differences in cough, headache, nausea/vomiting, or diarrhea.</p></li>
</ul>
<h3 data-number="9.132.6" id="significance-119"><span class="header-section-number">9.132.6</span> Significance</h3>
<p>Outline of pathological time course (implicating innate
immunity esp.) and identification key cytokines associated with disease
severity and prognosis (+ comorbidities). Anti-IP-10 as a possible
therapeutic intervention (ex: Eldelumab).</p>
<h3 data-number="9.132.7" id="credit-128"><span class="header-section-number">9.132.7</span> Credit</h3>
<p><em>Review by Natalie Vaninov as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.133" id="antibody-responses-to-sars-cov-2-in-patients-of-novel-coronavirus-disease-2019-1"><span class="header-section-number">9.133</span> Antibody responses to SARS-CoV-2 in patients of novel coronavirus disease 2019</h2>
<p>Zhao Jr. et al. <em>medRxiv</em>. <span class="citation" data-cites="5HXuCa5t">[<a href="#ref-5HXuCa5t" role="doc-biblioref">581</a>]</span></p>
<h3 data-number="9.133.1" id="keywords-132"><span class="header-section-number">9.133.1</span> Keywords</h3>
<ul>
<li>SARS-CoV-2 IgG</li>
<li>seroconversion rate</li>
<li>total Ab</li>
<li>Ig</li>
<li>IgM</li>
</ul>
<h3 data-number="9.133.2" id="main-findings-128"><span class="header-section-number">9.133.2</span> Main Findings</h3>
<p>This study examined antibody responses in
the blood of COVID-19 patients during the early SARS CoV2 outbreak in
China. Total 535 plasma samples were collected from 173 patients (51.4%
female) and were tested for seroconversion rate using ELISA. Authors
also compared the sensitivity of RNA and antibody tests over the course
of the disease . The key findings are:</p>
<ul>
<li><p>Among 173 patients, the seroconversion rates for total antibody
(Ab), IgM and IgG were 93.1% (161/173), 82.7% (143/173) and 64.7%
(112/173), respectively.</p></li>
<li><p>The seroconversion sequentially appeared for Ab, IgM and then IgG,
with a median time of 11, 12 and 14 days, respectively. Overall, the
seroconversion of Ab was significantly quicker than that of IgM (p =
0.012) and IgG (p &lt; 0.001). Comparisons of seroconversion rates
between critical and non-critical patients did not reveal any
significant differences.</p></li>
<li><p>RNA tests had higher sensitivity in early phase and within 7 days of
disease onset than antibody assays (66.7% Vs 38.3% respectively).</p></li>
<li><p>The sensitivity of the Ab assays was higher 8 days after disease
onset, reached 90% at day 13 and 100% at later time points (15-39
days). In contrast, RNA was only detectable in 45.5% of samples at
days 15-39.</p></li>
<li><p>In patients with undetectable RNA in nasal samples collected during
day 1-3, day 4-7, day 8-14 and day 15-39 since disease onset, 28.6%
(2/7), 53.6% (15/28), 98.2% (56/57) and 100% (30/30) had detectable
total Ab titers respectively Combining RNA and antibody tests
significantly raised the sensitivity for detecting COVID-19 patients
in different stages of the disease (p &lt; 0.001).</p></li>
<li><p>There was a strong positive correlation between clinical severity
and antibody titer 2-weeks after illness onset.</p></li>
<li><p>Dynamic profiling of viral RNA and antibodies in representative
COVID-19 patients (n=9) since onset of disease revealed that
antibodies may not be sufficient to clear the virus. It should be
noted that increases in of antibody titers were not always
accompanied by RNA clearance.</p></li>
</ul>
<h3 data-number="9.133.3" id="limitations-129"><span class="header-section-number">9.133.3</span> Limitations</h3>
<p>Because different types of ELISA assays were used for
determining antibody concentrations at different time points after
disease onset, sequential seroconversion of total Ab, IgM and IgG may
not represent actual temporal differences but rather differences in the
affinities of the assays used. Also, due to the lack of blood samples
collected from patients in the later stage of illness, how long the
antibodies could last remain unknown. For investigative dynamics of
antibodies, more samples were required.</p>
<h3 data-number="9.133.4" id="significance-120"><span class="header-section-number">9.133.4</span> Significance</h3>
<p>Total and IgG antibody titers could be used to understand
the epidemiology of SARS CoV-2 infection and to assist in determining
the level of humoral immune response in patients.</p>
<p>The findings provide strong clinical evidence for routine serological
and RNA testing in the diagnosis and clinical management of COVID-19
patients. The understanding of antibody responses and their half-life
during and after SARS CoV2 infection is important and warrants further
investigations.</p>
<h3 data-number="9.133.5" id="credit-129"><span class="header-section-number">9.133.5</span> Credit</h3>
<p><em>This review was undertaken by Zafar Mahmood and edited by K Alexandropoulos as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.134" id="restoration-of-leukomonocyte-counts-is-associated-with-viral-clearance-in-covid-19-hospitalized-patients-1"><span class="header-section-number">9.134</span> Restoration of leukomonocyte counts is associated with viral clearance in COVID-19 hospitalized patients</h2>
<p>Chen et al. <em>medRxiv</em> <span class="citation" data-cites="uCyoHtV3">[<a href="#ref-uCyoHtV3" role="doc-biblioref">582</a>]</span></p>
<h3 data-number="9.134.1" id="keywords-133"><span class="header-section-number">9.134.1</span> Keywords</h3>
<ul>
<li>COVID-19</li>
<li>T cell</li>
<li>B cell</li>
<li>NK cell</li>
<li>IL-6</li>
<li>pro-calcitonin</li>
<li>cytokine storm</li>
</ul>
<h3 data-number="9.134.2" id="main-findings-129"><span class="header-section-number">9.134.2</span> Main Findings</h3>
<p>The authors collected data on 25 COVID-19 patients (n=11 men, n=14
women) using standard laboratory tests and flow cytometry. All patients
were treated with antibiotics. Twenty-four of the 25 patients were also
treated with anti-viral Umefinovir and 14 of the patients were treated
with corticosteroids. 14 patients became negative for the virus after
8-14 days of treatment. The same treatment course was extended to 15-23
days for patients who were still positive for the virus at day 14.</p>
<p>The authors found a negative association between age and resolution of
infection. Patients with hypertension, diabetes, malignancy or chronic
liver disease were all unable to clear the virus at day 14, though not
statistically significant.</p>
<p>Elevated procalcitonin and a trend for increased IL-6 were also found in
peripheral blood prior to the treatment.</p>
<p>A trend for lower NK cell, T cell and B cell counts in patients was also
reported. B cell, CD4 and CD8 T cell counts were only increased upon
treatment in patients who cleared the virus. NK cell frequencies
remained unchanged after treatment in all the patients.</p>
<h3 data-number="9.134.3" id="limitations-130"><span class="header-section-number">9.134.3</span> Limitations</h3>
<p>73% of the patients who remained positive for SARS-CoV2 after the 1<sup>st</sup>
treatment, and 43% of all patients who cleared the virus were treated
with corticosteroids. Corticosteroids have strong effects on the immune
compartment in blood <span class="citation" data-cites="L6kwccjn">[<a href="#ref-L6kwccjn" role="doc-biblioref">583</a>]</span>. The authors should have accounted for
corticosteroid treatment when considering changes in T, NK and B cell
frequencies.</p>
<p>Assessing if IL-6 concentrations were back to baseline levels following
treatment would have provided insights into the COVID-19 cytokine storm
biology. Patients with higher baseline levels of IL-6 have been reported
to have lower CD8 and CD4 T cell frequencies <span class="citation" data-cites="1B0vBwv1q">[<a href="#ref-1B0vBwv1q" role="doc-biblioref">579</a>]</span>. Correlating IL-6 with
cell counts before and after treatment would thus have also been of
interest. The report of the laboratory measures in table 2 is incomplete
and should include the frequencies of patients with increased/decreased
levels for each parameter.</p>
<p>Correction is needed for the 1<sup>st</sup> paragraph of the discussion as data
does not support NK cell restoration upon treatment in patients who
cleared the virus. NK cells remain unchanged after the 1<sup>st</sup> treatment
course and only seem to increase in 2 out of 6 donors after the 2<sup>nd</sup>
treatment course in those patients.</p>
<h3 data-number="9.134.4" id="significance-121"><span class="header-section-number">9.134.4</span> Significance</h3>
<p>Previous reports suggest an association between disease severity and
elevated IL-6 or pro-calcitonin concentrations in COVID-19
patients <span class="citation" data-cites="T0BDaXkw K9G9QT8y">[<a href="#ref-T0BDaXkw" role="doc-biblioref">575</a>,<a href="#ref-K9G9QT8y" role="doc-biblioref">584</a>]</span>. IL-6 receptor blockade is also being administered to
patients enrolled in clinical trials (NCT04317092). This report thus
contributes to highlight elevated concentrations of these analytes in
COVID-19 patients. Mechanisms underlying the association between viral
clearance and restoration of the T cell and B cell frequencies suggests
viral-driven immune dysregulation, which needs to be investigated in
further studies.</p>
<h3 data-number="9.134.5" id="credit-130"><span class="header-section-number">9.134.5</span> Credit</h3>
<p><em>Review by Bérengère Salomé as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.135" id="clinical-findings-in-critically-ill-patients-infected-with-sars-cov-2-in-guangdong-province-china-a-multi-center-retrospective-observational-study-1"><span class="header-section-number">9.135</span> Clinical findings in critically ill patients infected with SARS-CoV-2 in Guangdong Province, China: a multi-center, retrospective, observational study</h2>
<p>Xu et al. <em>medRxiv</em>. <span class="citation" data-cites="USOnp7c0">[<a href="#ref-USOnp7c0" role="doc-biblioref">585</a>]</span></p>
<h3 data-number="9.135.1" id="keywords-134"><span class="header-section-number">9.135.1</span> Keywords</h3>
<ul>
<li>clinical outcomes</li>
<li>prognosis</li>
<li>critically ill patients</li>
<li>ICU</li>
<li>lymphopenia</li>
<li>LDH</li>
</ul>
<h3 data-number="9.135.2" id="main-findings-130"><span class="header-section-number">9.135.2</span> Main Findings:</h3>
<p>This work analyses laboratory and clinical data from 45 patients treated
in the in ICU in a single province in China. Overall, 44% of the
patients were intubated within 3 days of ICU admission with only 1
death.</p>
<p>Lymphopenia was noted in 91% of patient with an inverse correlation with
LDH.</p>
<p>Lymphocyte levels are negatively correlated with Sequential Organ
Failure Assessment (SOFA) score (clinical score, the higher the more
critical state), LDH levels are positively correlated to SOFA score.
Overall, older patients (&gt;60yo), with high SOFA score, high LDH
levels and low lymphocytes levels at ICU admission are at higher risk of
intubation.</p>
<p>Of note, convalescent plasma was administered to 6 patients but due to
limited sample size no conclusion can be made.</p>
<h3 data-number="9.135.3" id="limitations-131"><span class="header-section-number">9.135.3</span> Limitations</h3>
<p>While the study offers important insights into disease course and
clinical lab correlates of outcome, the cohort is relatively small and
is likely skewed towards a less-severe population compared to other ICU
reports given the outcomes observed. Analysis of laboratory values and
predictors of outcomes in larger cohorts will be important to make
triage and treatment decisions. As with many retrospective analyses,
pre-infection data is limited and thus it is not possible to understand
whether lymphopenia was secondary to underlying comorbidities or
infection.</p>
<p>Well-designed studies are necessary to evaluate the effect of
convalescent plasma administration.</p>
<h3 data-number="9.135.4" id="significance-122"><span class="header-section-number">9.135.4</span> Significance</h3>
<p>This clinical data enables the identification of at-risk
patients and gives guidance for research for treatment options. Indeed,
further work is needed to better understand the causes of the
lymphopenia and its correlation with outcome.</p>
<h3 data-number="9.135.5" id="credit-131"><span class="header-section-number">9.135.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.136" id="immune-multi-epitope-vaccine-design-using-an-immunoinformatics-approach-for-2019-novel-coronavirus-in-china-sars-cov-2-1"><span class="header-section-number">9.136</span> Immune Multi-epitope vaccine design using an immunoinformatics approach for 2019 novel coronavirus in China (SARS-CoV-2)</h2>
<p><span class="citation" data-cites="14OLQIYK0">[<a href="#ref-14OLQIYK0" role="doc-biblioref">586</a>]</span></p>
<h3 data-number="9.136.1" id="keywords-135"><span class="header-section-number">9.136.1</span> Keywords</h3>
<ul>
<li>Vaccine</li>
<li>in silico</li>
<li>B cell epitopes</li>
<li>T cell epitopes</li>
</ul>
<h3 data-number="9.136.2" id="main-findings-131"><span class="header-section-number">9.136.2</span> Main Findings</h3>
<p>Using in silico bioinformatic tools, this study identified putative
antigenic B-cell epitopes and HLA restricted T-cell epitopes from the
spike, envelope and membrane proteins of SARS-CoV-2, based on the genome
sequence available on the NCBI database. T cell epitopes were selected
based on predicted affinity for the more common HLA-I alleles in the
Chinese population. Subsequently, the authors designed vaccine peptides
by bridging selected B-cell epitopes and adjacent T-cell epitopes.
Vaccine peptides containing only T-cell epitopes were also generated.</p>
<p>From 61 predicted B-cell epitopes, only 19 were exposed on the surface
of the virion and had a high antigenicity score. A total of 499 T-cell
epitopes were predicted. Based on the 19 B-cell epitopes and their 121
adjacent T-cell epitopes, 17 candidate vaccine peptides were designed.
Additionally, another 102 vaccine peptides containing T-cell epitopes
only were generated. Based on the epitope counts and HLA score, 13 of
those were selected. Thus, a total of 30 peptide vaccine candidates were
designed.</p>
<h3 data-number="9.136.3" id="limitations-132"><span class="header-section-number">9.136.3</span> Limitations</h3>
<p>While this study provides candidates for the development of vaccines
against SARS-CoV-2, in vitro and in vivo trials are required to validate
the immunogenicity of the selected B and T cell epitopes. This could be
done using serum and cells from CoV-2-exposed individuals, and in
preclinical studies. The implication of this study for the current
epidemic are thus limited. Nevertheless, further research on this field
is greatly needed.</p>
<h3 data-number="9.136.4" id="credit-132"><span class="header-section-number">9.136.4</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.137" id="clinical-features-of-patients-infected-with-the-2019-novel-coronavirus-covid-19-in-shanghai-china-1"><span class="header-section-number">9.137</span> Clinical Features of Patients Infected with the 2019 Novel Coronavirus (COVID-19) in Shanghai, China</h2>
<p>Cao et al. <em>medRxiv</em> <span class="citation" data-cites="12ilBRq6I">[<a href="#ref-12ilBRq6I" role="doc-biblioref">587</a>]</span></p>
<h3 data-number="9.137.1" id="keywords-136"><span class="header-section-number">9.137.1</span> Keywords</h3>
<ul>
<li>Disease severity</li>
<li>clinical features</li>
<li>laboratory abnormalities</li>
</ul>
<h3 data-number="9.137.2" id="main-findings-132"><span class="header-section-number">9.137.2</span> Main Findings</h3>
<p>This single-center cohort study analyzes the clinical and
laboratory features of 198 patients with confirmed COVID-19 infection in
Shanghai, China and correlated these parameters with clinical disease
severity, including subsequent intensive care unit (ICU) admission. 19
cases (9.5%) required ICU admission after developing respiratory failure
or organ dysfunction. Age, male sex, underlying cardiovascular disease,
and high symptom severity (high fever, dyspnea) were all significantly
correlated with ICU admission. Additionally, ICU admission was more
common in patients who presented with lymphopenia and elevated
neutrophil counts, among other laboratory abnormalities. Flow cytometric
analysis revealed that patients admitted to the ICU had significantly
reduced circulating CD3+ T cell, CD4+ T cell, CD8+ T cell, and CD45+
leukocyte populations compared to the cohort of patients not requiring
ICU admission.</p>
<h3 data-number="9.137.3" id="limitations-133"><span class="header-section-number">9.137.3</span> Limitations</h3>
<p>The limitations of this study include the relatively
small sample size and lack of longitudinal testing. The authors also did
not assess whether respiratory comorbidity – such as asthma or chronic
obstructive lung disease – in addition to immunosuppression affected ICU
admission likelihood.</p>
<h3 data-number="9.137.4" id="significance-123"><span class="header-section-number">9.137.4</span> Significance</h3>
<p>COVID-19 has already sickened thousands across the globe,
though the severity of these infections is markedly diverse, ranging
from mild symptoms to respiratory failure requiring maximal
intervention. Understanding what clinical, laboratory, and immunologic
factors predict the clinical course of COVID-19 infection permits
frontline providers to distribute limited medical resources more
effectively.</p>
<h3 data-number="9.137.5" id="credit-133"><span class="header-section-number">9.137.5</span> Credit</h3>
<p><em>Review by Andrew Charap as part of a project by students, postdocs and
faculty at the Immunology Institute of the Icahn School of Medicine at
Mount Sinai.</em></p>
<h2 data-number="9.138" id="serological-detection-of-2019-ncov-respond-to-the-epidemic-a-useful-complement-to-nucleic-acid-testing-1"><span class="header-section-number">9.138</span> Serological detection of 2019-nCoV respond to the epidemic: A useful complement to nucleic acid testing</h2>
<p>Zhang et al. <em>medRxiv</em>. <span class="citation" data-cites="IPJURDxi">[<a href="#ref-IPJURDxi" role="doc-biblioref">588</a>]</span></p>
<h3 data-number="9.138.1" id="keywords-137"><span class="header-section-number">9.138.1</span> Keywords</h3>
<ul>
<li>Immunoassay</li>
<li>serum IgM and IgG</li>
<li>specific antibodies</li>
</ul>
<h3 data-number="9.138.2" id="main-finding-2"><span class="header-section-number">9.138.2</span> Main Finding</h3>
<p>This study showed that both anti-2019-nCov IgM and IgG were detected by
automated chemiluminescent immunoassay in the patients who had been
already confirmed as positive by nucleic acid detection, while single
positivity of IgM or IgG were detected in a very few cases in the other
population including 225 non-COVID-19 cases. In addition to the increase
of anti-2019-nCov IgM 7-12 days after morbidity, the increase of IgG was
detected in three patients with COVID-19 within a very short of time
(0-1 day).</p>
<h3 data-number="9.138.3" id="limitations-134"><span class="header-section-number">9.138.3</span> Limitations</h3>
<p>The limitation of this study is only
3 confirmed COVID-19 cases were included, so that the relationship
between anti-2019-nCov antibodies and disease progression might not be
clearly defined. Another limitation is that they did not show the course
of 2019-nCov specific antibodies in the cases with positive for COVID-19
but without clinical symptoms.</p>
<h3 data-number="9.138.4" id="significance-124"><span class="header-section-number">9.138.4</span> Significance</h3>
<p>The detection of anti-2019-nCov antibodies can be an alternative method
to diagnose and treat COVID-19 more comprehensively by distinguish non
COVID-19 patients. It may be helpful to understand the course of
individual cases with COVID-19 to predict the prognosis if more cases
will be evaluated.</p>
<h3 data-number="9.138.5" id="credit-134"><span class="header-section-number">9.138.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.139" id="human-kidney-is-a-target-for-novel-severe-acute-respiratory-syndrome-coronavirus-2-sars-cov-2-infection-1"><span class="header-section-number">9.139</span> Human Kidney is a Target for Novel Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) Infection</h2>
<p><span class="citation" data-cites="H0nvocJz">[<a href="#ref-H0nvocJz" role="doc-biblioref">589</a>]</span></p>
<h3 data-number="9.139.1" id="keywords-138"><span class="header-section-number">9.139.1</span> Keywords</h3>
<ul>
<li>Kidney/Renal Failure</li>
<li>Macrophage Infiltration</li>
<li>Complement Activation</li>
</ul>
<h3 data-number="9.139.2" id="main-finding-3"><span class="header-section-number">9.139.2</span> Main Finding</h3>
<p>Analyzing the eGFR (effective glomerular flow rate) of 85 Covid-19
patients and characterizing tissue damage and viral presence in
post-mortem kidney samples from 6 Covid-19 patients, the authors
conclude that significant damage occurs to the kidney, following
Covid-19 infection. This is in contrast to the SARS infection from the
2003 outbreak. They determine this damage to be more prevalent in
patients older than 60 years old, as determined by analysis of eGFR. H&amp;E
and IHC analysis in 6 Covid-19 patients revealed that damage was in the
tubules, not the glomeruli of the kidneys and suggested that macrophage
accumulation and C5b-9 deposition are key to this process. </p>
<h3 data-number="9.139.3" id="limitations-135"><span class="header-section-number">9.139.3</span> Limitations</h3>
<p>Severe limitations include that the H&amp;E and IHC samples were performed
on post-mortem samples of unknown age, thus we cannot assess how/if age
correlates with kidney damage, upon Covid-19 infection. Additionally,
eGFR was the only <em>in-vivo</em> measurement. Blood urea nitrogen and
proteinuria are amongst other measurements that could have been obtained
from patient records. An immune panel of the blood was not performed to
assess immune system activation. Additionally, patients are only from
one hospital. </p>
<h3 data-number="9.139.4" id="significance-125"><span class="header-section-number">9.139.4</span> Significance</h3>
<p>This report makes clear that kidney damage is prevalent
in Covid-19 patients and should be accounted for. </p>
<h3 data-number="9.139.5" id="credit-135"><span class="header-section-number">9.139.5</span> Credit</h3>
<p><em>Review by Dan Fu Ruan, Evan Cody and Venu Pothula as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine at Mount Sinai.</em></p>
<h2 data-number="9.140" id="covid-19-early-warning-score-a-multi-parameter-screening-tool-to-identify-highly-suspected-patients-1"><span class="header-section-number">9.140</span> COVID-19 early warning score: a multi-parameter screening tool to identify highly suspected patients</h2>
<p>Song et al. <em>medRxiv</em>. <span class="citation" data-cites="KHNnjP7M">[<a href="#ref-KHNnjP7M" role="doc-biblioref">590</a>]</span></p>
<h3 data-number="9.140.1" id="keywords-139"><span class="header-section-number">9.140.1</span> Keywords</h3>
<ul>
<li>retrospective</li>
<li>electronic health records</li>
<li>blood counts</li>
<li>diagnostic</li>
<li>prognostic</li>
<li>modeling</li>
</ul>
<h3 data-number="9.140.2" id="main-findings-133"><span class="header-section-number">9.140.2</span> Main Findings</h3>
<p>The aim of this study was to identify diagnostic or prognostic criteria
which could identify patients with COVID-19 and predict patients who
would go on to develop severe respiratory disease. The authors use EMR
data from individuals taking a COVID-19 test at Zhejiang hospital, China
in late January/Early February. A large number of clinical parameters
were different between individuals with COVID-19 and also between
‘severe’ and ‘non-severe’ infections and the authors combine these into
a multivariate linear model to derive a weighted score, presumably
intended for clinical use.</p>
<h3 data-number="9.140.3" id="limitations-136"><span class="header-section-number">9.140.3</span> Limitations</h3>
<p>Unfortunately, the paper is lacking a lot of crucial information, making
it impossible to determine the importance or relevance of the findings.
Most importantly, the timings of the clinical measurements are not
described relative to the disease course, so it is unclear if the
differences between ‘severe’ and ‘non-severe’ infections are occurring
before progression to severe disease (which would make them useful
prognostic markers), or after (which would not).</p>
<h3 data-number="9.140.4" id="significance-126"><span class="header-section-number">9.140.4</span> Significance</h3>
<p>This paper is one of many retrospective studies coming from hospitals in
China studying individuals with COVID-19. Because of the sparse
description of the study design, this paper offers little new
information. However, studies like this could be very valuable and we
would strongly encourage the authors to revise this manuscript to
include more information about the timeline of clinical measurements in
relation to disease onset and more details of patient outcomes.</p>
<h3 data-number="9.140.5" id="credit-136"><span class="header-section-number">9.140.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.141" id="ly6e-impairs-coronavirus-fusion-and-confers-immune-control-of-viral-disease-1"><span class="header-section-number">9.141</span> LY6E impairs coronavirus fusion and confers immune control of viral disease</h2>
<p>Pfaender et al. <em>bioRxiv</em>. <span class="citation" data-cites="19xwRoatu">[<a href="#ref-19xwRoatu" role="doc-biblioref">591</a>]</span></p>
<h3 data-number="9.141.1" id="keywords-140"><span class="header-section-number">9.141.1</span> Keywords</h3>
<ul>
<li>interferon-stimulated genes</li>
<li>antiviral interferons</li>
<li>human coronaviruses (CoV)</li>
<li>murine hepatitis virus (MHV)</li>
</ul>
<h3 data-number="9.141.2" id="main-findings-134"><span class="header-section-number">9.141.2</span> Main Findings</h3>
<p>Screening a cDNA library of &gt;350 human interferon-stimulated genes
for antiviral activity against endemic human coronavirus HCoV-229E
(associated with the common cold), Pfaender S &amp; Mar K <em>et al.</em> identify
lymphocyte antigen 6 complex, locus E (Ly6E) as an inhibitor of cellular
infection of Huh7 cells, a human hepatoma cell line susceptible to
HCoV-229E and other coronaviruses. In a series of consecutive <em>in vitro</em>
experiments including both stable Ly6E overexpression and
CRISPR-Cas9-mediated knockout the authors further demonstrate that Ly6E
reduces cellular infection by various other coronaviruses including
human SARS-CoV and SARS-CoV-2 as well as murine CoV mouse hepatitis
virus (MHV). Their experiments suggest that this effect is dependent on
Ly6E inhibition of CoV strain-specific spike protein-mediated membrane
fusion required for viral cell entry.</p>
<p>To address the function of Ly6E <em>in vivo</em>, hematopoietic stem
cell-specific Ly6E knock-out mice were generated by breeding Ly6E<sup>fl/fl</sup>
mice (referred to as functional wild-type mice) with transgenic
<em>Vav-iCre</em> mice (offspring referred to as Ly6E HSC ko mice); wild-type
and Ly6E HSC ko mice of both sexes were infected intraperitoneally with
varying doses of the natural murine coronavirus MHV, generally causing a
wide range of diseases in mice including hepatitis, enteritis and
encephalomyelitis. Briefly, compared to wild-type controls, mice lacking
hematopoietic cell-expressed Ly6E were found to present with a more
severe disease phenotype as based on serum ALT levels (prognostic of
liver damage), liver histopathology, and viral titers in the spleen.
Moreover, bulk RNAseq analysis of infected liver and spleen tissues
indicated changes in gene expression pathways related to tissue damage
and antiviral immune responses as well as a reduction of genes
associated with type I IFN response and inflammation. Finally, the
authors report substantial differences in the numbers of hepatic and
splenic APC subsets between wild-type and knockout mice following MHV
infection and show that Ly6E-deficient B cells and to a lesser extent
also DCs are particularly susceptible to MHV infection <em>in vitro</em>.</p>
<h3 data-number="9.141.3" id="limitations-137"><span class="header-section-number">9.141.3</span> Limitations</h3>
<p>Experiments and data in this study are presented in an overall logical
and coherent fashion; however, some observations and the conclusions
drawn are problematic and should be further addressed &amp; discussed by the
authors. Methodological &amp; formal limitations include relatively low
replicate numbers as well as missing technical replicates for some <em>in
vitro</em> experiments (<em>cf</em>. Fig. legend 1; Fig. legend 2e); the omission
of “outliers” in Fig. legend 2 without an apparent rationale as to why
this approach was chosen; the lack of detection of actual Ly6E protein
levels in Ly6E HSC ko or wild-type mice; and most importantly, missing
information on RNAseq data collection &amp; analysis in the method section
and throughout the paper. A more relevant concern though is that the
interpretation of the experimental data presented and the language used
tend to overrate and at times overgeneralize findings: for example,
while the authors demonstrate statistically significant, Ly6E-mediated
reduction of coronavirus titers in stable cells lines <em>in vitro</em>, it
remains unclear whether a viral titer reduction by one log decade would
be of actual biological relevance in face of high viral titers <em>in
vivo</em>. After high-dose intraperitoneal MHV infection <em>in vivo</em>, early
viral titers in Ly6E HSC knockout vs. wt mice only showed an elevation
in the spleen (~1.5 log decades) but not liver of the ko mice (other
tissue not evaluated), and while ko mice presented with only modestly
increased liver pathology, both male and female ko mice exhibited
significantly higher mortality. Thus, the manuscript tile statement that
“Ly6E … confers immune control of viral disease” is supported by only
limited <em>in vivo</em> data, and gain-of-function experiments (eg. Ly6E
overexpression) were not performed. Of additional note here, tissue
tropism and virulence differ greatly among various MHV strains and
isolates whereas dose, route of infection, age, genetic background and
sex of the mice used may additionally affect disease outcome and
phenotype (<em>cf</em>. Taguchi F &amp; Hirai-Yuki A,
<a href="https://doi.org/10.3389/fmicb.2012.00068" class="uri">https://doi.org/10.3389/fmicb.2012.00068</a>; Kanolkhar A et al,
<a href="https://jvi.asm.org/content/%2083/18/9258">https://jvi.asm.org/content/
83/18/9258</a>). Observations
attributed to hematopoietic stem cell-specific Ly6E deletion could
therefore be influenced by the different genetic backgrounds of floxed
and cre mice used, and although it appears that littermates wt and ko
littermates were used in the experiments, the potentially decisive
impact of strain differences should at least have been discussed. Along
these lines, it should also be taken into account that the majority of
human coronaviruses cause respiratory symptoms, which follow a different
clinical course engaging other primary cellular mediators than the
hepatotropic murine MHV disease studied here. It therefore remains
highly speculative how the findings reported in this study will
translate to human disease and it would therefore be important to test
other routes of MHV infection and doses that have been described to
produce a more comparable phenotype to human coronavirus disease (<em>cf.</em>
Kanolkhar A et al, <a href="https://jvi.asm.org/content/%2083/18/9258">https://jvi.asm.org/content/
83/18/9258</a>). Another
important shortcoming of this study is the lack of any information on
functional deficits or changes in Ly6E-deficient immune cells and how
this might relate to the phenotype observed. Overall, the <em>in vitro</em>
experiments are more convincing than the <em>in vivo</em> studies which appear
somewhat limited.</p>
<h3 data-number="9.141.4" id="significance-127"><span class="header-section-number">9.141.4</span> Significance</h3>
<p>Despite some shortcomings, the experiments performed in this study
suggest a novel and somewhat unexpected role of Ly6E in the protection
against coronaviruses across species. These findings are of relevance
and should be further explored in ongoing research on potential
coronavirus therapies. Yet an important caveat pertains to the authors’
suggestion that “therapeutic mimicking of Ly6E action” may constitute a
first line of defense against novel coronaviruses since their own prior
work demonstrated that Ly6E can enhance rather than curtail infection
with influenza A and other viruses.</p>
<h3 data-number="9.141.5" id="credit-137"><span class="header-section-number">9.141.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.142" id="a-preliminary-study-on-serological-assay-for-severe-acute-respiratory-syndrome-coronavirus-2-sars-cov-2-in-238-admitted-hospital-patients-1"><span class="header-section-number">9.142</span> A preliminary study on serological assay for severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) in 238 admitted hospital patients</h2>
<p>Liu et al. <em>medRxiv</em>. <span class="citation" data-cites="574aSa8C">[<a href="#ref-574aSa8C" role="doc-biblioref">592</a>]</span></p>
<h3 data-number="9.142.1" id="keywords-141"><span class="header-section-number">9.142.1</span> Keywords</h3>
<ul>
<li>diagnosis</li>
<li>serological assay</li>
<li>ELISA</li>
<li>RT-PCR</li>
</ul>
<h3 data-number="9.142.2" id="main-findings-135"><span class="header-section-number">9.142.2</span> Main Findings</h3>
<p>While RT-PCR is being used currently to routinely diagnose infection
with SARS-CoV-2, there are significant limitations to the use of a
nucleic acid test that lead to a high false-negative rate. This
article describes ELISAs that can measure IgM and IgG antibodies
against the N protein of SARS-CoV-2 to test samples from 238
patients (153 positive by RT-PCR and 85 negative by RT-PCR) at
different times after symptom onset. The positivity rate of the IgM
and/or IgG ELISAs was greater than that of the RT-PCR (81.5%
compared to 64.3%) with similar positive rates in the confirmed and
suspected cases (83% and 78.8%, respectively), suggesting that many
of the suspected but RT-PCR-negative cases were also infected. The
authors also found that the ELISAs have higher positive rates later
after symptom onset while RT-PCR is more effective as a diagnostic
test early during the infection.</p>
<h3 data-number="9.142.3" id="limitations-138"><span class="header-section-number">9.142.3</span> Limitations</h3>
<p>I cannot identify any limitations to this study.</p>
<h3 data-number="9.142.4" id="significance-128"><span class="header-section-number">9.142.4</span> Significance</h3>
<p>The authors make a strong case for using a combination of ELISA and
RT-PCR for diagnosis of infection with SARS-CoV-2, especially
considering the dynamics of positivity rates of RT-PCR and ELISA.
Fewer false-negative diagnoses would improve infection control and
patient management.</p>
<h3 data-number="9.142.5" id="credit-138"><span class="header-section-number">9.142.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<p><strong>Early, low-dose and short-term application of corticosteroid treatment
in patients with severe COVID-19 pneumonia: single-center experience
from Wuhan, China</strong></p>
<p>Corticosteroids, methylprednisolone, COVID-19 pneumonia, ARDS</p>
<p>This retrospective study evaluated the use of intravenous
methylprednisolone to treat severe COVID-19 pneumonia in a cohort of 46
patients. The severity of the disease was determined according to
version 5 of the Coronavirus Pneumonia Diagnosis and Treatment Plan by
the National Health Committee of the People’s Republic of China. The
percentage of patients with comorbidities was 32%, and it was reported
similar between methylprednisolone treated and untreated groups. The
results showed that the group of patients that received
methylprednisolone (n=26) had a shorter number of days with fever than
patients that did not received methylprednisolone (n=20); they also had
a faster improvement of peripheral capillary oxygen saturation (SpO2),
and better outcomes in follow-up CT scans of the lungs. The dosage of
methylprednisolone was reported to be 1-2mg/kg/d for 5-7 days, although
there is no information about the concrete dosages for each patient.
From the 46 patients, 43 recovered and were discharged, while 3 cases
were fatal. Patients without administration of methylprednisolone needed
longer periods of supplemental oxygen therapy, though there is no
reference to the number of patients requiring mechanical ventilation.
Interestingly, there were no significant differences in leucocyte and
lymphocyte counts nor in the levels of IL-2, IL-4, IL-6 or IL-10 after
treatment with methylprednisolone.</p>
<p>Some of our main criticisms to this study are also pointed-out by the
authors themselves: it is a retrospective single-center study with no
validation cohort and without mid- and long-term follow-ups. The
reported mortality was 7% (3/46) and did not appear to be affected by
corticosteroid treatment: one patient died in the group that did not
receive methylprednisolone, while two patients died in the
methylprednisolone treatment group. Additionally, although the authors
mention that patients received cotreatments, such as antiviral therapy
and antibiotics, there is no mention of differences between the
prevalence of other medications between the two groups. Unfortunately,
there is also no indication on whether the patients receiving
methylprednisolone were discharged earlier; the authors merely refer
that the symptoms and signs improved faster.</p>
<p>Corticosteroid have been widely used as therapy for acute respiratory
distress syndrome (ARDS), including in infections by SARS-CoV, so these
findings in COVID-19 patients are not unexpected. The implications of
this study for the current pandemic due to SARS-CoV-2 require evaluation
in future clinical trials, especially in a randomized way and in
combination with and comparison to other immunosuppressive and
immunomodulatory agents, including hydroxychloroquine. Nevertheless,
based on this report, the intravenous application of methylprednisolone
with the intention of strengthening the immunosuppressive treatment and
controlling the cytokine storm appears to be safe in COVID-19 patients,
and it might successfully shorten the recovery period.</p>
<p>This review was undertaken by Alvaro Moreira, MD as part of a project by
students, postdocs and faculty at the Immunology Institute of the Icahn
School of Medicine, Mount Sinai.</p>
<h2 data-number="9.143" id="monoclonal-antibodies-for-the-s2-subunit-of-spike-of-sars-cov-cross-react-with-the-newly-emerged-sars-cov-2-1"><span class="header-section-number">9.143</span> Monoclonal antibodies for the S2 subunit of spike of SARS-CoV cross-react with the newly-emerged SARS-CoV-2</h2>
<p><span class="citation" data-cites="NT3T9a8H">[<a href="#ref-NT3T9a8H" role="doc-biblioref">593</a>]</span></p>
<h3 data-number="9.143.1" id="keywords-142"><span class="header-section-number">9.143.1</span> Keywords</h3>
<ul>
<li>Coronavirus Disease 2019 (COVID-19)</li>
<li>SARS-CoV-2</li>
<li>Spike protein</li>
<li>Cross- reactive antibodies</li>
</ul>
<h3 data-number="9.143.2" id="main-findings-136"><span class="header-section-number">9.143.2</span> Main Findings</h3>
<p>Whole genome sequencing-based comparisons of the 2003
Severe Acute Respiratory Syndrome Coronavirus (SARS-CoV) and the 2019
SARS-CoV-2 revealed conserved receptor binding domain (RBD) and host
cell receptor, angiotensin-converting enzyme 2 (ACE2). In line with
this, the authors tested cross-reactivity of murine monoclonal
antibodies (mAbs) previously generated against the SARS-CoV spike (S)
glycoprotein involved in viral entry. One of the screened mAb, 1A9, was
able to bind and cross-neutralize multiple strains of SARS-CoV, as well
as, detect the S protein in SARS-CoV-2-infected cells. mAb 1A9 was
generated using an immunogenic fragment in the S2 subunit of SARS-CoV
and binds through a novel epitope within the S2 subunit at amino acids
1111-1130. It is important to note that CD8+ T lymphocyte epitopes
overlap with these residues, suggesting that S2 subunit could be
involved in inducing both, humoral and cell-mediated immunity.</p>
<h3 data-number="9.143.3" id="limitations-139"><span class="header-section-number">9.143.3</span> Limitations</h3>
<p>The authors used previously generated mouse mAbs
against the S protein in SARS-CoV expressed in mammalian cell line.
Future experimental validation using COVID-19 patient samples is needed
to validate these findings. In addition, the results of these studies
are predominantly based on in vitro experiments and so, evaluating the
effects of the mAb 1A9 in an animal model infected with this virus will
help us better understand the host immune responses in COVID-19 and
potential therapeutic vaccines.</p>
<h3 data-number="9.143.4" id="significance-129"><span class="header-section-number">9.143.4</span> Significance</h3>
<p>This study identified mAbs that recognize the new
coronavirus, SARS-Cov-2. These cross-reactive mAbs will help in
developing diagnostic assays for COVID-19.</p>
<h3 data-number="9.143.5" id="credit-139"><span class="header-section-number">9.143.5</span> Credit</h3>
<p><em>This review was undertaken by Tamar Plitt and Katherine Lindblad as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<p>Review of paper titled:</p>
<p><strong>SARS-CoV-2 is sensitive to type I interferon pretreatment</strong></p>
<p>Lokugamage et al. 2020</p>
<p>Keywords: Treatment, cytokines, interferons</p>
<p>Summary</p>
<p>Lokugamage et al have found that SARS-CoV-2 has sensitivity to type I
interferon treatment <em>in vitro</em>. They report a significant difference in
interferon stimulated genes (ISGs) and transcription factor expression
between SARS-CoV and SARS-CoV-2 infected VERO cells pretreated with
IFN-α. They attribute these differences to significant changes in the
viral genes ORF6 and ORF 3b, which have been reported to function as
interferon antagonist in SARS-CoV. They suggest using interferon type I
treatment as a potential pathway to further analyze in animal models and
humans.</p>
<p>Caveats</p>
<p>The study was carried out <em>in vitro</em> in an African monkey fibroblast
kidney cell line. They show sensitivity to IFN in cells infected <em>in
vitro</em> and do not mention any other pathway.</p>
<p>[]{#_heading=h.gjdgxs .anchor}This is extrapolated from sequence
homology of SARS-CoV and SARS-CoV-2 proteins. The study does not test
the ability of ORF6 and ORF 3b to inhibit the IFN pathway directly.</p>
<p>Importance of findings</p>
<p>Interferon type I therapies are readily available and have been tested
against several viral infections such as SARS-CoV without success.
Nevertheless, these results suggest that the difference in homologies
between viral proteins ORF6 and ORF 3b of SARS-CoV and SARS-CoV-2 is
enough to disrupt this viral defense mechanism.</p>
<p>[]{#_heading=h.1b4b0g63ddt .anchor}<em>Review by Jovani Catalan-Dibene as
part of a project by students, postdocs and faculty at the Immunology
Institute of the Icahn school of medicine, Mount Sinai.</em></p>
<h2 data-number="9.144" id="mortality-of-covid-19-is-associated-with-cellular-immune-function-compared-to-immune-function-in-chinese-han-population-1"><span class="header-section-number">9.144</span> Mortality of COVID-19 is Associated with Cellular Immune Function Compared to Immune Function in Chinese Han Population</h2>
<p>Zeng et al. <em>medRxiv</em>. <span class="citation" data-cites="1GEbBtBgh">[<a href="#ref-1GEbBtBgh" role="doc-biblioref">594</a>]</span></p>
<h3 data-number="9.144.1" id="keywords-143"><span class="header-section-number">9.144.1</span> Keywords</h3>
<ul>
<li>WBC</li>
<li>peripheral blood</li>
<li>CD4</li>
<li>CD8 T cells</li>
</ul>
<h3 data-number="9.144.2" id="main-findings-137"><span class="header-section-number">9.144.2</span> Main Findings</h3>
<p>Retrospective study of the clinical characteristics of 752
patients infected with COVID-19 at Chinese PLA General Hospital, Peking
Union Medical College Hospital, and affiliated hospitals at Shanghai
University of medicine &amp; Health Sciences. This study is the first one
that compares PB from healthy controls from the same regions in Shanghai
and Beijing, and infected COVID-19 patients to standardize a reference
range of WBCs of people at high risk.</p>
<h3 data-number="9.144.3" id="limitations-140"><span class="header-section-number">9.144.3</span> Limitations</h3>
<p>Lower levels of leukocyte counts -B cells, CD4 and CD8 T
cells- correlated with mortality (WBCs are significantly lower in severe
or critical UCI patients vs mild ones). Based on 14,117 normal controls
in Chinese Han population (ranging in age from 18-86) it is recommended
that reference ranges of people at high risk of COVID-19 infection are
CD3+ lymphocytes below 900 cells/mm3, CD4+ lymphocytes below 500
cells/mm3, and CD8+ lymphocytes below 300 cells/mm3. Importantly, this
study also reported that the levels of D-dimer, C-reactive protein and
IL-6 were elevated in COVID-19 pts., indicating clot formation, severe
inflammation and cytokine storm.</p>
<h3 data-number="9.144.4" id="significance-130"><span class="header-section-number">9.144.4</span> Significance</h3>
<p>This study sets a threshold to identify patients at risk
by analyzing their levels of leukocytes, which is an easy and fast
approach to stratify individuals that require hospitalization. Although
the study is limited (only counts of WBC are analyzed and not its
profile) the data is solid and statistically robust to correlate levels
of lymphopenia with mortality.</p>
<h3 data-number="9.144.5" id="credit-140"><span class="header-section-number">9.144.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.145" id="retrospective-analysis-of-clinical-features-in-101-death-cases-with-covid-19-1"><span class="header-section-number">9.145</span> Retrospective Analysis of Clinical Features in 101 Death Cases with COVID-19</h2>
<p>Chen et al. <em>medRxiv</em>. <span class="citation" data-cites="WuHcOc9T">[<a href="#ref-WuHcOc9T" role="doc-biblioref">595</a>]</span></p>
<h3 data-number="9.145.1" id="keywords-144"><span class="header-section-number">9.145.1</span> Keywords</h3>
<ul>
<li>death biomarkers</li>
<li>cardiac damage</li>
<li>Troponin</li>
<li>Blood type</li>
<li>respiratory failure</li>
<li>hypertension</li>
</ul>
<h3 data-number="9.145.2" id="main-findings-138"><span class="header-section-number">9.145.2</span> Main Findings</h3>
<p>This is a retrospective study involving 101 death cases with COVID-19 in
Wuhan Jinyintan Hospital. The aim was to describe clinical,
epidemiological and laboratory features of fatal cases in order to
identify the possible primary mortality causes related to COVID-19.</p>
<p>Among 101 death cases, 56.44% were confirmed by RT-PCR and 43.6% by
clinical diagnostics. Males dominated the number of deaths and the
average age was 65.46 years. All patients died of respiratory failure
and multiple organs failure, except one (acute coronary syndrome). The
predominant comorbidities were hypertension (42.57%) and diabetes
(22.77%). 25.74% of the patients presented more than two underlying
diseases. 82% of patients presented myocardial enzymes abnormalities at
admission and further increase in myocardial damage indicators with
disease progression: patients with elevated Troponin I progressed faster
to death. Alterations in coagulation were also detected. Indicators of
liver and kidney damage increased 48 hours before death. The authors
studied the deceased patients’ blood type and presented the following
results: type A (44.44%), type B (29.29%), type AB (8.08%) and type O
(18.19%), which is inconsistent with the distribution in Han population
in Wuhan.</p>
<p>Clinical analysis showed that the most common symptom was fever (91.9%),
followed by cough and dyspnea. The medium time from onset of symptoms to
acute respiratory distress syndrome (ARDS) development was 12 days.
Unlike SARS, only 2 patients with COVID-19 had diarrhea. 98% presented
abnormal lung imaging at admission and most had double-lung
abnormalities. Related to the laboratorial findings some inflammatory
indicators gradually increased during the disease progression, such as
IL-6 secretion in the circulation, procalcitonin (PCT) and C-reactive
protein (CRP), while platelets numbers decreased. The authors also
reported an initial lymphopenia that was followed by an increase in the
lymphocytes numbers. Neutrophil count increased with disease
progression.</p>
<p>The patients received different treatments such as antiviral drugs
(60.40%), glucocorticoids, thymosin and immunoglobulins. All patients
received antibiotic treatment and some received antifungal drugs. All
patients received oxygen therapy (invasive or non-invasive ones).</p>
<h3 data-number="9.145.3" id="limitations-141"><span class="header-section-number">9.145.3</span> Limitations</h3>
<p>This study involves just fatal patients, lacking comparisons with other
groups of patients e.g. patients that recovered from COVID-19. The
authors didn’t discuss the different approaches used for treatments and
how these may affect the several parameters measured. The possible
relationship between the increase of inflammatory indicators and
morbidities of COVID-19 are not discussed.</p>
<h3 data-number="9.145.4" id="significance-131"><span class="header-section-number">9.145.4</span> Significance</h3>
<p>This study has the largest cohort of fatal cases reported so far. The
authors show that COVID-19 causes fatal respiratory distress syndrome
and multiple organ failure. This study highlights prevalent myocardial
damage and indicates that cardiac function of COVID-19 patients should
be carefully monitored. The data suggest that Troponin I should be
further investigated as an early indicator of patients with high risk of
accelerated health deterioration. Secondary bacterial and fungal
infections were frequent in critically ill patients and these need to be
carefully monitored in severe COVID-19 patients. Differences in blood
type distribution were observed, suggesting that type A is detrimental
while type O is protective – but further studies are needed to confirm
these findings and elucidate if blood type influences infection or
disease severity. Several inflammatory indicators (neutrophils, PCT, CRP
and IL-6, D-dimer) increased according to disease severity and should be
assessed as biomarkers and to better understand the biology of
progression to severe disease.</p>
<h3 data-number="9.145.5" id="credit-141"><span class="header-section-number">9.145.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<p>10.1101/2020.03.09.983247</p>
<p><strong>Title:</strong> Inhibition of SARS-CoV-2 infection (previously 2019-nCoV) by
a highly potent pan-coronavirus fusion inhibitor targeting its spike
protein that harbors a high capacity to mediate membrane fusion</p>
<p><strong>Keywords:</strong> SARS-CoV2, S protein, fusion inhibitor, hACE2</p>
<p><strong>Main findings:</strong></p>
<p>Members of the coronavirus family rely on fusion with the host’s cell
membrane during infection. The spike (S) protein plays an essential role
in this step (1). This work sheds light on the particularities of the S
protein from SARS-CoV2 and compares it to previously existing
coronaviral S proteins. SARS-CoV2 S protein showed stronger affinity for
hACE2 compared to SARS-CoV. Previous work by this group had established
the potential of a previous version of the peptide (EK1) in reducing the
infectivity of several coronavirus by blocking the S protein
interactions with the host cell (2). In this work, authors present an
improved EK1C4 version of this peptide. In vitro experiments on human
cells showed EK1C4-mediated reduction of the infectivity of SARS-CoV-2
and other coronavirus. Administration of EK1C4 in prophylaxis or shortly
after exposure to the HCoV-OC43 coronavirus protected newborn mice from
virus-induced lethality.</p>
<p><strong>Limitations:</strong></p>
<p>This work suggests that the furin cleavage site from CoV2-S could be
related to its increased infectivity compared to CoV-S. However, a
previous report suggested that the enhanced cell-to-cell fusion mediated
by this furin cleavage might not necessarily translate into enhanced
viral infectivity (3). The mutation of this site, or inhibition of furin
in 293T cells expressing SARS-CoV or CoV-2 S proteins, could inform if
this mechanism is actually involved in SARS-CoV2 pathogenesis, and
potentially suggest additional routes for therapeutic intervention.</p>
<p>The in vivo model used to address the protective potential of EK1C4 is
based on the coronavirus HCoV-OC43. Further work should aim to establish
the potential of EK1C4 for prevention of SARS-CoV-2 replication in more
relevant animal models (e.g. Rhesus Macaques or hACE2 mice). It is,
however, acknowledged, that experimentation in vitro included both
SARS-CoV2 and the same HCoV-OC43 used in the in vivo infection model.</p>
<p>Even though the safety of the unmodified EK1 peptide in mice had been
previously examined (2), the potential side effects and
pharmacokinetics/dynamics of EK1C4 in the in vivo setting are not
addressed in this report.</p>
<p><strong>Relevance:</strong></p>
<p>The development of prophylactic drugs against viral infection is a
paramount element of epidemic control, specially in the time before a
vaccine can be readily available. Similar strategies have been followed
in the past in response to threats of this nature (4,5). This work’s
relevance is limited by the absence of relevant animal models of
SARS-CoV-2 infection. Further pharmacological analyses would also be
necessary to ensure the safety of EK1C4 in animal models before a
potential safety analysis in human subjects.</p>
<p>1. Heald-Sargent T, Gallagher T. Ready, set, fuse! the coronavirus spike
protein and acquisition of fusion competence. Vol. 4, Viruses.
Multidisciplinary Digital Publishing Institute (MDPI); 2012. p. 557–80.</p>
<p>2. Xia S, Yan L, Xu W, Agrawal AS, Algaissi A, Tseng CTK, et al. A
pan-coronavirus fusion inhibitor targeting the HR1 domain of human
coronavirus spike. Sci Adv. 2019 Apr 1;5(4):eaav4580.</p>
<p>3. Follis KE, York J, Nunberg JH. Furin cleavage of the SARS coronavirus
spike glycoprotein enhances cell-cell fusion but does not affect virion
entry. Virology. 2006 Jul 5;350(2):358–69.</p>
<p>4. Jones JC, Turpin EA, Bultmann H, Brandt CR, Schultz-Cherry S.
Inhibition of Influenza Virus Infection by a Novel Antiviral Peptide
That Targets Viral Attachment to Cells. J Virol. 2006 Dec
15;80(24):11960–7.</p>
<p>5. Ahmed A, Siman-Tov G, Hall G, Bhalla N, Narayanan A. Human
antimicrobial peptides as therapeutics for viral infections. Vol. 11,
Viruses. MDPI AG; 2019.</p>
<h2 data-number="9.146" id="relationship-between-the-abo-blood-group-and-the-covid-19-susceptibility-1"><span class="header-section-number">9.146</span> Relationship between the ABO Blood Group and the COVID-19 Susceptibility</h2>
<p>Zhao et al. <em>medRxiv</em>. <span class="citation" data-cites="1DbV444Pj">[<a href="#ref-1DbV444Pj" role="doc-biblioref">596</a>]</span></p>
<h3 data-number="9.146.1" id="keywords-145"><span class="header-section-number">9.146.1</span> Keywords</h3>
<ul>
<li>ABO blood group</li>
<li>COVID-19 susceptibility</li>
</ul>
<h3 data-number="9.146.2" id="main-findings-139"><span class="header-section-number">9.146.2</span> Main Findings</h3>
<p>These authors compared the ABO blood group of 2,173 patients with
RT-PCR-confirmed COVID-19 from hospitals in Wuhan and Shenzhen with the
ABO blood group distribution in unaffected people in the same cities
from previous studies (2015 and 2010 for Wuhan and Shenzhen,
respectively). They found that people with blood group A are
statistically over-represented in the number of those infected and who
succumb to death while those with blood group O are statistically
underrepresented with no influence of age or sex.</p>
<h3 data-number="9.146.3" id="limitations-142"><span class="header-section-number">9.146.3</span> Limitations</h3>
<p>This study compares patients with COVID-19 to the general population
but relies on data published 5 and 10 years ago for the control. The
mechanisms that the authors propose may underlie the differences they
observed require further study.</p>
<h3 data-number="9.146.4" id="significance-132"><span class="header-section-number">9.146.4</span> Significance</h3>
<p>Risk stratification based on blood group may be beneficial for
patients and also healthcare workers in infection control. Additionally,
investigating the mechanism behind these findings could lead to better
developing prophylactic and therapeutic targets for COVID-19.</p>
<h3 data-number="9.146.5" id="credit-142"><span class="header-section-number">9.146.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.147" id="the-inhaled-corticosteroid-ciclesonide-blocks-coronavirus-rna-replication-by-targeting-viral-nsp15-1"><span class="header-section-number">9.147</span> The inhaled corticosteroid ciclesonide blocks coronavirus RNA replication by targeting viral NSP15</h2>
<p>Matsuyama et al. <em>bioRxiv</em> <span class="citation" data-cites="JwL9G6bd">[<a href="#ref-JwL9G6bd" role="doc-biblioref">597</a>]</span></p>
<h3 data-number="9.147.1" id="keywords-146"><span class="header-section-number">9.147.1</span> Keywords</h3>
<ul>
<li>Corticosteroids</li>
<li>ciclesonide</li>
<li>mometasone</li>
<li>NSP15</li>
<li>MERS-CoV</li>
</ul>
<h3 data-number="9.147.2" id="main-findings-140"><span class="header-section-number">9.147.2</span> Main Findings</h3>
<p>This study reconsiders the use of inhaled corticosteroids in the
treatment of pneumonia by coronavirus. Corticosteroids were associated
with increased mortality for SARS in 2003 and for MERS in 2013, probably
due to that fact that systemic corticosteroids suppress the innate
immune system, resulting in increased viral replication. However, some
steroid compounds might block coronavirus replication. The authors
screened steroids from a chemical library and assessed the viral growth
suppression and drug cytotoxicity. Ciclesonide demonstrated low
cytotoxicity and potent suppression of MERS-CoV viral growth. The
commonly used systemic steroids cortisone, prednisolone and
dexamethasone did not suppress viral growth, nor did the commonly used
inhaled steroid fluticasone. To identify the drug target of virus
replication, the authors conducted 11 consecutive MERS-CoV passages in
the presence of ciclesonide or mometasone, and they could generate a
mutant virus that developed resistance to ciclesonide, but not to
mometasone. Afterwards, they performed next-generation sequencing and
identified an amino acid substitution in nonstructural protein 15
(NSP15) as the predicted mechanism for viral resistance to ciclesonide.
The authors were able to successfully generate a recombinant virus
carrying that amino acid substitution, which overcome the antiviral
effect of ciclesonide, suggesting that ciclosenide interacts with NSP15.
The mutant virus was inhibited by mometasone, suggesting that the
antiviral target of mometasone is different from that of ciclesonide.
Lastly, the effects of ciclesonide and mometason on suppressing the
replication of SARS-CoV-2 were evaluated. Both compounds were found to
suppress viral replication with a similar efficacy to lopinavir.</p>
<h3 data-number="9.147.3" id="limitations-143"><span class="header-section-number">9.147.3</span> Limitations</h3>
<p>Most of the experiments, including the identification of the mutation in
NSP15 were conducted with MERS-CoV. This is not the closest related
virus to SARS-CoV-2, as that would be SARS-CoV. Thus, to repeat the
initial experiments with SARS-CoV, or preferably SARS-CoV-2, is
essential. The manuscript should address this and, therefore, it will
require considerable editing for organization and clarity. Also, in
terms of cell immunogenic epitopes, while SARS-CoV-2 spike protein
contains several predicted B and T cell immunogenic epitopes that are
shared with other coronaviruses, some studies have shown critical
differences between MERS-CoV, SARS-CoV and SARS-CoV-2. A main criticism
is that the authors only used VeroE6/TMPRSS2 cells to gauge the direct
cytotoxic effects of viral replication. To evaluate this in other cell
lines, including human airway epithelial cells, is crucial, as the
infectivity of coronavirus strains greatly varies in different cell
lines,</p>
<h3 data-number="9.147.4" id="significance-133"><span class="header-section-number">9.147.4</span> Significance</h3>
<p>Nevertheless, these findings encourage evaluating ciclesonide and
mometasone as better options for patients with COVID-19 in need of
inhaled steroids, especially as an alternative to other corticosteroids
that have been shown to increase viral replication in vitro. This should
be evaluated in future clinical studies.</p>
<h3 data-number="9.147.5" id="credit-143"><span class="header-section-number">9.147.5</span> Credit</h3>
<p><em>This review was undertaken by Alvaro Moreira, MD as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.148" id="a-human-monoclonal-antibody-blocking-sars-cov-2-infection-1"><span class="header-section-number">9.148</span> A human monoclonal antibody blocking SARS-CoV-2 infection **</h2>
<p>Wang et al. <em>bioRxiv</em>. <span class="citation" data-cites="XcuzhxrJ">[<a href="#ref-XcuzhxrJ" role="doc-biblioref">460</a>]</span></p>
<h3 data-number="9.148.1" id="keywords-147"><span class="header-section-number">9.148.1</span> Keywords</h3>
<ul>
<li>Monoclonal antibodies</li>
<li>SARS-CoV2</li>
<li>cross-neutralization</li>
<li>potential treatment</li>
<li>spike receptor</li>
</ul>
<h3 data-number="9.148.2" id="main-findings-141"><span class="header-section-number">9.148.2</span> Main Findings</h3>
<p>The authors reported a human monoclonal
antibody that neutralizes SARS-CoV-2 and SARS-Cov which belong to same
family of corona viruses. For identifying mAbs, supernatants of a
collection of 51 hybridomas raised against the spike protein of SARS-CoV
(SARS-S) were screened by ELISA for cross-reactivity against the spike
protein of SARS-CoN2 (SARS2-S). Hybridomas were derived from immunized
transgenic H2L2 mice (chimeric for fully human VH-VL and rat constant
region). Four SARS-S hybridomas displayed cross-reactivity with SARS2-S,
one of which (47D11) exhibited cross-neutralizing activity for SARS-S
and SARS2-S pseudotyped VSV infection. A recombinant, fully human IgG1
isotype antibody was generated and used for further characterization.</p>
<p>The humanized 47D11 antibody inhibited infection of VeroE6 cells with
SARS-CoV and SARS-CoV-2 with IC50 values of 0.19 and 0.57 μg/ml
respectively. 47D11 mAb bound a conserved epitope on the spike receptor
binding domain (RBD) explaining its ability to cross-neutralize SARS-CoV
and SARS-CoV-2. 47D11 was shown to target the S1B RBD of SARS-S and
SARS2-S with similar affinities. Interestingly, binding of 47D11 to
SARS-S1B and SARS2-S1B did not interfere with S1B binding to ACE2
receptor-expressing cells assayed by flow cytometry.</p>
<h3 data-number="9.148.3" id="limitations-144"><span class="header-section-number">9.148.3</span> Limitations</h3>
<p>These results show that the human 47D11 antibody
neutralizes SARS-CoV and SARS-Cov2 infectivity via an as yet unknown
mechanism that is different from receptor binding interference.
Alternative mechanisms were proposed but these as yet remain to be
tested in the context of SARS-CoV2. From a therapeutic standpoint and in
the absence of in vivo data, it is unclear whether the 47D11 ab can
alter the course of infection in an infected host through virus
clearance or protect an uninfected host that is exposed to the virus.
There is a precedent for the latter possibility as it relates to
SARS-CoV that was cited by the authors and could turn out to be true for
SARS-CoV2.</p>
<h3 data-number="9.148.4" id="significance-134"><span class="header-section-number">9.148.4</span> Significance</h3>
<p>This study enabled the identification of novel neutralizing
antibody against COV-that could potentially be used as first line of
treatment in the near future to reduce the viral load and adverse
effects in infected patients. In addition, neutralizing antibodies such
as 47D11 represent promising reagents for developing
antigen-antibody-based detection test kits and assays.</p>
<h3 data-number="9.148.5" id="credit-144"><span class="header-section-number">9.148.5</span> Credit</h3>
<p><em>This review was edited by K. Alexandropoulos as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<p><strong>Heat inactivation of serum interferes with the immunoanalysis of
antibodies to SARS-CoV-2</strong></p>
<p>Heat inactivation, immunochromatography, diagnosis, serum antibodies,
IgM, IgG</p>
<p><em>Summary</em></p>
<p>The use of heat inactivation to neutralize pathogens in serum samples
collected from suspected COVID-19 patients reduces the sensitivity of a
fluorescent immunochromatographic assay to detect anti-SARS-CoV-2 IgM
and IgG.</p>
<p><em>Major findings</em></p>
<p>Coronaviruses can be killed by heat inactivation, and this is an
important safety precaution in laboratory manipulation of clinical
samples. However, the effect of this step on downstream
SARS-CoV-2-specific serum antibody assays has not been examined. The
authors tested the effect of heat inactivation (56 deg C for 30 minutes)
versus no heat inactivation on a fluorescence immunochromatography
assay. Heat inactivation reduced all IgM measurements by an average of
54% and most IgG measurements (22/36 samples, average reduction of 50%),
consistent with the lower thermal stability of IgM than that of IgG.
Heat inactivation caused a subset of IgM but not IgG readings to fall
below a specified positivity threshold.</p>
<p><em>Limitations</em></p>
<p>Limitations included the use of only one type of assay for testing heat
inactivated vs non-inactivated sera, and the use of the same baseline
for heat inactivated and non-inactivated sera. The results indicate that
heat inactivation affects the quantification of SARS-CoV-2-antibody
response, specially IgM, but still allows to distinguish positive
specific IgG. Therefore, the effect of heat inactivation should be
studied when designing assays that quantitatively associate
immunoglobulin levels (especially IgM) to immune state.</p>
<p><em>Review by Andrew M. Leader as part of a project by students, postdocs
and faculty at the Immunology Institute of the Icahn school of medicine,
Mount Sinai.</em></p>
<p><strong>Title: </strong></p>
<p><strong>Immunopathological characteristics of coronavirus disease 2019 cases
in</strong></p>
<p><strong>Guangzhou, China</strong></p>
<p><strong>Immunology keywords:</strong> SARS-COV-2, COVID-19, T cell, B cell, NK cell,
IL-2, IL-6, IL-10, TNF<span class="math inline">\(\alpha\)</span></p>
<p><strong>The main finding of the article: </strong></p>
<p>This study analyzed immune cell populations and multiple cytokines in 31
patients with mild/moderate COVID-19 (ave. 44.5 years) and 25 with
severe COVID-19 (ave. 66 years). Samples from patients with fever and
negative for the SARS-COV-2 test were used as control. At inpatient
admission, total lymphocytes number was decreased in severe patients but
not in mild patients, whereas neutrophils were increased in severe
patients. CD4+ and CD8+ T cells were diminished in all COVID-19
patients. CD19+ B cells and NK cells were decreased in both mild and
severe patients, however, severe patients showed a notable reduction.
These data might suggest a profound deregulation of lymphocytes in
COVID-19 patients. Further analysis showed significant increases of
IL-2, IL-6, IL-10 and TNF<span class="math inline">\(\alpha\)</span> in blood of severe patients at the
admission. Sequential samples revealed that IL-2 and IL-6 peaked on day
15-20 and declined thereafter. A moderate increase of IL-4 was seen in
mild/moderate patients. Thus, elevation of IL-2, IL-6 can be indicators
of severe COVID-19.</p>
<p><strong>Critical analysis of the study:</strong></p>
<p>There is no information on when the patients were assessed as severe or
mild/moderate, at inpatient admission or later. The authors could have
analyzed the correlation between immune cell population and cytokine
levels to see, for example, if severe lymphopenia correlated to higher
elevation of IL-2.</p>
<p><strong>The importance and implications for the current epidemics:</strong></p>
<p>While similar findings have already been shown, the data corroborates
alterations in circulating adaptive and innate immune cell populations
and cytokines, and its correlation to disease severity. The increase of
IL-2 and IL-6 at the admission might an indicator to start intensive
therapies (like convalescent serum) at an early time.</p>
<h2 data-number="9.149" id="immune-phenotyping-based-on-neutrophil-to-lymphocyte-ratio-and-igg-predicts-disease-severity-and-outcome-for-patients-with-covid-19-1"><span class="header-section-number">9.149</span> Immune phenotyping based on neutrophil-to-lymphocyte ratio and IgG predicts disease severity and outcome for patients with COVID-19</h2>
<p>Zhang et al. <em>medRxiv</em> <span class="citation" data-cites="1CwCTXPqj">[<a href="#ref-1CwCTXPqj" role="doc-biblioref">598</a>]</span></p>
<h3 data-number="9.149.1" id="keywords-148"><span class="header-section-number">9.149.1</span> Keywords</h3>
<ul>
<li>Biomarkers</li>
<li>cytokines</li>
<li>IgG</li>
<li>immune cells</li>
</ul>
<h3 data-number="9.149.2" id="main-findings-142"><span class="header-section-number">9.149.2</span> Main Findings</h3>
<p>In a cohort of 222 patients, anti-SARS-CoV-2 IgM and IgG levels were
analyzed during acute and convalescent phases (up to day 35) and
correlated to the diseases’ severity. The same was done with
neutrophil-to-lymphocyte ratio. High IgG levels and high
neutrophil-to-lymphocyte ratio in convalescence were both independently
associated to the severity of the disease. The simultaneous occurrence
of both of these laboratory findings correlated even stronger to the
diseases’ severity.</p>
<p>Severe cases with high neutrophil-to-lymphocyte ratios had clearly
higher levels of IL-6. The authors propose that a robust IgG response
leads to immune-mediated tissue damage, thus explaining the worse
outcome in patients with overexuberant antibody response.</p>
<h3 data-number="9.149.3" id="limitations-145"><span class="header-section-number">9.149.3</span> Limitations</h3>
<p>A main criticism is that the criteria for stratifying patients in severe
vs. non-severe are not described. The only reference related to this is
the difference between the percentage of patients who needed mechanical
ventilation, which was greater in patients with both high IgG levels and
high neutrophil-to-lymphocyte ratio. No patient with both low IgG levels
and low neutrophil-to-lymphocyte ratio was treated with mechanical
ventilation.</p>
<p>The proposed correlation of severity with IL-2 and IL-10 levels is not
very strong.</p>
<p>Furthermore, although mostly ignored in the paper’s discussion, one of
the most interesting findings is that an early increase in
anti-SARS-CoV-2 IgM levels also seems to correlate with severe disease.
However, as only median values are shown for antibody kinetics curves,
the extent of variation in acute phase cannot be assessed.</p>
<h3 data-number="9.149.4" id="significance-135"><span class="header-section-number">9.149.4</span> Significance</h3>
<p>Anti-SARS-CoV-2 IgG levels and with neutrophil-to-lymphocyte ratio
predict severity of COVID-19 independently of each other. An additive
predictive value of both variables is noticeable. Importantly, an
early-on increase in anti-SARS-CoV-2 IgM levels also seem to predict
outcome.</p>
<h3 data-number="9.149.5" id="credit-145"><span class="header-section-number">9.149.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.150" id="reinfection-could-not-occur-in-sars-2-cov-2-infected-rhesus-macaques-1"><span class="header-section-number">9.150</span> Reinfection could not occur in SARS-2 CoV-2 infected rhesus macaques</h2>
<p>Bao et al. <em>bioRxiv</em> <span class="citation" data-cites="3XqFhi8l">[<a href="#ref-3XqFhi8l" role="doc-biblioref">599</a>]</span></p>
<h3 data-number="9.150.1" id="keywords-149"><span class="header-section-number">9.150.1</span> Keywords</h3>
<ul>
<li>SARS-CoV-2</li>
<li>viral load</li>
<li>reinfection</li>
<li>relapse</li>
<li>non-human primate model</li>
</ul>
<h3 data-number="9.150.2" id="main-findings-143"><span class="header-section-number">9.150.2</span> Main Findings</h3>
<p>This study addresses the issue or acquired immunity after a primary
COVID-19 infection in rhesus monkeys. Four Chinese rhesus macaques were
intratracheally infected with SARS-CoV-2 and two out of the four were
re-infected at 28 days post initial infection (dpi) with the same viral
dose after confirming the recovery by the absence of clinical symptoms,
radiological abnormalities and viral detection (2 negative RT-PCR
tests). While the initial infection led the viral loads in nasal and
pharyngeal swabs that reach approximately 6.5 log<sub>10</sub> RNA copies/ml at 3
dpi in all four monkeys, viral loads in the swabs tested negative after
reinfection in the two reinfected monkeys. In addition, the necropsies
from a monkey (M1) at 7 days after primary infection, and another monkey
(M3) at 5 days post reinfection, revealed the histopathological damages
and viral replication in the examined tissues from M1, while no viral
replication as well as no histological damages were detected in the
tissues from M3. Furthermore, sera from three monkeys at 21 and 28 dpi
exhibited neutralizing activity against SARS-CoV-2 in vitro, suggesting
the production of protective neutralizing antibodies in these monkeys.
Overall, this study indicates that primary infection with SARS-CoV-2 may
protect from subsequent exposure to the same virus.</p>
<h3 data-number="9.150.3" id="limitations-146"><span class="header-section-number">9.150.3</span> Limitations</h3>
<p>In human, virus has been detected by nasopharyngeal swabs until 9 to 15
days after the onset of symptoms. In the infected monkeys in this study,
virus were detected from day 1 after the infection, declining to
undetectable level by day 15 post infection. It may suggest that there
is a faster viral clearance mechanism in monkeys, therefore the
conclusions of reinfection protection for humans need to be carefully
considered. In addition, only two monkeys were re-infected in this study
and the clinical signs of these monkeys were not similar: M3 did not
show weight loss and M4 showed relatively higher fever on the day of
infection and the day of re-challenge.</p>
<h3 data-number="9.150.4" id="significance-136"><span class="header-section-number">9.150.4</span> Significance</h3>
<p>This study showed clear viral clearance and no indications of relapse or
viremia after a secondary infection with SARS-CoV-2 in a Chinese rhesus
macaque model. These results support the idea that patients with full
recovery (two negative RT-PCR results) may also be protected from
secondary SARS-CoV-2 infection. Recovered patients may be able to
reintegrate to normal public life and provide protective serum perhaps
even if having had a mild infection. The results are also encouraging
for successful vaccine development against SARS-CoV-2.</p>
<h3 data-number="9.150.5" id="credit-146"><span class="header-section-number">9.150.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.151" id="a-highly-conserved-cryptic-epitope-in-the-receptor-binding-domains-of-sars-cov-2-and-sars-cov-1"><span class="header-section-number">9.151</span> A highly conserved cryptic epitope in the receptor-binding domains of SARS-CoV-2 and SARS-CoV</h2>
<p><span class="citation" data-cites="6gQxRkWM">[<a href="#ref-6gQxRkWM" role="doc-biblioref">600</a>]</span></p>
<h3 data-number="9.151.1" id="keywords-150"><span class="header-section-number">9.151.1</span> Keywords</h3>
<ul>
<li>neutralizing antibody</li>
<li>cross-reactivity</li>
</ul>
<h3 data-number="9.151.2" id="main-findings-144"><span class="header-section-number">9.151.2</span> Main Findings</h3>
<p>Given the sequence similarity of the surface spike glycoprotein (S) of
SARS-CoV-2 and SAR-CoV, Yuan et al. (2020) propose that neutralizing
antibodies isolated from convalescent SARS-CoV patients may offer
insight into cross-reactive antibodies targeting SARS-CoV-2. In
particular, they find that the receptor-binding domain (RBD) of
SARS-CoV-2 S protein shares 86% sequence similarity with the RBD of
SARS-CoV S protein that binds to the CR3022 neutralizing antibody.
CR3022 also displays increased affinity for the “up” conformation of the
SARS-CoV-2 S protein compared to the “down” conformation as it does for
the SARS-CoV S protein. Therefore, the authors propose that this
cross-reactive antibody may confer some degree of protection <em>in vivo</em>
even if it fails to neutralize <em>in vitro</em>.</p>
<h3 data-number="9.151.3" id="limitations-147"><span class="header-section-number">9.151.3</span> Limitations</h3>
<p>Although the authors offer a logical rationale for
identifying cross-reactive neutralizing antibodies derived from
SARS-CoV, their study using only CR3022 failed to demonstrate whether
this approach will be successful. After all, CR3022 failed to neutralize
<em>in vitro</em> despite the binding affinity to a similar epitope on
SARS-CoV-2. They would benefit from testing more candidates and using an
<em>in vivo</em> model to demonstrate their claim that protection may be
possible in the absence neutralization if combinations are used <em>in
vivo.</em></p>
<h3 data-number="9.151.4" id="significance-137"><span class="header-section-number">9.151.4</span> Significance</h3>
<p>The ability to make use of previously characterized
neutralizing antibodies for conserved epitopes can expedite drug design
and treatment options.</p>
<h3 data-number="9.151.5" id="credit-147"><span class="header-section-number">9.151.5</span> Credit</h3>
<p><em>This review was undertaken by Dan Fu Ruan, Evan Cody and Venu Pothula as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.152" id="highly-accurate-and-sensitive-diagnostic-detection-of-sars-cov-2-by-digital-pcr-1"><span class="header-section-number">9.152</span> Highly accurate and sensitive diagnostic detection of SARS-CoV-2 by digital PCR</h2>
<p>Dong et al. <em>medRxiv</em> <span class="citation" data-cites="UYD5T2mp">[<a href="#ref-UYD5T2mp" role="doc-biblioref">198</a>]</span></p>
<h3 data-number="9.152.1" id="keywords-151"><span class="header-section-number">9.152.1</span> Keywords</h3>
<ul>
<li>Diagnosis</li>
<li>digital PCR</li>
</ul>
<h3 data-number="9.152.2" id="main-findings-145"><span class="header-section-number">9.152.2</span> Main Findings</h3>
<p>The authors present a digital PCR (dPCR) diagnostic test for
SARS-CoV-2 infection. In 103 individuals that were confirmed in a
follow-up to be infected, the standard qPCR test had a positivity
rate of 28.2% while the dPCR test detected 87.4% of the infections
by detecting an additional 61 positive cases. The authors also
tested samples from close contacts (early in infection stage) and
convalescing individuals (late in infection stage) and were able to
detect SARS-CoV-2 nucleic acid in many more samples using dPCR
compared to qPCR.</p>
<h3 data-number="9.152.3" id="limitations-148"><span class="header-section-number">9.152.3</span> Limitations</h3>
<p>I did not detect limitations.</p>
<h3 data-number="9.152.4" id="significance-138"><span class="header-section-number">9.152.4</span> Significance</h3>
<p>The authors make a strong case for the need for a highly sensitive
and accurate confirmatory method for diagnosing COVID-19 during this
outbreak and present a potential addition to the diagnostic arsenal.
They propose a dPCR test that they present has a dramatically lower
false negative rate than the standard RT-qPCR tests and can be
especially beneficial in people with low viral load, whether they
are in the earlier or later stages of infection.</p>
<h3 data-number="9.152.5" id="credit-148"><span class="header-section-number">9.152.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.153" id="sars-cov-2-invades-host-cells-via-a-novel-route-cd147-spike-protein-1"><span class="header-section-number">9.153</span> SARS-CoV-2 invades host cells via a novel route: CD147-spike protein</h2>
<p>Wang et al. <em>bioRxiv</em> <span class="citation" data-cites="14y8UvAbH">[<a href="#ref-14y8UvAbH" role="doc-biblioref">601</a>]</span></p>
<h3 data-number="9.153.1" id="keywords-152"><span class="header-section-number">9.153.1</span> Keywords</h3>
<ul>
<li>spike protein</li>
<li>viral entry</li>
<li>CD147</li>
<li>SARS-CoV-2</li>
</ul>
<h3 data-number="9.153.2" id="main-findings-146"><span class="header-section-number">9.153.2</span> Main Findings</h3>
<p>The authors propose a novel mechanism of SARS-CoV-2 viral entry through
the interaction of the viral spike protein (SP) and the immunoglobulin
superfamily protein CD147 (also known as Basigin). Using an in-house
developed humanized antibody against CD147 (maplazumab), they show that
blocking CD147 decreases viral replication in Vero E6 cells. Using
surface plasmon resonance (SPR), ELISA, and Co-IP assays, they show that
the spike protein of SARS-CoV-2 directly interacts with CD147. Lastly,
they utilize immune-election microscopy to show spike protein and CD147
localize to viral inclusion bodies of Vero E6 cells.</p>
<h3 data-number="9.153.3" id="limitations-149"><span class="header-section-number">9.153.3</span> Limitations</h3>
<p>The authors claim that an anti-CD147 antibody (Meplazumab) inhibits
SARS-CoV-2 replication by testing cell growth and viral load in cells
infected with SARS-CoV-2, however there are key pieces of this
experiment that are missing. First, the authors fail to use a
non-specific antibody control. Second, the authors claim that viral
replication is inhibited, and that they test this by qPCR, however this
data is <strong>not shown.</strong> To further prove specificity, the authors should
introduce CD147 to non-susceptible cells and show that they become
permissive.</p>
<p>The authors claim that there is a direct interaction between CD147 and
SP through SPR, ELISA, and Co-IP, and this data seems generally
convincing. The electron microscopy provides further correlative
evidence that SARS-CoV-2 may interact with CD147 as they are both found
in the same viral inclusion body. A quantification of this data would
make the findings more robust.</p>
<p>Finally, the data in this paper lacks replicates, error bars, and
statistics to show that the data are reproducible and statistically
significant.</p>
<h3 data-number="9.153.4" id="significance-139"><span class="header-section-number">9.153.4</span> Significance</h3>
<p>It has been shown in various studies that SARS-CoV-2 binds to the cell
surface protein ACE2 for cell entry, yet ACE2 is highly expressed in
heart, kidney, and intestinal cells, raising the concern that blocking
ACE2 would result in harmful side effects <span class="citation" data-cites="BcOf65BE">[<a href="#ref-BcOf65BE" role="doc-biblioref">602</a>]</span> CD147 on the other hand
is highly expressed in various tumor types, inflamed tissues, and
pathogen infected cells, suggesting that the inhibition of CD147 would
not result in major side effects <span class="citation" data-cites="34nQJalJ KcThTnsm">[<a href="#ref-34nQJalJ" role="doc-biblioref">603</a>,<a href="#ref-KcThTnsm" role="doc-biblioref">604</a>]</span> The research in this paper has
resulted in an ongoing clinical trail in China to test the safety and
efficacy of anti-CD147 Meplazumab to treat COVID-19. (ClinicalTrails.gov
identifier NCT04275245).</p>
<h3 data-number="9.153.5" id="credit-149"><span class="header-section-number">9.153.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.154" id="blood-single-cell-immune-profiling-reveals-that-interferon-mapk-pathway-mediated-adaptive-immune-response-for-covid-19-1"><span class="header-section-number">9.154</span> Blood single cell immune profiling reveals that interferon-MAPK pathway mediated adaptive immune response for COVID-19</h2>
<p>Huang et al. <em>medRxiv</em> <span class="citation" data-cites="18l4l0nLX">[<a href="#ref-18l4l0nLX" role="doc-biblioref">605</a>]</span></p>
<h3 data-number="9.154.1" id="keywords-153"><span class="header-section-number">9.154.1</span> Keywords</h3>
<ul>
<li>COVID-19</li>
<li>SARS-CoV-2</li>
<li>PBMC</li>
<li>single cell</li>
<li>MAPK</li>
</ul>
<h3 data-number="9.154.2" id="main-findings-147"><span class="header-section-number">9.154.2</span> Main Findings</h3>
<p>The authors performed single-cell RNA sequencing (scRNAseq) of
peripheral blood mononuclear cells isolated from whole blood samples of
COVID-19 patients (n=10). Data was compared to scRNAseq of samples
collected from patients with influenza A (n=1), acute pharyngitis (n=1),
and cerebral infarction (n=1), as well as, three healthy controls.
COVID-19 patients were categorized into those with moderate (n=6),
severe (n=1), critical (n=1), and cured (n=2) disease. Analysis across
all COVID-19 disease levels revealed 56 different cellular subtypes,
among 17 immune cell types; comparisons between each category to the
normal controls revealed <strong>increased proportions of CD1c<sup>+</sup> dendritic
cells, CD8<sup>+</sup> CTLs, and plasmacytoid dendritic cells and a decrease in
proportions of B cells and CD4<sup>+</sup> T cells</strong>.</p>
<p><strong>TCR sequencing revealed that greater clonality is associated with
milder COVID-19 disease</strong>; BCR sequencing revealed that COVID-19
patients have circulating antibodies against known viral antigens,
including EBV, HIV, influenza A, and other RNA viruses. This may suggest
that the immune response to SARS-CoV-2 infection elicits production of
antibodies against known RNA viruses.</p>
<p>Excluding enriched pathways shared by COVID-19 patients and patients
with other conditions (influenza A, acute pharyngitis, and cerebral
infarction), the authors identified the <strong>interferon-MAPK signaling
pathway as a major response to SARS-CoV-2 infection</strong>. The authors
performed quantitative real-time reverse transcriptase polymerase chain
reaction (RT-PCR) for interferon-MAPK signaling genes: <em>IRF27, BST2,</em>
and <em>FOS</em>. These samples were collected from a separate cohort of
COVID-19 patients (critical, n=3; severe, n=3; moderate, n=19; mild,
n=3; and cured, n=10; and healthy controls, n=5). Notably, consistent
with the original scRNAseq data, <em>FOS</em> showed up-regulation in COVID-19
patients and down-regulation in cured patients. <strong>The authors propose
that <em>FOS</em> may be a candidate marker gene for curative COVID-19
disease</strong>.</p>
<h3 data-number="9.154.3" id="limitations-150"><span class="header-section-number">9.154.3</span> Limitations</h3>
<p>The sample size of this study is limited. To further delineate
differences in the immune profile of peripheral blood of COVID-19
patients, a greater sample size is needed, and longitudinal samples are
needed, as well. A better understanding of the immunological
interactions in cured patients, for example, would require a profile
before and after improvement.</p>
<p>Moreover, the conclusions drawn from this scRNAseq study point to
potential autoimmunity and immune deficiency to distinguish different
severities of COVID-19 disease. However, this requires an expanded
number of samples and a more robust organization of specific immune cell
subtypes that can be compared across different patients. Importantly,
this criterion is likely needed to ensure greater specificity in
identifying markers for COVID-19 infection and subsequent immune
response.</p>
<h3 data-number="9.154.4" id="significance-140"><span class="header-section-number">9.154.4</span> Significance</h3>
<p>At the single-cell level, COVID-19 disease has been characterized in the
lung, but a greater understanding of systemic immunological responses is
furthered in this study. Type I interferon is an important signaling
molecule for the anti-viral response. The identification of the
interferon-MAPK signaling pathway and the differential expression of
MAPK regulators between patients of differing COVID-19 severity and
compared to cured patients may underscore the importance of either
immune deficiency or autoimmunity in COVID-19 disease.</p>
<h3 data-number="9.154.5" id="credit-150"><span class="header-section-number">9.154.5</span> Credit</h3>
<p><em>This review was undertaken by Matthew D. Park as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.155" id="cross-reactive-antibody-response-between-sars-cov-2-and-sars-cov-infection.-1"><span class="header-section-number">9.155</span> Cross-reactive antibody response between SARS-CoV-2 and SARS-CoV infection.</h2>
<p>Lv et al. <em>bioRxiv</em> <span class="citation" data-cites="dM1rOHmH">[<a href="#ref-dM1rOHmH" role="doc-biblioref">606</a>]</span></p>
<h3 data-number="9.155.1" id="keywords-154"><span class="header-section-number">9.155.1</span> Keywords</h3>
<p>SARS-CoV-2, SARS-CoV, spike protein, RBD, cross-reactivity,
cross-neutralization, antibody, human patients, mouse</p>
<h3 data-number="9.155.2" id="main-findings-148"><span class="header-section-number">9.155.2</span> Main Findings</h3>
<p>The authors explore the antigenic differences between SARS-CoV-2 and
SARS-CoV by analyzing plasma samples from SARS-CoV-2 (n = 15) and
SARS-CoV (n = 7) patients. Cross-reactivity in antibody binding to the
spike protein between SARS-CoV-2 and SARS-CoV was found to be common,
mostly targeting non-RBD regions in plasma from SARS-CoV-2 patients.
Only one SARS-CoV-2 plasma sample was able to cross-neutralize SARS-CoV,
with low neutralization activity. No cross-neutralization response was
detected in plasma from SARS-CoV patients.</p>
<p>To further investigate the cross-reactivity of antibody responses to
SARS-CoV-2 and SARS-CoV, the authors analyzed the antibody response of
plasma collected from mice infected or immunized with SARS-CoV-2 or
SARS-CoV (n = 5 or 6 per group). Plasma from mice immunized with
SARS-CoV-2 displayed cross-reactive responses to SARS-CoV S ectodomain
and, to a lesser extent, SARS-CoV RBD. Similarly, plasma from mice
immunized with SARS-CoV displayed cross-reactive responses to SARS-CoV-2
S ectodomain. Cross-neutralization activity was not detected in any of
the mouse plasma samples.</p>
<h3 data-number="9.155.3" id="limitations-151"><span class="header-section-number">9.155.3</span> Limitations</h3>
<p>The size of each patient cohort is insufficient to accurately determine
the frequency of cross-reactivity and cross-neutralization in the
current SARS-CoV-2 pandemic. Recruitment of additional patients from a
larger range of geographical regions and time points would also enable
exploration into the effect of the genetic diversity and evolution of
the SARS-CoV-2 virus on cross-reactivity. This work would also benefit
from the mapping of specific epitopes for each sample. Future studies
may determine whether the non-neutralizing antibody responses can confer
<em>in vitro</em> protection or lead to antibody-dependent disease enhancement.</p>
<h3 data-number="9.155.4" id="significance-141"><span class="header-section-number">9.155.4</span> Significance</h3>
<p>The cross-reactive antibody responses to S protein in the majority of
SARS-CoV-2 patients is an important consideration for development of
serological assays and vaccine development during the current outbreak.
The limited extent of cross-neutralization demonstrated in this study
indicates that vaccinating to cross-reactive conserved epitopes may have
limited efficacy, presenting a key concern for the development of a more
universal coronavirus vaccine to address the global health risk of novel
coronavirus outbreaks.</p>
<h3 data-number="9.155.5" id="credit-151"><span class="header-section-number">9.155.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.156" id="the-feasibility-of-convalescent-plasma-therapy-in-severe-covid-19-patients-a-pilot-study-1"><span class="header-section-number">9.156</span> The feasibility of convalescent plasma therapy in severe COVID-19 patients: a pilot study</h2>
<p>Duan et al. <em>medRxiv</em> <span class="citation" data-cites="UENoLkhz">[<a href="#ref-UENoLkhz" role="doc-biblioref">607</a>]</span></p>
<h3 data-number="9.156.1" id="keywords-155"><span class="header-section-number">9.156.1</span> Keywords</h3>
<ul>
<li>COVID-19</li>
<li>SARS-CoV-2</li>
<li>convalescent plasma</li>
<li>treatment outcome</li>
<li>pilot</li>
<li>therapy</li>
<li>transfusion</li>
</ul>
<h3 data-number="9.156.2" id="main-findings-149"><span class="header-section-number">9.156.2</span> Main Findings</h3>
<p>This is the first report to date of convalescent plasma therapy as a
therapeutic against COVID-19 disease. This is a feasibility pilot study.
The authors report the administration and clinical benefit of 200 mL of
convalescent plasma (CP) (1:640 titer) derived from recently cured
donors (CP selected among 40 donors based on high neutralizing titer and
ABO compatibility) to 10 severe COVID-19 patients with confirmed
viremia. The primary endpoint was the safety of CP transfusion. The
secondary endpoint were clinical signs of improvement based on symptoms
and laboratory parameters.</p>
<p>The authors reported use of methylene blue photochemistry to inactivate
any potential residual virus in the plasma samples, without compromising
neutralizing antibodies, and no virus was detected before transfusion.</p>
<p>The authors report the following:</p>
<ul>
<li><p>No adverse events were observed in all patients, except 1 patient
who exhibited transient facial red spotting.</p></li>
<li><p>All patients showed significant improvement in or complete
disappearance of clinical symptoms, including fever, cough,
shortness of breath, and chest pain after 3 days of CP therapy.</p></li>
<li><p>Reduction of pulmonary lesions revealed by chest CT.</p></li>
<li><p>Elevation of lymphocyte counts in patients with lymphocytopenia.</p></li>
<li><p>Increase in SaO<sub>2</sub> in all patients, indicative of recuperating lung
function.</p></li>
<li><p>Resolution of SARS-CoV-2 viremia in 7 patients and increase in
neutralizing antibody titers in 5 patients. Persistence of
neutralizing antibody levels in 4 patients.</p></li>
</ul>
<h3 data-number="9.156.3" id="limitations-152"><span class="header-section-number">9.156.3</span> Limitations</h3>
<p>It is important to note that most recipients had high neutralization
titers of antibodies before plasma transfusion and even without
transfusion it would be expected to see an increase in neutralizing
antibodies over time. In addition to the small sample set number (n=10),
there are additional limitations to this pilot study:</p>
<ol type="1">
<li><p>All patients received concurrent therapy, in addition to the CP
transfusion. Therefore, it is unclear whether a combinatorial or
synergistic effect between these standards of care and CP
transfusion contributed to the clearance of viremia and improvement
of symptoms in these COVID-19 patients.</p></li>
<li><p>The kinetics of viral clearance was not investigated, with respect
to the administration of CP transfusion. So, the definitive impact
of CP transfusion on immune dynamics and subsequent viral load is
not well defined.</p></li>
<li><p>Comparison with a small historical control group is not ideal.</p></li>
</ol>
<h3 data-number="9.156.4" id="significance-142"><span class="header-section-number">9.156.4</span> Significance</h3>
<p>For the first time, a pilot study provides promising results involving
the use of convalescent plasma from cured COVID-19 patients to treat
others with more severe disease. The authors report that the
administration of a single, high-dose of neutralizing antibodies is
safe. In addition, there were encouraging results with regards to the
reduction of viral load and improvement of clinical outcomes. It is,
therefore, necessary to expand this type of study with more
participants, in order to determine optimal dose and treatment kinetics.
It is important to note that CP has been studied to treat H1N1
influenza, SARS-CoV-1, and MERS-CoV, although it has not been proven to
be effective in treating these infections.</p>
<h3 data-number="9.156.5" id="credit-152"><span class="header-section-number">9.156.5</span> Credit</h3>
<p><em>Review by Matthew D. Park and revised by Alice O. Kamphorst and Maria A. Curotto de Lafaille as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.157" id="hydroxychloroquine-and-azithromycin-as-a-treatment-of-covid-19-results-of-an-open-label-non-randomized-clinical-trial-1"><span class="header-section-number">9.157</span> Hydroxychloroquine and azithromycin as a treatment of COVID-19: results of an open label non-randomized clinical trial</h2>
<p><span class="citation" data-cites="Xbkby2Jr">[<a href="#ref-Xbkby2Jr" role="doc-biblioref">608</a>]</span></p>
<h3 data-number="9.157.1" id="keywords-156"><span class="header-section-number">9.157.1</span> Keywords</h3>
<ul>
<li>hydroxychloroquine</li>
<li>clearance</li>
<li>viral load</li>
<li>clinical trial</li>
</ul>
<h3 data-number="9.157.2" id="main-findings-150"><span class="header-section-number">9.157.2</span> Main Findings</h3>
<p>This study was a single-arm, open label clinical trial with
600 mg hydroxychloroquine (HCQ) in the treatment arm (n = 20). Patients
who refused participation or patients from another center not treated
with HCQ were included as negative controls (n = 16). Among the patients
in the treatment arm, 6 received concomitant azithromycin to prevent
superimposed bacterial infection. The primary endpoint was respiratory
viral loads on day 6 post enrollment, measured by nasopharyngeal swab
followed by real-time reverse transcription-PCR.</p>
<p>HCQ alone was able to significantly reduce viral loads by day 6 (n =
8/14, 57.1% complete clearance, p &lt; 0.001); azithromycin appears to
be synergistic with HCQ, as 6/6 patients receiving combined treatment
had complete viral clearance (p &lt; 0.001).</p>
<h3 data-number="9.157.3" id="limitations-153"><span class="header-section-number">9.157.3</span> Limitations</h3>
<p>Despite what is outlined above, this study has a number
of limitations that must be considered. First, there were originally n =
26 patients in the treatment arm, with 6 lost to follow up for the
following reasons: 3 transferred to ICU, 1 discharge, 1
self-discontinued treatment d/t side effects, and 1 patient expired.
Total length of clinical follow up was 14 days, but the data beyond day
6 post-inclusion are not shown.</p>
<p>Strikingly, in supplementary table 1, results of the real-time RT-PCR
are listed for the control and treatment arms from D0 – D6. However, the
data are not reported in a standard way, with a mix of broadly positive
or negative result delineation with Ct (cycle threshold) values, the
standard output of real time PCR. It is impossible to compare what is
defined as a positive value between the patients in the control and
treatment arms without a standardized threshold for a positive test.
Further, the starting viral loads reported at D0 in the groups receiving
HCQ or HCQ + azithromycin were significantly different (ct of 25.3 vs
26.8 respectively), which could explain in part the differences observed
in the response to treatment between 2 groups. Finally, patients in the
control arm from outside the primary medical center in this study
(Marseille) did not actually have samples tested by PCR daily. Instead,
positive test results from every other day were extrapolated to mean
positive results on the day before and after testing as well (Table 2,
footnote <sup>a</sup>).</p>
<p>Taken together, the results of this study suggest that HCQ represents a
promising treatment avenue for COVID-19 patients. However, the limited
size of the trial, and the way in which the results were reported does
not allow for other medical centers to extrapolate a positive or
negative result in the treatment of their own patients with HCQ +/-
azithromycin. Further larger randomized clinical trials will be required
to ascertain the efficacy of HCQ +/- azithromycin in the treatment of
COVID-19.</p>
<h3 data-number="9.157.4" id="significance-143"><span class="header-section-number">9.157.4</span> Significance</h3>
<p>Chloroquine is thought to inhibit viral infection,
including SARS-Cov-2, by increasing pH within endosomes and lysosomes,
altering the biochemical conditions required for viral fusion <span class="citation" data-cites="IhdcvbzC BIlqPh6M">[<a href="#ref-IhdcvbzC" role="doc-biblioref">246</a>,<a href="#ref-BIlqPh6M" role="doc-biblioref">609</a>]</span>.
However, chloroquine also has immuno-modulatory effects that I think may
play a role. Chloroquine has been shown to increase CTLA-4 expression at
the cell surface by decreasing its degradation in the endo-lysosome
pathway; AP-1 traffics the cytoplasmic tail of CTLA-4 to lysosomes, but
in conditions of increased pH, the protein machinery required for
degradation is less functional <span class="citation" data-cites="Q9tXjG1R">[<a href="#ref-Q9tXjG1R" role="doc-biblioref">610</a>]</span>. As such, more CTLA-4 remains in
endosomes and is trafficked back to the cell surface. It is possible
that this may also contribute to patient recovery via reduction of
cytokine storm, in addition to the direct anti-viral effects of HCQ.</p>
<h3 data-number="9.157.5" id="credit-153"><span class="header-section-number">9.157.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.158" id="recapitulation-of-sars-cov-2-infection-and-cholangiocyte-damage-with-human-liver-organoids-1"><span class="header-section-number">9.158</span> Recapitulation of SARS-CoV-2 Infection and Cholangiocyte Damage with Human Liver Organoids</h2>
<p><span class="citation" data-cites="OuDMtCRl">[<a href="#ref-OuDMtCRl" role="doc-biblioref">534</a>]</span></p>
<h3 data-number="9.158.1" id="keywords-157"><span class="header-section-number">9.158.1</span> Keywords</h3>
<ul>
<li>CoVID-19</li>
<li>2019-nCoV</li>
<li>SARS-CoV-2</li>
<li>liver</li>
<li>organoids</li>
<li>Cholangiocyte</li>
</ul>
<h3 data-number="9.158.2" id="main-findings-151"><span class="header-section-number">9.158.2</span> Main Findings</h3>
<ul>
<li><p>Used human liver ductal organoids to determine ACE2+ cholangiocytes
in healthy liver (2.92% of all cells) are infectable and support
SARS-CoV-2 viral replication.</p></li>
<li><p>Plaque-purified SARS-CoV-2 viral infection disrupted organoid
barrier and bile transporting functions of cholangiocytes through
dysregulation of genes involved in tight junction formation (CLDN1)
and bile acid transportation (ASBT and CFTR).</p></li>
</ul>
<h3 data-number="9.158.3" id="limitations-154"><span class="header-section-number">9.158.3</span> Limitations:</h3>
<ul>
<li><p>Unclear if liver damage observed in patients due to direct
cholangiocyte infection or due to secondary immune/cytokine effects.
This study argues for direct damage as it lacks immune contexture;
but further studies needed with autopsy samples or organoid-immune
cell co-culture to conclude strongly.</p></li>
<li><p>Would be important to measure cholangiocyte-intrinsic anti-viral
response and alarmins secreted upon infection, and furthermore study
tropism of various immune cells to conditioned media from organoids
infected with SARS-CoV-2.</p></li>
<li><p>Does not address how cirrhotic liver or
alcohol/smoking/obesity-associated liver organoids respond to
SARS-CoV-2 infectivity and replication, worth pursuing to
experimentally address clinical data indicating co-morbidities.</p></li>
</ul>
<h3 data-number="9.158.4" id="significance-144"><span class="header-section-number">9.158.4</span> Significance</h3>
<ul>
<li><p>Useful model to rapidly study drug activity against SARS-CoV-2
infection in liver, while monitoring baseline liver damage.</p></li>
<li><p>Liver abnormality observed in &gt;50% of CoVID-19 patients; the
results from this study could explain the bile acid accumulation and
consequent liver damage observed.</p></li>
</ul>
<h3 data-number="9.158.5" id="credit-154"><span class="header-section-number">9.158.5</span> Credit</h3>
<p><em>Review by Samarth Hegde as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.159" id="the-sequence-of-human-ace2-is-suboptimal-for-binding-the-s-spike-protein-of-sars-coronavirus-2-1"><span class="header-section-number">9.159</span> The sequence of human ACE2 is suboptimal for binding the S spike protein of SARS coronavirus 2</h2>
<p><span class="citation" data-cites="3VPi509z">[<a href="#ref-3VPi509z" role="doc-biblioref">611</a>]</span></p>
<h3 data-number="9.159.1" id="keywords-158"><span class="header-section-number">9.159.1</span> Keywords</h3>
<ul>
<li>Coronavirus Disease 2019 (COVID-19)</li>
<li>SARS-CoV-2</li>
<li>Spike protein S</li>
<li>ACE2</li>
</ul>
<h3 data-number="9.159.2" id="main-findings-152"><span class="header-section-number">9.159.2</span> Main Findings</h3>
<p>Severe Acute Respiratory Syndrome Coronavirus 2
(SARS-CoV-2) infects cells through S spike glycoprotein binding
angiotensin-converting enzyme (ACE2) on host cells. S protein can bind
both membrane-bound ACE2 and soluble ACE2 (sACE2), which can serve as a
decoy that neutralizes infection. Recombinant sACE2 is now being tested
in clinical trials for COVID-19. To determine if a therapeutic sACE2
with higher affinity for S protein could be designed, authors generated
a library containing every amino acid substitution possible at the 117
sites spanning the binding interface with S protein. The ACE2 library
was expressed in human Expi293F cells and cells were incubated with
medium containing the receptor binding domain (RBD) of SARS-CoV-2 fused
to GFP. Cells with high or low affinity mutant ACE2 receptor compared to
affinity of wild type ACE2 for the RBD were FACS sorted and transcripts
from these sorted populations were deep sequenced. Deep mutagenesis
identified numerous mutations in ACE2 that enhance RBD binding. This
work serves to identify putative high affinity ACE2 therapeutics for the
treatment of CoV-2.</p>
<h3 data-number="9.159.3" id="limitations-155"><span class="header-section-number">9.159.3</span> Limitations</h3>
<p>The authors generated a large library of mutated
ACE2, expressed them in human Expi293F cells, and performed deep
mutagenesis to identify enhanced binders for the RBD of SARS-CoV-2 S
protein. While these data serve as a useful resource, the ability of the
high affinity ACE2 mutants identified to serve as therapeutics needs
further validation in terms of conformational stability when purified as
well as efficacy/safety both <em>in vitro</em> and <em>in vivo</em>. Additionally,
authors mentioned fusing the therapeutic ACE2 to Fc receptors to elicit
beneficial host immune responses, which would need further design and
validation.</p>
<h3 data-number="9.159.4" id="significance-145"><span class="header-section-number">9.159.4</span> Significance</h3>
<p>This study identified structural ACE2 mutants that
have potential to serve as therapeutics in the treatment of SARS-CoV-2
upon further testing and validation.</p>
<h3 data-number="9.159.5" id="credit-155"><span class="header-section-number">9.159.5</span> Credit</h3>
<p><em>This review was undertaken by Katherine Lindblad and Tamar Plitt as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<p><strong><em>Title</em></strong>:  A serological assay to detect SARS-Cov-2 seroconversion in
humans</p>
<p><strong><em>Immunology keywords</em></strong>:  specific serological assay - ELISA -
seroconversion - antibody titers</p>
<p><em>Note: the authors of this review work in the same institution as the
authors of the study</em></p>
<p><strong><em>Main findings</em></strong>:  </p>
<p>Production of recombinant whole Spike (S) protein and the smaller
Receptor Binding Domain (RBD) based on the sequence of Wuhan-Hu-1
SARS-CoV-2 isolate. The S protein was modified to allow trimerization
and increase stability. The authors compared the antibody reactivity of
59 banked human serum samples (non-exposed) and 3 serum samples from
confirmed SARS-CoV-2 infected patients. All Covid-19 patient sera
reacted to the S protein and RBD domain compared to the control sera.</p>
<p>The authors also characterized the antibody isotypes from the Covid-19
patients, and observed stronger IgG3 response than IgG1. IgM and IgA
responses were also prevalent.</p>
<p><strong><em>Limitations of the study</em></strong>:  </p>
<p>The authors analyzed a total of 59 control human serum samples, and
samples from only three different patients to test for reactivity
against the RBD domain and full-length spike protein. It will be
important to follow up with a larger number of patient samples to
confirm the data obtained. Furthermore, it would be interesting to
assess people at different age groups and determine whether unexposed
control kids have a higher “background”.</p>
<p>Applications of the assay described in this study in diagnosis are
limited, since antibody response should start to be detectable only one
to two weeks after infection. Future studies will be required to assess
how long after infection this assay allow to detect anti-CoV2
antibodies. Finally, while likely, the association of seroconversion
with protective immunity against SARS-Cov-2 infection still needs to be
fully established.</p>
<p><strong><em>Relevance</em></strong>:</p>
<p>This study has strong implications in the research against SARS-Cov-2.
First, it is now possible to perform serosurveys and determine who has
been infected, allowing a more accurate estimate of infection prevalence
and death rate. Second, if it is confirmed that re-infection does not
happen (or is rare), this assay can be used as a tool to screen
healthcare workers and prioritize immune ones to work with infected
patients. Third, potential convalescent plasma donors can now be
screened to help treating currently infected patients. Of note, this
assay does not involve live virus handling. experimentally, this is an
advantage as the assay does not require the precautions required by
manipulation of live virus. Finally, the recombinant proteins described
in this study represent new tools that can be used for further
applications, including vaccine development.</p>
<h2 data-number="9.160" id="comparative-pathogenesis-of-covid-19-mers-and-sars-in-a-non-human-primate-model-1"><span class="header-section-number">9.160</span> COMPARATIVE PATHOGENESIS OF COVID-19, MERS AND SARS IN A NON-HUMAN PRIMATE MODEL</h2>
<p><span class="citation" data-cites="1EcOLnVXk">[<a href="#ref-1EcOLnVXk" role="doc-biblioref">612</a>]</span></p>
<h3 data-number="9.160.1" id="keywords-159"><span class="header-section-number">9.160.1</span> Keywords</h3>
<ul>
<li>SARS-CoV2</li>
<li>cynomolgus macaque</li>
<li>SARS-CoV</li>
</ul>
<h3 data-number="9.160.2" id="main-findings-153"><span class="header-section-number">9.160.2</span> Main Findings</h3>
<p>This work assesses SARS-CoV-2 infection in young adult and aged
cynomolgus macaques. 4 macaques per age group were infected with
low-passage clinical sample of SARS-CoV-2 by intranasal and
intratracheal administration. Viral presence was assessed in nose,
throat and rectum through RT-PCR and viral culture. SARS-CoV-2
replication was confirmed in the respiratory track (including nasal
samples), and it was also detected in ileum. Viral nucleocapsid
detection by IHC showed infection of type I and II pneumocytes and
epithelia. Virus was found to peak between 2 and 4 days after
administration and reached higher levels in aged vs. young animals. The
early peak is consistent with data in patients and contrasts to SARS-CoV
replication. SARS-CoV-2 reached levels below detection between 8 and 21
days after inoculation and macaques established antibody immunity
against the virus by day 14. There were histopathological alteration in
lung, but no overt clinical signs. At day 4 post inoculation of
SARS-CoV-2, two of four animals presented foci of pulmonary
consolidation, with limited areas of alveolar edema and pneumonia, as
well as immune cell infiltration. In sum, cynomolgus macaques are
permissive to SARS- CoV-2 and develop lung pathology (less severe than
SARC-CoV, but more severe than MERS-CoV).</p>
<h3 data-number="9.160.3" id="limitations-156"><span class="header-section-number">9.160.3</span> Limitations</h3>
<p>Even though cynomolgus macaques were permissive to SARS-CoV-2
replication, it is unclear if the viral load reaches levels comparable
to humans and there wasn’t overt clinical pathology.</p>
<h3 data-number="9.160.4" id="significance-146"><span class="header-section-number">9.160.4</span> Significance</h3>
<p>The development of platforms in which to carry out relevant
experimentation on SARS-CoV-2 pathophysiology is of great urgency.
Cynomolgus macaques offer an environment in which viral replication can
happen, albeit in a limited way and without translating into clinically
relevant symptoms. Other groups are contributing to SARS-CoV2 literature
using this animal model <span class="citation" data-cites="3XqFhi8l">[<a href="#ref-3XqFhi8l" role="doc-biblioref">599</a>]</span>, potentially showing protection against
reinfection in cured macaques. Therefore, this platform could be used to
examine SARS-CoV2 pathophysiology while studies in other animal models
are also underway <span class="citation" data-cites="NzFgXYvQ MgLBA5m">[<a href="#ref-MgLBA5m" role="doc-biblioref">536</a>,<a href="#ref-NzFgXYvQ" role="doc-biblioref">613</a>]</span>.</p>
<h3 data-number="9.160.5" id="credit-156"><span class="header-section-number">9.160.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.161" id="investigating-the-impact-of-asymptomatic-carriers-on-covid-19-transmission-1"><span class="header-section-number">9.161</span> Investigating the Impact of Asymptomatic Carriers on COVID-19 Transmission</h2>
<p><span class="citation" data-cites="OvFueDLZ">[<a href="#ref-OvFueDLZ" role="doc-biblioref">614</a>]</span></p>
<h3 data-number="9.161.1" id="keywords-160"><span class="header-section-number">9.161.1</span> Keywords</h3>
<ul>
<li>Coronavirus Disease 2019 (COVID-19)</li>
<li>SARS-CoV-2</li>
<li>asymptomatic carriers</li>
<li>mathematical model</li>
<li>transmission</li>
</ul>
<h3 data-number="9.161.2" id="main-findings-154"><span class="header-section-number">9.161.2</span> Main Findings</h3>
<p>Multiple studies reported the same level of infectiousness
between symptomatic and asymptomatic carriers of SARS-CoV-2. Given that
asymptomatic and undocumented carriers escape public health surveillance
systems, a better mathematical model of transmission is needed to
determine a more accurate estimate of the basic reproductive number
(R<sub>0</sub>) of the virus to assess the contagiousness of virus. The authors
developed a SEYAR dynamical model for transmission of the new
coronavirus that takes into account asymptomatic and undocumented
carriers. The model was validated using data reported from thirteen
countries during the first three weeks of community transmission. While
current studies estimate R<sub>0</sub> to be around 3, this model indicates that
the value could range between 5.5 to 25.4.</p>
<h3 data-number="9.161.3" id="limitations-157"><span class="header-section-number">9.161.3</span> Limitations</h3>
<p>The SEYAR model realistically depicts
transmission of the virus only during the initial stages of the disease.
More data is necessary to better fit the model with current trends. In
addition, multiple factors (e.g. behavioral patterns, surveillance
capabilities, environmental and socioeconomic factors) affect
transmission of the virus and so, these factors must be taken into
consideration when estimating the R<sub>0</sub>.</p>
<h3 data-number="9.161.4" id="significance-147"><span class="header-section-number">9.161.4</span> Significance</h3>
<p>Public health authorities use the basic reproductive
number to determine the severity of disease. An accurate estimate of
R<sub>0</sub> will inform intervention strategies. This model can be applied to
different locations to assess the potential impact of COVID-19.</p>
<h3 data-number="9.161.5" id="credit-157"><span class="header-section-number">9.161.5</span> Credit</h3>
<p><em>This review was undertaken by Tamar Plitt and Katherine Lindblad as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.162" id="antibody-responses-to-sars-cov-2-in-covid-19-patients-the-perspective-application-of-serological-tests-in-clinical-practice-1"><span class="header-section-number">9.162</span> Antibody responses to SARS-CoV-2 in COVID-19 patients: the perspective application of serological tests in clinical practice</h2>
<p>Long et al. <em>medRxiv</em> <span class="citation" data-cites="Y9g1sQ1Y">[<a href="#ref-Y9g1sQ1Y" role="doc-biblioref">615</a>]</span></p>
<h3 data-number="9.162.1" id="keywords-161"><span class="header-section-number">9.162.1</span> Keywords</h3>
<ul>
<li>Serum antibodies</li>
<li>IgM</li>
<li>IgG</li>
<li>immunoassay</li>
<li>diagnosis</li>
<li>seroconversion</li>
</ul>
<h3 data-number="9.162.2" id="main-findings-155"><span class="header-section-number">9.162.2</span> Main Findings</h3>
<p>This study investigated the profile of the acute antibody response
against SARS-CoV-2 and provided proposals for serologic tests in
clinical practice. Magnetic Chemiluminescence Enzyme Immunoassay was
used to evaluate IgM and IgG seroconversion in 285 hospital admitted
patients who tested positive for SARS-CoV-2 by RT-PCR and in 52 COVID-19
suspected patients that tested negative by RT-PCR. A follow up study
with 63 patients was performed to investigate longitudinal effects. In
addition, IgG and IgM titers were evaluated in a cohort of close
contacts (164 persons) of an infected couple.</p>
<p>The median day of seroconversion for both IgG and IgM was 13 days after
symptom onset. Patients varied in the order of IgM/ IgG seroconversion
and there was no apparent correlation of order with age, severity, or
hospitalization time. This led the authors to conclude that for
diagnosis IgM and IgG should be detected simultaneously at the early
phase of infection.</p>
<p>IgG titers, but not IgM titers were higher in severe patients compared
to non-severe patients after controlling for days post-symptom onset.
Importantly, 12% of COVID-19 patients (RT-PCR confirmed) did not meet
the WHO serological diagnosis criterion of either seroconversion or
<em>&gt;</em> 4-fold increase in IgG titer in sequential samples. This suggests
the current serological criteria may be too stringent for COVID-19
diagnosis.</p>
<p>Of note, 4 patients from a group of 52 suspects (negative RT-PCR test)
had anti-SARS-Cov-2 IgM and IgG. Similarly, 4.3% (7/162) of “close
contacts” who had negative RT-PCR tests were positive for IgG and/or
IgM. This highlights the usefulness of a serological assay to identify
asymptomatic infections and/or infections that are missed by RT-PCR.</p>
<h3 data-number="9.162.3" id="limitations-158"><span class="header-section-number">9.162.3</span> Limitations</h3>
<p>This group’s report generally confirms the findings of others that have
evaluated the acute antibody response to SARS-Cov-2. However, these data
would benefit from inclusion of data on whether the participants had a
documented history of viral infection. Moreover, serum samples that were
collected prior to SARS-Cov-2 outbreak from patients with other viral
infections would serve as a useful negative control for their assay.
Methodological limitations include that only one serum sample per case
was tested as well as the heat inactivation of serum samples prior to
testing. It has previously been reported that heat inactivation
interferes with the level of antibodies to SARS-Cov-2 and their protocol
may have resulted in diminished quantification of IgM, specifically <span class="citation" data-cites="p1c9Nilh">[<a href="#ref-p1c9Nilh" role="doc-biblioref">616</a>]</span>.</p>
<h3 data-number="9.162.4" id="significance-148"><span class="header-section-number">9.162.4</span> Significance</h3>
<p>Understanding the features of the antibody responses against SARS-CoV is
useful in the development of a serological test for the diagnosis of
COVID-19. This paper addresses the need for additional screening methods
that can detect the presence of infection despite lower viral titers.
Detecting the production of antibodies, especially IgM, which are
produced rapidly after infection can be combined with PCR to enhance
detection sensitivity and accuracy and map the full spread of infection
in communities, Moreover, serologic assays would be useful to screen
health care workers in order to identify those with immunity to care for
patients with COVID19.</p>
<h3 data-number="9.162.5" id="credit-158"><span class="header-section-number">9.162.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.163" id="sars-cov-2-specific-antibody-responses-in-covid-19-patients-1"><span class="header-section-number">9.163</span> SARS-CoV-2 specific antibody responses in COVID-19 patients</h2>
<p><span class="citation" data-cites="1BLNvgEao">[<a href="#ref-1BLNvgEao" role="doc-biblioref">617</a>]</span></p>
<h3 data-number="9.163.1" id="keywords-162"><span class="header-section-number">9.163.1</span> Keywords</h3>
<ul>
<li>immunoassay</li>
<li>antibody specificity</li>
<li>serology</li>
<li>cross-reactivity</li>
</ul>
<h3 data-number="9.163.2" id="main-findings-156"><span class="header-section-number">9.163.2</span> Main findings</h3>
<p>Antibodies specific to SARS-CoV-2 S protein, the S1 subunit and the RBD (receptor-binding domain) were detected in all SARS-CoV-2 patient sera by 13 to 21 days post onset of disease. Antibodies specific to SARS-CoV N protein (90% similarity to SARS-CoV-2) were able to neutralize SARS-CoV-2 by PRNT (plaque reduction neutralizing test). SARS-CoV-2 serum cross-reacted with SARS-CoV S and S1 proteins, and to a lower extent with MERS-CoV S protein, but not with the MERS-CoV S1 protein, consistent with an analysis of genetic similarity. No reactivity to SARS-CoV-2 antigens was observed in serum from patients with ubiquitous human CoV infections (common cold) or to non-CoV viral respiratory infections.</p>
<h3 data-number="9.163.3" id="limitations-159"><span class="header-section-number">9.163.3</span> Limitations</h3>
<p>Authors describe development of a serological ELISA based assay for the detection of neutralizing antibodies towards regions of the spike and nucleocapsid domains of the SARS-CoV-2 virus. Serum samples were obtained from PCR-confirmed COVID-19 patients. Negative control samples include a cohort of patients with confirmed recent exposure to non-CoV infections (i.e. adenovirus, bocavirus, enterovirus, influenza, RSV, CMV, EBV) as well as a cohort of patients with confirmed infections with ubiquitous human CoV infections known to cause the common cold. The study also included serum from patients with previous MERS-CoV and SARS-CoV zoonotic infections. This impressive patient cohort allowed the authors to determine the sensitivity and specificity of the development of their in-house ELISA assay. Of note, seroconversion was observed as early as 13 days following COVID-19 onset but the authors were not clear how disease onset was determined.</p>
<h3 data-number="9.163.4" id="significance-149"><span class="header-section-number">9.163.4</span> Significance</h3>
<p>Validated serological tests are urgently needed to map the full spread of SARS-CoV-2 in the population and to determine the kinetics of the antibody response to SARS-CoV-2. Furthermore, clinical trials are ongoing using plasma from patients who have recovered from SARS-CoV-2 as a therapeutic option. An assay such as the one described in this study could be used to screen for strong antibody responses in recovered patients. Furthermore, the assay could be used to screen health care workers for antibody responses to SARS-CoV-2 as personal protective equipment continues to dwindle. The challenge going forward will be to standardize and scale-up the various in-house ELISA’s being developed in independent laboratories across the world.</p>
<h2 data-number="9.164" id="a-brief-review-of-antiviral-drugs-evaluated-in-registered-clinical-trials-for-covid-19-1"><span class="header-section-number">9.164</span> A brief review of antiviral drugs evaluated in registered clinical trials for COVID-19</h2>
<p>Belhadi et al. <span class="citation" data-cites="K41YFM2N">[<a href="#ref-K41YFM2N" role="doc-biblioref">618</a>]</span></p>
<h3 data-number="9.164.1" id="keywords-163"><span class="header-section-number">9.164.1</span> Keywords</h3>
<ul>
<li>Clinical trials</li>
<li>COVID-19</li>
<li>SARS CoV-2</li>
<li>2019-nCoV</li>
<li>SARS Cov-2</li>
<li>Hcov-19</li>
<li>novel coronal virus</li>
<li>new corona virus</li>
<li>antiviral drugs</li>
</ul>
<h3 data-number="9.164.2" id="main-findings-157"><span class="header-section-number">9.164.2</span> Main Findings</h3>
<p>Summary of clinical trials registered as
of March7, 2020 from U.S, Chinese, Korean, Iranian and European
registries. Out of the 353 studies identified, 115 were selected for
data extraction. 80% of the trials were randomized with parallel
assignment and the median number of planned inclusions was 63 (IRQ,
36-120). Most frequent therapies in the trials included; 1) antiviral
drugs [lopinavir/ritonavir (n-15); umifenovir (n=9); favipiravir (n=7);
redmesivir (n=5)]; 2) anti-malaria drugs [chloroquine (n-11);
hydroxychloroquine (n=7)}; immunosuppressant drugs [methylprednisolone
(n=5)]; and stem cell therapies (n=23). Medians of the total number of
planned inclusions per trial for these therapies were also included.
Stem cells and lopunavir/ritonavir were the most frequently evaluated
candidate therapies (23 and 15 trials respectively), whereas remdesivir
was only tested in 5 trials but these trials had the highest median
number of planned inclusions per trial (400, IQR 394-453). Most of the
agents used in the different trials were chosen based on preclinical
assessments of antiviral activity against SARS CoV and MERS Cov corona
viruses.</p>
<p>The primary outcomes of the studies were clinical (66%); virological
(23%); radiological (8%); or immunological (3%). The trials were
classified as those that included patients with severe disease only;
trials that included patients with moderate disease; and trials that
included patients with severe or moderate disease.</p>
<h3 data-number="9.164.3" id="limitations-160"><span class="header-section-number">9.164.3</span> Limitations</h3>
<p>The trials evaluated provided incomplete information:
23% of these were phase IV trials but the bulk of the trials (54%) did
not describe the phase of the study. Only 52% of the trials (n=60)
reported treatment dose and only 34% (n=39) reported the duration. A lot
of the trials included a small number of patients and the trials are
still ongoing, therefore no insight was provided on the outcome of the
trials.</p>
<h3 data-number="9.164.4" id="significance-150"><span class="header-section-number">9.164.4</span> Significance</h3>
<p>Nonetheless, this review serves as framework for
identifying COVID-19 related trials, which can be expanded upon as new
trials begin at an accelerated rate as the disease spreads around the
world.</p>
<h3 data-number="9.164.5" id="credit-159"><span class="header-section-number">9.164.5</span> Credit</h3>
<p><em>This review was undertaken by K Alexandropoulos as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.165" id="ace-2-expression-in-the-small-airway-epithelia-of-smokers-and-copd-patients-implications-for-covid-19-1"><span class="header-section-number">9.165</span> ACE-2 Expression in the Small Airway Epithelia of Smokers and COPD Patients: Implications for COVID-19</h2>
<p>Leung et al. <em>medRxiv</em>. <span class="citation" data-cites="eTsOvBVf">[<a href="#ref-eTsOvBVf" role="doc-biblioref">619</a>]</span></p>
<h3 data-number="9.165.1" id="keywords-164"><span class="header-section-number">9.165.1</span> Keywords</h3>
<ul>
<li>chronic obstructive pulmonary disease</li>
<li>COPD</li>
<li>smokingE-2</li>
<li>risk factors</li>
</ul>
<h3 data-number="9.165.2" id="main-findings-158"><span class="header-section-number">9.165.2</span> Main Findings</h3>
<p>In bronchial epithelial samples from 3 different cohorts of
individuals, ACE-2 gene expression was found to be significantly
increased in both COPD patients and smokers relative to healthy
controls. Across all test subjects, ACE-2 gene expression was also
highly correlated with decreased forced expiratory volume in 1 second
(FEV1), which may explain the increased COVID-19 disease severity in
COPD patients. Former smokers were also found to show decreased ACE2
expression relative to current smokers and had no significant difference
when compared to non-smokers.</p>
<h3 data-number="9.165.3" id="limitations-161"><span class="header-section-number">9.165.3</span> Limitations</h3>
<p>While the upregulation of ACE-2 is an interesting
hypothesis for COVID-19 disease severity in COPD patients, this study
leaves many more unanswered questions than it addresses. Further studies
are required to show whether the specific cell type isolated in these
studies is relevant to the pathophysiology of COVID-19. Furthermore,
there is no attempt to show whether that increased ACE-2 expression
contributes to greater disease severity. Does the increased ACE-2
expression lead to greater infectivity with SARS-CoV-2? There is no
mechanistic explanation for why ACE-2 levels are increased in COPD
patients. The authors could also have considered the impact of
co-morbidities and interventions such as corticosteroids or
bronchodilators on ACE-2 expression. Finally, given the extensive
sequencing performed, the authors could have conducted significantly
more in-depth analyses into gene signature differences.</p>
<h3 data-number="9.165.4" id="significance-151"><span class="header-section-number">9.165.4</span> Significance</h3>
<p>This study attempts to address an
important clinical finding that both smokers and COPD patients show
increased mortality from COVID-19. The novel finding that ACE-2
expression is induced in smokers and COPD patients suggests not only a
mechanism for the clinical observation, but also highlights the
potential benefit of smoking cessation in reducing the risk of severe
COVID-19 disease.</p>
<h3 data-number="9.165.5" id="credit-160"><span class="header-section-number">9.165.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.166" id="dynamic-profile-of-severe-or-critical-covid-19-cases-1"><span class="header-section-number">9.166</span> Dynamic profile of severe or critical COVID-19 cases</h2>
<p><span class="citation" data-cites="18KZGwJ0J">[<a href="#ref-18KZGwJ0J" role="doc-biblioref">620</a>]</span></p>
<h3 data-number="9.166.1" id="keywords-165"><span class="header-section-number">9.166.1</span> Keywords</h3>
<ul>
<li>Coronavirus Disease 2019 (COVID-19)</li>
<li>SARS-CoV-2</li>
<li>progressive lymphopenia (PLD)</li>
<li>T-lymphocytes</li>
<li>clinical data</li>
<li>co-infection</li>
<li>influenza A</li>
</ul>
<h3 data-number="9.166.2" id="main-findings-159"><span class="header-section-number">9.166.2</span> Main Findings</h3>
<p>Authors evaluate clinical correlates of 10 patients (6 male
and 4 female) hospitalized for severe Acute Respiratory Syndrome
Coronavirus 2 (SARS-CoV-2). All patients required oxygen support and
received broad spectrum antibiotics and 6 patients received anti-viral
drugs. Additionally, 40% of patients were co-infected with influenza A.
All 10 patients developed lymphopenia, two of which developed
progressive lymphopenia (PLD) and died. Peripheral blood (PB)
lymphocytes were analyzed – low CD4 and CD8 counts were noted in most
patients, though CD4:CD8 ratio remained normal.</p>
<h3 data-number="9.166.3" id="limitations-162"><span class="header-section-number">9.166.3</span> Limitations</h3>
<p>The authors evaluated a small cohort of severe
SARS-CoV-2 cases and found an association between T cell lymphopenia and
adverse outcomes. However, this is an extremely small and diverse cohort
(40% of patients were co-infected with influenza A). These findings need
to be validated in a larger cohort. Additionally, the value of this data
would be greatly increased by adding individual data points for each
patient as well as by adding error bars to each of the figures.</p>
<h3 data-number="9.166.4" id="significance-152"><span class="header-section-number">9.166.4</span> Significance</h3>
<p>This study provides a collection of clinical data and
tracks evolution of T lymphocyte in 10 patients hospitalized for
SARS-CoV-2, of which 4 patients were co-infected with influenza A.</p>
<h3 data-number="9.166.5" id="credit-161"><span class="header-section-number">9.166.5</span> Credit</h3>
<p><em>This review was undertaken by Katherine Lindblad and Tamar Plitt as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.167" id="association-between-clinical-laboratory-and-ct-characteristics-and-rt-pcr-results-in-the-follow-up-of-covid-19-patients-1"><span class="header-section-number">9.167</span> Association between Clinical, Laboratory and CT Characteristics and RT-PCR Results in the Follow-up of COVID-19 patients</h2>
<p>Fu et al. <em>medRxiv</em>. <span class="citation" data-cites="1BJXDZ04c">[<a href="#ref-1BJXDZ04c" role="doc-biblioref">621</a>]</span></p>
<h3 data-number="9.167.1" id="keywords-166"><span class="header-section-number">9.167.1</span> Keywords</h3>
<ul>
<li>COVID-19</li>
<li>clinical</li>
<li>lymphocyte</li>
<li>CRP</li>
<li>LDH</li>
<li>HSST TNT</li>
<li>PCR test</li>
<li>readmission</li>
<li>CT</li>
<li>GGO</li>
<li>disease progression</li>
</ul>
<h3 data-number="9.167.2" id="study-description-2"><span class="header-section-number">9.167.2</span> Study description</h3>
<p>Data analyzed from 52 COVID-19 patients admitted
and then discharged with COVID-19. Clinical, laboratory, and
radiological data were longitudinally recorded with illness timecourse
(PCR + to PCR-) and 7 patients (13.5%) were readmitted with a follow up
positive test (PCR+) within two weeks of discharge.</p>
<h3 data-number="9.167.3" id="main-findings-160"><span class="header-section-number">9.167.3</span> Main Findings</h3>
<ul>
<li><p>At admission:</p>
<ul>
<li><p>The majority of patients had increased CRP at admission (63.5%).</p></li>
<li><p>LDH, and HSST TNT were significantly increased at admission.</p></li>
<li><p>Radiographic signs via chest CT showed increased involvement in
lower lobes: right lower lobe (47 cases, 90.4%), left lower lobe
(37 cases, 71.2%).</p></li>
<li><p>GGO (90.4%), interlobular septal thickening (42.3%), vascular
enlargement (42.3%), and reticulation (11.5%) were most commonly
observed.</p></li>
</ul></li>
<li><p>After negative PCR test (discharge):</p>
<ul>
<li><p>CRP levels decreased lymphocyte counts (#/L) increased
significantly (CD3+, CD3+/8+ and CD3+/4+) after negative PCR.</p></li>
<li><p>Consolidation and mixed GGO observed in longitudinal CT imaging
w different extents of inflammatory exudation in lungs, with
overall tendency for improvement (except 2/7 patients that were
readmitted after discharge with re-positive test) after negative
PCR.</p></li>
</ul></li>
<li><p>Seven patients repeated positive RT-PCR test and were readmitted to
the hospital (9 to 17 day after initial discharge).</p>
<ul>
<li><p>Follow up CT necessary to monitor improvement during recovery
and patients with lesion progression should be given more
attention.</p></li>
<li><p>Dynamic CT in addition to negative test essential in clinical
diagnosis due to nasal swab PCR sampling bias (false-negatives).</p></li>
<li><p>Increase in CRP occurred in 2 readmitted patients (and decr. in
lymphocytes in one patient), but was not correlated with new
lesions or disease progression vs. improvement (very low N).</p></li>
<li><p>Patients readmitted attributed to false-negative PCR vs.
re-exposure.</p></li>
</ul></li>
</ul>
<h3 data-number="9.167.4" id="limitations-163"><span class="header-section-number">9.167.4</span> Limitations</h3>
<p>Patients sampled in this study were generally
younger (65.4% &lt; 50 yrs) and less critically ill/all discharged.
Small number of recovered patients (N=18). Time of follow up was
relatively short.. Limited clinical information available about patients
with re-positive test (except CRP and lymph tracking).</p>
<h3 data-number="9.167.5" id="extended-results-1"><span class="header-section-number">9.167.5</span> Extended Results</h3>
<p><em>NOTE</em>: Patients sampled in this study were generally younger (65.4%
&lt; 50 yrs) and less critically ill/all discharged. After two
consecutive negative PCR tests, patients were discharged.</p>
<p>Clinical Results at Admission</p>
<ul>
<li><p>Median interval disease onset to admission (5 days, IQR: 3-7)</p></li>
<li><p>Most common symptoms included fever, fatigue, dry cough, and
expectoration.</p></li>
<li><p>Fifteen patients had reduced lymphocyte counts (28.8%).</p></li>
<li><p>No change in WBC or Neutrophil counts.</p></li>
<li><p><strong>The majority of patients had increased CRP at admission (63.5%).</strong></p></li>
<li><p><strong>LDH, and HSST TNT were significantly increased at admission.</strong></p></li>
<li><p>Fibrinogen was trending high though not significant.</p></li>
<li><p>No major changes in liver function observed.</p></li>
<li><p><strong>Radiographic signs via chest CT showed increased involvement in
lower lobes: right lower lobe (47 cases, 90.4%), left lower lobe (37
cases, 71.2%).</strong></p></li>
<li><p><strong>GGO (90.4%), interlobular septal thickening (42.3%), vascular
enlargement (42.3%)</strong>, and reticulation (11.5%) were most commonly
observed.</p></li>
</ul>
<p>Change in Clinical Results following Negative Test</p>
<ul>
<li><p><strong>CRP levels decreased after negative PCR.</strong></p></li>
<li><p><strong>Lymphocyte counts (#/L) increased significantly (CD3+, CD3+/8+
and CD3+/4+).</strong></p></li>
<li><p>No significant change to CD4/8 ratio.</p></li>
<li><p>LDH, HSST TNT, and Fibronegin remained high throughout, though range
observed decreased over time.</p></li>
<li><p><strong>Consolidation and mixed GGO observed in longitudinal CT imaging.</strong></p></li>
<li><p><strong>Patients showed different extents of inflammatory exudation in
lungs, with overall tendency for improvement (except 2/7 patients
that were readmitted after discharge with re-positive test).</strong></p></li>
</ul>
<p>Patients Readmitted with PCR+ test</p>
<ul>
<li><p><strong>Seven patients repeated positive RT-PCR test and were readmitted
to the hospital (9 to 17 day after initial discharge).</strong></p></li>
<li><p>Improvement during readmission in 4 patients and observation of
segmental progression CT in 2 patients (2/18 or 11.1% - re-positive
9 and 10 days post-discharge).</p></li>
<li><p>Two patients showed new GGO, while others improved greatly.</p></li>
<li><p><strong>Follow up CT necessary to monitor improvement during recovery and
patients with lesion progression should be given more attention.</strong></p></li>
<li><p><strong>Dynamic CT in addition to negative test essential in clinical
diagnosis due to nasal swab PCR sampling bias (false-negatives).</strong></p></li>
<li><p><strong>Increase in CRP occurred in 2 readmitted patients (and decr. in
lymphocytes in one patient), but was not correlated with new lesions
or disease progression vs. improvement (very low N).</strong></p></li>
</ul>
<h3 data-number="9.167.6" id="significance-153"><span class="header-section-number">9.167.6</span> Significance</h3>
<p>Study tracked key clinical features associated with
disease progression, recovery, and determinants of clinical
diagnosis/management of COVID-19 patients.</p>
<h3 data-number="9.167.7" id="credit-162"><span class="header-section-number">9.167.7</span> Credit</h3>
<p><em>This review was undertaken by Natalie Vaninov as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.168" id="an-orally-bioavailable-broad-spectrum-antiviral-inhibits-sars-cov-2-and-multiple-2-endemic-epidemic-and-bat-coronavirus-1"><span class="header-section-number">9.168</span> An orally bioavailable broad-spectrum antiviral inhibits SARS-CoV-2 and multiple 2 endemic, epidemic and bat coronavirus</h2>
<p>Sheahan et al. <em>bioRxiv</em>. <span class="citation" data-cites="iMCz5Twj">[<a href="#ref-iMCz5Twj" role="doc-biblioref">622</a>]</span></p>
<h3 data-number="9.168.1" id="keywords-167"><span class="header-section-number">9.168.1</span> Keywords</h3>
<ul>
<li>Treatment</li>
<li>Antiviral</li>
<li>Broad spectrum antiviral</li>
<li>ribonucleoside analog β-D-N4 30 hydroxycytidine (NHC)</li>
<li>Remdesivir</li>
</ul>
<h3 data-number="9.168.2" id="main-findings-161"><span class="header-section-number">9.168.2</span> Main Findings</h3>
<p>β-D-N4 30 –hydroxycytidine (NHC, EIDD-1931) is an
orally bioavailable ribonucleoside with antiviral activity against
various RNA viruses including Ebola, Influenza and CoV. NHC activity
introduce<em>d</em><del>s</del> mutations in the viral (but not cellular) RNA in a
dose dependent <em>manner</em> that directly correlated with a decrease in
viral titers. Authors show that NHC inhibited multiple genetically
distinct Bat-CoV viruses in human primary epithelial cells <em>without
affecting</em> <em>cell viability even at high concentrations (100</em> <em>µM)</em>.
Prophylactic oral administration of NHC in C57BL/6 mice reduce lung
titers of SARS-CoV and prevented weight loss and hemorrhage. Therapeutic
administration of NHC in C57BL/6 mice 12 hours post infected with
SARS-CoV reduced acute lung injury, viral titer, and lung hemorrhage.
The degree of clinical benefit was dependent on the time of treatment
initiation post infection. The authors also demonstrate that NHC reduces
MERS-CoV infection titers, pathogenesis, and viral RNA in prophylactic
and therapeutic settings.</p>
<h3 data-number="9.168.3" id="limitations-164"><span class="header-section-number">9.168.3</span> Limitations</h3>
<p>Most of the experiments were conducted using MERS-CoV, and
SARS-CoV and a few experiments were conducted using other strains of CoV
as opposed to SARS-CoV-2. The authors note the core residues that make
up the RNA interaction sites (which constitutes the NHC interaction
sites) are highly conserved among CoV and because of this conservation
their understanding is that NHC can inhibit a broad-spectrum of CoV
including SARS-CoV-2.</p>
<p>The increased viral mutation rates associated with NHC activity may have
adverse effects if mutations cause the virus to become drug resistant,
more infectious or speed-up immune evasion. <em>In addition, the temporal
diminishing effectiveness of NHC on clinical outcome when NHC was used
therapeutically is concerning. However, the longer window (7-10 days)
for clinical disease onset in human patients from the time of infection
compared to that of mice (24-48 hours), may associate with increased NHC
effectiveness in the clinic.</em></p>
<h3 data-number="9.168.4" id="significance-154"><span class="header-section-number">9.168.4</span> Significance</h3>
<p>Prophylactic or therapeutic oral administration of NHC
reduces lung titers and prevents acute lung failure in C57B\6 mice
infected with CoV. Given its <em>broad-spectrum antiviral activity, NHC
could turn out to be a useful drug for treating current, emerging and
future corona virus outbreaks.</em>
 
#### Credit</p>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<p><strong>Title:</strong> Anti-hypertensive Angiotensin II receptor blockers associated
to mitigation of disease severity in elderly COVID-19 patients</p>
<p><strong>Keywords:</strong> SARS-CoV-2, ACE-2, Renin-angiotensin system, Hypertension</p>
<p><strong>Main findings:</strong> The authors analyzed clinical data obtained from
COVID-19 patients and categorized them based on the antihypertensive
drugs they were taking. They then investigated its association with
morbidity and mortality of pneumonic COVID-19 patients. ARBs were found
to be associated with a reduced risk of pneumonia morbidity in a total
of 70,346 patients in three studies. They found that in the elderly
(age&gt;65) group of COVID-19 patients with hypertension comorbidity,
the risk of severe disease was significantly lower in patients who were
on ARB anti-hypertensive drugs prior to hospitalization compared to
patients who took no drugs. Also, through their meta-analysis of the
literature, the authors reported that ARB anti-hypertensive drugs were
associated with a decreased risk of severe disease in elderly COVID-19
patients.</p>
<p><strong>Critical Analyses</strong>:</p>
<ol type="1">
<li><p>Retrospective study with large potential for confounder bias.</p></li>
<li><p>Their inference that ARB is better than other anti-hypertensive
drugs is based on literature met-analysis.</p></li>
<li><p>P-values could not be computed for some subsets because of very
low/no patients in these categories(ref to table-1;ACEI, thiazide
and BB)</p></li>
</ol>
<p><strong>Relevance:</strong> Anti-hypertensive ARB drugs taken by COVID-19 patients
prior to entering the hospital may be associated with improved morbidity
and mortality of pneumonia in elderly COVID-19 patients although
confounders may bias results.</p>
<h2 data-number="9.169" id="identification-of-antiviral-drug-candidates-against-sars-cov-2-from-fda-approved-drugs-1"><span class="header-section-number">9.169</span> Identification of antiviral drug candidates against SARS-CoV-2 from FDA-approved drugs</h2>
<p>Sangeun Jeon et al. <span class="citation" data-cites="ZDP9vSpg">[<a href="#ref-ZDP9vSpg" role="doc-biblioref">623</a>]</span></p>
<h3 data-number="9.169.1" id="keywords-168"><span class="header-section-number">9.169.1</span> Keywords</h3>
<ul>
<li>COVID-19</li>
<li>SARS CoV-2</li>
<li>antiviral drugs</li>
<li>niclosamide</li>
<li>ciclesonide</li>
</ul>
<h3 data-number="9.169.2" id="main-findings-162"><span class="header-section-number">9.169.2</span> Main Findings</h3>
<p>A panel of ~3,000 FDA- and IND-approved
antiviral drugs were previously screened for inhibitory efficacy against
SARS CoV, a coronavirus related to the novel coronavirus SARS CoV-2
(79.5%) homology. 35 of these drugs along with another 15 (suggested by
infectious disease specialists) were tested in vitro for their ability
to inhibit SARS CoV-2 infectivity of Vero cells while preserving cell
viability. The infected cells were scored by immunofluorescence analysis
using an antibody against the N protein of SARS CoV-2. Chloroquine,
lopinavir and remdesivir were used as reference drugs.</p>
<p>Twenty four out of 50 drugs exhibited antiviral activity with IC50
values ranging from 0.1-10µM. Among these, two stood ou: 1) the-anti
helminthic drug niclosamide which exhibited potent antiviral activity
against SARS CoV-2 (IC50=0.28 µM). The broad-spectrum antiviral effect
of niclosamide against SARS and MERS-CoV have been previously documented
and recent evidence suggests that in may inhibit autophagy and reduce
MERS C0V replication. 2) Ciclesonide, a corticosteroid used to treat
asthma and allergic rhinitis, also exhibited antiviral efficacy but with
a lower IC50 (4.33µM) compared to niclosamide. The antiviral effects of
ciclesonide were directed against NSP15, a viral riboendonuclease which
is the molecular target of this drug.</p>
<h3 data-number="9.169.3" id="limitations-165"><span class="header-section-number">9.169.3</span> Limitations</h3>
<p>The drugs were tested against SARS CoV-2 infectivity in
vitro only, therefore preclinical studies in animals and clinical trials
in patients will be need for validation of these drugs as therapeutic
agents for COVID-19. In addition, niclosamide exhibits low adsorption
pharmatokinetically which could be alleviated with further development
of drug formulation to increase effective delivery of this drug to
target tissues. Nonetheless, niclosamide and ciclesonide represent
promising therapeutic agents against SARS CoV-2 given that other
compounds tested in the same study including favipiravir (currently used
in clinical trials) and atazanavir (predicted as the most potent
antiviral drug by AI-inference modeling) did not exhibit antiviral
activity in the current study.</p>
<h3 data-number="9.169.4" id="credit-163"><span class="header-section-number">9.169.4</span> Credit</h3>
<p><em>This review was undertaken by K Alexandropoulos as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.170" id="respiratory-disease-and-virus-shedding-in-rhesus-macaques-inoculated-with-sars-cov-2-1"><span class="header-section-number">9.170</span> Respiratory disease and virus shedding in rhesus macaques inoculated with SARS-CoV-2</h2>
<p>Munster et al. <em>bioRxiv</em>. <span class="citation" data-cites="LYFzvlzE">[<a href="#ref-LYFzvlzE" role="doc-biblioref">624</a>]</span></p>
<h3 data-number="9.170.1" id="keywords-169"><span class="header-section-number">9.170.1</span> Keywords</h3>
<p>animal model, pulmonary infiltrates, dynamic
of antibody response, cytokine</p>
<h3 data-number="9.170.2" id="main-findings-163"><span class="header-section-number">9.170.2</span> Main Findings</h3>
<p>Inoculation of 8 Resus macaques with SARS-Cov-2 , which
all showed clinical signs of infection (respiratory pattern, reduced
appetite, weight loss, elevated body temperature) resulting in moderate,
transient disease. Four animals were euthanized at 3dpi, the 4 others at
21 dpi. Study of viral loads in different organs showed that nose swab
and throat swabs were the most sensitive, with bronchio-alveolar lavage.
Interstitial pneumonia was visible in radiographies and at the
histological scale too. Clinically, the macaques had similar symptoms as
described in human patients with moderate disease.</p>
<p>Viral shedding was consistently detected in nose swabs and throat swabs
immediately after infection but less consistent thereafter which could
reflect virus administration route (intranasal, oral). Bronchoalveolar
lavages performed as a measure of virus replication in the lower
respiratory tract on animals maintained for 21 days, contained high
viral loads in 1 and 3dpi. The majority of the animals exhibited
pulmonary edema and mild to moderate interstitial pneumonia on terminal
bronchioles. In addition to the lung, viral RNA could also be detected
throughout the respiratory track where viral replication mainly
occurred.</p>
<p>Immunologic responses included leukocytosis, neutrophilia, monocytosis
and lymphopenia in the majority of the animals at 1dpi. Lymphocytes and
monocytes re-normalized at 2dpi. Neutrophils declined after 3dpi and
through 10dpi after which they started to recover. After infection,
serum analysis revealed significant increases in <strong>IL1ra, IL6, IL10,
IL15, MCP-1, MIP-1b, but quick normalization</strong> (3dpi). <strong>Antibody
response started around 7dpi, and the antibody titers stayed elevated
until 21dpi</strong> (day of animal euthanasia).</p>
<h3 data-number="9.170.3" id="limitations-166"><span class="header-section-number">9.170.3</span> Limitations</h3>
<p>The macaques were inoculated via a combination
of intratracheal, intranasal, ocular and oral routes, which might not
reproduce how humans get infected. Maybe this can lead to different
dynamics in the host immune response. Also, the authors noted that the
seroconversion was not directly followed by a decline in viral loads, as
observed in covid19 patients.</p>
<h3 data-number="9.170.4" id="significance-155"><span class="header-section-number">9.170.4</span> Significance</h3>
<p>This work confirms that rhesus
macaques can be a good model to study Covid-19, as it has been shown by
other groups <span class="citation" data-cites="3XqFhi8l unVgVIfU 1EcOLnVXk">[<a href="#ref-3XqFhi8l" role="doc-biblioref">599</a>,<a href="#ref-1EcOLnVXk" role="doc-biblioref">612</a>,<a href="#ref-unVgVIfU" role="doc-biblioref">625</a>]</span>. While these experiments recapitulate
moderate COVID-19 in humans, the mode of inoculation via a combination
of intratracheal, intranasal, ocular and oral routes, might not
reproduce how humans get infected and may lead to different dynamics in
the host immune response. For example, the authors noted that the
seroconversion was not directly followed by a decline in viral loads, as
observed in COVID-19 patients. Therefore, it will be interesting to
follow their antibody titers longer and further assess the
possibility/effect of reinfection in these macaques. It is essential to
be able to understand the dynamic of the disease and associated immune
responses, and to work on vaccine development and antiviral drug
testing.</p>
<h3 data-number="9.170.5" id="credit-164"><span class="header-section-number">9.170.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.171" id="ace2-expression-is-increased-in-the-lungs-of-patients-with-comorbidities-associated-with-severe-covid-19-1"><span class="header-section-number">9.171</span> ACE2 Expression is Increased in the Lungs of Patients with Comorbidities Associated with Severe COVID-19</h2>
<p><span class="citation" data-cites="wjtVy2v0">[<a href="#ref-wjtVy2v0" role="doc-biblioref">626</a>]</span></p>
<h3 data-number="9.171.1" id="keywords-170"><span class="header-section-number">9.171.1</span> Keywords</h3>
<ul>
<li>CoVID-19</li>
<li>2019-nCoV</li>
<li>SARS-CoV-2</li>
<li>lung</li>
<li>comorbidities</li>
<li>histone</li>
<li>epigenetics</li>
</ul>
<h3 data-number="9.171.2" id="main-findings-164"><span class="header-section-number">9.171.2</span> Main Findings</h3>
<ul>
<li><p>Transcriptomic analysis using systems-level meta-analysis and
network analysis of existing literature to determine ACE2 regulation
in patients who have frequent COVID-19 comorbidities [eg-
cardiovascular diseases, familial pulmonary hypertension, cancer].</p></li>
<li><p>Enrichment analyses indicated pathways associated with inflammation,
metabolism, macrophage autophagy, and ER stress.</p></li>
<li><p>ACE2 higher in adenocarcinoma compared to adjacent normal lung; ACE2
higher in COPD patients compared to normal.</p></li>
<li><p>Co-expression analysis identified genes important to viral entry
such as RAB1A, ADAM10, HMGBs, and TLR3 to be associated with ACE2 in
diseased lungs.</p></li>
<li><p>ACE2 expression could be potentially regulated by enzymes that
modify histones, including HAT1, HDAC2, and KDM5B.</p></li>
</ul>
<h3 data-number="9.171.3" id="limitations-167"><span class="header-section-number">9.171.3</span> Limitations:</h3>
<ul>
<li><p>Not actual CoVID-19 patients with co-morbidities, so interpretations
in this study need to be confirmed by analyzing upcoming
transcriptomics from CoVID-19 patients having co-morbidity metadata.</p></li>
<li><p>As mentioned by authors, study does not look at diabetes and
autoimmunity as risk factors in CoVID-19 patients due to lack of
data; would be useful to extend such analyses to those datasets when
available.</p></li>
<li><p>Co-expression analysis is perfunctory and needs
validation-experiments especially in CoVID-19 lung samples to mean
anything.</p></li>
<li><p>Epigenomic analyses are intriguing but incomplete, as existence of
histone marks does not necessarily mean occupancy. Would be
pertinent to check cell-line data (CCLE) or actual CoVID-19 patient
samples to confirm ACE2 epigenetic control.</p></li>
</ul>
<h3 data-number="9.171.4" id="significance-156"><span class="header-section-number">9.171.4</span> Significance</h3>
<ul>
<li><p>Study implies vulnerable populations have ACE2 upregulation that
could promote CoVID-19 severity. Shows important data-mining
strategy to find gene-networks associated with ACE2 upregulation in
co-morbid patients.</p></li>
<li><p>Several of the genes co-upregulated with ACE2 in diseased lung might
play an important role in CoVID-19 and can be preliminary targets
for therapeutics</p></li>
<li><p>If in silico findings hold true, epigenetic control of ACE2
expression could be a new target for CoVID-19 therapy with
strategies such as KDM5 demethylases.</p></li>
</ul>
<h3 data-number="9.171.5" id="credit-165"><span class="header-section-number">9.171.5</span> Credit</h3>
<p><em>Review by Samarth Hegde as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.172" id="meplazumab-treats-covid-19-pneumonia-an-open-labelled-concurrent-controlled-add-on-clinical-trial-1"><span class="header-section-number">9.172</span> Meplazumab treats COVID-19 pneumonia: an open-labelled, concurrent controlled add-on clinical trial</h2>
<p>Bian et al. <em>medRxiv</em>. <span class="citation" data-cites="DFhY5nfb">[<a href="#ref-DFhY5nfb" role="doc-biblioref">627</a>]</span></p>
<h3 data-number="9.172.1" id="keywords-171"><span class="header-section-number">9.172.1</span> Keywords</h3>
<ul>
<li>Meplazumab</li>
<li>CD147</li>
<li>humanized antibody</li>
<li>clinical trial</li>
</ul>
<h3 data-number="9.172.2" id="main-findings-165"><span class="header-section-number">9.172.2</span> Main Findings</h3>
<p>This work is based on previous work by the same group
that demonstrated that SARS-CoV-2can also enter host cells via CD147
(also called Basigin, part of the immunoglobulin superfamily, is
expressed by many cell types) consistent with their previous work with
SARS-CoV-1. <sup>1</sup> A prospective clinical trial was conducted with 17
patients receiving Meplazumab, a humanized anti-CD147 antibody, in
addition to all other treatments. 11 patients were included as a control
group (non-randomized).</p>
<p>They observed a faster overall improvement rate in the Meplazumab group
(e.g. at day 14 47% vs 17% improvement rate) compared to the control
patients. Also, virological clearance was more rapid with median of 3
days in the Meplazumab group vs 13 days in control group. In laboratory
values, a faster normalization of lymphocyte counts in the Meplazumab
group was observed, but no clear difference was observed for CRP levels.</p>
<h3 data-number="9.172.3" id="limitations-168"><span class="header-section-number">9.172.3</span> Limitations</h3>
<p>While the results from the study are encouraging, this
study was non-randomized, open-label and on a small number of patients,
all from the same hospital. It offers evidence to perform a larger scale
study. Selection bias as well as differences between treatment groups
(e.g. age 51yo vs 64yo) may have contributed to results. The authors
mention that there was no toxic effect to Meplazumab injection but more
patient and longer-term studies are necessary to assess this.</p>
<h3 data-number="9.172.4" id="significance-157"><span class="header-section-number">9.172.4</span> Significance</h3>
<p>These results seem promising as for now there are
limited treatments for Covid-19 patients, but a larger cohort of patient
is needed. CD147 has already been described to facilitate HIV <span class="citation" data-cites="yP8TXdq2">[<a href="#ref-yP8TXdq2" role="doc-biblioref">628</a>]</span>,
measles virus <span class="citation" data-cites="X8l1n4zY">[<a href="#ref-X8l1n4zY" role="doc-biblioref">629</a>]</span>, and malaria <span class="citation" data-cites="YcyNaIVC">[<a href="#ref-YcyNaIVC" role="doc-biblioref">630</a>]</span> entry into host cells. This group was
the first to describe the CD147-spike route of SARS-Cov-2 entry in host
cells <span class="citation" data-cites="14y8UvAbH">[<a href="#ref-14y8UvAbH" role="doc-biblioref">601</a>]</span> p147. Indeed, they had previously shown in 2005 that SARS-Cov
could enter host cells via this transmembrane protein <span class="citation" data-cites="m9yLJ5Da">[<a href="#ref-m9yLJ5Da" role="doc-biblioref">631</a>]</span>. Further
biological understanding of how SARS-CoV-2 can enter host cells and how
this integrates with ACE2R route of entry is needed. Also, the specific
cellular targets of the anti-CD147 antibody need to be assessed, as this
protein can be expressed by many cell types and has been shown to
involved in leukocytes aggregation <span class="citation" data-cites="J0ibHE3l">[<a href="#ref-J0ibHE3l" role="doc-biblioref">632</a>]</span>. Lastly, Meplazumab is not a
commercially-available drug and requires significant health resources to
generate and administer which might prevent rapid development and use.</p>
<h3 data-number="9.172.5" id="credit-166"><span class="header-section-number">9.172.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.173" id="potent-human-neutralizing-antibodies-elicited-1-by-sars-cov-2-infection-1"><span class="header-section-number">9.173</span> Potent human neutralizing antibodies elicited 1 by SARS-CoV-2 infection</h2>
<p>Ju et al. <em>bioRxiv</em>. <span class="citation" data-cites="GhJYjnft">[<a href="#ref-GhJYjnft" role="doc-biblioref">178</a>]</span></p>
<h3 data-number="9.173.1" id="keywords-172"><span class="header-section-number">9.173.1</span> Keywords</h3>
<ul>
<li>monoclonal antibodies</li>
<li>neutralization</li>
<li>antibody cross-reactivity</li>
<li>Receptor Binding Domain</li>
</ul>
<h3 data-number="9.173.2" id="main-findings-166"><span class="header-section-number">9.173.2</span> Main Findings</h3>
<p>In this study the authors report the affinity, cross reactivity (with
SARS-CoV and MERS-CoV virus) and viral neutralization capacity of 206
monoclonal antibodies engineered from isolated IgG memory B cells of
patients suffering from SARS-CoV-2 infection in Wuhan, China. All
patients but one recovered from disease. Interestingly, the patient that
did not recover had less SARS-CoV-2 specific B cells circulating
compared to other patients.</p>
<p>Plasma from all patients reacted to trimeric Spike proteins from
SARS-CoV-2, SARS-CoV and MERS-CoV but no HIV BG505 trimer. Furthermore,
plasma from patients recognized the receptor binding domain (RBD) from
SARS-CoV-2 but had little to no cross-reactivity against the RBD of
related viruses SARS-CoV and MERS-CoV, suggesting significant
differences between the RBDs of the different viruses. Negligible levels
of cross-neutralization using pseudoviruses bearing Spike proteins of
SARS-CoV-2, SARS-CoV or MERS-CoV, were observed, corroborating the ELISA
cross-reactivity assays on the RBDs.</p>
<p>SARS-CoV-2 RBD specific B cells constituted 0.005-0.065% of the total B
cell population and 0.023-0.329% of the memory subpopulation. SARS-CoV
specific IgG memory B cells were single cell sorted to sequence the
antibody genes that were subsequently expressed as recombinant IgG1
antibodies. From this library, 206 antibodies with different binding
capacities were obtained. No discernible patterns of VH usage were found
in the 206 antibodies suggesting immunologically distinct responses to
the infection. Nevertheless, most high-binding antibodies were derived
by clonal expansion. Further analyses in one of the patient derived
clones, showed that the antibodies from three different timepoints did
not group together in phylogenetic analysis, suggesting selection during
early infection.</p>
<p>Using surface plasmon resonance (SPR) 13 antibodies were found to have
10<sup>-8</sup> tp 10<sup>-9</sup> dissociation constants (Kd). Of the 13 antibodies, two
showed 98-99% blocking of SARS-CoV-2 RBD-ACE2 receptor binding in
competition assays. Thus, low Kd values alone did not predict ACE2
competing capacities. Consistent with competition assays the two
antibodies that show high ACE2 blocking (P2C-2F6 and P2C-1F11) were the
most capable of neutralizing pseudoviruses bearing SARS-CoV-2 spike
protein (IC<sub>50</sub> of 0.06 and 0.03 µg/mL, respectively). Finally, using
SPR the neutralizing antibodies were found to recognize both overlapping
and distinct epitopes of the RBD of SARS-CoV-2.</p>
<h3 data-number="9.173.3" id="limitations-169"><span class="header-section-number">9.173.3</span> Limitations</h3>
<ol type="1">
<li><p>Relatively low number of patients</p>
<ol type="a">
<li>No significant conclusion can be drawn about the possible
&gt; correlation between humoral response and disease severity</li>
</ol></li>
<li><p><em>In vitro</em> Cytopathic Effect Assay (CPE) for neutralization activity</p>
<ol type="a">
<li><p>Huh7 cells were used in neutralization assays with
&gt; pseudoviruses, and they may not entirely mimic what happens in
&gt; the upper respiratory tract</p></li>
<li><p>CPE assay is not quantitative</p></li>
</ol></li>
<li><p>Duplicated panel in Figure 4C reported (has been fixed in version 2)</p></li>
</ol>
<h3 data-number="9.173.4" id="significance-158"><span class="header-section-number">9.173.4</span> Significance</h3>
<p>This paper offers an explanation as to why
previously isolated antibodies against SARS-CoV do not effectively block
SARS-CoV-2. Also, it offers important insight into the development of
humoral responses at various time points during the first weeks of the
disease in small but clinically diverse group of patients. Furthermore,
it provides valuable information and well characterized antibody
candidates for the development of a recombinant antibody treatment for
SARS-CoV-2. Nevertheless, it also shows that plasmapheresis might have
variability in its effectiveness, depending on the donor’s antibody
repertoire at the time of donation.</p>
<h3 data-number="9.173.5" id="credit-167"><span class="header-section-number">9.173.5</span> Credit</h3>
<p><em>Review by Jovani Catalan-Dibene as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.174" id="characterisation-of-the-transcriptome-and-proteome-of-sars-cov-2-using-direct-rna-sequencing-and-tandem-mass-spectrometry-reveals-evidence-for-a-cell-passage-induced-in-frame-deletion-in-the-spike-glycoprotein-that-removes-the-furin-like-cleavage-site.-1"><span class="header-section-number">9.174</span> Characterisation of the transcriptome and proteome of SARS-CoV-2 using direct RNA sequencing and tandem mass spectrometry reveals evidence for a cell passage induced in-frame deletion in the spike glycoprotein that removes the furin-like cleavage site.</h2>
<p>Davidson et al. <span class="citation" data-cites="bCYYaFKM">[<a href="#ref-bCYYaFKM" role="doc-biblioref">633</a>]</span></p>
<h3 data-number="9.174.1" id="keywords-173"><span class="header-section-number">9.174.1</span> Keywords</h3>
<ul>
<li>Transcription</li>
<li>RNA-seq</li>
<li>proteomics</li>
<li>mass spec</li>
<li>furin cleavage site</li>
<li>mutation</li>
<li>pathogenicity</li>
</ul>
<h3 data-number="9.174.2" id="main-findings-167"><span class="header-section-number">9.174.2</span> Main Findings</h3>
<p>The authors performed long read RNA sequencing using an Oxford Nanopore
MinION as well as tandem mass spec (MS) on Vero cells (a cell line
derived from kidney cells of the African green monkey that is deficient
in interferon) infected with SARS-CoV-2.</p>
<p>The authors found that passage of the virus in Vero cells gave rise to a
spontaneous 9 amino acid deletion (679-NSPRRARSV-687 to I) in the spike
(S) protein. The deleted sequence overlaps a predicted furin cleavage
site at the S1 / S2 domain boundary that is present in SARS-CoV-2 but
not SARS-CoV or the closely related bat coronavirus RaTG13, which are
cleaved at S1 / S2 by other proteases <span class="citation" data-cites="CqQVkaqj">[<a href="#ref-CqQVkaqj" role="doc-biblioref">42</a>]</span>. Furin cleavage sites at
similar positions in other viruses have been linked to increased
pathogenicity and greater cell tropism <span class="citation" data-cites="16tHVjVzu">[<a href="#ref-16tHVjVzu" role="doc-biblioref">634</a>]</span>. Loss of this site in
SARS-CoV-2 has also already been shown to increase viral entry into Vero
but not BHK cells (which are also interferon deficient) <span class="citation" data-cites="qcVbT0w4">[<a href="#ref-qcVbT0w4" role="doc-biblioref">38</a>]</span>. The
authors therefore make an important contribution in demonstrating that
passage in Vero cells may lead to spontaneous loss of a key
pathogenicity-conferring element in SARS-CoV-2.</p>
<h3 data-number="9.174.3" id="limitations-170"><span class="header-section-number">9.174.3</span> Limitations</h3>
<p>As the authors note, a similar study posted earlier by Kim et al., which
also passaged SARS-CoV-2 in Vero cells, did not identify any loss in the
S protein furin cleavage site <span class="citation" data-cites="hRVfFbS">[<a href="#ref-hRVfFbS" role="doc-biblioref">635</a>]</span>. It therefore remains to be
determined how likely it is that this mutation spontaneously arises. A
quantitative investigation using multiple experimental replicas to
understand the spontaneous viral mutation rate at this site and
elsewhere would be informative. Also, the mechanistic basis for the
higher viral fitness conferred by loss of the furin cleavage site in
Vero cells – but, evidently, not in vivo in humans, as this site is
maintained in all currently sequenced circulating isolates - remains to
be understood.</p>
<p>Due to the high base-call error rate of MinION sequencing, the authors’
bioinformatic pipeline required aligning transcripts to a reference to
correct sequencing artifacts. This presumably made it difficult or
impossible to identify other kinds of mutations, such as single
nucleotide substitutions, which may occur even more frequently than the
deletions identified in this work. Pairing long read sequencing with
higher-accuracy short-read sequencing may be one approach to overcome
this issue.</p>
<h3 data-number="9.174.4" id="significance-159"><span class="header-section-number">9.174.4</span> Significance</h3>
<p>As the authors suggest, animal studies using live virus challenge may
need to periodically verify the genomic integrity of the virus, or
potentially risk unknowingly using a likely less-pathogenic variant of
the virus.</p>
<p>More broadly, the results emphasize the complexity and plasticity of the
SARS-CoV-2 viral transcriptome and proteome. For example, the authors
found multiple versions of transcripts encoding the nucleocapsid (N)
protein, each with different small internal deletions, some of which
were verified for translation by MS. A number of peptides arising from
translation of unexpected rearrangements of transcripts were also
detected. Additionally, the authors identified phosphorylation of a
number of viral proteins (N, M, ORF 3a, nsp3, nsp9, nsp12 and S). For
any cases where these have functional consequences, targeting the
kinases responsible could be an avenue for drug development.
Understanding the functional consequences of the mutations, transcript
variations, and post translational modifications identified in this
study will be important future work.</p>
<h3 data-number="9.174.5" id="credit-168"><span class="header-section-number">9.174.5</span> Credit</h3>
<p><em>This review was undertaken by Tim O’Donnell, Maria Kuksin as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.175" id="a-sars-cov-2-human-protein-protein-interaction-map-reveals-drug-targets-and-potential-drug--repurposing-1"><span class="header-section-number">9.175</span> A SARS-CoV-2-Human Protein-Protein Interaction Map Reveals Drug Targets and Potential Drug- Repurposing</h2>
<p>Gordon et al. <em>bioRxiv</em> <span class="citation" data-cites="phJM8g2Y">[<a href="#ref-phJM8g2Y" role="doc-biblioref">181</a>]</span></p>
<h3 data-number="9.175.1" id="keywords-174"><span class="header-section-number">9.175.1</span> Keywords</h3>
<ul>
<li>protein-protein interactions</li>
<li>mass spectrometry</li>
<li>drug targets</li>
</ul>
<h3 data-number="9.175.2" id="main-findings-168"><span class="header-section-number">9.175.2</span> Main Findings</h3>
<p>Gordon et al cloned, tagged and expressed 26 of the 29
SARS-CoV-2 proteins individually in HEK293T cells and used mass
spectrometry to identify protein-protein interactions. They identified
332 viral-host protein-protein interactions. Furthermore, they used
these interactions to identify 66 existing drugs known to target host
proteins or host pathways (eg SARS-CoV-2 N and Orf8 proteins interact
with proteins regulated by the mTOR pathway, so mTOR inhibitors
Silmitasertib and Rapamycin are possible drug candidates).</p>
<h3 data-number="9.175.3" id="limitations-171"><span class="header-section-number">9.175.3</span> Limitations</h3>
<p>The main limitation of the study stems from
the reductionist model: overexpression of plasmids encoding individual
viral proteins in HEK293T cells. This precludes any interactions between
the viral proteins, or the combined effects of multiple proteins on the
host, as they are expressed individually. Moreover, HEK293T cells come
from primary embryonic kidney and therefore might not reflect how
SARS-CoV-2 interacts with its primary target, the lung. However, the
authors found that the proteins found to interact with viral proteins in
their experiments are enriched in lung tissue compared to HEK293Ts.</p>
<h3 data-number="9.175.4" id="significance-160"><span class="header-section-number">9.175.4</span> Significance</h3>
<p>The authors provide a “SARS-CoV-2 interaction map,”
which may provide potential hypotheses as to how the virus interacts
with the host. Further, they identified existing drugs that could
disrupt these host-viral interactions and curb SARS-CoV-2 infection.
Although these interactions have not been validated, this paper acts as
a valuable resource.</p>
<h3 data-number="9.175.5" id="credit-169"><span class="header-section-number">9.175.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.176" id="first-clinical-study-using-hcv-protease-inhibitor-danoprevir-to-treat-naïve-and-experienced-covid-19-patients-1"><span class="header-section-number">9.176</span> First Clinical Study Using HCV Protease Inhibitor Danoprevir to Treat Naïve and Experienced COVID-19 Patients</h2>
<p>Chen et al. <em>medRxiv</em>.
<span class="citation" data-cites="RTIk8DTo">[<a href="#ref-RTIk8DTo" role="doc-biblioref">636</a>]</span></p>
<h3 data-number="9.176.1" id="keywords-175"><span class="header-section-number">9.176.1</span> Keywords</h3>
<ul>
<li>Clinical study</li>
<li>HCV protease inhibitor</li>
<li>Danoprevir</li>
<li>Ritonavir</li>
<li>Covid19 treatment</li>
</ul>
<h3 data-number="9.176.2" id="main-findings-169"><span class="header-section-number">9.176.2</span> Main Findings</h3>
<p>The authors treated 11 Covid-19 patients with Danoprevir, a
commercialized HCV protease inhibitor <span class="citation" data-cites="YoCPtq6P">[<a href="#ref-YoCPtq6P" role="doc-biblioref">637</a>]</span><sup>(p4)</sup>, boosted by ritonavir <span class="citation" data-cites="10GnhBctk">[<a href="#ref-10GnhBctk" role="doc-biblioref">638</a>]</span>, a
CYP3A4 inhibitor (which enhances the plasma concentration and
bioavailabilty of Danoprevir). Two patients had never received
anti-viral therapy before (=naïve), whereas nine patients were on
Lopinavir/Ritonavir treatment before switching to Danoprevir/Ritonavir
(=experienced). The age ranged from 18 to 66yo.</p>
<p>Naïve patients that received Danoprevir/Ritonavir treatment had a
decreased hospitalization time. Patients treated with
Lopinavir/Ritonavir did not have a negative PCR test, while after
switching to Danoprevir/Ritonavir treatment, the first negative PCR test
occurred at a median of two days.</p>
<h3 data-number="9.176.3" id="limitations-172"><span class="header-section-number">9.176.3</span> Limitations</h3>
<p>The results of the study are very hard to interpret as there is no
control group not receiving Danoprevir/Ritonavir treatment. This was
especially true in naïve patients who seemed to have more mild symptoms
before the start of the study and were younger (18 and 44yo) compared to
the experienced patients (18 to 66yo). The possibility that the patients
would have recovered without Danoprevir/Ritonavir treatment cannot be
excluded.</p>
<h3 data-number="9.176.4" id="significance-161"><span class="header-section-number">9.176.4</span> Significance</h3>
<p>The authors of the study treated patients with Danoprevir, with the
rational to that this is an approved and well tolerated drug for HCV
patients <span class="citation" data-cites="10GnhBctk">[<a href="#ref-10GnhBctk" role="doc-biblioref">638</a>]</span>, and that it could also target the protease from SARS-CoV-2
(essential for viral replication and transcription). Indeed, homology
modelling data indicated that HCV protease inhibitors have the highest
binding affinity to Sars-Cov2 protease among other approved drugs <span class="citation" data-cites="7zBwK1wn">[<a href="#ref-7zBwK1wn" role="doc-biblioref">639</a>]</span>.</p>
<p>While this study shows that the combination of Danoprevir and Ritonavir
might be beneficial for Covid-19 patients, additional clinical trials
with more patients and with better methodology (randomization and
control group) are needed to make further conclusions.</p>
<h3 data-number="9.176.5" id="credit-170"><span class="header-section-number">9.176.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.177" id="efficacy-of-hydroxychloroquine-in-patients-with-covid-19-results-of-a-randomized-clinical-trial-1"><span class="header-section-number">9.177</span> Efficacy of hydroxychloroquine in patients with COVID-19: results of a randomized clinical trial</h2>
<p><span class="citation" data-cites="wzHb7Mj5">[<a href="#ref-wzHb7Mj5" role="doc-biblioref">289</a>]</span></p>
<h3 data-number="9.177.1" id="keywords-176"><span class="header-section-number">9.177.1</span> Keywords</h3>
<ul>
<li>hydroxychloroquine</li>
</ul>
<h3 data-number="9.177.2" id="study-description-3"><span class="header-section-number">9.177.2</span> Study Description</h3>
<p>This is a randomized clinical trial of hydroxychloroquine (HCQ) efficacy
in the treatment of COVID-19. From February 4 – February 28, 2020 142
COVID-19 positive patients were admitted to Renmin Hospital of Wuhan
University. 62 patients met inclusion criteria and were enrolled in a
double blind, randomized control trial, with 31 patients in each arm.</p>
<p>Inclusion criteria:</p>
<ol type="1">
<li><p>Age ≥ 18 years</p></li>
<li><p>Positive diagnosis COVID-19 by detection of SARS-CoV-2 by RT-PCR</p></li>
<li><p>Diagnosis of pneumonia on chest CT</p></li>
<li><p>Mild respiratory illness, defined by SaO<sub>2</sub>/SPO<sub>2</sub> ratio &gt; 93% or
PaO<sub>2</sub>/FIO<sub>2</sub> ratio &gt; 300 mmHg in hospital room conditions (Note:
relevant clinical references described below.)</p>
<ol type="a">
<li><p>Hypoxia is defined as an SpO2 of 85-94%; severe hypoxia &lt;
85%.</p></li>
<li><p>The PaO<sub>2</sub>/FIO<sub>2</sub> (ratio of arterial oxygen tension to fraction
of inspired oxygen) is used to classify the severity of acute
respiratory distress syndrome (ARDS). Mild ARDS has a
PaO<sub>2</sub>/FIO<sub>2</sub> of 200-300 mmHg, moderate is 100-200, and severe
&lt; 100.</p></li>
</ol></li>
<li><p>Willing to receive a random assignment to any designated treatment
group; not participating in another study at the same time</p></li>
</ol>
<p>Exclusion criteria:</p>
<ol type="1">
<li><p>Severe or critical respiratory illness (not explicitly defined,
presumed to be respiratory function worse than outlined in inclusion
criteria); or participation in trial does not meet patient’s maximum
benefit or safe follow up criteria</p></li>
<li><p>Retinopathy or other retinal diseases</p></li>
<li><p>Conduction block or other arrhythmias</p></li>
<li><p>Severe liver disease, defined by Child-Pugh score ≥ C or AST &gt;
twice the upper limit</p></li>
<li><p>Pregnant or breastfeeding</p></li>
<li><p>Severe renal failure, defined by eGFR ≤ 30 mL/min/1.73m<sup>2</sup>, or on
dialysis</p></li>
<li><p>Potential transfer to another hospital within 72h of enrollment</p></li>
<li><p>Received any trial treatment for COVID-19 within 30 days before the
current study</p></li>
</ol>
<p>All patients received the standard of care: oxygen therapy, antiviral
agents, antibacterial agents, and immunoglobulin, with or without
corticosteroids. Patients in the HCQ treatment group received additional
oral HCQ 400 mg/day, given as 200 mg 2x/day. HCQ was administered from
days 1-5 of the trial. The primary endpoint was 5 days post enrollment
or a severe adverse reaction to HCQ. The primary outcome evaluated was
time to clinical recovery (TTCR), defined as return to normal body
temperature and cough cessation for &gt; 72h. Chest CT were imaged on
days 0 and 6 of the trial for both groups; body temperature and patient
reports of cough were collected 3x/day from day 0 – 6. The mean age and
sex distribution between the HCQ and control arms were comparable.</p>
<h3 data-number="9.177.3" id="main-findings-170"><span class="header-section-number">9.177.3</span> Main Findings</h3>
<p>There were 2 patients showing mild secondary effects of HCQ treatment.
More importantly, while 4 patients in the control group progressed to
severe disease, none progressed in the treatment group.</p>
<p>TTCR was significantly decreased in the HCQ treatment arm; recovery from
fever was shortened by one day (3.2 days control vs. 2.2 days HCQ, p =
0.0008); time to cessation of cough was similarly reduced (3.1 days
control vs. 2.0 days HCQ, p = 0.0016).</p>
<p>Overall, it appears that HCQ treatment of patients with mild COVID-19
has a modest effect on clinical recovery (symptom relief on average 1
day earlier) but may be more potent in reducing the progression from
mild to severe disease.</p>
<h3 data-number="9.177.4" id="limitations-173"><span class="header-section-number">9.177.4</span> Limitations</h3>
<p>This study is limited in its inclusion of only patients with mild
disease, and exclusion of those on any treatment other than the standard
of care. It would also have been important to include the laboratory
values of positive RT-PCR detection of SARS-CoV-2 to compare the
baseline and evolution of the patients’ viral load.</p>
<h3 data-number="9.177.5" id="limitations-174"><span class="header-section-number">9.177.5</span> Limitations</h3>
<p>Despite its limitations, the study design has good rigor as a double
blind RCT and consistent symptom checks on each day of the trail. Now
that the FDA has approved HCQ for treatment of COVID-19 in the USA, this
study supports the efficacy of HCQ use early in treatment of patients
showing mild symptoms, to improve time to clinical recovery, and
possibly reduce disease progression. However, most of the current
applications of HCQ have been in patients with severe disease and for
compassionate use, which are out of the scope of the findings presented
in this trial. Several additional clinical trials to examine
<a href="https://twitter.com/hashtag/hydroxychloroquine?src=hashtag_click">hydroxychloroquine</a>
are now undergoing; their results will be critical to further validate
these findings.</p>
<h3 data-number="9.177.6" id="credit-171"><span class="header-section-number">9.177.6</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<p><strong>10.1101 2020.03.22.20040832 </strong></p>
<p><strong>Title:</strong> Factors associated with prolonged viral shedding and impact
of Lopinavir/Ritonavir treatment in patients with SARS-CoV-2
infection<strong>
</strong></p>
<p><strong>Keywords</strong>: retrospective study – lopinavir/ritonavir – viral shedding</p>
<p><strong>Main findings:</strong></p>
<p>The aim of this retrospective study is to assess the potential impact of
earlier administration of <strong>lopinavir/ritonavir (LPV/r)</strong> treatment on
the duration of viral shedding in hospitalized non-critically ill
patients with SARS-CoV-2.</p>
<p>The analysis shows that administration of LPV/r treatment reduced the
duration of viral shedding (22 vs 28.5 days). Additionally, if the
treatment was started within 10 days of symptoms onset, an even shorter
duration of virus shedding was observed compared to patients that
started treatment after 10 days of symptoms s onset (19 vs 27.5 days).
Indeed, patients that started LPV/r treatment late did not have a
significant median duration of viral shedding compared to the control
group (27.5 vs 28.5 days). Old age and lack of LPV/r administration
independently associated with prolonged viral shedding in this cohort of
patients.</p>
<p><strong>Limitations:</strong></p>
<p>In this non-randomized study, the group not receiving LPV/r had a lower
proportion of severe and critical cases (14.3% vs 32.1%) and a lower
proportion of patients also receiving corticosteroid therapy and
antibiotics, which can make the results difficult to interpret.</p>
<p>The endpoint of the study is the end of viral shedding (when the swab
test comes back negative), not a clinical amelioration. The correlation
between viral shedding and clinical state needs to be further assessed
to confirm that early administration of LPV/r could be used in treating
COVID-19 patients.</p>
<p><strong>Relevance</strong>:</p>
<p>Lopinavir/ritonavir combination has been previously shown to be
efficient in treating SARS <sup>1,2</sup>. While this article raises an important
point of early administration of LPV/r being necessary to have an
effect, the study is retrospective, contains several sources of bias and
does not assess symptom improvement of patients. A previously published
randomized controlled trial including 200 severe COVID-19 patients did
not see a positive effect of LPV/r administration<sup>3</sup>, and treatment was
discontinued in 13.8% of the patients due to adverse events. Similarly,
another small randomized trial did not note a significant effect of
LPV/r treatment<sup>4</sup> in mild/moderate patients. A consequent European
clinical trial, “Discovery”, including among others LPV/r treatment is
under way and may provide conclusive evidence on the effect and timing
of LPV/r treatment on treating COVID-19.</p>
<p>1. Treatment of severe acute respiratory syndrome with
lopinavir/ritonavir: a multicentre retrospective matched cohort study. -
PubMed - NCBI.
https://www-ncbi-nlm-nih-gov.docelec.univ-lyon1.fr/pubmed/14660806.
Accessed March 30, 2020.</p>
<p>2. Role of lopinavir/ritonavir in the treatment of SARS: initial
virological and clinical findings. - PubMed - NCBI.
https://www-ncbi-nlm-nih-gov.docelec.univ-lyon1.fr/pubmed/14985565.
Accessed March 30, 2020.</p>
<p>3. Cao B, Wang Y, Wen D, et al. A Trial of Lopinavir–Ritonavir in Adults
Hospitalized with Severe Covid-19. <em>New England Journal of Medicine</em>.
March 2020. doi:10.1056/NEJMoa2001282</p>
<p>4. Li Y, Xie Z, Lin W, et al. <em>An Exploratory Randomized, Controlled
Study on the Efficacy and Safety of Lopinavir/Ritonavir or Arbidol
Treating Adult Patients Hospitalized with Mild/Moderate COVID-19
(ELACOI)</em>. Infectious Diseases (except HIV/AIDS); 2020.
doi:10.1101/2020.03.19.20038984</p>
<p><strong>Emma Risson Edited by K Alexandropoulos</strong></p>
<p><strong>Clinical features and the maternal and neonatal outcomes of pregnant
women with coronavirus disease 2019</strong></p>
<p><strong><em>Keywords</em></strong></p>
<p>Pregnancy, SARS-CoV2, neonatal and maternal Covid-19 outcome</p>
<p><strong><em>Key findings</em></strong></p>
<p>33 pregnant woman and 28 newborns were included in this retrospective
multicenter study, conducted at 5 hospitals in Wuhan and Hubei province,
China, between January 1 and February 20, 2020. All women were diagnosed
with Covid-19 by qPCR or viral gene sequencing based on the Chinese New
Corona Pneumonia Prevention and Control Program, 6<sup>th</sup> edition, and were
further subdivided into four groups based on clinical severity: (1)
mild, presence of mild clinical symptoms without radiological
abnormalities; (2) moderate, fever or upper respiratory symptoms as well
as radiological signs of pneumonia; (3) severe, at least one of the
following: shortness of breath/respiratory rate &gt;30/min, resting
oxygen saturation SaO<sub>2</sub>&lt;93%, Horowitz index paO2/FiO<sub>2</sub> &lt; 300
mmHg (indicating moderate pulmonary damage); and (4) severe-acute, acute
respiratory distress with need for mechanical ventilation; systemic
shock; multi-organ failure and transfer to ICU. Maternal admission to
ICU, mechanical ventilation or death were defined as primary outcomes;
secondary study outcomes comprised clinical Covid-19 severity in both
mothers and newborns, including development of ARDS, neonatal ICU
admission as well as mortality.</p>
<p><em>Maternal characteristics and outcome:</em> 3 out of 33 women were in their
second trimester of pregnancy (17, 20 and 26 weeks), and 15/33 (45.5%)
had a previous history of underlying chronic health disorders including
cardiovascular, cerebrovascular or nervous system disease. Common
Covid-19 symptoms at presentation were fever (63.6%), dry cough (39.4%),
fatigue (21.2%), and shortness of breath (21.2%). Less common symptoms
included diarrhea, post-partum fever, muscle ache, sore throat and chest
pain. 4 (12.1%) pregnant women had no apparent symptoms. The majority of
cases were classified as mild (39.4%) or moderate (57.6%); however, one
woman developed severe Covid-19. 40.6% of women were diagnosed with
bilateral pneumonia, 43.8% presented with unilateral pneumonia, and
15.6% showed radiological ground-glass opacity. 87.9% of women required
oxygen administration, and one (3%) woman had to be put on non-invasive
mechanical ventilation (primary outcome). 81.5% of women had a C-section
and only 5% had vaginal deliveries. Obstetrical complications were seen
in 22.2% of women, including three cases of preterm rupture of
membranes, two cases of hypertensive disorders of pregnancy, and one
case of spontaneous preterm labor. Five pregnancies were ongoing at the
end of the observation point of this study; one woman decided to have
her pregnancy terminated. <em>Neonatal outcome:</em> Out of 28 newborns
included in this study, 35.7% were born preterm at &lt;37 weeks of
gestation with Apgar scores ranging from 8-10/10 at 1 min and from
9-10/10 after 5 min, indicating normal heart and respiratory rates.
17.9% of newborns were of low birth weight (not specified) and 14.3%
showed signs of fetal distress (also not specified). According to the
authors of this study, none of the newborns presented with clinical
Covid-19 symptoms. However, one newborn, delivered at 34 weeks of
gestation, was diagnosed with (apparently Covid-19 unrelated?) ARDS and
transferred to NICU (secondary outcome). Of 26 newborns tested for
SARS-CoV2, only one was found positive and showed radiological signs of
pneumonia, but no clinical symptoms of Covid-19. It remains unclear
whether this was the same case as the newborn diagnosed with ARDS. The
affected newborn did not require any treatment and was discharged at 16
days post birth. In summary, the primary outcome “mechanical
ventilation” in pregnant women was rare (3%), no other primary outcomes
were reached. Most Covid-19 cases in pregnant women were described as
mild to moderate. Only one of 28 (3.57%) newborns was diagnosed with
ARDS (secondary outcome).</p>
<p><strong><em>Potential limitations</em></strong></p>
<p>Major limitations of this study are its small size and the rudimentary
and at times inadequate description of patient specifics. For example,
underlying health conditions that might be affecting Covid-19 outcome in
pregnant women should have been clearly specified (other than being of
be listed (not just &lt;37 weeks). Given that maternal infection status
seemed mostly unknown at the time of birth and, more importantly, that
the majority of cases in this study were clinically asymptomatic or mild
to moderate, it remains unclear whether the C-sections performed were a
medical necessity or elective procedures. This is of importance and
should have been discussed. With regard to neonatal outcome, it is also
not apparent whether the newborn found to be infected with SARS-CoV2 and
the case diagnosed with ARDS were the same individual. If this was the
case, it would be incorrect to refer to all newborns as asymptomatic.
Additionally, it seems somewhat unlikely that a newborn with a
near-perfect Apgar score would present with ARDS immediately after
birth. Likewise, any individual diagnosed with ARDS would certainly be
expected to receive supportive treatment including (invasive) mechanical
ventilation. While it is highly relevant that overall clinical outcome
in pregnant women diagnosed with Covid-19 seems better than in SARS or
MERS (as discussed by the authors), it nevertheless needs to be stressed
that more than 37% of newborns in this study were delivered preterm and
that the obstetric complication rate of 22% seems higher than
non-Covid-19 average.</p>
<p><strong><em>Overall relevance for the field</em></strong></p>
<p>Observations in this study confirm some of the findings published in a
case series by Yu N et al. (Lancet Infect Dis 2020;
https://doi.org/10.1016/ S1473-3099(20)30176-6). However, due to the
relatively small study size of 33 pregnant women and 28 newborns, this
study lacks statistical power and final conclusions on Covid-19 outcomes
in pregnant women and newborns cannot be drawn. Yet, the data collected
here are important and should be incorporated into larger data sets for
more insight. Understanding the clinical course and effects of Covid19
in both pregnant women and newborns is essential, and while there are
some recent publications on vertical SARS-CoV2 transmission between
mothers and newborns (Dong L et al, JAMA March 26, 2020,
doi:10.1001/jama.2020.4621; Zeng H et al, JAMA March 26, 2020,
doi:10.1001/jama.2020.4861) as well as on neonatal infection at birth
(Zeng L et al, JAMA March 26, 2020,
doi:10.1001/jamapediatrics.2020.0878), our knowledge of how these
patient subsets are affected is still very limited.</p>
<ol type="A">
<li><p>Elevated serum IgM levels indicate poor outcome in patients with
coronavirus disease 2019 pneumonia: A retrospective case-control
study</p></li>
<li><p><strong>Keywords:</strong> SAR-CoV-2, pneumonia, immunoglobulin, IgM, IgG.</p></li>
<li><p><strong>Key findings:</strong> The authors wanted to better understand the
dynamics of production SARS-CoV-2-specific IgM and IgG in COVID-19
pneumonia and the correlation of virus-specific antibody levels to
disease outcome in a case-control study paired by age. The
retrospective study included 116 hospitalized patients with COVID-19
pneumonia and with SAR-CoV-2 specific serum IgM and IgG detected.
From the study cohort, 15 cases died. SARS-CoV-2 specific IgG levels
increased over 8 weeks after onset of COVID-19 pneumonia, while
SARS-CoV-2 specific IgM levels peaked at 4 weeks. SARS-CoV-2
specific IgM levels were higher in the deceased group, and
correlated positively with the IgG levels and increased leucocyte
count in this group, a indication of severe inflammation. IgM levels
correlated negatively with clinical outcome and with albumin levels.
The authors suggest that IgM levels could be assessed to predict
clinical outcome.</p></li>
<li><p><strong>Potential limitations:</strong> There are limitations that should be
taken into account. First, the sample: small size, patients from a
single-center and already critically ill when they were admitted.
Second, the authors compared serum IgM levels in deceased patients
and mild-moderate patients and found that the levels were higher in
deceased group, however even if the difference is statistically
significant the number of patients in the two groups was very
different. Moreover, receiving operating characteristics (ROC)
curves were used to evaluate IgM and IgG as potential predictors for
clinical outcome. Given the low number of cases, especially in the
deceased group, it remains to be confirmed if IgM levels could be
predictive of worst outcome in patients with COVID-19 pneumonia. The
study did not explore the role of SARS-CoV-2-specific IgM and IgG in
COVID-19 pneumonia.</p></li>
<li><p><strong>Overall relevance for the field:</strong> Some results of this study have
been supported by subsequent studies that show that older age and
patients who have comorbidities are more likely to develop a more
severe clinical course with COVID-19, and severe SARS-CoV-2 may
trigger an exaggerated immune response. The study seems to
demonstrate that the increase of SARS-CoV-2-specific IgM could
indicate poor outcome in patients with COVID-19 pneumonia, however
given the very small sample size, the results are not yet
conclusive.</p></li>
</ol>
<p>Review by Meriem Belabed as part of a project by students, postdocs and
faculty at the Immunology Institute of the Icahn school of medicine,
Mount Sinai.</p>
<p><strong>Title:</strong> scRNA-seq reveals ACE2 and TMPRSS2 expression in TROP2+ Liver
Progenitor Cells: Implications in COVID-19 associated liver dysfunction</p>
<p><strong>Keywords:</strong> SARS-CoV-2, COVID-19, ACE2, TMPRSS2, TROP2, Liver,
scRNA-seq</p>
<p><strong>Main findings:</strong> The authors performed scRNA-seq on human liver tissue
and identified an epithelial progenitor cell type that co-express ACE2
and TMPRSS2. ACE2 receptor was highly expressed in cholangiocyte-biased
liver progenitors (EPCAM<sup>+</sup>) which also had a high expression for TROP2
gene. TMPRSS2 was expressed by both TROP2<sup>high</sup> and TROP2<sup>int</sup> liver
cells. The authors reported that these cholangiocyte-biased progenitor
cells had a high expression of TROP2 that correlated with high levels of
ACE2 and TMPRSS2 in liver. This suggests that SARS-CoV-2 can infect
TROP2<sup>high</sup> cells via ACE2 and TMPRSS2, contributing to liver
dysfunction by compromising the ability of the human liver to regenerate
cholangiocytes. The authors also note that ACE2 and TMPRSS2 positive
cells are absent in human fetal liver.</p>
<p><strong>Critical Analyses</strong>:</p>
<p>1. Normal tissue from hepatocellular carcinoma patients are
tumor-adjacent tissues from these patients and might not be truly
‘normal’.</p>
<p>2. Tissues from liver cancer patients may have an altered phenotype from
healthy controls.</p>
<p>3. ACE2 expression may be variable and inducible in other cell types in
the context of inflammation.</p>
<p><strong>Relevance: </strong></p>
<p>The apparent only cell population in liver that expresses ACE2 and
TMPRSS2 are EPCAM+ liver progenitor cells, making them suitable hosts
for viral infection and potentially resulting in liver dysfunction. This
cell population of liver progenitors with a cholangiocyte fate bias may
favor SARS-CoV-2 entry into liver, affecting cholangiocyte precursors,
thus leading to liver comorbidities in COVID-19 patients.</p>
<p><strong>Structure-based modeling of SARS-CoV-2 peptide/HLA-A02 antigens</strong></p>
<p><a href="https://doi.org/10.1101/2020.03.23.004176" class="uri">https://doi.org/10.1101/2020.03.23.004176</a></p>
<p><em>Immunology keywords</em>:</p>
<p>CoVID-19, 2019-nCoV, SARS-CoV-2, comparative, homology, peptide,
modeling, simulation, HLA-A, antigen</p>
<p><em>Summary of Findings</em>:</p>
<ul>
<li><p>The authors utilize homology modeling to identify peptides from the
SARS-CoV-2 proteome that potentially bind HLA-A*02:01.</p></li>
<li><p>They utilize high-resolution X-ray structures of peptide/MHC
complexes on Protein Data Bank, substitute homologous peptides with
SARS-CoV-2 peptides, and calculate MHC/SARS-CoV-2 peptide Rosetta
binding energy.</p></li>
<li><p>They select MHC/SARS-CoV-2 complex models with highest binding
energy for further study and publish models in an online database
(https://rosettamhc.chemistry.ucsc.edu).</p></li>
</ul>
<p><em>Limitations</em>:</p>
<ul>
<li><p>The authors only utilize computational methods and predicted
SARS-CoV-2 peptides must be validated experimentally for
immunogenicity and clinical response.</p></li>
<li><p>Due to computational burden and limited availability of high
resolution X-ray structures on PDB, authors only simulate 9-mer and
10-mer peptide binding to HLA-A*02:01.</p></li>
<li><p>Since the authors compare select existing X-ray structures as a
starting point, backbone conformations that deviate significantly
between test and template peptides are not captured. Furthermore,
Rosetta modeling protocols do not capture all possible structures
and binding energy scoring does not fully recapitulate fundamental
forces.<sup>1,2</sup></p></li>
</ul>
<p><em>Importance/Relevance:</em></p>
<ul>
<li><p>The authors identify and publish high-scoring SARS-CoV-2 peptides
that may direct a targeted, experimental validation approach toward
a COVID-19 vaccine.</p></li>
<li><p>The authors utilize Rosetta simulation to further filter results
from NetMHCpan 4.0, supporting machine learning prediction with
structural analysis.</p></li>
<li><p>The authors develop RosettaMHC, a computationally efficient method
of leveraging existing X-ray structures for identification of
immunogenic peptides.</p></li>
</ul>
<p><em>References:</em></p>
<ol type="1">
<li><p>Bender, B. J., Cisneros, A., 3rd, Duran, A. M., Finn, J. A., Fu, D.,
Lokits, A. D., . . . Moretti, R. (2016). Protocols for Molecular
Modeling with Rosetta3 and RosettaScripts. Biochemistry, 55(34),
4748-4763. doi:10.1021/acs.biochem.6b00444</p></li>
<li><p>Alford, R. F., Leaver-Fay, A., Jeliazkov, J. R., O’Meara, M. J.,
DiMaio, F. P., Park, H., . . . Gray, J. J. (2017). The Rosetta
All-Atom Energy Function for Macromolecular Modeling and Design. J
Chem Theory Comput, 13(6), 3031-3048. doi:10.1021/acs.jctc.7b00125</p></li>
</ol>
<p><em>Review by Jonathan Chung as part of a project by students, postdocs and
faculty at the Immunology Institute of the Icahn school of medicine,
Mount Sinai.</em></p>
<h2 data-number="9.178" id="serology-characteristics-of-sars-cov-2-infection-since-the-exposure-and-post-symptoms-onset-1"><span class="header-section-number">9.178</span> Serology characteristics of SARS-CoV-2 infection since the exposure and post symptoms onset</h2>
<p>Lou et al. <em>medRxiv</em>. <span class="citation" data-cites="lnuNSHEW">[<a href="#ref-lnuNSHEW" role="doc-biblioref">640</a>]</span></p>
<h3 data-number="9.178.1" id="keywords-177"><span class="header-section-number">9.178.1</span> Keywords</h3>
<ul>
<li>Seroconversion rate</li>
<li>Total Antibody</li>
<li>Ab</li>
<li>IgG and IgM</li>
<li>antibody</li>
</ul>
<h3 data-number="9.178.2" id="main-findings-171"><span class="header-section-number">9.178.2</span> Main Findings</h3>
<p>Currently, the diagnosis of SARS-CoV-2 infection
entirely depends on the detection of viral RNA using polymerase chain
reaction (PCR) assays. False negative results are common, particularly
when the samples are collected from upper respiratory. Serological
detection may be useful as an additional testing strategy. In this study
the authors reported that a typical acute antibody response was induced
during the SARS-CoV-2 infection, which was discuss earlier<sup>1</sup>. The
seroconversion rate for Ab, IgM and IgG in COVID-19 patients was 98.8%
(79/80), 93.8% (75/80) and 93.8% (75/80), respectively. The first
detectible serology marker was total antibody followed by IgM and IgG,
with a median seroconversion time of 15, 18 and 20 days-post exposure
(d.p.e) or 9, 10- and 12-days post-onset (d.p.o). Seroconversion was
first detected at day 7d.p.e in 98.9% of the patients. Interestingly
they found that viral load declined as antibody levels increased. This
was in contrast to a previous study <span class="citation" data-cites="5HXuCa5t">[<a href="#ref-5HXuCa5t" role="doc-biblioref">581</a>]</span>, showing that increased antibody
titers did not always correlate with RNA clearance (low number of
patient sample).</p>
<h3 data-number="9.178.3" id="limitations-175"><span class="header-section-number">9.178.3</span> Limitations</h3>
<p>Current knowledge of the antibody response to SAR-CoV-2
infection and its mechanism is not yet well elucidated. Similar to the
RNA test, the absence of antibody titers in the early stage of illness
could not exclude the possibility of infection. A diagnostic test, which
is the aim of the authors, would not be useful at the early time points
of infection but it could be used to screen asymptomatic patients or
patients with mild disease at later times after exposure.</p>
<h3 data-number="9.178.4" id="significance-162"><span class="header-section-number">9.178.4</span> Significance</h3>
<p>Understanding the antibody responses against SARS-CoV2 is
useful in the development of a serological test for the diagnosis of
COVID-19. This manuscript discussed acute antibody responses which can
be deducted in plasma for diagnostic as well as prognostic purposes.
Thus, patient-derived plasma with known antibody titers may be used
therapeutically for treating COVID-19 patients with severe illness.
<span id="OLE_LINK3" class="anchor"><span id="OLE_LINK4" class="anchor"></span></span></p>
<h3 data-number="9.178.5" id="credit-172"><span class="header-section-number">9.178.5</span> Credit</h3>
<p><em>This review was undertaken and edited by Konstantina A as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.179" id="sars-cov-2-launches-a-unique-transcriptional-signature-from-in-vitro-ex-vivo-and-in-vivo-systems-1"><span class="header-section-number">9.179</span> SARS-CoV-2 launches a unique transcriptional signature from in vitro, ex vivo, and in vivo systems</h2>
<p>Blanco-Melo et al. <em>bioRxiv</em>. <span class="citation" data-cites="aNSdARwe">[<a href="#ref-aNSdARwe" role="doc-biblioref">641</a>]</span></p>
<h3 data-number="9.179.1" id="keywords-178"><span class="header-section-number">9.179.1</span> Keywords</h3>
<ul>
<li>host cellular response</li>
<li>host-pathogen interaction</li>
<li>type I interferon</li>
<li>type III interferon</li>
<li>inflammation</li>
<li>RNA-seq</li>
<li>comparative analysis</li>
</ul>
<h3 data-number="9.179.2" id="main-findings-172"><span class="header-section-number">9.179.2</span> Main Findings</h3>
<p>Given the high mortality rate of SARS-CoV-2 relative to
other respiratory viruses such seasonal IAV and RSV, there may be
underlying host-pathogen interactions specific to SARS-CoV-2 that
predispose to a worse clinical outcome. Using <em>in vivo</em>, <em>ex vivo</em>, and
<em>in vitro</em> systems, the authors profiled host cell transcriptional
responses to SARS-CoV-2 and to other common respiratory viruses
(seasonal IAV and RSV). SARS-CoV-2 infection <em>in vitro</em> led to an
induction of type I interferon response signaling and the upregulation
of cytokine/chemokines transcripts. In comparison with IAV and RSV
infection, SARS-CoV-2 <em>in vitro</em> appears to uniquely induce less type I
and type III interferon expression and higher levels of two cytokines
previously implicated in respiratory inflammation. Lastly, <em>in vivo</em>
data from ferrets showed a reduced induction of cytokines and chemokines
by SARS-CoV-2 infection relative to IAV infection.</p>
<h3 data-number="9.179.3" id="limitations-176"><span class="header-section-number">9.179.3</span> Limitations</h3>
<p>While these results are promising, there are several
key weaknesses of this paper. 1) As the authors point out, there is an
undetectable level of SARS-CoV-2 putative receptor (ACE2) and protease
(TMPRSS2) expression in the lung epithelial cell line used for the <em>in
vitro</em> studies. This raises the important question of whether viral
replication actually occurs in any of the models used, which may explain
the lack of interferon production observed <em>in vitro</em> in SARS-CoV-2
treated cells. Further studies characterizing viral titers across
timepoints are needed. 2) Furthermore, these studies only characterize
the host response at a single dose and timepoint per virus, and it is
unclear why these doses/timepoints were chosen. This leaves open the
possibility that the observed differences between viruses could be due
to differences in dose, timing, host response, or a combination of all
of these. 3) It is unclear whether ferrets are productively infected,
which cell types are infected, and the extent/timing of the clinical
course of infection. Moreover, the <em>in vitro</em> and <em>in vivo</em> data do not
strongly correlate and the reasons for this are unclear.</p>
<h3 data-number="9.179.4" id="significance-163"><span class="header-section-number">9.179.4</span> Significance</h3>
<p>This paper describes potentially unique
transcriptional signatures of host cells exposed to SARS-CoV-2. If
validated, these findings may help explain clinical outcomes and could
be targeted in future therapeutic interventions.</p>
<h3 data-number="9.179.5" id="potential-conflicts-of-interest-disclosure-1"><span class="header-section-number">9.179.5</span> Potential Conflicts of Interest Disclosure</h3>
<p>The reviewers are also researchers at the Icahn School
of Medicine at Mount Sinai.</p>
<h3 data-number="9.179.6" id="credit-173"><span class="header-section-number">9.179.6</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<p><strong>Analysis of Serologic Cross-Reactivity Between Common Human
Coronaviruses and SARS-CoV-2 Using Coronavirus Antigen Microarray</strong></p>
<p><strong><em>Keywords</em></strong>: serum antibodies, cross-reactivity, antigen microarray</p>
<p><strong><em>Key Findings</em></strong></p>
<p>The authors of this study describe a common respiratory viral antigens
microarray that can be used to assess the cross-reactivity of antibodies
against the novel SARS-CoV-2 antigens and common human coronaviruses.
Sixty-seven (67) antigens from across subtypes of coronaviruses,
influenza viruses, adenoiruses, respiratory syncytial viruses and other
viruses, were printed onto microarrays, probed with human sera, and
analyzed. Five (5) serum samples collected before the SARS-CoV-2
outbreak were tested as part of a study that monitored acute respiratory
infection cases. Four out of five (4/5) sera samples showed high IgG
seroreactivity across the 4 common human coronaviruses. All sera showed
low IgG seroreactivity to SARS-CoV-2.</p>
<p>The S1 domain of the spike protein is suggested to be subtype specific
as it demonstrated very low cross-reactivity among the novel
coronaviruses (SARS-CoV-2, SARS-CoV and MERS-CoV) and between the novel
coronal viruses and common human coronaviruses. In contrast, the S2
domain of the spike protein and the nucleocapsid (NP) protein showed low
levels of cross-reactivity between the coronavirus subtypes, suggesting
the presence of more conserved antigenic domains. Therefore, the authors
conclude that the S1 domain is an ideal candidate for virus-specific
serologic assays.</p>
<p><strong><em>Importance</em></strong></p>
<p>This study provides insights into the potential cross-reactivity of
common human coronavirus antibodies for SARS-CoV-2 antigens.
Understanding cross-reactivity is useful because it can help in the
development of sensitive and specific serological assays that can test
for COVID-19. Additionally, studies such as this can help inform vaccine
development and indicate which antigens would be optimal to target.</p>
<p><strong><em>Limitations</em></strong></p>
<p>This study is limited by a small sample size (n=5) of serum samples that
all came from students who were part of the same college resident
community at The University of Maryland. A greatest limitation is the
lack of serum samples from COVID-19 patients. Additionally, the MFI of
2019-nCov from these naïve samples (in particular sera 4) is not zero.
This indicates that while relatively low, there is some cross-reactivity
of common human coronavirus antibodies for SARS-CoV-2 antigens,
including the S1 domain. A larger sample size and samples from SARS-CoV2
infected patients may be needed to confirm the sensitivity and
specificity of the assay.</p>
<p><strong>Lactate dehydrogenase - A Risk Factor of Severe COVID-19 Patients: A
Retrospective and Observational Study</strong></p>
<p>Han et al., medRxiv [10.1101:2020.03.24.20040162]</p>
<p>Keywords:</p>
<ul>
<li><p>lactate dehydrogenase</p></li>
<li><p>COVID-19</p></li>
<li><p>risk factor</p></li>
</ul>
<p><em>Main findings</em></p>
<p>This report is a retrospective analysis of clinical data collected from
47 patients with confirmed COVID-19 disease (median age 64.91 years; 26
males; 24 severe cases). Upon admission, patients were assessed with an
APACHE II score (mortality risk) and SOFA score (organ system
dysfunction). Among other clinical parameters, including lymphocyte
count, CRP, and enzyme levels, serum lactate dehydrogenase (LDH) level
was most positively correlated with APACHE II and SOFA. Additionally,
strong positive correlations were found between LDH and observational
assessment of severity of pneumonia and of underlying coronary disease.
Notably, serum LDH was also positively correlated with other univariate
parameters, including troponin and CRP. Meanwhile, LDH was negatively
correlated with lymphocyte count across different subsets, including
CD4<sup>+</sup> and CD8<sup>+</sup> T cells.</p>
<p><em>Limitations</em></p>
<p>Several studies have previously identified serum LDH as a predictive
biomarker for COVID-19 progression (see references). Therefore, it is
important to note that while LDH correlates with disease severity, it
does not suffice as a reason for severe disease, so whether it serves as
a robust <strong>predictive</strong> factor is not thoroughly supported; an
explanation is still needed between the release of LDH from infected
cells and disease severity.</p>
<p><em>Significance</em></p>
<p>The identification of serum LDH as a predictive biomarker does glean
insight into possible links between its release into the circulation and
immune response to SARS-CoV-2 pathogenesis. It is important to note that
LDH release itself requires significant damage to and permeabilization
of plasma membranes, as observed in pyroptosis and necrotic cell death.
Therefore, this report, and others like it, warrants investigation into
the link between SARS-CoV-2 infection and potential necrosis or
pyroptosis of infected host cells. It is important to note that previous
studies concerning biomarkers for SARS-CoV-1 disease have also
identified serum LDH as an important correlative factor with disease
severity<sup>1</sup>. And subsequent investigations have studied the activation
of inflammasome structures involved in pyroptosis in the presence of
SARS-CoV<sup>2-3</sup>.</p>
<p><em>Reviewed by Matthew D. Park as part of a project by students, postdocs,
and faculty at the Immunology Institute of the Icahn School of Medicine,
Mount Sinai.</em></p>
<p><em>References</em></p>
<p>1. Tsui PT, Kwok ML, Yuen H, Lai ST. Severe acute respiratory syndrome:
clinical outcome and prognostic correlates. <em>Emerg Infect Dis</em>.
2003;9(9):1064–1069. doi:10.3201/eid0909.030362</p>
<p>2. Yue, Y, Nabar, NR, Shi, C. <em>et al.</em> SARS-Coronavirus Open Reading
Frame-3a drives multimodal necrotic cell death. <em>Cell Death
Dis</em> 2018;<strong>9, </strong>904. doi: 10.1038/s41419-018-0917-y</p>
<p>3. Shi, C., Nabar, N.R., Huang, N. <em>et al.</em> SARS-Coronavirus Open
Reading Frame-8b triggers intracellular stress pathways and activates
NLRP3 inflammasomes. <em>Cell Death Discov.</em> 2019;<strong>5, </strong>101. doi:
10.1038/s41420-019-0181-7</p>
<h2 data-number="9.180" id="a-new-predictor-of-disease-severity-in-patients-with-covid-19-in-wuhan-china-1"><span class="header-section-number">9.180</span> A New Predictor of Disease Severity in Patients with COVID-19 in Wuhan, China</h2>
<p>Zhou et al. <em>bioRxiv</em>. <span class="citation" data-cites="EWZmuLvu">[<a href="#ref-EWZmuLvu" role="doc-biblioref">642</a>]</span></p>
<h3 data-number="9.180.1" id="keywords-179"><span class="header-section-number">9.180.1</span> Keywords</h3>
<ul>
<li>disease severity</li>
<li>clinical data</li>
<li>Neutrophils/lymphocytes ratio</li>
<li>CRP</li>
<li>D-dimer</li>
</ul>
<h3 data-number="9.180.2" id="main-findings-173"><span class="header-section-number">9.180.2</span> Main Findings</h3>
<p>377 hospitalized patients were divided into two groups: severe and
non-severe pneumonia. The laboratory results of their first day of
admission were retrospectively analyzed to identify predictors of
disease severity.</p>
<p>After adjusting for confounding factors from chronic comorbidities (such
as high blood pressure, type 2 diabetes, coronary heart disease, and
chronic obstructive pulmonary disease), the independent risk factors
identified for severe pneumonia were <strong>age</strong>, the <strong>ratio of
neutrophil/lymphocytes counts</strong>, <strong>CRP</strong> and <strong>D-dimer</strong> levels.</p>
<p>To further increase the specificity and sensibility of these markers,
they showed that their multiplication <strong>[(Neutrophil/lymphocyte count)
* CRP * D-dimer]</strong> was a better predictor of disease severity, with
higher sensitivity (95.7%) and specificity (63.3%), with a cutoff value
of 2.68.</p>
<h3 data-number="9.180.3" id="limitations-177"><span class="header-section-number">9.180.3</span> Limitations</h3>
<p>This study included 377 hospitalized patients. Among them, 45.6%
patients tested positive for SARS-Cov-2 nucleic acid test results, and
others were included in the study based on clinically diagnosis even if
the molecular diagnosis was negative. Thus, additional studies are
needed to verify this on a larger number of covid-19 certified patients
and the cutoff value might be adjusted. Also, all the patients that did
not have the clinical characteristics of severe pneumonia were included
in the non-severe pneumonia group, but usually patients are also divided
into moderate and mild disease.</p>
<p>Also, studying different subset of lymphocytes could lead to a more
specific predictor. Another study showed that the neutrophils to CD8+ T
cells ratio was a strong predictor of disease severity <span class="citation" data-cites="zTGzuiTI">[<a href="#ref-zTGzuiTI" role="doc-biblioref">551</a>]</span>. Another more
precise study showed that the percentage of helper T cells and
regulatory T cells decrease but the percentage of naïve helper T cells
increases in severe cases <span class="citation" data-cites="6Y81C4P3">[<a href="#ref-6Y81C4P3" role="doc-biblioref">544</a>]</span>. Taking these subpopulations into account
might make the predictor more powerful.</p>
<p>Other studies also noted an inverse correlation between disease severity
and LDH <span class="citation" data-cites="USOnp7c0">[<a href="#ref-USOnp7c0" role="doc-biblioref">585</a>]</span> or IL6 <span class="citation" data-cites="1GEbBtBgh">[<a href="#ref-1GEbBtBgh" role="doc-biblioref">594</a>]</span> levels, but the authors here do not discuss LDH nor
IL6 levels, although this could help to strengthen the predictor.</p>
<p>The study is based on the results obtained on the first day of
admission, studying the dynamic of the changes in patients might also be
interesting to better predict disease severity.</p>
<h3 data-number="9.180.4" id="significance-164"><span class="header-section-number">9.180.4</span> Significance</h3>
<p>This study confirms that the neutrophil to lymphocyte ratio can be a
predictor of disease severity as shown by many others <span class="citation" data-cites="6Y81C4P3 VRoblZFJ gK5e7Yhl">[<a href="#ref-gK5e7Yhl" role="doc-biblioref">543</a>,<a href="#ref-6Y81C4P3" role="doc-biblioref">544</a>,<a href="#ref-VRoblZFJ" role="doc-biblioref">558</a>]</span>. The
novelty here is that they show that a combination with other markers can
enhance the specificity and sensibility of the predictor, although the
study could be improved by taking into account sub-populations of
lymphocytes and more biological factors from patients such as LDH and
IL6.</p>
<h3 data-number="9.180.5" id="credit-174"><span class="header-section-number">9.180.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.181" id="metabolic-disturbances-and-inflammatory-dysfunction-predict-severity-of-coronavirus-disease-2019-covid-19-a-retrospective-study-1"><span class="header-section-number">9.181</span> Metabolic disturbances and inflammatory dysfunction predict severity of coronavirus disease 2019 (COVID-19): a retrospective study</h2>
<p>Shuke Nie et al. <em>medRxiv</em>. <span class="citation" data-cites="95sQSGzx">[<a href="#ref-95sQSGzx" role="doc-biblioref">643</a>]</span></p>
<h3 data-number="9.181.1" id="keywords-180"><span class="header-section-number">9.181.1</span> Keywords</h3>
<ul>
<li>metabolism</li>
<li>fasting blood glucose</li>
<li>serum total protein</li>
<li>albumin</li>
<li>blood lipid</li>
<li>HDL-C</li>
<li>APOA1</li>
<li>lymphocytopenia</li>
<li>IL-6</li>
<li>CRP</li>
<li>severity prediction of COVID19</li>
</ul>
<h3 data-number="9.181.2" id="main-findings-174"><span class="header-section-number">9.181.2</span> Main Findings</h3>
<p>Retrospective Study on 97 COVID-19 hospitalized patients
(25 severe and 72 non-severe) analyzing clinical and laboratory
parameter to predict transition from mild to severe disease based on
more accessible indicators (such as fasting blood glucose, serum protein
or blood lipid) than inflammatory indicators. In accordance with other
studies, age and hypertension were risk factors for disease severity,
and lymphopenia and increased IL-6 was observed in severe patients. The
authors show that fasting blood glucose (FBG) was altered and patients
with severe disease were often hyperglycemic. Data presented support
that hypoproteinaemia, hypoalbuminemia, and reduction in
high-densitylipoprotein (HDL-C) and ApoA1 were associated with disease
severity.</p>
<h3 data-number="9.181.3" id="limitations-178"><span class="header-section-number">9.181.3</span> Limitations</h3>
<p>In this study non-severe patients were divided in two groups based on
average course of the disease: mild group1 (14 days, n=28) and mild
group 2 (30 days, n=44). However mild patients with a longer disease
course did not show an intermediate phenotype (between mild patients
with shorter disease course and severe patients), hence it is unclear
whether this was a useful and how it impacted the analysis. Furthermore,
the non-exclusion of co-morbidity factors in the analysis may bias the
results (e.g. diabetic patients and glucose tests) It is not clear at
what point in time the laboratory parameters are sampled. In table 3, it
would have been interesting to explore a multivariate multiple
regression. The correlation lacks of positive control to assess the
specificity of the correlation to the disease vs. correlation in any
inflammatory case. The dynamic study assessing the predictability of the
laboratory parameter is limited to 2 patients. Hence there are several
associations with disease severity, but larger studies are necessary to
test the independent predictive value of these potential biomarkers.</p>
<h3 data-number="9.181.4" id="significance-165"><span class="header-section-number">9.181.4</span> Significance</h3>
<p>As hospital are getting overwhelmed a set of
easily accessible laboratory indicators (such as serum total protein)
would potentially provide a triage methodology between potentially
severe cases and mild ones. This paper also opens the question regarding
metabolic deregulation and COVID-19 severity.</p>
<h3 data-number="9.181.5" id="credit-175"><span class="header-section-number">9.181.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.182" id="viral-kinetics-and-antibody-responses-in-patients-with-covid-19-1"><span class="header-section-number">9.182</span> Viral Kinetics and Antibody Responses in Patients with COVID-19</h2>
<p><span class="citation" data-cites="btVuoX6V">[<a href="#ref-btVuoX6V" role="doc-biblioref">644</a>]</span></p>
<h3 data-number="9.182.1" id="keywords-181"><span class="header-section-number">9.182.1</span> Keywords</h3>
<ul>
<li>CoVID-19</li>
<li>2019-nCoV</li>
<li>SARS-CoV-2</li>
<li>IgG</li>
<li>IgM</li>
<li>clinical</li>
<li>kinetics</li>
<li>antibodies</li>
</ul>
<h3 data-number="9.182.2" id="main-findings-175"><span class="header-section-number">9.182.2</span> Main Findings</h3>
<ul>
<li><p>Prospective cohort of 67 patients, clinical specimens taken and
follow-up conducted.</p></li>
<li><p>Viral shedding, serum IgM, IgG antibody against NP evaluated and
correlated to disease severity and clinical outcome</p></li>
<li><p>Viral RNA levels peaked at 1 week from febrile/cough symptom onset
in sputum, nasal swabs, and stool samples. Shedding ranged from
12-19 days (median ranges) and was longer in severe patients.</p></li>
<li><p>IgM and IgG titers stratified patients into three archetypes as
‘strong vs weak vs non-responders’. Strong responders (with higher
IgM/IgG titers) were significantly higher in severe patients.</p></li>
</ul>
<h3 data-number="9.182.3" id="limitations-179"><span class="header-section-number">9.182.3</span> Limitations</h3>
<p>Specific for immune monitoring.</p>
<ul>
<li><p>Not clear if stool RNA captured from live infection in
intestine/liver or from swallowed sputum. Transmission electron
microscopy (TEM) carried out on sputum samples as proof of concept,
but not stools. TEM unreasonable for actual clinical diagnosis.</p></li>
<li><p>Several patients had co-morbidities (such as pulmonary and liver
disease) that were not accounted for when tracking antibody
responses. Viral kinetics and IgM/IgG titers in subsets of patients
with underlying conditions/undergoing certain medication would be
informative.</p></li>
</ul>
<h3 data-number="9.182.4" id="significance-166"><span class="header-section-number">9.182.4</span> Significance</h3>
<ul>
<li><p>Three archetypes of antibody response to SARS-CoV-2 with different
disease progression and kinetics is useful to stratify patients, and
for future serological tests.</p></li>
<li><p>Strong spike-IgG levels often correlate with lymphopenia and
CoVID-19 disease severity <span class="citation" data-cites="LIf5SSUs">[<a href="#ref-LIf5SSUs" role="doc-biblioref">645</a>]</span>, similar to macaque
studies in SARS <span class="citation" data-cites="tC5vJmwj">[<a href="#ref-tC5vJmwj" role="doc-biblioref">467</a>]</span>. It would be critical to see if anti-NP or
anti-Spike IgG antibodies for SARS-CoV-2 also elicit similar
detrimental effects before clinical use.</p></li>
</ul>
<h3 data-number="9.182.5" id="credit-176"><span class="header-section-number">9.182.5</span> Credit</h3>
<p><em>Review by Samarth Hegde as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.183" id="covid-19-infection-induces-readily-detectable-morphological-and-inflammation-related-phenotypic-changes-in-peripheral-blood-monocytes-the-severity-of-which-correlate-with-patient-outcome-1"><span class="header-section-number">9.183</span> COVID-19 infection induces readily detectable morphological and inflammation-related phenotypic changes in peripheral blood monocytes, the severity of which correlate with patient outcome</h2>
<p><span class="citation" data-cites="1FZgghazg">[<a href="#ref-1FZgghazg" role="doc-biblioref">646</a>]</span></p>
<h3 data-number="9.183.1" id="keywords-182"><span class="header-section-number">9.183.1</span> Keywords</h3>
<ul>
<li>Monocytes</li>
<li>FSC-high</li>
<li>PBMC</li>
<li>ACE2</li>
<li>inflammatory cytokines</li>
</ul>
<h3 data-number="9.183.2" id="main-findings-176"><span class="header-section-number">9.183.2</span> Main Findings</h3>
<p>This study is based on flow cytometry immunophenotyping of
PBMCs from 28 patients diagnosed positive for SARS-Cov2 (COVID19). The
authors identify a population of abnormally large (FSC-hi) monocytes,
present in COVID19 patients, but absent in PBMCs of healthy volunteers
(n=16) or patients with different infections (AIDS, malaria, TB). This
FSC-hi monocytic population contains classical, intermediate and
non-classical (monocytes (based on CD14 and CD16 expression) that
produce inflammatory cytokines (IL-6, TNF and IL-10). The authors
suggest an association of FSC-hi monocytes with poor outcome and
correlate a high percentage of FSC-low monocytes, or higher ratio of
FSC-low/hi monocytes, with faster hospital discharge.</p>
<h3 data-number="9.183.3" id="limitations-180"><span class="header-section-number">9.183.3</span> Limitations</h3>
<p>While identification of the monocytic population based
on FSC is rather robust, the characterization of these cells remains
weak. A comprehensive comparison of FSC-hi monocytes with FSC-low
monocytes from patients and healthy controls would be of high value. It
is unclear if percentages in blood are among CD45+ cells. Furthermore,
it would have been important to include absolute numbers of different
monocytic populations (in table 1 there are not enough samples and it is
unclear what the authors show).</p>
<p>The authors show expression of the ACE2 receptor on the surface of the
monocytes, and highlight these cells as potential targets of SARS-Cov2.
However, appropriate controls are needed. CD16 has high affinity to
rabbit IgG and it is unclear whether the authors considered unspecific
binding of rabbit anti-ACE2 to Fc receptors. Gene expression of ACE-2 on
monocytes needs to be assessed. Furthermore, it would be important to
confirm infection of monocytes by presence of viral proteins or viral
particles by microscopy.</p>
<p>Considering the predictive role of FSC-hi monocytes on the development
of the disease and its severity, some data expected at this level are
neither present nor addressed. Although the cohort is small, it does
include 3 ICU patients. What about their ratio of FSC-low vs FSC-hi
monocytes in comparison to other patients? Was this apparent early in
the disease course? Does this population of FSC-hi monocytes differ
between ICU patients and others in terms of frequency, phenotype or
cytokine secretion?</p>
<p>In general, figures need to revised to make the data clear. For example,
in Fig. 5, according to the legend it seems that patients with FSC-high
monocytes are discharged faster from the hospital. However according to
description in the text, patients were grouped in high or low levels of
FSC-low monocytes.</p>
<h3 data-number="9.183.4" id="significance-167"><span class="header-section-number">9.183.4</span> Significance</h3>
<p>Despite the limitations of this study, the discovery of
a FSC-high monocyte population in COVID-19 patients is of great
interest. With similar implication, a the recent study by Zhou et al. <span class="citation" data-cites="Bgo7Cmj3">[<a href="#ref-Bgo7Cmj3" role="doc-biblioref">549</a>]</span>
identified a connection between an inflammatory CD14+CD16+ monocyte
population and pulmonary immunopathology leading to deleterious clinical
manifestations and even acute mortality after SARS-CoV-2 infections.
Although the presence of these monocytes in the lungs has yet to be
demonstrated, such results support the importance of monocytes in the
critical inflammation observed in some COVID19 patients.</p>
<h3 data-number="9.183.5" id="credit-177"><span class="header-section-number">9.183.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.184" id="correlation-between-universal-bcg-vaccination-policy-and-reduced-morbidity-and-mortality-for-covid-19-an-epidemiological-study-1"><span class="header-section-number">9.184</span> Correlation between universal BCG vaccination policy and reduced morbidity and mortality for COVID-19: an epidemiological study</h2>
<p>Miller et al. <em>medRxiv</em>. <span class="citation" data-cites="15gdkEnfn">[<a href="#ref-15gdkEnfn" role="doc-biblioref">647</a>]</span></p>
<h3 data-number="9.184.1" id="keywords-183"><span class="header-section-number">9.184.1</span> Keywords</h3>
<ul>
<li>BCG vaccine</li>
<li>epidemiology</li>
<li>vaccination policy</li>
</ul>
<h3 data-number="9.184.2" id="main-findings-177"><span class="header-section-number">9.184.2</span> Main Findings</h3>
<p>The authors compared middle and high income countries that never had a
universal BCG vaccination policy (Italy, Lebanon, Nederland, Belgium)
and countries with a current policy (low income countries were excluded
from the analysis as their number of cases and deaths might be
underreported for the moment). <strong>Countries that never implement BCG
vaccination have a higher mortality rate than countries which have a BCG
vaccination policy</strong> (16.38 deaths per million people vs 0.78). Next,
<strong>the authors show that an earlier start of vaccination correlates with
a lower number of deaths per million inhabitants</strong>. They interpret this
as the vaccine protecting a larger fraction of elderly people, which are
usually more affected by COVID-19. Moreover, higher number of COVID-19
<strong>cases</strong> were presented in countries that never implemented a universal
BCG vaccination policy.</p>
<h3 data-number="9.184.3" id="limitations-181"><span class="header-section-number">9.184.3</span> Limitations</h3>
<p>While this study aims to test an intriguing hypothesis unfortunately,
the data is not sufficient at this time to accurately make any
determinations. Several caveats must be noted including: not all
countries are in the same stage of the pandemic, the number of
cases/deaths is still changing very rapidly in a lot of countries and
thus the association may only reflect exposure to the virus. This
analysis would need to be re-evaluated when all the countries are passed
the pandemic and more accurate numbers are available. Additionally, very
few middle and high-income countries ever implemented universal BCG
vaccination, which can be a source of bias (5 countries, vs 55 that have
a BCG vaccine policy). Effective screening and social isolation policies
also varied considerable across the countries tested and may reflect
another important confounder. The authors could consider analyzing the
Case Fatality Rate (CFR, % of patients with COVID-19 that die), to more
correct for exposure although testing availability will still bias this
result. Variability in mortality within countries or cities with
variable vaccination and similar exposure could also be appropriate
although confounders will still be present.</p>
<h3 data-number="9.184.4" id="significance-168"><span class="header-section-number">9.184.4</span> Significance</h3>
<p>BCG vaccine is a live attenuated strain derived from <em>Mycobacterium
bovis</em> and used for a vaccine for tuberculosis (TB). This vaccine has
been proven to be efficient in preventing childhood meningitis TB, but
doesn’t prevent adult TB as efficiently. For this reason, several
countries are now only recommending this vaccine for at-risk population
only.</p>
<p><strong>This study shows that there is a correlation between BCG vaccination
policy and reduced mortality for Covid-19</strong>. Indeed, BCG vaccine has
been shown to protect against several viruses and enhance innate
immunity <span class="citation" data-cites="16FOT89K5">[<a href="#ref-16FOT89K5" role="doc-biblioref">648</a>]</span>, which could explain why it could protect against SARS-CoV-2
infection, but the exact mechanism is still unknown. <strong>Moreover, the
efficiency of adult/older people vaccination and protection against
Covid-19 still needs to be assessed.</strong> Regarding this, Australian
researchers are starting a clinical trial of BCG vaccine for healthcare
workers <span class="citation" data-cites="MCuIRGFE">[<a href="#ref-MCuIRGFE" role="doc-biblioref">649</a>]</span>, to assess if it can protect them against Covid-19.</p>
<h3 data-number="9.184.5" id="credit-178"><span class="header-section-number">9.184.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="9.185" id="non-neural-expression-of-sars-cov-2-entry-genes-in-the-olfactory-epithelium-1"><span class="header-section-number">9.185</span> Non-neural expression of SARS-CoV-2 entry genes in the olfactory epithelium</h2>
<p><span class="citation" data-cites="12oDb7aki">[<a href="#ref-12oDb7aki" role="doc-biblioref">650</a>]</span></p>
<h3 data-number="9.185.1" id="keywords-184"><span class="header-section-number">9.185.1</span> Keywords</h3>
<ul>
<li>CoVID-19</li>
<li>2019-nCoV</li>
<li>SARS-CoV-2</li>
<li>anosmia</li>
<li>olfaction</li>
<li>scRNAseq</li>
</ul>
<h3 data-number="9.185.2" id="main-findings-178"><span class="header-section-number">9.185.2</span> Main Findings</h3>
<ul>
<li><p>Study analyzed bulk and scRNAseq data of olfactory cell types from
publicly-available mouse, nonhuman primate and human datasets.</p></li>
<li><p>show that <em>ACE2</em> and <em>TMPRSS2</em> (genes involved in SARS-CoV-2 entry)
are expressed in olfactory epithelial (OE) cells, basal stem cells
and respiratory epithelium (RE), but not sensory neurons.</p></li>
<li><p>Comparison of human RE and OE datasets (Deprez et al. 2019; Durante
et al. 2020) revealed that <em>ACE2</em> and <em>TMPRSS2</em> expression in OE
sustentacular cells was similar to expression in the remainder of
the non-nasal respiratory tract.</p></li>
</ul>
<h3 data-number="9.185.3" id="limitations-182"><span class="header-section-number">9.185.3</span> Limitations</h3>
<ul>
<li><p>Transcript data alone from healthy respiratory/olfactory cells is
not sufficient to confirm infectivity of nasal passage, or to
indicate damage to epithelia.</p></li>
<li><p>No mechanism defined for anosmia; it is not clear if epithelial
injury leads to reduced sensitivity or increased inflammation and
altered immune contexture drives neural/epithelial dysfunction. Will
be critical to test this in CoVID-19 patient samples or mouse
models.</p></li>
</ul>
<h3 data-number="9.185.4" id="significance-169"><span class="header-section-number">9.185.4</span> Significance</h3>
<ul>
<li><p>Study provides possible rationale for anosmia observed in several
CoVID-19 patients.</p></li>
<li><p>Raises possibility that nasal respiratory goblet, ciliated cells,
and olfactory epithelia may serve as a viral reservoir after initial
SARS-CoV-2 infection.</p></li>
<li><p>Human olfactory sensory neurons express several other molecules
important to CoV (not CoV-19) entry such as <em>FURIN, ST6GAL1,
ST3GAL4</em>; this suggests wider mechanism of neuronal infectivity in
other coronaviruses.</p></li>
</ul>
<h3 data-number="9.185.5" id="credit-179"><span class="header-section-number">9.185.5</span> Credit</h3>
<p><em>Review by Samarth Hegde as part of a project by students, postdocs and
faculty at the Immunology Institute of the Icahn School of Medicine,
Mount Sinai.</em></p>
<p><strong>Title: </strong></p>
<p><strong>SARS-CoV-2 proteome microarray for mapping COVID-19 antibody
interactions at amino acid resolution</strong></p>
<p><strong>Immunology keywords:</strong> SARS-CoV-2, COVID-19, high throughput, peptide
microarray, antibody epitope screening</p>
<p><strong>The main finding of the article: </strong></p>
<p>This study screened the viral protein epitopes recognized by antibodies
in the serum of 10 COVID-19 patients using a new SARS-CoV-2 proteome
peptide microarray. The peptide library was constructed with 966 linear
peptides, each 15 amino acids long with a 5 amino acid overlap, based on
the protein sequences encoded by the genome of the Wuhan-Hu-1 strain.</p>
<p>To investigate crossreactivity between SARS-CoV-1 and SARS-CoV-2, they
tested rabbit monoclonal and polyclonal antibodies against SARS-CoV-1
nucleocapsid (N) in the microarray. Antibodies against SARS-CoV-1 N
displayed binding to the SARS-CoV-2 nucleocapsid (N) peptides.
Polyclonal antibodies showed some crossreactivity to other epitopes from
membrane (M), spike (S), ORF1ab and ORF8. This suggests that previous
exposure to SARS-CoV-1 may induced antibodies recognizing both viruses.</p>
<p>Screening of IgM and IgG antibodies from 10 COVID-19 patients showed
that many antibodies targeted peptides on M, N, S, Orf1ab, Orf3a, Orf7a,
and Orf8 from SARS-CoV-2, while immunodominant epitopes with antibodies
in more than 80 % COVID-19 patients were present in N, S and Orf3. It is
shown that the receptor binding domain (RBD) resides on S protein and
RBD is important for SARS-CoV-2 to enter the host cells via ACE2. Among
six epitopes on S protein, structural analysis predicted that three
epitopes were located at the surface and three epitopes were located
inside of the protein. Furthermore, some IgM antibodies from 1 patient
and IgG antibodies from 2 patients bound to the same epitope (residue
456-460, FRKSN) which resided within the RBD, and structural analysis
determined that this epitope was located in the region of the RBD loop
that engages with ACE2.</p>
<p><strong>Critical analysis of the study:</strong></p>
<p>In addition to the limitations mentioned in the manuscript, it would
have been informative to do the analysis over the course of the disease.
The pattern of antibody recognition, especially on S protein, and the
course of antibodies of different isotypes recognizing the same peptide
might correlate to the clinical course in these patients. It would alos
have been informative to analyze the presence of cross-reactive
antibodies from pateints previously exposed to SARS-CoV-1.</p>
<p><strong>The importance and implications for the current epidemics:</strong></p>
<p>This study identified linear immunodominant epitopes on SARS-CoV-2,
Wuhan-Hu-1 strain. This is a valuable information to design vaccines
that will elicit desirable immune responses.</p>
<p><strong>The Novel Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2)
Directly Decimates Human Spleens and Lymph Nodes</strong></p>
<p>Review by Matthew D. Park</p>
<p>Revised by Miriam Merad</p>
<p>Keywords: COVID-19, SARS-CoV-2, spleen, lymph node, ACE2, macrophage</p>
<p><em>Main findings</em></p>
<p>It has been previously reported that COVID-19 patients exhibit severe
lymphocytopenia, but the mechanism through which this depletion occurs
has not been described. In order to characterize the cause and process
of lymphocyte depletion in COVID-19 patients, the authors performed
gross anatomical and <em>in situ</em> immune-histochemical analyses of spleens
and lymph nodes (hilar and subscapular) obtained from post-mortem
autopsies of 6 patients with confirmed positive viremia and 3 healthy
controls (deceased due to vehicle accidents).</p>
<p>Primary gross observations noted significant splenic and LN atrophy,
hemorrhaging, and necrosis with congestion of interstitial blood vessels
and large accumulation of mononuclear cells and massive lymphocyte
death. They found that CD68<sup>+</sup> CD169<sup>+</sup> cells in the spleens, hilar and
subscapular LN, and capillaries of these secondary lymphoid organs
expressed the ACE2 receptor and stain positive for the SARS-CoV-2
nucleoprotein (NP) antigen, while CD3<sup>+</sup> T cells and B220<sup>+</sup> B cells
lacked both the ACE2 receptor and SARS-CoV-2 NP antigen. ACE2<sup>+</sup> NP<sup>+</sup>
CD169<sup>+</sup> macrophages were positioned in the splenic marginal zone (MZ)
and in the marginal sinuses of LN, which suggests that these macrophages
were positioned to encounter invading pathogens first and may contribute
to virus dissemination.</p>
<p>Since SARS-CoV-2 does not directly infect lymphocytes, the authors
hypothesized that the NP<sup>+</sup> CD169<sup>+</sup> macrophages are responsible for
persistent activation of lymphocytes via Fas::FasL interactions that
would mediate activation-induced cell death (AICD). Indeed, the
expression of Fas was significantly higher in virus-infected tissue than
that of healthy controls, and TUNEL staining showed significant
lymphocytic apoptosis. Since pro-inflammatory cytokines like IL-6 and
TNF-α can also engage cellular apoptosis and necrosis, the authors
interrogated the cytokine expression of the secondary lymphoid organs
from COVID-19 patients; IL-6, not TNF-α, was elevated in virus-infected
splenic and lymph node tissues, compared to those of healthy controls,
and immunofluorescent staining showed that IL-6 is primarily produced by
the infected macrophages. <em>In vitro</em> infection of THP1 cells with
SARS-CoV-2 spike protein resulted in selectively increased <em>Il6</em>
expression, as opposed to <em>Il1b</em> and <em>Tnfa</em> transcription. Collectively,
the authors concluded that a combination of Fas up-regulation and IL-6
production by NP<sup>+</sup> CD169<sup>+</sup> macrophages induce AICD in lymphocytes in
secondary lymphoid organs, resulting in lymphocytopenia.</p>
<p>In summary, this study reports that CD169<sup>+</sup> macrophages in the splenic
MZ, subscapular LN, and the lining capillaries of the secondary lymphoid
tissues express ACE2 and are susceptible to SARS-CoV-2 infection. The
findings point to the potential role of these macrophages in viral
dissemination, immunopathology of these secondary lymphoid organs,
hyperinflammation and lymphopenia.</p>
<p><em>Limitations</em></p>
<p><em>Technical</em></p>
<p>A notable technical limitation is the small number of samples (n=6);
moreover, the analysis of these samples using multiplexed
immunohistochemistry and immunofluorescence do not necessarily provide
the depth of unbiased interrogation needed to better identify the cell
types involved.</p>
<p><em>Biological</em></p>
<p>The available literature and ongoing unpublished studies, including
single-cell experiments of spleen and LN from organ donors, do not
indicate that ACE2 is expressed by macrophages; however, it remains
possible that ACE2 expression may be triggered by type I IFN in COVID-19
patients. Importantly, the SARS-CoV-2 NP staining of the macrophages
does not necessarily reflect direct infection of these macrophages;
instead, positive staining only indicates that these macrophages carry
SARS-CoV-2 NP as antigen cargo, which may have been phagocytosed. Direct
viral culture of macrophages isolated from the secondary lymphoid organs
with SARS-CoV-2 is required to confirm the potential for direct
infection of macrophages by SARS-CoV-2. Additionally, it is important to
note that the low to negligible viremia reported in COVID-19 patients
to-date does not favor a dissemination route via the blood, as suggested
by this study, which would be necessary to explain the presence of
virally infected cells in the spleen.</p>
<p><em>Relevance</em></p>
<p>Excess inflammation in response to SARS-CoV-2 infection is characterized
by cytokine storm in many COVID-19 patients. The contribution of this
pathology to the overall fatality rate due to COVID-19, not even
necessarily directly due to SARS-CoV-2 infection, is significant. A
better understanding of the full effect and source of some of these
major cytokines, like IL-6, as well as the deficient immune responses,
like lymphocytopenia, is urgently needed. In this study, the authors
report severe tissue damage in spleens and lymph nodes of COVID-19
patients and identify the role that CD169<sup>+</sup> macrophages may play in the
hyperinflammation and lymphocytopenia that are both characteristic of
the disease. It may, therefore, be important to note the effects that
IL-6 inhibitors like Tocilizumab and Sarilumab may specifically have on
splenic and LN function. It is important to note that similar
observations of severe splenic and LN necrosis and inflammation in
patients infected with SARS-CoV-1 further support the potential
importance and relevance of this study.</p>
<h2 data-number="9.186" id="cigarette-smoke-triggers-the-expansion-of-a-subpopulation-of-respiratory-epithelial-cells-that-express-the-sars-cov-2-receptor-ace2-1"><span class="header-section-number">9.186</span> Cigarette smoke triggers the expansion of a subpopulation of respiratory epithelial cells that express the SARS-CoV-2 receptor ACE2</h2>
<p><span class="citation" data-cites="t4vofknR">[<a href="#ref-t4vofknR" role="doc-biblioref">651</a>]</span></p>
<h3 data-number="9.186.1" id="keywords-185"><span class="header-section-number">9.186.1</span> Keywords</h3>
<ul>
<li>CoVID-19</li>
<li>2019-nCoV</li>
<li>SARS-CoV-2</li>
<li>respiration</li>
<li>cigarette</li>
<li>ACE2</li>
<li>lung</li>
</ul>
<h3 data-number="9.186.2" id="main-findings-179"><span class="header-section-number">9.186.2</span> Main Findings</h3>
<ul>
<li><p>Study uses scRNAseq, bulk seq data and air-liquid interface culture
experiments to show that cigarette smoke causes a dose-dependent
upregulation of ACE2 in mouse and human lungs (transplantation,
tumor resection, or IPF datasets).</p></li>
<li><p>ACE2 was not up-regulated in patients with asthma or
lung-sarcoidosis or in mouse models of cystic fibrosis or carcinogen
exposure.</p></li>
<li><p>Cathepsin B (alternate protease involved in viral entry) is
increased in smoke-exposed mouse or human lungs.</p></li>
<li><p>Smoke triggers a protective expansion of mucus-secreting MUC5AC+
goblet and SCGB1A1+ club cells; ACE2 presence in these cells is
increased upon smoke exposure.</p></li>
</ul>
<h3 data-number="9.186.3" id="limitations-183"><span class="header-section-number">9.186.3</span> Limitations:</h3>
<ul>
<li><p>Long-term smokers usually have several co-morbidities including
immune dysfunction, which can affect interpretation of CoV-2
susceptibility in these datasets. Ideally, analyses can control for
major co-morbidities across smokers and non-smokers (immune
suppression, cardiovascular disease and atherosclerosis).</p></li>
<li><p>Hyperplasia of ACE2+ goblet cells upon smoking needs to be separated
from ACE2 upregulation in existing goblet cells.</p></li>
<li><p>ACE2 expression increase alone does not confirm increased viral
entry into goblet cells; future studies with air-liquid interface
cultures testing CoV-2 infectivity in <em>ex vivo</em> epithelial cells
from human epithelial lines, ex vivo samples or hACE2 mice will be
very informative.</p></li>
</ul>
<h3 data-number="9.186.4" id="significance-170"><span class="header-section-number">9.186.4</span> Significance</h3>
<ul>
<li><p>This study may partially explain why smokers are more likely to
develop severe SARS-CoV-2 infections. Also, the reversibility of
ACE2 expression upon smoking cessation suggests that quitting
smoking could lessen CoV-2 susceptibility.</p></li>
<li><p>Absence of ACE2 upregulation in other lung inflammation pathologies
implies CoV-2 susceptibility might be smoking-specific, and not
fibrosis-specific.</p></li>
<li><p>Another preprint showed ACE2 expression increases in lung of
patients with CoV-2 co-morbidities such as hypertension <span class="citation" data-cites="wjtVy2v0">[<a href="#ref-wjtVy2v0" role="doc-biblioref">626</a>]</span>; these studies
collectively paint a better picture of CoV-2 susceptibility before
actual experiments can be carried out.</p></li>
</ul>
<h3 data-number="9.186.5" id="credit-180"><span class="header-section-number">9.186.5</span> Credit</h3>
<p><em>Review by Samarth Hegde as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<p><strong>Structural Analysis of SARS-CoV-2 and Predictions of the Human
Interactome</strong></p>
<p><a href="https://www.biorxiv.org/content/10.1101/2020.03.28.013789v4.full.pdf"><em>https://www.biorxiv.org/content/10.1101/2020.03.28.013789v4.full.pdf</em></a></p>
<p><strong>Keywords:</strong></p>
<ul>
<li><p>SARS-CoV-2</p></li>
<li><p>Computational structural analysis</p></li>
<li><p>RNA-protein interactions</p></li>
<li><p>Binding partner predictions</p></li>
</ul>
<p><strong>Main findings:</strong></p>
<p>To elucidate mechanisms of viral replication within human hosts, the
authors used computational approaches to analyze the structural
properties of SARS-CoV-2 RNA and predict human proteins that bind to it.
They compared 2800 coronaviruses and 62 SARS-CoV-2 strains using
<em>CROSS</em> (Computational Recognition of Secondary Structure) and
<em>CROSS</em>align algorithms, which predict RNA structure using sequence
information and evaluate structural conservation, respectively. From
these structural comparisons, they found that the spike S protein that
interacts with the human receptor angiotensin-converting enzyme 2 (ACE2)
is highly conserved amongst coronaviruses.</p>
<p>The study also identified over 100,000 human protein interactions with
SARS-CoV-2 utilizing <em>cat</em>RAPID, an algorithm that determines binding
potentials of proteins for RNA using secondary structure, van der Waals,
and hydrogen bonding contributions. They found that the 5’ of SARS-CoV-2
is highly structured and has a strong propensity to bind to human
proteins with known involvement in viral RNA processing. Amongst the
proteins they identified can bind to 5’, there was significant
enrichment in proteins associated with HIV infection and replication,
including ATP-dependent RNA Helicase (DDX1), A-kinase anchor protein
8-like (AKAP8L), and dsRNA-specific Editase 1 (ADARB1).</p>
<p><strong>Limitations:</strong></p>
<p>The study was based off of computational structural predictions and
subsequent gene ontology enrichment analyses. Computational predictions
are less accurate than experimental observations, and while their
predicted binding partners prompt interesting hypotheses, they must be
experimentally confirmed. Furthermore, the gene ontology annotations
used to create their candidate list are based off prior studies and
inherently miss novel biological implications. Targeted biochemical and
structural studies that can be built off their identified targets will
be essential for elucidating viral-host complexes and informing
potential targeted drug designs.</p>
<p><strong>Significance:</strong></p>
<p>Their computational approach demonstrated findings confirmatory of other
studies in many respects. Their structural analysis findings of the high
conservation of the spike S protein amongst analyzed coronaviruses and
SARS-CoV-2 strains suggest that the spike S evolved to specifically
interact with host ACE2, supporting that the human engineering of
SARS-CoV2 is very unlikely and implicating spike S as a potential
therapeutic target. Their RNA-protein interaction predictions suggest
several relevant host-virus interactions that warrant further
investigation. If proved experimentally, their identified links to
proteins studied in the context of HIV and other viruses may be relevant
for the repurposing of existing antiviral drugs for SARS-CoV-2.</p>
<p><em>Review by Michelle Tran as part of a project by students, postdocs, and
faculty at the Immunology Institute of the Icahn School of Medicine at
Mount Sinai.</em></p>
<p>Review of paper titled:</p>
<p><strong>Virus shedding patterns in nasopharyngeal and fecal specimens
of1COVID-19 patients</strong></p>
<p>Zhang et al. 2020</p>
<p>Keywords: Viral shedding, testing specimens.</p>
<p><strong>Summary </strong></p>
<p>The authors tested feces, urine, plasma, nasal/throat swabs (respiratory
samples) for SARS-CoV-2 virus load analysis by qPCR several days after
illness onset (DAO).</p>
<ul>
<li><p>All plasma samples were negative (n=56).</p></li>
<li><p>Two urine samples were positive from patients that had severe
disease and were positive at least 16-21 DAO (n=56).</p></li>
<li><p>10 of 12 cases (83.3%) tested positive for fecal samples.</p></li>
<li><p>14 of 21 cases (66.7%) were positive for respiratory samples.</p></li>
<li><p>Respiratory samples were positive for 21 DAO, nevertheless fecal
samples were still positive.</p></li>
<li><p>Median duration of virus shedding was 10.0 days in respiratory
samples</p></li>
<li><p>For fecal samples median shedding was 22.0 days for the feces</p></li>
<li><p>Viral titers of respiratory samples peaked at six to nine DAO. and
at 14-18 DAO for fecal samples, and the highest virus titers at the
peak was significantly higher for feces (105.8 copies/ml, mean 5623
copies/ml) than of respiratory samples (106.363 copies/ml, mean 2535
copies/ml).</p></li>
</ul>
<p>Importance of findings and caveats</p>
<p>Notably, these data reflect viral RNA and there is mention of infectious
particles. Without these data, it is difficult to ascertain the
significance of this paper. While these data demonstrate that the
duration of fecal viral RNA shedding is significantly prolonged when
compared to viral RNA shedding from the respiratory tract, it is
critical for the field to ascertain if this represents infectious
virions or not. If viable virus were to be isolated from the GI tract,
that would indeed be very concerning for potential feco-oral
transmission of SARS-COV2. As such, this paper does not establish this
important fact.</p>
<p><em>Review by Jovani Catalan-Dibene and validated by Saurabh Mehandru as
part of a project by students, postdocs and faculty at the Immunology
Institute of the Icahn school of medicine, Mount Sinai.</em></p>
<h2 data-number="9.187" id="the-comparative-superiority-of-igm-igg-antibody-test-to-real-time-reverse-transcriptase-pcr-detection-for-sars-cov-2-infection-diagnosis-1"><span class="header-section-number">9.187</span> The comparative superiority of IgM-IgG antibody test to real-time reverse transcriptase PCR detection for SARS-CoV-2 infection diagnosis</h2>
<p>Liu et al. <em>medRxiv</em>.
<span class="citation" data-cites="k1E3DuFN">[<a href="#ref-k1E3DuFN" role="doc-biblioref">652</a>]</span></p>
<h3 data-number="9.187.1" id="keywords-186"><span class="header-section-number">9.187.1</span> Keywords</h3>
<ul>
<li>IgM/IgG antibody test</li>
<li>nucleic acid test</li>
<li>SARS-CoV-2 detection</li>
</ul>
<h3 data-number="9.187.2" id="main-findings-180"><span class="header-section-number">9.187.2</span> Main Findings</h3>
<p>The study compares IgM and IgG antibody testing to RT-PCR detection of
SARS-CoV-2 infection. 133 patients diagnosed with SARS-CoV-2 in Renmin
Hospital (Wuhan University, China) were analyzed. The positive ratio was
78.95% (105/133) in IgM antibody test (SARS-CoV-2 antibody detection kit
from YHLO Biotech) and 68.42% (91/133) in RT-PCR (SARS-CoV-2 ORF1ab/N
qPCR detection kit). There were no differences in the sensitivity of
SARS-CO-V2 diagnosis in patients grouped according to disease severity.
For example, IgG responses were detected in 93.18% of moderate cases,
100% of severe cases and 97.3% of critical cases. In sum, positive
ratios were higher in antibody testing compared to RT-PCR detection,
demonstrating a higher detection sensitivity of IgM-IgG testing for
patients hospitalized with COVID-19 symptoms.</p>
<h3 data-number="9.187.3" id="limitations-184"><span class="header-section-number">9.187.3</span> Limitations</h3>
<p>This analysis only included one-time point of 133 hospitalized patients,
and the time from symptom onset was not described. There was no
discussion about specificity of the tests and no healthy controls were
included. It would be important to perform similar studies with more
patients, including younger age groups and patients with mild symptoms
as well as asymptomatic individuals. It is critical to determine how
early after infection/symptom onset antibodies can be detected and the
duration of this immune response.</p>
<h3 data-number="9.187.4" id="significance-171"><span class="header-section-number">9.187.4</span> Significance</h3>
<p>The IgM-IgG combined testing is important to improve clinical
sensitivity and diagnose COVID-19 patients. The combined antibody test
shows higher sensitivity than individual IgM and IgG tests or nucleic
acid-based methods, at least in patients hospitalized with symptoms.</p>
<h3 data-number="9.187.5" id="credit-181"><span class="header-section-number">9.187.5</span> Credit</h3>
<p><em>Review by Erica Dalla as part of a project by students, postdocs and
faculty at the Immunology Institute of the Icahn School of Medicine,
Mount Sinai.</em></p>
<p><strong>Clinical observations of low molecular weight heparin in relieving
inflammation in COVID-19 patients: a retrospective cohort study</strong></p>
<p><strong><em>Key words</em></strong></p>
<p>Heparin, IL-6, cytokine storm, anti-inflammatory, anti-coagulant</p>
<p><strong><em>Key findings</em></strong></p>
<p>The authors investigated the use of a commercially available form of
heparin, low molecular weight heparin (LMWH), as a therapeutic drug for
patients with COVID-19. Previous studies showed that in addition to its
anticoagulant properties, LMWH exerts anti-inflammatory effects by
reducing the release of IL-6 and counteracting IL-6.</p>
<p>This was a retrospective single-center study conducted in Wuhan, China.
Forty-two (42) hospitalized patients with coronavirus pneumonia were
included, of which 21 underwent LMWH treatment (heparin group) and 21
did not (control). The general characteristics of the two groups of
patients were statistically comparable. Both control and LMWH had the
same hospitalization time and there were no critical cases in either
group.</p>
<p>This study found that treatment with LMWH significantly reduced IL-6
levels in patients in the heparin group compared to the control group.
However, LMWH treatment did not have an effect on the levels of other
inflammatory factors: CRP, IL-2, IL-4, IL-10, TNF-α, and IFN-γ. Compared
with the control group, patients in the heparin group had a
significantly increased percentage of lymphocytes after treatment,
further suggesting that LMWH treatment has anti-inflammatory effects and
can reduce the lymphopenia associated with COVID-19.</p>
<p>Consistent with other studies in COVID-19 patients, they found that LMWH
treatment can improve hypercoagulability. D-dimer and FDP levels
(biomarkers of coagulation) in the heparin group significantly decreased
from baseline after treatment, whereas there was no significant change
in levels for the control group. Of note however, patients in the
heparin group had a significantly higher level of D-dimer and FDP at
baseline compared to the control group.</p>
<p><strong><em>Importance</em></strong></p>
<p>Many studies have shown that severely ill COVID-19 patients have
significantly higher levels of IL-6 compared to patients with mild cases
and it has been proposed that progression to severe disease may be
caused by lymphopenia and cytokine storms. The anti-inflammatory effects
of heparin may help prevent or reverse a cytokine storm caused by the
virus and thus delay COVID-19 progression and improve overall condition
in patients. The pleiotropic effects of heparin may have a greater
therapeutic effect than compounds that are directed against a single
target, such as an anti-IL-6 therapy. This is because COVID-19 patients
commonly have complications such as coagulopathy and endothelial
dysfunction leading to cardiac pathology that may be mitigated by
heparin treatment (Li J, et.al; Wojnicz et.al).</p>
<p><strong><em>Limitations</em></strong></p>
<p>This study is limited by a small sample size (n=44) and a single-center
design. Double-blinded, randomized, placebo controlled clinical trials
of LMWH treatment are needed to understand the possible benefit of the
treatment. Additionally, this study was unable to control for the dose
and days of treatment of LMWH. Identifying the correct dose and timing
of LMWH is a matter of immediate interest. Of note, patients in the
heparin group received two types of LMWH, enoxaparin sodium or
nadroparin calcium, which have been reported to have differing
anticoagulant activity. The use of different LMWHs in the heparin group
warrants further explanation.</p>
<p>Another caveat of this study is that the levels of D-dimer and
fibrinogen degradation products were significantly higher at baseline
for patients in the heparin group compared to those in the control
group. Therefore, it is difficult to decipher whether some of the
positive effects of heparin treatment were due to its anti-coagulation
effects or direct anti-inflammatory effects. Future studies are that
delineate the anti-inflammatory functions of heparin independently of
its anticoagulant properties in cases of COVID-19 would be useful.</p>
<p>Lastly, this study did not discuss any side-effects of heparin, such as
the risk of bleeding. Moreover, coagulation can help to compartmentalize
pathogens and reduce their invasion, therefore anticoagulant treatments
like heparin may have risks and it remains to be determined which
patients would benefit from this therapy.</p>
<p><strong>References:</strong></p>
<p>Li J, Li Y, Yang B, Wang H, Li L. Low-molecular-weight heparin treatment
for acute lung injury/acute respiratory distress syndrome: a
meta-analysis of randomized controlled trials. Int J Clin Exp Med
2018;11(2):414-422</p>
<p>Wojnicz R, Nowak J, Szyguła-Jurkiewicz B, Wilczek K, Lekston A, Trzeciak
P, et al. Adjunctive therapy with low-molecular-weight heparin in
patients with chronic heart failure secondary to dilated cardiomyopathy:
oneyear follow-up results of the randomized trial. Am Heart J.
2006;152(4):713.e1-7</p>
<p><strong>Title</strong>: “Azithromycin and Ciprofloxacin have a chloroquine-like
effect on respiratory epithelial cells” Jens F. Poschet &amp; al, Posted:
2020-03-31</p>
<p><strong>Key words</strong>: Azithromycin (AZT) and ciprofloxacin (CPX),
hydroxychloroquine (HCQ), chloroquine (CQ), cystic fibrosis, lung
epithelial cells, pH, NFkB, IL-8, Furin, TGF-β</p>
<p><strong>Summary</strong>: The study assesses the effect of azithromycin (AZT) and its
potential mechanism of action in cystic fibrosis (CF) epithelial cells
beyond anti-bacterial activity and through changes in the pH of
intracellular organelles of lung epithelial cells. The authors also
discuss potential implications for COVID-19 therapy, as they compare AZT
to hydroxychloroquine (HCQ) and chloroquine (CQ), and further suggest
the use of ciprofloxacin (CPX) based on acido-tropic lipophilic weak
base properties of these drugs. The study examines AZT’s mechanism of
action in vitro on cystic fibrosis respiratory epithelial cells and
find:</p>
<ul>
<li><p>AZT elicits pH normalization in trans-Golgi network (TGN) and
endosomes from their subtly increased acidification in CF epithelial
cells.</p></li>
<li><p>AZT reduces inflammatory mediators in CF lung epithelial cells with
or without bacterial products challenge: basal NFκB activation was
reduced by 50% and by 40% in bacterial challenge; basal IL-8
secretion was reduced by almost 70 %, and by approximatively 33%
after bacterial product challenge.</p></li>
<li><p>AZT corrects Furin activity in CF cells’ intracellular organelles
and leads to a decrease in abundance of the profibrotic mediator
TGF-beta.</p></li>
</ul>
<p><strong>Limitations</strong>:</p>
<p>This study is conducted in-vitro on respiratory epithelial cells
affected by cystic fibrosis, and even though normal human epithelial
cells were included in some experiments, they were not treated with AZT
or CPX. The AZT results are derived from only 3 independent in-vitro
experiments. The number of experiments supporting the CPX is not clear,
nor why different cell types were used in AZT and CPX experiments. Very
little is shown, except for TGN pH normalization, to suggest replication
of AZT’s results with CPX.</p>
<p>The biological mechanism linking pH increase in the TGN and the endosome
and the decrease in pro-inflammatory and pro-fibrotic effect in CF
epithelial cell would need to be further explored.</p>
<p>The hypothesis of similar effect in pH shift in normal cells or in a
SARS-CoV2 affected cells would need to be substantiated by experimental
data.</p>
<p><strong>Findings implications</strong>: As we are desperately looking for a COVID-19
therapy, an hic and nunc approach identifying FDA-approved drugs with
known safety profiles, is appealing. The preliminary in-vitro findings
of this study warrant further in-vivo studies regarding these drugs.</p>
<p>Moreover, the biological mechanism underscored in this study could
provide insights in the link between SARS-Cov2 viral infection and
hyperinflammatory reaction as well as anti-viral effects. In particular,
the Furin activity reduction by AZT may result in hindering SARS-CoV2
entry, as its Spike protein possess a novel Furin cleavage site.</p>
<p><em>Review by Jaime Mateus-Tique as part of a project by students, postdocs
and faculty at the Immunology Institute of the Icahn school of medicine,
Mount Sinai.</em></p>
<p><strong>Title:</strong> Lectin-like Intestinal Defensin Inhibits 2019-nCoV Spike
binding to ACE2</p>
<p><strong>Immunology keywords:</strong> defensins, spike protein, intestinal Paneth
cells, ACE2 binding</p>
<p><strong>Main Findings:</strong></p>
<p>Human ACE2 was previously identified as the host receptor for
SARS-CoV-2. Despite ACE2 being expressed in both lung alveolar
epithelial cells and small intestine enterocytes, respiratory problems
are the most common symptom after viral infection while intestinal
symptoms are much less frequent. Thus, the authors here investigate the
biology behind the observed protection of the intestinal epithelium from
SARS-CoV-2. Human defensin 5 (HD5), produced by Paneth cells in the
small intestine, was shown to interact with human ACE2, with a binding
affinity of 39.3 nM by biolayer interferometry (BLI). A blocking
experiment using different doses of HD5 coating ACE2 showed that HD5
lowered viral spike protein S1 binding to ACE2. Further, a molecular
dynamic simulation demonstrated a strong intermolecular interaction
between HD5 and the ACE2 ligand binding domain. To test HD5 inhibitory
effect on S1 binding to ACE2, human intestinal epithelium Caco-2 cells
were preincubated with HD5. Preincubation strongly reduced adherence of
S1 to surface of cells. HD5 was effective at a concentration as low as
10 µg/mL, comparable to the concentration found in the intestinal fluid.</p>
<p><strong>Limitations:</strong></p>
<p>The study focuses exclusively on intestinal cells. However, HD5 could
have been tested to block ACE2-S1 binding in human lung epithelial cells
as a potential treatment strategy. It would be useful to know whether
HD5 could also prevent viral entry in lung cells.</p>
<p><strong>Relevance:</strong></p>
<p>This work provides the first understanding of the different efficiency
of viral entry and infection among ACE2-expressing cells and tissues.
Specifically, the authors show that human defensin 5 produced in the
small intestine is able to block binding between S1 and ACE2 necessary
for viral entry into cells. The study provides a plausible explanation
on why few patients show intestinal symptoms and suggests that patients
with intestinal disease that decrease defensins’ production may be more
susceptible to SARS-CoV-2. It also indicates that HD5 could be used as a
molecule to be exogenously administered to patients to prevent viral
infection in lung epithelial cells.</p>
<p>Highly pathogenic coronavirus N protein aggravates lung injury by MASP-2-mediated complement over-activation</p>
<p><em>Keywords:</em></p>
<p>Nucleocapsid protein, MASP-2, complement, pneumonia</p>
<p><em>Summary</em></p>
<p>In this manuscript, the authors describe direct complement system
activation through the SARS-CoV-2-N-protein as a common denominator of
coronavirus-induced lung injury.</p>
<p>The authors refer to previous evidence on SARS-CoV and MERS-CoV in which
they showed that the nucleocapsid (N-) protein binds to a variant of
MBL-associated serine protease-2 (MASP-2) that under physiologic
conditions initiates the lectin pathway of the complement cascade. The
authors hypothesized that SARS-CoV-2 activates the complement cascade in
severely ill patients through viral N-protein-mediated dimerization and
activating auto-cleavage of MASP-2 leading to respiratory failure in
some patients.</p>
<p>Following confirmation of MASP-2 binding to the N protein of SARS-CoV,
MERS-CoV and SARS-CoV2, the authors demonstrate the subsequent
complement activation in vitro as measured by C4b deposition in a
complement deposition assay and enhanced phagocyte activity. Translating
these findings into an in vivo model, the authors pre-infected mice with
an adenovirus expressing SARS CoV, MERS-CoV and control virus and
challenged the animals with LPS to activate the lectin pathway. This
effect was abolished when the N-protein was truncated or anti-MASP-2
antibodies or inhibitors were used. In mice infected with SARS- or
MERS-CoV, the authors found significantly higher C4b and C4 deposits in
the lungs and showed tremendously worse survival. However, when MASP-2
knockout mice were challenged with the above-mentioned viruses, survival
was substantially better as compared to virus-infected wildtype mice.</p>
<p>The authors next analyzed lung tissues from deceased COVID-19 patients
and found strong IHC stainings of complement components. In the final
step, the authors applied a clinically used C5-inhibitor to two
critically ill COVID-19 patients as a part of a clinical trial. Both
patients showed clinical improvement of their severe pneumoniae.</p>
<p><em>Critical analysis</em></p>
<p>The question if and in how far coronaviruses are capable of activating
the complement system has been controversial so far with conflicting
data published.</p>
<p>However, the authors performed a thorough and convincing analysis with
well-designed in vitro an in vivo experiments showing the ability of the
nucleocapsid to activate the complement cascade through the lectin
pathway.</p>
<p>However, the authors performed key experiments using adenoviruses only
containing SARS-CoV and MERS-CoV but not SARS-CoV-2. While the
assumption that due to the high sequence homology in the
N-protein-binding motif and the proven binding of the SARS-CoV-2
N-protein to MASP-2 strongly suggests that the N-protein in SARS-CoV-2
also plays a major role in complement activation in vivo remains to be
proven.</p>
<p><em>Implications of the findings in the context of current epidemics</em></p>
<p>Anti-inflammatory treatment in patients with severe viral pneumonia is
established as a clinical standard procedure. However, most
pathophysiologic aspects of immune hyperreactivity in these patients
remains unclear. The study reviewed here shows one mechanism by which
coronaviruses activate the complement cascade causing or at least
aggravating inflammation in the lung. Showing the successful treatment
of two patients with a complement C5 inhibitor should not be overrated
since these patients are enrolled in an ongoing clinical trial assessing
the significance of complement C5 inhibition in severely ill COVID-19
patients.</p>
<p><em>Additional resources used:</em></p>
<p>doi: 10.1038/emi.2015.28. Epub 2015 May 6.</p>
<p>doi: 10.1128/mBio.01753-18</p>
<p><strong>Title: </strong></p>
<p><strong>Susceptibility of ferrets, cats, dogs and different domestic animals
to SARS-coronavirus-2</strong></p>
<p><strong>Immunology keywords:</strong> SARS-CoV-2, ferret, cat, laboratory animal,
domestic animals</p>
<p><strong>The main finding of the article: </strong></p>
<p>This study evaluated the susceptibility of different model laboratory
animals (ferrets), as well as companion (cats and dogs), and domestic
animals (pigs, chickens and ducks) to SARS-CoV-2. They tested infection
with two SARS-CoV2 isolates, one from an environmental sample collected
in the Huanan Seafood Market in Wuhan (F13-E) and the other from a human
patient in Wuhan (CTan-H).</p>
<p><em>Ferrets</em> were inoculated with either of the two viruses by intranasal
route with 10<sup>5</sup> pfu, and the viral replication was evaluated. Two
ferrets from each group were euthanized on day 4 post infection (p.i.).
AT day 4 p.i., viral RNA and infectious viruses were detected only in
upper respiratory tract (nasal turbinate, upper palate, tonisls, but not
in the trachea, lungs or other tissues. Viral RNA and virus titer in the
remaining ferrets were monitored in nasal washes and rectal swabs on
days 2, 4, 6, 8 and 10 p.i. Viral RNA and infectious viruses were
detected in nasal washes until day 8 p.i. One ferret in each group
developed fever and loss of appetite on days 10 and 12 p.i., however,
viral RNA was practically undetactable. These two ferrets showed severe
lymphoplasmacytic perivasculitis and vasculitis in the lungs and lower
antibody titers compare to other 4 ferrets.</p>
<p><em>Cats</em>. Five <em>s</em>ubadult 8-month-old domestic cats were inoculated with
CTan-h virus and three uninfected cats were placed in a cage adjacent to
each of the infected cats to monitor respiratory droplet transmission.
Viral RNA was detected in the upper respiratory organs from all infected
cats and in one out of three exposed cats. All infected (inoculated and
exposed) cats developed elevated antibodies against SARS-CoV2. Viral
replication studies with juvenile cats (70-100 days) revealed massive
lesions in the nasal and tracheal mucosa epithelium and lungs of two
inoculated cats which died or were euthanized on day 3 p.i., and
infection in one out of three exposed cats. These results indicated
SARS-CoV2 could replicate in cats, that juvenile cats were more
susceptible that adults, and theat SARS-CoV2 could be transmit via
respiratory droplets between cats.</p>
<p><em>Dogs and others.</em> Five 3-month-old beagle dogs were inoculated and
housed with two uninoculated beagles in a room. Two virus inoculated
dogs seroconverted, but others including two contact dogs were all
seronegative for SARS-CoV2 and infectious virus was not detected in any
swabs collected. Viral RNA was not detected in swabs from pigs,
chickens, and ducks inoculated or contacted. These results indicated
that dogs, pigs, chickens, and ducks might have low or no susceptibility
to SARS-CoV2.</p>
<p><strong>Critical analysis of the study:</strong></p>
<p>This manuscript describes the viral replication and clinical symptoms of
SARS-CoV2 infection in ferrets, and the SARS-CoV2 infection and
transmission in cats. Clinical and pathological analysis was not
performed in cats, therefore the correlation of virus titer with
symptoms severity in the adult and juvenile cats could not be
determined.</p>
<p><strong>The importance and implications for the current epidemics:</strong></p>
<p>SARS-CoV-2 transmission to tigers, cats and dogs has been previously
reported. It should be noted that this manuscript did not evaluate the
transmission from cats to human. Nevertheless, it clearly showed higher
susceptibility of ferrets and domestic cats to SARS-CoV-2. This data
strongly indicates the need for surveillance of possible infection and
transmission of SARS-CoV-2 by domestic cats.</p>
<h2 data-number="9.188" id="prediction-of-sars-cov-2-epitopes-across-9360-hla-class-i-alleles"><span class="header-section-number">9.188</span> Prediction of SARS-CoV-2 epitopes across 9360 HLA class I alleles</h2>
<p>Campbell et al. <em>bioRxiv</em> <span class="citation" data-cites="1IPKmXjW">[<a href="#ref-1IPKmXjW" role="doc-biblioref">653</a>]</span></p>
<h3 data-number="9.188.1" id="keywords-187"><span class="header-section-number">9.188.1</span> Keywords</h3>
<ul>
<li><p>MHC class I</p></li>
<li><p>HLA</p></li>
<li><p>Haplotype</p></li>
<li><p>CD8+ T cells</p></li>
<li><p>CTL</p></li>
<li><p>SARS-CoV-2 peptidome</p></li>
</ul>
<h3 data-number="9.188.2" id="main-findings-181"><span class="header-section-number">9.188.2</span> Main Findings</h3>
<p>In this study, Campbell et al. investigate the immunogenic potential of
SARS-CoV-2-derived epitopes in the context of eliciting CD-8 T cell
responses. Using pVAC, a computational toolkit developed by the Parker
Institute for Cancer Immunotherapy to predict cancer neoantigens
presented by human leukocyte antigen (HLA) class I, they analyzed the
binding affinity of 9360 HLA-I alleles to 9-mer epitopes derived from
all 11 viral proteins spanning the SARS-CoV-2 proteome. They identified
6,748 epitope – HLA-I pairs, which consisted of 1,103 unique 9-mer
epitopes and 1,022 HLA class I alleles, with a predicted binding
affinity Kd &lt; 500 nM. Each viral protein was predicted to generate
immunogenic epitopes (12 to 684 epitopes, with a median of 31) and some
of these epitopes could bind to multiple HLA-I alleles (1 to 55 alleles,
with a median of 3). The study tested 2987 HLA-A, 3707 HLA-B and 2666
HLA-C alleles, of which majority of viral epitopes were predicted to
bind HLA-B alleles (295 HLA-A, 614 HLA-B and 113 HLA-C). Of note, 10 of
the 30 experimentally validated immunogenic SARS-CoV HLA-I epitopes were
identified among current SARS-CoV-2 predicted epitopes. Together, this
work shows that a diverse set of HLA alleles can potentially bind to a
wide variety of viral peptides, suggesting the presence of strong
cytotoxic T cell (CTL) responses against SARS-CoV-2 in COVID-19 patients
and serves as a valuable resource for SARS-CoV-2 epitope selection and
experimental validation efforts, which would guide the design of T-cell
based vaccination strategies against COVID-19.<span id="limitations" class="anchor"></span></p>
<h3 data-number="9.188.3" id="significance-172"><span class="header-section-number">9.188.3</span> Significance</h3>
<p>A limitation of this study is that it identifies only 9-mers peptides
with &lt;500 nM binding affinity, but it does not look for 8-mer or
10-mer peptides and excludes peptides with &gt;500 nM affinity but
within statistical rank-cutoff of 2%, which could still generate an
immune response to the virus. Another limitation is that the authors did
not evaluate epitope - HLA-II pairs, even though HLA class II is also
important in the context of viral immune responses. Studies have shown
that convalescent sera of recovered COVID-19 patients contain
neutralizing antibodies, which can be used as an early treatment for
recipient patients or as a prophylactic for at-risk or exposed patients.
Generation of these antibodies require viral detection by B cells, aided
by antigen presentation by HLA II molecules to CD4 T cells, which
stimulates B cells to differentiate into antibody-producing plasma
cells. Thus, further study could provide insight into the ability to
potentially initiate potent antibody responses by identifying longer
peptides and their affinity for class II variants.</p>
<h3 data-number="9.188.4" id="significance-173"><span class="header-section-number">9.188.4</span> Significance</h3>
<h3 data-number="9.188.5" id="this-study-predicts-6748-unique-pmhc-complexes-across-the-sars-cov-2-peptidome-and-global-variety-of-hla-alleles-highlighting-that-a-wide-variety-of-patients-can-generate-a-ctl-mediated-response-to-infection.-these-results-can-help-explain-the-heterogeneity-of-covid-19-disease-phenotypes-and-can-be-used-to-monitor-cd8-t-cell-responses-across-patients-with-many-different-haplotypes.-since-sars-cov-2-proteome-is-quite-large-to-be-screened-experimentally-for-immunogenic-epitopes-in-an-unbiased-fashion-these-in-silico-findings-guide-the-selection-of-immunogenic-epitopes-for-functional-validation-of-sars-cov-2-epitopes-which-can-be-used-in-vaccine-development-and-pre-clinical-treatment-studies.-moreover-all-data-generated-in-this-study-is-publicly-available-for-further-investigation-which-is-critical-for-clinical-translation-in-the-evolving-pandemic."><span class="header-section-number">9.188.5</span> This study predicts 6,748 unique pMHC complexes across the SARS-CoV-2 peptidome and global variety of HLA alleles, highlighting that a wide variety of patients can generate a CTL-mediated response to infection. These results can help explain the heterogeneity of COVID-19 disease phenotypes and can be used to monitor CD8+ T cell responses across patients with many different haplotypes. Since SARS-CoV-2 proteome is quite large to be screened experimentally for immunogenic epitopes in an unbiased fashion, these in silico findings guide the selection of immunogenic epitopes for functional validation of SARS-CoV-2 epitopes, which can be used in vaccine development and pre-clinical treatment studies. Moreover, all data generated in this study is publicly available for further investigation, which is critical for clinical translation in the evolving pandemic.</h3>
<h3 data-number="9.188.6" id="credit-182"><span class="header-section-number">9.188.6</span> Credit</h3>
<p><em>Reviewed </em>by Miriam Saffern* as part of a project by students, postdocs
and faculty at the Immunology Institute of the Icahn School of Medicine,
Mount Sinai.*</p>
<p><strong>Neutralizing antibody responses to SARS-CoV-2 in a COVID-19 recovered
patient cohort and their implications</strong></p>
<p>Fan Wu et al.;
medRxiv 2020.03.30.20047365; doi:https://doi.org/10.1101/2020.03.30.20047365</p>
<p><strong><em>Keywords</em></strong></p>
<ul>
<li><p>Neutralizing antibodies</p></li>
<li><p>SARS-CoV-2</p></li>
<li><p>pseudotype neutralization assay</p></li>
</ul>
<p><strong><em>Main findings </em></strong></p>
<p>In this study, plasma obtained from 175 convalescent patients with
laboratory-confirmed mild COVID-19 was screened for SARS-CoV-2-specific
neutralizing antibodies (nABs) by pseudotype-lentiviral-vector-based
neutralization assay as well as for binding antibodies (Abs) against
SARS-CoV-2 RBD, S1 and S2 proteins by ELISA. Kinetics of neutralizing
and binding Ab titers were assessed during the acute and convalescent
phase in the context of patient age as well as in relation to clinical
markers of inflammation (CRP and lymphocyte count at the time of
hospitalization). Across all age groups, SARS-CoV-2-specific nAbs titers
were low within the first 10 days of symptom onset, peaked between days
10-15, and persisted for at least two weeks post discharge. In contrast
to spike protein binding Abs, nAbs were not cross-reactive to
SARS-CoV-1. Moreover, nAb titers moderately correlated with the amount
of spike protein binding antibodies. Both neutralizing and binding Ab
titers varied across patient subsets of all ages, but were significantly
higher in middle-aged (40-59 yrs) and elderly (60-85) <em>vs</em>. younger
patients (15-39 yrs). However, plasma nAb titers were found to be below
detection level in 5.7% (10/175) of patients, i.e. a small number of
patients recovered without developing a robust nAb response. Conversely,
1.14% (2/175) of patients had substantially higher titers than the rest.
Notably, in addition to patient age, nAb titers correlated moderately
with serum CRP levels but were inversely related to lymphocyte count on
admission. In summary, the authors show that patients with clinically
mild COVID-19 disease mount a strong humoral response against the
SARS-CoV-2 spike protein. Compared to younger patients, middle-aged and
elderly patients had both higher neutralizing and binding Ab titers,
accompanied by increased CRP levels and lower lymphocyte counts. These
patients are usually considered at higher risk of severe disease.
Therefore, robust neutralizing and binding Ab responses may be
particularly important for recovery in this patient subset. Conversely,
patients who failed to produce high nAb/binding Ab titers against spike
protein did not progress to severe disease, indicating that binding Abs
against other viral epitopes as well as cellular immune responses are
equally important.</p>
<p><strong><em>Limitations </em></strong></p>
<p>This study provides valuable information on the kinetics of spike
protein-specific nAb as well as binding Ab titers in a cohort of
convalescent mild COVID-19 patients of all ages. However, similar
studies enrolling larger patient numbers, including those diagnosed with
moderate and severe disease as well as survivors and non-survivors,
especially in the elderly group (to rule out potential bias for more
favorable outcome), are warranted for reliable assumptions on the
potentially protective role of Abs and nAbs in COVID-19. Moreover,
longitudinal observation beyond the acute and convalescent phase in
addition to stringent clinical and immunological characterization is
urgently needed.</p>
<p>In their study, Wu et al. did not measure binding Abs against non-S
viral proteins, which are also induced in COVID-19 and therefore could
have added valuable diagnostic information with regard to patients who
seemingly failed to mount both binding and neutralizing Ab responses
against the SARS-CoV-2 spike protein. Likewise, while this study
excluded cross-reactivity of nAbs against SARS-CoV-1, no other
coronaviruses were tested. Of additional note, neutralizing activity of
plasma Abs was only assessed by pseudotype neutralization assay, not
against live SARS-CoV-2. Generally, while these are widely used and
reproducible assays, <em>in vitro</em> neutralization of pseudotyped viruses
does not necessarily translate to effective protection against the
respective live virus <em>in vivo</em> (<em>cf</em>. review by Burton, D. Antibodies,
viruses and vaccines. <em>Nat Rev Immunol</em> <strong>2, </strong>706–713 (2002)). Further
studies are therefore needed to assess the specificity and neutralizing
characteristics of these Abs to test whether they could be candidates
for prophylactic and therapeutic interventions. In this context, setting
arbitrary cut-off values (ID<sub>50</sub> &lt;500 <em>vs</em>. detection limit o ID<sub>50</sub>
&lt; 40) and thus classifying up to 30% of patients in this study as
“weak” responders does not take into account our currently limited
knowledge regarding protective capacity of these nAbs and should
therefore have been avoided by the authors.</p>
<p><strong><em>Significance</em></strong></p>
<p>This preprint is arguably the first report on neutralizing and binding
Ab titers in a larger cohort of mild COVID-19 patients. Assessing Ab
titers in these patients is not only important in order to confirm
whether mild COVID-19 infection elicits robust nAb responses, but also
adds further information regarding the use of plasma from mild disease
patients for convalescent plasma therapy as well as vaccine design in
general. Future studies will need to address now whether the nAb
responses generated in mild disease will be protective or (functionally)
different from nAbs generated in moderate and severe disease. The
findings in this study are therefore of great relevance and should be
further explored in ongoing research on potential coronavirus therapies
and prevention strategies.</p>
<p><em>This review was undertaken as part of a project by students, postdocs
and faculty at the Immunology Institute of the Icahn school of medicine,
Mount Sinai.</em> </p>
<p> </p>
<p>Title: Association of BCG vaccination policy with prevalence and
mortality of</p>
<p>COVID-19</p>
<p>Immunology Keywords</p>
<p>Bacillus Calmette–Guérin (BCG) Immunization, COVID-19 prevalence,
COVID-19 deaths</p>
<p>Main findings</p>
<p>Previously reported immunization programs using BCG vaccines have
demonstrated heterologous protection against other unrelated pathogens
that associated with lower mortality and morbidity risks [1].
Therefore this study investigated the possible correlation between
COVID-19 death cases or prevalence with BCG vaccination. The authors
used publicly available COVID-19 data from 136 countries as well as
vaccination demographics from the BCG World Atlas to perform a linear
regression modeling.</p>
<p>After correcting for life expectancy and the onset of the spread of the
virus (n=40), the analyses revealed a positive effect of current BCG
vaccination programs and controlling the number of COVID-19 cases and
deaths.</p>
<p>The amount of variance explained by BCG vaccination was 20% for number
of cases and significant for both groups of countries, the ones that
used to have a BCG immunization program in the past (b = 0.6122, p =
.0024) and the ones that never have it (b = 0.6511, p = .0326).</p>
<p>Only the group of countries that never vaccinated against BCG showed
significance in deaths/cases ratio but explains only 3.39% of the
observed variance.</p>
<p>The authors concluded that BCG immunization may provide protection
against COVID-19 probably due to the infection spread reduction. BCG
immunization doesn’t have a significant impact in the mortality induced
by COVID-19.</p>
<p>Limitations:</p>
<p>As acknowledged by the authors of this study, there are large number of
unexplained potential confounding variables such as BCG immunization
coverage, and onset of virus spread in different countries.</p>
<p>The authors cite that BCG immunization coverage could be variable among
countries, but they didn’t explore it. Further, vaccination coverage
changes at different rates over time across countries for different
reasons [2]. Additionally, the authors did not consider the variable
immunization coverage within countries, where unequal access to
healthcare is frequently observed [3, 4].</p>
<p>The authors do not adequately control for time of spread in infection
for each country [5].</p>
<p>The authors discuss the importance of validating experimentally the
results observed and claim that BCG vaccination could provide
non-specific protection against COVID-19. A stronger discussion of the
use of BCG vaccine would have included known considerations on efficacy
considering route of administration (intravenous, intradermal), vaccine
strains which are known to differ in the number of viable bacteria and
duration of protection.</p>
<p>Relevance:</p>
<p>This study presented preliminary data on possible non-specific
protection by BCG immunization on COVID-19 infection.</p>
<p>References</p>
<p>1. Aaby, P., T.R. Kollmann, and C.S. Benn, <em>Nonspecific effects of
neonatal and infant vaccination: public-health, immunological and
conceptual challenges.</em> Nat Immunol, 2014. <strong>15</strong>(10): p. 895-9.</p>
<p>2. Nuffieldtrust. <em>Vaccination coverage for children and mothers</em>. 2020
[cited 2020; Available from:
<a href="https://www.nuffieldtrust.org.uk/resource/vaccination-coverage-for-children-and-mothers-1?gclid=CjwKCAjw1cX0BRBmEiwAy9tKHhtZ6v3YmGA_N9MSd5IGM0rPHkb3vv4MViuWHpJjmBnKFJ0OAvNJtxoCCQAQAvD_BwE" class="uri">https://www.nuffieldtrust.org.uk/resource/vaccination-coverage-for-children-and-mothers-1?gclid=CjwKCAjw1cX0BRBmEiwAy9tKHhtZ6v3YmGA_N9MSd5IGM0rPHkb3vv4MViuWHpJjmBnKFJ0OAvNJtxoCCQAQAvD_BwE</a>.</p>
<p>3. WHO. <em>10 facts on health inequities and their causes</em>. 2017;
Available from:
<a href="https://www.who.int/features/factfiles/health_inequities/en/" class="uri">https://www.who.int/features/factfiles/health_inequities/en/</a>.</p>
<p>4. Balance. <em>Health Care Inequality in America</em>. 2020; Available from:
<a href="https://www.thebalance.com/health-care-inequality-facts-types-effect-solution-4174842" class="uri">https://www.thebalance.com/health-care-inequality-facts-types-effect-solution-4174842</a>.</p>
<p>5. Statista. <em>Rate of coronavirus (COVID-19) tests performed in select
countries worldwide as of April 8, 2020 (per thousand population)*</em>.
2020; Available from:
<a href="https://www.statista.com/statistics/1104645/covid19-testing-rate-select-countries-worldwide/" class="uri">https://www.statista.com/statistics/1104645/covid19-testing-rate-select-countries-worldwide/</a>.</p>
<p>Review by Alessandra Soares Schanoski as part of a project by students,
postdocs and faculty at the Immunology Institute of the Icahn school of
medicine, Mount Sinai.</p>
<p><strong>Title</strong>: Fatal toxicity of chloroquine or hydroxychloroquine with
metformin in mice</p>
<p><strong>Summary</strong>: Chloroquine (CQ) and hydroxychloroquine (HCQ) have been
presented as potential therapeutic strategies for acute respiratory
syndrome, a serious complication sometimes observed in COVID-19
patients. The drug was rapidly pushed to clinical testing and approved
within weeks as a recommended COVID-19 treatment by the National Health
Commission of the People’s Republic of China[1]. However, CQ or HCQ,
sometimes used in combination with anti-diabetic drug metformin, may
have serious toxic side-effects that have not been appropriately
addressed.</p>
<p>The authors, who study metformin with CQ or HCQ as potential anti-cancer
drugs in mice, study their toxicity. They found that the combination was
lethal in at least 40% of nude mice, tumour-bearing or not. Similar
results were obtained in non-immunocompromised mice. They observed that
nude mice treated with HCQ+metformin presented an increased number of
autophagosomes in the heart, liver and kidneys. Lactate dehydrogenase
(LDH) and creatine kinase (CK) were also elevated in treatment groups
compared to control vehicle-treated mice.</p>
<p><strong>Critical analysis</strong>: The authors of this article perform studies
exclusively in mice. They note that the lethal doses of CQ or HCQ are
similar to those in humans with allometric scaling. However, the
side-effects of the combinations observed in mice must yet be confirmed
in humans. The article acts as a word of caution for individuals and
physicians who have been taking and prescribing these drug combinations
prior to adequate clinical trials.</p>
<p><strong>Relevance to current epidemic</strong>: The authors note that both CQ and HCQ
can lead to the development of hypoglycemia, cardiomyopathy and
retinopathy. The dose of 500mg twice per day of CQ recommended in China
can rapidly approach a dangerous dose in humans[1]. CQ poisoning has
already been reported in Nigeria and resulted in at least one death in
Arizona; the clinical relevance of treating COVID-19 patients with
CQ+metformin or HCQ+metformin is yet to be confirmed or rejected.</p>
<p><strong>References</strong>:</p>
<p>1. Wong et al. Caution and clarity required in the use of chloroquine
for COVID-19, <em>The Lancet,</em> April 02 2020</p>
<p><strong>Title</strong>: Fatal toxicity of chloroquine or hydroxychloroquine with
metformin in mice</p>
<p><strong>Summary</strong>: Chloroquine (CQ) and hydroxychloroquine (HCQ) have been
presented as potential therapeutic strategies for acute respiratory
syndrome, a serious complication sometimes observed in COVID-19
patients. The drug was rapidly pushed to clinical testing and approved
within weeks as a recommended COVID-19 treatment by the National Health
Commission of the People’s Republic of China[1]. However, CQ or HCQ,
sometimes used in combination with anti-diabetic drug metformin, may
have serious toxic side-effects that have not been appropriately
addressed.</p>
<p>The authors, who study metformin with CQ or HCQ as potential anti-cancer
drugs in mice, study their toxicity. They found that the combination was
lethal in at least 40% of nude mice, tumour-bearing or not. Similar
results were obtained in non-immunocompromised mice. They observed that
nude mice treated with HCQ+metformin presented an increased number of
autophagosomes in the heart, liver and kidneys. Lactate dehydrogenase
(LDH) and creatine kinase (CK) were also elevated in treatment groups
compared to control vehicle-treated mice.</p>
<p><strong>Critical analysis</strong>: The authors of this article perform studies
exclusively in mice. They note that the lethal doses of CQ or HCQ are
similar to those in humans with allometric scaling. However, the
side-effects of the combinations observed in mice must yet be confirmed
in humans. The article acts as a word of caution for individuals and
physicians who have been taking and prescribing these drug combinations
prior to adequate clinical trials.</p>
<p><strong>Relevance to current epidemic</strong>: The authors note that both CQ and HCQ
can lead to the development of hypoglycemia, cardiomyopathy and
retinopathy. The dose of 500mg twice per day of CQ recommended in China
can rapidly approach a dangerous dose in humans[1]. CQ poisoning has
already been reported in Nigeria and resulted in at least one death in
Arizona; the clinical relevance of treating COVID-19 patients with
CQ+metformin or HCQ+metformin is yet to be confirmed or rejected.</p>
<p><strong>References</strong>:</p>
<p>1. Wong et al. Caution and clarity required in the use of chloroquine
for COVID-19, <em>The Lancet,</em> April 02 2020</p>
<h2 data-number="9.189" id="virus-host-interactome-and-proteomic-survey-of-pmbcs-from-covid-19-patients-reveal-potential-virulence-factors-influencing-sars-cov-2-pathogenesis-1"><span class="header-section-number">9.189</span> Virus-host interactome and proteomic survey of PMBCs from COVID-19 patients reveal potential virulence factors influencing SARS-CoV-2 pathogenesis</h2>
<p>Li et al. <em>bioRxiv</em>.
<span class="citation" data-cites="NLoN4aYj">[<a href="#ref-NLoN4aYj" role="doc-biblioref">182</a>]</span></p>
<h3 data-number="9.189.1" id="keywords-188"><span class="header-section-number">9.189.1</span> Keywords</h3>
<ul>
<li>PBMC</li>
<li>virulence factors
– interaction network
– nsp9</li>
<li>nsp10
– NKRF</li>
</ul>
<h3 data-number="9.189.2" id="main-findings-182"><span class="header-section-number">9.189.2</span> Main findings</h3>
<p>The authors identified <strong>intra-viral protein-protein interactions</strong>
(PPI) with two different approaches: genome wide yeast-two hybrid (Y2H)
and co-immunoprecipitation (co-IP). A total of 58 distinct PPI were
characterized. A screen of <strong>viral-host PPI</strong> was also established by
overexpressing all the SARS-CoV-2 genes with a Flag epitope into HEK293
cells and purifying each protein complex. Interacting host proteins were
then identified by liquid chromatography and tandem mass spectrometry.
251 cellular proteins were identified, such as subunits of ATPase, 40S
ribosomal proteins, T complex proteins and proteasome related proteins,
for a total of 631 viral-host PPI. Several interactions suggesting
protein-mediated modulation of the immune response were identified,
highlighting the multiple ways SARS-CoV-2 might reprogram infected
cells.</p>
<p>Subsequently, the authors compared global proteome profiles of PBMCs
from healthy donors (n=6) with PBMC from COVID-19 patients with mild
(n=22) or severe (n=13) symptoms. 220 proteins were found to be
differentially expressed between <em>healthy donors and mild COVID-19
patients</em>, and a pathway analysis showed <strong>a general activation of the
innate immune response.</strong> 553 proteins were differentially expressed
between the PBMC of <em>mild and severe COVID-19 patients</em>, most of them
(95%) being downregulated in severe patients<em>.</em> Functional pathway
analysis indicated a defect of T cell activation and function in severe
COVID-19. There was also evidence suggesting reduced antibody secretion
by B cells. Together, these results suggest a <strong>functional decline of
adaptive immunity</strong>. A FACS analysis of PBMC from severe patients
indicated higher levels of IL6 and IL8 but not IL17 compared to mild
patients.</p>
<p>Finally, the authors focused on NKRF, an endogenous repressor of IL8/IL6
synthesis that was previously identified as interacting with SARS-Cov-2
nsp9,10,12,13 and 15. Individually expressed nsp9 and nsp10 (but not
nsp12, nsp13, nsp15) induced both IL6 and IL8 in lung epithelial A459
cells, indicating that nsp9 and nsp10 may be directly involved in the
induction of these pro-inflammatory cytokines. The authors finally argue
that nsp9 and nsp10 represent potential drug targets to prevent
over-production of IL6 and IL8 in infected cells, and reducing the
over-activation of neutrophils.</p>
<h3 data-number="9.189.3" id="limitations-185"><span class="header-section-number">9.189.3</span> Limitations</h3>
<p>First, the authors seem to have forgotten to include the extended data
in the manuscript, and their proteomic data does not seem to be publicly
available for the moment, which limits greatly our analysis of their
results.</p>
<p>While this work provides important data on host and viral PPI, only 19
interactions were identified by Y2H system but 52 with co-IP. The
authors do not comment about what could lead to such differences between
the two techniques and they don’t specify whether they detected the same
interactions using the two techniques.</p>
<p>Moreover, the PBMC protein quantification was performed comparing bulk
PBMC. Consequently, protein differences likely reflect differences in
cell populations rather than cell-intrinsic differences in protein
expression. While this analysis is still interesting, a similar
experiment performed on pre-sorted specific cell populations would allow
measuring proteome dynamics at a higher resolution.</p>
<p>Finally, the authors did not discussed their results in regards to
another SARS-CoV-2 interactome of host-viral PPI that had been published
previously<sup>1</sup>. This study reported 332 host-virus PPI, but no
interaction of viral proteins with NKRF was found. Some interactions
were found in both studies (eg. N and G3BP1, Orf6 and RAE1). However,
the time point used to lyse the cells were different (40h previously vs
72h here), which could explain some of the differences.</p>
<h3 data-number="9.189.4" id="relevance-1"><span class="header-section-number">9.189.4</span> Relevance</h3>
<p>The identification of many interactions between intra-viral and
host-virus PPI provides an overview of host protein and pathways that
are modulated by SARS-CoV-2, which can lead to the identification of
potential targets for drug development.</p>
<p>In the model proposed by the authors, nsp9 and nsp10 from SARS-Cov-2
induce an over-expression of IL6 and IL8 by lung epithelial cells, which
recruits neutrophils and could lead to an excess in lung infiltration.
Nsp9 has been shown to be essential for viral replication for
SARS-Cov-1<sup>2</sup>, and shares a 97% homology with nsp9 from SARS-Cov-2<sup>3</sup>.
Further, nsp9 crystal structure was recently solved<sup>3</sup>, which can help
to develop drug inhibitors if this protein is further confirmed as being
important for the virulence of SARS-Cov-2.</p>
<p>1. Gordon DE, Jang GM, Bouhaddou M, et al. A SARS-CoV-2-Human
Protein-Protein Interaction Map Reveals Drug Targets and Potential
Drug-Repurposing. <em>bioRxiv</em>. March 2020:2020.03.22.002386.
doi:10.1101/2020.03.22.002386</p>
<p>2. Miknis ZJ, Donaldson EF, Umland TC, Rimmer RA, Baric RS, Schultz LW.
Severe acute respiratory syndrome coronavirus nsp9 dimerization is
essential for efficient viral growth. <em>J Virol</em>. 2009;83(7):3007-3018.
doi:10.1128/JVI.01505-08</p>
<p>3. Littler DR, Gully BS, Colson RN, Rossjohn J. <em>Crystal Structure of
the SARS-CoV-2 Non-Structural Protein 9, Nsp9</em>. Molecular Biology; 2020.
doi:10.1101/2020.03.28.013920</p>
<p><strong>Structural basis to design multi-epitope vaccines against Novel
Coranvirus 19 (COVID19) infection, the ongoing pandemic emergency: an in
silico approach</strong></p>
<p>S Srivastava et. al. 2020
https://www.biorxiv.org/content/10.1101/2020.04.01.019299v1.full.pdf</p>
<p><strong>Keywords: </strong></p>
<ul>
<li><p>In silico</p></li>
<li><p>Epitope prediction</p></li>
<li><p>Immunoinformatics</p></li>
</ul>
<p><strong>Main Findings:</strong></p>
<p>This study reports the identification of in-silico screened epitopes
capable of binding MHCI (CTL), MHCII (HTL), and B cells with high
immunogenicity that can be formulated with Ochocerca volvulus
activation-associated secreted protein-1 (Ov-ASP-1) adjuvant into two
multi-epitope vaccines (MEVs) for SARS-CoV-2. CTL, HTL, and B cell
linear epitopes were identified, scored, and percentile-ranked utilizing
respective IEDB server tools. SARS-CoV-2 polyprotein, surface (S)
glycoprotein, envelope (E) protein, membrane (M) protein, nucleocapsid
(N) protein, and several open reading frame proteins were screened in
silico for potential CTL, HTL, and B cell epitopes. CTL epitopes were
identified by the “MHC-I Binding Predictions” IEDB tool with default
parameters of 1<sup>st</sup>, 2<sup>nd</sup>, and C-term amino acids; epitopes were ranked
by total score combining proteasomal cleavage, TAP transport, and MHC
scores combined. HTL epitopes were identified by the “MHC-II Binding
Predictions” IEDB tool, which gives a percentile rank by combining 3
methods (viz. combinatorial library, SMM_align &amp; Sturniolo, score
comparison with random five million 15-mer peptides within SWISSPROT). B
cell linear epitopes were identified by the “B Cell epitope Prediction”
IEDB tool, which searches continuous epitopes based on propensity scales
for each amino acid.</p>
<p>From these proteins, 38 CTL top percentile ranked epitopes, 42 HTL top
scorers, and 12 B cell top scorers were used for further analysis.
Candidates were then analyzed for epitope conservation analysis (number
of protein sequences containing that particular epitope), toxicity,
population coverage, and overlap with one another. 9 epitopes that
overlapped among all three types (CTL, HTL, and B cell linear) were then
analyzed for interaction with HLA binders, showing stable binding with
A*11:01, A*31:01, B3*01:01, and B1*09:01, and TAP, demonstrating
ability to pass from cytoplasm into the ER. Two MEVs were formulated
using the top CTL and HTL epitopes, which were then analyzed for
physicochemical properties, allergenicity, and potential to induce
IFN-gamma production. Final 3D modeling, refinement, and discontinuous B
cell epitope analysis were completed to optimize the space-occupancy of
the MEVs. This rendering was used to assess docking with TLR3, the major
domain used by Ov-ASP-1. Codon adaptation optimization yielded cDNA
capable of high expression in mammalian host cells. Taken together, this
in-silico study produced two MEVs containing CTL, HTL, and B cell
epitopes capable of eliciting potent cell-mediated and humoral responses
for HLA types representing up to 96% (SD 31.17) of the population.
Further in vitro study is warranted to confirm its clinical potential.</p>
<p><strong>Limitations:</strong></p>
<p>In silico approaches are based upon models, however accurate, that make
certain assumptions and contain biases inherent to training data.
Synthesizing and testing a few candidates alongside their initial
findings would make this method far more robust. It remains to be seen
the efficacy of screened epitopes and corresponding multi-epitope
formulations function in vitro and in vivo models.</p>
<p><strong>Significance:</strong></p>
<p>This study reports an in silico approach to producing multi-epitope
vaccines that can produce potent adaptive immune responses. Utilizing
protein databases, established protein modeling, folding, and docking
algorithms, as well as population analysis, the team identifies 38
MHCI-binding, 42 MHCII-binding, and 12 B cell epitopes that can be
linked with Ov-ASP-1 adjuvant to form stable proteins. These proteins
are shown to dock well with HLA-alleles, TAP, TLR3, and to induce
IFN-gamma responses.</p>
<p><em>Review by Matthew Lin as part of a project by students, postdocs and
faculty at the Immunology Institute of the Icahn School of Medicine,
Mount Sinai.</em></p>
<p>Notes</p>
<p>11 Protein sequences</p>
<p>CTL, HTL, B cell epitopes</p>
<ul>
<li><p>MHCI (IC50s, percentile ranked; immunogenicity) 38 shortlisted</p></li>
<li><p>MHCII (affinity, percentile ranked) 42 shortlisted</p></li>
<li><p>Population coverage (score is % of population express HLA to which
epitope binds)</p></li>
<li><p>B cell epitope (hydrophilicity, flexibility, accessibility, turns,
exposed surface, polarity, antigenic propensity correlated with
location in protein score) 12 shortlisted</p></li>
</ul>
<p>Overlapping (partial and complete)</p>
<ul>
<li><p>Conservation analysis (number of protein sequences containing
particular epitope)</p></li>
<li><p>Toxicity (highly toxic vs non-toxic short peptides algorithm derived
from datasets from Swiss-Prot)</p></li>
<li><p><strong>Overlap of all three (multiple sequence alignment) 9 shortlisted
(2 CTL, 4 HTL, 3 B cell)</strong></p></li>
</ul>
<p><em>ONLY PARTIAL OR COMPLETE OVERLAP EPITOPES ARE NOW SCREENED BELOW</em></p>
<p>CTL/HTL HLA allele binders molecular docking</p>
<ul>
<li><p>Tertiary structure modeling (HLA structures based off IPD-IMGT/HLA;
de novo structure prediction for epitopes)</p></li>
<li><p>Molecular docking (PatchDock) for respective HLA binders</p></li>
</ul>
<p>CTL TAP interaction molecular docking</p>
<p><em>NOW GOING BACK TO USING ALL THE TOP HITS NOT JUST THE OVERLAPPING
SEQUENCES?</em></p>
<p>MEV – 38 CTL and 42 HTL then used to design MEV with EAAK and GGGS
linkers, with adjuvant at N-term (Fig 2)</p>
<ul>
<li><p>ProtParam used for physicochemical property analysis of designed MEV
(table 5)</p></li>
<li><p>IFNg screen (will the epitope induce IFNg?) 20 CTL and 20 HTL
shortlisted</p></li>
<li><p>Allergenicity (AlgPred analysis) both CTL and HTL MEVs
non-allergenic</p></li>
<li><p>Physiochemical analysis</p></li>
</ul>
<p>3D refinement</p>
<ul>
<li><p>Validation (RAMPAGE to show that residues are optimized for favored
confirmation and minimized for outlier regions)</p></li>
<li><p>Linear and Discontinuous B cell epitope prediction (from refined
structure) CTL MEV has 17 linear and 2 discontinuous; HTL MEV has 17
and 4 respectively</p></li>
</ul>
<p>TLR3 interaction molecular docking with ectodomain of TLR3 (PatchDock)
stable complex formation tendency for both MEVs</p>
<p>cDNA analysis of sequence for cloning and expression</p>
<ul>
<li>Codon adaptation tool cDNA predicted to have high expression in
mammalian host cell line</li>
</ul>
<p><strong>Title</strong>: <span id="page-title" class="anchor"></span>Level of IL-6 predicts respiratory
failure in hospitalized symptomatic COVID-19 patients</p>
<p><strong>Summary</strong>: Hospitals around the world being overwhelmed due the
COVID-19 pandemic calls for determination of predictive markers which
would accurately predict patient outcome at an early stage. Severe COVID
complications often include acute respiratory distress syndrome.</p>
<p>The authors analyse clinical and laboratory findings in 40 COVID
patients. 13 required mechanical ventilation, and 27 did not. Age,
comorbidities, radiological findings, respiratory rate or qSofa score
(mortality prediction score upon sepsis) did not discriminate between
both groups. However, all patients requiring intubation were male, and
the pulse as well as several laboratory parameters correlated with the
risk of respiratory failure. The strongest correlation was found to be
with serological IL-6. Elevated IL-6 was found to increase the risk of
respiratory failure by a factor of 22.</p>
<p><strong>Critical analysis</strong>: The value of 80pg/mL of IL-6 is identified as
decisive, as 92% of patients who reached this cut-off required
intubation 0-4 days afterwards. The study must be extended to a larger
cohort, however, to perhaps identify a more precise cut-off.</p>
<p>Moreover, the causality between IL-6 and respiratory distress remains
unclear. Understanding the mechanisms behind these correlations could
pinpoint new therapies to prevent respiratory failure.</p>
<p><strong>Relevance to current epidemic</strong>: Foreseeing respiratory distress at an
early stage in admitted COVID patients could allow hospitals to better
manage their resources, both human and material.</p>
<p><strong>Title</strong>: <span id="page-title" class="anchor"></span>Level of IL-6 predicts respiratory
failure in hospitalized symptomatic COVID-19 patients</p>
<p><strong>Summary</strong>: Hospitals around the world being overwhelmed due the
COVID-19 pandemic calls for determination of predictive markers which
would accurately predict patient outcome at an early stage. Severe COVID
complications often include acute respiratory distress syndrome.</p>
<p>The authors analyse clinical and laboratory findings in 40 COVID
patients. 13 required mechanical ventilation, and 27 did not. Age,
comorbidities, radiological findings, respiratory rate or qSofa score
(mortality prediction score upon sepsis) did not discriminate between
both groups. However, all patients requiring intubation were male, and
the pulse as well as several laboratory parameters correlated with the
risk of respiratory failure. The strongest correlation was found to be
with serological IL-6. Elevated IL-6 was found to increase the risk of
respiratory failure by a factor of 22.</p>
<p><strong>Critical analysis</strong>: The value of 80pg/mL of IL-6 is identified as
decisive, as 92% of patients who reached this cut-off required
intubation 0-4 days afterwards. The study must be extended to a larger
cohort, however, to perhaps identify a more precise cut-off.</p>
<p>Moreover, the causality between IL-6 and respiratory distress remains
unclear. Understanding the mechanisms behind these correlations could
pinpoint new therapies to prevent respiratory failure.</p>
<p><strong>Relevance to current epidemic</strong>: Foreseeing respiratory distress at an
early stage in admitted COVID patients could allow hospitals to better
manage their resources, both human and material.</p>
<h2 data-number="9.190" id="reduction-of-lymphocyte-at-early-stage-elevates-severity-and-death-risk"><span class="header-section-number">9.190</span> Reduction of lymphocyte at early stage elevates severity and death risk</h2>
<p>of COVID-19 patients: a hospital-based case-cohort study</p>
<h3 data-number="9.190.1" id="keywords-189"><span class="header-section-number">9.190.1</span> Keywords</h3>
<ul>
<li>COVID-19</li>
<li>lymphocyte</li>
<li>severity</li>
<li>organ injuries</li>
</ul>
<h3 data-number="9.190.2" id="main-findings-183"><span class="header-section-number">9.190.2</span> Main findings</h3>
<p>The aim of this study was to assess an association
between reduced blood lymphocyte counts at hospital admission and
prognosis of COVID-19 patients (n=192). The authors found:</p>
<ul>
<li><p>Patients with lymphopenia are more likely to progress to severe
disease or succumb to COVID-19 (32.1% of COVID-19 patients with
lymphocyte reduction died).</p></li>
<li><p>Reduction of lymphocytes mainly affects the elderly (&gt; 70 years
old).</p></li>
<li><p>Lymphocyte reduction is more prevalent in COVID-19 patients with
cardiac disease and pulmonary disease, patients with increase in the
chest CT score (key marker of lung injury) and a decrease in several
respiratory function markers (PaCO2, SpO2, oxygenation index).</p></li>
</ul>
<h3 data-number="9.190.3" id="limitations-of-the-study"><span class="header-section-number">9.190.3</span> Limitations of the study</h3>
<p>Reduced blood lymphocyte counts with aging
have been known
(<a href="https://www.medrxiv.org/content/10.1101/2020.03.08.20031229v2"><em>https://www.medrxiv.org/content/10.1101/2020.03.08.20031229v2</em></a>)
<a href="https://onlinelibrary.wiley.com/doi/epdf/10.1111/sji.12413"><em>https://onlinelibrary.wiley.com/doi/epdf/10.1111/sji.12413</em></a>).
Therefore, it is not unexpected that a larger fraction of COV ID-19
patients above 70 years old have lower lymphocytes counts. Since age has
been reported to be a major factor that determines outcome for COVID-19,
lymphocyte counts and prognosis should have been adjusted by age.
Multivariate analysis to identify independent risk factors is lacking.</p>
<h3 data-number="9.190.4" id="relevance-2"><span class="header-section-number">9.190.4</span> Relevance</h3>
<p>Previous studies demonstrated that SARS-CoV-2 infection
leads to a decrease of the T cell count. This study confirms these
results and shows that lymphocyte reduction mainly affects the elderly.
Lymphopenia was associated with disease severity as well as worse
prognosis. Future studies need to address if lymphopenia is a negative
predictive factor independent from age.</p>
<p>Review by Meriem Belabed as part of a project by students, postdocs and
faculty at the Immunology Institute of the Icahn school of medicine,
Mount Sinai.</p>
<p><strong><em>Title:</em></strong> Prediction and Evolution of B Cell Epitopes of Surface
Protein in SARS-CoV-2</p>
<p><strong><em>Keywords:</em></strong> SARS-CoV-2; Epitopes; Bioinformatics; Evolution</p>
<p><strong><em>Summary/Main findings: </em></strong></p>
<p>Lon et al. used a bioinformatic analysis of the published SARS-CoV-2
genomes in order to identify conserved linear and conformational B cell
epitopes found on the spike (S), envelope (E), and membrane (M)
proteins. The characterization of the surface proteins in this study
began with an assessment of the peptide sequences in order to identify
hydrophilicity indices and protein instability indices using the
Port-Param tool in ExPASy. All three surface proteins were calculated to
have an instability score under 40 indicating that they were stable.
Linear epitopes were identified on the basis of surface probability and
antigenicity, excluding regions of glycosylation. Using BepiPred 2.0
(with a cutoff value of 0.35) and ABCpred (with a cutoff value of 0.51),
4 linear B cell epitopes were predicted for the S protein, 1 epitope for
the E protein, and 1 epitope for the M protein. For structural analysis,
SARS-CoV assemblies published in the Protein Data Bank (PDB) acting as
scaffolds for the SARS-CoV-2 S and E amino acid sequences were used for
input into the SWISS-MODEL server in order to generate three-dimensional
structural models for the assessment of conformational epitopes. Using
Ellipro (cutoff value of 0.063) and SEPPA (cutoff value of 0.5), 1
conformational epitope was identified for the S protein and 1 epitope
was identified for the E protein, both of which are accessible on the
surface of the virus. Finally, the Consurf Server was used to assess the
conservation of these epitopes. All epitopes were conserved across the
published SARS-CoV-2 genomes and one epitope of the spike protein was
predicted to be the most stable across coronavirus phylogeny.</p>
<p><strong><em>Critical Analysis/Limitations:</em></strong></p>
<p>While this study provides a preliminary identification of potential
linear and conformational B cell epitopes, the translational value of
the epitopes described still needs extensive experimental validation to
ascertain whether these elicit a humoral immune response. The
conformational epitope analyses are also limited by the fact that they
are based off of predicted 3D structure from homology comparisons and
not direct crystal structures of the proteins themselves. Additionally,
since there was not a published M protein with a high homology to
SARS-CoV-2, no conformational epitopes were assessed for this protein.
Finally, while evolutionary conservation is an important consideration
in understanding the biology of the virus, conservation does not
necessarily imply that these sites neutralize the virus or aid in
non-neutralizing <em>in vivo</em> protection.</p>
<p><strong><em>Relevance/Implications:</em></strong></p>
<p>With further experimental validation that confirms that these epitopes
induce effective antibody responses to the virus, the epitopes described
can be used for the development of treatments and vaccines as well as
better characterize the viral structure to more deeply understand
pathogenesis.</p>
<p><em>Review of paper titled:</em></p>
<p><strong>Atazanavir inhibits SARS-CoV-2 replication and pro-inflammatory
cytokine production</strong></p>
<p>N Fintelman-Rodrigues et. al. 2020
<a href="https://www.biorxiv.org/content/10.1101/2020.04.04.020925v1"><em>https://www.biorxiv.org/content/10.1101/2020.04.04.020925v1</em></a></p>
<p><strong>Keywords: </strong></p>
<ul>
<li><p>Repurpose drug</p></li>
<li><p>Major protease inhibitor</p></li>
<li><p>Antiretroviral</p></li>
</ul>
<p><strong>Summary:</strong></p>
<p>Major protease (Mpro) is a targetable viral protease important in
processing polyprotein during the coronavirae replication phase.
Atazanavir (ATV), an FDA-approved HIV protease inhibitor with fewer side
effects than liponavir (currently in COVID-19 clinical trial:
NCT04307693), can reach lung tissue through intravenous administration
and suggests functional improvement in pulmonary fibrosis patients. In
this study, in vitro assays with VeroE6 and A549 cell lines demonstrate
ATV’s binding specificity for and inhibitory action on Mpro. Elevated
lactate dehydrogenase (LDH) levels have previously been shown to
correlate with COVID-19 mortality, and measured levels were reduced in
ATV-treated cells infected with SARS-CoV-2. In addition, co-treatment
with ATV and ritonavir prevented induction of IL-6 and TNF-a in
monocytes, which was as good or better than chloroquine treatment. These
data suggest it may be possible to repurpose ATV for SARS-CoV-2.</p>
<p><strong>Limitations:</strong></p>
<p>The study was performed entirely <em>in vitro</em> in VeroE6 and A549 cells.</p>
<p>Comparisons were done against chloroquine, a relatively toxic compound
yet to demonstrate efficacy during in vivo testing<sup>1–3</sup>. Molecular
docking stimulations are limited by previous data and training sets;
while unlikely, there is a possibility that the binding properties
modeled are not reflective of the true interactions.</p>
<p><strong>Importance of findings:</strong></p>
<p>This study identifies atazanavir, a HIV protease inhibitor, as an
alternative to liponavir/ritonavir to treat SARS-CoV-2 infection. A
safer toxicity profile, localization to lung tissue, and demonstrated in
vitro efficacy to reduce cell mortality markers and inflammatory markers
in monocytes make the case that ATV is a potential candidate to prevent
SARS-CoV-2 viral replication.</p>
<p>References</p>
<p>1. Vigerust, D. J. &amp; McCullers, J. A. Chloroquine is effective against
influenza A virus in vitro but not in vivo. <em>Influenza Other Respi.
Viruses</em> <strong>1</strong>, 189–192 (2007).</p>
<p>2. Dowall, S. D. <em>et al.</em> Chloroquine inhibited ebola virus replication
in vitro but failed to protect against infection and disease in the in
vivo guinea pig model. <em>J. Gen. Virol.</em> <strong>96</strong>, 3484–3492 (2015).</p>
<p>3. Wang, J., Yu, L. &amp; Li, K. Benefits and Risks of Chloroquine and
Hydroxychloroquine in The Treatment of Viral Diseases: A Meta-Analysis
of Placebo Randomized Controlled Trials. <em>medRxiv</em> 2020.04.13.20064295
(2020). doi:10.1101/2020.04.13.20064295</p>
<p><em>Review by Matthew Lin as part of a project by students, postdocs and
faculty at the Immunology Institute of the Icahn School of Medicine,
Mount Sinai.</em></p>
<h3 data-number="9.190.5" id="title-on-the-interactions-of-the-receptor-binding-domain-of-sars-cov-1-and-sars-cov-2-spike-proteins-with-monoclonal-antibodies-and-the-receptor-ace2"><span class="header-section-number">9.190.5</span> Title: <strong>On the interactions of the receptor-binding domain of SARS-CoV-1 and SARS-CoV-2 spike proteins with monoclonal antibodies and the receptor ACE2</strong></h3>
<p>Correa Giron et al., <em>bioRxiv</em> [<span class="citation" data-cites="doi">[<span class="citeproc-not-found" data-reference-id="doi"><strong>???</strong></span>]</span>: []{#_Hlk37590784
.anchor}10.1101/2020.04.05.026377]</p>
<h3 data-number="9.190.6" id="keywords-190"><span class="header-section-number">9.190.6</span> Keywords</h3>
<ul>
<li><p>Monoclonal antibodies</p></li>
<li><p>Binding affinities</p></li>
<li><p>RBD/ACE2 interaction</p></li>
</ul>
<h3 data-number="9.190.7" id="main-findings-184"><span class="header-section-number">9.190.7</span> Main Findings</h3>
<p>This study combined a theoretical approach, with structural and
computational methods to identify antibody epitopes on the ACE2
(angiotensin-converting enzyme 2) receptor-binding-domains of the Spike
(S) proteins of SARS CoV1 and SARS CoV2. The authors used constant-pH
Monte Carlo (MC) simulations and the PROCEEDpKa method, to determine
binding affinities of four SARS-Cov monoclonal antibody fragments
(CR3022, 80R, m39, F26G29<strong>)</strong> to SARS-Cov2, as well as the titratable
amino acids involved in the interactions. They also compared antibody
binding affinities to the binding affinity of S RBD of SARS-Cov1 and 2
to the ACE2 receptor to ensure that antibody binding affinities were
higher compared to those of S RBD/ACE2. They identified CR3022 as the Ab
having the highest affinity with SARS-Cov2 RBD. Based on their
simulations they were able to suggest some 3 amino acid modifications on
CR3022, to enhance its binding affinity to SARS-Cov2 RBD. The
computational analysis of interactions between SARS-CoV-1 S RBD proteins
and the S RBD-specific neutralizing mAbs (80R, F26G19, m396, CR3022)
recapitulated previous experimental results suggesting that the approach
was valid. The only mAb with measured affinity for the SARS-CoV-2 S RBD
protein by BLI assay was CR3020, which also exhibited the highest
theoretical binding affinity determined in the present study.</p>
<h3 data-number="9.190.8" id="limitations-186"><span class="header-section-number">9.190.8</span> Limitations</h3>
<p>Simulations were performed on S RBD proteins alone. However, given that
the S protein is trimeric, structure modifications and/or conformational
changes in the S protein trimer could alter antibody binding affinity.
In addition, while CR3022 has been shown to bind S RBD, it did not
neutralize SARS-Cov2 in vitro. The CR3022 modifications suggested by the
authors could enhance the binding affinitys and the neutralizing
potential of CR3022, but this is currently hypothetical and remains to
be tested.</p>
<h3 data-number="9.190.9" id="significance-174"><span class="header-section-number">9.190.9</span> Significance</h3>
<p>These results are in agreement with previous experimental studies where
CR3022 was shown to bind SARS-Cov2 RBD (10.1101:2020.01.28.923011,
10.1126/science.abb7269) and confirm the validity of the approach. A
human monoclonal antibody against SARS-Cov2 RBD could be used as
therapeutics to treat patients with severe COVID-19, by inhibiting
ACE2/RBD interaction.</p>
<h3 data-number="9.190.10" id="credit-183"><span class="header-section-number">9.190.10</span> Credit</h3>
<p><em>Reviewed by Emma Risson as part of a project by students, postdocs and
faculty at the Immunology Institute of the Icahn School of Medicine,
Mount Sinai.</em></p>
<p>Edited by K Alexandropoulos</p>
<p><em>.</em></p>
<p><strong>Title: </strong></p>
<p><strong>Genomic determinants of pathogenicity in SARS-CoV-2 and other human
coronaviruses</strong></p>
<p><strong>Immunology keywords: SARS-CoV-1, SARS-CoV-2, MERS-CoV, pathogenicity
sequences, comparative genomics</strong></p>
<p><strong>The main findings of the article: </strong></p>
<p>In this study, potential genomic determinants of pathogenicity of the
high case fatality rates (CFR) coronavirus strains were identified using
integrated comparative genomics and advanced machine learning methods.</p>
<p>A total 3001 coronavirus genomes were compared. Of those, 944 belong to
viruses that infect humans, including both viruses with low CFR (NL63,
229E, OC43 and HKU1) and those with high CFR (MERS, SARS-CoV-1 and
SARS-CoV-2). The full genomes of these coronaviruses were aligned to
detect high-confidence genomic features that were predictive of the
high-CFR. Eleven regions were detected in total, and the nucleocapsid
(N) protein and the spike (S) glycoprotein were significantly enriched
with those predictive regions. Further analysis revealed that 4 out of
11 differences identified were reflected in the protein alignment, 3 of
which resided in the N phosphoprotein and one in the S glycoprotein. The
deletions and insertions found in N proteins mapped to monopartite
nuclear localization signals (NLS), one bipartite NLS and a nuclear
export signal (NES). They resulted in the accumulation of positive
charges in the monopartite NLS, bipartite NLS and NES of SARS-CoV-1 and
SARS-CoV-2, whereas in MERS-CoV, positive charges accumulated primarily
in the first of the two monopartite NLS. The accumulation of positive
charges is known to result in the enhancement of these signals, thus
implying that the localization pattern of the N proteins differs between
high-CFR and low-CFR strains, and this may be a factor contributing to
the increased pathogenicity of high-CFR strains. The analysis of S
protein revealed a unique 4 amino acid insertion upstream of the heptad
repeat region in all high-CFR viruses but not in any of the low-CFR
ones. The structural analysis of SARS-CoV confirmed that this insertion
increased the length and flexibility of the region connecting the fusion
peptide and the first heptad repeat, likely affecting the membrane
fusion process of the high-CFR coronaviruses.</p>
<p>By the alignment of all 3001 collected coronavirus strain sequences,
they also identified a unique insert in the RBD and the RBM in the cases
of SARS-CoV and SARS-CoV-2, and within the subdomain that binds to DPP4
in the case of MERS-CoV. This unique insert was also found in the most
proximal zoonotic strains before the zoonotic strains jumped to human.
The unique insertions occurred independently in three high-CFR strains
and resulted in a larger hydrophobic surface and additional
interactions, which might contribute not only to the zoonotic
transmission of the high-CFR CoV strains to humans but also their high
CFR.</p>
<p><strong>Critical analysis of the study:</strong></p>
<p>This study provides predictions on unique sequences present in
coronaviruses of high pathogenicity, and in highly related zoonotic
species. Functional analysis of the unique elements is not carried out.</p>
<p>There is an inconsistency related to description of positive/negative
charged proteins between figure legends of 2 and 3, and the main text.</p>
<p><strong>The importance and implications for the current epidemics:</strong></p>
<p>The identification of unique sequences presents in the high-CFR but not
in the low-CFR human coronaviruses can be experimentally tested to
investigate the importance of each unique motive in the pathogenicity
and transmissibility of human coronaviruses. This information will be
useful to identify viruses with a potential to cause future pandemic,
and to develop novel therapeutic strategies.</p>
<p>Title: The phenotypic changes of γδ T cells in COVID-19 patients</p>
<p><em>Keywords</em></p>
<p>T cells, Innate immune response, CD25, γδ T cells.</p>
<p><em>Main findings</em></p>
<p>γδ T cells represent 1-10% of total circulating lymphocytes, and are
able to promptly respond to infections and other stimuli. To address the
effects of SARS-CoV-2 infection on γδ T cells, the authors collected
blood samples from 18 healthy donor (HD) s and 38 COVID-19 patients and
analyzed the frequency and phenotype of γδ T cells.</p>
<p>Unlike other viral infections, COVID-19 patients presented lower
frequency of circulating γδ T cells than HD. Among γδ T cells, there was
an increase in the CD4+ subset and CD25 expression compared to
non-infected volunteers. The authors also report that γδ T cells of
COVID-19 patients didn’t present higher frequencies of early
activation/exhaustion markers such as CD69 and PD-1, in comparison to
the control group.</p>
<p><em>Limitations</em></p>
<p>There is very little information on COVID-19 patients or healthy
volunteers (age and sex should be reported). Furthermore, since the time
after the onset of symptoms and viral load may be important parameters
in the innate immune response to SARS-CoV-2 the authors should have
shown this information in a table.</p>
<p>Since lymphopenia is an important characteristic of COVID-19 patients
[1] and the authors observed lower frequencies of γδ T cells, it would
have been important to show frequencies and cell numbers for different T
cell populations.</p>
<p><em>Relevance </em></p>
<p>This study indicates, for the first time, that γδ CD4+ CD25+ T cells can
be involved in the acute phase of COVID-19. The function of γδ T cells
in the context of COVID-19 should be further investigated.</p>
<p><em>References</em></p>
<p>1. Tan, L., et al., <em>Lymphopenia predicts disease severity of COVID-19:
a descriptive and predictive study.</em> Signal Transduct Target Ther, 2020.
<strong>5</strong>(1): p. 33.</p>
<p>Review by Alessandra Soares Schanoski as part of a project by students,
postdocs and faculty at the Immunology Institute of the Icahn school of
medicine, Mount Sinai.</p>
<h3 data-number="9.190.11" id="title-structural-and-functional-analysis-of-a-potent-sarbecovirus-neutralizing-antibody"><span class="header-section-number">9.190.11</span> <strong>Title</strong> <strong>Structural and functional analysis of a potent sarbecovirus neutralizing</strong> <strong>antibody </strong></h3>
<p>Pinto et al. <em>bioRxiv</em> [<span class="citation" data-cites="doi">[<span class="citeproc-not-found" data-reference-id="doi"><strong>???</strong></span>]</span>: 10.1101/2020.04.07.023903]</p>
<h3 data-number="9.190.12" id="keywords-191"><span class="header-section-number">9.190.12</span> Keywords</h3>
<ul>
<li><p>Neutralizing antibodies</p></li>
<li><p>ACE2/RBD binding</p></li>
<li><p>SARS-CoV-1</p></li>
</ul>
<h3 data-number="9.190.13" id="main-findings-185"><span class="header-section-number">9.190.13</span> Main Findings</h3>
<p><span id="limitations" class="anchor"></span>Antibodies were isolated from memory B cells of
a recovered SARS-CoV-1 infected individual. 8 out of 25 isolated
antibodies bound the SARS-CoV-2 Spike (S) protein, and 4 bound the S
receptor binding domain (RBD) from SARS-CoV-1 and SARS-CoV-2. One of
them (s309) neutralized SARS-CoV-1 and SARS-CoV-2 pseudoviruses with
similar potencies by binding to S RBD, as well as authentic SARS CoV2
(IC50‑69ng/ml). Single particle cryoEM structural mapping showed that
s309 bound to a strictly conserved protein/glycan epitope between
SARS-CoV-1 and SARS-CoV-2, which is distinct from the RBD. Biolayer
interferometry indeed confirmed that s309 did not interfere with ACE2
and S binding. The authors also showed that weakly neutralizing
antibodies when used together with s309, enhanced its ability to
neutralize SARS CoV2.</p>
<p>The authors further explored the effect of the different antibodies on
NK-mediated antibody-dependent cell toxicity (ADCC) and
antibody-dependent cellular phagocytosis (ADCP), both effector
mechanisms that can contribute to viral control in infected individuals.
s309 showed the highest ADCC and ADCP responses whereas other antibodies
showed limited or no activity. The authors propose that s309 besides
neutralization, may regulate protective mechanisms to mitigate viral
risk as shown for other antiviral antibodies. (ref 41,42 in the paper)</p>
<h3 data-number="9.190.14" id="limitations-1"><span class="header-section-number">9.190.14</span> Limitations</h3>
<p>Although s309 seems to be neutralizing in vivo, more studies need to be
performed with pre-clinical models. Also, it would have been interesting
to assess ADCC and ADCP in SARS-CoV-2 infected co-cultures of NK-cells
and ACE2-expressing cells. Although the mechanism of neutralization by
s309 is not understood, the authors postulate that S trimer
crosslinking, steric hindrance or aggregation of virions may all
contribute to the ability of the antibody to neutralize the virus.</p>
<p><em>(Of note, legend and panels of figure 3 are</em> <em>mismatched)</em></p>
<h3 data-number="9.190.15" id="significance-175"><span class="header-section-number">9.190.15</span> Significance</h3>
<p><span id="credit" class="anchor"></span>This study identifies s309 as a potential human
monoclonal neutralizing antibody. It doesn’t neutralize SARS-CoV-2 by
inhibiting ACE2/RBD binding, as shown in other neutralizing
antibodies<sup>1,2,3</sup>. While clinical data is still needed, it is a
promising candidate for further development and <em>in</em> <em>vivo</em> testing.
Moreover, s309 binds to a conserved epitope between SARS-CoV-1 and
SARS-CoV-2, so it could possibly also neutralize other zoonotic
sarbecovirus (sub-genus of coronavirus).</p>
<h3 data-number="9.190.16" id="credit-1"><span class="header-section-number">9.190.16</span> Credit</h3>
<p><em>Reviewed by Emma Risson as part of a project by students, postdocs and
faculty at the Immunology Institute of the Icahn School of Medicine,
Mount Sinai.</em></p>
<p><em>Edited by K Alexandropoulos</em></p>
<p>1. Sun C, Chen L, Yang J, et al. SARS-CoV-2 and SARS-CoV Spike-RBD
Structure and Receptor Binding Comparison and Potential Implications on
Neutralizing Antibody and Vaccine Development. Microbiology; 2020.
doi:10.1101/2020.02.16.951723</p>
<p>2. Wang C, Li W, Drabek D, et al. A Human Monoclonal Antibody Blocking
SARS-CoV-2 Infection. Microbiology; 2020. doi:10.1101/2020.03.11.987958</p>
<p>3. Wu Y, Li C, Xia S, et al. Fully Human Single-Domain Antibodies
against SARS-CoV-2. Microbiology; 2020. doi:10.1101/2020.03.30.015990</p>
<p><strong>Tocilizumab treatment in severe COVID-19 patients attenuates the
inflammatory storm incited by monocyte centric immune interactions
revealed by single-cell analysis</strong></p>
<p>Guo et al., bioRxiv [10.1101:2020.04.08.029769]</p>
<p>Keywords:</p>
<ul>
<li><p>Tocilizumab</p></li>
<li><p>COVID-19</p></li>
<li><p>SARS-CoV-2</p></li>
<li><p>monocytes</p></li>
</ul>
<p><em>Main findings</em></p>
<p>Single-cell RNA sequencing was performed on PBMCs from two COVID-19
patients treated with Tocilizumab, an interleukin (IL)-6 receptor
antagonist. One blood sample was taken from each patient within 12 hours
post-treatment (severe stage of disease), and a total of three blood
samples were taken 5 and 7 days post-treatment (recovery stage of
disease). A total of 13,289 cells were analyzed post-quality control and
compared to 55,948 previously published profiles of healthy PBMCs.</p>
<p>The authors identified a subpopulation of pro-inflammatory CD14<sup>+</sup>
monocyte/macrophages enriched in the severe stage of COVID-19,
characterized by the upregulation of TNF, IL-6, IL-1β, and CXCL8. In
addition, a cluster of plasma cells and subpopulations of effector
CD8<sup>+</sup> T cells were found to be enriched in both the severe and recovery
stages. The authors attribute this finding to the fact that Tocilizumab
therapy does not necessarily compromise the anti-viral immune response.</p>
<p><em>Limitations</em></p>
<p><em>Technical</em></p>
<p>Only two patients were included in this analysis. Though both patients
were classified under severe COVID-19 disease, at the time of therapy,
patients PZ and PW were at significantly different stages of the
disease; PZ had an Sp<sub>O2</sub> &lt; 93%, and PW presented with respiratory
failure, multiple organ dysfunction, and Sp<sub>O2</sub> &lt; 93% under high flow
oxygen.</p>
<p>Of note, PBMCs collected from patients were compared to previously
published healthy controls. However, this is not a proper control; cells
collected from COVID-19 patients that had not been treated with
Tocilizumab are needed. Therefore, this comparison is not sufficient to
attribute significant differences in clustering patterns to Tocilizumab
treatment. Moreover, without additional details concerning improvement
of clinical symptoms after Tocilizumab therapy at days 5 and 7, there is
a lack of data supporting the claim that these timepoints (only a week
into therapy) necessarily represent a stage of recovery in COVID-19
patients.</p>
<p>Finally, PBMCs attributed to severe disease (presumably to define the
pre-treatment stage) were collected after administration of Tocilizumab,
instead of before administration to establish an untreated baseline
control. In addition, PBMCs attributed to the recovery stage were
collected at timepoints that were not necessarily the same for both
patients. Recovery stage cells from patient PZ were collected 5 days
post-treatment, whereas recovery stage cells from patient PW were
collected 5 and 7 days post-treatment. Because it was not clarified, it
must be assumed that the 5 and 7 day samples from patient PW were
combined to represent the recovery stage for that patient, but this
yields an inconsistent comparison between the two patients.</p>
<p><em>Biological </em></p>
<p>Enrichment of pro-inflammatory monocyte/macrophages in patients with
severe infections is expected. The study is missing a more detailed
characterization of cluster 9 (identified as the group of myeloid cells
unique to the severe stage) to potentially establish a novel gene
program specific to COVID-19. This is furthered by the fact that despite
authors’ claims, cluster 9 technically belongs to both the severe and
recovery stages rather than being stage-specific. It is notable that
transcriptionally, the acute inflammatory response is reduced in the
defined recovery stage cells.</p>
<p>Other findings are expected, including the persistence of plasma B cells
and effector CD8<sup>+</sup> T cells in response to a viral infection. In
addition to the technical limitations aforementioned, to better assess
the impact of Tocilizumab on the inflammatory response, both
transcriptional and translational outcomes must be investigated. It is
not sufficient to account the downregulation of pro-inflammatory
cytokines and chemokines to Tocilizumab treatment without the proper
controls.</p>
<p><em>Significance</em></p>
<p>IL-6R antagonists, like Tocilizumab and Sarilumab, have been raised as
potential therapeutics for COVID-19 as a means of reducing the
hyper-inflammation seen in patients. Understanding the cellular changes
that occur during and after therapy is important to determining the
efficacy of such therapy and whether inhibition of IL-6 signaling
compromises other adaptive immune mechanisms needed for a complete
anti-viral response. This study contributes to that analysis at the
single-cell level.</p>
<p><em>Reviewed by Matthew D. Park and Assaf Magen as part of a project by
students, postdocs, and faculty at the Immunology Institute of the Icahn
School of Medicine, Mount Sinai.</em></p>
<p><strong>Title: </strong></p>
<p><strong>Comparison of SARS-CoV-2 infections among 3 species of non-human
primates</strong></p>
<p><strong>Immunology keywords: SARS-CoV-2, animal model</strong></p>
<p><strong>The main finding of the article: </strong></p>
<p>In this study, two species of old world monkeys, <em>Macaca mulatta</em> and
<em>Macaca facicularis</em>, and one of new world monkeys, <em>Callithrix
jacchus</em>, were used as potential animal models for SARS-CoV-2 infection.
4 adults and 4 old <em>M. mulatta</em> and 6 <em>M. facicularis</em> were inoculated
with 4.75 x 10<sup>6</sup> pfu of virus intratracheally (4.0 ml), intranasally
(0.5 ml) and on conjunctiva (0.25 ml), and half dosage of virus was
given to 4 young <em>M. Mulatta</em>. 6 <em>C. jacchus</em> were inoculated with 1.0 x
10<sup>6</sup> pfu intranasally.</p>
<p>The authors described clinical symptoms of fever and weight loss were
observed in 12/12 and 9/12 of <em>M. mulatta</em> and 2/6 and 5/6 of <em>M.
facicularis</em> post infection, respectively. Chest radiographs of <em>M.
mulatta</em> and <em>M. facicularis</em> revealed abnormalities in lungs in all
animals from 10 days post infection (dpi), and progressive pulmonary
infiltration was observed until 21 dpi. Along with clinical symptoms,
viral RNA was detected in nasal swabs, pharyngeal swabs, anal swabs,
feces, blood and tissues from <em>M. mulatta</em> and <em>M. facicularis</em> on 2 dpi
at high level. Most of the swabs showed a second peaks on 6-8 dpi and
viral RNA was detectable on 14 dpi in some swab samples. The analysis of
viral load in the tissues revealed higher level of viral genomes in
bronchus and trachea. Although no viral particles were detected by
ultrastructural examination, the authors described that
histopathological analysis revealed severe gross lesions on lung, heart
and stomach of <em>M. mulatta</em> and <em>M. facicularis</em>, but not <em>C. jacchus.</em>
While <em>C. jacchus</em> did not show any symptoms or viral RNA in the
tissues, lower levels of viral RNA were detected in the nasal swab
samples until 12 dpi. Virus-specific antibodies became detectable as
early as 4 dpi with gradual increase in most of the animals, however,
young <em>M. mulatta</em> showed overall lower levels of antibodies. Increase
of IL-2, IL-6 was not detected, but G-CSF, IL-1A, IL-8, IL-15, IL-18,
MCP-1, MIP-1B and sCD40L were detected in the serum from <em>M. mulatta</em>
and <em>M. facicularis</em>.</p>
<p>Although they refer to a decline in CD4+ T cells, CD8+ T cells and
monocytes in peripheral blood, the graphics shown are inconsistent with
their statements.</p>
<p>Overall, the study shows that, among the macaques tested, <em>M. mulatta</em>
has the highest susceptibility to SARS-CoV-2 infection, followed by <em>M.
facicularis</em> and <em>C. jacchus</em>.</p>
<p><strong>Critical analysis of the</strong> <strong>study:</strong></p>
<p>The manuscript needs clearer annotation and consistency among the main
text, methods and figure legends, and overall better scientific writing.
Because of the lack of uninfected control or baseline data for immune
cell numbers in healthy monkeys, neither the decline of T cells nor
stronger B cell response in young animals can be assessed from the data.
Cytokine levels in infected animals could also have been compared with
uninfected animals, or pre-infection levels. The histopathology and TEM
figures don’t have enough resolution and lack comparison with uninfected
controls. The authors report that they did not find virus perticles in
the TEM sections.</p>
<p><strong>The importance and implications for the current epidemics:</strong></p>
<p>While some of the observations of SARS-Cov2 infection in macaques were
not consistent with those of human patients, non-human primates may
still be a useful model for preclinical studies.</p>
<p>The Role of Vitamin D in Suppressing Cytokine Storm in COVID-19 Patients
and Associated Mortality</p>
<p><strong>Keywords:</strong> Vitamin D, cytokine storm, CRP, COVID-19</p>
<p><strong>Key findings:</strong> Using clinical data of patients and statistical
analysis, the authors wanted to explore the potential role of vitamin D
in decreased cytokine levels which may lead to the reduction of disease
severity in COVID-19 cases. The study shows that Spain and Italy present
a severe vitamin D deficiency. Concerning COVID-19, the two countries
have a higher mortality rate and a higher case fatality rate (CFR) for
patients age ≥ 70. The authors have also shown that subjects with a
severe deficiency of Vitamin D have 1.4 times higher risk for production
of high C reactive protein (CRP), used to indirectly evaluate cytokine
storm. They hypothesized that Vitamin D deficiency increases the risk
for cytokine storm in COVID-19 patients and they estimated a 15%
reduction in the number of severe COVID-19 cases after vitamin D
deficiency elimination.</p>
<p><strong>Potential limitations:</strong> The results of this study remain
hypothetical. Future studies should assess vitamin D and
pro-inflammatory cytokine levels in COVID-19 patients upon hospital
admission and the role of vitamin D in decreasing cytokine levels should
be evaluated. In addition, the authors did not take into account the
medical history of COVID-19 patients in their analysis, as previous
studies have shown that patients who have comorbidities (cardiovascular
disease, hypertension, chronic pulmonary disease, hepatic disease,
diabetes and other chronic disease) are more likely to develop a severe
COVID-19 <em>https://www.medrxiv.org/content/10.1101/2020.03.11.20030957v1</em></p>
<p><a href="https://www.medrxiv.org/content/10.1101/2020.03.30.20043919v1"><em>https://www.medrxiv.org/content/10.1101/2020.03.30.20043919v1</em></a></p>
<p>Furthermore, age has been reported to be a major factor that determines
outcome for COVID-19 and it has been demonstrated that aging affects the
formation of the active form of vitamin
D.<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4399494/pdf/nihms662547.pdf"><em>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4399494/pdf/nihms662547.pdf</em></a>
<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3782116/pdf/nihms466447.pdf"><em>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3782116/pdf/nihms466447.pdf</em></a>
It would be interesting to see if vitamin D deficiency in the elderly
associates with severe forms of COVID-19. Finally, the authors have
chosen to use CRP to evaluate the intensity of cytokine storm which is
an indirect approach. However, even if CRP has been regarded as a
pro-inflammatory molecule, it has some anti-inflammatory functions
(recruitment of complement inhibitors and release of anti-inflammatory
cytokines such as IL-10 and IL-1ra) and could be replaced by more
specific inflammatory markers.
<a href="https://www.fasebj.org/doi/pdf/10.1096/fj.11-186460"><em>https://www.fasebj.org/doi/pdf/10.1096/fj.11-186460</em></a>
<a href="https://www.sciencedirect.com/science/article/pii/S0889159118300266"><em>https://www.sciencedirect.com/science/article/pii/S0889159118300266</em></a></p>
<p><strong>Overall relevance for the field:</strong> The study results did not
demonstrate a direct and obvious link between vitamin D deficiency and
severe cases of COVID-19 that feature cytokine storm. The study is based
only on analysis of the reported clinical data from multiple studies,
the results are therefore hypothetical and require a more direct
demonstration to see if vitamin D supplementation could really suppress
cytokine storm in COVID-19 patients.</p>
<p>Review by Meriem Belabed as part of a project by students, postdocs and
faculty at the Immunology Institute of the Icahn school of medicine,
Mount Sinai.</p>
<h2 data-number="9.191" id="type-2-and-interferon-inflammation-strongly-regulate-sars-cov-2-related-gene-expression-in-the-airway-epithelium"><span class="header-section-number">9.191</span> Type 2 and interferon inflammation strongly regulate SARS-CoV-2 related gene expression in the airway epithelium</h2>
<p>Sajuthi S.P. et al.  [<span class="citation" data-cites="doi">[<span class="citeproc-not-found" data-reference-id="doi"><strong>???</strong></span>]</span>: 10.1101/2020.04.09.034454]</p>
<h3 data-number="9.191.1" id="keywords-192"><span class="header-section-number">9.191.1</span> Keywords</h3>
<ul>
<li><p>SARS-CoV-2</p></li>
<li><p>scRNAseq</p></li>
<li><p>Type 2 Inflammation</p></li>
<li><p>Nasal airway epithelium</p></li>
<li><p>Interleukin</p></li>
</ul>
<h3 data-number="9.191.2" id="main-findings-immunological"><span class="header-section-number">9.191.2</span> Main Findings (Immunological)</h3>
<ul>
<li><p>Analyzed nasal airway epithelial transcriptome data from 695
asthmatic and healthy children (GALA II study, no COVID-19) to
determine WGCNA networks. TMPRSS2 gene was contained in a set of
three highly correlated networks exhibiting strong enrichments for
IL13-induced mucus secretory cell genes and canonically Type-2
inflammation pathways. ACE2 gene was correlated with interferon
response and cytotoxic immune signaling eigengene enrichment.</p></li>
<li><p>TMPRSS2 upregulated and ACE2 downregulated upon rIL13 treatment in
muco-ciliary air-liquid interface (ALI) cell culture; supporting
transcriptome analyses. Also, scRNAseq data from tracheal airway
epithelial cultures chronically stimulated with IL-13 also showed
congruent changes in TMPRSS2 and ACE2.</p></li>
<li><p>Used metagenomic analysis from dataset to find 18 asymptomatic
children with four different coronaviral sequences (CoV; OC43, JKU1,
229E, NL63). Compared 11 subjects with highest CoV infection to 571
CoV-naïve subjects, with 37 rhinovirus (HRV)-infected subjects
serving as ‘basal/normal respiratory virus’ comparators.</p></li>
<li><p>Induction of cytotoxic immune response was considerably higher in
CoV-infected subjects, compared to HRV-infected individuals. IL10,
IL1B, IFNG, IFNA2, STAT1, and ACE2 upregulation shared for CoV or
HRV infections. IL6 upregulated and genes associated with CD8 T
cells, DCs and NK cells specifically enriched in CoV infections.</p></li>
<li><p>Identified eQTLs for ACE2 and TMPRSS2 using GALA II dataset and used
multi-variable modeling to estimate the relative contribution of
these factors to population variation in these genes (not reviewed
here).</p></li>
</ul>
<h3 data-number="9.191.3" id="limitations-187"><span class="header-section-number">9.191.3</span> Limitations</h3>
<ul>
<li><p>Dataset is large, but having asthmatic young subjects as part of
GALA II study might skew WGCNA analyses to enrich for T2 biomarkers
and secretory phenotype genes. Will be important to replicate
analyses in another independent non-COPD/allergy/asthma dataset and
reconcile ACE2+TMPRSS2 correlation with T2 emergence.</p></li>
<li><p>Provides rationale for increased cytotoxic responses in CoVID-19
cases, but no explanation for ACE2 and TMPRSS2 anti-correlative
trends in epithelia upon T2 inflammation.</p></li>
<li><p>Will need ALI experiments with IL13 addition in CoV-2 infection
models to ascertain changes in ACE2/TMPRSS2 to be consistent with
existing patient data for COVID-19.</p></li>
</ul>
<h3 data-number="9.191.4" id="significance-176"><span class="header-section-number">9.191.4</span> Significance</h3>
<ul>
<li><p>Study strongly suggests airway epithelial TMPRSS2 expression is
highly upregulated upon Type 2 inflammation, specifically by IL13
stimulation of epithelia. Also, ACE2 expression (and therefore viral
infectivity) is inextricably linked to interferon response and could
sabotage initial inflammation.</p></li>
<li><p>Changes in immune contextures observed in asymptomatic CoV+/HRV+
subjects suggest remodeling of airway epithelia even after viral
resolution, which can impact future infections with COVID-19 and
outcomes.</p></li>
<li><p>Results showing IL10, IL1B, IL6 enrichment and cytotoxic programs
(CD8, NK, DC) in CoV+ subjects supports other concurrent studies
showing their importance in cytokine storm and points to broader
conserved inflammatory pathways worth targeting in COVID-19.</p></li>
</ul>
<h3 data-number="9.191.5" id="section"><span class="header-section-number">9.191.5</span> </h3>
<h3 data-number="9.191.6" id="credit-184"><span class="header-section-number">9.191.6</span> Credit</h3>
<p><em>Reviewed by Samarth Hegde as part of a project by students, postdocs
and faculty at the Immunology Institute of the Icahn School of Medicine,
Mount Sinai.</em></p>
<p><strong>Single-cell atlas of a non-human primate reveals new pathogenic
mechanisms of COVID-19</strong></p>
<p>Han L. et al.  [<span class="citation" data-cites="doi">[<span class="citeproc-not-found" data-reference-id="doi"><strong>???</strong></span>]</span>: 10.1101/2020.04.10.022103]</p>
<h3 data-number="9.191.7" id="keywords-193"><span class="header-section-number">9.191.7</span> Keywords</h3>
<ul>
<li><p>scRNAseq</p></li>
<li><p>ScATACseq</p></li>
<li><p>ACE2</p></li>
<li><p>TMPRSS2</p></li>
<li><p>Non-human primate</p></li>
<li><p>interleukin</p></li>
</ul>
<h3 data-number="9.191.8" id="main-findings-186"><span class="header-section-number">9.191.8</span> Main Findings</h3>
<ul>
<li><p>Study profiled nine tissues from cynomolgus monkey (NHP model with
no COVID19 infection) by scRNAseq and analyzed ACE2 and TMPRSS2
expression findings to existing human scRNA datasets. Also conducted
scATACseq on kidney tissue specifically to map chromatin
accessibility relevant to above genes.</p></li>
<li><p>ACE2 and TMPRSS2 predominantly detected in ciliated, club cells and
type2 alveolar cells, as well as proximal tubule cells of kidney,
and cholangiocytes in liver (agreeing with human data). Some
differences exist in ACE2 and TMPRSS2 expression between human and
NHP samples, especially in liver but also in lung.</p></li>
<li><p>Correlation analyses show immune-modulatory genes such as TMEM27,
IDO2, DNAJC12, and ANPEP co-expressed with ACE2 upregulation in
kidney. Also, IL6R gene expression correlated well with ACE2 in
proximal tubule cells (agreeing with human data).</p></li>
<li><p>scATACseq of kidney cells determined open chromatin regions with
discrete ACE2 peaks in proximal tubule cells S3, and TF motifs for
these regions were enriched in STAT1/STAT3 and IRF1 binding sites.</p></li>
</ul>
<h3 data-number="9.191.9" id="limitations-188"><span class="header-section-number">9.191.9</span> Limitations</h3>
<ul>
<li><p>Variations in ACE2/TMPRSS2 gene expression between NHP and human
tissues could arise due to differences in digestion and/or
processing protocols hence biasing reads. Will be important to
analyze protein expression for relevant genes in NHP tissue to
confirm such differences.</p></li>
<li><p>Study requires ex vivo co-culture experiments with kidney tubule
epithelial cells and COV-2 to ascertain IL6R signaling axis is
linked to ACE upregulation and/or impacts downstream alarmin/PAMP
release.</p></li>
</ul>
<h3 data-number="9.191.10" id="significance-177"><span class="header-section-number">9.191.10</span> Significance</h3>
<ul>
<li><h3 id="single-cell-nhp-profiling-is-relevant-and-useful-considering-monkeys-are-a-preferred-model-for-studying-the-effectiveness-of-drug-treatments-and-of-vaccines-against-covid-19">Single-cell NHP profiling is relevant and useful considering monkeys are a preferred model for studying the effectiveness of drug treatments and of vaccines against COVID-19</h3></li>
<li><h3 id="scatacseq-results-with-irf1stat1-accessibility-suggests-a-link-between-paracrine-interferon-signaling-il6-and-enhanced-ace2-expression-in-kidney-that-can-exacerbate-covid-19-severity-due-to-increased-viral-entry-and-dissemination.">scATACseq results with IRF1/STAT1 accessibility suggests a link between paracrine interferon signaling + IL6 and enhanced ACE2 expression in kidney that can exacerbate COVID-19 severity due to increased viral entry and dissemination.</h3></li>
<li><h3 id="findings-supports-other-reports-of-cytokine-induced-tissue-damage-in-kidneys-of-covid-19-patients-as-well-as-provide-additional-support-for-anti-il6r-strategies-such-as-tocilizumab.">Findings supports other reports of cytokine-induced tissue damage in kidneys of COVID-19 patients, as well as provide additional support for anti-IL6R strategies such as Tocilizumab.</h3></li>
</ul>
<p><em>Reviewed by Samarth Hegde as part of a project by students, postdocs
and faculty at the Immunology Institute of the Icahn School of Medicine,
Mount Sinai.</em></p>
<blockquote>
<p>Validated by: Robert M Samstein</p>
<p>The SARS-CoV-2 receptor-binding domain elicits a potent neutralizing
response without antibody-dependent enhancement</p>
<p>Quinlan et al <strong>doi:</strong> https://doi.org/10.1101/2020.04.10.036418</p>
<p><a href="https://www.biorxiv.org/content/10.1101/2020.04.10.036418v1" class="uri">https://www.biorxiv.org/content/10.1101/2020.04.10.036418v1</a></p>
<p><em>Keywords:</em> antibody-dependent enhancement, ACE2, RBD-based vaccine.</p>
<p><span id="OLE_LINK4" class="anchor"><span id="OLE_LINK5" class="anchor"></span></span><em>Main findings:</em> <strong><em>The
author’s data suggest that a receptor-binding domain-based (RBD)
vaccine for SARS-Cov2 could be safe and effective</em></strong>. Similar to
SARS-CoV-1, SARS-CoV-2 requires expression of the cellular receptor
angiotensin-converting enzyme 2 (ACE2) to infect cells. The spike (S)
protein mediates entry of CoV-2 engages ACE2 through its
receptor-binding domain (RBD), an independently folded 197-amino acid
fragment of the 1273-amino acid S-protein promoter.</p>
<p>The authors immunized mice using RBD fusion protein, mixed with
adjuvant. Blood were collected at day 0(before injection), day 5 and
day10 and analyzed neutralization of viral entry. The data suggested
that antibodies to the RBD domain of SARS-CoV-1 potently neutralize
SARS-CoV-1 S-protein-mediated entry in rat, and the presence of
anti-RBD antibodies correlates with neutralization in SARS-CoV-2
convalescent sera.</p>
<p>Antibody dependent enhancement (ADE) can contribute to the
pathogenicity of viral infection a major concern observed with other
viruses but not with SARS-Cov-2 to date<em>.</em> Importantly, <strong>the
RBD-elicited antibodies may mediate ADE less efficiently than those
recognizing other neutralizing S-protein epitopes</strong>. Furthermore,
anti-RBD anti-serum did not promote ADE at serum dilutions and under a
condition in which is the <strong>ZIKV ADE could be observed.</strong></p>
<p>Relevance: The study highlights a potential strategy for SARS-CoV-2
vaccine, using RBD protein to promote neutralization and protection.
The work demonstrated limited evidence of ADE, that has been shown to
contribute to pathogenesis in zika and dengue.</p>
<p>Limitation: Data show that the RBD alone is sufficient to elicit a
potent neutralizing antibody response. It remains to be determined
whether trimer-based vaccines, or trimer/RBD combinations, can be more
effective, and whether they carry additional safety concerns.</p>
<p><span id="OLE_LINK1" class="anchor"><span id="OLE_LINK2" class="anchor"></span></span><strong>Credit:</strong></p>
<p><em>Reviewed by Zafar Mahmood as part of a project by students, postdocs
and faculty at the Immunology Institute of the Icahn School of
Medicine, Mount Sinai.</em></p>
</blockquote>
<p><strong>Presence of SARS-CoV-2 reactive T cells in COVID-19 patients and
healthy donors</strong></p>
<p>Braun J et al.; medRxiv  2020.04.17.20061440;
https://doi.org/10.1101/2020.04.17.20061440</p>
<p><strong><em>Keywords</em></strong></p>
<ul>
<li><p>SARS-CoV-2 specific CD4 T cells</p></li>
<li><p>Human endemic coronaviruses</p></li>
<li><p>COVID-19</p></li>
</ul>
<p><strong><em>Main findings </em></strong></p>
<p>In this preprint, Braun et al. report quantification of virus-specific
CD4 T cells in 18 patients with mild, severe and critical COVID-19,
including 10 patients admitted to ICU. Performing <em>in vitro</em> stimulation
of PBMCs with two sets of overlapping SARS-CoV-2 peptide pools – the S I
pool spanning the N-terminal region (aa 1-643) of the S protein,
including 21 predicted SARS-CoV-1 MHC-II epitopes, and the C-terminal S
II pool (aa 633-1273) containing 13 predicted SARS-CoV-1 MHC-II epitopes
– the authors detected S-protein-specific CD4 T cells in up to 83% of
COVID-19 patients based on intracellular 4-1BB (CD137) and CD40L (CD154)
induction. Notably, peptide pool S II shares higher homology with human
endemic coronaviruses (hCoVs) 229E, NL63, OC43, and HKU1 that may cause
the common cold, but it does not include the SARS-CoV-2 receptor-binding
domain (RBD), which has been identified as a critical target of
neutralizing antibodies in both SARS-CoV-1 and SARS-CoV-2. S I-reactive
CD4 T cells were found in 12 out of 18 (67%) patients, whereas CD4 T
cells against S II were detected in 15 patients (83%). Intriguingly,
S-specific CD4 T cells could also be found in 34% (n=23) of 68
SARS-CoV-2 seronegative donors, referred to as reactive healthy donors
(RHD), with a preference for S II over S I epitopes. Only 6 of 23 RHDs
also had detectable frequencies of S I-specific CD4 T cells, overall
suggesting S II-reactive CD4 T cells had likely developed in response to
prior infections with hCoVs. Of 18 out of 68 total healthy donors
tested, all were found to have anti-hCoV antibodies, although this was
independent of concomitant anti-S II CD4 T cell frequencies detected.
This finding mirrors observations of declining numbers of specific CD4 T
cells, but persistent humoral memory after certain vaccinations such as
against yellow fever. The authors further speculate that these
pre-existing virus-specific T cells against hCoVs might be one of the
reasons why children and younger patients, usually considered to have a
higher incidence of hCoV infections per year, are seemingly better
protected against SARS-CoV-2. Unlike specific CD4 T cells found in RHDs,
most S-specific CD4 T cells in COVID-19 patients displayed a phenotype
of recent <em>in vivo</em> activation with co-expression of HLA-DR and CD38, as
well as variable expression of Ki-67. In addition, a substantial
fraction of peripherally found HLA-DR<sup>+</sup>/CD38 bulk CD4 T cells was found
to be refractory to peptide stimulation, potentially indicating cellular
exhaustion.</p>
<p><strong><em>Limitations </em></strong></p>
<p>This is one of the first preprints reporting the detection of
virus-specific CD4 T cells in COVID-19 (also cf. Dong et al.,
<a href="https://www.medrxiv.org/content/10.1101/2020.03.17.20036640v1"><em>https://www.medrxiv.org/content/10.1101/2020.03.17.20036640v1</em></a>;
Weiskopf et al.,
<a href="https://www.medrxiv.org/content/10.1101/2020.04.11.20062349v1.article-info"><em>https://www.medrxiv.org/content/10.1101/2020.04.11.20062349v1.article-info</em></a>).
While it generally adds to our current knowledge about the potential
role of T cells in response to SARS-CoV-2, a few limitations, some of
which are discussed by the authors themselves, should be addressed.
Findings in this study pertain to a relatively small cohort of patients
of variable clinical disease. To corroborate the observations made here,
larger studies including greater numbers of both healthy donors and
patients of all clinical stages are needed to better assess the function
of virus-specific CD4 T cells in COVID-19. Specifically, the presence of
pre-existing, potentially hCoV-cross-reactive CD4 T cells in healthy
donors needs to be explored in the context of COVID-19
immunopathogenesis. While the authors suggest a potentially protective
role based on higher incidence of hCoV infection in children and younger
patients, and therefore a presumably larger pool of pre-existing
virus-specific memory T cells, the opposite could also be the case given
cumulatively increased number of hCoV infections in older patients. In
this context, it would therefore have been interesting to also measure
anti-hCoV antibodies in COVID-19 patients. Furthermore, this study did
not quantify virus-specific CD8 T cells. Based on observations in
SARS-CoV-1, virus-specific memory CD8 T cells are more likely to persist
long-term and confer protection than CD4 T cells, which were detected
only at lower frequencies six years post recovery from SARS-CoV-1 (<em>cf</em>.
[]{#_ENREF_18 .anchor}Li CK et al., Journal of immunology <em>181</em>,
5490-5500.) Morover, no other specifities such as against the N or M
epitopes were evaluated. Robust generation of virus-specific T cells
against the N protein was shown to be induced by SARS-CoV-2 in another
pre-print by Dong et al. (Dong et al.,
<a href="https://www.medrxiv.org/content/10.1101/2020.03.17.20036640v1"><em>https://www.medrxiv.org/content/10.1101/2020.03.17.20036640v1</em></a>),
while Weiskopf et al. recently reported preference of both CD8 and CD4 T
cells for S epitopes
<a href="https://www.medrxiv.org/content/10.1101/2020.04.11.20062349v1.article-info"><em>https://www.medrxiv.org/content/10.1101/2020.04.11.20062349v1.article-info</em></a>).
Moreover, the authors seem to suggest that some of the virus-specific
CD4 T cells detected could be potentially cross-reactive to predicted
SARS-CoV-1 epitopes present in the peptide pools used. Indeed, this has
been recently established for several SARS-CoV-2 binding antibodies,
while it was found not to be the case for RBD-targeting neutralizing
antibodies (<em>cf.</em> Wu et al.,
<a href="https://www.medrxiv.org/content/10.1101/2020.03.30.20047365v2"><em>https://www.medrxiv.org/content/10.1101/2020.03.30.20047365v2</em></a>.;
Ju et al.,
<a href="https://www.biorxiv.org/content/10.1101/2020.03.21.990770v2"><em>https://www.biorxiv.org/content/10.1101/2020.03.21.990770v2</em></a>).
A similar observation has not been made for T cells so far and should be
evaluated. Finally, since reactive healthy donors were only tested for
anti-S1 IgG, however not for other more ubiquitous binding antibodies,
e.g. against M, and only a fraction of these donors was additionally
confirmed to be negative by PCR, though unlikely, there is the
possibility that some of the seronegative reactive donors were
previously exposed to SARS-CoV-2.</p>
<p><strong><em>Significance</em></strong></p>
<p>Quantification of virus-specific T cells in peripheral blood is a useful
tool to determine the cellular immune response to SARS-CoV-2 both in
acute disease and even more so post recovery. Ideally, once immunogenic
T cell epitopes are better characterized, tetramer assays will allow for
faster and more efficient detection of their frequencies. Moreover,
assessing the potential role of pre-existing virus-specific CD4 T cells
in healthy donors in the context of COVID-19 pathogenesis will be of
particular importance. The observations made here are also highly
relevant for the design and development of potential vaccines and should
therefore be further explored in ongoing research on potential
coronavirus therapies and prevention strategies.</p>
<p><em>This review was undertaken by Verena van der Heide as part of a project
by students, postdocs and faculty at the Immunology Institute of the
Icahn school of medicine, Mount Sinai.</em> </p>
<p> </p>
<p>DOI: 10.1101/2020.04.11.20062349</p>
<p>Title: Phenotype of SARS-CoV-2-specific T-cells in COVID-19 patients
with acute respiratory distress syndrome</p>
<p>Keywords: SARS-CoV-2 specific T cells; Acute Respiratory Distress
Syndrome; Th1 response</p>
<p>Summary: Using peripheral blood samples collected from 8 COVID-19
patients with moderate to severe acute respiratory distress syndrome
(ARDS), the authors showed that COVID-19 patients exhibit lower CD3<sup>+</sup> T
cell counts as well as increased CD4:CD8 ratio. In addition to
population analysis, PBMCs were stimulated with three pools of either
overlapping peptides of SARS-CoV-2 spike (S) protein or HLA Class II or
I predicted epitopes covering all viral proteins except S designed to
activate CD4 and CD8 T cells, respectively. While stimulation of PBMCs
with all three peptide pools led to detection and activation of
SARS-CoV-2 specific CD4 and CD8 T cells, spike (S) peptide pool elicited
the strongest response, indicating that the S surface glycoprotein is a
strong inducer of both CD4 and CD8 T-cell responses. Phenotyping of
activated T cells (CD4<sup>+</sup>CD69<sup>+</sup>CD137<sup>+</sup> or CD8<sup>+</sup>CD69<sup>+</sup>CD137<sup>+</sup>)
showed that the majority of activated CD4 T cells were central memory
T-cells (CD45RA<sup>-</sup> and CCR7<sup>+</sup>) while activated CD8 T cells were mostly
effector memory T cells (CCR7<sup>-</sup>) and terminally differentiated effector
T cells. Cytokine analysis of cell culture supernatants from PBMCs
stimulated by S-peptide pool led to a strong production of Th1 cytokines
IFNγ, TNF⍺ and IL-2. Lastly, the authors profiled the kinetics of both
humoral and cell-mediated response in four different time points using
ELISA of virus-specific serum IgG and quantifying expression of cell
surface markers induced by S peptide pool activation. Throughout the
patients’ stay at the ICU, both levels of virus-specific IgG antibodies
and frequencies of virus-specific CD4 cells increased significantly over
time.</p>
<p>Limitations: A couple of additional assays done in parallel could have
further strengthened the paper’s findings. Simultaneous profiling of
cytokines from PBMCs could have easily answered whether these T cells
which are capable of producing Th1 cytokines upon activation are indeed
producing them in patients. Furthermore, it would have been informative
to have added a couple of functional exhaustion markers (i.e. PD-1,
Tim-3, etc.) and compare their expression between pre- and
post-activation by S peptide pool—thereby addressing the effect of
functional exhaustion in T cells reported in severe COVID-19 patients.
Follow-up studies investigating which epitopes out of the S protein
peptide pool elicited the most potent T-cell response would have yielded
informative results for possible vaccine design efforts against the
spike protein. Lastly, comparing the quality of virus specific T cells
immunity between patients in ICU (the focus of the study) and patients
with mild /moderate disease would have been very informative.</p>
<p>Significance of the finding: For the most part, the study design has
been well established to answer the following questions: a) are there T
cells reactive to spike (and other HLA-reactive) protein of SARS-CoV2
even in cases of COVID-19 with moderate to severe ARDS (which has been
characterized with lymphopenia)? b) what are the phenotype and cytokine
profile of CD4 and CD8 T cells upon activation? Answering these
questions do advance the field’s understanding of T cell response to
COVID-19 and add to much-needed effort to devise a vaccine against
SARS-CoV2. Building on this article’s findings, perhaps future studies
could perform mechanistic assays the function of T cells from COVID-19
patients in the context of systemic inflammation (i.e. adding IL-1,
IL-6, TNF⍺ in the culture media) as well as correlating epitope-specific
immune response of patients with their clinical severity.</p>
<p><em>Review by Chang Moon as part of a project by students, postdocs and
faculty at the<br />
Immunology Institute of the Icahn school of medicine, Mount Sinai.</em></p>
<p>DOI: 10.1101/2020.04.15.20065623</p>
<p>Title: Neutralizing Antibodies Responses to SARS-CoV-2 in COVID-19
Inpatients and Convalescent Patients</p>
<p>Keywords: Neutralizing Antibodies; Convalescent plasma</p>
<p>Summary: Based on peripheral blood samples from 117 COVID-19 inpatients
and convalescent patients, the authors demonstrate that all patients
sampled became seropositive with neutralizing antibodies within 20 days
since onset of symptoms and stayed seropositive until day 41-53.
Seropositivity of neutralizing antibodies was defined as a geometric
mean titer (GMT) of 1:4 higher and the titer was calculated using a
modified <em>in vitro</em> cytopathogenic assay where the dilution number of
50% protective condition from cytopathic effect of the virus represented
the titer. The GMT of neutralizing antibodies (average: 1:271.2) was the
highest at 31-40 days since onset and multivariate Generalized
Estimating Equations (GEE) model controlling for clinical variables
(i.e. gender, age, clinical severity, etc.) showed that antibody titer
at 31-40 days was significantly higher than 10-20 days past onset. In
addition, their multivariate GEE analysis showed that age- and clinical
severity-dependent rise in antibody titers with the youngest (age 16-30)
and patients with mild or asymptomatic conditions having a lower
antibody tier than its elderly and moderately-sick counterparts.</p>
<p>Limitations: Several shortcomings limit the impact of this study. While
it has been the intent of the authors to sample PBMCs from patients at
various time points in order to establish a robust profile of antibody
response against SARS-CoV2, in reality, sampling has been limited and
inconsistent across different time points. For instance, PBMCs of only
12 out of 117 patients have been collected three or more times and it is
not clear from the data whether samples from patients whose blood has
been collected only once (n = 37) are evenly distributed across the time
frames under analysis. Furthermore, the authors have tried to show
differences in kinetics of antibody response between patients with mild
and moderate conditions by sampling their blood at four different time
points. However, not only do two of eight patients sampled in this study
have only two data points, but also the authors have found that the
antibody response varies considerably across individuals—further
underscoring the need to have PBMCs sampled from each patient at
multiple time points and normalizing their response before comparing the
titers across individuals. In addition, due to the fact that patients
were enrolled using convenient sampling instead of random sampling
methods, it’s evident that the authors could not control for disease
severity as they only had four patients in severe condition. Beyond the
sampling issues, the modified cytopathic assay used to calculate the
neutralizing antibody titers may be less sensitive and specific than
ELISA-based assays that use purified antigens from the virus.</p>
<p>Significance of the finding: Limited. While it is informative to have
descriptive studies like this one showing the dynamics of the antibody
response against COVID-19, the failure of the study to collect samples
in controlled manner prevents the reader from using the data to answer
key questions regarding the humoral immune response against COVID-19: do
differences in clinical severity manifest in different kinetics of
antibody response? When controlled for age, is higher antibody titer
predictive of their clinical severity and prognosis? Future studies may
address those questions with more controlled experimental setup.</p>
<p><em>Review by Chang Moon as part of a project by students, postdocs and
faculty at the<br />
Immunology Institute of the Icahn school of medicine, Mount Sinai.</em></p>
<p>DOI: 10.1101/2020.04.16.045690</p>
<p>Title: The scRNA-seq expression profiling of the receptor ACE2 and the
cellular protease TMPRSS2 reveals human organs susceptible to COVID-19
infection</p>
<p>Keywords: ACE2, TMPRSS2, scRNA-seq expression profiling</p>
<p>Summary: Using publicly available scRNA-seq data of healthy human
samples from 31 organs, the authors compare the gene expression levels
of ACE2 and TMPRSS2 of each organ. The authors categorized each organ as
susceptible to SARS-CoV infection based on its expression of ACE2 and
further stratified 11 susceptible organs into three levels of risk for
infection based on each organ’s TMPRSS2 expression (i.e. TMPRSS2
expression ratio &gt;20% was defined as level 1). The authors claimed
that for the first time, their scRNA-seq analysis showed the brain, gall
bladder and fallopian tube as vulnerable to COVID-19 infection in
addition to confirming previous molecular and clinical data implicating
other organs susceptible to SARS-CoV2 (i.e. nose, heart, intestines,
etc.).</p>
<p>Limitations: The article advances its risk stratification strategy based
on a couple of naïve assumptions that are being actively contested in
literature: a) ACE2/TMPRSS2-mediated viral entry is the only route used
by SARS-CoV2 to infect cells and b) ACE2/TMPRSS2 expression is stable in
systemic inflammatory contexts such as COVID-19. Furthermore, their risk
stratification does not consider the immune contexture of each organ.
For instance, while several papers have suggested that reproductive
organs (i.e. testes) are also susceptible to SARS-CoV2 based on their
ACE2/TMPRSS2 expression, there are no reported clinical manifestations
of COVID-19 in those organs. Lastly, even though they were using
scRNA-seq data available from 13 different publications, their paper
does not discuss how they accounted for variations in scRNA-seq
protocols as well as how they normalized each data set for comparative
analysis.</p>
<p>Significance of the finding: Mostly confirmatory. While it is nice to
see that there were clinical cases describing the adverse effect of
COVID-19 for most of the eleven organs identified as SARS-CoV2
susceptible in this study, previous studies have already shown most of
these organs to be susceptible to SARS-CoV2 using various approaches
(including scRNA-seq analysis). Furthermore, their risk stratification
strategy is simplistic as it doesn’t seem to take account of recent
findings on SARS-CoV2’s alternative mechanisms of entry as well as
reported clinical manifestations of COVID-19.</p>
<p><em>Review by Chang Moon as part of a project by students, postdocs and
faculty at the<br />
Immunology Institute of the Icahn school of medicine, Mount Sinai.</em></p>
<p>Title Rapid development of an inactivated vaccine for SARS-CoV-2</p>
<p>Gao et al. bioRxiv [<span class="citation" data-cites="doi">[<span class="citeproc-not-found" data-reference-id="doi"><strong>???</strong></span>]</span>: 10.1101/2020.04.17.046375]</p>
<h3 data-number="9.191.11" id="keywords-194"><span class="header-section-number">9.191.11</span> Keywords</h3>
<ul>
<li><p>Inactivated vaccine</p></li>
<li><p>Neutralizing antibodies</p></li>
<li><p>Non-human primates</p></li>
</ul>
<h3 data-number="9.191.12" id="main-findings-187"><span class="header-section-number">9.191.12</span> Main Findings</h3>
<p>This study describes the development and pre-clinical tests on mice,
rats and rhesus macaques of a SARS-CoV-2 inactivated vaccine candidate.
Vaccine preparation (PiCoVacc) was generated from a viral strain (CN2)
isolated from a COVID-19 patient and cultured on Vero cells during 10
passages, before being inactivated with Beta-propiolactone, an
alkylating agent that damages nucleic acids and inhibits viral membrane
fusion.</p>
<p>Balb/c mice and Wistar rats were injected at d0 and d7 with PiCoVacc (0,
1.5 or 3 or 6 ug per dose) and alum adjuvant. SARS-CoV-2 S and RBD
specific IgG were detected after 1 week and continued to increase until
the end of the study (week 6). Micro-neutralization assays showed a
progressive augmentation of neutralizing antibodies (nAbs) against the
strain used to make the PiCoVacc from week 1 to week 6. These antibodies
(from week 3) also neutralized 9 others SARS-CoV-2 strains.</p>
<p>Next, rhesus macaques (n=4 per group) were immunized intramuscularly
three times at d0, d7 and d14 with 0 (sham), 3ug (medium dose) or 6ug
(high dose) of PiCoVacc with alum adjuvant, or only physiological
saline. S-specific IgG and nAbs were measured at week 2 in both
vaccinated groups. No fever or weight loss was observed after
immunization. Biochemical blood tests, lymphocytes subset percent and
cytokines showed no particular changed. Histopathological analysis of
several organs (kidney, brain, liver, heart, spleen) did not show any
changes either.</p>
<p>Macaques were then challenged by intra-tracheal inoculation of 10<sup>6</sup>
TCID<sub>50</sub> viral preparation of a different strain through the
intra-tracheal route, 8 days after the third vaccine inoculation. All
non-vaccinated controls showed important amount of viral RNA in pharynx,
crissum and lung until day 7 post inoculation (dpi) and had severe
interstitial pneumonia, whereas vaccinated macaques showed a quick
decrease in throat viral loads and small histopathological changes in
the lungs. Macaque vaccinated with the high dose had undetectable viral
loads in lungs and anal swab at any point of the study. No Antibody
Dependent Enhancement (ADE) was observed.</p>
<h3 data-number="9.191.13" id="limitations-189"><span class="header-section-number">9.191.13</span> Limitations</h3>
<p>The macaques were challenge at the peak of the antibody response induced
by the vaccine and no data was collected to assess the length of vaccine
protection. Furthermore, the macaques were euthanized as early as seven
days after the virus challenge and longer time of analysis will be
required to evaluate potential immune-pathological effects of the
vaccination and the dynamics of the infection in the different groups.
Finally, the size of the study is very limited with 4 animals vaccinated
per group.</p>
<p>While SARS-CoV-2 appears to be relatively stable, it would be
interesting to test cross reactivity and protection with heterologous
viruses, to confirm that the nAb induced by the vaccine can neutralize
many other SARS-CoV-2 strains, as it was performed in the mice and rat
studies. Indeed, although two different strains were used for the
immunization and the challenge in rhesus macaques, the S protein
sequences of those strains are identical, and the Genbank accession
number to compare the rest of the sequence was not made available yet.</p>
<p>Furthermore, as most of COVID-19 severe patients are of advanced age,
testing if this inactivated vaccine can also elicit antibody response in
aged macaques would be of great interest.</p>
<p>The timing of vaccination and challenge is not clear with contradictory
timelines indicated in the figures and main text. The authors do not
describe the inactivation process of the virus enough, nor show any
evidence of inactivation. More description and evidence of the
inactivation (by showing that the virus does not grow in Vero cells nor
in animals) is required to assess efficacy of vaccination</p>
<p><em>Finally, the resolution of the figures is really low and makes it hard
to analyze</em></p>
<h3 data-number="9.191.14" id="section-1"><span class="header-section-number">9.191.14</span> </h3>
<h3 data-number="9.191.15" id="significance-178"><span class="header-section-number">9.191.15</span> Significance</h3>
<p>Development of a vaccine against SARS-CoV-2 is strongly needed as no
effective treatment has been found yet. Among all the vaccine clinical
trials already started<sup>2</sup>, this pre-clinical study shows that this
inactivated vaccine could represent a good candidate for further
development as it protected rhesus macaques from viral challenge without
immediate side-effects. Although more pre-clinical data seem to be
needed before starting human trials, this is the first study to our
knowledge to show effectiveness in non-human primates. The
biopharmaceutical firm, Sinovac, has registered phase I and II clinical
trials <sup>1</sup>. Moreover, the inactivation technique for vaccine development
is well known and can be adapted for production in other facilities.</p>
<h3 data-number="9.191.16" id="section-2"><span class="header-section-number">9.191.16</span> </h3>
<h3 data-number="9.191.17" id="credit-185"><span class="header-section-number">9.191.17</span> Credit</h3>
<p><em>Reviewed by Emma Risson as part of a project by students, postdocs and
faculty at the Immunology Institute of the Icahn School of Medicine,
Mount Sinai.</em></p>
<p>1. Safety and Immunogenicity Study of 2019-nCoV Vaccine (Inactivated)
for Prophylaxis SARS CoV-2 Infection (COVID-19) - Full Text View -
ClinicalTrials.gov. https://clinicaltrials.gov/ct2/show/NCT04352608.
Accessed April 22, 2020.</p>
<p>2. <a href="http://paperpile.com/b/F9bKuX/LCEi">World Health Organization (2020). Draft of the landscape of COVID-19
candidate vaccines. World Health Organization
4–4.</a></p>
<p><strong>COVID-19 Critical Illness Pathophysiology Driven by Diffuse Pulmonary
Thrombi and Pulmonary Endothelial Dysfunction Responsive to
Thrombolysis</strong></p>
<p>Poor et al., medRxiv [10.1101/2020.04.17.20057125]</p>
<p>Keywords:</p>
<ul>
<li><p>COVID-19</p></li>
<li><p>pneumonia</p></li>
<li><p>thrombolysis</p></li>
<li><p>endothelium</p></li>
<li><p>platelet</p></li>
<li><p>D-dimer</p></li>
</ul>
<p><em>Main findings</em></p>
<p>This report describes the use of systemic tissue plasminogen activator
(tPA) to treat venous thromboembolism (VTE) seen in four critically ill
COVID-19 patients with respiratory failure. These patients all exhibited
gas exchange abnormalities, including shunt and dead-space ventilation,
despite well-preserved lung mechanics. A pulmonary vascular etiology was
suspected.</p>
<p>All four patients had elevated D-dimers and significant dead-space
ventilation. All patients were also obese, and 3/4 patients were
diabetic.</p>
<p>Not all patients exhibited an improvement in gas exchange or
hemodynamics during the infusion, but some did demonstrate improvements
in oxygenation after treatment. Two patients no longer required
vasopressors or could be weaned off them, while one patient became
hypoxemic and hypotensive and subsequently expired due to a cardiac
arrest. Echocardiogram showed large biventricular thrombi.</p>
<p><em>Limitations</em></p>
<p>In addition to the small sample size, all patients presented with
chronic conditions that are conducive to an inflammatory state. It is
unclear how this would have impacted the tPA therapy, but it is likely
not representative of all patients who present with COVID-19-induced
pneumonia. Moreover, each patient had received a different course of
therapy prior to receiving the tPA infusion. One patient received
hydroxychloroquine and ceftriaxone prior to tPA infusion, two patients
required external ventilator support, and another patient received
concurrent convalescent plasma therapy as part of a clinical trial. Each
patient received an infusion of tPA at 2 mg/hour but for variable
durations of time. One patient received an initial 50 mg infusion of tPA
over two hours. 3/4 patients were also given norepinephrine to manage
persistent, hypotensive shock. Of note, each patient was at a different
stage of the disease; One patient showed cardiac abnormalities and no
clots in transit on an echocardiogram, prior to tPA infusion.</p>
<p><em>Significance</em></p>
<p>The study describes emphasizes the importance of coagulopathies in
COVID-19 and describes clinical outcomes for four severe, COVID-19
patients, who received tPA infusions to manage poor gas exchange. While
the sample size is very limited and mixed benefits were observed,
thrombolysis seems to warrant further investigation as a therapeutic for
COVID-19-associated pneumonia that is characterized by D-dimer elevation
and dead-space ventilation. All four patients had normal platelet
levels, which may suggest that extrinsic triggers of the coagulation
cascade are involved.</p>
<p>The authors suspect that endothelial dysfunction and injury contribute
to the formation of pulmonary microthrombi, and these impair gas
exchange. Pulmonary thrombus formation has also been reported by other
groups; post-mortem analyses of 38 COVID-19 patients’ lungs showed
diffuse alveolar disease and platelet-fibrin thrombi (Carsana et al.,
2020). Inflammatory infiltrates were macrophages in the alveolar lumen
and lymphocytes in the interstitial space (Carsana et al., 2020).
Endothelial damage in COVID-19 patients has also been directly
described, noting the presence of viral elements in the endothelium and
inflammatory infiltrates within the intima (Varga et al., 2020). One
hypothesis may be that the combination of circulating inflammatory
monocytes (previously described to be enriched among PBMCs derived from
COVID-19 patients) that express tissue factor, damaged endothelium, and
complement elements that are also chemotactic for inflammatory cells may
contribute to the overall pro-coagulative state described in COVID-19
patients.</p>
<p><em>References</em></p>
<p>Carsana, L., Sonzogni, A., Nasr, A., Rossi, R.S., Pellegrinelli, A.,
Zerbi, P., Rech, R., Colombo, R., Antinori, S., Corbellino, M., et al.
(2020) Pulmonary post-mortem findings in a large series of COVID-19
cases from Northern Itality. medRxiv. 2020.04.19.20054262.</p>
<p>Varga, Z., Flammer, A.J., Steiger, P., Haberecker, M., Andermatt, R.,
Zinkernagal, A.S., Mehra, M.R., Schuepbach, R.A., Ruschitzka, F., Moch,
H. (2020) Endothelial cell infection and endotheliitis in COVID-19.
Lancet. 10.1016/S0140-6736(20)30937-5.</p>
<p><em>The study described in this review was conducted by physicians of the
Divisions of Pulmonary, Critical Care, and Sleep Medicine, Cardiology,
Nephrology, Surgery, and Neurosurgery and Neurology at the Icahn School
of Medicine at Mount Sinai. </em></p>
<p><em>Reviewed by Matthew D. Park as part of a project by students, postdocs,
and faculty at the Immunology Institute of the Icahn School of Medicine,
Mount Sinai.</em></p>
<h2 data-number="9.192" id="title-a-single-cell-atlas-of-the-peripheral-immune-response-to-severe-covid-19"><span class="header-section-number">9.192</span> Title: A single-cell atlas of the peripheral immune response to severe COVID-19</h2>
<h2 data-number="9.193" id="wilk-a.j.-et-al.-medrxiv-doi10.11012020.04.17.20069930"><span class="header-section-number">9.193</span> Wilk, A.J. et al. MedRxiv [<span class="citation" data-cites="5OE9jzTg"><a href="#ref-5OE9jzTg" role="doc-biblioref">654</a></span>]</h2>
<p><strong>Keywords:</strong> scRNAseq; Interferon-Stimulating Genes (ISGs);Activated
granulocytes</p>
<p><strong>Main Findings:</strong></p>
<p>The authors performed single-cell RNA-sequencing (scRNAseq) on
peripheral blood from 6 healthy donors and 7 patients, including 4
ventilated and 3 non-ventilated patients. 5 of the patients received
Remdesivir.</p>
<p>scRNAseq data reveal 30 gene clusters, distributed among granulocytes,
lymphocytes (NK, B, T cells), myeloid cells (dendritic cells DCs,
monocytes), platelets and red blood cells. Ventilated patients
specifically display cells containing neutrophil granule proteins that
appear closer to B cells than to neutrophils in dimensionality reduction
analyses. The authors named these cells “Activated Granulocytes” and
suggest them to be class-switched B cells that have lost the expression
of CD27, CD38 and BCMA and acquired neutrophil-associated genes, based
on RNA velocity studies.</p>
<p>SARS-CoV2 infection leads to decreased frequencies of myeloid cells,
including plasmacytoid DCs and CD16+ monocytes. CD14+ monocyte
frequencies are unchanged in the patients, though their transcriptome
reveals an increased activated profile and a downregulation of HLAE,
HLAF and class II HLA genes. NK cell transcriptomic signature suggests
lower CD56<sup>bright</sup> and CD56<sup>dim</sup> NK cell frequencies in COVID-19
patients. NK cells from patients have increased immune checkpoint
(Lag-3, Tim-3) and activation marker transcripts and decreased
maturation and cytotoxicity transcripts (CD16, Ksp-37, granulysin).
Granulysin transcripts are also decreased in CD8 T cells, yet immune
checkpoint transcripts remain unchanged in both CD8 and CD4 T cells upon
SARS-CoV2 infection. The frequencies of memory and naïve CD4 and CD8 T
cell subsets seem unchanged upon disease, though γδT cell proportions
are decreased. SARS-CoV2 infection also induces expansion of IgA and IgG
plasmablasts that do not share Ig V genes.</p>
<p>Interferon-signaling genes (ISGs) are upregulated in the monocyte, the
NK and the T cell compartment in a donor-dependent manner. ISG
transcripts in the monocytes tend to increase with the age, while
decreasing with the time to onset disease. No significant cytokine
transcripts are expressed by the circulating monocytes and IFNG, TNF,
CCL3, CCL4 transcript levels remain unchanged in NK and T cells upon
infection.</p>
<p><strong>Limitations:</strong></p>
<p>The sample size of the patients is limited (n=7) and gender-biased, as
all of them are men.</p>
<p>The activating and resting signatures in monocytes should be further
detailed. The authors did not detect IL1B transcripts in monocytes from
the patients, though preliminary studies suggest increased frequencies
of CD14+ IL1B+ monocytes in the blood of convalescent COVID-19
patients[1].</p>
<p>Decreased NK cells, B cells, DCs, CD16+ monocytes and γδT cells observed
in peripheral blood might not only reflect a direct SARS-CoV2-induced
impairment, but also the migration of these cells to the infected lung,
in line with preliminary data suggesting unchanged NK cell frequencies
in the patient lungs[2].</p>
<p>The authors identified platelets in their cluster analyses. Recent
reports of pulmonary complications secondary to COVID-19 describe
thrombus formation that is probably due, in part, to platelet
activation[3, 4]. A targeted characterization of the platelet
transcriptome may thus benefit an increased understanding of this
phenomenon.</p>
<p>The transcriptome of the Activated Granulocytes should be further
detailed. As discussed by the authors, IL24 and EGF might be involved in
the generation of the Activated Granulocytes, though these cytokines are
poorly represented in the blood of the patients. The generation of these
cells should therefore be further investigated in future studies.</p>
<p><strong>Significance:</strong></p>
<p>The authors show a SARS-CoV2-induced NK cell dysregulation, in
accordance with previous studies[5]. Alongside the upregulation of
ISGs in NK cells, these findings suggest an impaired capacity of the NK
cells to respond to activating signals in COVID-19 patients. The
unchanged expression of immune checkpoints on CD4 and CD8 T cells
suggest distinct SARS-CoV2 dysregulation pathways in the NK and the T
cell compartments. In particular, the downregulation of transcripts
encoding for class II HLA but not for the HLA-A, -B, -C molecules in
monocytes suggest an impaired antigen presentation capacity to CD4 T
cells, which should be further investigated.</p>
<p>The authors provide preliminary results suggesting an age-related
activation of the monocytes in the COVID-19 patients. Future studies
will be needed to evaluate if the age impacts the involvement of the
monocytes in the cytokine storm observed in COVID-19 patients.</p>
<h3 data-number="9.193.1" id="references-1"><span class="header-section-number">9.193.1</span> <strong>References:</strong></h3>
<p>1. Wen, W., et al., <em>Immune Cell Profiling of COVID-19 Patients in the
Recovery Stage by Single-Cell Sequencing.</em> MedRxiv, 2020.</p>
<p>2. Liao, M., et al., <em>The landscape of lung bronchoalveolar immune cells
in COVID-19 revealed by single-cell RNA sequencing</em>. 2020.</p>
<p>3. Giannis, D., I.A. Ziogas, and P. Gianni, <em>Coagulation disorders in
coronavirus infected patients: COVID-19, SARS-CoV-1, MERS-CoV and
lessons from the past.</em> J Clin Virol, 2020. <strong>127</strong>: p. 104362.</p>
<p>4. Dolhnikoff, M., et al., <em>Pathological evidence of pulmonary
thrombotic phenomena in severe COVID-19.</em> J Thromb Haemost, 2020.</p>
<p>5. Zheng, M., et al., <em>Functional exhaustion of antiviral lymphocytes in
COVID-19 patients.</em> Cell Mol Immunol, 2020.</p>
<p><strong>Credit:</strong></p>
<p><em>Reviewed by Bérengère Salomé and Zafar Mahmood as part of a project by
students, postdocs and faculty at the Immunology Institute of the Icahn
School of Medicine, Mount Sinai.</em></p>
<p>﻿<strong>National Consumption of Antimalarial Drugs and COVID-19 Deaths
Dynamics: An Ecological Study</strong></p>
<p>M. Izoulet, <em>medRxiv</em> [<span class="citation" data-cites="doi">[<span class="citeproc-not-found" data-reference-id="doi"><strong>???</strong></span>]</span>: 10.1101/2020.04.18.20063875]</p>
<p><span id="keywords" class="anchor"></span></p>
<p><em>Keywords</em></p>
<ul>
<li><p>hydroxychloroquine</p></li>
<li><p>modeling</p></li>
<li><p>ecological fallacy</p></li>
</ul>
<p><span id="main-findings" class="anchor"></span></p>
<p><em>Main Findings</em></p>
<p>This preprint sought to compare the daily deaths in countries using
CQ/HCQ as a treatment from the beginning of the COVID-19 pandemic to
those that did not. From a list of 60 countries in descending order by
number of confirmed cases, 16 countries were selected for inclusion into
either the high CQ/HCQ production or use group, versus not. Countries
were included if they met the criteria for having data from the day of
the 3<sup>rd</sup> death in the entire country and the daily deaths for the 10
days immediately following, until both groups were populated with a list
of 16 (Figure 1: Table with the CQ/HCQ group list; Figure 2: Table with
the “control” group list). For each group of countries, the average
daily deaths were determined, and the curves projected to illustrate
trajectories. In Figure 3, the author suggests that the deaths in the
countries belonging to the control group follow an exponential curve,
while the progression of average daily deaths in the countries with
greater use of CQ/HCQ follow a polynomial curve.</p>
<p>The author then applies Auto Regressive Integrated Moving Average
(ARIMA), a modeling tool used for time-series forecasting (i.e.,
predicting the future trajectory of data over time using the data from
previous time points as predictors in a linear regression). The Auto
Regressive component refers to each difference between two previous time
points that make the model “stationary” (current – previous); the Moving
Average is the number of forecast errors from calculating these
differences that should go in the model. The author uses ARIMA to
predict the next 10 days of mean deaths for the CQ/HCQ list (Figure 6)
and the control countries (Figure 8). In figures 9 and 10,
autocorrelations of residuals are performed to determine internal
validity of the model, here defined as no significant autocorrelations.</p>
<p>In conjunction, the author argues that these findings support major
differences in death rates between countries that use/mass produce
CQ/HCQ versus those that do not.</p>
<p><span id="limitations" class="anchor"></span><em>Limitations</em></p>
<p>The title of this study refers to itself as an ecological study, an
observational study in which the data are defined at the population
level, rather than individual. Although this study design allows for
rapid hypothesis testing in large datasets, a robust ecological study
should account for as many known risk-modifying factors or confounders
as possible. Subsequently, any results should be reviewed under strict
criteria for causality, since there is high probability of the outcomes
falling under the definition of ecological fallacy, which occurs when
inferences about individuals are determined from inferences about a
group to which they belong.</p>
<p>This study conflated the use and mass production of CQ/HCQ at the start
of the COVID-19 pandemic in each respective country, with that country’s
direct pandemic response. It is never explained whether use or
production is the key output for any given country, which are vastly
different metrics. The author fails to consider other reasons for having
existing infrastructure for the mass production of drugs like
hydroxychloroquine, whether the country was a global supplier of the
medication (India), or is a region where malaria is endemic (India,
Pakistan, Indonesia, Malaysia, South Korea), which may correlate to both
chloroquine production and use. Notably, the countries from which
studies of HCQ in the treatment of COVID-19 have been predominantly
performed (China, France, USA), are all in the control list of
countries. Additionally, the data for cases and deaths were collected
from reports accessed from <a href="https://www.worldometers.info/coronavirus/" class="uri">https://www.worldometers.info/coronavirus/</a>;
data were not selected from the top countries using a methodological
approach, but rather skipping certain countries to use only the most
complete death data for the timeframe of interest, allowing for bias
introduced by the reporting of each individual country.</p>
<p>With regards to the statistical methods applied, namely ARIMA, they are
non-standard practices for interpreting the results of an ecological
study. The first problem with this, in my opinion, is that the message
will be difficult to interpret and criticize for many scientists, as
ARIMA will be unfamiliar to most in the biological sciences. Further,
the models applied (Table 4) do not take into account any confounders,
which is a requirement for robust analysis of an ecological study. There
are only 3 variables in this type of model: p, the autoregressive
coefficient, q, the moving average coefficient, and d, the difference
between points in the time-series. Any flaws or bias inherent to the
input data are then upheld and propagated by the model, which does not
allow for any other variable that would contribute to the risk of death
(say, honest and complete reporting by country).</p>
<p><em>Significance</em></p>
<p>The faults of the stratification of countries into the groups proposed
in this study, together with the unorthodox application of ARIMA
modeling, make it challenging to accept the conclusion that the author
draws in this study. The apparent decrease in death rate in countries
with a high production/use/either/or of CQ/HCQ could be due to any
number of other factors for which this study did not account. The top 5
countries in both confirmed cases and reported deaths are all in the
control list, which has no relationship to the amount of CQ/HCQ
production within those countries yet skews the data to make the
dynamics of death rate appear more dramatic.</p>
<p><em>Reviewed by Rachel Levantovsky as part of a project by students,
postdocs and faculty at the Immunology Institute of the Icahn School of
Medicine at Mount Sinai.</em></p>
<p><strong>Title:</strong> Shotgun Transcriptome and Isothermal Profiling of SARS-CoV-2
Infection Reveals Unique Host Responses, Viral Diversification, and Drug
Interactions</p>
<p>Butler <em>et al.</em> [bioRxiv, <span class="citation" data-cites="doi">[<span class="citeproc-not-found" data-reference-id="doi"><strong>???</strong></span>]</span>: 10.1101/2020.04.20.048066]</p>
<p><strong>Key words:</strong> loop-mediated isothermal amplification (LAMP), RNA-Seq,
ACE2 expression, qRT-PCR</p>
<p><strong>Summary:</strong></p>
<p>The SARS-CoV-2 pandemic has created an urgent need for widely available
diagnostic tools to detect and understand host immune responses. This
study introduces a fast SARS-CoV-2 detection test from swabs, based on
colorimetric reverse transcriptase loop-mediated isothermal
amplification (RT-LAMP) assay technology, which amplifies viral RNA.
Results were validated using both qRT-PCR and total RNA sequencing. The
study also introduces a new large-scale total RNA sequencing platform
which enables comprehensive transcriptome profiling of the virus, host
and other microbiomes. Both technologies were used to analyze samples
from 735 confirmed/suspected COVID-19 patients, as well as 62
environmental swabs from NYC subway.</p>
<p>Through a viral evolution map based on the total RNA-sequencing data,
the study identified a novel subclade (A2-25563) enriched in NYC cases
with likely origin in western Europe. Overall, the host transcriptome
profiles revealed 216 qRT-PCR positive and 519 negative samples.
Integrative analysis together with the viral load levels revealed that
samples with high viral load exhibit increased expression of ACE2
(SARS-CoV-2 receptor), interferon pathway (SHFL, IFI6, IFI27, and IFIT1)
and HERC6 gene and decreased expression of olfactory receptor pathway
and TMPRSS-11B gene. Since ACE2 expression can be increased by
pharmacologic intake of angiotensin enzyme inhibition (ACEI) or
angiotensin receptor blockers (ARB), additional analysis of an
observational cohort of 8,856 suspected COVID-19 patients showed a
strong positive association between ACEI/ARB usage and SARS-CoV-2
infection.</p>
<p><strong>Limitations:</strong></p>
<p>While the host transcriptome analyses were performed on individuals who
tested positive and negative for SARS-CoV-2 infection and other
subgroups based on viral load, the influence of other clinical
characteristics/phenotypes (e.g. age, symptoms, severity) were not
discussed. It would also been useful to have the characteristics of the
735 confirmed/suspected COVID-19 patients.</p>
<p><strong>Findings implication:</strong></p>
<p>During this pandemic, large-scale rapid testing is key in tracking,
surveillance and containment of the viral spread. To aid diagnostics,
this study identified a fast LAMP assay that was applied to both naso/
oropharyngeal swabs and environmental samples and validated against gold
standard qRT-PCR approach. Furthermore, total RNA-Seq profiling enabled
the pursuit of multiple aspects of virus genetic features and evolution,
host response, co-infections and other possible correlations between
virus/host genotype and clinical phenotype. This is one of the first
studies based on sequencing data from SAR-CoV-2 patients and will serve
as an important resource to the research community.</p>
<p><em>Reviewed by Myvizhi Esai Selvan as part of a project by students,
postdocs and faculty at the Immunology Institute of the Icahn School of
Medicine, Mount Sinai.</em></p>
<p><strong>TMPRSS2 and TMPRSS4 mediate SARS-CoV-2 infection of human small
intestinal enterocytes</strong></p>
<p>Zang R. et al.  [<span class="citation" data-cites="doi">[<span class="citeproc-not-found" data-reference-id="doi"><strong>???</strong></span>]</span>: 10.1101/2020.04.21.054015]</p>
<h3 data-number="9.193.2" id="keywords-195"><span class="header-section-number">9.193.2</span> Keywords</h3>
<ul>
<li><p>TMPRSS2</p></li>
<li><p>scRNAseq</p></li>
<li><p>tropism</p></li>
<li><p>enteroid</p></li>
<li><p>intestine</p></li>
<li><p>fecal</p></li>
</ul>
<h3 data-number="9.193.3" id="main-findings-188"><span class="header-section-number">9.193.3</span> Main Findings</h3>
<ul>
<li><p>Study uses scRNAseq to profile viral-entry genes in intestinal cell
types and show confirmatory result of predominant ACE2 expression in
CD26+EPCAM+ mature enterocytes. The transcriptomics also map
TMPRSS2, TMPRSS4 and ST14 protease gene expression across mature
enterocytes and other enteric cells. They additionally show ACE2
protein localizing to the brush border of the intestinal epithelium.</p></li>
<li><p>Researchers use VSV chimera reporter virus with SARS-CoV-2 S protein
to assay viral tropism in culture and demonstrate active replication
in human duodenal enteroids as well as apical mature enterocytes
(ACE2+) in monolayer system.</p></li>
<li><p>Used HEK293 with ectopic expression of proteases to show that
TMPRSS2 or TMPRSS4 co-expression with ACE2 enhanced viral
infectivity by enhancing endocytosis. CRISPR-Cas9 based deletion or
pharmacological inhibition of TMPRSS proteases in human primary
enteroids limited viral replication, more so upon TMPRSS4 deletion.</p></li>
<li><p>Further, a co-culture system designed with ACE2/TMPRSS2/4 expressing
host cells and S-protein expressing donor cells, to mimic enteroid
cell population interactions. Importantly found that TMPRSS2 can
influence viral entry in trans through paracrine interaction, unlike
TMPRSS4 that functions in cis.</p></li>
<li><p>SARS-CoV-2 chimera virus were very unstable in colonic fluids,
compared to rotaviruses that were stable in gastric and colonic
fluids. Supporting these data, CoVID-19 patient fecal sample
analysis also consistently showed viral RNA load, but no infectious
particles.</p></li>
</ul>
<h3 data-number="9.193.4" id="limitations-190"><span class="header-section-number">9.193.4</span> Limitations</h3>
<ul>
<li><p>The studies use pseudotyped-VSV reporter virus instead of WT virus,
which could potentially have an impact on tropism. It will be
important to replicate some of the co-culture studies using active
SARS-CoV-2 virus as confirmation. Similarly, ectopic expression and
co-culture experiments can be strengthened by replicating studies in
primary cells.</p></li>
<li><p>Study mentions lack of infectious particle in fecal samples, but no
conclusive data presented to support this important point.</p></li>
</ul>
<h3 data-number="9.193.5" id="significance-179"><span class="header-section-number">9.193.5</span> Significance</h3>
<ul>
<li><h3 id="tmprss4-important-for-influenza-virus-tropism-but-not-previously-shown-to-be-important-in-cov-2-infectivity-in-the-gut-context.">TMPRSS4 important for influenza virus tropism, but not previously shown to be important in CoV-2 infectivity in the gut context.</h3></li>
<li><h3 id="co-culture-experiments-imply-that-paracrine-interaction-of-tmprss2-from-goblet-cells-for-example-on-neighboring-tmprss4ace2-expressing-enteric-cells-can-drive-synergistic-increase-in-viral-infectivity-in-gut-epithelia.-whether-such-gene-expression-distribution-and-protein-interactions-exist-in-other-tissue-contexts-lung-kidney-remains-to-be-seen.">Co-culture experiments imply that paracrine interaction of TMPRSS2 (from goblet cells for example) on neighboring TMPRSS4/ACE2 expressing enteric cells can drive synergistic increase in viral infectivity in gut epithelia. Whether such gene expression distribution and protein interactions exist in other tissue contexts (lung, kidney) remains to be seen.</h3></li>
<li>Fecal-oral transmission of SARS-CoV-2 has not been broadly observed,
but intestinal infection of ACE2++ cells has been reported
clinically; this study reconciles these paradoxical observations by
demonstrating that despite active infection and replication in
enterocytes the virus might not remain intact in colonic fluids.
Conversely, observed SARS-CoV-2 stability in small intestine might
play a role in clinically presented GI pathologies.</li>
</ul>
<p><em>Reviewed by Samarth Hegde as part of a project by students, postdocs
and faculty at the Immunology Institute of the Icahn School of Medicine,
Mount Sinai.</em></p>
<p><strong>STAT2 signaling as double-edged sword restricting viral dissemination
but driving severe pneumonia in SARS-CoV-2 infected hamsters</strong></p>
<p>Boudewijns et al., bioRxiv [10.1101/2020.04.23.056838]</p>
<p>Keywords:</p>
<ul>
<li><p>STAT2</p></li>
<li><p>SARS-CoV-2</p></li>
<li><p>animal model</p></li>
<li><p>transgenic hACE2</p></li>
<li><p>Syrian hamster</p></li>
</ul>
<p><em>Main findings</em></p>
<p>In this report, the authors provide an evaluation of available animal
models, based on their ability to recapitulate characteristics of
COVID-19 as seen in humans. As of now, both non-human primate models and
transgenic hACE2 mice have limitations. An immediate type I IFN
response, for one, has been notable in these models, even though the
dysregulated inflammatory response seen in SARS-CoV-2 infection in
humans is significant in its delayed IFN response.</p>
<p>Here, the authors describe the efficiency of SARS-CoV-2 viral
replication and viral load in wild-type (WT), <em>Ifnar1</em><sup>-/-</sup>, and
<em>Il28r</em><sup>-/-</sup> C57BL/6 mice. Results showed that wild-type mice that do
not bear the human ACE2 receptor can be infected with SARS-CoV-2,
“although inefficiently and likely transiently.” Comparison with mice
without the type I IFN receptor and the type III IFN receptor showed
that all strains presented with a mild lung pathology that did not
resemble COVID-19 bronchopneumonia; notably, though, <em>Ifnar1</em><sup>-/-</sup> mice
did present with increased levels of inflammatory infiltration, compared
to the WT strain, suggesting that the active and functional IFN response
in WT mice restricts pathogenesis. Hence, the disease is not fully
replicable in mice.</p>
<p>The authors then propose that Syrian hamsters may be a better model,
given previous reports that they are highly susceptible to SARS-CoV-2.
WT, <em>STAT2</em><sup>-/-</sup> (loss of type I and III IFN signaling), and
<em>Il28r</em><sup>-/-</sup> (loss of type III IFN signaling) hamsters were intranasally
inoculated with SARS-CoV-2. WT hamsters carried high viral loads and
infectious titers in the lungs, and histological analysis revealed
necrotizing bronchiolitis, massive immune infiltration, and edema. These
findings reflect the histopathological characteristics of human
bronchopneumonia seen in COVID-19.</p>
<p>Interestingly, while viral RNA levels were not significantly different
in the lungs of these hamsters, titers of infectious virus in the lung
and viremia were highest in the <em>STAT2</em><sup>-/-</sup> hamsters, which suggests
STAT2 plays a role in deterring viral dissemination. In fact, these
hamsters had the highest levels of viral RNA in the spleen, liver, and
upper and lower GI tracts. Still, <em>Il28r</em><sup>-/-</sup> hamsters exhibited
bronchopneumonia and lung inflammation, actually to a greater extent
than <em>STAT2</em><sup>-/-</sup> hamsters, suggesting that type III IFN indeed plays an
important, protective role in restricting pathogenesis and the inclusion
of a type I IFN exacerbates bronchopneumonia. Though, IL-6, IL-10, and
IFN-ɣ were not remarkably high in the serum of infected hamsters, IL-6
and IL-10 were elevated in the lungs of infected <em>STAT2</em><sup>-/-</sup> and
<em>Il28r</em><sup>-/-</sup> hamsters.</p>
<p><em>Limitations</em></p>
<p>Without comparing the aforementioned three murine strains to a
transgenic hACE2 mouse strain as a “control”, it is difficult to
properly assess the significance of viral replication and load in the
non-transgenic mice. A similar limitation can be identified in the
experiments involving the Syrian hamsters. A transgenic model with
hamsters would provide additional evidence to suggest that viral
replication in these animals in the absence of a human version of the
ACE2 receptor is not happenstance and is truly sufficient to reflect
COVID-19 disease. Or, alternatively, a transient knockout of the ACE2
receptor in hamsters before challenging them with SARS-CoV-2 to assess
for signs of reduced viral infection may be appropriate.</p>
<p>Another experiment that is absent in this study is the use of antibodies
against SARS-CoV-2 components required for viral entry as a means of
immunizing Syrian hamsters before challenging them with the virus. This,
followed by immunofluorescence or immunohistochemistry and flow
cytometry for ACE2-expressing cells, would provide further validity to
the concept of SARS-CoV-2 actually invading and replicating inside
ACE2<sup>+</sup> cells in Syrian hamsters, which has been described for
SARS-CoV-1, but not yet for SARS-CoV-2.</p>
<p><em>Significance</em></p>
<p>Ongoing studies have identified notable limitations to the use of mice
to recapitulate human COVID-19 following SARS-CoV-2 infection. Lung
pathology is not significant. Moreover, viral dissemination to
extrapulmonary organs, including the GI tract or the liver, and viremia
are not observed. Therefore, to address this unmet need for an improved
animal model for SARS-CoV-2 infection, the present study proposed the
use of golden Syrian hamsters, which have been previously described to
better reflect SARS-CoV-1 infection than mice. The histopathological
findings do provide convincing evidence to suggest that these animals
may indeed be superior than mice to also study SARS-CoV-2 infection and
COVID-19, but the role of STAT2 - as suggested by the experiments
involving Syrian hamsters - remains to be further elucidated. The
precise cellular mechanism for viral dissemination throughout the
vasculature and to other peripheral organs is not clear, so the supposed
dual impact of STAT2 deletion in mice or in hamsters is still phenotypic
observation.</p>
<p><em>Reviewed by Matthew D. Park as part of a project by students, postdocs,
and faculty at the Immunology Institute of the Icahn School of Medicine,
Mount Sinai.</em></p>
<p><strong>SUCCESSFUL MANUFACTURING OF CLINICAL-GRADE SARS-COV-2 SPECIFIC T CELLS
FOR ADOPTIVE CELL THERAPY</strong></p>
<p>Leung Wing et al.; medRxiv  2020.04.24.20077487;
https://doi.org/10.1101/2020.04.24. 20077487</p>
<p><strong><em>Keywords</em></strong></p>
<ul>
<li><p>SARS-CoV-2 specific T cells</p></li>
<li><p>Adoptive T cell transfer</p></li>
<li><p>COVID-19</p></li>
</ul>
<p><strong><em>Main findings </em></strong></p>
<p>In this preprint, Leung et al. report the isolation of
SARS-CoV-2-specific T cells from two convalescent COVID-19 donors (n=1
mild, n=1 severe; both Chinese Singapore residents), using Miltenyi
Biotec’s fully automated CliniMACS Cytokine Capture System: convalescent
donor PBMCs were stimulated with MHC class I and class II peptide pools,
covering immunodominant sequences of the SARS-CoV-2 S protein as well as
the complete N and M proteins; next, PBMCs were labeled with a
bi-specific antibody against human CD45, a common leukocyte marker
expressed on white blood cells, as well as against human IFN-γ,
capturing T cell-secreted IFN-γ in response to stimulation with
SARS-CoV-2 peptides. Post stimulation, IFN-γ<sup>+</sup> CD45<sup>+</sup> cells were
identified by a mouse anti-human IFN-γ antibody, coupled to
ferromagnetic dextrane microbeads, and magnetically labeled cells were
subsequently isolated by positive immunomagnetic cell separation.
Enriched IFN-γ<sup>+</sup> CD45<sup>+</sup> cells were mostly T cells (58%-71%;
CD4&gt;CD8), followed by smaller fractions of B (25-38%) and NK cells
(4%). Up to 74% of T cells were found to be IFN-γ<sup>+</sup>, and 17-22% of T
cells expressed the cytotoxic effector marker CD56. Very limited
phenotyping based on CD62L and CD45RO expression identified the majority
of enriched CD4 and CD8 T subsets as effector memory T cells. TCR
spectratyping of enriched T cells further revealed an oligoclonal TCR ß
distribution (vs. a polyclonal distribution pre-enrichment), with
increased representation of Vß3, Vß16 and Vß17. Based on limited
assumptions about HLA phenotype frequencies as well as estimated
haplotype sharing among Chinese Singaporeans, the authors suggest that
these enriched virus-specific T cells could be used for adoptive cell
therapy in severe COVID-19 patients.</p>
<p><strong><em>Limitations </em></strong></p>
<p>This preprint reports the technical adaptation of a previously described
approach to isolate virus-specific T cells for targeted therapy in
patients with CMV and EBV (reviewed by Houghtelin A et al.:
<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5641550/pdf/fimmu-08-01272.pdf"><em>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5641550/pdf/fimmu-08-01272.pdf</em></a>)
to PBMCs obtained from two convalescent COVID-19 patients. However, not
only is the title of this preprint misleading - no adoptive cell
transfer was performed -, but this study also lacks relevant information
- among others - on technical details such as the respective S epitopes
studied, on the precise identification of immune cell subsets (e.g. NK
cells: CD56<sup>+</sup> CD3<sup>-</sup>?), data pertaining to technical stimulation
controls (positive/negative controls used for assay validation and
potentially gating strategies), as well as on the percentage of live
enriched IFN-γ<sup>+</sup> CD45<sup>+</sup> cells. Generally, a more stringent
phenotypical and functional characterization (including coexpression
data of CD56 and IFN-γ as well as activation, effector, proliferation
and other markers) would be advisable. Similarly, in its current
context, the TCR spectratyping performed here remains of limited
relevance. Most importantly, though, as noted by the authors themselves,
this study is substantially impaired based on the inclusion of only two
convalescent donors from a relatively homogenous genetic population as
well as lack of any potential recipient data. In related terms, clinical
criteria, implications and potential perils of partially HLA-matched
cell transfers are generally not adequately addressed by this study and
even less so in the novel COVID-19 context.</p>
<p><strong><em>Significance</em></strong></p>
<p>Adoptive cell therapy with virus-specific T cells from partially
HLA-matched third-party donors into immunocompromised recipients has
been successfully performed in the past (cf.
<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5641550/pdf/fimmu-08-01272.pdf"><em>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5641550/pdf/fimmu-08-01272.pdf</em></a>;
<a href="https://www.jci.org/articles/view/121127"><em>https://www.jci.org/articles/view/121127</em></a>).
However, whether this approach might be clinically feasible for COVID-19
therapy remains unknown. Therefore, larger, more extensive studies
including heterogeneous patient populations are needed to assess and
balance potential risks vs. outcome in the new context of COVID-19.</p>
<p><em>This review was undertaken by V. van der Heide as part of a project by
students, postdocs and faculty at the Immunology Institute of the Icahn
school of medicine, Mount Sinai.</em> </p>
<p> </p>
<h2 data-number="9.194" id="title"><span class="header-section-number">9.194</span> Title</h2>
<p><strong>Disparate temperature-dependent virus – host dynamics for SARS-CoV-2
and SARS-CoV in the human respiratory epithelium</strong><span id="keywords" class="anchor"></span></p>
<p>V’kovski et al., bioRxiv/2020.04.27.062315
[https://doi.org/10.1101/2020.04.27.062315]</p>
<p><em>Keywords</em></p>
<ul>
<li><p>Upper and lower respiratory tracts</p></li>
<li><p>Temperature</p></li>
<li><p>Interferons</p></li>
</ul>
<p><span id="main-findings" class="anchor"></span><em>Main Findings</em></p>
<p>The anatomical distance between the upper and lower respiratory tracts,
as well as their different ambient temperatures (32-33°C and 37°C,
respectively) could influence viral replication and subsequent immune
responses, as previously reported in studies with influenza and
rhinovirus infection [1, 2]. By employing an <em>in vitro</em> model of
SARS-CoV-2 and SARS-CoV infection of human airway epithelial cells
(hAEC) originated from three different donors, this study aimed to
investigate the effect of temperature on virus-host innate immune
response dynamics in the epithelial lining of the human respiratory
tract.</p>
<p>The authors show comparable SARS-CoV and SARS-CoV-2 replication at 37°C.
However, when assessing viral replication efficiency at 33°C, SARS-CoV-2
infection resulted in 100-fold higher viral titers compared SARS-CoV, or
SARS-CoV-2 replication at 37°C. Data in transcriptional signature after
infection revealed that at 37°C, SARS-CoV-2-infection triggered earlier
and greater changes in the transcriptional program compared to infection
at 33°C. Among the differentially expressed genes (DEGs) after
SARS-CoV-2-infection, the authors identified immunological mediators,
such as chemokines and cytokines (<em>CXCL10, CXCL11, TNFSF13B, and
CCL5/RANTES</em>), Interferon Stimulating Genes (ISG, such as <em>OASL</em>), as
well as type I and III interferon genes. The authors tested SARS-CoV-and
SARS-CoV-2 replication sensitivity to exogenous type I or type III
interferon (IFN-I/III), and found a drastic reduction in viral load
after hAEC infection.</p>
<p>Altogether, these results suggest a temporal, temperature-dependent gene
expression profile that is inversely associated with SARS-CoV-2
replication which may account for the high replicative capacity of
SARS-CoV-2 in the upper airways.</p>
<p><span id="limitations" class="anchor"></span><em>Limitations</em></p>
<p>There is considerable variability between donors with regards to viral
replication at 37°C and 33°C (specially in Figure 1a), and it is unclear
how many times experiments were performed.</p>
<p>The observation that temperature influences both viral replication and
innate immune responses is essentially correlative. Further functional
studies evaluating whether the temperature-dependent viral replication
is a direct consequence of a delayed interferon response will be
required. The variability between samples may also explain the fact that
authors didn’t detect any DEGs in hAEC infected with SARS-CoV.</p>
<p>It was recently proposed that ACE2 can be an ISG [3] induced
specifically by IFNa, favoring SARS-CoV-2 cell entry. It would be
interesting to indicate the level of ACE2 expression after IFNs
exogeneous treatment of hAEC in the different culture temperatures.</p>
<p>Moreover, it would be important to perform immunofluorescence analysis
showing colocalization of ACE2 and SARS nucleocapsid protein.</p>
<p><em>Significance</em></p>
<p>This study provides insights about the impact of microenvironmental
temperature on virus-host interactions. Enhanced replication of
SARS-CoV-2 at 33°C, may influence its replication in the upper
respiratory tract and transmissibility in comparison to SARS-CoV.</p>
<p><span id="credit" class="anchor"></span><em>Credit</em></p>
<p><em>Reviewed </em>by Alessandra Soares-Schanoski* as part of a project by
students, postdocs and faculty at the Immunology Institute of the Icahn
School of Medicine, Mount Sinai.*</p>
<p><em>References</em></p>
<p>1. Holwerda, M., et al., <em>Determining the Replication Kinetics and
Cellular Tropism of Influenza D Virus on Primary Well-Differentiated
Human Airway Epithelial Cells.</em> Viruses, 2019. <strong>11</strong>(4).</p>
<p>2. Foxman, E.F., et al., <em>Temperature-dependent innate defense against
the common cold virus limits viral replication at warm temperature in
mouse airway cells.</em> Proc Natl Acad Sci U S A, 2015. <strong>112</strong>(3): p.
827-32.</p>
<p>3. Carly G.K. Ziegler, S.J.A., Sarah K. Nyquist, Ian M. Mbano, Vincent
N., et al., <em>SARS-CoV-2 receptor ACE2 is an interferon-stimulated gene
in human airway epithelial cells and is detected in specific cell
subsets across tissues.</em> Cell, 2020.</p>
<p><strong>A possible role of immunopathogenesis in COVID-19 progression</strong></p>
<p>Anft M., Paniskaki K, Blazquez-Navarro A t al.; medRxiv 
2020.04.28.20083089; https://doi.org/10.1101/2020.04.28.20083089</p>
<p><strong><em>Keywords</em></strong></p>
<ul>
<li><p>SARS-CoV-2 spike protein-specific T cells</p></li>
<li><p>COVID-19</p></li>
<li><p>adaptive immunity</p></li>
</ul>
<p><strong><em>Main findings </em></strong></p>
<p>In this preprint, 53 hospitalized COVID-19 patients, enrolled in a
prospective study at a tertiary care center in Germany, were assigned to
moderate (n=21; light pneumonia), severe (n=18; fever or respiratory
tract infection with respiratory rate &gt;30/min, severe dyspnea, or
resting SpO<sub>2</sub> &lt;90%), and critical subgroups (n=14; ARDS, sepsis, or
septic shock) according to clinical disease. Moderately and severely ill
patients with a PCR-confirmed diagnosis were recruited within four days
of clinical onset, whereas critically ill patients were enrolled on
average within 14 days of diagnosis on admission to ICU. To account for
the overall longer hospital stay in ICU cases prior to inclusion,
repeated blood samples were obtained from moderately and severely ill
donors within eight days post recruitment. For 10 out of 14 ICU
patients, no follow up blood samples were collected. At recruitment as
well as on follow-up, circulating lymphocyte counts were below reference
range in the majority of enrolled COVID-19 patients. Relative
frequencies were significantly reduced in critically <em>vs</em>. moderately,
but not <em>vs</em>. severely ill individuals, with substantially lower NK as
well as CD8 T cells counts, and a concomitant increase of the CD4:CD8 T
cell ratio in ICU patients. Basic phenotypic and immune cell subset
analysis by flow cytometry detected lower frequencies of central memory
CD4 T cells as well as reduced terminally differentiated CD8 Temra cells
in critical COVID-19. Moreover, a decrease in activated HLA-DR<sup>+</sup> CD4
and CD8 T cells as well as in cytolytic CD57<sup>+</sup> CD8 T cells was observed
in critical <em>vs</em>. severe/moderate disease. Similarly, frequencies of
CD11a<sup>+</sup> CD4 and CD8 T cells as well as CD28<sup>+</sup> CD4 T cells were lower
in critically ill donors, indicating a general loss of activated bulk T
cells in this subgroup. In addition, a reduction of both marginal and
transitional CD19<sup>+</sup> B cells was seen in patients with severe and
critical symptoms. Of note, on follow-up, recovering severe COVID-19
patients showed an increase in bulk T cell numbers with an activated
phenotype. Importantly, SARS-CoV-2 spike (S)-protein-specific CD4 and
CD8 T cells, identified following stimulation of PBMCs with 15-mer
overlapping S protein peptide pools by flow-cytometric detection of
intracellular CD154 and CD137, respectively, were found in the majority
of patients in all COVID-19 subgroups at the time of recruitment and
further increased in most subjects by the time of follow-up (antiviral
CD4 &gt;&gt; CD8 T cells). Most notably, frequencies of both antiviral
CD4 and CD8 T cells were substantially higher in critically ill
patients, and virus specific CD4 and CD8 T cells in both critically and
severely ill subgroups were shown to produce more pro-inflammatory Th1
cytokines (TNFa, IFNg, IL-2) and the effector molecule GzmB,
respectively, suggesting an overall increased magnitude of
virus-specific T cell inflammation in the context of more severe disease
courses. Furthermore, frequencies of antiviral CD4 T cells correlated
moderately with anti-S-protein IgG levels across all patient groups.</p>
<p><strong><em>Limitations </em></strong></p>
<p>In general, this is a well executed study and most of the observations
reported here pertaining to overall reduced bulk T cell frequencies
(along with lower NK and other immune cell counts) as well as diminished
numbers of T cells with an activated phenotype in ICU <em>vs</em>. non ICU
COVID-19 corroborate findings in several previous publications and
preprints (cf. https://www.jci.org/articles/view/137244;
<a href="https://academic.oup.com/jid/advance-article/doi/10.1093/infdis/jiaa150/5813618" class="uri">https://academic.oup.com/jid/advance-article/doi/10.1093/infdis/jiaa150/5813618</a>;
<a href="https://www.nature.com/articles/s41423-020-0401-3" class="uri">https://www.nature.com/articles/s41423-020-0401-3</a>;
<a href="https://www.medrxiv.org/content/10.1101/2020.04.11.20062349v1.full.pdf" class="uri">https://www.medrxiv.org/content/10.1101/2020.04.11.20062349v1.full.pdf</a>;
<a href="https://www.medrxiv.org/content/10.1101/2020.04.17.20061440v1.full.pdf" class="uri">https://www.medrxiv.org/content/10.1101/2020.04.17.20061440v1.full.pdf</a>).
Notably, in contrast to many previous reports, the prospective study by
Anft et al. enrolled a relatively larger number of COVID-19 patients of
variable clinical disease (with the exception of mild cases). However,
there are a few weaknesses that should be addressed. Most importantly,
the choice of statistical tests applied should be carefully revised:
e.g. comparison of more than two groups, as seems to be the case for
most of the figures, requires ANOVA testing, which should ideally be
followed by post-hoc testing (despite the somewhat confusing statement
that this was conceived as an exploratory study). Given the overall
limited case numbers per clinical subgroup, trends even though they
might not reach statistical significance are equally important.
Similarly, some statements are overgeneralized and should to be adjusted
based on the actual data shown (e.g. the authors continue to refer to
gradual reductions of activated T cell subset numbers in moderately
<em>vs.</em> severely <em>vs</em>. critically ill patients, but for the majority of
data shown substantial differences are apparent only in ICU <em>vs</em>.
non-ICU patients). Moreover, it would be helpful to include exemplary
FACS plots in addition to explanatory gating strategies provided in the
supplemental document. There are also several inconsistencies regarding
the order of data presented here (e.g. in the main manuscript, Fig S5 is
chronological referred to before Fig S4) as well as pertaining to
relevant technical details (according to both the main manuscript and
the gating strategy in Figure S5, virus-specific CD4 T cells were
identified by CD154 expression; however, in figure legend S5
virus-specific CD4 T cells are defined as CD4<sup>+</sup> CD154<sup>+</sup> CD137<sup>+</sup>).
Additionally, from a technical point of view, it is somewhat intriguing
that the percentages of virus-specific T cells identified by expression
of CD154 and CD137, respectively, following peptide simulation seem to
differ substantially from frequencies of CD154<sup>+</sup> or CD137<sup>+</sup> INFg<sup>+</sup>
virus-specific T cells. Assuming a somewhat lower extent of cellular
exhaustion in the moderate COVID-19 group, one would expect these cell
subsets to mostly overlap/match in frequencies, therefore suggesting
slight overestimation of actual virus-specific T cell numbers. In this
context, inclusion of positive controls, such as CMV pp65 peptide
stimulation of PBMCs from CMV seropositive donors, in addition to the
already included negative controls would also be helpful. Moreover, in
view of the observation that virus-specific T cells were found to be
increased in critically ill ICU over non-ICU patients, a more stringent
characterization of these patients as well as assessment of potential
associations with clinical characteristics such as mechanical
ventilation or death would add further impact to the findings described
here. Finally, this study is limited to anti-S protein specific T cells.
However, evaluation of N and also M-protein specific CD8 T cell
responses are likely of great interest as well based on current
knowledge about persistent M-protein specific memory CD8 T cells
following SARS-CoV-1 infection (cf.
<a href="https://www.microbiologyresearch.org/content/journal/jgv/10.1099/vir.0.82839-0" class="uri">https://www.microbiologyresearch.org/content/journal/jgv/10.1099/vir.0.82839-0</a>).</p>
<p><strong><em>Significance</em></strong></p>
<p>In addition to reduced frequencies of activated bulk T cell numbers, the
authors report an enhanced virus-specific T cell response against S
protein epitopes in critically ill COVID-19 patients compared to
severely and moderately ill individuals, which correlated with anti-S
protein antibody titers (also cf. Ni et al.:
https://doi.org/10.1016/j.immuni.2020.04.023).</p>
<p>This is an important observation that mirrors previous data about
SARS-CoV-1 (cf. Ka-fai Li C et al.:
<a href="https://www.jimmunol.org/content/jimmunol/181/8/5490.full.pdf" class="uri">https://www.jimmunol.org/content/jimmunol/181/8/5490.full.pdf</a>).
Furthermore, in accordance with a recent preprint by Weiskopf et al.
(<a href="https://www.medrxiv.org/content/10.1101/2020.04.11.20062349v1.full.pdf" class="uri">https://www.medrxiv.org/content/10.1101/2020.04.11.20062349v1.full.pdf</a>),
virus-specific CD4 T cells were found to increase in most patients over
time regardless of clinical disease, whereas antiviral CD8 T cell
kinetics seemed slightly less pronounced. Moreover, in the majority of
moderately and severely ill cases, virus-specific T cells against the S
protein could be detected early on - on average within 4 days of symptom
onset. Longitudinal studies including larger numbers of COVID-19
patients across all clinical subgroups are therefore needed to further
evaluate the potential impact of this observation, in particular in the
context of previously described pre-existing memory T cells
cross-reactive against human endemic coronaviruses (cf.
<a href="https://www.medrxiv.org/content/10.1101/2020.04.17.20061440v1.full.pdf" class="uri">https://www.medrxiv.org/content/10.1101/2020.04.17.20061440v1.full.pdf</a>;
<a href="https://journals.sagepub.com/doi/pdf/10.1177/039463200501800312" class="uri">https://journals.sagepub.com/doi/pdf/10.1177/039463200501800312</a>).</p>
<p><em>This review was undertaken by V. van der Heide as part of a project by
students, postdocs and faculty at the Immunology Institute of the Icahn
school of medicine, Mount Sinai.</em> </p>
<p> </p>
<p>10.1101/2020.03.09.983247</p>
<p><strong>Title:</strong> Inhibition of SARS-CoV-2 infection (previously 2019-nCoV) by
a highly potent pan-coronavirus fusion inhibitor targeting its spike
protein that harbors a high capacity to mediate membrane fusion</p>
<p><strong>Keywords:</strong> SARS-CoV2, S protein, fusion inhibitor, hACE2</p>
<p><strong>Main findings:</strong></p>
<p>Members of the coronavirus family rely on fusion with the host’s cell
membrane during infection. The spike (S) protein plays an essential role
in this step (1). This work sheds light on the particularities of the S
protein from SARS-CoV2 and compares it to previously existing
coronaviral S proteins. SARS-CoV2 S protein showed stronger affinity for
hACE2 compared to SARS-CoV. Previous work by this group had established
the potential of a previous version of the peptide (EK1) in reducing the
infectivity of several coronavirus by blocking the S protein
interactions with the host cell (2). In this work, authors present an
improved EK1C4 version of this peptide. In vitro experiments on human
cells showed EK1C4-mediated reduction of the infectivity of SARS-CoV-2
and other coronavirus. Administration of EK1C4 in prophylaxis or shortly
after exposure to the HCoV-OC43 coronavirus protected newborn mice from
virus-induced lethality.</p>
<p><strong>Limitations:</strong></p>
<p>This work suggests that the furin cleavage site from CoV2-S could be
related to its increased infectivity compared to CoV-S. However, a
previous report suggested that the enhanced cell-to-cell fusion mediated
by this furin cleavage might not necessarily translate into enhanced
viral infectivity (3). The mutation of this site, or inhibition of furin
in 293T cells expressing SARS-CoV or CoV-2 S proteins, could inform if
this mechanism is actually involved in SARS-CoV2 pathogenesis, and
potentially suggest additional routes for therapeutic intervention.</p>
<p>The in vivo model used to address the protective potential of EK1C4 is
based on the coronavirus HCoV-OC43. Further work should aim to establish
the potential of EK1C4 for prevention of SARS-CoV-2 replication in more
relevant animal models (e.g. Rhesus Macaques or hACE2 mice). It is,
however, acknowledged, that experimentation in vitro included both
SARS-CoV2 and the same HCoV-OC43 used in the in vivo infection model.</p>
<p>Even though the safety of the unmodified EK1 peptide in mice had been
previously examined (2), the potential side effects and
pharmacokinetics/dynamics of EK1C4 in the in vivo setting are not
addressed in this report.</p>
<p><strong>Relevance:</strong></p>
<p>The development of prophylactic drugs against viral infection is a
paramount element of epidemic control, specially in the time before a
vaccine can be readily available. Similar strategies have been followed
in the past in response to threats of this nature (4,5). This work’s
relevance is limited by the absence of relevant animal models of
SARS-CoV-2 infection. Further pharmacological analyses would also be
necessary to ensure the safety of EK1C4 in animal models before a
potential safety analysis in human subjects.</p>
<p>1. Heald-Sargent T, Gallagher T. Ready, set, fuse! the coronavirus spike
protein and acquisition of fusion competence. Vol. 4, Viruses.
Multidisciplinary Digital Publishing Institute (MDPI); 2012. p. 557–80.</p>
<p>2. Xia S, Yan L, Xu W, Agrawal AS, Algaissi A, Tseng CTK, et al. A
pan-coronavirus fusion inhibitor targeting the HR1 domain of human
coronavirus spike. Sci Adv. 2019 Apr 1;5(4):eaav4580.</p>
<p>3. Follis KE, York J, Nunberg JH. Furin cleavage of the SARS coronavirus
spike glycoprotein enhances cell-cell fusion but does not affect virion
entry. Virology. 2006 Jul 5;350(2):358–69.</p>
<p>4. Jones JC, Turpin EA, Bultmann H, Brandt CR, Schultz-Cherry S.
Inhibition of Influenza Virus Infection by a Novel Antiviral Peptide
That Targets Viral Attachment to Cells. J Virol. 2006 Dec
15;80(24):11960–7.</p>
<p>5. Ahmed A, Siman-Tov G, Hall G, Bhalla N, Narayanan A. Human
antimicrobial peptides as therapeutics for viral infections. Vol. 11,
Viruses. MDPI AG; 2019.</p>
<p>Reduction of lymphocyte at early stage elevates severity and death risk
of COVID-19 patients: a hospital-based case-cohort study</p>
<p><strong>Keywords:</strong> COVID-19, lymphocyte, severity, organ injuries.</p>
<p><strong>Main findings:</strong> The aim of this study was to assess an association
between reduced blood lymphocyte counts at hospital admission and
prognosis of COVID-19 patients (n=192). The authors found:</p>
<ul>
<li><p>Patients with lymphopenia are more likely to progress to severe
disease or succumb to COVID-19 (32.1% of COVID-19 patients with
lymphocyte reduction died).</p></li>
<li><p>Reduction of lymphocytes mainly affects the elderly (&gt; 70 years
old).</p></li>
<li><p>Lymphocyte reduction is more prevalent in COVID-19 patients with
cardiac disease and pulmonary disease, patients with increase in the
chest CT score (key marker of lung injury) and a decrease in several
respiratory function markers (PaCO2, SpO2, oxygenation index).</p></li>
</ul>
<p><strong>Limitations of the study:</strong> Reduced blood lymphocyte counts with aging
have been known
(<a href="https://www.medrxiv.org/content/10.1101/2020.03.08.20031229v2"><em>https://www.medrxiv.org/content/10.1101/2020.03.08.20031229v2</em></a>)
<a href="https://onlinelibrary.wiley.com/doi/epdf/10.1111/sji.12413"><em>https://onlinelibrary.wiley.com/doi/epdf/10.1111/sji.12413</em></a>).
Therefore, it is not unexpected that a larger fraction of COV ID-19
patients above 70 years old have lower lymphocytes counts. Since age has
been reported to be a major factor that determines outcome for COVID-19,
lymphocyte counts and prognosis should have been adjusted by age.
Multivariate analysis to identify independent risk factors is lacking.</p>
<p><strong>Relevance:</strong> Previous studies demonstrated that SARS-CoV-2 infection
leads to a decrease of the T cell count. This study confirms these
results and shows that lymphocyte reduction mainly affects the elderly.
Lymphopenia was associated with disease severity as well as worse
prognosis. Future studies need to address if lymphopenia is a negative
predictive factor independent from age.</p>
<p>Review by Meriem Belabed as part of a project by students, postdocs and
faculty at the Immunology Institute of the Icahn school of medicine,
Mount Sinai.</p>
<p><strong>COVID-19 Critical Illness Pathophysiology Driven by Diffuse Pulmonary
Thrombi and Pulmonary Endothelial Dysfunction Responsive to
Thrombolysis</strong></p>
<p>Poor et al., medRxiv [10.1101/2020.04.17.20057125]</p>
<p>Keywords:</p>
<ul>
<li><p>COVID-19</p></li>
<li><p>pneumonia</p></li>
<li><p>thrombolysis</p></li>
<li><p>endothelium</p></li>
<li><p>platelet</p></li>
<li><p>D-dimer</p></li>
</ul>
<p><em>Main findings</em></p>
<p>This report describes the use of systemic tissue plasminogen activator
(tPA) to treat venous thromboembolism (VTE) seen in four critically ill
COVID-19 patients with respiratory failure. These patients all exhibited
gas exchange abnormalities, including shunt and dead-space ventilation,
despite well-preserved lung mechanics. A pulmonary vascular etiology was
suspected.</p>
<p>All four patients had elevated D-dimers and significant dead-space
ventilation. All patients were also obese, and 3/4 patients were
diabetic.</p>
<p>Not all patients exhibited an improvement in gas exchange or
hemodynamics during the infusion, but some did demonstrate improvements
in oxygenation after treatment. Two patients no longer required
vasopressors or could be weaned off them, while one patient became
hypoxemic and hypotensive and subsequently expired due to a cardiac
arrest. Echocardiogram showed large biventricular thrombi.</p>
<p><em>Limitations</em></p>
<p>In addition to the small sample size, all patients presented with
chronic conditions that are conducive to an inflammatory state. It is
unclear how this would have impacted the tPA therapy, but it is likely
not representative of all patients who present with COVID-19-induced
pneumonia. Moreover, each patient had received a different course of
therapy prior to receiving the tPA infusion. One patient received
hydroxychloroquine and ceftriaxone prior to tPA infusion, two patients
required external ventilator support, and another patient received
concurrent convalescent plasma therapy as part of a clinical trial. Each
patient received an infusion of tPA at 2 mg/hour but for variable
durations of time. One patient received an initial 50 mg infusion of tPA
over two hours. 3/4 patients were also given norepinephrine to manage
persistent, hypotensive shock. Of note, each patient was at a different
stage of the disease; One patient showed cardiac abnormalities and no
clots in transit on an echocardiogram, prior to tPA infusion.</p>
<p><em>Significance</em></p>
<p>The study describes emphasizes the importance of coagulopathies in
COVID-19 and describes clinical outcomes for four severe, COVID-19
patients, who received tPA infusions to manage poor gas exchange. While
the sample size is very limited and mixed benefits were observed,
thrombolysis seems to warrant further investigation as a therapeutic for
COVID-19-associated pneumonia that is characterized by D-dimer elevation
and dead-space ventilation. All four patients had normal platelet
levels, which may suggest that extrinsic triggers of the coagulation
cascade are involved.</p>
<p>The authors suspect that endothelial dysfunction and injury contribute
to the formation of pulmonary microthrombi, and these impair gas
exchange. Pulmonary thrombus formation has also been reported by other
groups; post-mortem analyses of 38 COVID-19 patients’ lungs showed
diffuse alveolar disease and platelet-fibrin thrombi (Carsana et al.,
2020). Inflammatory infiltrates were macrophages in the alveolar lumen
and lymphocytes in the interstitial space (Carsana et al., 2020).
Endothelial damage in COVID-19 patients has also been directly
described, noting the presence of viral elements in the endothelium and
inflammatory infiltrates within the intima (Varga et al., 2020). One
hypothesis may be that the combination of circulating inflammatory
monocytes (previously described to be enriched among PBMCs derived from
COVID-19 patients) that express tissue factor, damaged endothelium, and
complement elements that are also chemotactic for inflammatory cells may
contribute to the overall pro-coagulative state described in COVID-19
patients.</p>
<p><em>References</em></p>
<p>Carsana, L., Sonzogni, A., Nasr, A., Rossi, R.S., Pellegrinelli, A.,
Zerbi, P., Rech, R., Colombo, R., Antinori, S., Corbellino, M., et al.
(2020) Pulmonary post-mortem findings in a large series of COVID-19
cases from Northern Itality. medRxiv. 2020.04.19.20054262.</p>
<p>Varga, Z., Flammer, A.J., Steiger, P., Haberecker, M., Andermatt, R.,
Zinkernagal, A.S., Mehra, M.R., Schuepbach, R.A., Ruschitzka, F., Moch,
H. (2020) Endothelial cell infection and endotheliitis in COVID-19.
Lancet. 10.1016/S0140-6736(20)30937-5.</p>
<p><em>The study described in this review was conducted by physicians of the
Divisions of Pulmonary, Critical Care, and Sleep Medicine, Cardiology,
Nephrology, Surgery, and Neurosurgery and Neurology at the Icahn School
of Medicine at Mount Sinai. </em></p>
<p><em>Reviewed by Matthew D. Park as part of a project by students, postdocs,
and faculty at the Immunology Institute of the Icahn School of Medicine,
Mount Sinai.</em></p>
<!-- default theme -->

<style>
    /* import google fonts */
    @import url("https://fonts.googleapis.com/css?family=Open+Sans:400,600,700");
    @import url("https://fonts.googleapis.com/css?family=Source+Code+Pro");

    /* -------------------------------------------------- */
    /* global */
    /* -------------------------------------------------- */

    /* all elements */
    * {
        /* force sans-serif font unless specified otherwise */
        font-family: "Open Sans", "Helvetica", sans-serif;

        /* prevent text inflation on some mobile browsers */
        -webkit-text-size-adjust: none !important;
        -moz-text-size-adjust: none !important;
        -o-text-size-adjust: none !important;
        text-size-adjust: none !important;
    }

    @media only screen {
        /* "page" element */
        body {
            position: relative;
            box-sizing: border-box;
            font-size: 12pt;
            line-height: 1.5;
            max-width: 8.5in;
            margin: 20px auto;
            padding: 40px;
            border-radius: 5px;
            border: solid 1px #bdbdbd;
            box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
            background: #ffffff;
        }
    }

    /* when on screen < 8.5in wide */
    @media only screen and (max-width: 8.5in) {
        /* "page" element */
        body {
            padding: 20px;
            margin: 0;
            border-radius: 0;
            border: none;
            box-shadow: 0 0 20px rgba(0, 0, 0, 0.05) inset;
            background: none;
        }
    }

    /* -------------------------------------------------- */
    /* headings */
    /* -------------------------------------------------- */

    /* all headings */
    h1,
    h2,
    h3,
    h4,
    h5,
    h6 {
        margin: 20px 0;
        padding: 0;
        font-weight: bold;
    }

    /* biggest heading */
    h1 {
        margin: 40px 0;
    }

    /* second biggest heading */
    h2 {
        margin-top: 30px;
        padding-bottom: 5px;
        border-bottom: solid 1px #bdbdbd;
    }

    /* -------------------------------------------------- */
    /* manuscript header */
    /* -------------------------------------------------- */

    /* manuscript title */
    header > h1 {
        margin: 0;
        text-align: center;
    }

    /* manuscript title caption text (ie "automatically generated on") */
    header + p {
        text-align: center;
        margin-top: 10px;
    }

    /* -------------------------------------------------- */
    /* text elements */
    /* -------------------------------------------------- */

    /* links */
    a {
        color: #2196f3;
        overflow-wrap: break-word;
    }

    /* normal links (not empty, not button link, not syntax highlighting link) */
    a:not(:empty):not(.button):not(.sourceLine) {
        padding-left: 1px;
        padding-right: 1px;
    }

    /* superscripts and subscripts */
    sub,
    sup {
        /* prevent from affecting line height */
        line-height: 0;
    }

    /* unordered and ordered lists*/
    ul,
    ol {
        padding-left: 20px;
    }

    /* class for styling text semibold */
    .semibold {
        font-weight: 600;
    }

    /* class for styling elements horizontally left aligned */
    .left {
        display: block;
        text-align: left;
        margin-left: auto;
        margin-right: 0;
        justify-content: left;
    }

    /* class for styling elements horizontally centered */
    .center {
        display: block;
        text-align: center;
        margin-left: auto;
        margin-right: auto;
        justify-content: center;
    }

    /* class for styling elements horizontally right aligned */
    .right {
        display: block;
        text-align: right;
        margin-left: 0;
        margin-right: auto;
        justify-content: right;
    }

    /* -------------------------------------------------- */
    /* section elements */
    /* -------------------------------------------------- */

    /* horizontal divider line */
    hr {
        border: none;
        height: 1px;
        background: #bdbdbd;
    }

    /* paragraphs, horizontal dividers, figures, tables, code */
    p,
    hr,
    figure,
    table,
    pre {
        /* treat all as "paragraphs", with consistent vertical margins */
        margin-top: 20px;
        margin-bottom: 20px;
    }

    /* -------------------------------------------------- */
    /* figures */
    /* -------------------------------------------------- */

    /* figure */
    figure {
        max-width: 100%;
        margin-left: auto;
        margin-right: auto;
    }

    /* figure caption */
    figcaption {
        padding: 0;
        padding-top: 10px;
    }

    /* figure image element */
    figure img {
        max-width: 100%;
        display: block;
        margin-left: auto;
        margin-right: auto;
    }

    /* figure auto-number */
    img + figcaption > span:first-of-type {
        font-weight: bold;
        margin-right: 5px;
    }

    /* -------------------------------------------------- */
    /* tables */
    /* -------------------------------------------------- */

    /* table */
    table {
        border-collapse: collapse;
        border-spacing: 0;
        width: 100%;
        margin-left: auto;
        margin-right: auto;
    }

    /* table cells */
    th,
    td {
        border: solid 1px #bdbdbd;
        padding: 10px;
        /* squash table if too wide for page by forcing line breaks */
        overflow-wrap: break-word;
        word-break: break-word;
    }

    /* header row and even rows */
    th,
    tr:nth-child(2n) {
        background-color: #fafafa;
    }

    /* odd rows */
    tr:nth-child(2n + 1) {
        background-color: #ffffff;
    }

    /* table caption */
    caption {
        text-align: left;
        padding: 0;
        padding-bottom: 10px;
    }

    /* table auto-number */
    table > caption > span:first-of-type,
    div.table_wrapper > table > caption > span:first-of-type {
        font-weight: bold;
        margin-right: 5px;
    }

    /* -------------------------------------------------- */
    /* code */
    /* -------------------------------------------------- */

    /* multi-line code block */
    pre {
        padding: 10px;
        background-color: #eeeeee;
        color: #000000;
        border-radius: 5px;
        break-inside: avoid;
        text-align: left;
    }

    /* inline code, ie code within normal text */
    :not(pre) > code {
        padding: 0 4px;
        background-color: #eeeeee;
        color: #000000;
        border-radius: 5px;
    }

    /* code text */
    /* apply all children, to reach syntax highlighting sub-elements */
    code,
    code * {
        /* force monospace font */
        font-family: "Source Code Pro", "Courier New", monospace;
    }

    /* -------------------------------------------------- */
    /* quotes */
    /* -------------------------------------------------- */

    /* quoted text */
    blockquote {
        margin: 0;
        padding: 0;
        border-left: 4px solid #bdbdbd;
        padding-left: 16px;
        break-inside: avoid;
    }

    /* -------------------------------------------------- */
    /* banners */
    /* -------------------------------------------------- */

    /* info banners */
    .banner {
        box-sizing: border-box;
        display: block;
        position: relative;
        width: 100%;
        margin-top: 20px;
        margin-bottom: 20px;
        padding: 20px;
        text-align: center;
    }

    /* paragraph in banner */
    .banner > p {
        margin: 0;
    }

    /* -------------------------------------------------- */
    /* highlight colors */
    /* -------------------------------------------------- */

    .white {
        background: #ffffff;
    }
    .lightgrey {
        background: #eeeeee;
    }
    .grey {
        background: #757575;
    }
    .darkgrey {
        background: #424242;
    }
    .black {
        background: #000000;
    }
    .lightred {
        background: #ffcdd2;
    }
    .lightyellow {
        background: #ffecb3;
    }
    .lightgreen {
        background: #dcedc8;
    }
    .lightblue {
        background: #e3f2fd;
    }
    .lightpurple {
        background: #f3e5f5;
    }
    .red {
        background: #f44336;
    }
    .orange {
        background: #ff9800;
    }
    .yellow {
        background: #ffeb3b;
    }
    .green {
        background: #4caf50;
    }
    .blue {
        background: #2196f3;
    }
    .purple {
        background: #9c27b0;
    }
    .white,
    .lightgrey,
    .lightred,
    .lightyellow,
    .lightgreen,
    .lightblue,
    .lightpurple,
    .orange,
    .yellow,
    .white a,
    .lightgrey a,
    .lightred a,
    .lightyellow a,
    .lightgreen a,
    .lightblue a,
    .lightpurple a,
    .orange a,
    .yellow a {
        color: #000000;
    }
    .grey,
    .darkgrey,
    .black,
    .red,
    .green,
    .blue,
    .purple,
    .grey a,
    .darkgrey a,
    .black a,
    .red a,
    .green a,
    .blue a,
    .purple a {
        color: #ffffff;
    }

    /* -------------------------------------------------- */
    /* buttons */
    /* -------------------------------------------------- */

    /* class for styling links like buttons */
    .button {
        display: inline-flex;
        justify-content: center;
        align-items: center;
        margin: 5px;
        padding: 10px 20px;
        font-size: 0.75em;
        font-weight: 600;
        text-transform: uppercase;
        text-decoration: none;
        letter-spacing: 1px;
        background: none;
        color: #2196f3;
        border: solid 1px #bdbdbd;
        border-radius: 5px;
    }

    /* buttons when hovered */
    .button:hover:not([disabled]),
    .icon_button:hover:not([disabled]) {
        cursor: pointer;
        background: #f5f5f5;
    }

    /* buttons when disabled */
    .button[disabled],
    .icon_button[disabled] {
        opacity: 0.35;
        pointer-events: none;
    }

    /* class for styling buttons containg only single icon */
    .icon_button {
        display: inline-flex;
        justify-content: center;
        align-items: center;
        text-decoration: none;
        margin: 0;
        padding: 0;
        background: none;
        border-radius: 5px;
        border: none;
        width: 20px;
        height: 20px;
        min-width: 20px;
        min-height: 20px;
    }

    /* icon button inner svg image */
    .icon_button > svg {
        height: 16px;
    }

    /* -------------------------------------------------- */
    /* icons */
    /* -------------------------------------------------- */

    /* class for styling icons inline with text */
    .inline_icon {
        height: 1em;
        position: relative;
        top: 0.125em;
    }

    /* -------------------------------------------------- */
    /* print control */
    /* -------------------------------------------------- */

    @media print {
        @page {
            /* suggested printing margin */
            margin: 0.5in;
        }

        /* document and "page" elements */
        html, body {
            margin: 0;
            padding: 0;
            width: 100%;
            height: 100%;
        }

        /* "page" element */
        body {
            font-size: 11pt !important;
            line-height: 1.35;
        }

        /* all headings */
        h1,
        h2,
        h3,
        h4,
        h5,
        h6 {
            margin: 15px 0;
        }

        /* heading 1 */
        h1 {
            font-size: 1.75em;
        }

        /* heading 2 */
        h2 {
            font-size: 1.25em;
            margin-top: 0;
        }

        /* heading 3 */
        h3 {
            font-size: 1.10em;
        }

        /* figures and tables */
        figure, table {
            font-size: 0.85em;
        }

        /* table cells */
        th,
        td {
            padding: 5px;
        }

        /* shrink font awesome icons */
        i.fas,
        i.fab,
        i.far,
        i.fal {
            transform: scale(0.85);
        }

        /* decrease banner margins */
        .banner {
            margin-top: 15px;
            margin-bottom: 15px;
            padding: 15px;
        }

        /* class for centering an element vertically on its own page */
        .page_center {
            margin: auto;
            width: 100%;
            height: 100%;
            display: flex;
            align-items: center;
            vertical-align: middle;
            break-before: page;
            break-after: page;
        }

        /* always insert a page break before the element */
        .page_break_before {
            break-before: page;
        }

        /* always insert a page break after the element */
        .page_break_after {
            break-after: page;
        }

        /* avoid page break before the element */
        .page_break_before_avoid {
            break-before: avoid;
        }

        /* avoid page break after the element */
        .page_break_after_avoid {
            break-after: avoid;
        }

        /* avoid page break inside the element */
        .page_break_inside_avoid {
            break-inside: avoid;
        }
    }

    /* -------------------------------------------------- */
    /* override pandoc css quirks */
    /* -------------------------------------------------- */

    .sourceCode {
        /* prevent unsightly overflow in wide code blocks */
        overflow: auto !important;
    }

    div.sourceCode {
        /* prevent background fill on top-most code block  container */
        background: none !important;
    }

    .sourceCode * {
        /* force consistent line spacing */
        line-height: 1.5 !important;
    }

    div.sourceCode {
        /* style code block margins same as <pre> element */
        margin-top: 20px;
        margin-bottom: 20px;
    }

    /* -------------------------------------------------- */
    /* tablenos */
    /* -------------------------------------------------- */

    /* tablenos wrapper */
    .tablenos {
        /* show scrollbar on tables if necessary to prevent overflow */
        width: 100%;
        margin: 20px 0;
    }

    .tablenos > table {
        /* move margins from table to table_wrapper to allow margin collapsing */
        margin: 0;
    }

    @media only screen {
        /* tablenos wrapper */
        .tablenos {
            /* show scrollbar on tables if necessary to prevent overflow */
            overflow-x: auto !important;
        }

        .tablenos th,
        .tablenos td {
            overflow-wrap: unset !important;
            word-break: unset !important;
        }

        /* table in wrapper */
        .tablenos table,
        .tablenos table * {
            /* don't break table words */
            overflow-wrap: normal !important;
        }
    }

    /* -------------------------------------------------- */
    /* mathjax */
    /* -------------------------------------------------- */

    /* mathjax containers */
    .math.display > span:not(.MathJax_Preview) {
        /* turn inline element (no dimensions) into block (allows fixed width and thus scrolling) */
        display: flex !important;
        overflow-x: auto !important;
        overflow-y: hidden !important;
        justify-content: center;
        align-items: center;
        margin: 0 !important;
    }

    /* right click menu */
    .MathJax_Menu {
        border-radius: 5px !important;
        border: solid 1px #bdbdbd !important;
        box-shadow: none !important;
    }

    /* equation auto-number */
    span[id^="eq:"] > span.math.display + span {
        font-weight: 600;
    }

    /* equation */
    span[id^="eq:"] > span.math.display > span {
        /* nudge to make room for equation auto-number and anchor */
        margin-right: 60px !important;
    }

    /* -------------------------------------------------- */
    /* anchors plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* anchor button */
        .anchor {
            opacity: 0;
            margin-left: 5px;
        }

        /* anchor buttons within <h2>'s */
        h2 .anchor {
            margin-left: 10px;
        }

        /* anchor buttons when hovered/focused and anything containing an anchor button when hovered */
        *:hover > .anchor,
        .anchor:hover,
        .anchor:focus {
            opacity: 1;
        }

        /* anchor button when hovered */
        .anchor:hover {
            cursor: pointer;
        }
    }

    /* always show anchor button on devices with no mouse/hover ability */
    @media (hover: none) {
        .anchor {
            opacity: 1;
        }
    }

    /* always hide anchor button on print */
    @media only print {
        .anchor {
            display: none;
        }
    }

    /* -------------------------------------------------- */
    /* accordion plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* accordion arrow button */
        .accordion_arrow {
            margin-right: 10px;
        }

        /* arrow icon when <h2> data-collapsed attribute true */
        h2[data-collapsed="true"] > .accordion_arrow > svg {
            transform: rotate(-90deg);
        }

        /* all elements (except <h2>'s) when data-collapsed attribute true */
        *:not(h2)[data-collapsed="true"] {
            display: none;
        }

        /* accordion arrow button when hovered and <h2>'s when hovered */
        .accordion_arrow:hover,
        h2[data-collapsed="true"]:hover,
        h2[data-collapsed="false"]:hover {
            cursor: pointer;
        }
    }

    /* always hide accordion arrow button on print */
    @media only print {
        .accordion_arrow {
            display: none;
        }
    }

    /* -------------------------------------------------- */
    /* tooltips plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* tooltip container */
        #tooltip {
            position: absolute;
            width: 50%;
            min-width: 240px;
            max-width: 75%;
            z-index: 1;
        }

        /* tooltip content */
        #tooltip_content {
            margin-bottom: 5px;
            padding: 20px;
            border-radius: 5px;
            border: solid 1px #bdbdbd;
            box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
            background: #ffffff;
            overflow-wrap: break-word;
        }

        /* tooltip copy of paragraphs and figures */
        #tooltip_content > p,
        #tooltip_content > figure {
            margin: 0;
            max-height: 320px;
            overflow-y: auto;
        }

        /* tooltip copy of <img> */
        #tooltip_content > figure > img {
            max-height: 260px;
        }

        /* navigation bar */
        #tooltip_nav_bar {
            margin-top: 10px;
            text-align: center;
        }

        /* navigation bar previous/next buton */
        #tooltip_nav_bar > .icon_button {
            position: relative;
            top: 3px;
        }

        /* navigation bar previous button */
        #tooltip_nav_bar > .icon_button:first-of-type {
            margin-right: 5px;
        }

        /* navigation bar next button */
        #tooltip_nav_bar > .icon_button:last-of-type {
            margin-left: 5px;
        }
    }

    /* always hide tooltip on print */
    @media only print {
        #tooltip {
            display: none;
        }
    }

    /* -------------------------------------------------- */
    /* jump to first plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* jump button */
        .jump_arrow {
            position: relative;
            top: 0.125em;
            margin-right: 5px;
        }
    }

    /* always hide jump button on print */
    @media only print {
        .jump_arrow {
            display: none;
        }
    }

    /* -------------------------------------------------- */
    /* link highlight plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* anything with data-highlighted attribute true */
        [data-highlighted="true"] {
            background: #ffeb3b;
        }

        /* anything with data-selected attribute true */
        [data-selected="true"] {
            background: #ff8a65 !important;
        }

        /* animation definition for glow */
        @keyframes highlight_glow {
            0% {
                background: none;
            }
            10% {
                background: #bbdefb;
            }
            100% {
                background: none;
            }
        }

        /* anything with data-glow attribute true */
        [data-glow="true"] {
            animation: highlight_glow 2s;
        }
    }

    /* -------------------------------------------------- */
    /* table of contents plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* toc panel */
        #toc_panel {
            box-sizing: border-box;
            position: fixed;
            top: 0;
            left: 0;
            background: #ffffff;
            box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
            z-index: 2;
        }

        /* toc panel when closed */
        #toc_panel[data-open="false"] {
            min-width: 60px;
            width: 60px;
            height: 60px;
            border-right: solid 1px #bdbdbd;
            border-bottom: solid 1px #bdbdbd;
        }

        /* toc panel when open */
        #toc_panel[data-open="true"] {
            min-width: 260px;
            max-width: 480px;
            /* keep panel edge consistent distance away from "page" edge */
            width: calc(((100vw - 8.5in) / 2) - 30px - 40px);
            bottom: 0;
            border-right: solid 1px #bdbdbd;
        }

        /* toc panel header */
        #toc_header {
            box-sizing: border-box;
            display: flex;
            flex-direction: row;
            align-items: center;
            height: 60px;
            margin: 0;
            padding: 20px;
        }

        /* toc panel header when hovered */
        #toc_header:hover {
            cursor: pointer;
        }

        /* toc panel header when panel open */
        #toc_panel[data-open="true"] > #toc_header {
            border-bottom: solid 1px #bdbdbd;
        }

        /* toc open/close header button */
        #toc_button {
            margin-right: 20px;
        }

        /* hide toc list and header text when closed */
        #toc_panel[data-open="false"] > #toc_header > *:not(#toc_button),
        #toc_panel[data-open="false"] > #toc_list {
            display: none;
        }

        /* toc list of entries */
        #toc_list {
            box-sizing: border-box;
            width: 100%;
            padding: 20px;
            position: absolute;
            top: calc(60px + 1px);
            bottom: 0;
            overflow: auto;
        }

        /* toc entry, link to section in document */
        .toc_link {
            display: block;
            padding: 5px;
            position: relative;
            font-weight: 600;
            text-decoration: none;
        }

        /* toc entry when hovered or when "viewed" */
        .toc_link:hover,
        .toc_link[data-viewing="true"] {
            background: #f5f5f5;
        }

        /* toc entry, level 1 indentation */
        .toc_link[data-level="1"] {
            margin-left: 0;
        }

        /* toc entry, level 2 indentation */
        .toc_link[data-level="2"] {
            margin-left: 20px;
        }

        /* toc entry, level 3 indentation */
        .toc_link[data-level="3"] {
            margin-left: 40px;
        }

        /* toc entry, level 4 indentation */
        .toc_link[data-level="4"] {
            margin-left: 60px;
        }

        /* toc entry bullets */
        #toc_panel[data-bullets="true"] .toc_link[data-level]:before {
            position: absolute;
            left: -15px;
            top: -1px;
            font-size: 1.5em;
        }

        /* toc entry, level 2 bullet */
        #toc_panel[data-bullets="true"] .toc_link[data-level="2"]:before {
            content: "\2022";
        }

        /* toc entry, level 3 bullet */
        #toc_panel[data-bullets="true"] .toc_link[data-level="3"]:before {
            content: "\25AB";
        }

        /* toc entry, level 4 bullet */
        #toc_panel[data-bullets="true"] .toc_link[data-level="4"]:before {
            content: "-";
        }
    }

    /* when on screen < 8.5in wide */
    @media only screen and (max-width: 8.5in) {
        /* push <body> ("page") element down to make room for toc icon */
        .toc_body_nudge {
            padding-top: 60px;
        }

        /* toc icon when panel closed and not hovered */
        #toc_panel[data-open="false"]:not(:hover) {
            background: rgba(255, 255, 255, 0.75);
        }
    }

    /* always hide toc panel on print */
    @media only print {
        #toc_panel {
            display: none;
        }
    }

    /* -------------------------------------------------- */
    /* lightbox plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* regular <img> in document when hovered */
        .lightbox_document_img:hover {
            cursor: pointer;
        }

        .body_no_scroll {
            overflow: hidden !important;
        }

        /* screen overlay */
        #lightbox_overlay {
            display: flex;
            flex-direction: column;
            position: fixed;
            left: 0;
            top: 0;
            right: 0;
            bottom: 0;
            background: rgba(0, 0, 0, 0.75);
            z-index: 3;
        }

        /* middle area containing lightbox image */
        #lightbox_image_container {
            flex-grow: 1;
            display: flex;
            justify-content: center;
            align-items: center;
            overflow: hidden;
            position: relative;
            padding: 20px;
        }

        /* bottom area containing caption */
        #lightbox_bottom_container {
            display: flex;
            justify-content: center;
            align-items: center;
            height: 100px;
            min-height: 100px;
            max-height: 100px;
            background: rgba(0, 0, 0, 0.5);
        }

        /* image number info text box */
        #lightbox_number_info {
            position: absolute;
            color: #ffffff;
            font-weight: 600;
            left: 2px;
            top: 0;
            z-index: 4;
        }

        /* zoom info text box */
        #lightbox_zoom_info {
            position: absolute;
            color: #ffffff;
            font-weight: 600;
            right: 2px;
            top: 0;
            z-index: 4;
        }

        /* copy of image caption */
        #lightbox_caption {
            box-sizing: border-box;
            display: inline-block;
            width: 100%;
            max-height: 100%;
            padding: 10px 0;
            text-align: center;
            overflow-y: auto;
            color: #ffffff;
        }

        /* navigation previous/next button */
        .lightbox_button {
            width: 100px;
            height: 100%;
            min-width: 100px;
            min-height: 100%;
            color: #ffffff;
        }

        /* navigation previous/next button when hovered */
        .lightbox_button:hover {
            background: none !important;
        }

        /* navigation button icon */
        .lightbox_button > svg {
            height: 25px;
        }

        /* figure auto-number */
        #lightbox_caption > span:first-of-type {
            font-weight: bold;
            margin-right: 5px;
        }

        /* lightbox image when hovered */
        #lightbox_img:hover {
            cursor: grab;
        }

        /* lightbox image when grabbed */
        #lightbox_img:active {
            cursor: grabbing;
        }
    }

    /* when on screen < 480px wide */
    @media only screen and (max-width: 480px) {
        /* make navigation buttons skinnier on small screens to make more room for caption text */
        .lightbox_button {
            width: 50px;
            min-width: 50px;
        }
    }

    /* always hide lightbox on print */
    @media only print {
        #lightbox_overlay {
            display: none;
        }
    }

    /* -------------------------------------------------- */
    /* hypothesis (annotations) plugin */
    /* -------------------------------------------------- */

    /* side panel */
    .annotator-frame {
        width: 280px !important;
        z-index: 0 !important;
    }

    /* match highlight color to rest of theme */
    .annotator-highlights-always-on .annotator-hl {
        background-color: #ffeb3b !important;
    }

    /* match focused color to rest of theme */
    .annotator-hl.annotator-hl-focused {
        background-color: #ff8a65 !important;
    }

    /* match bucket bar color to rest of theme */
    .annotator-bucket-bar {
        background: #f5f5f5 !important;
    }

    /* always hide toolbar and tooltip on print */
    @media only print {
        .annotator-frame {
            display: none !important;
        }

        hypothesis-adder {
            display: none !important;
        }
    }
</style>
<!-- anchors plugin -->

<script>
    (function() {
        // /////////////////////////
        // DESCRIPTION
        // /////////////////////////

        // This Manubot plugin adds an anchor next to each of a certain type
        // of element that provides a human-readable url to that specific
        // item/position in the document (eg "manuscript.html#abstract"). It
        // also makes it such that scrolling out of view of a target removes
        // its identifier from the url.

        // /////////////////////////
        // OPTIONS
        // /////////////////////////

        // plugin name prefix for url parameters
        const pluginName = 'anchors';

        // default plugin options
        const options = {
            // which types of elements to add anchors next to, in
            // "document.querySelector" format
            typesQuery: 'h1, h2, h3, [id^="fig:"], [id^="tbl:"], [id^="eq:"]',
            // whether plugin is on or not
            enabled: 'true'
        };

        // change options above, or override with url parameter, eg:
        // 'manuscript.html?pluginName-enabled=false'

        // /////////////////////////
        // SCRIPT
        // /////////////////////////

        // start script
        function start() {
            // add anchor to each element of specified types
            const elements = document.querySelectorAll(options.typesQuery);
            for (const element of elements)
                addAnchor(element);

            // attach scroll listener to window
            window.addEventListener('scroll', onScroll);
        }

        // when window is scrolled
        function onScroll() {
            // if url has hash and user has scrolled out of view of hash
            // target, remove hash from url
            const tolerance = 100;
            const target = getHashTarget();
            if (target) {
                if (
                    target.getBoundingClientRect().top >
                        window.innerHeight + tolerance ||
                    target.getBoundingClientRect().bottom < 0 - tolerance
                )
                    history.pushState(null, null, ' ');
            }
        }

        // add anchor to element
        function addAnchor(element) {
            let addTo; // element to add anchor button to

            // if figure or table, modify withId and addTo to get expected
            // elements
            if (element.id.indexOf('fig:') === 0) {
                addTo = element.querySelector('figcaption');
            } else if (element.id.indexOf('tbl:') === 0) {
                addTo = element.querySelector('caption');
            } else if (element.id.indexOf('eq:') === 0) {
                addTo = element.querySelector('.eqnos-number');
            }

            addTo = addTo || element;
            const id = element.id || null;

            // do not add anchor if element doesn't have assigned id.
            // id is generated by pandoc and is assumed to be unique and
            // human-readable
            if (!id)
                return;

            // create anchor button
            const anchor = document.createElement('a');
            anchor.innerHTML = document.querySelector('.icon_link').innerHTML;
            anchor.title = 'Link to this part of the document';
            anchor.classList.add('icon_button', 'anchor');
            anchor.dataset.ignore = 'true';
            anchor.href = '#' + id;
            addTo.appendChild(anchor);
        }

        // get element that is target of link or url hash
        function getHashTarget() {
            const hash = window.location.hash;
            const id = hash.slice(1);
            let target = document.querySelector('[id="' + id + '"]');
            if (!target)
                return;

            // if figure or table, modify target to get expected element
            if (id.indexOf('fig:') === 0)
                target = target.querySelector('figure');
            if (id.indexOf('tbl:') === 0)
                target = target.querySelector('table');

            return target;
        }

        // load options from url parameters
        function loadOptions() {
            const url = window.location.search;
            const params = new URLSearchParams(url);
            for (const optionName of Object.keys(options)) {
                const paramName = pluginName + '-' + optionName;
                const param = params.get(paramName);
                if (param !== '' && param !== null)
                    options[optionName] = param;
            }
        }
        loadOptions();

        // start script when document is finished loading
        if (options.enabled === 'true')
            window.addEventListener('load', start);
    })();
</script>

<!-- link icon -->

<template class="icon_link">
    <!-- modified from: https://fontawesome.com/icons/link -->
    <svg width="16" height="16" viewBox="0 0 512 512">
        <path
            fill="currentColor"
            d="M326.612 185.391c59.747 59.809 58.927 155.698.36 214.59-.11.12-.24.25-.36.37l-67.2 67.2c-59.27 59.27-155.699 59.262-214.96 0-59.27-59.26-59.27-155.7 0-214.96l37.106-37.106c9.84-9.84 26.786-3.3 27.294 10.606.648 17.722 3.826 35.527 9.69 52.721 1.986 5.822.567 12.262-3.783 16.612l-13.087 13.087c-28.026 28.026-28.905 73.66-1.155 101.96 28.024 28.579 74.086 28.749 102.325.51l67.2-67.19c28.191-28.191 28.073-73.757 0-101.83-3.701-3.694-7.429-6.564-10.341-8.569a16.037 16.037 0 0 1-6.947-12.606c-.396-10.567 3.348-21.456 11.698-29.806l21.054-21.055c5.521-5.521 14.182-6.199 20.584-1.731a152.482 152.482 0 0 1 20.522 17.197zM467.547 44.449c-59.261-59.262-155.69-59.27-214.96 0l-67.2 67.2c-.12.12-.25.25-.36.37-58.566 58.892-59.387 154.781.36 214.59a152.454 152.454 0 0 0 20.521 17.196c6.402 4.468 15.064 3.789 20.584-1.731l21.054-21.055c8.35-8.35 12.094-19.239 11.698-29.806a16.037 16.037 0 0 0-6.947-12.606c-2.912-2.005-6.64-4.875-10.341-8.569-28.073-28.073-28.191-73.639 0-101.83l67.2-67.19c28.239-28.239 74.3-28.069 102.325.51 27.75 28.3 26.872 73.934-1.155 101.96l-13.087 13.087c-4.35 4.35-5.769 10.79-3.783 16.612 5.864 17.194 9.042 34.999 9.69 52.721.509 13.906 17.454 20.446 27.294 10.606l37.106-37.106c59.271-59.259 59.271-155.699.001-214.959z"
        ></path>
    </svg>
</template>
<!-- accordion plugin -->

<script>
    (function() {
        // /////////////////////////
        // DESCRIPTION
        // /////////////////////////

        // This Manubot plugin allows sections of content under <h2> headings
        // to be collapsible.

        // /////////////////////////
        // OPTIONS
        // /////////////////////////

        // plugin name prefix for url parameters
        const pluginName = 'accordion';

        // default plugin options
        const options = {
            // whether to always start expanded ('false'), always start
            // collapsed ('true'), or start collapsed when screen small ('auto')
            startCollapsed: 'auto',
            // whether plugin is on or not
            enabled: 'true'
        };

        // change options above, or override with url parameter, eg:
        // 'manuscript.html?pluginName-enabled=false'

        // /////////////////////////
        // SCRIPT
        // /////////////////////////

        // start script
        function start() {
            // run through each <h2> heading
            const headings = document.querySelectorAll('h2');
            for (const heading of headings) {
                addArrow(heading);

                // start expanded/collapsed based on option
                if (
                    options.startCollapsed === 'true' ||
                    (options.startCollapsed === 'auto' && isSmallScreen())
                )
                    collapseHeading(heading);
                else
                    expandHeading(heading);
            }

            // attach hash change listener to window
            window.addEventListener('hashchange', onHashChange);
        }

        // when hash (eg manuscript.html#introduction) changes
        function onHashChange() {
            const target = getHashTarget();
            if (target)
                goToElement(target);
        }

        // add arrow to heading
        function addArrow(heading) {
            // add arrow button
            const arrow = document.createElement('button');
            arrow.innerHTML = document.querySelector(
                '.icon_angle_down'
            ).innerHTML;
            arrow.classList.add('icon_button', 'accordion_arrow');
            heading.insertBefore(arrow, heading.firstChild);

            // attach click listener to heading and button
            heading.addEventListener('click', onHeadingClick);
            arrow.addEventListener('click', onArrowClick);
        }

        // determine if on mobile-like device with small screen
        function isSmallScreen() {
            return Math.min(window.innerWidth, window.innerHeight) < 480;
        }

        // scroll to and focus element
        function goToElement(element, offset) {
            // expand accordion section if collapsed
            expandElement(element);
            const y =
                getRectInView(element).top -
                getRectInView(document.documentElement).top -
                (offset || 0);
            // trigger any function listening for "onscroll" event
            window.dispatchEvent(new Event('scroll'));
            window.scrollTo(0, y);
            document.activeElement.blur();
            element.focus();
        }

        // get element that is target of hash
        function getHashTarget(link) {
            const hash = link ? link.hash : window.location.hash;
            const id = hash.slice(1);
            let target = document.querySelector('[id="' + id + '"]');
            if (!target)
                return;

            // if figure or table, modify target to get expected element
            if (id.indexOf('fig:') === 0)
                target = target.querySelector('figure');
            if (id.indexOf('tbl:') === 0)
                target = target.querySelector('table');

            return target;
        }

        // when <h2> heading is clicked
        function onHeadingClick(event) {
            // only collapse if <h2> itself is target of click (eg, user did
            // not click on anchor within <h2>)
            if (event.target === this)
                toggleCollapse(this);
        }

        // when arrow button is clicked
        function onArrowClick() {
            toggleCollapse(this.parentNode);
        }

        // collapse section if expanded, expand if collapsed
        function toggleCollapse(heading) {
            if (heading.dataset.collapsed === 'false')
                collapseHeading(heading);
            else
                expandHeading(heading);
        }

        // elements to exclude from collapse, such as table of contents panel,
        // hypothesis panel, etc
        const exclude = '#toc_panel, div.annotator-frame, #lightbox_overlay';

        // collapse section
        function collapseHeading(heading) {
            heading.setAttribute('data-collapsed', 'true');
            const children = getChildren(heading);
            for (const child of children)
                child.setAttribute('data-collapsed', 'true');
        }

        // expand section
        function expandHeading(heading) {
            heading.setAttribute('data-collapsed', 'false');
            const children = getChildren(heading);
            for (const child of children)
                child.setAttribute('data-collapsed', 'false');
        }

        // get list of elements between this <h2> and next <h2> or <h1>
        // ("children" of the <h2> section)
        function getChildren(heading) {
            return nextUntil(heading, 'h2, h1', exclude);
        }

        // get position/dimensions of element or viewport
        function getRectInView(element) {
            let rect = {};
            rect.left = 0;
            rect.top = 0;
            rect.right = document.documentElement.clientWidth;
            rect.bottom = document.documentElement.clientHeight;
            let style = {};

            if (element instanceof HTMLElement) {
                rect = element.getBoundingClientRect();
                style = window.getComputedStyle(element);
            }

            const margin = {};
            margin.left = parseFloat(style.marginLeftWidth) || 0;
            margin.top = parseFloat(style.marginTopWidth) || 0;
            margin.right = parseFloat(style.marginRightWidth) || 0;
            margin.bottom = parseFloat(style.marginBottomWidth) || 0;

            const border = {};
            border.left = parseFloat(style.borderLeftWidth) || 0;
            border.top = parseFloat(style.borderTopWidth) || 0;
            border.right = parseFloat(style.borderRightWidth) || 0;
            border.bottom = parseFloat(style.borderBottomWidth) || 0;

            const newRect = {};
            newRect.left = rect.left + margin.left + border.left;
            newRect.top = rect.top + margin.top + border.top;
            newRect.right = rect.right + margin.right + border.right;
            newRect.bottom = rect.bottom + margin.bottom + border.bottom;
            newRect.width = newRect.right - newRect.left;
            newRect.height = newRect.bottom - newRect.top;

            return newRect;
        }

        // get list of elements after a start element up to element matching
        // query
        function nextUntil(element, query, exclude) {
            const elements = [];
            while (element = element.nextElementSibling, element) {
                if (element.matches(query))
                    break;
                if (!element.matches(exclude))
                    elements.push(element);
            }
            return elements;
        }

        // get closest element before specified element that matches query
        function firstBefore(element, query) {
            while (
                element &&
                element !== document.body &&
                !element.matches(query)
            )
                element = element.previousElementSibling || element.parentNode;

            return element;
        }

        // check if element is part of collapsed heading
        function isCollapsed(element) {
            while (element && element !== document.body) {
                if (element.dataset.collapsed === 'true')
                    return true;
                element = element.parentNode;
            }
            return false;
        }

        // expand heading containing element if necesary
        function expandElement(element) {
            if (isCollapsed(element)) {
                const heading = firstBefore(element, 'h2');
                if (heading)
                    heading.click();
            }
        }

        // load options from url parameters
        function loadOptions() {
            const url = window.location.search;
            const params = new URLSearchParams(url);
            for (const optionName of Object.keys(options)) {
                const paramName = pluginName + '-' + optionName;
                const param = params.get(paramName);
                if (param !== '' && param !== null)
                    options[optionName] = param;
            }
        }
        loadOptions();

        // start script when document is finished loading
        if (options.enabled === 'true')
            window.addEventListener('load', start);
    })();
</script>

<!-- angle down icon -->

<template class="icon_angle_down">
    <!-- modified from: https://fontawesome.com/icons/angle-down -->
    <svg width="16" height="16" viewBox="0 0 448 512">
        <path
            fill="currentColor"
            d="M207.029 381.476L12.686 187.132c-9.373-9.373-9.373-24.569 0-33.941l22.667-22.667c9.357-9.357 24.522-9.375 33.901-.04L224 284.505l154.745-154.021c9.379-9.335 24.544-9.317 33.901.04l22.667 22.667c9.373 9.373 9.373 24.569 0 33.941L240.971 381.476c-9.373 9.372-24.569 9.372-33.942 0z"
        ></path>
    </svg>
</template>
<!-- tooltips plugin -->

<script>
    (function() {
        // /////////////////////////
        // DESCRIPTION
        // /////////////////////////

        // This Manubot plugin makes it such that when the user hovers or
        // focuses a link to a citation or figure, a tooltip appears with a
        // preview of the reference content, along with arrows to navigate
        // between instances of the same reference in the document.

        // /////////////////////////
        // OPTIONS
        // /////////////////////////

        // plugin name prefix for url parameters
        const pluginName = 'tooltips';

        // default plugin options
        const options = {
            // whether user must click off to close tooltip instead of just
            // un-hovering
            clickClose: 'false',
            // delay (in ms) between opening and closing tooltip
            delay: '100',
            // whether plugin is on or not
            enabled: 'true'
        };

        // change options above, or override with url parameter, eg:
        // 'manuscript.html?pluginName-enabled=false'

        // /////////////////////////
        // SCRIPT
        // /////////////////////////

        // start script
        function start() {
            const links = getLinks();
            for (const link of links) {
                // attach hover and focus listeners to link
                link.addEventListener('mouseover', onLinkHover);
                link.addEventListener('mouseleave', onLinkUnhover);
                link.addEventListener('focus', onLinkFocus);
                link.addEventListener('touchend', onLinkTouch);
            }

            // attach mouse, key, and resize listeners to window
            window.addEventListener('mousedown', onClick);
            window.addEventListener('touchstart', onClick);
            window.addEventListener('keyup', onKeyUp);
            window.addEventListener('resize', onResize);
        }

        // when link is hovered
        function onLinkHover() {
            // function to open tooltip
            const delayOpenTooltip = function() {
                openTooltip(this);
            }.bind(this);

            // run open function after delay
            this.openTooltipTimer = window.setTimeout(
                delayOpenTooltip,
                options.delay
            );
        }

        // when mouse leaves link
        function onLinkUnhover() {
            // cancel opening tooltip
            window.clearTimeout(this.openTooltipTimer);

            // don't close on unhover if option specifies
            if (options.clickClose === 'true')
                return;

            // function to close tooltip
            const delayCloseTooltip = function() {
                // if tooltip open and if mouse isn't over tooltip, close
                const tooltip = document.getElementById('tooltip');
                if (tooltip && !tooltip.matches(':hover'))
                    closeTooltip();
            };

            // run close function after delay
            this.closeTooltipTimer = window.setTimeout(
                delayCloseTooltip,
                options.delay
            );
        }

        // when link is focused (tabbed to)
        function onLinkFocus(event) {
            openTooltip(this);
        }

        // when link is touched on touch screen
        function onLinkTouch(event) {
            // attempt to force hover state on first tap always, and trigger
            // regular link click (and navigation) on second tap
            if (event.target === document.activeElement)
                event.target.click();
            else {
                document.activeElement.blur();
                event.target.focus();
            }
            if (event.cancelable)
                event.preventDefault();
            event.stopPropagation();
            return false;
        }

        // when mouse is clicked anywhere in window
        function onClick(event) {
            closeTooltip();
        }

        // when key pressed
        function onKeyUp(event) {
            if (!event || !event.key)
                return;

            switch (event.key) {
                // trigger click of prev button
                case 'ArrowLeft':
                    const prevButton = document.getElementById(
                        'tooltip_prev_button'
                    );
                    if (prevButton)
                        prevButton.click();
                    break;
                // trigger click of next button
                case 'ArrowRight':
                    const nextButton = document.getElementById(
                        'tooltip_next_button'
                    );
                    if (nextButton)
                        nextButton.click();
                    break;
                // close on esc
                case 'Escape':
                    closeTooltip();
                    break;
            }
        }

        // when window is resized or zoomed
        function onResize() {
            closeTooltip();
        }

        // get all links of types we wish to handle
        function getLinks() {
            const queries = [];
            // exclude buttons, anchor links, toc links, etc
            const exclude =
                ':not(.button):not(.icon_button):not(.anchor):not(.toc_link)';
            queries.push('a[href^="#ref-"]' + exclude); // citation links
            queries.push('a[href^="#fig:"]' + exclude); // figure links
            const query = queries.join(', ');
            return document.querySelectorAll(query);
        }

        // get links with same target, get index of link in set, get total
        // same links
        function getSameLinks(link) {
            const sameLinks = [];
            const links = getLinks();
            for (const otherLink of links) {
                if (
                    otherLink.getAttribute('href') === link.getAttribute('href')
                )
                    sameLinks.push(otherLink);
            }

            return {
                elements: sameLinks,
                index: sameLinks.indexOf(link),
                total: sameLinks.length
            };
        }

        // open tooltip
        function openTooltip(link) {
            // delete tooltip if it exists, start fresh
            closeTooltip();

            // make tooltip element
            const tooltip = makeTooltip(link);

            // if source couldn't be found and tooltip not made, exit
            if (!tooltip)
                return;

            // make navbar elements
            const navBar = makeNavBar(link);
            if (navBar)
                tooltip.firstElementChild.appendChild(navBar);

            // attach tooltip to page
            document.body.appendChild(tooltip);

            // position tooltip
            const position = function() {
                positionTooltip(link);
            };
            position();

            // if tooltip contains images, position again after they've loaded
            const imgs = tooltip.querySelectorAll('img');
            for (const img of imgs)
                img.addEventListener('load', position);
        }

        // close (delete) tooltip
        function closeTooltip() {
            const tooltip = document.getElementById('tooltip');
            if (tooltip)
                tooltip.remove();
        }

        // make tooltip
        function makeTooltip(link) {
            // get target element that link points to
            const source = getSource(link);

            // if source can't be found, exit
            if (!source)
                return;

            // create new tooltip
            const tooltip = document.createElement('div');
            tooltip.id = 'tooltip';
            const tooltipContent = document.createElement('div');
            tooltipContent.id = 'tooltip_content';
            tooltip.appendChild(tooltipContent);

            // make copy of source node and put in tooltip
            const sourceCopy = makeCopy(source);
            tooltipContent.appendChild(sourceCopy);

            // attach mouse event listeners
            tooltip.addEventListener('click', onTooltipClick);
            tooltip.addEventListener('mousedown', onTooltipClick);
            tooltip.addEventListener('touchstart', onTooltipClick);
            tooltip.addEventListener('mouseleave', onTooltipUnhover);

            // (for interaction with lightbox plugin)
            // transfer click on tooltip copied img to original img
            const sourceImg = source.querySelector('img');
            const sourceCopyImg = sourceCopy.querySelector('img');
            if (sourceImg && sourceCopyImg) {
                const clickImg = function() {
                    sourceImg.click();
                    closeTooltip();
                };
                sourceCopyImg.addEventListener('click', clickImg);
            }

            return tooltip;
        }

        // make carbon copy of html dom element
        function makeCopy(source) {
            const sourceCopy = source.cloneNode(true);

            // delete elements marked with ignore (eg anchor and jump buttons)
            const deleteFromCopy = sourceCopy.querySelectorAll(
                '[data-ignore="true"]'
            );
            for (const element of deleteFromCopy)
                element.remove();

            // delete certain element attributes
            const attributes = [
                'id',
                'data-collapsed',
                'data-selected',
                'data-highlighted',
                'data-glow'
            ];
            for (const attribute of attributes) {
                sourceCopy.removeAttribute(attribute);
                const elements = sourceCopy.querySelectorAll(
                    '[' + attribute + ']'
                );
                for (const element of elements)
                    element.removeAttribute(attribute);
            }

            return sourceCopy;
        }

        // when tooltip is clicked
        function onTooltipClick(event) {
            // when user clicks on tooltip, stop click from transferring
            // outside of tooltip (eg, click off to close tooltip, or eg click
            // off to unhighlight same refs)
            event.stopPropagation();
        }

        // when tooltip is unhovered
        function onTooltipUnhover(event) {
            if (options.clickClose === 'true')
                return;

            // make sure new mouse/touch/focus no longer over tooltip or any
            // element within it
            const tooltip = document.getElementById('tooltip');
            if (!tooltip)
                return;
            if (this.contains(event.relatedTarget))
                return;

            closeTooltip();
        }

        // make nav bar to go betwen prev/next instances of same reference
        function makeNavBar(link) {
            // find other links to the same source
            const sameLinks = getSameLinks(link);

            // don't show nav bar when singular reference
            if (sameLinks.total <= 1)
                return;

            // find prev/next links with same target
            const prevLink = getPrevLink(link, sameLinks);
            const nextLink = getNextLink(link, sameLinks);

            // create nav bar
            const navBar = document.createElement('div');
            navBar.id = 'tooltip_nav_bar';
            const text = sameLinks.index + 1 + ' of ' + sameLinks.total;

            // create nav bar prev/next buttons
            const prevButton = document.createElement('button');
            const nextButton = document.createElement('button');
            prevButton.id = 'tooltip_prev_button';
            nextButton.id = 'tooltip_next_button';
            prevButton.title =
                'Jump to the previous occurence of this item in the document [←]';
            nextButton.title =
                'Jump to the next occurence of this item in the document [→]';
            prevButton.classList.add('icon_button');
            nextButton.classList.add('icon_button');
            prevButton.innerHTML = document.querySelector(
                '.icon_caret_left'
            ).innerHTML;
            nextButton.innerHTML = document.querySelector(
                '.icon_caret_right'
            ).innerHTML;
            navBar.appendChild(prevButton);
            navBar.appendChild(document.createTextNode(text));
            navBar.appendChild(nextButton);

            // attach click listeners to buttons
            prevButton.addEventListener('click', function() {
                onPrevNextClick(link, prevLink);
            });
            nextButton.addEventListener('click', function() {
                onPrevNextClick(link, nextLink);
            });

            return navBar;
        }

        // get previous link with same target
        function getPrevLink(link, sameLinks) {
            if (!sameLinks)
                sameLinks = getSameLinks(link);
            // wrap index to other side if < 1
            let index;
            if (sameLinks.index - 1 >= 0)
                index = sameLinks.index - 1;
            else
                index = sameLinks.total - 1;
            return sameLinks.elements[index];
        }

        // get next link with same target
        function getNextLink(link, sameLinks) {
            if (!sameLinks)
                sameLinks = getSameLinks(link);
            // wrap index to other side if > total
            let index;
            if (sameLinks.index + 1 <= sameLinks.total - 1)
                index = sameLinks.index + 1;
            else
                index = 0;
            return sameLinks.elements[index];
        }

        // get element that is target of link or url hash
        function getSource(link) {
            const hash = link ? link.hash : window.location.hash;
            const id = hash.slice(1);
            let target = document.querySelector('[id="' + id + '"]');
            if (!target)
                return;

            // if ref or figure, modify target to get expected element
            if (id.indexOf('ref-') === 0)
                target = target.querySelector('p');
            else if (id.indexOf('fig:') === 0)
                target = target.querySelector('figure');

            return target;
        }

        // when prev/next arrow button is clicked
        function onPrevNextClick(link, prevNextLink) {
            if (link && prevNextLink)
                goToElement(prevNextLink, window.innerHeight * 0.5);
        }

        // scroll to and focus element
        function goToElement(element, offset) {
            // expand accordion section if collapsed
            expandElement(element);
            const y =
                getRectInView(element).top -
                getRectInView(document.documentElement).top -
                (offset || 0);
            // trigger any function listening for "onscroll" event
            window.dispatchEvent(new Event('scroll'));
            window.scrollTo(0, y);
            document.activeElement.blur();
            element.focus();
        }

        // determine position to place tooltip based on link position in
        // viewport and tooltip size
        function positionTooltip(link, left, top) {
            const tooltipElement = document.getElementById('tooltip');
            if (!tooltipElement)
                return;

            // get convenient vars for position/dimensions of
            // link/tooltip/page/view
            link = getRectInPage(link);
            const tooltip = getRectInPage(tooltipElement);
            const view = getRectInPage();

            // horizontal positioning
            if (left)
                // use explicit value
                left = left;
            else if (link.left + tooltip.width < view.right)
                // fit tooltip to right of link
                left = link.left;
            else if (link.right - tooltip.width > view.left)
                // fit tooltip to left of link
                left = link.right - tooltip.width;
            // center tooltip in view
            else
                left = (view.right - view.left) / 2 - tooltip.width / 2;

            // vertical positioning
            if (top)
                // use explicit value
                top = top;
            else if (link.top - tooltip.height > view.top)
                // fit tooltip above link
                top = link.top - tooltip.height;
            else if (link.bottom + tooltip.height < view.bottom)
                // fit tooltip below link
                top = link.bottom;
            else {
                // center tooltip in view
                top = view.top + view.height / 2 - tooltip.height / 2;
                // nudge off of link to left/right if possible
                if (link.right + tooltip.width < view.right)
                    left = link.right;
                else if (link.left - tooltip.width > view.left)
                    left = link.left - tooltip.width;
            }

            tooltipElement.style.left = left + 'px';
            tooltipElement.style.top = top + 'px';
        }

        // get position/dimensions of element or viewport
        function getRectInView(element) {
            let rect = {};
            rect.left = 0;
            rect.top = 0;
            rect.right = document.documentElement.clientWidth;
            rect.bottom = document.documentElement.clientHeight;
            let style = {};

            if (element instanceof HTMLElement) {
                rect = element.getBoundingClientRect();
                style = window.getComputedStyle(element);
            }

            const margin = {};
            margin.left = parseFloat(style.marginLeftWidth) || 0;
            margin.top = parseFloat(style.marginTopWidth) || 0;
            margin.right = parseFloat(style.marginRightWidth) || 0;
            margin.bottom = parseFloat(style.marginBottomWidth) || 0;

            const border = {};
            border.left = parseFloat(style.borderLeftWidth) || 0;
            border.top = parseFloat(style.borderTopWidth) || 0;
            border.right = parseFloat(style.borderRightWidth) || 0;
            border.bottom = parseFloat(style.borderBottomWidth) || 0;

            const newRect = {};
            newRect.left = rect.left + margin.left + border.left;
            newRect.top = rect.top + margin.top + border.top;
            newRect.right = rect.right + margin.right + border.right;
            newRect.bottom = rect.bottom + margin.bottom + border.bottom;
            newRect.width = newRect.right - newRect.left;
            newRect.height = newRect.bottom - newRect.top;

            return newRect;
        }

        // get position of element relative to page
        function getRectInPage(element) {
            const rect = getRectInView(element);
            const body = getRectInView(document.body);

            const newRect = {};
            newRect.left = rect.left - body.left;
            newRect.top = rect.top - body.top;
            newRect.right = rect.right - body.left;
            newRect.bottom = rect.bottom - body.top;
            newRect.width = rect.width;
            newRect.height = rect.height;

            return newRect;
        }

        // (for interaction with accordion plugin)
        // get closest element before specified element that matches query
        function firstBefore(element, query) {
            while (
                element &&
                element !== document.body &&
                !element.matches(query)
            )
                element = element.previousElementSibling || element.parentNode;

            return element;
        }

        // (for interaction with accordion plugin)
        // check if element is part of collapsed heading
        function isCollapsed(element) {
            while (element && element !== document.body) {
                if (element.dataset.collapsed === 'true')
                    return true;
                element = element.parentNode;
            }
            return false;
        }

        // (for interaction with accordion plugin)
        // expand heading containing element if necesary
        function expandElement(element) {
            if (isCollapsed(element)) {
                const heading = firstBefore(element, 'h2');
                if (heading)
                    heading.click();
            }
        }

        // load options from url parameters
        function loadOptions() {
            const url = window.location.search;
            const params = new URLSearchParams(url);
            for (const optionName of Object.keys(options)) {
                const paramName = pluginName + '-' + optionName;
                const param = params.get(paramName);
                if (param !== '' && param !== null)
                    options[optionName] = param;
            }
        }
        loadOptions();

        // start script when document is finished loading
        if (options.enabled === 'true')
            window.addEventListener('load', start);
    })();
</script>

<!-- caret left icon -->

<template class="icon_caret_left">
    <!-- modified from: https://fontawesome.com/icons/caret-left -->
    <svg width="16" height="16" viewBox="0 0 192 512">
        <path
            fill="currentColor"
            d="M192 127.338v257.324c0 17.818-21.543 26.741-34.142 14.142L29.196 270.142c-7.81-7.81-7.81-20.474 0-28.284l128.662-128.662c12.599-12.6 34.142-3.676 34.142 14.142z"
        ></path>
    </svg>
</template>

<!-- caret right icon -->

<template class="icon_caret_right">
    <!-- modified from: https://fontawesome.com/icons/caret-right -->
    <svg width="16" height="16" viewBox="0 0 192 512">
        <path
            fill="currentColor"
            d="M0 384.662V127.338c0-17.818 21.543-26.741 34.142-14.142l128.662 128.662c7.81 7.81 7.81 20.474 0 28.284L34.142 398.804C21.543 411.404 0 402.48 0 384.662z"
        ></path>
    </svg>
</template>
<!-- jump to first plugin -->

<script>
    (function() {
        // /////////////////////////
        // DESCRIPTION
        // /////////////////////////

        // This Manubot plugin adds a button next to each reference entry,
        // figure, and table that jumps the page to the first occurrence of a
        // link to that item in the manuscript.

        // /////////////////////////
        // OPTIONS
        // /////////////////////////

        // plugin name prefix for url parameters
        const pluginName = 'jumpToFirst';

        // default plugin options
        const options = {
            // whether to add buttons next to reference entries
            references: 'true',
            // whether to add buttons next to figures
            figures: 'true',
            // whether to add buttons next to tables
            tables: 'true',
            // whether plugin is on or not
            enabled: 'true'
        };

        // change options above, or override with url parameter, eg:
        // 'manuscript.html?pluginName-enabled=false'

        // /////////////////////////
        // SCRIPT
        // /////////////////////////

        // start script
        function start() {
            if (options.references !== 'false')
                makeReferenceButtons();
            if (options.figures !== 'false')
                makeFigureButtons();
            if (options.tables !== 'false')
                makeTableButtons();
        }

        // when jump button clicked
        function onButtonClick() {
            const first = getFirstOccurrence(this.dataset.id);
            if (!first)
                return;

            // update url hash so navigating "back" in history will return
            // user to jump button
            window.location.hash = this.dataset.id;
            // scroll to link
            window.setTimeout(function() {
                goToElement(first, window.innerHeight * 0.5);
            }, 0);
        }

        // get first occurence of link to item in document
        function getFirstOccurrence(id) {
            let query = 'a';
            query += '[href="#' + id + '"]';
            // exclude buttons, anchor links, toc links, etc
            query +=
                ':not(.button):not(.icon_button):not(.anchor):not(.toc_link)';
            return document.querySelector(query);
        }

        // add button next to each reference entry
        function makeReferenceButtons() {
            const references = document.querySelectorAll('div[id^="ref-"]');
            for (const reference of references) {
                // get reference id and element to add button to
                const id = reference.id;
                const container = reference.firstElementChild;
                const first = getFirstOccurrence(id);

                // if can't find link to reference, ignore
                if (!first)
                    continue;

                // make jump button
                let button = document.createElement('button');
                button.classList.add('icon_button', 'jump_arrow');
                button.title =
                    'Jump to the first occurence of this reference in the document';
                button.innerHTML = document.querySelector(
                    '.icon_angle_double_up'
                ).innerHTML;
                button.dataset.id = id;
                button.dataset.ignore = 'true';
                container.innerHTML = button.outerHTML + container.innerHTML;
                button = container.firstElementChild;
                button.addEventListener('click', onButtonClick);
            }
        }

        // add button next to each figure
        function makeFigureButtons() {
            const figures = document.querySelectorAll('[id^="fig:"]');
            for (const figure of figures) {
                // get figure id and element to add button to
                const id = figure.id;
                const container = figure.querySelector('figcaption') || figure;
                const first = getFirstOccurrence(id);

                // if can't find link to figure, ignore
                if (!first)
                    continue;

                // make jump button
                const button = document.createElement('button');
                button.classList.add('icon_button', 'jump_arrow');
                button.title =
                    'Jump to the first occurence of this figure in the document';
                button.innerHTML = document.querySelector(
                    '.icon_angle_double_up'
                ).innerHTML;
                button.dataset.id = id;
                button.dataset.ignore = 'true';
                container.insertBefore(button, container.firstElementChild);
                button.addEventListener('click', onButtonClick);
            }
        }

        // add button next to each figure
        function makeTableButtons() {
            const tables = document.querySelectorAll('[id^="tbl:"]');
            for (const table of tables) {
                // get ref id and element to add button to
                const id = table.id;
                const container = table.querySelector('caption') || table;
                const first = getFirstOccurrence(id);

                // if can't find link to table, ignore
                if (!first)
                    continue;

                // make jump button
                const button = document.createElement('button');
                button.classList.add('icon_button', 'jump_arrow');
                button.title =
                    'Jump to the first occurence of this table in the document';
                button.innerHTML = document.querySelector(
                    '.icon_angle_double_up'
                ).innerHTML;
                button.dataset.id = id;
                button.dataset.ignore = 'true';
                container.insertBefore(button, container.firstElementChild);
                button.addEventListener('click', onButtonClick);
            }
        }

        // scroll to and focus element
        function goToElement(element, offset) {
            // expand accordion section if collapsed
            expandElement(element);
            const y =
                getRectInView(element).top -
                getRectInView(document.documentElement).top -
                (offset || 0);
            // trigger any function listening for "onscroll" event
            window.dispatchEvent(new Event('scroll'));
            window.scrollTo(0, y);
            document.activeElement.blur();
            element.focus();
        }

        // get position/dimensions of element or viewport
        function getRectInView(element) {
            let rect = {};
            rect.left = 0;
            rect.top = 0;
            rect.right = document.documentElement.clientWidth;
            rect.bottom = document.documentElement.clientHeight;
            let style = {};

            if (element instanceof HTMLElement) {
                rect = element.getBoundingClientRect();
                style = window.getComputedStyle(element);
            }

            const margin = {};
            margin.left = parseFloat(style.marginLeftWidth) || 0;
            margin.top = parseFloat(style.marginTopWidth) || 0;
            margin.right = parseFloat(style.marginRightWidth) || 0;
            margin.bottom = parseFloat(style.marginBottomWidth) || 0;

            const border = {};
            border.left = parseFloat(style.borderLeftWidth) || 0;
            border.top = parseFloat(style.borderTopWidth) || 0;
            border.right = parseFloat(style.borderRightWidth) || 0;
            border.bottom = parseFloat(style.borderBottomWidth) || 0;

            const newRect = {};
            newRect.left = rect.left + margin.left + border.left;
            newRect.top = rect.top + margin.top + border.top;
            newRect.right = rect.right + margin.right + border.right;
            newRect.bottom = rect.bottom + margin.bottom + border.bottom;
            newRect.width = newRect.right - newRect.left;
            newRect.height = newRect.bottom - newRect.top;

            return newRect;
        }

        // get closest element before specified element that matches query
        function firstBefore(element, query) {
            while (
                element &&
                element !== document.body &&
                !element.matches(query)
            )
                element = element.previousElementSibling || element.parentNode;

            return element;
        }

        // check if element is part of collapsed heading
        function isCollapsed(element) {
            while (element && element !== document.body) {
                if (element.dataset.collapsed === 'true')
                    return true;
                element = element.parentNode;
            }
            return false;
        }

        // (for interaction with accordion plugin)
        // expand heading containing element if necesary
        function expandElement(element) {
            if (isCollapsed(element)) {
                const heading = firstBefore(element, 'h2');
                if (heading)
                    heading.click();
            }
        }

        // load options from url parameters
        function loadOptions() {
            const url = window.location.search;
            const params = new URLSearchParams(url);
            for (const optionName of Object.keys(options)) {
                const paramName = pluginName + '-' + optionName;
                const param = params.get(paramName);
                if (param !== '' && param !== null)
                    options[optionName] = param;
            }
        }
        loadOptions();

        // start script when document is finished loading
        if (options.enabled === 'true')
            window.addEventListener('load', start);
    })();
</script>

<!-- angle double up icon -->

<template class="icon_angle_double_up">
    <!-- modified from: https://fontawesome.com/icons/angle-double-up -->
    <svg width="16" height="16" viewBox="0 0 320 512">
        <path
            fill="currentColor"
            d="M177 255.7l136 136c9.4 9.4 9.4 24.6 0 33.9l-22.6 22.6c-9.4 9.4-24.6 9.4-33.9 0L160 351.9l-96.4 96.4c-9.4 9.4-24.6 9.4-33.9 0L7 425.7c-9.4-9.4-9.4-24.6 0-33.9l136-136c9.4-9.5 24.6-9.5 34-.1zm-34-192L7 199.7c-9.4 9.4-9.4 24.6 0 33.9l22.6 22.6c9.4 9.4 24.6 9.4 33.9 0l96.4-96.4 96.4 96.4c9.4 9.4 24.6 9.4 33.9 0l22.6-22.6c9.4-9.4 9.4-24.6 0-33.9l-136-136c-9.2-9.4-24.4-9.4-33.8 0z"
        ></path>
    </svg>
</template>
<!-- link highlight plugin -->

<script>
    (function() {
        // /////////////////////////
        // DESCRIPTION
        // /////////////////////////

        // This Manubot plugin makes it such that when a user hovers or
        // focuses a link, other links that have the same target will be
        // highlighted. It also makes it such that when clicking a link, the
        // target of the link (eg reference, figure, table) is briefly
        // highlighted.

        // /////////////////////////
        // OPTIONS
        // /////////////////////////

        // plugin name prefix for url parameters
        const pluginName = 'linkHighlight';

        // default plugin options
        const options = {
            // whether to also highlight links that go to external urls
            externalLinks: 'false',
            // whether user must click off to unhighlight instead of just
            // un-hovering
            clickUnhighlight: 'false',
            // whether to also highlight links that are unique
            highlightUnique: 'true',
            // whether plugin is on or not
            enabled: 'true'
        };

        // change options above, or override with url parameter, eg:
        // 'manuscript.html?pluginName-enabled=false'

        // /////////////////////////
        // SCRIPT
        // /////////////////////////

        // start script
        function start() {
            const links = getLinks();
            for (const link of links) {
                // attach mouse and focus listeners to link
                link.addEventListener('mouseenter', onLinkFocus);
                link.addEventListener('focus', onLinkFocus);
                link.addEventListener('mouseleave', onLinkUnhover);
            }

            // attach click and hash change listeners to window
            window.addEventListener('click', onClick);
            window.addEventListener('touchstart', onClick);
            window.addEventListener('hashchange', onHashChange);

            // run hash change on window load in case user has navigated
            // directly to hash
            onHashChange();
        }

        // when link is focused (tabbed to) or hovered
        function onLinkFocus() {
            highlight(this);
        }

        // when link is unhovered
        function onLinkUnhover() {
            if (options.clickUnhighlight !== 'true')
                unhighlightAll();
        }

        // when the mouse is clicked anywhere in window
        function onClick(event) {
            unhighlightAll();
        }

        // when hash (eg manuscript.html#introduction) changes
        function onHashChange() {
            const target = getHashTarget();
            if (target)
                glowElement(target);
        }

        // get element that is target of link or url hash
        function getHashTarget(link) {
            const hash = link ? link.hash : window.location.hash;
            const id = hash.slice(1);
            let target = document.querySelector('[id="' + id + '"]');
            if (!target)
                return;

            return target;
        }

        // start glow sequence on an element
        function glowElement(element) {
            const startGlow = function() {
                onGlowEnd();
                element.dataset.glow = 'true';
                element.addEventListener('animationend', onGlowEnd);
            };
            const onGlowEnd = function() {
                element.removeAttribute('data-glow');
                element.removeEventListener('animationend', onGlowEnd);
            };
            startGlow();
        }

        // highlight link and all others with same target
        function highlight(link) {
            // force unhighlight all to start fresh
            unhighlightAll();

            // get links with same target
            if (!link)
                return;
            const sameLinks = getSameLinks(link);

            // if link unique and option is off, exit and don't highlight
            if (sameLinks.length <= 1 && options.highlightUnique !== 'true')
                return;

            // highlight all same links, and "select" (special highlight) this
            // one
            for (const sameLink of sameLinks) {
                if (sameLink === link)
                    sameLink.setAttribute('data-selected', 'true');
                else
                    sameLink.setAttribute('data-highlighted', 'true');
            }
        }

        // unhighlight all links
        function unhighlightAll() {
            const links = getLinks();
            for (const link of links) {
                link.setAttribute('data-selected', 'false');
                link.setAttribute('data-highlighted', 'false');
            }
        }

        // get links with same target
        function getSameLinks(link) {
            const results = [];
            const links = getLinks();
            for (const otherLink of links) {
                if (
                    otherLink.getAttribute('href') === link.getAttribute('href')
                )
                    results.push(otherLink);
            }
            return results;
        }

        // get all links of types we wish to handle
        function getLinks() {
            let query = 'a';
            if (options.externalLinks !== 'true')
                query += '[href^="#"]';
            // exclude buttons, anchor links, toc links, etc
            query +=
                ':not(.button):not(.icon_button):not(.anchor):not(.toc_link)';
            return document.querySelectorAll(query);
        }

        // load options from url parameters
        function loadOptions() {
            const url = window.location.search;
            const params = new URLSearchParams(url);
            for (const optionName of Object.keys(options)) {
                const paramName = pluginName + '-' + optionName;
                const param = params.get(paramName);
                if (param !== '' && param !== null)
                    options[optionName] = param;
            }
        }
        loadOptions();

        // start script when document is finished loading
        if (options.enabled === 'true')
            window.addEventListener('load', start);
    })();
</script>
<!-- table of contents plugin -->

<script>
    (function() {
        // /////////////////////////
        // DESCRIPTION
        // /////////////////////////

        // This Manubot plugin provides a "table of contents" (toc) panel on
        // the side of the document that allows the user to conveniently
        // navigate between sections of the document.

        // /////////////////////////
        // OPTIONS
        // /////////////////////////

        // plugin name prefix for url parameters
        const pluginName = 'tableOfContents';

        // default plugin options
        const options = {
            // which types of elements to add links for, in
            // "document.querySelector" format
            typesQuery: 'h1, h2, h3',
            // whether default behavior is to be closed ('false'), open
            // ('true'), or only open when screen wide enough to fit panel
            // ('auto'). note: still always starts closed when page loads.
            open: 'auto',
            // if list item is more than this many characters, text will be
            // truncated
            charLimit: '50',
            // whether or not to show bullets next to each toc item
            bullets: 'false',
            // whether plugin is on or not
            enabled: 'true'
        };

        // change options above, or override with url parameter, eg:
        // 'manuscript.html?pluginName-enabled=false'

        // /////////////////////////
        // SCRIPT
        // /////////////////////////

        // start script
        function start() {
            // make toc panel and populate with entries (links to document
            // sections)
            const panel = makePanel();
            if (!panel)
                return;
            makeEntries(panel);
            document.body.insertBefore(panel, document.body.firstChild);

            closePanel();

            // attach click, scroll, and hash change listeners to window
            window.addEventListener('click', onClick);
            window.addEventListener('touchstart', onClick);
            window.addEventListener('scroll', onScroll);
            window.addEventListener('hashchange', onScroll);
            window.addEventListener('keyup', onKeyUp);
            onScroll();

            // add class to push document body down out of way of toc button
            document.body.classList.add('toc_body_nudge');
        }

        // determine if screen wide enough to fit toc panel
        function isSmallScreen() {
            // in default theme:
            // 816px = 8.5in = width of "page" (<body>) element
            // 260px = min width of toc panel (*2 for both sides of <body>)
            return window.innerWidth < 816 + 260 * 2;
        }

        // open/close panel based on option and screen size
        function openOrClosePanel() {
            if (
                options.open === 'true' ||
                (options.open === 'auto' && !isSmallScreen())
            )
                openPanel();
            else
                closePanel();
        }

        // when mouse is clicked anywhere in window
        function onClick() {
            const panel = document.getElementById('toc_panel');
            if (!panel)
                return;

            if (panel.dataset.open === 'true')
                openOrClosePanel();
        }

        // when window is scrolled or hash changed
        function onScroll() {
            highlightViewed();
        }

        // when key pressed
        function onKeyUp(event) {
            if (!event || !event.key)
                return;

            // close on esc
            if (event.key === 'Escape')
                closePanel();
        }

        // find entry of currently viewed document section in toc and highlight
        function highlightViewed() {
            const firstId = getFirstInView(options.typesQuery);

            // get toc entries (links), unhighlight all, then highlight viewed
            const list = document.getElementById('toc_list');
            if (!firstId || !list)
                return;
            const links = list.querySelectorAll('a');
            for (const link of links)
                link.dataset.viewing = 'false';
            const link = list.querySelector('a[href="#' + firstId + '"]');
            if (!link)
                return;
            link.dataset.viewing = 'true';
        }

        // get first or previous toc listed element in top half of view
        function getFirstInView(query) {
            // get all elements matching query and with id
            const elements = document.querySelectorAll(query);
            const elementsWithIds = [];
            for (const element of elements) {
                if (element.id)
                    elementsWithIds.push(element);
            }


            // get first or previous element in top half of view
            for (let i = 0; i < elementsWithIds.length; i++) {
                const element = elementsWithIds[i];
                const prevElement = elementsWithIds[Math.max(0, i - 1)];
                if (element.getBoundingClientRect().top >= 0) {
                    if (
                        element.getBoundingClientRect().top <
                        window.innerHeight / 2
                    )
                        return element.id;
                    else
                        return prevElement.id;
                }
            }
        }

        // make panel
        function makePanel() {
            // create panel
            const panel = document.createElement('div');
            panel.id = 'toc_panel';
            if (options.bullets === 'true')
                panel.dataset.bullets = 'true';

            // create header
            const header = document.createElement('div');
            header.id = 'toc_header';

            // create toc button
            const button = document.createElement('button');
            button.id = 'toc_button';
            button.innerHTML = document.querySelector(
                '.icon_th_list'
            ).innerHTML;
            button.classList.add('icon_button');

            // create header text
            const text = document.createElement('h3');
            text.innerHTML = 'Table of Contents';

            // create container for toc list
            const list = document.createElement('div');
            list.id = 'toc_list';

            // attach click listeners
            panel.addEventListener('click', onPanelClick);
            header.addEventListener('click', onHeaderClick);
            button.addEventListener('click', onButtonClick);

            // attach elements
            header.appendChild(button);
            header.appendChild(text);
            panel.appendChild(header);
            panel.appendChild(list);

            return panel;
        }

        // create toc entries (links) to each element of the specified types
        function makeEntries(panel) {
            const elements = document.querySelectorAll(options.typesQuery);
            for (const element of elements) {
                // do not add link if element doesn't have assigned id
                if (!element.id)
                    continue;

                // create link/list item
                const link = document.createElement('a');
                link.classList.add('toc_link');
                switch (element.tagName.toLowerCase()) {
                    case 'h1':
                        link.dataset.level = '1';
                        break;
                    case 'h2':
                        link.dataset.level = '2';
                        break;
                    case 'h3':
                        link.dataset.level = '3';
                        break;
                    case 'h4':
                        link.dataset.level = '4';
                        break;
                }
                link.title = element.innerText;
                let text = element.innerText;
                if (text.length > options.charLimit)
                    text = text.slice(0, options.charLimit) + '...';
                link.innerHTML = text;
                link.href = '#' + element.id;
                link.addEventListener('click', onLinkClick);

                // attach link
                panel.querySelector('#toc_list').appendChild(link);
            }
        }

        // when panel is clicked
        function onPanelClick(event) {
            // stop click from propagating to window/document and closing panel
            event.stopPropagation();
        }

        // when header itself is clicked
        function onHeaderClick(event) {
            togglePanel();
        }

        // when button is clicked
        function onButtonClick(event) {
            togglePanel();
            // stop header underneath button from also being clicked
            event.stopPropagation();
        }

        // when link is clicked
        function onLinkClick() {
            openOrClosePanel();
        }

        // open panel if closed, close if opened
        function togglePanel() {
            const panel = document.getElementById('toc_panel');
            if (!panel)
                return;

            if (panel.dataset.open === 'true')
                closePanel();
            else
                openPanel();
        }

        // open panel
        function openPanel() {
            const panel = document.getElementById('toc_panel');
            if (panel)
                panel.dataset.open = 'true';
        }

        // close panel
        function closePanel() {
            const panel = document.getElementById('toc_panel');
            if (panel)
                panel.dataset.open = 'false';
        }

        // load options from url parameters
        function loadOptions() {
            const url = window.location.search;
            const params = new URLSearchParams(url);
            for (const optionName of Object.keys(options)) {
                const paramName = pluginName + '-' + optionName;
                const param = params.get(paramName);
                if (param !== '' && param !== null)
                    options[optionName] = param;
            }
        }
        loadOptions();

        // start script when document is finished loading
        if (options.enabled === 'true')
            window.addEventListener('load', start);
    })();
</script>

<!-- th list icon -->

<template class="icon_th_list">
    <!-- modified from: https://fontawesome.com/icons/th-list -->
    <svg width="16" height="16" viewBox="0 0 512 512" tabindex="-1">
        <path
            fill="currentColor"
            d="M96 96c0 26.51-21.49 48-48 48S0 122.51 0 96s21.49-48 48-48 48 21.49 48 48zM48 208c-26.51 0-48 21.49-48 48s21.49 48 48 48 48-21.49 48-48-21.49-48-48-48zm0 160c-26.51 0-48 21.49-48 48s21.49 48 48 48 48-21.49 48-48-21.49-48-48-48zm96-236h352c8.837 0 16-7.163 16-16V76c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h352c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h352c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16z"
            tabindex="-1"
        ></path>
    </svg>
</template>
<!-- lightbox plugin -->

<script>
    (function() {
        // /////////////////////////
        // DESCRIPTION
        // /////////////////////////

        // This Manubot plugin makes it such that when a user clicks on an
        // image, the image fills the screen and the user can pan/drag/zoom
        // the image and navigate between other images in the document.

        // /////////////////////////
        // OPTIONS
        // /////////////////////////

        // plugin name prefix for url parameters
        const pluginName = 'lightbox';

        // default plugin options
        const options = {
            // list of possible zoom/scale factors
            zoomSteps:
                '0.1, 0.25, 0.333333, 0.5, 0.666666, 0.75, 1,' +
                '1.25, 1.5, 1.75, 2, 2.5, 3, 3.5, 4, 5, 6, 7, 8',
            // whether to fit image to view ('fit'), display at 100% and shrink
            // if necessary ('shrink'), or always display at 100% ('100')
            defaultZoom: 'fit',
            // whether to zoom in/out toward center of view ('true') or mouse
            // ('false')
            centerZoom: 'false',
            // whether plugin is on or not
            enabled: 'true'
        };

        // change options above, or override with url parameter, eg:
        // 'manuscript.html?pluginName-enabled=false'

        // /////////////////////////
        // SCRIPT
        // /////////////////////////

        // start script
        function start() {
            // run through each <img> element
            const imgs = document.querySelectorAll('figure > img');
            let count = 1;
            for (const img of imgs) {
                img.classList.add('lightbox_document_img');
                img.dataset.number = count;
                img.dataset.total = imgs.length;
                img.addEventListener('click', openLightbox);
                count++;
            }

            // attach mouse and key listeners to window
            window.addEventListener('mousemove', onWindowMouseMove);
            window.addEventListener('keyup', onKeyUp);
        }

        // when mouse is moved anywhere in window
        function onWindowMouseMove(event) {
            window.mouseX = event.clientX;
            window.mouseY = event.clientY;
        }

        // when key pressed
        function onKeyUp(event) {
            if (!event || !event.key)
                return;

            switch (event.key) {
                // trigger click of prev button
                case 'ArrowLeft':
                    const prevButton = document.getElementById(
                        'lightbox_prev_button'
                    );
                    if (prevButton)
                        prevButton.click();
                    break;
                // trigger click of next button
                case 'ArrowRight':
                    const nextButton = document.getElementById(
                        'lightbox_next_button'
                    );
                    if (nextButton)
                        nextButton.click();
                    break;
                // close on esc
                case 'Escape':
                    closeLightbox();
                    break;
            }
        }

        // open lightbox
        function openLightbox() {
            const lightbox = makeLightbox(this);
            if (!lightbox)
                return;

            blurBody(lightbox);
            document.body.appendChild(lightbox);
        }

        // make lightbox
        function makeLightbox(img) {
            // delete lightbox if it exists, start fresh
            closeLightbox();

            // create screen overlay containing lightbox
            const overlay = document.createElement('div');
            overlay.id = 'lightbox_overlay';

            // create image info boxes
            const numberInfo = document.createElement('div');
            const zoomInfo = document.createElement('div');
            numberInfo.id = 'lightbox_number_info';
            zoomInfo.id = 'lightbox_zoom_info';

            // create container for image
            const imageContainer = document.createElement('div');
            imageContainer.id = 'lightbox_image_container';
            const lightboxImg = makeLightboxImg(
                img,
                imageContainer,
                numberInfo,
                zoomInfo
            );
            imageContainer.appendChild(lightboxImg);

            // create bottom container for caption and navigation buttons
            const bottomContainer = document.createElement('div');
            bottomContainer.id = 'lightbox_bottom_container';
            const caption = makeCaption(img);
            const prevButton = makePrevButton(img);
            const nextButton = makeNextButton(img);
            bottomContainer.appendChild(prevButton);
            bottomContainer.appendChild(caption);
            bottomContainer.appendChild(nextButton);

            // attach top middle and bottom to overlay
            overlay.appendChild(numberInfo);
            overlay.appendChild(zoomInfo);
            overlay.appendChild(imageContainer);
            overlay.appendChild(bottomContainer);

            return overlay;
        }

        // make <img> object that is intuitively draggable and zoomable
        function makeLightboxImg(
            sourceImg,
            container,
            numberInfoBox,
            zoomInfoBox
        ) {
            // create copy of source <img>
            const img = sourceImg.cloneNode(true);
            img.classList.remove('lightbox_document_img');
            img.removeAttribute('id');
            img.removeAttribute('width');
            img.removeAttribute('height');
            img.style.position = 'unset';
            img.style.margin = '0';
            img.style.padding = '0';
            img.style.width = '';
            img.style.height = '';
            img.style.minWidth = '';
            img.style.minHeight = '';
            img.style.maxWidth = '';
            img.style.maxHeight = '';
            img.id = 'lightbox_img';

            // build sorted list of unique zoomSteps, always including a 100%
            let zoomSteps = [];
            const optionsZooms = options.zoomSteps.split(/[^0-9.]/);
            for (const optionZoom of optionsZooms) {
                const newZoom = parseFloat(optionZoom);
                if (newZoom && !zoomSteps.includes(newZoom))
                    zoomSteps.push(newZoom);
            }
            if (!zoomSteps.includes(1))
                zoomSteps.push(1);
            zoomSteps = zoomSteps.sort(function sortNumber(a, b) {
                return a - b;
            });

            // <img> object property variables
            let zoom = 1;
            let translateX = 0;
            let translateY = 0;
            let clickMouseX = undefined;
            let clickMouseY = undefined;
            let clickTranslateX = undefined;
            let clickTranslateY = undefined;

            updateNumberInfo();

            // update image numbers displayed in info box
            function updateNumberInfo() {
                numberInfoBox.innerHTML =
                    sourceImg.dataset.number + ' of ' + sourceImg.dataset.total;
            }

            // update zoom displayed in info box
            function updateZoomInfo() {
                let zoomInfo = zoom * 100;
                if (!Number.isInteger(zoomInfo))
                    zoomInfo = zoomInfo.toFixed(2);
                zoomInfoBox.innerHTML = zoomInfo + '%';
            }

            // move to closest zoom step above current zoom
            const zoomIn = function() {
                for (const zoomStep of zoomSteps) {
                    if (zoomStep > zoom) {
                        zoom = zoomStep;
                        break;
                    }
                }
                updateTransform();
            };

            // move to closest zoom step above current zoom
            const zoomOut = function() {
                zoomSteps.reverse();
                for (const zoomStep of zoomSteps) {
                    if (zoomStep < zoom) {
                        zoom = zoomStep;
                        break;
                    }
                }
                zoomSteps.reverse();

                updateTransform();
            };

            // update display of <img> based on scale/translate properties
            const updateTransform = function() {
                // set transform
                img.style.transform =
                    'translate(' +
                    (translateX || 0) +
                    'px,' +
                    (translateY || 0) +
                    'px) scale(' +
                    (zoom || 1) +
                    ')';

                // get new width/height after scale
                const rect = img.getBoundingClientRect();
                // limit translate
                translateX = Math.max(translateX, -rect.width / 2);
                translateX = Math.min(translateX, rect.width / 2);
                translateY = Math.max(translateY, -rect.height / 2);
                translateY = Math.min(translateY, rect.height / 2);

                // set transform
                img.style.transform =
                    'translate(' +
                    (translateX || 0) +
                    'px,' +
                    (translateY || 0) +
                    'px) scale(' +
                    (zoom || 1) +
                    ')';

                updateZoomInfo();
            };

            // fit <img> to container
            const fit = function() {
                // no x/y offset, 100% zoom by default
                translateX = 0;
                translateY = 0;
                zoom = 1;

                // widths of <img> and container
                const imgWidth = img.naturalWidth;
                const imgHeight = img.naturalHeight;
                const containerWidth = parseFloat(
                    window.getComputedStyle(container).width
                );
                const containerHeight = parseFloat(
                    window.getComputedStyle(container).height
                );

                // how much zooming is needed to fit <img> to container
                const xRatio = imgWidth / containerWidth;
                const yRatio = imgHeight / containerHeight;
                const maxRatio = Math.max(xRatio, yRatio);
                const newZoom = 1 / maxRatio;

                // fit <img> to container according to option
                if (options.defaultZoom === 'shrink') {
                    if (maxRatio > 1)
                        zoom = newZoom;
                } else if (options.defaultZoom === 'fit')
                    zoom = newZoom;

                updateTransform();
            };

            // when mouse wheel is rolled anywhere in container
            const onContainerWheel = function(event) {
                if (!event)
                    return;

                // let ctrl + mouse wheel to zoom behave as normal
                if (event.ctrlKey)
                    return;

                // prevent normal scroll behavior
                event.preventDefault();
                event.stopPropagation();

                // point around which to scale img
                const viewRect = container.getBoundingClientRect();
                const viewX = (viewRect.left + viewRect.right) / 2;
                const viewY = (viewRect.top + viewRect.bottom) / 2;
                const originX = options.centerZoom === 'true' ? viewX : mouseX;
                const originY = options.centerZoom === 'true' ? viewY : mouseY;

                // get point on image under origin
                const oldRect = img.getBoundingClientRect();
                const oldPercentX = (originX - oldRect.left) / oldRect.width;
                const oldPercentY = (originY - oldRect.top) / oldRect.height;

                // increment/decrement zoom
                if (event.deltaY < 0)
                    zoomIn();
                if (event.deltaY > 0)
                    zoomOut();

                // get offset between previous image point and origin
                const newRect = img.getBoundingClientRect();
                const offsetX =
                    originX - (newRect.left + newRect.width * oldPercentX);
                const offsetY =
                    originY - (newRect.top + newRect.height * oldPercentY);

                // translate image to keep image point under origin
                translateX += offsetX;
                translateY += offsetY;

                // perform translate
                updateTransform();
            };

            // when container is clicked
            function onContainerClick(event) {
                // if container itself is target of click, and not other
                // element above it
                if (event.target === this)
                    closeLightbox();
            }

            // when mouse button is pressed on image
            const onImageMouseDown = function(event) {
                // store original mouse position relative to image
                clickMouseX = window.mouseX;
                clickMouseY = window.mouseY;
                clickTranslateX = translateX;
                clickTranslateY = translateY;
                event.stopPropagation();
                event.preventDefault();
            };

            // when mouse button is released anywhere in window
            const onWindowMouseUp = function(event) {
                // reset original mouse position
                clickMouseX = undefined;
                clickMouseY = undefined;
                clickTranslateX = undefined;
                clickTranslateY = undefined;

                // remove global listener if lightbox removed from document
                if (!document.body.contains(container))
                    window.removeEventListener('mouseup', onWindowMouseUp);
            };

            // when mouse is moved anywhere in window
            const onWindowMouseMove = function(event) {
                if (
                    clickMouseX === undefined ||
                    clickMouseY === undefined ||
                    clickTranslateX === undefined ||
                    clickTranslateY === undefined
                )
                    return;

                // offset image based on original and current mouse position
                translateX = clickTranslateX + window.mouseX - clickMouseX;
                translateY = clickTranslateY + window.mouseY - clickMouseY;
                updateTransform();
                event.preventDefault();

                // remove global listener if lightbox removed from document
                if (!document.body.contains(container))
                    window.removeEventListener('mousemove', onWindowMouseMove);
            };

            // when window is resized
            const onWindowResize = function(event) {
                fit();

                // remove global listener if lightbox removed from document
                if (!document.body.contains(container))
                    window.removeEventListener('resize', onWindowResize);
            };

            // attach the necessary event listeners
            img.addEventListener('dblclick', fit);
            img.addEventListener('mousedown', onImageMouseDown);
            container.addEventListener('wheel', onContainerWheel);
            container.addEventListener('mousedown', onContainerClick);
            container.addEventListener('touchstart', onContainerClick);
            window.addEventListener('mouseup', onWindowMouseUp);
            window.addEventListener('mousemove', onWindowMouseMove);
            window.addEventListener('resize', onWindowResize);

            // run fit() after lightbox atttached to document and <img> Loaded
            // so needed container and img dimensions available
            img.addEventListener('load', fit);

            return img;
        }

        // make caption
        function makeCaption(img) {
            const caption = document.createElement('div');
            caption.id = 'lightbox_caption';
            const captionSource = img.nextElementSibling;
            if (captionSource.tagName.toLowerCase() === 'figcaption') {
                const captionCopy = makeCopy(captionSource);
                caption.innerHTML = captionCopy.innerHTML;
            }

            caption.addEventListener('touchstart', function(event) {
                event.stopPropagation();
            });

            return caption;
        }

        // make carbon copy of html dom element
        function makeCopy(source) {
            const sourceCopy = source.cloneNode(true);

            // delete elements marked with ignore (eg anchor and jump buttons)
            const deleteFromCopy = sourceCopy.querySelectorAll(
                '[data-ignore="true"]'
            );
            for (const element of deleteFromCopy)
                element.remove();

            // delete certain element attributes
            const attributes = [
                'id',
                'data-collapsed',
                'data-selected',
                'data-highlighted',
                'data-glow'
            ];
            for (const attribute of attributes) {
                sourceCopy.removeAttribute(attribute);
                const elements = sourceCopy.querySelectorAll(
                    '[' + attribute + ']'
                );
                for (const element of elements)
                    element.removeAttribute(attribute);
            }

            return sourceCopy;
        }

        // make button to jump to previous image in document
        function makePrevButton(img) {
            const prevButton = document.createElement('button');
            prevButton.id = 'lightbox_prev_button';
            prevButton.title = 'Jump to the previous image in the document [←]';
            prevButton.classList.add('icon_button', 'lightbox_button');
            prevButton.innerHTML = document.querySelector(
                '.icon_caret_left'
            ).innerHTML;

            // attach click listeners to button
            prevButton.addEventListener('click', function() {
                getPrevImg(img).click();
            });

            return prevButton;
        }

        // make button to jump to next image in document
        function makeNextButton(img) {
            const nextButton = document.createElement('button');
            nextButton.id = 'lightbox_next_button';
            nextButton.title = 'Jump to the next image in the document [→]';
            nextButton.classList.add('icon_button', 'lightbox_button');
            nextButton.innerHTML = document.querySelector(
                '.icon_caret_right'
            ).innerHTML;

            // attach click listeners to button
            nextButton.addEventListener('click', function() {
                getNextImg(img).click();
            });

            return nextButton;
        }

        // get previous image in document
        function getPrevImg(img) {
            const imgs = document.querySelectorAll('.lightbox_document_img');

            // find index of provided img
            let index;
            for (index = 0; index < imgs.length; index++) {
                if (imgs[index] === img)
                    break;
            }


            // wrap index to other side if < 1
            if (index - 1 >= 0)
                index--;
            else
                index = imgs.length - 1;
            return imgs[index];
        }

        // get next image in document
        function getNextImg(img) {
            const imgs = document.querySelectorAll('.lightbox_document_img');

            // find index of provided img
            let index;
            for (index = 0; index < imgs.length; index++) {
                if (imgs[index] === img)
                    break;
            }


            // wrap index to other side if > total
            if (index + 1 <= imgs.length - 1)
                index++;
            else
                index = 0;
            return imgs[index];
        }

        // close lightbox
        function closeLightbox() {
            focusBody();

            const lightbox = document.getElementById('lightbox_overlay');
            if (lightbox)
                lightbox.remove();
        }

        // make all elements behind lightbox non-focusable
        function blurBody(overlay) {
            const all = document.querySelectorAll('*');
            for (const element of all)
                element.tabIndex = -1;
            document.body.classList.add('body_no_scroll');
        }

        // make all elements focusable again
        function focusBody() {
            const all = document.querySelectorAll('*');
            for (const element of all)
                element.removeAttribute('tabIndex');
            document.body.classList.remove('body_no_scroll');
        }

        // load options from url parameters
        function loadOptions() {
            const url = window.location.search;
            const params = new URLSearchParams(url);
            for (const optionName of Object.keys(options)) {
                const paramName = pluginName + '-' + optionName;
                const param = params.get(paramName);
                if (param !== '' && param !== null)
                    options[optionName] = param;
            }
        }
        loadOptions();

        // start script when document is finished loading
        if (options.enabled === 'true')
            window.addEventListener('load', start);
    })();
</script>

<!-- caret left icon -->

<template class="icon_caret_left">
    <!-- modified from: https://fontawesome.com/icons/caret-left -->
    <svg width="16" height="16" viewBox="0 0 192 512">
        <path
            fill="currentColor"
            d="M192 127.338v257.324c0 17.818-21.543 26.741-34.142 14.142L29.196 270.142c-7.81-7.81-7.81-20.474 0-28.284l128.662-128.662c12.599-12.6 34.142-3.676 34.142 14.142z"
        ></path>
    </svg>
</template>

<!-- caret right icon -->

<template class="icon_caret_right">
    <!-- modified from: https://fontawesome.com/icons/caret-right -->
    <svg width="16" height="16" viewBox="0 0 192 512">
        <path
            fill="currentColor"
            d="M0 384.662V127.338c0-17.818 21.543-26.741 34.142-14.142l128.662 128.662c7.81 7.81 7.81 20.474 0 28.284L34.142 398.804C21.543 411.404 0 402.48 0 384.662z"
        ></path>
    </svg>
</template>
<!-- attributes plugin -->

<script>
    (function() {
        // /////////////////////////
        // DESCRIPTION
        // /////////////////////////

        // This Manubot plugin allows arbitrary HTML attributes to be attached
        // to (almost) any element. Place an HTML comment inside or next to the
        // desired element in the format <!-- $attribute="value" -->

        // /////////////////////////
        // OPTIONS
        // /////////////////////////

        // plugin name prefix for url parameters
        const pluginName = 'attributes';

        // default plugin options
        const options = {
            // whether plugin is on or not
            enabled: 'true'
        };

        // change options above, or override with url parameter, eg:
        // 'manuscript.html?pluginName-enabled=false'

        // /////////////////////////
        // SCRIPT
        // /////////////////////////

        // start script
        function start() {
            // get list of comments in document
            const comments = findComments();

            for(const comment of comments)
                if (comment.parentElement)
                    addAttributes(
                        comment.parentElement,
                        comment.nodeValue.trim()
                    );
        }

        // add html attributes to specified element based on string of 
        // html attributes and values
        function addAttributes(element, text) {
            // regex's for finding attribute/value pairs in the format of
            // attribute="value" or attribute='value
            const regex2 = /\$([a-zA-Z\-]+)?=\"(.+?)\"/;
            const regex1 = /\$([a-zA-Z\-]+)?=\'(.+?)\'/;

            // loop through attribute/value pairs
            let match;
            while(match = text.match(regex2) || text.match(regex1)) {
                // get attribute and value from regex capture groups
                let attribute = match[1];
                let value = match[2];

                // remove from string
                text = text.substring(match.index + match[0].length);

                if (!attribute || !value)
                    break;

                // set attribute of parent element
                try {
                    element.setAttribute(attribute, value);
                } catch(error) {
                    console.log(error);
                }

                // special case for colspan
                if (attribute === 'colspan')
                    removeTableCells(element, value);
            }
        }

        // get list of comment elements in document
        function findComments() {
            const comments = [];

            // iterate over comment nodes in document
            function acceptNode(node) {
                return NodeFilter.FILTER_ACCEPT;
            }
            const iterator = document.createNodeIterator(
                document.body,
                NodeFilter.SHOW_COMMENT,
                acceptNode
            );
            let node;
            while(node = iterator.nextNode())
                comments.push(node);

            return comments;
        }

        // remove certain number of cells after specified cell
        function removeTableCells(cell, number) {
            number = parseInt(number);
            if (!number)
                return;

            // remove elements
            for(; number > 1; number--) {
                if (cell.nextElementSibling)
                    cell.nextElementSibling.remove();
            }
        }

        // load options from url parameters
        function loadOptions() {
            const url = window.location.search;
            const params = new URLSearchParams(url);
            for (const optionName of Object.keys(options)) {
                const paramName = pluginName + '-' + optionName;
                const param = params.get(paramName);
                if (param !== '' && param !== null)
                    options[optionName] = param;
            }
        }
        loadOptions();

        // start script when document is finished loading
        if (options.enabled === 'true')
            window.addEventListener('load', start);
    })();
</script>
<!-- math plugin configuration -->

<script type="text/x-mathjax-config">
    MathJax.Hub.Config({
        "CommonHTML": { linebreaks: { automatic: true } },
        "HTML-CSS": { linebreaks: { automatic: true } },
        "SVG": { linebreaks: { automatic: true } },
        "fast-preview": { disabled: true }
    });
</script>

<!-- math plugin -->

<script src='https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML'>
    // /////////////////////////
    // DESCRIPTION
    // /////////////////////////

    // This third-party plugin 'MathJax' allows the proper rendering of
    // math/equations written in LaTeX.

    // https://www.mathjax.org/
</script>
<!-- annotations plugin configuration -->

<script>
    window.hypothesisConfig = function() {
        return {
            branding: {
                accentColor: '#2196f3',
                appBackgroundColor: '#f8f8f8',
                ctaBackgroundColor: '#f8f8f8',
                ctaTextColor: '#000000',
                selectionFontFamily: 'Open Sans, Helvetica, sans serif',
                annotationFontFamily: 'Open Sans, Helvetica, sans serif'
            }
        };
    };
</script>

<!-- annotations plugin -->

<script src='https://hypothes.is/embed.js'>
    // /////////////////////////
    // DESCRIPTION
    // /////////////////////////

    // This third-party plugin 'Hypothesis' allows public annotation of the
    // manuscript.

    // https://web.hypothes.is/
</script>
<!-- analytics plugin -->

<!-- copy and paste code from Google Analytics or similar service here -->
</body>
</html>
