<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en-US" xml:lang="en-US">
<head>
  <meta charset="utf-8" />
  <meta name="generator" content="pandoc" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
  <meta name="author" content="Halie M. Rando" />
  <meta name="author" content="Casey S. Greene" />
  <meta name="author" content="Michael P. Robson" />
  <meta name="author" content="Simina M. Boca" />
  <meta name="author" content="Nils Wellhausen" />
  <meta name="author" content="Ronan Lordan" />
  <meta name="author" content="Christian Brueffer" />
  <meta name="author" content="Sandipan Ray" />
  <meta name="author" content="Lucy D'Agostino McGowan" />
  <meta name="author" content="Anthony Gitter" />
  <meta name="author" content="Anna Ada Dattoli" />
  <meta name="author" content="Ryan Velazquez" />
  <meta name="author" content="John P. Barton" />
  <meta name="author" content="Jeffrey M. Field" />
  <meta name="author" content="Bharath Ramsundar" />
  <meta name="author" content="Adam L. MacLean" />
  <meta name="author" content="Alexandra J. Lee" />
  <meta name="author" content="Immunology Institute of the Icahn School of Medicine" />
  <meta name="author" content="Fengling Hu" />
  <meta name="author" content="Nafisa M. Jadavji" />
  <meta name="author" content="Elizabeth Sell" />
  <meta name="author" content="Jinhui Wang" />
  <meta name="author" content="Diane N. Rafizadeh" />
  <meta name="author" content="Ashwin N. Skelly" />
  <meta name="author" content="Marouen Ben Guebila" />
  <meta name="author" content="Likhitha Kolla" />
  <meta name="author" content="David Manheim" />
  <meta name="author" content="Soumita Ghosh" />
  <meta name="author" content="Matthias Fax" />
  <meta name="author" content="James Brian Byrd" />
  <meta name="author" content="YoSon Park" />
  <meta name="author" content="Vikas Bansal" />
  <meta name="author" content="Stephen Capone" />
  <meta name="author" content="John J. Dziak" />
  <meta name="author" content="Yuchen Sun" />
  <meta name="author" content="Yanjun Qi" />
  <meta name="author" content="Lamonica Shinholster" />
  <meta name="author" content="Sergey Knyazev" />
  <meta name="author" content="Dimitri Perrin" />
  <meta name="author" content="Serghei Mangul" />
  <meta name="author" content="Shikta Das" />
  <meta name="author" content="Gregory L Szeto" />
  <meta name="author" content="Tiago Lubiana" />
  <meta name="dcterms.date" content="2020-11-12" />
  <meta name="keywords" content="covid-19, coronavirus, pandemic, viral infection" />
  <title>SARS-CoV-2 and COVID-19: An Evolving Review of Diagnostics and Therapeutics</title>
  <style>
    code{white-space: pre-wrap;}
    span.smallcaps{font-variant: small-caps;}
    span.underline{text-decoration: underline;}
    div.column{display: inline-block; vertical-align: top; width: 50%;}
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
    ul.task-list{list-style: none;}
  </style>
  <!--[if lt IE 9]>
    <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv-printshiv.min.js"></script>
  <![endif]-->
  <!--
  Manubot generated metadata rendered from header-includes-template.html.
  Suggest improvements at https://github.com/manubot/manubot/blob/master/manubot/process/header-includes-template.html
  -->
  <meta name="dc.format" content="text/html" />
  <meta name="dc.title" content="SARS-CoV-2 and COVID-19: An Evolving Review of Diagnostics and Therapeutics" />
  <meta name="citation_title" content="SARS-CoV-2 and COVID-19: An Evolving Review of Diagnostics and Therapeutics" />
  <meta property="og:title" content="SARS-CoV-2 and COVID-19: An Evolving Review of Diagnostics and Therapeutics" />
  <meta property="twitter:title" content="SARS-CoV-2 and COVID-19: An Evolving Review of Diagnostics and Therapeutics" />
  <meta name="dc.date" content="2020-11-12" />
  <meta name="citation_publication_date" content="2020-11-12" />
  <meta name="dc.language" content="en-US" />
  <meta name="citation_language" content="en-US" />
  <meta name="dc.relation.ispartof" content="Manubot" />
  <meta name="dc.publisher" content="Manubot" />
  <meta name="citation_journal_title" content="Manubot" />
  <meta name="citation_technical_report_institution" content="Manubot" />
  <meta name="citation_author" content="Halie M. Rando" />
  <meta name="citation_author_institution" content="Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America" />
  <meta name="citation_author_orcid" content="0000-0001-7688-1770" />
  <meta name="twitter:creator" content="@tamefoxtime" />
  <meta name="citation_author" content="Casey S. Greene" />
  <meta name="citation_author_institution" content="Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America" />
  <meta name="citation_author_institution" content="Childhood Cancer Data Lab, Alex&#39;s Lemonade Stand Foundation, Philadelphia, Pennsylvania, United States of America" />
  <meta name="citation_author_orcid" content="0000-0001-8713-9213" />
  <meta name="twitter:creator" content="@GreeneScientist" />
  <meta name="citation_author" content="Michael P. Robson" />
  <meta name="citation_author_institution" content="Department of Computing Sciences, Villanova University, Villanova, Pennsylvania, United States of America" />
  <meta name="citation_author_orcid" content="0000-0002-4859-0033" />
  <meta name="citation_author" content="Simina M. Boca" />
  <meta name="citation_author_institution" content="Innovation Center for Biomedical Informatics, Georgetown University Medical Center, Washington, District of Columbia, United States of America" />
  <meta name="citation_author_orcid" content="0000-0002-1400-3398" />
  <meta name="citation_author" content="Nils Wellhausen" />
  <meta name="citation_author_institution" content="Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America" />
  <meta name="citation_author_orcid" content="0000-0001-8955-7582" />
  <meta name="citation_author" content="Ronan Lordan" />
  <meta name="citation_author_institution" content="Institute for Translational Medicine and Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104-5158, USA" />
  <meta name="citation_author_orcid" content="0000-0001-9668-3368" />
  <meta name="twitter:creator" content="@el_ronan" />
  <meta name="citation_author" content="Christian Brueffer" />
  <meta name="citation_author_institution" content="Department of Clinical Sciences, Lund University, Lund, Sweden" />
  <meta name="citation_author_orcid" content="0000-0002-3826-0989" />
  <meta name="twitter:creator" content="@cbrueffer" />
  <meta name="citation_author" content="Sandipan Ray" />
  <meta name="citation_author_institution" content="Department of Systems Pharmacology &amp; Translational Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA" />
  <meta name="citation_author_institution" content="Institute for Translational Medicine and Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA" />
  <meta name="citation_author_orcid" content="0000-0002-9960-5768" />
  <meta name="citation_author" content="Lucy D\&#39;Agostino McGowan" />
  <meta name="citation_author_institution" content="Department of Mathematics and Statistics, Wake Forest University, Winston-Salem, North Carolina, United States of America" />
  <meta name="citation_author_orcid" content="0000-0001-7297-9359" />
  <meta name="twitter:creator" content="@LucyStats" />
  <meta name="citation_author" content="Anthony Gitter" />
  <meta name="citation_author_institution" content="Department of Biostatistics and Medical Informatics, University of Wisconsin-Madison, Madison, Wisconsin, United States of America" />
  <meta name="citation_author_institution" content="Morgridge Institute for Research, Madison, Wisconsin, United States of America" />
  <meta name="citation_author_orcid" content="0000-0002-5324-9833" />
  <meta name="twitter:creator" content="@anthonygitter" />
  <meta name="citation_author" content="Anna Ada Dattoli" />
  <meta name="citation_author_institution" content="Department of Systems Pharmacology &amp; Translational Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA" />
  <meta name="citation_author_orcid" content="0000-0003-1462-831X" />
  <meta name="twitter:creator" content="@aadattoli" />
  <meta name="citation_author" content="Ryan Velazquez" />
  <meta name="citation_author_institution" content="Azimuth1, McLean, VA" />
  <meta name="citation_author_orcid" content="0000-0002-3655-3403" />
  <meta name="citation_author" content="John P. Barton" />
  <meta name="citation_author_institution" content="Department of Physics and Astronomy, University of California-Riverside, Riverside, California, United States of America" />
  <meta name="citation_author_orcid" content="0000-0003-1467-421X" />
  <meta name="twitter:creator" content="@_jpbarton" />
  <meta name="citation_author" content="Jeffrey M. Field" />
  <meta name="citation_author_institution" content="Department of Pharmacology, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA" />
  <meta name="citation_author_orcid" content="0000-0001-7161-7284" />
  <meta name="citation_author" content="Bharath Ramsundar" />
  <meta name="citation_author_institution" content="The DeepChem Project, https://deepchem.io/" />
  <meta name="citation_author_orcid" content="0000-0001-8450-4262" />
  <meta name="twitter:creator" content="@rbhar90" />
  <meta name="citation_author" content="Adam L. MacLean" />
  <meta name="citation_author_institution" content="Department of Biological Sciences, University of Southern California, Los Angeles, California, United States of America" />
  <meta name="citation_author_orcid" content="0000-0003-0689-7907" />
  <meta name="twitter:creator" content="@adamlmaclean" />
  <meta name="citation_author" content="Alexandra J. Lee" />
  <meta name="citation_author_institution" content="Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America" />
  <meta name="citation_author_orcid" content="0000-0002-0208-3730" />
  <meta name="citation_author" content="Immunology Institute of the Icahn School of Medicine" />
  <meta name="citation_author_institution" content="Immunology Institute of the Icahn School of Medicine" />
  <meta name="citation_author" content="Fengling Hu" />
  <meta name="citation_author_institution" content="Department of Biostatistics, Epidemiology and Informatics, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America" />
  <meta name="citation_author_orcid" content="0000-0003-1081-5038" />
  <meta name="twitter:creator" content="@hufengling" />
  <meta name="citation_author" content="Nafisa M. Jadavji" />
  <meta name="citation_author_institution" content="Biomedical Science, Midwestern University, Glendale, AZ, United States of America" />
  <meta name="citation_author_institution" content="Department of Neuroscience, Carleton University, Ottawa, Ontario, Canada" />
  <meta name="citation_author_orcid" content="0000-0002-3557-7307" />
  <meta name="twitter:creator" content="@nafisajadavji" />
  <meta name="citation_author" content="Elizabeth Sell" />
  <meta name="citation_author_institution" content="Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America" />
  <meta name="citation_author_orcid" content="0000-0002-9658-1107" />
  <meta name="citation_author" content="Jinhui Wang" />
  <meta name="citation_author_institution" content="Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America" />
  <meta name="citation_author_orcid" content="0000-0002-5796-8130" />
  <meta name="citation_author" content="Diane N. Rafizadeh" />
  <meta name="citation_author_institution" content="Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America" />
  <meta name="citation_author_institution" content="Department of Chemistry, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America" />
  <meta name="citation_author_orcid" content="0000-0002-2838-067X" />
  <meta name="citation_author" content="Ashwin N. Skelly" />
  <meta name="citation_author_institution" content="Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America" />
  <meta name="citation_author_institution" content="Institute for Immunology, University of Pennsylvania Perelman School of Medicine, Philadelphia, United States of America" />
  <meta name="citation_author_orcid" content="0000-0002-1565-3376" />
  <meta name="citation_author" content="Marouen Ben Guebila" />
  <meta name="citation_author_institution" content="Department of Biostatistics, Harvard School of Public Health, Boston, Massachusetts, United States of America" />
  <meta name="citation_author_orcid" content="0000-0001-5934-966X" />
  <meta name="twitter:creator" content="@marouenbg" />
  <meta name="citation_author" content="Likhitha Kolla" />
  <meta name="citation_author_institution" content="Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America" />
  <meta name="citation_author_orcid" content="0000-0002-1169-906X" />
  <meta name="twitter:creator" content="@lkolla2018" />
  <meta name="citation_author" content="David Manheim" />
  <meta name="citation_author_institution" content="1DaySooner, Delaware, United States of America" />
  <meta name="citation_author_institution" content="Risk and Health Communication Research Center, School of Public Health, University of Haifa, Haifa, Israel" />
  <meta name="citation_author_orcid" content="0000-0001-8599-8380" />
  <meta name="twitter:creator" content="@davidmanheim" />
  <meta name="citation_author" content="Soumita Ghosh" />
  <meta name="citation_author_institution" content="Institute of Translational Medicine and Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America" />
  <meta name="citation_author_orcid" content="0000-0002-2783-2750" />
  <meta name="citation_author" content="Matthias Fax" />
  <meta name="citation_author_institution" content="greensolid technologies, Bavaria, Germany" />
  <meta name="citation_author_institution" content="IEEE Reliability Society, Vienna, Austria" />
  <meta name="citation_author_orcid" content="0000-0002-0064-395X" />
  <meta name="citation_author" content="James Brian Byrd" />
  <meta name="citation_author_institution" content="University of Michigan School of Medicine, Ann Arbor, Michigan, United States of America" />
  <meta name="citation_author_orcid" content="0000-0002-0509-3520" />
  <meta name="twitter:creator" content="@thebyrdlab" />
  <meta name="citation_author" content="YoSon Park" />
  <meta name="citation_author_institution" content="Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America" />
  <meta name="citation_author_orcid" content="0000-0002-0465-4744" />
  <meta name="twitter:creator" content="@__yoson__" />
  <meta name="citation_author" content="Vikas Bansal" />
  <meta name="citation_author_institution" content="Biomedical Data Science and Machine Learning Group, DZNE, Tübingen, Germany" />
  <meta name="citation_author_orcid" content="0000-0002-0944-7226" />
  <meta name="twitter:creator" content="@VikasBansal1989" />
  <meta name="citation_author" content="Stephen Capone" />
  <meta name="citation_author_institution" content="St. George&#39;s University School of Medicine, St. George&#39;s, Grenada" />
  <meta name="citation_author_orcid" content="0000-0001-7231-1535" />
  <meta name="citation_author" content="John J. Dziak" />
  <meta name="citation_author_institution" content="Edna Bennett Pierce Prevention Research Center, The Pennsylvania State University, University Park, PA, United States of America" />
  <meta name="citation_author_orcid" content="0000-0003-0762-5495" />
  <meta name="citation_author" content="Yuchen Sun" />
  <meta name="citation_author_institution" content="Department of Computer Science, University of Virginia, Charlottesville, VA, United States of America" />
  <meta name="citation_author" content="Yanjun Qi" />
  <meta name="citation_author_institution" content="Department of Computer Science, University of Virginia, Charlottesville, VA, United States of America" />
  <meta name="citation_author_orcid" content="0000-0002-5796-7453" />
  <meta name="citation_author" content="Lamonica Shinholster" />
  <meta name="citation_author_institution" content="Mercer University, Macon, GA, United States of America" />
  <meta name="citation_author_orcid" content="0000-0001-6285-005X" />
  <meta name="citation_author" content="Sergey Knyazev" />
  <meta name="citation_author_institution" content="Georgia State University, GA, United States of America" />
  <meta name="citation_author_orcid" content="0000-0003-0385-1831" />
  <meta name="twitter:creator" content="@SeKnyaz" />
  <meta name="citation_author" content="Dimitri Perrin" />
  <meta name="citation_author_institution" content="School of Computer Science, Queensland University of Technology, Brisbane, Australia" />
  <meta name="citation_author_institution" content="Centre for Data Science, Queensland University of Technology, Brisbane, Australia" />
  <meta name="citation_author_orcid" content="0000-0002-4007-5256" />
  <meta name="twitter:creator" content="@dperrin" />
  <meta name="citation_author" content="Serghei Mangul" />
  <meta name="citation_author_institution" content="Department of Clinical Pharmacy, School of Pharmacy, University of Southern California, CA, United States of America" />
  <meta name="citation_author_orcid" content="0000-0003-4770-3443" />
  <meta name="twitter:creator" content="@serghei_mangul" />
  <meta name="citation_author" content="Shikta Das" />
  <meta name="citation_author_institution" content="C4X Discovery, London, United Kingdom" />
  <meta name="citation_author_institution" content="Medical Research Council LHA, Institute of Cardiovascular Studies, University College London, London, United Kingdom" />
  <meta name="citation_author_orcid" content="0000-0002-8291-2788" />
  <meta name="twitter:creator" content="@shikta_das" />
  <meta name="citation_author" content="Gregory L Szeto" />
  <meta name="citation_author_institution" content="Allen Institute for Immunology, Seattle, WA, United States of America" />
  <meta name="citation_author_orcid" content="0000-0001-7604-1333" />
  <meta name="twitter:creator" content="@greg_szeto" />
  <meta name="citation_author" content="Tiago Lubiana" />
  <meta name="citation_author_institution" content="Department of Clinical and Toxicological Analyses, School of Pharmaceutical Sciences, University of São Paulo, São Paulo, Brazil" />
  <meta name="citation_author_orcid" content="0000-0003-2473-2313" />
  <meta name="twitter:creator" content="@lubianat" />
  <link rel="canonical" href="https://greenelab.github.io/covid19-review/" />
  <meta property="og:url" content="https://greenelab.github.io/covid19-review/" />
  <meta property="twitter:url" content="https://greenelab.github.io/covid19-review/" />
  <meta name="citation_fulltext_html_url" content="https://greenelab.github.io/covid19-review/" />
  <meta name="citation_pdf_url" content="https://greenelab.github.io/covid19-review/manuscript.pdf" />
  <link rel="alternate" type="application/pdf" href="https://greenelab.github.io/covid19-review/manuscript.pdf" />
  <link rel="alternate" type="text/html" href="https://greenelab.github.io/covid19-review/v/9712aaa67136c23ebc2d7adfeb17c66881e2e9e2/" />
  <meta name="manubot_html_url_versioned" content="https://greenelab.github.io/covid19-review/v/9712aaa67136c23ebc2d7adfeb17c66881e2e9e2/" />
  <meta name="manubot_pdf_url_versioned" content="https://greenelab.github.io/covid19-review/v/9712aaa67136c23ebc2d7adfeb17c66881e2e9e2/manuscript.pdf" />
  <meta property="og:type" content="article" />
  <meta property="twitter:card" content="summary_large_image" />
  <meta property="og:image" content="https://github.com/greenelab/covid19-review/raw/9712aaa67136c23ebc2d7adfeb17c66881e2e9e2/content/images/thumbnail.png" />
  <meta property="twitter:image" content="https://github.com/greenelab/covid19-review/raw/9712aaa67136c23ebc2d7adfeb17c66881e2e9e2/content/images/thumbnail.png" />
  <link rel="icon" type="image/png" sizes="192x192" href="https://manubot.org/favicon-192x192.png" />
  <link rel="mask-icon" href="https://manubot.org/safari-pinned-tab.svg" color="#ad1457" />
  <meta name="theme-color" content="#ad1457" />
  <!-- end Manubot generated metadata -->
</head>
<body>
<header id="title-block-header">
<h1 class="title">SARS-CoV-2 and COVID-19: An Evolving Review of Diagnostics and Therapeutics</h1>
</header>
<p><small><em>
This manuscript
(<a href="https://greenelab.github.io/covid19-review/v/9712aaa67136c23ebc2d7adfeb17c66881e2e9e2/">permalink</a>)
was automatically generated
from <a href="https://github.com/greenelab/covid19-review/tree/9712aaa67136c23ebc2d7adfeb17c66881e2e9e2">greenelab/covid19-review@9712aaa</a>
on November 12, 2020.
</em></small></p>
<!-- include the Font Awesome library, per: https://fontawesome.com/start -->
<p><link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.13.0/css/all.css">
<span class="banner lightred"><span class="fas fa-info-circle fa-lg"></span> <strong>This in progress manuscript is not intended for the general public.</strong><br>
This is a review paper that is authored by scientists for an audience of scientists to discuss research that is in progress.
If you are interested in guidelines on testing, therapies, or other issues related to your health, you should not use this document.
Instead, you should collect information from your local health department, the <a href="https://www.cdc.gov/coronavirus/2019-ncov/index.html">CDC’s guidance</a>, or your own government.</span></p>
<h1 class="unnumbered" data-number="" id="authors">Authors</h1>
<ul>
<li><p><strong>Halie M. Rando</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-7688-1770">0000-0001-7688-1770</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/rando2">rando2</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/tamefoxtime">tamefoxtime</a><br>
<small>
Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America
· Funded by the Gordon and Betty Moore Foundation (GBMF 4552)
</small></p></li>
<li><p><strong>Casey S. Greene</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-8713-9213">0000-0001-8713-9213</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/cgreene">cgreene</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/GreeneScientist">GreeneScientist</a><br>
<small>
Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America; Childhood Cancer Data Lab, Alex’s Lemonade Stand Foundation, Philadelphia, Pennsylvania, United States of America
· Funded by the Gordon and Betty Moore Foundation (GBMF 4552); the National Human Genome Research Institute (R01 HG010067)
</small></p></li>
<li><p><strong>Michael P. Robson</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-4859-0033">0000-0002-4859-0033</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/mprobson">mprobson</a><br>
<small>
Department of Computing Sciences, Villanova University, Villanova, Pennsylvania, United States of America
</small></p></li>
<li><p><strong>Simina M. Boca</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-1400-3398">0000-0002-1400-3398</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/SiminaB">SiminaB</a><br>
<small>
Innovation Center for Biomedical Informatics, Georgetown University Medical Center, Washington, District of Columbia, United States of America
</small></p></li>
<li><p><strong>Nils Wellhausen</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-8955-7582">0000-0001-8955-7582</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/nilswellhausen">nilswellhausen</a><br>
<small>
Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America
</small></p></li>
<li><p><strong>Ronan Lordan</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-9668-3368">0000-0001-9668-3368</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/RLordan">RLordan</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/el_ronan">el_ronan</a><br>
<small>
Institute for Translational Medicine and Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104-5158, USA
</small></p></li>
<li><p><strong>Christian Brueffer</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-3826-0989">0000-0002-3826-0989</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/cbrueffer">cbrueffer</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/cbrueffer">cbrueffer</a><br>
<small>
Department of Clinical Sciences, Lund University, Lund, Sweden
</small></p></li>
<li><p><strong>Sandipan Ray</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-9960-5768">0000-0002-9960-5768</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/rays1987">rays1987</a><br>
<small>
Department of Systems Pharmacology &amp; Translational Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA; Institute for Translational Medicine and Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA
</small></p></li>
<li><p><strong>Lucy D'Agostino McGowan</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-7297-9359">0000-0001-7297-9359</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/LucyMcGowan">LucyMcGowan</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/LucyStats">LucyStats</a><br>
<small>
Department of Mathematics and Statistics, Wake Forest University, Winston-Salem, North Carolina, United States of America
</small></p></li>
<li><p><strong>Anthony Gitter</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-5324-9833">0000-0002-5324-9833</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/agitter">agitter</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/anthonygitter">anthonygitter</a><br>
<small>
Department of Biostatistics and Medical Informatics, University of Wisconsin-Madison, Madison, Wisconsin, United States of America; Morgridge Institute for Research, Madison, Wisconsin, United States of America
· Funded by John W. and Jeanne M. Rowe Center for Research in Virology
</small></p></li>
<li><p><strong>Anna Ada Dattoli</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-1462-831X">0000-0003-1462-831X</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/aadattoli">aadattoli</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/aadattoli">aadattoli</a><br>
<small>
Department of Systems Pharmacology &amp; Translational Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA
</small></p></li>
<li><p><strong>Ryan Velazquez</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-3655-3403">0000-0002-3655-3403</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/rdvelazquez">rdvelazquez</a><br>
<small>
Azimuth1, McLean, VA
</small></p></li>
<li><p><strong>John P. Barton</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-1467-421X">0000-0003-1467-421X</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/johnbarton">johnbarton</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/_jpbarton">_jpbarton</a><br>
<small>
Department of Physics and Astronomy, University of California-Riverside, Riverside, California, United States of America
</small></p></li>
<li><p><strong>Jeffrey M. Field</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-7161-7284">0000-0001-7161-7284</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/Jeff-Field">Jeff-Field</a><br>
<small>
Department of Pharmacology, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA
</small></p></li>
<li><p><strong>Bharath Ramsundar</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-8450-4262">0000-0001-8450-4262</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/rbharath">rbharath</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/rbhar90">rbhar90</a><br>
<small>
The DeepChem Project, https://deepchem.io/
</small></p></li>
<li><p><strong>Adam L. MacLean</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-0689-7907">0000-0003-0689-7907</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/alavendelm">alavendelm</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/adamlmaclean">adamlmaclean</a><br>
<small>
Department of Biological Sciences, University of Southern California, Los Angeles, California, United States of America
</small></p></li>
<li><p><strong>Alexandra J. Lee</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-0208-3730">0000-0002-0208-3730</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/ajlee21">ajlee21</a><br>
<small>
Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America
· Funded by the Gordon and Betty Moore Foundation (GBMF 4552)
</small></p></li>
<li><p><strong>Immunology Institute of the Icahn School of Medicine</strong><br>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/ismms-himc">ismms-himc</a><br>
<small>
Immunology Institute of the Icahn School of Medicine
</small></p></li>
<li><p><strong>Fengling Hu</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-1081-5038">0000-0003-1081-5038</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/hufengling">hufengling</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/hufengling">hufengling</a><br>
<small>
Department of Biostatistics, Epidemiology and Informatics, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America
</small></p></li>
<li><p><strong>Nafisa M. Jadavji</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-3557-7307">0000-0002-3557-7307</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/nafisajadavji">nafisajadavji</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/nafisajadavji">nafisajadavji</a><br>
<small>
Biomedical Science, Midwestern University, Glendale, AZ, United States of America; Department of Neuroscience, Carleton University, Ottawa, Ontario, Canada
· Funded by the American Heart Association (20AIREA35050015)
</small></p></li>
<li><p><strong>Elizabeth Sell</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-9658-1107">0000-0002-9658-1107</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/esell17">esell17</a><br>
<small>
Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America
</small></p></li>
<li><p><strong>Jinhui Wang</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-5796-8130">0000-0002-5796-8130</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/jinhui2">jinhui2</a><br>
<small>
Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America
</small></p></li>
<li><p><strong>Diane N. Rafizadeh</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-2838-067X">0000-0002-2838-067X</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/dianerafi">dianerafi</a><br>
<small>
Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America; Department of Chemistry, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America
· Funded by NIH Medical Scientist Training Program T32 GM07170
</small></p></li>
<li><p><strong>Ashwin N. Skelly</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-1565-3376">0000-0002-1565-3376</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/anskelly">anskelly</a><br>
<small>
Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America; Institute for Immunology, University of Pennsylvania Perelman School of Medicine, Philadelphia, United States of America
· Funded by NIH Medical Scientist Training Program T32 GM07170
</small></p></li>
<li><p><strong>Marouen Ben Guebila</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-5934-966X">0000-0001-5934-966X</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/marouenbg">marouenbg</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/marouenbg">marouenbg</a><br>
<small>
Department of Biostatistics, Harvard School of Public Health, Boston, Massachusetts, United States of America
</small></p></li>
<li><p><strong>Likhitha Kolla</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-1169-906X">0000-0002-1169-906X</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/likhithakolla">likhithakolla</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/lkolla2018">lkolla2018</a><br>
<small>
Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America
· Funded by NIH Medical Scientist Training Program T32 GM07170
</small></p></li>
<li><p><strong>David Manheim</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-8599-8380">0000-0001-8599-8380</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/davidmanheim">davidmanheim</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/davidmanheim">davidmanheim</a><br>
<small>
1DaySooner, Delaware, United States of America; Risk and Health Communication Research Center, School of Public Health, University of Haifa, Haifa, Israel
</small></p></li>
<li><p><strong>Soumita Ghosh</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-2783-2750">0000-0002-2783-2750</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/soumitagh">soumitagh</a><br>
<small>
Institute of Translational Medicine and Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America
</small></p></li>
<li><p><strong>Matthias Fax</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-0064-395X">0000-0002-0064-395X</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/matfax">matfax</a><br>
<small>
greensolid technologies, Bavaria, Germany; IEEE Reliability Society, Vienna, Austria
</small></p></li>
<li><p><strong>James Brian Byrd</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-0509-3520">0000-0002-0509-3520</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/byrdjb">byrdjb</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/thebyrdlab">thebyrdlab</a><br>
<small>
University of Michigan School of Medicine, Ann Arbor, Michigan, United States of America
· Funded by NIH K23HL128909; FastGrants
</small></p></li>
<li><p><strong>YoSon Park</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-0465-4744">0000-0002-0465-4744</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/ypar">ypar</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/__yoson__"><strong>yoson</strong></a><br>
<small>
Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America
· Funded by NHGRI R01 HG10067
</small></p></li>
<li><p><strong>Vikas Bansal</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-0944-7226">0000-0002-0944-7226</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/bansalvi">bansalvi</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/VikasBansal1989">VikasBansal1989</a><br>
<small>
Biomedical Data Science and Machine Learning Group, DZNE, Tübingen, Germany
</small></p></li>
<li><p><strong>Stephen Capone</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-7231-1535">0000-0001-7231-1535</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/scapone01">scapone01</a><br>
<small>
St. George’s University School of Medicine, St. George’s, Grenada
</small></p></li>
<li><p><strong>John J. Dziak</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-0762-5495">0000-0003-0762-5495</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/dziakj1">dziakj1</a><br>
<small>
Edna Bennett Pierce Prevention Research Center, The Pennsylvania State University, University Park, PA, United States of America
</small></p></li>
<li><p><strong>Yuchen Sun</strong><br>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/kevinsunofficial">kevinsunofficial</a><br>
<small>
Department of Computer Science, University of Virginia, Charlottesville, VA, United States of America
</small></p></li>
<li><p><strong>Yanjun Qi</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-5796-7453">0000-0002-5796-7453</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/qiyanjun">qiyanjun</a><br>
<small>
Department of Computer Science, University of Virginia, Charlottesville, VA, United States of America
</small></p></li>
<li><p><strong>Lamonica Shinholster</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-6285-005X">0000-0001-6285-005X</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/LSH2126">LSH2126</a><br>
<small>
Mercer University, Macon, GA, United States of America
· Funded by the Center for Global Genomics and Health Equity at the University of Pennsylvania
</small></p></li>
<li><p><strong>Sergey Knyazev</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-0385-1831">0000-0003-0385-1831</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/Sergey-Knyazev">Sergey-Knyazev</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/SeKnyaz">SeKnyaz</a><br>
<small>
Georgia State University, GA, United States of America
</small></p></li>
<li><p><strong>Dimitri Perrin</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-4007-5256">0000-0002-4007-5256</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/SystemsResearch">SystemsResearch</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/dperrin">dperrin</a><br>
<small>
School of Computer Science, Queensland University of Technology, Brisbane, Australia; Centre for Data Science, Queensland University of Technology, Brisbane, Australia
</small></p></li>
<li><p><strong>Serghei Mangul</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-4770-3443">0000-0003-4770-3443</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/smangul1">smangul1</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/serghei_mangul">serghei_mangul</a><br>
<small>
Department of Clinical Pharmacy, School of Pharmacy, University of Southern California, CA, United States of America
</small></p></li>
<li><p><strong>Shikta Das</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-8291-2788">0000-0002-8291-2788</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/shiktadas">shiktadas</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/shikta_das">shikta_das</a><br>
<small>
C4X Discovery, London, United Kingdom; Medical Research Council LHA, Institute of Cardiovascular Studies, University College London, London, United Kingdom
</small></p></li>
<li><p><strong>Gregory L Szeto</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-7604-1333">0000-0001-7604-1333</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/gregszetoAI">gregszetoAI</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/greg_szeto">greg_szeto</a><br>
<small>
Allen Institute for Immunology, Seattle, WA, United States of America
</small></p></li>
<li><p><strong>Tiago Lubiana</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-2473-2313">0000-0003-2473-2313</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/lubianat">lubianat</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/lubianat">lubianat</a><br>
<small>
Department of Clinical and Toxicological Analyses, School of Pharmaceutical Sciences, University of São Paulo, São Paulo, Brazil
</small></p></li>
</ul>
<h1 class="page_break_before unnumbered" data-number="" id="abstract">Abstract</h1>
<p>Since late 2019, Coronavirus disease 2019 (COVID-19) has spread around the world, resulting in the declaration of a pandemic by the World Health Organization (WHO).
This infectious disease is caused by the newly identified severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2).
Research on the SARS-CoV-2 virus and the disease it causes is emerging rapidly through global scientific efforts.
Short-term mitigation of viral impacts will require public health interventions, and long-term mitigation will require new diagnostic and therapeutic technologies.
The urgency of the pandemic has led to a rapidly emerging scientific literature on SARS-CoV-2 and COVID-19.
This manuscript represents a collaborative effort to organize and consolidate this body of literature.
We present information about the virus in the context of what is known about related viruses, describe the pathogenesis of COVID-19, and synthesize studies emerging about the diagnosis and treatment of COVID-19 alongside literature about related illnesses.
We summarize this emerging literature with an eye towards discussing elements of the disease that will be fundamental to efforts to develop interventions.
Our review is a collaboratively-authored, evolving document into which we seek to incorporate the ever-expanding body of information on the topic.
This document provides a snapshot as of October, 2020.
We continue to accept new contributions and anticipate future snapshots until technologies to mitigate the pandemic are widely deployed.</p>
<h1 class="unnumbered" data-number="" id="how-to-contribute">How to Contribute</h1>
<p>We invite potential contributors to introduce themselves through GitHub: <a href="https://github.com/greenelab/covid19-review/issues/17">https://github.com/greenelab/covid19-review/issues/17</a></p>
<p>We have established a community chat room on a service called Gitter: <a href="https://gitter.im/covid19-review/community">https://gitter.im/covid19-review/community</a></p>
<p>More information about how to contribute is available in a README document on GitHub: <a href="https://github.com/greenelab/covid19-review#sars-cov-2-and-covid-19-an-evolving-review-of-diagnostics-and-therapeutics">https://github.com/greenelab/covid19-review#sars-cov-2-and-covid-19-an-evolving-review-of-diagnostics-and-therapeutics</a></p>
<h1 class="page_break_before" data-number="1" id="introduction"><span class="header-section-number">1</span> Introduction</h1>
<p>On January 21, 2020, the World Health Organization (WHO) released its first report concerning what is now known as the Coronavirus Disease 2019 (COVID-19) <span class="citation" data-cites="Vb5KFWnO">[<a href="#ref-Vb5KFWnO" role="doc-biblioref">1</a>]</span>.
This infectious disease came to international attention on December 31, 2019 following an announcement by national officials in China describing 44 cases of a respiratory infection of unknown cause.
The first known cases were located in Wuhan City within the Hubei province of China, but the disease spread rapidly throughout China and subsequently around the world.
At the time of the WHO’s first situation report <span class="citation" data-cites="Vb5KFWnO">[<a href="#ref-Vb5KFWnO" role="doc-biblioref">1</a>]</span>, 282 confirmed cases had been identified.
Most of these cases were in China, but one to two exported cases had also been identified in each of several neighboring countries (Thailand, Japan, and the Republic of Korea).
One week later, 4,593 confirmed cases had been identified, spanning not only Asia, but also Australia, North America, and Europe <span class="citation" data-cites="fLTiwToY">[<a href="#ref-fLTiwToY" role="doc-biblioref">2</a>]</span>.
On March 11, 2020, the WHO formally classified the situation as a pandemic <span class="citation" data-cites="x9975iBK">[<a href="#ref-x9975iBK" role="doc-biblioref">3</a>]</span>.
On April 4, 2020, the WHO reported that the global number of confirmed cases had surpassed one million <span class="citation" data-cites="1GACP7OZY">[<a href="#ref-1GACP7OZY" role="doc-biblioref">4</a>]</span>.
1,272,094 COVID-19 deaths had been reported worldwide as of November 10, 2020 (Figure <a href="#fig:csse-deaths">1</a>).</p>
<div id="fig:csse-deaths" class="fignos">
<figure>
<img src="https://github.com/greenelab/covid19-review/raw/65b2be6684a85cb83d462cdabcedda1d5393b8b7/csse/csse-deaths.svg" alt="" /><figcaption><span>Figure 1:</span> <strong>Cumulative global COVID-19 deaths since January 22, 2020.</strong>
Data are from the COVID-19 Data Repository by the Center for Systems Science and Engineering at Johns Hopkins University <span class="citation" data-cites="MrwDDw9R">[<a href="#ref-MrwDDw9R" role="doc-biblioref">5</a>]</span>.</figcaption>
</figure>
</div>
<p>As international attention remains focused on the ongoing public health crisis, the scientific community has responded by mobilizing resources and turning much of its attention to the virus and disease.
This rapid influx of information is disseminated by traditional publishing mechanisms, preprint servers, and press releases, which provide a venue for scientists to release findings without undergoing the formal publication process.
While having information available is valuable to efforts to understand and combat COVID-19, many contributions come from researchers across a wide range of fields who have varying degrees of experience working on coronaviruses and related topics.
The volume of information available, much of which has not gone through rigorous peer review, presents a significant challenge to individual efforts to keep abreast of the state of COVID-19 research <span class="citation" data-cites="1HZeeO4Cs">[<a href="#ref-1HZeeO4Cs" role="doc-biblioref">6</a>]</span>.
However, research on these topics is proceeding so rapidly that any static review is likely to quickly become dated.
Our goal as a community is to consolidate information about the virus in the context of related viruses and to synthesize rapidly emerging literature centered on the diagnosis and treatment of COVID-19.
We used an open publishing framework, Manubot <span class="citation" data-cites="YuJbg3zO">[<a href="#ref-YuJbg3zO" role="doc-biblioref">7</a>]</span>, to manage hundreds of contributions from the community to create a living, scholarly document.
We designed software to generate figures, such as Figure <a href="#fig:csse-deaths">1</a>, that automatically update using external data sources.
Our primary goal is to sort and distill informative content out of the overwhelming flood of information <span class="citation" data-cites="1HZeeO4Cs">[<a href="#ref-1HZeeO4Cs" role="doc-biblioref">6</a>]</span> and help the broader scientific community become more conversant on this critical subject.
Thus, our approach has been to develop a real-time, collaborative effort that welcomes submissions from scientists worldwide into this ongoing effort.
This document represents the first snapshot, which aims to reflect the state of the field as of October, 2020.
We plan to refine and expand this document until technologies to mitigate the pandemic are widely available.</p>
<h2 data-number="1.1" id="interdisciplinary-context"><span class="header-section-number">1.1</span> Interdisciplinary Context</h2>
<p>Collaboration across several broad areas of research is critical, as different areas provide different information and context necessary to understanding the virus and disease.
This review provides a biological perspective on the SARS-CoV-2 virus and efforts to develop diagnostic, prophylactic, and therapeutic responses to COVID-19.
We provide only brief summaries of two other important perspectives on this pandemic: epidemiology and public health.
Research in these areas often seeks to anticipate, model, and prevent outbreaks of infectious disease or to understand and manage human behavior relevant to health and disease.
Their insights are critical to mounting a global response to the pandemic.
Epidemiological analyses have investigated patterns of transmission within and between communities, the symptoms associated with and the duration of infection and/or contagiousness, and how the virus propagates, among other characteristics <span class="citation" data-cites="xdRgnrxv">[<a href="#ref-xdRgnrxv" role="doc-biblioref">8</a>]</span>.
Epidemiology also has a close relationship to public policy because it provides model-based insights into how preventative measures and public response can shift outcomes <span class="citation" data-cites="t1Gs6K2A">[<a href="#ref-t1Gs6K2A" role="doc-biblioref">9</a>]</span>.
Public health addresses social and human factors influencing individuals’ exposure and susceptibility to pathogens, such as resource availability, inequality, human behavior, and access to accurate information.
Strategies from public health and epidemiology for managing the current epidemic have included the promotion of hand hygiene, social distancing, and personal protective equipment such as masks to mitigate spread, as well as containment approaches such as test, trace, and isolate, which depends on widespread testing, contact tracing, and quarantining.
An effective public health management strategy involves response coordination, disease surveillance, intervention monitoring, risk communication, and health education (including the containment of “infodemics” of false information) <span class="citation" data-cites="13CJl7iIO">[<a href="#ref-13CJl7iIO" role="doc-biblioref">10</a>]</span>.
Epidemiology and public health intersect with the topics addressed in this manuscript because they both inform and benefit from relevant biotechnological developments.
For example, the development of accurate and fast diagnostic testing is relevant to test, trace, and isolate strategies for containment, and public education will be critical to deploying vaccines once they become available.
The present analysis focuses less on human and social factors and more on the basic biology of infection, diagnosis, and recovery, but these areas are inextricable in understanding and responding to the COVID-19 pandemic.</p>
<h2 data-number="1.2" id="initial-characterization-of-sars-cov-2"><span class="header-section-number">1.2</span> Initial Characterization of SARS-CoV-2</h2>
<p>The first genome sequence of the virus was released on January 3, 2020 and revealed that the cluster of pneumonia cases seen in Wuhan were caused by a novel coronavirus <span class="citation" data-cites="Bp847Lfa">[<a href="#ref-Bp847Lfa" role="doc-biblioref">11</a>]</span>.
Multiple research groups have drafted the genome sequence of SARS-CoV-2 based on sequences developed from clinical samples collected from the lower respiratory tract, namely bronchoalveolar lavage fluid (BALF), and the upper respiratory tract, in the form of throat swabs <span class="citation" data-cites="vHq0hOWQ VSkK7CeP 1Fqilxaum">[<a href="#ref-vHq0hOWQ" role="doc-biblioref">12</a>,<a href="#ref-VSkK7CeP" role="doc-biblioref">13</a>,<a href="#ref-1Fqilxaum" role="doc-biblioref">14</a>]</span>.
Analysis of the SARS-CoV-2 genome revealed significant sequence homology with two coronaviruses known to infect humans, with about 79% identity to SARS-CoV and 50% to MERS-CoV <span class="citation" data-cites="1Fqilxaum">[<a href="#ref-1Fqilxaum" role="doc-biblioref">14</a>]</span>.
However, the highest degree of similarity was observed between SARS-CoV-2 and bat-derived SARS-like coronaviruses (bat-SL-CoVZC45 and bat-SL-CoVZXC21) <span class="citation" data-cites="1Fqilxaum VSkK7CeP">[<a href="#ref-VSkK7CeP" role="doc-biblioref">13</a>,<a href="#ref-1Fqilxaum" role="doc-biblioref">14</a>]</span>, with identity between SARS-CoV-2 and RATG13 as high as 96.2% <span class="citation" data-cites="VSkK7CeP UUAeVUaR">[<a href="#ref-VSkK7CeP" role="doc-biblioref">13</a>,<a href="#ref-UUAeVUaR" role="doc-biblioref">15</a>]</span>.
This evidence therefore suggests the SARS-CoV-2 virus is the result of zoonotic transfer of a virus from bats to humans.
Nevertheless, some fragments between SARS-CoV-2 and RATG13 differ by up to 17%, suggesting a complex natural selection process during zoonotic transfer.
While the <em>S</em> region is highly similar to that of viruses found in pangolins <span class="citation" data-cites="eQIU1Nbs">[<a href="#ref-eQIU1Nbs" role="doc-biblioref">16</a>]</span>, there is no consensus about the origin of <em>S</em> in SARS-CoV-2, as it could potentially be the result either of recombination or coevolution <span class="citation" data-cites="UUAeVUaR wz7kPyn2">[<a href="#ref-UUAeVUaR" role="doc-biblioref">15</a>,<a href="#ref-wz7kPyn2" role="doc-biblioref">17</a>]</span>.
Though the intermediate host serving as the source for the zoonotic introduction of SARS-CoV-2 to human populations has not yet been identified, the SARS-CoV-2 virus has been placed within the coronavirus phylogeny through comparative genomic analyses.
Genomic analyses and comparisons to other known coronaviruses suggest that SARS-CoV-2 is unlikely to have originated in a laboratory – either purposely engineered and released, or escaped – and instead evolved naturally in an animal host <span class="citation" data-cites="NEIoHDjq">[<a href="#ref-NEIoHDjq" role="doc-biblioref">18</a>]</span>.
While the position of the SARS-CoV-2 virus within the coronavirus phylogeny has been largely resolved, the functional consequences of molecular variation between this virus and other viruses, such as its bat and pangolin sister taxa or SARS-CoV, remain unknown <span class="citation" data-cites="NEIoHDjq">[<a href="#ref-NEIoHDjq" role="doc-biblioref">18</a>]</span>.
Fortunately, the basic genome structure of coronaviruses is highly conserved, and insight into the mechanisms the virus uses to enter cells, replicate, and spread is likely to be available from prior research in coronaviruses.</p>
<h2 data-number="1.3" id="coronaviruses-and-humans-hosts"><span class="header-section-number">1.3</span> Coronaviruses and Humans Hosts</h2>
<p>Coronaviruses have long been known to infect animals and have been the subject of veterinary medical investigations and vaccine development efforts due to their effect on the health of companion and agricultural animals <span class="citation" data-cites="10oGRZVIE">[<a href="#ref-10oGRZVIE" role="doc-biblioref">19</a>]</span>.
Most coronaviruses show little to no transmission in humans.
However, today it is thought that approximately one-third of common cold infections are caused by four HCoV (HCoV-229E, HCoV-NL63, HCoV-OC43 and HCoV-HKU1) <span class="citation" data-cites="UBTUyMGV YAhVzI0A">[<a href="#ref-UBTUyMGV" role="doc-biblioref">20</a>,<a href="#ref-YAhVzI0A" role="doc-biblioref">21</a>]</span>.
The first coronavirus species infecting humans (human coronavirus or HCoV) were identified in the 1960s: HCoV-229E in 1965 <span class="citation" data-cites="gaAv88da">[<a href="#ref-gaAv88da" role="doc-biblioref">22</a>]</span> and HCoV-OC43 in 1967 <span class="citation" data-cites="q6rFk3Kl">[<a href="#ref-q6rFk3Kl" role="doc-biblioref">23</a>]</span>.
Both of these viruses cause cold-like symptoms <span class="citation" data-cites="cEFaGSlu Jnb5pce4">[<a href="#ref-cEFaGSlu" role="doc-biblioref">24</a>,<a href="#ref-Jnb5pce4" role="doc-biblioref">25</a>]</span>.
Two additional HCoV were subsequently identified <span class="citation" data-cites="URhDY7s9 VgGISeiE">[<a href="#ref-URhDY7s9" role="doc-biblioref">26</a>,<a href="#ref-VgGISeiE" role="doc-biblioref">27</a>]</span>.
In 2003, HCoV-NL63 <span class="citation" data-cites="URhDY7s9">[<a href="#ref-URhDY7s9" role="doc-biblioref">26</a>]</span> was first identified in a 7-month-old infant and then in clinical specimens collected from seven additional patients, five of whom were infants younger than 1 year old and the remainder of whom were adults.
CoV-HKU1 was identified in samples collected from a 71-year-old pneumonia patient in 2004 and then found in samples collected from a second adult patient <span class="citation" data-cites="VgGISeiE">[<a href="#ref-VgGISeiE" role="doc-biblioref">27</a>]</span>.
These viruses are associated with respiratory diseases of varying severity, ranging from common cold to severe pneumonia, with severe symptoms mostly observed in immunocompromised individuals <span class="citation" data-cites="rDtyU6cv">[<a href="#ref-rDtyU6cv" role="doc-biblioref">28</a>]</span>.
In addition to these relatively mild HCoV, however, highly pathogenic human coronaviruses have been identified, including the severe acute respiratory syndrome coronavirus 1 (SARS-CoV or SARS-CoV-1) and the Middle East respiratory syndrome coronavirus (MERS-CoV) <span class="citation" data-cites="G5NJrE75 UBTUyMGV uXEsLlzb">[<a href="#ref-UBTUyMGV" role="doc-biblioref">20</a>,<a href="#ref-G5NJrE75" role="doc-biblioref">29</a>,<a href="#ref-uXEsLlzb" role="doc-biblioref">30</a>]</span>.</p>
<p>At the time that SARS-CoV emerged in the early 2000s, no HCoV had been identified in almost 40 years <span class="citation" data-cites="G5NJrE75">[<a href="#ref-G5NJrE75" role="doc-biblioref">29</a>]</span>.
The first case of SARS was reported in November 2002 in the Guangdong Province of China, and over the following month, the disease spread more widely within China and then into several countries across multiple continents <span class="citation" data-cites="sP4wQEiM G5NJrE75">[<a href="#ref-G5NJrE75" role="doc-biblioref">29</a>,<a href="#ref-sP4wQEiM" role="doc-biblioref">31</a>]</span>.
Unlike previously identified HCoV, SARS was much more severe, with an estimated death rate of 9.5% <span class="citation" data-cites="sP4wQEiM">[<a href="#ref-sP4wQEiM" role="doc-biblioref">31</a>]</span>.
It was also highly contagious via droplet transmission, with a basic reproduction number (R<sub>0</sub>) of 4 (i.e., each person infected was estimated to infect four other people) <span class="citation" data-cites="sP4wQEiM">[<a href="#ref-sP4wQEiM" role="doc-biblioref">31</a>]</span>.
However, the identity of the virus behind the infection remained unknown until April of 2003, when the SARS-CoV virus was identified through a worldwide scientific effort spearheaded by the WHO <span class="citation" data-cites="G5NJrE75">[<a href="#ref-G5NJrE75" role="doc-biblioref">29</a>]</span>.
SARS-CoV belonged to a distinct lineage from the two other HCoV known at the time <span class="citation" data-cites="sP4wQEiM">[<a href="#ref-sP4wQEiM" role="doc-biblioref">31</a>]</span>.
By July 2003, the SARS outbreak was officially determined to be under control, with the success credited to infection management practices <span class="citation" data-cites="G5NJrE75">[<a href="#ref-G5NJrE75" role="doc-biblioref">29</a>]</span>.
A decade later, a second outbreak of severe respiratory illness associated with a coronavirus emerged, this time in the Arabian Peninsula.
This disease, known as Middle East respiratory syndrome (MERS), was linked to another novel coronavirus, MERS-CoV.
The fatality rate associated with MERS is much higher than that of SARS, at almost 35%, but the disease is much less easily transmitted, with an R<sub>0</sub> of 1 <span class="citation" data-cites="sP4wQEiM">[<a href="#ref-sP4wQEiM" role="doc-biblioref">31</a>]</span>.
Although MERS is still circulating, its low reproduction number has allowed for its spread to be contained <span class="citation" data-cites="sP4wQEiM">[<a href="#ref-sP4wQEiM" role="doc-biblioref">31</a>]</span>.
The COVID-19 pandemic is thus associated with the seventh HCoV to be identified and the fifth since the turn of the millennium, though additional HCoVs may be in circulation but remain undetected.</p>
<p>SARS-CoV and MERS-CoV were ultimately managed largely through infection management practices (e.g., mask wearing) and properties of the virus itself (i.e., low rate of transmission), respectively <span class="citation" data-cites="G5NJrE75 sP4wQEiM">[<a href="#ref-G5NJrE75" role="doc-biblioref">29</a>,<a href="#ref-sP4wQEiM" role="doc-biblioref">31</a>]</span>.
Vaccines were not used to control either virus, although vaccine development programs were established for SARS-CoV <span class="citation" data-cites="NHVzYmXx">[<a href="#ref-NHVzYmXx" role="doc-biblioref">32</a>]</span>.
In general, care for SARS and MERS patients focuses on supportive care and symptom management <span class="citation" data-cites="sP4wQEiM">[<a href="#ref-sP4wQEiM" role="doc-biblioref">31</a>]</span>.
Clinical treatments for SARS and MERS developed during the outbreaks generally do not have strong evidence supporting their use.
Common treatments included Ribavirin, an antiviral, often in combination with corticosteroids or sometimes interferon (IFN) medications, which would both be expected to have immunomodulatory effects <span class="citation" data-cites="G5NJrE75">[<a href="#ref-G5NJrE75" role="doc-biblioref">29</a>]</span>.
However, retrospective and <em>in vitro</em> analyses have reported inconclusive results of these treatments on SARS and the SARS-CoV virus, respectively <span class="citation" data-cites="G5NJrE75">[<a href="#ref-G5NJrE75" role="doc-biblioref">29</a>]</span>.
IFNs and Ribavirin have shown promise in <em>in vitro</em> analyses of MERS, but their clinical effectiveness remains unknown <span class="citation" data-cites="G5NJrE75">[<a href="#ref-G5NJrE75" role="doc-biblioref">29</a>]</span>.
Therefore, only limited strategy for the pharmaceutical management of COVID-19 can be adopted from previous severe HCoV infections.
Research in response to prior outbreaks of HCoV-borne infections, such as SARS and MERS, have, however, provided a strong foundation for hypotheses about the pathogenesis of SARS-CoV-2 as well as potential diagnostic and therapeutic approaches.</p>
<h3 data-number="1.3.1" id="human-immune-response-to-viral-threats"><span class="header-section-number">1.3.1</span> Human Immune Response to Viral Threats</h3>
<p>Understanding the fundamental organization of the human immune response to viral threats is critical to understanding the varied response to SARS-CoV-2.
The human immune system utilizes a variety of innate and adaptive responses to protect against the pathogens it encounters.
The innate immune system consists of barriers, such as the skin, mucous secretions, neutrophils, macrophages, and dendritic cells.
It also includes cell-surface receptors that can recognize the molecular patterns of pathogens.
The adaptive immune system utilizes antigen-specific receptors that are expressed on B and T lymphocytes.
These components of the immune system typically act together; the innate response acts first, and the adaptive response begins to act several days after initial infection following the clonal expansion of T and B cells <span class="citation" data-cites="icy0PeD7">[<a href="#ref-icy0PeD7" role="doc-biblioref">33</a>]</span>.
After a virus enters into a host cell, its antigen is presented by major histocompatibility complex 1 (MHC 1) molecules and is then recognized by cytotoxic T lymphocytes.</p>
<p>In the case of COVID-19, there is also concern about the immune system becoming over-active.
One of the main immune responses contributing to the onset of acute respiratory distress syndrome (ARDS) in COVID-19 patients is the cytokine storm, which causes an extreme inflammatory response due to a release of pro-inflammatory cytokines and chemokines by immune effector cells.
In addition to respiratory distress, this mechanism can lead to organ failure and death in severe COVID-19 cases <span class="citation" data-cites="OlEbK4fc">[<a href="#ref-OlEbK4fc" role="doc-biblioref">34</a>]</span>.
Details of how the human body responds to SARS-CoV-2, both in healthy and pathological ways, and how these mechanisms can inform the identification of diagnostic, prophylactic, and therapeutic responses are explored in detail throughout this manuscript.</p>
<h3 data-number="1.3.2" id="clinical-presentation-of-covid-19"><span class="header-section-number">1.3.2</span> Clinical Presentation of COVID-19</h3>
<p>A great diversity of symptom profiles has been observed for COVID-19, although a large study from Wuhan, China suggests fever and cough as the two most common symptoms on admission <span class="citation" data-cites="15IDA5kX6">[<a href="#ref-15IDA5kX6" role="doc-biblioref">35</a>]</span>.
One early retrospective study in China described the clinical presentations of patients infected with SARS-CoV-2 as including lower respiratory tract infection with fever, dry cough, and dyspnea <span class="citation" data-cites="10THxyeCg">[<a href="#ref-10THxyeCg" role="doc-biblioref">36</a>]</span>.
This study <span class="citation" data-cites="10THxyeCg">[<a href="#ref-10THxyeCg" role="doc-biblioref">36</a>]</span> noted that upper respiratory tract symptoms were less common, which suggests that the virus targets cells located in the lower respiratory tract.
However, data from the New York City region <span class="citation" data-cites="19ytsiSpq 1H8gSsSkn">[<a href="#ref-19ytsiSpq" role="doc-biblioref">37</a>,<a href="#ref-1H8gSsSkn" role="doc-biblioref">38</a>]</span> showed variable rates of fever as a presenting symptom, suggesting that symptoms may not be consistent across samples.
These differences are present when comparing both between institutions in similar locations and between different regions experiencing COVID-19 outbreaks, leading to conflicting reports of the frequency of fever as a presenting symptom for patients upon hospital admission.
For example, even within New York City, one study <span class="citation" data-cites="19ytsiSpq">[<a href="#ref-19ytsiSpq" role="doc-biblioref">37</a>]</span> identified low oxygen saturation (&lt;90% without the use of supplemental oxygen or ventilation support) in a significant percentage of patients upon presentation, while another study <span class="citation" data-cites="1H8gSsSkn">[<a href="#ref-1H8gSsSkn" role="doc-biblioref">38</a>]</span> reported cough, fever, and dyspnea as the most common presenting symptoms.
The variability of both which symptoms present and their severity makes it difficult for public health agencies to provide clear recommendations for citizens regarding what symptoms indicate SARS-CoV-2 infection and should prompt isolation.</p>
<h2 data-number="1.4" id="role-of-the-covid-19-review"><span class="header-section-number">1.4</span> Role of the COVID-19 Review</h2>
<div id="fig:overview" class="fignos">
<figure>
<img src="images/N000-overview.png" alt="" /><figcaption><span>Figure 2:</span> <strong>Summary of the relationships among topics covered in this review.</strong></figcaption>
</figure>
</div>
<p>Several review articles on aspects of COVID-19 have already been published.
These have included reviews on the disease epidemiology <span class="citation" data-cites="I2EsJmfs">[<a href="#ref-I2EsJmfs" role="doc-biblioref">39</a>]</span>, immunological response <span class="citation" data-cites="5x25saIz">[<a href="#ref-5x25saIz" role="doc-biblioref">40</a>]</span>, diagnostics <span class="citation" data-cites="evtsR3C5">[<a href="#ref-evtsR3C5" role="doc-biblioref">41</a>]</span>, and pharmacological treatments <span class="citation" data-cites="5x25saIz 18eCxyLhx">[<a href="#ref-5x25saIz" role="doc-biblioref">40</a>,<a href="#ref-18eCxyLhx" role="doc-biblioref">42</a>]</span>.
Others <span class="citation" data-cites="SAE5ME3N xOs5ctsW">[<a href="#ref-SAE5ME3N" role="doc-biblioref">43</a>,<a href="#ref-xOs5ctsW" role="doc-biblioref">44</a>]</span> provide narrative reviews of progress on some important ongoing COVID-19 research questions.
With the worldwide scientific community uniting during 2020 to investigate SARS-CoV-2 and COVID-19 from a wide range of perspectives, findings from many disciplines are relevant on a rapid timescale to a broad scientific audience.
Additionally, many findings are published as preprints, which are available prior to going through the peer review process.
As a result, centralizing, summarizing, and critiquing new literature broadly relevant to COVID-19 can help to expedite the interdisciplinary scientific process that is currently happening at an advanced pace.
We are particularly interested in providing background to the development of diagnostic, prophylactic, and therapeutic approaches to COVID-19.
Two major concerns within diagnosis include the detection of current infections in individuals with and without symptoms, and the detection of past exposure without an active infection.
In the latter category, identifying whether individuals can develop or have developed sustained immunity is also a major consideration.
The development of high-throughput, affordable methods for detecting active infections and sustained immunity will be critical to understanding and controlling the disease.
The identification of interventions that can mitigate the effect of the virus on exposed and infected individuals is a significant research priority.
Some possible approaches include the identification of existing pharmaceuticals that reduce the severity of infection, either by reducing the virus’ virulence (e.g., antivirals) or managing the most severe symptoms of infection.
Due to the long timeline for the development of novel pharmaceuticals, in most cases, research surrounding possible pharmaceutical interventions focuses on the identification and investigation of existing compounds whose mechanisms may be relevant to COVID-19.
Other foci of current research include the identification of antibodies produced by survivors of COVID-19 and the development of vaccines.
Understanding the mechanisms describing host-virus interactions between humans and SARS-CoV-2 is thus critical to identifying candidate therapeutics.
An overview of the topics covered is visualized in Figure <a href="#fig:overview">2</a>.
Thus, in this review, we seek to consolidate information about efforts to develop strategies for diagnosis and therapeutics as new information is released by the scientific community.
We include information from both traditional peer-reviewed scientific literature and from preprints, which typically have not undergone peer review but have been critically evaluated by the scientists involved in this effort.
The goal of this manuscript is to present preliminary findings within the broader context of COVID-19 research and to identify the broad interpretations of new research, as well as limitations to interpretability.</p>
<h1 data-number="2" id="pathogenesis"><span class="header-section-number">2</span> Pathogenesis</h1>
<p>The current COVID-19 pandemic, caused by the SARS-CoV-2 virus, represents an acute global health crisis where symptoms can range from mild to severe or fatal <span class="citation" data-cites="gHWlMufv">[<a href="#ref-gHWlMufv" role="doc-biblioref">45</a>]</span>, can affect a variety of organs and systems, and includes outcomes such as acute respiratory distress and acute lung injury, among other complications.
Viral pathogenesis is typically broken down into three major components: entry, replication, and spread <span class="citation" data-cites="jsMmdH4D">[<a href="#ref-jsMmdH4D" role="doc-biblioref">46</a>]</span>.
However, in order to draw a more complete picture of pathogenesis, it is also necessary to examine how infection manifests clinically, identify systems-level interactions between the virus and the human body, and consider the possible effects of variation or evolutionary change on pathogenesis and virulence.
Though hCoV are not common and the SARS-CoV-2 virus appears to have emerged only recently, the rapid release of the genomic sequence of the virus in January 2020 provided early opportunities for comparative genomic analysis of the virus compared to its close phylogenetic relatives.
Because many mechanisms of pathogenicity are conserved among coronaviruses, this phylogenetic analysis provided an important basis for forming hypotheses about how the virus interacts with hosts.
Additionally, worldwide sequencing of viral samples has provided some preliminary insights into possible mechanisms of adaptation in the virus, and omics-based analysis of patient samples has elucidated some of the biological changes the virus induces in its human hosts.
In this way, both biomedicine and genomics are important pieces of the puzzle of SARS-CoV-2 presentation and pathogenesis.</p>
<h2 data-number="2.1" id="fundamental-viral-pathogenesis"><span class="header-section-number">2.1</span> Fundamental Viral Pathogenesis</h2>
<p>As discussed above, the virus clusters with known coronaviruses (order <em>Nidovirales</em>, family <em>Coronaviridae</em>, subfamily <em>Orthocoronavirinae</em>).
Coronaviruses are large viruses that can be identified by their distinctive “crown-like” shape.
Their spherical virions are made from lipid envelopes ranging from 100 to 160 nanometers (nm) in which peplomers of two to three spike (S) glycoproteins are anchored, creating the crown <span class="citation" data-cites="RGAUx68v DlBAegL1">[<a href="#ref-RGAUx68v" role="doc-biblioref">47</a>,<a href="#ref-DlBAegL1" role="doc-biblioref">48</a>]</span>.
These spikes, which are critical to both viral pathogenesis and the host immune response, have been visualized using cryo-electron microscopy <span class="citation" data-cites="CqQVkaqj">[<a href="#ref-CqQVkaqj" role="doc-biblioref">49</a>]</span>.
Because they induce the human immune response, they are the target of many proposed therapeutic agents.
Phylogenetic analysis of the coronaviruses reveals four major subclades, each corresponding to a genus: the alpha, beta, delta and gamma coronaviruses.
Among them, alpha and beta coronaviruses infect mammalian species, gamma coronaviruses infect avian species, and delta coronaviruses infect both mammalian and avian species <span class="citation" data-cites="17DSmRo9H">[<a href="#ref-17DSmRo9H" role="doc-biblioref">50</a>]</span>.
Phylogenetic analysis of a polymerase chain reaction (PCR) amplicon fragment from five patients along with the full genomic sequence revealed SARS-CoV-2 to be a novel betacoronavirus belonging to the B lineage, also known as sarbecovirus, which also includes the coronavirus SARS-CoV that causes SARS in humans <span class="citation" data-cites="IFLf8rsY">[<a href="#ref-IFLf8rsY" role="doc-biblioref">51</a>]</span>.
Because viral structure and mechanisms of pathogenicity are highly conserved within the order, this phylogenetic analysis provides a basis for forming hypotheses about how the virus interacts with hosts, including which tissues, organs, and systems would be most susceptible to SARS-CoV-2 infection.</p>
<h3 data-number="2.1.1" id="genomic-structure-of-coronaviruses"><span class="header-section-number">2.1.1</span> Genomic Structure of Coronaviruses</h3>
<p>Genome structure is highly conserved among coronaviruses, meaning that the relationship between the SARS-CoV-2 genome and its pathogenesis can be inferred from prior research in related viral species.
The genomes of viruses in the <em>Nidovirales</em> order share several fundamental characteristics.
They are non-segmented, which means the viral genome is contained in a single capsid, and are enveloped, which means that the genome and capsid are encased by a lipid bilayer.
Coronaviruses have large positive-sense RNA (ssRNA+) genomes ranging from 27 kilobases (Kb) to 32 Kb in length <span class="citation" data-cites="47hnjvDb 1Fqilxaum">[<a href="#ref-1Fqilxaum" role="doc-biblioref">14</a>,<a href="#ref-47hnjvDb" role="doc-biblioref">52</a>]</span>.
The SARS-CoV-2 genome lies in the middle of this range at 29,903 bp <span class="citation" data-cites="1Fqilxaum">[<a href="#ref-1Fqilxaum" role="doc-biblioref">14</a>]</span>.
Genome organization is highly conserved within the order <span class="citation" data-cites="47hnjvDb">[<a href="#ref-47hnjvDb" role="doc-biblioref">52</a>]</span>.
There are three major regions: one containing the replicase gene and one containing the genes encoding structural proteins <span class="citation" data-cites="47hnjvDb">[<a href="#ref-47hnjvDb" role="doc-biblioref">52</a>]</span>.
The replicase gene comprises about two-thirds of the genome of coronaviruses and consists of two open reading frames that are translated with ribosomal frameshifting <span class="citation" data-cites="47hnjvDb">[<a href="#ref-47hnjvDb" role="doc-biblioref">52</a>]</span>.
This polypeptide is then translated into 16 non-structural proteins (nsp1-16, except in Gammacoronaviruses, where nsp1 is absent) that form replication machinery used to synthesize viral RNA <span class="citation" data-cites="18GflyMj">[<a href="#ref-18GflyMj" role="doc-biblioref">53</a>]</span>.
The remaining third of the genome encodes structural proteins, including the spike (S), membrane (M), envelope (E), and nucleocapsid (N) proteins.
Additional accessory genes are sometimes present between these two regions, depending on the species or strain. <!-- TODO: Add a figure showing the structure of the genome.-->
Much attention has been focused on the S protein, which is a critical structure involved in cell entry.</p>
<h3 data-number="2.1.2" id="pathogenic-mechanisms-of-coronaviruses"><span class="header-section-number">2.1.2</span> Pathogenic Mechanisms of Coronaviruses</h3>
<p>While, like most viruses, it is possible that SARS-CoV and SARS-CoV-2 can enter cells through endocytosis, coronaviruses are able to target cells for entry through fusion with the plasma membrane <span class="citation" data-cites="FFwktXgx FNetQkRI">[<a href="#ref-FFwktXgx" role="doc-biblioref">54</a>,<a href="#ref-FNetQkRI" role="doc-biblioref">55</a>]</span>.
This process is conserved among coronaviruses and is closely associated with the content of their genomes.
Cell entry proceeds in three steps: binding, cleavage, and fusion.
First, the viral spike protein binds to a host cell via a recognized receptor.
Coronaviruses can bind to a range of host receptors <span class="citation" data-cites="13wWdgODZ OVsxrEuX">[<a href="#ref-13wWdgODZ" role="doc-biblioref">56</a>,<a href="#ref-OVsxrEuX" role="doc-biblioref">57</a>]</span>, with binding conserved only at the genus level <span class="citation" data-cites="17DSmRo9H">[<a href="#ref-17DSmRo9H" role="doc-biblioref">50</a>]</span>.
Viruses in the betacoronavirus genus, to which SARS-CoV-2 belongs, are known to bind to CEACAM1, Neu 5,9 Ac2, and Angiotensin-Converting Enzyme 2 (ACE2) <span class="citation" data-cites="13wWdgODZ">[<a href="#ref-13wWdgODZ" role="doc-biblioref">56</a>]</span>.
SARS-CoV-2 has a high affinity for the human ACE2 receptor, which is expressed in the vascular epithelium, other epithelial cells, and cardiovascular and renal tissues <span class="citation" data-cites="So6ceUwB NodtSApl">[<a href="#ref-So6ceUwB" role="doc-biblioref">58</a>,<a href="#ref-NodtSApl" role="doc-biblioref">59</a>]</span>.
The binding process is guided by the molecular structure of the spike protein, which is structured in three segments: an ectodomain, a transmembrane anchor, and an intracellular tail <span class="citation" data-cites="skHKTFMe">[<a href="#ref-skHKTFMe" role="doc-biblioref">60</a>]</span>.
The ectodomain forms the crown-like structures on the viral membrane and contains two subdomains known as the S1 and S2 subunits <span class="citation" data-cites="ueMPuUMe">[<a href="#ref-ueMPuUMe" role="doc-biblioref">61</a>]</span>.
The S1 (N-terminal) domain forms the head of the crown and contains the receptor binding motif (RBM), and the S2 (C-terminal) domain forms the stalk that supports the head <span class="citation" data-cites="ueMPuUMe">[<a href="#ref-ueMPuUMe" role="doc-biblioref">61</a>]</span>.
The S1 subunit guides the binding of the virus to a host cell receptor, and the S2 subunit guides the fusion process <span class="citation" data-cites="skHKTFMe">[<a href="#ref-skHKTFMe" role="doc-biblioref">60</a>]</span>.</p>
<p>After the binding of the S1 subunit to a receptor, the spike protein is often cleaved at the S1-S2 boundary by a host protease <span class="citation" data-cites="qcVbT0w4 kCVy8vjj YeKaMzdJ">[<a href="#ref-qcVbT0w4" role="doc-biblioref">62</a>,<a href="#ref-kCVy8vjj" role="doc-biblioref">63</a>,<a href="#ref-YeKaMzdJ" role="doc-biblioref">64</a>]</span>.
Similar to SARS-CoV, SARS-CoV-2 exhibits redundancy in which host proteases can cleave the S protein <span class="citation" data-cites="15l3di3Wj">[<a href="#ref-15l3di3Wj" role="doc-biblioref">65</a>]</span>.
Specifically, both transmembrane protease serine protease-2 (TMPRSS2) and cathepsins B/L have been shown to mediate SARS-CoV-2 S protein proteolytic priming, and small molecule inhibition of these enzymes fully inhibited viral entry <em>in vitro</em> <span class="citation" data-cites="15l3di3Wj JOJ2n3gC">[<a href="#ref-15l3di3Wj" role="doc-biblioref">65</a>,<a href="#ref-JOJ2n3gC" role="doc-biblioref">66</a>]</span>.
Proteolytic priming prepares the S protein for fusion <span class="citation" data-cites="kCVy8vjj YeKaMzdJ">[<a href="#ref-kCVy8vjj" role="doc-biblioref">63</a>,<a href="#ref-YeKaMzdJ" role="doc-biblioref">64</a>]</span>.
The two subunits remain bound by van der Waals forces, with the S1 subunit stabilizing the S2 subunit during the membrane fusion process <span class="citation" data-cites="qcVbT0w4">[<a href="#ref-qcVbT0w4" role="doc-biblioref">62</a>]</span>.
Electron microscopy suggests that in some coronaviruses, including SARS-CoV and MERS-CoV, a six-helix bundle separates the two subunits in the postfusion conformation, and the unusual length of this bundle facilitates membrane fusion through the release of additional energy <span class="citation" data-cites="17DSmRo9H">[<a href="#ref-17DSmRo9H" role="doc-biblioref">50</a>]</span>.
Cleavage at a second site within S2 by these same proteases activates <em>S</em> for fusion by inducing conformational changes <span class="citation" data-cites="qcVbT0w4">[<a href="#ref-qcVbT0w4" role="doc-biblioref">62</a>]</span>.
The viral membrane can then fuse with the endosomal membrane to release the viral genome into the host cytoplasm.
Once the virus enters a host cell, the replicase gene is translated and assembled into the viral replicase complex.
This complex then synthesizes the double-stranded RNA (dsRNA) genome from the genomic ssRNA(+).
The dsRNA genome is transcribed and replicated to create viral mRNAs and new ssRNA(+) genomes <span class="citation" data-cites="47hnjvDb ZUuGRxk9">[<a href="#ref-47hnjvDb" role="doc-biblioref">52</a>,<a href="#ref-ZUuGRxk9" role="doc-biblioref">67</a>]</span>.
From there, the virus can spread in other cells.
In this way, the genome of SARS-CoV-2 provides some insight into the pathogenic behavior of the virus.</p>
<h3 data-number="2.1.3" id="immune-evasion-strategies"><span class="header-section-number">2.1.3</span> Immune Evasion Strategies</h3>
<p>Research in other coronaviruses that affect humans provides some indication of how SARS-CoV-2 infection proceeds in spite of the human immune response.
By infecting the epithelium, viruses such as SARS-CoV are known to bypass the physical barriers such as skin and mucus that comprise the immune systems’ first line of defense <span class="citation" data-cites="1XbF4GLn">[<a href="#ref-1XbF4GLn" role="doc-biblioref">68</a>]</span>.
Once the virus infiltrates host cells, it is adept at evading detection.
CD163+ and CD68+ macrophage cells especially are crucial for the establishment of SARS-CoV in the body <span class="citation" data-cites="1XbF4GLn">[<a href="#ref-1XbF4GLn" role="doc-biblioref">68</a>]</span>.
These cells most likely serve as viral reservoirs that help shield SARS-CoV from the innate immune response.
According to a study on the viral dissemination of SARS-CoV in Chinese macaques, viral RNA could be detected in some monocytes throughout the process of differentiation into dendritic cells <span class="citation" data-cites="1XbF4GLn">[<a href="#ref-1XbF4GLn" role="doc-biblioref">68</a>]</span>.
This lack of active viral replication allows SARS-CoV to escape innate immunity because reduced levels of detectable viral RNA allow the virus to avoid both natural killer (NK) cells and Toll-like receptors <span class="citation" data-cites="1XbF4GLn">[<a href="#ref-1XbF4GLn" role="doc-biblioref">68</a>]</span>.
Even during replication, SARS-CoV is able to mask its dsRNA from detection by the immune system.
Although dsRNA is a pathogen-associated molecular pattern (PAMP) that would typically initiate a response from the innate immune system <span class="citation" data-cites="159GOVLV1">[<a href="#ref-159GOVLV1" role="doc-biblioref">69</a>]</span>, <em>in vitro</em> analysis of nidoviruses including SARS-CoV suggests that these viruses can induce the development of double-membrane vesicles that protect the dsRNA signature from being detected by the host immune system <span class="citation" data-cites="yQO4HFTZ">[<a href="#ref-yQO4HFTZ" role="doc-biblioref">70</a>]</span>.
This protective envelope can therefore insulate these coronaviruses from the innate immune response’s detection mechanism <span class="citation" data-cites="OlEbK4fc">[<a href="#ref-OlEbK4fc" role="doc-biblioref">34</a>]</span>.</p>
<p>HCoV are also known to interfere with the host immune response, rather than just evade it.
For example, the virulence of SARS-CoV is increased by nonstructural protein 1 (Nsp1), which can suppress host gene expression.
Nsp1 accomplishes this by stalling mRNA translation and inducing endonucleolytic cleavage and mRNA degradation <span class="citation" data-cites="Q5Q9k6Nc">[<a href="#ref-Q5Q9k6Nc" role="doc-biblioref">71</a>]</span>.
SARS-CoV also evades the immune response by interfering with type I IFN induction signaling, which is a mechanism that leads to cellular resistance to viral infections.
SARS-CoV employs methods such as ubiquitination and degradation of RNA sensor adaptor molecules MAVS and TRAF3/6 <span class="citation" data-cites="G6A0wWdn">[<a href="#ref-G6A0wWdn" role="doc-biblioref">72</a>]</span>.
Also, MERS-CoV downregulates antigen presentation via MHC class I and MHC class II, which leads to a reduction in T cell activation <span class="citation" data-cites="G6A0wWdn">[<a href="#ref-G6A0wWdn" role="doc-biblioref">72</a>]</span>.
These evasion mechanisms, in turn, can lead to systemic infection.
Coronaviruses such as SARS-CoV are also able to evade the humoral immune response through other mechanisms, such as inhibiting certain cytokine pathways or down-regulating antigen presentation by the cells <span class="citation" data-cites="yQO4HFTZ">[<a href="#ref-yQO4HFTZ" role="doc-biblioref">70</a>]</span>.</p>
<h3 data-number="2.1.4" id="host-cell-susceptibility"><span class="header-section-number">2.1.4</span> Host Cell Susceptibility</h3>
<p>ACE2 and TMPRSS2 have been identified as a primary receptor and a critical protease, respectively, facilitating the entry of SARS-CoV/CoV-2 into a target cell <span class="citation" data-cites="2QTH37Xi CqQVkaqj MXWaV7SO 15l3di3Wj JjrO6YbD">[<a href="#ref-CqQVkaqj" role="doc-biblioref">49</a>,<a href="#ref-15l3di3Wj" role="doc-biblioref">65</a>,<a href="#ref-2QTH37Xi" role="doc-biblioref">73</a>,<a href="#ref-MXWaV7SO" role="doc-biblioref">74</a>,<a href="#ref-JjrO6YbD" role="doc-biblioref">75</a>]</span>.
This finding has led to a hypothesized role for ACE2 and TMPRSS2 expression in determining which cells, tissues, and organs are most likely to be infected by SARS-CoV-2.
The ACE2 receptor is expressed in numerous organs, such as the heart, kidney, and intestine, but it is most prominently expressed in alveolar epithelial cells; this pattern of expression is expected to contribute to the virus’ association with lung pathology <span class="citation" data-cites="UwVweB2M So6ceUwB">[<a href="#ref-So6ceUwB" role="doc-biblioref">58</a>,<a href="#ref-UwVweB2M" role="doc-biblioref">76</a>]</span>.
Clinical investigations of COVID-19 patients have detected SARS-CoV-2 transcripts in BALF (93% of specimens), sputum (72%), nasal swabs (63%), fibrobronchoscopy brush biopsies (46%), pharyngeal swabs (32%), feces (29%) and blood (1%) <span class="citation" data-cites="vr7AH83b">[<a href="#ref-vr7AH83b" role="doc-biblioref">77</a>]</span>.
Two studies reported that SARS-CoV-2 could not be detected in the urine specimens <span class="citation" data-cites="vr7AH83b azgJqujy">[<a href="#ref-vr7AH83b" role="doc-biblioref">77</a>,<a href="#ref-azgJqujy" role="doc-biblioref">78</a>]</span>; however, a third study identified four urine samples (out of 58) that were positive for SARS-CoV-2 nucleic acids <span class="citation" data-cites="F7nSMvZk">[<a href="#ref-F7nSMvZk" role="doc-biblioref">79</a>]</span>.
Although respiratory failure remains the leading cause of death for COVID-19 patients <span class="citation" data-cites="1AJFJxzmJ">[<a href="#ref-1AJFJxzmJ" role="doc-biblioref">80</a>]</span>, SARS-CoV-2 infection can damage many other organ systems including the heart <span class="citation" data-cites="5gnCuPzp">[<a href="#ref-5gnCuPzp" role="doc-biblioref">81</a>]</span>, kidneys <span class="citation" data-cites="5ET1D3cK w5B6qRKv">[<a href="#ref-5ET1D3cK" role="doc-biblioref">82</a>,<a href="#ref-w5B6qRKv" role="doc-biblioref">83</a>]</span>, liver <span class="citation" data-cites="TTSFlLVC">[<a href="#ref-TTSFlLVC" role="doc-biblioref">84</a>]</span>, and gastrointestinal tract <span class="citation" data-cites="PmE9xedP 3Ak4Mata">[<a href="#ref-PmE9xedP" role="doc-biblioref">85</a>,<a href="#ref-3Ak4Mata" role="doc-biblioref">86</a>]</span>.
As it becomes clear that SARS-CoV-2 infection can damage multiple organs, the scientific community is pursuing multiple avenues of investigation in order to build a consensus about how the virus affects the human body.</p>
<h2 data-number="2.2" id="clinical-presentation-of-covid-19-1"><span class="header-section-number">2.2</span> Clinical Presentation of COVID-19</h2>
<p>Reports have described diverse symptom profiles associated with COVID-19.
A large study from Wuhan, China suggested that fever and cough are the two most common symptoms at hospital admission <span class="citation" data-cites="15IDA5kX6">[<a href="#ref-15IDA5kX6" role="doc-biblioref">35</a>]</span>, and another early retrospective study in China described the clinical presentations of patients infected with SARS-CoV-2 as including lower respiratory tract infection with fever, dry cough, and dyspnea <span class="citation" data-cites="10THxyeCg">[<a href="#ref-10THxyeCg" role="doc-biblioref">36</a>]</span>.
This study <span class="citation" data-cites="10THxyeCg">[<a href="#ref-10THxyeCg" role="doc-biblioref">36</a>]</span> noted that upper respiratory tract symptoms were less common, which suggested that the virus was targeting cells located in the lower respiratory tract.
A later study reported radiographic findings such as ground-glass opacity and bilateral patchy shadowing in the lungs of many hospitalized patients, and most COVID-19 patients had lymphocytopenia, meaning they had low levels of lymphocytes (a type of white blood cell) <span class="citation" data-cites="15IDA5kX6">[<a href="#ref-15IDA5kX6" role="doc-biblioref">35</a>]</span>.
However, data from the New York City region <span class="citation" data-cites="19ytsiSpq 1H8gSsSkn">[<a href="#ref-19ytsiSpq" role="doc-biblioref">37</a>,<a href="#ref-1H8gSsSkn" role="doc-biblioref">38</a>]</span> showed variable rates of fever as a presenting symptom, suggesting that symptoms may not be consistent across samples.
These differences persist when comparing both between institutions in similar locations and between different regions experiencing COVID-19 outbreaks, leading to conflicting reports of the frequency of fever at hospital admission.
For example, even within New York City, one study <span class="citation" data-cites="19ytsiSpq">[<a href="#ref-19ytsiSpq" role="doc-biblioref">37</a>]</span> identified low oxygen saturation (&lt;90% without the use of supplemental oxygen or ventilation support) in a significant percentage of patients upon presentation, while another study <span class="citation" data-cites="1H8gSsSkn">[<a href="#ref-1H8gSsSkn" role="doc-biblioref">38</a>]</span> reported cough, fever, and dyspnea as the most common presenting symptoms.
Patients may also experience loss of smell, myalgias (muscle aches), fatigue, or headache.
Radiographic findings such as ground-glass opacity and bilateral patchy shadowing in the lungs of many hospitalized patients have also been reported, and most COVID-19 patients had lymphocytopenia, meaning they had low levels of lymphocytes (a type of white blood cell) <span class="citation" data-cites="15IDA5kX6">[<a href="#ref-15IDA5kX6" role="doc-biblioref">35</a>]</span>.
The variability of both which symptoms present and their severity makes it difficult for public health agencies to provide clear recommendations for citizens regarding what symptoms indicate SARS-CoV-2 infection and should prompt isolation.
For example, evidence suggests that gastrointestinal symptoms presentation does occur <span class="citation" data-cites="18tT1tLJI">[<a href="#ref-18tT1tLJI" role="doc-biblioref">87</a>]</span>, and the CDC has updated its list of symptoms suggestive of COVID-19 to include nausea and vomiting, as well congestion and runny nose <span class="citation" data-cites="gHWlMufv">[<a href="#ref-gHWlMufv" role="doc-biblioref">45</a>]</span>.
A recent preprint using data from an app-based survey of 500,000 individuals in the US found that among those tested for SARS-CoV-2, a loss of taste or smell, fever, and a cough were significant predictors of a positive test result <span class="citation" data-cites="19IBzQBhB">[<a href="#ref-19IBzQBhB" role="doc-biblioref">88</a>]</span>.
It is important to note that in this study, the predictive value of symptoms may be underestimated if they are not specific to COVID-19 because the outcome measured was a positive, as opposed to a negative, COVID-19 test result.
At the time the surveys were conducted, due to limits in US testing infrastructure, respondents typically needed to have some symptoms known to be specific to COVID-19 in order to qualify for testing in the first place.
Widespread testing of asymptomatic individuals may therefore provide additional insight into the range of symptoms associated with COVID-19.</p>
<p>COVID-19 can affect diverse body systems in addition to causing respiratory problems <span class="citation" data-cites="wK2afyL8">[<a href="#ref-wK2afyL8" role="doc-biblioref">89</a>]</span>.
For example, COVID-19 can lead to acute kidney injury, especially in patients with severe respiratory symptoms or certain preexisting conditions <span class="citation" data-cites="17hdoiBo9">[<a href="#ref-17hdoiBo9" role="doc-biblioref">90</a>]</span>.
It can also cause neurological complications <span class="citation" data-cites="1CqoZGSKK 1EF9EpbZ5">[<a href="#ref-1CqoZGSKK" role="doc-biblioref">91</a>,<a href="#ref-1EF9EpbZ5" role="doc-biblioref">92</a>]</span>, potentially including stroke, seizures or meningitis <span class="citation" data-cites="nBGGUV9s LLm9d62d">[<a href="#ref-nBGGUV9s" role="doc-biblioref">93</a>,<a href="#ref-LLm9d62d" role="doc-biblioref">94</a>]</span>.
COVID-19 has also been associated with an increased incidence of large vessel stroke, particularly in patients under the age of 40 <span class="citation" data-cites="aRNqjkMg">[<a href="#ref-aRNqjkMg" role="doc-biblioref">95</a>]</span>, and other thrombotic events including pulmonary embolism and deep vein thrombosis <span class="citation" data-cites="3UBbDrG2">[<a href="#ref-3UBbDrG2" role="doc-biblioref">96</a>]</span>.
The mechanism behind these complications has been suggested to be related to coagulopathy, with reports indicating the presence of antiphospholipid antibodies <span class="citation" data-cites="8OnbWuhF">[<a href="#ref-8OnbWuhF" role="doc-biblioref">97</a>]</span> and elevated levels of d-dimer and fibrinogen degradation products (FDP) in deceased patients <span class="citation" data-cites="dUfws1q0">[<a href="#ref-dUfws1q0" role="doc-biblioref">98</a>]</span>.
Other viral infections have been associated with coagulation defects and changes to the coagulation cascade; notably SARS was also found to lead to disseminated intravascular coagulation (DIC) and was associated with both pulmonary embolism and deep vein thrombosis <span class="citation" data-cites="p5EhN9Qd">[<a href="#ref-p5EhN9Qd" role="doc-biblioref">99</a>]</span>.
The mechanism behind these insults has been suggested to be related to inflammation-induced increases in the von Willebrand factor (VWF) clotting protein, leading to a pro-coagulative state <span class="citation" data-cites="p5EhN9Qd">[<a href="#ref-p5EhN9Qd" role="doc-biblioref">99</a>]</span>.
Abnormal clotting (thromboinflammation or coagulopathy) has been increasingly discussed recently as a possible key mechanism in many cases of severe COVID-19, and may be associated with the high d-dimer levels often observed in severe cases <span class="citation" data-cites="v1EIzwfx 18AiyvhO8 PyKMLraw">[<a href="#ref-v1EIzwfx" role="doc-biblioref">100</a>,<a href="#ref-18AiyvhO8" role="doc-biblioref">101</a>,<a href="#ref-PyKMLraw" role="doc-biblioref">102</a>]</span>.
This excessive clotting in lung capillaries has been suggested to be related to a dysregulated activation of the complement system, part of the innate immune system <span class="citation" data-cites="14dyYQY7s 17y6YeJ6R">[<a href="#ref-14dyYQY7s" role="doc-biblioref">103</a>,<a href="#ref-17y6YeJ6R" role="doc-biblioref">104</a>]</span>.</p>
<h3 data-number="2.2.1" id="cytokine-release-syndrome"><span class="header-section-number">2.2.1</span> Cytokine Release Syndrome</h3>
<p>Symptoms of a disease can be caused by a pathogen, but they can also be caused by the immune system’s reaction to the pathogen.
A dysregulated immune response can cause significant damage to the host <span class="citation" data-cites="o6BQnEt7 b1QVdemU 1GnFL9zeN">[<a href="#ref-o6BQnEt7" role="doc-biblioref">105</a>,<a href="#ref-b1QVdemU" role="doc-biblioref">106</a>,<a href="#ref-1GnFL9zeN" role="doc-biblioref">107</a>]</span>.
The inflammatory response has received particular attention for its role in both a healthy response to infection and a pathogenic one.
Inflammation is one of the most visible components of the immune response, as it is responsible for hallmarks of injury such as pain, heat, and swelling <span class="citation" data-cites="sXusUkLI">[<a href="#ref-sXusUkLI" role="doc-biblioref">108</a>]</span>.
In response to injury or to signaling by pattern recognition receptors (PRRs) indicating the detection of a molecular pattern associated with a pathogen or foreign body, the immune system stimulates leukocytes that travel to the site of the threat, where they then produce cytokines <span class="citation" data-cites="sXusUkLI">[<a href="#ref-sXusUkLI" role="doc-biblioref">108</a>]</span>.
Cytokines are a diverse group of small proteins that play an important role in intercellular signaling <span class="citation" data-cites="bZMKqj6e">[<a href="#ref-bZMKqj6e" role="doc-biblioref">109</a>]</span>.
Cytokines can be both pro- and anti-inflammatory, which means they can either stimulate or inhibit the production of additional cytokines <span class="citation" data-cites="bZMKqj6e DYbswZ6D">[<a href="#ref-bZMKqj6e" role="doc-biblioref">109</a>,<a href="#ref-DYbswZ6D" role="doc-biblioref">110</a>]</span>.
Some notable pro-inflammatory cytokines include the interleukins IL-1β and IL-6 and tumor necrosis factor α (TNF-α) <span class="citation" data-cites="DYbswZ6D">[<a href="#ref-DYbswZ6D" role="doc-biblioref">110</a>]</span>.
Anti-inflammatory cytokines play an immunoregulatory role complementary to the cascading effect of pro-inflammatory cytokines <span class="citation" data-cites="bZMKqj6e DYbswZ6D">[<a href="#ref-bZMKqj6e" role="doc-biblioref">109</a>,<a href="#ref-DYbswZ6D" role="doc-biblioref">110</a>]</span>.
A number of interleukins and interferons play anti-inflammatory roles, and receptors or receptor antagonists for inflammatory cytokines are also important for regulating inflammation <span class="citation" data-cites="DYbswZ6D">[<a href="#ref-DYbswZ6D" role="doc-biblioref">110</a>]</span>.
IL-10 is an anti-inflammatory cytokines of particular note because it regulates the expression of TNF-α, IL-1, and IL-6 <span class="citation" data-cites="DYbswZ6D">[<a href="#ref-DYbswZ6D" role="doc-biblioref">110</a>]</span>.
When the pro- and anti-inflammatory responses are both commensurate with the threat posed, the immune system drives a shift back to homeostasis <span class="citation" data-cites="16CYY7vzG">[<a href="#ref-16CYY7vzG" role="doc-biblioref">111</a>]</span>.
However, when the responses are disproportionate, the cytokine response can become dysregulated.
Too low of an inflammatory response will not eliminate the immune threat <span class="citation" data-cites="16CYY7vzG">[<a href="#ref-16CYY7vzG" role="doc-biblioref">111</a>]</span>.
In contrast, if the response is dysregulated towards excessive pro-inflammatory cytokine activity, inflammation can cascade <span class="citation" data-cites="xuMYmc7W">[<a href="#ref-xuMYmc7W" role="doc-biblioref">112</a>]</span> and cause cell damage, among other problems <span class="citation" data-cites="sXusUkLI">[<a href="#ref-sXusUkLI" role="doc-biblioref">108</a>]</span>.
Elevated levels of inflammation over the long-term are associated with many chronic health conditions, including type 2 diabetes, dementia and Alzheimer’s, and arthritis, among others <span class="citation" data-cites="OvhepUrA">[<a href="#ref-OvhepUrA" role="doc-biblioref">113</a>]</span>.
On a shorter timescale, dysregulated systemic inflammation can cause sepsis, which can lead to multi-organ failure and death <span class="citation" data-cites="bZMKqj6e 6eWgCrWj">[<a href="#ref-bZMKqj6e" role="doc-biblioref">109</a>,<a href="#ref-6eWgCrWj" role="doc-biblioref">114</a>]</span>.</p>
<p>Cytokines have been investigated for their role in the immune response to lung infections long before the COVID-19 pandemic.
Dysregulation of the inflammatory response, including elevated levels of pro-inflammatory cytokines, is found in patients with ARDS, which is a severe condition that can arise from pneumonia, SARS, and COVID-19 <span class="citation" data-cites="xuMYmc7W">[<a href="#ref-xuMYmc7W" role="doc-biblioref">112</a>]</span>.
One study of patients with and at risk for ARDS (specifically, those who were intubated for medical ventilation) found that shortly after the onset of ARDS, anti-inflammatory cytokine concentration in BALF increased relative to the concentration of pro-inflammatory cytokines <span class="citation" data-cites="f61jsRKY">[<a href="#ref-f61jsRKY" role="doc-biblioref">115</a>]</span>.
However, patients with severe ARDS were excluded from this study.
The results suggest that an increase in pro-inflammatory cytokines such as IL-6 may signal the onset of ARDS, but recovery depends on an increased anti-inflammatory response <span class="citation" data-cites="f61jsRKY">[<a href="#ref-f61jsRKY" role="doc-biblioref">115</a>]</span>.
Acute phase response to an infection can also cause damage to the capillary endothelium, allowing leaks that disrupt the balance between pro-inflammatory cytokines and their regulators <span class="citation" data-cites="f61jsRKY">[<a href="#ref-f61jsRKY" role="doc-biblioref">115</a>]</span>.
Hyperactivity of the pro-inflammatory response due to lung infection is commonly associated with acute lung injury and more rarely with the more severe manifestation, ARDS <span class="citation" data-cites="bZMKqj6e">[<a href="#ref-bZMKqj6e" role="doc-biblioref">109</a>]</span>.
The heightened inflammatory response in the lungs can also serve as a source for systemic inflammation, or sepsis, which can lead to multi-organ failure <span class="citation" data-cites="bZMKqj6e">[<a href="#ref-bZMKqj6e" role="doc-biblioref">109</a>]</span>.
The shift from local to systemic inflammation is a phenomenon often referred to broadly as a cytokine storm <span class="citation" data-cites="bZMKqj6e">[<a href="#ref-bZMKqj6e" role="doc-biblioref">109</a>]</span> or, more precisely, as cytokine release syndrome (CRS) <span class="citation" data-cites="NITOJ0Ka">[<a href="#ref-NITOJ0Ka" role="doc-biblioref">116</a>]</span>.
Sepsis is a known possible complication of pneumonia, and in an analysis of over 1400 US pneumonia patients, IL-6, tumor necrosis factor (TNF), and IL-1β were found to be elevated at intake in patients who developed severe sepsis and/or ultimately deceased <span class="citation" data-cites="oSOG8N0f">[<a href="#ref-oSOG8N0f" role="doc-biblioref">117</a>]</span>.
IL-6 and TNF are pro-inflammatory cytokines, while IL-1β is anti-inflammatory <span class="citation" data-cites="oSOG8N0f">[<a href="#ref-oSOG8N0f" role="doc-biblioref">117</a>]</span>.
However, this study reported that unbalanced pro-/anti-inflammatory cytokine profiles were rare, although they measured only the three cytokines listed above.
Prior work therefore made it clear that pulmonary infection and injury were associated with systemic inflammation through sepsis.
While IL-6 is a biomarker sometimes used to assess cytokine storm activity in sepsis <span class="citation" data-cites="bZMKqj6e">[<a href="#ref-bZMKqj6e" role="doc-biblioref">109</a>]</span>, the relationship between cytokine profiles and the risks associated with sepsis may be more complex.
IL-6 is a pleiotropic cytokine that plays an integral role in both the inflammatory and anti-inflammatory responses and is associated with both healthy and pathological responses to viral threat <span class="citation" data-cites="dBAe8aYi">[<a href="#ref-dBAe8aYi" role="doc-biblioref">118</a>]</span>.</p>
<p>The inflammatory response was identified early on as a potential driver of COVID-19 outcomes due to existing research in SARS and emerging research in COVID-19.
In addition to the known role of cytokines in ARDS and lung infection more broadly, immunohistological analysis at autopsy of patients who deceased from SARS revealed that ACE2-expressing cells that were infected by SARS-CoV showed elevated expression of IL-6, IL-1β, and TNF-α <span class="citation" data-cites="NNFGje9g">[<a href="#ref-NNFGje9g" role="doc-biblioref">119</a>]</span>.
Similarly, the introduction of the S-protein from SARS-CoV to mouse macrophages was found to increase production of IL-6 and TNF-α <span class="citation" data-cites="QkTGQUcj">[<a href="#ref-QkTGQUcj" role="doc-biblioref">120</a>]</span>.
For SARS-CoV-2 infection leading to COVID-19, early reports described a cytokine storm-like response in patients with particularly severe infections <span class="citation" data-cites="UwVweB2M 3HdlV9Vf cyUG1Zi2">[<a href="#ref-UwVweB2M" role="doc-biblioref">76</a>,<a href="#ref-3HdlV9Vf" role="doc-biblioref">121</a>,<a href="#ref-cyUG1Zi2" role="doc-biblioref">122</a>]</span>.
Among patients hospitalized with COVID-19 in Wuhan, China, 112 out of 191 (59%) developed sepsis, including all 54 of the non-survivors <span class="citation" data-cites="10THxyeCg">[<a href="#ref-10THxyeCg" role="doc-biblioref">36</a>]</span>.
However, the argument has been made that while the cytokine levels observed in COVID-19 patients fall outside of the normal range, they are not be as high as typically found in patients with ARDS <span class="citation" data-cites="Xcrxlxnp">[<a href="#ref-Xcrxlxnp" role="doc-biblioref">123</a>]</span>.
Regardless, inflammation has received significant interest both in regards to the pathology of COVID-19 as well as potential avenues for treatment, as the similarities between the cytokine storm and the pathophysiology of COVID-19 has led to the suggestion that a number of immunomodulatory pharmaceutical interventions could hold therapeutic value for the treatment of COVID-19 <span class="citation" data-cites="RIMWgsWt">[<a href="#ref-RIMWgsWt" role="doc-biblioref">124</a>]</span>.</p>
<h3 data-number="2.2.2" id="pediatric-presentation"><span class="header-section-number">2.2.2</span> Pediatric Presentation</h3>
<p>The presentation of COVID-19 infection can vary greatly among pediatric patients and in some cases manifests in distinct ways from COVID-19 in adults.
A review examined symptoms reported in 17 studies of children infected with COVID-19 during the early months of the COVID-19 epidemic in China and one study from Singapore <span class="citation" data-cites="1BhN2uq3f">[<a href="#ref-1BhN2uq3f" role="doc-biblioref">125</a>]</span>.
Of the more than a thousand cases described, the most commonly reports were for mild symptoms such as fever, dry cough, fatigue, nasal congestion and/or runny nose, while three children were reported to be asymptomatic.
Severe lower respiratory infection was described in only one of the pediatric cases reviewed.
Gastrointestinal symptoms such as vomiting or diarrhea were occasionally reported.
Radiologic findings were not always reported in the case studies reviewed, but when they were mentioned they included bronchial thickening, ground-glass opacities, and/or inflammatory lesions <span class="citation" data-cites="1BhN2uq3f">[<a href="#ref-1BhN2uq3f" role="doc-biblioref">125</a>]</span>.
Neurological symptoms have also been reported <span class="citation" data-cites="12lVfu2Qe">[<a href="#ref-12lVfu2Qe" role="doc-biblioref">126</a>]</span>.</p>
<p>These analyses indicate that most pediatric cases of COVID-19 are not severe.
However, serious complications and, in rare cases, deaths have occurred <span class="citation" data-cites="MCVe25tf">[<a href="#ref-MCVe25tf" role="doc-biblioref">127</a>]</span>.
Of particular interest, children have occasionally experienced a serious inflammatory syndrome, multisystem inflammatory syndrome in children (MIS-C), following COVID-19 infection.
This syndrome is similar in some respects to Kawasaki disease or to Kawasaki disease shock syndrome <span class="citation" data-cites="fZwrv8M AmvwCWm3 68JorBeU">[<a href="#ref-fZwrv8M" role="doc-biblioref">128</a>,<a href="#ref-AmvwCWm3" role="doc-biblioref">129</a>,<a href="#ref-68JorBeU" role="doc-biblioref">130</a>]</span> and is thought to be a distinct clinical manifestation of SARS-CoV-2 due to its distinct cytokine profile and the presence of burr cells in peripheral blood smears <span class="citation" data-cites="10gSXfOKm 10QhaDu6N">[<a href="#ref-10gSXfOKm" role="doc-biblioref">131</a>,<a href="#ref-10QhaDu6N" role="doc-biblioref">132</a>]</span>.
MIS-C has been associated with heart failure in some cases <span class="citation" data-cites="FYgN6gUL">[<a href="#ref-FYgN6gUL" role="doc-biblioref">133</a>]</span>.
One case study <span class="citation" data-cites="71To4laE">[<a href="#ref-71To4laE" role="doc-biblioref">134</a>]</span> described an adult who appeared to show symptoms similar to MIS-C after exposure to COVID-19, but cautioned against broad conclusions; a second possible adult case has also been reported <span class="citation" data-cites="UBDAxWWv">[<a href="#ref-UBDAxWWv" role="doc-biblioref">135</a>]</span>.
The presentation of SARS-CoV-2 infection is therefore likely to be largely distinct between adult and pediatric populations.</p>
<h2 data-number="2.3" id="systems-level-effects"><span class="header-section-number">2.3</span> Systems-Level Effects</h2>
<p>Systems biology provides a cross-disciplinary analytical paradigm through which the host response to an infection can be analyzed.
This field integrates the “omics” (genomics, transcriptomics, proteomics, metabolomics, etc.) with bioinformatics and other computational strategies.
These cutting-edge research approaches hold enormous potential for the study of the complexity of biological systems and human diseases <span class="citation" data-cites="5MhhHQO0">[<a href="#ref-5MhhHQO0" role="doc-biblioref">136</a>]</span>.
Over the last decade, systems biology approaches have been used widely to study the pathogenesis of diverse types of life-threatening acute and chronic infectious diseases <span class="citation" data-cites="8esIIWt9">[<a href="#ref-8esIIWt9" role="doc-biblioref">137</a>]</span>.
Omics-based studies have also provided meaningful information regarding host immune responses and surrogate protein markers in several viral, bacterial and protozoan infections <span class="citation" data-cites="8vUHj9cm">[<a href="#ref-8vUHj9cm" role="doc-biblioref">138</a>]</span>.
Though the complex pathogenesis and clinical manifestations of SARS-CoV-2 infection are not yet fully understood, “omics” technologies offer the opportunity for discovery-driven analysis of biological changes associated with SARS-CoV-2 infection.
For example, previous studies suggest that infection by coronaviruses such as SARS-CoV and MERS-CoV and other viruses is associated with the upregulation of ACE2.
In several preliminary assays and an analysis of previous microarray data, ACE2 expression was reported to be significantly upregulated following infection of human embryonic kidney cells and human airway epithelial cells <span class="citation" data-cites="UwVweB2M">[<a href="#ref-UwVweB2M" role="doc-biblioref">76</a>]</span>.
This study also reported that direct stimulation with inflammatory cytokines such as type I interferons (e.g., IFNβ) resulted in the upregulation of ACE2 in human bronchial epithelial cells, with treated groups showing four-fold higher ACE2 expression than control groups at 18 hours post-treatment <span class="citation" data-cites="UwVweB2M">[<a href="#ref-UwVweB2M" role="doc-biblioref">76</a>]</span>.
Whether SARS-CoV-2 facilitates the positive regulation of its own transmission between host cells is still unclear, the host immune response itself likely plays a key role in mediating infection-associated pathologies.
A systems-biology approach allows for analyses such as these to identify possible phenotypic and endophenotypic responses to SARS-CoV-2 infection and to develop new hypotheses about how pathogenesis proceeds.</p>
<h3 data-number="2.3.1" id="transcriptomics"><span class="header-section-number">2.3.1</span> Transcriptomics</h3>
<p>In addition to the study described above, two other studies have profiled expression following SARS-CoV-2 infection using human cell lines.
The first study <span class="citation" data-cites="pHrstqMQ">[<a href="#ref-pHrstqMQ" role="doc-biblioref">139</a>]</span> compared transcriptional responses to SARS-CoV-2 and to other respiratory viruses, including MERS-CoV, SARS-CoV, human parainfluenza virus 3 (HPIV3), respiratory syncytial virus (RSV), and influenza A virus (IAV).
They analyzed the responses of three human cell lines: A549 (adenocarcinomic human alveolar basal epithelial cells), Calu-3 (human airway epithelial cells derived from human bronchial submucosal glands), and MRC-5 (human fetal lung fibroblast cells).
As the viral receptor ACE2 has low expression in A549 cells, they supplemented the A549 cells with adenovirus (AdV)-based vectors expressing either mCherry (a fluorescent protein used as a control) or ACE2 (A549-ACE2).
The authors also measured host transcriptional responses to SARS-CoV-2 in primary normal human bronchial epithelial cells (HBEC or NHBE cells), nasal washes from an animal model (ferret), and lung samples from two COVID-19 patients.
Differential expression (DE) analysis was then carried out to compare infected cells with control cells that underwent only a mock treatment.
In the hosts where SARS-CoV-2 was able to replicate efficiently, DE analysis revealed that the transcriptional response was significantly different from the response to all of the other viruses tested.
A unique proinflammatory cytokine signature associated with SARS-CoV-2 was present under in cells exposed to both high and low doses of the virus, with the cytokines IL-6 and IL1RA uniquely elevated in response to SARS-CoV-2 relative to other viruses.
However, the A549-ACE2 cells showed significant IFN-I or IFN-III expression when exposed to high, but not low, doses of SARS-CoV-2.
This finding suggests that IFN induction is dependent on the extent of exposure.
Similarly, in cells from the NHBE line, ferrets, and COVID-19 patients, chemokine signaling was significantly enriched, but there was no significant induction of IFN-I or IFN-III.
Together, these results suggest that SARS-CoV-2 induces a limited antiviral state with low IFN-I or IFN-III expression and a moderate IFN-stimulated gene response, in contrast to other viruses.
However, in ACE2-expressing A549 cells, this state could be overcome by using a 10-fold increase in SARS-CoV-2 exposure.
This finding suggests that the SARS-CoV-2 antagonist is insufficient for large doses of the virus <span class="citation" data-cites="pHrstqMQ">[<a href="#ref-pHrstqMQ" role="doc-biblioref">139</a>]</span>.
This hypothesis was further supported by a recent study <span class="citation" data-cites="PJgYPcKM">[<a href="#ref-PJgYPcKM" role="doc-biblioref">140</a>]</span> that showed that the SARS-CoV-2 <em>ORF3b</em> gene suppresses IFNB1 promoter activity (IFN-I induction) more efficiently than the SARS-CoV_ORF3b_ gene.
Taken together, these findings suggest that a unique cytokine profile is associated with the response to the SARS-CoV-2 virus, and that this response differs depending on the extent of exposure.</p>
<p>Another study <span class="citation" data-cites="Gj8vlc0W">[<a href="#ref-Gj8vlc0W" role="doc-biblioref">141</a>]</span> analyzed cells’ transcriptional response to SARS-CoV-2 and SARS-CoV over time.
They characterized the response of three human cell lines, H1299 (human non-small cell lung carcinoma cell line), Calu-3, and Caco-2 (human epithelial colorectal adenocarcinoma cell line), at 4 to 36 hpi.
Using poly(A) bulk RNA-seq, the authors found negligible susceptibility of H1299 cells (&lt; 0.08 viral read percentage of total reads) compared to Caco-2 and Calu-3 cells (&gt;10% of viral reads).
This finding suggests that the risk of infection varies among cell types, and that cell type could influence which hosts are more or less susceptible.
Based on visual inspection (microscopy images) and transcriptional profiling, the authors also showed distinct responses among the host cell lines evaluated.
In contrast to Caco-2, Calu-3 cells infected with SARS-CoV-2 showed signs of impaired growth and cell death at 24 hpi, as well as moderate IFN induction with a strong up-regulation of IFN-stimulated genes.
Interestingly, the results were similar to those reported in Calu-3 cells exposed to much higher levels of SARS-CoV-2 <span class="citation" data-cites="pHrstqMQ">[<a href="#ref-pHrstqMQ" role="doc-biblioref">139</a>]</span>, as described above.
This finding suggests that IFN induction in Calu-3 cells is not dependent on the level of exposure, in contrast to A549-ACE2 cells.
The discrepancy could be explained by the observations that Calu-3 cells are highly susceptible to SARS-CoV-2 and show rapid viral replication <span class="citation" data-cites="JOJ2n3gC">[<a href="#ref-JOJ2n3gC" role="doc-biblioref">66</a>]</span>, whereas A549 cells are incompatible with SARS-CoV-2 infection <span class="citation" data-cites="16EFyBURq">[<a href="#ref-16EFyBURq" role="doc-biblioref">142</a>]</span>.
This discrepancy raises the concern that <em>in vitro</em> models may vary in their similarity to the human response, underscoring the importance of follow-up studies in additional models.</p>
<h3 data-number="2.3.2" id="proteomics"><span class="header-section-number">2.3.2</span> Proteomics</h3>
<p>No comprehensive proteomic analyses of the pathogen or of patients suffering from its infection have yet been reported.
One study investigated proteomics associated with <em>in vitro</em> infection using Caco-2 cells infected with SARS-CoV-2 <span class="citation" data-cites="11xZWeHN3">[<a href="#ref-11xZWeHN3" role="doc-biblioref">143</a>]</span>.
This study reported that SARS-CoV-2 induced alterations in multiple vital physiological pathways, including translation, splicing, carbon metabolism and nucleic acid metabolism in the host cells.
Another area of interest is whether SARS-CoV-2 is likely to induce similar changes to other hCoV.
For example, because of the high level of sequence homology between SARS-CoV-2 and SARS-CoV, it has been hypothesized that sera from convalescent SARS-CoV patients might show some efficacy in cross-neutralizing SARS-CoV-2-S-driven entry <span class="citation" data-cites="15l3di3Wj">[<a href="#ref-15l3di3Wj" role="doc-biblioref">65</a>]</span>.
However, despite the high level of sequence homology, certain protein structures might be immunologically distinct, which would be likely to prohibit effective cross-neutralization across different SARS species <span class="citation" data-cites="GhJYjnft">[<a href="#ref-GhJYjnft" role="doc-biblioref">144</a>]</span>.
Consequently, proteomic analyses of SARS-CoV might also provide some essential information regarding the new pathogen <span class="citation" data-cites="GpnngtWK JLTf2Fwb">[<a href="#ref-GpnngtWK" role="doc-biblioref">145</a>,<a href="#ref-JLTf2Fwb" role="doc-biblioref">146</a>]</span>.</p>
<p>Considering the paucity of omics-level big data sets for SARS-CoV-2 currently available, existing data hubs that contain information for other coronaviruses such as UniProt, NCBI Genome Database, The Immune Epitope Database and Analysis Resource (IEDB), and The Virus Pathogen Resource (ViPR) will serve as useful resources for computational and bioinformatics research on SARS-CoV-2.
Using such databases, the systems-level reconstruction of the protein-protein interaction (PPI) will enable the generation of hypotheses about the mechanism of action of SARS-CoV-2 and suggest potential drug targets.
In an initial study <span class="citation" data-cites="phJM8g2Y">[<a href="#ref-phJM8g2Y" role="doc-biblioref">147</a>]</span>, 26 of the 29 SARS-CoV-2 proteins were cloned and expressed in HEK293T kidney cells, allowing for the identification of 332 high-confidence human proteins interacting with them.
Notably, this study suggested that SARS-CoV-2 interacts with innate immunity pathways.
Ranking pathogens by the similarity between their interactomes and that of SARS-CoV-2 suggested <em>West Nile Virus</em>, <em>Mycobacterium tuberculosis</em>, and <em>human papillomavirus</em> as the top three hits.
Therefore, given the lung symptoms associated with COVID-19, the <em>Mycobacterium tuberculosis</em> host-pathogen interactome in particular might provide new insights to the mechanism of SARS-CoV-2 infection.
Additionally, it was suggested that the envelope protein (E) could disrupt host bromodomain-containing proteins, i.e., BRD2 and BRD4, that bind to histones, and the spike protein could likely intervene in viral fusion by modulating the GOLGA7-ZDHHC5 acyl-transferase complex to increase palmitoylation, which is a post-translational modification that affects how proteins interact with membranes <span class="citation" data-cites="mXUCjmCh">[<a href="#ref-mXUCjmCh" role="doc-biblioref">148</a>]</span>.
Another study <span class="citation" data-cites="NLoN4aYj">[<a href="#ref-NLoN4aYj" role="doc-biblioref">149</a>]</span> used patient-derived peripheral blood mononuclear cells (PBMCs) to identify 251 host proteins targeted by SARS-CoV-2.
This study also reported that more than 200 host proteins were disrupted following infection.
In particular, a network analysis showed that non-structural proteins 9 and 10 (nsp9 and nsp10) interacted with NF-Kappa-B-Repressing Factor (NKRF), which encodes a transcriptional repressor that mediates repression of genes responsive to Nuclear Factor kappa-light-chain-enhancer of activated B-cells (NF-kB).
These genes are important to pro-, and potentially also anti-, inflammatory signaling <span class="citation" data-cites="KAqOiTeZ">[<a href="#ref-KAqOiTeZ" role="doc-biblioref">150</a>]</span>.
This finding could explain the exacerbation of the immune response that shapes the pathology and the high cytokine levels characteristic of COVID-19, possibly due to the chemotaxis of neutrophils mediated by IL-8 and IL-6.
Finally, it was suggested <span class="citation" data-cites="1ccnm0N9">[<a href="#ref-1ccnm0N9" role="doc-biblioref">151</a>]</span> that the E protein of both SARS-CoV and SARS-CoV-2 has a conserved Bcl-2 Homology 3 (BH3)-like motif, which could inhibit anti-apoptosis proteins, e.g., BCL2, and trigger the apoptosis of T cells.
Several compounds are known to disrupt the host-pathogen protein interactome, largely through the inhibition of host proteins.
Therefore, preliminary research in the proteomics of SARS-CoV-2 infection suggests that drugs modulating the protein-level interactions between virus and host might be worth investigating.</p>
<h2 data-number="2.4" id="viral-evolution-and-virulence"><span class="header-section-number">2.4</span> Viral Evolution and Virulence</h2>
<p>Like that of SARS-CoV, the entry of SARS-CoV-2 into host cells is mediated by interactions between the viral spike glycoprotein (S) and human angiotensin-converting enzyme 2 (ACE2) <span class="citation" data-cites="15EIBRhef qcVbT0w4 12TZ7hPMA wCbhn23d VX8OWaGj 15l3di3Wj 15Q2XgkK7 t1e4CW9A">[<a href="#ref-qcVbT0w4" role="doc-biblioref">62</a>,<a href="#ref-15l3di3Wj" role="doc-biblioref">65</a>,<a href="#ref-15EIBRhef" role="doc-biblioref">152</a>,<a href="#ref-12TZ7hPMA" role="doc-biblioref">153</a>,<a href="#ref-wCbhn23d" role="doc-biblioref">154</a>,<a href="#ref-VX8OWaGj" role="doc-biblioref">155</a>,<a href="#ref-15Q2XgkK7" role="doc-biblioref">156</a>,<a href="#ref-t1e4CW9A" role="doc-biblioref">157</a>]</span>.
Differences in how the S proteins of the two viruses interact with the human ACE2 (hACE2) receptor could also partially account for the increased transmissibility of SARS-CoV-2.
Recent studies have reported conflicting binding constants for the S-hACE2 interaction, though they have agreed that the SARS-CoV-2 S protein binds with equal, if not greater, affinity to the SARS-CoV S protein does <span class="citation" data-cites="qcVbT0w4 VX8OWaGj CqQVkaqj">[<a href="#ref-CqQVkaqj" role="doc-biblioref">49</a>,<a href="#ref-qcVbT0w4" role="doc-biblioref">62</a>,<a href="#ref-VX8OWaGj" role="doc-biblioref">155</a>]</span>.
The C-terminal domain of the SARS-CoV-2 S protein in particular was identified as the key region of the virus that interacts with hACE2, and the crystal structure of the C-terminal domain of the SARS-CoV-2 S protein in complex with hACE2 reveals stronger interaction and a higher affinity for receptor binding than that of SARS-CoV <span class="citation" data-cites="15Q2XgkK7">[<a href="#ref-15Q2XgkK7" role="doc-biblioref">156</a>]</span>.
Among the 14 key binding residues identified in the SARS-CoV S protein, 8 are conserved in SARS-CoV-2, and the remaining 6 are semi-conservatively substituted, potentially explaining variation in binding affinity <span class="citation" data-cites="qcVbT0w4 VX8OWaGj">[<a href="#ref-qcVbT0w4" role="doc-biblioref">62</a>,<a href="#ref-VX8OWaGj" role="doc-biblioref">155</a>]</span>.
Recent crystal structures have shown that the receptor-binding domain (RBD) of the SARS-CoV-2 S protein, like that of other coronaviruses, undergoes stochastic hinge-like movement that flips it from a “closed” conformation, in which key binding residues are hidden at the interface between protomers, to an “open” one <span class="citation" data-cites="qcVbT0w4 CqQVkaqj">[<a href="#ref-CqQVkaqj" role="doc-biblioref">49</a>,<a href="#ref-qcVbT0w4" role="doc-biblioref">62</a>]</span>.
Because the RBD plays such a critical role in viral entry, blocking its interaction with ACE2 represents a promising therapeutic approach.
Nevertheless, despite the high structural homology between SARS-CoV-2 RBD and that of SARS-CoV, monoclonal antibodies targeting SARS-CoV-RBD failed to bind to SARS-CoV-2-RBD <span class="citation" data-cites="CqQVkaqj">[<a href="#ref-CqQVkaqj" role="doc-biblioref">49</a>]</span>.
Promisingly, though, sera from convalescent SARS patients inhibited SARS-CoV-2 viral entry <em>in vitro</em>, albeit with lower efficiency than it inhibited SARS-CoV <span class="citation" data-cites="15l3di3Wj">[<a href="#ref-15l3di3Wj" role="doc-biblioref">65</a>]</span>.</p>
<p>Comparative genomic analysis reveals that several regions of the coronavirus genome are likely critical to virulence.
The S1 domain of the spike protein, which contains the RBM, evolves more rapidly than <em>S</em>’s S2 domain <span class="citation" data-cites="13wWdgODZ OVsxrEuX">[<a href="#ref-13wWdgODZ" role="doc-biblioref">56</a>,<a href="#ref-OVsxrEuX" role="doc-biblioref">57</a>]</span>.
However, even within the S1 domain, some regions are more conserved than others, with the receptors in S1’s N-terminal domain (S1-NTD) evolving more rapidly than those in its C-terminal domain (S1-CTD) <span class="citation" data-cites="OVsxrEuX">[<a href="#ref-OVsxrEuX" role="doc-biblioref">57</a>]</span>.
Both S1-NTD and S1-CTD are involved in receptor binding and can function as RBDs to bind proteins and sugars <span class="citation" data-cites="13wWdgODZ">[<a href="#ref-13wWdgODZ" role="doc-biblioref">56</a>]</span>, but RBDs in the S1-NTD typically bind to sugars, while those in the S1-CTD recognize protein receptors <span class="citation" data-cites="17DSmRo9H">[<a href="#ref-17DSmRo9H" role="doc-biblioref">50</a>]</span>.
Viral receptors show higher affinity with protein receptors than sugar receptors <span class="citation" data-cites="17DSmRo9H">[<a href="#ref-17DSmRo9H" role="doc-biblioref">50</a>]</span>, which suggests that positive selection on or relaxed conservation of the S1-NTD might reduce the risk of a deleterious mutation that would prevent binding.
The SARS-CoV-2 S protein also contains a an RRAR furin recognition site at the S1/S2 junction <span class="citation" data-cites="qcVbT0w4 CqQVkaqj">[<a href="#ref-CqQVkaqj" role="doc-biblioref">49</a>,<a href="#ref-qcVbT0w4" role="doc-biblioref">62</a>]</span>, setting it apart from both bat coronavirus RaTG13, with which it shares 96% genome sequence identity, and SARS-CoV <span class="citation" data-cites="VSkK7CeP">[<a href="#ref-VSkK7CeP" role="doc-biblioref">13</a>]</span>.
Such furin cleavage sites are commonly found in highly virulent influenza viruses, and as such may contribute to the heightened pathogenicity of SARS-CoV-2 <span class="citation" data-cites="NsORsLig vhHB3yyS">[<a href="#ref-NsORsLig" role="doc-biblioref">158</a>,<a href="#ref-vhHB3yyS" role="doc-biblioref">159</a>]</span>.
Effective cell entry is a critical component to pathogenesis and therefore an important process to understand when examining possible therapeutics.</p>
<p>Evolution in SARS-CoV-2 has also been observed over a short timescale.
After zoonotic transfer, SARS-CoV-2 continued evolving in the human population <span class="citation" data-cites="p9LUiyCN">[<a href="#ref-p9LUiyCN" role="doc-biblioref">160</a>]</span>.
The SARS-CoV-2 mutation rate is moderate compared to other RNA viruses <span class="citation" data-cites="2w7lNKxQ">[<a href="#ref-2w7lNKxQ" role="doc-biblioref">161</a>]</span>, which likely restricts the pace of evolution in SARS-CoV-2.
Nevertheless, genomic analyses have yielded statistical evidence of ongoing evolution.
There are two known variants of the spike protein that differ by a single amino acid at position 614 (G614 and D614), and there is evidence that G614 had become more prevalent than D614 by June 2020 <span class="citation" data-cites="RIpPhJ1g">[<a href="#ref-RIpPhJ1g" role="doc-biblioref">162</a>]</span>.
While there is a hypothesis that this genomic change increased the SARS-CoV-2 infectivity and virulence, this hypothesis has not yet been tested due to a lack of data <span class="citation" data-cites="DyG1eyK4">[<a href="#ref-DyG1eyK4" role="doc-biblioref">163</a>]</span>.
Another study <span class="citation" data-cites="2w7lNKxQ">[<a href="#ref-2w7lNKxQ" role="doc-biblioref">161</a>]</span> identified 198 recurrent mutations in a dataset of 7,666 curated sequences.
This pattern of convergent evolution at some sites could indicate that certain mutations confer an adaptive advantage.
While it is evident that SARS-CoV-2 exhibits moderate potential for ongoing and future evolution, the relationship between mutations and pathogenicity is not yet known.
Additional data is needed in order to understand patterns of evolutionary change and whether they are likely to affect virulence.</p>
<h1 data-number="3" id="transmission-and-susceptibility"><span class="header-section-number">3</span> Transmission and Susceptibility</h1>
<h2 data-number="3.1" id="mechanism-of-transmission"><span class="header-section-number">3.1</span> Mechanism of Transmission</h2>
<p>Person-to-person viral transmission of a virus can occur through several possible mechanisms.
The primary mechanisms associated with respiratory viruses are contact, droplet, and aerosol transmission <span class="citation" data-cites="CJbohYmK">[<a href="#ref-CJbohYmK" role="doc-biblioref">164</a>]</span>.
Contact transmission can occur through either direct contact with a contagious person or indirect contact with active viral particles on a contaminated surface <span class="citation" data-cites="KMb6knfE">[<a href="#ref-KMb6knfE" role="doc-biblioref">165</a>]</span>.
This latter mode of transmission is also called fomite transmission <span class="citation" data-cites="1FLLN4PSl">[<a href="#ref-1FLLN4PSl" role="doc-biblioref">166</a>]</span>.
Viral particles can enter the body if they then come in contact with the oral, nasal, eye, or other mucus membranes <span class="citation" data-cites="KMb6knfE">[<a href="#ref-KMb6knfE" role="doc-biblioref">165</a>]</span>.
Droplet transmission occurs when a contagious individual sneezes, coughs, or exhales and produces respiratory droplets that can contain a large number of viral particles <span class="citation" data-cites="KMb6knfE">[<a href="#ref-KMb6knfE" role="doc-biblioref">165</a>]</span>.
Contact with these droplets can occur either through direct exposure to the droplets, such as breathing in droplets produced by a sneeze, or exposure to particles that have settled on a surface <span class="citation" data-cites="KMb6knfE">[<a href="#ref-KMb6knfE" role="doc-biblioref">165</a>]</span>.
Aerosol transmission refers to much smaller particles (less than 5 micrometers) that are also produced by sneezing, coughing, or exhaling <span class="citation" data-cites="CJbohYmK KMb6knfE">[<a href="#ref-CJbohYmK" role="doc-biblioref">164</a>,<a href="#ref-KMb6knfE" role="doc-biblioref">165</a>]</span>.
The small size of these particles allows them to remain suspended over a longer period of time and potentially to be moved by air currents <span class="citation" data-cites="KMb6knfE">[<a href="#ref-KMb6knfE" role="doc-biblioref">165</a>]</span>.
Additionally, viral particles deposited on surfaces via large respiratory droplets can also later be aerosolized <span class="citation" data-cites="KMb6knfE">[<a href="#ref-KMb6knfE" role="doc-biblioref">165</a>]</span>.
Droplet and/or contact transmission are both well-accepted modes of transmission for many viruses associated with common human illness, including influenza and rhinovirus <span class="citation" data-cites="KMb6knfE">[<a href="#ref-KMb6knfE" role="doc-biblioref">165</a>]</span>.
The extent to which aerosol transmission contributes to the spread of respiratory viruses is less clear.
In influenza A, for example, viral particles can be detected in aerosols produced by infected individuals, but the extent to which these particles drive the spread of influenza A infection remains under debate <span class="citation" data-cites="KMb6knfE CJbohYmK AC4okoVf Hct9jRcb i6tTpqwA">[<a href="#ref-CJbohYmK" role="doc-biblioref">164</a>,<a href="#ref-KMb6knfE" role="doc-biblioref">165</a>,<a href="#ref-AC4okoVf" role="doc-biblioref">167</a>,<a href="#ref-Hct9jRcb" role="doc-biblioref">168</a>,<a href="#ref-i6tTpqwA" role="doc-biblioref">169</a>]</span>.
Regardless of its role in the spread of influenza A, however, aerosol transmission likely played a role in outbreaks such as the 1918 Spanish Influenza (H1N1) and 2009 “swine flu” (pH1N1) <span class="citation" data-cites="i6tTpqwA">[<a href="#ref-i6tTpqwA" role="doc-biblioref">169</a>]</span>.
Contact, droplet, and aerosol transmission are therefore all worth evaluating when considering possible modes of transmission for a respiratory virus like SARS-CoV-2.</p>
<h3 data-number="3.1.1" id="transmission-of-hcov"><span class="header-section-number">3.1.1</span> Transmission of HCoV</h3>
<p>All three of these mechanisms have been identified as contributors to the transmission of HCoV <span class="citation" data-cites="KMb6knfE">[<a href="#ref-KMb6knfE" role="doc-biblioref">165</a>]</span>, including the highly pathogenic coronaviruses SARS-CoV and MERS-CoV <span class="citation" data-cites="rjVw7V94 G5NJrE75">[<a href="#ref-G5NJrE75" role="doc-biblioref">29</a>,<a href="#ref-rjVw7V94" role="doc-biblioref">170</a>]</span>.
Transmission of SARS-CoV is thought to proceed primarily through droplet transmission, but aerosol transmission is also considered possible <span class="citation" data-cites="KMb6knfE">[<a href="#ref-KMb6knfE" role="doc-biblioref">165</a>]</span>, and fomite transmission may have also played an important role in some outbreaks <span class="citation" data-cites="yR57NFIB">[<a href="#ref-yR57NFIB" role="doc-biblioref">171</a>]</span>.
Similarly, the primary mechanism of MERS transmission is thought to be droplets because inter-individual transmission appears to be associated with close interpersonal contact (e.g., household or healthcare settings), but aerosolized particles of the MERS virus have been reported to persist much more robustly than influenza A under a range of environmental conditions <span class="citation" data-cites="s3oVNbGV 1FjDLPCye">[<a href="#ref-s3oVNbGV" role="doc-biblioref">172</a>,<a href="#ref-1FjDLPCye" role="doc-biblioref">173</a>]</span>.
While droplet-based and contact transmission were initially considered to be the primary modes by which SARS-CoV-2 spread <span class="citation" data-cites="vP6yTZ0y">[<a href="#ref-vP6yTZ0y" role="doc-biblioref">174</a>]</span>, as additional information has emerged, the possibility of aerosol transmission has also been raised <span class="citation" data-cites="16SDeiudC yXKcviw8 9ouDX5IN">[<a href="#ref-16SDeiudC" role="doc-biblioref">175</a>,<a href="#ref-yXKcviw8" role="doc-biblioref">176</a>,<a href="#ref-9ouDX5IN" role="doc-biblioref">177</a>]</span>.
For example, the detection of SARS-CoV-2 viral particles in air samples taken from hospitals treating COVID-19 patients led to the concern that the virus could be spreading via aerosols <span class="citation" data-cites="bQLMgMGC">[<a href="#ref-bQLMgMGC" role="doc-biblioref">178</a>]</span>.
The stability of the virus both in aerosols and on a variety of surfaces appeared similar to that of SARS-CoV <span class="citation" data-cites="yXKcviw8">[<a href="#ref-yXKcviw8" role="doc-biblioref">176</a>]</span>, and fomite transmission could also play a role in transmission (e.g., <span class="citation" data-cites="1HOZHff6">[<a href="#ref-1HOZHff6" role="doc-biblioref">179</a>]</span>).
However, while the possibility of aerosol transmission seems plausible, the evidence suggests that droplet and contact transmission are the dominant mechanisms driving the spread of the virus <span class="citation" data-cites="82XnTbtX">[<a href="#ref-82XnTbtX" role="doc-biblioref">180</a>]</span>, and the risk of fomite transmission under real-world conditions is likely to be substantially lower than the conditions used for experimental analyses <span class="citation" data-cites="cw5j7x80">[<a href="#ref-cw5j7x80" role="doc-biblioref">181</a>]</span>.
These mechanisms may differ in their relevance to different types of transmission events, such as transmission within households, nosocomial transmissions, and transmission in indoor versus outdoor spaces.</p>
<h3 data-number="3.1.2" id="symptoms-and-viral-spread"><span class="header-section-number">3.1.2</span> Symptoms and Viral Spread</h3>
<p>Other aspects of pathogenesis are also important to understanding how the virus spreads, especially the relationship between symptoms, viral shedding, and contagiousness.
Symptoms associated with reported cases of COVID-19 range from mild to severe <span class="citation" data-cites="gHWlMufv">[<a href="#ref-gHWlMufv" role="doc-biblioref">45</a>]</span>, but some individuals who contract COVID-19 remain asymptomatic throughout the duration of the illness <span class="citation" data-cites="wxVni9Hz">[<a href="#ref-wxVni9Hz" role="doc-biblioref">182</a>]</span>.
The incubation period, or time period between exposure and the onset of symptoms, has been estimated at five to eight days (4.91 and 7.54 in two different cities) <span class="citation" data-cites="NIxttl2v QFVSrboR">[<a href="#ref-NIxttl2v" role="doc-biblioref">183</a>,<a href="#ref-QFVSrboR" role="doc-biblioref">184</a>]</span>, and estimates suggest that viral shedding may begin long before the onset of symptoms (12.3 days with a 95% CI of 5.9 - 17.0) and peak around the onset of symptoms <span class="citation" data-cites="1654TaAK4">[<a href="#ref-1654TaAK4" role="doc-biblioref">185</a>]</span>.
As these trends became apparent, concerns arose due to the potential for individuals who did not or did not yet show symptoms to transmit the virus <span class="citation" data-cites="Oyy1OUTT">[<a href="#ref-Oyy1OUTT" role="doc-biblioref">186</a>]</span>.
Recovered individuals may also be able to transmit the virus after their symptoms cease.
Estimates of the communicable period based on twenty-four individuals who tested positive for SARS-CoV-2 prior to or without developing symptoms estimated that individuals may be contagious for one to twenty-one days, but they note that this estimate may be low <span class="citation" data-cites="wxVni9Hz">[<a href="#ref-wxVni9Hz" role="doc-biblioref">182</a>]</span>.
Initially, viral nucleic acids were reported to remain at observable levels in the respiratory specimens of recovering hospitalized COVID-19 patients for a median of 20 days and with a maximum observed duration through 37 days, when data collection for the study ceased <span class="citation" data-cites="10THxyeCg">[<a href="#ref-10THxyeCg" role="doc-biblioref">36</a>]</span>.
As more estimates of the duration of viral shedding are released, they are beginning to converge around approximately three weeks from first positive PCR test and/or onset of symptoms (which, if present, are usually identified within three days of the initial PCR test).
For example, viral shedding was reported for up to 28 days following symptom onset by a second study <span class="citation" data-cites="Xp5HT2SF">[<a href="#ref-Xp5HT2SF" role="doc-biblioref">187</a>]</span> and for one to 24 days from first positive PCR test with a median of 12 days <span class="citation" data-cites="azgJqujy">[<a href="#ref-azgJqujy" role="doc-biblioref">78</a>]</span>.
On the other hand, almost 70% of patients were reported to still have symptoms at the time that viral shedding ceased, although all symptoms reduced in prevalence between onset and cessation of viral shedding (CVS) <span class="citation" data-cites="14fWuiUKS">[<a href="#ref-14fWuiUKS" role="doc-biblioref">188</a>]</span>.
They also reported that the median time that elapsed between the onset of symptoms and cessation of viral RNA shedding (CVS) was 23 days and between first positive PCR test and CVS was 17 days <span class="citation" data-cites="14fWuiUKS">[<a href="#ref-14fWuiUKS" role="doc-biblioref">188</a>]</span>.
The fact that this study reported symptom onset to predate the first positive PCR test by an average of three days, however, suggests that there may be some methodological differences between it and related studies.
Furthermore, an analysis of residents of a nursing home with a known SARS-CoV-2 case measured similar viral load in residents who were asymptomatic regardless of whether they later developed symptoms, and the load in the asymptomatic residents was comparable to that of residents who displayed either typical of atypical symptoms <span class="citation" data-cites="ac2kt3rh">[<a href="#ref-ac2kt3rh" role="doc-biblioref">189</a>]</span>.
Taken together, these results suggest that the presence or absence of symptoms are not reliable predictors of viral shedding or of SARS-CoV-2 status (e.g, <span class="citation" data-cites="lVDiHYp">[<a href="#ref-lVDiHYp" role="doc-biblioref">190</a>]</span>.
However, viral shedding is not necessarily indicative of contagiousness.
The risk of spreading the infection was low after ten days from the onset of symptoms, as viral load in sputum was found to be unlikely to pose a significant risk based on their efforts to culture samples <em>in vitro</em> <span class="citation" data-cites="Xp5HT2SF">[<a href="#ref-Xp5HT2SF" role="doc-biblioref">187</a>]</span>.</p>
<p>The extent to which asymptomatic (or presymptomatic) individuals are able to transmit SARS-CoV-2 has been a question of both scientific and community interest.
Early reports (February and March 2020) described transmission from presymptomatic SARS-CoV-2-positive individuals to close family contacts <span class="citation" data-cites="E6Qv6YQ0 18Orbxz8t">[<a href="#ref-E6Qv6YQ0" role="doc-biblioref">191</a>,<a href="#ref-18Orbxz8t" role="doc-biblioref">192</a>]</span>.
One of these reports <span class="citation" data-cites="18Orbxz8t">[<a href="#ref-18Orbxz8t" role="doc-biblioref">192</a>]</span> also included a description of an individual who tested positive for SARS-CoV-2 but never developed symptoms.
Later analyses also sought to estimate the proportion of infections that could be traced back to a presymptomatic or asymptomatic individual (e.g., <span class="citation" data-cites="4M3GM1sg">[<a href="#ref-4M3GM1sg" role="doc-biblioref">193</a>]</span>).
Estimates of the proportion of individuals with asymptomatic infections have varied widely.
The proportion of asymptomatic individuals on board the Diamond Princess cruise ship, which was the site of an early COVID-19 outbreak, was estimated at 17.9% <span class="citation" data-cites="6K5lsF5i">[<a href="#ref-6K5lsF5i" role="doc-biblioref">194</a>]</span>.
In contrast, a model using the prevalence of antibodies among residents of Wuhan, China estimated a much higher of asymptomatic cases, at approximately 7 in 8, or 87.5% <span class="citation" data-cites="sk1NbA7K">[<a href="#ref-sk1NbA7K" role="doc-biblioref">195</a>]</span>.
An estimate of the population of care homes in London found that, among the residents (median age 85), the rate of asymptomatic infection was 43.8%, and among the caretakers (median age 47), the rate was 49.1% <span class="citation" data-cites="kIumgXPI">[<a href="#ref-kIumgXPI" role="doc-biblioref">196</a>]</span>.
The duration of viral shedding may also be longer in individuals with asymptomatic cases of COVID-19 compared to those who do show symptoms <span class="citation" data-cites="10OGkFiGJ">[<a href="#ref-10OGkFiGJ" role="doc-biblioref">197</a>]</span>.
As a result, the potential for individuals who do not know they have COVID-19 to spread the virus raises significant concerns.
In Singapore and Tianjin, two cities studied to estimate incubation period, an estimated 40-50% and 60-80% of cases, respectively, were estimated to be caused by contact with an asymptomatic individuals <span class="citation" data-cites="NIxttl2v">[<a href="#ref-NIxttl2v" role="doc-biblioref">183</a>]</span>.
An analysis of viral spread in the Italian town of Vo’, which was the site of an early COVID-19 outbreak, revealed that 42.5% of cases were asymptomatic and that the rate was similar across age groups <span class="citation" data-cites="dMjSbAQV">[<a href="#ref-dMjSbAQV" role="doc-biblioref">198</a>]</span>.
They argued that the town’s lockdown was imperative for controlling the spread of COVID-19 because it isolated the asymptomatic individuals.
While more models are likely to emerge to better explore the effect of asymptomatic individuals on SARS-CoV-2 transmission, these results suggest that strategies for identifying and containing asymptomatic but contagious individuals are important for managing community spread.</p>
<h3 data-number="3.1.3" id="estimating-the-fatality-rate"><span class="header-section-number">3.1.3</span> Estimating the Fatality Rate</h3>
<p>Estimating the occurrence of asymptomatic and mild COVID-19 cases is important to identifying the mortality rate associated with COVID-19.
The mortality rate of greatest interest would be the total number of fatalities as a fraction of the total number of people infected.
One metric reported is often case fatality rate (CFR), which simply compares the number of COVID-19 related deaths to the number of confirmed or suspected cases.
However, in locations without universal testing protocols, it is impossible to identify all exposed or all infected individuals because so many asymptomatic or mild cases go undetected.
Therefore, a more informative metric is the infection fatality rate (IFR), which compares the known deaths to the estimated number of cases.
Meta-analyses have produced estimates of global IFR ranging from as low as 0.1% to as high as 1.04% <span class="citation" data-cites="BENQIwOM AavOV1He 12CiweatZ BENQIwOM">[<a href="#ref-BENQIwOM" role="doc-biblioref">199</a>,<a href="#ref-BENQIwOM" role="doc-biblioref">199</a>,<a href="#ref-AavOV1He" role="doc-biblioref">200</a>,<a href="#ref-12CiweatZ" role="doc-biblioref">201</a>]</span>, and this estimate was also supported by a repeated cross-sectional serosurvey conducted in New York City that revealed an estimated IFR of 0.97% <span class="citation" data-cites="tIjAygts">[<a href="#ref-tIjAygts" role="doc-biblioref">202</a>]</span>.
All of these estimates note that IFR varies widely around the world.
Estimates of infection rates are becoming more feasible as more data becomes available for modeling and will be bolstered as serological testing becomes more common and available.</p>
<h2 data-number="3.2" id="dynamics-of-transmission"><span class="header-section-number">3.2</span> Dynamics of Transmission</h2>
<p>Disease spread dynamics can be estimated using R<sub>0</sub>, the basic reproduction number, and R<sub>t</sub>, the effective reproduction number.
Thus, accurate estimates are crucial to understanding the dynamics of infection and to predicting the effects of different interventions.
R<sub>0</sub> and the timescale of infection (measured by the infectious period and the exposed period) govern population-level epidemic dynamics, with R<sub>0</sub> being one of the most important epidemiological parameters <span class="citation" data-cites="14l4fvU71">[<a href="#ref-14l4fvU71" role="doc-biblioref">203</a>]</span>.
R<sub>0</sub> is the average number of new (secondary) infections caused by one infected person, assuming a wholly susceptible population <span class="citation" data-cites="16CD4voW1">[<a href="#ref-16CD4voW1" role="doc-biblioref">204</a>]</span>.
A simple mechanistic model used to describe infectious disease dynamics is a susceptible-infected-recovered (SIR) compartmental model.
In this formulation individuals move through three states: susceptible, infected, and recovered; two parameters, <span class="math inline">\(\gamma\)</span> and <span class="math inline">\(\beta\)</span>, specify the rate at which the infectious recover, and the infection transmission rate, respectively.
In this simple formulation, R<sub>0</sub> is estimated as the ratio of <span class="math inline">\(\beta\)</span> and <span class="math inline">\(\gamma\)</span>.<span class="citation" data-cites="YubbhU7G 14l4fvU71">[<a href="#ref-14l4fvU71" role="doc-biblioref">203</a>,<a href="#ref-YubbhU7G" role="doc-biblioref">205</a>]</span>.
A pathogen can invade a susceptible population only if R<sub>0</sub> &gt; 1 <span class="citation" data-cites="14l4fvU71 JRaVgcNd">[<a href="#ref-14l4fvU71" role="doc-biblioref">203</a>,<a href="#ref-JRaVgcNd" role="doc-biblioref">206</a>]</span>.
The spread of an infectious disease at a particular time t can be quantified by R<sub>t</sub>, the effective reproduction number, which assumes that part of the population has already recovered (and thus gained immunity to reinfection) or that mitigating interventions were put into place.
For example, if only a fraction S<sub>t</sub> of the population is still susceptible, R<sub>t</sub> = S<sub>t</sub> x R<sub>0</sub>.
When R<sub>t</sub> is greater than 1, an epidemic grows (i.e., the proportion of the population that is infectious increases); when R<sub>t</sub> is less than 1, the proportion of the population that is infectious decreases.
R<sub>0</sub> and R<sub>t</sub> can be estimated directly from epidemiological data or inferred using mathematical modeling.
Modeling approaches are typically based upon a classic epidemiological model structure: the SIR model and its extensions <span class="citation" data-cites="1Aozf7KBW SPInPew0">[<a href="#ref-1Aozf7KBW" role="doc-biblioref">207</a>,<a href="#ref-SPInPew0" role="doc-biblioref">208</a>]</span>.
In the context of SARS-CoV-2, more complex modified susceptible-exposed-infectious-recovered (SEIR) models are commonly used. <!--TO DO: [need citations]--></p>
<p>Estimates of R<sub>0</sub> for COVID-19 lie in the range R<sub>0</sub>=1.4-6.5 <span class="citation" data-cites="1E0r4uZy9 LHtuVmaq mHYmt0mv">[<a href="#ref-1E0r4uZy9" role="doc-biblioref">209</a>,<a href="#ref-LHtuVmaq" role="doc-biblioref">210</a>,<a href="#ref-mHYmt0mv" role="doc-biblioref">211</a>]</span>.
Variation in R<sub>0</sub> is expected between different populations, and the estimated values of R<sub>0</sub> discussed below are for specific populations in specific environments.
The different estimates of R<sub>0</sub> should not necessarily be interpreted as a range of estimates of the same underlying parameter. <!--I am not quite sure about this next paragraph. Most sophisticated models now incorporate "data" in their SEIR formulation, so I'm not sure it's accurate to draw the distinction.-->
In one study of international cases, the predicted value was R<sub>0</sub>=1.7 <span class="citation" data-cites="vITui6ac">[<a href="#ref-vITui6ac" role="doc-biblioref">212</a>]</span>.
In China (both Hubei province and nationwide), the value was predicted to lie in the range R<sub>0</sub>=2.0-3.6 <span class="citation" data-cites="nxM0rP5R WLc2UMgQ 1E0r4uZy9">[<a href="#ref-1E0r4uZy9" role="doc-biblioref">209</a>,<a href="#ref-nxM0rP5R" role="doc-biblioref">213</a>,<a href="#ref-WLc2UMgQ" role="doc-biblioref">214</a>]</span>.
Another estimate based on a cruise ship where an outbreak occurred predicted R<sub>0</sub>=2.28 <span class="citation" data-cites="10bBqMHH7">[<a href="#ref-10bBqMHH7" role="doc-biblioref">215</a>]</span>.
SEIR model-derived estimates of R<sub>0</sub> range from 2.0 - 6.5 in China <span class="citation" data-cites="ITh0Anof 1BMU7sKbs Yj8Xh4Wz ZzrrVDoE">[<a href="#ref-ITh0Anof" role="doc-biblioref">216</a>,<a href="#ref-1BMU7sKbs" role="doc-biblioref">217</a>,<a href="#ref-Yj8Xh4Wz" role="doc-biblioref">218</a>,<a href="#ref-ZzrrVDoE" role="doc-biblioref">219</a>]</span> to R<sub>0</sub>=4.8 in France <span class="citation" data-cites="itj26agd">[<a href="#ref-itj26agd" role="doc-biblioref">220</a>]</span>.
Using the same model as for the French population, a study estimated R<sub>0</sub>=2.6 in South Korea <span class="citation" data-cites="itj26agd">[<a href="#ref-itj26agd" role="doc-biblioref">220</a>]</span>, which is consistent with other studies <span class="citation" data-cites="YfOcGRRa">[<a href="#ref-YfOcGRRa" role="doc-biblioref">221</a>]</span>.
From a meta-analysis of studies estimating R<sub>0</sub>, <span class="citation" data-cites="LHtuVmaq">[<a href="#ref-LHtuVmaq" role="doc-biblioref">210</a>]</span> predict the median as R<sub>0</sub>=2.79.</p>
<p>Inference of the effective reproduction number can provide insight into how populations respond to an infection and the effectiveness of interventions.
In China, R<sub>t</sub> was predicted to lie in the range 1.6-2.6 in January 2020, before travel restrictions <span class="citation" data-cites="tiRfUgvs">[<a href="#ref-tiRfUgvs" role="doc-biblioref">222</a>]</span>.
R<sub>t</sub> decreased from 2.35 one week before travel restrictions were imposed (January 23, 2020), to 1.05 one week after.
Using their model, the authors also estimated the probability of new outbreaks occurring: the probability of a single individual exporting the virus and causing a large outbreak is 17-25%, assuming MERS-like or SARS-like transmission, and the probability of a large outbreak occurring after ≥4 infections exist at a new location is greater than 50%.
An independent study came to similar conclusions, finding R<sub>t</sub>=2.38 in the two-week period before January 23 with a decrease to R<sub>t</sub> = 1.34 (using data from January 24 to February 3) or R<sub>t</sub>=0.98 (using data from January 24 to February 8) <span class="citation" data-cites="mHYmt0mv">[<a href="#ref-mHYmt0mv" role="doc-biblioref">211</a>]</span>.
In South Korea, R<sub>t</sub> was inferred for February through March 2020 in two cities, Daegu (the center of the outbreak) and Seoul <span class="citation" data-cites="YfOcGRRa">[<a href="#ref-YfOcGRRa" role="doc-biblioref">221</a>]</span>.
Metro data was also analyzed to estimate the effects of social distancing measures.
R<sub>t</sub> decreased in Daegu from around 3 to &lt;1 over the period that social distancing measures were introduced.
In Seoul, R<sub>t</sub> decreased slightly, but remained close to 1 (and larger than R<sub>t</sub> in Daegu).
These findings indicate that social distancing measures appeared to be effective in containing the infection in Daegu, but in Seoul, R<sub>t</sub> remained above 1, meaning secondary outbreaks remained possible.
The study also shows the importance of region-specific analysis: the large decline in case load nationwide was mainly due to the Daegu region and could mask persistence of the epidemic in other regions, such as Seoul and Gyeonggi-do.
In Iran, estimates of R<sub>t</sub> declined from 4.86 in the first week to 2.1 by the fourth week after the first cases were reported <span class="citation" data-cites="TH4ymqvJ">[<a href="#ref-TH4ymqvJ" role="doc-biblioref">223</a>]</span>.
In Europe, analysis of 11 countries inferred the dynamics of R<sub>t</sub> over a time range from the beginning of the outbreak until March 28, 2020, by which point most countries had implemented major interventions (such as school closures, public gathering bans, and stay-at-home orders) <span class="citation" data-cites="Frya4XA4">[<a href="#ref-Frya4XA4" role="doc-biblioref">224</a>]</span>.
Across all countries, the mean R<sub>t</sub> before interventions began was estimated as 3.87; R<sub>t</sub> varied considerably, from below 3 in Norway to above 4.5 in Spain.
After interventions, R<sub>t</sub> decreased by an average of 64% across all countries, with mean R<sub>t</sub>=1.43.
The lowest predicted value was 0.97 for Norway and the highest was 2.64 for Sweden, which may be in part because Sweden did not implement social distancing measures on the same scale as other countries.
The study concludes that while large changes in R<sub>t</sub> are observed, it is too early to tell whether the interventions put into place are sufficient to decrease R<sub>t</sub> below 1.</p>
<p>More generally, population-level epidemic dynamics can be both observed and modelled <span class="citation" data-cites="YubbhU7G">[<a href="#ref-YubbhU7G" role="doc-biblioref">205</a>]</span>.
Data and empirically determined biological mechanisms inform models, while models can be used to try to understand data and systems of interest or to make predictions about possible future dynamics, such as the estimation of capacity needs <span class="citation" data-cites="NCcssjLV">[<a href="#ref-NCcssjLV" role="doc-biblioref">225</a>]</span> or the comparison of predicted outcomes among prevention and control strategies <span class="citation" data-cites="11CWFYFDL J4RCpntH">[<a href="#ref-11CWFYFDL" role="doc-biblioref">226</a>,<a href="#ref-J4RCpntH" role="doc-biblioref">227</a>]</span>.
Many current efforts to model R<sub>t</sub> have led to tools that assist the visualization of estimates in real time or over recent intervals <span class="citation" data-cites="WA1R5R95 17CQnstjh">[<a href="#ref-WA1R5R95" role="doc-biblioref">228</a>,<a href="#ref-17CQnstjh" role="doc-biblioref">229</a>]</span>.
While these may be valuable resources, it is important to note that the estimates arise from models containing many assumptions and are dependent on the quality of the data they use, which varies widely by region.</p>
<h2 data-number="3.3" id="molecular-signatures-and-transmission"><span class="header-section-number">3.3</span> Molecular Signatures and Transmission</h2>
<p>Genetic variation in SARS-CoV-2 has been used to elucidate patterns over time and space.
Mutations observed in individual SARS-CoV-2 genome sequences can be used to trace transmission patterns and have provided insights during outbreak investigations <span class="citation" data-cites="p9LUiyCN UUAeVUaR 2w7lNKxQ">[<a href="#ref-UUAeVUaR" role="doc-biblioref">15</a>,<a href="#ref-p9LUiyCN" role="doc-biblioref">160</a>,<a href="#ref-2w7lNKxQ" role="doc-biblioref">161</a>]</span>.
Similar mutations observed in several patients may indicate that the patients belong to the same transmission group.
The tracking of SARS-CoV-2 mutations is recognized as an essential tool for controlling outbreaks that may facilitate tracing the paths of SARS-CoV-2’s spread <span class="citation" data-cites="jVbH3kJR">[<a href="#ref-jVbH3kJR" role="doc-biblioref">230</a>]</span>.
Several studies used phylogenetic analysis to determine the source of local COVID-19 outbreaks in Connecticut (USA), <span class="citation" data-cites="W5Z7Ztlg">[<a href="#ref-W5Z7Ztlg" role="doc-biblioref">231</a>]</span>, the New York City area (USA) <span class="citation" data-cites="FVmbE2oW">[<a href="#ref-FVmbE2oW" role="doc-biblioref">232</a>]</span>, and Iceland <span class="citation" data-cites="WJrnUyZf">[<a href="#ref-WJrnUyZf" role="doc-biblioref">233</a>]</span>.
There is an ongoing effort to collect SARS-CoV-2 genomes throughout the COVID-19 outbreak, and as of August 7, 2020, more than 78,000 genome sequences have been collected from patients. <!-- TODO: consider auto-updating this -->
The sequencing data can be found at GISAID <span class="citation" data-cites="qIXPib3m">[<a href="#ref-qIXPib3m" role="doc-biblioref">234</a>]</span>, NCBI <span class="citation" data-cites="cyzrC7qd">[<a href="#ref-cyzrC7qd" role="doc-biblioref">235</a>]</span>, and COVID-19 data portal <span class="citation" data-cites="BjPoOYSA">[<a href="#ref-BjPoOYSA" role="doc-biblioref">236</a>]</span>.</p>
<h2 data-number="3.4" id="factors-influencing-susceptibility"><span class="header-section-number">3.4</span> Factors Influencing Susceptibility</h2>
<p>As COVID-19 has spread into communities around the globe, it has become clear that the risks associated with this disease are not equally shared by all individuals or all communities.
Significant disparities in outcomes have led to interest in the demographic, biomedical, and social factors that influence COVID-19 severity.
Untangling the factors influencing COVID-19 susceptibility is a complex undertaking.
Among patients who are admitted to the hospital, outcomes have generally been poor, with rates of admission to the intensive care unit (ICU) upwards of 15% in both Wuhan, China and Italy <span class="citation" data-cites="15IDA5kX6 11xfoqpEl qlJ8hCUs">[<a href="#ref-15IDA5kX6" role="doc-biblioref">35</a>,<a href="#ref-11xfoqpEl" role="doc-biblioref">237</a>,<a href="#ref-qlJ8hCUs" role="doc-biblioref">238</a>]</span>.
However, hospitalization rates vary by location <span class="citation" data-cites="xZBEFXiG">[<a href="#ref-xZBEFXiG" role="doc-biblioref">239</a>]</span>.
This variation may be influenced by demographic (e.g., average age in the area), medical (e.g., the prevalence of comorbid conditions such as diabetes), and social (e.g., income or healthcare availability) factors that vary geographically.
Additionally, some of the same factors may influence an individual’s probability of exposure to SARS-CoV-2, their risk of developing a more serious case of COVID-19 that would require hospitalization, and their access to medical support.
As a result, quantifying or comparing susceptibility among individuals, communities, or other groups requires consideration of a number of complex phenomena that intersect across many disciplines of research.
In this section, the term “risk factors” is used to refer to variables that are statistically associated with more severe COVID-19 outcomes.
Some are intrinsic characteristics that have been observed to carry an association with variation in outcomes, whereas others may be more functionally linked to the pathophysiology of COVID-19.</p>
<h3 data-number="3.4.1" id="patient-traits-associated-with-increased-risk"><span class="header-section-number">3.4.1</span> Patient Traits Associated with Increased Risk</h3>
<p>Two traits that have been consistently associated with more severe COVID-19 outcomes are male sex and advanced age.
In the United States, males and older individuals diagnosed with COVID-19 were found to be more likely to require hospitalization <span class="citation" data-cites="1D6lVXWbJ Zk5CfYeW">[<a href="#ref-1D6lVXWbJ" role="doc-biblioref">240</a>,<a href="#ref-Zk5CfYeW" role="doc-biblioref">241</a>]</span>.
A retrospective study of hospitalized Chinese patients <span class="citation" data-cites="10THxyeCg">[<a href="#ref-10THxyeCg" role="doc-biblioref">36</a>]</span> found that a higher probability of mortality was associated with older age, and world-wide, population age structure has been found to be an important variable for explaining differences in outbreak severity <span class="citation" data-cites="BQ4pZGxJ">[<a href="#ref-BQ4pZGxJ" role="doc-biblioref">242</a>]</span>.
The CFR for adults over 80 has been estimated upwards of 14% or even 20% <span class="citation" data-cites="1DnayDKc4">[<a href="#ref-1DnayDKc4" role="doc-biblioref">243</a>]</span>.
Male sex has also been identified as a risk factor for severe COVID-19 outcomes, including death <span class="citation" data-cites="gObtnWSD DqBeNrR5 ConBOYBB">[<a href="#ref-gObtnWSD" role="doc-biblioref">244</a>,<a href="#ref-DqBeNrR5" role="doc-biblioref">245</a>,<a href="#ref-ConBOYBB" role="doc-biblioref">246</a>]</span>.
Early reports from China and Europe indicated that even though the case rates were similar across males and females, males were at elevated risk for hospital admission, ICU admission, and death <span class="citation" data-cites="DqBeNrR5">[<a href="#ref-DqBeNrR5" role="doc-biblioref">245</a>]</span>, although data from some US states indicates more cases among females, potentially due to gender representation in care-taking professions <span class="citation" data-cites="fH3qecwy">[<a href="#ref-fH3qecwy" role="doc-biblioref">247</a>]</span>.
In older age groups (e.g., age 60 and older), comparable absolute numbers of male and female cases actually suggests a higher rate of occurrence in males, due to increased skew in the sex ratio <span class="citation" data-cites="DqBeNrR5">[<a href="#ref-DqBeNrR5" role="doc-biblioref">245</a>]</span>.
Current estimates based on worldwide data suggest that, compared to females, males may be 30% more likely to be hospitalized, 80% more likely to be admitted to the ICU, and 40% more likely to die as a result of COVID-19 <span class="citation" data-cites="ConBOYBB">[<a href="#ref-ConBOYBB" role="doc-biblioref">246</a>]</span>.
There also may be a compounding effect of advanced age and male sex, with differences time to recovery worst for males over 60 years old relative to female members of their age cohort <span class="citation" data-cites="RcYBFjm1">[<a href="#ref-RcYBFjm1" role="doc-biblioref">248</a>]</span>.</p>
<p>Both of these risk factors can be approached through the lens of biology.
The biological basis for greater susceptibility with age is likely linked to the prevalence of extenuating health conditions such as heart failure or diabetes <span class="citation" data-cites="1DnayDKc4">[<a href="#ref-1DnayDKc4" role="doc-biblioref">243</a>]</span>.
Several hypotheses have been proposed to account for differences in severity between males and females.
For example, some evidence suggests that female sex hormones may be protective <span class="citation" data-cites="DqBeNrR5 fH3qecwy">[<a href="#ref-DqBeNrR5" role="doc-biblioref">245</a>,<a href="#ref-fH3qecwy" role="doc-biblioref">247</a>]</span>.
ACE2 expression in the kidneys of male mice was observed to be twice as high as that of females, and a regulatory effect of estradiol on ACE2 expression was demonstrated by removing the gonads and then supplementing with estradiol <span class="citation" data-cites="OJvjTsWz fH3qecwy">[<a href="#ref-fH3qecwy" role="doc-biblioref">247</a>,<a href="#ref-OJvjTsWz" role="doc-biblioref">249</a>]</span>.
Other work in mice has shown an inverse association between mortality due to SARS-CoV and estradiol, suggesting a protective role for the sex hormone <span class="citation" data-cites="fH3qecwy">[<a href="#ref-fH3qecwy" role="doc-biblioref">247</a>]</span>.
Similarly, evidence suggests that similar patterns might be found in other tissues: preliminary analysis suggested that male patients with aortic valve stenosis may show higher levels of ACE2 expression in the myocardium than females, although this pattern was not found in controls, and research has indicated that females may respond to lower doses of heart medications that act on a pathway shared with ACE2 (Renin angiotensin aldosterone system or RAAS) <span class="citation" data-cites="DqBeNrR5">[<a href="#ref-DqBeNrR5" role="doc-biblioref">245</a>]</span>.
Additionally, several components of the immune response, including the inflammatory response, may differ in intensity and timing between males and females <span class="citation" data-cites="fH3qecwy OJvjTsWz">[<a href="#ref-fH3qecwy" role="doc-biblioref">247</a>,<a href="#ref-OJvjTsWz" role="doc-biblioref">249</a>]</span>.
This hypothesis is supported by some preliminary evidence showing that female patients who recovered from severe COVID-19 had higher antibody titers than males <span class="citation" data-cites="fH3qecwy">[<a href="#ref-fH3qecwy" role="doc-biblioref">247</a>]</span>.
Sex steroids can also bind to immune cell receptors to influence cytokine production <span class="citation" data-cites="DqBeNrR5">[<a href="#ref-DqBeNrR5" role="doc-biblioref">245</a>]</span>.
Additionally, social factors may influence risks related to both age and sex: for example, older adults are more likely to live in care facilities, which have been a source for a large number of outbreaks <span class="citation" data-cites="ujWsdPdH">[<a href="#ref-ujWsdPdH" role="doc-biblioref">250</a>]</span>, and gender roles may also influence exposure and/or susceptibility due to differences in care-taking and/or risky behaviors (e.g., caring for elder relatives and smoking, respectively) <span class="citation" data-cites="DqBeNrR5">[<a href="#ref-DqBeNrR5" role="doc-biblioref">245</a>]</span> among men and women (however, it should be noted that both transgender men and women are suspected to be at heightened risk <span class="citation" data-cites="agpMLmxt">[<a href="#ref-agpMLmxt" role="doc-biblioref">251</a>]</span>.)</p>
<h3 data-number="3.4.2" id="comorbid-health-conditions"><span class="header-section-number">3.4.2</span> Comorbid Health Conditions</h3>
<p>A number of pre-existing or comorbid conditions have repeatedly been identified as risk factors for more severe COVID-19 outcomes.
Several underlying health conditions were identified at high prevalence among hospitalized patients, including obesity, diabetes, hypertension, lung disease, and cardiovascular disease <span class="citation" data-cites="xZBEFXiG">[<a href="#ref-xZBEFXiG" role="doc-biblioref">239</a>]</span>.
Higher Sequential Organ Failure Assessment (SOFA) scores have been associated with a higher probability of mortality <span class="citation" data-cites="10THxyeCg">[<a href="#ref-10THxyeCg" role="doc-biblioref">36</a>]</span>, and comorbid conditions such as cardiovascular and lung disease as well as obesity were also associated with an increased risk of hospitalization and death, even when correcting for age and sex <span class="citation" data-cites="gObtnWSD">[<a href="#ref-gObtnWSD" role="doc-biblioref">244</a>]</span>.
Diabetes may increase the risk of lengthy hospitalization <span class="citation" data-cites="9NfXSHjH">[<a href="#ref-9NfXSHjH" role="doc-biblioref">252</a>]</span> or of death <span class="citation" data-cites="9NfXSHjH 13BpwZaaq">[<a href="#ref-9NfXSHjH" role="doc-biblioref">252</a>,<a href="#ref-13BpwZaaq" role="doc-biblioref">253</a>]</span>.
<span class="citation" data-cites="UkCdtLwF">[<a href="#ref-UkCdtLwF" role="doc-biblioref">254</a>]</span> and <span class="citation" data-cites="OT5GEOej">[<a href="#ref-OT5GEOej" role="doc-biblioref">255</a>]</span> discuss possible ways in which COVID-19 and diabetes may interact.
Obesity also appears to be associated with higher risk of severe outcomes from SARS-CoV-2 <span class="citation" data-cites="ZLmNMDrD 2Ahpn6R2">[<a href="#ref-ZLmNMDrD" role="doc-biblioref">256</a>,<a href="#ref-2Ahpn6R2" role="doc-biblioref">257</a>]</span>.
Obesity is considered an underlying risk factor for other health problems, and the mechanism for its contributions to COVID-19 hospitalization or mortality is not yet clear <span class="citation" data-cites="1AmBg80Un">[<a href="#ref-1AmBg80Un" role="doc-biblioref">258</a>]</span>.
Dementia and cancer were also associated with the risk of death in an analysis of a large number (more than 20,000) COVID-19 patients in the United Kingdom <span class="citation" data-cites="gObtnWSD">[<a href="#ref-gObtnWSD" role="doc-biblioref">244</a>]</span>.
It should be noted that comorbid conditions are inextricably tied to age, as conditions tend to be accumulated over time, but that the prevalence of individual comorbidities or of population health overall can vary regionally <span class="citation" data-cites="1GakM8xhV">[<a href="#ref-1GakM8xhV" role="doc-biblioref">259</a>]</span>.
Several comorbidities that are highly prevalent in older adults, such as COPD, hypertension, cardiovascular disease, and diabetes, have been associated with CFRs upwards of 8% compared to an estimate of 1.4% in people without comorbidities <span class="citation" data-cites="1DnayDKc4 H6dfv6Wv">[<a href="#ref-1DnayDKc4" role="doc-biblioref">243</a>,<a href="#ref-H6dfv6Wv" role="doc-biblioref">260</a>]</span>. <!-- will this url work for Manubot?-->
Therefore, both age and health are important considerations when predicting the impact of COVID-19 on a population <span class="citation" data-cites="1GakM8xhV">[<a href="#ref-1GakM8xhV" role="doc-biblioref">259</a>]</span>.
However, other associations may exist, such as patients with sepsis having higher SOFA scores – in fact, SOFA was developed for the assessment of organ failure in the context of sepsis, and the acronym originally stood for Sepsis-Related Organ Failure Assessment <span class="citation" data-cites="TuzwDCPt a0h975P6">[<a href="#ref-TuzwDCPt" role="doc-biblioref">261</a>,<a href="#ref-a0h975P6" role="doc-biblioref">262</a>]</span>.
Additionally, certain conditions are likely to be more prevalent under or exacerbated by social conditions, especially poverty, as is discussed further below.</p>
<h3 data-number="3.4.3" id="ancestry"><span class="header-section-number">3.4.3</span> Ancestry</h3>
<p>A number of studies have suggested associations between individuals’ racial and ethnic backgrounds and their COVID-19 risk.
In particular, Black Americans are consistently identified as carrying a higher burden of COVID-19 than white Americans <span class="citation" data-cites="1D6lVXWbJ Zk5CfYeW">[<a href="#ref-1D6lVXWbJ" role="doc-biblioref">240</a>,<a href="#ref-Zk5CfYeW" role="doc-biblioref">241</a>]</span>, with differences in the rates of kidney complications from COVID-19 particularly pronounced <span class="citation" data-cites="17hdoiBo9">[<a href="#ref-17hdoiBo9" role="doc-biblioref">90</a>]</span>.
Statistics from a number of cities indicate significant discrepancies between the proportion of COVID-19 cases and deaths in Black Americans relative to their representation in the general population <span class="citation" data-cites="jKKxa9Yy">[<a href="#ref-jKKxa9Yy" role="doc-biblioref">263</a>]</span>.
In addition to Black Americans, disproportionate harm and mortality from COVID-19 has also been noted in Latino/Hispanic Americans and in Native American and Alaskan Native communities, including the Navajo nation <span class="citation" data-cites="yK5bczRb u4MMV8aJ JAo2QxY 12WZObWRf Pp8GWnQm J9T8eR5H">[<a href="#ref-yK5bczRb" role="doc-biblioref">264</a>,<a href="#ref-u4MMV8aJ" role="doc-biblioref">265</a>,<a href="#ref-JAo2QxY" role="doc-biblioref">266</a>,<a href="#ref-12WZObWRf" role="doc-biblioref">267</a>,<a href="#ref-Pp8GWnQm" role="doc-biblioref">268</a>,<a href="#ref-J9T8eR5H" role="doc-biblioref">269</a>]</span>.
In Brazil indigenous communities likewise carry an increased burden of COVID-19 <span class="citation" data-cites="ODm1CaEq">[<a href="#ref-ODm1CaEq" role="doc-biblioref">270</a>]</span>.
In the United Kingdom, nonwhite ethnicity (principally Black or South Asian) was one of several factors found to be associated with a higher risk of death from COVID-19 <span class="citation" data-cites="17gnxmOnf">[<a href="#ref-17gnxmOnf" role="doc-biblioref">271</a>]</span>.</p>
<p>From a genetic standpoint, it is highly unlikely that ancestry itself predisposes individuals to contracting COVID-19 or to experiencing severe COVID-19 outcomes.
Examining human genetic diversity indicates variation over a geographic continuum, and that most human genetic variation is associated with the African continent <span class="citation" data-cites="pwbrxYv8">[<a href="#ref-pwbrxYv8" role="doc-biblioref">272</a>]</span>.
African-Americans are also a more genetically diverse group relative to European-Americans, with a large number of rare alleles and a much smaller fraction of common alleles identified in African-Americans <span class="citation" data-cites="CMAPEmTp">[<a href="#ref-CMAPEmTp" role="doc-biblioref">273</a>]</span>.
Therefore, the idea that African ancestry (at the continent level) might convey some sort of genetic risk for severe COVID-19 contrasts with what is known about worldwide human genetic diversity <span class="citation" data-cites="q4yhes98">[<a href="#ref-q4yhes98" role="doc-biblioref">274</a>]</span>.
The possibility for genetic variants that confer some risk or some protection remains possible, but has not been widely explored, especially at a global level.
Research in Beijing of a small number (n=80) hospitalized COVID-19 patients revealed an association between severe COVID-19 outcomes and homozygosity for an allele in the interferon-induced transmembrane protein 3 (IFITM3) gene, which was selected as a candidate because it was previously found to be associated with influenza outcomes in Chinese patients <span class="citation" data-cites="FbeY1L1s">[<a href="#ref-FbeY1L1s" role="doc-biblioref">275</a>]</span>.
Genetic factors may also play a role in the risk of respiratory failure for COVID-19 <span class="citation" data-cites="owevcpwb nn67UTjV I2mIU0wn">[<a href="#ref-owevcpwb" role="doc-biblioref">276</a>,<a href="#ref-nn67UTjV" role="doc-biblioref">277</a>,<a href="#ref-I2mIU0wn" role="doc-biblioref">278</a>]</span>.
However, genetic variants associated with outcomes within ancestral groups are far less surprising than genetic variants explaining outcomes between groups.
Alleles in <em>ACE2</em> and <em>TMPRSS2</em> have been identified that vary in frequency among ancestral groups <span class="citation" data-cites="KbDdcSgo">[<a href="#ref-KbDdcSgo" role="doc-biblioref">279</a>]</span>, but whether these variants are associated with COVID-19 susceptibility has not been explored.</p>
<p>Instead, examining patterns of COVID-19 susceptibility on a global scale that suggest that social factors are of primary importance in predicting mortality.
Reports from several sub-Saharan African countries have indicated that the effects of the COVID-19 pandemic have been less severe than expected based on the outbreaks in China and Italy.
In Kenya, for example, estimates of national prevalence based on testing blood donors for SARS-CoV-2 antibodies were consistent with 5% of Kenyan adults having recovered from COVID-19 <span class="citation" data-cites="4SqaP0Ub">[<a href="#ref-4SqaP0Ub" role="doc-biblioref">280</a>]</span>.
This high seroprevalence of antibodies lies in sharp contrast to the low number of COVID-19 fatalities in Kenya, which at the time was 71 out of 2093 known cases <span class="citation" data-cites="4SqaP0Ub">[<a href="#ref-4SqaP0Ub" role="doc-biblioref">280</a>]</span>.
Likewise, a serosurvey of health care workers in Blantyre City, Malawi reported an adjusted antibody prevalence of 12.3%, suggesting that the virus had been circulating more widely than thought and that the death rate was up eight times lower than models had predicted <span class="citation" data-cites="eWIOKa0u">[<a href="#ref-eWIOKa0u" role="doc-biblioref">281</a>]</span>.
While several possible hypotheses for the apparent reduced impact of COVID-19 on the African continent are being explored, such as young demographics in many places <span class="citation" data-cites="p4AYZss0">[<a href="#ref-p4AYZss0" role="doc-biblioref">282</a>]</span>, these reports present a stark contrast to the severity of COVID-19 in Americans and Europeans of African descent.
Additionally, ethnic minorities in the United Kingdom also tend to be younger than white British living in the same areas, yet the burden of COVID-19 is still more serious for minorities, especially people of Black Caribbean ancestry, both in absolute numbers and when controlling for age and location <span class="citation" data-cites="xFKmPg6l">[<a href="#ref-xFKmPg6l" role="doc-biblioref">283</a>]</span>.
Furthermore, the groups in the United States and United Kingdom that have been identified as carrying elevated COVID-19 burden, namely Black American, indigenous American, and Black and South Asian British, are quite distinct in their position on the human ancestral tree.
What is shared across these groups is instead a history of disenfranchisement under colonialism and ongoing systematic racism.
A large analysis of over 11,000 COVID-19 patients hospitalized in 92 hospitals across U.S. states revealed that Black patients were younger, more often female, more likely to be on Medicaid, more likely to have comorbidities, and came from neighborhoods identified as more economically deprived than white patients <span class="citation" data-cites="hrOv770z">[<a href="#ref-hrOv770z" role="doc-biblioref">284</a>]</span>.
This study reported that when these factors were accounted for, the differences in mortality between Black and white patients were no longer significant.
Thus, the current evidence suggests that the apparent correlations between ancestry and health outcomes must be examined in the appropriate social context.
We explore the role these social factors have played in shaping the COVID-19 pandemic broadly in the Discussion.</p>
<h1 data-number="4" id="diagnostics"><span class="header-section-number">4</span> Diagnostics</h1>
<p>Identifying individuals who have contracted COVID-19 is crucial to slowing down the global pandemic.
Given the high transmissibility of SARS-CoV-2, the development of reliable assays to detect SARS-CoV-2 infection even in asymptomatic carriers is vitally important.
For instance, the deployment of wide-scale diagnostic testing followed by the isolation of infected people has been a key factor in South Korea’s successful strategy for controlling the spread of the virus.
Following the first release of the genetic sequence of the virus by Chinese officials on January 10, 2020, the first test was released about 13 days later <span class="citation" data-cites="1AHtg71LX">[<a href="#ref-1AHtg71LX" role="doc-biblioref">285</a>]</span>.
Diagnostic approaches utilizing a variety of methods are currently or have been developed.
There are two main classes of diagnostic tests: molecular tests, which can diagnose an active infection by identifying the presence of SARS-CoV-2, and serological tests, which can assess whether an individual was infected in the past via the presence or absence of antibodies against SARS-CoV-2.
Molecular tests are essential for identifying individuals for treatment and alerting their contacts to quarantine and be alert for possible symptoms.
While serological tests may be of interest to individuals who wish to confirm they were infected with SARS-CoV-2 in the past, their potential for false positives means that they are not currently recommended for this use.
However, serological tests provide population-level information for epidemiological analysis, as they can be used to estimate the extent of the infection in a given area.
Thus, they may be useful in efforts to better understand the percent of cases that manifest as severe versus mild and for guiding public health and economic decisions regarding resource allocation and counter-disease measures.</p>
<h2 data-number="4.1" id="molecular-tests"><span class="header-section-number">4.1</span> Molecular Tests</h2>
<p>Molecular tests are used to identify distinct genomic subsequences of a viral molecule in a sample and thus to diagnose an active viral infection.
This first requires identifying which biospecimens are likely to contain the virus during infection and then acquiring these samples from the patient(s) to be tested.
Common sampling sources for molecular tests include nasopharyngeal cavity samples, such as throat wash and saliva <span class="citation" data-cites="fCZXxoas">[<a href="#ref-fCZXxoas" role="doc-biblioref">286</a>]</span>, and stool samples <span class="citation" data-cites="oNFnjRBI">[<a href="#ref-oNFnjRBI" role="doc-biblioref">287</a>]</span>.
Once a sample is acquired from a patient, molecular testing will utilize a number of steps, described below, to analyze a sample and identify whether evidence of SARS-CoV-2 is present.
When testing for RNA viruses like SARS-CoV-2, pre-processing is necessary in order to create DNA from the RNA sample.
The DNA can then be amplified with PCR.
Some tests use the results of the PCR itself to determine whether the pathogen is present, but in other cases, it may be necessary to sequence the amplified DNA.
Sequencing requires an additional pre-processing step: library preparation.
Library preparation is the process of preparing the sample for sequencing, typically by fragmenting the sequences and adding adapters <span class="citation" data-cites="jr3z6jGU">[<a href="#ref-jr3z6jGU" role="doc-biblioref">288</a>]</span>.
In some cases, library preparation can involve other modifications of the sample, such as adding “barcoding” to identify a particular sample in the sequence data, which is useful for pooling samples from multiple sources.
There are different reagents used for library preparation that are specific to identifying one or more target sections with PCR <span class="citation" data-cites="43modeVu">[<a href="#ref-43modeVu" role="doc-biblioref">289</a>]</span>.
Sequential pattern matching is then used to identify unique subsequences of the virus that identify it in specific.
If sufficient subsequences are found, the test is considered positive.</p>
<h3 data-number="4.1.1" id="rt-pcr"><span class="header-section-number">4.1.1</span> RT-PCR</h3>
<p>Real-Time Polymerase Chain Reaction (RT-PCR) tests determine whether a target is present by measuring the rate of amplification during PCR compared to a standard.
When the target is RNA, such as in the case of RNA viruses, the RNA must be converted into complementary DNA during pre-processing.
The Drosten Lab, from Germany, was the first lab to establish and validate a diagnostic test to detect SARS-CoV-2.
This test uses RT-PCR with reverse transcription <span class="citation" data-cites="1AHtg71LX">[<a href="#ref-1AHtg71LX" role="doc-biblioref">285</a>]</span> to detect several regions of the viral genome: the <em>ORF1b</em> of the RNA-dependent RNA polymerase (RdRP), the Envelope protein gene (<em>E</em>), and the Nucleocapsid protein gene (<em>N</em>).
In collaboration with several other labs in Europe and in China, the researchers confirmed the specificity of this test with respect to other coronaviruses against specimens from 297 patients infected with a broad range of respiratory agents.
Specifically this test utilizes two probes against RdRP, one of which is specific to SARS-CoV-2 <span class="citation" data-cites="1AHtg71LX">[<a href="#ref-1AHtg71LX" role="doc-biblioref">285</a>]</span>.
Importantly, this assay was not found to return false positive results.</p>
<h3 data-number="4.1.2" id="qrt-pcr"><span class="header-section-number">4.1.2</span> qRT-PCR</h3>
<p>Chinese researchers developed a quantitative real-time reverse transcription PCR (qRT-PCR) test to identify two gene regions of the viral genome, <em>ORF1b</em> and <em>N</em> <span class="citation" data-cites="COEfjKo5">[<a href="#ref-COEfjKo5" role="doc-biblioref">290</a>]</span>.
This assay was tested on samples coming two COVID-19 patients and a panel of positive and negative controls consisting of RNA extracted from several cultured viruses.
The assay uses the <em>N</em> gene to screen patients, while the <em>ORF1b</em> gene region is used to confirm the infection <span class="citation" data-cites="COEfjKo5">[<a href="#ref-COEfjKo5" role="doc-biblioref">290</a>]</span>.
In this case the test was designed to detect sequences conserved across sarbecoviruses, or viruses within the same subgenus as SARS-CoV-2.
Considering that SARS-CoV and SARS-CoV-2 are the only sarbecoviruses currently known to infect humans, a positive test can be assumed to indicate that the patient is infected with SARS-CoV-2.
However, this test is not able to discriminate the genetics of viruses within the sarbecovirus clade.</p>
<h4 data-number="4.1.2.1" id="dpcr"><span class="header-section-number">4.1.2.1</span> dPCR</h4>
<p>Digital PCR (dPCR) is a new generation of PCR technologies offering an alternative to traditional real-time quantitative PCR.
In dPCR, a sample is partitioned into thousands of compartments, such as nanodroplets (droplet dPCR or ddPCR) or nanowells, and a PCR reaction takes place in each compartment.
This design allows for a digital read-out where each partition is either positive or negative for the nucleic acid sequence being tested for, allowing for much higher throughput.
While dPCR equipment is not yet as common as that for RT-PCR, dPCR for DNA targets generally achieves higher sensitivity than other PCR technologies while maintaining high specificity, though sensitivity is slightly lower for RNA targets <span class="citation" data-cites="Hl2pASeg">[<a href="#ref-Hl2pASeg" role="doc-biblioref">291</a>]</span>.
High sensitivity is particularly relevant for SARS-CoV-2 detection, since low viral load in clinical samples can lead to false negatives.
Suo et al. <span class="citation" data-cites="2WgqQLF8">[<a href="#ref-2WgqQLF8" role="doc-biblioref">292</a>]</span> performed a double-blind evaluation of ddPCR for SARS-CoV-2 detection on 57 samples, comprised by 43 samples from suspected positive patients and 14 from supposed convalescents, that had all tested negative for SARS-CoV-2 using RT-PCR.
Despite the initial negative results, 33 out of 35 (94.3%) patients were later clinically confirmed positive.
All of these individuals tested positive using ddPCR.
Additionally, of 14 supposed convalescents who had received two consecutive negative RT-PCR tests, nine (64.2%) tested positive for SARS-CoV-2 using ddPCR.
Two symptomatic patients tested negative with both RT-PCR and ddPCR, but were later clinically diagnosed positive, and 5 of the 14 suspected convalescents tested negative by ddPCR.
While this study did not provide a complete head-to-head comparison to RT-PCR in all aspects, e.g., no samples testing positive using RT-PCR were evaluated by ddPCR, the study shows the potential of dPCR for viral detection even in highly diluted samples.
In a second study, Dong et al. <span class="citation" data-cites="UYD5T2mp">[<a href="#ref-UYD5T2mp" role="doc-biblioref">293</a>]</span> compared the results of qRT-PCR and ddPCR testing for SARS-CoV-2 in 194 samples, including 103 samples from suspected patients, 75 from contacts and close contacts, and 16 from suspected convalescents.
Of the 103 suspected patient samples, 29 were reported as positive, 25 as negative, and 49 as suspected by qRT-PCR; all patients were later confirmed to be SARS-CoV-2 positive.
Of the qRT-PCR negative or suspected samples, a total of 61 (17 negative and 44 suspected) were later confirmed to be positive by ddPCR, improving the overall detection rate among these patients from 28.2% to 87.4%.
Of 75 patient samples from contacts and close contacts, 48 tested negative with both methods, and these patients were observed to remain healthy.
Within the remaining 27 patient samples, 10 tested positive, 1 negative, and 16 suspect with qRT-PCR.
Fifteen out of 16 suspect samples and the negative test results were overturned by ddPCR, decreasing the rate of suspect cases from 21% to 1%.
Importantly, all samples that tested positive using qRT-PCR also tested positive using ddPCR.
Among the 16 convalescent patients, qRT-PCR identified 12 as positive, three as suspect, and one as negative, but RT-dPCR identified all 16 as positive.
This evidence further indicates that the lower limit of detection made possible by ddPCR may be useful for identifying when COVID-19 patients are cleared of the virus.
Overall, these studies suggest that dPCR is a promising tool for overcoming the problem of false-negative SARS-CoV-2 testing.</p>
<h3 data-number="4.1.3" id="pooled-and-automated-pcr-testing"><span class="header-section-number">4.1.3</span> Pooled and Automated PCR Testing</h3>
<p>Due to limited supplies and the need for more tests, several labs have found ways to pool or otherwise strategically design tests to increase throughput.
The first such result came from Yelin et al. <span class="citation" data-cites="93NBANq2">[<a href="#ref-93NBANq2" role="doc-biblioref">294</a>]</span>, who found they could pool up to 32 samples in a single qPCR run.
This was followed by larger-scale pooling with slightly different methods <span class="citation" data-cites="IQ5kUQEO">[<a href="#ref-IQ5kUQEO" role="doc-biblioref">295</a>]</span>.
Although these approaches are also PCR based, they allow for more rapid scaling and higher efficiency for testing than the initial PCR-based methods developed.</p>
<h4 data-number="4.1.3.1" id="crispr-based-detection"><span class="header-section-number">4.1.3.1</span> CRISPR-based Detection</h4>
<p>Two CRISPR-associated nucleases, Cas12 and Cas13, have been used for nucleic acid detection.
Multiple assays exploiting these nucleases have emerged as potential diagnostic tools for the rapid detection of SARS-CoV-2 genetic material and therefore SARS-CoV-2 infection.
The SHERLOCK method (Specific High-sensitivity Enzymatic Reporter unLOCKing) from Sherlock Biosciences relies on Cas13a to discriminate between inputs that differ by a single nucleotide at very low concentrations <span class="citation" data-cites="vVBrQqWB">[<a href="#ref-vVBrQqWB" role="doc-biblioref">296</a>]</span>.
The target RNA is amplified by RT-RPA and T7 transcription, and the amplified product activates Cas13a.
The nuclease then cleaves a reporter RNA, which liberates a fluorescent dye from a quencher.
Several groups have used the SHERLOCK method to detect SARS-CoV-2 viral RNA.
An early study reported that the method could detect 7.5 copies of viral RNA in all 10 replicates, 2.5 copies in 6 out of 10, and 1.25 copies in 2 out of 10 runs <span class="citation" data-cites="K6w6Llt2">[<a href="#ref-K6w6Llt2" role="doc-biblioref">297</a>]</span>.
It also reported 100% specificity and sensitivity on 114 RNA samples from clinical respiratory samples (61 suspected cases, among which 52 were confirmed and nine were ruled out by metagenomic next-generation sequencing, 17 nCoV-/hCoV+ cases and 36 samples from healthy subjects), and a reaction turnaround time of 40 minutes.
A separate study screened four designs of SHERLOCK and extensively tested the best-performing assay.
They determined the limit of detection to be 10 copies/μl using both fluorescent and lateral flow detection <span class="citation" data-cites="zSefi0ye">[<a href="#ref-zSefi0ye" role="doc-biblioref">298</a>]</span>.
Lateral flow test strips are simple to use and read, but there are limitations in terms of availability and cost per test.
Another group therefore proposed the CREST protocol (Cas13-based, Rugged, Equitable, Scalable Testing), which uses a P51 cardboard fluorescence visualizer, powered by a 9V battery, for the detection of Cas13 activity instead of immunochromatography <span class="citation" data-cites="GlLismDj">[<a href="#ref-GlLismDj" role="doc-biblioref">299</a>]</span>.
CREST can be run, from RNA sample to result, with no need for AC power or a dedicated facility, with minimal handling in approximately 2 hours.
Testing was performed on 14 nasopharyngeal swabs.
CREST picked up the same positives as the CDC-recommended TaqMan assay with the exception of one borderline sample that displayed low-quality RNA.</p>
<p>The DETECTR method (DNA Endonuclease-Targeted CRISPR Trans Reporter) from Mammoth Biosciences involves purification of RNA extracted from patient specimens, amplification of extracted RNAs by loop-mediated amplification, which is a rapid, isothermal nucleic acid amplification technique, and application of their Cas12-based technology.
In their assay, guide RNAs were designed to recognize portions of sequences corresponding to the SARS-CoV-2 genome, specifically the N2 and E regions <span class="citation" data-cites="XWCinLld">[<a href="#ref-XWCinLld" role="doc-biblioref">300</a>]</span>.
In the presence of SARS-CoV-2 genetic material, sequence recognition by the guide RNAs results in double-stranded DNA cleavage by Cas12, as well as cleavage of a single-stranded DNA molecular beacon.
The cleavage of this molecular beacon acts as a colorimetric reporter that is subsequently read out in a lateral flow assay and indicates the positive presence of SARS-CoV-2 genetic material and therefore SARS-CoV-2 infection.
The 40-minute assay is considered positive if there is detection of both the E and N genes or presumptive positive if there is detection of either of them.
The assay had 95% positive predictive agreement and 100% negative predictive agreement with the US Centers for Disease Control and Prevention SARS-CoV-2 real-time RT–PCR assay.
The estimated limit of detection was 10 copies per μl reaction, versus 1 copy per μl reaction for the CDC assay.
These results have been confirmed by other DETECTR approaches.
Using RTRPA for amplification, another group detected 10 copies of synthetic SARS-CoV-2 RNA per μl of input within 60 minutes of RNA sample preparation in a proof-of-principle evaluation <span class="citation" data-cites="4T3aId7a">[<a href="#ref-4T3aId7a" role="doc-biblioref">301</a>]</span>.
The DETECTR protocol was improved by combining RT-RPA and CRISPR-based detection in a one-pot reaction that incubates at a single temperature, and by using dual crRNAs (which increases sensitivity).
This new assay, known as All-In-One Dual CRISPR-Cas12a (AIOD-CRISPR), detected 4.6 copies of SARS-CoV-2 RNA per μl of input in 40 minutes <span class="citation" data-cites="Q4ylhpEC">[<a href="#ref-Q4ylhpEC" role="doc-biblioref">302</a>]</span>.
Another single-tube, constant-temperature approach using Cas12b instead of Cas12a achieved a detection limit of 5 copies/μl in 40-60 minutes <span class="citation" data-cites="1FqpXpmAD">[<a href="#ref-1FqpXpmAD" role="doc-biblioref">303</a>]</span>.
It was also reported that that electric field gradients can be used to control and accelerate CRISPR assays by co-focusing Cas12-gRNA, reporters, and target <span class="citation" data-cites="4y01WM4x">[<a href="#ref-4y01WM4x" role="doc-biblioref">304</a>]</span>.
The authors generated an appropriate electric field gradient using a selective ionic focusing technique known as isotachophoresis (ITP) implemented on a microfluidic chip.
They also used ITP for automated purification of target RNA from raw nasopharyngeal swab samples.
Combining this ITP purification with loop-mediated isothermal amplification, their ITP-enhanced assay to achieved detection of SARS-CoV-2 RNA (from raw sample to result) in 30 minutes.</p>
<p>There is an increasing body of evidence that CRISPR-based assays offer a practical solution for rapid, low-barrier testing in areas that are at greater risk of infection, such as airports and local community hospitals.
In the largest study to date, DETECTR was compared to qRT-PCR on 378 patient samples <span class="citation" data-cites="1BCUNYRvv">[<a href="#ref-1BCUNYRvv" role="doc-biblioref">305</a>]</span>.
The authors reported a 95% reproducibility.
Both techniques were equally sensitive in detecting SARS-CoV-2.
Lateral flow strips showed a 100% correlation to the high-throughput DETECTR assay.
Importantly, DETECTR was 100% specific for SARS-CoV-2 and did not detect other human coronaviruses.</p>
<h3 data-number="4.1.4" id="limitations-of-molecular-tests"><span class="header-section-number">4.1.4</span> Limitations of Molecular Tests</h3>
<p>Tests that identify SARS-CoV-2 using nucleic-acid-based technologies will identify only individuals with current infections and are not appropriate for identifying individuals who have recovered from a previous infection.
Within this category, different types of tests have different limitations.
For example, PCR-based test can be highly sensitive, but in high-throughput settings they can show several problems:</p>
<ol type="1">
<li>False-negative responses, which can present a significant problem to large-scale testing.
To reduce occurrence of false negatives, correct execution of the analysis is crucial <span class="citation" data-cites="Fdc7seLs">[<a href="#ref-Fdc7seLs" role="doc-biblioref">306</a>]</span>.</li>
<li>Uncertainty surrounding the SARS-CoV-2 viral shedding kinetics, which could affect the result of a test depending on when it was taken <span class="citation" data-cites="Fdc7seLs">[<a href="#ref-Fdc7seLs" role="doc-biblioref">306</a>]</span>.</li>
<li>Type of specimen, as it is not clear which clinical samples are best for detecting the virus <span class="citation" data-cites="Fdc7seLs">[<a href="#ref-Fdc7seLs" role="doc-biblioref">306</a>]</span>.</li>
<li>Expensive machinery, which might be present in major hospitals and/or diagnostic centers but is often not available to smaller facilities <span class="citation" data-cites="NT5mrVGY">[<a href="#ref-NT5mrVGY" role="doc-biblioref">307</a>]</span>.</li>
<li>Timing of the test, which might take up to 4 days to return results <span class="citation" data-cites="NT5mrVGY">[<a href="#ref-NT5mrVGY" role="doc-biblioref">307</a>]</span>.</li>
<li>The availability of supplies for testing, including swabs and testing media, has been limited <span class="citation" data-cites="AvQI3GSm">[<a href="#ref-AvQI3GSm" role="doc-biblioref">308</a>]</span>.</li>
<li>Because the guide RNA can recognize other interspersed sequences on the patient’s genome, false positives and a loss of specificity can occur.</li>
</ol>
<p>Similarly, in tests that use CRISPR, false positives can occur due to the specificity of the technique, as the guide RNA can recognize other interspersed sequences on the patient’s genome.
As noted above, false negatives are a significant concern for several reason.
Importantly, clinical reports indicate that it is imperative to exercise caution when interpreting the results of molecular tests for SARS-CoV-2 because negative results do not necessarily mean a patient is virus-free <span class="citation" data-cites="11vIuqmS2">[<a href="#ref-11vIuqmS2" role="doc-biblioref">309</a>]</span>.</p>
<h2 data-number="4.2" id="serological-tests"><span class="header-section-number">4.2</span> Serological Tests</h2>
<p>Although diagnostic tests based on the detection of genetic material can be quite sensitive, they cannot provide information about the extent of the disease over time.
Most importantly, they would not work on a patient who has fully recovered from the virus at the time of sample collection.
In this context, serological tests, which use serum to test for the presence of antibodies against SARS-CoV-2, are significantly more informative.
Additionally, serological tests can help scientists to understand why the disease has a different course among patients, as well as which strategies might work to manage the spread of the infection.
Furthermore, serological tests hold significant interest because of the possibility that they could provide information relevant to advancing economic recovery and allowing reopenings.
For instance, people that had developed antibodies might be able to return to work, assuming (still-unproven) protective immunity <span class="citation" data-cites="EUrJt06r">[<a href="#ref-EUrJt06r" role="doc-biblioref">310</a>]</span>.
Some infectious agents can be controlled through “herd immunity”, which is when a critical mass within the population acquires immunity through vaccination and/or infection, preventing an infectious agent from spreading widely.
A simple SIR model predicts that to achieve the required level of exposure for herd immunity to be effective, at least (1-(1/R0)) fraction of the population must be immune or, equivalently, less than (1/R0) fraction of the population susceptible <span class="citation" data-cites="JRaVgcNd">[<a href="#ref-JRaVgcNd" role="doc-biblioref">206</a>]</span>.
However, for SARS-CoV-2 and COVID-19, the R0 and mortality rates that have been observed suggest that relying on herd immunity without some combination of vaccines, proven treatment options, and strong non-pharmaceutical measures of prevention and control would likely result in a significant loss of life.</p>
<h3 data-number="4.2.1" id="sustained-immunity-to-covid-19"><span class="header-section-number">4.2.1</span> Sustained Immunity to COVID-19</h3>
<p>In the process of mounting a response to a pathogen, the immune system produces antibodies specific to the pathogen.
Understanding the acquisition and retention of antibodies is important both to the diagnosis of prior (inactive) infections and to the development of vaccines.
The two immunoglobulin classes most pertinent to these goals are immunoglobulin M (IgM), which are the first antibodies produced in response to an infection, and immunoglobulin G (IgG), which are the most abundant antibodies found in the blood.
Following SARS infection, IgM and IgG antibodies were detected in the second week post-infection.
IgM titers peaked by the first month post-infection, and then declined to undetectable levels after day 180.
IgG titers peaked by day 60, and persisted in all donors through the two-year duration of study <span class="citation" data-cites="pGXsPCCz">[<a href="#ref-pGXsPCCz" role="doc-biblioref">311</a>]</span>.
A two-year longitudinal study following convalesced SARS patients with a mean age of 29 found that IgG antibodies were detectable in all 56 patients surveyed for at least 16 months, and remained detectable in all but 4 (11.8%) of patients through the full two-year study period <span class="citation" data-cites="11UpvND0K">[<a href="#ref-11UpvND0K" role="doc-biblioref">312</a>]</span>.
These results suggest that immunity to SARS-CoV is sustained for at least a year.</p>
<p>The persistence of antibodies to SARS-CoV-2 remains under investigation.
Circulating antibody titers to other coronaviruses have been reported to decline significantly after 1 year <span class="citation" data-cites="FIOysZLl">[<a href="#ref-FIOysZLl" role="doc-biblioref">313</a>]</span>.
Autopsies of lymph nodes and spleens from severe acute COVID-19 patients showed a loss of T follicular helper cells and germinal centers that may explain some of the impaired development of antibody responses <span class="citation" data-cites="JcpiBiQt">[<a href="#ref-JcpiBiQt" role="doc-biblioref">314</a>]</span>.
An early study (initially released on <em>medRxiv</em> on February 25, 2020) presented a chemiluminescence immunoassay to a synthetic peptide derived from the amino acid sequence of the SARS_CoV-2 <em>S</em> protein <span class="citation" data-cites="ciLSdA3x">[<a href="#ref-ciLSdA3x" role="doc-biblioref">315</a>]</span>.
This method was highly specific to SARS-CoV-2 and detected IgM in 57.2% and IgG in 71.4% and 57.2% of sera samples from 276 confirmed COVID-19 patients.
They reported that they could detect IgG within two days of the onset of fever and were not able to detect IgM any earlier, a pattern they compared to findings in MERS.
<!--TODO: add https://www.nature.com/articles/s41591-020-0965-6#Sec9 and evalute reports of recurrence.-->
The duration over which these antibodies persist remains unknown.</p>
<h3 data-number="4.2.2" id="current-approaches"><span class="header-section-number">4.2.2</span> Current Approaches</h3>
<p>Several countries are now focused on implementing antibody tests, and in the United States, the FDA recently approved a serological test by Cellex for use under emergency conditions <span class="citation" data-cites="n6sKSpIk">[<a href="#ref-n6sKSpIk" role="doc-biblioref">316</a>]</span>.
Specifically, the Cellex qSARS-CoV-2 IgG/IgM Rapid Test is a chromatographic immunoassay designed to qualitatively detect IgM and IgG antibodies against SARS-CoV-2 in the plasma (from a blood sample) of patients suspected to have developed the SARS-CoV-2 infection <span class="citation" data-cites="n6sKSpIk">[<a href="#ref-n6sKSpIk" role="doc-biblioref">316</a>]</span>.
Such tests illuminate the progression of viral disease, as IgM are the first antibodies produced by the body and indicate that the infection is active.
Once the body has responded to the infection, IgG are produced and gradually replace IgM, indicating that the body has developed immunogenic memory <span class="citation" data-cites="FFMfMDRN">[<a href="#ref-FFMfMDRN" role="doc-biblioref">317</a>]</span>.
The test cassette contains a pad of SARS-CoV-2 antigens and a nitrocellulose strip with lines for each of IgG and IgM, as well as a control (goat IgG) <span class="citation" data-cites="n6sKSpIk">[<a href="#ref-n6sKSpIk" role="doc-biblioref">316</a>]</span>.
In a specimen that contains antibodies against the SARS-CoV-2 antigen, the antibodies will bind to the strip and be captured by the IgM and/or IgG line(s), resulting in a change of color <span class="citation" data-cites="n6sKSpIk">[<a href="#ref-n6sKSpIk" role="doc-biblioref">316</a>]</span>.
With this particular assay, results can be read within 15-20 minutes <span class="citation" data-cites="n6sKSpIk">[<a href="#ref-n6sKSpIk" role="doc-biblioref">316</a>]</span>.
Other research groups, such as the Krammer lab of the Icahn School of Medicine at Mount Sinai, proposed an ELISA test that detects IgG and IgM that react against the receptor binding domain (RBD) of the spike proteins (S) of the virus <span class="citation" data-cites="yJW1RVQb">[<a href="#ref-yJW1RVQb" role="doc-biblioref">318</a>]</span>.
The authors are now working to get the assay into clinical use <span class="citation" data-cites="Gk9WwhYB">[<a href="#ref-Gk9WwhYB" role="doc-biblioref">319</a>]</span>.</p>
<h3 data-number="4.2.3" id="limitations-of-serological-tests"><span class="header-section-number">4.2.3</span> Limitations of Serological Tests</h3>
<p>Importantly, false-positives can occur due to the cross-reactivity with other antibodies according to the clinical condition of the patient <span class="citation" data-cites="n6sKSpIk">[<a href="#ref-n6sKSpIk" role="doc-biblioref">316</a>]</span>.
Therefore, this test should be used in combination with RNA detection tests <span class="citation" data-cites="n6sKSpIk">[<a href="#ref-n6sKSpIk" role="doc-biblioref">316</a>]</span>.
Due to the long incubation times and delayed immune responses of infected patients, serological tests are insufficiently sensitive for a diagnosis in the early stages of an infection.
The limitations due to timing make serological tests far less useful for enabling test-and-trace strategies.</p>
<h2 data-number="4.3" id="possible-alternatives-to-current-practices-for-identifying-active-cases"><span class="header-section-number">4.3</span> Possible Alternatives to Current Practices for Identifying Active Cases</h2>
<p>Clinical symptoms are too similar to other types of pneumonia to be sufficient as a sole diagnostics criterion.
In addition, as noted above, identifying asymptomatic cases is critical.
Even among mildly symptomatic patients, a predictive model based on clinical symptoms had a sensitivity of only 56% and a specificity of 91% <span class="citation" data-cites="91dto3z3">[<a href="#ref-91dto3z3" role="doc-biblioref">320</a>]</span>.
More problematic is that clinical symptom-based tests are only able to identify already symptomatic cases, not presymptomatic or asymptomatic cases.
They may still be important for clinical practice, and for reducing tests needed for patients deemed unlikely to have COVID-19.</p>
<p>X-ray diagnostics have been reported to have high sensitivity but low specificity in some studies <span class="citation" data-cites="itCGVb5b">[<a href="#ref-itCGVb5b" role="doc-biblioref">321</a>]</span>.
Other studies have shown that specificity varies between radiologists <span class="citation" data-cites="tkuUFNIX">[<a href="#ref-tkuUFNIX" role="doc-biblioref">322</a>]</span>, though the sensitivity reported here was lower than that published in the previous paper.
However, preliminary machine-learning results have shown far higher sensitivity and specificity from analyzing chest X-rays than was possible with clinical examination <span class="citation" data-cites="nN9zfzPe">[<a href="#ref-nN9zfzPe" role="doc-biblioref">323</a>]</span>.
X-ray tests with machine learning can potentially detect asymptomatic or presymptomatic infections that show lung manifestations.
This approach would still not recognize entirely asymptomatic cases.
Given the above, the widespread use of X-ray tests on otherwise healthy adults is likely inadvisable.</p>
<h2 data-number="4.4" id="strategies-and-considerations-for-determining-whom-to-test"><span class="header-section-number">4.4</span> Strategies and Considerations for Determining Whom to Test</h2>
<p>Early in the COVID-19 pandemic, testing was typically limited to individuals considered high risk for developing serious illness <span class="citation" data-cites="OB8BJiwj">[<a href="#ref-OB8BJiwj" role="doc-biblioref">324</a>]</span>.
This approach often involved limiting testing to people with severe symptoms and people showing mild symptoms that had been in contact with a person who had tested positive.
Individuals who are asymptomatic (i.e., potential spreaders) and individuals who are able to recover at home are therefore often unaware of their status.
However, this method of testing administration misses a high proportion of infections and does not allow for test-and-trace methods to be used.
For instance, a recent study from Imperial College estimates that in Italy, the true number of infections was around 5.9 million in a total population ~60 million, compared to the 70,000 detected as of March 28th <span class="citation" data-cites="Frya4XA4">[<a href="#ref-Frya4XA4" role="doc-biblioref">224</a>]</span>.
Another analysis, which examined New York state, indicated that as of May, 2020, approximately 300,000 cases had been reported in a total population of approximately 20 million <span class="citation" data-cites="IldxlHwA">[<a href="#ref-IldxlHwA" role="doc-biblioref">325</a>]</span>.
This corresponded to ~1.5% of the population, but ~12% of individuals sampled statewide were estimated as positive through antibody tests (along with indications of spatial heterogeneity at higher resolution) <span class="citation" data-cites="IldxlHwA">[<a href="#ref-IldxlHwA" role="doc-biblioref">325</a>]</span>.
Technological advancements that facilitate widespread, rapid testing will therefore improve the potential to accurately assess the rate of infection and aid in controlling the virus’ spread.</p>
<h1 data-number="5" id="prophylactics-and-therapeutics"><span class="header-section-number">5</span> Prophylactics and Therapeutics</h1>
<p>Given the worldwide spread of COVID-19, the identification and development of prophylactic and therapeutic interventions hold valuable potential for controlling the impact of the disease.
Such interventions fall into two categories: prophylactics, which are meant to prevent a disease from occurring, and therapeutics, which are meant to treat existing disease.
For infectious diseases such as COVID-19, the main prophylactics of interest are vaccines.
Several types of vaccines are currently under development, as detailed below.
While vaccines are expected to save the largest number of lives by bolstering the immune response to the virus of the population broadly, resulting in a lower rate of infection, the vaccine development process is long, and they fail to provide immediate prophylactic protection or treat ongoing infections <span class="citation" data-cites="181QWa7HL">[<a href="#ref-181QWa7HL" role="doc-biblioref">326</a>]</span>.
Thus, there is also an immediate need for treatments that palliate symptoms and prevent the most severe outcomes from infection.
Therapeutics are generally considered either for the treatment and reduction of symptoms or as a direct way of targeting the virus (e.g., antivirals).
The goal of the former is to reduce the severity and risks of an active infection while for the latter, it is to inhibit the development of the virus once an individual is infected.
In the context of COVID-19, uncertainty often surrounds a treatment’s exact mechanism of action, as most therapies have secondary or off-target effects.
Thus, in this section, we describe some of the early treatments considered and classify both therapeutics and prophylactics according to their biological properties, specifically whether they are biologics (produced from components of organisms) or small molecules.
Biologics include antibodies, interferons, and vaccines, while small molecules include drugs targeted at viral particles, drugs targeted at host proteins, and broad spectrum pharmaceuticals.
Broad-based pharmaceuticals include the much-discussed drugs hydroxychloroquine and chloroquine, as well as nutraceuticals, which are dietary supplement interventions that may prime an individual’s immune system to lessen the impact of RNA virus infections <span class="citation" data-cites="eyBAUUCl GT7yDBjv">[<a href="#ref-eyBAUUCl" role="doc-biblioref">327</a>,<a href="#ref-GT7yDBjv" role="doc-biblioref">328</a>]</span>.
In the following sections, we critically appraise the literature and clinical trials (Figure <a href="#fig:ebm-trials">3</a>) surrounding the repurposing of existing treatments and development of novel approaches for the prevention, mitigation, and treatment of coronavirus infections.</p>
<div id="fig:ebm-trials" class="fignos">
<figure>
<img src="https://github.com/greenelab/covid19-review/raw/65b2be6684a85cb83d462cdabcedda1d5393b8b7/ebmdatalab/ebmdatalab-trials.svg" alt="" /><figcaption><span>Figure 3:</span> <strong>COVID-19 clinical trials.</strong>
There are 5,257 COVID-19 clinical trials and 157 trials with results as of September 2, 2020.
The recruitment statuses and trial phases are shown only for trials in which the status or phase is recorded.
The study types include only types used in at least five trials.
The common interventions are all interventions used in at least ten trials.
Combinations of interventions, such as Hydroxychloroquine + Azithromycin, are tallied separately from the individual interventions.
Trials data are from the University of Oxford Evidence-Based Medicine Data Lab’s COVID-19 TrialsTracker <span class="citation" data-cites="SSbnPnzT">[<a href="#ref-SSbnPnzT" role="doc-biblioref">329</a>]</span>.</figcaption>
</figure>
</div>
<h2 data-number="5.1" id="small-molecule-drugs"><span class="header-section-number">5.1</span> Small Molecule Drugs</h2>
<h3 data-number="5.1.1" id="small-molecule-drugs-targeting-sars-cov-2"><span class="header-section-number">5.1.1</span> Small Molecule Drugs Targeting SARS-CoV-2</h3>
<p>The replication cycle of a virus within an epithelial host cell includes six basic steps that can be summarized as follows: i) attachment of the virus to the host cell; ii) penetration by endocytosis; iii) uncoating, classically defined as the release of viral contents into the host cell; iv) biosynthesis, during which the viral genetic material enters the nucleus where it gets replicated; v) assembly, where viral proteins are translated and new viral particles are assembled; vi) release, when the new viruses are released into the extracellular environment <span class="citation" data-cites="14gthSuhm">[<a href="#ref-14gthSuhm" role="doc-biblioref">330</a>]</span>.
Antiviral drugs do not kill the virus; rather, they inhibit its amplification by impairing one of these steps.
Today, many of these drugs act during the biosynthesis step in order to inhibit the replication of viral genetic material.
In contrast to DNA viruses, which can use the host enzymes to propagate themselves, RNA viruses like SARS-CoV-2 depends on their own polymerase, the RNA-dependent RNA polymerase (RdRP), for replication <span class="citation" data-cites="tn6j6IZ0 Sh1FwWps">[<a href="#ref-tn6j6IZ0" role="doc-biblioref">331</a>,<a href="#ref-Sh1FwWps" role="doc-biblioref">332</a>]</span>.
As noted above, even if a drug is meant to target the virus, it can also impact other processes in the host.</p>
<h4 data-number="5.1.1.1" id="nucleoside-and-nucleotide-analogs"><span class="header-section-number">5.1.1.1</span> Nucleoside and Nucleotide Analogs</h4>
<h5 data-number="5.1.1.1.1" id="favipiravir"><span class="header-section-number">5.1.1.1.1</span> Favipiravir</h5>
<p>Favipiravir (Avigan), also known as T-705, was discovered by Toyama Chemical Co., Ltd. <span class="citation" data-cites="IkAU62Pd">[<a href="#ref-IkAU62Pd" role="doc-biblioref">333</a>]</span>.
The drug was found to be effective at blocking viral amplification in several influenza subtypes as well as other RNA viruses, such as <em>Flaviviridae</em> and <em>Picornaviridae</em>, through a reduction in plaque formation <span class="citation" data-cites="EgoJKtW4">[<a href="#ref-EgoJKtW4" role="doc-biblioref">334</a>]</span> and viral replication in Madin-Darby Canine Kidney (MDCK) cells <span class="citation" data-cites="rG5RCRdJ">[<a href="#ref-rG5RCRdJ" role="doc-biblioref">335</a>]</span>.
Furthermore, inoculation of mice with favipiravir was shown to increase survival <span class="citation" data-cites="EgoJKtW4 rG5RCRdJ">[<a href="#ref-EgoJKtW4" role="doc-biblioref">334</a>,<a href="#ref-rG5RCRdJ" role="doc-biblioref">335</a>]</span>.
In 2014, the drug was approved in Japan for the treatment of patients infected with influenza that was resistant to conventional treatments like neuraminidase inhibitors <span class="citation" data-cites="14p3btkRH">[<a href="#ref-14p3btkRH" role="doc-biblioref">336</a>]</span>.
Favipiravir (6-fluoro-3-hydroxy-2-pyrazinecarboxamide) acts as a purine and purine nucleoside analogue that inhibits viral RNA polymerase in a dose-dependent manner across a range of RNA viruses, including Influenza virus <span class="citation" data-cites="DpdKJbki 9j7NVPJS IYEoCGHy 7d1TnpE3 15kQ4QYbx">[<a href="#ref-DpdKJbki" role="doc-biblioref">337</a>,<a href="#ref-9j7NVPJS" role="doc-biblioref">338</a>,<a href="#ref-IYEoCGHy" role="doc-biblioref">339</a>,<a href="#ref-7d1TnpE3" role="doc-biblioref">340</a>,<a href="#ref-15kQ4QYbx" role="doc-biblioref">341</a>]</span>.
Nucleotides/sides are the natural building blocks for RNA synthesis.
Because of this, modifications to these nucleotides/sides can disrupt key processes including replication <span class="citation" data-cites="MPJdSF1p">[<a href="#ref-MPJdSF1p" role="doc-biblioref">342</a>]</span>.
Biochemical experiments showed that favipiravir was recognized as a purine nucleoside analogue and incorporated into the viral RNA template.
A single incorporation does not influence RNA transcription; however, multiple events of incorporation lead to the arrest of RNA synthesis <span class="citation" data-cites="ukZRKllk">[<a href="#ref-ukZRKllk" role="doc-biblioref">343</a>]</span>.
Evidence for T-705 inhibiting viral RNA polymerase are based on time-of-drug addition studies that found that viral loads were reduced with the addition of Favipiravir in early times post-infection <span class="citation" data-cites="DpdKJbki 7d1TnpE3 15kQ4QYbx">[<a href="#ref-DpdKJbki" role="doc-biblioref">337</a>,<a href="#ref-7d1TnpE3" role="doc-biblioref">340</a>,<a href="#ref-15kQ4QYbx" role="doc-biblioref">341</a>]</span>.</p>
<p>The effectiveness of favipiravir for treating patients with COVID-19 is currently under investigation.
An open-label, nonrandomized, before-after controlled study was recently conducted <span class="citation" data-cites="13Oz7yRPx">[<a href="#ref-13Oz7yRPx" role="doc-biblioref">344</a>]</span>.
The study included 80 COVID-19 patients (35 treated with favipiravir, 45 control) from the isolation ward of the National Clinical Research Center for Infectious Diseases (The Third People’s Hospital of Shenzhen), Shenzhen, China.
The patients in the control group were treated with other antivirals, such as lopinavir and ritonavir.
Treatment was applied on days 2-14; treatment stopped either when viral clearance was confirmed or at day 14.
The efficacy of the treatment was measured by 1) the time until viral clearance using Kaplan-Meier survival curves, and 2) the improvement rate of chest computed tomography (CT) scans on day 14 after treatment.
The study found that favipiravir increased the speed of recovery (measured as viral clearance from the patient by RT-PCR) to 4 days compared to 11 days using other antivirals such as lopinavir and ritonavir.
Additionally, the lung CT scans of patients treated with favipiravir showed significantly higher improvement rates (91%) on day 14 compared to control patients (62%).
However, there were adverse side effects in 4 (11%) favipiravir-treated patients and 25 (56%) control patients.
The adverse side effects included: diarrhea, vomiting, nausea, rash, and liver and kidney injury.
Overall, despite the study reporting clinical improvement in favipiravir-treated patients, due to some issues with study design, it cannot be determined whether treatment with favipiravir had an effect or whether these patients would have recovered regardless of any treatment.
For example, despite the significant differences observed between the two treatment groups, follow-up analysis is necessary due to the small sample size.
The selection of patients did not take into consideration important factors such as previous clinical conditions or sex, and there was no age categorization.
The study was neither randomized nor blinded, and the baseline control group was another antiviral instead of a placebo.
Therefore, randomized controlled trials are still required.</p>
<h5 data-number="5.1.1.1.2" id="remdesivir"><span class="header-section-number">5.1.1.1.2</span> Remdesivir</h5>
<p>Remdesivir (GS-5734) is an intravenous antiviral that was developed by Gilead Sciences to treat Ebola Virus Disease.
At the outset of the COVID-19 pandemic, it did not have any have any FDA-approved use.
However, on May 1, 2020, the FDA issued an Emergency Use Authorization (EUA) for remdesivir for the treatment of hospitalized COVID-19 patients <span class="citation" data-cites="1BYoe0y2H">[<a href="#ref-1BYoe0y2H" role="doc-biblioref">345</a>]</span>.
The EUA was based on information from two clinical trials, NCT04280705 and NCT04292899 <span class="citation" data-cites="yTCAmOyt 1ATTPbCCO DtpdBrgt 16zLDIOyk">[<a href="#ref-yTCAmOyt" role="doc-biblioref">346</a>,<a href="#ref-1ATTPbCCO" role="doc-biblioref">347</a>,<a href="#ref-DtpdBrgt" role="doc-biblioref">348</a>,<a href="#ref-16zLDIOyk" role="doc-biblioref">349</a>]</span>.
Remdesivir is metabolized to GS-441524, an adenosine analog that inhibits a broad range of polymerases and then evades exonuclease repair, causing chain termination <span class="citation" data-cites="YaxKcqq8 lIko9Imd IhdcvbzC">[<a href="#ref-YaxKcqq8" role="doc-biblioref">350</a>,<a href="#ref-lIko9Imd" role="doc-biblioref">351</a>,<a href="#ref-IhdcvbzC" role="doc-biblioref">352</a>]</span>.
A clinical trial in the Democratic Republic of Congo found some evidence of effectiveness against Ebola, but two antibody preparations were found to be more effective, and remdesivir was not pursued <span class="citation" data-cites="gnfviuGp">[<a href="#ref-gnfviuGp" role="doc-biblioref">353</a>]</span>.
Although it was developed against Ebola, remdesivir also inhibits polymerase and replication of the coronaviruses MERS-CoV and SARS-CoV in cell culture assays with submicromolar IC50s <span class="citation" data-cites="1H0NPTMw9">[<a href="#ref-1H0NPTMw9" role="doc-biblioref">354</a>]</span>.
It has also been found to inhibit SARS-CoV-2, showing synergy with chloroquine <em>in vitro</em> <span class="citation" data-cites="IhdcvbzC">[<a href="#ref-IhdcvbzC" role="doc-biblioref">352</a>]</span>.
The effectiveness of remdesivir for treating patients with COVID-19 is currently under investigation.</p>
<p>Remdesivir was first used on some COVID-19 patients under compassionate use guidelines <span class="citation" data-cites="D4HwvRMd zoLViMNg">[<a href="#ref-D4HwvRMd" role="doc-biblioref">355</a>,<a href="#ref-zoLViMNg" role="doc-biblioref">356</a>]</span>.
All were in late stages of COVID-19 infection, and these reports were inconclusive about the drug’s efficacy.
Gilead Sciences, the maker of remdesivir, led a recent publication that reported outcomes for compassionate use of the drug in 61 patients hospitalized with confirmed COVID-19.
Here, 200 mg of remdesivir was administered intravenously on day 1, followed by a further 100 mg/day for 9 days <span class="citation" data-cites="16zLDIOyk">[<a href="#ref-16zLDIOyk" role="doc-biblioref">349</a>]</span>.
There were significant issues with the study design, or lack thereof.
There was no randomized control group.
The inclusion criteria were variable: some patients only required low doses of oxygen, while others required ventilation.
The study included many sites, potentially with variable inclusion criteria and treatment protocols.
The patients analyzed had mixed demographics.
There was a short follow-up period of investigation.
Some patients worsened, some patients died, and eight were excluded from the analysis mainly due to missing post-baseline information; thus, their health is unaccounted for.
Therefore, even though the study reported clinical improvement in 68% of the 53 patients ultimately evaluated, due to the significant issues with study design, it could not be determined whether treatment with remdesivir had an effect or whether these patients would have recovered regardless of treatment.
Another study comparing 5- and 10-day treatment regimens had similar results but was also limited because of the lack of a placebo control <span class="citation" data-cites="w5y1LWUR">[<a href="#ref-w5y1LWUR" role="doc-biblioref">358</a>]</span>.
The study did not alter our understanding of the efficacy of remdesivir in treating COVID-19, but the encouraging results provided motivation for placebo-controlled studies.</p>
<p>Remdesivir was later tested in a double-blind placebo-controlled phase 3 clinical trial performed at 60 trial sites, 45 of which were in the United states.
The trial recruited 1,063 patients and randomly assigned them to placebo treatment or treatment with remdesivir.
The treatment was 200 mg on day 1, followed by 100 mg on days 2 through 10.
Data was analyzed from 1,059 patients (538 assigned to remdesivir and 521 to placebo).
The two groups were well matched demographically and clinically at baseline.
Those who received remdesivir had a median recovery time of 11 days (95% confidence interval [CI], 9 to 12), as compared with 15 days (95% CI, 13 to 19) in those who received placebo (rate ratio for recovery, 1.32; 95% CI, 1.12 to 1.55; P&lt;0.001).
The Kaplan-Meier estimates of mortality by 14 days were 7.1% with remdesivir and 11.9% with placebo (hazard ratio for death, 0.70; 95% CI, 0.47 to 1.04).
Though mortality was lower in the remdesivir group, the difference was not significant.
Serious adverse events were reported for 114 of the 541 patients in the remdesivir group who underwent randomization (21.1%) and in 141 of the 522 patients in the placebo group who underwent randomization (27.0%).
The median time to recovery in patients in the subgroup receiving invasive mechanical ventilation or extracorporeal membrane oxygenation (ECMO) could not be established, which may indicate that the follow up time was too short for this group (272 patients).
Largely on the results of this trial, the FDA issued an Emergency Use Authorization (EUA) for remdesivir for the treatment of hospitalized COVID-19 patients.
Clinical trials are currently underway to evaluate the use of remdesivir to treat COVID-19 patients at both early and late stages of infection and in combinations with other drugs (Figure <a href="#fig:ebm-trials">3</a>).
These trials include <span class="citation" data-cites="IhdcvbzC oso9r21L 1q4PmqTm rUTsIlZs JSoOOX98">[<a href="#ref-IhdcvbzC" role="doc-biblioref">352</a>,<a href="#ref-oso9r21L" role="doc-biblioref">359</a>,<a href="#ref-1q4PmqTm" role="doc-biblioref">360</a>,<a href="#ref-rUTsIlZs" role="doc-biblioref">361</a>,<a href="#ref-JSoOOX98" role="doc-biblioref">362</a>]</span>.
As of October 22, 2020, remdesivir received FDA approval based on three clinical trials <span class="citation" data-cites="1DTFVDWTn">[<a href="#ref-1DTFVDWTn" role="doc-biblioref">363</a>]</span>.</p>
<p>In summary, remdesivir is a first in class drug due to its FDA approval.
Early investigations of this drug establish proof of principle that drugs targeting the virus can benefit patients.
It also shows proof of principle that the virus can be targeted at the level of viral replication, since remdesivir targets the viral RNA polymerase at high potency.
Moreover, one of the most successful strategies for developing therapeutics for viral diseases is to target the viral replication machinery, which are typically virally encoded polymerases.
Small molecule drugs targeting viral polymerases are the backbones of treatments for other viral diseases including HIV and Herpes.
Notably, the HIV and herpes polymerases are a reverse transcriptase and a DNA polymerase, respectively, whereas SARS-CoV-2 encodes an RNA-dependent RNA polymerase, so most of the commonly used polymerase inhibitors are not likely to be active against SARS-CoV-2.
In clinical use, polymerase inhibitors show short term benefits for HIV patients but for long term benefits they must be part of combination regimens.
They are typically combined with protease inhibitors, integrase inhibitors, and even other polymerase inhibitors.
Additional clinical trials of remdesivir on different patient pools and in combination with other therapies will refine its use in the clinic.</p>
<h4 data-number="5.1.1.2" id="protease-inhibitors"><span class="header-section-number">5.1.1.2</span> Protease Inhibitors</h4>
<p>Several studies showed that viral proteases play an important role in the life cycle of viruses, including coronaviruses, by modulating the cleavage of viral polyprotein precursors <span class="citation" data-cites="26dpptLw">[<a href="#ref-26dpptLw" role="doc-biblioref">364</a>]</span>.
Several FDA-approved drugs target proteases, including lopinavir and ritonavir for HIV infection and simeprevir for hepatitis C virus infection.
In particular, serine protease inhibitors were suggested for the treatment of SARS and MERS viruses <span class="citation" data-cites="f7csoyEu">[<a href="#ref-f7csoyEu" role="doc-biblioref">365</a>]</span>.
Recently, a study <span class="citation" data-cites="15l3di3Wj">[<a href="#ref-15l3di3Wj" role="doc-biblioref">65</a>]</span> suggested that camostat mesylate, an FDA-approved protease inhibitor (PI) could block the entry of SARS-CoV-2 into lung cells <em>in vitro</em>.
However, to test the efficacy of PIs in patients, randomized clinical trials will need to be conducted on patients and healthy volunteers.
Investigation of possible PIs that would work against SARS-CoV-2 has been driven by computational predictions, leading to the computer-aided design of a Michael acceptor inhibitor, N3, to target a protease critical to SARS-CoV-2 replication.</p>
<p>Discovery of the N3 mechanism arose from interest in the two polyproteins encoded by the SARS-CoV-2 replicase gene, pp1a and pp1ab, that are critical for viral replication and transcription <span class="citation" data-cites="4Sja6dIz">[<a href="#ref-4Sja6dIz" role="doc-biblioref">366</a>]</span>.
These polyproteins must undergo proteolytic processing.
This processing is usually conducted by Mpro, a 33.8-kilodalton (kDa) SARS-CoV-2 protease that is therefore fundamental to viral replication and transcription.
N3 was designed computationally <span class="citation" data-cites="15JYG0AIU">[<a href="#ref-15JYG0AIU" role="doc-biblioref">367</a>]</span> to bind in the substrate binding pocket of the Mpro protease of SARS-like coronaviruses <span class="citation" data-cites="XfOTPQPJ">[<a href="#ref-XfOTPQPJ" role="doc-biblioref">368</a>]</span>, therefore inhibiting proteolytic processing.
Subsequently, the structure of N3-bound SARS-CoV-2 Mpro was solved <span class="citation" data-cites="4Sja6dIz">[<a href="#ref-4Sja6dIz" role="doc-biblioref">366</a>]</span>, confirming the computational prediction.
N3 was tested <em>in vitro</em> on SARS-CoV-2-infected Vero cells, which belong to a line of cells established from the kidney epithelial cells of an African green monkey, and was found to inhibit SARS-CoV-2 <span class="citation" data-cites="4Sja6dIz">[<a href="#ref-4Sja6dIz" role="doc-biblioref">366</a>]</span>.</p>
<p>Although N3 is a strong inhibitor of SARS-CoV-2 <em>in vitro</em>, its safety and efficacy still need to be tested in healthy volunteers and patients.
After the design and confirmation of N3 as a highly potent Michael acceptor inhibitor and the identification of Mpro’s structure <span class="citation" data-cites="4Sja6dIz 10NcfBSiT">[<a href="#ref-4Sja6dIz" role="doc-biblioref">366</a>,<a href="#ref-10NcfBSiT" role="doc-biblioref">369</a>]</span>, 10,000 compounds were screened for their <em>in vitro</em> anti-Mpro activity.
The six leads that were identified were ebselen, disulfiram, tideglusib, carmofur, and PX-12.
<em>In vitro</em> analysis revealed that Ebselen had the strongest potency in reducing the viral load in SARS-CoV-2-infected Vero cells <span class="citation" data-cites="4Sja6dIz">[<a href="#ref-4Sja6dIz" role="doc-biblioref">366</a>]</span>.
Ebselen is an organoselenium compound with anti-inflammatory and antioxidant properties <span class="citation" data-cites="17EErh01r">[<a href="#ref-17EErh01r" role="doc-biblioref">370</a>]</span>
It has been proposed as a possible treatment for conditions ranging from bipolar disorder to diabetes to heart disease <span class="citation" data-cites="17EErh01r">[<a href="#ref-17EErh01r" role="doc-biblioref">370</a>]</span>, and a preliminary investigation of ebselen as a treatment for noise-induced hearing loss provided promising reports of its safety <span class="citation" data-cites="17QBZngoP">[<a href="#ref-17QBZngoP" role="doc-biblioref">371</a>]</span>.
For COVID-19, the NSP5 in SARS-CoV-2 contains a cysteine at the active site of Mpro, and ebselen is able to inactivate the protease by bonding covalently with this cysteine to form a selenosulfide <span class="citation" data-cites="17EErh01r FBE30Uqz">[<a href="#ref-17EErh01r" role="doc-biblioref">370</a>,<a href="#ref-FBE30Uqz" role="doc-biblioref">372</a>]</span>.
Interestingly there has been some argument that selenium deficiency may be associated with more severe COVID-19 outcomes <span class="citation" data-cites="1355hoYr1 bvMB3ySR 11F21sY2C">[<a href="#ref-1355hoYr1" role="doc-biblioref">373</a>,<a href="#ref-bvMB3ySR" role="doc-biblioref">374</a>,<a href="#ref-11F21sY2C" role="doc-biblioref">375</a>]</span>, possibly indicating that its antioxidative properties are protective <span class="citation" data-cites="FBE30Uqz">[<a href="#ref-FBE30Uqz" role="doc-biblioref">372</a>]</span>.
On the other hand, ebselen and the other compounds identified are likely to be promiscuous binders, which could diminish their therapeutic potential <span class="citation" data-cites="4Sja6dIz">[<a href="#ref-4Sja6dIz" role="doc-biblioref">366</a>]</span>.
While there is clear computational and <em>in vitro</em> support for ebselen’s potential as a COVID-19 therapeutic, results from clinical trials are not yet available for this compound.</p>
<p>In summary, N3 is a computationally designed molecule that inhibits the viral transcription through inhibiting Mpro.
Ebselen is both a strong Mpro inhibitor and strong inhibitor of viral replication <em>in vitro</em> that was found to reduce SARS-CoV-2 viral load even more effectively than N3.
Ebselen is a very promising compound since its safety has been demonstrated in other indications.
However, Ebselen is likely a false positive since it is a promiscuous compound that can have many targets <span class="citation" data-cites="vXFGEybh">[<a href="#ref-vXFGEybh" role="doc-biblioref">376</a>]</span>.
Therefore, compounds with higher specificity may be required to effectively translate to clinical trials.</p>
<!--##### Molecules Targeting the Viral Envelope-->
<h3 data-number="5.1.2" id="drugs-targeting-host-proteins"><span class="header-section-number">5.1.2</span> Drugs Targeting Host Proteins</h3>
<p>When a virus enters a host, the host becomes the virus’ environment.
Therefore, the state of the host can also influence whether a virus is able to replicate and spread.
Traditionally, viral targets have been favored because altering host processes is likely to be less specific than targeting the virus directly <span class="citation" data-cites="FT3Nw9xJ">[<a href="#ref-FT3Nw9xJ" role="doc-biblioref">377</a>]</span>.
On the other hand, targeting the host offers potential for a complementary strategy to anti-virals that could broadly limit the ability of viruses to replicate <span class="citation" data-cites="FT3Nw9xJ">[<a href="#ref-FT3Nw9xJ" role="doc-biblioref">377</a>]</span>.
As a result, therapeutic approaches that target host proteins have become an area of interest for SARS-CoV-2.
Viral entry receptors in particular have emerged as a target of interest.
Entry of SARS-CoV-2 into the cell depends on the ACE2 receptor and the enzyme encoded by <em>TMPRSS2</em> <span class="citation" data-cites="15l3di3Wj">[<a href="#ref-15l3di3Wj" role="doc-biblioref">65</a>]</span>.
In principle, drugs that reduce the expression of these proteins or sterically hinder viral interactions with them might reduce viral entry into cells.
Angiotensin-converting enzyme (ACE) inhibitors and angiotensin II receptor blockers (ARBs) are among the most commonly prescribed medications <span class="citation" data-cites="CIeWPFHG pbfxOvnK">[<a href="#ref-CIeWPFHG" role="doc-biblioref">378</a>,<a href="#ref-pbfxOvnK" role="doc-biblioref">379</a>]</span>.
In the United States, for example, they are prescribed well over 100,000,000 times annually.
Data from some animal models suggest that several, but not all, ACE inhibitors and several ARBs increase ACE2 expression in the cells of some organs <span class="citation" data-cites="17xTGqKp2">[<a href="#ref-17xTGqKp2" role="doc-biblioref">380</a>]</span>.
Clinical studies have not established whether plasma ACE2 expression is increased in humans treated with these medications <span class="citation" data-cites="ZhhV4ztO">[<a href="#ref-ZhhV4ztO" role="doc-biblioref">381</a>]</span>.
While randomized clinical trials are ongoing, a variety of observational studies have examined the relationship between exposure to ACE inhibitors or ARBs and outcomes in patients with COVID-19.
An observational study of the association of exposure to ACE inhibitors or ARB with outcomes in COVID-19 was retracted from the <em>New England Journal of Medicine</em> <span class="citation" data-cites="10q7vcaWw">[<a href="#ref-10q7vcaWw" role="doc-biblioref">382</a>]</span>.
Moreover, because observational studies are subject to confounding, randomized controlled trials are the standard means of assessing the effects of medications, and the findings of the various observational studies bearing on this topic cannot be interpreted as indicating a protective effect of the drug <span class="citation" data-cites="19quOF2kK k5xEN3Ke">[<a href="#ref-19quOF2kK" role="doc-biblioref">383</a>,<a href="#ref-k5xEN3Ke" role="doc-biblioref">384</a>]</span>.
Clinical trials testing the effects of ACE inhibitors or ARBs on COVID-19 outcomes are ongoing <span class="citation" data-cites="H0oEjZR8 kuUcA1G9 3ccJAyrG NCb9Wq2z 5fkIarR7 c7V9rAFR">[<a href="#ref-H0oEjZR8" role="doc-biblioref">385</a>,<a href="#ref-kuUcA1G9" role="doc-biblioref">386</a>,<a href="#ref-3ccJAyrG" role="doc-biblioref">387</a>,<a href="#ref-NCb9Wq2z" role="doc-biblioref">388</a>,<a href="#ref-5fkIarR7" role="doc-biblioref">389</a>,<a href="#ref-c7V9rAFR" role="doc-biblioref">390</a>]</span>.
These studies of randomized intervention will provide important data for understanding whether exposure to ACEis or ARBs is associated with COVID-19 outcomes.
Additional information about ACE2, observational studies of ACE inhibitors and ARBs in COVID-19, and clinical trials on this topic have been summarized <span class="citation" data-cites="1Hrsy8Hiq">[<a href="#ref-1Hrsy8Hiq" role="doc-biblioref">391</a>]</span>.</p>
<h3 data-number="5.1.3" id="broad-spectrum-pharmaceuticals"><span class="header-section-number">5.1.3</span> Broad-Spectrum Pharmaceuticals</h3>
<p>Due to the urgent nature of the COVID-19 pandemic, many of the pharmaceutical agents that have been widely publicized as having possible therapeutic or prophylactic effects are broad-spectrum pharmaceuticals that pre-date the COVID-19 pandemic.
These treatments are not specifically targeted at the virus itself or at the host receptors it relies on, but rather induce broad shifts in host biology that are hypothesized to be potential inhibitors of the virus.
In most cases, interest in particular candidate medications or nutraceuticals arises because they are already available for other purposes.
However, the fact that the targets of these agents are non-specific means that the mechanism of action can sound relevant to COVID-19 without a therapeutic or prophylactic effect being observed in rigorous testing.
This category of drugs has received significant attention from the media and general public.</p>
<h4 data-number="5.1.3.1" id="hydroxychloroquine-and-chloroquine"><span class="header-section-number">5.1.3.1</span> Hydroxychloroquine and Chloroquine</h4>
<p>Chloroquine (CQ) and hydroxychloroquine (HCQ) are lysosomotropic agents, meaning they are weak bases that can pass through the plasma membrane.
Both drugs increase cellular pH by accumulating in their protonated form inside lysosomes <span class="citation" data-cites="ggYRK4FU 16IRfGFKb">[<a href="#ref-ggYRK4FU" role="doc-biblioref">392</a>,<a href="#ref-16IRfGFKb" role="doc-biblioref">393</a>]</span>.
This shift in pH inhibits the breakdown of proteins and peptides by the lysosomes during the process of proteolysis <span class="citation" data-cites="16IRfGFKb">[<a href="#ref-16IRfGFKb" role="doc-biblioref">393</a>]</span>.
A number of mechanisms have been proposed through which these drugs could influence the immune response to pathogen challenge.
For example, CQ/HCQ can interfere with digestion of antigens within the lysosome and inhibit CD4 T-cell stimulation while promoting the stimulation of CD8 T-cells <span class="citation" data-cites="16IRfGFKb">[<a href="#ref-16IRfGFKb" role="doc-biblioref">393</a>]</span>.
CQ/HCQ can also decrease the production of certain key cytokines involved in the immune response including IL-6 and inhibit the stimulation of toll-like receptors (TLR) and TLR signaling <span class="citation" data-cites="16IRfGFKb">[<a href="#ref-16IRfGFKb" role="doc-biblioref">393</a>]</span>.
The drugs also have anti-inflammatory and photoprotective effects and may also affect rates of cell death, blood clotting, glucose tolerance, and cholesterol levels <span class="citation" data-cites="16IRfGFKb">[<a href="#ref-16IRfGFKb" role="doc-biblioref">393</a>]</span>.</p>
<p>Interest in CQ and HCQ for treating COVID-19 was catalyzed by a mechanism observed in <em>in vitro</em> studies of both SARS-CoV and SARS-CoV-2.
In one study, CQ inhibited viral entry of SARS-CoV into Vero E6 cells, a cell line that was derived from Vero cells in 1968, through the elevation of endosomal pH and the terminal glycosylation of angiotensin-converting enzyme 2 (ACE2), which is the cellular entry receptor <span class="citation" data-cites="x6nInyCf">[<a href="#ref-x6nInyCf" role="doc-biblioref">394</a>]</span>.
Increased pH within the cell, as discussed above, inhibits proteolysis, and terminal glycosylation of ACE2 is thought to interfere with virus-receptor binding.
An <em>in vitro</em> study of SARS-CoV-2 infection of Vero cells found both HCQ and CQ to be effective in inhibiting viral replication, with HCQ being more potent <span class="citation" data-cites="1GBsnkeLK">[<a href="#ref-1GBsnkeLK" role="doc-biblioref">395</a>]</span>.
Additionally, an early case study of three COVID-19 patients reported the presence of antiphospholipid antibodies in all three patients <span class="citation" data-cites="8OnbWuhF">[<a href="#ref-8OnbWuhF" role="doc-biblioref">97</a>]</span>.
Antiphospholipid antibodies are central to the diagnosis of the antiphospholipid syndrome, a disorder that HCQ has often been used to treat <span class="citation" data-cites="nEaG534R rL8owM5Y 134vCyU3E">[<a href="#ref-nEaG534R" role="doc-biblioref">396</a>,<a href="#ref-rL8owM5Y" role="doc-biblioref">397</a>,<a href="#ref-134vCyU3E" role="doc-biblioref">398</a>]</span>.
Together, these studies triggered initial enthusiasm about the therapeutic potential for HCQ and CQ against COVID-19.
HCQ/CQ has been proposed both as a treatment for COVID-19 and a prophylaxis against SARS-CoV-2 exposure.
HCQ/CQ are sometimes administered with azithromycin (AZ) and/or zinc supplementation.
<!--
rishirajgoel to add text on Azithromycin
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4262884/
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5918160/
https://www.ncbi.nlm.nih.gov/pubmed/28099856
https://www.ncbi.nlm.nih.gov/pubmed/27911847
--></p>
<h5 data-number="5.1.3.1.1" id="therapeutic-administration-of-hcqcq"><span class="header-section-number">5.1.3.1.1</span> Therapeutic Administration of HCQ/CQ</h5>
<p>The initial study evaluating HCQ as a treatment for COVID-19 patients was published on March 20, 2020 by Gautret et al. <span class="citation" data-cites="ovHLMvCi">[<a href="#ref-ovHLMvCi" role="doc-biblioref">399</a>]</span>.
This non-randomized, non-blinded, non-placebo clinical trial compared HCQ to standard of care in 42 hospitalized patients in southern France.
They reported that patients who received HCQ showed higher rates of virological clearance by nasopharyngeal swab on Days 3-6 when compared to standard care.
This study also treated six patients with both HCQ + AZ and found this combination therapy to be more effective than HCQ alone.
However, the design and analyses used showed weaknesses that severely limit interpretability of results, including the lack of randomization, lack of blinding, lack of placebo, lack of Intention-To-Treat analysis, lack of correction for sequential multiple comparisons, trial arms entirely confounded by hospital, false negatives in outcome measurements, lack of trial pre-registration, and small sample size.
Two of these weaknesses are due to inappropriate data analysis and can therefore be corrected <em>post hoc</em> by recalculating the p-values (lack of Intention-To-Treat analysis and multiple comparisons.)
However, all other weaknesses are fundamental design flaws and cannot be corrected for.
Thus, conclusions cannot be generalized outside of the study.
The International Society of Antimicrobial Chemotherapy, the scientific organization that publishes <em>International Journal of Antimicrobial Agents</em> where the article appeared, has announced that the article does not meet its expected standard for publications <span class="citation" data-cites="10J3hBiMo">[<a href="#ref-10J3hBiMo" role="doc-biblioref">400</a>]</span>, although it has not been officially retracted.
Because of the preliminary data presented in this study, the use of HCQ in COVID-19 treatment has subsequently been explored by other researchers.
About one week later, a follow-up case study reported that 11 consecutive patients were treated with HCQ + AZ using the same dosing regimen <span class="citation" data-cites="18KmvBOFr">[<a href="#ref-18KmvBOFr" role="doc-biblioref">401</a>]</span>.
One patient died, two were transferred to the ICU, and one developed a prolonged QT interval, leading to discontinuation of HCQ + AZ.
As in the Gautret et al. study, the outcome assessed was virological clearance at Day 6 post-treatment, as measured in nasopharyngeal swabs.
Of the ten living patients on Day 6, eight remained positive for SARS-CoV-2 RNA.
Like in the original study, interpretability was severely limited by lack of comparison group and the small sample size.
However, these results stand in contrast to the claims by Gautret et al. that all six patients treated with HCQ + AZ tested negative for SARS-CoV-2 RNA by Day 6 post-treatment.
This case study illustrated the need for further investigation using robust study design before the original proposed benefits of HCQ could be widely accepted.</p>
<p>On April 10, 2020, a randomized, non-placebo trial of 62 COVID-19 patients at the Renmin Hospital of Wuhan University was released <span class="citation" data-cites="wzHb7Mj5">[<a href="#ref-wzHb7Mj5" role="doc-biblioref">402</a>]</span>.
This study investigated whether HCQ decreased time to fever break or time to cough relief when compared to standard of care <span class="citation" data-cites="wzHb7Mj5">[<a href="#ref-wzHb7Mj5" role="doc-biblioref">402</a>]</span>.
This trial found HCQ decreased both average time to fever break and average time to cough relief, defined as mild or no cough.
While this study improved on some of the methodological flaws in Gautret et al. by randomizing patients, it also had several flaws in trial design and data analysis that prevent generalization of the results.
These weaknesses include the lack of placebo, lack of correction for multiple primary outcomes, inappropriate choice of outcomes, lack of sufficient detail to understand analysis, drastic disparities between pre-registration and published protocol, and small sample size.
The choice of outcomes may be inappropriate as both fevers and cough may break periodically without resolution of illness.
Additionally, for these outcomes, the authors reported that 23 of 62 patients did not have a fever and 25 of 62 patients did not have a cough at the start of the study, but the authors failed to describe how these patients were included in a study assessing time to fever break and time to cough relief.
It is important to note here that the authors claimed “neither the research performers nor the patients were aware of the treatment assignments.”
This blinding seems impossible in a non-placebo trial because at the very least, providers would know whether they were administering a medication or not, and this knowledge could lead to systematic differences in the administration of care.
Correction for multiple primary outcomes can be adjusted <em>post hoc</em> by recalculating p-values, but all of the other issues were design and statistical weaknesses that cannot be corrected for.
Additionally, the observation of drastic disparities between pre-registration and published protocol could indicate p-hacking <span class="citation" data-cites="XdCJ4u8O">[<a href="#ref-XdCJ4u8O" role="doc-biblioref">403</a>]</span>.
The design limitations mean that the conclusions cannot be generalized outside of the study.
A second randomized trial, conducted by the Shanghai Public Health Clinical Center, analyzed whether HCQ increased rates of virological clearance at day 7 in respiratory pharyngeal swabs compared to standard care <span class="citation" data-cites="o5gwQA7W">[<a href="#ref-o5gwQA7W" role="doc-biblioref">404</a>]</span>.
This trial was published in Chinese along with an abstract in English, and only the English abstract was read and interpreted for this review.
The trial found comparable outcomes in virological clearance rate, time to virological clearance, and time to body temperature normalization between the treatment and control groups.
A known weakness is small sample size, with only 30 patients enrolled and 15 in each arm.
This problem suggests the study is underpowered to detect potentially useful differences and precludes interpretation of results.
Additionally, because only the abstract could be read, other design and analysis issues could be present.
Thus, though these studies added randomization to their assessment of HCQ, their conclusions should be interpreted very cautiously.
These two studies assessed different outcomes and reached differing conclusions about the efficacy of HCQ for treating COVID-19; the designs of both studies, especially with respect to sample size, meant that no general conclusions can be made about the efficacy of the drug.</p>
<p>Several widely reported studies on HCQ have issues with data integrity and/or provenance.
A Letter to the Editor published in <em>BioScience Trends</em> on March 16, 2020 claimed that numerous clinical trials have shown that HCQ is superior to control treatment in inhibiting the exacerbation of COVID-19 pneumonia <span class="citation" data-cites="aiItPxSY">[<a href="#ref-aiItPxSY" role="doc-biblioref">405</a>]</span>.
This letter has been cited by numerous primary literature, review articles, and media alike <span class="citation" data-cites="1BdQKtXnY z8XnLYWM">[<a href="#ref-1BdQKtXnY" role="doc-biblioref">406</a>,<a href="#ref-z8XnLYWM" role="doc-biblioref">407</a>]</span>.
However, the letter referred to 15 pre-registration identifiers from the Chinese Clinical Trial Registry.
When these identifiers are followed back to the registry, most trials claim they are not yet recruiting patients or are currently recruiting patients.
For all of these 15 identifiers, no data uploads or links to publications could be located on the pre-registrations.
At the very least, the lack of availability of the primary data means the claim that HCQ is efficacious against COVID-19 pneumonia cannot be verified.
Similarly, a recent multinational registry analysis <span class="citation" data-cites="1BApBNHvd">[<a href="#ref-1BApBNHvd" role="doc-biblioref">408</a>]</span> analyzed the efficacy of CQ and HCQ with and without a macrolide, which is a class of antibiotics that includes Azithromycin, for the treatment of COVID-19.
The study observed 96032 patients split into a control and four treatment conditions (CQ with and without a macrolide; HCQ with and without a macrolide).
They concluded that treatment with CQ and HCQ was associated with increased risk of <em>de novo</em> ventricular arrhythmia during hospitalization.
However, this study has since been retracted by <em>The Lancet</em> due to an inability to validate the data used <span class="citation" data-cites="11FsxWalS">[<a href="#ref-11FsxWalS" role="doc-biblioref">409</a>]</span>.
These studies demonstrate that increased skepticism in evaluation of the HCQ/CQ and COVID-19 literature may be warranted, possible because of the significant attention HCQ and CQ have received as possible treatments for COVID-19 and the politicization of these drugs.</p>
<p>Despite the fact that the study suggesting that CQ/HCQ increased risk of ventricular arrhythmia in COVID-19 patients has now been retracted, previous studies have identified risks with HCQ/CQ.
A patient with systemic lupus erythematosus developed a prolonged QT interval that was likely exacerbated by use of HCQ in combination with renal failure <span class="citation" data-cites="dY9kXLAS">[<a href="#ref-dY9kXLAS" role="doc-biblioref">410</a>]</span>.
A prolonged QT interval is associated with ventricular arrhythmia <span class="citation" data-cites="wy3S7Cpa">[<a href="#ref-wy3S7Cpa" role="doc-biblioref">411</a>]</span>.
Furthermore, a separate study <span class="citation" data-cites="OQ0b31BA">[<a href="#ref-OQ0b31BA" role="doc-biblioref">412</a>]</span> investigated the safety associated with the use of HCQ with and without macrolides between 2000 and 2020.
The study involved 900,000 cases treated with HCQ and 300,000 cases treated with HCQ + AZ.
The results indicated that short-term use of HCQ was not associated with additional risk, but that HCQ + AZ was associated with an enhanced risk of cardiovascular complications (15-20% increased risk of chest pain) and a two-fold increased risk of mortality.
Therefore, whether studies utilize HCQ alone or HCQ in combination with a macrolide may be an important consideration in assessing risk.
As results from initial investigations of these drug combinations have emerged, concerns about the efficacy and risks of treating COVID-19 with HCQ and CQ has led to the removal of CQ/HCQ from standard of care practices in several countries <span class="citation" data-cites="1CgZR55GZ 1EcscrT32">[<a href="#ref-1CgZR55GZ" role="doc-biblioref">413</a>,<a href="#ref-1EcscrT32" role="doc-biblioref">414</a>]</span>.
As of May 25, 2020, WHO had suspended administration of HCQ as part of the worldwide Solidarity Trial <span class="citation" data-cites="AMQ5402v">[<a href="#ref-AMQ5402v" role="doc-biblioref">415</a>]</span>.</p>
<p>As additional research has emerged, HCQ/CQ have increasingly been demonstrated to be ineffective against COVID-19 and to carry a number of significant side effects.
A randomized, open-label, non-placebo trial of 150 COVID-19 patients was conducted in parallel at 16 government-designated COVID-19 centers in China to assess the safety and efficacy of HCQ <span class="citation" data-cites="RXr9BKKT">[<a href="#ref-RXr9BKKT" role="doc-biblioref">416</a>]</span>.
The trial compared treatment with HCQ in conjunction with standard of care (SOC) to SOC alone in 150 infected patients who were assigned randomly to the two groups (75 per group).
The primary endpoint of the study was the negative conversion rate of SARS-CoV-2 in 28 days, and the investigators found no difference in this parameter between the groups.
The secondary endpoints were an amelioration of the symptoms of the disease such as axillary temperature ≤36.6°C, SpO2 &gt;94% on room air, and disappearance of symptoms like shortness of breath, cough, and sore throat.
The median time to symptom alleviation was similar across different conditions (19 days in HCQ+SOC vs. 21 days in SOC).
Additionally, 30% of the patients receiving SOC+HCQ reported adverse outcomes compared to 8.8% of patients receiving only SOC, with the most common adverse outcome in the SOC+HCQ group being diarrhea (10% vs. 0% in the SOC group, p=0.004).
Furthermore, there are several factors that limit the interpretability of this study.
Most of the enrolled patients had mild-to-moderate symptoms (98%), and the average age was 46.
SOC in this study included the use of antivirals (Lopinavir-Ritonavir, Arbidol, Oseltamivir, Virazole, Entecavir, Ganciclovir, and Interferon alfa), which appeared to introduce confounding effects.
Thus, to isolate the effect of HCQ, SOC would need to exclude the use of antivirals.
In this trial, the samples used to test for the presence of the SARS-CoV-2 virus were collected from the upper respiratory tract, and the authors indicated that the use of upper respiratory samples may have introduced false negatives (e.g., <span class="citation" data-cites="vr7AH83b">[<a href="#ref-vr7AH83b" role="doc-biblioref">77</a>]</span>); thus, the identification of biomarkers that can be collected non-invasively would be valuable to studies such as this one.
Another limitation of the study that the authors acknowledge was that the HCQ treatment began, on average, at a 16-day delay from the symptom onset.
The fact that this study was open-label and lacked a placebo limits interpretation, and additional analysis is required to determine whether HCQ reduces inflammatory response.
Therefore, despite some potential areas of investigation identified in <em>post hoc</em> analysis, this study cannot be interpreted as providing support for HCQ as a therapeutic against COVID-19.</p>
<p>Additional evidence comes from a retrospective analysis <span class="citation" data-cites="MENbRfOl">[<a href="#ref-MENbRfOl" role="doc-biblioref">417</a>]</span> that examined data from 368 COVID-19 patients across all United States Veteran Health Administration medical centers.
The study retrospectively investigated the effect of the administration of HCQ (n=97), HCQ + AZ (n=113), and no HCQ (n=158) on 368 patients.
The primary outcomes assessed were death and the need for mechanical ventilation.
Standard supportive care was rendered to all patients.
Due to the low representation of women (N=17) in the available data, the study included only men, and the median age was 65 years.
The rate of death in the HCQ-only treatment condition was 27.8% and in the HCQ + AZ treatment condition, it was 22.1%.
In comparison to the 14.1% rate of death in the no-HCQ cohort, these data indicated a statistically significant elevation in the risk of death for the HCQ-only group compared to the no-HCQ group (adjusted hazard ratio: 2.61, p=0.03), but not for the HCQ + AZ group compared to the no-HCQ group (adjusted hazard ratio: 1.14; p=0.72).
Further, the risk of ventilation was similar across all three groups (adjusted hazard ratio: 1.43, p=0.48 (HCQ) and 0.43, p=0.09 (HCQ + AZ) compared to no HCQ).
The study thus showed evidence of an association between increased mortality and HCQ in this cohort of COVID-19 patients but no change in rates of mechanical ventilation among the treatment conditions.
The study had a few limitations: it was not randomized, and the baseline vital signs, laboratory tests, and prescription drug use were significantly different among the three groups.
All of these factors could potentially influence treatment outcome.
Furthermore, the authors acknowledge that the effect of the drugs might be different in females and pediatric subjects, since these subjects were not part of the study.
The reported result that HCQ + AZ is safer than HCQ contradicts the findings of the previous large-scale analysis of twenty years of records that found HCQ + AZ to be more frequently associated with cardiac arrhythmia than HCQ alone <span class="citation" data-cites="OQ0b31BA">[<a href="#ref-OQ0b31BA" role="doc-biblioref">412</a>]</span>; whether this discrepancy is caused by the pathology of COVID-19, is influenced by age or sex, or is a statistical artifact is not presently known.</p>
<p>Finally, findings from the Randomized Evaluation of COVID-19 Therapy (RECOVERY) trial were released on October 8, 2020.
This study used a randomized, open-label design to study the effects of HCQ compared to standard care at 176 hospitals in the United Kingdom <span class="citation" data-cites="KwFZyfFx">[<a href="#ref-KwFZyfFx" role="doc-biblioref">418</a>]</span>.
This large study enrolled 11,197 hospitalized patients whose physicians believed it would not harm them to participate.
Patients were randomized into either the control group or one of the treatment arms, with twice as many patients enrolled in the control group as any treatment group.
Of the patients eligible to receive HCQ, 1,561 were randomized into the HCQ arm while 3,155 were randomized into the control arm.
The demographics of the HCQ and control groups were similar in terms of average age (65 years), proportion female (approximately 38%), ethnic make-up (73% versus 76% white), and prevalence of pre-existing conditions (56% versus 57% overall).
In the HCQ arm of the study, patients received 800 mg at baseline and again after 6 hours, then 400 mg at 12 hours and every subsequent 12 hours.
The primary outcome analyzed was all-cause mortality, and patient vital statistics were reported by physicians upon discharge or death, or else at 28 days following HCQ administration if they remained hospitalized.
The secondary outcome assessed was the combined risk of progression to invasive mechanical ventilation or death within 28 days.
By the advice of an external data monitoring committee, the HCQ arm of the study was reviewed early, leading to it being closed due a lack of support for HCQ as a treatment for COVID-19.
Patients who received HCQ had a longer duration of hospitalization than patients receiving usual care, were less likely to be discharged alive within 28 days, and were more likely to progress to mechanical ventilation.
This large-scale study thus builds upon studies in the United States and China to suggest that HCQ is not an effective treatment, and in fact may negatively impact COVID-19 patients due to its side effects.
The rates of COVID-19-related reported in the RECOVERY trial did not differ between the control and HCQ arms, but patients receiving HCQ were more likely to die due to cardiac events.
Therefore, though none of the studies have been blinded, taken together it becomes clear that all of the available evidence points to significant concerns about the administration of HCQ to hospitalized COVID-19 patients, without providing any support for its efficacy.</p>
<h5 data-number="5.1.3.1.2" id="hcq-for-the-treatment-of-mild-cases"><span class="header-section-number">5.1.3.1.2</span> HCQ for the Treatment of Mild Cases</h5>
<p>One additional possible therapeutic application considered was the treatment of mild COVID-19 cases in otherwise healthy individuals.
This possibility was assessed in a randomized, open-label, multi-center analysis conducted in Catalonia (Spain) <span class="citation" data-cites="V8fkkVZg">[<a href="#ref-V8fkkVZg" role="doc-biblioref">419</a>]</span>.
This analysis enrolled adults 18 and older who had been experiencing mild symptoms of COVID-19 for fewer than five days.
Participants were randomized into an HCQ arm (N=136) and a control arm (N=157), and those in the treatment arm were administered 800 mg of HCQ on the first day of treatment followed by 400 mg on each of the subsequent six days.
The primary outcome assessed was viral clearance at days 3 and 7 following the onset of treatment, and secondary outcomes were clinical progression and time to complete resolution of symptoms.
They found no significant differences between the two groups.
This study thus suggests that COVID-19 does not improve recovery from COVID-19, even in otherwise healthy adult patients with mild symptoms.</p>
<h5 data-number="5.1.3.1.3" id="prophylactic-administration-of-hcq"><span class="header-section-number">5.1.3.1.3</span> Prophylactic Administration of HCQ</h5>
<p>An initial study of the possible prophylactic application of HCQ utilized a randomized, double-blind, placebo-controlled design to analyze the administration of HCQ prophylactically <span class="citation" data-cites="NiVdIN7Z">[<a href="#ref-NiVdIN7Z" role="doc-biblioref">420</a>]</span>.
Asymptomatic adults in the United States and Canada who had been exposed to SARS-CoV-2 within the past four days were enrolled in an online study to see whether administration of HCQ over five days would influence the probability of developing COVID-19 symptoms over a 14-day period.
Of the participants, 414 received HCQ and 407 received a placebo.
The participants averaged 40 years of age and 51.6% were women.
They found no significant difference in the rate of symptomatic illness between the two groups (11.8% HCQ, 14.3% placebo, p=0.35).
The HCQ condition was associated with side effects, with 40.1% of patients reporting side effects compared to 16.8% in the control group (p&lt;0.001).
However, likely due to the high enrollment of healthcare workers (66% of participants) and the well-known side effects associated with HCQ, a large number of participants were able to correctly identify whether they were receiving HCQ or a placebo (46.5% and 35.7%, respectively).
Furthermore, due to a lack of availability of diagnostic testing, only 20 of the 107 cases were confirmed with a PCR-based test to be positive for SARS-CoV-2.
The rest were categorized as “probable” or “possible” cases by a panel of four physicians who were blind to the treatment status.
One possible confounder is that a patient presenting one or more symptoms, which included diarrhea, was defined as a “possible” case, but diarrhea is also a common side effect of HCQ.
Additionally, four of the twenty PCR-confirmed cases did not develop symptoms until after the observation period had completed, suggesting that the 14-day trial period may not have been long enough or that some participants also encountered secondary exposure events.
Finally, in addition to the young age of the participants in this study, which ranged from 32 to 51, there were possible impediments to generalization introduced by the selection process, as 2,237 patients who were eligible but had already developed symptoms by day 4 were enrolled in a separate study.
It is therefore likely that asymptomatic cases were over-represented in this sample, which would not have been detected based on the diagnostic criteria used.
Therefore, while this study does represent the first effort to conduct a randomized, double-blind, placebo-controlled investigation of HCQ’s effect on COVID-19 symptoms in a large sample, the lack of PCR tests and several other design flaws significantly impedes interpretation of the results.
However, in line with the results from therapeutic studies, there is no evidence that HCQ provides protection against COVID-19.</p>
<p>A second study <span class="citation" data-cites="oaqFxV4W">[<a href="#ref-oaqFxV4W" role="doc-biblioref">421</a>]</span> examined the effect of administering HCQ to healthcare workers as a pre-exposure prophylactic.
The primary outcome assessed was the conversion from SARS-CoV-2 negative to SARS-CoV-2 positive status over the 8 week study period.
This study was also randomized, double-blind, and placebo-controlled and sough to address some of the limitations of the first prophylactic study.
They sought to enroll 200 healthcare workers, preferentially those working with COVID-19 patients, at two hospitals within the University of Pennsylvania hospital system in Philadelphia, PA.
Participants were randomized 1:1 to receive either 600 mg of HCQ daily or a placebo, and their SARS-CoV-2 infection status and antibody status were assessed using RT-PCR and serological testing, respectively, at baseline, 4 weeks, and 8 weeks following the beginning of the treatment period.
The statistical design of the study accounted for interim analyses at 50 and 100 participants in case efficacy or futility of HCQ for prophylaxis became clear earlier than completion of enrollment.
The 139 individuals enrolled comprised a study population that was fairly young (average age 33) and made of largely of people who were white, women, and without pre-existing conditions.
At the second interim analysis, more individuals in the treatment group than the control group had contracted COVID-19 (4 versus 3), causing the estimate z-score to fall below the pre-established threshold for futility.
As a result, the trial was terminated early, offering additional evidence against the use of HCQ for prophylaxis.</p>
<h5 data-number="5.1.3.1.4" id="summary"><span class="header-section-number">5.1.3.1.4</span> Summary</h5>
<p>Early <em>in vitro</em> evidence indicated that HCQ could be an effective therapeutic against SARS-CoV-2 and COVID-19, leading to early interest in its potential as both a therapeutic and prophylaxis.
Because the 90% effective concentration (EC<sub>90</sub>) of CQ in Vero E6 cells (6.90 μM) can be achieved in and tolerated by rheumatoid arthritis patients, it was hypothesized that it might also be possible to achieve the effective concentration in COVID-19 patients <span class="citation" data-cites="WFGcbZhe">[<a href="#ref-WFGcbZhe" role="doc-biblioref">422</a>]</span>.
Additionally, HCQ has been found to be effective in treating HIV <span class="citation" data-cites="hFGjQQN8">[<a href="#ref-hFGjQQN8" role="doc-biblioref">423</a>]</span> and chronic Hepatitis C <span class="citation" data-cites="rQNzKxRd">[<a href="#ref-rQNzKxRd" role="doc-biblioref">424</a>]</span>.
Therefore, initially it was hypothesized that CQ/HCQ might be effective against SARS-CoV-2, especially since CQ and HCQ have both been found to inhibit the expression of CD154 in T-cells and to reduce toll-like receptor signaling that leads to the production of pro-inflammatory cytokines <span class="citation" data-cites="j3rGUH1f">[<a href="#ref-j3rGUH1f" role="doc-biblioref">425</a>]</span>.
Clinical trials for COVID-19 have more often used HCQ rather than CQ because it offers the advantages of being cheaper and having fewer side effects than CQ.
Several countries have removed CQ from their standard of care for COVID-19 due to the frequency of adverse effects.
However, a large analysis of patients receiving HCQ from January 2000 through March 2020 reported that the combination of HCQ and azithromycin, but not other macrolides, was associated with an elevated risk of cardiovascular complications and mortality.
Multiple clinical studies have already been carried out to assess HCQ as a therapeutic agent for COVID-19, and many more are in progress.
To date, none of these studies have used randomized, double-blind, placebo-controlled designs with a large sample size, which would be the gold standard.
Despite the design limitations (which would be more likely to produce false positives than false negatives), initial optimism about HCQ has not been substantiated by these studies.
The most methodologically rigorous analysis to date <span class="citation" data-cites="NiVdIN7Z">[<a href="#ref-NiVdIN7Z" role="doc-biblioref">420</a>]</span> evaluated the prophylactic potential of HCQ and found no significant differences between the treatment and control groups.
Thus, HCQ/CQ are not likely to be effective therapeutic or prophylactic agents against COVID-19.
Additionally, one study identified an increased risk of mortality in older men receiving HCQ, and administration of HCQ and HCQ+AZ did not decrease the use of mechanical ventilation in these patients <span class="citation" data-cites="MENbRfOl">[<a href="#ref-MENbRfOl" role="doc-biblioref">417</a>]</span>.
HCQ use for COVID-19 also leads to shortages for anti-malarial or anti-rheumatic use, where it has been definitively proven to be effective.
Despite significant attention, these drugs appear to be ineffective against COVID-19.</p>
<h4 data-number="5.1.3.2" id="dexamethasone"><span class="header-section-number">5.1.3.2</span> Dexamethasone</h4>
<p>Dexamethasone (9α-fluoro-16α-methylprednisolone) is a synthetic corticosteroid that binds to the glucocorticoid receptors <span class="citation" data-cites="q0TqfpEr 2M8wLShq">[<a href="#ref-q0TqfpEr" role="doc-biblioref">426</a>,<a href="#ref-2M8wLShq" role="doc-biblioref">427</a>]</span>.
It was first synthesized in the late 1950s as an anti-inflammatory and has been used to treat rheumatoid arthritis and other inflammatory conditions <span class="citation" data-cites="KQ7uhAaa bSNVCQbh">[<a href="#ref-KQ7uhAaa" role="doc-biblioref">428</a>,<a href="#ref-bSNVCQbh" role="doc-biblioref">429</a>]</span>.
Steroids such as dexamethasone are widely available and affordable, and they are often used to treat community-acquired pneumonia <span class="citation" data-cites="19dEKfvUE">[<a href="#ref-19dEKfvUE" role="doc-biblioref">430</a>]</span>.
A clinical trial that began in 2012 recently reported that dexamethasone may improve outcomes for patients with ARDS <span class="citation" data-cites="1DUXbHUEW">[<a href="#ref-1DUXbHUEW" role="doc-biblioref">431</a>]</span>.
However, meta-analysis of a small amount of available data regarding the use of dexamethasone to treat SARS suggested that it may, in fact, be associated with patient harm <span class="citation" data-cites="Ji2fchyS">[<a href="#ref-Ji2fchyS" role="doc-biblioref">432</a>]</span>, although these findings may have been biased by the fact that all of the studies examined were observational and a large number of inconclusive studies were not included <span class="citation" data-cites="xndHds7d">[<a href="#ref-xndHds7d" role="doc-biblioref">433</a>]</span>.
Dexamethasone works as an anti-inflammatory by binding to glucocorticoid receptors with higher affinity than endogenous cortisol <span class="citation" data-cites="1ETGa7JyL">[<a href="#ref-1ETGa7JyL" role="doc-biblioref">434</a>]</span>.
In order to understand how dexamethasone works to reduce inflammation, it is necessary to consider the stress response more broadly.
In response to stress, corticotropin‐releasing hormone stimulates the release of neurotransmitters known as catecholamines, such as epinephrine, and steroid hormones known as glucocorticoids, such as cortisol <span class="citation" data-cites="jMzODpwS Riev3em1">[<a href="#ref-jMzODpwS" role="doc-biblioref">435</a>,<a href="#ref-Riev3em1" role="doc-biblioref">436</a>]</span>.
While catecholamines are often associated with the characteristic fight-or-flight response, the specific role that glucocorticoids play is less clear, although they are thought to be important to restoring homeostasis <span class="citation" data-cites="JZyVNRCN">[<a href="#ref-JZyVNRCN" role="doc-biblioref">437</a>]</span>.
Immune challenge is a stressor that is known to interact closely with the stress response.
The immune system can therefore interact with the central nervous system; for example, macrophages can both respond to and produce catecholamines <span class="citation" data-cites="jMzODpwS">[<a href="#ref-jMzODpwS" role="doc-biblioref">435</a>]</span>.
Additionally, the production of both catecholamines and glucocorticoids is associated with inhibition of proinflammatory cytokines such as IL-6, IL-12, and TNF‐α and the stimulation of anti-inflammatory cytokines such as IL-10, meaning that the stress response can regulate inflammatory immune activity <span class="citation" data-cites="Riev3em1">[<a href="#ref-Riev3em1" role="doc-biblioref">436</a>]</span>.
Administration of dexamethasone has been found to correspond to dose-dependent inhibition of IL-12 production, but not to affect IL-10 <span class="citation" data-cites="SiXrQUgV">[<a href="#ref-SiXrQUgV" role="doc-biblioref">438</a>]</span>; the fact that this relationship could be disrupted by administration of a glucocorticoid-receptor antagonist suggests that it is regulated by the receptor itself <span class="citation" data-cites="SiXrQUgV">[<a href="#ref-SiXrQUgV" role="doc-biblioref">438</a>]</span>. <!-- Note: waiting for full text of this paper, this is based on abstract and citations-->
Thus, the administration of dexamethasone for COVID-19 is likely to simulate the release of glucocorticoids endogenously during stress, resulting in binding of the synthetic steroid to the glucocorticoid receptor and the associated inhibition of the production of proinflammatory cytokines.
In this model, dexamethasone reduces inflammation by stimulating the mechanism used to reduce inflammation following a threat such as immune challenge.
Immunosuppressive drugs such as steroids are typically contraindicated in the setting of infection <span class="citation" data-cites="nbiwprc">[<a href="#ref-nbiwprc" role="doc-biblioref">439</a>]</span>, but because COVID-19 results in hyperinflammation that appears to contribute to mortality via lung damage, immunosuppression may be a helpful approach to treatment <span class="citation" data-cites="3HdlV9Vf">[<a href="#ref-3HdlV9Vf" role="doc-biblioref">121</a>]</span>.
The decision of whether and/or when to counter hyperinflammation with immunosuppression in the setting of COVID-19 was an area of intense debate, as the risks of inhibiting antiviral immunity needed to be weighed against the beneficial anti-inflammatory effects <span class="citation" data-cites="suHq6Qnm">[<a href="#ref-suHq6Qnm" role="doc-biblioref">440</a>]</span>.
As a result, guidelines early in the pandemic typically recommended avoiding treating COVID-19 patients with corticosteroids such as dexamethasone <span class="citation" data-cites="Ji2fchyS">[<a href="#ref-Ji2fchyS" role="doc-biblioref">432</a>]</span>.</p>
<p>The application of dexamethasone for the treatment of COVID-19 was evaluated as part of the multi-site RECOVERY trial in the United Kingdom <span class="citation" data-cites="pzQoUwz3">[<a href="#ref-pzQoUwz3" role="doc-biblioref">441</a>]</span>.
Over 6,000 hospitalized COVID-19 patients were assigned into standard care or treatment (dexamethasone) arms of the trial with a 2:1 ratio.
At the time of randomization, some patients were ventilated (16%), others were on non-invasive oxygen (60%), and others were breathing independently (24%).
Patients in the treatment arm were administered dexamethasone either orally or intravenously at 6 mg per day for up to 10 days.
The primary end-point was the patient’s status at 28-days post-randomization (mortality, discharge, or continued hospitalization), and secondary outcomes analyzed included the progression to invasive mechanical ventilation over the same period.
The 28-day mortality rate was found to be lower in the treatment group than in the standard care group (21.6% vs 24.6%, p &lt;0.001).
However, this finding was driven by differences in mortality among patients who were receiving mechanical ventilation or supplementary oxygen at the start of the study.
The report indicated that dexamethasone reduced 28-day mortality relative to standard care in patients who were ventilated (29.3% vs. 41.4%) and among those who were receiving oxygen supplementation (23.3% vs. 26.2%) at randomization, but not in patients who were breathing independently (17.8% vs. 14.0%).
One possible confounder is that patients receiving mechanical ventilation tended to be younger than patients who were not receiving respiratory support (by 10 years on average) and to have had symptoms for a longer period.
However, adjusting for age did not change their conclusions, although the duration of symptoms was found to be significantly associated with the effect of dexamethasone administration.
These findings also suggested that dexamethasone may have reduced progression to mechanical ventilation, especially among patients who were receiving oxygen support at randomization.
Thus, this large, randomized, and multi-site, albeit not placebo-controlled, study suggests that administration of dexamethasone to patients who are unable to breathe independently may significantly improve survival outcomes.
Additionally, dexamethasone is a widely available and affordable medication, raising the hope that it could be made available to COVID-19 patients globally.</p>
<p>The results of the RECOVERY trial’s analysis of dexamethasone suggest that this therapeutic is effective primarily in patients who had been experiencing symptoms for at least seven days and patients who were not breathing independently <span class="citation" data-cites="H3C2PalE">[<a href="#ref-H3C2PalE" role="doc-biblioref">442</a>]</span>.
Thus, it seems likely that dexamethasone is useful for treating inflammation associated with immunopathy or cytokine release syndrome.
In fact, corticosteroids such as dexamethasone are sometimes used to treat CRS <span class="citation" data-cites="H70g2GF9">[<a href="#ref-H70g2GF9" role="doc-biblioref">443</a>]</span>.
It is not surprising that administration of an immunosuppressant would be most beneficial when the immune system was dysregulated towards inflammation.
However, it is also unsurprising that care must be taken in administering an immunosuppressant to patients fighting a viral infection.
In particular, the concern has been raised that treatment with dexamethasone might increase patient susceptibility to concurrent (e.g., nosocomial) infections <span class="citation" data-cites="EgvIETTq">[<a href="#ref-EgvIETTq" role="doc-biblioref">444</a>]</span>.
Additionally, the drug could potentially slow viral clearance and inhibit patients’ ability to develop antibodies to SARS-CoV-2 <span class="citation" data-cites="EgvIETTq Ji2fchyS">[<a href="#ref-Ji2fchyS" role="doc-biblioref">432</a>,<a href="#ref-EgvIETTq" role="doc-biblioref">444</a>]</span>.
Furthermore, dexamethasone has been associated with side effects that include psychosis, glucocorticoid-induced diabetes, and avascular necrosis <span class="citation" data-cites="Ji2fchyS">[<a href="#ref-Ji2fchyS" role="doc-biblioref">432</a>]</span>, and the RECOVERY trial did not report outcomes with enough detail to be able to determine whether they observed similar complications.
However, since these results were released, strategies have been proposed for administering dexamethasone alongside more targeted treatments to minimize the likelihood of negative side effects <span class="citation" data-cites="EgvIETTq">[<a href="#ref-EgvIETTq" role="doc-biblioref">444</a>]</span>.
Given the available evidence, dexamethasone is currently one of the most promising treatments for severe COVID-19.</p>
<!--#### Toll-Like Receptor Agonists and Antagonists
Note: I think this section was added elsewhere, so commenting out to be moved or removed

Due to the relationship between coronaviral infections, ARDS, and the toll-like receptors TLR3 and TLR4, TLR agonists and antagonists hold some potential for broad-spectrum prophylaxis.
In mouse models, poly(I:C) and lipopolysaccharide, which are agonists of TLR3 and TLR4, respectively, showed protective effects when administered prior to SARS-CoV exposure [@doi:10.1128/JVI.01410-12].
-->
<h3 data-number="5.1.4" id="nutraceuticals-and-dietary-supplements"><span class="header-section-number">5.1.4</span> Nutraceuticals and Dietary Supplements</h3>
<p>Given the current pandemic, scientists and the medical community are scrambling to repurpose or discover novel host-directed therapies.
For the general public in particular, whether nutraceuticals or dietary supplements can provide any prophylactic or therapeutic benefit has been a topic of interest.
Nutraceuticals are classified as supplements with health benefits beyond their basic nutritional value <span class="citation" data-cites="xLPeszeO 2Us0gNMr">[<a href="#ref-xLPeszeO" role="doc-biblioref">445</a>,<a href="#ref-2Us0gNMr" role="doc-biblioref">446</a>]</span>.
The key difference between a dietary supplement and a nutraceutical is that nutraceuticals should not only supplement the diet, but also aid in the prophylaxis and/or treatment of a disorder or disease <span class="citation" data-cites="X0jEogv4">[<a href="#ref-X0jEogv4" role="doc-biblioref">447</a>]</span>.
Unlike pharmaceuticals, nutraceuticals do not entirely fall under the responsibility of the FDA, but they are monitored as dietary supplements according to the Dietary Supplement, Health and Education Act 1994 (DSHEA) <span class="citation" data-cites="10Tnnl6nF">[<a href="#ref-10Tnnl6nF" role="doc-biblioref">448</a>]</span> and the Food and Drug Administration Modernization Act 1997 <span class="citation" data-cites="16eEQf0HM">[<a href="#ref-16eEQf0HM" role="doc-biblioref">449</a>]</span>.
Due to the increased sales of dietary supplements and nutraceuticals, the FDA established the Office of Dietary Supplement Programs (ODSP) in 1996 to increase surveillance.
However, there is significant concern that these acts do not adequately protect the consumer as they ascribe responsibility on the manufacturers to ensure safety of the product before manufacturing or marketing <span class="citation" data-cites="5TmtJWVQ">[<a href="#ref-5TmtJWVQ" role="doc-biblioref">450</a>]</span>.
Likewise, manufacturers are not required to register or seek approval from the FDA to produce or sell food supplements or nutraceuticals.
Health or nutrient content claims for labeling purposes are approved based on an authoritative statement from the Academy of Sciences or relevant federal authorities once the FDA has been notified and on the basis that the information is known to be true and not deceptive <span class="citation" data-cites="5TmtJWVQ">[<a href="#ref-5TmtJWVQ" role="doc-biblioref">450</a>]</span>.
In Europe, health claims are permitted on a product label only following compliance and authorization according to the European Food Safety Authority (EFSA) guidelines on nutrition and health claims <span class="citation" data-cites="dzOveIQY">[<a href="#ref-dzOveIQY" role="doc-biblioref">451</a>]</span>.
EFSA does not currently distinguish between food supplements and nutraceuticals for health claim applications of new products, as claim authorization is dependent on the availability of clinical data in order to substantiate efficacy <span class="citation" data-cites="MhvDTmeu">[<a href="#ref-MhvDTmeu" role="doc-biblioref">452</a>]</span>.
These guidelines seem to provide more protection to consumers.
Currently, there is a debate among scientists and regulatory authorities over developing a regulatory framework to deal with the challenges of safety and health claim substantiation for nutraceuticals <span class="citation" data-cites="MhvDTmeu 5TmtJWVQ">[<a href="#ref-5TmtJWVQ" role="doc-biblioref">450</a>,<a href="#ref-MhvDTmeu" role="doc-biblioref">452</a>]</span>.
As a result, studies of supplements and nutraceuticals do not necessarily follow the same rigorous clinical trial framework used for pharmaceuticals.</p>
<p>Nutraceuticals and supplements purported to “boost” the immune response, reduce immunopathology, exhibit antiviral activities, or prevent acute respiratory distress syndrome (ARDS) are being considered for their potential therapeutic value <span class="citation" data-cites="GT7yDBjv">[<a href="#ref-GT7yDBjv" role="doc-biblioref">328</a>]</span>.
A host of potential candidates have been highlighted in the literature that target various aspects of the COVID-19 viral pathology, while others are thought to prime the host immune system.
These candidates include vitamins and minerals along with extracts and omega-3 polyunsaturated fatty acids (n-3 PUFA) <span class="citation" data-cites="2hMFPQJR">[<a href="#ref-2hMFPQJR" role="doc-biblioref">453</a>]</span>.
<em>In vitro</em> and <em>in vivo</em> studies suggest that nutraceuticals containing phycocyanobilin, N-acetylcysteine, glucosamine, selenium or phase 2 inductive nutraceuticals (e.g. ferulic acid, lipoic acid, or sulforaphane) can prevent or modulate RNA virus infections via amplification of the signaling activity of mitochondrial antiviral-signaling protein (MAVS) and activation of toll-like receptor 7 (TLR7) <span class="citation" data-cites="eyBAUUCl">[<a href="#ref-eyBAUUCl" role="doc-biblioref">327</a>]</span>.
While promising, further animal and human studies are required to assess the therapeutic potential of these various nutraceuticals against COVID-19.</p>
<h4 data-number="5.1.4.1" id="n-3-pufa"><span class="header-section-number">5.1.4.1</span> n-3 PUFA</h4>
<p>One supplement that has been explored for beneficial effects against various viral infections is n-3 PUFA <span class="citation" data-cites="2hMFPQJR">[<a href="#ref-2hMFPQJR" role="doc-biblioref">453</a>]</span>, such as eicosapentaenoic acid (EPA) and docosahexaenoic acid (DHA).
EPA and DHA intake can come from a diet high in fish or through dietary supplementation with fish oils or purified oils <span class="citation" data-cites="Ufvsmc3N">[<a href="#ref-Ufvsmc3N" role="doc-biblioref">454</a>]</span>.
They can mediate inflammation and therefore may have the capacity to modulate the adaptive immune response <span class="citation" data-cites="JNyhLAfe Ufvsmc3N 2Us0gNMr">[<a href="#ref-2Us0gNMr" role="doc-biblioref">446</a>,<a href="#ref-Ufvsmc3N" role="doc-biblioref">454</a>,<a href="#ref-JNyhLAfe" role="doc-biblioref">455</a>]</span>.
Another potential mechanism that has led to interest in n-3 PUFA for viral infections is its potential as a precursor molecule for the biosynthesis of endogenous specialized proresolving mediators (SPM), such as protectins and resolvins, that actively resolve inflammation and infection <span class="citation" data-cites="BIVORG2X">[<a href="#ref-BIVORG2X" role="doc-biblioref">456</a>]</span>.
Finally, some COVID-19 patients, particularly those with comorbidities, are at a significant risk of thrombotic complications including arterial and venous thrombosis <span class="citation" data-cites="17jegDTvF PyKMLraw">[<a href="#ref-PyKMLraw" role="doc-biblioref">102</a>,<a href="#ref-17jegDTvF" role="doc-biblioref">457</a>]</span>.
Therefore, the use of prophylactic and therapeutic anticoagulants and antithrombotic agents is under consideration <span class="citation" data-cites="SxPCbOB1">[<a href="#ref-SxPCbOB1" role="doc-biblioref">458</a>]</span>, which could potentially include n-3 PUFA.</p>
<p>SPM have exhibited beneficial effects against a variety of lung infections, including some caused by RNA viruses <span class="citation" data-cites="uPOTvvjR">[<a href="#ref-uPOTvvjR" role="doc-biblioref">459</a>]</span>.
Indeed, protectin D1 has been shown to increase survival from H1N1 viral infection in mice by affecting the viral replication machinery <span class="citation" data-cites="fIo3ESPM">[<a href="#ref-fIo3ESPM" role="doc-biblioref">460</a>]</span>.
Several mechanisms for SPM have been proposed, including preventing the release of pro-inflammatory cytokines and chemokines or increasing phagocytosis of cellular debris by macrophages <span class="citation" data-cites="OSEnHt6H">[<a href="#ref-OSEnHt6H" role="doc-biblioref">461</a>]</span>.
In influenza, SPM promote antiviral B lymphocytic activities <span class="citation" data-cites="MK0GQ55y">[<a href="#ref-MK0GQ55y" role="doc-biblioref">462</a>]</span>, and protectin D1 has been shown to increase survival from H1N1 viral infection in mice by affecting the viral replication machinery <span class="citation" data-cites="fIo3ESPM">[<a href="#ref-fIo3ESPM" role="doc-biblioref">460</a>]</span>.
It is hypothesized that SPM could aid in the resolution of the cytokine storm and pulmonary inflammation associated with COVID-19 <span class="citation" data-cites="14LjKv9ZH">[<a href="#ref-14LjKv9ZH" role="doc-biblioref">463</a>]</span>.
However, not all studies are in agreement that n-3 PUFA is effective against infections <span class="citation" data-cites="B8toe6Mr">[<a href="#ref-B8toe6Mr" role="doc-biblioref">464</a>]</span>.
At a minimum, the effectiveness of n-3 PUFA against infections would be dependent on the dosage, timing, and the specific pathogens responsible <span class="citation" data-cites="3Xht24k3">[<a href="#ref-3Xht24k3" role="doc-biblioref">465</a>]</span>.
On another level, there is still the question of whether fish oils can raise the levels of SPM levels upon ingestion and in response to acute inflammation in humans <span class="citation" data-cites="Ge00xeR3">[<a href="#ref-Ge00xeR3" role="doc-biblioref">466</a>]</span>.</p>
<p>The increased risk of thrombotic complications in COVID-19 infected patients was reported relatively late in comparison to other manifestations of COVID-19 <span class="citation" data-cites="3UBbDrG2 17jegDTvF">[<a href="#ref-3UBbDrG2" role="doc-biblioref">96</a>,<a href="#ref-17jegDTvF" role="doc-biblioref">457</a>]</span>.
Considering that there is significant evidence that n-3 fatty acids and other fish oil-derived lipids possess antithrombotic properties and anti-inflammatory properties <span class="citation" data-cites="umzPtQG7 V9MDMfpj">[<a href="#ref-umzPtQG7" role="doc-biblioref">467</a>,<a href="#ref-V9MDMfpj" role="doc-biblioref">468</a>]</span>, they may have therapeutic value against the prothrombotic complications of COVID-19.
Based on concern among the medical community that the use of investigational therapeutics on COVID-19 patients already on antiplatelet therapies due to a pre-existing comorbidities may lead to issues with dosing and drug choice, and/or negative drug-drug interactions <span class="citation" data-cites="SxPCbOB1">[<a href="#ref-SxPCbOB1" role="doc-biblioref">458</a>]</span>, this supplementation holds particular interest for the treatment of patients already receiving pharmaceutical antiplatelet therapies.
As a result, the use of other therapeutics, such as dietary sources of n-3 fatty acids or nutraceuticals with antiplatelet activities, may be beneficial and warrant further investigation.
A new clinical trial <span class="citation" data-cites="QKHdIx2w">[<a href="#ref-QKHdIx2w" role="doc-biblioref">469</a>]</span> is currently recruiting COVID-19 positive patients to investigate the anti-inflammatory activity of a recently developed, highly purified nutraceutical derivative of EPA known as icosapent ethyl (Vascepa<sup>TM</sup>) <span class="citation" data-cites="12Q4bY0vF">[<a href="#ref-12Q4bY0vF" role="doc-biblioref">470</a>]</span>.
Other randomized controlled trials are in the preparatory stages with the intention of investigating the administration of EPA and other bioactive compounds to COVID-19 positive patients to determine whether anti-inflammatory effects or disease state improvements are observed <span class="citation" data-cites="8lPjdnE7 1AGVAbQuf">[<a href="#ref-8lPjdnE7" role="doc-biblioref">471</a>,<a href="#ref-1AGVAbQuf" role="doc-biblioref">472</a>]</span>.
Finally, while there have been studies investigating the therapeutic value of n-3 fatty acids against ARDS in humans, there is still limited evidence of their effectiveness <span class="citation" data-cites="XHtzuR5c">[<a href="#ref-XHtzuR5c" role="doc-biblioref">473</a>]</span>.
It should be noted that the overall lack of human studies in this area means there is limited evidence as to whether these supplements and nutraceuticals could affect COVID-19 infection.
Consequently, clinical trials have been proposed and many are in the preparatory stages.
These trials will investigate whether the anti-inflammatory potential of n-3 PUFA and their derivatives is beneficial in the treatment of COVID-19.
Therefore, the evidence is not present to draw conclusions about whether n-3 PUFA will be useful in treating COVID-19, but as there is likely little harm associated with a diet rich in fish oils, interest in these products by the general public is unlikely to have negative effects.</p>
<h4 data-number="5.1.4.2" id="zinc"><span class="header-section-number">5.1.4.2</span> Zinc</h4>
<p>There is evidence that certain nutrient supplements may exhibit some benefit against RNA viral infections.
Zinc is a trace metal obtained from dietary sources or supplementation that is important for the maintenance of immune cells involved in adaptive and innate immunity <span class="citation" data-cites="VMI1lOFR">[<a href="#ref-VMI1lOFR" role="doc-biblioref">474</a>]</span>.
Zinc supplements can be administered orally as a tablet or as a lozenge and they are available in many forms, such as zinc picolinate, zinc acetate, and zinc citrate.
Zinc is also available from dietary sources including meat, seafood, nuts, seeds, legumes, and dairy.
The role of zinc in immune function has been extensively reviewed <span class="citation" data-cites="VMI1lOFR">[<a href="#ref-VMI1lOFR" role="doc-biblioref">474</a>]</span>.
Zinc is an important signaling molecule and zinc levels can alter host defense systems.
In inflammatory situations such as an infection, zinc can regulate leukocyte immune responses and it can activate the nuclear factor kappa-light-chain-enhancer of activated B cells (NF-kB), thus altering cytokine production <span class="citation" data-cites="Qif5U0Ig 1CEzU1G34">[<a href="#ref-Qif5U0Ig" role="doc-biblioref">475</a>,<a href="#ref-1CEzU1G34" role="doc-biblioref">476</a>]</span>.
In particular, zinc supplementation can increase natural killer cell levels, which are important cells for host defense against viral infections <span class="citation" data-cites="7ElRN12v VMI1lOFR">[<a href="#ref-VMI1lOFR" role="doc-biblioref">474</a>,<a href="#ref-7ElRN12v" role="doc-biblioref">477</a>]</span>.</p>
<p>Adequate zinc intake has been associated with reduced incidence of infection <span class="citation" data-cites="kxuSwrCI">[<a href="#ref-kxuSwrCI" role="doc-biblioref">478</a>]</span> and antiviral immunity <span class="citation" data-cites="IAOnO0Gj">[<a href="#ref-IAOnO0Gj" role="doc-biblioref">479</a>]</span>.
Similarly, a randomized, double-blind, placebo-controlled trial that administered zinc supplementation to elderly subjects over the course of a year found zinc deficiency to be associated with increased susceptibility to infection and that zinc deficiency could be prevented through supplementation <span class="citation" data-cites="kxuSwrCI">[<a href="#ref-kxuSwrCI" role="doc-biblioref">478</a>]</span>.
Clinical trial data supports the utility of zinc to diminish the duration and severity of symptoms associated with common colds when it is provided within 24 hours of the onset of symptoms <span class="citation" data-cites="sgr0mlJK lPexOhy6">[<a href="#ref-sgr0mlJK" role="doc-biblioref">480</a>,<a href="#ref-lPexOhy6" role="doc-biblioref">481</a>]</span>.
In coronaviruses specifically, <em>in vitro</em> evidence demonstrates that the combination of zinc (Zn2+) and zinc ionophores (pyrithione) can interrupt the replication mechanisms of SARS-CoV-GFP (a fluorescently tagged SARS-CoV) and a variety of other RNA viruses <span class="citation" data-cites="6aOUJdHa NB1udFHF">[<a href="#ref-6aOUJdHa" role="doc-biblioref">482</a>,<a href="#ref-NB1udFHF" role="doc-biblioref">483</a>]</span>.
Currently, there are over twenty clinical trials registered with the intention to use zinc in a preventative or therapeutic manner.
However, many of these trials proposed the use of zinc in conjunction with hydroxychloroquine and azithromycin <span class="citation" data-cites="XJrj3MiZ 1EqFW3G4X Hc9mjVB2 T0aXNTIp">[<a href="#ref-XJrj3MiZ" role="doc-biblioref">484</a>,<a href="#ref-1EqFW3G4X" role="doc-biblioref">485</a>,<a href="#ref-Hc9mjVB2" role="doc-biblioref">486</a>,<a href="#ref-T0aXNTIp" role="doc-biblioref">487</a>]</span>, and it is not known how the lack of evidence supporting the use of hydroxychloroquine will affect investigation of zinc.
Other trials, however, are investigating zinc in conjunction with other supplements such as vitamin C or n-3 PUFA <span class="citation" data-cites="1GGbb9IZt 1AGVAbQuf">[<a href="#ref-1AGVAbQuf" role="doc-biblioref">472</a>,<a href="#ref-1GGbb9IZt" role="doc-biblioref">488</a>]</span>.
Though there is, overall, encouraging data for zinc supplementation against the common cold and viral infections, there is currently limited evidence to suggest zinc supplementation has any beneficial effects against the current novel COVID-19; thus, the clinical trials that are currently underway will provide vital information on the efficacious use of zinc in COVID-19 prevention and/or treatment.
However, given the limited risk, maintaining a healthy diet to ensure an adequate zinc status may be advisable for individuals seeking to reduce their likelihood of infection.</p>
<h4 data-number="5.1.4.3" id="vitamin-c"><span class="header-section-number">5.1.4.3</span> Vitamin C</h4>
<p>Vitamins B, C, D, and E have also been suggested as potential nutrient supplement interventions for COVID-19 <span class="citation" data-cites="2hMFPQJR 3RCeQRRc">[<a href="#ref-2hMFPQJR" role="doc-biblioref">453</a>,<a href="#ref-3RCeQRRc" role="doc-biblioref">489</a>]</span>.
In particular vitamin C has been proposed as a potential therapeutic agent against COVID-19.
Vitamin C can be obtained via dietary sources such as fruit and vegetable or via supplementation.
Vitamin C plays a significant role in promoting immune function due to its effects on various immune cells.
It affects inflammation by modulating cytokine production, decreasing histamine levels, enhancing the differentiation and proliferation of T- and B-lymphocytes, increasing antibody levels, and protecting against the negative effects of reactive oxygen species amongst other effects <span class="citation" data-cites="9czQt5cR RjFxOURM 13M3VAfY9">[<a href="#ref-9czQt5cR" role="doc-biblioref">490</a>,<a href="#ref-RjFxOURM" role="doc-biblioref">491</a>,<a href="#ref-13M3VAfY9" role="doc-biblioref">492</a>]</span>.
Vitamin C is utilized by the body during viral infections, as evinced by lower concentrations in leukocytes and lower concentrations of urinary vitamin C.
Post-infection, these levels return to baseline ranges <span class="citation" data-cites="uQFCDDIy 10nBT1HbX 1Bj8VMlSk aEEZ8Pq9 gZRPoTM9">[<a href="#ref-uQFCDDIy" role="doc-biblioref">493</a>,<a href="#ref-10nBT1HbX" role="doc-biblioref">494</a>,<a href="#ref-1Bj8VMlSk" role="doc-biblioref">495</a>,<a href="#ref-aEEZ8Pq9" role="doc-biblioref">496</a>,<a href="#ref-gZRPoTM9" role="doc-biblioref">497</a>]</span>.</p>
<p>A recent meta-analysis found consistent support for regular vitamin C supplementation reducing the duration of the common cold, but that supplementation with vitamin C (&gt; 200 mg) failed to reduce the incidence of colds <span class="citation" data-cites="Pb5PCH7y">[<a href="#ref-Pb5PCH7y" role="doc-biblioref">498</a>]</span>.
Individual studies have found Vitamin C to reduce the susceptibility of patients to lower respiratory tract infections such as pneumonia <span class="citation" data-cites="JARvpqRw">[<a href="#ref-JARvpqRw" role="doc-biblioref">499</a>]</span>.
Another meta-analysis has demonstrated in twelve trials that vitamin C supplementation reduced the length of stay of patients in intensive care units (ICUs) by 7.8% (95% CI: 4.2% to 11.2%; p = 0.00003).
Furthermore, high doses (1-3 g/day) significantly reduced the length of an ICU stay by 8.6% in six trials (p = 0.003).
Vitamin C also shortened the duration of mechanical ventilation by 18.2% in three trials in which patients required intervention for over 24 hours (95% CI 7.7% to 27%; p = 0.001) <span class="citation" data-cites="DPhRTQw0">[<a href="#ref-DPhRTQw0" role="doc-biblioref">500</a>]</span>.
Despite these findings, the CITRUS ALI study failed to show a benefit of a 96-hour infusion of vitamin C to treat ARDS, which is a severe complication of COVID-19 infection <span class="citation" data-cites="Rt5Aik4p">[<a href="#ref-Rt5Aik4p" role="doc-biblioref">501</a>]</span>.
Nevertheless, a randomized placebo-controlled trial <span class="citation" data-cites="wBwLIQkU">[<a href="#ref-wBwLIQkU" role="doc-biblioref">502</a>]</span> has begun in Wuhan, China to investigate the intravenous infusion of vitamin C to treat pneumonia in 140 severe COVID-19 infected patients.
As summarized by Carr <span class="citation" data-cites="rSyBZFhE">[<a href="#ref-rSyBZFhE" role="doc-biblioref">503</a>]</span> the trial will not be completed until September 2020.
Another trial in Italy <span class="citation" data-cites="16435N02u">[<a href="#ref-16435N02u" role="doc-biblioref">504</a>]</span> intends to deliver a 10 g infusion of vitamin C to 500 severe COVID-19 patients with pneumonia to assess in-hospital mortality over a 72 hr period as the primary outcome.
The trial is currently recruiting and is due to run until March 2021.
We will not know how effective vitamin C is as a therapeutic for quite some time due to the length of both trials.
These are not the only trials investigating the potential value of vitamin C, as there are currently (as of October 2020) over fifteen trials registered with clinicaltrials.gov that are either recruiting or are currently in preparation.
When completed, the trials will provide crucial evidence on the efficacy of vitamin C as a therapeutic for COVID-19 infection.
Thus, some evidence suggests that vitamin C supplementation or infusion can shorten the duration of a cold, reduce an individual’s susceptibility to infections, and shorten a patient’s stay in an ICU when administered at high doses, but we don’t yet understand if these findings apply to COVID-19.
There are ongoing trials in China and Italy that will inform our understanding of the therapeutic value of vitamin C supplementation for COVID-19.
Once again, vitamin C intake is likely to be part of a healthy diet and the vitamin likely presents minimal risk, but its potential prophylactic or therapeutic effects against COVID-19 are unknown.</p>
<h4 data-number="5.1.4.4" id="vitamin-d"><span class="header-section-number">5.1.4.4</span> Vitamin D</h4>
<p>In terms of other dietary supplements, vitamin D can modulate the adaptive and innate immune system and has been associated with various aspects of health.
Vitamin D can be sourced through diet or supplementation, but it is mainly biosynthesized by the body on exposure to sunlight.
Vitamin D deficiency is associated with an increased susceptibility to infection <span class="citation" data-cites="n3v9Thlz">[<a href="#ref-n3v9Thlz" role="doc-biblioref">505</a>]</span>.
In particular, vitamin D deficient patients are at risk of developing acute respiratory infections <span class="citation" data-cites="iWLsflrG">[<a href="#ref-iWLsflrG" role="doc-biblioref">506</a>]</span> and ARDS <span class="citation" data-cites="iWLsflrG">[<a href="#ref-iWLsflrG" role="doc-biblioref">506</a>]</span>.
1,25-dihydroxyvitamin D3 is the more active form of vitamin D that is involved in adaptive and innate responses, however, due to a short half life of a few hours, it is measured by its longer lasting precursor 25-hydroxyvitamin D. The vitamin D receptor is expressed in various immune cells and vitamin D is an immunomodulator of antigen presenting cells, dendritic cells, macrophages, monocytes, and T- and B-lymphocytes <span class="citation" data-cites="n3v9Thlz nwEQBjV4">[<a href="#ref-n3v9Thlz" role="doc-biblioref">505</a>,<a href="#ref-nwEQBjV4" role="doc-biblioref">507</a>]</span>.
Due to its potential immunomodulating properties, vitamin D supplementation may be advantageous to maintain a healthy immune system.</p>
<p>One influential review postulated that an individual’s vitamin D status may significantly affect their risk of developing COVID-19 <span class="citation" data-cites="9ac0GYir">[<a href="#ref-9ac0GYir" role="doc-biblioref">508</a>]</span>.
This hypothesis was derived from the fact that the current pandemic emerged in winter in Wuhan China when 25-hydroxyvitamin D concentrations are at their lowest due to a lack of sunlight, whereas in the Southern Hemisphere, where it was nearing the end of the summer and higher 25-hydroxyvitamin D concentrations would be higher, the number of cases was low.
The authors suggested that people at risk of developing COVID-19 should increase their vitamin D3 intake to reach 25-hydroxyvitamin D plasma concentrations above 40–60 ng/ml.
The authors also suggest high-dose supplementation of vitamin D to treat infected patients and to prevent infection in hospital staff <span class="citation" data-cites="9ac0GYir">[<a href="#ref-9ac0GYir" role="doc-biblioref">508</a>]</span>.
While vitamin D is relatively inexpensive and safe to consume, caution is warranted when interpreting this review as it has yet to be determined whether vitamin D levels affect COVID-19 outside of this geographic/climatic correlation.
Likewise, though it is assumed that COVID-19 may be seasonal, multiple other factors that can affect vitamin D levels should also be considered.
These factors include an individual’s nutritional status, their age, their occupation, skin pigmentation, potential comorbidities, and the variation of exposure to sunlight due to latitude amongst others.
As the pandemic evolves, further research has investigated some of the potential links identified in the Grant et al. review <span class="citation" data-cites="9ac0GYir">[<a href="#ref-9ac0GYir" role="doc-biblioref">508</a>]</span> between vitamin D and COVID-19 and sought to shed light on whether there is any prophylactic and/or therapeutic relationship.
A study in Switzerland demonstrated that 27 SARS-CoV-2 positive patients exhibited 25-hydroxyvitamin D plasma concentrations that were significantly lower (11.1 ng/ml) than those of SARS-CoV-2 negative patients (24.6 ng/ml; p = 0.004), an association that held when stratifying for patients greater than 70 years old <span class="citation" data-cites="1Gy7Svl72">[<a href="#ref-1Gy7Svl72" role="doc-biblioref">509</a>]</span>.
These findings seem to be supported by a Belgian observational study of 186 SARS-CoV-2 positive patients exhibiting symptoms of pneumonia, where 25-hydroxyvitamin D plasma concentrations were measured and a CT scan of the lungs was obtained upon hospitalization <span class="citation" data-cites="GMAEaWD1">[<a href="#ref-GMAEaWD1" role="doc-biblioref">510</a>]</span>.
A significant difference in 25-hydroxyvitamin D levels was observed between the SARS-CoV-2 patients and 2,717 season-matched diseased controls.
Both female and male patients possessed lower median 25-hydroxyvitamin D concentrations than the control group (18.6 ng/ml versus 21.5 ng/ml; p = 0.0016) and a higher rate of vitamin D deficiency (58.6% versus 42.5%).
Evidence of sexual dimorphism was apparent, as female patients had equivalent levels of 23-hydroxyvitamin D to the control group, whereas male patients were deficient in 25-hydroxyvitamin D relative to male controls (67% versus 49%; p = 0.0006).
Notably, vitamin D deficiency was progressively lower in males with advancing radiological disease stages (p = 0.001).
These studies are supported by several others that indicate that vitamin D status may be an independent risk factor for the severity of COVID-19 <span class="citation" data-cites="GM9W6HPG kN3eqnZW 5MvUc8IL 18O6t6Jo">[<a href="#ref-GM9W6HPG" role="doc-biblioref">511</a>,<a href="#ref-kN3eqnZW" role="doc-biblioref">512</a>,<a href="#ref-5MvUc8IL" role="doc-biblioref">513</a>,<a href="#ref-18O6t6Jo" role="doc-biblioref">514</a>]</span> and in COVID-19 patients relative to population-based controls <span class="citation" data-cites="8ZxZKpYS">[<a href="#ref-8ZxZKpYS" role="doc-biblioref">515</a>]</span>.
Indeed, serum concentrations of 25-hydroxyvitamin D above 30 ng/ml, which indicate vitamin D sufficiency, seems to be associated with a reduction in serum C-reactive protein (CRP), an inflammatory marker, along with increased lymphocyte levels, which suggests that vitamin D levels may modulate the immune response by reducing risk for cytokine storm in response to SARS-CoV-2 infection.
Despite these studies potentially linking vitamin D status with COVID-19 severity, an examination of the UK Biobank did not support this thesis <span class="citation" data-cites="FZ8yXmxQ">[<a href="#ref-FZ8yXmxQ" role="doc-biblioref">516</a>]</span>.
This analysis examined 25-hydroxyvitamin D concentrations in 348,598 UK Biobank participants, of which 449 were SARS-CoV-2 positive.
However, this study has caused considerable debate that will likely be settled following further studies <span class="citation" data-cites="EtCW6lkq ClClnlJb">[<a href="#ref-EtCW6lkq" role="doc-biblioref">517</a>,<a href="#ref-ClClnlJb" role="doc-biblioref">518</a>]</span>.
There is significant interest in the link between vitamin D and COVID-19, hence the multitude of studies published in the literature of varying quality and the clinical trials underway.
One trial is currently investigating the utility of vitamin D as an immune-modulating agent by monitoring whether administration of vitamin D precipitates an improvement of health status in non-severe symptomatic patients infected with COVID-19 or whether vitamin D prevents patient deterioration <span class="citation" data-cites="162t7zdGv">[<a href="#ref-162t7zdGv" role="doc-biblioref">519</a>]</span>.
Other trials are also underway examining various factors including mortality, symptom recovery, severity of disease, rates of ventilation, inflammatory markers such as CRP and IL-6, blood cell counts, and the prophylactic capacity of vitamin D administration <span class="citation" data-cites="2K0jV5Bn 162t7zdGv rBiojRHk 6kLoX5qT">[<a href="#ref-162t7zdGv" role="doc-biblioref">519</a>,<a href="#ref-2K0jV5Bn" role="doc-biblioref">520</a>,<a href="#ref-rBiojRHk" role="doc-biblioref">521</a>,<a href="#ref-6kLoX5qT" role="doc-biblioref">522</a>]</span>.
Concomitant administration of vitamin D with pharmaceuticals such as aspirin <span class="citation" data-cites="32C2erIe">[<a href="#ref-32C2erIe" role="doc-biblioref">523</a>]</span> and bioactive molecules such as resveratrol <span class="citation" data-cites="tLVliODQ">[<a href="#ref-tLVliODQ" role="doc-biblioref">524</a>]</span> are also under investigation.</p>
<p>While there is increasing evidence that vitamin D status is linked to COVID-19 outcomes, the effectiveness of its supplementation remains open for debate.
Once again, supplementation of vitamin D and maintaining a healthy diet for optimum vitamin D status is unlikely to carry major health risks while the possible link between vitamin D status and COVID-19 is investigated.
However, pursuing to elevate vitamin D levels through sunlight exposure does carry additional risks, as many densely populated cities and countries around the world are sporadically utilizing “shelter in place” orders to enforce social distancing guidelines, especially with cases rising again as winter 2020 approaches.
Given the lack of conclusive evidence in support of vitamin D supplementation, it is not clear that these guidelines present additional risk.
However, to the extent that people are able to maintain safe exposure to sunlight, there is a possibility that it could improve endogenous synthesis of vitamin D, potentially strengthening the immune system.</p>
<h4 data-number="5.1.4.5" id="probiotics"><span class="header-section-number">5.1.4.5</span> Probiotics</h4>
<p>Probiotics are “live microorganisms that, when administered in adequate amounts, confer a health benefit on the host” <span class="citation" data-cites="1FyHVZ9yx">[<a href="#ref-1FyHVZ9yx" role="doc-biblioref">525</a>]</span>.
Some studies suggest that probiotics are beneficial against common viral infections and there is modest evidence to suggest that they can modulate the immune response <span class="citation" data-cites="pvZU55k9 HKeroTXj">[<a href="#ref-pvZU55k9" role="doc-biblioref">526</a>,<a href="#ref-HKeroTXj" role="doc-biblioref">527</a>]</span>, and as a result it has been hypothesized that probiotics may have therapeutic value worthy of investigation against SARS-CoV-2 <span class="citation" data-cites="10e8cHDs8">[<a href="#ref-10e8cHDs8" role="doc-biblioref">528</a>]</span>.
Probiotics and next-generation probiotics, which are more akin to pharmacological-grade supplements, have been associated with multiple potential beneficial effects for allergies, digestive tract disorders, and even metabolic diseases through their anti-inflammatory and immunomodulatory effects <span class="citation" data-cites="5fU7dexB 1GCrGCp5P">[<a href="#ref-5fU7dexB" role="doc-biblioref">529</a>,<a href="#ref-1GCrGCp5P" role="doc-biblioref">530</a>]</span>.
However, the mechanisms by which probiotics affect these various conditions would likely differ among strains, with the ultimate effect of the probiotic depending on the heterogeneous set of bacteria present <span class="citation" data-cites="1GCrGCp5P">[<a href="#ref-1GCrGCp5P" role="doc-biblioref">530</a>]</span>.
Some of the beneficial effects of probiotics include reducing inflammation by promoting the expression of anti-inflammatory mediators, inhibiting toll-like receptors (TLR) 2 and 4, direct competition with pathogens, the synthesis of antimicrobial substances or other metabolites, improving intestinal barrier function, and/or favorably altering the gut microbiota and the brain-gut axis <span class="citation" data-cites="B2GeSSQo 1GCrGCp5P nmXF8vkS">[<a href="#ref-1GCrGCp5P" role="doc-biblioref">530</a>,<a href="#ref-B2GeSSQo" role="doc-biblioref">531</a>,<a href="#ref-nmXF8vkS" role="doc-biblioref">532</a>]</span>.
However, there is also a bi-directional relationship between the lungs and gut microbiota known as the gut-lung axis <span class="citation" data-cites="13AWoYvqB">[<a href="#ref-13AWoYvqB" role="doc-biblioref">533</a>]</span>, whereby gut microbial metabolites and endotoxins may affect the lungs via the circulatory system and the lung microbiota in return may affect the gut <span class="citation" data-cites="Smqu1EYN">[<a href="#ref-Smqu1EYN" role="doc-biblioref">534</a>]</span>.
Therefore, the gut-lung axis may play role in our future understanding of COVID-19 pathogenesis and become a target for probiotic treatments <span class="citation" data-cites="d27mQ4kO">[<a href="#ref-d27mQ4kO" role="doc-biblioref">535</a>]</span>.</p>
<p>Probiotics have tentatively been associated with the reduction of risk and duration of viral upper respiratory tract infections <span class="citation" data-cites="1MIGrWIP 58eXvYEV kjavvCzp">[<a href="#ref-1MIGrWIP" role="doc-biblioref">536</a>,<a href="#ref-58eXvYEV" role="doc-biblioref">537</a>,<a href="#ref-kjavvCzp" role="doc-biblioref">538</a>]</span>.
Some meta-analyses that have assessed the efficacy of probiotics in viral respiratory infections have reported moderate reductions in the incidence and duration of infection <span class="citation" data-cites="58eXvYEV dJIGDt4k">[<a href="#ref-58eXvYEV" role="doc-biblioref">537</a>,<a href="#ref-dJIGDt4k" role="doc-biblioref">539</a>]</span>.
Indeed, randomized controlled trials have shown that administering <em>Bacillus subtilis</em> and <em>Enterococcus faecalis</em> <span class="citation" data-cites="AJav6c61">[<a href="#ref-AJav6c61" role="doc-biblioref">540</a>]</span>, <em>Lactobacillus rhamnosus GG</em> <span class="citation" data-cites="1E2aDJp4T">[<a href="#ref-1E2aDJp4T" role="doc-biblioref">541</a>]</span>, or <em>Lactobacillus casei</em> and <em>Bifidobacterium breve</em> with galactooligosaccharides <span class="citation" data-cites="zccFgajh">[<a href="#ref-zccFgajh" role="doc-biblioref">542</a>]</span> via the nasogastric tube to ventilated patients reduced the occurrence of ventilator-associated pneumonia in comparison to the respective control groups in studies of viral infections and sepsis.
These findings are supported by a recently published meta-analysis <span class="citation" data-cites="1DI06gFPc">[<a href="#ref-1DI06gFPc" role="doc-biblioref">543</a>]</span>.
There is a significant risk of ventilator-associated bacterial pneumonia in COVID-19 patients <span class="citation" data-cites="164mVBFZx">[<a href="#ref-164mVBFZx" role="doc-biblioref">544</a>]</span>, but it can be challenging for clinicians to diagnose this infection due to the fact that severe COVID-19 infection presents with the symptoms of pneumonia <span class="citation" data-cites="w2VFo0h1">[<a href="#ref-w2VFo0h1" role="doc-biblioref">545</a>]</span>.
Therefore, an effective prophylactic therapy for ventilator-associated pneumonia in severe COVID-19 patients would be of significant therapeutic value.</p>
<p>Probiotics are generally synonymous with the treatment of gastrointestinal issues due to their supposed anti-inflammatory and immunomodulatory effects <span class="citation" data-cites="gcyQ73v7">[<a href="#ref-gcyQ73v7" role="doc-biblioref">546</a>]</span>.
Notably, gastrointestinal symptoms commonly occur in COVID-19 patients <span class="citation" data-cites="67ihxDiV">[<a href="#ref-67ihxDiV" role="doc-biblioref">547</a>]</span>, and the ACE2 receptor is highly expressed in enterocytes of the ileum and colon, suggesting that these organs may be a potential route of infection <span class="citation" data-cites="8Hvgbl3n Nhxoefru">[<a href="#ref-8Hvgbl3n" role="doc-biblioref">548</a>,<a href="#ref-Nhxoefru" role="doc-biblioref">549</a>]</span>.
Indeed, SARS-CoV-2 viral RNA has been detected in human feces <span class="citation" data-cites="TPWaxApy">[<a href="#ref-TPWaxApy" role="doc-biblioref">550</a>]</span>, and fecal-oral transmission of the virus has not yet been ruled out <span class="citation" data-cites="r9HNMqTS">[<a href="#ref-r9HNMqTS" role="doc-biblioref">551</a>]</span>.
Rectal swabs of some SARS-CoV-2 positive pediatric patients persistently tested positive for several days despite negative nasopharyngeal tests, indicating the potential for fecal viral shedding <span class="citation" data-cites="eOcc1BUx">[<a href="#ref-eOcc1BUx" role="doc-biblioref">552</a>]</span>.
However, there is conflicting evidence for the therapeutic value of various probiotics against the incidence or severity of gastrointestinal symptoms in viral or bacterial infections such as gastroenteritis <span class="citation" data-cites="VwStn9Ps UtAKolza">[<a href="#ref-VwStn9Ps" role="doc-biblioref">553</a>,<a href="#ref-UtAKolza" role="doc-biblioref">554</a>]</span>.
Nevertheless, it has been proposed that the administration of probiotics to COVID-19 patients and healthcare workers may prevent or ameliorate the gastrointestinal symptoms of COVID-19, a hypothesis that several clinical trials are now preparing to investigate <span class="citation" data-cites="1AqF1ESKq 9UYsmETk">[<a href="#ref-1AqF1ESKq" role="doc-biblioref">555</a>,<a href="#ref-9UYsmETk" role="doc-biblioref">556</a>]</span>.
Other studies are investigating whether probiotics may affect patient outcomes following SARS-CoV-2 infection <span class="citation" data-cites="15pDlYtom">[<a href="#ref-15pDlYtom" role="doc-biblioref">557</a>]</span>.</p>
<p>Generally, the efficacy of probiotic use is a controversial topic among scientists.
In Europe, EFSA has banned the term probiotics on products labels, which has elicited either criticism for EFSA or support for probiotics from researchers in the field <span class="citation" data-cites="m3SexO9F 5FTOk7ol 1FyHVZ9yx">[<a href="#ref-1FyHVZ9yx" role="doc-biblioref">525</a>,<a href="#ref-m3SexO9F" role="doc-biblioref">558</a>,<a href="#ref-5FTOk7ol" role="doc-biblioref">559</a>]</span>.
This is due to the hyperbolic claims placed on the labels of various probiotic products, which lack rigorous scientific data to support their efficacy.
Overall, the data supporting probiotics in the treatment or prevention of many different disorders and diseases is not conclusive as the quality of the evidence is generally considered low <span class="citation" data-cites="1MIGrWIP">[<a href="#ref-1MIGrWIP" role="doc-biblioref">536</a>]</span>.
However, in the case of probiotics and respiratory infections, the evidence seems to be supportive of their potential therapeutic value.
Consequently, several investigations are underway to investigate the prophylactic and therapeutic potential of probiotics for COVID-19.
However, blind use of conventional probiotics for COVID-19 is cautioned against until the pathogenesis of SARS-CoV-2 is further established <span class="citation" data-cites="kv94VRLF">[<a href="#ref-kv94VRLF" role="doc-biblioref">560</a>]</span>.
Until clinical trials investigating the prophylactic and therapeutic potential of probiotics for COVID-19 are complete, it is not possible to provide an evidence-based recommendation for their use.</p>
<h4 data-number="5.1.4.6" id="nutraceutical-and-dietary-supplement-conclusions"><span class="header-section-number">5.1.4.6</span> Nutraceutical and Dietary Supplement Conclusions</h4>
<p>Despite all the potential benefits of nutraceutical and dietary supplement interventions presented, currently there is a paucity of clinical evidence to support their use for the prevention or mitigation of COVID-19 infections.
Nevertheless, optimal nutritional status can prime an individual’s immune system to protect against the effects of acute respiratory viral infections by supporting normal maintenance of the immune system <span class="citation" data-cites="5tHMedSi Lyd8s4Mk">[<a href="#ref-5tHMedSi" role="doc-biblioref">561</a>,<a href="#ref-Lyd8s4Mk" role="doc-biblioref">562</a>]</span>.
Nutritional strategies can also play a role in the treatment of hospitalized patients, as malnutrition is a risk to COVID-19 patients <span class="citation" data-cites="1DmkRczqw">[<a href="#ref-1DmkRczqw" role="doc-biblioref">563</a>]</span>.
Overall, supplementation of vitamin C, vitamin D, and zinc may be an effective method of ensuring their adequate intake to maintain optimal immune function, which may also convey beneficial effects against viral infections due to their immunomodulatory effects.
However, many supplements and nutraceuticals designed for various ailments are available in the United States and beyond that are not strictly regulated <span class="citation" data-cites="lMhFSIiF">[<a href="#ref-lMhFSIiF" role="doc-biblioref">564</a>]</span>.
Indeed, there can be safety and efficacy concerns associated with many of these products.
Often, the vulnerable members of society can be exploited in this regard and unfortunately the COVID-19 pandemic is no different.
The Food and Drug Administration (FDA) has issued warnings to several companies for advertising falsified claims in relation to the preventative and therapeutic capabilities of their products against COVID-19 <span class="citation" data-cites="fez8aZAu">[<a href="#ref-fez8aZAu" role="doc-biblioref">565</a>]</span>.
In light of these serious occurrences, it is pertinent to clarify that the nutraceuticals discussed in this review have been selected because of their possible relevance to the biological mechanisms that can beneficially affect viral and respiratory infections and because they are currently under clinical investigation.
Therefore, further intensive investigation is required to establish the effects of these nutraceuticals, if any, against COVID-19.
Until effective therapeutics are established, the most effective mitigation strategies consist of encouraging standard public health practices such as regular hand washing with soap, wearing a face mask, and covering a cough with your elbow <span class="citation" data-cites="2Xu90WtD">[<a href="#ref-2Xu90WtD" role="doc-biblioref">566</a>]</span>, along with following social distancing measures, “shelter in place” guidelines, expansive testing, and contact tracing <span class="citation" data-cites="EhbZIMo0 12Met0WvX">[<a href="#ref-EhbZIMo0" role="doc-biblioref">567</a>,<a href="#ref-12Met0WvX" role="doc-biblioref">568</a>]</span>.</p>
<h2 data-number="5.2" id="biologics"><span class="header-section-number">5.2</span> Biologics</h2>
<p>Biologics are produced from components of living organisms or viruses.
They include treatments such as humanized monoclonal antibodies (mAb) tocilizumab (TCZ), and neutralizing antibodies (nAbs), and prophylactics such as vaccines.
Historically produced from animal tissue, biologics have become increasingly feasible to produce as recombinant DNA technologies have advanced <span class="citation" data-cites="1ALDN7rfe">[<a href="#ref-1ALDN7rfe" role="doc-biblioref">569</a>]</span>.
Often, they are glycoproteins or peptides <span class="citation" data-cites="OgYDNJt3">[<a href="#ref-OgYDNJt3" role="doc-biblioref">570</a>]</span>, but whole viruses can also be used therapeutically or prophylactically, not only for vaccines but also as vectors for gene therapy or therapeutic proteins or for oncolytic virotherapy <span class="citation" data-cites="GO8FwtPx">[<a href="#ref-GO8FwtPx" role="doc-biblioref">571</a>]</span>.
They are typically catabolized by the body to their amino acid components <span class="citation" data-cites="OgYDNJt3">[<a href="#ref-OgYDNJt3" role="doc-biblioref">570</a>]</span>.
There are many differences on the development side between biologics and synthesized pharmaceuticals, such as small molecule drugs.
Biologics are typically orders of magnitude larger than small molecule drugs and their physiochemical properties are often much less understood <span class="citation" data-cites="OgYDNJt3">[<a href="#ref-OgYDNJt3" role="doc-biblioref">570</a>]</span>.
They are often heat-sensitive and the toxicity can vary, as it is not directly associated with the primary effects of the drug <span class="citation" data-cites="OgYDNJt3">[<a href="#ref-OgYDNJt3" role="doc-biblioref">570</a>]</span>.
However, this class includes some extremely significant medical breakthroughs, including insulin for the management of diabetes and the smallpox vaccine.
As a result, it is another possible avenue through which the pharmacological management of SARS-CoV-2 infection can be explored.</p>
<h3 data-number="5.2.1" id="tocilizumab"><span class="header-section-number">5.2.1</span> Tocilizumab</h3>
<p>TCZ is a receptor antibody that was developed to manage chronic inflammation caused by the continuous synthesis of the cytokine interleukin-6 (IL-6) <span class="citation" data-cites="NDRyNd61">[<a href="#ref-NDRyNd61" role="doc-biblioref">572</a>]</span>.
IL-6 is a pro-inflammatory cytokine belonging to the interleukin family, immune system regulators that are primarily responsible for immune cell differentiation.
Often used to treat conditions such as rheumatoid arthritis <span class="citation" data-cites="NDRyNd61">[<a href="#ref-NDRyNd61" role="doc-biblioref">572</a>]</span>, TCZ has become a pharmaceutical of interest for the treatment of COVID-19.
While secretion of IL-6 can be associated with chronic conditions, it is a key player in the innate immune response and is secreted by macrophages in response to the detection of PAMPs or DAMPs <span class="citation" data-cites="NDRyNd61">[<a href="#ref-NDRyNd61" role="doc-biblioref">572</a>]</span>.
An analysis of 191 in-patients at two Wuhan hospitals revealed that blood concentrations of IL-6 differed between patients who did and did not recover from COVID-19.
Patients who ultimately deceased had higher IL-6 levels at admission than those who recovered <span class="citation" data-cites="10THxyeCg">[<a href="#ref-10THxyeCg" role="doc-biblioref">36</a>]</span>.
Additionally, IL-6 levels remained higher throughout the course of hospitalization in the patients who ultimately deceased <span class="citation" data-cites="10THxyeCg">[<a href="#ref-10THxyeCg" role="doc-biblioref">36</a>]</span>.
This finding provided some early evidence that COVID-19 deaths may be induced by the hyperactive immune response, which can be either the cytokine release syndrome (CRS) or cytokine storm syndrome (CSS), as IL-6 plays a key role in this response <span class="citation" data-cites="bZMKqj6e">[<a href="#ref-bZMKqj6e" role="doc-biblioref">109</a>]</span>.
In this context, the observation of elevated IL-6 in patients who died may reflect an over-production of proinflammatory interleukins, suggesting that TCZ may palliate some of the most severe symptoms of COVID-19 associated with increased cytokine production.</p>
<p>Human IL-6 is a 26-kDa glycoprotein that consists of 184 amino acids and contains two potential N-glycosylation sites and four cysteine residues.
It binds to a type I cytokine receptor (IL-6Rα or glycoprotein 80) that exists in both membrane-bound (IL-6Rα) and soluble (sIL-6Rα) forms <span class="citation" data-cites="VXDsRwHC">[<a href="#ref-VXDsRwHC" role="doc-biblioref">573</a>]</span>.
It is not the binding of IL-6 to the receptor that initiates pro- and/or anti-inflammatory signaling, but rather the binding of the complex to another subunit, known as IL-6Rβ or glycoprotein 130 (gp13) <span class="citation" data-cites="14xZS2Gic VXDsRwHC">[<a href="#ref-VXDsRwHC" role="doc-biblioref">573</a>,<a href="#ref-14xZS2Gic" role="doc-biblioref">574</a>]</span>.
Unlike membrane-bound IL-6Rα, which is only found on hepatocytes and some types of leukocytes, gp130 is found on most cells <span class="citation" data-cites="LIqxTdN">[<a href="#ref-LIqxTdN" role="doc-biblioref">575</a>]</span>.
When IL-6 binds to sIL-6Rα, the complex can then bind to a gp130 protein on any cell <span class="citation" data-cites="LIqxTdN">[<a href="#ref-LIqxTdN" role="doc-biblioref">575</a>]</span>.
The binding of IL-6 to IL-6Rα is termed classical signaling, while its binding to sIL-6Rα is termed trans-signaling <span class="citation" data-cites="1BxkPZYnj 1EEloBjzn LIqxTdN">[<a href="#ref-LIqxTdN" role="doc-biblioref">575</a>,<a href="#ref-1BxkPZYnj" role="doc-biblioref">576</a>,<a href="#ref-1EEloBjzn" role="doc-biblioref">577</a>]</span>.
These two signaling processes are thought to play different roles in health and illness.
For example, trans-signaling may play a role in the proliferation of mucosal T-helper TH2 cells associated with asthma, while an earlier step in this proliferation of process may be regulated by classical signaling <span class="citation" data-cites="LIqxTdN">[<a href="#ref-LIqxTdN" role="doc-biblioref">575</a>]</span>.
Similarly, IL-6 is known to play a role in Crohn’s Disease via trans-, but not classical, signaling <span class="citation" data-cites="LIqxTdN">[<a href="#ref-LIqxTdN" role="doc-biblioref">575</a>]</span>.
Both classical and trans-signaling can occur through three independent pathways: the Janus-activated kinase (JAK)-STAT3 pathway, the Ras/Mitogen-Activated Protein Kinases (MAPK) pathway and the Phosphoinositol-3 Kinase (PI3K)/Akt pathway <span class="citation" data-cites="VXDsRwHC">[<a href="#ref-VXDsRwHC" role="doc-biblioref">573</a>]</span>.
These signaling pathways are involved in a variety of different functions, including cell type differentiation, immunoglobulin synthesis, and cellular survival signaling pathways, respectively <span class="citation" data-cites="VXDsRwHC">[<a href="#ref-VXDsRwHC" role="doc-biblioref">573</a>]</span>.
The ultimate result of the IL-6 cascade is to direct transcriptional activity of various promoters of pro-inflammatory cytokines, such as IL-1, TFN, and even IL-6 itself, through the activity of NF-κB <span class="citation" data-cites="VXDsRwHC">[<a href="#ref-VXDsRwHC" role="doc-biblioref">573</a>]</span>.
IL-6 synthesis is tightly regulated both transcriptionally and post-transcriptionally, and it has been shown that viral proteins can enhance transcription of the IL-6 gene by strengthening the DNA-binding activity between several transcription factors and IL-6 gene-cis-regulatory elements <span class="citation" data-cites="1H8o8TFtM">[<a href="#ref-1H8o8TFtM" role="doc-biblioref">578</a>]</span>.
Therefore, drugs inhibiting the binding of IL-6 to IL-6Rα or sIL-6Rα are of interest for combating the hyperactive inflammatory response characteristic of CRS and CSS.
TCZ is a humanized monoclonal antibody that binds both to the insoluble and soluble receptor of IL-6, providing de facto inhibition of the IL-6 immune cascade.</p>
<p>Tocilizumab is being administered either as an intervention or as concomitant medication in 77 COVID-19 clinical trials (Figure <a href="#fig:ebm-trials">3</a>).
No randomized, placebo-controlled studies have been released yet for TCZ.
Therefore, no conclusions can be drawn about its efficacy for the treatment of COVID-19.
However, early interest in TCZ as a possible treatment for COVID-19 emerged from a very small retrospective study in China that examined 20 patients with severe symptoms in early February 2020 and reported rapid improvement in symptoms following treatment with TCZ <span class="citation" data-cites="bj2feMy4">[<a href="#ref-bj2feMy4" role="doc-biblioref">579</a>]</span>.
Subsequently, a number of retrospective studies have been conducted in several countries.
<!--
| Reference | Site(s) | Observation Period | Primary Outcome | Patients Receiving TCZ / Total Enrolled | Matched Control | PCR-confirmed | Main Finding | Hazard Ratio (p) |
|-----------|---------|--------------------|-----------------|-----------------------------------------|-----------------|---------------|--------------|
| [@doi:10.1002/jmv.25801] | Jan. 27 - Mar. 5 |1 site (Wuhan, China) | CRP & IL-6 Levels | 15/15 | No | Not Specified | Clinical stabilization | N/A |
| [@doi:10.1073/pnas.2005615117] | Feb. 5 - 14 | 2 sites (Anhui, China) | Clinical Response | 20/20 | No | Yes | Clinical stabilization | N/A |
| [@doi:10.1016/j.ejim.2020.05.009] | Feb. 26 - Apr. 2 | 1 site (Brescia, Italy) | Survival | 62/85 | No | Yes | Improved survival | 0.035 (0.004) | 
| [@doi:10/d2pk] | Feb. 21 - Mar. 24 or Apr. 30 (Modena) | 3 sites (Bologna, Reggio Emilia, & Modena, Italy) | Survival + IMV | 179/544 | No | Yes | Reduced risk of IMV/death | 0.61 (0.02) | 
| [@doi:10.1016/j.chest.2020.06.006] | Mar. 10 - 31 | 1 site (Connecticut, USA) | Clinical Response | 153/239 | No | Yes | Disease severity at intake less associated with survival | N/A | 
| [@doi:10.1002/jmv.25964] | Feb. 27 - Apr. 17 | Qatar | Clinical Response | 25/25 | No | Yes | Clinical stabilization | N/A | 
| [@doi:10.3390/microorganisms8050695] | Mar. 14 - 27 | 1 site (Pavia, Italy) | Survival & ICU Admission | 21/42 | Yes | Yes | No effect | N/A (OR used) | 

Table: Comparison of some early observational studies administering TCZ to COVID-19 patients.
The above is not a systematic review of TCZ studies but instead provides some perspective on the diversity of retrospective studies of TCZ that have been released. 
Standard of Care administered to all patients is defined by each study within the manuscript.
Site(s) indicates the number and location(s) of the hospitals included in the study, except in the case where the analysis included all cases within a geographic area (Qatar, which had only 25 confirmed COVID-19 cases treated with TCZ in the observation period).
The observation period refers to the timeperiod examined retrospectively to identify eligible patients.
Primary outcomes, when specified, are indicated; clinical response is used to indicate studies that did not explicitly define a primary outcome and mainly reported changes in biomarkers, secondary infections, etc.
Matched control refers to studies that sought to retrospectively identify pairs of comparable patients who received and didn't receive TCZ.
PCR-confirmed indicates whether patients' COVID-19 diagnosis was confirmed by a laboratory test (RT-PCR).
IMV = Invasive Medical Ventilation {#tbl:tcz-table}
-->
Many studies use a retrospective, observational design, where they compare outcomes for COVID-19 patients who received TCZ to those who did not over a set period of time.
For example, one of the largest retrospective, observational analysis released to date <span class="citation" data-cites="ayc7zYqi">[<a href="#ref-ayc7zYqi" role="doc-biblioref">580</a>]</span> compared the rates at which patients who received TCZ deceased or progressed to invasive medical ventilation over a 14-day period compared to patients receiving only SOC.</p>
<p>Under this definition, SOC could include other drugs such as HCQ, azithromycin, lopinavir-ritonavir or darunavir-cobicistat, or heparin.
While this study was not randomized, a subset of patients who were eligible to receive TCZ were unable to obtain it due to shortages; however, these groups were not directly compared in the analysis.
After adjusting for variables such as age, sex, and SOFA score, they found that patients treated with TCZ were less likely to progress to invasive medical ventilation and/or death (adjusted hazard ratio = 0.61, CI 0.40-0.92, p=0.020), although analysis of death and ventilation separately suggests that this effect may have been driven by differences in the death rate (20% of control versus 7% of TCZ-treated patients).
They reported particular benefits for patients whose PaO<sub>2</sub>/FiO<sub>2</sub> (P/F) ratio, also known as the Horowitz Index for Lung Function, fell below a 150 mm Hg threshold.
They found no differences between groups administered subcutaneous versus intravenous TCZ.
Another retrospective observational analysis of interest examined the charts of patients at a hospital in Connecticut, USA where 64% of all 239 COVID-19 patients in the study period were administered TCZ based on assignment by a standardized algorithm <span class="citation" data-cites="Wt9vgKLY">[<a href="#ref-Wt9vgKLY" role="doc-biblioref">581</a>]</span>.
They found that TCZ administration was associated with more similar rates of survivorship in patients with severe versus nonsevere COVID-19 at intake, defined based on the amount of supplemental oxygen needed.
They therefore proposed that their algorithm was able to identify patients presenting with or likely to develop CRS as good candidates for TCZ.
This study also reported higher survivorship in Black and Hispanic patients compared to white patients when adjusted for age.
The major limitation with interpretation for these studies is that there may be clinical characteristics that influenced medical practitioners decisions to administer TCZ to some patients and not others.
One interesting example therefore comes from an analysis of patients at a single hospital in Brescia, Italy, where TCZ was not available for a period of time <span class="citation" data-cites="luPHOS0m">[<a href="#ref-luPHOS0m" role="doc-biblioref">582</a>]</span>.
This study compared COVID-19 patients admitted to the hospital before and after March 13, 2020, when the hospital received TCZ.
Therefore, patients who would have been eligible for TCZ prior to this arbitrary date did not receive it as treatment, making this retrospective analysis something of a natural experiment.
Despite this design, demographic factors did not appear to be consistent between the two groups, and the average age of the control group was older than the TCZ group.
The control group also had a higher percentage of males and a higher incidence of comorbidities such as diabetes and heart disease.
All the same, the multivariate hazard ratio, which adjusted for these clinical and demographic factors, found a significant difference between survival in the two groups (HR=0.035, CI=0.004-0.347, p = 0.004).
They reported improvement of survival outcomes after the addition of TCZ to their SOC regime, with 11 of 23 patients (47.8%) admitted prior to March 13th dying compared to 2 of 62 (3.2%) admitted afterwards.
They also reported a reduced progression to mechanical ventilation in the TCZ group.
However, this study also holds a significant limitation: the time delay between the two groups means that knowledge about how to treat the disease likely improved over this timeframe as well.
All the same, the results of these observational retrospective studies provide support for TCZ as a pharmaceutical of interest for follow-up in clinical trials.</p>
<p>In addition to the retrospective observational studies, other analysis have utilized a retrospective case-control design to match pairs of patients with similar baseline characteristics, only one of whom received TCZ for COVID-19.
In one such study, TCZ was significantly associated with a reduced risk of progression to ICU admission or death <span class="citation" data-cites="17hgtBpcV">[<a href="#ref-17hgtBpcV" role="doc-biblioref">583</a>]</span>.
This study examined only 20 patients treated with TCZ compared (all but one of the patients treated with TCZ in the hospital during the study period) to 25 patients receiving standard of care.
For the combined primary endpoint of death and/or ICU admission, only 25% of patients receiving TCZ progressed to an endpoint compared to 72% in the SOC group (p = 0.002 presumably based on a chi-square test).
When the two endpoints were examined separately, progression to invasive medical ventilation remained significant (32% SOC compared to 0% TCZ, p = 0.006) but not for mortality (48% SOC compared to 25% TCZ, p = 0.066).
In contrast, a study that compared 96 patients treated with TCZ to 97 patients treated with SOC only in New York City found that differences in mortality did not differ between the two groups, but that this difference did become significant when intubated patients were excluded from the analysis <span class="citation" data-cites="WEqfULN1">[<a href="#ref-WEqfULN1" role="doc-biblioref">584</a>]</span>.
Taken together, these findings suggest that future clinical trials of TCZ may want to include intubation as an endpoint.
However, these studies should be approached with caution, not only because of the small number of patients enrolled and retrospective design, but also because they performed a large number of statistical tests and did not account for multiple hypothesis testing.
These last findings highlight the need to search for a balance between impairing a harmful immune response, such as the one generated during CRS and CSS, and preventing the worsening of the clinical picture of the patients by potential new viral infections.</p>
<p>Though data about TCZ for COVID-19 is still only just emerging, some meta-analyses and systematic reviews have investigated the available data.
One meta-analysis <span class="citation" data-cites="3yV2bL9S">[<a href="#ref-3yV2bL9S" role="doc-biblioref">585</a>]</span> evaluated 19 studies published or released as preprints prior to July 1, 2020 and found that the overall trends supportive of the frequent conclusion that TCZ does improve survivorship, with a significant hazard ratio of 0.41.
This trend improved when they excluded studies that administered a steroid alongside TCZ, with a significant hazard ratio of 0.04.
They also found some evidence for reduced IMV or ICU admission, but only when excluding all studies except a small number reporting estimates that were adjusted for possible bias introduced by the challenges of stringency during the enrollment process.
A systematic analysis of nine case-control studies estimated a hazard ratio of 0.482, which was also significant <span class="citation" data-cites="ZHqWXFcN">[<a href="#ref-ZHqWXFcN" role="doc-biblioref">586</a>]</span>.
Although these estimates are similar, it is important to note that they are drawing from the same literature and are therefore likely to be affected by the same biases in publication.
A second systematic review of studies investigating TCZ treatment for COVID-19 analyzed 31 studies that had been published or released as pre-prints and reported that none carried a low risk of bias (RoB) <span class="citation" data-cites="h4reluu">[<a href="#ref-h4reluu" role="doc-biblioref">587</a>]</span>.
Therefore, the present evidence is not likely to be sufficient for conclusions about the efficacy of TCZ.</p>
<p>TCZ has been used for over a decade to treat RA <span class="citation" data-cites="CRriJXDk">[<a href="#ref-CRriJXDk" role="doc-biblioref">588</a>]</span>, and a recent study shows that the drug is safe for pregnant and breastfeeding women <span class="citation" data-cites="6xLhs9st">[<a href="#ref-6xLhs9st" role="doc-biblioref">589</a>]</span>.
However, TCZ may increase the risk of developing infections <span class="citation" data-cites="CRriJXDk">[<a href="#ref-CRriJXDk" role="doc-biblioref">588</a>]</span>, and RA patients with chronic hepatitis B (HB) infections had a high risk of HB virus reactivation when TCZ was administered in combination with other RA drugs <span class="citation" data-cites="IJzmtJOr">[<a href="#ref-IJzmtJOr" role="doc-biblioref">590</a>]</span>.
As a result, TCZ is contraindicated in patients with active infections such as tuberculosis <span class="citation" data-cites="SiOKIbl3">[<a href="#ref-SiOKIbl3" role="doc-biblioref">591</a>]</span>.
Previous studies have investigated, with varying results, a possible increased risk of infection in RA patients administered TCZ <span class="citation" data-cites="6Agll71m zRRSvckT">[<a href="#ref-6Agll71m" role="doc-biblioref">592</a>,<a href="#ref-zRRSvckT" role="doc-biblioref">593</a>]</span>, although another study reported that the incidence rate of infections was higher in clinical practice RA patients treated with TCZ than in the rates reported by clinical trials <span class="citation" data-cites="TgV8gb30">[<a href="#ref-TgV8gb30" role="doc-biblioref">594</a>]</span>.
In the investigation of 544 Italian COVID-19 patients, the group treated with TCZ was found to be more likely to develop secondary infections, with 24% compared to 4% in the control group <span class="citation" data-cites="ayc7zYqi">[<a href="#ref-ayc7zYqi" role="doc-biblioref">580</a>]</span>.
Reactivation of hepatitis B and herpes simplex virus 1 was also reported in a small number of patients in this study, all of whom were receiving TCZ.
A July 2020 case report described negative outcomes of two COVID-19 patients after receiving TCZ, including one death; however, both patients were intubated and had entered septic shock prior to receiving TCZ <span class="citation" data-cites="p0tzgJvA">[<a href="#ref-p0tzgJvA" role="doc-biblioref">595</a>]</span>, likely indicating a severe level of cytokine production.
Additionally, D-dimer and sIL2R levels were reported by one study to increase in patients treated with TCZ, which raised concerns because of the potential association between elevated D-dimer levels and thrombosis and between sIL2R and diseases where T-cell regulation is compromised <span class="citation" data-cites="Wt9vgKLY">[<a href="#ref-Wt9vgKLY" role="doc-biblioref">581</a>]</span>.
An increased risk of bacterial infection was also identified in a systematic review of the literature, based on the unadjusted estimates reported <span class="citation" data-cites="3yV2bL9S">[<a href="#ref-3yV2bL9S" role="doc-biblioref">585</a>]</span>.
In summary, TCZ administration to COVID-19 patients is not without risks, may introduce additional risk of developing secondary infections, and should be approached especially cautiously for patients who have latent viral infections.</p>
<p>In summary, approximately 25% of coronavirus patients develop ARDS, which is caused by an excessive early response of the immune system which can be a component of cytokine release syndrome <span class="citation" data-cites="Wt9vgKLY">[<a href="#ref-Wt9vgKLY" role="doc-biblioref">581</a>]</span> and cytokine storm syndrome <span class="citation" data-cites="SiOKIbl3">[<a href="#ref-SiOKIbl3" role="doc-biblioref">591</a>]</span>.
This overwhelming inflammation is triggered by IL-6.
TCZ is an inhibitor of IL-6 and therefore may be able to neutralize the inflammatory pathway that leads to the cytokine storm.
While the mechanism suggests TCZ may be beneficial for the treatment of COVID-19 patients experiencing excessive immune activity, no randomized controlled trials are available assessing its effect.
However, small initial studies have found preliminary indications that TCZ may reduce progression to invasive medical ventilation and/or death.
It should be noted that SOC varied widely across retrospective studies, with one study administering HCQ, lopinavir-ritonavir, antibiotics, and/or heparin as part of standard care.
Interest in TCZ as a treatment for COVID-19 was supported by two meta-analyses that converged on a hazard ratio estimate of approximately 0.45 <span class="citation" data-cites="ZHqWXFcN 3yV2bL9S">[<a href="#ref-3yV2bL9S" role="doc-biblioref">585</a>,<a href="#ref-ZHqWXFcN" role="doc-biblioref">586</a>]</span>, but a third meta-analysis found that all of the available literature carries a risk of bias, with even the largest available TCZ studies to date carrying a moderate risk of bias under the ROBINS-I criteria <span class="citation" data-cites="h4reluu">[<a href="#ref-h4reluu" role="doc-biblioref">587</a>]</span>.
Additionally, different studies used different dosages, number of doses, and methods of administration; ongoing research may be needed to optimize administration of TCZ <span class="citation" data-cites="hvOwSJvK">[<a href="#ref-hvOwSJvK" role="doc-biblioref">596</a>]</span>, although similar results were reported by one study for intravenous and subcutaneous administration <span class="citation" data-cites="ayc7zYqi">[<a href="#ref-ayc7zYqi" role="doc-biblioref">580</a>]</span>.
Clinical trials that are in progress are likely to provide additional insight into the effectiveness of this drug for the treatment of COVID-19 along with how it should be administered.</p>
<h3 data-number="5.2.2" id="neutralizing-antibodies"><span class="header-section-number">5.2.2</span> Neutralizing Antibodies</h3>
<p>Monoclonal antibodies have revolutionized the way we treat human diseases.
They have become some of the best-selling drugs in the pharmaceutical market in recent years <span class="citation" data-cites="1AKi0FYUB">[<a href="#ref-1AKi0FYUB" role="doc-biblioref">597</a>]</span>.
There are currently 79 FDA approved mAbs on the market including antibodies for viral infections (e.g. Ibalizumab for HIV and Palivizumab for RSV) <span class="citation" data-cites="1AKi0FYUB yumBaJ6U">[<a href="#ref-1AKi0FYUB" role="doc-biblioref">597</a>,<a href="#ref-yumBaJ6U" role="doc-biblioref">598</a>]</span>.
Virus-specific neutralizing antibodies commonly target viral surface glycoproteins or host structures, thereby inhibiting viral entry through receptor binding interference <span class="citation" data-cites="dUnB3gD6 cwbHiM1o">[<a href="#ref-dUnB3gD6" role="doc-biblioref">599</a>,<a href="#ref-cwbHiM1o" role="doc-biblioref">600</a>]</span>.
This section discusses current efforts in developing neutralizing antibodies against SARS-CoV-2 and how expertise gained from previous approaches for MERS-CoV and SARS-CoV may benefit antibody development.</p>
<h4 data-number="5.2.2.1" id="spike-s-neutralizing-antibody"><span class="header-section-number">5.2.2.1</span> Spike (S) Neutralizing Antibody</h4>
<p>During the first SARS epidemic in 2002, nAbs were found in SARS-CoV infected patients <span class="citation" data-cites="3gp7t98X KULxo48U">[<a href="#ref-3gp7t98X" role="doc-biblioref">601</a>,<a href="#ref-KULxo48U" role="doc-biblioref">602</a>]</span>.
Several studies following up on these findings identified various S-glycoprotein epitopes as the major targets of neutralizing antibodies against SARS-CoV <span class="citation" data-cites="MTcTiS7i">[<a href="#ref-MTcTiS7i" role="doc-biblioref">603</a>]</span>.
The passive transfer of immune serum containing nAbs from SARS-CoV-infected mice resulted in protection of naïve mice from viral lower respiratory tract infection upon intranasal challenge <span class="citation" data-cites="rXzkS3gA">[<a href="#ref-rXzkS3gA" role="doc-biblioref">604</a>]</span>.
Similarly, a meta-analysis suggested that administration of plasma from recovered SARS-CoV patients reduced mortality upon SARS-CoV infection <span class="citation" data-cites="RRx6YovP">[<a href="#ref-RRx6YovP" role="doc-biblioref">605</a>]</span>.
Similar results were observed in MERS-CoV infection during the second coronavirus-related epidemic of the 21st century.
In these cases, neutralizing antibodies were identified against various epitopes of the RBD of the S glycoprotein <span class="citation" data-cites="Ou3P9pGV b4TFEgIi">[<a href="#ref-Ou3P9pGV" role="doc-biblioref">606</a>,<a href="#ref-b4TFEgIi" role="doc-biblioref">607</a>]</span>.
Coronaviruses use trimeric spike (S) glycoproteins on their surface to bind to host cell receptors, such as ACE2, allowing for cell entry <span class="citation" data-cites="15l3di3Wj qcVbT0w4">[<a href="#ref-qcVbT0w4" role="doc-biblioref">62</a>,<a href="#ref-15l3di3Wj" role="doc-biblioref">65</a>]</span>.
Each S glycoprotein protomer is comprised of an S1 domain, also called the receptor binding domain (RBD), and an S2 domain.
The S1 domain binds to host cell receptors while the S2 domain facilitates the fusion between the viral envelope and host cell membranes <span class="citation" data-cites="MTcTiS7i">[<a href="#ref-MTcTiS7i" role="doc-biblioref">603</a>]</span>.
Although targeting of the host cell receptor ACE2 shows efficacy in inhibiting SARS-CoV-2 infection <span class="citation" data-cites="2QTH37Xi">[<a href="#ref-2QTH37Xi" role="doc-biblioref">73</a>]</span>, given the physiological relevance of ACE2 <span class="citation" data-cites="1GLr0EJU">[<a href="#ref-1GLr0EJU" role="doc-biblioref">608</a>]</span>, it would be favorable to target virus-specific structures rather than host receptors.
This forms the rationale of developing neutralizing antibodies against the S glycoprotein, disrupting its interaction with ACE2 and other receptors and thereby inhibiting viral entry.</p>
<p>The first human neutralizing antibody against SARS-CoV-2 targeting the trimeric spike (S) glycoproteins was developed using hybridoma technology <span class="citation" data-cites="XcuzhxrJ">[<a href="#ref-XcuzhxrJ" role="doc-biblioref">609</a>]</span>, where antibody-producing B-cells developed by mice can be inserted into myeloma cells to produce a hybrid cell line (the hybridoma) that is grown in culture.
The 47D11 clone was able to cross-neutralize SARS-CoV and SARS-CoV2 by a mechanism that is different from receptor binding interference.
The exact mechanism of how this clone neutralizes SARS-CoV-2 and inhibits infection <em>in vitro</em> remains unknown, but a potential mechanism might be antibody-induced destabilization of the membrane prefusion structure <span class="citation" data-cites="XcuzhxrJ Ftbm1M9p">[<a href="#ref-XcuzhxrJ" role="doc-biblioref">609</a>,<a href="#ref-Ftbm1M9p" role="doc-biblioref">610</a>]</span>.
The ability of this antibody to prevent infection at a feasible dose needs to be validated <em>in vivo</em>, especially since <em>in vitro</em> neutralization effects have been shown to not be reflective of <em>in vivo</em> efficacy <span class="citation" data-cites="O1whWg6Q">[<a href="#ref-O1whWg6Q" role="doc-biblioref">611</a>]</span>.
Only a week later, a different group successfully isolated multiple nAbs targeting the RBD of the S glycoprotein from blood samples taken from COVID-19 patients in China <span class="citation" data-cites="GhJYjnft">[<a href="#ref-GhJYjnft" role="doc-biblioref">144</a>]</span>.
Interestingly, the patient-isolated antibodies did not cross-react with RBDs from SARS-CoV and MERS-CoV, although cross-reactivity to the trimeric spike proteins of SARS-CoV and MERS-CoV was observed.
This finding suggests that the RBDs between the three coronavirus species are immunologically distinct and that the isolated nAbs targeting the RBD of SARS-CoV-2 are species specific.
While this specificity is desirable, it also raises the question of whether these antibodies are more susceptible to viral escape mechanisms.
Viral escape is a common resistance mechanism to nAbs therapy due to selective pressure from neutralizing antibodies <span class="citation" data-cites="pUnzB8wV 14motyCOm">[<a href="#ref-pUnzB8wV" role="doc-biblioref">612</a>,<a href="#ref-14motyCOm" role="doc-biblioref">613</a>]</span>.
For HIV, broadly neutralizing antibodies (bnAbs) targeting the CD4 binding site (CD4bs) show greater neutralization breadth than monoclonal antibodies, which target only specific HIV strains <span class="citation" data-cites="oGHFF5Cm">[<a href="#ref-oGHFF5Cm" role="doc-biblioref">614</a>]</span>.
For MERS-CoV, a combination of multiple neutralizing antibodies targeting different antigenic sites prevented neutralization escape <span class="citation" data-cites="hvjXMm9H">[<a href="#ref-hvjXMm9H" role="doc-biblioref">615</a>]</span>.
It was found that the different antibody isolates did not target the same epitopes, suggesting that using them in combination might produce a synergistic effect that prevents viral escape <span class="citation" data-cites="GhJYjnft">[<a href="#ref-GhJYjnft" role="doc-biblioref">144</a>]</span>.
It was also demonstrated that binding affinity of the antibodies does not reflect their capability to compete with ACE2 binding.
Furthermore, no conclusions about correlations between the severity of disease and the ability to produce neutralizing antibodies can be drawn at this point.
Rather, higher neutralizing antibody titers were more frequently found in patients with severe disease.
Correspondingly, higher levels of anti-spike IgG were observed in patients that deceased from infection compared to patient that recovered <span class="citation" data-cites="tC5vJmwj">[<a href="#ref-tC5vJmwj" role="doc-biblioref">616</a>]</span>.</p>
<p>Results from the SARS and MERS epidemics thus provide valuable lessons for the design of neutralizing antibodies for the current outbreak.
The findings for SARS-CoV and MERS-CoV can aid in identifying which structures constitute suitable targets for nAbs, despite the fact that the RBD appears to be distinct between the three coronavirus species.
These studies also suggest that a combination of nAbs targeting distinct antigens might be necessary to provide protection <span class="citation" data-cites="hvjXMm9H">[<a href="#ref-hvjXMm9H" role="doc-biblioref">615</a>]</span>.
The biggest challenge remains identifying antibodies that not only bind to their target, but also prove to be beneficial for disease management.
On that note, a recently published study indicates that anti-spike antibodies could make the disease worse rather than eliminating the virus <span class="citation" data-cites="tC5vJmwj">[<a href="#ref-tC5vJmwj" role="doc-biblioref">616</a>]</span>.
These findings underscores our current lack of understanding the full immune response to SARS-CoV-2.</p>
<h3 data-number="5.2.3" id="interferons"><span class="header-section-number">5.2.3</span> Interferons</h3>
<p>Interferons (IFNs) are a family of cytokines crucial to activating the innate immune system response against viral infections.
Interferons are classified into three categories based on their receptor specificity: types I, II and III <span class="citation" data-cites="bZMKqj6e">[<a href="#ref-bZMKqj6e" role="doc-biblioref">109</a>]</span>.
Specifically, IFNs I (IFN-𝛼 and 𝛽) and II (IFN-𝛾) induce the expression of antiviral proteins that bring the viral RNA to degradation <span class="citation" data-cites="LtFtUo2P">[<a href="#ref-LtFtUo2P" role="doc-biblioref">617</a>]</span>.
Among these IFNs, IFN-𝛽 has already been found to strongly inhibit the replication of other coronaviruses, such as SARS-CoV, in cell culture, while IFN-𝛼 and 𝛾 were shown to be less effective in this context <span class="citation" data-cites="LtFtUo2P">[<a href="#ref-LtFtUo2P" role="doc-biblioref">617</a>]</span>.
There is evidence that patients with higher susceptibility to ARDS indeed show deficiency in IFN- 𝛽.
For instance, infection with other coronaviruses impairs IFN-𝛽 expression and synthesis, allowing the virus to escape the innate immune response <span class="citation" data-cites="Nfu9kiae">[<a href="#ref-Nfu9kiae" role="doc-biblioref">618</a>]</span>.
On March 18 2020, Synairgen plc received approval to start a phase II trial for SNG001, an IFN-𝛽-1a formulation to be delivered to the lungs via inhalation.
SNG001 was already shown to be effective in reducing viral load in an <em>in vivo</em> model of swine flu and <em>in vitro</em> models of other coronavirus infections <span class="citation" data-cites="s9C2iFSR">[<a href="#ref-s9C2iFSR" role="doc-biblioref">619</a>]</span>.
In July, a press release from Synairgen stated that SNG001 reduced progression to ventilation in a double-blind, placebo-controlled, multi-center study of 101 patients with an average age in the late 50s <span class="citation" data-cites="4Tx92JMv">[<a href="#ref-4Tx92JMv" role="doc-biblioref">620</a>]</span>.
They also reported that patients in the treatment group showed greater recovery and lower breathlessness.
However, given that this information was released in a press release rather than in a manuscript and thus cannot be thoroughly reviewed, these findings should be considered preliminary.</p>
<h3 data-number="5.2.4" id="vaccines"><span class="header-section-number">5.2.4</span> Vaccines</h3>
<h4 data-number="5.2.4.1" id="vaccine-development"><span class="header-section-number">5.2.4.1</span> Vaccine Development</h4>
<p>Flu-like illnesses caused by viruses are a common target of vaccine development, and influenza vaccine technology in particular has made many strides.
During the H1N1 influenza outbreak, vaccine development was accelerated because of the existing infrastructure, along with the fact that regulatory agencies had already decided that vaccines produced using egg- and cell-based platforms could be licensed under the regulations used for a strain change.
Critiques of the production and distribution of the H1N1 vaccine have stressed the need for alternative development-and-manufacturing platforms that can be readily adapted to new pathogens.
Although a monovalent H1N1 vaccine was not available before the pandemic peaked in the United States and Europe, it was available soon afterward as a stand-alone vaccine that was eventually incorporated into commercially available seasonal influenza vaccines <span class="citation" data-cites="HyYY2agc">[<a href="#ref-HyYY2agc" role="doc-biblioref">621</a>]</span>.
If H1N1 vaccine development provides any indication, considering developing and manufacturing platforms for promising COVID-19 vaccine trials early could hasten the emergence of an effective prophylactic vaccine against SARS-CoV-2.</p>
<p>The first critical step towards developing a vaccine against SARS-CoV-2 was characterizing the target, which fortunately happened early in the COVID-19 outbreak with the sequencing and dissemination of the viral genome <span class="citation" data-cites="SvOLLYfw">[<a href="#ref-SvOLLYfw" role="doc-biblioref">622</a>]</span>.
The Coalition for Epidemic Preparedness Innovations (CEPI) is coordinating global health agencies and pharmaceutical companies to develop vaccines against SARS-CoV-2.
As of September 2020, there are over 180 vaccine candidates against SARS-CoV-2 in development <span class="citation" data-cites="dqpEe5Lz">[<a href="#ref-dqpEe5Lz" role="doc-biblioref">623</a>]</span>.
Of the 78 active vaccine programs in April 2020, 73 were in the preclinical or exploratory stage <span class="citation" data-cites="1GA95MF2m">[<a href="#ref-1GA95MF2m" role="doc-biblioref">624</a>]</span>.
Unlike many global vaccine development programs previously, such as with H1N1, the vaccine development landscape for COVID-19 includes vaccines produced by a wide array of technologies.
Experience in the field of oncology is encouraging COVID-19 vaccine developers to use next-generation approaches to vaccine development, which have led to the great diversity of vaccine development programs <span class="citation" data-cites="wPl93ATP">[<a href="#ref-wPl93ATP" role="doc-biblioref">625</a>]</span>.
Diverse technology platforms include DNA, RNA, virus-like particle, recombinant protein, both replicating and non-replicating viral vectors, live attenuated virus, and inactivated virus approaches (Figure <a href="#fig:vaccines">4</a>).
Given the wide range of vaccines under development, it is possible that some vaccine products may eventually be shown to be more effective in certain subpopulations, such as children, pregnant women, immunocompromised patients, the elderly, etc.
The requirements for a successful vaccine trial and deployment are complex and may require coordination between government, industry, academia, and philanthropic entities <span class="citation" data-cites="plfPrQP7">[<a href="#ref-plfPrQP7" role="doc-biblioref">626</a>]</span>.
While little is currently known about immunity to SARS-CoV-2, vaccine development typically tests for serum neutralizing activity, as this has been established as a biomarker for adaptive immunity in other respiratory illnesses <span class="citation" data-cites="wiGjCZC8">[<a href="#ref-wiGjCZC8" role="doc-biblioref">627</a>]</span>.</p>
<div id="fig:vaccines" class="fignos">
<figure>
<img src="images/N002-Vaccines.png" alt="" /><figcaption><span>Figure 4:</span> <strong>Vaccine Development Strategies.</strong>
Several different strategies can and are being employed for the development of vaccines.
Each approach capitalizes on different features of the SARS-CoV-2 virus and delivery through a different platform.</figcaption>
</figure>
</div>
<h5 data-number="5.2.4.1.1" id="dna-vaccines"><span class="header-section-number">5.2.4.1.1</span> DNA Vaccines</h5>
<p>This vaccination method involves the direct introduction of a plasmid containing a DNA sequence encoding the antigen(s) against which an immune response is sought into appropriate tissues <span class="citation" data-cites="XnrBoKVk">[<a href="#ref-XnrBoKVk" role="doc-biblioref">628</a>]</span>.
This approach may offer several advantages over traditional vaccination approaches, such as the stimulation of both B- as well as T-cell responses and the absence of any infectious agent.
Currently, a Phase I safety and immunogenicity clinical trial of INO-4800, a prophylactic vaccine against SARS-CoV-2, is underway <span class="citation" data-cites="xuzLfS0y">[<a href="#ref-xuzLfS0y" role="doc-biblioref">629</a>]</span>.
The vaccine developer Inovio Pharmaceuticals Technology is overseeing administration of INO-4800 by intradermal injection followed by electroporation with the CELLECTRA® device to healthy volunteers.
Electroporation is the application of brief electric pulses to tissues in order to permeabilize cell membranes in a transient and reversible manner.
It has been shown that electroporation can enhance vaccine efficacy by up to 100-fold, as measured by increases in antigen-specific antibody titers <span class="citation" data-cites="H6tWVs5R">[<a href="#ref-H6tWVs5R" role="doc-biblioref">630</a>]</span>.
The safety of the CELLECTRA® device has been studied for over seven years, and these studies support the further development of electroporation as a safe vaccine delivery method <span class="citation" data-cites="1Hsm2J1sc">[<a href="#ref-1Hsm2J1sc" role="doc-biblioref">631</a>]</span>.
The temporary formation of pores through electroporation facilitates the successful transportation of macromolecules into cells, allowing cells to robustly take up INO-4800 for the production of an antibody response.</p>
<p>Approved by the U.S. FDA on April 6, 2020, the Phase I study is enrolling up to 40 healthy adult volunteers in Philadelphia, PA at the Perelman School of Medicine and at the Center for Pharmaceutical Research in Kansas City, MO.
The trial has two experimental arms corresponding to the two locations.
Participants in Experimental Group 1 will receive one intradermal injection of 1.0 milligram (mg) of INO-4800 followed by electroporation using the CELLECTRA® 2000 device twice, administered at Day 0 and Week 4.
Participants in Experimental Group 2 will receive two intradermal injections of 1.0 mg (total 2.0 mg per dosing visit) of INO-4800 followed by electroporation using the CELLECTRA® 2000 device, administered at Day 0 and Week 4.
Safety data and the initial immune responses of participants from the trial are expected by the end of the summer of 2020.
The development of a DNA vaccine against SARS-CoV-2 by Inovio could be an important step forward in the world’s search for a COVID-19 vaccine.
Although exciting, the cost of vaccine manufacturing and electroporation may make scaling the use of this technology for prophylactic use for the general public difficult.</p>
<h5 data-number="5.2.4.1.2" id="rna-vaccines"><span class="header-section-number">5.2.4.1.2</span> RNA Vaccines</h5>
<p>RNA vaccines are nucleic-acid based modalities that code for viral antigens against which the human body elicits a humoral and cellular immune response.
The mRNA technology is transcribed <em>in vitro</em> and delivered to cells via lipid nanoparticles (LNP) <span class="citation" data-cites="HCImhzy8">[<a href="#ref-HCImhzy8" role="doc-biblioref">632</a>]</span>.
They are recognized by ribosomes <em>in vivo</em> and then translated and modified into functional proteins <span class="citation" data-cites="K0Ltu31S">[<a href="#ref-K0Ltu31S" role="doc-biblioref">633</a>]</span>.
The resulting intracellular viral proteins are displayed on surface MHC proteins, provoking a strong CD8+ T cell response as well as a CD4+ T cell and B cell-associated antibody responses <span class="citation" data-cites="K0Ltu31S">[<a href="#ref-K0Ltu31S" role="doc-biblioref">633</a>]</span>.
Naturally, mRNA is not very stable and can degrade quickly in the extracellular environment or the cytoplasm.
The LNP covering protects the mRNA from enzymatic degradation outside of the cell <span class="citation" data-cites="zNKWlCwE">[<a href="#ref-zNKWlCwE" role="doc-biblioref">634</a>]</span>.
Codon optimization to prevent secondary structure formation and modifications of the poly-A tail as well as the 5’ untranslated region to promote ribosomal complex binding can increase mRNA expression in cells.
Furthermore, purifying out dsRNA and immature RNA with FPLC (fast performance liquid chromatography) and HPLC (high performance liquid chromatography) technology will improve translation of the mRNA in the cell <span class="citation" data-cites="K0Ltu31S pRoqjur8">[<a href="#ref-K0Ltu31S" role="doc-biblioref">633</a>,<a href="#ref-pRoqjur8" role="doc-biblioref">635</a>]</span>.
Vaccines based on mRNA delivery confer many advantages over traditional viral vectored vaccines and DNA vaccines.
In comparison to live attenuated viruses, mRNA vaccines are non-infectious and can be synthetically produced in an egg-free, cell-free environment, thereby reducing the risk of a detrimental immune response in the host <span class="citation" data-cites="wYZ6qJMu">[<a href="#ref-wYZ6qJMu" role="doc-biblioref">636</a>]</span>.
Unlike DNA vaccines, mRNA technologies are naturally degradable and non-integrating, and they do not need to cross the nuclear membrane in addition to the plasma membrane for their effects to be seen <span class="citation" data-cites="K0Ltu31S">[<a href="#ref-K0Ltu31S" role="doc-biblioref">633</a>]</span>.
Furthermore, mRNA vaccines are easily, affordably, and rapidly scalable.</p>
<p>Although mRNA vaccines have been developed for therapeutic and prophylactic purposes, none have been licensed or commercialized thus far.
Nevertheless, they have shown promise in animal models and preliminary clinical trials for several indications, including rabies, coronavirus, influenza, and cytomegalovirus <span class="citation" data-cites="3EUiWZdN">[<a href="#ref-3EUiWZdN" role="doc-biblioref">637</a>]</span>.
Preclinical data from Pardi et al. identified effective antibody generation against full-length FPLC-purified influenza hemagglutinin stalk-encoding mRNA in mice, rabbits, and ferrets <span class="citation" data-cites="6wZy2mn8">[<a href="#ref-6wZy2mn8" role="doc-biblioref">638</a>]</span>.
Similar immunological responses for mRNA vaccines were observed in humans in Phase I and II clinical trials operated by the pharmaceutical-development companies Curevac and Moderna for rabies, flu, and zika <span class="citation" data-cites="pRoqjur8">[<a href="#ref-pRoqjur8" role="doc-biblioref">635</a>]</span>.
Positively charged bilayer LNPs carrying the mRNA attract negatively charged cell membranes, endocytose into the cytoplasm <span class="citation" data-cites="zNKWlCwE">[<a href="#ref-zNKWlCwE" role="doc-biblioref">634</a>]</span>, and facilitate endosomal escape.
LNPs can be coated with modalities recognized and engulfed by specific cell types.
LNPs 150nm or less effectively enter into lymphatic vessels.</p>
<p>There are three types of RNA vaccines: non-replicating, <em>in vivo</em> self-replicating, and <em>in vitro</em> dendritic cell non-replicating <span class="citation" data-cites="1EM5nGaYd">[<a href="#ref-1EM5nGaYd" role="doc-biblioref">639</a>]</span>.
Non-replicating mRNA vaccines consist of a simple open reading frame (ORF) for the viral antigen flanked by the 5’ UTR and 3’ poly-A tail.
<em>In vivo</em> self-replicating vaccines encode a modified viral genome derived from single-stranded, positive sense RNA alphaviruses <span class="citation" data-cites="K0Ltu31S pRoqjur8">[<a href="#ref-K0Ltu31S" role="doc-biblioref">633</a>,<a href="#ref-pRoqjur8" role="doc-biblioref">635</a>]</span>.
The RNA genome encodes the viral antigen along with proteins of the genome replication machinery, including an RNA polymerase.
Structural proteins required for viral assembly are not included in the engineered genome <span class="citation" data-cites="K0Ltu31S">[<a href="#ref-K0Ltu31S" role="doc-biblioref">633</a>]</span>.
Self-replicating vaccines produce more viral antigens over a longer period of time, thereby evoking a more robust immune response <span class="citation" data-cites="1EM5nGaYd">[<a href="#ref-1EM5nGaYd" role="doc-biblioref">639</a>]</span>.
Finally, <em>in vitro</em> dendritic cell non-replicating RNA vaccines limit transfection to dendritic cells.
Dendritic cells are potent antigen-presenting immune cells that easily take up mRNA and present fragments of the translated peptide on their MHC proteins, which can then interact with T cell receptors.
Ultimately, primed T follicular helper cells can stimulate germinal center B cells that also present the viral antigen to produce antibodies against the virus <span class="citation" data-cites="3LMMW7F0">[<a href="#ref-3LMMW7F0" role="doc-biblioref">640</a>]</span>.
These cells are isolated from the patient, grown and transfected <em>ex vivo</em>, and reintroduced to the patient <span class="citation" data-cites="ENBWnhAh">[<a href="#ref-ENBWnhAh" role="doc-biblioref">641</a>]</span>.</p>
<p>mRNA-1273 was the first COVID-19 vaccine to enter a phase I clinical in the United States.
ModernaTX, Inc. is currently spearheading an investigation on the immunogenicity and reactogenicity of mRNA-1273, a conventional lipid nanoparticle encapsulated RNA encoding a full-length prefusion stabilized spike (S) protein for SARS-CoV-2 <span class="citation" data-cites="Biu1CQeQ">[<a href="#ref-Biu1CQeQ" role="doc-biblioref">642</a>]</span>.
In a study that is ongoing, forty-five participants were enrolled and given intramuscular injections of mRNA-1273 in their deltoid muscle on day 1 and day 29, and then followed for the next twelve months.
Healthy males and non-pregnant females aged 18-55 years were recruited for this study and divided into three groups receiving 25, 100, or 250 micrograms (μg) of mRNA-1273.
IgG ELISA assays on patient serology samples are being used to examine the immunogenicity of the vaccine <span class="citation" data-cites="Biu1CQeQ">[<a href="#ref-Biu1CQeQ" role="doc-biblioref">642</a>]</span>.</p>
<p>A preliminary report describing initial safety and immunogenicity findings is now available <span class="citation" data-cites="wiGjCZC8">[<a href="#ref-wiGjCZC8" role="doc-biblioref">627</a>]</span>.
Binding antibodies were observed at two weeks after the first dose at all concentrations.
At the time point one week after the second dose was administered on day 29, the pseudotyped lentivirus reporter single-round-of-infection neutralization assay (PsVNA), which was used to assess neutralizing activity, reached a median level similar to the median observed in convalescent plasma samples.
Participants reported mild and moderate systemic adverse events after the day 1 injection, and one severe local event was observed in each of the two highest dose levels.
The second injection led to severe systemic adverse events for three of the participants at the highest dose levels, with one participant in the group being evaluated at an urgent care center on the day after the second dose.
The reported localized adverse events from the second dose were similar to those from the first.</p>
<p>In summary, mRNA vaccines are promising tools in the prevention and control of pandemics because they may sidestep many challenges associated with traditional vaccine design and manufacturing.
<!-- mRNA-1273 is the only RNA vaccine for SARS-CoV-2 for which preliminary results are available. -->
As of August 2020, placebo-blinded and randomized results for mRNA vaccines are not yet available, but the initial results are promising.
In the dose ranging study, even the lowest dose of 25 μg of mRNA-1273 appeared to be sufficient to induce an immunogenicity profile comparable to that observed in convalescent plasma and appeared to be well tolerated among the population of healthy, non-pregnant participants from 18-55 years of age.
Based on the preliminary results from mRNA-1273 <span class="citation" data-cites="wiGjCZC8">[<a href="#ref-wiGjCZC8" role="doc-biblioref">627</a>]</span>, a new trial aims to enroll 30,000 participants who will be randomized for two injections of 100 μg of mRNA-1273 or placebo, again spaced 28 days apart <span class="citation" data-cites="m4J5hWsz">[<a href="#ref-m4J5hWsz" role="doc-biblioref">643</a>]</span>.</p>
<h4 data-number="5.2.4.2" id="adjuvants-for-vaccines"><span class="header-section-number">5.2.4.2</span> Adjuvants for Vaccines</h4>
<p>Adjuvants include a variety of molecules or larger microbial-related products that have an effect on the immune system or an immune response of interest.
They can either be comprised of or contain immunostimulants or immunomodulators.
Adjuvants are sometimes included within vaccines, especially vaccines other than live-attenuated and inactivated viruses, in order to enhance the immune response.
A review on the development of SARS-CoV-2 vaccines <span class="citation" data-cites="ouOXy0wH">[<a href="#ref-ouOXy0wH" role="doc-biblioref">644</a>]</span> also included a brief summary of the potential of adjuvants for these vaccines, including a brief description of some already commonly used adjuvants.
Different adjuvants can regulate different types of immune responses, so the type or combination of adjuvants used in a vaccine will depend on both the type of vaccine and concern related to efficacy and safety.
A variety of possible mechanisms for adjuvants have been researched <span class="citation" data-cites="13bVbfc5h 122h6fIxE uO0uqhxc">[<a href="#ref-13bVbfc5h" role="doc-biblioref">645</a>,<a href="#ref-122h6fIxE" role="doc-biblioref">646</a>,<a href="#ref-uO0uqhxc" role="doc-biblioref">647</a>]</span>, including the following: induction of DAMPs (damage-associated molecular patterns) that can be recognized by certain pattern recognition receptors (PRRs) of the innate immune system; functioning as PAMPs (pathogen-associated molecular patterns) that can also be recognized by certain pattern recognition receptors (PRRs); and more generally enhancing the humoral or cellular immune responses.
Selection of one or more adjuvants requires considering how to promote the advantageous effects of the components and/or immune response and, likewise, to inhibit possible deleterious effects.
There are also considerations related to the method of delivering (or co-delivering) the adjuvant and antigen components of a vaccine.</p>
<h4 data-number="5.2.4.3" id="trained-immunity"><span class="header-section-number">5.2.4.3</span> Trained Immunity</h4>
<p>Another approach that is being investigated explores the potential for vaccines that are not made from the SARS-CoV-2 virus to confer what has been termed trained immunity.
In a recent review <span class="citation" data-cites="103fS7Kz2">[<a href="#ref-103fS7Kz2" role="doc-biblioref">648</a>]</span>, trained immunity was defined as forms of memory that are temporary (e.g., months or years) and reversible.
It is induced by exposure to whole-microorganism vaccines or other microbial stimuli that generates heterologous protective effects.
Trained immunity can be displayed by innate immune cells or innate immune features of other cells, and it is characterized by alterations to immune responsiveness to future immune challenges due to epigenetic and metabolic mechanisms.
These alterations can take the form of either an increased or decreased response to immune challenge by a pathogen.
Trained immunity elicited by non-SARS-CoV-2 whole-microorganism vaccines could potentially improve SARS-CoV-2 susceptibility or severity <span class="citation" data-cites="Vu1VILWK">[<a href="#ref-Vu1VILWK" role="doc-biblioref">649</a>]</span>.</p>
<p>One type of stimulus which research indicates can induce trained immunity is bacillus Calmette-Guerin (BCG) vaccination.
BCG is an attenuated form of bacteria <em>Mycobacterium bovis</em>.
The vaccine is most commonly administered for the prevention of tuberculosis in humans.
Clinical trials in non-SARS-CoV-2-infected adults have been designed to assess whether BCG vaccination could have prophylactic effects against SARS-CoV-2 by reducing susceptibility, preventing infection, or reducing disease severity.
A number of trials are now evaluating the effects of the BCG vaccine or the related vaccine VPM1002 <span class="citation" data-cites="Vu1VILWK 9m3rP633 y9IYdfM3 xdqxBruc 962rELVS EuwTWcPi dQtUeruv DjXsPR8O 10OE6y3Pv 86OjIybR ITO15LIz VkZGZxLn 13JVjMfQI nk2MVsld 1E2t9tr8h">[<a href="#ref-Vu1VILWK" role="doc-biblioref">649</a>,<a href="#ref-9m3rP633" role="doc-biblioref">650</a>,<a href="#ref-y9IYdfM3" role="doc-biblioref">651</a>,<a href="#ref-xdqxBruc" role="doc-biblioref">652</a>,<a href="#ref-962rELVS" role="doc-biblioref">653</a>,<a href="#ref-EuwTWcPi" role="doc-biblioref">654</a>,<a href="#ref-dQtUeruv" role="doc-biblioref">655</a>,<a href="#ref-DjXsPR8O" role="doc-biblioref">656</a>,<a href="#ref-10OE6y3Pv" role="doc-biblioref">657</a>,<a href="#ref-86OjIybR" role="doc-biblioref">658</a>,<a href="#ref-ITO15LIz" role="doc-biblioref">659</a>,<a href="#ref-VkZGZxLn" role="doc-biblioref">660</a>,<a href="#ref-13JVjMfQI" role="doc-biblioref">661</a>,<a href="#ref-nk2MVsld" role="doc-biblioref">662</a>,<a href="#ref-1E2t9tr8h" role="doc-biblioref">663</a>]</span>.</p>
<p>The ongoing trials are using a number of different approaches.
Some trials enroll healthcare workers, other trials hospitalized elderly adults without immunosuppression who get vaccinated with placebo or BCG at hospital discharge, and yet another set of trials older adults (&gt;50 years) under chronic care for conditions like hypertension and diabetes.
One set of trials, for example, uses time until first infection as the primary study endpoint; more generally, outcomes measured in some of these trials are related to incidence of disease and disease severity or symptoms.
Some analyses have suggested a possible correlation at the country level between the frequency of BCG vaccination (or BCG vaccination policies) and the severity of COVID-19 <span class="citation" data-cites="Vu1VILWK">[<a href="#ref-Vu1VILWK" role="doc-biblioref">649</a>]</span>.
Currently it is unclear whether this correlation has any connection to trained immunity.
Many possible confounding factors are also like to vary among countries, such as age distribution, detection efficiency, stochastic epidemic dynamic effects, differences in healthcare capacity over time in relation to epidemic dynamics, and these have not been adequately accounted for in current analyses. <!-- TO DO: add in study that debunks this-->
It is unclear whether there is an effect of the timing of BCG vaccination, both during an individual’s life cycle and relative to the COVID-19 pandemic.
Additionally, given that severe SARS-CoV-2 may be associated with a dysregulated immune response, it is unclear what alterations to the immune response would be most likely to be protective versus pathogenic (e.g., <span class="citation" data-cites="i5k18bpX 1GnFL9zeN Vu1VILWK 1228YjPRv">[<a href="#ref-1GnFL9zeN" role="doc-biblioref">107</a>,<a href="#ref-Vu1VILWK" role="doc-biblioref">649</a>,<a href="#ref-i5k18bpX" role="doc-biblioref">664</a>,<a href="#ref-1228YjPRv" role="doc-biblioref">665</a>]</span>).
The article <span class="citation" data-cites="Vu1VILWK">[<a href="#ref-Vu1VILWK" role="doc-biblioref">649</a>]</span> proposes that trained immunity might lead to an earlier and stronger response, which could in turn reduce viremia and the risk of later, detrimental immunopathology.
While trained immunity is an interesting possible avenue to complement vaccine development efforts through the use of an existing vaccine, additional research is required to assess whether the BCG vaccine is likely to confer trained immunity in the case of SARS-CoV-2.</p>
<h2 data-number="5.3" id="summary-of-prophylactics-and-therapeutics"><span class="header-section-number">5.3</span> Summary of Prophylactics and Therapeutics</h2>
<p>Currently, there are several possible prophylactics and therapeutics under investigation.
Early public attention focused on potential treatments whose therapeutic/prophylactic value has subsequently been disproved or remains under question, such as HCQ and some nutraceuticals, respectively.
Advances in prophylactics continue to develop, with several vaccines currently in clinical trials.
As research has progressed, several potential approaches to treatment have emerged (Figure <a href="#fig:therapeutics">5</a>).
Most notably, remdesivir has been approved by the FDA for the treatment of COVID-19, and dexamethasone, which was approved by the FDA in 1958, has been found to improve outcomes for patients with severe COVID-19.
As more evidence becomes available, the potential for existing and novel therapies to improve outcomes for COVID-19 patients will become better understood.
At present, a number of promising avenues continue to be explored.</p>
<div id="fig:therapeutics" class="fignos">
<figure>
<img src="images/N001-LifeCyclePlusDrugs.png" alt="" /><figcaption><span>Figure 5:</span> <strong>Mechanism of Action for Potential Therapeutics</strong>
Potential therapeutics currently being studied target the SARS-CoV-2 or modify the host environment through many different mechanisms.
Here, the relationship between the virus and several therapeutics described above are visualized.</figcaption>
</figure>
</div>
<h1 data-number="6" id="discussion"><span class="header-section-number">6</span> Discussion</h1>
<p>As of October 2020 the SARS-CoV-2 virus remains a serious worldwide threat.
The scientific community has responded by rapidly collecting and disseminating information about the SARS-CoV-2 virus and the associated illness, COVID-19.
The rapid identification of the genomic sequence of the virus allowed for early contextualization of SARS-CoV-2 among other known respiratory viruses.
The pathogen is a coronavirus that is closely related to SARS-CoV, which caused the SARS pandemics of the early 2000s.
Knowing the phylogenetic context and genomic sequence of the virus then allowed for rapid insights into its structure and pathogenesis.
As with other HCoV, the immune response to SARS-CoV-2 is likely driven by detection of its spike protein, which allows it to enter cells through the ACE2 receptor.
Epithelial cells have also emerged as the major cellular target of the virus, contextualizing the respiratory and gastrointestinal symptoms that are frequently observed in COVID-19.
However, as COVID-19 cases have been more widely characterized, it has become clear that the disease presentation is highly heterogeneous.
Many cases, especially in younger adults, present with mild symptoms or even asymptomatically, while others, especially in older adults, can be severe or fatal.
In children, the SARS-CoV-2 virus can present as two distinct diseases, COVID-19 or MIS-C.
While the overall infection fatality rate remains unknown, estimates suggest that it is not more than 1%; however, the severity of many non-lethal cases makes COVID-19 an ongoing, significant concern.</p>
<p>Characterizing the rate of infection and fatality rates hinges on the availability of rapid and accurate diagnostic testing.
Major advancements have been made in identifying diagnostic approaches.
The development of diagnostic technologies have been rapid, beginning with the release of the SARS-CoV-2 viral genome sequence in January.
As of October 2020, a range of diagnostic tests have become available.
One class of tests uses PCR (RT-PCR or qRT-PCR) to assess the presence of SARS-CoV-2 RNA, while another typically uses ELISA to test for the presence of antibodies to SARS-CoV-2.
The former approach is useful for identifying active infections, while the latter measures hallmarks of the immune response and therefore can detect either active infections or immunity gained from prior infection.
Combining these tests leads to extremely accurate detection of SARS-CoV-2 infection (98.6%), but when used alone, PCR-based tests are recommended before 5.5 days after the onset of the illness and antibody tests after 5.5 days <span class="citation" data-cites="CC4sPeuO">[<a href="#ref-CC4sPeuO" role="doc-biblioref">666</a>]</span>.
Other strategies for testing can also influence the tests’ accuracy, such as the use of nasopharyngeal swabs versus BALF <span class="citation" data-cites="CC4sPeuO">[<a href="#ref-CC4sPeuO" role="doc-biblioref">666</a>]</span>, which allow for trade-offs between patient’s comfort and test sensitivity.
Additionally, technologies such as digital PCR may allow for scale-up in the throughput of diagnostic testing, facilitating widespread testing.
One major question that remains is whether people who recover from SARS-CoV-2 develop sustained immunity, and over what period this immunity is expected to last.
Some reports have suggested that some patients may develop COVID-19 reinfections (e.g., <span class="citation" data-cites="80JdM9px">[<a href="#ref-80JdM9px" role="doc-biblioref">667</a>]</span>), but the rates of reinfection are currently unknown.
Serologic testing combined with PCR testing will be critical to confirming purported cases of reinfection and to identifying the duration over which immunity is retained and to understanding reinfection risks.</p>
<p>One of the goals of characterizing the immune response is to identify strategies for the prophylactic enhancements of immunity.
Though some concerns remain about the duration of sustained immunity for convalescents, vaccine development efforts are ongoing and show initial promising results.
The Moderna trial, for example, reported that the neutralizing activity in participants who received two doses of the vaccine was similar to that observed in convalescent plasma.
Vaccine development for COVID-19 is progressing rapidly compared to typical timelines, but vaccine development is still a lengthy process.
In the meantime, some advances have also been made in the treatment of patients with COVID-19.
As cases have become better characterized, it has become evident that many patients experience an initial immune response to the virus that is typically characterized by fever, cough, dyspnea, and related symptoms.
However, the most serious concern is cytokine release syndrome, when the body’s immune response becomes dysregulated, resulting in an extreme inflammatory response.
The RECOVERY trial, a large-scale, multi-arm trial enrolling about 15% of all COVID-19 patients in the United Kingdom, was the first to identify that the widely available steroid dexamethasone seems to be beneficial for patients suffering from this immune dysregulation <span class="citation" data-cites="pzQoUwz3">[<a href="#ref-pzQoUwz3" role="doc-biblioref">441</a>]</span>.
Efforts to identify therapeutic treatments to treat patients early in the course of infection have been more ambiguous.
Early interest in the drugs hydroxychloroquine and chloroquine yielded no promising results from studies with robust experimental designs.
The experimental drug remdesivir, which was developed for Ebola, has received enough support from early analyses to receive FDA approval, but results have been mixed.
The potential for other drugs, such as tocilizumab, to reduce recovery time remains unclear, but some early results were promising.</p>
<h2 data-number="6.1" id="additional-therapeutics-of-interest"><span class="header-section-number">6.1</span> Additional Therapeutics of Interest</h2>
<p>Given what is currently known about these therapeutics for COVID-19, a number of related therapies beyond those explored above may also prove to be of interest.
For example, the demonstrated benefit of dexamethasone and the ongoing potential of tocilizumab for treatment of COVID-19 suggests that other anti-inflammatory agents might also hold value for the treatment of COVID-19.
Given that current evidence about treating COVID-19 with dexamethasone suggests that the need to curtail the cytokine storm inflammatory response to the virus can transcend the risks of immunosuppression, exploration of more anti-inflammatory agents may be warranted.
While dexamethasone is considered widely available and generally affordable, the high costs of biologics such as tocilizumab therapy may present obstacles to wide-scale distribution of this drug if it proves of value.
At the doses used for rheumatoid arthritis patients, the cost for tocilizumab ranges from $179.20 to $896 per dose for the IV form and $355 for the pre-filled syringe <span class="citation" data-cites="7fZ9eyMk">[<a href="#ref-7fZ9eyMk" role="doc-biblioref">668</a>]</span>.
There are several anti-inflammatory agents used for the treatment of autoimmune diseases that may also be able to counter the effects of the cytokine storm induced by the virus, some of which, such as cyclosporine, are likely to be more cost-effective and readily available than biologics <span class="citation" data-cites="trWLoLFz">[<a href="#ref-trWLoLFz" role="doc-biblioref">669</a>]</span>.
While tocilizumab targets IL-6, several other inflammatory markers could be potential targets, including TNF-alpha.
Inhibition of TNF-alpha by an inhibitor such as Etanercept has been previously suggested for treatment of SARS-CoV <span class="citation" data-cites="LDQlolY3">[<a href="#ref-LDQlolY3" role="doc-biblioref">670</a>]</span> and may be relevant for SARS-CoV-2 as well.
Another anti-IL-6 antibody, sarilumab, is also being investigated <span class="citation" data-cites="4DAIEbQF amy3rDsj">[<a href="#ref-4DAIEbQF" role="doc-biblioref">671</a>,<a href="#ref-amy3rDsj" role="doc-biblioref">672</a>]</span>.
Baricitinib and other small molecule inhibitors of the JAK kinase pathway also curtail the inflammatory response and have been suggested as potential options for SARS-CoV-2 infections <span class="citation" data-cites="WF7ymA4m">[<a href="#ref-WF7ymA4m" role="doc-biblioref">673</a>]</span>.
Baricitinib in particular may be able to reduce the ability of SARS-CoV-2 to infect lung cells <span class="citation" data-cites="uCns3aFw">[<a href="#ref-uCns3aFw" role="doc-biblioref">674</a>]</span>.
Clinical trials studying baricitinib in COVID-19 have already begun in the US and in Italy <span class="citation" data-cites="jKRGxazA JCMhG5r9">[<a href="#ref-jKRGxazA" role="doc-biblioref">675</a>,<a href="#ref-JCMhG5r9" role="doc-biblioref">676</a>]</span>.
Identification and targeting of further inflammatory markers that are relevant in SARS-CoV-2 infection may be of value for curtailing the inflammatory response and lung damage.</p>
<p>In addition to immunosuppressive treatments that are most beneficial late in disease progression, much research is focused on identifying treatments would be likely to benefit early-stage patients.
For example, although studies of hydroxychloroquine have not supported the early theory-driven interest in this antiviral treatment, alternative compounds with related mechanisms may still have potential.
Hydroxyferroquine derivatives of HCQ have been described as a class of bioorganometallic compounds that exert antiviral effects with some selectivity for SARS-CoV <em>in vitro</em> <span class="citation" data-cites="Cd5uMaAr">[<a href="#ref-Cd5uMaAr" role="doc-biblioref">677</a>]</span>.
Future work could explore whether such compounds exert antiviral effects against SARS-CoV-2 and whether they would be safer for use in COVID-19.
Another potential approach is the development of antivirals, which could be broad-spectrum, specific to coronaviruses, or targeted to SARS-CoV-2.
Development of new antivirals is complicated by the fact that none have yet been approved for human coronaviruses.
Intriguing new options are emerging, however.
Beta-D-N4-hydroxycytidine (NHC) is an orally bioavailable ribonucleotide analog showing broad-spectrum activity against RNA viruses, which may inhibit SARS-CoV-2 replication <em>in vitro</em> and <em>in vivo</em> in mouse models of HCoVs <span class="citation" data-cites="TZleuwHX">[<a href="#ref-TZleuwHX" role="doc-biblioref">678</a>]</span>.
A range of other antivirals are also in development.
Development of antivirals will be further facilitated as research reveals more information about the interaction of SARS-CoV-2 with the host cell and host cell genome, mechanisms of viral replication, mechanisms of viral assembly, and mechanisms of viral release to other cells; this can allow researchers to target specific stages and structures of the viral life cycle.
Many researchers have also focused their attention to the potential use of dietary supplements and nutraceuticals.
Indeed, there has been recent interest for the potential use of vitamin D as a prophylactic and therapeutic agent against COVID-19 as several observational studies have linked low vitamin D status to its incidence <span class="citation" data-cites="LpqYmd4C 18O6t6Jo">[<a href="#ref-18O6t6Jo" role="doc-biblioref">514</a>,<a href="#ref-LpqYmd4C" role="doc-biblioref">679</a>]</span>.
These associations have yet to be confirmed and rigorous trials are required before considering supplementation recommendations.
However, the nutritional status and general health of a patient can affect their outcomes in various diseases, thus it would be pertinent to advise people to follow a healthy diet and life style to the best of their ability to prevent nutrient deficiencies and insufficiencies and to maintain a healthy immune system <span class="citation" data-cites="Hd5V5Usd">[<a href="#ref-Hd5V5Usd" role="doc-biblioref">680</a>]</span>.
Finally, antibodies against viruses, also known as antiviral monoclonal antibodies, could be an alternative as well and are described in detail in an above section.
The goal of antiviral antibodies is to neutralize viruses through either cell-killing activity or blocking of viral replication <span class="citation" data-cites="1FTnvyZlf">[<a href="#ref-1FTnvyZlf" role="doc-biblioref">681</a>]</span>.
They may also engage the host immune response, encouraging the immune system to hone in on the virus.
Given the cytokine storm that results from immune system activation in response to the virus, which has been implicated in worsening of the disease, a neutralizing antibody (nAb) may be preferable.
Upcoming work may explore the specificity of nAbs for their target, mechanisms by which the nAbs impede the virus, and improvements to antibody structure that may enhance the ability of the antibody to block viral activity.</p>
<p>Some research is also investigating potential therapeutics and prophylactics that would interact with components of the innate immune response.
For example, there are a variety of TLRs, pattern recognition receptors (PRRs) that recognize pathogen-associated molecular patterns (PAMPs) and damage-associated molecular patterns (DAMPs).
TLRs form a part of innate immune recognition and can more generally contribute to promoting both innate and adaptive responses <span class="citation" data-cites="o6BQnEt7">[<a href="#ref-o6BQnEt7" role="doc-biblioref">105</a>]</span>.
In mouse models, poly(I:C) and CpG, which are agonists of toll-like receptors TLR3 and TLR9, respectively, showed protective effects when administered prior to SARS-CoV infection <span class="citation" data-cites="Ull2rQ5L">[<a href="#ref-Ull2rQ5L" role="doc-biblioref">682</a>]</span>.
Therefore, TLR agonists hold some potential for broad-spectrum prophylaxis.</p>
<p>Given that a large number of clinical trials are currently in progress, more information about the potential of these and other therapeutics should become available over time.
This information, combined with advances in understanding the molecular structure and viral pathogenesis of SARS-CoV-2, may lead to a more complete understanding of how the virus affects the human host and what strategies can improve outcomes.
To date, investigations of potential therapeutics for COVID-19 have focused primarily on repurposing existing drugs.
This approach is necessary given the urgency of the situation as well as the extensive time required for developing and testing new therapies.
However, in the long-term, new drugs specific for treatment of COVID-19 may also enter development.
Development of novel drugs is likely to be guided by what is known about the pathogenesis and molecular structure of SARS-CoV-2.
For example, understanding the various structural components of SARS-CoV-2 may allow for the development of small molecule inhibitors of those components.
Currently, crystal structures of the SARS-CoV-2 main protease have recently been resolved <span class="citation" data-cites="4Sja6dIz AJLaaguT">[<a href="#ref-4Sja6dIz" role="doc-biblioref">366</a>,<a href="#ref-AJLaaguT" role="doc-biblioref">683</a>]</span>, and efforts are already in place to perform screens for small molecule inhibitors of the main protease, which have yielded potential hits <span class="citation" data-cites="4Sja6dIz">[<a href="#ref-4Sja6dIz" role="doc-biblioref">366</a>]</span>.
Much work remains to be done to determine further crystal structures of other viral components, understand the relative utility of targeting different viral components, perform additional small molecule inhibitor screens, and determine the safety and efficacy of the potential inhibitors.
While still nascent, work in this area is promising.
Over the longer term, this approach and others may lead to the development of novel therapeutics specifically for COVID-19 and SARS-CoV-2.</p>
<h2 data-number="6.2" id="social-factors-influencing-covid-19-outcomes"><span class="header-section-number">6.2</span> Social Factors Influencing COVID-19 Outcomes</h2>
<p>In addition to understanding the fundamental biology of the SARS-CoV-2 virus and COVID-19, it is critical to consider how the broader environment can influence both COVID-19 outcomes and efforts to develop and implement treatments for the disease.
The evidence clearly indicates that social environmental factors are critical determinants of individuals’ and communities’ risks related to COVID-19.
There are distinct components to COVID-19 susceptibility, and an individual’s risk can be elevated at one or all stages from exposure to recovery/mortality: an individual may be more likely to be exposed to the virus, more likely to get infected once exposed, more likely to have serious complications once infected, and be less likely to receive adequate care once they are seriously ill.
The fact that differences in survival between Black and white patients were no longer significant after controlling for comorbidities and socioeconomic status (type of insurance, neighborhood deprivation score, and hospital where treatment was received) in addition to sex and age <span class="citation" data-cites="hrOv770z">[<a href="#ref-hrOv770z" role="doc-biblioref">284</a>]</span> underscores the relevance of social factors to understanding mortality differences between racial and ethnic groups.
Moreover, the Black patients were younger and more likely to be female than white patients, yet still had a higher mortality rate without correction for the other variables <span class="citation" data-cites="hrOv770z">[<a href="#ref-hrOv770z" role="doc-biblioref">284</a>]</span>.
Here, we outline a few systemic reasons that may exacerbate the COVID-19 pandemic in communities of color.</p>
<h3 data-number="6.2.1" id="exposure-to-covid-19"><span class="header-section-number">6.2.1</span> Exposure to COVID-19</h3>
<p>Social distancing has emerged as one of the main social policies used to manage the COVID-19 epidemic in many countries.
Many governments issued stay-at-home orders, especially in the initial months of the crisis.
However, data clearly indicates that these orders impacted different socioeconomic groups differently.
In U.S. counties with and without stay-at-home orders, smartphone tracking indicated a significant decrease in the general population’s mobility in April relative to February through March of 2020 (-52.3% and -60.8%, respectively) <span class="citation" data-cites="bDNDEYNn">[<a href="#ref-bDNDEYNn" role="doc-biblioref">684</a>]</span>.
A linear relationship was observed between counties’ reduction in mobility and their wealth and health, as measured by access to health care, food security, income, space, and other factors <span class="citation" data-cites="bDNDEYNn">[<a href="#ref-bDNDEYNn" role="doc-biblioref">684</a>]</span>.
Counties with greater reductions in mobility were also found to have much lower child poverty and household crowding and to be more racially segregated, and to have fewer youth and more elderly residents <span class="citation" data-cites="bDNDEYNn">[<a href="#ref-bDNDEYNn" role="doc-biblioref">684</a>]</span>.
Similar associations between wealth and decreased mobility were observed in cellphone GPS data from Colombia, Indonesia, and Mexico collected between January and May 2020 <span class="citation" data-cites="RdnghmVU">[<a href="#ref-RdnghmVU" role="doc-biblioref">685</a>]</span>, as well as in a very large data set from several US cities <span class="citation" data-cites="9OksxLmM">[<a href="#ref-9OksxLmM" role="doc-biblioref">686</a>]</span>.
These disparities in mobility are likely to be related to the role that essential workers have played during the pandemic.
Essential workers are disproportionately likely to be female, people of color, immigrants, and to have an income below 200% of the poverty line <span class="citation" data-cites="15qfRPnUG">[<a href="#ref-15qfRPnUG" role="doc-biblioref">687</a>]</span>.
Black Americans in particular are over-represented among front-line workers and in professions where social distancing is infeasible <span class="citation" data-cites="HihYbn6k">[<a href="#ref-HihYbn6k" role="doc-biblioref">688</a>]</span>.
Health care work in particular presents an increased risk of exposure to SARS-CoV-2 <span class="citation" data-cites="HihYbn6k F4laSOXM hAYohR8m DOyiuaup 13qkVV6eN">[<a href="#ref-HihYbn6k" role="doc-biblioref">688</a>,<a href="#ref-F4laSOXM" role="doc-biblioref">689</a>,<a href="#ref-hAYohR8m" role="doc-biblioref">690</a>,<a href="#ref-DOyiuaup" role="doc-biblioref">691</a>,<a href="#ref-13qkVV6eN" role="doc-biblioref">692</a>]</span>.
In the United Kingdom, (South) Asians are more likely than their white counterparts to be medical professionals <span class="citation" data-cites="xFKmPg6l">[<a href="#ref-xFKmPg6l" role="doc-biblioref">283</a>]</span>, although BAME medical professionals are still disproportionately represented in the proportion of National Health Service staff deaths <span class="citation" data-cites="4zjepZ8T">[<a href="#ref-4zjepZ8T" role="doc-biblioref">693</a>]</span>.
Similar trends have been reported for nurses, especially nurses of color, in the United States <span class="citation" data-cites="2gxJPENM">[<a href="#ref-2gxJPENM" role="doc-biblioref">694</a>]</span>.
Furthermore, beyond the risks associated with work itself, use of public transportation may also impact COVID-19 risk <span class="citation" data-cites="Moc3BREQ">[<a href="#ref-Moc3BREQ" role="doc-biblioref">695</a>]</span>.
The socioeconomic and racial/ethnic gaps in who is working on the front lines of the pandemic make it clear that socioeconomic privilege is likely to decrease the probability of exposure to SARS-CoV-2.</p>
<p>Increased risk of exposure can also arise outside the workplace.
Nursing homes and skilled nursing facilities received attention early on as high-risk locations for COVID-19 outbreaks <span class="citation" data-cites="aM2XPE3J">[<a href="#ref-aM2XPE3J" role="doc-biblioref">696</a>]</span>.
Prisons and detention centers also confer a high risk of exposure or infection <span class="citation" data-cites="BeedQx3I 2uUcva6Y">[<a href="#ref-BeedQx3I" role="doc-biblioref">697</a>,<a href="#ref-2uUcva6Y" role="doc-biblioref">698</a>]</span>.
Populations in care facilities are largely older adults, and in the United States, incarcerated people are more likely to be male and persons of color, especially Black <span class="citation" data-cites="zuipD7ZP">[<a href="#ref-zuipD7ZP" role="doc-biblioref">699</a>]</span>.
Additionally, multi-generational households are less common among non-Hispanic white Americans than people of other racial and ethnic backgrounds <span class="citation" data-cites="OzhmHOxJ">[<a href="#ref-OzhmHOxJ" role="doc-biblioref">700</a>]</span>, increasing the risk of exposure for more susceptible family members.
Analysis suggests that household crowding may also be associated with increased risk of COVID-19 exposure <span class="citation" data-cites="bDNDEYNn">[<a href="#ref-bDNDEYNn" role="doc-biblioref">684</a>]</span>, and household crowding is associated with poverty <span class="citation" data-cites="13wfXDXqr">[<a href="#ref-13wfXDXqr" role="doc-biblioref">701</a>]</span>.
Forms of economic insecurity like housing insecurity, which is associated with poverty and more pronounced in communities subjected to racism <span class="citation" data-cites="1Axz3a1Hj 64YOO3yt">[<a href="#ref-1Axz3a1Hj" role="doc-biblioref">702</a>,<a href="#ref-64YOO3yt" role="doc-biblioref">703</a>]</span>, would be likely to increase household crowding and other possible sources of exposure.
As a result, facets of systemic inequality such as mass incarceration of Black Americans and poverty are likely to increase the risk of exposure outside of the workplace.</p>
<h3 data-number="6.2.2" id="severity-of-covid-19-following-exposure"><span class="header-section-number">6.2.2</span> Severity of COVID-19 Following Exposure</h3>
<p>Following exposure to SARS-CoV-2, the likelihood that an individual develops COVID-19 and the severity of the disease presentation can be influenced by a number of social factors.
As discussed above, a number of patient characteristics are associated with the likelihood of severe COVID-19 symptoms.
In some cases, these trends run counter to those expected given rates of exposure: for example, although women are more likely to be exposed, men are more likely to be diagnosed with, hospitalized from, or die from COVID-19 <span class="citation" data-cites="fH3qecwy">[<a href="#ref-fH3qecwy" role="doc-biblioref">247</a>]</span>.
In the case of comorbid conditions and racial/ethnic demographics, however, social factors are highly likely to modulate or at least influence the apparent association between these traits and the increased risk from COVID-19.
In particular, the comorbidities and racial/ethnic correlates of severe COVID-19 outcomes suggest that poverty confers additional risk for COVID-19.</p>
<p>In order to explore the relationship between poverty and COVID-19 outcomes, it is necessary to consider how poverty impacts biology.
In particular, we focus on the United States and the United Kingdom.
Comorbidities that increase risk for COVID-19, including obesity, type II diabetes, hypertension, and cardiovascular disease, are known to be intercorrelated <span class="citation" data-cites="sRZCzj6j">[<a href="#ref-sRZCzj6j" role="doc-biblioref">704</a>]</span>.
Metabolic conditions related to heightened inflammation, like obesity, type II diabetes, and hypertension, are more strongly associated with negative COVID-19 outcomes than other comorbid conditions, such as chronic heart disease <span class="citation" data-cites="JizR1ocj">[<a href="#ref-JizR1ocj" role="doc-biblioref">705</a>]</span>.
As discussed above, dysregulated inflammation characteristic of cytokine release syndrome is one of the greatest concerns for COVID-19-related death.
Therefore, it is possible that chronic inflammation characteristic of these metabolic conditions predisposes patients to COVID-19-related death <span class="citation" data-cites="JizR1ocj">[<a href="#ref-JizR1ocj" role="doc-biblioref">705</a>]</span>.
The association between these diseases and severe COVID-19 outcomes is a concern from a health equity perspective because poverty exposes people to “obesogenic” conditions <span class="citation" data-cites="nYLB9EMw">[<a href="#ref-nYLB9EMw" role="doc-biblioref">706</a>]</span> and is therefore unsurprisingly associated with higher incidence of obesity and associated disorders <span class="citation" data-cites="h1e9QTlK">[<a href="#ref-h1e9QTlK" role="doc-biblioref">707</a>]</span>.
Furthermore, cell phone GPS data suggests that lower socioeconomic status may also be associated with decreased access to healthy food choices during the COVID-19 pandemic <span class="citation" data-cites="94eOAGnA 5G5UozXd">[<a href="#ref-94eOAGnA" role="doc-biblioref">708</a>,<a href="#ref-5G5UozXd" role="doc-biblioref">709</a>]</span>, suggesting that health-related risk factors for COVID-19 may be exacerbated as the pandemic continues <span class="citation" data-cites="17yfPiYAS">[<a href="#ref-17yfPiYAS" role="doc-biblioref">710</a>]</span>.
Chronic inflammation is a known outcome of chronic stress (e.g., <span class="citation" data-cites="Hu7PyNNp P2wUkjrR 109EtOYzw 11oGPv27i">[<a href="#ref-Hu7PyNNp" role="doc-biblioref">711</a>,<a href="#ref-P2wUkjrR" role="doc-biblioref">712</a>,<a href="#ref-109EtOYzw" role="doc-biblioref">713</a>,<a href="#ref-11oGPv27i" role="doc-biblioref">714</a>]</span>).
Therefore, the chronic stress of poverty is likely to influence health broadly (as summarized in <span class="citation" data-cites="KZSJklnl">[<a href="#ref-KZSJklnl" role="doc-biblioref">715</a>]</span>) and especially during the stress of the ongoing pandemic.</p>
<p>A preprint <span class="citation" data-cites="5BMDcXz3">[<a href="#ref-5BMDcXz3" role="doc-biblioref">716</a>]</span> provided observational evidence that geographical areas in the United States that suffer from worse air pollution by fine particulate matter have also suffered more COVID-19 deaths per capita, after adjusting for demographic covariates.
Although lack of individual-level exposure data and the impossibility of randomization make it difficult to elucidate the exact causal mechanism, this finding would be consistent with similar findings for all-cause mortality (e.g., <span class="citation" data-cites="vImXUPNS">[<a href="#ref-vImXUPNS" role="doc-biblioref">717</a>]</span>).
Exposure to air pollution is associated with both poverty (e.g., <span class="citation" data-cites="6ACdDJOV">[<a href="#ref-6ACdDJOV" role="doc-biblioref">718</a>]</span>) and chronic inflammation <span class="citation" data-cites="11Ift2eIl">[<a href="#ref-11Ift2eIl" role="doc-biblioref">719</a>]</span>.
Other outcomes of environmental racism, such as the proximity of abandoned uranium mines to Navajo land, can also cause respiratory illnesses and other health issues <span class="citation" data-cites="J9T8eR5H">[<a href="#ref-J9T8eR5H" role="doc-biblioref">269</a>]</span>.
Similarly, preliminary findings indicate that nutritional status (e.g., vitamin D deficiency <span class="citation" data-cites="8ZxZKpYS">[<a href="#ref-8ZxZKpYS" role="doc-biblioref">515</a>]</span>) may be associated with COVID-19 outcomes, and reduced access to grocery stores and fresh food often co-occurs with environmental racism <span class="citation" data-cites="J9T8eR5H aHM6WZzd">[<a href="#ref-J9T8eR5H" role="doc-biblioref">269</a>,<a href="#ref-aHM6WZzd" role="doc-biblioref">720</a>]</span>.
Taken together, the evidence suggests that low-income workers who face greater exposure to SARS-CoV-2 due to their home or work conditions are also more likely to face environmental and social stressors associated with increased inflammation, and therefore with increased risk from COVID-19.
In particular, structural racism can play an important role on disease severity after SARS-CoV-2 exposure, due to consequences of racism which include an increased likelihood of poverty and its associated food and housing instability.
COVID-19 can thus be considered a “syndemic”, or a synergistic interaction between several epidemics <span class="citation" data-cites="11Ldf3Ch3">[<a href="#ref-11Ldf3Ch3" role="doc-biblioref">721</a>]</span>.
As a result, it is not surprising that people from minoritized backgrounds and/or with certain pre-existing conditions are more likely to suffer severe effects of COVID-19, but these “risk factors” are likely to be causally linked to poverty.</p>
<h3 data-number="6.2.3" id="access-to-treatment"><span class="header-section-number">6.2.3</span> Access to Treatment</h3>
<p>Finally, COVID-19 outcomes can be influenced by access to healthcare.
Receiving care for COVID-19 can, but does not always, include receiving a positive test for the SARS-CoV-2 virus.
For example, it is common to see treatment guidelines for suspected cases regardless of whether the presence of SARS-CoV-2 has been confirmed (e.g., <span class="citation" data-cites="cSA2h9hO">[<a href="#ref-cSA2h9hO" role="doc-biblioref">722</a>]</span>).
Whether and where a patient is diagnosed can depend on their access to testing, which can vary both between and within countries.
In the United States, it is not always clear whether an individual will have access to free testing <span class="citation" data-cites="1Intrlax jc9vc7ws">[<a href="#ref-1Intrlax" role="doc-biblioref">723</a>,<a href="#ref-jc9vc7ws" role="doc-biblioref">724</a>]</span>.
The concern has been raised that more economic privilege is likely to correspond to increased access to testing, at least within the United States <span class="citation" data-cites="13MGsHBtw">[<a href="#ref-13MGsHBtw" role="doc-biblioref">725</a>]</span>.
This is supported by the fact that African Americans seem to be more likely to be diagnosed in the hospital, while individuals from other groups were more likely to have been diagnosed in ambulatory settings in the community <span class="citation" data-cites="1D6lVXWbJ">[<a href="#ref-1D6lVXWbJ" role="doc-biblioref">240</a>]</span>.
Any delays in treatment are a cause for concern <span class="citation" data-cites="13MGsHBtw">[<a href="#ref-13MGsHBtw" role="doc-biblioref">725</a>]</span>, which could potentially be increased by an inability to acquire testing because in the United States, insurance coverage for care received can depend on a positive test <span class="citation" data-cites="P8VNkYTE">[<a href="#ref-P8VNkYTE" role="doc-biblioref">726</a>]</span>.</p>
<p>Another important question is whether patients with moderate to severe cases are able to access hospital facilities and treatments, to the extent that they have been identified.
Early findings from China as of February 2020 suggested the COVID-19 mortality rate to be much lower in the most developed regions of the country <span class="citation" data-cites="tLVYjfMk">[<a href="#ref-tLVYjfMk" role="doc-biblioref">727</a>]</span>, although reported mortality is generally an estimate of CFR, which is dependent on rates of testing.
Efforts to make treatment accessible for all confirmed and suspected cases of COVID-19 in China are credited with expanding care to people with fewer economic resources <span class="citation" data-cites="cOQK0zNa">[<a href="#ref-cOQK0zNa" role="doc-biblioref">728</a>]</span>.
In the United States, access to healthcare varies widely, with certain sectors of the workforce less likely to have health insurance; many essential workers in transportation, food service, and other frontline fields are among those likely to be uninsured or underinsured <span class="citation" data-cites="13MGsHBtw">[<a href="#ref-13MGsHBtw" role="doc-biblioref">725</a>]</span>.
As of 2018, Hispanic Americans of all races were much less likely to have health insurance than people from non-Hispanic backgrounds <span class="citation" data-cites="gOiB5HDd">[<a href="#ref-gOiB5HDd" role="doc-biblioref">729</a>]</span>.
Therefore, access to diagnostics and care prior to the development of severe COVID-19 is likely to vary depending on socioeconomic and social factors, many of which overlap with the risks of exposure and of developing more severe COVID-19 symptoms.
This discrepancy ties into concerns about broad infrastructural challenges imposed by COVID-19.
A major concern in many countries has been the saturation of healthcare systems due to the volume of COVID-19 hospitalizations (e.g., <span class="citation" data-cites="NCcssjLV">[<a href="#ref-NCcssjLV" role="doc-biblioref">225</a>]</span>).
Similarly, there have been shortages of supplies such as ventilators that are critical to the survival of many COVID-19 patients, leading to extensive ethical discussions about how to allocate limited resources among patients <span class="citation" data-cites="15Yaf3fc8 1As9kWVvA FaGAJ7AI B3awlFvs">[<a href="#ref-15Yaf3fc8" role="doc-biblioref">730</a>,<a href="#ref-1As9kWVvA" role="doc-biblioref">731</a>,<a href="#ref-FaGAJ7AI" role="doc-biblioref">732</a>,<a href="#ref-B3awlFvs" role="doc-biblioref">733</a>]</span>.
Although it is generally considered unethical to consider demographic factors such as age, sex, race, or ethnicity while making such decisions, and ideally this information would not be shared with triage teams tasked with allocating limited resources among patients <span class="citation" data-cites="2i1Y7SYn">[<a href="#ref-2i1Y7SYn" role="doc-biblioref">734</a>]</span>, there are substantial concerns about implicit and explicit biases against older adults <span class="citation" data-cites="GTL02UT6">[<a href="#ref-GTL02UT6" role="doc-biblioref">735</a>]</span>, premature infants <span class="citation" data-cites="KQfJ4llw">[<a href="#ref-KQfJ4llw" role="doc-biblioref">736</a>]</span>, and people with disabilities or comorbidities <span class="citation" data-cites="K6yc6elC 1EaLRFk21 2i1Y7SYn">[<a href="#ref-2i1Y7SYn" role="doc-biblioref">734</a>,<a href="#ref-K6yc6elC" role="doc-biblioref">737</a>,<a href="#ref-1EaLRFk21" role="doc-biblioref">738</a>]</span>.
Because of the greater burden of chronic disease in populations subjected to systemic racism, algorithms intended to be blind to race and ethnicity could, in fact, reinforce systemic inequalities caused by structural racism <span class="citation" data-cites="cPsuWT7U 1D1FX4zQZ LrqhFVIz">[<a href="#ref-cPsuWT7U" role="doc-biblioref">739</a>,<a href="#ref-1D1FX4zQZ" role="doc-biblioref">740</a>,<a href="#ref-LrqhFVIz" role="doc-biblioref">741</a>]</span>.
Because of this inequality, it has been argued that groups facing health disparities should be prioritized by these algorithms <span class="citation" data-cites="PFHCN5ep">[<a href="#ref-PFHCN5ep" role="doc-biblioref">742</a>]</span>.
This approach would carry its own ethical concerns, including the fact that many resources that need to be distributed do not have well-established risks and benefits <span class="citation" data-cites="PFHCN5ep">[<a href="#ref-PFHCN5ep" role="doc-biblioref">742</a>]</span>.</p>
<p>As the pandemic has progressed, it has become clear that ICU beds and ventilators are not the only limited resources that needs to be allocated, and, in fact, the survival rate for patients who receive mechanical ventilation is lower than these discussions would suggest <span class="citation" data-cites="1ExRX1x4C">[<a href="#ref-1ExRX1x4C" role="doc-biblioref">743</a>]</span>.
Allocation of interventions that may reduce suffering, including palliative care, has become critically important <span class="citation" data-cites="1ExRX1x4C ztTunmqF">[<a href="#ref-1ExRX1x4C" role="doc-biblioref">743</a>,<a href="#ref-ztTunmqF" role="doc-biblioref">744</a>]</span>.
The ambiguities surrounding the risks and benefits associated with therapeutics that have been approved under emergency use authorizations also present ethical concerns related to the distribution of resources <span class="citation" data-cites="PFHCN5ep">[<a href="#ref-PFHCN5ep" role="doc-biblioref">742</a>]</span>.
For example, remdesivir, discussed above, is currently available for the treatment of COVID-19 under compassionate use guidelines and through expanded access programs, and in many cases has been donated to hospitals by Gilead <span class="citation" data-cites="18et5lcBc 11gslRJJU">[<a href="#ref-18et5lcBc" role="doc-biblioref">745</a>,<a href="#ref-11gslRJJU" role="doc-biblioref">746</a>]</span>.
Regulations guiding the distribution of drugs in situations like these typically do not address how to determine which patients receive them <span class="citation" data-cites="11gslRJJU">[<a href="#ref-11gslRJJU" role="doc-biblioref">746</a>]</span>.
Prioritizing marginalized groups for treatment with a drug like remdesivir would also be unethical because it would entail disproportionately exposing these groups to a therapeutic that may or not be beneficial <span class="citation" data-cites="PFHCN5ep">[<a href="#ref-PFHCN5ep" role="doc-biblioref">742</a>]</span>.
On the other hand, given that the drug is one of the most promising treatments available for many patients, using a framework that tacitly feeds into structural biases would also be unethical.
At present, the report prepared for the Director of the CDC by Ethics Subcommittee of the CDC fails to address the complexity of this ethical question given the state of structural racism in the United States, instead stating that “prioritizing individuals according to their chances for short-term survival also avoids ethically irrelevant considerations, such as race or socioeconomic status” <span class="citation" data-cites="ugEOYgfx">[<a href="#ref-ugEOYgfx" role="doc-biblioref">747</a>]</span>.
In many cases, experimental therapeutics are made available only through participation in clinical trials <span class="citation" data-cites="1A5E2TkBS">[<a href="#ref-1A5E2TkBS" role="doc-biblioref">748</a>]</span>.
However, given the history of medical trials abusing minority communities, especially Black Americans, there is a history of unequal representation in clinical trial enrollment <span class="citation" data-cites="1A5E2TkBS">[<a href="#ref-1A5E2TkBS" role="doc-biblioref">748</a>]</span>.
As a result, the standard practice of requiring enrollment in a clinical trial in order to receive experimental treatment may also reinforce patterns established by systemic racism.
<!--Risk of comorbid health conditions associated with more severe outcomes may be influenced by long-term damage caused by chronic stress related to traumatic social experiences [@doi:10.1007/s12170-013-0338-5], perhaps mediated by cardiovascular risk factors [@doi:10.1101/2020.05.10.20097253], although the effects of chronic stress have not yet been researched in the specific case of COVID-19 disparities.--></p>
<h3 data-number="6.2.4" id="access-to-and-representation-in-clinical-trials"><span class="header-section-number">6.2.4</span> Access to and Representation in Clinical Trials</h3>
<!-- Add #600 here -->
<!--Table for future version: ### Indirect Effects of the Pandemic
Besides the direct harms caused by infection, many populations are in indirect risk of serious harm due to the social and economic effects of the pandemic and of the efforts needed to fight it.
These might include individuals with substance use disorder [@doi:10.7326/M20-1141], victims of natural disasters [@doi:10.1038/s41558-020-0804-2], and victims of human trafficking [@doi:10.1016/j.eclinm.2020.100409].
The pandemic could also delay the fight against other major infectious diseases, such as HIV, malaria and tuberculosis, potentially leading to a further increase of mortality [@doi:10.25561/78670].
Although they are beyond the scope of this paper, further research is needed in order to prevent these harms.
[There are already some policy recommendations in literature that we cite - we should probably direct folks there. I think we probably want a stronger synthesis and closing than this] -->
<h2 data-number="6.3" id="conclusions-and-future-directions"><span class="header-section-number">6.3</span> Conclusions and Future Directions</h2>
<p>As the COVID-19 pandemic evolves, the scientific community’s response will be critical for identifying potential pharmacological and biotechnological developments that may aid in combating the virus and the disease it causes.
However, this global crisis highlights the importance of mounting a response based on collaboration among a wide variety of disciplines.
Understanding the basic science of the virus and its pathogenesis is imperative for identifying and envisioning possible diagnostic and therapeutic approaches; understanding how social factors can influence outcomes and shape implementation of a response is critical to disseminating any scientific advancements.
Summarizing such a complex and ever-changing topic presents a number of challenges.
This review represents the effort of over 50 contributors <!-- TO DO: check with final number -->to distill and interpret the available information.
However, this text represents a dynamic and evolving document, and we welcome continued contributions from all researchers who have insights into how these topics intersect.
A multidisciplinary perspective is critical to understanding this evolving crisis, and in this review we seek to use open science tools to coordinate a response among a variety of researchers.
We intend to publish additional updates as the situation evolves.</p>
<h1 data-number="7" id="methods"><span class="header-section-number">7</span> Methods</h1>
<p>In an effort to keep pace as new information about COVID-19 and SARS-CoV-2 becomes available, this project is an open, collaborative effort that invited contributions from the scientific community broadly, similar to previous efforts to develop collaborative reviews <span class="citation" data-cites="PZMP42Ak heDA5StF">[<a href="#ref-PZMP42Ak" role="doc-biblioref">749</a>,<a href="#ref-heDA5StF" role="doc-biblioref">750</a>]</span>.
Contributors were recruited by word of mouth and on Twitter.
Existing efforts to train early-career scientists were also integrated: Appendix A contains summaries written by the students, post-docs, and faculty of the Immunology Institute at the Mount Sinai School of Medicine <span class="citation" data-cites="cYo4O2qX">[<a href="#ref-cYo4O2qX" role="doc-biblioref">751</a>]</span> and two of the authors were recruited through the American Physician Scientist Association’s Virtual Summer Research Program <span class="citation" data-cites="DGTDsJZy">[<a href="#ref-DGTDsJZy" role="doc-biblioref">752</a>]</span>.
The project was managed through GitHub <span class="citation" data-cites="11MwmdOKi">[<a href="#ref-11MwmdOKi" role="doc-biblioref">753</a>]</span> using Manubot <span class="citation" data-cites="YuJbg3zO">[<a href="#ref-YuJbg3zO" role="doc-biblioref">7</a>]</span> to continuously generate a version of the manuscript online <span class="citation" data-cites="yTsmmAYC">[<a href="#ref-yTsmmAYC" role="doc-biblioref">754</a>]</span>.
Contributors developed text that was proposed through GitHub’s pull request system and then reviewed and approved by at least one other author.
While this document reflects the current version of record, the online version will continue to be developed as information about the pandemic emerges.
Below, we will describe the processes used to synthesize the literature.</p>
<h2 data-number="7.1" id="technical-infrastructure"><span class="header-section-number">7.1</span> Technical Infrastructure</h2>
<h3 data-number="7.1.1" id="collaborative-writing-and-manuscript-generation"><span class="header-section-number">7.1.1</span> Collaborative Writing and Manuscript Generation</h3>
<p>Manubot <span class="citation" data-cites="YuJbg3zO">[<a href="#ref-YuJbg3zO" role="doc-biblioref">7</a>]</span> is a collaborative framework developed to adapt open-source software development techniques and version control for manuscript writing.
Here, Manubot was used to generate a manuscript from text maintained using GitHub, a popular, online version control interface.
The GitHub implementation allowed users to contribute either using git on the command line or using the GitHub user interface, and we developed documentation for users with less experience with this platform.
Manubot also provides a functionality to create a bibliography using digital object identifiers (DOIs), website URLs, or other identifiers such as PubMed identifiers and arXiv IDs.
Due to the needs of this project, project contributors also implemented new features in Manubot and <a href="https://www.zotero.org/">Zotero</a>, which Manubot uses to extract metadata for some types of citations.
These features support directly citing clinical trial identifiers such as <code>clinicaltrials:NCT04292899</code> <span class="citation" data-cites="yTCAmOyt">[<a href="#ref-yTCAmOyt" role="doc-biblioref">346</a>]</span>.
Finally, Manubot and GitHub Actions continuous integration allowed for scripted updates to be run each time the manuscript was generated.
These scripts were used to check that the manuscript was built correctly, run spellchecking, and cross-reference the manuscripts cited in this review, summarized in Appendix A, and discussed in the project’s issues and pull requests.</p>
<h3 data-number="7.1.2" id="data-analysis-and-visualization"><span class="header-section-number">7.1.2</span> Data Analysis and Visualization</h3>
<p>The combination of Manubot and GitHub Actions also made it possible to dynamically update information such as statistics and visualizations in the manuscript.
Data about worldwide cases and deaths from the COVID-19 Data Repository by the Center for Systems Science and Engineering at Johns Hopkins University <span class="citation" data-cites="MrwDDw9R">[<a href="#ref-MrwDDw9R" role="doc-biblioref">5</a>]</span> were read using a Python script to generate Figure <a href="#fig:csse-deaths">1</a>.
Similarly, Figure <a href="#fig:ebm-trials">3</a> and clinical trials statistics were generated based on data from the University of Oxford Evidence-Based Medicine Data Lab’s COVID-19 TrialsTracker <span class="citation" data-cites="SSbnPnzT">[<a href="#ref-SSbnPnzT" role="doc-biblioref">329</a>]</span>.
In both cases, frequency data were plotted using Matplotlib <span class="citation" data-cites="1026Gxdsi">[<a href="#ref-1026Gxdsi" role="doc-biblioref">755</a>]</span> in Python.
Figure ?? was generated using the countries associated with the trials listed in the COVID-19 TrialsTracker, converting the country names to 3-letter ISO codes using pycountry or manual adjustment when necessary, and visualizing the geographic distribution of trial recruitment using geopandas.
<!-- Is this figure going to be included? #673 --></p>
<p>GitHub Actions runs a nightly workflow to update these external data and regenerate the statistics and figures for the manuscript.
The workflow uses the GitHub API to detect and save the latest commit of the external data sources, which are both GitHub repositories.
It then downloads versioned data from that snapshot of the external repositories and runs bash and Python scripts to calculate the desired statistics and produce the summary figures.
The statistics are stored in JSON files that are accessed by Manubot to populate the values of placeholder template variables dynamically every time the manuscript is built.
For instance, the template variable <code>{{ebm_trials_results}}</code> in the manuscript is replaced by the actual number of clinical trials with results, 157.
The template variables also include versioned URLs to the dynamically updated figures.
The JSON files and figures are stored in the <code>external-resources</code> branch of the manuscript’s GitHub repository, which acts as versioned storage.
The GitHub Actions workflow automatically adds and commits the new JSON files and figures to the <code>external-resources</code> branch every time it runs, and Manubot uses the latest version of these resources when it builds the manuscript.</p>
<p>The workflow file is available from <a href="https://github.com/greenelab/covid19-review/blob/master/.github/workflows/update-external-resources.yaml" class="uri">https://github.com/greenelab/covid19-review/blob/master/.github/workflows/update-external-resources.yaml</a> and the scripts are available from <a href="https://github.com/greenelab/covid19-review/tree/external-resources" class="uri">https://github.com/greenelab/covid19-review/tree/external-resources</a>.
The Python package versions are available in <a href="https://github.com/greenelab/covid19-review/blob/external-resources/environment.yml" class="uri">https://github.com/greenelab/covid19-review/blob/external-resources/environment.yml</a>.
<!-- These files are archived with [Software Heritage](...). --></p>
<h2 data-number="7.2" id="article-selection-and-evaluation"><span class="header-section-number">7.2</span> Article Selection and Evaluation</h2>
<p>Relevant articles were identified and submitted as issues on <a href="https://github.com/greenelab/covid19-review">GitHub</a> for review.
Articles were classified as <em>diagnostic</em>, <em>therapeutic</em>, or <em>other</em>, and a template was developed to guide the review of papers and preprints in each category.
Following a framework often used for assessing medical literature, the review consisted of examining methods used in each relevant article, assignment (whether the study was observational or randomized), assessment, results, interpretation, and how well the study extrapolates <span class="citation" data-cites="17OQtAY4l">[<a href="#ref-17OQtAY4l" role="doc-biblioref">756</a>]</span>.
For examples of each template, please see Appendices B-D.</p>
<h3 data-number="7.2.1" id="diagnostic-papers"><span class="header-section-number">7.2.1</span> Diagnostic Papers</h3>
<h4 data-number="7.2.1.1" id="methods-1"><span class="header-section-number">7.2.1.1</span> Methods</h4>
<p>Reviewers began by describing the study question(s) being investigated by the article.
They then described the study population, the sample size, the prevalence of the disease in the study population, countries / regions considered in case of human subjects, demographics of participants, the setting, and any remaining inclusion / exclusion criteria considered.
They then described the reference test or “gold standard,” if one was utilized.</p>
<h4 data-number="7.2.1.2" id="assignment"><span class="header-section-number">7.2.1.2</span> Assignment</h4>
<p>Reviewers described how new and reference tests were assigned, including additional relevant details about the study design.
For example, reviewers were asked whether the diagnostic test resulted in rigorous assignments of case status or was biased towards sicker or healthier individuals.</p>
<h4 data-number="7.2.1.3" id="assessment"><span class="header-section-number">7.2.1.3</span> Assessment</h4>
<p>Reviewers described how the test was performed.
For example, for both standard and reference tests, reviewers described technical details of assays used, when measurements were taken and by whom.
Subsequently, they described how individuals were classified as positive or negative cases and whether results were precise and reproducible with repeated tests.
Reviewers described whether there were any missing data, whether some participants underwent only one test, or whether there were individuals with inconclusive results.</p>
<h4 data-number="7.2.1.4" id="results"><span class="header-section-number">7.2.1.4</span> Results</h4>
<p>Reviewers reported the estimated sensitivity, specificity, positive predictive value (PPV), and negative predicted value (NPV), as well as confidence bounds around these measures, if provided.</p>
<h4 data-number="7.2.1.5" id="interpretation"><span class="header-section-number">7.2.1.5</span> Interpretation</h4>
<p>Reviewers reported how well the test ruled in or ruled out disease based on the population, if there were identified side effects, and patient adherence.</p>
<h4 data-number="7.2.1.6" id="extrapolation"><span class="header-section-number">7.2.1.6</span> Extrapolation</h4>
<p>Reviewers described how well this test will extrapolate outside the measured population.</p>
<h3 data-number="7.2.2" id="therapeutic-papers"><span class="header-section-number">7.2.2</span> Therapeutic Papers</h3>
<h4 data-number="7.2.2.1" id="methods-2"><span class="header-section-number">7.2.2.1</span> Methods</h4>
<p>Reviewers began by describing the study question(s) being investigated by the article.
They then described the study population, the sample size, the prevalence of the disease in the study population, countries / regions considered in case of human subjects, demographics of participants, the setting, and any remaining inclusion / exclusion criteria considered.</p>
<h4 data-number="7.2.2.2" id="assignment-1"><span class="header-section-number">7.2.2.2</span> Assignment</h4>
<p>Reviewers described how the treatment is assigned, whether it was an interventional or observational study, whether randomization took place, etc.</p>
<h4 data-number="7.2.2.3" id="assessment-1"><span class="header-section-number">7.2.2.3</span> Assessment</h4>
<h5 data-number="7.2.2.3.1" id="outcome-assessment"><span class="header-section-number">7.2.2.3.1</span> Outcome Assessment</h5>
<p>Reviewers described the outcome that was assessed and evaluated whether it was appropriate given the underlying study question.
They described whether there were any missing data such as whether there were individuals lost to follow up.
They then describe whether there were any potential sources of bias such as lack of blinding in a randomized controlled trial.</p>
<h5 data-number="7.2.2.3.2" id="statistical-methods-assessment"><span class="header-section-number">7.2.2.3.2</span> Statistical Methods Assessment</h5>
<p>Reviewers described which statistical methods were used for inference and whether applied methods were appropriate for the study.
They then described whether adjustments were made for possible confounders.</p>
<h4 data-number="7.2.2.4" id="results-1"><span class="header-section-number">7.2.2.4</span> Results</h4>
<p>Reviewers described the estimated association between the treatment and outcome.
They described measures of confidence or statistical significance, if provided.</p>
<h4 data-number="7.2.2.5" id="interpretation-1"><span class="header-section-number">7.2.2.5</span> Interpretation</h4>
<p>Reviewers described whether a causal claim could be made.
They described whether any side effects or interactions with other drugs were identified, as well as any subgroup findings.</p>
<h4 data-number="7.2.2.6" id="extrapolation-1"><span class="header-section-number">7.2.2.6</span> Extrapolation</h4>
<p>Reviewers describe how the study may extrapolate to a different species or population.</p>
<h1 class="page_break_before" data-number="8" id="additional-items"><span class="header-section-number">8</span> Additional Items</h1>
<h2 data-number="8.1" id="competing-interests"><span class="header-section-number">8.1</span> Competing Interests</h2>
<table>
<colgroup>
<col style="width: 33%" />
<col style="width: 33%" />
<col style="width: 33%" />
</colgroup>
<thead>
<tr class="header">
<th>Author</th>
<th>Competing Interests</th>
<th>Last Reviewed</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Halie M. Rando</td>
<td>None</td>
<td>2020-03-22</td>
</tr>
<tr class="even">
<td>Casey S. Greene</td>
<td>None</td>
<td>2020-11-07</td>
</tr>
<tr class="odd">
<td>Michael P. Robson</td>
<td>None</td>
<td>2020-03-23</td>
</tr>
<tr class="even">
<td>Simina M. Boca</td>
<td>None</td>
<td>2020-11-07</td>
</tr>
<tr class="odd">
<td>Nils Wellhausen</td>
<td>None</td>
<td>2020-11-03</td>
</tr>
<tr class="even">
<td>Ronan Lordan</td>
<td>None</td>
<td>2020-11-03</td>
</tr>
<tr class="odd">
<td>Christian Brueffer</td>
<td>Employee and shareholder of SAGA Diagnostics AB.</td>
<td>2020-11-11</td>
</tr>
<tr class="even">
<td>Sandipan Ray</td>
<td>None</td>
<td>2020-11-11</td>
</tr>
<tr class="odd">
<td>Lucy D'Agostino McGowan</td>
<td>Received consulting fees from Acelity and Sanofi in the past five years</td>
<td>2020-11-10</td>
</tr>
<tr class="even">
<td>Anthony Gitter</td>
<td>Filed a patent application with the Wisconsin Alumni Research Foundation related to classifying activated T cells</td>
<td>2020-11-10</td>
</tr>
<tr class="odd">
<td>Anna Ada Dattoli</td>
<td>None</td>
<td>2020-03-26</td>
</tr>
<tr class="even">
<td>Ryan Velazquez</td>
<td>None</td>
<td>2020-11-10</td>
</tr>
<tr class="odd">
<td>John P. Barton</td>
<td>None</td>
<td>2020-11-11</td>
</tr>
<tr class="even">
<td>Jeffrey M. Field</td>
<td>None</td>
<td>2020-03-30</td>
</tr>
<tr class="odd">
<td>Bharath Ramsundar</td>
<td>None</td>
<td>2020-11-11</td>
</tr>
<tr class="even">
<td>Adam L. MacLean</td>
<td>None</td>
<td>2020-11-11</td>
</tr>
<tr class="odd">
<td>Alexandra J. Lee</td>
<td>None</td>
<td>2020-11-09</td>
</tr>
<tr class="even">
<td>Immunology Institute of the Icahn School of Medicine</td>
<td>None</td>
<td>2020-04-07</td>
</tr>
<tr class="odd">
<td>Fengling Hu</td>
<td>None</td>
<td>2020-04-08</td>
</tr>
<tr class="even">
<td>Nafisa M. Jadavji</td>
<td>None</td>
<td>2020-04-09</td>
</tr>
<tr class="odd">
<td>Elizabeth Sell</td>
<td>None</td>
<td>2020-11-11</td>
</tr>
<tr class="even">
<td>Jinhui Wang</td>
<td>None</td>
<td>2020-04-13</td>
</tr>
<tr class="odd">
<td>Diane N. Rafizadeh</td>
<td>None</td>
<td>2020-11-11</td>
</tr>
<tr class="even">
<td>Ashwin N. Skelly</td>
<td>None</td>
<td>2020-11-11</td>
</tr>
<tr class="odd">
<td>Marouen Ben Guebila</td>
<td>None</td>
<td>2020-11-11</td>
</tr>
<tr class="even">
<td>Likhitha Kolla</td>
<td>None</td>
<td>2020-04-23</td>
</tr>
<tr class="odd">
<td>David Manheim</td>
<td>None</td>
<td>2020-11-11</td>
</tr>
<tr class="even">
<td>Soumita Ghosh</td>
<td>None</td>
<td>2020-11-09</td>
</tr>
<tr class="odd">
<td>Matthias Fax</td>
<td>None</td>
<td>2020-04-30</td>
</tr>
<tr class="even">
<td>James Brian Byrd</td>
<td>Funded by FastGrants to conduct a COVID-19-related clinical trial</td>
<td>2020-04-23</td>
</tr>
<tr class="odd">
<td>YoSon Park</td>
<td>None</td>
<td>2020-04-30</td>
</tr>
<tr class="even">
<td>Vikas Bansal</td>
<td>None</td>
<td>2020-05-26</td>
</tr>
<tr class="odd">
<td>Stephen Capone</td>
<td>None</td>
<td>2020-11-11</td>
</tr>
<tr class="even">
<td>John J. Dziak</td>
<td>None</td>
<td>2020-11-11</td>
</tr>
<tr class="odd">
<td>Yuchen Sun</td>
<td>None</td>
<td>2020-11-11</td>
</tr>
<tr class="even">
<td>Yanjun Qi</td>
<td>None</td>
<td>2020-07-09</td>
</tr>
<tr class="odd">
<td>Lamonica Shinholster</td>
<td>None</td>
<td>2020-11-11</td>
</tr>
<tr class="even">
<td>Sergey Knyazev</td>
<td>None</td>
<td>2020-11-11</td>
</tr>
<tr class="odd">
<td>Dimitri Perrin</td>
<td>None</td>
<td>2020-11-11</td>
</tr>
<tr class="even">
<td>Serghei Mangul</td>
<td>None</td>
<td>2020-11-11</td>
</tr>
<tr class="odd">
<td>Shikta Das</td>
<td>None</td>
<td>2020-08-13</td>
</tr>
<tr class="even">
<td>Gregory L Szeto</td>
<td>None</td>
<td>2020-09-04</td>
</tr>
<tr class="odd">
<td>Tiago Lubiana</td>
<td>None</td>
<td>2020-11-11</td>
</tr>
</tbody>
</table>
<h2 data-number="8.2" id="author-contributions"><span class="header-section-number">8.2</span> Author Contributions</h2>
<table>
<colgroup>
<col style="width: 50%" />
<col style="width: 50%" />
</colgroup>
<thead>
<tr class="header">
<th>Author</th>
<th>Contributions</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Halie M. Rando</td>
<td>Project Administration, Writing - Original Draft, Writing - Review &amp; Editing, Methodology</td>
</tr>
<tr class="even">
<td>Casey S. Greene</td>
<td>Conceptualization, Software, Writing - Review &amp; Editing</td>
</tr>
<tr class="odd">
<td>Michael P. Robson</td>
<td>Software</td>
</tr>
<tr class="even">
<td>Simina M. Boca</td>
<td>Methodology, Writing - Review &amp; Editing</td>
</tr>
<tr class="odd">
<td>Nils Wellhausen</td>
<td>Writing - Original Draft, Writing - Review &amp; Editing, Project Administration, Visualization</td>
</tr>
<tr class="even">
<td>Ronan Lordan</td>
<td>Writing - Original Draft, Writing - Review &amp; Editing</td>
</tr>
<tr class="odd">
<td>Christian Brueffer</td>
<td>Writing - Original Draft, Writing - Review &amp; Editing, Project Administration</td>
</tr>
<tr class="even">
<td>Sandipan Ray</td>
<td>Writing - Original Draft, Writing - Review &amp; Editing</td>
</tr>
<tr class="odd">
<td>Lucy D'Agostino McGowan</td>
<td>Methodology, Writing - Original Draft</td>
</tr>
<tr class="even">
<td>Anthony Gitter</td>
<td>Methodology, Software, Project Administration, Writing - Original Draft, Writing - Review &amp; Editing, Visualization</td>
</tr>
<tr class="odd">
<td>Anna Ada Dattoli</td>
<td>Writing - Original Draft</td>
</tr>
<tr class="even">
<td>Ryan Velazquez</td>
<td>Methodology, Software</td>
</tr>
<tr class="odd">
<td>John P. Barton</td>
<td>Writing - Original Draft, Writing - Review &amp; Editing</td>
</tr>
<tr class="even">
<td>Jeffrey M. Field</td>
<td>Writing - Original Draft</td>
</tr>
<tr class="odd">
<td>Bharath Ramsundar</td>
<td>Investigation, Writing - Review &amp; Editing</td>
</tr>
<tr class="even">
<td>Adam L. MacLean</td>
<td>Writing - Original Draft, Writing - Review &amp; Editing</td>
</tr>
<tr class="odd">
<td>Alexandra J. Lee</td>
<td>Writing - Original Draft, Writing - Review &amp; Editing</td>
</tr>
<tr class="even">
<td>Immunology Institute of the Icahn School of Medicine</td>
<td>Data Curation</td>
</tr>
<tr class="odd">
<td>Fengling Hu</td>
<td>Writing - Original Draft, Writing - Review &amp; Editing</td>
</tr>
<tr class="even">
<td>Nafisa M. Jadavji</td>
<td>Writing - Original Draft, Writing - Review &amp; Editing</td>
</tr>
<tr class="odd">
<td>Elizabeth Sell</td>
<td>Writing - Original Draft, Writing - Review &amp; Editing</td>
</tr>
<tr class="even">
<td>Jinhui Wang</td>
<td>Writing - Revising &amp; Editing</td>
</tr>
<tr class="odd">
<td>Diane N. Rafizadeh</td>
<td>Writing - Original Draft, Writing - Review &amp; Editing</td>
</tr>
<tr class="even">
<td>Ashwin N. Skelly</td>
<td>Writing - Original Draft, Writing - Review &amp; Editing</td>
</tr>
<tr class="odd">
<td>Marouen Ben Guebila</td>
<td>Writing - Original Draft</td>
</tr>
<tr class="even">
<td>Likhitha Kolla</td>
<td>Writing - Original Draft</td>
</tr>
<tr class="odd">
<td>David Manheim</td>
<td>Writing - Original Draft, Investigation</td>
</tr>
<tr class="even">
<td>Soumita Ghosh</td>
<td>Writing - Original Draft</td>
</tr>
<tr class="odd">
<td>Matthias Fax</td>
<td>Writing - Review &amp; Editing</td>
</tr>
<tr class="even">
<td>James Brian Byrd</td>
<td>Writing - Original Draft, Writing - Review &amp; Editing</td>
</tr>
<tr class="odd">
<td>YoSon Park</td>
<td>Writing - Original Draft, Writing - Review &amp; Editing, Investigation</td>
</tr>
<tr class="even">
<td>Vikas Bansal</td>
<td>Writing - Original Draft, Investigation</td>
</tr>
<tr class="odd">
<td>Stephen Capone</td>
<td>Writing - Review &amp; Editing, Writing - Original Draft</td>
</tr>
<tr class="even">
<td>John J. Dziak</td>
<td>Writing - Original Draft, Writing - Review &amp; Editing</td>
</tr>
<tr class="odd">
<td>Yuchen Sun</td>
<td>Visualization</td>
</tr>
<tr class="even">
<td>Yanjun Qi</td>
<td>Visualization</td>
</tr>
<tr class="odd">
<td>Lamonica Shinholster</td>
<td>Writing - Original Draft</td>
</tr>
<tr class="even">
<td>Sergey Knyazev</td>
<td>Writing - Original Draft, Writing - Review &amp; Editing</td>
</tr>
<tr class="odd">
<td>Dimitri Perrin</td>
<td>Writing - Original Draft, Writing - Review &amp; Editing</td>
</tr>
<tr class="even">
<td>Serghei Mangul</td>
<td>Writing - Review &amp; Editing</td>
</tr>
<tr class="odd">
<td>Shikta Das</td>
<td>Writing - Review &amp; Editing</td>
</tr>
<tr class="even">
<td>Gregory L Szeto</td>
<td>Writing - Review &amp; Editing</td>
</tr>
<tr class="odd">
<td>Tiago Lubiana</td>
<td>Writing - Review &amp; Editing</td>
</tr>
</tbody>
</table>
<h2 data-number="8.3" id="acknowledgements"><span class="header-section-number">8.3</span> Acknowledgements</h2>
<p>We thank Nick DeVito for assistance with the Evidence-Based Medicine Data Lab COVID-19 TrialsTracker data.
We thank Yael Evelyn Marshall who contributed writing (original draft) as well as reviewing and editing of pieces of the text but who did not formally approve the manuscript, as well as Ronnie Russell, who contributed text to and helped develop the structure of the manuscript early in the writing process.
We are grateful to the following contributors for reviewing pieces of the text: Nadia Danilova, James Eberwine and Ipsita Krishnan.</p>
<h1 class="page_break_before" data-number="9" id="references"><span class="header-section-number">9</span> References</h1>
<!-- Explicitly insert bibliography here -->
<div id="refs" class="references hanging-indent" role="doc-bibliography">
<div id="ref-Vb5KFWnO">
<p>1. <strong>Novel Coronavirus (2019-nCoV) SITUATION REPORT - 1</strong> <br />
World Health Organization<br />
(2020-01-21) <a href="https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200121-sitrep-1-2019-ncov.pdf">https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200121-sitrep-1-2019-ncov.pdf</a></p>
</div>
<div id="ref-fLTiwToY">
<p>2. <strong>Novel Coronavirus (2019-nCoV) Situation Report - 8</strong> <br />
World Health Organization<br />
(2020-01-28) <a href="https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200128-sitrep-8-ncov-cleared.pdf">https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200128-sitrep-8-ncov-cleared.pdf</a></p>
</div>
<div id="ref-x9975iBK">
<p>3. <strong>Novel Coronavirus (2019-nCoV) Situation Report - 51</strong> <br />
World Health Organization<br />
(2020-03-11) <a href="https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200311-sitrep-51-covid-19.pdf">https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200311-sitrep-51-covid-19.pdf</a></p>
</div>
<div id="ref-1GACP7OZY">
<p>4. <strong>Novel Coronavirus (2019-nCoV) Situation Report - 75</strong> <br />
World Health Organization<br />
(2020-04-04) <a href="https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200404-sitrep-75-covid-19.pdf">https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200404-sitrep-75-covid-19.pdf</a></p>
</div>
<div id="ref-MrwDDw9R">
<p>5. <strong>COVID-19 Data Repository</strong> <br />
Center for Systems Science and Engineering at Johns Hopkins University<br />
<em>GitHub</em> <a href="https://github.com/CSSEGISandData/COVID-19/tree/master/csse_covid_19_data/csse_covid_19_time_series">https://github.com/CSSEGISandData/COVID-19/tree/master/csse_covid_19_data/csse_covid_19_time_series</a></p>
</div>
<div id="ref-1HZeeO4Cs">
<p>6. <strong>How you can help with COVID-19 modelling</strong> <br />
Julia R. Gog<br />
<em>Nature Reviews Physics</em> (2020-04-08) <a href="https://doi.org/ggrsq3">https://doi.org/ggrsq3</a> <br />
DOI: <a href="https://doi.org/10.1038/s42254-020-0175-7">10.1038/s42254-020-0175-7</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7144181">PMC7144181</a></p>
</div>
<div id="ref-YuJbg3zO">
<p>7. <strong>Open collaborative writing with Manubot</strong> <br />
Daniel S. Himmelstein, Vincent Rubinetti, David R. Slochower, Dongbo Hu, Venkat S. Malladi, Casey S. Greene, Anthony Gitter<br />
<em>PLOS Computational Biology</em> (2019-06-24) <a href="https://doi.org/c7np">https://doi.org/c7np</a> <br />
DOI: <a href="https://doi.org/10.1371/journal.pcbi.1007128">10.1371/journal.pcbi.1007128</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31233491">31233491</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6611653">PMC6611653</a></p>
</div>
<div id="ref-xdRgnrxv">
<p>8. <strong>Coronavirus Disease 2019 (COVID-19)</strong> <br />
CDC<br />
<em>Centers for Disease Control and Prevention</em> (2020-02-11) <a href="https://www.cdc.gov/coronavirus/2019-ncov/cases-updates/about-epidemiology/index.html">https://www.cdc.gov/coronavirus/2019-ncov/cases-updates/about-epidemiology/index.html</a></p>
</div>
<div id="ref-t1Gs6K2A">
<p>9. <strong>IHME | COVID-19 Projections</strong> <br />
Institute for Health Metrics and Evaluation<br />
<a href="https://covid19.healthdata.org/">https://covid19.healthdata.org/</a></p>
</div>
<div id="ref-13CJl7iIO">
<p>10. <strong>Managing epidemics: key facts about major deadly diseases.</strong> <br />
World Health Organization<br />
(2018) <br />
ISBN: <a href="https://worldcat.org/isbn/9789241565530">9789241565530</a></p>
</div>
<div id="ref-Bp847Lfa">
<p>11. <strong>A Novel Coronavirus Genome Identified in a Cluster of Pneumonia Cases — Wuhan, China 2019−2020</strong> <br />
Wenjie Tan, Xiang Zhao, Xuejun Ma, Wenling Wang, Peihua Niu, Wenbo Xu, George F. Gao, Guizhen Wu, MHC Key Laboratory of Biosafety, National Institute for Viral Disease Control and Prevention, China CDC, Beijing, China, Center for Biosafety Mega-Science, Chinese Academy of Sciences, Beijing, China<br />
<em>China CDC Weekly</em> (2020) <a href="https://doi.org/gg8z47">https://doi.org/gg8z47</a> <br />
DOI: <a href="https://doi.org/10.46234/ccdcw2020.017">10.46234/ccdcw2020.017</a></p>
</div>
<div id="ref-vHq0hOWQ">
<p>12. <strong>A new coronavirus associated with human respiratory disease in China</strong> <br />
Fan Wu, Su Zhao, Bin Yu, Yan-Mei Chen, Wen Wang, Zhi-Gang Song, Yi Hu, Zhao-Wu Tao, Jun-Hua Tian, Yuan-Yuan Pei, … Yong-Zhen Zhang<br />
<em>Nature</em> (2020-02-03) <a href="https://doi.org/dk2w">https://doi.org/dk2w</a> <br />
DOI: <a href="https://doi.org/10.1038/s41586-020-2008-3">10.1038/s41586-020-2008-3</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32015508">32015508</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7094943">PMC7094943</a></p>
</div>
<div id="ref-VSkK7CeP">
<p>13. <strong>A pneumonia outbreak associated with a new coronavirus of probable bat origin</strong> <br />
Peng Zhou, Xing-Lou Yang, Xian-Guang Wang, Ben Hu, Lei Zhang, Wei Zhang, Hao-Rui Si, Yan Zhu, Bei Li, Chao-Lin Huang, … Zheng-Li Shi<br />
<em>Nature</em> (2020-02-03) <a href="https://doi.org/ggj5cg">https://doi.org/ggj5cg</a> <br />
DOI: <a href="https://doi.org/10.1038/s41586-020-2012-7">10.1038/s41586-020-2012-7</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32015507">32015507</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7095418">PMC7095418</a></p>
</div>
<div id="ref-1Fqilxaum">
<p>14. <strong>Genomic characterisation and epidemiology of 2019 novel coronavirus: implications for virus origins and receptor binding</strong> <br />
Roujian Lu, Xiang Zhao, Juan Li, Peihua Niu, Bo Yang, Honglong Wu, Wenling Wang, Hao Song, Baoying Huang, Na Zhu, … Wenjie Tan<br />
<em>The Lancet</em> (2020-02) <a href="https://doi.org/ggjr43">https://doi.org/ggjr43</a> <br />
DOI: <a href="https://doi.org/10.1016/s0140-6736(20)30251-8">10.1016/s0140-6736(20)30251-8</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32007145">32007145</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7159086">PMC7159086</a></p>
</div>
<div id="ref-UUAeVUaR">
<p>15. <strong>On the origin and continuing evolution of SARS-CoV-2</strong> <br />
Xiaolu Tang, Changcheng Wu, Xiang Li, Yuhe Song, Xinmin Yao, Xinkai Wu, Yuange Duan, Hong Zhang, Yirong Wang, Zhaohui Qian, … Jian Lu<br />
<em>National Science Review</em> (2020-03-03) <a href="https://doi.org/ggndzn">https://doi.org/ggndzn</a> <br />
DOI: <a href="https://doi.org/10.1093/nsr/nwaa036">10.1093/nsr/nwaa036</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7107875">PMC7107875</a></p>
</div>
<div id="ref-eQIU1Nbs">
<p>16. <strong>Pangolin homology associated with 2019-nCoV</strong> <br />
Tao Zhang, Qunfu Wu, Zhigang Zhang<br />
<em>bioRxiv</em> (2020-02-20) <a href="https://doi.org/ggpvpt">https://doi.org/ggpvpt</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.02.19.950253">10.1101/2020.02.19.950253</a></p>
</div>
<div id="ref-wz7kPyn2">
<p>17. <strong>Emergence of SARS-CoV-2 through recombination and strong purifying selection</strong> <br />
Xiaojun Li, Elena E. Giorgi, Manukumar Honnayakanahalli Marichannegowda, Brian Foley, Chuan Xiao, Xiang-Peng Kong, Yue Chen, S. Gnanakaran, Bette Korber, Feng Gao<br />
<em>Science Advances</em> (2020-07) <a href="https://doi.org/gg6r93">https://doi.org/gg6r93</a> <br />
DOI: <a href="https://doi.org/10.1126/sciadv.abb9153">10.1126/sciadv.abb9153</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32937441">32937441</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7458444">PMC7458444</a></p>
</div>
<div id="ref-NEIoHDjq">
<p>18. <strong>The proximal origin of SARS-CoV-2</strong> <br />
Kristian G. Andersen, Andrew Rambaut, W. Ian Lipkin, Edward C. Holmes, Robert F. Garry<br />
<em>Nature Medicine</em> (2020-03-17) <a href="https://doi.org/ggn4dn">https://doi.org/ggn4dn</a> <br />
DOI: <a href="https://doi.org/10.1038/s41591-020-0820-9">10.1038/s41591-020-0820-9</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32284615">32284615</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7095063">PMC7095063</a></p>
</div>
<div id="ref-10oGRZVIE">
<p>19. <strong>Coronavirus immunogens</strong> <br />
Linda J. Saif<br />
<em>Veterinary Microbiology</em> (1993-11) <a href="https://doi.org/ckfn8b">https://doi.org/ckfn8b</a> <br />
DOI: <a href="https://doi.org/10.1016/0378-1135(93)90030-b">10.1016/0378-1135(93)90030-b</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/8116187">8116187</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7117163">PMC7117163</a></p>
</div>
<div id="ref-UBTUyMGV">
<p>20. <strong>Origin and evolution of pathogenic coronaviruses</strong> <br />
Jie Cui, Fang Li, Zheng-Li Shi<br />
<em>Nature Reviews Microbiology</em> (2018-12-10) <a href="https://doi.org/ggh4vb">https://doi.org/ggh4vb</a> <br />
DOI: <a href="https://doi.org/10.1038/s41579-018-0118-9">10.1038/s41579-018-0118-9</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30531947">30531947</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7097006">PMC7097006</a></p>
</div>
<div id="ref-YAhVzI0A">
<p>21. <strong>Human Coronaviruses: A Review of Virus–Host Interactions</strong> <br />
Yvonne Lim, Yan Ng, James Tam, Ding Liu<br />
<em>Diseases</em> (2016-07-25) <a href="https://doi.org/ggjs23">https://doi.org/ggjs23</a> <br />
DOI: <a href="https://doi.org/10.3390/diseases4030026">10.3390/diseases4030026</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28933406">28933406</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5456285">PMC5456285</a></p>
</div>
<div id="ref-gaAv88da">
<p>22. <strong>A New Virus Isolated from the Human Respiratory Tract.</strong> <br />
D. Hamre, J. J. Procknow<br />
<em>Experimental Biology and Medicine</em> (1966-01-01) <a href="https://doi.org/gg84fc">https://doi.org/gg84fc</a> <br />
DOI: <a href="https://doi.org/10.3181/00379727-121-30734">10.3181/00379727-121-30734</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/4285768">4285768</a></p>
</div>
<div id="ref-q6rFk3Kl">
<p>23. <strong>Recovery in tracheal organ cultures of novel viruses from patients with respiratory disease.</strong> <br />
K. McIntosh, J. H. Dees, W. B. Becker, A. Z. Kapikian, R. M. Chanock<br />
<em>Proceedings of the National Academy of Sciences</em> (1967-04-01) <a href="https://doi.org/bhfd6w">https://doi.org/bhfd6w</a> <br />
DOI: <a href="https://doi.org/10.1073/pnas.57.4.933">10.1073/pnas.57.4.933</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/5231356">5231356</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC224637">PMC224637</a></p>
</div>
<div id="ref-cEFaGSlu">
<p>24.<strong>:{unav)</strong> <br />
Krzysztof Pyrc, Maarten F Jebbink, Ben Berkhout, Lia van der Hoek<br />
<em>Virology Journal</em> (2004) <a href="https://doi.org/dnmj8m">https://doi.org/dnmj8m</a> <br />
DOI: <a href="https://doi.org/10.1186/1743-422x-1-7">10.1186/1743-422x-1-7</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15548333">15548333</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC538260">PMC538260</a></p>
</div>
<div id="ref-Jnb5pce4">
<p>25. <strong>Understanding Human Coronavirus HCoV-NL63.</strong> <br />
Sahar Abdul-Rasool, Burtram C Fielding<br />
<em>The open virology journal</em> (2010-05-25) <a href="https://www.ncbi.nlm.nih.gov/pubmed/20700397">https://www.ncbi.nlm.nih.gov/pubmed/20700397</a> <br />
DOI: <a href="https://doi.org/10.2174/1874357901004010076">10.2174/1874357901004010076</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20700397">20700397</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2918871">PMC2918871</a></p>
</div>
<div id="ref-URhDY7s9">
<p>26. <strong>Identification of a new human coronavirus</strong> <br />
Lia van der Hoek, Krzysztof Pyrc, Maarten F Jebbink, Wilma Vermeulen-Oost, Ron JM Berkhout, Katja C Wolthers, Pauline ME Wertheim-van Dillen, Jos Kaandorp, Joke Spaargaren, Ben Berkhout<br />
<em>Nature Medicine</em> (2004-03-21) <a href="https://doi.org/b5wtsn">https://doi.org/b5wtsn</a> <br />
DOI: <a href="https://doi.org/10.1038/nm1024">10.1038/nm1024</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15034574">15034574</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7095789">PMC7095789</a></p>
</div>
<div id="ref-VgGISeiE">
<p>27. <strong>Characterization and Complete Genome Sequence of a Novel Coronavirus, Coronavirus HKU1, from Patients with Pneumonia</strong> <br />
P. C. Y. Woo, S. K. P. Lau, C.-m. Chu, K.-h. Chan, H.-w. Tsoi, Y. Huang, B. H. L. Wong, R. W. S. Poon, J. J. Cai, W.-k. Luk, … K.-y. Yuen<br />
<em>Journal of Virology</em> (2004-12-20) <a href="https://doi.org/bk7m7h">https://doi.org/bk7m7h</a> <br />
DOI: <a href="https://doi.org/10.1128/jvi.79.2.884-895.2005">10.1128/jvi.79.2.884-895.2005</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15613317">15613317</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC538593">PMC538593</a></p>
</div>
<div id="ref-rDtyU6cv">
<p>28. <strong>Coronavirus 229E-Related Pneumonia in Immunocompromised Patients</strong> <br />
F. Pene, A. Merlat, A. Vabret, F. Rozenberg, A. Buzyn, F. Dreyfus, A. Cariou, F. Freymuth, P. Lebon<br />
<em>Clinical Infectious Diseases</em> (2003-10-01) <a href="https://doi.org/dcjk64">https://doi.org/dcjk64</a> <br />
DOI: <a href="https://doi.org/10.1086/377612">10.1086/377612</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/13130404">13130404</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7107892">PMC7107892</a></p>
</div>
<div id="ref-G5NJrE75">
<p>29. <strong>SARS and MERS: recent insights into emerging coronaviruses</strong> <br />
Emmie de Wit, Neeltje van Doremalen, Darryl Falzarano, Vincent J. Munster<br />
<em>Nature Reviews Microbiology</em> (2016-06-27) <a href="https://doi.org/f8v5cv">https://doi.org/f8v5cv</a> <br />
DOI: <a href="https://doi.org/10.1038/nrmicro.2016.81">10.1038/nrmicro.2016.81</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27344959">27344959</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7097822">PMC7097822</a></p>
</div>
<div id="ref-uXEsLlzb">
<p>30. <strong>Hosts and Sources of Endemic Human Coronaviruses</strong> <br />
Victor M. Corman, Doreen Muth, Daniela Niemeyer, Christian Drosten<br />
<em>Advances in Virus Research</em> (2018) <a href="https://doi.org/ggwx4j">https://doi.org/ggwx4j</a> <br />
DOI: <a href="https://doi.org/10.1016/bs.aivir.2018.01.001">10.1016/bs.aivir.2018.01.001</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29551135">29551135</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7112090">PMC7112090</a></p>
</div>
<div id="ref-sP4wQEiM">
<p>31. <strong>Three Emerging Coronaviruses in Two Decades</strong> <br />
Jeannette Guarner<br />
<em>American Journal of Clinical Pathology</em> (2020-02-13) <a href="https://doi.org/ggppq3">https://doi.org/ggppq3</a> <br />
DOI: <a href="https://doi.org/10.1093/ajcp/aqaa029">10.1093/ajcp/aqaa029</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32053148">32053148</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7109697">PMC7109697</a></p>
</div>
<div id="ref-NHVzYmXx">
<p>32. <strong>From SARS to MERS, Thrusting Coronaviruses into the Spotlight</strong> <br />
Zhiqi Song, Yanfeng Xu, Linlin Bao, Ling Zhang, Pin Yu, Yajin Qu, Hua Zhu, Wenjie Zhao, Yunlin Han, Chuan Qin<br />
<em>Viruses</em> (2019-01-14) <a href="https://doi.org/ggqp7h">https://doi.org/ggqp7h</a> <br />
DOI: <a href="https://doi.org/10.3390/v11010059">10.3390/v11010059</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30646565">30646565</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6357155">PMC6357155</a></p>
</div>
<div id="ref-icy0PeD7">
<p>33. <strong>1. Overview of the human immune response</strong> <br />
D CHAPLIN<br />
<em>Journal of Allergy and Clinical Immunology</em> (2006-02) <a href="https://doi.org/b6zghf">https://doi.org/b6zghf</a> <br />
DOI: <a href="https://doi.org/10.1016/j.jaci.2005.09.034">10.1016/j.jaci.2005.09.034</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16455341">16455341</a></p>
</div>
<div id="ref-OlEbK4fc">
<p>34. <strong>Molecular immune pathogenesis and diagnosis of COVID-19</strong> <br />
Xiaowei Li, Manman Geng, Yizhao Peng, Liesu Meng, Shemin Lu<br />
<em>Journal of Pharmaceutical Analysis</em> (2020-03) <a href="https://doi.org/ggppqg">https://doi.org/ggppqg</a> <br />
DOI: <a href="https://doi.org/10.1016/j.jpha.2020.03.001">10.1016/j.jpha.2020.03.001</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32282863">32282863</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7104082">PMC7104082</a></p>
</div>
<div id="ref-15IDA5kX6">
<p>35. <strong>Clinical Characteristics of Coronavirus Disease 2019 in China</strong> <br />
Wei-jie Guan, Zheng-yi Ni, Yu Hu, Wen-hua Liang, Chun-quan Ou, Jian-xing He, Lei Liu, Hong Shan, Chun-liang Lei, David S. C. Hui, … Nan-shan Zhong<br />
<em>New England Journal of Medicine</em> (2020-02-28) <a href="https://doi.org/ggm6dh">https://doi.org/ggm6dh</a> <br />
DOI: <a href="https://doi.org/10.1056/nejmoa2002032">10.1056/nejmoa2002032</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32109013">32109013</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7092819">PMC7092819</a></p>
</div>
<div id="ref-10THxyeCg">
<p>36. <strong>Clinical course and risk factors for mortality of adult inpatients with COVID-19 in Wuhan, China: a retrospective cohort study</strong> <br />
Fei Zhou, Ting Yu, Ronghui Du, Guohui Fan, Ying Liu, Zhibo Liu, Jie Xiang, Yeming Wang, Bin Song, Xiaoying Gu, … Bin Cao<br />
<em>The Lancet</em> (2020-03) <a href="https://doi.org/ggnxb3">https://doi.org/ggnxb3</a> <br />
DOI: <a href="https://doi.org/10.1016/s0140-6736(20)30566-3">10.1016/s0140-6736(20)30566-3</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32171076">32171076</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7270627">PMC7270627</a></p>
</div>
<div id="ref-19ytsiSpq">
<p>37. <strong>Presenting Characteristics, Comorbidities, and Outcomes Among 5700 Patients Hospitalized With COVID-19 in the New York City Area</strong> <br />
Safiya Richardson, Jamie S. Hirsch, Mangala Narasimhan, James M. Crawford, Thomas McGinn, Karina W. Davidson, Douglas P. Barnaby, Lance B. Becker, John D. Chelico, Stuart L. Cohen, … and the Northwell COVID-19 Research Consortium<br />
<em>JAMA</em> (2020-05-26) <a href="https://doi.org/ggsrkd">https://doi.org/ggsrkd</a> <br />
DOI: <a href="https://doi.org/10.1001/jama.2020.6775">10.1001/jama.2020.6775</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32320003">32320003</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7177629">PMC7177629</a></p>
</div>
<div id="ref-1H8gSsSkn">
<p>38. <strong>Clinical Characteristics of Covid-19 in New York City</strong> <br />
Parag Goyal, Justin J. Choi, Laura C. Pinheiro, Edward J. Schenck, Ruijun Chen, Assem Jabri, Michael J. Satlin, Thomas R. Campion, Musarrat Nahid, Joanna B. Ringel, … Monika M. Safford<br />
<em>New England Journal of Medicine</em> (2020-06-11) <a href="https://doi.org/ggtsjc">https://doi.org/ggtsjc</a> <br />
DOI: <a href="https://doi.org/10.1056/nejmc2010419">10.1056/nejmc2010419</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32302078">32302078</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7182018">PMC7182018</a></p>
</div>
<div id="ref-I2EsJmfs">
<p>39. <strong>COVID-19: Epidemiology, Evolution, and Cross-Disciplinary Perspectives</strong> <br />
Jiumeng Sun, Wan-Ting He, Lifang Wang, Alexander Lai, Xiang Ji, Xiaofeng Zhai, Gairu Li, Marc A. Suchard, Jin Tian, Jiyong Zhou, … Shuo Su<br />
<em>Trends in Molecular Medicine</em> (2020-03) <a href="https://doi.org/ggqgwq">https://doi.org/ggqgwq</a> <br />
DOI: <a href="https://doi.org/10.1016/j.molmed.2020.02.008">10.1016/j.molmed.2020.02.008</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32359479">32359479</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7118693">PMC7118693</a></p>
</div>
<div id="ref-5x25saIz">
<p>40. <strong>Immunology of COVID-19: Current State of the Science</strong> <br />
Nicolas Vabret, Graham J. Britton, Conor Gruber, Samarth Hegde, Joel Kim, Maria Kuksin, Rachel Levantovsky, Louise Malle, Alvaro Moreira, Matthew D. Park, … Uri Laserson<br />
<em>Immunity</em> (2020-06) <a href="https://doi.org/ggt54g">https://doi.org/ggt54g</a> <br />
DOI: <a href="https://doi.org/10.1016/j.immuni.2020.05.002">10.1016/j.immuni.2020.05.002</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32505227">32505227</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7200337">PMC7200337</a></p>
</div>
<div id="ref-evtsR3C5">
<p>41. <strong>COVID-19 diagnostics in context</strong> <br />
Ralph Weissleder, Hakho Lee, Jina Ko, Mikael J. Pittet<br />
<em>Science Translational Medicine</em> (2020-06-03) <a href="https://doi.org/gg339m">https://doi.org/gg339m</a> <br />
DOI: <a href="https://doi.org/10.1126/scitranslmed.abc1931">10.1126/scitranslmed.abc1931</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32493791">32493791</a></p>
</div>
<div id="ref-18eCxyLhx">
<p>42. <strong>Pharmacologic Treatments for Coronavirus Disease 2019 (COVID-19)</strong> <br />
James M. Sanders, Marguerite L. Monogue, Tomasz Z. Jodlowski, James B. Cutrell<br />
<em>JAMA</em> (2020-04-13) <a href="https://doi.org/ggr27x">https://doi.org/ggr27x</a> <br />
DOI: <a href="https://doi.org/10.1001/jama.2020.6019">10.1001/jama.2020.6019</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32282022">32282022</a></p>
</div>
<div id="ref-SAE5ME3N">
<p>43. <strong>COVID-19 Research in Brief: December, 2019 to June, 2020</strong> <br />
Thiago Carvalho<br />
<em>Nature Medicine</em> (2020-06-26) <a href="https://doi.org/gg3kd2">https://doi.org/gg3kd2</a> <br />
DOI: <a href="https://doi.org/10.1038/d41591-020-00026-w">10.1038/d41591-020-00026-w</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32778824">32778824</a></p>
</div>
<div id="ref-xOs5ctsW">
<p>44. <strong>Pathophysiology, Transmission, Diagnosis, and Treatment of Coronavirus Disease 2019 (COVID-19)</strong> <br />
W. Joost Wiersinga, Andrew Rhodes, Allen C. Cheng, Sharon J. Peacock, Hallie C. Prescott<br />
<em>JAMA</em> (2020-08-25) <a href="https://doi.org/gg4ht4">https://doi.org/gg4ht4</a> <br />
DOI: <a href="https://doi.org/10.1001/jama.2020.12839">10.1001/jama.2020.12839</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32648899">32648899</a></p>
</div>
<div id="ref-gHWlMufv">
<p>45. <strong>Coronavirus Disease 2019 (COVID-19) – Symptoms</strong> <br />
CDC<br />
<em>Centers for Disease Control and Prevention</em> (2020-05-13) <a href="https://www.cdc.gov/coronavirus/2019-ncov/symptoms-testing/symptoms.html">https://www.cdc.gov/coronavirus/2019-ncov/symptoms-testing/symptoms.html</a></p>
</div>
<div id="ref-jsMmdH4D">
<p>46. <strong>Medical microbiology</strong> <br />
Samuel Baron (editor)<br />
<em>University of Texas Medical Branch at Galveston</em> (1996) <br />
ISBN: <a href="https://worldcat.org/isbn/9780963117212">9780963117212</a></p>
</div>
<div id="ref-RGAUx68v">
<p>47. <strong>Fields virology</strong> <br />
Bernard N. Fields, David M. Knipe, Peter M. Howley (editors)<br />
<em>Wolters Kluwer Health/Lippincott Williams &amp; Wilkins</em> (2007) <br />
ISBN: <a href="https://worldcat.org/isbn/9780781760607">9780781760607</a></p>
</div>
<div id="ref-DlBAegL1">
<p>48. <strong>Important Role for the Transmembrane Domain of Severe Acute Respiratory Syndrome Coronavirus Spike Protein during Entry</strong> <br />
Rene Broer, Bertrand Boson, Willy Spaan, François-Loïc Cosset, Jeroen Corver<br />
<em>Journal of Virology</em> (2006-02-01) <a href="https://doi.org/dvvg2h">https://doi.org/dvvg2h</a> <br />
DOI: <a href="https://doi.org/10.1128/jvi.80.3.1302-1310.2006">10.1128/jvi.80.3.1302-1310.2006</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16415007">16415007</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1346921">PMC1346921</a></p>
</div>
<div id="ref-CqQVkaqj">
<p>49. <strong>Cryo-EM structure of the 2019-nCoV spike in the prefusion conformation</strong> <br />
Daniel Wrapp, Nianshuang Wang, Kizzmekia S. Corbett, Jory A. Goldsmith, Ching-Lin Hsieh, Olubukola Abiona, Barney S. Graham, Jason S. McLellan<br />
<em>Science</em> (2020-02-19) <a href="https://doi.org/ggmtk2">https://doi.org/ggmtk2</a> <br />
DOI: <a href="https://doi.org/10.1126/science.abb2507">10.1126/science.abb2507</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32075877">32075877</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7164637">PMC7164637</a></p>
</div>
<div id="ref-17DSmRo9H">
<p>50. <strong>Structure, Function, and Evolution of Coronavirus Spike Proteins</strong> <br />
Fang Li<br />
<em>Annual Review of Virology</em> (2016-09-29) <a href="https://doi.org/ggr7gv">https://doi.org/ggr7gv</a> <br />
DOI: <a href="https://doi.org/10.1146/annurev-virology-110615-042301">10.1146/annurev-virology-110615-042301</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27578435">27578435</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5457962">PMC5457962</a></p>
</div>
<div id="ref-IFLf8rsY">
<p>51. <strong>A familial cluster of pneumonia associated with the 2019 novel coronavirus indicating person-to-person transmission: a study of a family cluster</strong> <br />
Jasper Fuk-Woo Chan, Shuofeng Yuan, Kin-Hang Kok, Kelvin Kai-Wang To, Hin Chu, Jin Yang, Fanfan Xing, Jieling Liu, Cyril Chik-Yan Yip, Rosana Wing-Shan Poon, … Kwok-Yung Yuen<br />
<em>The Lancet</em> (2020-02) <a href="https://doi.org/ggjs7j">https://doi.org/ggjs7j</a> <br />
DOI: <a href="https://doi.org/10.1016/s0140-6736(20)30154-9">10.1016/s0140-6736(20)30154-9</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31986261">31986261</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7159286">PMC7159286</a></p>
</div>
<div id="ref-47hnjvDb">
<p>52. <strong>Coronaviruses: An Overview of Their Replication and Pathogenesis</strong> <br />
Anthony R. Fehr, Stanley Perlman<br />
<em>Methods in Molecular Biology</em> (2015) <a href="https://doi.org/ggpc6n">https://doi.org/ggpc6n</a> <br />
DOI: <a href="https://doi.org/10.1007/978-1-4939-2438-7_1">10.1007/978-1-4939-2438-7_1</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25720466">25720466</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4369385">PMC4369385</a></p>
</div>
<div id="ref-18GflyMj">
<p>53. <strong>Emerging coronaviruses: Genome structure, replication, and pathogenesis</strong> <br />
Yu Chen, Qianyun Liu, Deyin Guo<br />
<em>Journal of Medical Virology</em> (2020-02-07) <a href="https://doi.org/ggjvwj">https://doi.org/ggjvwj</a> <br />
DOI: <a href="https://doi.org/10.1002/jmv.25681">10.1002/jmv.25681</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31967327">31967327</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7167049">PMC7167049</a></p>
</div>
<div id="ref-FFwktXgx">
<p>54. <strong>SARS coronavirus entry into host cells through a novel clathrin- and caveolae-independent endocytic pathway</strong> <br />
Hongliang Wang, Peng Yang, Kangtai Liu, Feng Guo, Yanli Zhang, Gongyi Zhang, Chengyu Jiang<br />
<em>Cell Research</em> (2008-01-29) <a href="https://doi.org/bp9275">https://doi.org/bp9275</a> <br />
DOI: <a href="https://doi.org/10.1038/cr.2008.15">10.1038/cr.2008.15</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/18227861">18227861</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7091891">PMC7091891</a></p>
</div>
<div id="ref-FNetQkRI">
<p>55. <strong>Virus Entry by Endocytosis</strong> <br />
Jason Mercer, Mario Schelhaas, Ari Helenius<br />
<em>Annual Review of Biochemistry</em> (2010-06-07) <a href="https://doi.org/cw4dnb">https://doi.org/cw4dnb</a> <br />
DOI: <a href="https://doi.org/10.1146/annurev-biochem-060208-104626">10.1146/annurev-biochem-060208-104626</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20196649">20196649</a></p>
</div>
<div id="ref-13wWdgODZ">
<p>56. <strong>Mechanisms of Coronavirus Cell Entry Mediated by the Viral Spike Protein</strong> <br />
Sandrine Belouzard, Jean K. Millet, Beth N. Licitra, Gary R. Whittaker<br />
<em>Viruses</em> (2012-06-20) <a href="https://doi.org/gbbktb">https://doi.org/gbbktb</a> <br />
DOI: <a href="https://doi.org/10.3390/v4061011">10.3390/v4061011</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22816037">22816037</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3397359">PMC3397359</a></p>
</div>
<div id="ref-OVsxrEuX">
<p>57. <strong>Phylogenetic Analysis and Structural Modeling of SARS-CoV-2 Spike Protein Reveals an Evolutionary Distinct and Proteolytically Sensitive Activation Loop</strong> <br />
Javier A. Jaimes, Nicole M. André, Joshua S. Chappie, Jean K. Millet, Gary R. Whittaker<br />
<em>Journal of Molecular Biology</em> (2020-05) <a href="https://doi.org/ggtxhr">https://doi.org/ggtxhr</a> <br />
DOI: <a href="https://doi.org/10.1016/j.jmb.2020.04.009">10.1016/j.jmb.2020.04.009</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32320687">32320687</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7166309">PMC7166309</a></p>
</div>
<div id="ref-So6ceUwB">
<p>58. <strong>Angiotensin-converting enzyme 2 (ACE2) as a SARS-CoV-2 receptor: molecular mechanisms and potential therapeutic target</strong> <br />
Haibo Zhang, Josef M. Penninger, Yimin Li, Nanshan Zhong, Arthur S. Slutsky<br />
<em>Intensive Care Medicine</em> (2020-03-03) <a href="https://doi.org/ggpx6p">https://doi.org/ggpx6p</a> <br />
DOI: <a href="https://doi.org/10.1007/s00134-020-05985-9">10.1007/s00134-020-05985-9</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32125455">32125455</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7079879">PMC7079879</a></p>
</div>
<div id="ref-NodtSApl">
<p>59. <strong>Infection of Human Airway Epithelia by Sars Coronavirus is Associated with ACE2 Expression and Localization</strong> <br />
Hong Peng Jia, Dwight C. Look, Melissa Hickey, Lei Shi, Lecia Pewe, Jason Netland, Michael Farzan, Christine Wohlford-Lenane, Stanley Perlman, Paul B. McCray<br />
<em>Advances in Experimental Medicine and Biology</em> (2006) <a href="https://doi.org/dhh5tp">https://doi.org/dhh5tp</a> <br />
DOI: <a href="https://doi.org/10.1007/978-0-387-33012-9_85">10.1007/978-0-387-33012-9_85</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17037581">17037581</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7123641">PMC7123641</a></p>
</div>
<div id="ref-skHKTFMe">
<p>60. <strong>Receptor Recognition Mechanisms of Coronaviruses: a Decade of Structural Studies</strong> <br />
Fang Li<br />
<em>Journal of Virology</em> (2015-02-15) <a href="https://doi.org/f633jb">https://doi.org/f633jb</a> <br />
DOI: <a href="https://doi.org/10.1128/jvi.02615-14">10.1128/jvi.02615-14</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25428871">25428871</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4338876">PMC4338876</a></p>
</div>
<div id="ref-ueMPuUMe">
<p>61. <strong>The spike protein of SARS-CoV — a target for vaccine and therapeutic development</strong> <br />
Lanying Du, Yuxian He, Yusen Zhou, Shuwen Liu, Bo-Jian Zheng, Shibo Jiang<br />
<em>Nature Reviews Microbiology</em> (2009-02-09) <a href="https://doi.org/d4tq4t">https://doi.org/d4tq4t</a> <br />
DOI: <a href="https://doi.org/10.1038/nrmicro2090">10.1038/nrmicro2090</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19198616">19198616</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2750777">PMC2750777</a></p>
</div>
<div id="ref-qcVbT0w4">
<p>62. <strong>Structure, Function, and Antigenicity of the SARS-CoV-2 Spike Glycoprotein</strong> <br />
Alexandra C. Walls, Young-Jun Park, M. Alejandra Tortorici, Abigail Wall, Andrew T. McGuire, David Veesler<br />
<em>Cell</em> (2020-03) <a href="https://doi.org/dpvh">https://doi.org/dpvh</a> <br />
DOI: <a href="https://doi.org/10.1016/j.cell.2020.02.058">10.1016/j.cell.2020.02.058</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32155444">32155444</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7102599">PMC7102599</a></p>
</div>
<div id="ref-kCVy8vjj">
<p>63. <strong>Molecular Interactions in the Assembly of Coronaviruses</strong> <br />
Cornelis A. M. de Haan, Peter J. M. Rottier<br />
<em>Advances in Virus Research</em> (2005) <a href="https://doi.org/cf8chz">https://doi.org/cf8chz</a> <br />
DOI: <a href="https://doi.org/10.1016/s0065-3527(05)64006-7">10.1016/s0065-3527(05)64006-7</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16139595">16139595</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7112327">PMC7112327</a></p>
</div>
<div id="ref-YeKaMzdJ">
<p>64. <strong>Coronavirus membrane fusion mechanism offers a potential target for antiviral development</strong> <br />
Tiffany Tang, Miya Bidon, Javier A. Jaimes, Gary R. Whittaker, Susan Daniel<br />
<em>Antiviral Research</em> (2020-06) <a href="https://doi.org/ggr23b">https://doi.org/ggr23b</a> <br />
DOI: <a href="https://doi.org/10.1016/j.antiviral.2020.104792">10.1016/j.antiviral.2020.104792</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32272173">32272173</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7194977">PMC7194977</a></p>
</div>
<div id="ref-15l3di3Wj">
<p>65. <strong>SARS-CoV-2 Cell Entry Depends on ACE2 and TMPRSS2 and Is Blocked by a Clinically Proven Protease Inhibitor</strong> <br />
Markus Hoffmann, Hannah Kleine-Weber, Simon Schroeder, Nadine Krüger, Tanja Herrler, Sandra Erichsen, Tobias S. Schiergens, Georg Herrler, Nai-Huei Wu, Andreas Nitsche, … Stefan Pöhlmann<br />
<em>Cell</em> (2020-03) <a href="https://doi.org/ggnq74">https://doi.org/ggnq74</a> <br />
DOI: <a href="https://doi.org/10.1016/j.cell.2020.02.052">10.1016/j.cell.2020.02.052</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32142651">32142651</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7102627">PMC7102627</a></p>
</div>
<div id="ref-JOJ2n3gC">
<p>66. <strong>Characterization of spike glycoprotein of SARS-CoV-2 on virus entry and its immune cross-reactivity with SARS-CoV</strong> <br />
Xiuyuan Ou, Yan Liu, Xiaobo Lei, Pei Li, Dan Mi, Lili Ren, Li Guo, Ruixuan Guo, Ting Chen, Jiaxin Hu, … Zhaohui Qian<br />
<em>Nature Communications</em> (2020-03-27) <a href="https://doi.org/ggqsrf">https://doi.org/ggqsrf</a> <br />
DOI: <a href="https://doi.org/10.1038/s41467-020-15562-9">10.1038/s41467-020-15562-9</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32221306">32221306</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7100515">PMC7100515</a></p>
</div>
<div id="ref-ZUuGRxk9">
<p>67. <strong>Coronaviridae ~ ViralZone page</strong> <a href="https://viralzone.expasy.org/30?outline">https://viralzone.expasy.org/30?outline</a></p>
</div>
<div id="ref-1XbF4GLn">
<p>68. <strong>Spatiotemporal interplay of severe acute respiratory syndrome coronavirus and respiratory mucosal cells drives viral dissemination in rhesus macaques</strong> <br />
L Liu, Q Wei, K Nishiura, J Peng, H Wang, C Midkiff, X Alvarez, C Qin, A Lackner, Z Chen<br />
<em>Mucosal Immunology</em> (2015-12-09) <a href="https://doi.org/f8r7dk">https://doi.org/f8r7dk</a> <br />
DOI: <a href="https://doi.org/10.1038/mi.2015.127">10.1038/mi.2015.127</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26647718">26647718</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4900951">PMC4900951</a></p>
</div>
<div id="ref-159GOVLV1">
<p>69. <strong>Understanding Viral dsRNA-Mediated Innate Immune Responses at the Cellular Level Using a Rainbow Trout Model</strong> <br />
Sarah J. Poynter, Stephanie J. DeWitte-Orr<br />
<em>Frontiers in Immunology</em> (2018-04-23) <a href="https://doi.org/gdhpbs">https://doi.org/gdhpbs</a> <br />
DOI: <a href="https://doi.org/10.3389/fimmu.2018.00829">10.3389/fimmu.2018.00829</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29740439">29740439</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5924774">PMC5924774</a></p>
</div>
<div id="ref-yQO4HFTZ">
<p>70. <strong>Ultrastructure and Origin of Membrane Vesicles Associated with the Severe Acute Respiratory Syndrome Coronavirus Replication Complex</strong> <br />
Eric J. Snijder, Yvonne van der Meer, Jessika Zevenhoven-Dobbe, Jos J. M. Onderwater, Jannes van der Meulen, Henk K. Koerten, A. Mieke Mommaas<br />
<em>Journal of Virology</em> (2006-06-15) <a href="https://doi.org/b2rh4r">https://doi.org/b2rh4r</a> <br />
DOI: <a href="https://doi.org/10.1128/jvi.02501-05">10.1128/jvi.02501-05</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16731931">16731931</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1472606">PMC1472606</a></p>
</div>
<div id="ref-Q5Q9k6Nc">
<p>71. <strong>Structural basis for translational shutdown and immune evasion by the Nsp1 protein of SARS-CoV-2</strong> <br />
Matthias Thoms, Robert Buschauer, Michael Ameismeier, Lennart Koepke, Timo Denk, Maximilian Hirschenberger, Hanna Kratzat, Manuel Hayn, Timur Mackens-Kiani, Jingdong Cheng, … Roland Beckmann<br />
<em>Science</em> (2020-07-17) <a href="https://doi.org/gg69nq">https://doi.org/gg69nq</a> <br />
DOI: <a href="https://doi.org/10.1126/science.abc8665">10.1126/science.abc8665</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32680882">32680882</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7402621">PMC7402621</a></p>
</div>
<div id="ref-G6A0wWdn">
<p>72. <strong>Immune responses in COVID-19 and potential vaccines: Lessons learned from SARS and MERS epidemic</strong> <br />
Asian Pacific Journal of Allergy and Immunology<br />
(2020) <a href="https://doi.org/ggpvxw">https://doi.org/ggpvxw</a> <br />
DOI: <a href="https://doi.org/10.12932/ap-200220-0772">10.12932/ap-200220-0772</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32105090">32105090</a></p>
</div>
<div id="ref-2QTH37Xi">
<p>73. <strong>Angiotensin-converting enzyme 2 is a functional receptor for the SARS coronavirus</strong> <br />
Wenhui Li, Michael J. Moore, Natalya Vasilieva, Jianhua Sui, Swee Kee Wong, Michael A. Berne, Mohan Somasundaran, John L. Sullivan, Katherine Luzuriaga, Thomas C. Greenough, … Michael Farzan<br />
<em>Nature</em> (2003-11) <a href="https://doi.org/bqvpjh">https://doi.org/bqvpjh</a> <br />
DOI: <a href="https://doi.org/10.1038/nature02145">10.1038/nature02145</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/14647384">14647384</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7095016">PMC7095016</a></p>
</div>
<div id="ref-MXWaV7SO">
<p>74. <strong>Efficient Activation of the Severe Acute Respiratory Syndrome Coronavirus Spike Protein by the Transmembrane Protease TMPRSS2</strong> <br />
S. Matsuyama, N. Nagata, K. Shirato, M. Kawase, M. Takeda, F. Taguchi<br />
<em>Journal of Virology</em> (2010-10-06) <a href="https://doi.org/d4hnfr">https://doi.org/d4hnfr</a> <br />
DOI: <a href="https://doi.org/10.1128/jvi.01542-10">10.1128/jvi.01542-10</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20926566">20926566</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3004351">PMC3004351</a></p>
</div>
<div id="ref-JjrO6YbD">
<p>75. <strong>Evidence that TMPRSS2 Activates the Severe Acute Respiratory Syndrome Coronavirus Spike Protein for Membrane Fusion and Reduces Viral Control by the Humoral Immune Response</strong> <br />
I. Glowacka, S. Bertram, M. A. Muller, P. Allen, E. Soilleux, S. Pfefferle, I. Steffen, T. S. Tsegaye, Y. He, K. Gnirss, … S. Pohlmann<br />
<em>Journal of Virology</em> (2011-02-16) <a href="https://doi.org/bg97wb">https://doi.org/bg97wb</a> <br />
DOI: <a href="https://doi.org/10.1128/jvi.02232-10">10.1128/jvi.02232-10</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21325420">21325420</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3126222">PMC3126222</a></p>
</div>
<div id="ref-UwVweB2M">
<p>76. <strong>Increasing Host Cellular Receptor—Angiotensin-Converting Enzyme 2 (ACE2) Expression by Coronavirus may Facilitate 2019-nCoV Infection</strong> <br />
Pei-Hui Wang, Yun Cheng<br />
<em>bioRxiv</em> (2020-02-27) <a href="https://doi.org/ggscwd">https://doi.org/ggscwd</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.02.24.963348">10.1101/2020.02.24.963348</a></p>
</div>
<div id="ref-vr7AH83b">
<p>77. <strong>Detection of SARS-CoV-2 in Different Types of Clinical Specimens</strong> <br />
Wenling Wang, Yanli Xu, Ruqin Gao, Roujian Lu, Kai Han, Guizhen Wu, Wenjie Tan<br />
<em>JAMA</em> (2020-03-11) <a href="https://doi.org/ggpp6h">https://doi.org/ggpp6h</a> <br />
DOI: <a href="https://doi.org/10.1001/jama.2020.3786">10.1001/jama.2020.3786</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32159775">32159775</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7066521">PMC7066521</a></p>
</div>
<div id="ref-azgJqujy">
<p>78. <strong>Epidemiologic Features and Clinical Course of Patients Infected With SARS-CoV-2 in Singapore</strong> <br />
Barnaby Edward Young, Sean Wei Xiang Ong, Shirin Kalimuddin, Jenny G. Low, Seow Yen Tan, Jiashen Loh, Oon-Tek Ng, Kalisvar Marimuthu, Li Wei Ang, Tze Minn Mak, … for the Singapore 2019 Novel Coronavirus Outbreak Research Team<br />
<em>JAMA</em> (2020-03-03) <a href="https://doi.org/ggnb37">https://doi.org/ggnb37</a> <br />
DOI: <a href="https://doi.org/10.1001/jama.2020.3204">10.1001/jama.2020.3204</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32125362">32125362</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7054855">PMC7054855</a></p>
</div>
<div id="ref-F7nSMvZk">
<p>79. <strong>Persistence and clearance of viral RNA in 2019 novel coronavirus disease rehabilitation patients</strong> <br />
Yun Ling, Shui-Bao Xu, Yi-Xiao Lin, Di Tian, Zhao-Qin Zhu, Fa-Hui Dai, Fan Wu, Zhi-Gang Song, Wei Huang, Jun Chen, … Hong-Zhou Lu<br />
<em>Chinese Medical Journal</em> (2020-02) <a href="https://doi.org/ggnnz8">https://doi.org/ggnnz8</a> <br />
DOI: <a href="https://doi.org/10.1097/cm9.0000000000000774">10.1097/cm9.0000000000000774</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32118639">32118639</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7147278">PMC7147278</a></p>
</div>
<div id="ref-1AJFJxzmJ">
<p>80. <strong>Postmortem examination of COVID‐19 patients reveals diffuse alveolar damage with severe capillary congestion and variegated findings in lungs and other organs suggesting vascular dysfunction</strong> <br />
Thomas Menter, Jasmin D Haslbauer, Ronny Nienhold, Spasenija Savic, Helmut Hopfer, Nikolaus Deigendesch, Stephan Frank, Daniel Turek, Niels Willi, Hans Pargger, … Alexandar Tzankov<br />
<em>Histopathology</em> (2020-07-05) <a href="https://doi.org/ggwr32">https://doi.org/ggwr32</a> <br />
DOI: <a href="https://doi.org/10.1111/his.14134">10.1111/his.14134</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32364264">32364264</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7496150">PMC7496150</a></p>
</div>
<div id="ref-5gnCuPzp">
<p>81. <strong>Association of Cardiac Injury With Mortality in Hospitalized Patients With COVID-19 in Wuhan, China</strong> <br />
Shaobo Shi, Mu Qin, Bo Shen, Yuli Cai, Tao Liu, Fan Yang, Wei Gong, Xu Liu, Jinjun Liang, Qinyan Zhao, … Congxin Huang<br />
<em>JAMA Cardiology</em> (2020-03-25) <a href="https://doi.org/ggq8qf">https://doi.org/ggq8qf</a> <br />
DOI: <a href="https://doi.org/10.1001/jamacardio.2020.0950">10.1001/jamacardio.2020.0950</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32211816">32211816</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7097841">PMC7097841</a></p>
</div>
<div id="ref-5ET1D3cK">
<p>82. <strong>The need for urogenital tract monitoring in COVID-19</strong> <br />
Shangqian Wang, Xiang Zhou, Tongtong Zhang, Zengjun Wang<br />
<em>Nature Reviews Urology</em> (2020-04-20) <a href="https://doi.org/ggv4xb">https://doi.org/ggv4xb</a> <br />
DOI: <a href="https://doi.org/10.1038/s41585-020-0319-7">10.1038/s41585-020-0319-7</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32313110">32313110</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7186932">PMC7186932</a></p>
</div>
<div id="ref-w5B6qRKv">
<p>83. <strong>Acute kidney injury in SARS-CoV-2 infected patients</strong> <br />
Vito Fanelli, Marco Fiorentino, Vincenzo Cantaluppi, Loreto Gesualdo, Giovanni Stallone, Claudio Ronco, Giuseppe Castellano<br />
<em>Critical Care</em> (2020-04-16) <a href="https://doi.org/ggv45f">https://doi.org/ggv45f</a> <br />
DOI: <a href="https://doi.org/10.1186/s13054-020-02872-z">10.1186/s13054-020-02872-z</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32299479">32299479</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7161433">PMC7161433</a></p>
</div>
<div id="ref-TTSFlLVC">
<p>84. <strong>Liver injury in COVID-19: management and challenges</strong> <br />
Chao Zhang, Lei Shi, Fu-Sheng Wang<br />
<em>The Lancet Gastroenterology &amp; Hepatology</em> (2020-03) <a href="https://doi.org/ggpx6s">https://doi.org/ggpx6s</a> <br />
DOI: <a href="https://doi.org/10.1016/s2468-1253(20)30057-1">10.1016/s2468-1253(20)30057-1</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32145190">32145190</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7129165">PMC7129165</a></p>
</div>
<div id="ref-PmE9xedP">
<p>85. <strong>Evidence for gastrointestinal infection of SARS-CoV-2</strong> <br />
Fei Xiao, Meiwen Tang, Xiaobin Zheng, Ye Liu, Xiaofeng Li, Hong Shan<br />
<em>Gastroenterology</em> (2020-03) <a href="https://doi.org/ggpx27">https://doi.org/ggpx27</a> <br />
DOI: <a href="https://doi.org/10.1053/j.gastro.2020.02.055">10.1053/j.gastro.2020.02.055</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32142773">32142773</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7130181">PMC7130181</a></p>
</div>
<div id="ref-3Ak4Mata">
<p>86. <strong>2019 Novel coronavirus infection and gastrointestinal tract</strong> <br />
Qin Yan Gao, Ying Xuan Chen, Jing Yuan Fang<br />
<em>Journal of Digestive Diseases</em> (2020-03-10) <a href="https://doi.org/ggqr86">https://doi.org/ggqr86</a> <br />
DOI: <a href="https://doi.org/10.1111/1751-2980.12851">10.1111/1751-2980.12851</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32096611">32096611</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7162053">PMC7162053</a></p>
</div>
<div id="ref-18tT1tLJI">
<p>87. <strong>Symptom Profiles of a Convenience Sample of Patients with COVID-19 — United States, January–April 2020</strong> <br />
Rachel M. Burke, Marie E. Killerby, Suzanne Newton, Candace E. Ashworth, Abby L. Berns, Skyler Brennan, Jonathan M. Bressler, Erica Bye, Richard Crawford, Laurel Harduar Morano, … Case Investigation Form Working Group<br />
<em>MMWR. Morbidity and Mortality Weekly Report</em> (2020-07-17) <a href="https://doi.org/gg8r2m">https://doi.org/gg8r2m</a> <br />
DOI: <a href="https://doi.org/10.15585/mmwr.mm6928a2">10.15585/mmwr.mm6928a2</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32673296">32673296</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7366851">PMC7366851</a></p>
</div>
<div id="ref-19IBzQBhB">
<p>88. <strong>Population-scale Longitudinal Mapping of COVID-19 Symptoms, Behavior, and Testing Identifies Contributors to Continued Disease Spread in the United States</strong> <br />
William E Allen, Han Altae-Tran, James Briggs, Xin Jin, Glen McGee, Rumya Raghavan, Andy Shi, Mireille Kamariza, Nicole Nova, Albert Pereta, … Xihong Lin<br />
<em>Cold Spring Harbor Laboratory</em> (2020-06-11) <a href="https://doi.org/gg8skn">https://doi.org/gg8skn</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.06.09.20126813">10.1101/2020.06.09.20126813</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32577674">32577674</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7302230">PMC7302230</a></p>
</div>
<div id="ref-wK2afyL8">
<p>89. <strong>Extrapulmonary manifestations of COVID-19</strong> <br />
Aakriti Gupta, Mahesh V. Madhavan, Kartik Sehgal, Nandini Nair, Shiwani Mahajan, Tejasav S. Sehrawat, Behnood Bikdeli, Neha Ahluwalia, John C. Ausiello, Elaine Y. Wan, … Donald W. Landry<br />
<em>Nature Medicine</em> (2020-07-10) <a href="https://doi.org/gg4r37">https://doi.org/gg4r37</a> <br />
DOI: <a href="https://doi.org/10.1038/s41591-020-0968-3">10.1038/s41591-020-0968-3</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32651579">32651579</a></p>
</div>
<div id="ref-17hdoiBo9">
<p>90. <strong>Acute kidney injury in patients hospitalized with COVID-19</strong> <br />
Jamie S. Hirsch, Jia H. Ng, Daniel W. Ross, Purva Sharma, Hitesh H. Shah, Richard L. Barnett, Azzour D. Hazzan, Steven Fishbane, Kenar D. Jhaveri, Mersema Abate, … Jia Hwei. Ng<br />
<em>Kidney International</em> (2020-07) <a href="https://doi.org/ggx24k">https://doi.org/ggx24k</a> <br />
DOI: <a href="https://doi.org/10.1016/j.kint.2020.05.006">10.1016/j.kint.2020.05.006</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32416116">32416116</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7229463">PMC7229463</a></p>
</div>
<div id="ref-1CqoZGSKK">
<p>91. <strong>Nervous system involvement after infection with COVID-19 and other coronaviruses</strong> <br />
Yeshun Wu, Xiaolin Xu, Zijun Chen, Jiahao Duan, Kenji Hashimoto, Ling Yang, Cunming Liu, Chun Yang<br />
<em>Brain, Behavior, and Immunity</em> (2020-07) <a href="https://doi.org/ggq7s2">https://doi.org/ggq7s2</a> <br />
DOI: <a href="https://doi.org/10.1016/j.bbi.2020.03.031">10.1016/j.bbi.2020.03.031</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32240762">32240762</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7146689">PMC7146689</a></p>
</div>
<div id="ref-1EF9EpbZ5">
<p>92. <strong>Neurological associations of COVID-19</strong> <br />
Mark A Ellul, Laura Benjamin, Bhagteshwar Singh, Suzannah Lant, Benedict Daniel Michael, Ava Easton, Rachel Kneen, Sylviane Defres, Jim Sejvar, Tom Solomon<br />
<em>The Lancet Neurology</em> (2020-09) <a href="https://doi.org/d259">https://doi.org/d259</a> <br />
DOI: <a href="https://doi.org/10.1016/s1474-4422(20)30221-0">10.1016/s1474-4422(20)30221-0</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32622375">32622375</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7332267">PMC7332267</a></p>
</div>
<div id="ref-nBGGUV9s">
<p>93. <strong>Update on the neurology of COVID‐19</strong> <br />
Josef Finsterer, Claudia Stollberger<br />
<em>Journal of Medical Virology</em> (2020-06-02) <a href="https://doi.org/gg2qnn">https://doi.org/gg2qnn</a> <br />
DOI: <a href="https://doi.org/10.1002/jmv.26000">10.1002/jmv.26000</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32401352">32401352</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7272942">PMC7272942</a></p>
</div>
<div id="ref-LLm9d62d">
<p>94. <strong>‐19: A Global Threat to the Nervous System</strong> <br />
Igor J. Koralnik, Kenneth L. Tyler<br />
<em>Annals of Neurology</em> (2020-06-23) <a href="https://doi.org/gg3hzh">https://doi.org/gg3hzh</a> <br />
DOI: <a href="https://doi.org/10.1002/ana.25807">10.1002/ana.25807</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32506549">32506549</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7300753">PMC7300753</a></p>
</div>
<div id="ref-aRNqjkMg">
<p>95. <strong>Large-Vessel Stroke as a Presenting Feature of Covid-19 in the Young</strong> <br />
Thomas J. Oxley, J. Mocco, Shahram Majidi, Christopher P. Kellner, Hazem Shoirah, I. Paul Singh, Reade A. De Leacy, Tomoyoshi Shigematsu, Travis R. Ladner, Kurt A. Yaeger, … Johanna T. Fifi<br />
<em>New England Journal of Medicine</em> (2020-05-14) <a href="https://doi.org/ggtsjg">https://doi.org/ggtsjg</a> <br />
DOI: <a href="https://doi.org/10.1056/nejmc2009787">10.1056/nejmc2009787</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32343504">32343504</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7207073">PMC7207073</a></p>
</div>
<div id="ref-3UBbDrG2">
<p>96. <strong>Incidence of thrombotic complications in critically ill ICU patients with COVID-19</strong> <br />
F. A. Klok, M. J. H. A. Kruip, N. J. M. van der Meer, M. S. Arbous, D. A. M. P. J. Gommers, K. M. Kant, F. H. J. Kaptein, J. van Paassen, M. A. M. Stals, M. V. Huisman, H. Endeman<br />
<em>Thrombosis Research</em> (2020-07) <a href="https://doi.org/dt2q">https://doi.org/dt2q</a> <br />
DOI: <a href="https://doi.org/10.1016/j.thromres.2020.04.013">10.1016/j.thromres.2020.04.013</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32291094">32291094</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7146714">PMC7146714</a></p>
</div>
<div id="ref-8OnbWuhF">
<p>97. <strong>Coagulopathy and Antiphospholipid Antibodies in Patients with Covid-19</strong> <br />
Yan Zhang, Meng Xiao, Shulan Zhang, Peng Xia, Wei Cao, Wei Jiang, Huan Chen, Xin Ding, Hua Zhao, Hongmin Zhang, … Shuyang Zhang<br />
<em>New England Journal of Medicine</em> (2020-04-23) <a href="https://doi.org/ggrgz7">https://doi.org/ggrgz7</a> <br />
DOI: <a href="https://doi.org/10.1056/nejmc2007575">10.1056/nejmc2007575</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32268022">32268022</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7161262">PMC7161262</a></p>
</div>
<div id="ref-dUfws1q0">
<p>98. <strong>Abnormal coagulation parameters are associated with poor prognosis in patients with novel coronavirus pneumonia</strong> <br />
Ning Tang, Dengju Li, Xiong Wang, Ziyong Sun<br />
<em>Journal of Thrombosis and Haemostasis</em> (2020-04) <a href="https://doi.org/ggqxf6">https://doi.org/ggqxf6</a> <br />
DOI: <a href="https://doi.org/10.1111/jth.14768">10.1111/jth.14768</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32073213">32073213</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7166509">PMC7166509</a></p>
</div>
<div id="ref-p5EhN9Qd">
<p>99. <strong>Review: Viral infections and mechanisms of thrombosis and bleeding</strong> <br />
M. Goeijenbier, M. van Wissen, C. van de Weg, E. Jong, V. E. A. Gerdes, J. C. M. Meijers, D. P. M. Brandjes, E. C. M. van Gorp<br />
<em>Journal of Medical Virology</em> (2012-10) <a href="https://doi.org/f37tfr">https://doi.org/f37tfr</a> <br />
DOI: <a href="https://doi.org/10.1002/jmv.23354">10.1002/jmv.23354</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22930518">22930518</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7166625">PMC7166625</a></p>
</div>
<div id="ref-v1EIzwfx">
<p>100. <strong>Immune mechanisms of pulmonary intravascular coagulopathy in COVID-19 pneumonia</strong> <br />
Dennis McGonagle, James S O’Donnell, Kassem Sharif, Paul Emery, Charles Bridgewood<br />
<em>The Lancet Rheumatology</em> (2020-07) <a href="https://doi.org/ggvd74">https://doi.org/ggvd74</a> <br />
DOI: <a href="https://doi.org/10.1016/s2665-9913(20)30121-1">10.1016/s2665-9913(20)30121-1</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32835247">32835247</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7252093">PMC7252093</a></p>
</div>
<div id="ref-18AiyvhO8">
<p>101. <strong>“War to the knife” against thromboinflammation to protect endothelial function of COVID-19 patients</strong> <br />
Gabriele Guglielmetti, Marco Quaglia, Pier Paolo Sainaghi, Luigi Mario Castello, Rosanna Vaschetto, Mario Pirisi, Francesco Della Corte, Gian Carlo Avanzi, Piero Stratta, Vincenzo Cantaluppi<br />
<em>Critical Care</em> (2020-06-19) <a href="https://doi.org/gg35w7">https://doi.org/gg35w7</a> <br />
DOI: <a href="https://doi.org/10.1186/s13054-020-03060-9">10.1186/s13054-020-03060-9</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32560665">32560665</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7303575">PMC7303575</a></p>
</div>
<div id="ref-PyKMLraw">
<p>102. <strong>COVID-19 update: Covid-19-associated coagulopathy</strong> <br />
Richard C. Becker<br />
<em>Journal of Thrombosis and Thrombolysis</em> (2020-05-15) <a href="https://doi.org/ggwpp5">https://doi.org/ggwpp5</a> <br />
DOI: <a href="https://doi.org/10.1007/s11239-020-02134-3">10.1007/s11239-020-02134-3</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32415579">32415579</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7225095">PMC7225095</a></p>
</div>
<div id="ref-14dyYQY7s">
<p>103. <strong>The complement system in COVID-19: friend and foe?</strong> <br />
Anuja Java, Anthony J. Apicelli, M. Kathryn Liszewski, Ariella Coler-Reilly, John P. Atkinson, Alfred H. J. Kim, Hrishikesh S. Kulkarni<br />
<em>JCI Insight</em> (2020-08-06) <a href="https://doi.org/gg4b5b">https://doi.org/gg4b5b</a> <br />
DOI: <a href="https://doi.org/10.1172/jci.insight.140711">10.1172/jci.insight.140711</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32554923">32554923</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7455060">PMC7455060</a></p>
</div>
<div id="ref-17y6YeJ6R">
<p>104. <strong>COVID-19, microangiopathy, hemostatic activation, and complement</strong> <br />
Wen-Chao Song, Garret A. FitzGerald<br />
<em>Journal of Clinical Investigation</em> (2020-06-22) <a href="https://doi.org/gg4b5c">https://doi.org/gg4b5c</a> <br />
DOI: <a href="https://doi.org/10.1172/jci140183">10.1172/jci140183</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32459663">32459663</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7410042">PMC7410042</a></p>
</div>
<div id="ref-o6BQnEt7">
<p>105. <strong>Molecular biology of the cell</strong> <br />
Bruce Alberts (editor)<br />
<em>Garland Science</em> (2002) <br />
ISBN: <a href="https://worldcat.org/isbn/9780815332183">9780815332183</a></p>
</div>
<div id="ref-b1QVdemU">
<p>106. <strong>Vander’s human physiology: the mechanisms of body function</strong> <br />
Eric P Widmaier, Hershel Raff, Kevin T Strang<br />
<em>McGraw-Hill Higher Education</em> (2008) <br />
ISBN: <a href="https://worldcat.org/isbn/9780071283663">9780071283663</a></p>
</div>
<div id="ref-1GnFL9zeN">
<p>107. <strong>The Innate Immune System: Fighting on the Front Lines or Fanning the Flames of COVID-19?</strong> <br />
Julia L. McKechnie, Catherine A. Blish<br />
<em>Cell Host &amp; Microbe</em> (2020-06) <a href="https://doi.org/gg28pq">https://doi.org/gg28pq</a> <br />
DOI: <a href="https://doi.org/10.1016/j.chom.2020.05.009">10.1016/j.chom.2020.05.009</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32464098">32464098</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7237895">PMC7237895</a></p>
</div>
<div id="ref-sXusUkLI">
<p>108. <strong>Inflammatory responses and inflammation-associated diseases in organs</strong> <br />
Linlin Chen, Huidan Deng, Hengmin Cui, Jing Fang, Zhicai Zuo, Junliang Deng, Yinglun Li, Xun Wang, Ling Zhao<br />
<em>Oncotarget</em> (2017-12-14) <a href="https://doi.org/ggps2p">https://doi.org/ggps2p</a> <br />
DOI: <a href="https://doi.org/10.18632/oncotarget.23208">10.18632/oncotarget.23208</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29467962">29467962</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5805548">PMC5805548</a></p>
</div>
<div id="ref-bZMKqj6e">
<p>109. <strong>Into the Eye of the Cytokine Storm</strong> <br />
J. R. Tisoncik, M. J. Korth, C. P. Simmons, J. Farrar, T. R. Martin, M. G. Katze<br />
<em>Microbiology and Molecular Biology Reviews</em> (2012-03-05) <a href="https://doi.org/f4n9h2">https://doi.org/f4n9h2</a> <br />
DOI: <a href="https://doi.org/10.1128/mmbr.05015-11">10.1128/mmbr.05015-11</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22390970">22390970</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3294426">PMC3294426</a></p>
</div>
<div id="ref-DYbswZ6D">
<p>110. <strong>Cytokines, Inflammation, and Pain</strong> <br />
Jun-Ming Zhang, Jianxiong An<br />
<em>International Anesthesiology Clinics</em> (2007-Spring) <a href="https://doi.org/ft5b7h">https://doi.org/ft5b7h</a> <br />
DOI: <a href="https://doi.org/10.1097/aia.0b013e318034194e">10.1097/aia.0b013e318034194e</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17426506">17426506</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2785020">PMC2785020</a></p>
</div>
<div id="ref-16CYY7vzG">
<p>111. <strong>Dynamic balance of pro- and anti-inflammatory signals controls disease and limits pathology</strong> <br />
Joseph M. Cicchese, Stephanie Evans, Caitlin Hult, Louis R. Joslyn, Timothy Wessler, Jess A. Millar, Simeone Marino, Nicholas A. Cilfone, Joshua T. Mattila, Jennifer J. Linderman, Denise E. Kirschner<br />
<em>Immunological Reviews</em> (2018-09) <a href="https://doi.org/gd4g4p">https://doi.org/gd4g4p</a> <br />
DOI: <a href="https://doi.org/10.1111/imr.12671">10.1111/imr.12671</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30129209">30129209</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6292442">PMC6292442</a></p>
</div>
<div id="ref-xuMYmc7W">
<p>112. <strong>Cytokine Dysregulation, Inflammation and Well-Being</strong> <br />
Ilia J. Elenkov, Domenic G. Iezzoni, Adrian Daly, Alan G. Harris, George P. Chrousos<br />
<em>Neuroimmunomodulation</em> (2005) <a href="https://doi.org/bsn7kn">https://doi.org/bsn7kn</a> <br />
DOI: <a href="https://doi.org/10.1159/000087104">10.1159/000087104</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16166805">16166805</a></p>
</div>
<div id="ref-OvhepUrA">
<p>113. <strong>Chronic inflammation in the etiology of disease across the life span</strong> <br />
David Furman, Judith Campisi, Eric Verdin, Pedro Carrera-Bastos, Sasha Targ, Claudio Franceschi, Luigi Ferrucci, Derek W. Gilroy, Alessio Fasano, Gary W. Miller, … George M. Slavich<br />
<em>Nature Medicine</em> (2019-12-05) <a href="https://doi.org/gghdcf">https://doi.org/gghdcf</a> <br />
DOI: <a href="https://doi.org/10.1038/s41591-019-0675-0">10.1038/s41591-019-0675-0</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31806905">31806905</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7147972">PMC7147972</a></p>
</div>
<div id="ref-6eWgCrWj">
<p>114. <strong>Inpatient care for septicemia or sepsis: a challenge for patients and hospitals.</strong> <br />
Margaret Jean Hall, Sonja N Williams, Carol J DeFrances, Aleksandr Golosinskiy<br />
<em>NCHS data brief</em> (2011-06) <a href="https://www.ncbi.nlm.nih.gov/pubmed/22142805">https://www.ncbi.nlm.nih.gov/pubmed/22142805</a> <br />
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22142805">22142805</a></p>
</div>
<div id="ref-f61jsRKY">
<p>115. <strong>Cytokine Balance in the Lungs of Patients with Acute Respiratory Distress Syndrome</strong> <br />
WILLIAM Y. PARK, RICHARD B. GOODMAN, KENNETH P. STEINBERG, JOHN T. RUZINSKI, FRANK RADELLA, DAVID R. PARK, JEROME PUGIN, SHAWN J. SKERRETT, LEONARD D. HUDSON, THOMAS R. MARTIN<br />
<em>American Journal of Respiratory and Critical Care Medicine</em> (2001-11-15) <a href="https://doi.org/ggqfq7">https://doi.org/ggqfq7</a> <br />
DOI: <a href="https://doi.org/10.1164/ajrccm.164.10.2104013">10.1164/ajrccm.164.10.2104013</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/11734443">11734443</a></p>
</div>
<div id="ref-NITOJ0Ka">
<p>116. <strong>Cytokine release syndrome</strong> <br />
Alexander Shimabukuro-Vornhagen, Philipp Gödel, Marion Subklewe, Hans Joachim Stemmler, Hans Anton Schlößer, Max Schlaak, Matthias Kochanek, Boris Böll, Michael S. von Bergwelt-Baildon<br />
<em>Journal for ImmunoTherapy of Cancer</em> (2018-06-15) <a href="https://doi.org/ghbncj">https://doi.org/ghbncj</a> <br />
DOI: <a href="https://doi.org/10.1186/s40425-018-0343-9">10.1186/s40425-018-0343-9</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29907163">29907163</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6003181">PMC6003181</a></p>
</div>
<div id="ref-oSOG8N0f">
<p>117. <strong>Understanding the Inflammatory Cytokine Response in Pneumonia and Sepsis</strong> <br />
John A. Kellum<br />
<em>Archives of Internal Medicine</em> (2007-08-13) <a href="https://doi.org/dbxb66">https://doi.org/dbxb66</a> <br />
DOI: <a href="https://doi.org/10.1001/archinte.167.15.1655">10.1001/archinte.167.15.1655</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17698689">17698689</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4495652">PMC4495652</a></p>
</div>
<div id="ref-dBAe8aYi">
<p>118. <strong>The Role of Interleukin 6 During Viral Infections</strong> <br />
Lauro Velazquez-Salinas, Antonio Verdugo-Rodriguez, Luis L. Rodriguez, Manuel V. Borca<br />
<em>Frontiers in Microbiology</em> (2019-05-10) <a href="https://doi.org/ghbnck">https://doi.org/ghbnck</a> <br />
DOI: <a href="https://doi.org/10.3389/fmicb.2019.01057">10.3389/fmicb.2019.01057</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31134045">31134045</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6524401">PMC6524401</a></p>
</div>
<div id="ref-NNFGje9g">
<p>119. <strong>Expression of elevated levels of pro-inflammatory cytokines in SARS-CoV-infected ACE2 <sup>+</sup> cells in SARS patients: relation to the acute lung injury and pathogenesis of SARS</strong> <br />
L He, Y Ding, Q Zhang, X Che, Y He, H Shen, H Wang, Z Li, L Zhao, J Geng, … S Jiang<br />
<em>The Journal of Pathology</em> (2006-11) <a href="https://doi.org/bwb8ns">https://doi.org/bwb8ns</a> <br />
DOI: <a href="https://doi.org/10.1002/path.2067">10.1002/path.2067</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17031779">17031779</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7167655">PMC7167655</a></p>
</div>
<div id="ref-QkTGQUcj">
<p>120. <strong>Up-regulation of IL-6 and TNF-α induced by SARS-coronavirus spike protein in murine macrophages via NF-κB pathway</strong> <br />
Wei Wang, Linbai Ye, Li Ye, Baozong Li, Bo Gao, Yingchun Zeng, Lingbao Kong, Xiaonan Fang, Hong Zheng, Zhenghui Wu, Yinglong She<br />
<em>Virus Research</em> (2007-09) <a href="https://doi.org/bm7m55">https://doi.org/bm7m55</a> <br />
DOI: <a href="https://doi.org/10.1016/j.virusres.2007.02.007">10.1016/j.virusres.2007.02.007</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17532082">17532082</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7114322">PMC7114322</a></p>
</div>
<div id="ref-3HdlV9Vf">
<p>121. <strong>COVID-19: consider cytokine storm syndromes and immunosuppression</strong> <br />
Puja Mehta, Daniel F McAuley, Michael Brown, Emilie Sanchez, Rachel S Tattersall, Jessica J Manson<br />
<em>The Lancet</em> (2020-03) <a href="https://doi.org/ggnzmc">https://doi.org/ggnzmc</a> <br />
DOI: <a href="https://doi.org/10.1016/s0140-6736(20)30628-0">10.1016/s0140-6736(20)30628-0</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32192578">32192578</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7270045">PMC7270045</a></p>
</div>
<div id="ref-cyUG1Zi2">
<p>122. <strong>Cytokine Storms: Understanding COVID-19</strong> <br />
Nilam Mangalmurti, Christopher A. Hunter<br />
<em>Immunity</em> (2020-07) <a href="https://doi.org/gg4fd7">https://doi.org/gg4fd7</a> <br />
DOI: <a href="https://doi.org/10.1016/j.immuni.2020.06.017">10.1016/j.immuni.2020.06.017</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32610079">32610079</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7321048">PMC7321048</a></p>
</div>
<div id="ref-Xcrxlxnp">
<p>123. <strong>Is a “Cytokine Storm” Relevant to COVID-19?</strong> <br />
Pratik Sinha, Michael A. Matthay, Carolyn S. Calfee<br />
<em>JAMA Internal Medicine</em> (2020-09-01) <a href="https://doi.org/gg3k6r">https://doi.org/gg3k6r</a> <br />
DOI: <a href="https://doi.org/10.1001/jamainternmed.2020.3313">10.1001/jamainternmed.2020.3313</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32602883">32602883</a></p>
</div>
<div id="ref-RIMWgsWt">
<p>124. <strong>Can we use interleukin-6 (IL-6) blockade for coronavirus disease 2019 (COVID-19)-induced cytokine release syndrome (CRS)?</strong> <br />
Bingwen Liu, Min Li, Zhiguang Zhou, Xuan Guan, Yufei Xiang<br />
<em>Journal of Autoimmunity</em> (2020-07) <a href="https://doi.org/ggr79c">https://doi.org/ggr79c</a> <br />
DOI: <a href="https://doi.org/10.1016/j.jaut.2020.102452">10.1016/j.jaut.2020.102452</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32291137">32291137</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7151347">PMC7151347</a></p>
</div>
<div id="ref-1BhN2uq3f">
<p>125. <strong>Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) Infection in Children and Adolescents</strong> <br />
Riccardo Castagnoli, Martina Votto, Amelia Licari, Ilaria Brambilla, Raffaele Bruno, Stefano Perlini, Francesca Rovida, Fausto Baldanti, Gian Luigi Marseglia<br />
<em>JAMA Pediatrics</em> (2020-09-01) <a href="https://doi.org/dswz">https://doi.org/dswz</a> <br />
DOI: <a href="https://doi.org/10.1001/jamapediatrics.2020.1467">10.1001/jamapediatrics.2020.1467</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32320004">32320004</a></p>
</div>
<div id="ref-12lVfu2Qe">
<p>126. <strong>Neurologic and Radiographic Findings Associated With COVID-19 Infection in Children</strong> <br />
Omar Abdel-Mannan, Michael Eyre, Ulrike Löbel, Alasdair Bamford, Christin Eltze, Biju Hameed, Cheryl Hemingway, Yael Hacohen<br />
<em>JAMA Neurology</em> (2020-11-01) <a href="https://doi.org/gg339f">https://doi.org/gg339f</a> <br />
DOI: <a href="https://doi.org/10.1001/jamaneurol.2020.2687">10.1001/jamaneurol.2020.2687</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32609336">32609336</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7330822">PMC7330822</a></p>
</div>
<div id="ref-MCVe25tf">
<p>127. <strong>COVID-19 in 7780 pediatric patients: A systematic review</strong> <br />
Ansel Hoang, Kevin Chorath, Axel Moreira, Mary Evans, Finn Burmeister-Morton, Fiona Burmeister, Rija Naqvi, Matthew Petershack, Alvaro Moreira<br />
<em>EClinicalMedicine</em> (2020-07) <a href="https://doi.org/gg4hn2">https://doi.org/gg4hn2</a> <br />
DOI: <a href="https://doi.org/10.1016/j.eclinm.2020.100433">10.1016/j.eclinm.2020.100433</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32766542">32766542</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7318942">PMC7318942</a></p>
</div>
<div id="ref-fZwrv8M">
<p>128. <strong>Multisystem Inflammatory Syndrome in Children During the Coronavirus 2019 Pandemic: A Case Series</strong> <br />
Kathleen Chiotos, Hamid Bassiri, Edward M Behrens, Allison M Blatz, Joyce Chang, Caroline Diorio, Julie C Fitzgerald, Alexis Topjian, Audrey R Odom John<br />
<em>Journal of the Pediatric Infectious Diseases Society</em> (2020-07) <a href="https://doi.org/ggx4pd">https://doi.org/ggx4pd</a> <br />
DOI: <a href="https://doi.org/10.1093/jpids/piaa069">10.1093/jpids/piaa069</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32463092">32463092</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7313950">PMC7313950</a></p>
</div>
<div id="ref-AmvwCWm3">
<p>129. <strong>Clinical Characteristics of 58 Children With a Pediatric Inflammatory Multisystem Syndrome Temporally Associated With SARS-CoV-2</strong> <br />
Elizabeth Whittaker, Alasdair Bamford, Julia Kenny, Myrsini Kaforou, Christine E. Jones, Priyen Shah, Padmanabhan Ramnarayan, Alain Fraisse, Owen Miller, Patrick Davies, … for the PIMS-TS Study Group and EUCLIDS and PERFORM Consortia<br />
<em>JAMA</em> (2020-07-21) <a href="https://doi.org/gg2v75">https://doi.org/gg2v75</a> <br />
DOI: <a href="https://doi.org/10.1001/jama.2020.10369">10.1001/jama.2020.10369</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32511692">32511692</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7281356">PMC7281356</a></p>
</div>
<div id="ref-68JorBeU">
<p>130. <strong>Toxic shock-like syndrome and COVID-19: A case report of multisystem inflammatory syndrome in children (MIS-C)</strong> <br />
Andrea G. Greene, Mona Saleh, Eric Roseman, Richard Sinert<br />
<em>The American Journal of Emergency Medicine</em> (2020-06) <a href="https://doi.org/gg2586">https://doi.org/gg2586</a> <br />
DOI: <a href="https://doi.org/10.1016/j.ajem.2020.05.117">10.1016/j.ajem.2020.05.117</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32532619">32532619</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7274960">PMC7274960</a></p>
</div>
<div id="ref-10gSXfOKm">
<p>131. <strong>Multisystem inflammatory syndrome in children and COVID-19 are distinct presentations of SARS-CoV-2</strong> <br />
Caroline Diorio, Sarah E. Henrickson, Laura A. Vella, Kevin O. McNerney, Julie M. Chase, Chakkapong Burudpakdee, Jessica H. Lee, Cristina Jasen, Fran Balamuth, David M. Barrett, … Hamid Bassiri<br />
<em>Journal of Clinical Investigation</em> (2020-07-30) <a href="https://doi.org/gg7mz2">https://doi.org/gg7mz2</a> <br />
DOI: <a href="https://doi.org/10.1172/jci140970">10.1172/jci140970</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32730233">32730233</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7598044">PMC7598044</a></p>
</div>
<div id="ref-10QhaDu6N">
<p>132. <strong>The Immunology of Multisystem Inflammatory Syndrome in Children with COVID-19</strong> <br />
Camila Rosat Consiglio, Nicola Cotugno, Fabian Sardh, Christian Pou, Donato Amodio, Lucie Rodriguez, Ziyang Tan, Sonia Zicari, Alessandra Ruggiero, Giuseppe Rubens Pascucci, … Petter Brodin<br />
<em>Cell</em> (2020-11) <a href="https://doi.org/d8fh">https://doi.org/d8fh</a> <br />
DOI: <a href="https://doi.org/10.1016/j.cell.2020.09.016">10.1016/j.cell.2020.09.016</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32966765">32966765</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7474869">PMC7474869</a></p>
</div>
<div id="ref-FYgN6gUL">
<p>133. <strong>Acute Heart Failure in Multisystem Inflammatory Syndrome in Children in the Context of Global SARS-CoV-2 Pandemic</strong> <br />
Zahra Belhadjer, Mathilde Méot, Fanny Bajolle, Diala Khraiche, Antoine Legendre, Samya Abakka, Johanne Auriau, Marion Grimaud, Mehdi Oualha, Maurice Beghetti, … Damien Bonnet<br />
<em>Circulation</em> (2020-08-04) <a href="https://doi.org/ggwkv6">https://doi.org/ggwkv6</a> <br />
DOI: <a href="https://doi.org/10.1161/circulationaha.120.048360">10.1161/circulationaha.120.048360</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32418446">32418446</a></p>
</div>
<div id="ref-71To4laE">
<p>134. <strong>An adult with Kawasaki-like multisystem inflammatory syndrome associated with COVID-19</strong> <br />
Sheila Shaigany, Marlis Gnirke, Allison Guttmann, Hong Chong, Shane Meehan, Vanessa Raabe, Eddie Louie, Bruce Solitar, Alisa Femia<br />
<em>The Lancet</em> (2020-07) <a href="https://doi.org/gg4sd6">https://doi.org/gg4sd6</a> <br />
DOI: <a href="https://doi.org/10.1016/s0140-6736(20)31526-9">10.1016/s0140-6736(20)31526-9</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32659211">32659211</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7351414">PMC7351414</a></p>
</div>
<div id="ref-UBDAxWWv">
<p>135. <strong>COVID-19 associated Kawasaki-like multisystem inflammatory disease in an adult</strong> <br />
Sabrina Sokolovsky, Parita Soni, Taryn Hoffman, Philip Kahn, Joshua Scheers-Masters<br />
<em>The American Journal of Emergency Medicine</em> (2020-06) <a href="https://doi.org/gg5tf4">https://doi.org/gg5tf4</a> <br />
DOI: <a href="https://doi.org/10.1016/j.ajem.2020.06.053">10.1016/j.ajem.2020.06.053</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32631771">32631771</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7315983">PMC7315983</a></p>
</div>
<div id="ref-5MhhHQO0">
<p>136. <strong>Systems Approaches to Biology and Disease Enable Translational Systems Medicine</strong> <br />
Leroy Hood, Qiang Tian<br />
<em>Genomics, Proteomics &amp; Bioinformatics</em> (2012-08) <a href="https://doi.org/f4f599">https://doi.org/f4f599</a> <br />
DOI: <a href="https://doi.org/10.1016/j.gpb.2012.08.004">10.1016/j.gpb.2012.08.004</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23084773">23084773</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3844613">PMC3844613</a></p>
</div>
<div id="ref-8esIIWt9">
<p>137. <strong>A systems approach to infectious disease</strong> <br />
Manon Eckhardt, Judd F. Hultquist, Robyn M. Kaake, Ruth Hüttenhain, Nevan J. Krogan<br />
<em>Nature Reviews Genetics</em> (2020-02-14) <a href="https://doi.org/ggnv63">https://doi.org/ggnv63</a> <br />
DOI: <a href="https://doi.org/10.1038/s41576-020-0212-5">10.1038/s41576-020-0212-5</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32060427">32060427</a></p>
</div>
<div id="ref-8vUHj9cm">
<p>138. <strong>Differential expression of serum/plasma proteins in various infectious diseases: Specific or nonspecific signatures</strong> <br />
Sandipan Ray, Sandip K. Patel, Vipin Kumar, Jagruti Damahe, Sanjeeva Srivastava<br />
<em>PROTEOMICS - Clinical Applications</em> (2014-02) <a href="https://doi.org/f2px3h">https://doi.org/f2px3h</a> <br />
DOI: <a href="https://doi.org/10.1002/prca.201300074">10.1002/prca.201300074</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24293340">24293340</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7168033">PMC7168033</a></p>
</div>
<div id="ref-pHrstqMQ">
<p>139. <strong>Imbalanced Host Response to SARS-CoV-2 Drives Development of COVID-19</strong> <br />
Daniel Blanco-Melo, Benjamin E. Nilsson-Payant, Wen-Chun Liu, Skyler Uhl, Daisy Hoagland, Rasmus Møller, Tristan X. Jordan, Kohei Oishi, Maryline Panis, David Sachs, … Benjamin R. tenOever<br />
<em>Cell</em> (2020-05) <a href="https://doi.org/ggw5tq">https://doi.org/ggw5tq</a> <br />
DOI: <a href="https://doi.org/10.1016/j.cell.2020.04.026">10.1016/j.cell.2020.04.026</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32416070">32416070</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7227586">PMC7227586</a></p>
</div>
<div id="ref-PJgYPcKM">
<p>140. <strong>SARS-CoV-2 ORF3b is a potent interferon antagonist whose activity is further increased by a naturally occurring elongation variant</strong> <br />
Yoriyuki Konno, Izumi Kimura, Keiya Uriu, Masaya Fukushi, Takashi Irie, Yoshio Koyanagi, So Nakagawa, Kei Sato<br />
<em>Cold Spring Harbor Laboratory</em> (2020-05-11) <a href="https://doi.org/gg2qgs">https://doi.org/gg2qgs</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.05.11.088179">10.1101/2020.05.11.088179</a></p>
</div>
<div id="ref-Gj8vlc0W">
<p>141. <strong>Bulk and single-cell gene expression profiling of SARS-CoV-2 infected human cell lines identifies molecular targets for therapeutic intervention</strong> <br />
Wyler Emanuel, Mösbauer Kirstin, Franke Vedran, Diag Asija, Gottula Lina Theresa, Arsie Roberto, Klironomos Filippos, Koppstein David, Ayoub Salah, Buccitelli Christopher, … Landthaler Markus<br />
<em>Cold Spring Harbor Laboratory</em> (2020-05-05) <a href="https://doi.org/ggxd2g">https://doi.org/ggxd2g</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.05.05.079194">10.1101/2020.05.05.079194</a></p>
</div>
<div id="ref-16EFyBURq">
<p>142. <strong>Isolation and characterization of SARS-CoV-2 from the first US COVID-19 patient</strong> <br />
Jennifer Harcourt, Azaibi Tamin, Xiaoyan Lu, Shifaq Kamili, Senthil Kumar. Sakthivel, Janna Murray, Krista Queen, Ying Tao, Clinton R. Paden, Jing Zhang, … Natalie J. Thornburg<br />
<em>Cold Spring Harbor Laboratory</em> (2020-03-07) <a href="https://doi.org/gg2fkm">https://doi.org/gg2fkm</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.02.972935">10.1101/2020.03.02.972935</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32511316">32511316</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7239045">PMC7239045</a></p>
</div>
<div id="ref-11xZWeHN3">
<p>143. <strong>Proteomics of SARS-CoV-2-infected host cells reveals therapy targets</strong> <br />
Denisa Bojkova, Kevin Klann, Benjamin Koch, Marek Widera, David Krause, Sandra Ciesek, Jindrich Cinatl, Christian Münch<br />
<em>Nature</em> (2020-05-14) <a href="https://doi.org/dw7s">https://doi.org/dw7s</a> <br />
DOI: <a href="https://doi.org/10.1038/s41586-020-2332-7">10.1038/s41586-020-2332-7</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32408336">32408336</a></p>
</div>
<div id="ref-GhJYjnft">
<p>144. <strong>Potent human neutralizing antibodies elicited by SARS-CoV-2 infection</strong> <br />
Bin Ju, Qi Zhang, Xiangyang Ge, Ruoke Wang, Jiazhen Yu, Sisi Shan, Bing Zhou, Shuo Song, Xian Tang, Jinfang Yu, … Linqi Zhang<br />
<em>bioRxiv</em> (2020-03-26) <a href="https://doi.org/ggp7t4">https://doi.org/ggp7t4</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.21.990770">10.1101/2020.03.21.990770</a></p>
</div>
<div id="ref-GpnngtWK">
<p>145. <strong>Plasma proteome of severe acute respiratory syndrome analyzed by two-dimensional gel electrophoresis and mass spectrometry</strong> <br />
J.-H. Chen, Y.-W. Chang, C.-W. Yao, T.-S. Chiueh, S.-C. Huang, K.-Y. Chien, A. Chen, F.-Y. Chang, C.-H. Wong, Y.-J. Chen<br />
<em>Proceedings of the National Academy of Sciences</em> (2004-11-30) <a href="https://doi.org/dtv8sx">https://doi.org/dtv8sx</a> <br />
DOI: <a href="https://doi.org/10.1073/pnas.0407992101">10.1073/pnas.0407992101</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15572443">15572443</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC535397">PMC535397</a></p>
</div>
<div id="ref-JLTf2Fwb">
<p>146. <strong>Analysis of multimerization of the SARS coronavirus nucleocapsid protein</strong> <br />
Runtao He, Frederick Dobie, Melissa Ballantine, Andrew Leeson, Yan Li, Nathalie Bastien, Todd Cutts, Anton Andonov, Jingxin Cao, Timothy F. Booth, … Xuguang Li<br />
<em>Biochemical and Biophysical Research Communications</em> (2004-04) <a href="https://doi.org/dbfwr9">https://doi.org/dbfwr9</a> <br />
DOI: <a href="https://doi.org/10.1016/j.bbrc.2004.02.074">10.1016/j.bbrc.2004.02.074</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15020242">15020242</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7111152">PMC7111152</a></p>
</div>
<div id="ref-phJM8g2Y">
<p>147. <strong>A SARS-CoV-2-Human Protein-Protein Interaction Map Reveals Drug Targets and Potential Drug-Repurposing</strong> <br />
David E. Gordon, Gwendolyn M. Jang, Mehdi Bouhaddou, Jiewei Xu, Kirsten Obernier, Matthew J. O’Meara, Jeffrey Z. Guo, Danielle L. Swaney, Tia A. Tummino, Ruth Hüttenhain, … Nevan J. Krogan<br />
<em>bioRxiv</em> (2020-03-22) <a href="https://doi.org/ggpptg">https://doi.org/ggpptg</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.22.002386">10.1101/2020.03.22.002386</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32511329">32511329</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7239059">PMC7239059</a></p>
</div>
<div id="ref-mXUCjmCh">
<p>148. <strong>Protein Palmitoylation and Its Role in Bacterial and Viral Infections</strong> <br />
Justyna Sobocińska, Paula Roszczenko-Jasińska, Anna Ciesielska, Katarzyna Kwiatkowska<br />
<em>Frontiers in Immunology</em> (2018-01-19) <a href="https://doi.org/gcxpp2">https://doi.org/gcxpp2</a> <br />
DOI: <a href="https://doi.org/10.3389/fimmu.2017.02003">10.3389/fimmu.2017.02003</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29403483">29403483</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5780409">PMC5780409</a></p>
</div>
<div id="ref-NLoN4aYj">
<p>149. <strong>Virus-host interactome and proteomic survey of PMBCs from COVID-19 patients reveal potential virulence factors influencing SARS-CoV-2 pathogenesis</strong> <br />
Jingjiao Li, Mingquan Guo, Xiaoxu Tian, Chengrong Liu, Xin Wang, Xing Yang, Ping Wu, Zixuan Xiao, Yafei Qu, Yue Yin, … Qiming Liang<br />
<em>Cold Spring Harbor Laboratory</em> (2020-04-02) <a href="https://doi.org/ggrgbv">https://doi.org/ggrgbv</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.31.019216">10.1101/2020.03.31.019216</a></p>
</div>
<div id="ref-KAqOiTeZ">
<p>150. <strong>The Nuclear Factor NF- B Pathway in Inflammation</strong> <br />
T. Lawrence<br />
<em>Cold Spring Harbor Perspectives in Biology</em> (2009-10-07) <a href="https://doi.org/fptfvp">https://doi.org/fptfvp</a> <br />
DOI: <a href="https://doi.org/10.1101/cshperspect.a001651">10.1101/cshperspect.a001651</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20457564">20457564</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2882124">PMC2882124</a></p>
</div>
<div id="ref-1ccnm0N9">
<p>151. <strong>The severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) envelope (E) protein harbors a conserved BH3-like sequence</strong> <br />
Vincent Navratil, Loïc Lionnard, Sonia Longhi, J. Marie Hardwick, Christophe Combet, Abdel Aouacheria<br />
<em>Cold Spring Harbor Laboratory</em> (2020-06-09) <a href="https://doi.org/ggrp43">https://doi.org/ggrp43</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.04.09.033522">10.1101/2020.04.09.033522</a></p>
</div>
<div id="ref-15EIBRhef">
<p>152. <strong>Structure of SARS Coronavirus Spike Receptor-Binding Domain Complexed with Receptor</strong> <br />
F. Li<br />
<em>Science</em> (2005-09-16) <a href="https://doi.org/fww324">https://doi.org/fww324</a> <br />
DOI: <a href="https://doi.org/10.1126/science.1116480">10.1126/science.1116480</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16166518">16166518</a></p>
</div>
<div id="ref-12TZ7hPMA">
<p>153. <strong>Structural basis for the recognition of SARS-CoV-2 by full-length human ACE2</strong> <br />
Renhong Yan, Yuanyuan Zhang, Yaning Li, Lu Xia, Yingying Guo, Qiang Zhou<br />
<em>Science</em> (2020-03-27) <a href="https://doi.org/ggpxc8">https://doi.org/ggpxc8</a> <br />
DOI: <a href="https://doi.org/10.1126/science.abb2762">10.1126/science.abb2762</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32132184">32132184</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7164635">PMC7164635</a></p>
</div>
<div id="ref-wCbhn23d">
<p>154. <strong>Structural basis of receptor recognition by SARS-CoV-2</strong> <br />
Jian Shang, Gang Ye, Ke Shi, Yushun Wan, Chuming Luo, Hideki Aihara, Qibin Geng, Ashley Auerbach, Fang Li<br />
<em>Nature</em> (2020-03-30) <a href="https://doi.org/ggqspv">https://doi.org/ggqspv</a> <br />
DOI: <a href="https://doi.org/10.1038/s41586-020-2179-y">10.1038/s41586-020-2179-y</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32225175">32225175</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7328981">PMC7328981</a></p>
</div>
<div id="ref-VX8OWaGj">
<p>155. <strong>Crystal structure of the 2019-nCoV spike receptor-binding domain bound with the ACE2 receptor</strong> <br />
Jun Lan, Jiwan Ge, Jinfang Yu, Sisi Shan, Huan Zhou, Shilong Fan, Qi Zhang, Xuanling Shi, Qisheng Wang, Linqi Zhang, Xinquan Wang<br />
<em>bioRxiv</em> (2020-02-20) <a href="https://doi.org/ggqzp5">https://doi.org/ggqzp5</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.02.19.956235">10.1101/2020.02.19.956235</a></p>
</div>
<div id="ref-15Q2XgkK7">
<p>156. <strong>Structural and Functional Basis of SARS-CoV-2 Entry by Using Human ACE2</strong> <br />
Qihui Wang, Yanfang Zhang, Lili Wu, Sheng Niu, Chunli Song, Zengyuan Zhang, Guangwen Lu, Chengpeng Qiao, Yu Hu, Kwok-Yung Yuen, … Jianxun Qi<br />
<em>Cell</em> (2020-05) <a href="https://doi.org/ggr2cz">https://doi.org/ggr2cz</a> <br />
DOI: <a href="https://doi.org/10.1016/j.cell.2020.03.045">10.1016/j.cell.2020.03.045</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32275855">32275855</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7144619">PMC7144619</a></p>
</div>
<div id="ref-t1e4CW9A">
<p>157. <strong>Receptor recognition by novel coronavirus from Wuhan: An analysis based on decade-long structural studies of SARS</strong> <br />
Yushun Wan, Jian Shang, Rachel Graham, Ralph S. Baric, Fang Li<br />
<em>Journal of Virology</em> (2020-01-29) <a href="https://doi.org/ggjvwn">https://doi.org/ggjvwn</a> <br />
DOI: <a href="https://doi.org/10.1128/jvi.00127-20">10.1128/jvi.00127-20</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31996437">31996437</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7081895">PMC7081895</a></p>
</div>
<div id="ref-NsORsLig">
<p>158. <strong>Structure of the Hemagglutinin Precursor Cleavage Site, a Determinant of Influenza Pathogenicity and the Origin of the Labile Conformation</strong> <br />
Jue Chen, Kon Ho Lee, David A Steinhauer, David J Stevens, John J Skehel, Don C Wiley<br />
<em>Cell</em> (1998-10) <a href="https://doi.org/bvgh5b">https://doi.org/bvgh5b</a> <br />
DOI: <a href="https://doi.org/10.1016/s0092-8674(00)81771-7">10.1016/s0092-8674(00)81771-7</a></p>
</div>
<div id="ref-vhHB3yyS">
<p>159. <strong>Role of Hemagglutinin Cleavage for the Pathogenicity of Influenza Virus</strong> <br />
David A. Steinhauer<br />
<em>Virology</em> (1999-05) <a href="https://doi.org/fw3jz4">https://doi.org/fw3jz4</a> <br />
DOI: <a href="https://doi.org/10.1006/viro.1999.9716">10.1006/viro.1999.9716</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/10329563">10329563</a></p>
</div>
<div id="ref-p9LUiyCN">
<p>160. <strong>Emerging SARS-CoV-2 mutation hot spots include a novel RNA-dependent-RNA polymerase variant</strong> <br />
Maria Pachetti, Bruna Marini, Francesca Benedetti, Fabiola Giudici, Elisabetta Mauro, Paola Storici, Claudio Masciovecchio, Silvia Angeletti, Massimo Ciccozzi, Robert C. Gallo, … Rudy Ippodrino<br />
<em>Journal of Translational Medicine</em> (2020-04-22) <a href="https://doi.org/ggtzrr">https://doi.org/ggtzrr</a> <br />
DOI: <a href="https://doi.org/10.1186/s12967-020-02344-6">10.1186/s12967-020-02344-6</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32321524">32321524</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7174922">PMC7174922</a></p>
</div>
<div id="ref-2w7lNKxQ">
<p>161. <strong>Emergence of genomic diversity and recurrent mutations in SARS-CoV-2</strong> <br />
Lucy van Dorp, Mislav Acman, Damien Richard, Liam P. Shaw, Charlotte E. Ford, Louise Ormond, Christopher J. Owen, Juanita Pang, Cedric C. S. Tan, Florencia A. T. Boshier, … François Balloux<br />
<em>Infection, Genetics and Evolution</em> (2020-09) <a href="https://doi.org/ggvz4h">https://doi.org/ggvz4h</a> <br />
DOI: <a href="https://doi.org/10.1016/j.meegid.2020.104351">10.1016/j.meegid.2020.104351</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32387564">32387564</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7199730">PMC7199730</a></p>
</div>
<div id="ref-RIpPhJ1g">
<p>162. <strong>Tracking changes in SARS-CoV-2 Spike: evidence that D614G increases infectivity of the COVID-19 virus</strong> <br />
B. Korber, W. M. Fischer, S. Gnanakaran, H. Yoon, J. Theiler, W. Abfalterer, N. Hengartner, E. E. Giorgi, T. Bhattacharya, B. Foley, … Matthew D. Wyles<br />
<em>Cell</em> (2020-07) <a href="https://doi.org/gg3wqn">https://doi.org/gg3wqn</a> <br />
DOI: <a href="https://doi.org/10.1016/j.cell.2020.06.043">10.1016/j.cell.2020.06.043</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32697968">32697968</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7332439">PMC7332439</a></p>
</div>
<div id="ref-DyG1eyK4">
<p>163. <strong>Making Sense of Mutation: What D614G Means for the COVID-19 Pandemic Remains Unclear</strong> <br />
Nathan D. Grubaugh, William P. Hanage, Angela L. Rasmussen<br />
<em>Cell</em> (2020-08) <a href="https://doi.org/gg4gqt">https://doi.org/gg4gqt</a> <br />
DOI: <a href="https://doi.org/10.1016/j.cell.2020.06.040">10.1016/j.cell.2020.06.040</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32697970">32697970</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7332445">PMC7332445</a></p>
</div>
<div id="ref-CJbohYmK">
<p>164. <strong>Monogenic, polygenic, and oligogenic familial hypercholesterolemia</strong> <br />
Hayato Tada, Atsushi Nohara, Masa-aki Kawashiri<br />
<em>Current Opinion in Lipidology</em> (2019-08) <a href="https://doi.org/ggbwdb">https://doi.org/ggbwdb</a> <br />
DOI: <a href="https://doi.org/10.1097/qco.0000000000000563">10.1097/qco.0000000000000563</a></p>
</div>
<div id="ref-KMb6knfE">
<p>165. <strong>Viral infections acquired indoors through airborne, droplet or contact transmission.</strong> <br />
Giuseppina La Rosa, Marta Fratini, Simonetta Della Libera, Marcello Iaconelli, Michele Muscillo<br />
<em>Annali dell’Istituto superiore di sanita</em> (2013) <a href="https://www.ncbi.nlm.nih.gov/pubmed/23771256">https://www.ncbi.nlm.nih.gov/pubmed/23771256</a> <br />
DOI: <a href="https://doi.org/10.4415/ann_13_02_03">10.4415/ann_13_02_03</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23771256">23771256</a></p>
</div>
<div id="ref-1FLLN4PSl">
<p>166. <strong>Transmission of SARS-CoV-2: implications for infection prevention precautions</strong> <a href="https://www.who.int/news-room/commentaries/detail/transmission-of-sars-cov-2-implications-for-infection-prevention-precautions">https://www.who.int/news-room/commentaries/detail/transmission-of-sars-cov-2-implications-for-infection-prevention-precautions</a></p>
</div>
<div id="ref-AC4okoVf">
<p>167. <strong>Questioning Aerosol Transmission of Influenza</strong> <br />
Camille Lemieux, Gabrielle Brankston, Leah Gitterman, Zahir Hirji, Michael Gardam<br />
<em>Emerging Infectious Diseases</em> (2007-01) <a href="https://doi.org/c2skj8">https://doi.org/c2skj8</a> <br />
DOI: <a href="https://doi.org/10.3201/eid1301.061202">10.3201/eid1301.061202</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17370541">17370541</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2725811">PMC2725811</a></p>
</div>
<div id="ref-Hct9jRcb">
<p>168. <strong>Assessing the Dynamics and Control of Droplet- and Aerosol-Transmitted Influenza Using an Indoor Positioning System</strong> <br />
Timo Smieszek, Gianrocco Lazzari, Marcel Salathé<br />
<em>Scientific Reports</em> (2019-02-18) <a href="https://doi.org/ggnqbc">https://doi.org/ggnqbc</a> <br />
DOI: <a href="https://doi.org/10.1038/s41598-019-38825-y">10.1038/s41598-019-38825-y</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30778136">30778136</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6379436">PMC6379436</a></p>
</div>
<div id="ref-i6tTpqwA">
<p>169. <strong>Influenza A virus transmission via respiratory aerosols or droplets as it relates to pandemic potential</strong> <br />
Mathilde Richard, Ron A. M. Fouchier<br />
<em>FEMS Microbiology Reviews</em> (2016-01) <a href="https://doi.org/f8cp4h">https://doi.org/f8cp4h</a> <br />
DOI: <a href="https://doi.org/10.1093/femsre/fuv039">10.1093/femsre/fuv039</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26385895">26385895</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5006288">PMC5006288</a></p>
</div>
<div id="ref-rjVw7V94">
<p>170. <strong>Coronavirus Pathogenesis</strong> <br />
Susan R. Weiss, Julian L. Leibowitz<br />
<em>Advances in Virus Research</em> (2011) <a href="https://doi.org/ggvvd7">https://doi.org/ggvvd7</a> <br />
DOI: <a href="https://doi.org/10.1016/b978-0-12-385885-6.00009-2">10.1016/b978-0-12-385885-6.00009-2</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22094080">22094080</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7149603">PMC7149603</a></p>
</div>
<div id="ref-yR57NFIB">
<p>171. <strong>Role of fomites in SARS transmission during the largest hospital outbreak in Hong Kong</strong> <br />
Shenglan Xiao, Yuguo Li, Tze-wai Wong, David S. C. Hui<br />
<em>PLOS ONE</em> (2017-07-20) <a href="https://doi.org/gbpgv7">https://doi.org/gbpgv7</a> <br />
DOI: <a href="https://doi.org/10.1371/journal.pone.0181558">10.1371/journal.pone.0181558</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28727803">28727803</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5519164">PMC5519164</a></p>
</div>
<div id="ref-s3oVNbGV">
<p>172. <strong>Stability of Middle East respiratory syndrome coronavirus (MERS-CoV) under different environmental conditions</strong> <br />
N van Doremalen, T Bushmaker, V Munster<br />
<em>Eurosurveillance</em> (2013-09-19) <a href="https://doi.org/ggnnjt">https://doi.org/ggnnjt</a> <br />
DOI: <a href="https://doi.org/10.2807/1560-7917.es2013.18.38.20590">10.2807/1560-7917.es2013.18.38.20590</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24084338">24084338</a></p>
</div>
<div id="ref-1FjDLPCye">
<p>173. <strong>MERS coronavirus: diagnostics, epidemiology and transmission</strong> <br />
Ian M. Mackay, Katherine E. Arden<br />
<em>Virology Journal</em> (2015-12-22) <a href="https://doi.org/f745px">https://doi.org/f745px</a> <br />
DOI: <a href="https://doi.org/10.1186/s12985-015-0439-5">10.1186/s12985-015-0439-5</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26695637">26695637</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4687373">PMC4687373</a></p>
</div>
<div id="ref-vP6yTZ0y">
<p>174. <strong>Transmission routes of 2019-nCoV and controls in dental practice</strong> <br />
Xian Peng, Xin Xu, Yuqing Li, Lei Cheng, Xuedong Zhou, Biao Ren<br />
<em>International Journal of Oral Science</em> (2020-03-03) <a href="https://doi.org/ggnf47">https://doi.org/ggnf47</a> <br />
DOI: <a href="https://doi.org/10.1038/s41368-020-0075-9">10.1038/s41368-020-0075-9</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32127517">32127517</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7054527">PMC7054527</a></p>
</div>
<div id="ref-16SDeiudC">
<p>175. <strong>Reducing transmission of SARS-CoV-2</strong> <br />
Kimberly A. Prather, Chia C. Wang, Robert T. Schooley<br />
<em>Science</em> (2020-06-26) <a href="https://doi.org/ggxp9w">https://doi.org/ggxp9w</a> <br />
DOI: <a href="https://doi.org/10.1126/science.abc6197">10.1126/science.abc6197</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32461212">32461212</a></p>
</div>
<div id="ref-yXKcviw8">
<p>176. <strong>Aerosol and Surface Stability of SARS-CoV-2 as Compared with SARS-CoV-1</strong> <br />
Neeltje van Doremalen, Trenton Bushmaker, Dylan H. Morris, Myndi G. Holbrook, Amandine Gamble, Brandi N. Williamson, Azaibi Tamin, Jennifer L. Harcourt, Natalie J. Thornburg, Susan I. Gerber, … Vincent J. Munster<br />
<em>New England Journal of Medicine</em> (2020-03-17) <a href="https://doi.org/ggn88w">https://doi.org/ggn88w</a> <br />
DOI: <a href="https://doi.org/10.1056/nejmc2004973">10.1056/nejmc2004973</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32182409">32182409</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7121658">PMC7121658</a></p>
</div>
<div id="ref-9ouDX5IN">
<p>177. <strong>It Is Time to Address Airborne Transmission of Coronavirus Disease 2019 (COVID-19)</strong> <br />
Lidia Morawska, Donald K Milton<br />
<em>Clinical Infectious Diseases</em> (2020-07-06) <a href="https://doi.org/gg34zn">https://doi.org/gg34zn</a> <br />
DOI: <a href="https://doi.org/10.1093/cid/ciaa939">10.1093/cid/ciaa939</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32628269">32628269</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7454469">PMC7454469</a></p>
</div>
<div id="ref-bQLMgMGC">
<p>178. <strong>Aerodynamic analysis of SARS-CoV-2 in two Wuhan hospitals</strong> <br />
Yuan Liu, Zhi Ning, Yu Chen, Ming Guo, Yingle Liu, Nirmal Kumar Gali, Li Sun, Yusen Duan, Jing Cai, Dane Westerdahl, … Ke Lan<br />
<em>Nature</em> (2020-04-27) <a href="https://doi.org/ggtgng">https://doi.org/ggtgng</a> <br />
DOI: <a href="https://doi.org/10.1038/s41586-020-2271-3">10.1038/s41586-020-2271-3</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32340022">32340022</a></p>
</div>
<div id="ref-1HOZHff6">
<p>179. <strong>Indirect Virus Transmission in Cluster of COVID-19 Cases, Wenzhou, China, 2020</strong> <br />
Jing Cai, Wenjie Sun, Jianping Huang, Michelle Gamber, Jing Wu, Guiqing He<br />
<em>Emerging Infectious Diseases</em> (2020-06) <a href="https://doi.org/ggpxkx">https://doi.org/ggpxkx</a> <br />
DOI: <a href="https://doi.org/10.3201/eid2606.200412">10.3201/eid2606.200412</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32163030">32163030</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7258486">PMC7258486</a></p>
</div>
<div id="ref-82XnTbtX">
<p>180. <strong>Airborne Transmission of SARS-CoV-2</strong> <br />
Michael Klompas, Meghan A. Baker, Chanu Rhee<br />
<em>JAMA</em> (2020-08-04) <a href="https://doi.org/gg4ttq">https://doi.org/gg4ttq</a> <br />
DOI: <a href="https://doi.org/10.1001/jama.2020.12458">10.1001/jama.2020.12458</a></p>
</div>
<div id="ref-cw5j7x80">
<p>181. <strong>Exaggerated risk of transmission of COVID-19 by fomites</strong> <br />
Emanuel Goldman<br />
<em>The Lancet Infectious Diseases</em> (2020-08) <a href="https://doi.org/gg6br7">https://doi.org/gg6br7</a> <br />
DOI: <a href="https://doi.org/10.1016/s1473-3099(20)30561-2">10.1016/s1473-3099(20)30561-2</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32628907">32628907</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7333993">PMC7333993</a></p>
</div>
<div id="ref-wxVni9Hz">
<p>182. <strong>Clinical characteristics of 24 asymptomatic infections with COVID-19 screened among close contacts in Nanjing, China</strong> <br />
Zhiliang Hu, Ci Song, Chuanjun Xu, Guangfu Jin, Yaling Chen, Xin Xu, Hongxia Ma, Wei Chen, Yuan Lin, Yishan Zheng, … Hongbing Shen<br />
<em>Science China Life Sciences</em> (2020-03-04) <a href="https://doi.org/dqbn">https://doi.org/dqbn</a> <br />
DOI: <a href="https://doi.org/10.1007/s11427-020-1661-4">10.1007/s11427-020-1661-4</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32146694">32146694</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7088568">PMC7088568</a></p>
</div>
<div id="ref-NIxttl2v">
<p>183. <strong>Evidence for transmission of COVID-19 prior to symptom onset</strong> <br />
Lauren C Tindale, Jessica E Stockdale, Michelle Coombe, Emma S Garlock, Wing Yin Venus Lau, Manu Saraswat, Louxin Zhang, Dongxuan Chen, Jacco Wallinga, Caroline Colijn<br />
<em>eLife</em> (2020-06-22) <a href="https://doi.org/gg6dtw">https://doi.org/gg6dtw</a> <br />
DOI: <a href="https://doi.org/10.7554/elife.57149">10.7554/elife.57149</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32568070">32568070</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7386904">PMC7386904</a></p>
</div>
<div id="ref-QFVSrboR">
<p>184. <strong>Time Kinetics of Viral Clearance and Resolution of Symptoms in Novel Coronavirus Infection</strong> <br />
De Chang, Guoxin Mo, Xin Yuan, Yi Tao, Xiaohua Peng, Fu-Sheng Wang, Lixin Xie, Lokesh Sharma, Charles S. Dela Cruz, Enqiang Qin<br />
<em>American Journal of Respiratory and Critical Care Medicine</em> (2020-05-01) <a href="https://doi.org/ggq8xs">https://doi.org/ggq8xs</a> <br />
DOI: <a href="https://doi.org/10.1164/rccm.202003-0524le">10.1164/rccm.202003-0524le</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32200654">32200654</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7193851">PMC7193851</a></p>
</div>
<div id="ref-1654TaAK4">
<p>185. <strong>Temporal dynamics in viral shedding and transmissibility of COVID-19</strong> <br />
Xi He, Eric H. Y. Lau, Peng Wu, Xilong Deng, Jian Wang, Xinxin Hao, Yiu Chung Lau, Jessica Y. Wong, Yujuan Guan, Xinghua Tan, … Gabriel M. Leung<br />
<em>Nature Medicine</em> (2020-04-15) <a href="https://doi.org/ggr99q">https://doi.org/ggr99q</a> <br />
DOI: <a href="https://doi.org/10.1038/s41591-020-0869-5">10.1038/s41591-020-0869-5</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32296168">32296168</a></p>
</div>
<div id="ref-Oyy1OUTT">
<p>186. <strong>Coronavirus Disease 2019 (COVID-19) - Transmission</strong> <br />
CDC<br />
<em>Centers for Disease Control and Prevention</em> (2020-10-28) <a href="https://www.cdc.gov/coronavirus/2019-ncov/prevent-getting-sick/how-covid-spreads.html">https://www.cdc.gov/coronavirus/2019-ncov/prevent-getting-sick/how-covid-spreads.html</a></p>
</div>
<div id="ref-Xp5HT2SF">
<p>187. <strong>Virological assessment of hospitalized patients with COVID-2019</strong> <br />
Roman Wölfel, Victor M. Corman, Wolfgang Guggemos, Michael Seilmaier, Sabine Zange, Marcel A. Müller, Daniela Niemeyer, Terry C. Jones, Patrick Vollmar, Camilla Rothe, … Clemens Wendtner<br />
<em>Nature</em> (2020-04-01) <a href="https://doi.org/ggqrv7">https://doi.org/ggqrv7</a> <br />
DOI: <a href="https://doi.org/10.1038/s41586-020-2196-x">10.1038/s41586-020-2196-x</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32235945">32235945</a></p>
</div>
<div id="ref-14fWuiUKS">
<p>188. <strong>Clinical predictors and timing of cessation of viral RNA shedding in patients with COVID-19</strong> <br />
Cristina Corsini Campioli, Edison Cano Cevallos, Mariam Assi, Robin Patel, Matthew J. Binnicker, John C. O’Horo<br />
<em>Journal of Clinical Virology</em> (2020-09) <a href="https://doi.org/gg7m96">https://doi.org/gg7m96</a> <br />
DOI: <a href="https://doi.org/10.1016/j.jcv.2020.104577">10.1016/j.jcv.2020.104577</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32777762">32777762</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7405830">PMC7405830</a></p>
</div>
<div id="ref-ac2kt3rh">
<p>189. <strong>Presymptomatic SARS-CoV-2 Infections and Transmission in a Skilled Nursing Facility</strong> <br />
Melissa M. Arons, Kelly M. Hatfield, Sujan C. Reddy, Anne Kimball, Allison James, Jesica R. Jacobs, Joanne Taylor, Kevin Spicer, Ana C. Bardossy, Lisa P. Oakley, … John A. Jernigan<br />
<em>New England Journal of Medicine</em> (2020-05-28) <a href="https://doi.org/ggszfg">https://doi.org/ggszfg</a> <br />
DOI: <a href="https://doi.org/10.1056/nejmoa2008457">10.1056/nejmoa2008457</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32329971">32329971</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7200056">PMC7200056</a></p>
</div>
<div id="ref-lVDiHYp">
<p>190. <strong>Prevalence of SARS-CoV-2 Infection in Residents of a Large Homeless Shelter in Boston</strong> <br />
Travis P. Baggett, Harrison Keyes, Nora Sporn, Jessie M. Gaeta<br />
<em>JAMA</em> (2020-06-02) <a href="https://doi.org/ggtsh3">https://doi.org/ggtsh3</a> <br />
DOI: <a href="https://doi.org/10.1001/jama.2020.6887">10.1001/jama.2020.6887</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32338732">32338732</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7186911">PMC7186911</a></p>
</div>
<div id="ref-E6Qv6YQ0">
<p>191. <strong>Presumed Asymptomatic Carrier Transmission of COVID-19</strong> <br />
Yan Bai, Lingsheng Yao, Tao Wei, Fei Tian, Dong-Yan Jin, Lijuan Chen, Meiyun Wang<br />
<em>JAMA</em> (2020-02-21) <a href="https://doi.org/ggmbs8">https://doi.org/ggmbs8</a> <br />
DOI: <a href="https://doi.org/10.1001/jama.2020.2565">10.1001/jama.2020.2565</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32083643">32083643</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7042844">PMC7042844</a></p>
</div>
<div id="ref-18Orbxz8t">
<p>192. <strong>Transmission of COVID-19 in the terminal stage of incubation period: a familial cluster</strong> <br />
Peng Li, Ji-Bo Fu, Ke-Feng Li, Yan Chen, Hong-Ling Wang, Lei-Jie Liu, Jie-Nan Liu, Yong-Li Zhang, She-Lan Liu, An Tang, … Jian-Bo Yan<br />
<em>International Journal of Infectious Diseases</em> (2020-03) <a href="https://doi.org/ggq844">https://doi.org/ggq844</a> <br />
DOI: <a href="https://doi.org/10.1016/j.ijid.2020.03.027">10.1016/j.ijid.2020.03.027</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32194239">32194239</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7264481">PMC7264481</a></p>
</div>
<div id="ref-4M3GM1sg">
<p>193. <strong>A Cohort of SARS-CoV-2 Infected Asymptomatic and Pre-Symptomatic Contacts from COVID-19 Contact Tracing in Hubei Province, China: Short-Term Outcomes</strong> <br />
Peng Zhang, Fei Tian, Yuan Wan, Jing Cai, Zhengmin Qian, Ran Wu, Yunquan Zhang, Shiyu Zhang, Huan Li, Mingyan Li, … Hualiang Lin<br />
<em>SSRN Electronic Journal</em> (2020) <a href="https://doi.org/ghf3n2">https://doi.org/ghf3n2</a> <br />
DOI: <a href="https://doi.org/10.2139/ssrn.3678556">10.2139/ssrn.3678556</a></p>
</div>
<div id="ref-6K5lsF5i">
<p>194. <strong>Estimating the asymptomatic proportion of coronavirus disease 2019 (COVID-19) cases on board the Diamond Princess cruise ship, Yokohama, Japan, 2020</strong> <br />
Kenji Mizumoto, Katsushi Kagaya, Alexander Zarebski, Gerardo Chowell<br />
<em>Eurosurveillance</em> (2020-03-12) <a href="https://doi.org/ggn4bd">https://doi.org/ggn4bd</a> <br />
DOI: <a href="https://doi.org/10.2807/1560-7917.es.2020.25.10.2000180">10.2807/1560-7917.es.2020.25.10.2000180</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32183930">32183930</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7078829">PMC7078829</a></p>
</div>
<div id="ref-sk1NbA7K">
<p>195. <strong>Estimated prevalence and viral transmissibility in subjects with asymptomatic SARS-CoV-2 infections in Wuhan, China</strong> <br />
Kang Zhang, Weiwei Tong, Xinghuan Wang, Johnson Yiu-Nam Lau<br />
<em>Precision Clinical Medicine</em> (2020-09-04) <a href="https://doi.org/ghjmks">https://doi.org/ghjmks</a> <br />
DOI: <a href="https://doi.org/10.1093/pcmedi/pbaa032">10.1093/pcmedi/pbaa032</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7499683">PMC7499683</a></p>
</div>
<div id="ref-kIumgXPI">
<p>196. <strong>Investigation of SARS-CoV-2 outbreaks in six care homes in London, April 2020</strong> <br />
Shamez N Ladhani, J.Yimmy Chow, Roshni Janarthanan, Jonathan Fok, Emma Crawley-Boevey, Amoolya Vusirikala, Elena Fernandez, Marina Sanchez Perez, Suzanne Tang, Kate Dun-Campbell, … Maria Zambon<br />
<em>EClinicalMedicine</em> (2020-09) <a href="https://doi.org/ghbj9v">https://doi.org/ghbj9v</a> <br />
DOI: <a href="https://doi.org/10.1016/j.eclinm.2020.100533">10.1016/j.eclinm.2020.100533</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32923993">32923993</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7480335">PMC7480335</a></p>
</div>
<div id="ref-10OGkFiGJ">
<p>197. <strong>Clinical and immunological assessment of asymptomatic SARS-CoV-2 infections</strong> <br />
Quan-Xin Long, Xiao-Jun Tang, Qiu-Lin Shi, Qin Li, Hai-Jun Deng, Jun Yuan, Jie-Li Hu, Wei Xu, Yong Zhang, Fa-Jin Lv, … Ai-Long Huang<br />
<em>Nature Medicine</em> (2020-06-18) <a href="https://doi.org/gg26dx">https://doi.org/gg26dx</a> <br />
DOI: <a href="https://doi.org/10.1038/s41591-020-0965-6">10.1038/s41591-020-0965-6</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32555424">32555424</a></p>
</div>
<div id="ref-dMjSbAQV">
<p>198. <strong>Suppression of a SARS-CoV-2 outbreak in the Italian municipality of Vo’</strong> <br />
Enrico Lavezzo, Elisa Franchin, Constanze Ciavarella, Gina Cuomo-Dannenburg, Luisa Barzon, Claudia Del Vecchio, Lucia Rossi, Riccardo Manganelli, Arianna Loregian, Nicolò Navarin, … Imperial College COVID-19 Response Team<br />
<em>Nature</em> (2020-06-30) <a href="https://doi.org/gg3w87">https://doi.org/gg3w87</a> <br />
DOI: <a href="https://doi.org/10.1038/s41586-020-2488-1">10.1038/s41586-020-2488-1</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32604404">32604404</a></p>
</div>
<div id="ref-BENQIwOM">
<p>199. <strong>A systematic review and meta-analysis of published research data on COVID-19 infection-fatality rates</strong> <br />
Gideon Meyerowitz-Katz, Lea Merone<br />
<em>Cold Spring Harbor Laboratory</em> (2020-07-07) <a href="https://doi.org/dv2r">https://doi.org/dv2r</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.05.03.20089854">10.1101/2020.05.03.20089854</a></p>
</div>
<div id="ref-AavOV1He">
<p>200. <strong>Global Covid-19 Case Fatality Rates</strong> <br />
The Centre for Evidence-Based Medicine<br />
<a href="https://www.cebm.net/covid-19/global-covid-19-case-fatality-rates/">https://www.cebm.net/covid-19/global-covid-19-case-fatality-rates/</a></p>
</div>
<div id="ref-12CiweatZ">
<p>201. <strong>Estimating the Global Infection Fatality Rate of COVID-19</strong> <br />
Richard Grewelle, Giulio De Leo<br />
<em>Cold Spring Harbor Laboratory</em> (2020-05-18) <a href="https://doi.org/ghbvcj">https://doi.org/ghbvcj</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.05.11.20098780">10.1101/2020.05.11.20098780</a></p>
</div>
<div id="ref-tIjAygts">
<p>202. <strong>Repeated cross-sectional sero-monitoring of SARS-CoV-2 in New York City</strong> <br />
Daniel Stadlbauer, Jessica Tan, Kaijun Jiang, Matthew M. Hernandez, Shelcie Fabre, Fatima Amanat, Catherine Teo, Guha Asthagiri Arunkumar, Meagan McMahon, Christina Capuano, … Florian Krammer<br />
<em>Nature</em> (2020-11-03) <a href="https://doi.org/ghhtq9">https://doi.org/ghhtq9</a> <br />
DOI: <a href="https://doi.org/10.1038/s41586-020-2912-6">10.1038/s41586-020-2912-6</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33142304">33142304</a></p>
</div>
<div id="ref-14l4fvU71">
<p>203. <strong>Modeling infectious diseases in humans and animals</strong> <br />
Matthew James Keeling, Pejman Rohani<br />
<em>Princeton University Press</em> (2008) <br />
ISBN: <a href="https://worldcat.org/isbn/9780691116174">9780691116174</a></p>
</div>
<div id="ref-16CD4voW1">
<p>204. <strong>The concept of R <sub>o</sub> in epidemic theory</strong> <br />
J. A. P. Heesterbeek, K. Dietz<br />
<em>Statistica Neerlandica</em> (1996-03) <a href="https://doi.org/d29ch4">https://doi.org/d29ch4</a> <br />
DOI: <a href="https://doi.org/10.1111/j.1467-9574.1996.tb01482.x">10.1111/j.1467-9574.1996.tb01482.x</a></p>
</div>
<div id="ref-YubbhU7G">
<p>205. <strong>Modeling infectious disease dynamics</strong> <br />
Sarah Cobey<br />
<em>Science</em> (2020-05-15) <a href="https://doi.org/ggsztw">https://doi.org/ggsztw</a> <br />
DOI: <a href="https://doi.org/10.1126/science.abb5659">10.1126/science.abb5659</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32332062">32332062</a></p>
</div>
<div id="ref-JRaVgcNd">
<p>206. <strong>Theoretical ecology: principles and applications</strong> <br />
Robert M. May, Angela R. McLean (editors)<br />
<em>Oxford University Press</em> (2007) <br />
ISBN: <a href="https://worldcat.org/isbn/9780199209989">9780199209989</a></p>
</div>
<div id="ref-1Aozf7KBW">
<p>207. <strong>A contribution to the mathematical theory of epidemics</strong> <br />
William Ogilvy Kermack, A. G. McKendrick<br />
<em>Proceedings of the Royal Society of London. Series A, Containing Papers of a Mathematical and Physical Character</em> (1927-08) <a href="https://doi.org/10.1098/rspa.1927.0118">https://doi.org/10.1098/rspa.1927.0118</a> <br />
DOI: <a href="https://doi.org/10.1098/rspa.1927.0118">10.1098/rspa.1927.0118</a></p>
</div>
<div id="ref-SPInPew0">
<p>208. <strong>Population biology of infectious diseases: Part I</strong> <br />
Roy M. Anderson, Robert M. May<br />
<em>Nature</em> (1979-08) <a href="https://doi.org/b6z9hc">https://doi.org/b6z9hc</a> <br />
DOI: <a href="https://doi.org/10.1038/280361a0">10.1038/280361a0</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/460412">460412</a></p>
</div>
<div id="ref-1E0r4uZy9">
<p>209. <strong>Nowcasting and forecasting the potential domestic and international spread of the 2019-nCoV outbreak originating in Wuhan, China: a modelling study</strong> <br />
Joseph T Wu, Kathy Leung, Gabriel M Leung<br />
<em>The Lancet</em> (2020-01) <a href="https://doi.org/ggjvr7">https://doi.org/ggjvr7</a> <br />
DOI: <a href="https://doi.org/10.1016/s0140-6736(20)30260-9">10.1016/s0140-6736(20)30260-9</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32014114">32014114</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7159271">PMC7159271</a></p>
</div>
<div id="ref-LHtuVmaq">
<p>210. <strong>The reproductive number of COVID-19 is higher compared to SARS coronavirus</strong> <br />
Ying Liu, Albert A Gayle, Annelies Wilder-Smith, Joacim Rocklöv<br />
<em>Journal of Travel Medicine</em> (2020-02-13) <a href="https://doi.org/ggnntv">https://doi.org/ggnntv</a> <br />
DOI: <a href="https://doi.org/10.1093/jtm/taaa021">10.1093/jtm/taaa021</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32052846">32052846</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7074654">PMC7074654</a></p>
</div>
<div id="ref-mHYmt0mv">
<p>211. <strong>Substantial undocumented infection facilitates the rapid dissemination of novel coronavirus (SARS-CoV2)</strong> <br />
Ruiyun Li, Sen Pei, Bin Chen, Yimeng Song, Tao Zhang, Wan Yang, Jeffrey Shaman<br />
<em>Science</em> (2020-03-16) <a href="https://doi.org/ggn6c2">https://doi.org/ggn6c2</a> <br />
DOI: <a href="https://doi.org/10.1126/science.abb3221">10.1126/science.abb3221</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32179701">32179701</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7164387">PMC7164387</a></p>
</div>
<div id="ref-vITui6ac">
<p>212. <strong>Epidemiological parameters of coronavirus disease 2019: a pooled analysis of publicly reported individual data of 1155 cases from seven countries</strong> <br />
Shujuan Ma, Jiayue Zhang, Minyan Zeng, Qingping Yun, Wei Guo, Yixiang Zheng, Shi Zhao, Maggie H Wang, Zuyao Yang<br />
<em>Cold Spring Harbor Laboratory</em> (2020-03-24) <a href="https://doi.org/ggqhzz">https://doi.org/ggqhzz</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.21.20040329">10.1101/2020.03.21.20040329</a></p>
</div>
<div id="ref-nxM0rP5R">
<p>213. <strong>Early Transmissibility Assessment of a Novel Coronavirus in Wuhan, China</strong> <br />
Maimuna Majumder, Kenneth D. Mandl<br />
<em>SSRN Electronic Journal</em> (2020) <a href="https://doi.org/ggqhz3">https://doi.org/ggqhz3</a> <br />
DOI: <a href="https://doi.org/10.2139/ssrn.3524675">10.2139/ssrn.3524675</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32714102">32714102</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7366781">PMC7366781</a></p>
</div>
<div id="ref-WLc2UMgQ">
<p>214. <strong>Time-varying transmission dynamics of Novel Coronavirus Pneumonia in China</strong> <br />
Tao Liu, Jianxiong Hu, Jianpeng Xiao, Guanhao He, Min Kang, Zuhua Rong, Lifeng Lin, Haojie Zhong, Qiong Huang, Aiping Deng, … Wenjun Ma<br />
<em>bioRxiv</em> (2020-02-13) <a href="https://doi.org/dkx9">https://doi.org/dkx9</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.01.25.919787">10.1101/2020.01.25.919787</a></p>
</div>
<div id="ref-10bBqMHH7">
<p>215. <strong>Estimation of the reproductive number of novel coronavirus (COVID-19) and the probable outbreak size on the Diamond Princess cruise ship: A data-driven analysis</strong> <br />
Sheng Zhang, MengYuan Diao, Wenbo Yu, Lei Pei, Zhaofen Lin, Dechang Chen<br />
<em>International Journal of Infectious Diseases</em> (2020-04) <a href="https://doi.org/ggpx56">https://doi.org/ggpx56</a> <br />
DOI: <a href="https://doi.org/10.1016/j.ijid.2020.02.033">10.1016/j.ijid.2020.02.033</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32097725">32097725</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7110591">PMC7110591</a></p>
</div>
<div id="ref-ITh0Anof">
<p>216. <strong>Estimation of the Transmission Risk of the 2019-nCoV and Its Implication for Public Health Interventions</strong> <br />
Tang, Wang, Li, Bragazzi, Tang, Xiao, Wu<br />
<em>Journal of Clinical Medicine</em> (2020-02-07) <a href="https://doi.org/ggmkf4">https://doi.org/ggmkf4</a> <br />
DOI: <a href="https://doi.org/10.3390/jcm9020462">10.3390/jcm9020462</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32046137">32046137</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7074281">PMC7074281</a></p>
</div>
<div id="ref-1BMU7sKbs">
<p>217. <strong>Estimating the effective reproduction number of the 2019-nCoV in China</strong> <br />
Zhidong Cao, Qingpeng Zhang, Xin Lu, Dirk Pfeiffer, Zhongwei Jia, Hongbing Song, Daniel Dajun Zeng<br />
<em>medRxiv</em> (2020-01) <a href="https://www.medrxiv.org/content/10.1101/2020.01.27.20018952v1">https://www.medrxiv.org/content/10.1101/2020.01.27.20018952v1</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.01.27.20018952">10.1101/2020.01.27.20018952</a></p>
</div>
<div id="ref-Yj8Xh4Wz">
<p>218. <strong>Modelling the epidemic trend of the 2019 novel coronavirus outbreak in China</strong> <br />
Mingwang Shen, Zhihang Peng, Yanni Xiao, Lei Zhang<br />
<em>bioRxiv</em> (2020-01-25) <a href="https://doi.org/ggqhzw">https://doi.org/ggqhzw</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.01.23.916726">10.1101/2020.01.23.916726</a></p>
</div>
<div id="ref-ZzrrVDoE">
<p>219. <strong>Novel coronavirus 2019-nCoV: early estimation of epidemiological parameters and epidemic predictions</strong> <br />
Jonathan M Read, Jessica RE Bridgen, Derek AT Cummings, Antonia Ho, Chris P Jewell<br />
<em>medRxiv</em> (2020-01-28) <a href="https://doi.org/dkzb">https://doi.org/dkzb</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.01.23.20018549">10.1101/2020.01.23.20018549</a></p>
</div>
<div id="ref-itj26agd">
<p>220. <strong>Using early data to estimate the actual infection fatality ratio from COVID-19 in France</strong> <br />
Lionel Roques, Etienne Klein, Julien Papaix, Antoine Sar, Samuel Soubeyrand<br />
<em>Cold Spring Harbor Laboratory</em> (2020-05-07) <a href="https://doi.org/ggqhz2">https://doi.org/ggqhz2</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.22.20040915">10.1101/2020.03.22.20040915</a></p>
</div>
<div id="ref-YfOcGRRa">
<p>221. <strong>Potential roles of social distancing in mitigating the spread of coronavirus disease 2019 (COVID-19) in South Korea</strong> <br />
Sang Woo Park, Kaiyuan Sun, Cécile Viboud, Bryan T Grenfell, Jonathan Dushoff<br />
<em>GitHub</em> (2020) <a href="https://github.com/parksw3/Korea-analysis/blob/master/v1/korea.pdf">https://github.com/parksw3/Korea-analysis/blob/master/v1/korea.pdf</a></p>
</div>
<div id="ref-tiRfUgvs">
<p>222. <strong>Early dynamics of transmission and control of COVID-19: a mathematical modelling study</strong> <br />
Adam J Kucharski, Timothy W Russell, Charlie Diamond, Yang Liu, John Edmunds, Sebastian Funk, Rosalind M Eggo, Fiona Sun, Mark Jit, James D Munday, … Stefan Flasche<br />
<em>The Lancet Infectious Diseases</em> (2020-03) <a href="https://doi.org/ggptcf">https://doi.org/ggptcf</a> <br />
DOI: <a href="https://doi.org/10.1016/s1473-3099(20)30144-4">10.1016/s1473-3099(20)30144-4</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32171059">32171059</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7158569">PMC7158569</a></p>
</div>
<div id="ref-TH4ymqvJ">
<p>223. <strong>Estimating the reproduction number of COVID-19 in Iran using epidemic modeling</strong> <br />
Ebrahim Sahafizadeh, Samaneh Sartoli<br />
<em>Cold Spring Harbor Laboratory</em> (2020-04-23) <a href="https://doi.org/ggqhzx">https://doi.org/ggqhzx</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.20.20038422">10.1101/2020.03.20.20038422</a></p>
</div>
<div id="ref-Frya4XA4">
<p>224. <strong>Report 13: Estimating the number of infections and the impact of non-pharmaceutical interventions on COVID-19 in 11 European countries</strong> <br />
S Flaxman, S Mishra, A Gandy, H Unwin, H Coupland, T Mellan, H Zhu, T Berah, J Eaton, P Perez Guzman, … S Bhatt<br />
<em>Imperial College London</em> (2020-03-30) <a href="https://doi.org/ggrbmf">https://doi.org/ggrbmf</a> <br />
DOI: <a href="https://doi.org/10.25561/77731">10.25561/77731</a></p>
</div>
<div id="ref-NCcssjLV">
<p>225. <strong>Projecting hospital utilization during the COVID-19 outbreaks in the United States</strong> <br />
Seyed M. Moghadas, Affan Shoukat, Meagan C. Fitzpatrick, Chad R. Wells, Pratha Sah, Abhishek Pandey, Jeffrey D. Sachs, Zheng Wang, Lauren A. Meyers, Burton H. Singer, Alison P. Galvani<br />
<em>Proceedings of the National Academy of Sciences</em> (2020-04-21) <a href="https://doi.org/ggq7jc">https://doi.org/ggq7jc</a> <br />
DOI: <a href="https://doi.org/10.1073/pnas.2004064117">10.1073/pnas.2004064117</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32245814">32245814</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7183199">PMC7183199</a></p>
</div>
<div id="ref-11CWFYFDL">
<p>226. <strong>The effect of control strategies to reduce social mixing on outcomes of the COVID-19 epidemic in Wuhan, China: a modelling study</strong> <br />
Kiesha Prem, Yang Liu, Timothy W Russell, Adam J Kucharski, Rosalind M Eggo, Nicholas Davies, Mark Jit, Petra Klepac, Stefan Flasche, Samuel Clifford, … Joel Hellewell<br />
<em>The Lancet Public Health</em> (2020-03) <a href="https://doi.org/ggp3xq">https://doi.org/ggp3xq</a> <br />
DOI: <a href="https://doi.org/10.1016/s2468-2667(20)30073-6">10.1016/s2468-2667(20)30073-6</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32220655">32220655</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7158905">PMC7158905</a></p>
</div>
<div id="ref-J4RCpntH">
<p>227. <strong>Spread and dynamics of the COVID-19 epidemic in Italy: Effects of emergency containment measures</strong> <br />
Marino Gatto, Enrico Bertuzzo, Lorenzo Mari, Stefano Miccoli, Luca Carraro, Renato Casagrandi, Andrea Rinaldo<br />
<em>Proceedings of the National Academy of Sciences</em> (2020-05-12) <a href="https://doi.org/ggv4j6">https://doi.org/ggv4j6</a> <br />
DOI: <a href="https://doi.org/10.1073/pnas.2004978117">10.1073/pnas.2004978117</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32327608">32327608</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7229754">PMC7229754</a></p>
</div>
<div id="ref-WA1R5R95">
<p>228. <strong>Covid-19: Temporal variation in transmission during the COVID-19 outbreak</strong> <br />
EpiForecasts and the CMMID Covid working group<br />
<a href="https://epiforecasts.io/covid/">https://epiforecasts.io/covid/</a></p>
</div>
<div id="ref-17CQnstjh">
<p>229. <strong>Rt COVID-19</strong> <br />
Kevin Systrom, Thomas Vladeck, Mike Krieger<br />
<a href="https://rt.live/">https://rt.live/</a></p>
</div>
<div id="ref-jVbH3kJR">
<p>230. <strong>An integrated national scale SARS-CoV-2 genomic surveillance network</strong> <br />
The Lancet Microbe<br />
(2020-07) <a href="https://doi.org/d5mg">https://doi.org/d5mg</a> <br />
DOI: <a href="https://doi.org/10.1016/s2666-5247(20)30054-9">10.1016/s2666-5247(20)30054-9</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32835336">32835336</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7266609">PMC7266609</a></p>
</div>
<div id="ref-W5Z7Ztlg">
<p>231. <strong>Coast-to-Coast Spread of SARS-CoV-2 during the Early Epidemic in the United States</strong> <br />
Joseph R. Fauver, Mary E. Petrone, Emma B. Hodcroft, Kayoko Shioda, Hanna Y. Ehrlich, Alexander G. Watts, Chantal B. F. Vogels, Anderson F. Brito, Tara Alpert, Anthony Muyombwe, … Nathan D. Grubaugh<br />
<em>Cell</em> (2020-05) <a href="https://doi.org/gg6r9x">https://doi.org/gg6r9x</a> <br />
DOI: <a href="https://doi.org/10.1016/j.cell.2020.04.021">10.1016/j.cell.2020.04.021</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32386545">32386545</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7204677">PMC7204677</a></p>
</div>
<div id="ref-FVmbE2oW">
<p>232. <strong>Introductions and early spread of SARS-CoV-2 in the New York City area</strong> <br />
Ana S. Gonzalez-Reiche, Matthew M. Hernandez, Mitchell J. Sullivan, Brianne Ciferri, Hala Alshammary, Ajay Obla, Shelcie Fabre, Giulio Kleiner, Jose Polanco, Zenab Khan, … Harm van Bakel<br />
<em>Science</em> (2020-05-29) <a href="https://doi.org/gg5gv7">https://doi.org/gg5gv7</a> <br />
DOI: <a href="https://doi.org/10.1126/science.abc1917">10.1126/science.abc1917</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32471856">32471856</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7259823">PMC7259823</a></p>
</div>
<div id="ref-WJrnUyZf">
<p>233. <strong>Spread of SARS-CoV-2 in the Icelandic Population</strong> <br />
Daniel F. Gudbjartsson, Agnar Helgason, Hakon Jonsson, Olafur T. Magnusson, Pall Melsted, Gudmundur L. Norddahl, Jona Saemundsdottir, Asgeir Sigurdsson, Patrick Sulem, Arna B. Agustsdottir, … Kari Stefansson<br />
<em>New England Journal of Medicine</em> (2020-06-11) <a href="https://doi.org/ggr6wx">https://doi.org/ggr6wx</a> <br />
DOI: <a href="https://doi.org/10.1056/nejmoa2006100">10.1056/nejmoa2006100</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32289214">32289214</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7175425">PMC7175425</a></p>
</div>
<div id="ref-qIXPib3m">
<p>234. <strong>GISAID - Initiative</strong> <a href="https://www.gisaid.org/">https://www.gisaid.org/</a></p>
</div>
<div id="ref-cyzrC7qd">
<p>235. <strong>NCBI SARS-CoV-2 Resources</strong> <a href="https://www.ncbi.nlm.nih.gov/sars-cov-2/">https://www.ncbi.nlm.nih.gov/sars-cov-2/</a></p>
</div>
<div id="ref-BjPoOYSA">
<p>236. <strong>COVID-19 Data Portal - accelerating scientific research through data</strong> <a href="https://www.covid19dataportal.org/">https://www.covid19dataportal.org/</a></p>
</div>
<div id="ref-11xfoqpEl">
<p>237. <strong>Characteristics of and Important Lessons From the Coronavirus Disease 2019 (COVID-19) Outbreak in China</strong> <br />
Zunyou Wu, Jennifer M. McGoogan<br />
<em>JAMA</em> (2020-02-24) <a href="https://doi.org/ggmq43">https://doi.org/ggmq43</a> <br />
DOI: <a href="https://doi.org/10.1001/jama.2020.2648">10.1001/jama.2020.2648</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32091533">32091533</a></p>
</div>
<div id="ref-qlJ8hCUs">
<p>238. <strong>Critical Care Utilization for the COVID-19 Outbreak in Lombardy, Italy</strong> <br />
Giacomo Grasselli, Antonio Pesenti, Maurizio Cecconi<br />
<em>JAMA</em> (2020-03-13) <a href="https://doi.org/ggqf6g">https://doi.org/ggqf6g</a> <br />
DOI: <a href="https://doi.org/10.1001/jama.2020.4031">10.1001/jama.2020.4031</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32167538">32167538</a></p>
</div>
<div id="ref-xZBEFXiG">
<p>239. <strong>Hospitalization Rates and Characteristics of Patients Hospitalized with Laboratory-Confirmed Coronavirus Disease 2019 — COVID-NET, 14 States, March 1–30, 2020</strong> <br />
Shikha Garg, Lindsay Kim, Michael Whitaker, Alissa O’Halloran, Charisse Cummings, Rachel Holstein, Mila Prill, Shua J. Chai, Pam D. Kirley, Nisha B. Alden, … Alicia Fry<br />
<em>MMWR. Morbidity and Mortality Weekly Report</em> (2020-04-17) <a href="https://doi.org/ggsppz">https://doi.org/ggsppz</a> <br />
DOI: <a href="https://doi.org/10.15585/mmwr.mm6915e3">10.15585/mmwr.mm6915e3</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32298251">32298251</a></p>
</div>
<div id="ref-1D6lVXWbJ">
<p>240. <strong>Disparities In Outcomes Among COVID-19 Patients In A Large Health Care System In California</strong> <br />
Kristen M. J. Azar, Zijun Shen, Robert J. Romanelli, Stephen H. Lockhart, Kelly Smits, Sarah Robinson, Stephanie Brown, Alice R. Pressman<br />
<em>Health Affairs</em> (2020-07-01) <a href="https://doi.org/ggx4mf">https://doi.org/ggx4mf</a> <br />
DOI: <a href="https://doi.org/10.1377/hlthaff.2020.00598">10.1377/hlthaff.2020.00598</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32437224">32437224</a></p>
</div>
<div id="ref-Zk5CfYeW">
<p>241. <strong>Characteristics Associated with Hospitalization Among Patients with COVID-19 — Metropolitan Atlanta, Georgia, March–April 2020</strong> <br />
Marie E. Killerby, Ruth Link-Gelles, Sarah C. Haight, Caroline A. Schrodt, Lucinda England, Danica J. Gomes, Mays Shamout, Kristen Pettrone, Kevin O’Laughlin, Anne Kimball, … CDC COVID-19 Response Clinical Team<br />
<em>MMWR. Morbidity and Mortality Weekly Report</em> (2020-06-26) <a href="https://doi.org/gg3k6h">https://doi.org/gg3k6h</a> <br />
DOI: <a href="https://doi.org/10.15585/mmwr.mm6925e1">10.15585/mmwr.mm6925e1</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32584797">32584797</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7316317">PMC7316317</a></p>
</div>
<div id="ref-BQ4pZGxJ">
<p>242. <strong>Demographic science aids in understanding the spread and fatality rates of COVID-19</strong> <br />
Jennifer Beam Dowd, Liliana Andriano, David M. Brazel, Valentina Rotondi, Per Block, Xuejie Ding, Yan Liu, Melinda C. Mills<br />
<em>Proceedings of the National Academy of Sciences</em> (2020-05-05) <a href="https://doi.org/ggsd5b">https://doi.org/ggsd5b</a> <br />
DOI: <a href="https://doi.org/10.1073/pnas.2004911117">10.1073/pnas.2004911117</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32300018">32300018</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7211934">PMC7211934</a></p>
</div>
<div id="ref-1DnayDKc4">
<p>243. <strong>‐19 and Older Adults: What We Know</strong> <br />
Zainab Shahid, Ricci Kalayanamitra, Brendan McClafferty, Douglas Kepko, Devyani Ramgobin, Ravi Patel, Chander Shekher Aggarwal, Ramarao Vunnam, Nitasa Sahu, Dhirisha Bhatt, … Rohit Jain<br />
<em>Journal of the American Geriatrics Society</em> (2020-04-20) <a href="https://doi.org/ggxgsb">https://doi.org/ggxgsb</a> <br />
DOI: <a href="https://doi.org/10.1111/jgs.16472">10.1111/jgs.16472</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32255507">32255507</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7262251">PMC7262251</a></p>
</div>
<div id="ref-gObtnWSD">
<p>244. <strong>Features of 20 133 UK patients in hospital with covid-19 using the ISARIC WHO Clinical Characterisation Protocol: prospective observational cohort study</strong> <br />
Annemarie B Docherty, Ewen M Harrison, Christopher A Green, Hayley E Hardwick, Riinu Pius, Lisa Norman, Karl A Holden, Jonathan M Read, Frank Dondelinger, Gail Carson, … Malcolm G Semple<br />
<em>BMJ</em> (2020-05-22) <a href="https://doi.org/ggw4nh">https://doi.org/ggw4nh</a> <br />
DOI: <a href="https://doi.org/10.1136/bmj.m1985">10.1136/bmj.m1985</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32444460">32444460</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7243036">PMC7243036</a></p>
</div>
<div id="ref-DqBeNrR5">
<p>245. <strong>Impact of sex and gender on COVID-19 outcomes in Europe</strong> <br />
Catherine Gebhard, Vera Regitz-Zagrosek, Hannelore K. Neuhauser, Rosemary Morgan, Sabra L. Klein<br />
<em>Biology of Sex Differences</em> (2020-05-25) <a href="https://doi.org/ghbvck">https://doi.org/ghbvck</a> <br />
DOI: <a href="https://doi.org/10.1186/s13293-020-00304-9">10.1186/s13293-020-00304-9</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32450906">32450906</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7247289">PMC7247289</a></p>
</div>
<div id="ref-ConBOYBB">
<p>246. <strong>The Sex, Gender and COVID-19 Project | Global Health 50/50</strong> <a href="https://globalhealth5050.org/the-sex-gender-and-covid-19-project/">https://globalhealth5050.org/the-sex-gender-and-covid-19-project/</a></p>
</div>
<div id="ref-fH3qecwy">
<p>247. <strong>Biological sex impacts COVID-19 outcomes</strong> <br />
Sabra L. Klein, Santosh Dhakal, Rebecca L. Ursin, Sharvari Deshpande, Kathryn Sandberg, Franck Mauvais-Jarvis<br />
<em>PLOS Pathogens</em> (2020-06-22) <a href="https://doi.org/gg3hwv">https://doi.org/gg3hwv</a> <br />
DOI: <a href="https://doi.org/10.1371/journal.ppat.1008570">10.1371/journal.ppat.1008570</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32569293">32569293</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7307725">PMC7307725</a></p>
</div>
<div id="ref-RcYBFjm1">
<p>248. <strong>Sex-specific clinical characteristics and prognosis of coronavirus disease-19 infection in Wuhan, China: A retrospective study of 168 severe patients</strong> <br />
Yifan Meng, Ping Wu, Wanrong Lu, Kui Liu, Ke Ma, Liang Huang, Jiaojiao Cai, Hong Zhang, Yu Qin, Haiying Sun, … Peng Wu<br />
<em>PLOS Pathogens</em> (2020-04-28) <a href="https://doi.org/ggv3zn">https://doi.org/ggv3zn</a> <br />
DOI: <a href="https://doi.org/10.1371/journal.ppat.1008520">10.1371/journal.ppat.1008520</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32343745">32343745</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7209966">PMC7209966</a></p>
</div>
<div id="ref-OJvjTsWz">
<p>249. <strong>Sex differences in renal angiotensin converting enzyme 2 (ACE2) activity are 17β-oestradiol-dependent and sex chromosome-independent</strong> <br />
Jun Liu, Hong Ji, Wei Zheng, Xie Wu, Janet J Zhu, Arthur P Arnold, Kathryn Sandberg<br />
<em>Biology of Sex Differences</em> (2010) <a href="https://doi.org/bbx3r6">https://doi.org/bbx3r6</a> <br />
DOI: <a href="https://doi.org/10.1186/2042-6410-1-6">10.1186/2042-6410-1-6</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21208466">21208466</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3010099">PMC3010099</a></p>
</div>
<div id="ref-ujWsdPdH">
<p>250. <strong>COVID-19 in nursing homes</strong> <br />
A Fallon, T Dukelow, SP Kennelly, D O’Neill<br />
<em>QJM: An International Journal of Medicine</em> (2020-06) <a href="https://doi.org/ggv4xx">https://doi.org/ggv4xx</a> <br />
DOI: <a href="https://doi.org/10.1093/qjmed/hcaa136">10.1093/qjmed/hcaa136</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32311049">32311049</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7188176">PMC7188176</a></p>
</div>
<div id="ref-agpMLmxt">
<p>251. <strong>Vulnerabilities to COVID-19 Among Transgender Adults in the U.S.</strong> <br />
Jody L. Herman, Kathryn O’Neill<br />
(2020-04-01) <a href="https://escholarship.org/uc/item/55t297mc">https://escholarship.org/uc/item/55t297mc</a></p>
</div>
<div id="ref-9NfXSHjH">
<p>252. <strong>Association of Blood Glucose Control and Outcomes in Patients with COVID-19 and Pre-existing Type 2 Diabetes</strong> <br />
Lihua Zhu, Zhi-Gang She, Xu Cheng, Juan-Juan Qin, Xiao-Jing Zhang, Jingjing Cai, Fang Lei, Haitao Wang, Jing Xie, Wenxin Wang, … Hongliang Li<br />
<em>Cell Metabolism</em> (2020-06) <a href="https://doi.org/ggvcc9">https://doi.org/ggvcc9</a> <br />
DOI: <a href="https://doi.org/10.1016/j.cmet.2020.04.021">10.1016/j.cmet.2020.04.021</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32369736">32369736</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7252168">PMC7252168</a></p>
</div>
<div id="ref-13BpwZaaq">
<p>253. <strong>Diabetes increases the mortality of patients with COVID-19: a meta-analysis</strong> <br />
Zeng-hong Wu, Yun Tang, Qing Cheng<br />
<em>Acta Diabetologica</em> (2020-06-24) <a href="https://doi.org/gg3k55">https://doi.org/gg3k55</a> <br />
DOI: <a href="https://doi.org/10.1007/s00592-020-01546-0">10.1007/s00592-020-01546-0</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32583078">32583078</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7311595">PMC7311595</a></p>
</div>
<div id="ref-UkCdtLwF">
<p>254. <strong>COVID-19 infection may cause ketosis and ketoacidosis</strong> <br />
Juyi Li, Xiufang Wang, Jian Chen, Xiuran Zuo, Hongmei Zhang, Aiping Deng<br />
<em>Diabetes, Obesity and Metabolism</em> (2020-05-18) <a href="https://doi.org/ggv4tm">https://doi.org/ggv4tm</a> <br />
DOI: <a href="https://doi.org/10.1111/dom.14057">10.1111/dom.14057</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32314455">32314455</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7264681">PMC7264681</a></p>
</div>
<div id="ref-OT5GEOej">
<p>255. <strong>COVID-19 pandemic, coronaviruses, and diabetes mellitus</strong> <br />
Ranganath Muniyappa, Sriram Gubbi<br />
<em>American Journal of Physiology-Endocrinology and Metabolism</em> (2020-05-01) <a href="https://doi.org/ggq79v">https://doi.org/ggq79v</a> <br />
DOI: <a href="https://doi.org/10.1152/ajpendo.00124.2020">10.1152/ajpendo.00124.2020</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32228322">32228322</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7191633">PMC7191633</a></p>
</div>
<div id="ref-ZLmNMDrD">
<p>256. <strong>Severe obesity, increasing age and male sex are independently associated with worse in-hospital outcomes, and higher in-hospital mortality, in a cohort of patients with COVID-19 in the Bronx, New York</strong> <br />
Leonidas Palaiodimos, Damianos G. Kokkinidis, Weijia Li, Dimitrios Karamanis, Jennifer Ognibene, Shitij Arora, William N. Southern, Christos S. Mantzoros<br />
<em>Metabolism</em> (2020-07) <a href="https://doi.org/ggx229">https://doi.org/ggx229</a> <br />
DOI: <a href="https://doi.org/10.1016/j.metabol.2020.154262">10.1016/j.metabol.2020.154262</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32422233">32422233</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7228874">PMC7228874</a></p>
</div>
<div id="ref-2Ahpn6R2">
<p>257. <strong>Features of 16,749 hospitalised UK patients with COVID-19 using the ISARIC WHO Clinical Characterisation Protocol</strong> <br />
Annemarie B Docherty, Ewen M Harrison, Christopher A Green, Hayley E Hardwick, Riinu Pius, Lisa Norman, Karl A Holden, Jonathan M Read, Frank Dondelinger, Gail Carson, … Malcolm Gracie Semple<br />
<em>Cold Spring Harbor Laboratory</em> (2020-04-28) <a href="https://doi.org/ggtdtb">https://doi.org/ggtdtb</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.04.23.20076042">10.1101/2020.04.23.20076042</a></p>
</div>
<div id="ref-1AmBg80Un">
<p>258. <strong>When Two Pandemics Meet: Why Is Obesity Associated with Increased COVID-19 Mortality?</strong> <br />
Sam M. Lockhart, Stephen O’Rahilly<br />
<em>Med</em> (2020-06) <a href="https://doi.org/gg3k57">https://doi.org/gg3k57</a> <br />
DOI: <a href="https://doi.org/10.1016/j.medj.2020.06.005">10.1016/j.medj.2020.06.005</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32838359">32838359</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7323660">PMC7323660</a></p>
</div>
<div id="ref-1GakM8xhV">
<p>259. <strong>Besides population age structure, health and other demographic factors can contribute to understanding the COVID-19 burden</strong> <br />
Marília R. Nepomuceno, Enrique Acosta, Diego Alburez-Gutierrez, José Manuel Aburto, Alain Gagnon, Cássio M. Turra<br />
<em>Proceedings of the National Academy of Sciences</em> (2020-06-23) <a href="https://doi.org/gg33qx">https://doi.org/gg33qx</a> <br />
DOI: <a href="https://doi.org/10.1073/pnas.2008760117">10.1073/pnas.2008760117</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32576710">32576710</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7322063">PMC7322063</a></p>
</div>
<div id="ref-H6dfv6Wv">
<p>260. <a href="https://www.who.int/publications/i/item/report-of-the-who-china-joint-mission-on-coronavirus-disease-2019">https://www.who.int/publications/i/item/report-of-the-who-china-joint-mission-on-coronavirus-disease-2019</a></p>
</div>
<div id="ref-TuzwDCPt">
<p>261. <strong>The SOFA (Sepsis-related Organ Failure Assessment) score to describe organ dysfunction/failure</strong> <br />
J. -L. Vincent, R. Moreno, J. Takala, S. Willatts, A. De Mendonça, H. Bruining, C. K. Reinhart, P. M. Suter, L. G. Thijs<br />
<em>Intensive Care Medicine</em> (1996-07) <a href="https://doi.org/bpkxdw">https://doi.org/bpkxdw</a> <br />
DOI: <a href="https://doi.org/10.1007/bf01709751">10.1007/bf01709751</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/8844239">8844239</a></p>
</div>
<div id="ref-a0h975P6">
<p>262. <strong>The Third International Consensus Definitions for Sepsis and Septic Shock (Sepsis-3)</strong> <br />
Mervyn Singer, Clifford S. Deutschman, Christopher Warren Seymour, Manu Shankar-Hari, Djillali Annane, Michael Bauer, Rinaldo Bellomo, Gordon R. Bernard, Jean-Daniel Chiche, Craig M. Coopersmith, … Derek C. Angus<br />
<em>JAMA</em> (2016-02-23) <a href="https://doi.org/gdrcdh">https://doi.org/gdrcdh</a> <br />
DOI: <a href="https://doi.org/10.1001/jama.2016.0287">10.1001/jama.2016.0287</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26903338">26903338</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4968574">PMC4968574</a></p>
</div>
<div id="ref-jKKxa9Yy">
<p>263. <strong>COVID-19 and African Americans</strong> <br />
Clyde W. Yancy<br />
<em>JAMA</em> (2020-05-19) <a href="https://doi.org/ggv494">https://doi.org/ggv494</a> <br />
DOI: <a href="https://doi.org/10.1001/jama.2020.6548">10.1001/jama.2020.6548</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32293639">32293639</a></p>
</div>
<div id="ref-yK5bczRb">
<p>264. <strong>COVID-19 and Racial/Ethnic Disparities</strong> <br />
Monica Webb Hooper, Anna María Nápoles, Eliseo J. Pérez-Stable<br />
<em>JAMA</em> (2020-06-23) <a href="https://doi.org/ggvzqn">https://doi.org/ggvzqn</a> <br />
DOI: <a href="https://doi.org/10.1001/jama.2020.8598">10.1001/jama.2020.8598</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32391864">32391864</a></p>
</div>
<div id="ref-u4MMV8aJ">
<p>265. <strong>Covid-19: Black people and other minorities are hardest hit in US</strong> <br />
Owen Dyer<br />
<em>BMJ</em> (2020-04-14) <a href="https://doi.org/ggv5br">https://doi.org/ggv5br</a> <br />
DOI: <a href="https://doi.org/10.1136/bmj.m1483">10.1136/bmj.m1483</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32291262">32291262</a></p>
</div>
<div id="ref-JAo2QxY">
<p>266. <strong>Susceptibility of Southwestern American Indian Tribes to Coronavirus Disease 2019 (COVID‐19)</strong> <br />
Monika Kakol, Dona Upson, Akshay Sood<br />
<em>The Journal of Rural Health</em> (2020-06) <a href="https://doi.org/ggtzkq">https://doi.org/ggtzkq</a> <br />
DOI: <a href="https://doi.org/10.1111/jrh.12451">10.1111/jrh.12451</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32304251">32304251</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7264672">PMC7264672</a></p>
</div>
<div id="ref-12WZObWRf">
<p>267. <strong>The Fullest Look Yet at the Racial Inequity of Coronavirus</strong> <br />
Richard A. Oppel Jr, Robert Gebeloff, K. K. Rebecca Lai, Will Wright, Mitch Smith<br />
<em>The New York Times</em> (2020-07-05) <a href="https://www.nytimes.com/interactive/2020/07/05/us/coronavirus-latinos-african-americans-cdc-data.html">https://www.nytimes.com/interactive/2020/07/05/us/coronavirus-latinos-african-americans-cdc-data.html</a></p>
</div>
<div id="ref-Pp8GWnQm">
<p>268. <strong>Addressing inequities in COVID-19 morbidity and mortality: research and policy recommendations</strong> <br />
Monica L Wang, Pamela Behrman, Akilah Dulin, Monica L Baskin, Joanna Buscemi, Kassandra I Alcaraz, Carly M Goldstein, Tiffany L Carson, Megan Shen, Marian Fitzgibbon<br />
<em>Translational Behavioral Medicine</em> (2020-06) <a href="https://doi.org/gg3389">https://doi.org/gg3389</a> <br />
DOI: <a href="https://doi.org/10.1093/tbm/ibaa055">10.1093/tbm/ibaa055</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32542349">32542349</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7337775">PMC7337775</a></p>
</div>
<div id="ref-J9T8eR5H">
<p>269. <strong>Historical Environmental Racism, Structural Inequalities, and Dik’os Ntsaaígíí-19 (COVID-19) on Navajo Nation</strong> <br />
Nicholet A. Deschine Parkhurst, Kimberly R. Huyser, Aggie J. Yellow Horse<br />
<em>Journal of Indigenous Social Development</em> (2020-11-02) <a href="https://journalhosting.ucalgary.ca/index.php/jisd/article/view/70753">https://journalhosting.ucalgary.ca/index.php/jisd/article/view/70753</a></p>
</div>
<div id="ref-ODm1CaEq">
<p>270. <strong>Protect Indigenous peoples from COVID-19</strong> <br />
Lucas Ferrante, Philip M. Fearnside<br />
<em>Science</em> (2020-04-16) <a href="https://doi.org/gg3k6f">https://doi.org/gg3k6f</a> <br />
DOI: <a href="https://doi.org/10.1126/science.abc0073">10.1126/science.abc0073</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32299940">32299940</a></p>
</div>
<div id="ref-17gnxmOnf">
<p>271. <strong>Factors associated with COVID-19-related death using OpenSAFELY</strong> <br />
Elizabeth J. Williamson, Alex J. Walker, Krishnan Bhaskaran, Seb Bacon, Chris Bates, Caroline E. Morton, Helen J. Curtis, Amir Mehrkar, David Evans, Peter Inglesby, … Ben Goldacre<br />
<em>Nature</em> (2020-07-08) <a href="https://doi.org/gg39n7">https://doi.org/gg39n7</a> <br />
DOI: <a href="https://doi.org/10.1038/s41586-020-2521-4">10.1038/s41586-020-2521-4</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32640463">32640463</a></p>
</div>
<div id="ref-pwbrxYv8">
<p>272. <strong>Implications of biogeography of human populations for “race” and medicine</strong> <br />
Sarah A Tishkoff, Kenneth K Kidd<br />
<em>Nature Genetics</em> (2004-10-26) <a href="https://doi.org/d2xq92">https://doi.org/d2xq92</a> <br />
DOI: <a href="https://doi.org/10.1038/ng1438">10.1038/ng1438</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15507999">15507999</a></p>
</div>
<div id="ref-CMAPEmTp">
<p>273. <strong>African Genetic Diversity: Implications for Human Demographic History, Modern Human Origins, and Complex Disease Mapping</strong> <br />
Michael C. Campbell, Sarah A. Tishkoff<br />
<em>Annual Review of Genomics and Human Genetics</em> (2008-09) <a href="https://doi.org/cphggp">https://doi.org/cphggp</a> <br />
DOI: <a href="https://doi.org/10.1146/annurev.genom.9.081307.164258">10.1146/annurev.genom.9.081307.164258</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/18593304">18593304</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2953791">PMC2953791</a></p>
</div>
<div id="ref-q4yhes98">
<p>274. <strong>NIH must confront the use of race in science</strong> <br />
Michael Yudell, Dorothy Roberts, Rob DeSalle, Sarah Tishkoff, 70 signatories<br />
<em>Science</em> (2020-09-10) <a href="https://doi.org/ghcm7s">https://doi.org/ghcm7s</a> <br />
DOI: <a href="https://doi.org/10.1126/science.abd4842">10.1126/science.abd4842</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32913094">32913094</a></p>
</div>
<div id="ref-FbeY1L1s">
<p>275. <strong>Interferon-Induced Transmembrane Protein 3 Genetic Variant rs12252-C Associated With Disease Severity in Coronavirus Disease 2019</strong> <br />
Yonghong Zhang, Ling Qin, Yan Zhao, Ping Zhang, Bin Xu, Kang Li, Lianchun Liang, Chi Zhang, Yanchao Dai, Yingmei Feng, … Ronghua Jin<br />
<em>The Journal of Infectious Diseases</em> (2020-07-01) <a href="https://doi.org/ggv3tj">https://doi.org/ggv3tj</a> <br />
DOI: <a href="https://doi.org/10.1093/infdis/jiaa224">10.1093/infdis/jiaa224</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32348495">32348495</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7197559">PMC7197559</a></p>
</div>
<div id="ref-owevcpwb">
<p>276. <strong>Genomewide Association Study of Severe Covid-19 with Respiratory Failure</strong> <br />
David Ellinghaus, Frauke Degenhardt, Luis Bujanda, Maria Buti, Agustín Albillos, Pietro Invernizzi, Javier Fernández, Daniele Prati, Guido Baselli, Rosanna Asselta, … Tom H. Karlsen<br />
<em>New England Journal of Medicine</em> (2020-06-17) <a href="https://doi.org/gg2pqx">https://doi.org/gg2pqx</a> <br />
DOI: <a href="https://doi.org/10.1056/nejmoa2020283">10.1056/nejmoa2020283</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32558485">32558485</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7315890">PMC7315890</a></p>
</div>
<div id="ref-nn67UTjV">
<p>277. <strong>APOE e4 Genotype Predicts Severe COVID-19 in the UK Biobank Community Cohort</strong> <br />
Chia-Ling Kuo, Luke C Pilling, Janice L Atkins, Jane AH Masoli, João Delgado, George A Kuchel, David Melzer<br />
<em>The Journals of Gerontology: Series A</em> (2020-05-26) <a href="https://doi.org/ggx4ng">https://doi.org/ggx4ng</a> <br />
DOI: <a href="https://doi.org/10.1093/gerona/glaa131">10.1093/gerona/glaa131</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32451547">32451547</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7314139">PMC7314139</a></p>
</div>
<div id="ref-I2mIU0wn">
<p>278. <strong>Genome-wide CRISPR screen reveals host genes that regulate SARS-CoV-2 infection</strong> <br />
Jin Wei, Mia Madel Alfajaro, Ruth E. Hanna, Peter C. DeWeirdt, Madison S. Strine, William J. Lu-Culligan, Shang-Min Zhang, Vincent R. Graziano, Cameron O. Schmitz, Jennifer S. Chen, … Craig B. Wilen<br />
<em>Cold Spring Harbor Laboratory</em> (2020-06-17) <a href="https://doi.org/dzz3">https://doi.org/dzz3</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.06.16.155101">10.1101/2020.06.16.155101</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32869025">32869025</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7457610">PMC7457610</a></p>
</div>
<div id="ref-KbDdcSgo">
<p>279. <strong>New insights into genetic susceptibility of COVID-19: an ACE2 and TMPRSS2 polymorphism analysis</strong> <br />
Yuan Hou, Junfei Zhao, William Martin, Asha Kallianpur, Mina K. Chung, Lara Jehi, Nima Sharifi, Serpil Erzurum, Charis Eng, Feixiong Cheng<br />
<em>BMC Medicine</em> (2020-07-15) <a href="https://doi.org/gg445n">https://doi.org/gg445n</a> <br />
DOI: <a href="https://doi.org/10.1186/s12916-020-01673-z">10.1186/s12916-020-01673-z</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32664879">32664879</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7360473">PMC7360473</a></p>
</div>
<div id="ref-4SqaP0Ub">
<p>280. <strong>Seroprevalence of anti-SARS-CoV-2 IgG antibodies in Kenyan blood donors</strong> <br />
Sophie Uyoga, Ifedayo M. O. Adetifa, Henry K. Karanja, James Nyagwange, James Tuju, Perpetual Wanjiku, Rashid Aman, Mercy Mwangangi, Patrick Amoth, Kadondi Kasera, … George M Warimwe<br />
<em>Cold Spring Harbor Laboratory</em> (2020-07-29) <a href="https://doi.org/ghcm7p">https://doi.org/ghcm7p</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.07.27.20162693">10.1101/2020.07.27.20162693</a></p>
</div>
<div id="ref-eWIOKa0u">
<p>281. <strong>High SARS-CoV-2 seroprevalence in Health Care Workers but relatively low numbers of deaths in urban Malawi</strong> <br />
Marah Grace Chibwana, Khuzwayo Chidiwa Jere, Raphael kamng’ona, Jonathan Mandolo, Vincent Katunga-Phiri, Dumizulu Tembo, Ndaona Mitole, Samantha Musasa, Simon Sichone, Agness Lakudzala, … Kondwani Charles Jambo<br />
<em>Cold Spring Harbor Laboratory</em> (2020-08-05) <a href="https://doi.org/ghcm7q">https://doi.org/ghcm7q</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.07.30.20164970">10.1101/2020.07.30.20164970</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32766597">32766597</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7402052">PMC7402052</a></p>
</div>
<div id="ref-p4AYZss0">
<p>282. <strong>Africa’s pandemic puzzle: why so few cases and deaths?</strong> <br />
Linda Nordling<br />
<em>Science</em> (2020-08-13) <a href="https://doi.org/ghcm7r">https://doi.org/ghcm7r</a> <br />
DOI: <a href="https://doi.org/10.1126/science.369.6505.756">10.1126/science.369.6505.756</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32792376">32792376</a></p>
</div>
<div id="ref-xFKmPg6l">
<p>283. <strong>Are some ethnic groups more vulnerable to COVID-19 than others?</strong> <a href="https://www.ifs.org.uk/inequality/chapter/are-some-ethnic-groups-more-vulnerable-to-covid-19-than-others/">https://www.ifs.org.uk/inequality/chapter/are-some-ethnic-groups-more-vulnerable-to-covid-19-than-others/</a></p>
</div>
<div id="ref-hrOv770z">
<p>284. <strong>Association of Race With Mortality Among Patients Hospitalized With Coronavirus Disease 2019 (COVID-19) at 92 US Hospitals</strong> <br />
Baligh R. Yehia, Angela Winegar, Richard Fogel, Mohamad Fakih, Allison Ottenbacher, Christine Jesser, Angelo Bufalino, Ren-Huai Huang, Joseph Cacchione<br />
<em>JAMA Network Open</em> (2020-08-18) <a href="https://doi.org/ghcspt">https://doi.org/ghcspt</a> <br />
DOI: <a href="https://doi.org/10.1001/jamanetworkopen.2020.18039">10.1001/jamanetworkopen.2020.18039</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32809033">32809033</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7435340">PMC7435340</a></p>
</div>
<div id="ref-1AHtg71LX">
<p>285. <strong>Detection of 2019 novel coronavirus (2019-nCoV) by real-time RT-PCR</strong> <br />
Victor M Corman, Olfert Landt, Marco Kaiser, Richard Molenkamp, Adam Meijer, Daniel KW Chu, Tobias Bleicker, Sebastian Brünink, Julia Schneider, Marie Luisa Schmidt, … Christian Drosten<br />
<em>Eurosurveillance</em> (2020-01-23) <a href="https://doi.org/ggjs7g">https://doi.org/ggjs7g</a> <br />
DOI: <a href="https://doi.org/10.2807/1560-7917.es.2020.25.3.2000045">10.2807/1560-7917.es.2020.25.3.2000045</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31992387">31992387</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6988269">PMC6988269</a></p>
</div>
<div id="ref-fCZXxoas">
<p>286. <strong>Temporal profiles of viral load in posterior oropharyngeal saliva samples and serum antibody responses during infection by SARS-CoV-2: an observational cohort study</strong> <br />
Kelvin Kai-Wang To, Owen Tak-Yin Tsang, Wai-Shing Leung, Anthony Raymond Tam, Tak-Chiu Wu, David Christopher Lung, Cyril Chik-Yan Yip, Jian-Piao Cai, Jacky Man-Chun Chan, Thomas Shiu-Hong Chik, … Kwok-Yung Yuen<br />
<em>The Lancet Infectious Diseases</em> (2020-03) <a href="https://doi.org/ggp4qx">https://doi.org/ggp4qx</a> <br />
DOI: <a href="https://doi.org/10.1016/s1473-3099(20)30196-1">10.1016/s1473-3099(20)30196-1</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32213337">32213337</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7158907">PMC7158907</a></p>
</div>
<div id="ref-oNFnjRBI">
<p>287. <strong>Fecal specimen diagnosis 2019 novel coronavirus–infected pneumonia</strong> <br />
JingCheng Zhang, SaiBin Wang, YaDong Xue<br />
<em>Journal of Medical Virology</em> (2020-03-12) <a href="https://doi.org/ggpx6d">https://doi.org/ggpx6d</a> <br />
DOI: <a href="https://doi.org/10.1002/jmv.25742">10.1002/jmv.25742</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32124995">32124995</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7228355">PMC7228355</a></p>
</div>
<div id="ref-jr3z6jGU">
<p>288. <strong>Library preparation for next generation sequencing: A review of automation strategies</strong> <br />
J. F. Hess, T. A. Kohl, M. Kotrová, K. Rönsch, T. Paprotka, V. Mohr, T. Hutzenlaub, M. Brüggemann, R. Zengerle, S. Niemann, N. Paust<br />
<em>Biotechnology Advances</em> (2020-07) <a href="https://doi.org/ggth2v">https://doi.org/ggth2v</a> <br />
DOI: <a href="https://doi.org/10.1016/j.biotechadv.2020.107537">10.1016/j.biotechadv.2020.107537</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32199980">32199980</a></p>
</div>
<div id="ref-43modeVu">
<p>289. <strong>Diagnosing COVID-19: The Disease and Tools for Detection</strong> <br />
Buddhisha Udugama, Pranav Kadhiresan, Hannah N. Kozlowski, Ayden Malekjahani, Matthew Osborne, Vanessa Y. C. Li, Hongmin Chen, Samira Mubareka, Jonathan B. Gubbay, Warren C. W. Chan<br />
<em>ACS Nano</em> (2020-03-30) <a href="https://doi.org/ggq8ds">https://doi.org/ggq8ds</a> <br />
DOI: <a href="https://doi.org/10.1021/acsnano.0c02624">10.1021/acsnano.0c02624</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32223179">32223179</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7144809">PMC7144809</a></p>
</div>
<div id="ref-COEfjKo5">
<p>290. <strong>Molecular Diagnosis of a Novel Coronavirus (2019-nCoV) Causing an Outbreak of Pneumonia</strong> <br />
Daniel KW Chu, Yang Pan, Samuel MS Cheng, Kenrie PY Hui, Pavithra Krishnan, Yingzhi Liu, Daisy YM Ng, Carrie KC Wan, Peng Yang, Quanyi Wang, … Leo LM Poon<br />
<em>Clinical Chemistry</em> (2020-01-31) <a href="https://doi.org/ggnbpp">https://doi.org/ggnbpp</a> <br />
DOI: <a href="https://doi.org/10.1093/clinchem/hvaa029">10.1093/clinchem/hvaa029</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32031583">32031583</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7108203">PMC7108203</a></p>
</div>
<div id="ref-Hl2pASeg">
<p>291. <strong>dPCR: A Technology Review</strong> <br />
Phenix-Lan Quan, Martin Sauzade, Eric Brouzes<br />
<em>Sensors</em> (2018-04-20) <a href="https://doi.org/ggr39c">https://doi.org/ggr39c</a> <br />
DOI: <a href="https://doi.org/10.3390/s18041271">10.3390/s18041271</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29677144">29677144</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5948698">PMC5948698</a></p>
</div>
<div id="ref-2WgqQLF8">
<p>292. <strong>ddPCR: a more accurate tool for SARS-CoV-2 detection in low viral load specimens</strong> <br />
Tao Suo, Xinjin Liu, Jiangpeng Feng, Ming Guo, Wenjia Hu, Dong Guo, Hafiz Ullah, Yang Yang, Qiuhan Zhang, Xin Wang, … Yu Chen<br />
<em>Emerging Microbes &amp; Infections</em> (2020-06-07) <a href="https://doi.org/ggx2t2">https://doi.org/ggx2t2</a> <br />
DOI: <a href="https://doi.org/10.1080/22221751.2020.1772678">10.1080/22221751.2020.1772678</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32438868">32438868</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7448897">PMC7448897</a></p>
</div>
<div id="ref-UYD5T2mp">
<p>293. <strong>Highly accurate and sensitive diagnostic detection of SARS-CoV-2 by digital PCR</strong> <br />
Lianhua Dong, Junbo Zhou, Chunyan Niu, Quanyi Wang, Yang Pan, Sitong Sheng, Xia Wang, Yongzhuo Zhang, Jiayi Yang, Manqing Liu, … Xiang Fang<br />
<em>Cold Spring Harbor Laboratory</em> (2020-03-30) <a href="https://doi.org/ggqnqh">https://doi.org/ggqnqh</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.14.20036129">10.1101/2020.03.14.20036129</a></p>
</div>
<div id="ref-93NBANq2">
<p>294. <strong>Evaluation of COVID-19 RT-qPCR test in multi-sample pools</strong> <br />
Idan Yelin, Noga Aharony, Einat Shaer-Tamar, Amir Argoetti, Esther Messer, Dina Berenbaum, Einat Shafran, Areen Kuzli, Nagam Gandali, Tamar Hashimshony, … Roy Kishony<br />
<em>medRxiv</em> (2020-03-27) <a href="https://doi.org/ggrn74">https://doi.org/ggrn74</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.26.20039438">10.1101/2020.03.26.20039438</a></p>
</div>
<div id="ref-IQ5kUQEO">
<p>295. <strong>Analytical Validation of a COVID-19 qRT-PCR Detection Assay Using a 384-well Format and Three Extraction Methods</strong> <br />
Andrew C. Nelson, Benjamin Auch, Matthew Schomaker, Daryl M. Gohl, Patrick Grady, Darrell Johnson, Robyn Kincaid, Kylene E. Karnuth, Jerry Daniel, Jessica K. Fiege, … Sophia Yohe<br />
<em>Cold Spring Harbor Laboratory</em> (2020-04-05) <a href="https://doi.org/ggs45d">https://doi.org/ggs45d</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.04.02.022186">10.1101/2020.04.02.022186</a></p>
</div>
<div id="ref-vVBrQqWB">
<p>296. <strong>Nucleic acid detection with CRISPR-Cas13a/C2c2</strong> <br />
Jonathan S. Gootenberg, Omar O. Abudayyeh, Jeong Wook Lee, Patrick Essletzbichler, Aaron J. Dy, Julia Joung, Vanessa Verdine, Nina Donghia, Nichole M. Daringer, Catherine A. Freije, … Feng Zhang<br />
<em>Science</em> (2017-04-28) <a href="https://doi.org/f93x8p">https://doi.org/f93x8p</a> <br />
DOI: <a href="https://doi.org/10.1126/science.aam9321">10.1126/science.aam9321</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28408723">28408723</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5526198">PMC5526198</a></p>
</div>
<div id="ref-K6w6Llt2">
<p>297. <strong>Development and Evaluation of A CRISPR-based Diagnostic For 2019-novel Coronavirus</strong> <br />
Tieying Hou, Weiqi Zeng, Minling Yang, Wenjing Chen, Lili Ren, Jingwen Ai, Ji Wu, Yalong Liao, Xuejing Gou, Yongjun Li, … Teng Xu<br />
<em>Cold Spring Harbor Laboratory</em> (2020-02-23) <a href="https://doi.org/gg7km8">https://doi.org/gg7km8</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.02.22.20025460">10.1101/2020.02.22.20025460</a></p>
</div>
<div id="ref-zSefi0ye">
<p>298. <strong>CRISPR-based surveillance for COVID-19 using genomically-comprehensive machine learning design</strong> <br />
Hayden C. Metsky, Catherine A. Freije, Tinna-Solveig F. Kosoko-Thoroddsen, Pardis C. Sabeti, Cameron Myhrvold<br />
<em>Cold Spring Harbor Laboratory</em> (2020-03-02) <a href="https://doi.org/ggr3zf">https://doi.org/ggr3zf</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.02.26.967026">10.1101/2020.02.26.967026</a></p>
</div>
<div id="ref-GlLismDj">
<p>299. <strong>A Scalable, Easy-to-Deploy, Protocol for Cas13-Based Detection of SARS-CoV-2 Genetic Material</strong> <br />
Jennifer N. Rauch, Eric Valois, Sabrina C. Solley, Friederike Braig, Ryan S. Lach, Morgane Audouard, Jose Carlos Ponce-Rojas, Michael S. Costello, Naomi J. Baxter, Kenneth S. Kosik, … Maxwell Z. Wilson<br />
<em>Cold Spring Harbor Laboratory</em> (2020-08-29) <a href="https://doi.org/gg7km7">https://doi.org/gg7km7</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.04.20.052159">10.1101/2020.04.20.052159</a></p>
</div>
<div id="ref-XWCinLld">
<p>300. <strong>CRISPR–Cas12-based detection of SARS-CoV-2</strong> <br />
James P. Broughton, Xianding Deng, Guixia Yu, Clare L. Fasching, Venice Servellita, Jasmeet Singh, Xin Miao, Jessica A. Streithorst, Andrea Granados, Alicia Sotomayor-Gonzalez, … Charles Y. Chiu<br />
<em>Nature Biotechnology</em> (2020-04-16) <a href="https://doi.org/ggv47f">https://doi.org/ggv47f</a> <br />
DOI: <a href="https://doi.org/10.1038/s41587-020-0513-4">10.1038/s41587-020-0513-4</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32300245">32300245</a></p>
</div>
<div id="ref-4T3aId7a">
<p>301. <strong>An ultrasensitive, rapid, and portable coronavirus SARS-CoV-2 sequence detection method based on CRISPR-Cas12</strong> <br />
Curti Lucia, Pereyra-Bonnet Federico, Gimenez Carla Alejandra<br />
<em>Cold Spring Harbor Laboratory</em> (2020-03-02) <a href="https://doi.org/gg7km6">https://doi.org/gg7km6</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.02.29.971127">10.1101/2020.02.29.971127</a></p>
</div>
<div id="ref-Q4ylhpEC">
<p>302. <strong>All-in-One Dual CRISPR-Cas12a (AIOD-CRISPR) Assay: A Case for Rapid, Ultrasensitive and Visual Detection of Novel Coronavirus SARS-CoV-2 and HIV virus</strong> <br />
Xiong Ding, Kun Yin, Ziyue Li, Changchun Liu<br />
<em>Cold Spring Harbor Laboratory</em> (2020-03-21) <a href="https://doi.org/gg7km5">https://doi.org/gg7km5</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.19.998724">10.1101/2020.03.19.998724</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32511323">32511323</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7239053">PMC7239053</a></p>
</div>
<div id="ref-1FqpXpmAD">
<p>303. <strong>SARS-CoV-2 detection with CRISPR diagnostics</strong> <br />
Lu Guo, Xuehan Sun, Xinge Wang, Chen Liang, Haiping Jiang, Qingqin Gao, Moyu Dai, Bin Qu, Sen Fang, Yihuan Mao, … Wei Li<br />
<em>Cold Spring Harbor Laboratory</em> (2020-04-11) <a href="https://doi.org/gg7kns">https://doi.org/gg7kns</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.04.10.023358">10.1101/2020.04.10.023358</a></p>
</div>
<div id="ref-4y01WM4x">
<p>304. <strong>Electric-field-driven microfluidics for rapid CRISPR-based diagnostics and its application to detection of SARS-CoV-2</strong> <br />
Ashwin Ramachandran, Diego A. Huyke, Eesha Sharma, Malaya K. Sahoo, Niaz Banaei, Benjamin A. Pinsky, Juan G. Santiago<br />
<em>Cold Spring Harbor Laboratory</em> (2020-05-22) <a href="https://doi.org/gg7knt">https://doi.org/gg7knt</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.05.21.109637">10.1101/2020.05.21.109637</a></p>
</div>
<div id="ref-1BCUNYRvv">
<p>305. <strong>Rapid, sensitive and specific SARS coronavirus-2 detection: a multi-center comparison between standard qRT-PCR and CRISPR based DETECTR.</strong> <br />
Eelke Brandsma, Han JMP Verhagen, Thijs J. W. van de Laar, Eric C. J. Claas, Marion Cornelissen, Emile van den Akker<br />
<em>Cold Spring Harbor Laboratory</em> (2020-07-29) <a href="https://doi.org/gg7km4">https://doi.org/gg7km4</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.07.27.20147249">10.1101/2020.07.27.20147249</a></p>
</div>
<div id="ref-Fdc7seLs">
<p>306. <strong>Coronavirus and the race to distribute reliable diagnostics</strong> <br />
Cormac Sheridan<br />
<em>Nature Biotechnology</em> (2020-02-19) <a href="https://doi.org/ggm4nt">https://doi.org/ggm4nt</a> <br />
DOI: <a href="https://doi.org/10.1038/d41587-020-00002-2">10.1038/d41587-020-00002-2</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32265548">32265548</a></p>
</div>
<div id="ref-NT5mrVGY">
<p>307. <strong>The standard coronavirus test, if available, works well—but can new diagnostics help in this pandemic?</strong> <br />
Robert Service<br />
<em>Science</em> (2020-03-22) <a href="https://doi.org/ggq9wm">https://doi.org/ggq9wm</a> <br />
DOI: <a href="https://doi.org/10.1126/science.abb8400">10.1126/science.abb8400</a></p>
</div>
<div id="ref-AvQI3GSm">
<p>308. <strong>Laboratory Diagnosis of COVID-19: Current Issues and Challenges</strong> <br />
Yi-Wei Tang, Jonathan E. Schmitz, David H. Persing, Charles W. Stratton<br />
<em>Journal of Clinical Microbiology</em> (2020-05-26) <a href="https://doi.org/ggq7h8">https://doi.org/ggq7h8</a> <br />
DOI: <a href="https://doi.org/10.1128/jcm.00512-20">10.1128/jcm.00512-20</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32245835">32245835</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7269383">PMC7269383</a></p>
</div>
<div id="ref-11vIuqmS2">
<p>309. <strong>Negative Nasopharyngeal and Oropharyngeal Swab Does Not Rule Out COVID-19</strong> <br />
Poramed Winichakoon, Romanee Chaiwarith, Chalerm Liwsrisakun, Parichat Salee, Aree Goonna, Atikun Limsukon, Quanhathai Kaewpoowat<br />
<em>Journal of Clinical Microbiology</em> (2020-02-26) <a href="https://doi.org/ggpw9m">https://doi.org/ggpw9m</a> <br />
DOI: <a href="https://doi.org/10.1128/jcm.00297-20">10.1128/jcm.00297-20</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32102856">32102856</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7180262">PMC7180262</a></p>
</div>
<div id="ref-EUrJt06r">
<p>310. <strong>A systematic review of antibody mediated immunity to coronaviruses: antibody kinetics, correlates of protection, and association of antibody responses with severity of disease</strong> <br />
Angkana T. Huang, Bernardo Garcia-Carreras, Matt D. T. Hitchings, Bingyi Yang, Leah Katzelnick, Susan M Rattigan, Brooke Borgert, Carlos Moreno, Benjamin D. Solomon, Isabel Rodriguez-Barraquer, … Derek A. T. Cummings<br />
<em>Cold Spring Harbor Laboratory</em> (2020-04-17) <a href="https://doi.org/ggsfmz">https://doi.org/ggsfmz</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.04.14.20065771">10.1101/2020.04.14.20065771</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32511434">32511434</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7217088">PMC7217088</a></p>
</div>
<div id="ref-pGXsPCCz">
<p>311. <strong>Longitudinal profile of antibodies against SARS-coronavirus in SARS patients and their clinical significance</strong> <br />
Hongying MO, Guangqiao ZENG, Xiaolan REN, Hui LI, Changwen KE, Yaxia TAN, Chaoda CAI, Kefang LAI, Rongchang CHEN, Moira CHAN-YEUNG, Nanshan ZHONG<br />
<em>Respirology</em> (2006-01) <a href="https://doi.org/dn23vj">https://doi.org/dn23vj</a> <br />
DOI: <a href="https://doi.org/10.1111/j.1440-1843.2006.00783.x">10.1111/j.1440-1843.2006.00783.x</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16423201">16423201</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7192223">PMC7192223</a></p>
</div>
<div id="ref-11UpvND0K">
<p>312. <strong>Two‐Year Prospective Study of the Humoral Immune Response of Patients with Severe Acute Respiratory Syndrome</strong> <br />
Wei Liu, Arnaud Fontanet, Pan‐He Zhang, Lin Zhan, Zhong‐Tao Xin, Laurence Baril, Fang Tang, Hui Lv, Wu‐Chun Cao<br />
<em>The Journal of Infectious Diseases</em> (2006-03-15) <a href="https://doi.org/cmzn2k">https://doi.org/cmzn2k</a> <br />
DOI: <a href="https://doi.org/10.1086/500469">10.1086/500469</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16479513">16479513</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7109932">PMC7109932</a></p>
</div>
<div id="ref-FIOysZLl">
<p>313. <strong>The time course of the immune response to experimental coronavirus infection of man</strong> <br />
K. A. Callow, H. F. Parry, M. Sergeant, D. A. J. Tyrrell<br />
<em>Epidemiology and Infection</em> (2009-05-15) <a href="https://doi.org/c9pnmg">https://doi.org/c9pnmg</a> <br />
DOI: <a href="https://doi.org/10.1017/s0950268800048019">10.1017/s0950268800048019</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/2170159">2170159</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2271881">PMC2271881</a></p>
</div>
<div id="ref-JcpiBiQt">
<p>314. <strong>Loss of Bcl-6-Expressing T Follicular Helper Cells and Germinal Centers in COVID-19</strong> <br />
Naoki Kaneko, Hsiao-Hsuan Kuo, Julie Boucau, Jocelyn R. Farmer, Hugues Allard-Chamard, Vinay S. Mahajan, Alicja Piechocka-Trocha, Kristina Lefteri, Matthew Osborn, Julia Bals, … Shiv Pillai<br />
<em>Cell</em> (2020-10) <a href="https://doi.org/gg9rdv">https://doi.org/gg9rdv</a> <br />
DOI: <a href="https://doi.org/10.1016/j.cell.2020.08.025">10.1016/j.cell.2020.08.025</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32877699">32877699</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7437499">PMC7437499</a></p>
</div>
<div id="ref-ciLSdA3x">
<p>315. <strong>A Peptide-Based Magnetic Chemiluminescence Enzyme Immunoassay for Serological Diagnosis of Coronavirus Disease 2019</strong> <br />
Xue-fei Cai, Juan Chen, Jie- li Hu, Quan-xin Long, Hai-jun Deng, Ping Liu, Kai Fan, Pu Liao, Bei-zhong Liu, Gui-cheng Wu, … De-qiang Wang<br />
<em>The Journal of Infectious Diseases</em> (2020-07-15) <a href="https://doi.org/ggv2fx">https://doi.org/ggv2fx</a> <br />
DOI: <a href="https://doi.org/10.1093/infdis/jiaa243">10.1093/infdis/jiaa243</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32382737">32382737</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7239108">PMC7239108</a></p>
</div>
<div id="ref-n6sKSpIk">
<p>316. <strong>Cellex qSARS-CoV-2 IgG/IgM Rapid Test</strong> <br />
Cellex<br />
(2020-04-07) <a href="https://www.fda.gov/media/136625/download">https://www.fda.gov/media/136625/download</a></p>
</div>
<div id="ref-FFMfMDRN">
<p>317. <strong>Detection of antibodies against SARS‐CoV‐2 in patients with COVID‐19</strong> <br />
Zhe Du, Fengxue Zhu, Fuzheng Guo, Bo Yang, Tianbing Wang<br />
<em>Journal of Medical Virology</em> (2020-04-10) <a href="https://doi.org/ggq7m2">https://doi.org/ggq7m2</a> <br />
DOI: <a href="https://doi.org/10.1002/jmv.25820">10.1002/jmv.25820</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32243608">32243608</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7228383">PMC7228383</a></p>
</div>
<div id="ref-yJW1RVQb">
<p>318. <strong>A serological assay to detect SARS-CoV-2 seroconversion in humans</strong> <br />
Fatima Amanat, Daniel Stadlbauer, Shirin Strohmeier, Thi Nguyen, Veronika Chromikova, Meagan McMahon, Kaijun Jiang, Guha Asthagiri-Arunkumar, Denise Jurczyszak, Jose Polanco, … Thomas Moran<br />
<em>medRxiv</em> (2020-04-16) <a href="https://doi.org/ggpn83">https://doi.org/ggpn83</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.17.20037713">10.1101/2020.03.17.20037713</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32511441">32511441</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7239062">PMC7239062</a></p>
</div>
<div id="ref-Gk9WwhYB">
<p>319. <strong>Coronavirus testing is ramping up. Here are the new tests and how they work.</strong> <br />
Stephanie Pappas-Live Science Contributor 31 March 2020<br />
<em>livescience.com</em> <a href="https://www.livescience.com/coronavirus-tests-available.html">https://www.livescience.com/coronavirus-tests-available.html</a></p>
</div>
<div id="ref-91dto3z3">
<p>320. <strong>Strong associations and moderate predictive value of early symptoms for SARS-CoV-2 test positivity among healthcare workers, the Netherlands, March 2020</strong> <br />
Alma Tostmann, John Bradley, Teun Bousema, Wing-Kee Yiek, Minke Holwerda, Chantal Bleeker-Rovers, Jaap ten Oever, Corianne Meijer, Janette Rahamat-Langendoen, Joost Hopman, … Heiman Wertheim<br />
<em>Eurosurveillance</em> (2020-04-23) <a href="https://doi.org/ggthwx">https://doi.org/ggthwx</a> <br />
DOI: <a href="https://doi.org/10.2807/1560-7917.es.2020.25.16.2000508">10.2807/1560-7917.es.2020.25.16.2000508</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32347200">32347200</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7189649">PMC7189649</a></p>
</div>
<div id="ref-itCGVb5b">
<p>321. <strong>Correlation of Chest CT and RT-PCR Testing in Coronavirus Disease 2019 (COVID-19) in China: A Report of 1014 Cases</strong> <br />
Tao Ai, Zhenlu Yang, Hongyan Hou, Chenao Zhan, Chong Chen, Wenzhi Lv, Qian Tao, Ziyong Sun, Liming Xia<br />
<em>Radiology</em> (2020-02-26) <a href="https://doi.org/ggmw6p">https://doi.org/ggmw6p</a> <br />
DOI: <a href="https://doi.org/10.1148/radiol.2020200642">10.1148/radiol.2020200642</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32101510">32101510</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7233399">PMC7233399</a></p>
</div>
<div id="ref-tkuUFNIX">
<p>322. <strong>Performance of radiologists in differentiating COVID-19 from viral pneumonia on chest CT</strong> <br />
Harrison X. Bai, Ben Hsieh, Zeng Xiong, Kasey Halsey, Ji Whae Choi, Thi My Linh Tran, Ian Pan, Lin-Bo Shi, Dong-Cui Wang, Ji Mei, … Wei-Hua Liao<br />
<em>Radiology</em> (2020-03-10) <a href="https://doi.org/ggnqw4">https://doi.org/ggnqw4</a> <br />
DOI: <a href="https://doi.org/10.1148/radiol.2020200823">10.1148/radiol.2020200823</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32155105">32155105</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7233414">PMC7233414</a></p>
</div>
<div id="ref-nN9zfzPe">
<p>323. <strong>Covid-19: automatic detection from X-ray images utilizing transfer learning with convolutional neural networks</strong> <br />
Ioannis D. Apostolopoulos, Tzani A. Mpesiana<br />
<em>Physical and Engineering Sciences in Medicine</em> (2020-04-03) <a href="https://doi.org/ggs448">https://doi.org/ggs448</a> <br />
DOI: <a href="https://doi.org/10.1007/s13246-020-00865-4">10.1007/s13246-020-00865-4</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32524445">32524445</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7118364">PMC7118364</a></p>
</div>
<div id="ref-OB8BJiwj">
<p>324. <strong>Coronavirus Disease 2019 (COVID-19)</strong> <br />
CDC<br />
<em>Centers for Disease Control and Prevention</em> (2020-02-11) <a href="https://www.cdc.gov/coronavirus/2019-ncov/hcp/testing-overview.html">https://www.cdc.gov/coronavirus/2019-ncov/hcp/testing-overview.html</a></p>
</div>
<div id="ref-IldxlHwA">
<p>325. <strong>NY Forward: a guide to reopening New York &amp; building back better</strong> (2020-05-15) <a href="https://www.governor.ny.gov/sites/governor.ny.gov/files/atoms/files/NYForwardReopeningGuide.pdf">https://www.governor.ny.gov/sites/governor.ny.gov/files/atoms/files/NYForwardReopeningGuide.pdf</a></p>
</div>
<div id="ref-181QWa7HL">
<p>326. <strong>Neutralizing Monoclonal Antibodies as Promising Therapeutics against Middle East Respiratory Syndrome Coronavirus Infection</strong> <br />
Hui-Ju Han, Jian-Wei Liu, Hao Yu, Xue-Jie Yu<br />
<em>Viruses</em> (2018-11-30) <a href="https://doi.org/ggp87v">https://doi.org/ggp87v</a> <br />
DOI: <a href="https://doi.org/10.3390/v10120680">10.3390/v10120680</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30513619">30513619</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6315345">PMC6315345</a></p>
</div>
<div id="ref-eyBAUUCl">
<p>327. <strong>Nutraceuticals have potential for boosting the type 1 interferon response to RNA viruses including influenza and coronavirus</strong> <br />
Mark F. McCarty, James J. DiNicolantonio<br />
<em>Progress in Cardiovascular Diseases</em> (2020-05) <a href="https://doi.org/ggpwx2">https://doi.org/ggpwx2</a> <br />
DOI: <a href="https://doi.org/10.1016/j.pcad.2020.02.007">10.1016/j.pcad.2020.02.007</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32061635">32061635</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7130854">PMC7130854</a></p>
</div>
<div id="ref-GT7yDBjv">
<p>328. <strong>Reducing mortality from 2019-nCoV: host-directed therapies should be an option</strong> <br />
Alimuddin Zumla, David S Hui, Esam I Azhar, Ziad A Memish, Markus Maeurer<br />
<em>The Lancet</em> (2020-02) <a href="https://doi.org/ggkd3b">https://doi.org/ggkd3b</a> <br />
DOI: <a href="https://doi.org/10.1016/s0140-6736(20)30305-6">10.1016/s0140-6736(20)30305-6</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32035018">32035018</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7133595">PMC7133595</a></p>
</div>
<div id="ref-SSbnPnzT">
<p>329. <strong>Evidence-Based Medicine Data Lab COVID-19 TrialsTracker</strong> <br />
Nick DeVito, Peter Inglesby<br />
<em>GitHub</em> (2020-03-29) <a href="https://github.com/ebmdatalab/covid_trials_tracker-covid">https://github.com/ebmdatalab/covid_trials_tracker-covid</a> <br />
DOI: <a href="https://doi.org/10.5281/zenodo.3732709">10.5281/zenodo.3732709</a></p>
</div>
<div id="ref-14gthSuhm">
<p>330. <strong>Introduction to modern virology</strong> <br />
N. J. Dimmock, A. J. Easton, K. N. Leppard<br />
<em>Blackwell Pub</em> (2007) <br />
ISBN: <a href="https://worldcat.org/isbn/9781405136457">9781405136457</a></p>
</div>
<div id="ref-tn6j6IZ0">
<p>331. <strong>Coronaviruses</strong> <br />
Helena Jane Maier, Erica Bickerton, Paul Britton (editors)<br />
<em>Methods in Molecular Biology</em> (2015) <a href="https://doi.org/ggqfqx">https://doi.org/ggqfqx</a> <br />
DOI: <a href="https://doi.org/10.1007/978-1-4939-2438-7">10.1007/978-1-4939-2438-7</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25870870">25870870</a> · ISBN: <a href="https://worldcat.org/isbn/9781493924370">9781493924370</a></p>
</div>
<div id="ref-Sh1FwWps">
<p>332. <strong>The potential chemical structure of anti‐SARS‐CoV‐2 RNA‐dependent RNA polymerase</strong> <br />
Jrhau Lung, Yu‐Shih Lin, Yao‐Hsu Yang, Yu‐Lun Chou, Li‐Hsin Shu, Yu‐Ching Cheng, Hung Te Liu, Ching‐Yuan Wu<br />
<em>Journal of Medical Virology</em> (2020-03-18) <a href="https://doi.org/ggp6fm">https://doi.org/ggp6fm</a> <br />
DOI: <a href="https://doi.org/10.1002/jmv.25761">10.1002/jmv.25761</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32167173">32167173</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7228302">PMC7228302</a></p>
</div>
<div id="ref-IkAU62Pd">
<p>333. <strong>Favipiravir</strong> <br />
DrugBank<br />
(2020-06-12) <a href="https://www.drugbank.ca/drugs/DB12466">https://www.drugbank.ca/drugs/DB12466</a></p>
</div>
<div id="ref-EgoJKtW4">
<p>334. <strong>In Vitro and In Vivo Activities of Anti-Influenza Virus Compound T-705</strong> <br />
Y. Furuta, K. Takahashi, Y. Fukuda, M. Kuno, T. Kamiyama, K. Kozaki, N. Nomura, H. Egawa, S. Minami, Y. Watanabe, … K. Shiraki<br />
<em>Antimicrobial Agents and Chemotherapy</em> (2002-04-01) <a href="https://doi.org/cndw7n">https://doi.org/cndw7n</a> <br />
DOI: <a href="https://doi.org/10.1128/aac.46.4.977-981.2002">10.1128/aac.46.4.977-981.2002</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/11897578">11897578</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC127093">PMC127093</a></p>
</div>
<div id="ref-rG5RCRdJ">
<p>335. <strong>Efficacy of Orally Administered T-705 on Lethal Avian Influenza A (H5N1) Virus Infections in Mice</strong> <br />
R. W. Sidwell, D. L. Barnard, C. W. Day, D. F. Smee, K. W. Bailey, M.-H. Wong, J. D. Morrey, Y. Furuta<br />
<em>Antimicrobial Agents and Chemotherapy</em> (2006-12-28) <a href="https://doi.org/dm9xr2">https://doi.org/dm9xr2</a> <br />
DOI: <a href="https://doi.org/10.1128/aac.01051-06">10.1128/aac.01051-06</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17194832">17194832</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1803113">PMC1803113</a></p>
</div>
<div id="ref-14p3btkRH">
<p>336. <strong>Favipiravir (T-705), a broad spectrum inhibitor of viral RNA polymerase</strong> <br />
Yousuke FURUTA, Takashi KOMENO, Takaaki NAKAMURA<br />
<em>Proceedings of the Japan Academy, Series B</em> (2017) <a href="https://doi.org/gbxcxw">https://doi.org/gbxcxw</a> <br />
DOI: <a href="https://doi.org/10.2183/pjab.93.027">10.2183/pjab.93.027</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28769016">28769016</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5713175">PMC5713175</a></p>
</div>
<div id="ref-DpdKJbki">
<p>337. <strong>Mechanism of Action of T-705 against Influenza Virus</strong> <br />
Y. Furuta, K. Takahashi, M. Kuno-Maekawa, H. Sangawa, S. Uehara, K. Kozaki, N. Nomura, H. Egawa, K. Shiraki<br />
<em>Antimicrobial Agents and Chemotherapy</em> (2005-02-23) <a href="https://doi.org/dgbwdh">https://doi.org/dgbwdh</a> <br />
DOI: <a href="https://doi.org/10.1128/aac.49.3.981-986.2005">10.1128/aac.49.3.981-986.2005</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15728892">15728892</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC549233">PMC549233</a></p>
</div>
<div id="ref-9j7NVPJS">
<p>338. <strong>Activity of T-705 in a Hamster Model of Yellow Fever Virus Infection in Comparison with That of a Chemically Related Compound, T-1106</strong> <br />
Justin G. Julander, Kristiina Shafer, Donald F. Smee, John D. Morrey, Yousuke Furuta<br />
<em>Antimicrobial Agents and Chemotherapy</em> (2009-01) <a href="https://doi.org/brknds">https://doi.org/brknds</a> <br />
DOI: <a href="https://doi.org/10.1128/aac.01074-08">10.1128/aac.01074-08</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/18955536">18955536</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2612161">PMC2612161</a></p>
</div>
<div id="ref-IYEoCGHy">
<p>339. <strong>In Vitro and In Vivo Activities of T-705 against Arenavirus and Bunyavirus Infections</strong> <br />
Brian B. Gowen, Min-Hui Wong, Kie-Hoon Jung, Andrew B. Sanders, Michelle Mendenhall, Kevin W. Bailey, Yousuke Furuta, Robert W. Sidwell<br />
<em>Antimicrobial Agents and Chemotherapy</em> (2007-09) <a href="https://doi.org/d98c87">https://doi.org/d98c87</a> <br />
DOI: <a href="https://doi.org/10.1128/aac.00356-07">10.1128/aac.00356-07</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17606691">17606691</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2043187">PMC2043187</a></p>
</div>
<div id="ref-7d1TnpE3">
<p>340. <strong>Favipiravir (T-705) inhibits in vitro norovirus replication</strong> <br />
J. Rocha-Pereira, D. Jochmans, K. Dallmeier, P. Leyssen, M. S. J. Nascimento, J. Neyts<br />
<em>Biochemical and Biophysical Research Communications</em> (2012-08) <a href="https://doi.org/f369j7">https://doi.org/f369j7</a> <br />
DOI: <a href="https://doi.org/10.1016/j.bbrc.2012.07.034">10.1016/j.bbrc.2012.07.034</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22809499">22809499</a></p>
</div>
<div id="ref-15kQ4QYbx">
<p>341. <strong>T-705 (Favipiravir) Inhibition of Arenavirus Replication in Cell Culture</strong> <br />
Michelle Mendenhall, Andrew Russell, Terry Juelich, Emily L. Messina, Donald F. Smee, Alexander N. Freiberg, Michael R. Holbrook, Yousuke Furuta, Juan-Carlos de la Torre, Jack H. Nunberg, Brian B. Gowen<br />
<em>Antimicrobial Agents and Chemotherapy</em> (2011-02) <a href="https://doi.org/cppwsc">https://doi.org/cppwsc</a> <br />
DOI: <a href="https://doi.org/10.1128/aac.01219-10">10.1128/aac.01219-10</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21115797">21115797</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3028760">PMC3028760</a></p>
</div>
<div id="ref-MPJdSF1p">
<p>342. <strong>The evolution of nucleoside analogue antivirals: A review for chemists and non-chemists. Part 1: Early structural modifications to the nucleoside scaffold</strong> <br />
Katherine L. Seley-Radtke, Mary K. Yates<br />
<em>Antiviral Research</em> (2018-06) <a href="https://doi.org/gdpn35">https://doi.org/gdpn35</a> <br />
DOI: <a href="https://doi.org/10.1016/j.antiviral.2018.04.004">10.1016/j.antiviral.2018.04.004</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29649496">29649496</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6396324">PMC6396324</a></p>
</div>
<div id="ref-ukZRKllk">
<p>343. <strong>The Ambiguous Base-Pairing and High Substrate Efficiency of T-705 (Favipiravir) Ribofuranosyl 5′-Triphosphate towards Influenza A Virus Polymerase</strong> <br />
Zhinan Jin, Lucas K. Smith, Vivek K. Rajwanshi, Baek Kim, Jerome Deval<br />
<em>PLoS ONE</em> (2013-07-10) <a href="https://doi.org/f5br92">https://doi.org/f5br92</a> <br />
DOI: <a href="https://doi.org/10.1371/journal.pone.0068347">10.1371/journal.pone.0068347</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23874596">23874596</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3707847">PMC3707847</a></p>
</div>
<div id="ref-13Oz7yRPx">
<p>344. <strong>TEMPORARY REMOVAL: Experimental Treatment with Favipiravir for COVID-19: An Open-Label Control Study</strong> <br />
Qingxian Cai, Minghui Yang, Dongjing Liu, Jun Chen, Dan Shu, Junxia Xia, Xuejiao Liao, Yuanbo Gu, Qiue Cai, Yang Yang, … Lei Liu<br />
<em>Engineering</em> (2020-03) <a href="https://doi.org/ggpprd">https://doi.org/ggpprd</a> <br />
DOI: <a href="https://doi.org/10.1016/j.eng.2020.03.007">10.1016/j.eng.2020.03.007</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32346491">32346491</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7185795">PMC7185795</a></p>
</div>
<div id="ref-1BYoe0y2H">
<p>345. <strong>Remdesivir EUA Letter of Authorization</strong> <br />
Denise M Hinton<br />
(2020-05-01) <a href="https://www.fda.gov/media/137564/download">https://www.fda.gov/media/137564/download</a></p>
</div>
<div id="ref-yTCAmOyt">
<p>346. <strong>A Phase 3 Randomized Study to Evaluate the Safety and Antiviral Activity of Remdesivir (GS-5734™) in Participants With Severe COVID-19</strong> <br />
Gilead Sciences<br />
<em>clinicaltrials.gov</em> (2020-07-23) <a href="https://clinicaltrials.gov/ct2/show/NCT04292899">https://clinicaltrials.gov/ct2/show/NCT04292899</a></p>
</div>
<div id="ref-1ATTPbCCO">
<p>347. <strong>A Multicenter, Adaptive, Randomized Blinded Controlled Trial of the Safety and Efficacy of Investigational Therapeutics for the Treatment of COVID-19 in Hospitalized Adults</strong> <br />
National Institute of Allergy and Infectious Diseases (NIAID)<br />
<em>clinicaltrials.gov</em> (2020-11-01) <a href="https://clinicaltrials.gov/ct2/show/NCT04280705">https://clinicaltrials.gov/ct2/show/NCT04280705</a></p>
</div>
<div id="ref-DtpdBrgt">
<p>348. <strong>Remdesivir for the Treatment of Covid-19 — Preliminary Report</strong> <br />
John H. Beigel, Kay M. Tomashek, Lori E. Dodd, Aneesh K. Mehta, Barry S. Zingman, Andre C. Kalil, Elizabeth Hohmann, Helen Y. Chu, Annie Luetkemeyer, Susan Kline, … H. Clifford Lane<br />
<em>New England Journal of Medicine</em> (2020-05-22) <a href="https://doi.org/dwkd">https://doi.org/dwkd</a> <br />
DOI: <a href="https://doi.org/10.1056/nejmoa2007764">10.1056/nejmoa2007764</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32445440">32445440</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7262788">PMC7262788</a></p>
</div>
<div id="ref-16zLDIOyk">
<p>349. <strong>Compassionate Use of Remdesivir for Patients with Severe Covid-19</strong> <br />
Jonathan Grein, Norio Ohmagari, Daniel Shin, George Diaz, Erika Asperges, Antonella Castagna, Torsten Feldt, Gary Green, Margaret L. Green, François-Xavier Lescure, … Timothy Flanigan<br />
<em>New England Journal of Medicine</em> (2020-04-10) <a href="https://doi.org/ggrm99">https://doi.org/ggrm99</a> <br />
DOI: <a href="https://doi.org/10.1056/nejmoa2007016">10.1056/nejmoa2007016</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32275812">32275812</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7169476">PMC7169476</a></p>
</div>
<div id="ref-YaxKcqq8">
<p>350. <strong>The antiviral compound remdesivir potently inhibits RNA-dependent RNA polymerase from Middle East respiratory syndrome coronavirus</strong> <br />
Calvin J Gordon, Egor P Tchesnokov, Joy Y. Feng, Danielle P Porter, Matthias Gotte<br />
<em>Journal of Biological Chemistry</em> (2020-02-24) <a href="https://doi.org/ggqm6x">https://doi.org/ggqm6x</a> <br />
DOI: <a href="https://doi.org/10.1074/jbc.ac120.013056">10.1074/jbc.ac120.013056</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32094225">32094225</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7152756">PMC7152756</a></p>
</div>
<div id="ref-lIko9Imd">
<p>351. <strong>Coronavirus Susceptibility to the Antiviral Remdesivir (GS-5734) Is Mediated by the Viral Polymerase and the Proofreading Exoribonuclease</strong> <br />
Maria L. Agostini, Erica L. Andres, Amy C. Sims, Rachel L. Graham, Timothy P. Sheahan, Xiaotao Lu, Everett Clinton Smith, James Brett Case, Joy Y. Feng, Robert Jordan, … Mark R. Denison<br />
<em>mBio</em> (2018-03-06) <a href="https://doi.org/gc45v6">https://doi.org/gc45v6</a> <br />
DOI: <a href="https://doi.org/10.1128/mbio.00221-18">10.1128/mbio.00221-18</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29511076">29511076</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5844999">PMC5844999</a></p>
</div>
<div id="ref-IhdcvbzC">
<p>352. <strong>Remdesivir and chloroquine effectively inhibit the recently emerged novel coronavirus (2019-nCoV) in vitro</strong> <br />
Manli Wang, Ruiyuan Cao, Leike Zhang, Xinglou Yang, Jia Liu, Mingyue Xu, Zhengli Shi, Zhihong Hu, Wu Zhong, Gengfu Xiao<br />
<em>Cell Research</em> (2020-02-04) <a href="https://doi.org/ggkbsg">https://doi.org/ggkbsg</a> <br />
DOI: <a href="https://doi.org/10.1038/s41422-020-0282-0">10.1038/s41422-020-0282-0</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32020029">32020029</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7054408">PMC7054408</a></p>
</div>
<div id="ref-gnfviuGp">
<p>353. <strong>A Randomized, Controlled Trial of Ebola Virus Disease Therapeutics</strong> <br />
Sabue Mulangu, Lori E. Dodd, Richard T. Davey, Olivier Tshiani Mbaya, Michael Proschan, Daniel Mukadi, Mariano Lusakibanza Manzo, Didier Nzolo, Antoine Tshomba Oloma, Augustin Ibanda, … the PALM Writing Group<br />
<em>New England Journal of Medicine</em> (2019-12-12) <a href="https://doi.org/ggqmx4">https://doi.org/ggqmx4</a> <br />
DOI: <a href="https://doi.org/10.1056/nejmoa1910993">10.1056/nejmoa1910993</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31774950">31774950</a></p>
</div>
<div id="ref-1H0NPTMw9">
<p>354. <strong>Broad-spectrum antiviral GS-5734 inhibits both epidemic and zoonotic coronaviruses</strong> <br />
Timothy P. Sheahan, Amy C. Sims, Rachel L. Graham, Vineet D. Menachery, Lisa E. Gralinski, James B. Case, Sarah R. Leist, Krzysztof Pyrc, Joy Y. Feng, Iva Trantcheva, … Ralph S. Baric<br />
<em>Science Translational Medicine</em> (2017-06-28) <a href="https://doi.org/gc3grb">https://doi.org/gc3grb</a> <br />
DOI: <a href="https://doi.org/10.1126/scitranslmed.aal3653">10.1126/scitranslmed.aal3653</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28659436">28659436</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5567817">PMC5567817</a></p>
</div>
<div id="ref-D4HwvRMd">
<p>355. <strong>Did an experimental drug help a U.S. coronavirus patient?</strong> <br />
Jon Cohen<br />
<em>Science</em> (2020-03-13) <a href="https://doi.org/ggqm62">https://doi.org/ggqm62</a> <br />
DOI: <a href="https://doi.org/10.1126/science.abb7243">10.1126/science.abb7243</a></p>
</div>
<div id="ref-zoLViMNg">
<p>356. <strong>First 12 patients with coronavirus disease 2019 (COVID-19) in the United States</strong> <br />
Stephanie A. Kujawski, Karen K Wong, Jennifer P. Collins, Lauren Epstein, Marie E. Killerby, Claire M. Midgley, Glen R. Abedi, N. Seema Ahmed, Olivia Almendares, Francisco N. Alvarez, … Jing Zhang<br />
<em>Cold Spring Harbor Laboratory</em> (2020-03-12) <a href="https://doi.org/ggqm6z">https://doi.org/ggqm6z</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.09.20032896">10.1101/2020.03.09.20032896</a></p>
</div>
<div id="ref-gleq16qf">
<p>357. <strong>First Case of 2019 Novel Coronavirus in the United States</strong> <br />
Michelle L. Holshue, Chas DeBolt, Scott Lindquist, Kathy H. Lofy, John Wiesman, Hollianne Bruce, Christopher Spitters, Keith Ericson, Sara Wilkerson, Ahmet Tural, … Satish K. Pillai<br />
<em>New England Journal of Medicine</em> (2020-01-31) <a href="https://doi.org/ggjvr6">https://doi.org/ggjvr6</a> <br />
DOI: <a href="https://doi.org/10.1056/nejmoa2001191">10.1056/nejmoa2001191</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32004427">32004427</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7092802">PMC7092802</a></p>
</div>
<div id="ref-w5y1LWUR">
<p>358. <strong>Remdesivir for 5 or 10 Days in Patients with Severe Covid-19</strong> <br />
Jason D. Goldman, David C. B. Lye, David S. Hui, Kristen M. Marks, Raffaele Bruno, Rocio Montejano, Christoph D. Spinner, Massimo Galli, Mi-Young Ahn, Ronald G. Nahass, … Aruna Subramanian<br />
<em>New England Journal of Medicine</em> (2020-05-27) <a href="https://doi.org/ggz7qv">https://doi.org/ggz7qv</a> <br />
DOI: <a href="https://doi.org/10.1056/nejmoa2015301">10.1056/nejmoa2015301</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32459919">32459919</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7377062">PMC7377062</a></p>
</div>
<div id="ref-oso9r21L">
<p>359. <strong>A Phase 3 Randomized Study to Evaluate the Safety and Antiviral Activity of Remdesivir (GS-5734™) in Participants With Moderate COVID-19 Compared to Standard of Care Treatment</strong> <br />
Gilead Sciences<br />
<em>clinicaltrials.gov</em> (2020-07-23) <a href="https://clinicaltrials.gov/ct2/show/NCT04292730">https://clinicaltrials.gov/ct2/show/NCT04292730</a></p>
</div>
<div id="ref-1q4PmqTm">
<p>360. <strong>Multi-centre, adaptive, randomized trial of the safety and efficacy of treatments of COVID-19 in hospitalized adults</strong> <br />
EU Clinical Trials Register<br />
(2020-03-09) <a href="https://www.clinicaltrialsregister.eu/ctr-search/trial/2020-000936-23/FR">https://www.clinicaltrialsregister.eu/ctr-search/trial/2020-000936-23/FR</a></p>
</div>
<div id="ref-rUTsIlZs">
<p>361. <strong>A Trial of Remdesivir in Adults With Mild and Moderate COVID-19 - Full Text View - ClinicalTrials.gov</strong> <a href="https://clinicaltrials.gov/ct2/show/NCT04252664">https://clinicaltrials.gov/ct2/show/NCT04252664</a></p>
</div>
<div id="ref-JSoOOX98">
<p>362. <strong>A Phase 3 Randomized, Double-blind, Placebo-controlled, Multicenter Study to Evaluate the Efficacy and Safety of Remdesivir in Hospitalized Adult Patients With Severe COVID-19.</strong> <br />
Bin Cao<br />
<em>clinicaltrials.gov</em> (2020-04-13) <a href="https://clinicaltrials.gov/ct2/show/NCT04257656">https://clinicaltrials.gov/ct2/show/NCT04257656</a></p>
</div>
<div id="ref-1DTFVDWTn">
<p>363. <strong>FDA Approves First Treatment for COVID-19</strong> <br />
Office of the Commissioner<br />
<em>FDA</em> (2020-10-22) <a href="https://www.fda.gov/news-events/press-announcements/fda-approves-first-treatment-covid-19">https://www.fda.gov/news-events/press-announcements/fda-approves-first-treatment-covid-19</a></p>
</div>
<div id="ref-26dpptLw">
<p>364. <strong>Proteases Essential for Human Influenza Virus Entry into Cells and Their Inhibitors as Potential Therapeutic Agents</strong> <br />
Hiroshi Kido, Yuushi Okumura, Hiroshi Yamada, Trong Quang Le, Mihiro Yano<br />
<em>Current Pharmaceutical Design</em> (2007-02-01) <a href="https://doi.org/bts3xp">https://doi.org/bts3xp</a> <br />
DOI: <a href="https://doi.org/10.2174/138161207780162971">10.2174/138161207780162971</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17311557">17311557</a></p>
</div>
<div id="ref-f7csoyEu">
<p>365. <strong>Protease inhibitors targeting coronavirus and filovirus entry</strong> <br />
Yanchen Zhou, Punitha Vedantham, Kai Lu, Juliet Agudelo, Ricardo Carrion, Jerritt W. Nunneley, Dale Barnard, Stefan Pöhlmann, James H. McKerrow, Adam R. Renslo, Graham Simmons<br />
<em>Antiviral Research</em> (2015-04) <a href="https://doi.org/ggr984">https://doi.org/ggr984</a> <br />
DOI: <a href="https://doi.org/10.1016/j.antiviral.2015.01.011">10.1016/j.antiviral.2015.01.011</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25666761">25666761</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4774534">PMC4774534</a></p>
</div>
<div id="ref-4Sja6dIz">
<p>366. <strong>Structure of Mpro from SARS-CoV-2 and discovery of its inhibitors</strong> <br />
Zhenming Jin, Xiaoyu Du, Yechun Xu, Yongqiang Deng, Meiqin Liu, Yao Zhao, Bing Zhang, Xiaofeng Li, Leike Zhang, Chao Peng, … Haitao Yang<br />
<em>Nature</em> (2020-04-09) <a href="https://doi.org/ggrp42">https://doi.org/ggrp42</a> <br />
DOI: <a href="https://doi.org/10.1038/s41586-020-2223-y">10.1038/s41586-020-2223-y</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32272481">32272481</a></p>
</div>
<div id="ref-15JYG0AIU">
<p>367. <strong>Design of Wide-Spectrum Inhibitors Targeting Coronavirus Main Proteases</strong> <br />
Haitao Yang, Weiqing Xie, Xiaoyu Xue, Kailin Yang, Jing Ma, Wenxue Liang, Qi Zhao, Zhe Zhou, Duanqing Pei, John Ziebuhr, … Zihe Rao<br />
<em>PLoS Biology</em> (2005-09-06) <a href="https://doi.org/bcm9k7">https://doi.org/bcm9k7</a> <br />
DOI: <a href="https://doi.org/10.1371/journal.pbio.0030324">10.1371/journal.pbio.0030324</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16128623">16128623</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1197287">PMC1197287</a></p>
</div>
<div id="ref-XfOTPQPJ">
<p>368. <strong>The newly emerged SARS-Like coronavirus HCoV-EMC also has an “Achilles’ heel”: current effective inhibitor targeting a 3C-like protease</strong> <br />
Zhilin Ren, Liming Yan, Ning Zhang, Yu Guo, Cheng Yang, Zhiyong Lou, Zihe Rao<br />
<em>Protein &amp; Cell</em> (2013-04-03) <a href="https://doi.org/ggr7vh">https://doi.org/ggr7vh</a> <br />
DOI: <a href="https://doi.org/10.1007/s13238-013-2841-3">10.1007/s13238-013-2841-3</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23549610">23549610</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4875521">PMC4875521</a></p>
</div>
<div id="ref-10NcfBSiT">
<p>369. <strong>Crystal structure of SARS-CoV-2 main protease provides a basis for design of improved α-ketoamide inhibitors</strong> <br />
Linlin Zhang, Daizong Lin, Xinyuanyuan Sun, Ute Curth, Christian Drosten, Lucie Sauerhering, Stephan Becker, Katharina Rox, Rolf Hilgenfeld<br />
<em>Science</em> (2020-03-20) <a href="https://doi.org/ggp9sb">https://doi.org/ggp9sb</a> <br />
DOI: <a href="https://doi.org/10.1126/science.abb3405">10.1126/science.abb3405</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32198291">32198291</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7164518">PMC7164518</a></p>
</div>
<div id="ref-17EErh01r">
<p>370. <strong>Ebselen, a promising antioxidant drug: mechanisms of action and targets of biological pathways</strong> <br />
Gajendra Kumar Azad, Raghuvir S. Tomar<br />
<em>Molecular Biology Reports</em> (2014-05-28) <a href="https://doi.org/f6cnq3">https://doi.org/f6cnq3</a> <br />
DOI: <a href="https://doi.org/10.1007/s11033-014-3417-x">10.1007/s11033-014-3417-x</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24867080">24867080</a></p>
</div>
<div id="ref-17QBZngoP">
<p>371. <strong>Safety and efficacy of ebselen for the prevention of noise-induced hearing loss: a randomised, double-blind, placebo-controlled, phase 2 trial</strong> <br />
Jonathan Kil, Edward Lobarinas, Christopher Spankovich, Scott K Griffiths, Patrick J Antonelli, Eric D Lynch, Colleen G Le Prell<br />
<em>The Lancet</em> (2017-09) <a href="https://doi.org/gbwnbv">https://doi.org/gbwnbv</a> <br />
DOI: <a href="https://doi.org/10.1016/s0140-6736(17)31791-9">10.1016/s0140-6736(17)31791-9</a></p>
</div>
<div id="ref-FBE30Uqz">
<p>372. <strong>Potential therapeutic use of ebselen for COVID-19 and other respiratory viral infections</strong> <br />
Helmut Sies, Michael J. Parnham<br />
<em>Free Radical Biology and Medicine</em> (2020-08) <a href="https://doi.org/ghdx7s">https://doi.org/ghdx7s</a> <br />
DOI: <a href="https://doi.org/10.1016/j.freeradbiomed.2020.06.032">10.1016/j.freeradbiomed.2020.06.032</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32598985">32598985</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7319625">PMC7319625</a></p>
</div>
<div id="ref-1355hoYr1">
<p>373. <strong>Selenium Deficiency Is Associated with Mortality Risk from COVID-19</strong> <br />
Arash Moghaddam, Raban Arved Heller, Qian Sun, Julian Seelig, Asan Cherkezov, Linda Seibert, Julian Hackler, Petra Seemann, Joachim Diegmann, Maximilian Pilz, … Lutz Schomburg<br />
<em>Nutrients</em> (2020-07-16) <a href="https://doi.org/gg5kbc">https://doi.org/gg5kbc</a> <br />
DOI: <a href="https://doi.org/10.3390/nu12072098">10.3390/nu12072098</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32708526">32708526</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7400921">PMC7400921</a></p>
</div>
<div id="ref-bvMB3ySR">
<p>374. <strong>Selenium and viral infection: are there lessons for COVID-19?</strong> <br />
Giovanna Bermano, Catherine Méplan, Derry K. Mercer, John E. Hesketh<br />
<em>British Journal of Nutrition</em> (2020-08-06) <a href="https://doi.org/ghdx7w">https://doi.org/ghdx7w</a> <br />
DOI: <a href="https://doi.org/10.1017/s0007114520003128">10.1017/s0007114520003128</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32758306">32758306</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7503044">PMC7503044</a></p>
</div>
<div id="ref-11F21sY2C">
<p>375. <strong>Selenium and selenoproteins in viral infection with potential relevance to COVID-19</strong> <br />
Jinsong Zhang, Ramy Saad, Ethan Will Taylor, Margaret P. Rayman<br />
<em>Redox Biology</em> (2020-10) <a href="https://doi.org/ghdx7t">https://doi.org/ghdx7t</a> <br />
DOI: <a href="https://doi.org/10.1016/j.redox.2020.101715">10.1016/j.redox.2020.101715</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32992282">32992282</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7481318">PMC7481318</a></p>
</div>
<div id="ref-vXFGEybh">
<p>376. <strong>Target discovery of ebselen with a biotinylated probe</strong> <br />
Zhenzhen Chen, Zhongyao Jiang, Nan Chen, Qian Shi, Lili Tong, Fanpeng Kong, Xiufen Cheng, Hao Chen, Chu Wang, Bo Tang<br />
<em>Chemical Communications</em> (2018) <a href="https://doi.org/ggrtcm">https://doi.org/ggrtcm</a> <br />
DOI: <a href="https://doi.org/10.1039/c8cc04258f">10.1039/c8cc04258f</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30091742">30091742</a></p>
</div>
<div id="ref-FT3Nw9xJ">
<p>377. <strong>Curing a viral infection by targeting the host: The example of cyclophilin inhibitors</strong> <br />
Kai Lin, Philippe Gallay<br />
<em>Antiviral Research</em> (2013-07) <a href="https://doi.org/f4237c">https://doi.org/f4237c</a> <br />
DOI: <a href="https://doi.org/10.1016/j.antiviral.2013.03.020">10.1016/j.antiviral.2013.03.020</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23578729">23578729</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4332838">PMC4332838</a></p>
</div>
<div id="ref-CIeWPFHG">
<p>378. <strong>Lisinopril - Drug Usage Statistics</strong> <br />
ClinCalc DrugStats Database<br />
<a href="https://clincalc.com/DrugStats/Drugs/Lisinopril">https://clincalc.com/DrugStats/Drugs/Lisinopril</a></p>
</div>
<div id="ref-pbfxOvnK">
<p>379. <strong>Hypertension Hot Potato — Anatomy of the Angiotensin-Receptor Blocker Recalls</strong> <br />
J. Brian Byrd, Glenn M. Chertow, Vivek Bhalla<br />
<em>New England Journal of Medicine</em> (2019-04-25) <a href="https://doi.org/ggvc7g">https://doi.org/ggvc7g</a> <br />
DOI: <a href="https://doi.org/10.1056/nejmp1901657">10.1056/nejmp1901657</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30865819">30865819</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7066505">PMC7066505</a></p>
</div>
<div id="ref-17xTGqKp2">
<p>380. <strong>Hypertension, the renin–angiotensin system, and the risk of lower respiratory tract infections and lung injury: implications for COVID-19</strong> <br />
Reinhold Kreutz, Engi Abd El-Hady Algharably, Michel Azizi, Piotr Dobrowolski, Tomasz Guzik, Andrzej Januszewicz, Alexandre Persu, Aleksander Prejbisz, Thomas Günther Riemer, Ji-Guang Wang, Michel Burnier<br />
<em>Cardiovascular Research</em> (2020-08-01) <a href="https://doi.org/ggtwpj">https://doi.org/ggtwpj</a> <br />
DOI: <a href="https://doi.org/10.1093/cvr/cvaa097">10.1093/cvr/cvaa097</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32293003">32293003</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7184480">PMC7184480</a></p>
</div>
<div id="ref-ZhhV4ztO">
<p>381. <strong>Angiotensin converting enzyme 2 activity and human atrial fibrillation: increased plasma angiotensin converting enzyme 2 activity is associated with atrial fibrillation and more advanced left atrial structural remodelling</strong> <br />
Tomos E. Walters, Jonathan M. Kalman, Sheila K. Patel, Megan Mearns, Elena Velkoska, Louise M. Burrell<br />
<em>Europace</em> (2016-10-12) <a href="https://doi.org/gbt2jw">https://doi.org/gbt2jw</a> <br />
DOI: <a href="https://doi.org/10.1093/europace/euw246">10.1093/europace/euw246</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27738071">27738071</a></p>
</div>
<div id="ref-10q7vcaWw">
<p>382. <strong>Cardiovascular Disease, Drug Therapy, and Mortality in Covid-19</strong> <br />
Mandeep R. Mehra, Sapan S. Desai, SreyRam Kuy, Timothy D. Henry, Amit N. Patel<br />
<em>New England Journal of Medicine</em> (2020-06-18) <a href="https://doi.org/ggtp6v">https://doi.org/ggtp6v</a> <br />
DOI: <a href="https://doi.org/10.1056/nejmoa2007621">10.1056/nejmoa2007621</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32356626">32356626</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7206931">PMC7206931</a></p>
</div>
<div id="ref-19quOF2kK">
<p>383. <strong>Response by Cohen et al to Letter Regarding Article, “Association of Inpatient Use of Angiotensin-Converting Enzyme Inhibitors and Angiotensin II Receptor Blockers With Mortality Among Patients With Hypertension Hospitalized With COVID-19”</strong> <br />
Jordana B. Cohen, Thomas C. Hanff, Andrew M. South, Matthew A. Sparks, Swapnil Hiremath, Adam P. Bress, J. Brian Byrd, Julio A. Chirinos<br />
<em>Circulation Research</em> (2020-06-05) <a href="https://doi.org/gg3xsg">https://doi.org/gg3xsg</a> <br />
DOI: <a href="https://doi.org/10.1161/circresaha.120.317205">10.1161/circresaha.120.317205</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32496917">32496917</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7265880">PMC7265880</a></p>
</div>
<div id="ref-k5xEN3Ke">
<p>384. <strong>Sound Science before Quick Judgement Regarding RAS Blockade in COVID-19</strong> <br />
Matthew A. Sparks, Andrew South, Paul Welling, J. Matt Luther, Jordana Cohen, James Brian Byrd, Louise M. Burrell, Daniel Batlle, Laurie Tomlinson, Vivek Bhalla, … Swapnil Hiremath<br />
<em>Clinical Journal of the American Society of Nephrology</em> (2020-05-07) <a href="https://doi.org/ggq8gn">https://doi.org/ggq8gn</a> <br />
DOI: <a href="https://doi.org/10.2215/cjn.03530320">10.2215/cjn.03530320</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32220930">32220930</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7269218">PMC7269218</a></p>
</div>
<div id="ref-H0oEjZR8">
<p>385. <strong>Elimination or Prolongation of ACE Inhibitors and ARB in Coronavirus Disease 2019 - Full Text View - ClinicalTrials.gov</strong> <a href="https://clinicaltrials.gov/ct2/show/NCT04338009">https://clinicaltrials.gov/ct2/show/NCT04338009</a></p>
</div>
<div id="ref-kuUcA1G9">
<p>386. <strong>Stopping ACE-inhibitors in COVID-19 - Full Text View - ClinicalTrials.gov</strong> <a href="https://clinicaltrials.gov/ct2/show/NCT04353596">https://clinicaltrials.gov/ct2/show/NCT04353596</a></p>
</div>
<div id="ref-3ccJAyrG">
<p>387. <strong>Losartan for Patients With COVID-19 Not Requiring Hospitalization - Full Text View - ClinicalTrials.gov</strong> <a href="https://clinicaltrials.gov/ct2/show/NCT04311177">https://clinicaltrials.gov/ct2/show/NCT04311177</a></p>
</div>
<div id="ref-NCb9Wq2z">
<p>388. <strong>Losartan for Patients With COVID-19 Requiring Hospitalization - Full Text View - ClinicalTrials.gov</strong> <a href="https://clinicaltrials.gov/ct2/show/NCT04312009">https://clinicaltrials.gov/ct2/show/NCT04312009</a></p>
</div>
<div id="ref-5fkIarR7">
<p>389. <strong>The CORONAvirus Disease 2019 Angiotensin Converting Enzyme Inhibitor/Angiotensin Receptor Blocker InvestigatiON (CORONACION) Randomized Clinical Trial</strong> <br />
Prof John William McEvoy<br />
<em>clinicaltrials.gov</em> (2020-06-26) <a href="https://clinicaltrials.gov/ct2/show/NCT04330300">https://clinicaltrials.gov/ct2/show/NCT04330300</a></p>
</div>
<div id="ref-c7V9rAFR">
<p>390. <strong>Ramipril for the Treatment of COVID-19 - Full Text View - ClinicalTrials.gov</strong> <a href="https://clinicaltrials.gov/ct2/show/NCT04366050">https://clinicaltrials.gov/ct2/show/NCT04366050</a></p>
</div>
<div id="ref-1Hrsy8Hiq">
<p>391. <strong>The Coronavirus Conundrum: ACE2 and Hypertension Edition</strong> <br />
Matthew Sparks, Swapnil Hiremath<br />
<em>NephJC</em> <a href="http://www.nephjc.com/news/covidace2">http://www.nephjc.com/news/covidace2</a></p>
</div>
<div id="ref-ggYRK4FU">
<p>392. <strong>Lysosomotropic agents as HCV entry inhibitors</strong> <br />
Usman A Ashfaq, Tariq Javed, Sidra Rehman, Zafar Nawaz, Sheikh Riazuddin<br />
<em>Virology Journal</em> (2011-04-12) <a href="https://doi.org/dr5g4m">https://doi.org/dr5g4m</a> <br />
DOI: <a href="https://doi.org/10.1186/1743-422x-8-163">10.1186/1743-422x-8-163</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21481279">21481279</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3090357">PMC3090357</a></p>
</div>
<div id="ref-16IRfGFKb">
<p>393. <strong>New concepts in antimalarial use and mode of action in dermatology</strong> <br />
Sunil Kalia, Jan P Dutz<br />
<em>Dermatologic Therapy</em> (2007-07) <a href="https://doi.org/fv69cb">https://doi.org/fv69cb</a> <br />
DOI: <a href="https://doi.org/10.1111/j.1529-8019.2007.00131.x">10.1111/j.1529-8019.2007.00131.x</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17970883">17970883</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7163426">PMC7163426</a></p>
</div>
<div id="ref-x6nInyCf">
<p>394. <strong>Chloroquine is a potent inhibitor of SARS coronavirus infection and spread</strong> <br />
Martin J Vincent, Eric Bergeron, Suzanne Benjannet, Bobbie R Erickson, Pierre E Rollin, Thomas G Ksiazek, Nabil G Seidah, Stuart T Nichol<br />
<em>Virology Journal</em> (2005) <a href="https://doi.org/dvbds4">https://doi.org/dvbds4</a> <br />
DOI: <a href="https://doi.org/10.1186/1743-422x-2-69">10.1186/1743-422x-2-69</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16115318">16115318</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1232869">PMC1232869</a></p>
</div>
<div id="ref-1GBsnkeLK">
<p>395. <strong>In Vitro Antiviral Activity and Projection of Optimized Dosing Design of Hydroxychloroquine for the Treatment of Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2)</strong> <br />
Xueting Yao, Fei Ye, Miao Zhang, Cheng Cui, Baoying Huang, Peihua Niu, Xu Liu, Li Zhao, Erdan Dong, Chunli Song, … Dongyang Liu<br />
<em>Clinical Infectious Diseases</em> (2020-03-09) <a href="https://doi.org/ggpx7z">https://doi.org/ggpx7z</a> <br />
DOI: <a href="https://doi.org/10.1093/cid/ciaa237">10.1093/cid/ciaa237</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32150618">32150618</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7108130">PMC7108130</a></p>
</div>
<div id="ref-nEaG534R">
<p>396. <strong>Mechanism of Action of Hydroxychloroquine in the Antiphospholipid Syndrome</strong> <br />
Nadine Müller-Calleja, Davit Manukyan, Wolfram Ruf, Karl Lackner<br />
<em>Blood</em> (2016-12-02) <a href="https://doi.org/ggrm82">https://doi.org/ggrm82</a> <br />
DOI: <a href="https://doi.org/10.1182/blood.v128.22.5023.5023">10.1182/blood.v128.22.5023.5023</a></p>
</div>
<div id="ref-rL8owM5Y">
<p>397. <strong>14th International Congress on Antiphospholipid Antibodies Task Force Report on Antiphospholipid Syndrome Treatment Trends</strong> <br />
Doruk Erkan, Cassyanne L. Aguiar, Danieli Andrade, Hannah Cohen, Maria J. Cuadrado, Adriana Danowski, Roger A. Levy, Thomas L. Ortel, Anisur Rahman, Jane E. Salmon, … Michael D. Lockshin<br />
<em>Autoimmunity Reviews</em> (2014-06) <a href="https://doi.org/ggp8r8">https://doi.org/ggp8r8</a> <br />
DOI: <a href="https://doi.org/10.1016/j.autrev.2014.01.053">10.1016/j.autrev.2014.01.053</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24468415">24468415</a></p>
</div>
<div id="ref-134vCyU3E">
<p>398. <strong>What is the role of hydroxychloroquine in reducing thrombotic risk in patients with antiphospholipid antibodies?</strong> <br />
Tzu-Fei Wang, Wendy Lim<br />
<em>Hematology</em> (2016-12-02) <a href="https://doi.org/ggrn3k">https://doi.org/ggrn3k</a> <br />
DOI: <a href="https://doi.org/10.1182/asheducation-2016.1.714">10.1182/asheducation-2016.1.714</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27913551">27913551</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6142483">PMC6142483</a></p>
</div>
<div id="ref-ovHLMvCi">
<p>399. <strong>Hydroxychloroquine and azithromycin as a treatment of COVID-19: results of an open-label non-randomized clinical trial</strong> <br />
Philippe Gautret, Jean-Christophe Lagier, Philippe Parola, Van Thuan Hoang, Line Meddeb, Morgane Mailhe, Barbara Doudier, Johan Courjon, Valérie Giordanengo, Vera Esteves Vieira, … Didier Raoult<br />
<em>International Journal of Antimicrobial Agents</em> (2020-03) <a href="https://doi.org/dp7d">https://doi.org/dp7d</a> <br />
DOI: <a href="https://doi.org/10.1016/j.ijantimicag.2020.105949">10.1016/j.ijantimicag.2020.105949</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32205204">32205204</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7102549">PMC7102549</a></p>
</div>
<div id="ref-10J3hBiMo">
<p>400. <strong>Official Statement from International Society of Antimicrobial Chemotherapy</strong> <br />
Andreas Voss<br />
(2020-04-03) <a href="https://www.isac.world/news-and-publications/official-isac-statement">https://www.isac.world/news-and-publications/official-isac-statement</a></p>
</div>
<div id="ref-18KmvBOFr">
<p>401. <strong>No Evidence of Rapid Antiviral Clearance or Clinical Benefit with the Combination of Hydroxychloroquine and Azithromycin in Patients with Severe COVID-19 Infection</strong> <br />
Jean Michel Molina, Constance Delaugerre, Jerome Le Goff, Breno Mela-Lima, Diane Ponscarme, Lauriane Goldwirt, Nathalie de Castro<br />
<em>Médecine et Maladies Infectieuses</em> (2020-03) <a href="https://doi.org/ggqzrb">https://doi.org/ggqzrb</a> <br />
DOI: <a href="https://doi.org/10.1016/j.medmal.2020.03.006">10.1016/j.medmal.2020.03.006</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32240719">32240719</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7195369">PMC7195369</a></p>
</div>
<div id="ref-wzHb7Mj5">
<p>402. <strong>Efficacy of hydroxychloroquine in patients with COVID-19: results of a randomized clinical trial</strong> <br />
Zhaowei Chen, Jijia Hu, Zongwei Zhang, Shan Jiang, Shoumeng Han, Dandan Yan, Ruhong Zhuang, Ben Hu, Zhan Zhang<br />
<em>medRxiv</em> (2020-03-31) <a href="https://doi.org/ggqm4v">https://doi.org/ggqm4v</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.22.20040758">10.1101/2020.03.22.20040758</a></p>
</div>
<div id="ref-XdCJ4u8O">
<p>403. <strong>The Extent and Consequences of P-Hacking in Science</strong> <br />
Megan L. Head, Luke Holman, Rob Lanfear, Andrew T. Kahn, Michael D. Jennions<br />
<em>PLOS Biology</em> (2015-03-13) <a href="https://doi.org/4z7">https://doi.org/4z7</a> <br />
DOI: <a href="https://doi.org/10.1371/journal.pbio.1002106">10.1371/journal.pbio.1002106</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25768323">25768323</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4359000">PMC4359000</a></p>
</div>
<div id="ref-o5gwQA7W">
<p>404. <strong>A pilot study of hydroxychloroquine in treatment of patients with common coronavirus disease-19 (COVID-19)</strong> <br />
CHEN Jun, LIU Danping, LIU Li, LIU Ping, XU Qingnian, XIA Lu, LING Yun, HUANG Dan, SONG Shuli, ZHANG Dandan, … LU Hongzhou<br />
<em>Journal of Zhejiang University (Medical Sciences)</em> (2020-03) <a href="https://doi.org/10.3785/j.issn.1008-9292.2020.03.03">https://doi.org/10.3785/j.issn.1008-9292.2020.03.03</a> <br />
DOI: <a href="https://doi.org/10.3785/j.issn.1008-9292.2020.03.03">10.3785/j.issn.1008-9292.2020.03.03</a></p>
</div>
<div id="ref-aiItPxSY">
<p>405. <strong>Breakthrough: Chloroquine phosphate has shown apparent efficacy in treatment of COVID-19 associated pneumonia in clinical studies</strong> <br />
Jianjun Gao, Zhenxue Tian, Xu Yang<br />
<em>BioScience Trends</em> (2020) <a href="https://doi.org/ggm3mv">https://doi.org/ggm3mv</a> <br />
DOI: <a href="https://doi.org/10.5582/bst.2020.01047">10.5582/bst.2020.01047</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32074550">32074550</a></p>
</div>
<div id="ref-1BdQKtXnY">
<p>406. <strong>Targeting the Endocytic Pathway and Autophagy Process as a Novel Therapeutic Strategy in COVID-19</strong> <br />
Naidi Yang, Han-Ming Shen<br />
<em>International Journal of Biological Sciences</em> (2020) <a href="https://doi.org/ggqspm">https://doi.org/ggqspm</a> <br />
DOI: <a href="https://doi.org/10.7150/ijbs.45498">10.7150/ijbs.45498</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32226290">32226290</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7098027">PMC7098027</a></p>
</div>
<div id="ref-z8XnLYWM">
<p>407. <strong>SARS-CoV-2: an Emerging Coronavirus that Causes a Global Threat</strong> <br />
Jun Zheng<br />
<em>International Journal of Biological Sciences</em> (2020) <a href="https://doi.org/ggqspr">https://doi.org/ggqspr</a> <br />
DOI: <a href="https://doi.org/10.7150/ijbs.45053">10.7150/ijbs.45053</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32226285">32226285</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7098030">PMC7098030</a></p>
</div>
<div id="ref-1BApBNHvd">
<p>408. <strong>Hydroxychloroquine or chloroquine with or without a macrolide for treatment of COVID-19: a multinational registry analysis</strong> <br />
Mandeep R Mehra, Sapan S Desai, Frank Ruschitzka, Amit N Patel<br />
<em>The Lancet</em> (2020-05) <a href="https://doi.org/ggwzsb">https://doi.org/ggwzsb</a> <br />
DOI: <a href="https://doi.org/10.1016/s0140-6736(20)31180-6">10.1016/s0140-6736(20)31180-6</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32450107">32450107</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7255293">PMC7255293</a></p>
</div>
<div id="ref-11FsxWalS">
<p>409. <strong>Retraction—Hydroxychloroquine or chloroquine with or without a macrolide for treatment of COVID-19: a multinational registry analysis</strong> <br />
Mandeep R Mehra, Frank Ruschitzka, Amit N Patel<br />
<em>The Lancet</em> (2020-06) <a href="https://doi.org/ggzqng">https://doi.org/ggzqng</a> <br />
DOI: <a href="https://doi.org/10.1016/s0140-6736(20)31324-6">10.1016/s0140-6736(20)31324-6</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32511943">32511943</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7274621">PMC7274621</a></p>
</div>
<div id="ref-dY9kXLAS">
<p>410. <strong>Life Threatening Severe QTc Prolongation in Patient with Systemic Lupus Erythematosus due to Hydroxychloroquine</strong> <br />
John P. O’Laughlin, Parag H. Mehta, Brian C. Wong<br />
<em>Case Reports in Cardiology</em> (2016) <a href="https://doi.org/ggqzrc">https://doi.org/ggqzrc</a> <br />
DOI: <a href="https://doi.org/10.1155/2016/4626279">10.1155/2016/4626279</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27478650">27478650</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4960328">PMC4960328</a></p>
</div>
<div id="ref-wy3S7Cpa">
<p>411. <strong>Keep the QT interval: It is a reliable predictor of ventricular arrhythmias</strong> <br />
Dan M. Roden<br />
<em>Heart Rhythm</em> (2008-08) <a href="https://doi.org/d5rchx">https://doi.org/d5rchx</a> <br />
DOI: <a href="https://doi.org/10.1016/j.hrthm.2008.05.008">10.1016/j.hrthm.2008.05.008</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/18675237">18675237</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3212752">PMC3212752</a></p>
</div>
<div id="ref-OQ0b31BA">
<p>412. <strong>Safety of hydroxychloroquine, alone and in combination with azithromycin, in light of rapid wide-spread use for COVID-19: a multinational, network cohort and self-controlled case series study</strong> <br />
Jennifer C. E Lane, James Weaver, Kristin Kostka, Talita Duarte-Salles, Maria Tereza F. Abrahao, Heba Alghoul, Osaid Alser, Thamir M Alshammari, Patricia Biedermann, Edward Burn, … Daniel Prieto-Alhambra<br />
<em>Cold Spring Harbor Laboratory</em> (2020-05-31) <a href="https://doi.org/ggrn7s">https://doi.org/ggrn7s</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.04.08.20054551">10.1101/2020.04.08.20054551</a></p>
</div>
<div id="ref-1CgZR55GZ">
<p>413. <strong>Chloroquine diphosphate in two different dosages as adjunctive therapy of hospitalized patients with severe respiratory syndrome in the context of coronavirus (SARS-CoV-2) infection: Preliminary safety results of a randomized, double-blinded, phase IIb clinical trial (CloroCovid-19 Study)</strong> <br />
Mayla Gabriela Silva Borba, Fernando de Almeida Val, Vanderson Sousa Sampaio, Marcia Almeida Ara&amp;uacutejo Alexandre, Gisely Cardoso Melo, Marcelo Brito, Maria Paula Gomes Mour&amp;atildeo, José Diego Brito Sousa, Djane Clarys Baia-da-Silva, Marcus Vinitius Farias Guerra, … CloroCovid-19 Team<br />
<em>medRxiv</em> (2020-04-16) <a href="https://doi.org/ggr3nj">https://doi.org/ggr3nj</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.04.07.20056424">10.1101/2020.04.07.20056424</a></p>
</div>
<div id="ref-1EcscrT32">
<p>414. <strong>Heart risk concerns mount around use of chloroquine and hydroxychloroquine for Covid-19 treatment</strong> <br />
Jacqueline Howard, Elizabeth Cohen, Nadia Kounang, Per Nyberg<br />
<em>CNN</em> (2020-04-14) <a href="https://www.cnn.com/2020/04/13/health/chloroquine-risks-coronavirus-treatment-trials-study/index.html">https://www.cnn.com/2020/04/13/health/chloroquine-risks-coronavirus-treatment-trials-study/index.html</a></p>
</div>
<div id="ref-AMQ5402v">
<p>415. <strong>WHO Director-General’s opening remarks at the media briefing on COVID-19</strong> <br />
World Health Organization<br />
(2020-05-25) <a href="https://www.who.int/dg/speeches/detail/who-director-general-s-opening-remarks-at-the-media-briefing-on-covid-19---25-may-2020">https://www.who.int/dg/speeches/detail/who-director-general-s-opening-remarks-at-the-media-briefing-on-covid-19---25-may-2020</a></p>
</div>
<div id="ref-RXr9BKKT">
<p>416. <strong>Hydroxychloroquine in patients mainly with mild to moderate COVID-19: an open-label, randomized, controlled trial</strong> <br />
Wei Tang, Zhujun Cao, Mingfeng Han, Zhengyan Wang, Junwen Chen, Wenjin Sun, Yaojie Wu, Wei Xiao, Shengyong Liu, Erzhen Chen, … Qing Xie<br />
<em>Cold Spring Harbor Laboratory</em> (2020-05-07) <a href="https://doi.org/ggr68m">https://doi.org/ggr68m</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.04.10.20060558">10.1101/2020.04.10.20060558</a></p>
</div>
<div id="ref-MENbRfOl">
<p>417. <strong>Outcomes of hydroxychloroquine usage in United States veterans hospitalized with Covid-19</strong> <br />
Joseph Magagnoli, Siddharth Narendran, Felipe Pereira, Tammy Cummings, James W Hardin, S Scott Sutton, Jayakrishna Ambati<br />
<em>Cold Spring Harbor Laboratory</em> (2020-04-23) <a href="https://doi.org/ggspt6">https://doi.org/ggspt6</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.04.16.20065920">10.1101/2020.04.16.20065920</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32511622">32511622</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7276049">PMC7276049</a></p>
</div>
<div id="ref-KwFZyfFx">
<p>418. <strong>Effect of Hydroxychloroquine in Hospitalized Patients with Covid-19</strong> <br />
The RECOVERY Collaborative Group<br />
<em>New England Journal of Medicine</em> (2020-10-08) <a href="https://doi.org/ghd8c7">https://doi.org/ghd8c7</a> <br />
DOI: <a href="https://doi.org/10.1056/nejmoa2022926">10.1056/nejmoa2022926</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33031652">33031652</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7556338">PMC7556338</a></p>
</div>
<div id="ref-V8fkkVZg">
<p>419. <strong>Hydroxychloroquine for Early Treatment of Adults with Mild Covid-19: A Randomized-Controlled Trial</strong> <br />
Oriol Mitjà, Marc Corbacho-Monné, Maria Ubals, Cristian Tebe, Judith Peñafiel, Aurelio Tobias, Ester Ballana, Andrea Alemany, Núria Riera-Martí, Carla A Pérez, … BCN PEP-CoV-2 RESEARCH GROUP<br />
<em>Clinical Infectious Diseases</em> (2020-07-16) <a href="https://doi.org/gg5f9x">https://doi.org/gg5f9x</a> <br />
DOI: <a href="https://doi.org/10.1093/cid/ciaa1009">10.1093/cid/ciaa1009</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32674126">32674126</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7454406">PMC7454406</a></p>
</div>
<div id="ref-NiVdIN7Z">
<p>420. <strong>A Randomized Trial of Hydroxychloroquine as Postexposure Prophylaxis for Covid-19</strong> <br />
David R. Boulware, Matthew F. Pullen, Ananta S. Bangdiwala, Katelyn A. Pastick, Sarah M. Lofgren, Elizabeth C. Okafor, Caleb P. Skipper, Alanna A. Nascene, Melanie R. Nicol, Mahsa Abassi, … Kathy H. Hullsiek<br />
<em>New England Journal of Medicine</em> (2020-08-06) <a href="https://doi.org/dxkv">https://doi.org/dxkv</a> <br />
DOI: <a href="https://doi.org/10.1056/nejmoa2016638">10.1056/nejmoa2016638</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32492293">32492293</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7289276">PMC7289276</a></p>
</div>
<div id="ref-oaqFxV4W">
<p>421. <strong>Efficacy and Safety of Hydroxychloroquine vs Placebo for Pre-exposure SARS-CoV-2 Prophylaxis Among Health Care Workers</strong> <br />
Benjamin S. Abella, Eliana L. Jolkovsky, Barbara T. Biney, Julie E. Uspal, Matthew C. Hyman, Ian Frank, Scott E. Hensley, Saar Gill, Dan T. Vogl, Ivan Maillard, … and the Prevention and Treatment of COVID-19 With Hydroxychloroquine (PATCH) Investigators<br />
<em>JAMA Internal Medicine</em> (2020-09-30) <a href="https://doi.org/ghd6nj">https://doi.org/ghd6nj</a> <br />
DOI: <a href="https://doi.org/10.1001/jamainternmed.2020.6319">10.1001/jamainternmed.2020.6319</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33001138">33001138</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7527945">PMC7527945</a></p>
</div>
<div id="ref-WFGcbZhe">
<p>422. <strong>COVID-19: a recommendation to examine the effect of hydroxychloroquine in preventing infection and progression</strong> <br />
Dan Zhou, Sheng-Ming Dai, Qiang Tong<br />
<em>Journal of Antimicrobial Chemotherapy</em> (2020-03-20) <a href="https://doi.org/ggq84c">https://doi.org/ggq84c</a> <br />
DOI: <a href="https://doi.org/10.1093/jac/dkaa114">10.1093/jac/dkaa114</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32196083">32196083</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7184499">PMC7184499</a></p>
</div>
<div id="ref-hFGjQQN8">
<p>423. <strong>Hydroxychloroquine treatment of patients with human immunodeficiency virus type 1</strong> <br />
Kirk Sperber, Michael Louie, Thomas Kraus, Jacqueline Proner, Erica Sapira, Su Lin, Vera Stecher, Lloyd Mayer<br />
<em>Clinical Therapeutics</em> (1995-07) <a href="https://doi.org/cq2hx9">https://doi.org/cq2hx9</a> <br />
DOI: <a href="https://doi.org/10.1016/0149-2918(95)80039-5">10.1016/0149-2918(95)80039-5</a></p>
</div>
<div id="ref-rQNzKxRd">
<p>424. <strong>Hydroxychloroquine augments early virological response to pegylated interferon plus ribavirin in genotype-4 chronic hepatitis C patients</strong> <br />
Gouda Kamel Helal, Magdy Abdelmawgoud Gad, Mohamed Fahmy Abd-Ellah, Mahmoud Saied Eid<br />
<em>Journal of Medical Virology</em> (2016-12) <a href="https://doi.org/f889nt">https://doi.org/f889nt</a> <br />
DOI: <a href="https://doi.org/10.1002/jmv.24575">10.1002/jmv.24575</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27183377">27183377</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7167065">PMC7167065</a></p>
</div>
<div id="ref-j3rGUH1f">
<p>425. <strong>Mechanisms of action of hydroxychloroquine and chloroquine: implications for rheumatology</strong> <br />
Eva Schrezenmeier, Thomas Dörner<br />
<em>Nature Reviews Rheumatology</em> (2020-02-07) <a href="https://doi.org/ggzjnh">https://doi.org/ggzjnh</a> <br />
DOI: <a href="https://doi.org/10.1038/s41584-020-0372-x">10.1038/s41584-020-0372-x</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32034323">32034323</a></p>
</div>
<div id="ref-q0TqfpEr">
<p>426. <strong>Synthesis and Pharmacology of Anti-Inflammatory Steroidal Antedrugs</strong> <br />
M. Omar F. Khan, Henry J. Lee<br />
<em>Chemical Reviews</em> (2008-12-10) <a href="https://doi.org/cmkrtc">https://doi.org/cmkrtc</a> <br />
DOI: <a href="https://doi.org/10.1021/cr068203e">10.1021/cr068203e</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19035773">19035773</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2650492">PMC2650492</a></p>
</div>
<div id="ref-2M8wLShq">
<p>427. <strong>Dexamethasone</strong> <br />
The Centre for Evidence-Based Medicine<br />
<a href="https://www.cebm.net/covid-19/dexamethasone/">https://www.cebm.net/covid-19/dexamethasone/</a></p>
</div>
<div id="ref-KQ7uhAaa">
<p>428. <strong>16-METHYLATED STEROIDS. I. 16α-METHYLATED ANALOGS OF CORTISONE, A NEW GROUP OF ANTI-INFLAMMATORY STEROIDS</strong> <br />
Glen E. Arth, David B. R. Johnston, John Fried, William W. Spooncer, Dale R. Hoff, Lewis H. Sarett<br />
<em>Journal of the American Chemical Society</em> (1958-06) <a href="https://doi.org/cj5c82">https://doi.org/cj5c82</a> <br />
DOI: <a href="https://doi.org/10.1021/ja01545a061">10.1021/ja01545a061</a></p>
</div>
<div id="ref-bSNVCQbh">
<p>429. <strong>Treatment of Rheumatoid Arthritis with Dexamethasone</strong> <br />
Abraham Cohen<br />
<em>JAMA</em> (1960-10-15) <a href="https://doi.org/csfmhc">https://doi.org/csfmhc</a> <br />
DOI: <a href="https://doi.org/10.1001/jama.1960.03030070009002">10.1001/jama.1960.03030070009002</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/13694317">13694317</a></p>
</div>
<div id="ref-19dEKfvUE">
<p>430. <strong>Potential benefits of precise corticosteroids therapy for severe 2019-nCoV pneumonia</strong> <br />
Wei Zhou, Yisi Liu, Dongdong Tian, Cheng Wang, Sa Wang, Jing Cheng, Ming Hu, Minghao Fang, Yue Gao<br />
<em>Signal Transduction and Targeted Therapy</em> (2020-02-21) <a href="https://doi.org/ggqr84">https://doi.org/ggqr84</a> <br />
DOI: <a href="https://doi.org/10.1038/s41392-020-0127-9">10.1038/s41392-020-0127-9</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32296012">32296012</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7035340">PMC7035340</a></p>
</div>
<div id="ref-1DUXbHUEW">
<p>431. <strong>Dexamethasone treatment for the acute respiratory distress syndrome: a multicentre, randomised controlled trial</strong> <br />
Jesús Villar, Carlos Ferrando, Domingo Martínez, Alfonso Ambrós, Tomás Muñoz, Juan A Soler, Gerardo Aguilar, Francisco Alba, Elena González-Higueras, Luís A Conesa, … Jesús Villar<br />
<em>The Lancet Respiratory Medicine</em> (2020-03) <a href="https://doi.org/ggpxzc">https://doi.org/ggpxzc</a> <br />
DOI: <a href="https://doi.org/10.1016/s2213-2600(19)30417-5">10.1016/s2213-2600(19)30417-5</a></p>
</div>
<div id="ref-Ji2fchyS">
<p>432. <strong>Clinical evidence does not support corticosteroid treatment for 2019-nCoV lung injury</strong> <br />
Clark D Russell, Jonathan E Millar, J Kenneth Baillie<br />
<em>The Lancet</em> (2020-02) <a href="https://doi.org/ggks86">https://doi.org/ggks86</a> <br />
DOI: <a href="https://doi.org/10.1016/s0140-6736(20)30317-2">10.1016/s0140-6736(20)30317-2</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32043983">32043983</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7134694">PMC7134694</a></p>
</div>
<div id="ref-xndHds7d">
<p>433. <strong>On the use of corticosteroids for 2019-nCoV pneumonia</strong> <br />
Lianhan Shang, Jianping Zhao, Yi Hu, Ronghui Du, Bin Cao<br />
<em>The Lancet</em> (2020-02) <a href="https://doi.org/ggq356">https://doi.org/ggq356</a> <br />
DOI: <a href="https://doi.org/10.1016/s0140-6736(20)30361-5">10.1016/s0140-6736(20)30361-5</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32122468">32122468</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7159292">PMC7159292</a></p>
</div>
<div id="ref-1ETGa7JyL">
<p>434. <strong>Pharmacology of Postoperative Nausea and Vomiting</strong> <br />
Eric S. Zabirowicz, Tong J. Gan<br />
<em>Elsevier BV</em> (2019) <a href="https://doi.org/ghfkjw">https://doi.org/ghfkjw</a> <br />
DOI: <a href="https://doi.org/10.1016/b978-0-323-48110-6.00034-x">10.1016/b978-0-323-48110-6.00034-x</a></p>
</div>
<div id="ref-jMzODpwS">
<p>435. <strong>Non-traditional cytokines: How catecholamines and adipokines influence macrophages in immunity, metabolism and the central nervous system</strong> <br />
Mark A. Barnes, Monica J. Carson, Meera G. Nair<br />
<em>Cytokine</em> (2015-04) <a href="https://doi.org/f65c59">https://doi.org/f65c59</a> <br />
DOI: <a href="https://doi.org/10.1016/j.cyto.2015.01.008">10.1016/j.cyto.2015.01.008</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25703786">25703786</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4590987">PMC4590987</a></p>
</div>
<div id="ref-Riev3em1">
<p>436. <strong>Stress Hormones, Proinflammatory and Antiinflammatory Cytokines, and Autoimmunity</strong> <br />
ILIA J. ELENKOV, GEORGE P. CHROUSOS<br />
<em>Annals of the New York Academy of Sciences</em> (2002-06) <a href="https://doi.org/fmwpx2">https://doi.org/fmwpx2</a> <br />
DOI: <a href="https://doi.org/10.1111/j.1749-6632.2002.tb04229.x">10.1111/j.1749-6632.2002.tb04229.x</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/12114286">12114286</a></p>
</div>
<div id="ref-JZyVNRCN">
<p>437. <strong>Recovery of the Hypothalamic-Pituitary-Adrenal Response to Stress</strong> <br />
Arantxa García, Octavi Martí, Astrid Vallès, Silvina Dal-Zotto, Antonio Armario<br />
<em>Neuroendocrinology</em> (2000) <a href="https://doi.org/b2cq8n">https://doi.org/b2cq8n</a> <br />
DOI: <a href="https://doi.org/10.1159/000054578">10.1159/000054578</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/10971146">10971146</a></p>
</div>
<div id="ref-SiXrQUgV">
<p>438. <strong>Modulatory effects of glucocorticoids and catecholamines on human interleukin-12 and interleukin-10 production: clinical implications.</strong> <br />
IJ Elenkov, DA Papanicolaou, RL Wilder, GP Chrousos<br />
<em>Proceedings of the Association of American Physicians</em> (1996-09) <a href="https://www.ncbi.nlm.nih.gov/pubmed/8902882">https://www.ncbi.nlm.nih.gov/pubmed/8902882</a> <br />
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/8902882">8902882</a></p>
</div>
<div id="ref-nbiwprc">
<p>439. <strong>Prevention of infection caused by immunosuppressive drugs in gastroenterology</strong> <br />
Katarzyna Orlicka, Eleanor Barnes, Emma L. Culver<br />
<em>Therapeutic Advances in Chronic Disease</em> (2013-04-22) <a href="https://doi.org/ggrqd3">https://doi.org/ggrqd3</a> <br />
DOI: <a href="https://doi.org/10.1177/2040622313485275">10.1177/2040622313485275</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23819020">23819020</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3697844">PMC3697844</a></p>
</div>
<div id="ref-suHq6Qnm">
<p>440. <strong>Immunosuppression for hyperinflammation in COVID-19: a double-edged sword?</strong> <br />
Andrew I Ritchie, Aran Singanayagam<br />
<em>The Lancet</em> (2020-04) <a href="https://doi.org/ggq8hs">https://doi.org/ggq8hs</a> <br />
DOI: <a href="https://doi.org/10.1016/s0140-6736(20)30691-7">10.1016/s0140-6736(20)30691-7</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32220278">32220278</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7138169">PMC7138169</a></p>
</div>
<div id="ref-pzQoUwz3">
<p>441. <strong>Effect of Dexamethasone in Hospitalized Patients with COVID-19: Preliminary Report</strong> <br />
Peter Horby, Wei Shen Lim, Jonathan Emberson, Marion Mafham, Jennifer Bell, Louise Linsell, Natalie Staplin, Christopher Brightling, Andrew Ustianowski, Einas Elmahi, … RECOVERY Collaborative Group<br />
<em>Cold Spring Harbor Laboratory</em> (2020-06-22) <a href="https://doi.org/dz5x">https://doi.org/dz5x</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.06.22.20137273">10.1101/2020.06.22.20137273</a></p>
</div>
<div id="ref-H3C2PalE">
<p>442. <strong>Dexamethasone in Hospitalized Patients with Covid-19 — Preliminary Report</strong> <br />
The RECOVERY Collaborative Group<br />
<em>New England Journal of Medicine</em> (2020-07-17) <a href="https://doi.org/gg5c8p">https://doi.org/gg5c8p</a> <br />
DOI: <a href="https://doi.org/10.1056/nejmoa2021436">10.1056/nejmoa2021436</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32678530">32678530</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7383595">PMC7383595</a></p>
</div>
<div id="ref-H70g2GF9">
<p>443. <strong>Current concepts in the diagnosis and management of cytokine release syndrome</strong> <br />
Daniel W. Lee, Rebecca Gardner, David L. Porter, Chrystal U. Louis, Nabil Ahmed, Michael Jensen, Stephan A. Grupp, Crystal L. Mackall<br />
<em>Blood</em> (2014-07-10) <a href="https://doi.org/ggsrwk">https://doi.org/ggsrwk</a> <br />
DOI: <a href="https://doi.org/10.1182/blood-2014-05-552729">10.1182/blood-2014-05-552729</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24876563">24876563</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4093680">PMC4093680</a></p>
</div>
<div id="ref-EgvIETTq">
<p>444. <strong>Dexamethasone for COVID-19? Not so fast.</strong> <br />
T. C. Theoharides<br />
<em>JOURNAL OF BIOLOGICAL REGULATORS AND HOMEOSTATIC AGENTS</em> (2020-08-31) <a href="https://doi.org/ghfkjx">https://doi.org/ghfkjx</a> <br />
DOI: <a href="https://doi.org/10.23812/20-editorial_1-5">10.23812/20-editorial_1-5</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32551464">32551464</a></p>
</div>
<div id="ref-xLPeszeO">
<p>445. <strong>Structural Design Principles for Delivery of Bioactive Components in Nutraceuticals and Functional Foods</strong> <br />
David Julian McClements, Eric Andrew Decker, Yeonhwa Park, Jochen Weiss<br />
<em>Critical Reviews in Food Science and Nutrition</em> (2009-06-16) <a href="https://doi.org/dt68m4">https://doi.org/dt68m4</a> <br />
DOI: <a href="https://doi.org/10.1080/10408390902841529">10.1080/10408390902841529</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19484636">19484636</a></p>
</div>
<div id="ref-2Us0gNMr">
<p>446. <strong>Nutraceutical therapies for atherosclerosis</strong> <br />
Joe W. E. Moss, Dipak P. Ramji<br />
<em>Nature Reviews Cardiology</em> (2016-07-07) <a href="https://doi.org/f9g389">https://doi.org/f9g389</a> <br />
DOI: <a href="https://doi.org/10.1038/nrcardio.2016.103">10.1038/nrcardio.2016.103</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27383080">27383080</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5228762">PMC5228762</a></p>
</div>
<div id="ref-X0jEogv4">
<p>447. <strong>Nutraceutical-definition and introduction</strong> <br />
Ekta K. Kalra<br />
<em>AAPS PharmSci</em> (2015-07-10) <a href="https://doi.org/cg5wc9">https://doi.org/cg5wc9</a> <br />
DOI: <a href="https://doi.org/10.1208/ps050325">10.1208/ps050325</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/14621960">14621960</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2750935">PMC2750935</a></p>
</div>
<div id="ref-10Tnnl6nF">
<p>448. <strong>Dietary Supplement Health and Education Act of 1994</strong> <br />
National Institutes of Health Office of Dietary Supplements<br />
<a href="https://ods.od.nih.gov/About/DSHEA_Wording.aspx">https://ods.od.nih.gov/About/DSHEA_Wording.aspx</a></p>
</div>
<div id="ref-16eEQf0HM">
<p>449. <strong>Food and Drug Administration Modernization Act (FDAMA) of 1997</strong> <br />
Office of the Commissioner<br />
<em>FDA</em> (2018-11-03) <a href="https://www.fda.gov/regulatory-information/selected-amendments-fdc-act/food-and-drug-administration-modernization-act-fdama-1997">https://www.fda.gov/regulatory-information/selected-amendments-fdc-act/food-and-drug-administration-modernization-act-fdama-1997</a></p>
</div>
<div id="ref-5TmtJWVQ">
<p>450. <strong>Nutraceuticals - shedding light on the grey area between pharmaceuticals and food</strong> <br />
Antonello Santini, Ettore Novellino<br />
<em>Expert Review of Clinical Pharmacology</em> (2018-04-23) <a href="https://doi.org/ggwztk">https://doi.org/ggwztk</a> <br />
DOI: <a href="https://doi.org/10.1080/17512433.2018.1464911">10.1080/17512433.2018.1464911</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29667442">29667442</a></p>
</div>
<div id="ref-dzOveIQY">
<p>451. <strong>EU Register of nutrition and health claims made on foods (v.3.5)</strong> <a href="https://ec.europa.eu/food/safety/labelling_nutrition/claims/register/public/">https://ec.europa.eu/food/safety/labelling_nutrition/claims/register/public/</a></p>
</div>
<div id="ref-MhvDTmeu">
<p>452. <strong>Nutraceuticals: opening the debate for a regulatory framework</strong> <br />
Antonello Santini, Silvia Miriam Cammarata, Giacomo Capone, Angela Ianaro, Gian Carlo Tenore, Luca Pani, Ettore Novellino<br />
<em>British Journal of Clinical Pharmacology</em> (2018-04) <a href="https://doi.org/ggwztm">https://doi.org/ggwztm</a> <br />
DOI: <a href="https://doi.org/10.1111/bcp.13496">10.1111/bcp.13496</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29433155">29433155</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5867125">PMC5867125</a></p>
</div>
<div id="ref-2hMFPQJR">
<p>453. <strong>Potential interventions for novel coronavirus in China: A systematic review</strong> <br />
Lei Zhang, Yunhui Liu<br />
<em>Journal of Medical Virology</em> (2020-03-03) <a href="https://doi.org/ggpx57">https://doi.org/ggpx57</a> <br />
DOI: <a href="https://doi.org/10.1002/jmv.25707">10.1002/jmv.25707</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32052466">32052466</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7166986">PMC7166986</a></p>
</div>
<div id="ref-Ufvsmc3N">
<p>454. <strong>Inflammation and cardiovascular disease: are marine phospholipids the answer?</strong> <br />
Ronan Lordan, Shane Redfern, Alexandros Tsoupras, Ioannis Zabetakis<br />
<em>Food &amp; Function</em> (2020) <a href="https://doi.org/gg29hg">https://doi.org/gg29hg</a> <br />
DOI: <a href="https://doi.org/10.1039/c9fo01742a">10.1039/c9fo01742a</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32270798">32270798</a></p>
</div>
<div id="ref-JNyhLAfe">
<p>455. <strong>Omega-3 polyunsaturated fatty acids and inflammatory processes: nutrition or pharmacology?</strong> <br />
Philip C. Calder<br />
<em>British Journal of Clinical Pharmacology</em> (2013-03) <a href="https://doi.org/ggqmgg">https://doi.org/ggqmgg</a> <br />
DOI: <a href="https://doi.org/10.1111/j.1365-2125.2012.04374.x">10.1111/j.1365-2125.2012.04374.x</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22765297">22765297</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3575932">PMC3575932</a></p>
</div>
<div id="ref-BIVORG2X">
<p>456. <strong>Proresolving Lipid Mediators and Mechanisms in the Resolution of Acute Inflammation</strong> <br />
Christopher D. Buckley, Derek W. Gilroy, Charles N. Serhan<br />
<em>Immunity</em> (2014-03) <a href="https://doi.org/f5wntr">https://doi.org/f5wntr</a> <br />
DOI: <a href="https://doi.org/10.1016/j.immuni.2014.02.009">10.1016/j.immuni.2014.02.009</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24656045">24656045</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4004957">PMC4004957</a></p>
</div>
<div id="ref-17jegDTvF">
<p>457. <strong>COVID-19 and its implications for thrombosis and anticoagulation</strong> <br />
Jean M. Connors, Jerrold H. Levy<br />
<em>Blood</em> (2020-06-04) <a href="https://doi.org/ggv35b">https://doi.org/ggv35b</a> <br />
DOI: <a href="https://doi.org/10.1182/blood.2020006000">10.1182/blood.2020006000</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32339221">32339221</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7273827">PMC7273827</a></p>
</div>
<div id="ref-SxPCbOB1">
<p>458. <strong>COVID-19 and Thrombotic or Thromboembolic Disease: Implications for Prevention, Antithrombotic Therapy, and Follow-Up</strong> <br />
Behnood Bikdeli, Mahesh V. Madhavan, David Jimenez, Taylor Chuich, Isaac Dreyfus, Elissa Driggin, Caroline Der Nigoghossian, Walter Ageno, Mohammad Madjid, Yutao Guo, … Gregory Y. H. Lip<br />
<em>Journal of the American College of Cardiology</em> (2020-06) <a href="https://doi.org/ggsppk">https://doi.org/ggsppk</a> <br />
DOI: <a href="https://doi.org/10.1016/j.jacc.2020.04.031">10.1016/j.jacc.2020.04.031</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32311448">32311448</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7164881">PMC7164881</a></p>
</div>
<div id="ref-uPOTvvjR">
<p>459. <strong>Specialized pro-resolving mediators: endogenous regulators of infection and inflammation</strong> <br />
Maria C. Basil, Bruce D. Levy<br />
<em>Nature Reviews Immunology</em> (2015-12-21) <a href="https://doi.org/f9fgtd">https://doi.org/f9fgtd</a> <br />
DOI: <a href="https://doi.org/10.1038/nri.2015.4">10.1038/nri.2015.4</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26688348">26688348</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5242505">PMC5242505</a></p>
</div>
<div id="ref-fIo3ESPM">
<p>460. <strong>The Lipid Mediator Protectin D1 Inhibits Influenza Virus Replication and Improves Severe Influenza</strong> <br />
Masayuki Morita, Keiji Kuba, Akihiko Ichikawa, Mizuho Nakayama, Jun Katahira, Ryo Iwamoto, Tokiko Watanebe, Saori Sakabe, Tomo Daidoji, Shota Nakamura, … Yumiko Imai<br />
<em>Cell</em> (2013-03) <a href="https://doi.org/f4rbgb">https://doi.org/f4rbgb</a> <br />
DOI: <a href="https://doi.org/10.1016/j.cell.2013.02.027">10.1016/j.cell.2013.02.027</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23477864">23477864</a></p>
</div>
<div id="ref-OSEnHt6H">
<p>461. <strong>Pro-resolving lipid mediators are leads for resolution physiology</strong> <br />
Charles N. Serhan<br />
<em>Nature</em> (2014-06-04) <a href="https://doi.org/ggv53d">https://doi.org/ggv53d</a> <br />
DOI: <a href="https://doi.org/10.1038/nature13479">10.1038/nature13479</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24899309">24899309</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4263681">PMC4263681</a></p>
</div>
<div id="ref-MK0GQ55y">
<p>462. <strong>The Specialized Proresolving Mediator 17-HDHA Enhances the Antibody-Mediated Immune Response against Influenza Virus: A New Class of Adjuvant?</strong> <br />
Sesquile Ramon, Steven F. Baker, Julie M. Sahler, Nina Kim, Eric A. Feldsott, Charles N. Serhan, Luis Martínez-Sobrido, David J. Topham, Richard P. Phipps<br />
<em>The Journal of Immunology</em> (2014-12-15) <a href="https://doi.org/f6spr8">https://doi.org/f6spr8</a> <br />
DOI: <a href="https://doi.org/10.4049/jimmunol.1302795">10.4049/jimmunol.1302795</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25392529">25392529</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4258475">PMC4258475</a></p>
</div>
<div id="ref-14LjKv9ZH">
<p>463. <strong>Inflammation resolution: a dual-pronged approach to averting cytokine storms in COVID-19?</strong> <br />
Dipak Panigrahy, Molly M. Gilligan, Sui Huang, Allison Gartung, Irene Cortés-Puch, Patricia J. Sime, Richard P. Phipps, Charles N. Serhan, Bruce D. Hammock<br />
<em>Cancer and Metastasis Reviews</em> (2020-05-08) <a href="https://doi.org/ggvv7w">https://doi.org/ggvv7w</a> <br />
DOI: <a href="https://doi.org/10.1007/s10555-020-09889-4">10.1007/s10555-020-09889-4</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32385712">32385712</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7207990">PMC7207990</a></p>
</div>
<div id="ref-B8toe6Mr">
<p>464. <strong>Fish Oil-Fed Mice Have Impaired Resistance to Influenza Infection</strong> <br />
Nicole M. J. Schwerbrock, Erik A. Karlsson, Qing Shi, Patricia A. Sheridan, Melinda A. Beck<br />
<em>The Journal of Nutrition</em> (2009-08) <a href="https://doi.org/dv45f4">https://doi.org/dv45f4</a> <br />
DOI: <a href="https://doi.org/10.3945/jn.109.108027">10.3945/jn.109.108027</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19549756">19549756</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2709305">PMC2709305</a></p>
</div>
<div id="ref-3Xht24k3">
<p>465. <strong>Modulation of host defence against bacterial and viral infections by omega-3 polyunsaturated fatty acids</strong> <br />
Marie-Odile Husson, Delphine Ley, Céline Portal, Madeleine Gottrand, Thomas Hueso, Jean-Luc Desseyn, Frédéric Gottrand<br />
<em>Journal of Infection</em> (2016-12) <a href="https://doi.org/f9pp2h">https://doi.org/f9pp2h</a> <br />
DOI: <a href="https://doi.org/10.1016/j.jinf.2016.10.001">10.1016/j.jinf.2016.10.001</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27746159">27746159</a></p>
</div>
<div id="ref-Ge00xeR3">
<p>466. <strong>Bioactive products formed in humans from fish oils</strong> <br />
Carsten Skarke, Naji Alamuddin, John A. Lawson, Xuanwen Li, Jane F. Ferguson, Muredach P. Reilly, Garret A. FitzGerald<br />
<em>Journal of Lipid Research</em> (2015-09) <a href="https://doi.org/f7pm5g">https://doi.org/f7pm5g</a> <br />
DOI: <a href="https://doi.org/10.1194/jlr.m060392">10.1194/jlr.m060392</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26180051">26180051</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4548785">PMC4548785</a></p>
</div>
<div id="ref-umzPtQG7">
<p>467. <strong>Regulation of platelet function and thrombosis by omega-3 and omega-6 polyunsaturated fatty acids</strong> <br />
Reheman Adili, Megan Hawley, Michael Holinstat<br />
<em>Prostaglandins &amp; Other Lipid Mediators</em> (2018-11) <a href="https://doi.org/ggvv73">https://doi.org/ggvv73</a> <br />
DOI: <a href="https://doi.org/10.1016/j.prostaglandins.2018.09.005">10.1016/j.prostaglandins.2018.09.005</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30266534">30266534</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6242736">PMC6242736</a></p>
</div>
<div id="ref-V9MDMfpj">
<p>468. <strong>Platelet activation and prothrombotic mediators at the nexus of inflammation and atherosclerosis: Potential role of antiplatelet agents</strong> <br />
Ronan Lordan, Alexandros Tsoupras, Ioannis Zabetakis<br />
<em>Blood Reviews</em> (2020-04) <a href="https://doi.org/ggvv7x">https://doi.org/ggvv7x</a> <br />
DOI: <a href="https://doi.org/10.1016/j.blre.2020.100694">10.1016/j.blre.2020.100694</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32340775">32340775</a></p>
</div>
<div id="ref-QKHdIx2w">
<p>469. <strong>An Investigation on the Effects of Icosapent Ethyl (VascepaTM) on Inflammatory Biomarkers in Individuals With COVID-19 - Full Text View - ClinicalTrials.gov</strong> <a href="https://clinicaltrials.gov/ct2/show/NCT04412018">https://clinicaltrials.gov/ct2/show/NCT04412018</a></p>
</div>
<div id="ref-12Q4bY0vF">
<p>470. <strong>Cardiovascular Risk Reduction with Icosapent Ethyl for Hypertriglyceridemia</strong> <br />
Deepak L. Bhatt, P. Gabriel Steg, Michael Miller, Eliot A. Brinton, Terry A. Jacobson, Steven B. Ketchum, Ralph T. Doyle, Rebecca A. Juliano, Lixia Jiao, Craig Granowitz, … Christie M. Ballantyne<br />
<em>New England Journal of Medicine</em> (2019-01-03) <a href="https://doi.org/gfj3w9">https://doi.org/gfj3w9</a> <br />
DOI: <a href="https://doi.org/10.1056/nejmoa1812792">10.1056/nejmoa1812792</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30415628">30415628</a></p>
</div>
<div id="ref-8lPjdnE7">
<p>471. <strong>A Randomised, Double-blind, Placebo Controlled Study of Eicosapentaenoic Acid (EPA-FFA) Gastro-resistant Capsules to Treat Hospitalised Subjects With Confirmed SARS-CoV-2</strong> <br />
S.L.A. Pharma AG<br />
<em>clinicaltrials.gov</em> (2020-10-29) <a href="https://clinicaltrials.gov/ct2/show/NCT04335032">https://clinicaltrials.gov/ct2/show/NCT04335032</a></p>
</div>
<div id="ref-1AGVAbQuf">
<p>472. <strong>Anti-inflammatory/Antioxidant Oral Nutrition Supplementation on the Cytokine Storm and Progression of COVID-19: A Randomized Controlled Trial</strong> <br />
Mahmoud Abulmeaty FACN M. D.<br />
<em>clinicaltrials.gov</em> (2020-09-18) <a href="https://clinicaltrials.gov/ct2/show/NCT04323228">https://clinicaltrials.gov/ct2/show/NCT04323228</a></p>
</div>
<div id="ref-XHtzuR5c">
<p>473. <strong>Functional Role of Dietary Intervention to Improve the Outcome of COVID-19: A Hypothesis of Work</strong> <br />
Giovanni Messina, Rita Polito, Vincenzo Monda, Luigi Cipolloni, Nunzio Di Nunno, Giulio Di Mizio, Paolo Murabito, Marco Carotenuto, Antonietta Messina, Daniela Pisanelli, … Francesco Sessa<br />
<em>International Journal of Molecular Sciences</em> (2020-04-28) <a href="https://doi.org/ggvb88">https://doi.org/ggvb88</a> <br />
DOI: <a href="https://doi.org/10.3390/ijms21093104">10.3390/ijms21093104</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32354030">32354030</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7247152">PMC7247152</a></p>
</div>
<div id="ref-VMI1lOFR">
<p>474. <strong>Zinc and immunity: An essential interrelation</strong> <br />
Maria Maares, Hajo Haase<br />
<em>Archives of Biochemistry and Biophysics</em> (2016-12) <a href="https://doi.org/f9c9b5">https://doi.org/f9c9b5</a> <br />
DOI: <a href="https://doi.org/10.1016/j.abb.2016.03.022">10.1016/j.abb.2016.03.022</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27021581">27021581</a></p>
</div>
<div id="ref-Qif5U0Ig">
<p>475. <strong>Zinc-Dependent Suppression of TNF-α Production Is Mediated by Protein Kinase A-Induced Inhibition of Raf-1, IκB Kinase β, and NF-κB</strong> <br />
Verena von Bülow, Svenja Dubben, Gabriela Engelhardt, Silke Hebel, Birgit Plümäkers, Holger Heine, Lothar Rink, Hajo Haase<br />
<em>The Journal of Immunology</em> (2007-09-15) <a href="https://doi.org/f3vs45">https://doi.org/f3vs45</a> <br />
DOI: <a href="https://doi.org/10.4049/jimmunol.179.6.4180">10.4049/jimmunol.179.6.4180</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17785857">17785857</a></p>
</div>
<div id="ref-1CEzU1G34">
<p>476. <strong>Zinc activates NF-κB in HUT-78 cells</strong> <br />
Ananda S. Prasad, Bin Bao, Frances W. J. Beck, Fazlul H. Sarkar<br />
<em>Journal of Laboratory and Clinical Medicine</em> (2001-10) <a href="https://doi.org/cnc6fr">https://doi.org/cnc6fr</a> <br />
DOI: <a href="https://doi.org/10.1067/mlc.2001.118108">10.1067/mlc.2001.118108</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/11574819">11574819</a></p>
</div>
<div id="ref-7ElRN12v">
<p>477. <strong>Innate or Adaptive Immunity? The Example of Natural Killer Cells</strong> <br />
E. Vivier, D. H. Raulet, A. Moretta, M. A. Caligiuri, L. Zitvogel, L. L. Lanier, W. M. Yokoyama, S. Ugolini<br />
<em>Science</em> (2011-01-06) <a href="https://doi.org/ckzg9g">https://doi.org/ckzg9g</a> <br />
DOI: <a href="https://doi.org/10.1126/science.1198687">10.1126/science.1198687</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21212348">21212348</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3089969">PMC3089969</a></p>
</div>
<div id="ref-kxuSwrCI">
<p>478. <strong>Zinc supplementation decreases incidence of infections in the elderly: effect of zinc on generation of cytokines and oxidative stress</strong> <br />
Ananda S Prasad, Frances WJ Beck, Bin Bao, James T Fitzgerald, Diane C Snell, Joel D Steinberg, Lavoisier J Cardozo<br />
<em>The American Journal of Clinical Nutrition</em> (2007-03) <a href="https://doi.org/ggqmgs">https://doi.org/ggqmgs</a> <br />
DOI: <a href="https://doi.org/10.1093/ajcn/85.3.837">10.1093/ajcn/85.3.837</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17344507">17344507</a></p>
</div>
<div id="ref-IAOnO0Gj">
<p>479. <strong>The Role of Zinc in Antiviral Immunity</strong> <br />
Scott A Read, Stephanie Obeid, Chantelle Ahlenstiel, Golo Ahlenstiel<br />
<em>Advances in Nutrition</em> (2019-07) <a href="https://doi.org/ggqmgr">https://doi.org/ggqmgr</a> <br />
DOI: <a href="https://doi.org/10.1093/advances/nmz013">10.1093/advances/nmz013</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31305906">31305906</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6628855">PMC6628855</a></p>
</div>
<div id="ref-sgr0mlJK">
<p>480. <strong>Efficacy of Zinc Against Common Cold Viruses: An Overview</strong> <br />
Darrell Hulisz<br />
<em>Journal of the American Pharmacists Association</em> (2004-09) <a href="https://doi.org/cf6pmt">https://doi.org/cf6pmt</a> <br />
DOI: <a href="https://doi.org/10.1331/1544-3191.44.5.594.hulisz">10.1331/1544-3191.44.5.594.hulisz</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15496046">15496046</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7185598">PMC7185598</a></p>
</div>
<div id="ref-lPexOhy6">
<p>481. <strong>Zinc Lozenges May Shorten the Duration of Colds: A Systematic Review</strong> <br />
Harri Hemilä<br />
<em>The Open Respiratory Medicine Journal</em> (2011-06-23) <a href="https://doi.org/bndmfq">https://doi.org/bndmfq</a> <br />
DOI: <a href="https://doi.org/10.2174/1874306401105010051">10.2174/1874306401105010051</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21769305">21769305</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3136969">PMC3136969</a></p>
</div>
<div id="ref-6aOUJdHa">
<p>482. <strong>Zn2+ Inhibits Coronavirus and Arterivirus RNA Polymerase Activity In Vitro and Zinc Ionophores Block the Replication of These Viruses in Cell Culture</strong> <br />
Aartjan J. W. te Velthuis, Sjoerd H. E. van den Worm, Amy C. Sims, Ralph S. Baric, Eric J. Snijder, Martijn J. van Hemert<br />
<em>PLoS Pathogens</em> (2010-11-04) <a href="https://doi.org/d95x4g">https://doi.org/d95x4g</a> <br />
DOI: <a href="https://doi.org/10.1371/journal.ppat.1001176">10.1371/journal.ppat.1001176</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21079686">21079686</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2973827">PMC2973827</a></p>
</div>
<div id="ref-NB1udFHF">
<p>483. <strong>The SARS-coronavirus papain-like protease: Structure, function and inhibition by designed antiviral compounds</strong> <br />
Yahira M. Báez-Santos, Sarah E. St. John, Andrew D. Mesecar<br />
<em>Antiviral Research</em> (2015-03) <a href="https://doi.org/f63hjp">https://doi.org/f63hjp</a> <br />
DOI: <a href="https://doi.org/10.1016/j.antiviral.2014.12.015">10.1016/j.antiviral.2014.12.015</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25554382">25554382</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5896749">PMC5896749</a></p>
</div>
<div id="ref-XJrj3MiZ">
<p>484. <strong>A Randomized Study Evaluating the Safety and Efficacy of Hydroxychloroquine and Zinc in Combination With Either Azithromycin or Doxycycline for the Treatment of COVID-19 in the Outpatient Setting</strong> <br />
Avni Thakore MD<br />
<em>clinicaltrials.gov</em> (2020-05-14) <a href="https://clinicaltrials.gov/ct2/show/NCT04370782">https://clinicaltrials.gov/ct2/show/NCT04370782</a></p>
</div>
<div id="ref-1EqFW3G4X">
<p>485. <strong>A Study of Hydroxychloroquine and Zinc in the Prevention of COVID-19 Infection in Military Healthcare Workers - Full Text View - ClinicalTrials.gov</strong> <a href="https://clinicaltrials.gov/ct2/show/NCT04377646">https://clinicaltrials.gov/ct2/show/NCT04377646</a></p>
</div>
<div id="ref-Hc9mjVB2">
<p>486. <strong>Therapies to Prevent Progression of COVID-19, Including Hydroxychloroquine, Azithromycin, Zinc, Vitamin D, Vitamin B12 With or Without Vitamin C, a Multi-centre, International, Randomized Trial: The International ALLIANCE Study</strong> <br />
National Institute of Integrative Medicine, Australia<br />
<em>clinicaltrials.gov</em> (2020-09-09) <a href="https://clinicaltrials.gov/ct2/show/NCT04395768">https://clinicaltrials.gov/ct2/show/NCT04395768</a></p>
</div>
<div id="ref-T0aXNTIp">
<p>487. <strong>Early Intervention in COVID-19: Favipiravir Verses Standard Care - Full Text View - ClinicalTrials.gov</strong> <a href="https://clinicaltrials.gov/ct2/show/NCT04373733">https://clinicaltrials.gov/ct2/show/NCT04373733</a></p>
</div>
<div id="ref-1GGbb9IZt">
<p>488. <strong>Coronavirus Disease 2019- Using Ascorbic Acid and Zinc Supplementation (COVIDAtoZ) Research Study A Randomized, Open Label Single Center Study</strong> <br />
Milind Desai<br />
<em>clinicaltrials.gov</em> (2020-08-12) <a href="https://clinicaltrials.gov/ct2/show/NCT04342728">https://clinicaltrials.gov/ct2/show/NCT04342728</a></p>
</div>
<div id="ref-3RCeQRRc">
<p>489. <strong>Vitamin B12 May Inhibit RNA-Dependent-RNA Polymerase Activity of nsp12 from the COVID-19 Virus</strong> <br />
Naveen Narayanan, Deepak T. Nair<br />
<em>Preprints</em> (2020-03-22) <a href="https://doi.org/ggqmjc">https://doi.org/ggqmjc</a> <br />
DOI: <a href="https://doi.org/10.20944/preprints202003.0347.v1">10.20944/preprints202003.0347.v1</a></p>
</div>
<div id="ref-9czQt5cR">
<p>490. <strong>Vitamin C Mitigates Oxidative Stress and Tumor Necrosis Factor-Alpha in Severe Community-Acquired Pneumonia and LPS-Induced Macrophages</strong> <br />
Yuanyuan Chen, Guangyan Luo, Jiao Yuan, Yuanyuan Wang, Xiaoqiong Yang, Xiaoyun Wang, Guoping Li, Zhiguang Liu, Nanshan Zhong<br />
<em>Mediators of Inflammation</em> (2014) <a href="https://doi.org/f6nb5f">https://doi.org/f6nb5f</a> <br />
DOI: <a href="https://doi.org/10.1155/2014/426740">10.1155/2014/426740</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25253919">25253919</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4165740">PMC4165740</a></p>
</div>
<div id="ref-RjFxOURM">
<p>491. <strong>Intravenous infusion of ascorbic acid decreases serum histamine concentrations in patients with allergic and non-allergic diseases</strong> <br />
Alexander F. Hagel, Christian M. Layritz, Wolfgang H. Hagel, Hans-Jürgen Hagel, Edith Hagel, Wolfgang Dauth, Jürgen Kressel, Tanja Regnet, Andreas Rosenberg, Markus F. Neurath, … Martin Raithel<br />
<em>Naunyn-Schmiedeberg’s Archives of Pharmacology</em> (2013-05-11) <a href="https://doi.org/f48jsb">https://doi.org/f48jsb</a> <br />
DOI: <a href="https://doi.org/10.1007/s00210-013-0880-1">10.1007/s00210-013-0880-1</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23666445">23666445</a></p>
</div>
<div id="ref-13M3VAfY9">
<p>492. <strong>Vitamin C and Immune Function</strong> <br />
Anitra Carr, Silvia Maggini<br />
<em>Nutrients</em> (2017-11-03) <a href="https://doi.org/gfzrjs">https://doi.org/gfzrjs</a> <br />
DOI: <a href="https://doi.org/10.3390/nu9111211">10.3390/nu9111211</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29099763">29099763</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5707683">PMC5707683</a></p>
</div>
<div id="ref-uQFCDDIy">
<p>493. <strong>Changes in Leucocyte Ascorbic Acid during the Common Cold</strong> <br />
R. Hume, Elspeth Weyers<br />
<em>Scottish Medical Journal</em> (2016-06-25) <a href="https://doi.org/ggqrfj">https://doi.org/ggqrfj</a> <br />
DOI: <a href="https://doi.org/10.1177/003693307301800102">10.1177/003693307301800102</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/4717661">4717661</a></p>
</div>
<div id="ref-10nBT1HbX">
<p>494. <strong>ASCORBIC ACID FUNCTION AND METABOLISM DURING COLDS</strong> <br />
C. W. M. Wilson<br />
<em>Annals of the New York Academy of Sciences</em> (1975-09) <a href="https://doi.org/bjfdtb">https://doi.org/bjfdtb</a> <br />
DOI: <a href="https://doi.org/10.1111/j.1749-6632.1975.tb29312.x">10.1111/j.1749-6632.1975.tb29312.x</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/1106304">1106304</a></p>
</div>
<div id="ref-1Bj8VMlSk">
<p>495. <strong>Metabolism of ascorbic acid (vitamin C) in subjects infected with common cold viruses</strong> <br />
J. E. W. Davies, R. E. Hughes, Eleri Jones, Sylvia E. Reed, J. W. Craig, D. A. J. Tyrrell<br />
<em>Biochemical Medicine</em> (1979-02) <a href="https://doi.org/fd22sv">https://doi.org/fd22sv</a> <br />
DOI: <a href="https://doi.org/10.1016/0006-2944(79)90058-9">10.1016/0006-2944(79)90058-9</a></p>
</div>
<div id="ref-aEEZ8Pq9">
<p>496. <strong>Vitamin C and Infections</strong> <br />
Harri Hemilä<br />
<em>Nutrients</em> (2017-03-29) <a href="https://doi.org/gfkb9n">https://doi.org/gfkb9n</a> <br />
DOI: <a href="https://doi.org/10.3390/nu9040339">10.3390/nu9040339</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28353648">28353648</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5409678">PMC5409678</a></p>
</div>
<div id="ref-gZRPoTM9">
<p>497. <strong>Vitamin C and the common cold</strong> <br />
Harri Hemilä<br />
<em>British Journal of Nutrition</em> (2007-03-09) <a href="https://doi.org/fszhc6">https://doi.org/fszhc6</a> <br />
DOI: <a href="https://doi.org/10.1079/bjn19920004">10.1079/bjn19920004</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/1547201">1547201</a></p>
</div>
<div id="ref-Pb5PCH7y">
<p>498. <strong>Vitamin C for preventing and treating the common cold</strong> <br />
Harri Hemilä, Elizabeth Chalker<br />
<em>Cochrane Database of Systematic Reviews</em> (2013-01-31) <a href="https://doi.org/xz5">https://doi.org/xz5</a> <br />
DOI: <a href="https://doi.org/10.1002/14651858.cd000980.pub4">10.1002/14651858.cd000980.pub4</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23440782">23440782</a></p>
</div>
<div id="ref-JARvpqRw">
<p>499. <strong>Vitamin C intake and susceptibility to pneumonia</strong> <br />
HARRI HEMILÄ<br />
<em>The Pediatric Infectious Disease Journal</em> (1997-09) <a href="https://doi.org/fkvs9d">https://doi.org/fkvs9d</a> <br />
DOI: <a href="https://doi.org/10.1097/00006454-199709000-00003">10.1097/00006454-199709000-00003</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/9306475">9306475</a></p>
</div>
<div id="ref-DPhRTQw0">
<p>500. <strong>Vitamin C Can Shorten the Length of Stay in the ICU: A Meta-Analysis</strong> <br />
Harri Hemilä, Elizabeth Chalker<br />
<em>Nutrients</em> (2019-03-27) <a href="https://doi.org/gfzscg">https://doi.org/gfzscg</a> <br />
DOI: <a href="https://doi.org/10.3390/nu11040708">10.3390/nu11040708</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30934660">30934660</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6521194">PMC6521194</a></p>
</div>
<div id="ref-Rt5Aik4p">
<p>501. <strong>Effect of Vitamin C Infusion on Organ Failure and Biomarkers of Inflammation and Vascular Injury in Patients With Sepsis and Severe Acute Respiratory Failure</strong> <br />
Alpha A. Fowler, Jonathon D. Truwit, R. Duncan Hite, Peter E. Morris, Christine DeWilde, Anna Priday, Bernard Fisher, Leroy R. Thacker, Ramesh Natarajan, Donald F. Brophy, … Matthew Halquist<br />
<em>JAMA</em> (2019-10-01) <a href="https://doi.org/ggqmh8">https://doi.org/ggqmh8</a> <br />
DOI: <a href="https://doi.org/10.1001/jama.2019.11825">10.1001/jama.2019.11825</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31573637">31573637</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6777268">PMC6777268</a></p>
</div>
<div id="ref-wBwLIQkU">
<p>502. <strong>Vitamin C Infusion for the Treatment of Severe 2019-nCoV Infected Pneumonia: a Prospective Randomized Clinical Trial</strong> <br />
ZhiYong Peng<br />
<em>clinicaltrials.gov</em> (2020-10-08) <a href="https://clinicaltrials.gov/ct2/show/NCT04264533">https://clinicaltrials.gov/ct2/show/NCT04264533</a></p>
</div>
<div id="ref-rSyBZFhE">
<p>503. <strong>A new clinical trial to test high-dose vitamin C in patients with COVID-19</strong> <br />
Anitra C. Carr<br />
<em>Critical Care</em> (2020-04-07) <a href="https://doi.org/ggsbmg">https://doi.org/ggsbmg</a> <br />
DOI: <a href="https://doi.org/10.1186/s13054-020-02851-4">10.1186/s13054-020-02851-4</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32264963">32264963</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7137406">PMC7137406</a></p>
</div>
<div id="ref-16435N02u">
<p>504. <strong>Use of Ascorbic Acid in Patients With COVID 19</strong> <br />
Salvatore Corrao MD<br />
<em>clinicaltrials.gov</em> (2020-03-24) <a href="https://clinicaltrials.gov/ct2/show/NCT04323514">https://clinicaltrials.gov/ct2/show/NCT04323514</a></p>
</div>
<div id="ref-n3v9Thlz">
<p>505. <strong>Vitamin D and the Immune System</strong> <br />
Cynthia Aranow<br />
<em>Journal of Investigative Medicine</em> (2015-12-15) <a href="https://doi.org/f3wh87">https://doi.org/f3wh87</a> <br />
DOI: <a href="https://doi.org/10.2310/jim.0b013e31821b8755">10.2310/jim.0b013e31821b8755</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21527855">21527855</a></p>
</div>
<div id="ref-iWLsflrG">
<p>506. <strong>Vitamin D in the prevention of acute respiratory infection: Systematic review of clinical studies</strong> <br />
David A. Jolliffe, Christopher J. Griffiths, Adrian R. Martineau<br />
<em>The Journal of Steroid Biochemistry and Molecular Biology</em> (2013-07) <a href="https://doi.org/ggqmh9">https://doi.org/ggqmh9</a> <br />
DOI: <a href="https://doi.org/10.1016/j.jsbmb.2012.11.017">10.1016/j.jsbmb.2012.11.017</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23220552">23220552</a></p>
</div>
<div id="ref-nwEQBjV4">
<p>507. <strong>Vitamin D: modulator of the immune system</strong> <br />
Femke Baeke, Tatiana Takiishi, Hannelie Korf, Conny Gysemans, Chantal Mathieu<br />
<em>Current Opinion in Pharmacology</em> (2010-08) <a href="https://doi.org/d43qtf">https://doi.org/d43qtf</a> <br />
DOI: <a href="https://doi.org/10.1016/j.coph.2010.04.001">10.1016/j.coph.2010.04.001</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20427238">20427238</a></p>
</div>
<div id="ref-9ac0GYir">
<p>508. <strong>Evidence that Vitamin D Supplementation Could Reduce Risk of Influenza and COVID-19 Infections and Deaths</strong> <br />
William B. Grant, Henry Lahore, Sharon L. McDonnell, Carole A. Baggerly, Christine B. French, Jennifer L. Aliano, Harjit P. Bhattoa<br />
<em>Nutrients</em> (2020-04-02) <a href="https://doi.org/ggr2v5">https://doi.org/ggr2v5</a> <br />
DOI: <a href="https://doi.org/10.3390/nu12040988">10.3390/nu12040988</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32252338">32252338</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7231123">PMC7231123</a></p>
</div>
<div id="ref-1Gy7Svl72">
<p>509. <strong>25-Hydroxyvitamin D Concentrations Are Lower in Patients with Positive PCR for SARS-CoV-2</strong> <br />
Antonio D’Avolio, Valeria Avataneo, Alessandra Manca, Jessica Cusato, Amedeo De Nicolò, Renzo Lucchini, Franco Keller, Marco Cantù<br />
<em>Nutrients</em> (2020-05-09) <a href="https://doi.org/ggvv76">https://doi.org/ggvv76</a> <br />
DOI: <a href="https://doi.org/10.3390/nu12051359">10.3390/nu12051359</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32397511">32397511</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7285131">PMC7285131</a></p>
</div>
<div id="ref-GMAEaWD1">
<p>510. <strong>Vitamin D deficiency as risk factor for severe COVID-19: a convergence of two pandemics</strong> <br />
Dieter De Smet, Kristof De Smet, Pauline Herroelen, Stefaan Gryspeerdt, Geert Antoine Martens<br />
<em>Cold Spring Harbor Laboratory</em> (2020-05-05) <a href="https://doi.org/ggvv75">https://doi.org/ggvv75</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.05.01.20079376">10.1101/2020.05.01.20079376</a></p>
</div>
<div id="ref-GM9W6HPG">
<p>511. <strong>Vitamin D sufficiency, a serum 25-hydroxyvitamin D at least 30 ng/mL reduced risk for adverse clinical outcomes in patients with COVID-19 infection</strong> <br />
Zhila Maghbooli, Mohammad Ali Sahraian, Mehdi Ebrahimi, Marzieh Pazoki, Samira Kafan, Hedieh Moradi Tabriz, Azar Hadadi, Mahnaz Montazeri, Mehrad Nasiri, Arash Shirvani, Michael F. Holick<br />
<em>PLOS ONE</em> (2020-09-25) <a href="https://doi.org/ghdzx8">https://doi.org/ghdzx8</a> <br />
DOI: <a href="https://doi.org/10.1371/journal.pone.0239799">10.1371/journal.pone.0239799</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32976513">32976513</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7518605">PMC7518605</a></p>
</div>
<div id="ref-kN3eqnZW">
<p>512. <strong>Role of vitamin D in preventing of COVID-19 infection, progression and severity</strong> <br />
Nurshad Ali<br />
<em>Journal of Infection and Public Health</em> (2020-06) <a href="https://doi.org/ghdzw9">https://doi.org/ghdzw9</a> <br />
DOI: <a href="https://doi.org/10.1016/j.jiph.2020.06.021">10.1016/j.jiph.2020.06.021</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32605780">32605780</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7305922">PMC7305922</a></p>
</div>
<div id="ref-5MvUc8IL">
<p>513. <strong>Low plasma 25(OH) vitamin D level is associated with increased risk of COVID‐19 infection: an Israeli population‐based study</strong> <br />
Eugene Merzon, Dmitry Tworowski, Alessandro Gorohovski, Shlomo Vinker, Avivit Golan Cohen, Ilan Green, Milana Frenkel Morgenstern<br />
<em>The FEBS Journal</em> (2020-07-23) <a href="https://doi.org/gg7b5c">https://doi.org/gg7b5c</a> <br />
DOI: <a href="https://doi.org/10.1111/febs.15495">10.1111/febs.15495</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32700398">32700398</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7404739">PMC7404739</a></p>
</div>
<div id="ref-18O6t6Jo">
<p>514. <strong>Association of Vitamin D Status and Other Clinical Characteristics With COVID-19 Test Results</strong> <br />
David O. Meltzer, Thomas J. Best, Hui Zhang, Tamara Vokes, Vineet Arora, Julian Solway<br />
<em>JAMA Network Open</em> (2020-09-03) <a href="https://doi.org/ghdzw6">https://doi.org/ghdzw6</a> <br />
DOI: <a href="https://doi.org/10.1001/jamanetworkopen.2020.19722">10.1001/jamanetworkopen.2020.19722</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32880651">32880651</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7489852">PMC7489852</a></p>
</div>
<div id="ref-8ZxZKpYS">
<p>515. <strong>Vitamin D Status in Hospitalized Patients with SARS-CoV-2 Infection</strong> <br />
José L Hernández, Daniel Nan, Marta Fernandez-Ayala, Mayte García-Unzueta, Miguel A Hernández-Hernández, Marcos López-Hoyos, Pedro Muñoz-Cacho, José M Olmos, Manuel Gutiérrez-Cuadra, Juan J Ruiz-Cubillán, … Víctor M Martínez-Taboada<br />
<em>The Journal of Clinical Endocrinology &amp; Metabolism</em> (2020-10-27) <a href="https://doi.org/ghh737">https://doi.org/ghh737</a> <br />
DOI: <a href="https://doi.org/10.1210/clinem/dgaa733">10.1210/clinem/dgaa733</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33159440">33159440</a></p>
</div>
<div id="ref-FZ8yXmxQ">
<p>516. <strong>Vitamin D concentrations and COVID-19 infection in UK Biobank</strong> <br />
Claire E. Hastie, Daniel F. Mackay, Frederick Ho, Carlos A. Celis-Morales, Srinivasa Vittal Katikireddi, Claire L. Niedzwiedz, Bhautesh D. Jani, Paul Welsh, Frances S. Mair, Stuart R. Gray, … Jill P. Pell<br />
<em>Diabetes &amp; Metabolic Syndrome: Clinical Research &amp; Reviews</em> (2020-07) <a href="https://doi.org/ggvv72">https://doi.org/ggvv72</a> <br />
DOI: <a href="https://doi.org/10.1016/j.dsx.2020.04.050">10.1016/j.dsx.2020.04.050</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32413819">32413819</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7204679">PMC7204679</a></p>
</div>
<div id="ref-EtCW6lkq">
<p>517. <strong>Low serum 25‐hydroxyvitamin D (25[OH]D) levels in patients hospitalized with COVID‐19 are associated with greater disease severity</strong> <br />
Grigorios Panagiotou, Su Ann Tee, Yasir Ihsan, Waseem Athar, Gabriella Marchitelli, Donna Kelly, Christopher S. Boot, Nadia Stock, James Macfarlane, Adrian R. Martineau, … Richard Quinton<br />
<em>Clinical Endocrinology</em> (2020-08-06) <a href="https://doi.org/gg5gbj">https://doi.org/gg5gbj</a> <br />
DOI: <a href="https://doi.org/10.1111/cen.14276">10.1111/cen.14276</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32621392">32621392</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7361912">PMC7361912</a></p>
</div>
<div id="ref-ClClnlJb">
<p>518. <strong>Letter in response to the article: Vitamin D concentrations and COVID-19 infection in UK biobank (Hastie et al.)</strong> <br />
W. B. Grant, S. L. McDonnell<br />
<em>Diabetes &amp; Metabolic Syndrome: Clinical Research &amp; Reviews</em> (2020-09) <a href="https://doi.org/ghc7p4">https://doi.org/ghc7p4</a> <br />
DOI: <a href="https://doi.org/10.1016/j.dsx.2020.05.046">10.1016/j.dsx.2020.05.046</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32563941">32563941</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7293469">PMC7293469</a></p>
</div>
<div id="ref-162t7zdGv">
<p>519. <strong>Effect of Vitamin D Administration on Prevention and Treatment of Mild Forms of Suspected Covid-19</strong> <br />
Manuel Castillo Garzón<br />
<em>clinicaltrials.gov</em> (2020-04-03) <a href="https://clinicaltrials.gov/ct2/show/NCT04334005">https://clinicaltrials.gov/ct2/show/NCT04334005</a></p>
</div>
<div id="ref-2K0jV5Bn">
<p>520. <strong>Improving Vitamin D Status in the Management of COVID-19</strong> <br />
Aldo Montano-Loza<br />
<em>clinicaltrials.gov</em> (2020-06-03) <a href="https://clinicaltrials.gov/ct2/show/NCT04385940">https://clinicaltrials.gov/ct2/show/NCT04385940</a></p>
</div>
<div id="ref-rBiojRHk">
<p>521. <strong>Cholecalciferol to Improve the Outcomes of COVID-19 Patients - Full Text View - ClinicalTrials.gov</strong> <a href="https://clinicaltrials.gov/ct2/show/NCT04411446">https://clinicaltrials.gov/ct2/show/NCT04411446</a></p>
</div>
<div id="ref-6kLoX5qT">
<p>522. <strong>COvid-19 and Vitamin D Supplementation: a Multicenter Randomized Controlled Trial of High Dose Versus Standard Dose Vitamin D3 in High-risk COVID-19 Patients (CoVitTrial) - Full Text View - ClinicalTrials.gov</strong> <a href="https://clinicaltrials.gov/ct2/show/NCT04344041">https://clinicaltrials.gov/ct2/show/NCT04344041</a></p>
</div>
<div id="ref-32C2erIe">
<p>523. <strong>The LEAD COVID-19 Trial: Low-risk, Early Aspirin and Vitamin D to Reduce COVID-19 Hospitalizations</strong> <br />
Louisiana State University Health Sciences Center in New Orleans<br />
<em>clinicaltrials.gov</em> (2020-04-24) <a href="https://clinicaltrials.gov/ct2/show/NCT04363840">https://clinicaltrials.gov/ct2/show/NCT04363840</a></p>
</div>
<div id="ref-tLVliODQ">
<p>524. <strong>Randomized Double-Blind Placebo-Controlled Proof-of-Concept Trial of a Plant Polyphenol for the Outpatient Treatment of Mild Coronavirus Disease (COVID-19)</strong> <br />
Marvin McCreary MD<br />
<em>clinicaltrials.gov</em> (2020-09-22) <a href="https://clinicaltrials.gov/ct2/show/NCT04400890">https://clinicaltrials.gov/ct2/show/NCT04400890</a></p>
</div>
<div id="ref-1FyHVZ9yx">
<p>525. <strong>The International Scientific Association for Probiotics and Prebiotics consensus statement on the scope and appropriate use of the term probiotic</strong> <br />
Colin Hill, Francisco Guarner, Gregor Reid, Glenn R. Gibson, Daniel J. Merenstein, Bruno Pot, Lorenzo Morelli, Roberto Berni Canani, Harry J. Flint, Seppo Salminen, … Mary Ellen Sanders<br />
<em>Nature Reviews Gastroenterology &amp; Hepatology</em> (2014-06-10) <a href="https://doi.org/f6ndv7">https://doi.org/f6ndv7</a> <br />
DOI: <a href="https://doi.org/10.1038/nrgastro.2014.66">10.1038/nrgastro.2014.66</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24912386">24912386</a></p>
</div>
<div id="ref-pvZU55k9">
<p>526. <strong>The Effect of Probiotics on Prevention of Common Cold: A Meta-Analysis of Randomized Controlled Trial Studies</strong> <br />
En-Jin Kang, Soo Young Kim, In-Hong Hwang, Yun-Jeong Ji<br />
<em>Korean Journal of Family Medicine</em> (2013) <a href="https://doi.org/gg3knf">https://doi.org/gg3knf</a> <br />
DOI: <a href="https://doi.org/10.4082/kjfm.2013.34.1.2">10.4082/kjfm.2013.34.1.2</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23372900">23372900</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3560336">PMC3560336</a></p>
</div>
<div id="ref-HKeroTXj">
<p>527. <strong>Probiotics and Paraprobiotics in Viral Infection: Clinical Application and Effects on the Innate and Acquired Immune Systems</strong> <br />
Osamu Kanauchi, Akira Andoh, Sazaly AbuBakar, Naoki Yamamoto<br />
<em>Current Pharmaceutical Design</em> (2018-05-10) <a href="https://doi.org/gdjnpk">https://doi.org/gdjnpk</a> <br />
DOI: <a href="https://doi.org/10.2174/1381612824666180116163411">10.2174/1381612824666180116163411</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29345577">29345577</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6006794">PMC6006794</a></p>
</div>
<div id="ref-10e8cHDs8">
<p>528. <strong>Using Probiotics to Flatten the Curve of Coronavirus Disease COVID-2019 Pandemic</strong> <br />
David Baud, Varvara Dimopoulou Agri, Glenn R. Gibson, Gregor Reid, Eric Giannoni<br />
<em>Frontiers in Public Health</em> (2020-05-08) <a href="https://doi.org/gg3knd">https://doi.org/gg3knd</a> <br />
DOI: <a href="https://doi.org/10.3389/fpubh.2020.00186">10.3389/fpubh.2020.00186</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32574290">32574290</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7227397">PMC7227397</a></p>
</div>
<div id="ref-5fU7dexB">
<p>529. <strong>Next-generation probiotics: the spectrum from probiotics to live biotherapeutics</strong> <br />
Paul W. O’Toole, Julian R. Marchesi, Colin Hill<br />
<em>Nature Microbiology</em> (2017-04-25) <a href="https://doi.org/ggzggv">https://doi.org/ggzggv</a> <br />
DOI: <a href="https://doi.org/10.1038/nmicrobiol.2017.57">10.1038/nmicrobiol.2017.57</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28440276">28440276</a></p>
</div>
<div id="ref-1GCrGCp5P">
<p>530. <strong>Mechanisms of Action of Probiotics</strong> <br />
Julio Plaza-Diaz, Francisco Javier Ruiz-Ojeda, Mercedes Gil-Campos, Angel Gil<br />
<em>Advances in Nutrition</em> (2019-01) <a href="https://doi.org/gft8sh">https://doi.org/gft8sh</a> <br />
DOI: <a href="https://doi.org/10.1093/advances/nmy063">10.1093/advances/nmy063</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30721959">30721959</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6363529">PMC6363529</a></p>
</div>
<div id="ref-B2GeSSQo">
<p>531. <strong>Probiotic mechanisms of action</strong> <br />
Katrina Halloran, Mark A. Underwood<br />
<em>Early Human Development</em> (2019-08) <a href="https://doi.org/gg3jc4">https://doi.org/gg3jc4</a> <br />
DOI: <a href="https://doi.org/10.1016/j.earlhumdev.2019.05.010">10.1016/j.earlhumdev.2019.05.010</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31174927">31174927</a></p>
</div>
<div id="ref-nmXF8vkS">
<p>532. <strong>Probiotic Mechanisms of Action</strong> <br />
Miriam Bermudez-Brito, Julio Plaza-Díaz, Sergio Muñoz-Quezada, Carolina Gómez-Llorente, Angel Gil<br />
<em>Annals of Nutrition and Metabolism</em> (2012) <a href="https://doi.org/gg3knb">https://doi.org/gg3knb</a> <br />
DOI: <a href="https://doi.org/10.1159/000342079">10.1159/000342079</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23037511">23037511</a></p>
</div>
<div id="ref-13AWoYvqB">
<p>533. <strong>Pulmonary-intestinal cross-talk in mucosal inflammatory disease</strong> <br />
S Keely, NJ Talley, PM Hansbro<br />
<em>Mucosal Immunology</em> (2011-11-16) <a href="https://doi.org/b5knk2">https://doi.org/b5knk2</a> <br />
DOI: <a href="https://doi.org/10.1038/mi.2011.55">10.1038/mi.2011.55</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22089028">22089028</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3243663">PMC3243663</a></p>
</div>
<div id="ref-Smqu1EYN">
<p>534. <strong>The role of the lung microbiota and the gut-lung axis in respiratory infectious diseases</strong> <br />
Alexia Dumas, Lucie Bernard, Yannick Poquet, Geanncarlo Lugo-Villarino, Olivier Neyrolles<br />
<em>Cellular Microbiology</em> (2018-12) <a href="https://doi.org/gfjds9">https://doi.org/gfjds9</a> <br />
DOI: <a href="https://doi.org/10.1111/cmi.12966">10.1111/cmi.12966</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30329198">30329198</a></p>
</div>
<div id="ref-d27mQ4kO">
<p>535. <strong>Gut microbiota and Covid-19- possible link and implications</strong> <br />
Debojyoti Dhar, Abhishek Mohanty<br />
<em>Virus Research</em> (2020-08) <a href="https://doi.org/gg3jc5">https://doi.org/gg3jc5</a> <br />
DOI: <a href="https://doi.org/10.1016/j.virusres.2020.198018">10.1016/j.virusres.2020.198018</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32430279">32430279</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7217790">PMC7217790</a></p>
</div>
<div id="ref-1MIGrWIP">
<p>536. <strong>Probiotics in respiratory virus infections</strong> <br />
L. Lehtoranta, A. Pitkäranta, R. Korpela<br />
<em>European Journal of Clinical Microbiology &amp; Infectious Diseases</em> (2014-03-18) <a href="https://doi.org/f583jr">https://doi.org/f583jr</a> <br />
DOI: <a href="https://doi.org/10.1007/s10096-014-2086-y">10.1007/s10096-014-2086-y</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24638909">24638909</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7088122">PMC7088122</a></p>
</div>
<div id="ref-58eXvYEV">
<p>537. <strong>Probiotics for preventing acute upper respiratory tract infections</strong> <br />
Qiukui Hao, Bi Rong Dong, Taixiang Wu<br />
<em>Cochrane Database of Systematic Reviews</em> (2015-02-03) <a href="https://doi.org/gg3jc3">https://doi.org/gg3jc3</a> <br />
DOI: <a href="https://doi.org/10.1002/14651858.cd006895.pub3">10.1002/14651858.cd006895.pub3</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25927096">25927096</a></p>
</div>
<div id="ref-kjavvCzp">
<p>538. <strong>Probiotics for the prevention of respiratory tract infections: a systematic review</strong> <br />
Evridiki K. Vouloumanou, Gregory C. Makris, Drosos E. Karageorgopoulos, Matthew E. Falagas<br />
<em>International Journal of Antimicrobial Agents</em> (2009-09) <a href="https://doi.org/dn8kw8">https://doi.org/dn8kw8</a> <br />
DOI: <a href="https://doi.org/10.1016/j.ijantimicag.2008.11.005">10.1016/j.ijantimicag.2008.11.005</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19179052">19179052</a></p>
</div>
<div id="ref-dJIGDt4k">
<p>539. <strong>Effectiveness of probiotics on the duration of illness in healthy children and adults who develop common acute respiratory infectious conditions: a systematic review and meta-analysis</strong> <br />
Sarah King, Julie Glanville, Mary Ellen Sanders, Anita Fitzgerald, Danielle Varley<br />
<em>British Journal of Nutrition</em> (2014-04-29) <a href="https://doi.org/f57hq5">https://doi.org/f57hq5</a> <br />
DOI: <a href="https://doi.org/10.1017/s0007114514000075">10.1017/s0007114514000075</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24780623">24780623</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4054664">PMC4054664</a></p>
</div>
<div id="ref-AJav6c61">
<p>540. <strong>Effect of probiotics on the incidence of ventilator-associated pneumonia in critically ill patients: a randomized controlled multicenter trial</strong> <br />
Juan Zeng, Chun-Ting Wang, Fu-Shen Zhang, Feng Qi, Shi-Fu Wang, Shuang Ma, Tie-Jun Wu, Hui Tian, Zhao-Tao Tian, Shu-Liu Zhang, … Yu-Ping Wang<br />
<em>Intensive Care Medicine</em> (2016-04-04) <a href="https://doi.org/f8jnrt">https://doi.org/f8jnrt</a> <br />
DOI: <a href="https://doi.org/10.1007/s00134-016-4303-x">10.1007/s00134-016-4303-x</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27043237">27043237</a></p>
</div>
<div id="ref-1E2aDJp4T">
<p>541. <strong>Probiotic Prophylaxis of Ventilator-associated Pneumonia</strong> <br />
Lee E. Morrow, Marin H. Kollef, Thomas B. Casale<br />
<em>American Journal of Respiratory and Critical Care Medicine</em> (2010-10-15) <a href="https://doi.org/d5hh4t">https://doi.org/d5hh4t</a> <br />
DOI: <a href="https://doi.org/10.1164/rccm.200912-1853oc">10.1164/rccm.200912-1853oc</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20522788">20522788</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2970846">PMC2970846</a></p>
</div>
<div id="ref-zccFgajh">
<p>542. <strong>Synbiotics modulate gut microbiota and reduce enteritis and ventilator-associated pneumonia in patients with sepsis: a randomized controlled trial</strong> <br />
Kentaro Shimizu, Tomoki Yamada, Hiroshi Ogura, Tomoyoshi Mohri, Takeyuki Kiguchi, Satoshi Fujimi, Takashi Asahara, Tomomi Yamada, Masahiro Ojima, Mitsunori Ikeda, Takeshi Shimazu<br />
<em>Critical Care</em> (2018-09-27) <a href="https://doi.org/gfdggj">https://doi.org/gfdggj</a> <br />
DOI: <a href="https://doi.org/10.1186/s13054-018-2167-x">10.1186/s13054-018-2167-x</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30261905">30261905</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6161427">PMC6161427</a></p>
</div>
<div id="ref-1DI06gFPc">
<p>543. <strong>Probiotics for the Prevention of Ventilator-Associated Pneumonia: A Meta-Analysis of Randomized Controlled Trials</strong> <br />
Minmin Su, Ying Jia, Yan Li, Dianyou Zhou, Jinsheng Jia<br />
<em>Respiratory Care</em> (2020-03-03) <a href="https://doi.org/gg3kng">https://doi.org/gg3kng</a> <br />
DOI: <a href="https://doi.org/10.4187/respcare.07097">10.4187/respcare.07097</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32127415">32127415</a></p>
</div>
<div id="ref-164mVBFZx">
<p>544. <strong>COVID-19: An Alert to Ventilator-Associated Bacterial Pneumonia</strong> <br />
Helvécio Cardoso Corrêa Póvoa, Gabriela Ceccon Chianca, Natalia Lopes Pontes Póvoa Iorio<br />
<em>Adis Journals</em> (2020) <a href="https://doi.org/gg3knh">https://doi.org/gg3knh</a> <br />
DOI: <a href="https://doi.org/10.6084/m9.figshare.12340496">10.6084/m9.figshare.12340496</a></p>
</div>
<div id="ref-w2VFo0h1">
<p>545. <strong>The challenge of ventilator-associated pneumonia diagnosis in COVID-19 patients</strong> <br />
Bruno François, Pierre-François Laterre, Charles-Edouard Luyt, Jean Chastre<br />
<em>Critical Care</em> (2020-06-05) <a href="https://doi.org/gg3knc">https://doi.org/gg3knc</a> <br />
DOI: <a href="https://doi.org/10.1186/s13054-020-03013-2">10.1186/s13054-020-03013-2</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32503590">32503590</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7273812">PMC7273812</a></p>
</div>
<div id="ref-gcyQ73v7">
<p>546. <strong>Prophylactic use of probiotics for gastrointestinal disorders in children</strong> <br />
Celine Perceval, Hania Szajewska, Flavia Indrio, Zvi Weizman, Yvan Vandenplas<br />
<em>The Lancet Child &amp; Adolescent Health</em> (2019-09) <a href="https://doi.org/d2qp">https://doi.org/d2qp</a> <br />
DOI: <a href="https://doi.org/10.1016/s2352-4642(19)30182-8">10.1016/s2352-4642(19)30182-8</a></p>
</div>
<div id="ref-67ihxDiV">
<p>547. <strong>Effect of gastrointestinal symptoms on patients infected with COVID-19</strong> <br />
Zili Zhou, Ning Zhao, Yan Shu, Shengbo Han, Bin Chen, Xiaogang Shu<br />
<em>Gastroenterology</em> (2020-03) <a href="https://doi.org/ggq8x8">https://doi.org/ggq8x8</a> <br />
DOI: <a href="https://doi.org/10.1053/j.gastro.2020.03.020">10.1053/j.gastro.2020.03.020</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32199880">32199880</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7270807">PMC7270807</a></p>
</div>
<div id="ref-8Hvgbl3n">
<p>548. <strong>The digestive system is a potential route of 2019-nCov infection: a bioinformatics analysis based on single-cell transcriptomes</strong> <br />
Hao Zhang, Zijian Kang, Haiyi Gong, Da Xu, Jing Wang, Zifu Li, Xingang Cui, Jianru Xiao, Tong Meng, Wang Zhou, … Huji Xu<br />
<em>bioRxiv</em> (2020-01-31) <a href="https://doi.org/ggjvx2">https://doi.org/ggjvx2</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.01.30.927806">10.1101/2020.01.30.927806</a></p>
</div>
<div id="ref-Nhxoefru">
<p>549. <strong>Cryo-electron microscopy structures of the SARS-CoV spike glycoprotein reveal a prerequisite conformational state for receptor binding</strong> <br />
Miao Gui, Wenfei Song, Haixia Zhou, Jingwei Xu, Silian Chen, Ye Xiang, Xinquan Wang<br />
<em>Cell Research</em> (2016-12-23) <a href="https://doi.org/f9m247">https://doi.org/f9m247</a> <br />
DOI: <a href="https://doi.org/10.1038/cr.2016.152">10.1038/cr.2016.152</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28008928">28008928</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5223232">PMC5223232</a></p>
</div>
<div id="ref-TPWaxApy">
<p>550. <strong>Prolonged presence of SARS-CoV-2 viral RNA in faecal samples</strong> <br />
Yongjian Wu, Cheng Guo, Lantian Tang, Zhongsi Hong, Jianhui Zhou, Xin Dong, Huan Yin, Qiang Xiao, Yanping Tang, Xiujuan Qu, … Xi Huang<br />
<em>The Lancet Gastroenterology &amp; Hepatology</em> (2020-05) <a href="https://doi.org/ggq8zp">https://doi.org/ggq8zp</a> <br />
DOI: <a href="https://doi.org/10.1016/s2468-1253(20)30083-2">10.1016/s2468-1253(20)30083-2</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32199469">32199469</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7158584">PMC7158584</a></p>
</div>
<div id="ref-r9HNMqTS">
<p>551. <strong>Enteric involvement of coronaviruses: is faecal–oral transmission of SARS-CoV-2 possible?</strong> <br />
Charleen Yeo, Sanghvi Kaushal, Danson Yeo<br />
<em>The Lancet Gastroenterology &amp; Hepatology</em> (2020-02) <a href="https://doi.org/ggpx7s">https://doi.org/ggpx7s</a> <br />
DOI: <a href="https://doi.org/10.1016/s2468-1253(20)30048-0">10.1016/s2468-1253(20)30048-0</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32087098">32087098</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7130008">PMC7130008</a></p>
</div>
<div id="ref-eOcc1BUx">
<p>552. <strong>Characteristics of pediatric SARS-CoV-2 infection and potential evidence for persistent fecal viral shedding</strong> <br />
Yi Xu, Xufang Li, Bing Zhu, Huiying Liang, Chunxiao Fang, Yu Gong, Qiaozhi Guo, Xin Sun, Danyang Zhao, Jun Shen, … Sitang Gong<br />
<em>Nature Medicine</em> (2020-03-13) <a href="https://doi.org/ggpwx5">https://doi.org/ggpwx5</a> <br />
DOI: <a href="https://doi.org/10.1038/s41591-020-0817-4">10.1038/s41591-020-0817-4</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32284613">32284613</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7095102">PMC7095102</a></p>
</div>
<div id="ref-VwStn9Ps">
<p>553. <strong>Modulation of rotavirus severe gastroenteritis by the combination of probiotics and prebiotics</strong> <br />
Guadalupe Gonzalez-Ochoa, Lilian K. Flores-Mendoza, Ramona Icedo-Garcia, Ricardo Gomez-Flores, Patricia Tamez-Guerra<br />
<em>Archives of Microbiology</em> (2017-06-20) <a href="https://doi.org/gbsb4d">https://doi.org/gbsb4d</a> <br />
DOI: <a href="https://doi.org/10.1007/s00203-017-1400-3">10.1007/s00203-017-1400-3</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28634691">28634691</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5548957">PMC5548957</a></p>
</div>
<div id="ref-UtAKolza">
<p>554. <strong>Multicenter Trial of a Combination Probiotic for Children with Gastroenteritis</strong> <br />
Stephen B. Freedman, Sarah Williamson-Urquhart, Ken J. Farion, Serge Gouin, Andrew R. Willan, Naveen Poonai, Katrina Hurley, Philip M. Sherman, Yaron Finkelstein, Bonita E. Lee, … Suzanne Schuh<br />
<em>New England Journal of Medicine</em> (2018-11-22) <a href="https://doi.org/gfkbsf">https://doi.org/gfkbsf</a> <br />
DOI: <a href="https://doi.org/10.1056/nejmoa1802597">10.1056/nejmoa1802597</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30462939">30462939</a></p>
</div>
<div id="ref-1AqF1ESKq">
<p>555. <strong>Synbiotic Therapy of Gastrointestinal Symptoms During Covid-19 Infection: A Randomized, Double-blind, Placebo Controlled, Telemedicine Study (SynCov Study)</strong> <br />
Medical University of Graz<br />
<em>clinicaltrials.gov</em> (2020-10-20) <a href="https://clinicaltrials.gov/ct2/show/NCT04420676">https://clinicaltrials.gov/ct2/show/NCT04420676</a></p>
</div>
<div id="ref-9UYsmETk">
<p>556. <strong>Multicentric Study to Assess the Effect of Consumption of Lactobacillus Coryniformis K8 on Healthcare Personnel Exposed to COVID-19</strong> <br />
Biosearch S.A.<br />
<em>clinicaltrials.gov</em> (2020-04-28) <a href="https://clinicaltrials.gov/ct2/show/NCT04366180">https://clinicaltrials.gov/ct2/show/NCT04366180</a></p>
</div>
<div id="ref-15pDlYtom">
<p>557. <strong>The Intestinal Microbiota as a Therapeutic Target in Hospitalized Patients With COVID-19 Infection</strong> <br />
Bioithas SL<br />
<em>clinicaltrials.gov</em> (2020-05-14) <a href="https://clinicaltrials.gov/ct2/show/NCT04390477">https://clinicaltrials.gov/ct2/show/NCT04390477</a></p>
</div>
<div id="ref-m3SexO9F">
<p>558. <strong>Probiotics: definition, scope and mechanisms of action</strong> <br />
Gregor Reid<br />
<em>Best Practice &amp; Research Clinical Gastroenterology</em> (2016-02) <a href="https://doi.org/f8m79k">https://doi.org/f8m79k</a> <br />
DOI: <a href="https://doi.org/10.1016/j.bpg.2015.12.001">10.1016/j.bpg.2015.12.001</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27048893">27048893</a></p>
</div>
<div id="ref-5FTOk7ol">
<p>559. <strong>Health benefits and health claims of probiotics: bridging science and marketing</strong> <br />
Ger T. Rijkers, Willem M. de Vos, Robert-Jan Brummer, Lorenzo Morelli, Gerard Corthier, Philippe Marteau<br />
<em>British Journal of Nutrition</em> (2011-08-24) <a href="https://doi.org/cb78rx">https://doi.org/cb78rx</a> <br />
DOI: <a href="https://doi.org/10.1017/s000711451100287x">10.1017/s000711451100287x</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21861940">21861940</a></p>
</div>
<div id="ref-kv94VRLF">
<p>560. <strong>Probiotics and COVID-19: one size does not fit all</strong> <br />
Joyce WY Mak, Francis KL Chan, Siew C Ng<br />
<em>The Lancet Gastroenterology &amp; Hepatology</em> (2020-07) <a href="https://doi.org/d2qq">https://doi.org/d2qq</a> <br />
DOI: <a href="https://doi.org/10.1016/s2468-1253(20)30122-9">10.1016/s2468-1253(20)30122-9</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32339473">32339473</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7182525">PMC7182525</a></p>
</div>
<div id="ref-5tHMedSi">
<p>561. <strong>Optimal Nutritional Status for a Well-Functioning Immune System Is an Important Factor to Protect against Viral Infections</strong> <br />
Philip C. Calder, Anitra C. Carr, Adrian F. Gombart, Manfred Eggersdorfer<br />
<em>Nutrients</em> (2020-04-23) <a href="https://doi.org/gg29hh">https://doi.org/gg29hh</a> <br />
DOI: <a href="https://doi.org/10.3390/nu12041181">10.3390/nu12041181</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32340216">32340216</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7230749">PMC7230749</a></p>
</div>
<div id="ref-Lyd8s4Mk">
<p>562. <strong>COVID-19: The Inflammation Link and the Role of Nutrition in Potential Mitigation</strong> <br />
Ioannis Zabetakis, Ronan Lordan, Catherine Norton, Alexandros Tsoupras<br />
<em>Nutrients</em> (2020-05-19) <a href="https://doi.org/ggxdq3">https://doi.org/ggxdq3</a> <br />
DOI: <a href="https://doi.org/10.3390/nu12051466">10.3390/nu12051466</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32438620">32438620</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7284818">PMC7284818</a></p>
</div>
<div id="ref-1DmkRczqw">
<p>563. <strong>Nutritional management of COVID-19 patients in a rehabilitation unit</strong> <br />
Luigia Brugliera, Alfio Spina, Paola Castellazzi, Paolo Cimino, Pietro Arcuri, Alessandra Negro, Elise Houdayer, Federica Alemanno, Alessandra Giordani, Pietro Mortini, Sandro Iannaccone<br />
<em>European Journal of Clinical Nutrition</em> (2020-05-20) <a href="https://doi.org/gg29hf">https://doi.org/gg29hf</a> <br />
DOI: <a href="https://doi.org/10.1038/s41430-020-0664-x">10.1038/s41430-020-0664-x</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32433599">32433599</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7237874">PMC7237874</a></p>
</div>
<div id="ref-lMhFSIiF">
<p>564. <strong>The frontier between nutrition and pharma: The international regulatory framework of functional foods, food supplements and nutraceuticals</strong> <br />
Laura Domínguez Díaz, Virginia Fernández-Ruiz, Montaña Cámara<br />
<em>Critical Reviews in Food Science and Nutrition</em> (2019-03-29) <a href="https://doi.org/ggqs3w">https://doi.org/ggqs3w</a> <br />
DOI: <a href="https://doi.org/10.1080/10408398.2019.1592107">10.1080/10408398.2019.1592107</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30924346">30924346</a></p>
</div>
<div id="ref-fez8aZAu">
<p>565. <strong>Coronavirus Update: FDA and FTC Warn Seven Companies Selling Fraudulent Products that Claim to Treat or Prevent COVID-19</strong> <br />
Office of the Commissioner<br />
<em>FDA</em> (2020-03-27) <a href="https://www.fda.gov/news-events/press-announcements/coronavirus-update-fda-and-ftc-warn-seven-companies-selling-fraudulent-products-claim-treat-or">https://www.fda.gov/news-events/press-announcements/coronavirus-update-fda-and-ftc-warn-seven-companies-selling-fraudulent-products-claim-treat-or</a></p>
</div>
<div id="ref-2Xu90WtD">
<p>566. <strong>Coronavirus Disease 2019 (COVID-19) – Prevention &amp; Treatment</strong> <br />
CDC<br />
<em>Centers for Disease Control and Prevention</em> (2020-11-04) <a href="https://www.cdc.gov/coronavirus/2019-ncov/prevent-getting-sick/prevention.html">https://www.cdc.gov/coronavirus/2019-ncov/prevent-getting-sick/prevention.html</a></p>
</div>
<div id="ref-EhbZIMo0">
<p>567. <strong>Potential roles of social distancing in mitigating the spread of coronavirus disease 2019 (COVID-19) in South Korea</strong> <br />
Sang Woo Park, Kaiyuan Sun, Cécile Viboud, Bryan T Grenfell, Jonathan Dushoff<br />
<em>Cold Spring Harbor Laboratory</em> (2020-03-30) <a href="https://doi.org/gg3mhg">https://doi.org/gg3mhg</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.27.20045815">10.1101/2020.03.27.20045815</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32511429">32511429</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7217070">PMC7217070</a></p>
</div>
<div id="ref-12Met0WvX">
<p>568. <strong>Evaluating the Effectiveness of Social Distancing Interventions to Delay or Flatten the Epidemic Curve of Coronavirus Disease</strong> <br />
Laura Matrajt, Tiffany Leung<br />
<em>Emerging Infectious Diseases</em> (2020-08) <a href="https://doi.org/ggtx3k">https://doi.org/ggtx3k</a> <br />
DOI: <a href="https://doi.org/10.3201/eid2608.201093">10.3201/eid2608.201093</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32343222">32343222</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7392458">PMC7392458</a></p>
</div>
<div id="ref-1ALDN7rfe">
<p>569. <strong>Formulation and manufacturability of biologics</strong> <br />
Steven J Shire<br />
<em>Current Opinion in Biotechnology</em> (2009-12) <a href="https://doi.org/cjk8p6">https://doi.org/cjk8p6</a> <br />
DOI: <a href="https://doi.org/10.1016/j.copbio.2009.10.006">10.1016/j.copbio.2009.10.006</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19880308">19880308</a></p>
</div>
<div id="ref-OgYDNJt3">
<p>570. <strong>Early Development of Therapeutic Biologics - Pharmacokinetics</strong> <br />
A. Baumann<br />
<em>Current Drug Metabolism</em> (2006-01-01) <a href="https://doi.org/bhcz79">https://doi.org/bhcz79</a> <br />
DOI: <a href="https://doi.org/10.2174/138920006774832604">10.2174/138920006774832604</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16454690">16454690</a></p>
</div>
<div id="ref-GO8FwtPx">
<p>571. <strong>Deriving Immune Modulating Drugs from Viruses—A New Class of Biologics</strong> <br />
Jordan R. Yaron, Liqiang Zhang, Qiuyun Guo, Michelle Burgin, Lauren N. Schutz, Enkidia Awo, Lyn Wise, Kurt L. Krause, Cristhian J. Ildefonso, Jacek M. Kwiecien, … Alexandra R. Lucas<br />
<em>Journal of Clinical Medicine</em> (2020-03-31) <a href="https://doi.org/ghdx73">https://doi.org/ghdx73</a> <br />
DOI: <a href="https://doi.org/10.3390/jcm9040972">10.3390/jcm9040972</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32244484">32244484</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7230489">PMC7230489</a></p>
</div>
<div id="ref-NDRyNd61">
<p>572. <strong>IL-6 in Inflammation, Immunity, and Disease</strong> <br />
T. Tanaka, M. Narazaki, T. Kishimoto<br />
<em>Cold Spring Harbor Perspectives in Biology</em> (2014-09-04) <a href="https://doi.org/gftpjs">https://doi.org/gftpjs</a> <br />
DOI: <a href="https://doi.org/10.1101/cshperspect.a016295">10.1101/cshperspect.a016295</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25190079">25190079</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4176007">PMC4176007</a></p>
</div>
<div id="ref-VXDsRwHC">
<p>573. <strong>Hall of Fame among Pro-inflammatory Cytokines: Interleukin-6 Gene and Its Transcriptional Regulation Mechanisms</strong> <br />
Yang Luo, Song Guo Zheng<br />
<em>Frontiers in Immunology</em> (2016-12-19) <a href="https://doi.org/ggqmgv">https://doi.org/ggqmgv</a> <br />
DOI: <a href="https://doi.org/10.3389/fimmu.2016.00604">10.3389/fimmu.2016.00604</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28066415">28066415</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5165036">PMC5165036</a></p>
</div>
<div id="ref-14xZS2Gic">
<p>574. <strong>IL-6 Trans-Signaling via the Soluble IL-6 Receptor: Importance for the Pro-Inflammatory Activities of IL-6</strong> <br />
Stefan Rose-John<br />
<em>International Journal of Biological Sciences</em> (2012) <a href="https://doi.org/f4c4hf">https://doi.org/f4c4hf</a> <br />
DOI: <a href="https://doi.org/10.7150/ijbs.4989">10.7150/ijbs.4989</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23136552">23136552</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3491447">PMC3491447</a></p>
</div>
<div id="ref-LIqxTdN">
<p>575. <strong>Interleukin-6 and its receptor: from bench to bedside</strong> <br />
Jürgen Scheller, Stefan Rose-John<br />
<em>Medical Microbiology and Immunology</em> (2006-05-31) <a href="https://doi.org/ck8xch">https://doi.org/ck8xch</a> <br />
DOI: <a href="https://doi.org/10.1007/s00430-006-0019-9">10.1007/s00430-006-0019-9</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16741736">16741736</a></p>
</div>
<div id="ref-1BxkPZYnj">
<p>576. <strong>Plasticity and cross-talk of Interleukin 6-type cytokines</strong> <br />
Christoph Garbers, Heike M. Hermanns, Fred Schaper, Gerhard Müller-Newen, Joachim Grötzinger, Stefan Rose-John, Jürgen Scheller<br />
<em>Cytokine &amp; Growth Factor Reviews</em> (2012-06) <a href="https://doi.org/f3z743">https://doi.org/f3z743</a> <br />
DOI: <a href="https://doi.org/10.1016/j.cytogfr.2012.04.001">10.1016/j.cytogfr.2012.04.001</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22595692">22595692</a></p>
</div>
<div id="ref-1EEloBjzn">
<p>577. <strong>Soluble receptors for cytokines and growth factors: generation and biological function</strong> <br />
S Rose-John, PC Heinrich<br />
<em>Biochemical Journal</em> (1994-06-01) <a href="https://doi.org/ggqmgd">https://doi.org/ggqmgd</a> <br />
DOI: <a href="https://doi.org/10.1042/bj3000281">10.1042/bj3000281</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/8002928">8002928</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1138158">PMC1138158</a></p>
</div>
<div id="ref-1H8o8TFtM">
<p>578. <strong>Interleukin-6; pathogenesis and treatment of autoimmune inflammatory diseases</strong> <br />
Toshio Tanaka, Masashi Narazaki, Kazuya Masuda, Tadamitsu Kishimoto<br />
<em>Inflammation and Regeneration</em> (2013) <a href="https://doi.org/ggqmgt">https://doi.org/ggqmgt</a> <br />
DOI: <a href="https://doi.org/10.2492/inflammregen.33.054">10.2492/inflammregen.33.054</a></p>
</div>
<div id="ref-bj2feMy4">
<p>579. <strong>Effective treatment of severe COVID-19 patients with tocilizumab</strong> <br />
Xiaoling Xu, Mingfeng Han, Tiantian Li, Wei Sun, Dongsheng Wang, Binqing Fu, Yonggang Zhou, Xiaohu Zheng, Yun Yang, Xiuyong Li, … Haiming Wei<br />
<em>Proceedings of the National Academy of Sciences</em> (2020-05-19) <a href="https://doi.org/ggv3r3">https://doi.org/ggv3r3</a> <br />
DOI: <a href="https://doi.org/10.1073/pnas.2005615117">10.1073/pnas.2005615117</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32350134">32350134</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7245089">PMC7245089</a></p>
</div>
<div id="ref-ayc7zYqi">
<p>580. <strong>Tocilizumab in patients with severe COVID-19: a retrospective cohort study</strong> <br />
Giovanni Guaraldi, Marianna Meschiari, Alessandro Cozzi-Lepri, Jovana Milic, Roberto Tonelli, Marianna Menozzi, Erica Franceschini, Gianluca Cuomo, Gabriella Orlando, Vanni Borghi, … Cristina Mussini<br />
<em>The Lancet Rheumatology</em> (2020-08) <a href="https://doi.org/d2pk">https://doi.org/d2pk</a> <br />
DOI: <a href="https://doi.org/10.1016/s2665-9913(20)30173-9">10.1016/s2665-9913(20)30173-9</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32835257">32835257</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7314456">PMC7314456</a></p>
</div>
<div id="ref-Wt9vgKLY">
<p>581. <strong>Tocilizumab Treatment for Cytokine Release Syndrome in Hospitalized Patients With Coronavirus Disease 2019</strong> <br />
Christina C. Price, Frederick L. Altice, Yu Shyr, Alan Koff, Lauren Pischel, George Goshua, Marwan M. Azar, Dayna Mcmanus, Sheau-Chiann Chen, Shana E. Gleeson, … Maricar Malinis<br />
<em>Chest</em> (2020-10) <a href="https://doi.org/gg2789">https://doi.org/gg2789</a> <br />
DOI: <a href="https://doi.org/10.1016/j.chest.2020.06.006">10.1016/j.chest.2020.06.006</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32553536">32553536</a></p>
</div>
<div id="ref-luPHOS0m">
<p>582. <strong>Impact of low dose tocilizumab on mortality rate in patients with COVID-19 related pneumonia</strong> <br />
Ruggero Capra, Nicola De Rossi, Flavia Mattioli, Giuseppe Romanelli, Cristina Scarpazza, Maria Pia Sormani, Stefania Cossi<br />
<em>European Journal of Internal Medicine</em> (2020-06) <a href="https://doi.org/ggx4fm">https://doi.org/ggx4fm</a> <br />
DOI: <a href="https://doi.org/10.1016/j.ejim.2020.05.009">10.1016/j.ejim.2020.05.009</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32405160">32405160</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7219361">PMC7219361</a></p>
</div>
<div id="ref-17hgtBpcV">
<p>583. <strong>Tocilizumab therapy reduced intensive care unit admissions and/or mortality in COVID-19 patients</strong> <br />
T. Klopfenstein, S. Zayet, A. Lohse, J.-C. Balblanc, J. Badie, P.-Y. Royer, L. Toko, C. Mezher, N. J. Kadiane-Oussou, M. Bossert, … T. Conrozier<br />
<em>Médecine et Maladies Infectieuses</em> (2020-08) <a href="https://doi.org/ggvz45">https://doi.org/ggvz45</a> <br />
DOI: <a href="https://doi.org/10.1016/j.medmal.2020.05.001">10.1016/j.medmal.2020.05.001</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32387320">32387320</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7202806">PMC7202806</a></p>
</div>
<div id="ref-WEqfULN1">
<p>584. <strong>Outcomes in patients with severe COVID-19 disease treated with tocilizumab: a case–controlled study</strong> <br />
G Rojas-Marte, M Khalid, O Mukhtar, AT Hashmi, MA Waheed, S Ehrlich, A Aslam, S Siddiqui, C Agarwal, Y Malyshev, … J Shani<br />
<em>QJM: An International Journal of Medicine</em> (2020-08) <a href="https://doi.org/gg496t">https://doi.org/gg496t</a> <br />
DOI: <a href="https://doi.org/10.1093/qjmed/hcaa206">10.1093/qjmed/hcaa206</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32569363">32569363</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7337835">PMC7337835</a></p>
</div>
<div id="ref-3yV2bL9S">
<p>585. <strong>Systematic Review and Meta-Analysis of Case-Control Studies from 7,000 COVID-19 Pneumonia Patients Suggests a Beneficial Impact of Tocilizumab with Benefit Most Evident in Non-Corticosteroid Exposed Subjects.</strong> <br />
Abdulla Watad, Nicola Luigi Bragazzi, Charlie Bridgewood, Muhammad Mansour, Naim Mahroum, Matteo Riccò, Ahmed Nasr, Amr Hussein, Omer Gendelman, Yehuda Shoenfeld, … Dennis McGonagle<br />
<em>SSRN Electronic Journal</em> (2020) <a href="https://doi.org/gg62hz">https://doi.org/gg62hz</a> <br />
DOI: <a href="https://doi.org/10.2139/ssrn.3642653">10.2139/ssrn.3642653</a></p>
</div>
<div id="ref-ZHqWXFcN">
<p>586. <strong>The Efficacy of IL-6 Inhibitor Tocilizumab in Reducing Severe COVID-19 Mortality: A Systematic Review</strong> <br />
Avi Gurion Kaye, Robert Siegel<br />
<em>Cold Spring Harbor Laboratory</em> (2020-09-03) <a href="https://doi.org/gg62hv">https://doi.org/gg62hv</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.07.10.20150938">10.1101/2020.07.10.20150938</a></p>
</div>
<div id="ref-h4reluu">
<p>587. <strong>Rationale and evidence on the use of tocilizumab in COVID-19: a systematic review</strong> <br />
A. Cortegiani, M. Ippolito, M. Greco, V. Granone, A. Protti, C. Gregoretti, A. Giarratano, S. Einav, M. Cecconi<br />
<em>Pulmonology</em> (2020-07) <a href="https://doi.org/gg5xv3">https://doi.org/gg5xv3</a> <br />
DOI: <a href="https://doi.org/10.1016/j.pulmoe.2020.07.003">10.1016/j.pulmoe.2020.07.003</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32713784">32713784</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7369580">PMC7369580</a></p>
</div>
<div id="ref-CRriJXDk">
<p>588. <strong>New insights and long-term safety of tocilizumab in rheumatoid arthritis</strong> <br />
Graeme Jones, Elena Panova<br />
<em>Therapeutic Advances in Musculoskeletal Disease</em> (2018-10-07) <a href="https://doi.org/gffsdt">https://doi.org/gffsdt</a> <br />
DOI: <a href="https://doi.org/10.1177/1759720x18798462">10.1177/1759720x18798462</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30327685">30327685</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6178374">PMC6178374</a></p>
</div>
<div id="ref-6xLhs9st">
<p>589. <strong>Tocilizumab during pregnancy and lactation: drug levels in maternal serum, cord blood, breast milk and infant serum</strong> <br />
Jumpei Saito, Naho Yakuwa, Kayoko Kaneko, Chinatsu Takai, Mikako Goto, Ken Nakajima, Akimasa Yamatani, Atsuko Murashima<br />
<em>Rheumatology</em> (2019-08) <a href="https://doi.org/ggzhks">https://doi.org/ggzhks</a> <br />
DOI: <a href="https://doi.org/10.1093/rheumatology/kez100">10.1093/rheumatology/kez100</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30945743">30945743</a></p>
</div>
<div id="ref-IJzmtJOr">
<p>590. <strong>Short-course tocilizumab increases risk of hepatitis B virus reactivation in patients with rheumatoid arthritis: a prospective clinical observation</strong> <br />
Le-Feng Chen, Ying-Qian Mo, Jun Jing, Jian-Da Ma, Dong-Hui Zheng, Lie Dai<br />
<em>International Journal of Rheumatic Diseases</em> (2017-07) <a href="https://doi.org/f9pbc5">https://doi.org/f9pbc5</a> <br />
DOI: <a href="https://doi.org/10.1111/1756-185x.13010">10.1111/1756-185x.13010</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28160426">28160426</a></p>
</div>
<div id="ref-SiOKIbl3">
<p>591. <strong>Why tocilizumab could be an effective treatment for severe COVID-19?</strong> <br />
Binqing Fu, Xiaoling Xu, Haiming Wei<br />
<em>Journal of Translational Medicine</em> (2020-04-14) <a href="https://doi.org/ggv5c8">https://doi.org/ggv5c8</a> <br />
DOI: <a href="https://doi.org/10.1186/s12967-020-02339-3">10.1186/s12967-020-02339-3</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32290839">32290839</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7154566">PMC7154566</a></p>
</div>
<div id="ref-6Agll71m">
<p>592. <strong>Risk of adverse events including serious infections in rheumatoid arthritis patients treated with tocilizumab: a systematic literature review and meta-analysis of randomized controlled trials</strong> <br />
L. Campbell, C. Chen, S. S. Bhagat, R. A. Parker, A. J. K. Ostor<br />
<em>Rheumatology</em> (2010-11-14) <a href="https://doi.org/crqn7c">https://doi.org/crqn7c</a> <br />
DOI: <a href="https://doi.org/10.1093/rheumatology/keq343">10.1093/rheumatology/keq343</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21078627">21078627</a></p>
</div>
<div id="ref-zRRSvckT">
<p>593. <strong>Risk of serious infections in tocilizumab versus other biologic drugs in patients with rheumatoid arthritis: a multidatabase cohort study</strong> <br />
Ajinkya Pawar, Rishi J Desai, Daniel H Solomon, Adrian J Santiago Ortiz, Sara Gale, Min Bao, Khaled Sarsour, Sebastian Schneeweiss, Seoyoung C Kim<br />
<em>Annals of the Rheumatic Diseases</em> (2019-04) <a href="https://doi.org/gg62hx">https://doi.org/gg62hx</a> <br />
DOI: <a href="https://doi.org/10.1136/annrheumdis-2018-214367">10.1136/annrheumdis-2018-214367</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30679153">30679153</a></p>
</div>
<div id="ref-TgV8gb30">
<p>594. <strong>Risk of infections in rheumatoid arthritis patients treated with tocilizumab</strong> <br />
Veronika R. Lang, Matthias Englbrecht, Jürgen Rech, Hubert Nüsslein, Karin Manger, Florian Schuch, Hans-Peter Tony, Martin Fleck, Bernhard Manger, Georg Schett, Jochen Zwerina<br />
<em>Rheumatology</em> (2012-05) <a href="https://doi.org/d3b3rh">https://doi.org/d3b3rh</a> <br />
DOI: <a href="https://doi.org/10.1093/rheumatology/ker223">10.1093/rheumatology/ker223</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21865281">21865281</a></p>
</div>
<div id="ref-p0tzgJvA">
<p>595. <strong>Use of Tocilizumab for COVID-19-Induced Cytokine Release Syndrome</strong> <br />
Jared Radbel, Navaneeth Narayanan, Pinki J. Bhatt<br />
<em>Chest</em> (2020-07) <a href="https://doi.org/ggtxvs">https://doi.org/ggtxvs</a> <br />
DOI: <a href="https://doi.org/10.1016/j.chest.2020.04.024">10.1016/j.chest.2020.04.024</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32343968">32343968</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7195070">PMC7195070</a></p>
</div>
<div id="ref-hvOwSJvK">
<p>596. <strong>Utilizing tocilizumab for the treatment of cytokine release syndrome in COVID-19</strong> <br />
Ali Hassoun, Elizabeth Dilip Thottacherry, Justin Muklewicz, Qurrat-ul-ain Aziz, Jonathan Edwards<br />
<em>Journal of Clinical Virology</em> (2020-07) <a href="https://doi.org/ggx359">https://doi.org/ggx359</a> <br />
DOI: <a href="https://doi.org/10.1016/j.jcv.2020.104443">10.1016/j.jcv.2020.104443</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32425661">32425661</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7229471">PMC7229471</a></p>
</div>
<div id="ref-1AKi0FYUB">
<p>597. <strong>Development of therapeutic antibodies for the treatment of diseases</strong> <br />
Ruei-Min Lu, Yu-Chyi Hwang, I-Ju Liu, Chi-Chiu Lee, Han-Zen Tsai, Hsin-Jung Li, Han-Chung Wu<br />
<em>Journal of Biomedical Science</em> (2020-01-02) <a href="https://doi.org/ggqbpx">https://doi.org/ggqbpx</a> <br />
DOI: <a href="https://doi.org/10.1186/s12929-019-0592-z">10.1186/s12929-019-0592-z</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31894001">31894001</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6939334">PMC6939334</a></p>
</div>
<div id="ref-yumBaJ6U">
<p>598. <strong>Broadly Neutralizing Antiviral Antibodies</strong> <br />
Davide Corti, Antonio Lanzavecchia<br />
<em>Annual Review of Immunology</em> (2013-03-21) <a href="https://doi.org/gf25g8">https://doi.org/gf25g8</a> <br />
DOI: <a href="https://doi.org/10.1146/annurev-immunol-032712-095916">10.1146/annurev-immunol-032712-095916</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23330954">23330954</a></p>
</div>
<div id="ref-dUnB3gD6">
<p>599. <strong>Ibalizumab Targeting CD4 Receptors, An Emerging Molecule in HIV Therapy</strong> <br />
Simona A. Iacob, Diana G. Iacob<br />
<em>Frontiers in Microbiology</em> (2017-11-27) <a href="https://doi.org/gcn3kh">https://doi.org/gcn3kh</a> <br />
DOI: <a href="https://doi.org/10.3389/fmicb.2017.02323">10.3389/fmicb.2017.02323</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29230203">29230203</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5711820">PMC5711820</a></p>
</div>
<div id="ref-cwbHiM1o">
<p>600. <strong>Product review on the monoclonal antibody palivizumab for prevention of respiratory syncytial virus infection</strong> <br />
Bernhard Resch<br />
<em>Human Vaccines &amp; Immunotherapeutics</em> (2017-06-12) <a href="https://doi.org/ggqbps">https://doi.org/ggqbps</a> <br />
DOI: <a href="https://doi.org/10.1080/21645515.2017.1337614">10.1080/21645515.2017.1337614</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28605249">28605249</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5612471">PMC5612471</a></p>
</div>
<div id="ref-3gp7t98X">
<p>601. <strong>Chronological evolution of IgM, IgA, IgG and neutralisation antibodies after infection with SARS-associated coronavirus</strong> <br />
P.-R. Hsueh, L.-M. Huang, P.-J. Chen, C.-L. Kao, P.-C. Yang<br />
<em>Clinical Microbiology and Infection</em> (2004-12) <a href="https://doi.org/cwwg87">https://doi.org/cwwg87</a> <br />
DOI: <a href="https://doi.org/10.1111/j.1469-0691.2004.01009.x">10.1111/j.1469-0691.2004.01009.x</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15606632">15606632</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7129952">PMC7129952</a></p>
</div>
<div id="ref-KULxo48U">
<p>602. <strong>Neutralizing Antibodies in Patients with Severe Acute Respiratory Syndrome-Associated Coronavirus Infection</strong> <br />
Nie Yuchun, Wang Guangwen, Shi Xuanling, Zhang Hong, Qiu Yan, He Zhongping, Wang Wei, Lian Gewei, Yin Xiaolei, Du Liying, … Ding Mingxiao<br />
<em>The Journal of Infectious Diseases</em> (2004-09) <a href="https://doi.org/cgqj5b">https://doi.org/cgqj5b</a> <br />
DOI: <a href="https://doi.org/10.1086/423286">10.1086/423286</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15319862">15319862</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7199490">PMC7199490</a></p>
</div>
<div id="ref-MTcTiS7i">
<p>603. <strong>Potent human monoclonal antibodies against SARS CoV, Nipah and Hendra viruses</strong> <br />
Ponraj Prabakaran, Zhongyu Zhu, Xiaodong Xiao, Arya Biragyn, Antony S Dimitrov, Christopher C Broder, Dimiter S Dimitrov<br />
<em>Expert Opinion on Biological Therapy</em> (2009-04-08) <a href="https://doi.org/b88kw8">https://doi.org/b88kw8</a> <br />
DOI: <a href="https://doi.org/10.1517/14712590902763755">10.1517/14712590902763755</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19216624">19216624</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2705284">PMC2705284</a></p>
</div>
<div id="ref-rXzkS3gA">
<p>604. <strong>Prior Infection and Passive Transfer of Neutralizing Antibody Prevent Replication of Severe Acute Respiratory Syndrome Coronavirus in the Respiratory Tract of Mice</strong> <br />
Kanta Subbarao, Josephine McAuliffe, Leatrice Vogel, Gary Fahle, Steven Fischer, Kathleen Tatti, Michelle Packard, Wun-Ju Shieh, Sherif Zaki, Brian Murphy<br />
<em>Journal of Virology</em> (2004-04-01) <a href="https://doi.org/b8wr7c">https://doi.org/b8wr7c</a> <br />
DOI: <a href="https://doi.org/10.1128/jvi.78.7.3572-3577.2004">10.1128/jvi.78.7.3572-3577.2004</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15016880">15016880</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC371090">PMC371090</a></p>
</div>
<div id="ref-RRx6YovP">
<p>605. <strong>The Effectiveness of Convalescent Plasma and Hyperimmune Immunoglobulin for the Treatment of Severe Acute Respiratory Infections of Viral Etiology: A Systematic Review and Exploratory Meta-analysis</strong> <br />
John Mair-Jenkins, Maria Saavedra-Campos, J. Kenneth Baillie, Paul Cleary, Fu-Meng Khaw, Wei Shen Lim, Sophia Makki, Kevin D. Rooney, Jonathan S. Nguyen-Van-Tam, Charles R. Beck, Convalescent Plasma Study Group<br />
<em>Journal of Infectious Diseases</em> (2015-01-01) <a href="https://doi.org/f632n7">https://doi.org/f632n7</a> <br />
DOI: <a href="https://doi.org/10.1093/infdis/jiu396">10.1093/infdis/jiu396</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25030060">25030060</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4264590">PMC4264590</a></p>
</div>
<div id="ref-Ou3P9pGV">
<p>606. <strong>Identification of human neutralizing antibodies against MERS-CoV and their role in virus adaptive evolution</strong> <br />
X.-C. Tang, S. S. Agnihothram, Y. Jiao, J. Stanhope, R. L. Graham, E. C. Peterson, Y. Avnir, A. S. C. Tallarico, J. Sheehan, Q. Zhu, … W. A. Marasco<br />
<em>Proceedings of the National Academy of Sciences</em> (2014-04-28) <a href="https://doi.org/smr">https://doi.org/smr</a> <br />
DOI: <a href="https://doi.org/10.1073/pnas.1402074111">10.1073/pnas.1402074111</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24778221">24778221</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4024880">PMC4024880</a></p>
</div>
<div id="ref-b4TFEgIi">
<p>607. <strong>Exceptionally Potent Neutralization of Middle East Respiratory Syndrome Coronavirus by Human Monoclonal Antibodies</strong> <br />
Tianlei Ying, Lanying Du, Tina W. Ju, Ponraj Prabakaran, Candy C. Y. Lau, Lu Lu, Qi Liu, Lili Wang, Yang Feng, Yanping Wang, … Dimiter S. Dimitrov<br />
<em>Journal of Virology</em> (2014-07-15) <a href="https://doi.org/ggzf5p">https://doi.org/ggzf5p</a> <br />
DOI: <a href="https://doi.org/10.1128/jvi.00912-14">10.1128/jvi.00912-14</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24789777">24789777</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4097770">PMC4097770</a></p>
</div>
<div id="ref-1GLr0EJU">
<p>608. <strong>The Role of ACE2 in Cardiovascular Physiology</strong> <br />
Gavin Y. Oudit, Michael A. Crackower, Peter H. Backx, Josef M. Penninger<br />
<em>Trends in Cardiovascular Medicine</em> (2003-04) <a href="https://doi.org/bsbp49">https://doi.org/bsbp49</a> <br />
DOI: <a href="https://doi.org/10.1016/s1050-1738(02)00233-5">10.1016/s1050-1738(02)00233-5</a></p>
</div>
<div id="ref-XcuzhxrJ">
<p>609. <strong>A human monoclonal antibody blocking SARS-CoV-2 infection</strong> <br />
Chunyan Wang, Wentao Li, Dubravka Drabek, Nisreen M. A. Okba, Rien van Haperen, Albert D. M. E. Osterhaus, Frank J. M. van Kuppeveld, Bart L. Haagmans, Frank Grosveld, Berend-Jan Bosch<br />
<em>bioRxiv</em> (2020-03-12) <a href="https://doi.org/ggnw4t">https://doi.org/ggnw4t</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.11.987958">10.1101/2020.03.11.987958</a></p>
</div>
<div id="ref-Ftbm1M9p">
<p>610. <strong>Unexpected Receptor Functional Mimicry Elucidates Activation of Coronavirus Fusion</strong> <br />
Alexandra C. Walls, Xiaoli Xiong, Young-Jun Park, M. Alejandra Tortorici, Joost Snijder, Joel Quispe, Elisabetta Cameroni, Robin Gopal, Mian Dai, Antonio Lanzavecchia, … David Veesler<br />
<em>Cell</em> (2019-02) <a href="https://doi.org/gft3jg">https://doi.org/gft3jg</a> <br />
DOI: <a href="https://doi.org/10.1016/j.cell.2018.12.028">10.1016/j.cell.2018.12.028</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30712865">30712865</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6751136">PMC6751136</a></p>
</div>
<div id="ref-O1whWg6Q">
<p>611. <strong><em>In Vitro</em> Neutralization Is Not Predictive of Prophylactic Efficacy of Broadly Neutralizing Monoclonal Antibodies CR6261 and CR9114 against Lethal H2 Influenza Virus Challenge in Mice</strong> <br />
Troy C. Sutton, Elaine W. Lamirande, Kevin W. Bock, Ian N. Moore, Wouter Koudstaal, Muniza Rehman, Gerrit Jan Weverling, Jaap Goudsmit, Kanta Subbarao<br />
<em>Journal of Virology</em> (2017-12-15) <a href="https://doi.org/ggqbpt">https://doi.org/ggqbpt</a> <br />
DOI: <a href="https://doi.org/10.1128/jvi.01603-17">10.1128/jvi.01603-17</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29046448">29046448</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5709608">PMC5709608</a></p>
</div>
<div id="ref-pUnzB8wV">
<p>612. <strong>Human neutralizing antibodies against MERS coronavirus: implications for future immunotherapy</strong> <br />
Xian-Chun Tang, Wayne A Marasco<br />
<em>Immunotherapy</em> (2015-07) <a href="https://doi.org/ggqbpz">https://doi.org/ggqbpz</a> <br />
DOI: <a href="https://doi.org/10.2217/imt.15.33">10.2217/imt.15.33</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26098703">26098703</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5068219">PMC5068219</a></p>
</div>
<div id="ref-14motyCOm">
<p>613. <strong>A Potent and Broad Neutralizing Antibody Recognizes and Penetrates the HIV Glycan Shield</strong> <br />
R. Pejchal, K. J. Doores, L. M. Walker, R. Khayat, P.-S. Huang, S.-K. Wang, R. L. Stanfield, J.-P. Julien, A. Ramos, M. Crispin, … I. A. Wilson<br />
<em>Science</em> (2011-10-13) <a href="https://doi.org/bzqv8c">https://doi.org/bzqv8c</a> <br />
DOI: <a href="https://doi.org/10.1126/science.1213256">10.1126/science.1213256</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21998254">21998254</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3280215">PMC3280215</a></p>
</div>
<div id="ref-oGHFF5Cm">
<p>614. <strong>Broadly Neutralizing Antibodies against HIV: Back to Blood</strong> <br />
Amir Dashti, Anthony L. DeVico, George K. Lewis, Mohammad M. Sajadi<br />
<em>Trends in Molecular Medicine</em> (2019-03) <a href="https://doi.org/ggqbpr">https://doi.org/ggqbpr</a> <br />
DOI: <a href="https://doi.org/10.1016/j.molmed.2019.01.007">10.1016/j.molmed.2019.01.007</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30792120">30792120</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6401214">PMC6401214</a></p>
</div>
<div id="ref-hvjXMm9H">
<p>615. <strong>Importance of Neutralizing Monoclonal Antibodies Targeting Multiple Antigenic Sites on the Middle East Respiratory Syndrome Coronavirus Spike Glycoprotein To Avoid Neutralization Escape</strong> <br />
Lingshu Wang, Wei Shi, James D. Chappell, M. Gordon Joyce, Yi Zhang, Masaru Kanekiyo, Michelle M. Becker, Neeltje van Doremalen, Robert Fischer, Nianshuang Wang, … Barney S. Graham<br />
<em>Journal of Virology</em> (2018-04-27) <a href="https://doi.org/ggqbpv">https://doi.org/ggqbpv</a> <br />
DOI: <a href="https://doi.org/10.1128/jvi.02002-17">10.1128/jvi.02002-17</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29514901">29514901</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5923077">PMC5923077</a></p>
</div>
<div id="ref-tC5vJmwj">
<p>616. <strong>Anti–spike IgG causes severe acute lung injury by skewing macrophage responses during acute SARS-CoV infection</strong> <br />
Li Liu, Qiang Wei, Qingqing Lin, Jun Fang, Haibo Wang, Hauyee Kwok, Hangying Tang, Kenji Nishiura, Jie Peng, Zhiwu Tan, … Zhiwei Chen<br />
<em>JCI Insight</em> (2019-02-21) <a href="https://doi.org/ggqbpw">https://doi.org/ggqbpw</a> <br />
DOI: <a href="https://doi.org/10.1172/jci.insight.123158">10.1172/jci.insight.123158</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30830861">30830861</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6478436">PMC6478436</a></p>
</div>
<div id="ref-LtFtUo2P">
<p>617. <strong>The antiviral effect of interferon-beta against SARS-Coronavirus is not mediated by MxA protein</strong> <br />
Martin Spiegel, Andreas Pichlmair, Elke Mühlberger, Otto Haller, Friedemann Weber<br />
<em>Journal of Clinical Virology</em> (2004-07) <a href="https://doi.org/cmc3ds">https://doi.org/cmc3ds</a> <br />
DOI: <a href="https://doi.org/10.1016/j.jcv.2003.11.013">10.1016/j.jcv.2003.11.013</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15135736">15135736</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7128634">PMC7128634</a></p>
</div>
<div id="ref-Nfu9kiae">
<p>618. <strong>Coronavirus virulence genes with main focus on SARS-CoV envelope gene</strong> <br />
Marta L. DeDiego, Jose L. Nieto-Torres, Jose M. Jimenez-Guardeño, Jose A. Regla-Nava, Carlos Castaño-Rodriguez, Raul Fernandez-Delgado, Fernando Usera, Luis Enjuanes<br />
<em>Virus Research</em> (2014-12) <a href="https://doi.org/f6wm24">https://doi.org/f6wm24</a> <br />
DOI: <a href="https://doi.org/10.1016/j.virusres.2014.07.024">10.1016/j.virusres.2014.07.024</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25093995">25093995</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4261026">PMC4261026</a></p>
</div>
<div id="ref-s9C2iFSR">
<p>619. <strong>people</strong> <br />
Richard marsden<br />
(2020-03) <a href="https://www.synairgen.com/wp-content/uploads/2020/03/200318-Synairgen-to-start-trial-of-SNG001-in-COVID-19-imminently-.pdf">https://www.synairgen.com/wp-content/uploads/2020/03/200318-Synairgen-to-start-trial-of-SNG001-in-COVID-19-imminently-.pdf</a></p>
</div>
<div id="ref-4Tx92JMv">
<p>620. <strong>people</strong> <br />
Richard marsden<br />
(2020-07) <a href="https://www.synairgen.com/wp-content/uploads/2020/07/200720-Synairgen-announces-positive-results-from-trial-of-SNG001-in-hospitalised-COVID-19-patients.pdf">https://www.synairgen.com/wp-content/uploads/2020/07/200720-Synairgen-announces-positive-results-from-trial-of-SNG001-in-hospitalised-COVID-19-patients.pdf</a></p>
</div>
<div id="ref-HyYY2agc">
<p>621. <strong>Developing Covid-19 Vaccines at Pandemic Speed</strong> <br />
Nicole Lurie, Melanie Saville, Richard Hatchett, Jane Halton<br />
<em>New England Journal of Medicine</em> (2020-03-30) <a href="https://doi.org/ggq8bc">https://doi.org/ggq8bc</a> <br />
DOI: <a href="https://doi.org/10.1056/nejmp2005630">10.1056/nejmp2005630</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32227757">32227757</a></p>
</div>
<div id="ref-SvOLLYfw">
<p>622. <strong>WHO | Novel Coronavirus – China</strong> <br />
WHO<br />
<a href="http://www.who.int/csr/don/12-january-2020-novel-coronavirus-china/en/">http://www.who.int/csr/don/12-january-2020-novel-coronavirus-china/en/</a></p>
</div>
<div id="ref-dqpEe5Lz">
<p>623. <strong>SARS-CoV-2 vaccines in development</strong> <br />
Florian Krammer<br />
<em>Nature</em> (2020-09-23) <a href="https://doi.org/ghdprn">https://doi.org/ghdprn</a> <br />
DOI: <a href="https://doi.org/10.1038/s41586-020-2798-3">10.1038/s41586-020-2798-3</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32967006">32967006</a></p>
</div>
<div id="ref-1GA95MF2m">
<p>624. <strong>The COVID-19 vaccine development landscape</strong> <br />
Tung Thanh Le, Zacharias Andreadakis, Arun Kumar, Raúl Gómez Román, Stig Tollefsen, Melanie Saville, Stephen Mayhew<br />
<em>Nature Reviews Drug Discovery</em> (2020-04-09) <a href="https://doi.org/ggrnbr">https://doi.org/ggrnbr</a> <br />
DOI: <a href="https://doi.org/10.1038/d41573-020-00073-5">10.1038/d41573-020-00073-5</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32273591">32273591</a></p>
</div>
<div id="ref-wPl93ATP">
<p>625. <strong>Newer Vaccine Technologies Deployed to Develop COVID-19 Shot</strong> <br />
Abby Olena<br />
<em>The Scientist Magazine</em> (2020-02-21) <a href="https://www.the-scientist.com/news-opinion/newer-vaccine-technologies-deployed-to-develop-covid-19-shot-67152">https://www.the-scientist.com/news-opinion/newer-vaccine-technologies-deployed-to-develop-covid-19-shot-67152</a></p>
</div>
<div id="ref-plfPrQP7">
<p>626. <strong>A strategic approach to COVID-19 vaccine R&amp;D</strong> <br />
Lawrence Corey, John R. Mascola, Anthony S. Fauci, Francis S. Collins<br />
<em>Science</em> (2020-05-29) <a href="https://doi.org/ggwfck">https://doi.org/ggwfck</a> <br />
DOI: <a href="https://doi.org/10.1126/science.abc5312">10.1126/science.abc5312</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32393526">32393526</a></p>
</div>
<div id="ref-wiGjCZC8">
<p>627. <strong>An mRNA Vaccine against SARS-CoV-2 — Preliminary Report</strong> <br />
Lisa A. Jackson, Evan J. Anderson, Nadine G. Rouphael, Paul C. Roberts, Mamodikoe Makhene, Rhea N. Coler, Michele P. McCullough, James D. Chappell, Mark R. Denison, Laura J. Stevens, … John H. Beigel<br />
<em>New England Journal of Medicine</em> (2020-07-14) <a href="https://doi.org/d3tt">https://doi.org/d3tt</a> <br />
DOI: <a href="https://doi.org/10.1056/nejmoa2022483">10.1056/nejmoa2022483</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32663912">32663912</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7377258">PMC7377258</a></p>
</div>
<div id="ref-XnrBoKVk">
<p>628. <strong>DNA</strong> <a href="https://www.who.int/teams/health-product-and-policy-standards/standards-and-specifications/vaccines-quality/dna">https://www.who.int/teams/health-product-and-policy-standards/standards-and-specifications/vaccines-quality/dna</a></p>
</div>
<div id="ref-xuzLfS0y">
<p>629. <strong>Safety, Tolerability and Immunogenicity of INO-4800 for COVID-19 in Healthy Volunteers - Full Text View - ClinicalTrials.gov</strong> <a href="https://clinicaltrials.gov/ct2/show/NCT04336410">https://clinicaltrials.gov/ct2/show/NCT04336410</a></p>
</div>
<div id="ref-H6tWVs5R">
<p>630. <strong>Electroporation delivery of DNA vaccines: prospects for success</strong> <br />
Niranjan Y Sardesai, David B Weiner<br />
<em>Current Opinion in Immunology</em> (2011-06) <a href="https://doi.org/cq8b4p">https://doi.org/cq8b4p</a> <br />
DOI: <a href="https://doi.org/10.1016/j.coi.2011.03.008">10.1016/j.coi.2011.03.008</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21530212">21530212</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3109217">PMC3109217</a></p>
</div>
<div id="ref-1Hsm2J1sc">
<p>631. <strong>Tolerability of intramuscular and intradermal delivery by CELLECTRA <sup>®</sup> adaptive constant current electroporation device in healthy volunteers</strong> <br />
Malissa C Diehl, Jessica C Lee, Stephen E Daniels, Pablo Tebas, Amir S Khan, Mary Giffear, Niranjan Y Sardesai, Mark L Bagarazzi<br />
<em>Human Vaccines &amp; Immunotherapeutics</em> (2014-10-27) <a href="https://doi.org/ggrj7h">https://doi.org/ggrj7h</a> <br />
DOI: <a href="https://doi.org/10.4161/hv.24702">10.4161/hv.24702</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24051434">24051434</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3906411">PMC3906411</a></p>
</div>
<div id="ref-HCImhzy8">
<p>632. <strong>Induction of virus-specific cytotoxic T lymphocytesin vivo by liposome-entrapped mRNA</strong> <br />
Frédéric Martinon, Sivadasan Krishnan, Gerlinde Lenzen, Rémy Magné, Elisabeth Gomard, Jean-Gérard Guillet, Jean-Paul Lévy, Pierre Meulien<br />
<em>European Journal of Immunology</em> (1993-07) <a href="https://doi.org/b6jb3z">https://doi.org/b6jb3z</a> <br />
DOI: <a href="https://doi.org/10.1002/eji.1830230749">10.1002/eji.1830230749</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/8325342">8325342</a></p>
</div>
<div id="ref-K0Ltu31S">
<p>633. <strong>Advances in mRNA Vaccines for Infectious Diseases</strong> <br />
Cuiling Zhang, Giulietta Maruggi, Hu Shan, Junwei Li<br />
<em>Frontiers in Immunology</em> (2019-03-27) <a href="https://doi.org/ggsnm7">https://doi.org/ggsnm7</a> <br />
DOI: <a href="https://doi.org/10.3389/fimmu.2019.00594">10.3389/fimmu.2019.00594</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30972078">30972078</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6446947">PMC6446947</a></p>
</div>
<div id="ref-zNKWlCwE">
<p>634. <strong>mRNA vaccine delivery using lipid nanoparticles</strong> <br />
Andreas M Reichmuth, Matthias A Oberli, Ana Jaklenec, Robert Langer, Daniel Blankschtein<br />
<em>Therapeutic Delivery</em> (2016-05) <a href="https://doi.org/f8xfzc">https://doi.org/f8xfzc</a> <br />
DOI: <a href="https://doi.org/10.4155/tde-2016-0006">10.4155/tde-2016-0006</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27075952">27075952</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5439223">PMC5439223</a></p>
</div>
<div id="ref-pRoqjur8">
<p>635. <strong>Mechanism of action of mRNA-based vaccines</strong> <br />
Carlo Iavarone, Derek T. O’hagan, Dong Yu, Nicolas F. Delahaye, Jeffrey B. Ulmer<br />
<em>Expert Review of Vaccines</em> (2017-07-28) <a href="https://doi.org/ggsnm6">https://doi.org/ggsnm6</a> <br />
DOI: <a href="https://doi.org/10.1080/14760584.2017.1355245">10.1080/14760584.2017.1355245</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28701102">28701102</a></p>
</div>
<div id="ref-wYZ6qJMu">
<p>636. <strong>SARS-CoV-2 Vaccines: Status Report</strong> <br />
Fatima Amanat, Florian Krammer<br />
<em>Immunity</em> (2020-04) <a href="https://doi.org/ggrdj4">https://doi.org/ggrdj4</a> <br />
DOI: <a href="https://doi.org/10.1016/j.immuni.2020.03.007">10.1016/j.immuni.2020.03.007</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32259480">32259480</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7136867">PMC7136867</a></p>
</div>
<div id="ref-3EUiWZdN">
<p>637. <strong>Study in Healthy Adults to Evaluate Gene Activation After Vaccination With GlaxoSmithKline (GSK) Biologicals’ Candidate Tuberculosis (TB) Vaccine GSK 692342 - Full Text View - ClinicalTrials.gov</strong> <a href="https://clinicaltrials.gov/ct2/show/NCT01669096">https://clinicaltrials.gov/ct2/show/NCT01669096</a></p>
</div>
<div id="ref-6wZy2mn8">
<p>638. <strong>Nucleoside-modified mRNA immunization elicits influenza virus hemagglutinin stalk-specific antibodies</strong> <br />
Norbert Pardi, Kaela Parkhouse, Ericka Kirkpatrick, Meagan McMahon, Seth J. Zost, Barbara L. Mui, Ying K. Tam, Katalin Karikó, Christopher J. Barbosa, Thomas D. Madden, … Drew Weissman<br />
<em>Nature Communications</em> (2018-08-22) <a href="https://doi.org/gd49qt">https://doi.org/gd49qt</a> <br />
DOI: <a href="https://doi.org/10.1038/s41467-018-05482-0">10.1038/s41467-018-05482-0</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30135514">30135514</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6105651">PMC6105651</a></p>
</div>
<div id="ref-1EM5nGaYd">
<p>639. <strong>RNA vaccines: an introduction</strong> <br />
PHG Foundation<br />
<a href="https://www.phgfoundation.org/briefing/rna-vaccines">https://www.phgfoundation.org/briefing/rna-vaccines</a></p>
</div>
<div id="ref-3LMMW7F0">
<p>640. <strong>T Follicular Helper Cell Differentiation, Function, and Roles in Disease</strong> <br />
Shane Crotty<br />
<em>Immunity</em> (2014-10) <a href="https://doi.org/ggsp64">https://doi.org/ggsp64</a> <br />
DOI: <a href="https://doi.org/10.1016/j.immuni.2014.10.004">10.1016/j.immuni.2014.10.004</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25367570">25367570</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4223692">PMC4223692</a></p>
</div>
<div id="ref-ENBWnhAh">
<p>641. <strong>mRNA vaccines — a new era in vaccinology</strong> <br />
Norbert Pardi, Michael J. Hogan, Frederick W. Porter, Drew Weissman<br />
<em>Nature Reviews Drug Discovery</em> (2018-01-12) <a href="https://doi.org/gcsmgr">https://doi.org/gcsmgr</a> <br />
DOI: <a href="https://doi.org/10.1038/nrd.2017.243">10.1038/nrd.2017.243</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29326426">29326426</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5906799">PMC5906799</a></p>
</div>
<div id="ref-Biu1CQeQ">
<p>642. <strong>Phase I, Open-Label, Dose-Ranging Study of the Safety and Immunogenicity of 2019-nCoV Vaccine (mRNA-1273) in Healthy Adults</strong> <br />
National Institute of Allergy and Infectious Diseases (NIAID)<br />
<em>clinicaltrials.gov</em> (2020-07-16) <a href="https://clinicaltrials.gov/ct2/show/NCT04283461">https://clinicaltrials.gov/ct2/show/NCT04283461</a></p>
</div>
<div id="ref-m4J5hWsz">
<p>643. <strong>A Study to Evaluate Efficacy, Safety, and Immunogenicity of mRNA-1273 Vaccine in Adults Aged 18 Years and Older to Prevent COVID-19 - Full Text View - ClinicalTrials.gov</strong> <a href="https://clinicaltrials.gov/ct2/show/NCT04470427">https://clinicaltrials.gov/ct2/show/NCT04470427</a></p>
</div>
<div id="ref-ouOXy0wH">
<p>644. <strong>Progress and Prospects on Vaccine Development against SARS-CoV-2</strong> <br />
Jinyong Zhang, Hao Zeng, Jiang Gu, Haibo Li, Lixin Zheng, Quanming Zou<br />
<em>Vaccines</em> (2020-03-29) <a href="https://doi.org/ggq726">https://doi.org/ggq726</a> <br />
DOI: <a href="https://doi.org/10.3390/vaccines8020153">10.3390/vaccines8020153</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32235387">32235387</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7349596">PMC7349596</a></p>
</div>
<div id="ref-13bVbfc5h">
<p>645. <strong>Towards an understanding of the adjuvant action of aluminium</strong> <br />
Philippa Marrack, Amy S. McKee, Michael W. Munks<br />
<em>Nature Reviews Immunology</em> (2009-04) <a href="https://doi.org/drcwvf">https://doi.org/drcwvf</a> <br />
DOI: <a href="https://doi.org/10.1038/nri2510">10.1038/nri2510</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19247370">19247370</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3147301">PMC3147301</a></p>
</div>
<div id="ref-122h6fIxE">
<p>646. <strong>DAMP-Inducing Adjuvant and PAMP Adjuvants Parallelly Enhance Protective Type-2 and Type-1 Immune Responses to Influenza Split Vaccination</strong> <br />
Tomoya Hayashi, Masatoshi Momota, Etsushi Kuroda, Takato Kusakabe, Shingo Kobari, Kotaro Makisaka, Yoshitaka Ohno, Yusuke Suzuki, Fumika Nakagawa, Michelle S. J. Lee, … Hidetoshi Arima<br />
<em>Frontiers in Immunology</em> (2018-11-20) <a href="https://doi.org/gfqq89">https://doi.org/gfqq89</a> <br />
DOI: <a href="https://doi.org/10.3389/fimmu.2018.02619">10.3389/fimmu.2018.02619</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30515151">30515151</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6255964">PMC6255964</a></p>
</div>
<div id="ref-uO0uqhxc">
<p>647. <strong>Better Adjuvants for Better Vaccines: Progress in Adjuvant Delivery Systems, Modifications, and Adjuvant–Antigen Codelivery</strong> <br />
Zhi-Biao Wang, Jing Xu<br />
<em>Vaccines</em> (2020-03-13) <a href="https://doi.org/gg35vj">https://doi.org/gg35vj</a> <br />
DOI: <a href="https://doi.org/10.3390/vaccines8010128">10.3390/vaccines8010128</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32183209">32183209</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7157724">PMC7157724</a></p>
</div>
<div id="ref-103fS7Kz2">
<p>648. <strong>Defining trained immunity and its role in health and disease</strong> <br />
Mihai G. Netea, Jorge Domínguez-Andrés, Luis B. Barreiro, Triantafyllos Chavakis, Maziar Divangahi, Elaine Fuchs, Leo A. B. Joosten, Jos W. M. van der Meer, Musa M. Mhlanga, Willem J. M. Mulder, … Eicke Latz<br />
<em>Nature Reviews Immunology</em> (2020-03-04) <a href="https://doi.org/gg28pr">https://doi.org/gg28pr</a> <br />
DOI: <a href="https://doi.org/10.1038/s41577-020-0285-6">10.1038/s41577-020-0285-6</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32132681">32132681</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7186935">PMC7186935</a></p>
</div>
<div id="ref-Vu1VILWK">
<p>649. <strong>Trained Immunity: a Tool for Reducing Susceptibility to and the Severity of SARS-CoV-2 Infection</strong> <br />
Mihai G. Netea, Evangelos J. Giamarellos-Bourboulis, Jorge Domínguez-Andrés, Nigel Curtis, Reinout van Crevel, Frank L. van de Veerdonk, Marc Bonten<br />
<em>Cell</em> (2020-05) <a href="https://doi.org/gg2584">https://doi.org/gg2584</a> <br />
DOI: <a href="https://doi.org/10.1016/j.cell.2020.04.042">10.1016/j.cell.2020.04.042</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32437659">32437659</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7196902">PMC7196902</a></p>
</div>
<div id="ref-9m3rP633">
<p>650. <strong>BCG Vaccination to Protect Healthcare Workers Against COVID-19 - Full Text View - ClinicalTrials.gov</strong> <a href="https://clinicaltrials.gov/ct2/show/NCT04327206">https://clinicaltrials.gov/ct2/show/NCT04327206</a></p>
</div>
<div id="ref-y9IYdfM3">
<p>651. <strong>Reducing Health Care Workers Absenteeism in Covid-19 Pandemic Through BCG Vaccine - Full Text View - ClinicalTrials.gov</strong> <a href="https://clinicaltrials.gov/ct2/show/NCT04328441">https://clinicaltrials.gov/ct2/show/NCT04328441</a></p>
</div>
<div id="ref-xdqxBruc">
<p>652. <strong>BCG Vaccine for Health Care Workers as Defense Against COVID 19 - Full Text View - ClinicalTrials.gov</strong> <a href="https://clinicaltrials.gov/ct2/show/NCT04348370">https://clinicaltrials.gov/ct2/show/NCT04348370</a></p>
</div>
<div id="ref-962rELVS">
<p>653. <strong>Application of BCG Vaccine for Immune-prophylaxis Among Egyptian Healthcare Workers During the Pandemic of COVID-19</strong> <br />
Adel Khattab<br />
<em>clinicaltrials.gov</em> (2020-04-17) <a href="https://clinicaltrials.gov/ct2/show/NCT04350931">https://clinicaltrials.gov/ct2/show/NCT04350931</a></p>
</div>
<div id="ref-EuwTWcPi">
<p>654. <strong>Performance Evaluation of BCG Vaccination in Healthcare Personnel to Reduce the Severity of SARS-COV-2 Infection in Medellín, Colombia, 2020</strong> <br />
Universidad de Antioquia<br />
<em>clinicaltrials.gov</em> (2020-04-24) <a href="https://clinicaltrials.gov/ct2/show/NCT04362124">https://clinicaltrials.gov/ct2/show/NCT04362124</a></p>
</div>
<div id="ref-dQtUeruv">
<p>655. <strong>COVID-19: BCG As Therapeutic Vaccine, Transmission Limitation, and Immunoglobulin Enhancement - Full Text View - ClinicalTrials.gov</strong> <a href="https://clinicaltrials.gov/ct2/show/NCT04369794">https://clinicaltrials.gov/ct2/show/NCT04369794</a></p>
</div>
<div id="ref-DjXsPR8O">
<p>656. <strong>Using BCG Vaccine to Protect Health Care Workers in the COVID-19 Pandemic - Full Text View - ClinicalTrials.gov</strong> <a href="https://clinicaltrials.gov/ct2/show/NCT04373291">https://clinicaltrials.gov/ct2/show/NCT04373291</a></p>
</div>
<div id="ref-10OE6y3Pv">
<p>657. <strong>Reducing Morbidity and Mortality in Health Care Workers Exposed to SARS-CoV-2 by Enhancing Non-specific Immune Responses Through Bacillus Calmette-Guérin Vaccination, a Randomized Controlled Trial</strong> <br />
TASK Applied Science<br />
<em>clinicaltrials.gov</em> (2020-05-06) <a href="https://clinicaltrials.gov/ct2/show/NCT04379336">https://clinicaltrials.gov/ct2/show/NCT04379336</a></p>
</div>
<div id="ref-86OjIybR">
<p>658. <strong>Randomized Controlled Trial Evaluating the Efficacy of Vaccination With Bacillus Calmette and Guérin (BCG) in the Prevention of COVID-19 Via the Strengthening of Innate Immunity in Health Care Workers</strong> <br />
Assistance Publique - Hôpitaux de Paris<br />
<em>clinicaltrials.gov</em> (2020-08-17) <a href="https://clinicaltrials.gov/ct2/show/NCT04384549">https://clinicaltrials.gov/ct2/show/NCT04384549</a></p>
</div>
<div id="ref-ITO15LIz">
<p>659. <strong>Study to Assess VPM1002 in Reducing Healthcare Professionals’ Absenteeism in COVID-19 Pandemic - Full Text View - ClinicalTrials.gov</strong> <a href="https://clinicaltrials.gov/ct2/show/NCT04387409">https://clinicaltrials.gov/ct2/show/NCT04387409</a></p>
</div>
<div id="ref-VkZGZxLn">
<p>660. <strong>A Randomized Clinical Trial for Enhanced Trained Immune Responses Through Bacillus Calmette-Guérin Vaccination to Prevent Infections by COVID-19: The ACTIVATE II Trial</strong> <br />
Hellenic Institute for the Study of Sepsis<br />
<em>clinicaltrials.gov</em> (2020-07-10) <a href="https://clinicaltrials.gov/ct2/show/NCT04414267">https://clinicaltrials.gov/ct2/show/NCT04414267</a></p>
</div>
<div id="ref-13JVjMfQI">
<p>661. <strong>Reducing Hospital Admission of Elderly in SARS-CoV-2 Pandemic Via the Induction of Trained Immunity by Bacillus Calmette-Guérin Vaccination, a Randomized Controlled Trial</strong> <br />
Radboud University<br />
<em>clinicaltrials.gov</em> (2020-06-03) <a href="https://clinicaltrials.gov/ct2/show/NCT04417335">https://clinicaltrials.gov/ct2/show/NCT04417335</a></p>
</div>
<div id="ref-nk2MVsld">
<p>662. <strong>Study to Assess VPM1002 in Reducing Hospital Admissions and/or Severe Respiratory Infectious Diseases in Elderly in COVID-19 Pandemic - Full Text View - ClinicalTrials.gov</strong> <a href="https://clinicaltrials.gov/ct2/show/NCT04435379">https://clinicaltrials.gov/ct2/show/NCT04435379</a></p>
</div>
<div id="ref-1E2t9tr8h">
<p>663. <strong>Efficacy and Safety of VPM1002 in Reducing SARS-CoV-2 (COVID-19) Infection Rate and Severity - Full Text View - ClinicalTrials.gov</strong> <a href="https://clinicaltrials.gov/ct2/show/NCT04439045">https://clinicaltrials.gov/ct2/show/NCT04439045</a></p>
</div>
<div id="ref-i5k18bpX">
<p>664. <strong>Complex Immune Dysregulation in COVID-19 Patients with Severe Respiratory Failure</strong> <br />
Evangelos J. Giamarellos-Bourboulis, Mihai G. Netea, Nikoletta Rovina, Karolina Akinosoglou, Anastasia Antoniadou, Nikolaos Antonakos, Georgia Damoraki, Theologia Gkavogianni, Maria-Evangelia Adami, Paraskevi Katsaounou, … Antonia Koutsoukou<br />
<em>Cell Host &amp; Microbe</em> (2020-06) <a href="https://doi.org/ggthxs">https://doi.org/ggthxs</a> <br />
DOI: <a href="https://doi.org/10.1016/j.chom.2020.04.009">10.1016/j.chom.2020.04.009</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32320677">32320677</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7172841">PMC7172841</a></p>
</div>
<div id="ref-1228YjPRv">
<p>665. <strong>Type I and Type III Interferons – Induction, Signaling, Evasion, and Application to Combat COVID-19</strong> <br />
Annsea Park, Akiko Iwasaki<br />
<em>Cell Host &amp; Microbe</em> (2020-06) <a href="https://doi.org/gg2ccp">https://doi.org/gg2ccp</a> <br />
DOI: <a href="https://doi.org/10.1016/j.chom.2020.05.008">10.1016/j.chom.2020.05.008</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32464097">32464097</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7255347">PMC7255347</a></p>
</div>
<div id="ref-CC4sPeuO">
<p>666. <strong>Interpreting Diagnostic Tests for SARS-CoV-2</strong> <br />
Nandini Sethuraman, Sundararaj Stanleyraj Jeremiah, Akihide Ryo<br />
<em>JAMA</em> (2020-06-09) <a href="https://doi.org/ggt6cw">https://doi.org/ggt6cw</a> <br />
DOI: <a href="https://doi.org/10.1001/jama.2020.8259">10.1001/jama.2020.8259</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32374370">32374370</a></p>
</div>
<div id="ref-80JdM9px">
<p>667. <strong>Genomic evidence for reinfection with SARS-CoV-2: a case study</strong> <br />
Richard L Tillett, Joel R Sevinsky, Paul D Hartley, Heather Kerwin, Natalie Crawford, Andrew Gorzalski, Chris Laverdure, Subhash C Verma, Cyprian C Rossetto, David Jackson, … Mark Pandori<br />
<em>The Lancet Infectious Diseases</em> (2020-10) <a href="https://doi.org/ghfgkt">https://doi.org/ghfgkt</a> <br />
DOI: <a href="https://doi.org/10.1016/s1473-3099(20)30764-7">10.1016/s1473-3099(20)30764-7</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33058797">33058797</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7550103">PMC7550103</a></p>
</div>
<div id="ref-7fZ9eyMk">
<p>668. <strong>Table 1, Cost-Comparison Table for Biologic Disease-Modifying Drugs for Rheumatoid Arthritis</strong> <br />
National Center for Biotechnology Information, U. S. National Library of Medicine 8600 Rockville Pike, Bethesda MD, 20894 Usa<br />
(2015-08) <a href="https://www.ncbi.nlm.nih.gov/books/NBK349513/table/T43/">https://www.ncbi.nlm.nih.gov/books/NBK349513/table/T43/</a></p>
</div>
<div id="ref-trWLoLFz">
<p>669. <strong>A Cost Comparison of Treatments of Moderate to Severe Psoriasis</strong> <br />
Cheryl Hankin, Steven Feldman, Andy Szczotka, Randolph Stinger, Leslie Fish, David Hankin<br />
<em>Drug Benefit Trends</em> (2005-05) <a href="https://escholarship.umassmed.edu/meyers_pp/385">https://escholarship.umassmed.edu/meyers_pp/385</a></p>
</div>
<div id="ref-LDQlolY3">
<p>670. <strong>TNF-α inhibition for potential therapeutic modulation of SARS coronavirus infection</strong> <br />
Edward Tobinick<br />
<em>Current Medical Research and Opinion</em> (2008-09-22) <a href="https://doi.org/bq4cx2">https://doi.org/bq4cx2</a> <br />
DOI: <a href="https://doi.org/10.1185/030079903125002757">10.1185/030079903125002757</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/14741070">14741070</a></p>
</div>
<div id="ref-4DAIEbQF">
<p>671. <strong>Sanofi and Regeneron begin global Kevzara® (sarilumab) clinical trial program in patients with severe COVID-19</strong> <br />
Sanofi<br />
(2020-03-16) <a href="http://www.news.sanofi.us/2020-03-16-Sanofi-and-Regeneron-begin-global-Kevzara-R-sarilumab-clinical-trial-program-in-patients-with-severe-COVID-19">http://www.news.sanofi.us/2020-03-16-Sanofi-and-Regeneron-begin-global-Kevzara-R-sarilumab-clinical-trial-program-in-patients-with-severe-COVID-19</a></p>
</div>
<div id="ref-amy3rDsj">
<p>672. <strong>Sarilumab COVID-19 - Full Text View - ClinicalTrials.gov</strong> <a href="https://clinicaltrials.gov/ct2/show/NCT04327388">https://clinicaltrials.gov/ct2/show/NCT04327388</a></p>
</div>
<div id="ref-WF7ymA4m">
<p>673. <strong>COVID-19: combining antiviral and anti-inflammatory treatments</strong> <br />
Justin Stebbing, Anne Phelan, Ivan Griffin, Catherine Tucker, Olly Oechsle, Dan Smith, Peter Richardson<br />
<em>The Lancet Infectious Diseases</em> (2020-04) <a href="https://doi.org/dph5">https://doi.org/dph5</a> <br />
DOI: <a href="https://doi.org/10.1016/s1473-3099(20)30132-8">10.1016/s1473-3099(20)30132-8</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32113509">32113509</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7158903">PMC7158903</a></p>
</div>
<div id="ref-uCns3aFw">
<p>674. <strong>Baricitinib as potential treatment for 2019-nCoV acute respiratory disease</strong> <br />
Peter Richardson, Ivan Griffin, Catherine Tucker, Dan Smith, Olly Oechsle, Anne Phelan, Justin Stebbing<br />
<em>The Lancet</em> (2020-02) <a href="https://doi.org/ggnrsx">https://doi.org/ggnrsx</a> <br />
DOI: <a href="https://doi.org/10.1016/s0140-6736(20)30304-4">10.1016/s0140-6736(20)30304-4</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32032529">32032529</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7137985">PMC7137985</a></p>
</div>
<div id="ref-jKRGxazA">
<p>675. <strong>Lilly Begins Clinical Testing of Therapies for COVID-19 | Eli Lilly and Company</strong> <a href="https://investor.lilly.com/news-releases/news-release-details/lilly-begins-clinical-testing-therapies-covid-19">https://investor.lilly.com/news-releases/news-release-details/lilly-begins-clinical-testing-therapies-covid-19</a></p>
</div>
<div id="ref-JCMhG5r9">
<p>676. <strong>Baricitinib Combined With Antiviral Therapy in Symptomatic Patients Infected by COVID-19: an Open-label, Pilot Study</strong> <br />
Fabrizio Cantini<br />
<em>clinicaltrials.gov</em> (2020-04-19) <a href="https://clinicaltrials.gov/ct2/show/NCT04320277">https://clinicaltrials.gov/ct2/show/NCT04320277</a></p>
</div>
<div id="ref-Cd5uMaAr">
<p>677. <strong>Design and Synthesis of Hydroxyferroquine Derivatives with Antimalarial and Antiviral Activities</strong> <br />
Christophe Biot, Wassim Daher, Natascha Chavain, Thierry Fandeur, Jamal Khalife, Daniel Dive, Erik De Clercq<br />
<em>Journal of Medicinal Chemistry</em> (2006-05) <a href="https://doi.org/db4n83">https://doi.org/db4n83</a> <br />
DOI: <a href="https://doi.org/10.1021/jm0601856">10.1021/jm0601856</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16640347">16640347</a></p>
</div>
<div id="ref-TZleuwHX">
<p>678. <strong>An orally bioavailable broad-spectrum antiviral inhibits SARS-CoV-2 in human airway epithelial cell cultures and multiple coronaviruses in mice</strong> <br />
Timothy P. Sheahan, Amy C. Sims, Shuntai Zhou, Rachel L. Graham, Andrea J. Pruijssers, Maria L. Agostini, Sarah R. Leist, Alexandra Schäfer, Kenneth H. Dinnon, Laura J. Stevens, … Ralph S. Baric<br />
<em>Science Translational Medicine</em> (2020-04-29) <a href="https://doi.org/ggrqd2">https://doi.org/ggrqd2</a> <br />
DOI: <a href="https://doi.org/10.1126/scitranslmed.abb5883">10.1126/scitranslmed.abb5883</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32253226">32253226</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7164393">PMC7164393</a></p>
</div>
<div id="ref-LpqYmd4C">
<p>679. <strong>Perspective: Vitamin D deficiency and COVID‐19 severity – plausibly linked by latitude, ethnicity, impacts on cytokines, ACE2 and thrombosis</strong> <br />
J. M. Rhodes, S. Subramanian, E. Laird, G. Griffin, R. A. Kenny<br />
<em>Journal of Internal Medicine</em> (2020-07-22) <a href="https://doi.org/ghc7dh">https://doi.org/ghc7dh</a> <br />
DOI: <a href="https://doi.org/10.1111/joim.13149">10.1111/joim.13149</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32613681">32613681</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7361294">PMC7361294</a></p>
</div>
<div id="ref-Hd5V5Usd">
<p>680. <strong>Coronavirus Disease 2019 (COVID-19) and Nutritional Status: The Missing Link?</strong> <br />
Renata Silverio, Daniela Caetano Gonçalves, Márcia Fábia Andrade, Marilia Seelaender<br />
<em>Advances in Nutrition</em> (2020-09-25) <a href="https://doi.org/ghhqjd">https://doi.org/ghhqjd</a> <br />
DOI: <a href="https://doi.org/10.1093/advances/nmaa125">10.1093/advances/nmaa125</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32975565">32975565</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7543263">PMC7543263</a></p>
</div>
<div id="ref-1FTnvyZlf">
<p>681. <strong>Antiviral Monoclonal Antibodies: Can They Be More Than Simple Neutralizing Agents?</strong> <br />
Mireia Pelegrin, Mar Naranjo-Gomez, Marc Piechaczyk<br />
<em>Trends in Microbiology</em> (2015-10) <a href="https://doi.org/f7vzrf">https://doi.org/f7vzrf</a> <br />
DOI: <a href="https://doi.org/10.1016/j.tim.2015.07.005">10.1016/j.tim.2015.07.005</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26433697">26433697</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7127033">PMC7127033</a></p>
</div>
<div id="ref-Ull2rQ5L">
<p>682. <strong>Intranasal Treatment with Poly(I{middle dot}C) Protects Aged Mice from Lethal Respiratory Virus Infections</strong> <br />
J. Zhao, C. Wohlford-Lenane, J. Zhao, E. Fleming, T. E. Lane, P. B. McCray, S. Perlman<br />
<em>Journal of Virology</em> (2012-08-22) <a href="https://doi.org/f4bzfp">https://doi.org/f4bzfp</a> <br />
DOI: <a href="https://doi.org/10.1128/jvi.01410-12">10.1128/jvi.01410-12</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22915814">22915814</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3486278">PMC3486278</a></p>
</div>
<div id="ref-AJLaaguT">
<p>683. <strong>Main protease structure and XChem fragment screen</strong> <br />
Diamond<br />
(2020-05-05) <a href="https://www.diamond.ac.uk/covid-19/for-scientists/Main-protease-structure-and-XChem.html">https://www.diamond.ac.uk/covid-19/for-scientists/Main-protease-structure-and-XChem.html</a></p>
</div>
<div id="ref-bDNDEYNn">
<p>684. <strong>Quantifying the social distancing privilege gap: a longitudinal study of smartphone movement</strong> <br />
Nabarun Dasgupta, Michele Jonsson Funk, Allison Lazard, Benjamin Eugene White, Stephen W. Marshall<br />
<em>Cold Spring Harbor Laboratory</em> (2020-05-08) <a href="https://doi.org/gg79qk">https://doi.org/gg79qk</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.05.03.20084624">10.1101/2020.05.03.20084624</a></p>
</div>
<div id="ref-RdnghmVU">
<p>685. <strong>Uncovering socioeconomic gaps in mobility reduction during the COVID-19 pandemic using location data</strong> <br />
Samuel P. Fraiberger, Pablo Astudillo, Lorenzo Candeago, Alex Chunet, Nicholas K. W. Jones, Maham Faisal Khan, Bruno Lepri, Nancy Lozano Gracia, Lorenzo Lucchini, Emanuele Massaro, Aleister Montfort<br />
<em>arXiv</em> (2020-07-28) <a href="https://arxiv.org/abs/2006.15195">https://arxiv.org/abs/2006.15195</a></p>
</div>
<div id="ref-9OksxLmM">
<p>686. <strong>Mobility network models of COVID-19 explain inequities and inform reopening</strong> <br />
Serina Chang, Emma Pierson, Pang Wei Koh, Jaline Gerardin, Beth Redbird, David Grusky, Jure Leskovec<br />
<em>Nature</em> (2020-11-10) <a href="https://doi.org/ghjmt2">https://doi.org/ghjmt2</a> <br />
DOI: <a href="https://doi.org/10.1038/s41586-020-2923-3">10.1038/s41586-020-2923-3</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33171481">33171481</a></p>
</div>
<div id="ref-15qfRPnUG">
<p>687. <strong>A Basic Demographic Profile of Workers in Frontline Industries</strong> <br />
Hye Jin Rho;Shawn Fremstad;Hayley Brown<br />
(2020-06) <a href="https://mronline.org/wp-content/uploads/2020/06/2020-04-Frontline-Workers.pdf">https://mronline.org/wp-content/uploads/2020/06/2020-04-Frontline-Workers.pdf</a></p>
</div>
<div id="ref-HihYbn6k">
<p>688. <strong>Differential occupational risk for COVID‐19 and other infection exposure according to race and ethnicity</strong> <br />
Devan Hawkins<br />
<em>American Journal of Industrial Medicine</em> (2020-06-15) <a href="https://doi.org/gg3rb2">https://doi.org/gg3rb2</a> <br />
DOI: <a href="https://doi.org/10.1002/ajim.23145">10.1002/ajim.23145</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32539166">32539166</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7323065">PMC7323065</a></p>
</div>
<div id="ref-F4laSOXM">
<p>689. <strong>Estimating the burden of United States workers exposed to infection or disease: A key factor in containing risk of COVID-19 infection</strong> <br />
Marissa G. Baker, Trevor K. Peckham, Noah S. Seixas<br />
<em>PLOS ONE</em> (2020-04-28) <a href="https://doi.org/ggtx7c">https://doi.org/ggtx7c</a> <br />
DOI: <a href="https://doi.org/10.1371/journal.pone.0232452">10.1371/journal.pone.0232452</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32343747">32343747</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7188235">PMC7188235</a></p>
</div>
<div id="ref-hAYohR8m">
<p>690. <strong>Coronavirus (COVID-19) related deaths by occupation, England and Wales: deaths registered up to and including 20 April 2020</strong> <br />
Ben Windsor-Shellard, Jasveer Kaur<br />
(2020-05-11) <a href="https://www.ons.gov.uk/peoplepopulationandcommunity/healthandsocialcare/causesofdeath/bulletins/coronaviruscovid19relateddeathsbyoccupationenglandandwales/deathsregistereduptoandincluding20april2020">https://www.ons.gov.uk/peoplepopulationandcommunity/healthandsocialcare/causesofdeath/bulletins/coronaviruscovid19relateddeathsbyoccupationenglandandwales/deathsregistereduptoandincluding20april2020</a></p>
</div>
<div id="ref-DOyiuaup">
<p>691. <strong>Which occupations have the highest potential exposure to the coronavirus (COVID-19)?</strong> <br />
Office for National Statistics<br />
(2020-05-11) <a href="https://www.ons.gov.uk/employmentandlabourmarket/peopleinwork/employmentandemployeetypes/articles/whichoccupationshavethehighestpotentialexposuretothecoronaviruscovid19/2020-05-11">https://www.ons.gov.uk/employmentandlabourmarket/peopleinwork/employmentandemployeetypes/articles/whichoccupationshavethehighestpotentialexposuretothecoronaviruscovid19/2020-05-11</a></p>
</div>
<div id="ref-13qkVV6eN">
<p>692. <strong>Disparities in the risk and outcomes from COVID-19</strong> <br />
Public Health England<br />
(2020-06-12) <a href="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/892085/disparities_review.pdf">https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/892085/disparities_review.pdf</a></p>
</div>
<div id="ref-4zjepZ8T">
<p>693. <strong>Exclusive: deaths of NHS staff from covid-19 analysed</strong> <br />
Tim Cook, Emira Kursumovic, Simon Lennane2020-04-22T12:42:00+01:00<br />
<em>Health Service Journal</em> <a href="https://www.hsj.co.uk/exclusive-deaths-of-nhs-staff-from-covid-19-analysed/7027471.article">https://www.hsj.co.uk/exclusive-deaths-of-nhs-staff-from-covid-19-analysed/7027471.article</a></p>
</div>
<div id="ref-2gxJPENM">
<p>694. <a href="https://act.nationalnursesunited.org/page">https://act.nationalnursesunited.org/page</a></p>
</div>
<div id="ref-Moc3BREQ">
<p>695. <strong>Racial Disparity in COVID-19 Deaths: Seeking Economic Roots with Census data.</strong> <br />
John McLaren<br />
<em>National Bureau of Economic Research</em> (2020-06-22) <a href="https://www.nber.org/papers/w27407">https://www.nber.org/papers/w27407</a></p>
</div>
<div id="ref-aM2XPE3J">
<p>696. <strong>Mortality, Admissions, and Patient Census at SNFs in 3 US Cities During the COVID-19 Pandemic</strong> <br />
Michael L. Barnett, Lissy Hu, Thomas Martin, David C. Grabowski<br />
<em>JAMA</em> (2020-08-04) <a href="https://doi.org/gg3387">https://doi.org/gg3387</a> <br />
DOI: <a href="https://doi.org/10.1001/jama.2020.11642">10.1001/jama.2020.11642</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32579161">32579161</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7315390">PMC7315390</a></p>
</div>
<div id="ref-BeedQx3I">
<p>697. <strong>COVID-19 in Prisons and Jails in the United States</strong> <br />
Laura Hawks, Steffie Woolhandler, Danny McCormick<br />
<em>JAMA Internal Medicine</em> (2020-08-01) <a href="https://doi.org/ggtxw6">https://doi.org/ggtxw6</a> <br />
DOI: <a href="https://doi.org/10.1001/jamainternmed.2020.1856">10.1001/jamainternmed.2020.1856</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32343355">32343355</a></p>
</div>
<div id="ref-2uUcva6Y">
<p>698. <strong>COVID-19 Cases and Deaths in Federal and State Prisons</strong> <br />
Brendan Saloner, Kalind Parish, Julie A. Ward, Grace DiLaura, Sharon Dolovich<br />
<em>JAMA</em> (2020-08-11) <a href="https://doi.org/gg4dcv">https://doi.org/gg4dcv</a> <br />
DOI: <a href="https://doi.org/10.1001/jama.2020.12528">10.1001/jama.2020.12528</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32639537">32639537</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7344796">PMC7344796</a></p>
</div>
<div id="ref-zuipD7ZP">
<p>699. <strong>State Rates of Incarceration Race &amp; Ethnicity_updated2</strong> <br />
nlizanna<br />
(2018-03-24) <a href="https://www.issuelab.org/resources/695/695.pdf">https://www.issuelab.org/resources/695/695.pdf</a></p>
</div>
<div id="ref-OzhmHOxJ">
<p>700. <strong>Under One Roof: A Review of Research on Intergenerational Coresidence and Multigenerational Households in the United States</strong> <br />
Jennifer Reid Keene, Christie D. Batson<br />
<em>Sociology Compass</em> (2010-08) <a href="https://doi.org/fsrcr7">https://doi.org/fsrcr7</a> <br />
DOI: <a href="https://doi.org/10.1111/j.1751-9020.2010.00306.x">10.1111/j.1751-9020.2010.00306.x</a></p>
</div>
<div id="ref-13wfXDXqr">
<p>701. <strong>Chaos and the macrosetting: The role of poverty and socioeconomic status.</strong> <br />
Gary W. Evans, John Eckenrode, Lyscha A. Marcynyszyn<br />
<em>American Psychological Association (APA)</em> (2010-01-12) <a href="https://doi.org/c76n3g">https://doi.org/c76n3g</a> <br />
DOI: <a href="https://doi.org/10.1037/12057-014">10.1037/12057-014</a></p>
</div>
<div id="ref-1Axz3a1Hj">
<p>702. <strong>Housing insecurity among urban fathers</strong> <br />
Marah A. Curtis, Amanda B. Geller<br />
<em>Columbia University</em> (2010) <a href="https://doi.org/ghdjn2">https://doi.org/ghdjn2</a> <br />
DOI: <a href="https://doi.org/10.7916/d8wh2w9t">10.7916/d8wh2w9t</a></p>
</div>
<div id="ref-64YOO3yt">
<p>703. <strong>Housing and Employment Insecurity among the Working Poor</strong> <br />
Matthew Desmond, Carl Gershenson<br />
<em>Social Problems</em> (2016-02) <a href="https://doi.org/f8crm2">https://doi.org/f8crm2</a> <br />
DOI: <a href="https://doi.org/10.1093/socpro/spv025">10.1093/socpro/spv025</a></p>
</div>
<div id="ref-sRZCzj6j">
<p>704. <strong>Obesity and its comorbid conditions</strong> <br />
Lalita Khaodhiar, Karen C. McCowen, George L. Blackburn<br />
<em>Clinical Cornerstone</em> (1999-01) <a href="https://doi.org/bpp37d">https://doi.org/bpp37d</a> <br />
DOI: <a href="https://doi.org/10.1016/s1098-3597(99)90002-9">10.1016/s1098-3597(99)90002-9</a></p>
</div>
<div id="ref-JizR1ocj">
<p>705. <strong>Aging, Male Sex, Obesity, and Metabolic Inflammation Create the Perfect Storm for COVID-19</strong> <br />
Franck Mauvais-Jarvis<br />
<em>Diabetes</em> (2020-09) <a href="https://doi.org/gg47zk">https://doi.org/gg47zk</a> <br />
DOI: <a href="https://doi.org/10.2337/dbi19-0023">10.2337/dbi19-0023</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32669390">32669390</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7458034">PMC7458034</a></p>
</div>
<div id="ref-nYLB9EMw">
<p>706. <strong>Non-communicable disease syndemics: poverty, depression, and diabetes among low-income populations</strong> <br />
Emily Mendenhall, Brandon A Kohrt, Shane A Norris, David Ndetei, Dorairaj Prabhakaran<br />
<em>The Lancet</em> (2017-03) <a href="https://doi.org/gddg84">https://doi.org/gddg84</a> <br />
DOI: <a href="https://doi.org/10.1016/s0140-6736(17)30402-6">10.1016/s0140-6736(17)30402-6</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28271846">28271846</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5491333">PMC5491333</a></p>
</div>
<div id="ref-h1e9QTlK">
<p>707. <strong>Obesity and poverty paradox in developed countries</strong> <br />
Wioletta Żukiewicz-Sobczak, Paula Wróblewska, Jacek Zwoliński, Jolanta Chmielewska-Badora, Piotr Adamczuk, Ewelina Krasowska, Jerzy Zagórski, Anna Oniszczuk, Jacek Piątek, Wojciech Silny<br />
<em>Annals of Agricultural and Environmental Medicine</em> (2014-09-04) <a href="https://doi.org/f6jhzc">https://doi.org/f6jhzc</a> <br />
DOI: <a href="https://doi.org/10.5604/12321966.1120608">10.5604/12321966.1120608</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25292135">25292135</a></p>
</div>
<div id="ref-94eOAGnA">
<p>708. <strong>The Impact of the COVID‐19 Pandemic on Unhealthy Eating in Populations with Obesity</strong> <br />
Nathaniel J. S. Ashby<br />
<em>Obesity</em> (2020-06-26) <a href="https://doi.org/ghd6qc">https://doi.org/ghd6qc</a> <br />
DOI: <a href="https://doi.org/10.1002/oby.22940">10.1002/oby.22940</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32589788">32589788</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7361200">PMC7361200</a></p>
</div>
<div id="ref-5G5UozXd">
<p>709. <strong>Fast‐Food Patronage and Obesity Prevalence during the COVID‐19 Pandemic: An Alternative Explanation</strong> <br />
Candice A. Myers, Stephanie T. Broyles<br />
<em>Obesity</em> (2020-08-02) <a href="https://doi.org/gg6v84">https://doi.org/gg6v84</a> <br />
DOI: <a href="https://doi.org/10.1002/oby.22993">10.1002/oby.22993</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32741130">32741130</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7435526">PMC7435526</a></p>
</div>
<div id="ref-17yfPiYAS">
<p>710. <strong>The global food syndemic: The impact of food insecurity, Malnutrition and obesity on the healthspan amid the COVID-19 pandemic</strong> <br />
Martha I. Huizar, Ross Arena, Deepika R. Laddu<br />
<em>Progress in Cardiovascular Diseases</em> (2020-07) <a href="https://doi.org/gg4r3h">https://doi.org/gg4r3h</a> <br />
DOI: <a href="https://doi.org/10.1016/j.pcad.2020.07.002">10.1016/j.pcad.2020.07.002</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32653438">32653438</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7347484">PMC7347484</a></p>
</div>
<div id="ref-Hu7PyNNp">
<p>711. <strong>Stress, chronic inflammation, and emotional and physical well-being: Concurrent effects and chronic sequelae</strong> <br />
George P. Chrousos<br />
<em>Journal of Allergy and Clinical Immunology</em> (2000-11) <a href="https://doi.org/bgx7hn">https://doi.org/bgx7hn</a> <br />
DOI: <a href="https://doi.org/10.1067/mai.2000.110163">10.1067/mai.2000.110163</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/11080744">11080744</a></p>
</div>
<div id="ref-P2wUkjrR">
<p>712. <strong>Chronic psychological stress and the regulation of pro-inflammatory cytokines: A glucocorticoid-resistance model.</strong> <br />
Gregory E. Miller, Sheldon Cohen, A. Kim Ritchey<br />
<em>Health Psychology</em> (2002) <a href="https://doi.org/dj5r8b">https://doi.org/dj5r8b</a> <br />
DOI: <a href="https://doi.org/10.1037/0278-6133.21.6.531">10.1037/0278-6133.21.6.531</a></p>
</div>
<div id="ref-109EtOYzw">
<p>713. <strong>Chronic stress, daily stressors, and circulating inflammatory markers.</strong> <br />
Jean-Philippe Gouin, Ronald Glaser, William B. Malarkey, David Beversdorf, Janice Kiecolt-Glaser<br />
<em>Health Psychology</em> (2012-03) <a href="https://doi.org/dkz9tr">https://doi.org/dkz9tr</a> <br />
DOI: <a href="https://doi.org/10.1037/a0025536">10.1037/a0025536</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21928900">21928900</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3253267">PMC3253267</a></p>
</div>
<div id="ref-11oGPv27i">
<p>714. <strong>Turning Up the Heat</strong> <br />
Gregory E. Miller, Ekin Blackwell<br />
<em>Current Directions in Psychological Science</em> (2016-06-24) <a href="https://doi.org/bft9mv">https://doi.org/bft9mv</a> <br />
DOI: <a href="https://doi.org/10.1111/j.1467-8721.2006.00450.x">10.1111/j.1467-8721.2006.00450.x</a></p>
</div>
<div id="ref-KZSJklnl">
<p>715. <strong>Sick of Poverty</strong> <br />
Robert Sapolsky<br />
<em>Scientific American</em> (2005-12) <a href="https://doi.org/fxf5kp">https://doi.org/fxf5kp</a> <br />
DOI: <a href="https://doi.org/10.1038/scientificamerican1205-92">10.1038/scientificamerican1205-92</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16323696">16323696</a></p>
</div>
<div id="ref-5BMDcXz3">
<p>716. <strong>Exposure to air pollution and COVID-19 mortality in the United States: A nationwide cross-sectional study</strong> <br />
Xiao Wu, Rachel C. Nethery, Benjamin M. Sabath, Danielle Braun, Francesca Dominici<br />
<em>Cold Spring Harbor Laboratory</em> (2020-04-27) <a href="https://doi.org/ggrpcj">https://doi.org/ggrpcj</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.04.05.20054502">10.1101/2020.04.05.20054502</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32511651">32511651</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7277007">PMC7277007</a></p>
</div>
<div id="ref-vImXUPNS">
<p>717. <strong>Global estimates of mortality associated with long-term exposure to outdoor fine particulate matter</strong> <br />
Richard Burnett, Hong Chen, Mieczysław Szyszkowicz, Neal Fann, Bryan Hubbell, C. Arden Pope, Joshua S. Apte, Michael Brauer, Aaron Cohen, Scott Weichenthal, … Joseph V. Spadaro<br />
<em>Proceedings of the National Academy of Sciences</em> (2018-09-18) <a href="https://doi.org/gfgbcx">https://doi.org/gfgbcx</a> <br />
DOI: <a href="https://doi.org/10.1073/pnas.1803222115">10.1073/pnas.1803222115</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30181279">30181279</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6156628">PMC6156628</a></p>
</div>
<div id="ref-6ACdDJOV">
<p>718. <strong>Early-Life Air Pollution Exposure, Neighborhood Poverty, and Childhood Asthma in the United States, 1990–2014</strong> <br />
Nicole Kravitz-Wirtz, Samantha Teixeira, Anjum Hajat, Bongki Woo, Kyle Crowder, David Takeuchi<br />
<em>International Journal of Environmental Research and Public Health</em> (2018-05-30) <a href="https://doi.org/gdvwp9">https://doi.org/gdvwp9</a> <br />
DOI: <a href="https://doi.org/10.3390/ijerph15061114">10.3390/ijerph15061114</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29848979">29848979</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6025399">PMC6025399</a></p>
</div>
<div id="ref-11Ift2eIl">
<p>719. <strong>Early life stress, air pollution, inflammation, and disease: An integrative review and immunologic model of social-environmental adversity and lifespan health</strong> <br />
Hector A. Olvera Alvarez, Laura D. Kubzansky, Matthew J. Campen, George M. Slavich<br />
<em>Neuroscience &amp; Biobehavioral Reviews</em> (2018-09) <a href="https://doi.org/gd46bm">https://doi.org/gd46bm</a> <br />
DOI: <a href="https://doi.org/10.1016/j.neubiorev.2018.06.002">10.1016/j.neubiorev.2018.06.002</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29874545">29874545</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6082389">PMC6082389</a></p>
</div>
<div id="ref-aHM6WZzd">
<p>720. <strong>Covid-19 and Disparities in Nutrition and Obesity</strong> <br />
Matthew J. Belanger, Michael A. Hill, Angeliki M. Angelidi, Maria Dalamaga, James R. Sowers, Christos S. Mantzoros<br />
<em>New England Journal of Medicine</em> (2020-07-15) <a href="https://doi.org/gg475x">https://doi.org/gg475x</a> <br />
DOI: <a href="https://doi.org/10.1056/nejmp2021264">10.1056/nejmp2021264</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32668105">32668105</a></p>
</div>
<div id="ref-11Ldf3Ch3">
<p>721. <strong>Systemic racism, chronic health inequities, and ‐19: A syndemic in the making?</strong> <br />
Clarence C. Gravlee<br />
<em>American Journal of Human Biology</em> (2020-08-04) <a href="https://doi.org/ghcxwk">https://doi.org/ghcxwk</a> <br />
DOI: <a href="https://doi.org/10.1002/ajhb.23482">10.1002/ajhb.23482</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32754945">32754945</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7441277">PMC7441277</a></p>
</div>
<div id="ref-cSA2h9hO">
<p>722. <strong>Coronavirus Disease (COVID-19): A primer for emergency physicians</strong> <br />
Summer Chavez, Brit Long, Alex Koyfman, Stephen Y. Liang<br />
<em>The American Journal of Emergency Medicine</em> (2020-03) <a href="https://doi.org/ggr22z">https://doi.org/ggr22z</a> <br />
DOI: <a href="https://doi.org/10.1016/j.ajem.2020.03.036">10.1016/j.ajem.2020.03.036</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32265065">32265065</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7102516">PMC7102516</a></p>
</div>
<div id="ref-1Intrlax">
<p>723. <strong>Insurers May Only Pay For Coronavirus Tests When They’re “Medically Necessary”</strong> <br />
NPR.org<br />
<a href="https://www.npr.org/sections/health-shots/2020/06/19/880543755/insurers-may-only-pay-for-coronavirus-tests-when-theyre-medically-necessary">https://www.npr.org/sections/health-shots/2020/06/19/880543755/insurers-may-only-pay-for-coronavirus-tests-when-theyre-medically-necessary</a></p>
</div>
<div id="ref-jc9vc7ws">
<p>724. <strong>Private Health Insurance Coverage in the COVID-19 Public Health Emergency | Commonwealth Fund</strong> <a href="https://www.commonwealthfund.org/blog/2020/private-health-insurance-coverage-covid-19-public-health-emergency">https://www.commonwealthfund.org/blog/2020/private-health-insurance-coverage-covid-19-public-health-emergency</a></p>
</div>
<div id="ref-13MGsHBtw">
<p>725. <strong>COVID-19 and racial disparities</strong> <br />
Monica Shah, Muskaan Sachdeva, Roni P. Dodiuk-Gad<br />
<em>Journal of the American Academy of Dermatology</em> (2020-07) <a href="https://doi.org/ggtwm7">https://doi.org/ggtwm7</a> <br />
DOI: <a href="https://doi.org/10.1016/j.jaad.2020.04.046">10.1016/j.jaad.2020.04.046</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32305444">32305444</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7162783">PMC7162783</a></p>
</div>
<div id="ref-P8VNkYTE">
<p>726. <strong>FAQs for COVID-19 Claims Reimbursement to Health Care Providers and Facilities for Testing and Treatment of the Uninsured</strong> <br />
Official web site of the U.S. Health Resources &amp; Services Administration<br />
<a href="https://www.hrsa.gov/coviduninsuredclaim/frequently-asked-questions">https://www.hrsa.gov/coviduninsuredclaim/frequently-asked-questions</a></p>
</div>
<div id="ref-tLVYjfMk">
<p>727. <strong>Potential association between COVID-19 mortality and health-care resource availability</strong> <br />
Yunpeng Ji, Zhongren Ma, Maikel P Peppelenbosch, Qiuwei Pan<br />
<em>The Lancet Global Health</em> (2020-02) <a href="https://doi.org/ggqscd">https://doi.org/ggqscd</a> <br />
DOI: <a href="https://doi.org/10.1016/s2214-109x(20)30068-1">10.1016/s2214-109x(20)30068-1</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32109372">32109372</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7128131">PMC7128131</a></p>
</div>
<div id="ref-cOQK0zNa">
<p>728. <strong>Combating COVID-19: health equity matters</strong> <br />
Zhicheng Wang, Kun Tang<br />
<em>Nature Medicine</em> (2020-03-26) <a href="https://doi.org/ggs4p6">https://doi.org/ggs4p6</a> <br />
DOI: <a href="https://doi.org/10.1038/s41591-020-0823-6">10.1038/s41591-020-0823-6</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32284617">32284617</a></p>
</div>
<div id="ref-gOiB5HDd">
<p>729. <strong>lockup black</strong> <br />
chen0307<br />
(2020-10-14) <a href="https://www.census.gov/content/dam/Census/library/publications/2019/demo/p60-267.pdf">https://www.census.gov/content/dam/Census/library/publications/2019/demo/p60-267.pdf</a></p>
</div>
<div id="ref-15Yaf3fc8">
<p>730. <strong>An Ethical Dilemma in SARS-Cov-2 Pandemic : Who Gets the Ventilator?</strong> <br />
Dumache Raluca, Ciocan Veronica, Muresan Camelia Oana, Enache Alexandra<br />
<em>European Scientific Journal ESJ</em> (2020-07-31) <a href="https://doi.org/ghfprk">https://doi.org/ghfprk</a> <br />
DOI: <a href="https://doi.org/10.19044/esj.2020.v16n21p24">10.19044/esj.2020.v16n21p24</a></p>
</div>
<div id="ref-1As9kWVvA">
<p>731. <strong>Planning Hospital Needs for Ventilators and Respiratory Therapists in the COVID-19 Crisis</strong> <br />
John Raffensperger, Marygail Brauner, R. Briggs<br />
<em>Rand Corporation</em> (2020) <a href="https://doi.org/ghfprp">https://doi.org/ghfprp</a> <br />
DOI: <a href="https://doi.org/10.7249/pea228-1">10.7249/pea228-1</a></p>
</div>
<div id="ref-FaGAJ7AI">
<p>732. <strong>Fair Allocation of Vaccines, Ventilators and Antiviral Treatments: Leaving No Ethical Value Behind in Health Care Rationing</strong> <br />
Parag A. Pathak, Tayfun Sönmez, M. Utku Ünver, M. Bumin Yenmez<br />
<em>arXiv</em> (2020-08-04) <a href="https://arxiv.org/abs/2008.00374">https://arxiv.org/abs/2008.00374</a></p>
</div>
<div id="ref-B3awlFvs">
<p>733. <strong>Reallocating ventilators during the coronavirus disease 2019 pandemic: Is it ethical?</strong> <br />
Quyen Chu, Ricardo Correa, Tracey L. Henry, Kyle A. McGregor, Hanni Stoklosa, Loren Robinson, Sachin Jha, Alagappan Annamalai, Benson S. Hsu, Rohit Gupta, … SreyRam Kuy<br />
<em>Surgery</em> (2020-09) <a href="https://doi.org/ghfprb">https://doi.org/ghfprb</a> <br />
DOI: <a href="https://doi.org/10.1016/j.surg.2020.04.044">10.1016/j.surg.2020.04.044</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32616345">32616345</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7205622">PMC7205622</a></p>
</div>
<div id="ref-2i1Y7SYn">
<p>734. <strong>Ethics Lessons From Seattle’s Early Experience With COVID-19</strong> <br />
Denise M. Dudzinski, Benjamin Y. Hoisington, Crystal E. Brown<br />
<em>The American Journal of Bioethics</em> (2020-06-18) <a href="https://doi.org/ghfprc">https://doi.org/ghfprc</a> <br />
DOI: <a href="https://doi.org/10.1080/15265161.2020.1764137">10.1080/15265161.2020.1764137</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32552455">32552455</a></p>
</div>
<div id="ref-GTL02UT6">
<p>735. <strong>Rationing Limited Healthcare Resources in the COVID‐19 Era and Beyond: Ethical Considerations Regarding Older Adults</strong> <br />
Timothy W. Farrell, Leslie Francis, Teneille Brown, Lauren E. Ferrante, Eric Widera, Ramona Rhodes, Tony Rosen, Ula Hwang, Leah J. Witt, Niranjan Thothala, … Debra Saliba<br />
<em>Journal of the American Geriatrics Society</em> (2020-06-14) <a href="https://doi.org/ggvt7z">https://doi.org/ggvt7z</a> <br />
DOI: <a href="https://doi.org/10.1111/jgs.16539">10.1111/jgs.16539</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32374466">32374466</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7267288">PMC7267288</a></p>
</div>
<div id="ref-KQfJ4llw">
<p>736. <strong>Paediatric ethical issues during the COVID‐19 pandemic are not just about ventilator triage</strong> <br />
Marlyse F. Haward, Gregory P. Moore, John Lantos, Annie Janvier<br />
<em>Acta Paediatrica</em> (2020-05-20) <a href="https://doi.org/ggv24n">https://doi.org/ggv24n</a> <br />
DOI: <a href="https://doi.org/10.1111/apa.15334">10.1111/apa.15334</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32364256">32364256</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7267437">PMC7267437</a></p>
</div>
<div id="ref-K6yc6elC">
<p>737. <strong>Ethical Challenges Arising in the COVID-19 Pandemic: An Overview from the Association of Bioethics Program Directors (ABPD) Task Force</strong> <br />
Amy L. McGuire, Mark P. Aulisio, F. Daniel Davis, Cheryl Erwin, Thomas D. Harter, Reshma Jagsi, Robert Klitzman, Robert Macauley, Eric Racine, Susan M. Wolf, … The COVID-19 Task Force of the Association of Bioethics Program Directors (ABPD)<br />
<em>The American Journal of Bioethics</em> (2020-06-08) <a href="https://doi.org/gg6c5k">https://doi.org/gg6c5k</a> <br />
DOI: <a href="https://doi.org/10.1080/15265161.2020.1764138">10.1080/15265161.2020.1764138</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32511078">32511078</a></p>
</div>
<div id="ref-1EaLRFk21">
<p>738. <strong>Disability, Ethics, and Health Care in the COVID-19 Pandemic</strong> <br />
Maya Sabatello, Teresa Blankmeyer Burke, Katherine E. McDonald, Paul S. Appelbaum<br />
<em>American Journal of Public Health</em> (2020-10) <a href="https://doi.org/ghfprm">https://doi.org/ghfprm</a> <br />
DOI: <a href="https://doi.org/10.2105/ajph.2020.305837">10.2105/ajph.2020.305837</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32816541">32816541</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7483109">PMC7483109</a></p>
</div>
<div id="ref-cPsuWT7U">
<p>739. <strong>Allocating Ventilators During the COVID-19 Pandemic and Conscientious Objection</strong> <br />
Mark Wicclair<br />
<em>The American Journal of Bioethics</em> (2020-07-27) <a href="https://doi.org/gg6nk4">https://doi.org/gg6nk4</a> <br />
DOI: <a href="https://doi.org/10.1080/15265161.2020.1777347">10.1080/15265161.2020.1777347</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32716798">32716798</a></p>
</div>
<div id="ref-1D1FX4zQZ">
<p>740. <strong>Colorblind Algorithms: Racism in the Era of COVID-19</strong> <br />
J. Corey Williams, Nientara Anderson, Myra Mathis, Ezelle Samford, Jeffrey Eugene, Jessica Isom<br />
<em>Journal of the National Medical Association</em> (2020-06) <a href="https://doi.org/ghfpq8">https://doi.org/ghfpq8</a> <br />
DOI: <a href="https://doi.org/10.1016/j.jnma.2020.05.010">10.1016/j.jnma.2020.05.010</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32563687">32563687</a></p>
</div>
<div id="ref-LrqhFVIz">
<p>741. <strong>Structural Racism, Social Risk Factors, and Covid-19 — A Dangerous Convergence for Black Americans</strong> <br />
Leonard E. Egede, Rebekah J. Walker<br />
<em>New England Journal of Medicine</em> (2020-09-17) <a href="https://doi.org/gg56nc">https://doi.org/gg56nc</a> <br />
DOI: <a href="https://doi.org/10.1056/nejmp2023616">10.1056/nejmp2023616</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32706952">32706952</a></p>
</div>
<div id="ref-PFHCN5ep">
<p>742. <strong>Allocating Remdesivir Under Scarcity: Social Justice or More Systemic Racism</strong> <br />
Eli Weber, Mark J. Bliton<br />
<em>The American Journal of Bioethics</em> (2020-08-25) <a href="https://doi.org/ghfprf">https://doi.org/ghfprf</a> <br />
DOI: <a href="https://doi.org/10.1080/15265161.2020.1795538">10.1080/15265161.2020.1795538</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32840451">32840451</a></p>
</div>
<div id="ref-1ExRX1x4C">
<p>743. <strong>Revisiting the equity debate in COVID-19: ICU is no panacea</strong> <br />
Angela Ballantyne, Wendy A Rogers, Vikki Entwistle, Cindy Towns<br />
<em>Journal of Medical Ethics</em> (2020-06-22) <a href="https://doi.org/gg33nq">https://doi.org/gg33nq</a> <br />
DOI: <a href="https://doi.org/10.1136/medethics-2020-106460">10.1136/medethics-2020-106460</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32571847">32571847</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7335695">PMC7335695</a></p>
</div>
<div id="ref-ztTunmqF">
<p>744. <strong>Ethical Dilemmas in Covid-19 Medical Care: Is a Problematic Triage Protocol Better or Worse than No Protocol at All?</strong> <br />
Sheri Fink<br />
<em>The American Journal of Bioethics</em> (2020-07-27) <a href="https://doi.org/gg6nqn">https://doi.org/gg6nqn</a> <br />
DOI: <a href="https://doi.org/10.1080/15265161.2020.1788663">10.1080/15265161.2020.1788663</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32716771">32716771</a></p>
</div>
<div id="ref-18et5lcBc">
<p>745. <strong>Developing an Ethics Framework for Allocating Remdesivir in the COVID-19 Pandemic</strong> <br />
Sarah Lim, Debra A. DeBruin, Jonathon P. Leider, Nneka Sederstrom, Ruth Lynfield, Jason V. Baker, Susan Kline, Sarah Kesler, Stacey Rizza, Joel Wu, … Susan M. Wolf<br />
<em>Mayo Clinic Proceedings</em> (2020-09) <a href="https://doi.org/ghfpq9">https://doi.org/ghfpq9</a> <br />
DOI: <a href="https://doi.org/10.1016/j.mayocp.2020.06.016">10.1016/j.mayocp.2020.06.016</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32861338">32861338</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7305893">PMC7305893</a></p>
</div>
<div id="ref-11gslRJJU">
<p>746. <strong>Ethically Allocating COVID-19 Drugs Via Pre-approval Access and Emergency Use Authorization</strong> <br />
Jamie Webb, Lesha D. Shah, Holly Fernandez Lynch<br />
<em>The American Journal of Bioethics</em> (2020-08-25) <a href="https://doi.org/ghfprd">https://doi.org/ghfprd</a> <br />
DOI: <a href="https://doi.org/10.1080/15265161.2020.1795529">10.1080/15265161.2020.1795529</a></p>
</div>
<div id="ref-ugEOYgfx">
<p>747. (2011-07-27) <a href="https://www.cdc.gov/about/advisory/pdf/VentDocument_Release.pdf">https://www.cdc.gov/about/advisory/pdf/VentDocument_Release.pdf</a></p>
</div>
<div id="ref-1A5E2TkBS">
<p>748. <strong>Adopting an Anti-Racist Model of COVID-19 Drug Allocation and Prioritization</strong> <br />
Akilah A. Jefferson<br />
<em>The American Journal of Bioethics</em> (2020-08-25) <a href="https://doi.org/ghggz5">https://doi.org/ghggz5</a> <br />
DOI: <a href="https://doi.org/10.1080/15265161.2020.1795541">10.1080/15265161.2020.1795541</a></p>
</div>
<div id="ref-PZMP42Ak">
<p>749. <strong>Opportunities and obstacles for deep learning in biology and medicine</strong> <br />
Travers Ching, Daniel S. Himmelstein, Brett K. Beaulieu-Jones, Alexandr A. Kalinin, Brian T. Do, Gregory P. Way, Enrico Ferrero, Paul-Michael Agapow, Michael Zietz, Michael M. Hoffman, … Casey S. Greene<br />
<em>Journal of The Royal Society Interface</em> (2018-04-04) <a href="https://doi.org/gddkhn">https://doi.org/gddkhn</a> <br />
DOI: <a href="https://doi.org/10.1098/rsif.2017.0387">10.1098/rsif.2017.0387</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29618526">29618526</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5938574">PMC5938574</a></p>
</div>
<div id="ref-heDA5StF">
<p>750. <strong>Opportunities and obstacles for deep learning in biology and medicine [update in progress]</strong> <br />
Casey S. Greene, Daniel S. Himmelstein, Daniel C. Elton, Brock C. Christensen, Anthony Gitter, Alexander J. Titus, Joshua J. Levy<br />
<em>Manubot</em> (2020-11-10) <a href="https://greenelab.github.io/deep-review/">https://greenelab.github.io/deep-review/</a></p>
</div>
<div id="ref-cYo4O2qX">
<p>751. <strong>ismms-himc/covid-19_sinai_reviews</strong> <br />
Human Immune Monitoring Center at Mount Sinai<br />
(2020-07-09) <a href="https://github.com/ismms-himc/covid-19_sinai_reviews">https://github.com/ismms-himc/covid-19_sinai_reviews</a></p>
</div>
<div id="ref-DGTDsJZy">
<p>752. <strong>Undergraduate Mentoring - American Physician Scientists Association</strong> <a href="https://www.physicianscientists.org/page/summer-research-pilot-program">https://www.physicianscientists.org/page/summer-research-pilot-program</a></p>
</div>
<div id="ref-11MwmdOKi">
<p>753. <strong>greenelab/covid19-review</strong> <br />
Greene Laboratory<br />
(2020-11-12) <a href="https://github.com/greenelab/covid19-review">https://github.com/greenelab/covid19-review</a></p>
</div>
<div id="ref-yTsmmAYC">
<p>754. <strong>SARS-CoV-2 and COVID-19: An Evolving Review of Diagnostics and Therapeutics</strong> <br />
Halie M. Rando, Casey S. Greene, Michael P. Robson, Simina M. Boca, Nils Wellhausen, Ronan Lordan, Christian Brueffer, Sandipan Ray, Lucy D\’Agostino McGowan, Anthony Gitter, … Tiago Lubiana<br />
<em>Manubot</em> (2020-11-12) <a href="https://greenelab.github.io/covid19-review/">https://greenelab.github.io/covid19-review/</a></p>
</div>
<div id="ref-1026Gxdsi">
<p>755. <strong>Matplotlib: A 2D Graphics Environment</strong> <br />
John D. Hunter<br />
<em>Computing in Science &amp; Engineering</em> (2007) <a href="https://doi.org/drbjhg">https://doi.org/drbjhg</a> <br />
DOI: <a href="https://doi.org/10.1109/mcse.2007.55">10.1109/mcse.2007.55</a></p>
</div>
<div id="ref-17OQtAY4l">
<p>756. <strong>Using the MAARIE Framework To Read the Research Literature</strong> <br />
M. Corcoran<br />
<em>American Journal of Occupational Therapy</em> (2006-07-01) <a href="https://doi.org/bqh97x">https://doi.org/bqh97x</a> <br />
DOI: <a href="https://doi.org/10.5014/ajot.60.4.367">10.5014/ajot.60.4.367</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16915865">16915865</a></p>
</div>
<div id="ref-1HLSeqnY8">
<p>757. <strong>Potent binding of 2019 novel coronavirus spike protein by a SARS coronavirus-specific human monoclonal antibody</strong> <br />
Xiaolong Tian, Cheng Li, Ailing Huang, Shuai Xia, Sicong Lu, Zhengli Shi, Lu Lu, Shibo Jiang, Zhenlin Yang, Yanling Wu, Tianlei Ying<br />
<em>Cold Spring Harbor Laboratory</em> (2020-01-28) <a href="https://doi.org/ggjqfd">https://doi.org/ggjqfd</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.01.28.923011">10.1101/2020.01.28.923011</a></p>
</div>
<div id="ref-lOQPScZt">
<p>758. <strong>Integrative Bioinformatics Analysis Provides Insight into the Molecular Mechanisms of 2019-nCoV</strong> <br />
Xiang He, Lei Zhang, Qin Ran, Junyi Wang, Anying Xiong, Dehong Wu, Feng Chen, Guoping Li<br />
<em>Cold Spring Harbor Laboratory</em> (2020-02-05) <a href="https://doi.org/ggrbd8">https://doi.org/ggrbd8</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.02.03.20020206">10.1101/2020.02.03.20020206</a></p>
</div>
<div id="ref-rL9IRhVy">
<p>759. <strong>Diarrhea may be underestimated: a missing link in 2019 novel coronavirus</strong> <br />
Weicheng Liang, Zhijie Feng, Shitao Rao, Cuicui Xiao, Ze-Xiao Lin, Qi Zhang, Qi Wei<br />
<em>Cold Spring Harbor Laboratory</em> (2020-02-17) <a href="https://doi.org/ggrbdw">https://doi.org/ggrbdw</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.02.03.20020289">10.1101/2020.02.03.20020289</a></p>
</div>
<div id="ref-BHQvMYPG">
<p>760. <strong>Specific ACE2 Expression in Cholangiocytes May Cause Liver Damage After 2019-nCoV Infection</strong> <br />
Xiaoqiang Chai, Longfei Hu, Yan Zhang, Weiyu Han, Zhou Lu, Aiwu Ke, Jian Zhou, Guoming Shi, Nan Fang, Jia Fan, … Fei Lan<br />
<em>bioRxiv</em> (2020-02-04) <a href="https://doi.org/ggq626">https://doi.org/ggq626</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.02.03.931766">10.1101/2020.02.03.931766</a></p>
</div>
<div id="ref-OuDMtCRl">
<p>761. <strong>Recapitulation of SARS-CoV-2 Infection and Cholangiocyte Damage with Human Liver Organoids</strong> <br />
Bing Zhao, Chao Ni, Ran Gao, Yuyan Wang, Li Yang, Jinsong Wei, Ting Lv, Jianqing Liang, Qisheng Zhang, Wei Xu, … Xinhua Lin<br />
<em>Cold Spring Harbor Laboratory</em> (2020-03-17) <a href="https://doi.org/ggq648">https://doi.org/ggq648</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.16.990317">10.1101/2020.03.16.990317</a></p>
</div>
<div id="ref-ViWCb8MJ">
<p>762. <strong>ACE2 expression by colonic epithelial cells is associated with viral infection, immunity and energy metabolism</strong> <br />
Jun Wang, Shanmeizi Zhao, Ming Liu, Zhiyao Zhao, Yiping Xu, Ping Wang, Meng Lin, Yanhui Xu, Bing Huang, Xiaoyu Zuo, … Yuxia Zhang<br />
<em>Cold Spring Harbor Laboratory</em> (2020-02-07) <a href="https://doi.org/ggrfbx">https://doi.org/ggrfbx</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.02.05.20020545">10.1101/2020.02.05.20020545</a></p>
</div>
<div id="ref-MgLBA5m">
<p>763. <strong>The Pathogenicity of SARS-CoV-2 in hACE2 Transgenic Mice</strong> <br />
Linlin Bao, Wei Deng, Baoying Huang, Hong Gao, Jiangning Liu, Lili Ren, Qiang Wei, Pin Yu, Yanfeng Xu, Feifei Qi, … Chuan Qin<br />
<em>bioRxiv</em> (2020-02-28) <a href="https://doi.org/dph2">https://doi.org/dph2</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.02.07.939389">10.1101/2020.02.07.939389</a></p>
</div>
<div id="ref-1Acv5UMSt">
<p>764. <strong>Caution on Kidney Dysfunctions of COVID-19 Patients</strong> <br />
Zhen Li, Ming Wu, Jiwei Yao, Jie Guo, Xiang Liao, Siji Song, Jiali Li, Guangjie Duan, Yuanxiu Zhou, Xiaojun Wu, … Junan Yan<br />
<em>Cold Spring Harbor Laboratory</em> (2020-03-27) <a href="https://doi.org/ggq627">https://doi.org/ggq627</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.02.08.20021212">10.1101/2020.02.08.20021212</a></p>
</div>
<div id="ref-DMaiZBE1">
<p>765. <strong>Acute renal impairment in coronavirus-associated severe acute respiratory syndrome</strong> <br />
Kwok Hong Chu, Wai Kay Tsang, Colin S. Tang, Man Fai Lam, Fernand M. Lai, Ka Fai To, Ka Shun Fung, Hon Lok Tang, Wing Wa Yan, Hilda W. H. Chan, … Kar Neng Lai<br />
<em>Kidney International</em> (2005-02) <a href="https://doi.org/b7tgtx">https://doi.org/b7tgtx</a> <br />
DOI: <a href="https://doi.org/10.1111/j.1523-1755.2005.67130.x">10.1111/j.1523-1755.2005.67130.x</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15673319">15673319</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7112337">PMC7112337</a></p>
</div>
<div id="ref-16Xj27QYj">
<p>766. <strong>Single-cell Analysis of ACE2 Expression in Human Kidneys and Bladders Reveals a Potential Route of 2019-nCoV Infection</strong> <br />
Wei Lin, Longfei Hu, Yan Zhang, Joshua D. Ooi, Ting Meng, Peng Jin, Xiang Ding, Longkai Peng, Lei Song, Zhou Xiao, … Yong Zhong<br />
<em>Cold Spring Harbor Laboratory</em> (2020-02-18) <a href="https://doi.org/ggq629">https://doi.org/ggq629</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.02.08.939892">10.1101/2020.02.08.939892</a></p>
</div>
<div id="ref-QhE5mVCc">
<p>767. <strong>The immune vulnerability landscape of the 2019 Novel Coronavirus, SARS-CoV-2</strong> <br />
James Zhu, Jiwoong Kim, Xue Xiao, Yunguan Wang, Danni Luo, Shuang Jiang, Ran Chen, Lin Xu, He Zhang, Lenny Moise, … Yang Xie<br />
<em>Cold Spring Harbor Laboratory</em> (2020-09-04) <a href="https://doi.org/ggq628">https://doi.org/ggq628</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.02.08.939553">10.1101/2020.02.08.939553</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32908981">32908981</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7480032">PMC7480032</a></p>
</div>
<div id="ref-1FE1YGsaV">
<p>768. <strong>Tissue distribution of ACE2 protein, the functional receptor for SARS coronavirus. A first step in understanding SARS pathogenesis</strong> <br />
I Hamming, W Timens, MLC Bulthuis, AT Lely, GJ Navis, H van Goor<br />
<em>The Journal of Pathology</em> (2004-06) <a href="https://doi.org/bhpzc3">https://doi.org/bhpzc3</a> <br />
DOI: <a href="https://doi.org/10.1002/path.1570">10.1002/path.1570</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15141377">15141377</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7167720">PMC7167720</a></p>
</div>
<div id="ref-iObQU2el">
<p>769. <strong>Clinical Course and Outcomes of Critically Ill Patients With Middle East Respiratory Syndrome Coronavirus Infection</strong> <br />
Yaseen M. Arabi, Ahmed A. Arifi, Hanan H. Balkhy, Hani Najm, Abdulaziz S. Aldawood, Alaa Ghabashi, Hassan Hawa, Adel Alothman, Abdulaziz Khaldi, Basel Al Raiy<br />
<em>Annals of Internal Medicine</em> (2014-03-18) <a href="https://doi.org/ggptxw">https://doi.org/ggptxw</a> <br />
DOI: <a href="https://doi.org/10.7326/m13-2486">10.7326/m13-2486</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24474051">24474051</a></p>
</div>
<div id="ref-gK5e7Yhl">
<p>770. <strong>Neutrophil-to-Lymphocyte Ratio Predicts Severe Illness Patients with 2019 Novel Coronavirus in the Early Stage</strong> <br />
Jingyuan Liu, Yao Liu, Pan Xiang, Lin Pu, Haofeng Xiong, Chuansheng Li, Ming Zhang, Jianbo Tan, Yanli Xu, Rui Song, … Xianbo Wang<br />
<em>medRxiv</em> (2020-02-12) <a href="https://doi.org/ggrbdx">https://doi.org/ggrbdx</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.02.10.20021584">10.1101/2020.02.10.20021584</a></p>
</div>
<div id="ref-6Y81C4P3">
<p>771. <strong>Dysregulation of immune response in patients with COVID-19 in Wuhan, China</strong> <br />
Chuan Qin, Luoqi Zhou, Ziwei Hu, Shuoqi Zhang, Sheng Yang, Yu Tao, Cuihong Xie, Ke Ma, Ke Shang, Wei Wang, Dai-Shi Tian<br />
<em>Clinical Infectious Diseases</em> (2020-03-12) <a href="https://doi.org/ggpxcf">https://doi.org/ggpxcf</a> <br />
DOI: <a href="https://doi.org/10.1093/cid/ciaa248">10.1093/cid/ciaa248</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32161940">32161940</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7108125">PMC7108125</a></p>
</div>
<div id="ref-P4assQHj">
<p>772. <strong>Characteristics of lymphocyte subsets and cytokines in peripheral blood of 123 hospitalized patients with 2019 novel coronavirus pneumonia (NCP)</strong> <br />
Suxin Wan, Qingjie Yi, Shibing Fan, Jinglong Lv, Xianxiang Zhang, Lian Guo, Chunhui Lang, Qing Xiao, Kaihu Xiao, Zhengjun Yi, … Yongping Chen<br />
<em>medRxiv</em> (2020-02-12) <a href="https://doi.org/ggq63b">https://doi.org/ggq63b</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.02.10.20021832">10.1101/2020.02.10.20021832</a></p>
</div>
<div id="ref-zskgSnle">
<p>773. <strong>Longitudinal Characteristics of Lymphocyte Responses and Cytokine Profiles in the Peripheral Blood of SARS-CoV-2 Infected Patients</strong> <br />
Jing Liu, Sumeng Li, Jia Liu, Boyun Liang, Xiaobei Wang, Wei Li, Hua Wang, Qiaoxia Tong, Jianhua Yi, Lei Zhao, … Xin Zheng<br />
<em>SSRN Electronic Journal</em> (2020) <a href="https://doi.org/ggq655">https://doi.org/ggq655</a> <br />
DOI: <a href="https://doi.org/10.2139/ssrn.3539682">10.2139/ssrn.3539682</a></p>
</div>
<div id="ref-TXlNzmXd">
<p>774. <strong>Epidemiological and Clinical Characteristics of 17 Hospitalized Patients with 2019 Novel Coronavirus Infections Outside Wuhan, China</strong> <br />
Jie Li, Shilin Li, Yurui Cai, Qin Liu, Xue Li, Zhaoping Zeng, Yanpeng Chu, Fangcheng Zhu, Fanxin Zeng<br />
<em>Cold Spring Harbor Laboratory</em> (2020-02-12) <a href="https://doi.org/ggq63c">https://doi.org/ggq63c</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.02.11.20022053">10.1101/2020.02.11.20022053</a></p>
</div>
<div id="ref-15K5Ovntj">
<p>775. <strong>ACE2 Expression in Kidney and Testis May Cause Kidney and Testis Damage After 2019-nCoV Infection</strong> <br />
Caibin Fan, Kai Li, Yanhong Ding, Wei Lu Lu, Jianqing Wang<br />
<em>medRxiv</em> (2020-02-13) <a href="https://doi.org/ggq63d">https://doi.org/ggq63d</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.02.12.20022418">10.1101/2020.02.12.20022418</a></p>
</div>
<div id="ref-Bgo7Cmj3">
<p>776. <strong>Aberrant pathogenic GM-CSF <sup>+</sup> T cells and inflammatory CD14 <sup>+</sup> CD16 <sup>+</sup> monocytes in severe pulmonary syndrome patients of a new coronavirus</strong> <br />
Yonggang Zhou, Binqing Fu, Xiaohu Zheng, Dongsheng Wang, Changcheng Zhao, Yingjie qi, Rui Sun, Zhigang Tian, Xiaoling Xu, Haiming Wei<br />
<em>bioRxiv</em> (2020-02-20) <a href="https://doi.org/ggq63f">https://doi.org/ggq63f</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.02.12.945576">10.1101/2020.02.12.945576</a></p>
</div>
<div id="ref-4Z1pEioX">
<p>777. <strong>Clinical Characteristics of 2019 Novel Infected Coronavirus Pneumonia：A Systemic Review and Meta-analysis</strong> <br />
Kai Qian, Yi Deng, Yonghang Tai, Jun Peng, Hao Peng, Lihong Jiang<br />
<em>medRxiv</em> (2020-02-17) <a href="https://doi.org/ggrgbq">https://doi.org/ggrgbq</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.02.14.20021535">10.1101/2020.02.14.20021535</a></p>
</div>
<div id="ref-zTGzuiTI">
<p>778. <strong>Longitudinal characteristics of lymphocyte responses and cytokine profiles in the peripheral blood of SARS-CoV-2 infected patients</strong> <br />
Jing Liu, Sumeng Li, Jia Liu, Boyun Liang, Xiaobei Wang, Hua Wang, Wei Li, Qiaoxia Tong, Jianhua Yi, Lei Zhao, … Xin Zheng<br />
<em>medRxiv</em> (2020-02-22) <a href="https://doi.org/ggq63g">https://doi.org/ggq63g</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.02.16.20023671">10.1101/2020.02.16.20023671</a></p>
</div>
<div id="ref-1EIEx8Qe4">
<p>779. <strong>Clinical and immunologic features in severe and moderate forms of Coronavirus Disease 2019</strong> <br />
Guang Chen, Di Wu, Wei Guo, Yong Cao, Da Huang, Hongwu Wang, Tao Wang, Xiaoyun Zhang, Huilong Chen, Haijing Yu, … Qin Ning<br />
<em>Cold Spring Harbor Laboratory</em> (2020-02-19) <a href="https://doi.org/ggq63h">https://doi.org/ggq63h</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.02.16.20023903">10.1101/2020.02.16.20023903</a></p>
</div>
<div id="ref-41WAtZmB">
<p>780. <strong>SARS-CoV-2 and SARS-CoV Spike-RBD Structure and Receptor Binding Comparison and Potential Implications on Neutralizing Antibody and Vaccine Development</strong> <br />
Chunyun Sun, Long Chen, Ji Yang, Chunxia Luo, Yanjing Zhang, Jing Li, Jiahui Yang, Jie Zhang, Liangzhi Xie<br />
<em>bioRxiv</em> (2020-02-20) <a href="https://doi.org/ggq63j">https://doi.org/ggq63j</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.02.16.951723">10.1101/2020.02.16.951723</a></p>
</div>
<div id="ref-NPxs7bbq">
<p>781. <strong>Protection of Rhesus Macaque from SARS-Coronavirus challenge by recombinant adenovirus vaccine</strong> <br />
Yiyou Chen, Qiang Wei, Ruobing Li, Hong Gao, Hua Zhu, Wei Deng, Linlin Bao, Wei Tong, Zhe Cong, Hong Jiang, Chuan Qin<br />
<em>Cold Spring Harbor Laboratory</em> (2020-02-21) <a href="https://doi.org/ggq63k">https://doi.org/ggq63k</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.02.17.951939">10.1101/2020.02.17.951939</a></p>
</div>
<div id="ref-f50ahNxF">
<p>782. <strong>Reduction and Functional Exhaustion of T Cells in Patients with Coronavirus Disease 2019 (COVID-19)</strong> <br />
Bo Diao, Chenhui Wang, Yingjun Tan, Xiewan Chen, Ying Liu, Lifeng Ning, Li Chen, Min Li, Yueping Liu, Gang Wang, … Yongwen Chen<br />
<em>medRxiv</em> (2020-02-20) <a href="https://doi.org/ggq63m">https://doi.org/ggq63m</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.02.18.20024364">10.1101/2020.02.18.20024364</a></p>
</div>
<div id="ref-EwNi3WJq">
<p>783. <strong>Clinical characteristics of 25 death cases with COVID-19: a retrospective review of medical records in a single medical center, Wuhan, China</strong> <br />
Xun Li, Luwen Wang, Shaonan Yan, Fan Yang, Longkui Xiang, Jiling Zhu, Bo Shen, Zuojiong Gong<br />
<em>Cold Spring Harbor Laboratory</em> (2020-02-25) <a href="https://doi.org/ggq63n">https://doi.org/ggq63n</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.02.19.20025239">10.1101/2020.02.19.20025239</a></p>
</div>
<div id="ref-KG5Lg65z">
<p>784. <strong>SARS-CoV-2 infection does not significantly cause acute renal injury: an analysis of 116 hospitalized patients with COVID-19 in a single hospital, Wuhan, China</strong> <br />
Luwen Wang, Xun Li, Hui Chen, Shaonan Yan, Yan Li, Dong Li, Zuojiong Gong<br />
<em>Cold Spring Harbor Laboratory</em> (2020-02-23) <a href="https://doi.org/ggq63p">https://doi.org/ggq63p</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.02.19.20025288">10.1101/2020.02.19.20025288</a></p>
</div>
<div id="ref-VRoblZFJ">
<p>785. <strong>Clinical Characteristics of 138 Hospitalized Patients With 2019 Novel Coronavirus–Infected Pneumonia in Wuhan, China</strong> <br />
Dawei Wang, Bo Hu, Chang Hu, Fangfang Zhu, Xing Liu, Jing Zhang, Binbin Wang, Hui Xiang, Zhenshun Cheng, Yong Xiong, … Zhiyong Peng<br />
<em>JAMA</em> (2020-02-07) <a href="https://doi.org/ggkh48">https://doi.org/ggkh48</a> <br />
DOI: <a href="https://doi.org/10.1001/jama.2020.1585">10.1001/jama.2020.1585</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32031570">32031570</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7042881">PMC7042881</a></p>
</div>
<div id="ref-bgrzawWp">
<p>786. <strong>Clinical characteristics of 2019 novel coronavirus infection in China</strong> <br />
Wei-jie Guan, Zheng-yi Ni, Yu Hu, Wen-hua Liang, Chun-quan Ou, Jian-xing He, Lei Liu, Hong Shan, Chun-liang Lei, David SC Hui, … Nan-shan Zhong<br />
<em>medRxiv</em> (2020-02-09) <a href="https://doi.org/ggkj9s">https://doi.org/ggkj9s</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.02.06.20020974">10.1101/2020.02.06.20020974</a></p>
</div>
<div id="ref-hdSafq1o">
<p>787. <strong>Potential T-cell and B-cell Epitopes of 2019-nCoV</strong> <br />
Ethan Fast, Russ B. Altman, Binbin Chen<br />
<em>Cold Spring Harbor Laboratory</em> (2020-03-18) <a href="https://doi.org/ggq63q">https://doi.org/ggq63q</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.02.19.955484">10.1101/2020.02.19.955484</a></p>
</div>
<div id="ref-8xnxgpag">
<p>788. <strong>Structure, function and antigenicity of the SARS-CoV-2 spike glycoprotein</strong> <br />
Alexandra C. Walls, Young-Jun Park, M. Alexandra Tortorici, Abigail Wall, Andrew T. McGuire, David Veesler<br />
<em>bioRxiv</em> (2020-02-20) <a href="https://doi.org/ggrgbr">https://doi.org/ggrgbr</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.02.19.956581">10.1101/2020.02.19.956581</a></p>
</div>
<div id="ref-QdimT8mN">
<p>789. <strong>Breadth of concomitant immune responses underpinning viral clearance and patient recovery in a non-severe case of COVID-19</strong> <br />
Irani Thevarajan, Thi HO Nguyen, Marios Koutsakos, Julian Druce, Leon Caly, Carolien E van de Sandt, Xiaoxiao Jia, Suellen Nicholson, Mike Catton, Benjamin Cowie, … Katherine Kedzierska<br />
<em>Cold Spring Harbor Laboratory</em> (2020-02-23) <a href="https://doi.org/ggq63r">https://doi.org/ggq63r</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.02.20.20025841">10.1101/2020.02.20.20025841</a></p>
</div>
<div id="ref-tVkKuIJt">
<p>790. <strong>The landscape of lung bronchoalveolar immune cells in COVID-19 revealed by single-cell RNA sequencing</strong> <br />
Minfeng Liao, Yang Liu, Jin Yuan, Yanling Wen, Gang Xu, Juanjuan Zhao, Lin Chen, Jinxiu Li, Xin Wang, Fuxiang Wang, … Zheng Zhang<br />
<em>medRxiv</em> (2020-02-26) <a href="https://doi.org/ggq63s">https://doi.org/ggq63s</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.02.23.20026690">10.1101/2020.02.23.20026690</a></p>
</div>
<div id="ref-87CJN2CG">
<p>791. <strong>Influenza A Virus Infection Induces Hyperresponsiveness in Human Lung Tissue-Resident and Peripheral Blood NK Cells</strong> <br />
Marlena Scharenberg, Sindhu Vangeti, Eliisa Kekäläinen, Per Bergman, Mamdoh Al-Ameri, Niclas Johansson, Klara Sondén, Sara Falck-Jones, Anna Färnert, Hans-Gustaf Ljunggren, … Nicole Marquardt<br />
<em>Frontiers in Immunology</em> (2019-05-17) <a href="https://doi.org/ggq656">https://doi.org/ggq656</a> <br />
DOI: <a href="https://doi.org/10.3389/fimmu.2019.01116">10.3389/fimmu.2019.01116</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31156653">31156653</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6534051">PMC6534051</a></p>
</div>
<div id="ref-vasHgjGH">
<p>792. <strong>Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China</strong> <br />
Chaolin Huang, Yeming Wang, Xingwang Li, Lili Ren, Jianping Zhao, Yi Hu, Li Zhang, Guohui Fan, Jiuyang Xu, Xiaoying Gu, … Bin Cao<br />
<em>The Lancet</em> (2020-02) <a href="https://doi.org/ggjfnn">https://doi.org/ggjfnn</a> <br />
DOI: <a href="https://doi.org/10.1016/s0140-6736(20)30183-5">10.1016/s0140-6736(20)30183-5</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31986264">31986264</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7159299">PMC7159299</a></p>
</div>
<div id="ref-9GEyqic3">
<p>793. <strong>Alveolar Macrophages in the Resolution of Inflammation, Tissue Repair, and Tolerance to Infection</strong> <br />
Benoit Allard, Alice Panariti, James G. Martin<br />
<em>Frontiers in Immunology</em> (2018-07-31) <a href="https://doi.org/gd3bnz">https://doi.org/gd3bnz</a> <br />
DOI: <a href="https://doi.org/10.3389/fimmu.2018.01777">10.3389/fimmu.2018.01777</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30108592">30108592</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6079255">PMC6079255</a></p>
</div>
<div id="ref-WBN5f37b">
<p>794. <strong>PPAR-γ in Macrophages Limits Pulmonary Inflammation and Promotes Host Recovery following Respiratory Viral Infection</strong> <br />
Su Huang, Bibo Zhu, In Su Cheon, Nick P. Goplen, Li Jiang, Ruixuan Zhang, R. Stokes Peebles, Matthias Mack, Mark H. Kaplan, Andrew H. Limper, Jie Sun<br />
<em>Journal of Virology</em> (2019-04-17) <a href="https://doi.org/ggq652">https://doi.org/ggq652</a> <br />
DOI: <a href="https://doi.org/10.1128/jvi.00030-19">10.1128/jvi.00030-19</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30787149">30787149</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6475778">PMC6475778</a></p>
</div>
<div id="ref-FOVDaThy">
<p>795. <strong>Can routine laboratory tests discriminate 2019 novel coronavirus infected pneumonia from other community-acquired pneumonia?</strong> <br />
Yunbao Pan, Guangming Ye, Xiantao Zeng, Guohong Liu, Xiaojiao Zeng, Xianghu Jiang, Jin Zhao, Liangjun Chen, Shuang Guo, Qiaoling Deng, … Xinghuan Wang<br />
<em>Cold Spring Harbor Laboratory</em> (2020-02-25) <a href="https://doi.org/ggq63t">https://doi.org/ggq63t</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.02.25.20024711">10.1101/2020.02.25.20024711</a></p>
</div>
<div id="ref-1Du40IOOH">
<p>796. <strong>Correlation Analysis Between Disease Severity and Inflammation-related Parameters in Patients with COVID-19 Pneumonia</strong> <br />
Jing Gong, Hui Dong, Qingsong Xia, Zhaoyi Huang, Dingkun Wang, Yan Zhao, Wenhua Liu, Shenghao Tu, Mingmin Zhang, Qi Wang, Fuer Lu<br />
<em>Cold Spring Harbor Laboratory</em> (2020-02-26) <a href="https://doi.org/ggq63v">https://doi.org/ggq63v</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.02.25.20025643">10.1101/2020.02.25.20025643</a></p>
</div>
<div id="ref-1DTLMIDoM">
<p>797. <strong>An Effective CTL Peptide Vaccine for Ebola Zaire Based on Survivors’ CD8+ Targeting of a Particular Nucleocapsid Protein Epitope with Potential Implications for COVID-19 Vaccine Design</strong> <br />
CV Herst, S Burkholz, J Sidney, A Sette, PE Harris, S Massey, T Brasel, E Cunha-Neto, DS Rosa, WCH Chao, … R Rubsamen<br />
<em>Cold Spring Harbor Laboratory</em> (2020-04-06) <a href="https://doi.org/ggq63x">https://doi.org/ggq63x</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.02.25.963546">10.1101/2020.02.25.963546</a></p>
</div>
<div id="ref-9spK49tt">
<p>798. <strong>Epitope-based peptide vaccine design and target site characterization against novel coronavirus disease caused by SARS-CoV-2</strong> <br />
Lin Li, Ting Sun, Yufei He, Wendong Li, Yubo Fan, Jing Zhang<br />
<em>Cold Spring Harbor Laboratory</em> (2020-02-27) <a href="https://doi.org/ggnqwt">https://doi.org/ggnqwt</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.02.25.965434">10.1101/2020.02.25.965434</a></p>
</div>
<div id="ref-qg1bmQng">
<p>799. <strong>The definition and risks of Cytokine Release Syndrome-Like in 11 COVID-19-Infected Pneumonia critically ill patients: Disease Characteristics and Retrospective Analysis</strong> <br />
Wang Wenjun, Liu Xiaoqing, Wu Sipei, Lie Puyi, Huang Liyan, Li Yimin, Cheng Linling, Chen Sibei, Nong Lingbo, Lin Yongping, He Jianxing<br />
<em>Cold Spring Harbor Laboratory</em> (2020-02-27) <a href="https://doi.org/ggrgbs">https://doi.org/ggrgbs</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.02.26.20026989">10.1101/2020.02.26.20026989</a></p>
</div>
<div id="ref-YODgfrtM">
<p>800. <strong>Clinical characteristics of 36 non-survivors with COVID-19 in Wuhan, China</strong> <br />
Ying Huang, Rui Yang, Ying Xu, Ping Gong<br />
<em>Cold Spring Harbor Laboratory</em> (2020-03-05) <a href="https://doi.org/ggq63z">https://doi.org/ggq63z</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.02.27.20029009">10.1101/2020.02.27.20029009</a></p>
</div>
<div id="ref-5CGbn9Jj">
<p>801. <strong>Risk factors related to hepatic injury in patients with corona virus disease 2019</strong> <br />
Lu Li, Shuang Li, Manman Xu, Pengfei Yu, Sujun Zheng, Zhongping Duan, Jing Liu, Yu Chen, Junfeng Li<br />
<em>Cold Spring Harbor Laboratory</em> (2020-03-10) <a href="https://doi.org/ggq632">https://doi.org/ggq632</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.02.28.20028514">10.1101/2020.02.28.20028514</a></p>
</div>
<div id="ref-T0BDaXkw">
<p>802. <strong>Detectable serum SARS-CoV-2 viral load (RNAaemia) is closely associated with drastically elevated interleukin 6 (IL-6) level in critically ill COVID-19 patients</strong> <br />
Xiaohua Chen, Binghong Zhao, Yueming Qu, Yurou Chen, Jie Xiong, Yong Feng, Dong Men, Qianchuan Huang, Ying Liu, Bo Yang, … Feng Li<br />
<em>medRxiv</em> (2020-03-03) <a href="https://doi.org/ggq633">https://doi.org/ggq633</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.02.29.20029520">10.1101/2020.02.29.20029520</a></p>
</div>
<div id="ref-l3CzJ5ni">
<p>803. <strong>Prognostic factors in the acute respiratory distress syndrome</strong> <br />
Wei Chen, Lorraine B Ware<br />
<em>Clinical and Translational Medicine</em> (2015-07-02) <a href="https://doi.org/ggq653">https://doi.org/ggq653</a> <br />
DOI: <a href="https://doi.org/10.1186/s40169-015-0065-2">10.1186/s40169-015-0065-2</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26162279">26162279</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4534483">PMC4534483</a></p>
</div>
<div id="ref-uIwTzZkJ">
<p>804. <strong>Lymphopenia predicts disease severity of COVID-19: a descriptive and predictive study</strong> <br />
Li Tan, Qi Wang, Duanyang Zhang, Jinya Ding, Qianchuan Huang, Yi-Quan Tang, Qiongshu Wang, Hongming Miao<br />
<em>Cold Spring Harbor Laboratory</em> (2020-03-03) <a href="https://doi.org/ggq634">https://doi.org/ggq634</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.01.20029074">10.1101/2020.03.01.20029074</a></p>
</div>
<div id="ref-1GMxHWnxN">
<p>805. <strong>The potential role of IL-6 in monitoring severe case of coronavirus disease 2019</strong> <br />
Tao Liu, Jieying Zhang, Yuhui Yang, Hong Ma, Zhengyu Li, Jiaoyue Zhang, Ji Cheng, Xiaoyun Zhang, Yanxia Zhao, Zihan Xia, … Jianhua Yi<br />
<em>Cold Spring Harbor Laboratory</em> (2020-03-10) <a href="https://doi.org/ggq635">https://doi.org/ggq635</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.01.20029769">10.1101/2020.03.01.20029769</a></p>
</div>
<div id="ref-1B0vBwv1q">
<p>806. <strong>Clinical and Laboratory Profiles of 75 Hospitalized Patients with Novel Coronavirus Disease 2019 in Hefei, China</strong> <br />
Zonghao Zhao, Jiajia Xie, Ming Yin, Yun Yang, Hongliang He, Tengchuan Jin, Wenting Li, Xiaowu Zhu, Jing Xu, Changcheng Zhao, … Xiaoling Ma<br />
<em>Cold Spring Harbor Laboratory</em> (2020-03-06) <a href="https://doi.org/ggq636">https://doi.org/ggq636</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.01.20029785">10.1101/2020.03.01.20029785</a></p>
</div>
<div id="ref-RyXNw87v">
<p>807. <strong>Exuberant elevation of IP-10, MCP-3 and IL-1ra during SARS-CoV-2 infection is associated with disease severity and fatal outcome</strong> <br />
Yang Yang, Chenguang Shen, Jinxiu Li, Jing Yuan, Minghui Yang, Fuxiang Wang, Guobao Li, Yanjie Li, Li Xing, Ling Peng, … Yingxia Liu<br />
<em>medRxiv</em> (2020-03-06) <a href="https://doi.org/ggq637">https://doi.org/ggq637</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.02.20029975">10.1101/2020.03.02.20029975</a></p>
</div>
<div id="ref-5HXuCa5t">
<p>808. <strong>Antibody responses to SARS-CoV-2 in patients of novel coronavirus disease 2019</strong> <br />
Juanjuan Zhao, Quan Yuan, Haiyan Wang, Wei Liu, Xuejiao Liao, Yingying Su, Xin Wang, Jing Yuan, Tingdong Li, Jinxiu Li, … Zheng Zhang<br />
<em>Cold Spring Harbor Laboratory</em> (2020-03-02) <a href="https://doi.org/ggrbj6">https://doi.org/ggrbj6</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.02.20030189">10.1101/2020.03.02.20030189</a></p>
</div>
<div id="ref-uCyoHtV3">
<p>809. <strong>Restoration of leukomonocyte counts is associated with viral clearance in COVID-19 hospitalized patients</strong> <br />
Xiaoping Chen, Jiaxin Ling, Pingzheng Mo, Yongxi Zhang, Qunqun Jiang, Zhiyong Ma, Qian Cao, Wenjia Hu, Shi Zou, Liangjun Chen, … Yong Xiong<br />
<em>medRxiv</em> (2020-03-06) <a href="https://doi.org/ggq639">https://doi.org/ggq639</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.03.20030437">10.1101/2020.03.03.20030437</a></p>
</div>
<div id="ref-L6kwccjn">
<p>810. <strong>Effects of Systemically Administered Hydrocortisone on the Human Immunome</strong> <br />
Matthew J. Olnes, Yuri Kotliarov, Angélique Biancotto, Foo Cheung, Jinguo Chen, Rongye Shi, Huizhi Zhou, Ena Wang, John S. Tsang, Robert Nussenblatt, The CHI Consortium<br />
<em>Scientific Reports</em> (2016-03-14) <a href="https://doi.org/f8dmvw">https://doi.org/f8dmvw</a> <br />
DOI: <a href="https://doi.org/10.1038/srep23002">10.1038/srep23002</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26972611">26972611</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4789739">PMC4789739</a></p>
</div>
<div id="ref-K9G9QT8y">
<p>811. <strong>Procalcitonin in patients with severe coronavirus disease 2019 (COVID-19): A meta-analysis</strong> <br />
Giuseppe Lippi, Mario Plebani<br />
<em>Clinica Chimica Acta</em> (2020-06) <a href="https://doi.org/ggpxp7">https://doi.org/ggpxp7</a> <br />
DOI: <a href="https://doi.org/10.1016/j.cca.2020.03.004">10.1016/j.cca.2020.03.004</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32145275">32145275</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7094472">PMC7094472</a></p>
</div>
<div id="ref-USOnp7c0">
<p>812. <strong>Clinical findings in critical ill patients infected with SARS-Cov-2 in Guangdong Province, China: a multi-center, retrospective, observational study</strong> <br />
Yonghao Xu, Zhiheng Xu, Xuesong Liu, Lihua Cai, Haichong Zheng, Yongbo Huang, Lixin Zhou, Linxi Huang, Yun Lin, Liehua Deng, … Yimin Li<br />
<em>Cold Spring Harbor Laboratory</em> (2020-03-06) <a href="https://doi.org/ggq64b">https://doi.org/ggq64b</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.03.20030668">10.1101/2020.03.03.20030668</a></p>
</div>
<div id="ref-14OLQIYK0">
<p>813. <strong>Multi-epitope vaccine design using an immunoinformatics approach for 2019 novel coronavirus (SARS-CoV-2)</strong> <br />
Ye Feng, Min Qiu, Liang Liu, Shengmei Zou, Yun Li, Kai Luo, Qianpeng Guo, Ning Han, Yingqiang Sun, Kui Wang, … Fan Mo<br />
<em>Cold Spring Harbor Laboratory</em> (2020-06-30) <a href="https://doi.org/ggq64c">https://doi.org/ggq64c</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.03.962332">10.1101/2020.03.03.962332</a></p>
</div>
<div id="ref-12ilBRq6I">
<p>814. <strong>Clinical Features of Patients Infected with the 2019 Novel Coronavirus (COVID-19) in Shanghai, China</strong> <br />
Min Cao, Dandan Zhang, Youhua Wang, Yunfei Lu, Xiangdong Zhu, Ying Li, Honghao Xue, Yunxiao Lin, Min Zhang, Yiguo Sun, … Longping Peng<br />
<em>Cold Spring Harbor Laboratory</em> (2020-03-06) <a href="https://doi.org/ggq64d">https://doi.org/ggq64d</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.04.20030395">10.1101/2020.03.04.20030395</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32511465">32511465</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7255784">PMC7255784</a></p>
</div>
<div id="ref-IPJURDxi">
<p>815. <strong>Serological detection of 2019-nCoV respond to the epidemic: A useful complement to nucleic acid testing</strong> <br />
Jin Zhang, Jianhua Liu, Na Li, Yong Liu, Rui Ye, Xiaosong Qin, Rui Zheng<br />
<em>Cold Spring Harbor Laboratory</em> (2020-03-10) <a href="https://doi.org/ggq64f">https://doi.org/ggq64f</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.04.20030916">10.1101/2020.03.04.20030916</a></p>
</div>
<div id="ref-H0nvocJz">
<p>816. <strong>Human Kidney is a Target for Novel Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) Infection</strong> <br />
Bo Diao, Chenhui Wang, Rongshuai Wang, Zeqing Feng, Yingjun Tan, Huiming Wang, Changsong Wang, Liang Liu, Ying Liu, Yueping Liu, … Yongwen Chen<br />
<em>medRxiv</em> (2020-04-10) <a href="https://doi.org/ggq64g">https://doi.org/ggq64g</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.04.20031120">10.1101/2020.03.04.20031120</a></p>
</div>
<div id="ref-KHNnjP7M">
<p>817. <strong>COVID-19 early warning score: a multi-parameter screening tool to identify highly suspected patients</strong> <br />
Cong-Ying Song, Jia Xu, Jian-Qin He, Yuan-Qiang Lu<br />
<em>medRxiv</em> (2020-03-08) <a href="https://doi.org/ggq64h">https://doi.org/ggq64h</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.05.20031906">10.1101/2020.03.05.20031906</a></p>
</div>
<div id="ref-19xwRoatu">
<p>818. <strong>LY6E impairs coronavirus fusion and confers immune control of viral disease</strong> <br />
Stephanie Pfaender, Katrina B. Mar, Eleftherios Michailidis, Annika Kratzel, Dagny Hirt, Philip V’kovski, Wenchun Fan, Nadine Ebert, Hanspeter Stalder, Hannah Kleine-Weber, … Volker Thiel<br />
<em>Cold Spring Harbor Laboratory</em> (2020-03-07) <a href="https://doi.org/dpvn">https://doi.org/dpvn</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.05.979260">10.1101/2020.03.05.979260</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32511345">32511345</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7255780">PMC7255780</a></p>
</div>
<div id="ref-574aSa8C">
<p>819. <strong>A preliminary study on serological assay for severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) in 238 admitted hospital patients</strong> <br />
Lei Liu, Wanbing Liu, Shengdian Wang, Shangen Zheng<br />
<em>Cold Spring Harbor Laboratory</em> (2020-03-08) <a href="https://doi.org/ggq64j">https://doi.org/ggq64j</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.06.20031856">10.1101/2020.03.06.20031856</a></p>
</div>
<div id="ref-NT3T9a8H">
<p>820. <strong>Monoclonal antibodies for the S2 subunit of spike of SARS-CoV cross-react with the newly-emerged SARS-CoV-2</strong> <br />
Zhiqiang Zheng, Vanessa M. Monteil, Sebastian Maurer-Stroh, Chow Wenn Yew, Carol Leong, Nur Khairiah Mohd-Ismail, Suganya Cheyyatraivendran Arularasu, Vincent Tak Kwong Chow, Raymond Lin Tzer Pin, Ali Mirazimi, … Yee-Joo Tan<br />
<em>Cold Spring Harbor Laboratory</em> (2020-03-07) <a href="https://doi.org/ggrbj7">https://doi.org/ggrbj7</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.06.980037">10.1101/2020.03.06.980037</a></p>
</div>
<div id="ref-1GEbBtBgh">
<p>821. <strong>Mortality of COVID-19 is Associated with Cellular Immune Function Compared to Immune Function in Chinese Han Population</strong> <br />
Qiang Zeng, Yong-zhe Li, Gang Huang, Wei Wu, Sheng-yong Dong, Yang Xu<br />
<em>medRxiv</em> (2020-03-13) <a href="https://doi.org/ggq64k">https://doi.org/ggq64k</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.08.20031229">10.1101/2020.03.08.20031229</a></p>
</div>
<div id="ref-WuHcOc9T">
<p>822. <strong>Retrospective Analysis of Clinical Features in 101 Death Cases with COVID-19</strong> <br />
JIan Chen, Hua Fan, Lin Zhang, Bin Huang, Muxin Zhu, Yong Zhou, WenHu Yu, Liping Zhu, Shaohui Cheng, Xiaogen Tao, Huan Zhang<br />
<em>Cold Spring Harbor Laboratory</em> (2020-03-17) <a href="https://doi.org/ggq64n">https://doi.org/ggq64n</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.09.20033068">10.1101/2020.03.09.20033068</a></p>
</div>
<div id="ref-1DbV444Pj">
<p>823. <strong>Relationship between the ABO Blood Group and the COVID-19 Susceptibility</strong> <br />
Jiao Zhao, Yan Yang, Hanping Huang, Dong Li, Dongfeng Gu, Xiangfeng Lu, Zheng Zhang, Lei Liu, Ting Liu, Yukun Liu, … Peng George Wang<br />
<em>Cold Spring Harbor Laboratory</em> (2020-03-27) <a href="https://doi.org/ggpn3d">https://doi.org/ggpn3d</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.11.20031096">10.1101/2020.03.11.20031096</a></p>
</div>
<div id="ref-JwL9G6bd">
<p>824. <strong>The inhaled corticosteroid ciclesonide blocks coronavirus RNA replication by targeting viral NSP15</strong> <br />
Shutoku Matsuyama, Miyuki Kawase, Naganori Nao, Kazuya Shirato, Makoto Ujike, Wataru Kamitani, Masayuki Shimojima, Shuetsu Fukushi<br />
<em>bioRxiv</em> (2020-03-12) <a href="https://doi.org/ggq64p">https://doi.org/ggq64p</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.11.987016">10.1101/2020.03.11.987016</a></p>
</div>
<div id="ref-1CwCTXPqj">
<p>825. <strong>Immune phenotyping based on neutrophil-to-lymphocyte ratio and IgG predicts disease severity and outcome for patients with COVID-19</strong> <br />
Bicheng Zhang, Xiaoyang Zhou, Chengliang Zhu, Fan Feng, Yanru Qiu, Jia Feng, Qingzhu Jia, Qibin Song, Bo Zhu, Jun Wang<br />
<em>medRxiv</em> (2020-03-16) <a href="https://doi.org/ggq64q">https://doi.org/ggq64q</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.12.20035048">10.1101/2020.03.12.20035048</a></p>
</div>
<div id="ref-3XqFhi8l">
<p>826. <strong>Reinfection could not occur in SARS-CoV-2 infected rhesus macaques</strong> <br />
Linlin Bao, Wei Deng, Hong Gao, Chong Xiao, Jiayi Liu, Jing Xue, Qi Lv, Jiangning Liu, Pin Yu, Yanfeng Xu, … Chuan Qin<br />
<em>bioRxiv</em> (2020-03-14) <a href="https://doi.org/ggn8r8">https://doi.org/ggn8r8</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.13.990226">10.1101/2020.03.13.990226</a></p>
</div>
<div id="ref-6gQxRkWM">
<p>827. <strong>A highly conserved cryptic epitope in the receptor-binding domains of SARS-CoV-2 and SARS-CoV</strong> <br />
Meng Yuan, Nicholas C. Wu, Xueyong Zhu, Chang-Chun D. Lee, Ray T. Y. So, Huibin Lv, Chris K. P. Mok, Ian A. Wilson<br />
<em>Cold Spring Harbor Laboratory</em> (2020-03-14) <a href="https://doi.org/ggq64s">https://doi.org/ggq64s</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.13.991570">10.1101/2020.03.13.991570</a></p>
</div>
<div id="ref-14y8UvAbH">
<p>828. <strong>SARS-CoV-2 invades host cells via a novel route: CD147-spike protein</strong> <br />
Ke Wang, Wei Chen, Yu-Sen Zhou, Jian-Qi Lian, Zheng Zhang, Peng Du, Li Gong, Yang Zhang, Hong-Yong Cui, Jie-Jie Geng, … Zhi-Nan Chen<br />
<em>bioRxiv</em> (2020-03-14) <a href="https://doi.org/ggq64t">https://doi.org/ggq64t</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.14.988345">10.1101/2020.03.14.988345</a></p>
</div>
<div id="ref-BcOf65BE">
<p>829. <strong>CD147 (EMMPRIN/Basigin) in kidney diseases: from an inflammation and immune system viewpoint</strong> <br />
Tomoki Kosugi, Kayaho Maeda, Waichi Sato, Shoichi Maruyama, Kenji Kadomatsu<br />
<em>Nephrology Dialysis Transplantation</em> (2015-07) <a href="https://doi.org/ggq624">https://doi.org/ggq624</a> <br />
DOI: <a href="https://doi.org/10.1093/ndt/gfu302">10.1093/ndt/gfu302</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25248362">25248362</a></p>
</div>
<div id="ref-34nQJalJ">
<p>830. <strong>The roles of CyPA and CD147 in cardiac remodelling</strong> <br />
Hongyan Su, Yi Yang<br />
<em>Experimental and Molecular Pathology</em> (2018-06) <a href="https://doi.org/ggq622">https://doi.org/ggq622</a> <br />
DOI: <a href="https://doi.org/10.1016/j.yexmp.2018.05.001">10.1016/j.yexmp.2018.05.001</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29772453">29772453</a></p>
</div>
<div id="ref-KcThTnsm">
<p>831. <strong>Cancer-related issues of CD147.</strong> <br />
Ulrich H Weidle, Werner Scheuer, Daniela Eggle, Stefan Klostermann, Hannes Stockinger<br />
<em>Cancer genomics &amp; proteomics</em> <a href="https://www.ncbi.nlm.nih.gov/pubmed/20551248">https://www.ncbi.nlm.nih.gov/pubmed/20551248</a> <br />
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20551248">20551248</a></p>
</div>
<div id="ref-18l4l0nLX">
<p>832. <strong>Blood single cell immune profiling reveals the interferon-MAPK pathway mediated adaptive immune response for COVID-19</strong> <br />
Lulin Huang, Yi Shi, Bo Gong, Li Jiang, Xiaoqi Liu, Jialiang Yang, Juan Tang, Chunfang You, Qi Jiang, Bo Long, … Zhenglin Yang<br />
<em>Cold Spring Harbor Laboratory</em> (2020-03-17) <a href="https://doi.org/ggq64v">https://doi.org/ggq64v</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.15.20033472">10.1101/2020.03.15.20033472</a></p>
</div>
<div id="ref-dM1rOHmH">
<p>833. <strong>Cross-reactive antibody response between SARS-CoV-2 and SARS-CoV infections</strong> <br />
Huibin Lv, Nicholas C. Wu, Owen Tak-Yin Tsang, Meng Yuan, Ranawaka A. P. M. Perera, Wai Shing Leung, Ray T. Y. So, Jacky Man Chun Chan, Garrick K. Yip, Thomas Shiu Hong Chik, … Chris K. P. Mok<br />
<em>Cold Spring Harbor Laboratory</em> (2020-03-17) <a href="https://doi.org/ggq64w">https://doi.org/ggq64w</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.15.993097">10.1101/2020.03.15.993097</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32511317">32511317</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7239046">PMC7239046</a></p>
</div>
<div id="ref-UENoLkhz">
<p>834. <strong>The feasibility of convalescent plasma therapy in severe COVID-19 patients: a pilot study</strong> <br />
Kai Duan, Bende Liu, Cesheng Li, Huajun Zhang, Ting Yu, Jieming Qu, Min Zhou, Li Chen, Shengli Meng, Yong Hu, … Xiaoming Yang<br />
<em>Cold Spring Harbor Laboratory</em> (2020-03-23) <a href="https://doi.org/dqrs">https://doi.org/dqrs</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.16.20036145">10.1101/2020.03.16.20036145</a></p>
</div>
<div id="ref-Xbkby2Jr">
<p>835. <strong>Hydroxychloroquine and Azithromycin as a treatment of COVID-19: preliminary results of an open-label non-randomized clinical trial</strong> <br />
Philippe GAUTRET, Jean Christophe LAGIER, Philippe PAROLA, Van Thuan HOANG, Line MEDDED, Morgan MAILHE, Barbara DOUDIER, Johan COURJON, Valerie GIORDANENGO, Vera ESTEVES VIEIRA, … Didier RAOULT<br />
<em>medRxiv</em> (2020-03-20) <a href="https://doi.org/dqbv">https://doi.org/dqbv</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.16.20037135">10.1101/2020.03.16.20037135</a></p>
</div>
<div id="ref-BIlqPh6M">
<p>836. <strong>Chloroquine: Modes of action of an undervalued drug</strong> <br />
Rodolfo Thomé, Stefanie Costa Pinto Lopes, Fabio Trindade Maranhão Costa, Liana Verinaud<br />
<em>Immunology Letters</em> (2013-06) <a href="https://doi.org/f5b5cr">https://doi.org/f5b5cr</a> <br />
DOI: <a href="https://doi.org/10.1016/j.imlet.2013.07.004">10.1016/j.imlet.2013.07.004</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23891850">23891850</a></p>
</div>
<div id="ref-Q9tXjG1R">
<p>837. <strong>Patients with LRBA deficiency show CTLA4 loss and immune dysregulation responsive to abatacept therapy</strong> <br />
B. Lo, K. Zhang, W. Lu, L. Zheng, Q. Zhang, C. Kanellopoulou, Y. Zhang, Z. Liu, J. M. Fritz, R. Marsh, … M. B. Jordan<br />
<em>Science</em> (2015-07-23) <a href="https://doi.org/f7kc8d">https://doi.org/f7kc8d</a> <br />
DOI: <a href="https://doi.org/10.1126/science.aaa1663">10.1126/science.aaa1663</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26206937">26206937</a></p>
</div>
<div id="ref-3VPi509z">
<p>838. <strong>The sequence of human ACE2 is suboptimal for binding the S spike protein of SARS coronavirus 2</strong> <br />
Erik Procko<br />
<em>Cold Spring Harbor Laboratory</em> (2020-05-11) <a href="https://doi.org/ggrbj8">https://doi.org/ggrbj8</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.16.994236">10.1101/2020.03.16.994236</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32511321">32511321</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7239051">PMC7239051</a></p>
</div>
<div id="ref-1EcOLnVXk">
<p>839. <strong>Comparative Pathogenesis Of COVID-19, MERS And SARS In A Non-Human Primate Model</strong> <br />
Barry Rockx, Thijs Kuiken, Sander Herfst, Theo Bestebroer, Mart M. Lamers, Dennis de Meulder, Geert van Amerongen, Judith van den Brand, Nisreen M. A. Okba, Debby Schipper, … Bart L. Haagmans<br />
<em>bioRxiv</em> (2020-03-17) <a href="https://doi.org/ggq649">https://doi.org/ggq649</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.17.995639">10.1101/2020.03.17.995639</a></p>
</div>
<div id="ref-NzFgXYvQ">
<p>840. <strong>Lethal Infection of K18-hACE2 Mice Infected with Severe Acute Respiratory Syndrome Coronavirus</strong> <br />
P. B. McCray, L. Pewe, C. Wohlford-Lenane, M. Hickey, L. Manzel, L. Shi, J. Netland, H. P. Jia, C. Halabi, C. D. Sigmund, … S. Perlman<br />
<em>Journal of Virology</em> (2006-11-01) <a href="https://doi.org/b2dr3s">https://doi.org/b2dr3s</a> <br />
DOI: <a href="https://doi.org/10.1128/jvi.02012-06">10.1128/jvi.02012-06</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17079315">17079315</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1797474">PMC1797474</a></p>
</div>
<div id="ref-OvFueDLZ">
<p>841. <strong>A Model Describing COVID-19 Community Transmission Taking into Account Asymptomatic Carriers and Risk Mitigation</strong> <br />
Jacob B Aguilar, Jeremy Samuel Faust, Lauren M. Westafer, Juan B. Gutierrez<br />
<em>Cold Spring Harbor Laboratory</em> (2020-08-11) <a href="https://doi.org/ggqnvp">https://doi.org/ggqnvp</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.18.20037994">10.1101/2020.03.18.20037994</a></p>
</div>
<div id="ref-Y9g1sQ1Y">
<p>842. <strong>Antibody responses to SARS-CoV-2 in COVID-19 patients: the perspective application of serological tests in clinical practice</strong> <br />
Quan-xin Long, Hai-jun Deng, Juan Chen, Jieli Hu, Bei-zhong Liu, Pu Liao, Yong Lin, Li-hua Yu, Zhan Mo, Yin-yin Xu, … Ai-long Huang<br />
<em>Cold Spring Harbor Laboratory</em> (2020-03-20) <a href="https://doi.org/ggpvz3">https://doi.org/ggpvz3</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.18.20038018">10.1101/2020.03.18.20038018</a></p>
</div>
<div id="ref-p1c9Nilh">
<p>843. <strong>Heat inactivation of serum interferes with the immunoanalysis of antibodies to SARS-CoV-2</strong> <br />
Xiumei Hu, Taixue An, Bo Situ, Yuhai Hu, Zihao Ou, Qiang Li, Xiaojing He, Ye Zhang, Peifu Tian, Dehua Sun, … Lei Zheng<br />
<em>Cold Spring Harbor Laboratory</em> (2020-03-16) <a href="https://doi.org/ggq646">https://doi.org/ggq646</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.12.20034231">10.1101/2020.03.12.20034231</a></p>
</div>
<div id="ref-1BLNvgEao">
<p>844. <strong>SARS-CoV-2 specific antibody responses in COVID-19 patients</strong> <br />
NISREEN M. A. OKBA, Marcel A Muller, Wentao Li, Chunyan Wang, Corine H. GeurtsvanKessel, Victor M. Corman, Mart M. Lamers, Reina S. Sikkema, Erwin de Bruin, Felicity D. Chandler, … Bart L. Haagmans<br />
<em>medRxiv</em> (2020-03-20) <a href="https://doi.org/ggpvz2">https://doi.org/ggpvz2</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.18.20038059">10.1101/2020.03.18.20038059</a></p>
</div>
<div id="ref-K41YFM2N">
<p>845. <strong>A brief review of antiviral drugs evaluated in registered clinical trials for COVID-19</strong> <br />
Drifa Belhadi, Nathan Peiffer-Smadja, François-Xavier Lescure, Yazdan Yazdanpanah, France Mentré, Cédric Laouénan<br />
<em>Cold Spring Harbor Laboratory</em> (2020-03-27) <a href="https://doi.org/ggq65b">https://doi.org/ggq65b</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.18.20038190">10.1101/2020.03.18.20038190</a></p>
</div>
<div id="ref-eTsOvBVf">
<p>846. <strong>ACE-2 Expression in the Small Airway Epithelia of Smokers and COPD Patients: Implications for COVID-19</strong> <br />
Janice M Leung, Chen Xi Yang, Anthony Tam, Tawimas Shaipanich, Tillie L Hackett, Gurpreet K Singhera, Delbert R Dorscheid, Don D Sin<br />
<em>Cold Spring Harbor Laboratory</em> (2020-03-23) <a href="https://doi.org/dqx2">https://doi.org/dqx2</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.18.20038455">10.1101/2020.03.18.20038455</a></p>
</div>
<div id="ref-18KZGwJ0J">
<p>847. <strong>Dynamic profile of severe or critical COVID-19 cases</strong> <br />
Yang Xu<br />
<em>Cold Spring Harbor Laboratory</em> (2020-03-20) <a href="https://doi.org/ggrbj9">https://doi.org/ggrbj9</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.18.20038513">10.1101/2020.03.18.20038513</a></p>
</div>
<div id="ref-1BJXDZ04c">
<p>848. <strong>Association between Clinical, Laboratory and CT Characteristics and RT-PCR Results in the Follow-up of COVID-19 patients</strong> <br />
Hang Fu, Huayan Xu, Na Zhang, Hong Xu, Zhenlin Li, Huizhu Chen, Rong Xu, Ran Sun, Lingyi Wen, Linjun Xie, … Yingkun Guo<br />
<em>Cold Spring Harbor Laboratory</em> (2020-03-23) <a href="https://doi.org/ggq65c">https://doi.org/ggq65c</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.19.20038315">10.1101/2020.03.19.20038315</a></p>
</div>
<div id="ref-iMCz5Twj">
<p>849. <strong>An orally bioavailable broad-spectrum antiviral inhibits SARS-CoV-2 and multiple endemic, epidemic and bat coronavirus</strong> <br />
Timothy P. Sheahan, Amy C. Sims, Shuntai Zhou, Rachel L. Graham, Collin S. Hill, Sarah R. Leist, Alexandra Schäfer, Kenneth H. Dinnon, Stephanie A. Montgomery, Maria L. Agostini, … Ralph S. Baric<br />
<em>Cold Spring Harbor Laboratory</em> (2020-03-20) <a href="https://doi.org/ggrbkb">https://doi.org/ggrbkb</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.19.997890">10.1101/2020.03.19.997890</a></p>
</div>
<div id="ref-ZDP9vSpg">
<p>850. <strong>Identification of antiviral drug candidates against SARS-CoV-2 from FDA-approved drugs</strong> <br />
Sangeun Jeon, Meehyun Ko, Jihye Lee, Inhee Choi, Soo Young Byun, Soonju Park, David Shum, Seungtaek Kim<br />
<em>bioRxiv</em> (2020-03-28) <a href="https://doi.org/ggq65h">https://doi.org/ggq65h</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.20.999730">10.1101/2020.03.20.999730</a></p>
</div>
<div id="ref-LYFzvlzE">
<p>851. <strong>Respiratory disease and virus shedding in rhesus macaques inoculated with SARS-CoV-2</strong> <br />
Vincent J. Munster, Friederike Feldmann, Brandi N. Williamson, Neeltje van Doremalen, Lizzette Pérez-Pérez, Jonathan Schulz, Kimberly Meade-White, Atsushi Okumura, Julie Callison, Beniah Brumbaugh, … Emmie de Wit<br />
<em>bioRxiv</em> (2020-03-21) <a href="https://doi.org/ggq65j">https://doi.org/ggq65j</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.21.001628">10.1101/2020.03.21.001628</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32511299">32511299</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7217148">PMC7217148</a></p>
</div>
<div id="ref-unVgVIfU">
<p>852. <strong>Ocular conjunctival inoculation of SARS-CoV-2 can cause mild COVID-19 in Rhesus macaques</strong> <br />
Wei Deng, Linlin Bao, Hong Gao, Zhiguang Xiang, Yajin Qu, Zhiqi Song, Shunran Gong, Jiayi Liu, Jiangning Liu, Pin Yu, … Chuan Qin<br />
<em>Cold Spring Harbor Laboratory</em> (2020-03-30) <a href="https://doi.org/ggq64r">https://doi.org/ggq64r</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.13.990036">10.1101/2020.03.13.990036</a></p>
</div>
<div id="ref-wjtVy2v0">
<p>853. <strong>ACE2 Expression is Increased in the Lungs of Patients with Comorbidities Associated with Severe COVID-19</strong> <br />
Bruna GG Pinto, Antonio ER Oliveira, Youvika Singh, Leandro Jimenez, Andre NA Goncalves, Rodrigo LT Ogava, Rachel Creighton, Jean PS Peron, Helder I Nakaya<br />
<em>Cold Spring Harbor Laboratory</em> (2020-03-27) <a href="https://doi.org/ggq65k">https://doi.org/ggq65k</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.21.20040261">10.1101/2020.03.21.20040261</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32511627">32511627</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7276054">PMC7276054</a></p>
</div>
<div id="ref-DFhY5nfb">
<p>854. <strong>Meplazumab treats COVID-19 pneumonia: an open-labelled, concurrent controlled add-on clinical trial</strong> <br />
Huijie Bian, Zhao-Hui Zheng, Ding Wei, Zheng Zhang, Wen-Zhen Kang, Chun-Qiu Hao, Ke Dong, Wen Kang, Jie-Lai Xia, Jin-Lin Miao, … Ping Zhu<br />
<em>Cold Spring Harbor Laboratory</em> (2020-09-27) <a href="https://doi.org/ggq65m">https://doi.org/ggq65m</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.21.20040691">10.1101/2020.03.21.20040691</a></p>
</div>
<div id="ref-yP8TXdq2">
<p>855. <strong>CD147 facilitates HIV-1 infection by interacting with virus-associated cyclophilin A</strong> <br />
T. Pushkarsky, G. Zybarth, L. Dubrovsky, V. Yurchenko, H. Tang, H. Guo, B. Toole, B. Sherry, M. Bukrinsky<br />
<em>Proceedings of the National Academy of Sciences</em> (2001-05-15) <a href="https://doi.org/cc4c7p">https://doi.org/cc4c7p</a> <br />
DOI: <a href="https://doi.org/10.1073/pnas.111583198">10.1073/pnas.111583198</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/11353871">11353871</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC33473">PMC33473</a></p>
</div>
<div id="ref-X8l1n4zY">
<p>856. <strong>CD147/EMMPRIN Acts as a Functional Entry Receptor for Measles Virus on Epithelial Cells</strong> <br />
Akira Watanabe, Misako Yoneda, Fusako Ikeda, Yuri Terao-Muto, Hiroki Sato, Chieko Kai<br />
<em>Journal of Virology</em> (2010-05-01) <a href="https://doi.org/dpcsqg">https://doi.org/dpcsqg</a> <br />
DOI: <a href="https://doi.org/10.1128/jvi.02168-09">10.1128/jvi.02168-09</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20147391">20147391</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2863760">PMC2863760</a></p>
</div>
<div id="ref-YcyNaIVC">
<p>857. <strong>Basigin is a receptor essential for erythrocyte invasion by Plasmodium falciparum</strong> <br />
Cécile Crosnier, Leyla Y. Bustamante, S. Josefin Bartholdson, Amy K. Bei, Michel Theron, Makoto Uchikawa, Souleymane Mboup, Omar Ndir, Dominic P. Kwiatkowski, Manoj T. Duraisingh, … Gavin J. Wright<br />
<em>Nature</em> (2011-11-09) <a href="https://doi.org/dm59hf">https://doi.org/dm59hf</a> <br />
DOI: <a href="https://doi.org/10.1038/nature10606">10.1038/nature10606</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22080952">22080952</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3245779">PMC3245779</a></p>
</div>
<div id="ref-m9yLJ5Da">
<p>858. <strong>Function of HAb18G/CD147 in Invasion of Host Cells by Severe Acute Respiratory Syndrome Coronavirus</strong> <br />
Zhinan Chen, Li Mi, Jing Xu, Jiyun Yu, Xianhui Wang, Jianli Jiang, Jinliang Xing, Peng Shang, Airong Qian, Yu Li, … Ping Zhu<br />
<em>The Journal of Infectious Diseases</em> (2005-03) <a href="https://doi.org/cd8snd">https://doi.org/cd8snd</a> <br />
DOI: <a href="https://doi.org/10.1086/427811">10.1086/427811</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15688292">15688292</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7110046">PMC7110046</a></p>
</div>
<div id="ref-J0ibHE3l">
<p>859. <strong>CD147 mediates intrahepatic leukocyte aggregation and determines the extent of liver injury</strong> <br />
Christine Yee, Nathan M. Main, Alexandra Terry, Igor Stevanovski, Annette Maczurek, Alison J. Morgan, Sarah Calabro, Alison J. Potter, Tina L. Iemma, David G. Bowen, … Nicholas A. Shackel<br />
<em>PLOS ONE</em> (2019-07-10) <a href="https://doi.org/ggq654">https://doi.org/ggq654</a> <br />
DOI: <a href="https://doi.org/10.1371/journal.pone.0215557">10.1371/journal.pone.0215557</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31291257">31291257</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6619953">PMC6619953</a></p>
</div>
<div id="ref-bCYYaFKM">
<p>860. <strong>Characterisation of the transcriptome and proteome of SARS-CoV-2 using direct RNA sequencing and tandem mass spectrometry reveals evidence for a cell passage induced in-frame deletion in the spike glycoprotein that removes the furin-like cleavage site</strong> <br />
Andrew D. Davidson, Maia Kavanagh Williamson, Sebastian Lewis, Deborah Shoemark, Miles W. Carroll, Kate Heesom, Maria Zambon, Joanna Ellis, Phillip A. Lewis, Julian A. Hiscox, David A. Matthews<br />
<em>Cold Spring Harbor Laboratory</em> (2020-03-24) <a href="https://doi.org/ggq65n">https://doi.org/ggq65n</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.22.002204">10.1101/2020.03.22.002204</a></p>
</div>
<div id="ref-16tHVjVzu">
<p>861. <strong>Modifications to the Hemagglutinin Cleavage Site Control the Virulence of a Neurotropic H1N1 Influenza Virus</strong> <br />
X. Sun, L. V. Tse, A. D. Ferguson, G. R. Whittaker<br />
<em>Journal of Virology</em> (2010-06-16) <a href="https://doi.org/drs2zt">https://doi.org/drs2zt</a> <br />
DOI: <a href="https://doi.org/10.1128/jvi.00797-10">10.1128/jvi.00797-10</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20554779">20554779</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2919019">PMC2919019</a></p>
</div>
<div id="ref-hRVfFbS">
<p>862. <strong>The architecture of SARS-CoV-2 transcriptome</strong> <br />
Dongwan Kim, Joo-Yeon Lee, Jeong-Sun Yang, Jun Won Kim, V. Narry Kim, Hyeshik Chang<br />
<em>Cold Spring Harbor Laboratory</em> (2020-03-14) <a href="https://doi.org/ggpx9q">https://doi.org/ggpx9q</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.12.988865">10.1101/2020.03.12.988865</a></p>
</div>
<div id="ref-RTIk8DTo">
<p>863. <strong>First Clinical Study Using HCV Protease Inhibitor Danoprevir to Treat Naive and Experienced COVID-19 Patients</strong> <br />
Hongyi Chen, Zhicheng Zhang, Li Wang, Zhihua Huang, Fanghua Gong, Xiaodong Li, Yahong Chen, Jinzi J. WU<br />
<em>Cold Spring Harbor Laboratory</em> (2020-03-24) <a href="https://doi.org/ggrgbt">https://doi.org/ggrgbt</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.22.20034041">10.1101/2020.03.22.20034041</a></p>
</div>
<div id="ref-YoCPtq6P">
<p>864. <strong>Preclinical Characteristics of the Hepatitis C Virus NS3/4A Protease Inhibitor ITMN-191 (R7227)</strong> <br />
S. D. Seiwert, S. W. Andrews, Y. Jiang, V. Serebryany, H. Tan, K. Kossen, P. T. R. Rajagopalan, S. Misialek, S. K. Stevens, A. Stoycheva, … L. M. Blatt<br />
<em>Antimicrobial Agents and Chemotherapy</em> (2008-09-29) <a href="https://doi.org/btpg52">https://doi.org/btpg52</a> <br />
DOI: <a href="https://doi.org/10.1128/aac.00699-08">10.1128/aac.00699-08</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/18824605">18824605</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2592891">PMC2592891</a></p>
</div>
<div id="ref-10GnhBctk">
<p>865. <strong>Efficacy and Safety of All-oral, 12-week Ravidasvir Plus Ritonavir-boosted Danoprevir and Ribavirin in Treatment-naïve Noncirrhotic HCV Genotype 1 Patients: Results from a Phase 2/3 Clinical Trial in China</strong> <br />
Xiaoyuan Xu, Bo Feng, Yujuan Guan, Sujun Zheng, Jifang Sheng, Xingxiang Yang, Yuanji Ma, Yan Huang, Yi Kang, Xiaofeng Wen, … Lai Wei<br />
<em>Journal of Clinical and Translational Hepatology</em> (2019-09-30) <a href="https://doi.org/ggrbkd">https://doi.org/ggrbkd</a> <br />
DOI: <a href="https://doi.org/10.14218/jcth.2019.00033">10.14218/jcth.2019.00033</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31608212">31608212</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6783683">PMC6783683</a></p>
</div>
<div id="ref-7zBwK1wn">
<p>866. <strong>Potentially highly potent drugs for 2019-nCoV</strong> <br />
Duc Duy Nguyen, Kaifu Gao, Jiahui Chen, Rui Wang, Guo-Wei Wei<br />
<em>Cold Spring Harbor Laboratory</em> (2020-02-13) <a href="https://doi.org/ggrbj5">https://doi.org/ggrbj5</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.02.05.936013">10.1101/2020.02.05.936013</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32511344">32511344</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7255774">PMC7255774</a></p>
</div>
<div id="ref-lnuNSHEW">
<p>867. <strong>Serology characteristics of SARS-CoV-2 infection since the exposure and post symptoms onset</strong> <br />
Bin Lou, Tigndong Li, Shufa Zheng, Yingying Su, Zhiyong Li, Wei Liu, Fei Yu, Shengxiang Ge, Qianda Zou, Quan Yuan, … Yu Chen<br />
<em>Cold Spring Harbor Laboratory</em> (2020-03-27) <a href="https://doi.org/ggrbkc">https://doi.org/ggrbkc</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.23.20041707">10.1101/2020.03.23.20041707</a></p>
</div>
<div id="ref-aNSdARwe">
<p>868. <strong>SARS-CoV-2 launches a unique transcriptional signature from in vitro, ex vivo, and in vivo systems</strong> <br />
Daniel Blanco-Melo, Benjamin E. Nilsson-Payant, Wen-Chun Liu, Rasmus Møller, Maryline Panis, David Sachs, Randy A. Albrecht, Benjamin R. tenOever<br />
<em>Cold Spring Harbor Laboratory</em> (2020-03-24) <a href="https://doi.org/ggq65q">https://doi.org/ggq65q</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.24.004655">10.1101/2020.03.24.004655</a></p>
</div>
<div id="ref-EWZmuLvu">
<p>869. <strong>A New Predictor of Disease Severity in Patients with COVID-19 in Wuhan, China</strong> <br />
Ying Zhou, Zhen Yang, Yanan Guo, Shuang Geng, Shan Gao, Shenglan Ye, Yi Hu, Yafei Wang<br />
<em>Cold Spring Harbor Laboratory</em> (2020-03-27) <a href="https://doi.org/ggq65r">https://doi.org/ggq65r</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.24.20042119">10.1101/2020.03.24.20042119</a></p>
</div>
<div id="ref-95sQSGzx">
<p>870. <strong>Metabolic disturbances and inflammatory dysfunction predict severity of coronavirus disease 2019 (COVID-19): a retrospective study</strong> <br />
Shuke Nie, Xueqing Zhao, Kang Zhao, Zhaohui Zhang, Zhentao Zhang, Zhan Zhang<br />
<em>Cold Spring Harbor Laboratory</em> (2020-03-26) <a href="https://doi.org/ggq65s">https://doi.org/ggq65s</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.24.20042283">10.1101/2020.03.24.20042283</a></p>
</div>
<div id="ref-btVuoX6V">
<p>871. <strong>Viral Kinetics and Antibody Responses in Patients with COVID-19</strong> <br />
Wenting Tan, Yanqiu Lu, Juan Zhang, Jing Wang, Yunjie Dan, Zhaoxia Tan, Xiaoqing He, Chunfang Qian, Qiangzhong Sun, Qingli Hu, … Guohong Deng<br />
<em>medRxiv</em> (2020-03-26) <a href="https://doi.org/ggq65t">https://doi.org/ggq65t</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.24.20042382">10.1101/2020.03.24.20042382</a></p>
</div>
<div id="ref-LIf5SSUs">
<p>872. <strong>Global profiling of SARS-CoV-2 specific IgG/ IgM responses of convalescents using a proteome microarray</strong> <br />
He-wei Jiang, Yang Li, Hai-nan Zhang, Wei Wang, Dong Men, Xiao Yang, Huan Qi, Jie Zhou, Sheng-ce Tao<br />
<em>Cold Spring Harbor Laboratory</em> (2020-03-27) <a href="https://doi.org/ggq65g">https://doi.org/ggq65g</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.20.20039495">10.1101/2020.03.20.20039495</a></p>
</div>
<div id="ref-1FZgghazg">
<p>873. <strong>COVID-19 infection induces readily detectable morphological and inflammation-related phenotypic changes in peripheral blood monocytes, the severity of which correlate with patient outcome</strong> <br />
Dan Zhang, Rui Guo, Lei Lei, Hongjuan Liu, Yawen Wang, Yili Wang, Tongxin Dai, Tianxiao Zhang, Yanjun Lai, Jingya Wang, … Jinsong Hu<br />
<em>Cold Spring Harbor Laboratory</em> (2020-03-26) <a href="https://doi.org/ggq65v">https://doi.org/ggq65v</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.24.20042655">10.1101/2020.03.24.20042655</a></p>
</div>
<div id="ref-15gdkEnfn">
<p>874. <strong>Correlation between universal BCG vaccination policy and reduced morbidity and mortality for COVID-19: an epidemiological study</strong> <br />
Aaron Miller, Mac Josh Reandelar, Kimberly Fasciglione, Violeta Roumenova, Yan Li, Gonzalo H Otazu<br />
<em>medRxiv</em> (2020-03-28) <a href="https://doi.org/ggq65w">https://doi.org/ggq65w</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.24.20042937">10.1101/2020.03.24.20042937</a></p>
</div>
<div id="ref-16FOT89K5">
<p>875. <strong>Non-specific effects of BCG vaccine on viral infections</strong> <br />
S. J. C. F. M. Moorlag, R. J. W. Arts, R. van Crevel, M. G. Netea<br />
<em>Clinical Microbiology and Infection</em> (2019-12) <a href="https://doi.org/ggq62z">https://doi.org/ggq62z</a> <br />
DOI: <a href="https://doi.org/10.1016/j.cmi.2019.04.020">10.1016/j.cmi.2019.04.020</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31055165">31055165</a></p>
</div>
<div id="ref-MCuIRGFE">
<p>876. <strong>BCG vaccination to reduce the impact of COVID-19 in healthcare workers (The BRACE Trial)</strong> <br />
Murdoch Children’s Research Institute<br />
<a href="https://www.mcri.edu.au/BRACE">https://www.mcri.edu.au/BRACE</a></p>
</div>
<div id="ref-12oDb7aki">
<p>877. <strong>Non-neural expression of SARS-CoV-2 entry genes in the olfactory epithelium suggests mechanisms underlying anosmia in COVID-19 patients</strong> <br />
David H. Brann, Tatsuya Tsukahara, Caleb Weinreb, Darren W. Logan, Sandeep Robert Datta<br />
<em>bioRxiv</em> (2020-03-28) <a href="https://doi.org/ggqr4m">https://doi.org/ggqr4m</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.25.009084">10.1101/2020.03.25.009084</a></p>
</div>
<div id="ref-t4vofknR">
<p>878. <strong>Cigarette smoke triggers the expansion of a subpopulation of respiratory epithelial cells that express the SARS-CoV-2 receptor ACE2</strong> <br />
Joan C Smith, Jason Meyer Sheltzer<br />
<em>bioRxiv</em> (2020-03-31) <a href="https://doi.org/ggq65x">https://doi.org/ggq65x</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.28.013672">10.1101/2020.03.28.013672</a></p>
</div>
<div id="ref-k1E3DuFN">
<p>879. <strong>The comparative superiority of IgM-IgG antibody test to real-time reverse transcriptase PCR detection for SARS-CoV-2 infection diagnosis</strong> <br />
Rui Liu, Xinghui Liu, Huan Han, Muhammad Adnan Shereen, Zhili Niu, Dong Li, Fang Liu, Kailang Wu, Zhen Luo, Chengliang Zhu<br />
<em>Cold Spring Harbor Laboratory</em> (2020-03-30) <a href="https://doi.org/ggqtp5">https://doi.org/ggqtp5</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.28.20045765">10.1101/2020.03.28.20045765</a></p>
</div>
</div>
<!-- Define citation tags below -->
<h1 class="page_break_before" data-number="10" id="appendix-1"><span class="header-section-number">10</span> Appendix 1</h1>
<p>This appendix contains reviews produced by the Immunology Institute of the Icahn School of Medicine</p>
<!-- Appendix source: https://github.com/ismms-himc/covid-19_sinai_reviews/tree/05d387bf10b829dad6bedb4bf313686750b78cc9 -->
<h2 data-number="10.1" id="potent-binding-of-2019-novel-coronavirus-spike-protein-by-a-sars-coronavirus-specific-human-monoclonal-antibody"><span class="header-section-number">10.1</span> Potent binding of 2019 novel coronavirus spike protein by a SARS coronavirus-specific human monoclonal antibody</h2>
<p>Tian et al. <em>Emerg Microbes Infect 2020</em> <span class="citation" data-cites="1HLSeqnY8">[<a href="#ref-1HLSeqnY8" role="doc-biblioref">757</a>]</span></p>
<h3 data-number="10.1.1" id="keywords"><span class="header-section-number">10.1.1</span> Keywords</h3>
<ul>
<li>Monoclonal antibody</li>
<li>Cross-reactivity</li>
<li>receptor binding domain</li>
</ul>
<h3 data-number="10.1.2" id="summary-1"><span class="header-section-number">10.1.2</span> Summary</h3>
<p>Considering the relatively high identity of the receptor binding domain
(RBD) of the spike proteins from 2019-nCoV and SARS-CoV (73%), this
study aims to assess the cross-reactivity of several anti-SARS-CoV
monoclonal antibodies with 2019-nCoV. The results showed that the
SARS-CoV-specific antibody CR3022 can potently bind 2019-nCoV RBD.</p>
<h3 data-number="10.1.3" id="main-findings"><span class="header-section-number">10.1.3</span> Main Findings</h3>
<p>The structure of the 2019-nCoV spike RBD and its conformation in complex
with the receptor angiotensin-converting enzyme (ACE2) was modeled <em>in
silico</em> and compared with the SARS-CoV RBD structure. The models
predicted very similar RBD-ACE2 interactions for both viruses. The
binding capacity of representative SARS-CoV-RBD specific monoclonal
antibodies (m396, CR3014, and CR3022) to recombinant 2019-nCoV RBD was
then investigated by ELISA and their binding kinetics studied using
biolayer interferometry. The analysis showed that only CR3022 was able
to bind 2019-nCoV RBD with high affinity (KD of 6.3 nM), however it did
not interfere with ACE2 binding. Antibodies m396 and CR3014, which
target the ACE2 binding site of SARS-CoV failed to bind 2019-nCoV spike
protein.</p>
<h3 data-number="10.1.4" id="limitations"><span class="header-section-number">10.1.4</span> Limitations</h3>
<p>The 2019-nCoV RBD largely differ from the SARS-CoV at the C-terminus
residues, which drastically impact the cross-reactivity of antibodies
described for other B beta-coronaviruses, including SARS-CoV. This study
claims that CR3022 antibody could be a potential candidate for therapy.
However, none of the antibodies assayed in this work showed
cross-reactivity with the ACE2 binding site of 2019-nCoV, essential for
the replication of this virus. Furthermore, neutralization assays with
2019-nCoV virus or pseudovirus were not performed. Although the use of
neutralizing antibodies is an interesting approach, these results
suggest that it is critical the development of novel monoclonal
antibodies able to specifically bind 2019-nCoV spike protein.</p>
<h3 data-number="10.1.5" id="credit"><span class="header-section-number">10.1.5</span> Credit</h3>
<p><em>Review by D.L.O as part of a project by students, postdocs and faculty
at the Immunology Institute of the Icahn School of Medicine, Mount
Sinai.</em></p>
<h2 data-number="10.2" id="integrative-bioinformatics-analysis-provides-insight-into-the-molecular-mechanisms-of-2019-ncov"><span class="header-section-number">10.2</span> Integrative Bioinformatics Analysis Provides Insight into the Molecular Mechanisms of 2019-nCoV</h2>
<p>He et al. <em>medRxiv</em> <span class="citation" data-cites="lOQPScZt">[<a href="#ref-lOQPScZt" role="doc-biblioref">758</a>]</span></p>
<h3 data-number="10.2.1" id="keywords-1"><span class="header-section-number">10.2.1</span> Keywords</h3>
<ul>
<li>ACE2</li>
<li>lungs</li>
<li>smoking</li>
<li>COPD</li>
<li>asthma</li>
<li>SARS-Cov</li>
<li>IL-1</li>
<li>IL-10</li>
<li>IL-6</li>
<li>IL-8</li>
</ul>
<h3 data-number="10.2.2" id="main-findings-1"><span class="header-section-number">10.2.2</span> Main Findings</h3>
<p>The authors used bioinformatics tools to identify features
of ACE2 expression in the lungs of different patent groups: healthy,
smokers, patients with chronic airway disease (i.e., COPD) or asthma.
They used gene expression data publicly available from GEO that included
lung tissues, bronchoalveolar lavage, bronchial epithelial cells, small
airway epithelial cells, or SARS-Cov infected cells.</p>
<p>The authors describe no significant differences in ACE2 expression in
lung tissues of Healthy, COPD, and Asthma groups (p=0.85); or in BAL of
Healthy and COPD (p=0.48); or in epithelial brushings of Healthy and
Mild/Moderate/Severe Asthma (p=0.99). ACE2 was higher in the small
airway epithelium of long-term smokers vs non-smokers (p&lt;0.001).
Consistently, there was a trend of higher ACE2 expression in the
bronchial airway epithelial cells 24h post-acute smoking exposure
(p=0.073). Increasing ACE2 expression at 24h and 48h compared to 12h
post SARS-Cov infection (p=0.026; n=3 at each time point) was also
detected.</p>
<p>15 lung samples’ data from healthy participants were separated into high
and low ACE2 expression groups. “High” ACE2 expression was associated
with the following GO pathways: innate and adaptive immune responses, B
cell mediated immunity, cytokine secretion, and IL-1, IL-10, IL-6, IL-8
cytokines. The authors speculate that a high basal ACE2 expression will
increase susceptibility to SARS-CoV infection.</p>
<p>In 3 samples SARS-Cov infection was associated with IL-1, IL-10 and IL-6
cytokine production (GO pathways) at 24h. And later, at 48h, with T-cell
activation and T-cell cytokine production. It is unclear whether those
changes were statistically significant.</p>
<p>The authors describe a time course quantification of immune infiltrates
in epithelial cells infected with SARS-Cov infection. They state that in
healthy donors ACE2 expression did not correlate with the immune cell
infiltration. However, in SARS-Cov samples, at 48h they found that ACE2
correlated with neutrophils, NK-, Th17-, Th2-, Th1- cells, and DCs.
Again, while authors claim significance, the corresponding correlation
coefficients and p-values are not presented in the text or figures. In
addition, the source of the data for this analysis is not clear.</p>
<p>Using network analysis, proteins SRC, FN1, MAPK3, LYN, MBP, NLRC4, NLRP1
and PRKCD were found to be central (Hub proteins) in the regulating
network of cytokine secretion after coronavirus infection. Authors
conclude this indicates that these molecules were critically important
in ACE2-induced inflammatory response. Additionally, authors speculate
that the increased expression of ACE2 affected RPS3 and SRC, which were
the two hub genes involved in viral replication and inflammatory
response.</p>
<h3 data-number="10.2.3" id="limitations-1"><span class="header-section-number">10.2.3</span> Limitations</h3>
<p>The methods section is very limited and does not
describe any of the statistical analyses; and description of the
construction of the regulatory protein networks is also limited. For the
findings in Figures 2 authors claim significance, which is not supported
by p-values or coefficients. For the sample selection, would be useful
if sample sizes and some of the patients’ demographics (e.g. age) were
described.</p>
<p>For the analysis of high vs low ACE2 expression in healthy subjects, it
is not clear what was the cut off for ‘high’ expression and how it was
determined. Additionally, further laboratory studies are warranted to
confirm that high ACE2 gene expression would have high correlation with
the amount of ACE2 protein on cell surface. For the GO pathway analysis
significance was set at p&lt;0.05, but not adjusted for multiple
comparisons.</p>
<p>There were no samples with SARS-CoV-2 infection. While SARS-Cov and
SARS-CoV-2 both use ACE2 to enter the host cells, the analysis only
included data on SARS-Cov and any conclusions about SARS-CoV2 are
limited.</p>
<p>Upon checking GSE accession numbers of the datasets references, two
might not be cited correctly: GSE37758 (“A spergillus niger: Control
(fructose) vs. steam-exploded sugarcane induction (SEB)”" was used in
this paper as “lung tissue” data) and GSE14700 (“Steroid Pretreatment
of Organ Donors to Prevent Postischemic Renal Allograft Failure: A
Randomized, Controlled Trial” – was used as SARS-Cov infection data).</p>
<h3 data-number="10.2.4" id="credit-1"><span class="header-section-number">10.2.4</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.3" id="diarrhea-may-be-underestimated-a-missing-link-in-2019-novel-coronavirus"><span class="header-section-number">10.3</span> Diarrhea may be underestimated: a missing link in 2019 novel coronavirus</h2>
<p>Liang et al. <em>medRxiv</em> <span class="citation" data-cites="rL9IRhVy">[<a href="#ref-rL9IRhVy" role="doc-biblioref">759</a>]</span></p>
<h3 data-number="10.3.1" id="keywords-2"><span class="header-section-number">10.3.1</span> Keywords</h3>
<ul>
<li>SARS-CoV-2</li>
<li>diarrhea</li>
<li>ACE2</li>
<li>scRNA-seq</li>
</ul>
<h3 data-number="10.3.2" id="main-findings-2"><span class="header-section-number">10.3.2</span> Main Findings</h3>
<p>This study examined the incidence of diarrhea in
patients infected with SARS-CoV-2 across three recently published
cohorts and found that there are statistically significant differences
by Fisher’s exact test. They report that this could be due to subjective
diagnosis criterion for diarrhea or from patients first seeking medical
care from gastroenterologist. In order to minimize nosocomial infections
arising from unsuspected patients with diarrhea and gain comprehensive
understanding of transmission routes for this viral pathogen, they
compared the transcriptional levels of ACE2 of various human tissues
from NCBI public database as well as in small intestine tissue from
CD57BL/6 mice using single cell sequencing. They show that ACE2
expression is not only increased in the human small intestine, but
demonstrate a particular increase in mice enterocytes positioned on the
surface of the intestinal lining exposed to viral pathogens. Given that
ACE2 is the viral receptor for SARS-CoV-2 and also reported to regulate
diarrhea, their data suggests the small intestine as a potential
transmission route and diarrhea as a potentially underestimated symptom
in COVID19 patients that must be carefully monitored. Interestingly,
however, they show that ACE2 expression level is not elevated in human
lung tissue.</p>
<h3 data-number="10.3.3" id="limitations-2"><span class="header-section-number">10.3.3</span> Limitations</h3>
<p>Although this study demonstrates a
statistical difference in the incidence of diarrhea across three
separate COVID19 patient cohorts, their conclusions are limited by a
small sample size. Specifically, the p-value computed by Fisher’s exact
test is based on a single patient cohort of only six cases of which 33%
are reported to have diarrhea, while the remaining two larger cohorts
with 41 and 99 cases report 3% and 2% diarrhea incidence, respectively.
Despite showing significance, they would need to acquire larger sample
sizes and cohorts to minimize random variability and draw meaningful
conclusions. Furthermore, they do not address why ACE2 expression level
is not elevated in human lung tissue despite it being a major
established route of transmission for SARS-CoV-2. It could be helpful to
validate this result by looking at ACE2 expression in mouse lung tissue.
Finally, although this study is descriptive and shows elevated ACE2
expression in small intestinal epithelial cells, it does not establish a
mechanistic link to SARS-CoV-2 infection of the host. Overall, their
claim that infected patients exhibiting diarrhea pose an increased risk
to hospital staff needs to be further substantiated.</p>
<h3 data-number="10.3.4" id="significance"><span class="header-section-number">10.3.4</span> Significance</h3>
<p>This study provides a possible transmission route and a
potentially underappreciated clinical symptom for SARS-CoV-2 for better
clinical management and control of COVID19.</p>
<h3 data-number="10.3.5" id="credit-2"><span class="header-section-number">10.3.5</span> Credit</h3>
<p><em>Summary generated as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.4" id="specific-ace2-expression-in-cholangiocytes-may-cause-liver-damage-after-2019-ncov-infection"><span class="header-section-number">10.4</span> Specific ACE2 Expression in Cholangiocytes May Cause Liver Damage After 2019-nCoV Infection</h2>
<p>Chai et al. <em>bioRxiv</em> <span class="citation" data-cites="BHQvMYPG">[<a href="#ref-BHQvMYPG" role="doc-biblioref">760</a>]</span></p>
<h3 data-number="10.4.1" id="keywords-3"><span class="header-section-number">10.4.1</span> Keywords</h3>
<ul>
<li>ACE2</li>
<li>Cholangiocytes</li>
<li>COVID-associated Liver Damage</li>
</ul>
<h3 data-number="10.4.2" id="summary-2"><span class="header-section-number">10.4.2</span> Summary</h3>
<p>Using both publicly available scRNA-seq dataset of liver
samples from colorectal patients and scRNA-sequencing of four liver
samples from healthy volunteers, the authors show that ACE2 is
significantly enriched in the majority of cholangiocytes (59.7 %) but
not in hepatocytes (2.6%).</p>
<h3 data-number="10.4.3" id="main-findings-3"><span class="header-section-number">10.4.3</span> Main Findings</h3>
<p>Using bioinformatics approaches of
RNASeq analysis, this study reveals that ACE2 dominates in
cholangiocytes and is present at very low levels in hepatocytes.</p>
<h3 data-number="10.4.4" id="limitations-3"><span class="header-section-number">10.4.4</span> Limitations</h3>
<p>The
study does not provide mechanistic insights into how SARS-CoV-2 can
infect and replicate in cholangiocytes and the types of intrinsic
anti-viral responses induced by cholangiocytes when infected. In
addition, because the study relies on the assumption that SARS-CoV-2
infects cells only through ACE2, it cannot discount the possibility that
the virus can infect hepatocytes through mechanisms other than
ACE2-mediated entry. Furthermore, because the scRNA-seq analysis were
performed on healthy liver samples, one cannot draw any definitive
conclusions about gene expression states (including ACE2 expression in
liver cell types) in system-wide inflammatory contexts.</p>
<h3 data-number="10.4.5" id="significance-1"><span class="header-section-number">10.4.5</span> Significance</h3>
<p>This article with other studies on liver
damage in COVID patients suggests that liver damage observed in COVID
patients is more due to inflammatory cytokines than direct infection of
the liver. Even if cholangiocytes are infectable by SARS-CoV-2 (which
was demonstrated by human liver ductal organoid study
(<span class="citation" data-cites="OuDMtCRl">[<a href="#ref-OuDMtCRl" role="doc-biblioref">761</a>]</span>),
published clinical data show no significant increase in bile duct injury
related indexes (i.e. alkaline phosphatase, gamma-glutamyl
transpeptidase and total bilirubin). In sum, it underscores the
importance of future studies characterizing cellular responses of
extra-pulmonary organs in the context of COVID or at least in viral lung
infections..</p>
<h3 data-number="10.4.6" id="credit-3"><span class="header-section-number">10.4.6</span> Credit</h3>
<p><em>Summary generated by Chang Moon as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.5" id="ace2-expression-by-colonic-epithelial-cells-is-associated-with-viral-infection-immunity-and-energy-metabolism"><span class="header-section-number">10.5</span> ACE2 expression by colonic epithelial cells is associated with viral infection, immunity and energy metabolism</h2>
<p>Wang et al. <em>medRxiv</em>. <span class="citation" data-cites="ViWCb8MJ">[<a href="#ref-ViWCb8MJ" role="doc-biblioref">762</a>]</span></p>
<h3 data-number="10.5.1" id="keywords-4"><span class="header-section-number">10.5.1</span> Keywords</h3>
<ul>
<li>single cell RNA seq</li>
<li>ACE2 expression</li>
<li>human colonic biopsy</li>
</ul>
<h3 data-number="10.5.2" id="main-findings-4"><span class="header-section-number">10.5.2</span> Main Findings</h3>
<p>Colonic enterocytes primarily express ACE2. Cellular pathways associated with ACE2 expression include innate immune signaling, HLA up regulation, energy metabolism and apoptotic signaling.</p>
<h3 data-number="10.5.3" id="limitations-4"><span class="header-section-number">10.5.3</span> Limitations</h3>
<p>This is a study of colonic biopsies taken from 17 children with and without IBD and analyzed using scRNAseq to look at ACE2 expression and identify gene families correlated with ACE2 expression. The authors find ACE2 expression to be primarily in colonocytes. It is not clear why both healthy and IBD patients were combined for the analysis. Biopsies were all of children so extrapolation to adults is limited. The majority of genes found to be negatively correlated with ACE2 expression include immunoglobulin genes (IGs). IG expression will almost certainly be low in colonocytes irrespective of ACE2 expression.</p>
<h3 data-number="10.5.4" id="significance-2"><span class="header-section-number">10.5.4</span> Significance</h3>
<p>This study performs a retrospective analysis of ACE2 expression using an RNAseq dataset from intestinal biopsies of children with and without IBD. The implications for the CoV-19 epidemic are modest, but do provide support that ACE2 expression is specific to colonocytes in the intestines. The ontological pathway analysis provides some limited insights into gene expression associated with ACE2.</p>
<h3 data-number="10.5.5" id="credit-4"><span class="header-section-number">10.5.5</span> Credit</h3>
<p><em>Summary generated as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.6" id="the-pathogenicity-of-2019-novel-coronavirus-in-hace2-transgenic-mice"><span class="header-section-number">10.6</span> The Pathogenicity of 2019 Novel Coronavirus in hACE2 Transgenic Mice</h2>
<p>Bao et al. <em>bioRxiv</em> <span class="citation" data-cites="MgLBA5m">[<a href="#ref-MgLBA5m" role="doc-biblioref">763</a>]</span></p>
<h3 data-number="10.6.1" id="keywords-5"><span class="header-section-number">10.6.1</span> Keywords</h3>
<ul>
<li>Covid-19 mouse model</li>
<li>hACE2 mice</li>
<li>2019-nCoV model</li>
<li>ACE2</li>
<li>2019-nCoV </li>
</ul>
<h3 data-number="10.6.2" id="main-findings-5"><span class="header-section-number">10.6.2</span> Main Findings</h3>
<p>Using a transgenic human Angiotensin-converting enzyme 2
(hACE2) mouse that has previously been shown susceptible to infection by
SARS-CoV, Bao et al. create a model of pandemic 2019-nCoV strain
coronavirus. The model includes interstitial hyperplasia in lung tissue,
moderate inflammation in bronchioles and blood vessels, and histology
consistent with viral pneumonia at 3 days post infection. Wildtype did
not experience these symptoms. In addition, viral antigen and hACE2
receptor were found to co-localize the lung by immunofluorescence 3-10
days post infection only in the hACE2 infected mice.</p>
<h3 data-number="10.6.3" id="limitations-5"><span class="header-section-number">10.6.3</span> Limitations</h3>
<p>The characterization of the infection remains incomplete,
as well as lacking characterization of the immune response other than
the presence of a single antiviral antibody. Though they claim to
fulfill Koch’s postulates, they only isolate the virus and re-infect
Vero cells, rather than naive mice.</p>
<h3 data-number="10.6.4" id="significance-3"><span class="header-section-number">10.6.4</span> Significance</h3>
<p>This paper establishes a murine model for 2019-nCoV
infection with symptoms consistent with viral pneumonia. Though not
fully characterized, this model allows in vivo analysis of viral entry
and pathology that is important for the development of vaccines and
antiviral therapeutics.</p>
<h3 data-number="10.6.5" id="credit-5"><span class="header-section-number">10.6.5</span> Credit</h3>
<p><em>Review by Dan Fu Ruan, Evan Cody and Venu Pothula as part of a project by students, postdocs and faculty
at the Immunology Institute of the Icahn School of Medicine, Mount
Sinai.</em></p>
<h2 data-number="10.7" id="caution-on-kidney-dysfunctions-of-2019-ncov-patients"><span class="header-section-number">10.7</span> Caution on Kidney Dysfunctions of 2019-nCoV Patients</h2>
<p>Li et al. <em>medRxiv</em>. <span class="citation" data-cites="1Acv5UMSt">[<a href="#ref-1Acv5UMSt" role="doc-biblioref">764</a>]</span></p>
<h3 data-number="10.7.1" id="keywords-6"><span class="header-section-number">10.7.1</span> Keywords</h3>
<p>CoVID-19, 2019-nCoV, SARS-CoV-2, kidney, clinical, creatinine,
proteinuria, albuminuria, CT</p>
<h3 data-number="10.7.2" id="main-findings-6"><span class="header-section-number">10.7.2</span> Main Findings</h3>
<ul>
<li><p>Retrospective study of 59 patients assayed key function indicators
of the kidney–including urine protein, blood urea nitrogen (BUN),
plasma creatinine (Cre), and renal CT scan data.</p></li>
<li><p>Found that 34% of patients developed massive albuminuria on the
first day of admission, and 63% developed proteinuria during their
stay in hospital; and 19% of patients had high plasma creatinine,
especially the terminal cases.</p></li>
<li><p>CT analyses of 27 patients showed all patients to have abnormal
kidney damage; indicate that inflammation and edema of the renal
parenchyma very common.</p></li>
</ul>
<h3 data-number="10.7.3" id="limitations-6"><span class="header-section-number">10.7.3</span> Limitations</h3>
<ul>
<li><p>No analysis of immunity-dependent damage and cytokines in
blood/plasma/urine. Will be worth correlating disease progression
with cytokine production, immune activity and kidney function.</p></li>
<li><p>Extrapolating to earlier SARS-CoV studies provides the only
rationale for viral-damage in kidney and resultant pathologic immune
response <em>(understandable for this clinical study)</em>.</p></li>
</ul>
<h3 data-number="10.7.4" id="significance-4"><span class="header-section-number">10.7.4</span> Significance</h3>
<ul>
<li><p>Multiple lines of evidence along this study’s finding point to the
idea that renal impairment/injury is a key risk factor in 2019-nCoV
patients similar to what has been reported for SARS-CoV<span class="citation" data-cites="DMaiZBE1">[<a href="#ref-DMaiZBE1" role="doc-biblioref">765</a>]</span>; this
may be one of the major causes of virally-induced damage and
contribute to multiorgan failure.</p></li>
<li><p>ACE2 expression in kidney proximal tubule epithelia and bladder
epithelia <span class="citation" data-cites="16Xj27QYj">[<a href="#ref-16Xj27QYj" role="doc-biblioref">766</a>]</span> support
these clinical findings.</p></li>
<li><p>Study argues for closely monitoring kidney function, and applying
potential interventions including continuous renal replacement
therapies (CRRT) for protecting kidney functions as early as
possible, particularly for those with rising plasma creatinine.</p></li>
</ul>
<h3 data-number="10.7.5" id="credit-6"><span class="header-section-number">10.7.5</span> Credit</h3>
<p><em>Review by Samarth Hegde as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.8" id="profiling-the-immune-vulnerability-landscape-of-the-2019-novel-coronavirus"><span class="header-section-number">10.8</span> Profiling the immune vulnerability landscape of the 2019 Novel Coronavirus</h2>
<p>Zhu et al. <em>bioRxiv</em> <span class="citation" data-cites="QhE5mVCc">[<a href="#ref-QhE5mVCc" role="doc-biblioref">767</a>]</span></p>
<h3 data-number="10.8.1" id="keywords-7"><span class="header-section-number">10.8.1</span> Keywords</h3>
<ul>
<li>epitope prediction</li>
<li>vaccine development.</li>
</ul>
<h3 data-number="10.8.2" id="main-findings-7"><span class="header-section-number">10.8.2</span> Main Findings</h3>
<p>This study harnesses bioinformatic profiling to predict the potential of
COV2 viral proteins to be presented on MHC I and II and to form linear
B-cell epitopes. These estimates suggest a T-cell antigenic profile
distinct from SARS-CoV or MERS-CoV, identify focused regions of the
virus with a high density of predicted epitopes, and provide preliminary
evidence for adaptive immune pressure in the genetic evolution of the
virus.</p>
<h3 data-number="10.8.3" id="limitations-7"><span class="header-section-number">10.8.3</span> Limitations</h3>
<p>While the study performs a comprehensive analysis of potential epitopes
within the virus genome, the analysis relies solely on bioinformatic
prediction to examine MHC binding affinity and B-cell epitope potential
and does not capture the immunogenicity or recognition of
these epitopes. Future experimental validation in data from patients
infected with SARS-CoV-2 will be important to validate and refine these
findings. Thus some of the potential conclusions stated, including viral
evolution toward lower immunogenicity or a dominant role for CD4+
T-cells rather than CD8+ T-cells in viral clearance, require further
valiadtion.</p>
<h3 data-number="10.8.4" id="significance-5"><span class="header-section-number">10.8.4</span> Significance</h3>
<p>These findings may help direct peptide vaccine design toward relevant
epitopes and provide intriguing evidence of viral evolution in response
to immune pressure.</p>
<h3 data-number="10.8.5" id="credit-7"><span class="header-section-number">10.8.5</span> Credit</h3>
<p><em>Summary generated as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.9" id="single-cell-analysis-of-ace2-expression-in-human-kidneys-and-bladders-reveals-a-potential-route-of-2019-ncov-infection"><span class="header-section-number">10.9</span> Single-cell Analysis of ACE2 Expression in Human Kidneys and Bladders Reveals a Potential Route of 2019-nCoV Infection</h2>
<p>Lin et al. <em>bioRxiv</em> <span class="citation" data-cites="16Xj27QYj">[<a href="#ref-16Xj27QYj" role="doc-biblioref">766</a>]</span></p>
<h3 data-number="10.9.1" id="keywords-8"><span class="header-section-number">10.9.1</span> Keywords</h3>
<ul>
<li>CoVID-19</li>
<li>2019-nCoV</li>
<li>SARS-CoV-2</li>
<li>ACE2</li>
<li>scRNAseq</li>
<li>kidney</li>
<li>bladder</li>
<li>public dataset</li>
</ul>
<h3 data-number="10.9.2" id="main-findings-8"><span class="header-section-number">10.9.2</span> Main Findings</h3>
<ul>
<li><p>To investigate the possible cause of kidney damage in 2019-nCoV
patients, authors used published kidney and bladder cell atlas data
(GSE131685, GSE108097; 3 healthy donors each) as well as an
unpublished kidney single-cell RNA-Seq data (in-house from 2
transplant donors) to evaluate ACE2 gene expressions in all cell
types of healthy kidneys and bladders.</p></li>
<li><p>They find enriched expression of ACE2 transcript in all subtypes of
proximal tubule cells of kidney, with 5%-15% of both straight and
convoluted proximal tubule cells expressing ACE2.</p></li>
<li><p>They also find detectable levels of ACE2 in bladder epithelial
cells, noting expression from around 1.5% of cells in the outer
layer umbrella cells of the bladder epithelium and decreasing in the
basal cells.</p></li>
<li><p>Importantly endothelial or immune cells in kidney/bladder do not
express ACE2.</p></li>
</ul>
<h3 data-number="10.9.3" id="limitations-8"><span class="header-section-number">10.9.3</span> Limitations</h3>
<ul>
<li><p>This study primarily characterizes ACE2 expression (amongst other
genes) from a small healthy-donor dataset, and will benefit from
supporting data in (expired) patient samples to show functional
viral damage. ACE2 transcript does not necessarily translate to
viral permissiveness in kidney/bladder epithelia or cytokine
release.</p></li>
<li><p>This study focuses on only healthy tissue; it will be useful to
analyze kidney/bladder epithelial ACE2 expression under inflammatory
conditions or in patients with underlying kidney conditions.</p></li>
<li><p>Given what is known about protease TMPRSS2 expression during
SARS-CoV-2 infection, ACE2+TMPRSS2+ double-positive cell
identification would be useful in these datasets.</p></li>
</ul>
<h3 data-number="10.9.4" id="significance-6"><span class="header-section-number">10.9.4</span> Significance</h3>
<ul>
<li><p>ACE2 protein is spatially restricted to brush border of proximal
tubules and in bladder umbrella cells <span class="citation" data-cites="1FE1YGsaV">[<a href="#ref-1FE1YGsaV" role="doc-biblioref">768</a>]</span>, such cells in direct
contact with viral particles are likely to be highly sensitive to
viral-induced damage.</p></li>
<li><p>SARS-CoV and MERS-CoV have been shown to be detected in urine of
patients and associate with higher mortality <span class="citation" data-cites="DMaiZBE1 iObQU2el">[<a href="#ref-DMaiZBE1" role="doc-biblioref">765</a>,<a href="#ref-iObQU2el" role="doc-biblioref">769</a>]</span>, thus worth
understanding kidney damage and resultant immune response in
SARS-CoV-2 as well.</p></li>
<li><p>This study argues for a potential mode of viral infectivity and
resultant inflammatory responses in these tissue in addition to
reported infectivity in the lung and digestive system, which is
supported by clinical data showing acute and early kidney
complications in 2019-nCoV patients <span class="citation" data-cites="1Acv5UMSt">[<a href="#ref-1Acv5UMSt" role="doc-biblioref">764</a>]</span>.</p></li>
<li><p>Clinically, thus very important to track urinary CoVID-19 shedding
as well as study acute kidney injury-related co-morbidities.</p></li>
</ul>
<h3 data-number="10.9.5" id="credit-8"><span class="header-section-number">10.9.5</span> Credit</h3>
<p><em>Review by Samarth Hegde as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.10" id="neutrophil-to-lymphocyte-ratio-predicts-severe-illness-patients-with-2019-novel-coronavirus-in-the-early-stage"><span class="header-section-number">10.10</span> Neutrophil-to-Lymphocyte Ratio Predicts Severe Illness Patients with 2019 Novel Coronavirus in the Early Stage</h2>
<p>Liu et al. <em>medRxiv</em> <span class="citation" data-cites="gK5e7Yhl">[<a href="#ref-gK5e7Yhl" role="doc-biblioref">770</a>]</span></p>
<h3 data-number="10.10.1" id="keywords-9"><span class="header-section-number">10.10.1</span> Keywords</h3>
<ul>
<li>severe disease</li>
<li>pneumonia</li>
<li>lymphocytes</li>
<li>neutrophils</li>
</ul>
<h3 data-number="10.10.2" id="main-findings-9"><span class="header-section-number">10.10.2</span> Main Findings</h3>
<p>This study aimed to find prognostic biomarkers of COVID-19
pneumonia severity. Sixty-one (61) patients with COVID-19 treated in
January at a hospital in Beijing, China were included. On average,
patients were seen within 5 days from illness onset. Samples were
collected on admission; and then patients were monitored for the
development of severe illness with a median follow-up of 10 days].</p>
<p>Patients were grouped as “mild” (N=44) or “moderate/severe” (N=17)
according to symptoms on admission and compared for different
clinical/laboratory features. “Moderate/severe” patients were
significantly older (median of 56 years old, compared to 41 years old).
Whereas comorbidies rates were largely similar between the groups,
except for hypertension, which was more frequent in the severe group (p=
0.056). ‘Severe’ patients had higher counts of neutrophils, and serum
glucose levels; but lower lymphocyte counts, sodium and serum chlorine
levels. The ratio of neutrophils to lymphocytes (NLR) was also higher
for the ‘severe’ group. ‘Severe’ patients had a higher rate of bacterial
infections (and antibiotic treatment) and received more intensive
respiratory support and treatment.</p>
<p>26 clinical/laboratory variables were used to select NLR and age as the
best predictors of the severe disease. Predictive cutoffs for a severe
illness as NLR ≥ 3.13 or age ≥ 50 years.</p>
<h3 data-number="10.10.3" id="limitations-9"><span class="header-section-number">10.10.3</span> Limitations</h3>
<p>Identification of early biomarkers is important for
making clinical decisions, but large sample size and validation cohorts
are necessary to confirm findings. It is worth noting that patients
classified as “mild” showed pneumonia by imaging and fever, and in
accordance with current classifications this would be consistent with
“moderate” cases. Hence it would be more appropriate to refer to the
groups as “moderate” vs “severe/critical”. Furthermore, there are
several limitations that could impact the interpretation of the results:
e.g. classification of patients was based on symptoms presented on
admission and not based on disease progression, small sample size,
especially the number of ‘severe’ cases (with no deaths among these
patients). Given the small sample size, the proposed NLR and age cut
offs might not hold for a slightly different set of patients. For
example, in a study of &gt;400 patients, ‘non-severe’ and ‘severe’ NLR
were 3.2 and 5.5, respectively <span class="citation" data-cites="6Y81C4P3">[<a href="#ref-6Y81C4P3" role="doc-biblioref">771</a>]</span>.</p>
<h3 data-number="10.10.4" id="credit-9"><span class="header-section-number">10.10.4</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.11" id="characteristics-of-lymphocyte-subsets-and-cytokines-in-peripheral-blood-of-123-hospitalized-patients-with-2019-novel-coronavirus-pneumonia-ncp"><span class="header-section-number">10.11</span> Characteristics of lymphocyte subsets and cytokines in peripheral blood of 123 hospitalized patients with 2019 novel coronavirus pneumonia (NCP)</h2>
<p>Wan et. al. <em>medRxiv</em> <span class="citation" data-cites="P4assQHj">[<a href="#ref-P4assQHj" role="doc-biblioref">772</a>]</span></p>
<h3 data-number="10.11.1" id="keywords-10"><span class="header-section-number">10.11.1</span> Keywords</h3>
<ul>
<li>Cytokines</li>
<li>lymphocyte subsets</li>
<li>CD8 + T</li>
<li>B cells</li>
<li>NK cells,</li>
<li>PBMCs</li>
<li>IL-6</li>
<li>IL-10</li>
</ul>
<h3 data-number="10.11.2" id="main-findings-10"><span class="header-section-number">10.11.2</span> Main Findings</h3>
<p>The authors analyzed lymphocyte subsets and cytokines of 102 patients
with mild disease and 21 with severe disease. CD8+T cells and CD4+T
cells were significantly reduced in both cohort. particularly in severe
patients. The cytokines IL6 and IL10 were significantly elevated in
severe patients as compared to mild. No significant differences were
observed in frequency of B cells and NK cells.</p>
<p>The authors argue that the measurement of T cell frequencies and
cytokine levels of IL6 and IL10 can be used to predict progression of
disease from Mild to severe Cov-2 infection.</p>
<h3 data-number="10.11.3" id="limitations-10"><span class="header-section-number">10.11.3</span> Limitations</h3>
<p>The study demonstrates in a limited cohort
similar associations to several other reported studies. The authors
didn’t compare the changes in lymphocyte and cytokine with healthy
individual (Covid-19 Negative) rather used an internal standard value.
The recently preprint in LANCET shows The degree of lymphopenia and a
pro-inflammatory cytokine storm is higher in severe COVID-19 patients
than in mild cases, and is associated with the disease severity <span class="citation" data-cites="zskgSnle">[<a href="#ref-zskgSnle" role="doc-biblioref">773</a>]</span>.</p>
<h3 data-number="10.11.4" id="significance-7"><span class="header-section-number">10.11.4</span> Significance</h3>
<p>This translational data identifies key cytokines and lymphopenia
associated with disease severity although mechanism and key cellular
players are still unknown. Higher level IL-6 production in severe
patient suggests potential role of Tocilizumab (anti-IL6R) biologic
although clinical trial will be necessary.</p>
<h3 data-number="10.11.5" id="credit-10"><span class="header-section-number">10.11.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.12" id="epidemiological-and-clinical-characteristics-of-17-hospitalized-patients-with-2019-novel-coronavirus-infections-outside-wuhan-china"><span class="header-section-number">10.12</span> Epidemiological and Clinical Characteristics of 17 Hospitalized Patients with 2019 Novel Coronavirus Infections Outside Wuhan, China</h2>
<p>Li et al. <em>medRxiv</em> <span class="citation" data-cites="TXlNzmXd">[<a href="#ref-TXlNzmXd" role="doc-biblioref">774</a>]</span></p>
<h3 data-number="10.12.1" id="keywords-11"><span class="header-section-number">10.12.1</span> Keywords</h3>
<ul>
<li>epidemiology</li>
<li>clinical characteristics</li>
</ul>
<h3 data-number="10.12.2" id="major-findings"><span class="header-section-number">10.12.2</span> Major Findings</h3>
<p>These authors looked at 17 hospitalized patients with COVID-19
confirmed by RT-PCR in Dazhou, Sichuan. Patients were admitted between
January 22 and February 10 and the final data were collected on February
11. Of the 17 patients, 12 remained hospitalized while 5 were discharged
after meeting national standards. The authors observed no differences
based on the sex of the patients but found that the discharged patients
were younger in age (p = 0.026) and had higher lymphocyte counts (p =
0.005) and monocyte counts (p = 0.019) upon admission.</p>
<h3 data-number="10.12.3" id="limitations-11"><span class="header-section-number">10.12.3</span> Limitations</h3>
<p>This study is limited in the sample size of the study and the last
data collection point was only one day after some of the patients were
admitted.</p>
<h3 data-number="10.12.4" id="significance-8"><span class="header-section-number">10.12.4</span> Significance</h3>
<p>These findings have been somewhat supported by subsequent studies
that show that older age and an immunocompromised state are more likely
to result in a more severe clinical course with COVID-19. However, other
studies have been published that report on larger numbers of cases.</p>
<h3 data-number="10.12.5" id="credit-11"><span class="header-section-number">10.12.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.13" id="ace2-expression-in-kidney-and-testis-may-cause-kidney-and-testis-damage-after-2019-ncov-infection"><span class="header-section-number">10.13</span> ACE2 Expression in Kidney and Testis May Cause Kidney and Testis Damage After 2019-nCoV Infection</h2>
<p><span class="citation" data-cites="15K5Ovntj">[<a href="#ref-15K5Ovntj" role="doc-biblioref">775</a>]</span></p>
<h3 data-number="10.13.1" id="keywords-12"><span class="header-section-number">10.13.1</span> Keywords</h3>
<ul>
<li>CoVID-19</li>
<li>2019-nCoV</li>
<li>SARS-CoV-2</li>
<li>kidney</li>
<li>testis</li>
<li>ACE2</li>
<li>scRNAseq</li>
</ul>
<h3 data-number="10.13.2" id="main-findings-11"><span class="header-section-number">10.13.2</span> Main Findings</h3>
<ul>
<li><p>Study used online datasets (scRNAseq GSE131685, scRNAseq GSE107585,
Human Protein Atlas, GTEx portal, CCLE) to analyze ACE2 expression
in different human organs.</p></li>
<li><p>Study re-analyzed three clinical datasets (n=6, n=99, and n=41) to
show 3~10% of 2019-nCoV patients present with abnormal renal
function.</p></li>
<li><p>results indicate ACE2 highly expressed in renal tubular cells,
Leydig cells and seminiferous ductal cells of testis.</p></li>
</ul>
<h3 data-number="10.13.3" id="limitations-12"><span class="header-section-number">10.13.3</span> Limitations</h3>
<ul>
<li><p>Very preliminary transcript/protein dataset analysis in healthy
cohorts; does not necessarily translate to actual viral tropism and
permissiveness.</p></li>
<li><p>Clinically, would be important to determine with larger longitudinal
dataset if SARS-CoV-2 infection changes sperm quality or testicular
inflammation.</p></li>
<li><p>Similarly, would be important to determine if simultaneous HBV or
syphilis infection and orchitis impacts SARS-CoV-2 severity.</p></li>
<li><p>Examination and follow-up of renal function and viral orchitis/sperm
quality of CoVID-19 patients not done in this preliminary study.</p></li>
</ul>
<h3 data-number="10.13.4" id="significance-9"><span class="header-section-number">10.13.4</span> Significance</h3>
<ul>
<li><p>Kidney ACE2 result supports other concurrent sequencing studies
<span class="citation" data-cites="16Xj27QYj">[<a href="#ref-16Xj27QYj" role="doc-biblioref">766</a>]</span> and clinical reports
of abnormal renal function or even kidney damage in patients
infected with 2019-nCoV
<span class="citation" data-cites="1Acv5UMSt">[<a href="#ref-1Acv5UMSt" role="doc-biblioref">764</a>]</span>.</p></li>
<li><p>High ACE2 expression in testis suggests potential tropism of the
virus to testicular tissues and indicates potential risks for male
fertility. Viral orchitis reported for SARS-CoV previously [1],
but no clear evidence so far of infertility in SARS, MERS or
CoVID-19 patients.</p></li>
</ul>
<h3 data-number="10.13.5" id="credit-12"><span class="header-section-number">10.13.5</span> Credit</h3>
<p><em>Review by Samarth Hegde as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.14" id="aberrant-pathogenic-gm-csf-t-cells-and-inflammatory-cd14cd16-monocytes-in-severe-pulmonary-syndrome-patients-of-a-new-coronavirus"><span class="header-section-number">10.14</span> Aberrant pathogenic GM-CSF+ T cells and inflammatory CD14+CD16+ monocytes in severe pulmonary syndrome patients of a new coronavirus</h2>
<p><span class="citation" data-cites="Bgo7Cmj3">[<a href="#ref-Bgo7Cmj3" role="doc-biblioref">776</a>]</span></p>
<h3 data-number="10.14.1" id="keywords-13"><span class="header-section-number">10.14.1</span> Keywords</h3>
<ul>
<li>immunopathology</li>
<li>Th1</li>
<li>inflammatory monocytes</li>
<li>GM-CSF</li>
<li>IFN-γ</li>
<li>IL-6</li>
</ul>
<h3 data-number="10.14.2" id="main-findings-12"><span class="header-section-number">10.14.2</span> Main Findings</h3>
<p>The authors of this study sought to characterize the immune
mechanism causing severe pulmonary disease and mortality in 2019-nCoV
(COVID-19) patients. Peripheral blood was collected from hospitalized
ICU (n=12) and non-ICU (n=21) patients with confirmed 2019-nCoV and from
healthy controls (n=10) in The First Affiliated Hospital of University
of Science and Technology China (Hefei, Anhui). Immune analysis was
conducted by flow cytometry. 2019-nCoV patients had decreased
lymphocyte, monocyte, and CD4 T cell counts compared to healthy
controls. ICU patients had fewer lymphocytes than non-ICU patients. CD4
T cells of 2019-nCoV patients expressed higher levels of activation
markers (OX40, CD69, CD38, CD44) and exhaustion markers (PD-1 and Tim3)
than those of healthy controls. CD4 cells of ICU patients expressed
significantly higher levels of OX40, PD-1, and Tim3 than those of
non-ICU patients. 2019-nCoV patients had higher percentages of CD4 T
cells co-expressing GM-CSF and IL-6 compared to healthy controls, while
ICU patients had a markedly higher percentage of GM-CSF+ IFN-γ+ CD4 T
cells than non-ICU patients. The CD4 T cells of nCoV patients and
healthy controls showed no differences in TNF-α secretion.</p>
<p>The CD8 T cells of 2019-nCoV patients also showed higher expression of
activation markers CD69, CD38, and CD44, as well as exhaustion markers
PD-1 and Tim3, compared to healthy controls. CD8 T cells of ICU patients
expressed higher levels of GM-CSF than those of non-ICU patients and
healthy controls. No IL-6 or TNF-α was found in the CD8 T cells of any
group. There were no differences in numbers of NK cells or B cells in
2019-nCoV patients and healthy controls, nor was there any GM-CSF or
IL-6 secretion from these cells in either group.</p>
<p>Percentages of CD14+ CD16+ GM-CSF+ and CD14+ CD16+ IL-6+ inflammatory
monocytes were significantly increased in nCoV patients compared to
healthy controls; in particular, patients in the ICU had greater
percentages of CD14+ CD16+ IL-6+ monocytes than non-ICU patients. The
authors suggest that in 2019-nCoV patients, pathogenic Th1 cells produce
GM-CSF, recruiting CD14+ CD16+ inflammatory monocytes that secrete high
levels of IL-6. These may enter pulmonary circulation and damage lung
tissue while initiating the cytokine storm that causes mortality in
severe cases. This is consistent with the cytokine storm seen in similar
coronaviruses, as IL-6, IFN-γ, and GM-CSF are key inflammatory mediators
seen in patients with SARS-CoV-1 and MERS-CoV.</p>
<h3 data-number="10.14.3" id="limitations-13"><span class="header-section-number">10.14.3</span> Limitations</h3>
<p>Though the results of this study open
questions for further investigation, this is an early study on a small
cohort of patients, and as such there are a number of limitations. The
study included only 12 ICU patients and 21 non-ICU patients, and ideally
would be repeated with a much larger patient cohort. Though the authors
make claims about differences in lymphocyte and monocyte counts between
patients and healthy controls, they did not report baseline laboratory
findings for the control group. Additionally, severity of disease was
classified based on whether or not patients were in the ICU. It would be
interesting to contextualize the authors’ immunological findings with
more specific metrics of disease severity or time course. Noting
mortality, time from disease onset, pre-existing conditions, or severity
of lung pathology in post-mortem tissue samples would paint a fuller
picture of how to assess risk level and the relationship between
severity of disease and immunopathology. Another limitation is the
selection of cytokines and immune markers for analysis, as the selection
criteria were based on the cell subsets and cytokine storm typically
seen in SARS-CoV-1 and MERS-CoV patients. Unbiased cytokine screens and
immune profiling may reveal novel therapeutic targets that were not
included in this study.</p>
<h3 data-number="10.14.4" id="significance-10"><span class="header-section-number">10.14.4</span> Significance</h3>
<p>This study identifies potential therapeutic targets
that could prevent acute respiratory disease syndrome (ARDS) and
mortality in patients most severely affected by COVID-19. The authors
propose testing monoclonal antibodies against IL6-R or GM-CSF to block
recruitment of inflammatory monocytes and the subsequent cytokine storm
in these patients.</p>
<h3 data-number="10.14.5" id="credit-13"><span class="header-section-number">10.14.5</span> Credit</h3>
<p><em>Review by Gabrielle Lubitz as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.15" id="clinical-characteristics-of-2019-novel-infected-coronavirus-pneumoniaa-systemic-review-and-meta-analysis"><span class="header-section-number">10.15</span> Clinical Characteristics of 2019 Novel Infected Coronavirus Pneumonia：A Systemic Review and Meta-analysis</h2>
<p>Qian et al. <em>medRxiv</em>.
<span class="citation" data-cites="4Z1pEioX">[<a href="#ref-4Z1pEioX" role="doc-biblioref">777</a>]</span></p>
<h3 data-number="10.15.1" id="keywords-14"><span class="header-section-number">10.15.1</span> Keywords</h3>
<ul>
<li>White Blood Cells</li>
<li>Lymphocytes</li>
<li>Neutrophils</li>
</ul>
<h3 data-number="10.15.2" id="main-findings-13"><span class="header-section-number">10.15.2</span> Main Findings</h3>
<p>The authors performed a meta analysis of literature on clinical,
laboratory and radiologic characteristics of patients presenting with
pneumonia related to SARSCoV2 infection, published up to Feb 6 2020.
They found that symptoms that were mostly consistent among studies were
sore throat, headache, diarrhea and rhinorrhea. Fever, cough, malaise
and muscle pain were highly variable across studies. Leukopenia (mostly
lymphocytopenia) and increased white blood cells were highly variable
across studies. They identified three most common patterns seen on CT
scan, but there was high variability across studies. Consistently across
the studies examined, the authors found that about 75% of patients need
supplemental oxygen therapy, about 23% mechanical ventilation and about
5% extracorporeal membrane oxygenation (ECMO). The authors calculated a
staggering pooled mortality incidence of 78% for these patients.</p>
<h3 data-number="10.15.3" id="limitations-14"><span class="header-section-number">10.15.3</span> Limitations</h3>
<p>The authors mention that the total number of studies included in this
meta analysis is nine, however they also mentioned that only three
studies reported individual patient data. It is overall unclear how many
patients in total were included in their analysis. This is mostly
relevant as they reported an incredibly high mortality (78%) and mention
an absolute number of deaths of 26 cases overall. It is not clear from
their report how the mortality rate was calculated.</p>
<p>The data is based on reports from China and mostly from the Wuhan area,
which somewhat limits the overall generalizability and applicability of
these results.</p>
<h3 data-number="10.15.4" id="significance-11"><span class="header-section-number">10.15.4</span> Significance</h3>
<p>This meta analysis offers some important data for clinicians to refer to
when dealing with patients with COVID-19 and specifically with
pneumonia. It is very helpful to set expectations about the course of
the disease.</p>
<h3 data-number="10.15.5" id="credit-14"><span class="header-section-number">10.15.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.16" id="longitudinal-characteristics-of-lymphocyte-responses-and-cytokine-profiles-in-the-peripheral-blood-of-sars-cov-2-infected-patients"><span class="header-section-number">10.16</span> Longitudinal characteristics of lymphocyte responses and cytokine profiles in the peripheral blood of SARS-CoV-2 infected patients</h2>
<p>Liu et al. <em>medRxiv</em> <span class="citation" data-cites="zTGzuiTI">[<a href="#ref-zTGzuiTI" role="doc-biblioref">778</a>]</span></p>
<h3 data-number="10.16.1" id="keywords-15"><span class="header-section-number">10.16.1</span> Keywords</h3>
<ul>
<li>Lymphopenia</li>
<li>Neutrophil to CD8 T cell ratio (N8R)</li>
<li>inflammatory cytokines</li>
</ul>
<h3 data-number="10.16.2" id="main-findings-14"><span class="header-section-number">10.16.2</span> Main Findings</h3>
<p>Liu et al. enrolled a cohort of 40 patients from Wuhan
including 27 mild cases and 13 severe cases of COVID-19. They performed
a 16-day kinetic analysis of peripheral blood from time of disease
onset. Patients in the severe group were older (medium age of 59.7,
compared to 48.7 in mild group) and more likely to have hypertension as
a co-morbidity. Lymphopenia was observed in 44.4% of the mild patients
and 84.6% of the severe patients. Lymphopenia was due to low T cell
count, specially CD8 T cells. Severe patients showed higher neutrophil
counts and an increase of cytokines in the serum (IL2, IL6, IL10 and
IFNγ). The authors measured several other clinical laboratory parameters
were also higher in severe cases compared to mild, but concluded that
neutrophil to CD8 T cell ratio (N8R) as the best prognostic factor to
identify the severe cases compared to other receiver operating
characteristic (ROC).</p>
<h3 data-number="10.16.3" id="limitations-15"><span class="header-section-number">10.16.3</span> Limitations</h3>
<p>This was a small cohort (N=40), and two of the patients
initially included in the severe group (N=13) passed away and were
excluded from the analysis due to lack of longitudinal data. However, it
would be most important to be able to identify patients with severe
disease with higher odds of dying. It seems that the different time
points analyzed relate to hospital admission, which the authors describe
as disease onset. The time between first symptoms and first data points
is not described. It would have been important to analyze how the
different measured parameters change according to health condition, and
not just time (but that would require a larger cohort). The predictive
value of N8R compared to the more commonly used NLR needs to be assessed
in other independent and larger cohorts. Lastly, it is important to note
that pneumonia was detected in patients included in the “mild” group,
but according to the Chinese Clinical Guidance for COVID-19 Pneumonia
Diagnosis and Treatment (7th edition) this group should be considered
“moderate”.</p>
<h3 data-number="10.16.4" id="significance-12"><span class="header-section-number">10.16.4</span> Significance</h3>
<p>Lymphopenia and cytokine storm have been described to be
detrimental in many other infections including SARS-CoV1 and MERS-CoV.
However, it was necessary to confirm that this dramatic immune response
was also observed in the SARS-CoV2 infected patients. These results and
further validation of the N8R ratio as a predictor of disease severity
will contribute for the management of COVID19 patients and potential
development of therapies.</p>
<h3 data-number="10.16.5" id="credit-15"><span class="header-section-number">10.16.5</span> Credit</h3>
<p><em>Review by Pauline Hamon as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.17" id="clinical-and-immunologic-features-in-severe-and-moderate-forms-of-coronavirus-disease-2019"><span class="header-section-number">10.17</span> Clinical and immunologic features in severe and moderate forms of Coronavirus Disease 2019</h2>
<p>Chen et al. <em>medRxiv</em> <span class="citation" data-cites="1EIEx8Qe4">[<a href="#ref-1EIEx8Qe4" role="doc-biblioref">779</a>]</span></p>
<h3 data-number="10.17.1" id="keywords-16"><span class="header-section-number">10.17.1</span> Keywords</h3>
<ul>
<li>severe disease</li>
<li>lymphocytes</li>
<li>cytokines</li>
<li>IFNy</li>
<li>CD4 Tcells</li>
<li>HLA-DR CD8</li>
<li>Tcells</li>
</ul>
<h3 data-number="10.17.2" id="main-findings-15"><span class="header-section-number">10.17.2</span> Main Findings</h3>
<p>This study retrospectively evaluated clinical, laboratory,
hematological, biochemical and immunologic data from 21 subjects
admitted to the hospital in Wuhan, China (late December/January) with
confirmed SARS-CoV-2 infection. The aim of the study was to compare
‘severe’ (n=11, ~64 years old) and ‘moderate’ (n=10, ~51 years old)
COVID-19 cases. Disease severity was defined by patients’ blood oxygen
level and respiratory output. They were classified as ‘severe’ if SpO2
93% or respiratory rates 30 per min.</p>
<p>In terms of the clinical laboratory measures, ‘severe’ patients had
higher CRP and ferritin, alanine and aspartate aminotransferases, and
lactate dehydrogenase but lower albumin concentrations.</p>
<p>The authors then compared plasma cytokine levels (ELISA) and immune cell
populations (PBMCs, Flow Cytometry). ‘Severe’ cases had higher levels of
IL-2R, IL-10, TNFa, and IL-6 (marginally significant). For the immune
cell counts, ‘severe’ group had higher neutrophils, HLA-DR+ CD8 T cells
and total B cells; and lower total lymphocytes, CD4 and CD8 T cells
(except for HLA-DR+), CD45RA Tregs, and IFNy-expressing CD4 T cells. No
significant differences were observed for IL-8, counts of NK cells,
CD45+RO Tregs, IFNy-expressing CD8 T and NK cells.</p>
<h3 data-number="10.17.3" id="limitations-16"><span class="header-section-number">10.17.3</span> Limitations</h3>
<p>Several potential limitations should be noted: 1) Blood samples were
collected 2 days post hospital admission and no data on viral loads were
available; 2) Most patients were administered medications (e.g.
corticosteroids), which could have affected lymphocyte counts.
Medications are briefly mentioned in the text of the manuscript; authors
should include medications as part of Table 1. 3) ‘Severe’ cases were
significantly older and 4/11 ‘severe’ patients died within 20 days.
Authors should consider a sensitivity analysis of biomarkers with the
adjustment for patients’ age.</p>
<h3 data-number="10.17.4" id="significance-13"><span class="header-section-number">10.17.4</span> Significance</h3>
<p>Although the sample size was small, this paper presented a broad range
of clinical, biochemical, and immunologic data on patients with
COVID-19. One of the main findings is that SARS-CoV-2 may affect T
lymphocytes, primarily CD4+ T cells, resulting in decreased IFNy
production. Potentially, diminished T lymphocytes and elevated cytokines
can serve as biomarkers of severity of COVID-19.</p>
<h3 data-number="10.17.5" id="credit-16"><span class="header-section-number">10.17.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.18" id="sars-cov-2-and-sars-cov-spike-rbd-structure-and-receptor-binding-comparison-and-potential-implications-on-neutralizing-antibody-and-vaccine-development"><span class="header-section-number">10.18</span> SARS-CoV-2 and SARS-CoV Spike-RBD Structure and Receptor Binding Comparison and Potential Implications on Neutralizing Antibody and Vaccine Development</h2>
<p>Sun et al. <em>bioRxiv</em> <span class="citation" data-cites="41WAtZmB">[<a href="#ref-41WAtZmB" role="doc-biblioref">780</a>]</span></p>
<h3 data-number="10.18.1" id="keywords-17"><span class="header-section-number">10.18.1</span> Keywords</h3>
<ul>
<li>SARS-CoV</li>
<li>SARS-CoV-2</li>
<li>ACE2</li>
<li>Spike (S) protein</li>
<li>receptor binding domain (RBD)</li>
<li>receptor binding motif (RBM)</li>
<li>neutralizing antibody</li>
</ul>
<h3 data-number="10.18.2" id="main-findings-16"><span class="header-section-number">10.18.2</span> Main Findings</h3>
<p>This study compared the structure of SARS-CoV and SARS-CoV-2 Spike (S)
protein receptor binding domain (RBD) and interactions with ACE2 using
computational modeling, and interrogated cross-reactivity and
cross-neutralization of SARS-CoV-2 by antibodies against SARS-CoV. While
SARS-CoV and SARS-CoV-2 have over 70 % sequence homology and share the
same human receptor ACE2, the receptor binding motif (RBM) is only 50%
homologous.</p>
<p>Computational prediction of the SARS-CoV-2 and ACE2 interactions based
on the previous crystal structure data of SARS-CoV, and measurement of
binding affinities against human ACE2 using recombinant SARS-CoV and
SARS-CoV-2 S1 peptides, demonstrated similar binding of the two S1
peptides to ACE2, explaining the similar transmissibility of SARS-CoV
and SARS-CoV-2 and consistent with previous data (Wall et al Cell
2020).</p>
<p>The neutralization activity of SARS-CoV-specific rabbit polyclonal
antibodies were about two-order of magnitude less efficient to
neutralize SARS-CoV-2 than SARS-CoV, and four potently neutralizing
monoclonal antibodies against SARS-CoV had poor binding and neutralizing
activity against SARS-CoV-2. In contrast, 3 poor SARS-CoV-binding
monoclonal antibodies show some efficiency to bind and neutralize
SARS-CoV-2. The results suggest that that antibodies to more conserved
regions outside the RBM motif might possess better cross-protective
neutralizing activities between two strains.</p>
<h3 data-number="10.18.3" id="limitations-17"><span class="header-section-number">10.18.3</span> Limitations</h3>
<p>It would have been helpful to show the epitopes recognized by the
monoclonal antibodies tested on both SARS-CoV, SARS-CoV-2 to be able to
make predictions for induction of broadly neutralizing antibodies. The
data on monoclonal antibody competition with ACE2 for binding to
SARS-CoV RBD should have also included binding on SARS-CoV2, especially
for the three monoclonal antibodies that showed neutralization activity
for SARS-CoV2. Because of the less homology in RBM sequences between
viruses, it still may be possible that these antibodies would recognize
the ACE2 RBD in SARS-CoV-2.</p>
<h3 data-number="10.18.4" id="significance-14"><span class="header-section-number">10.18.4</span> Significance</h3>
<p>It is noteworthy that immunization to mice and rabbit with SARS-CoV S1
or RBD protein could induce monoclonal antibodies to cross-bind and
cross-neutralize SARS-CoV-2 even if they are not ACE2-blocking. If these
types of antibodies could be found in human survivors or in the
asymptomatic populations as well, it might suggest that exposure to
previous Coronavirus strains could have induced cross-neutralizing
antibodies and resulted in the protection from severe symptoms in some
cases of SARS-CoV2.</p>
<h3 data-number="10.18.5" id="credit-17"><span class="header-section-number">10.18.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.19" id="protection-of-rhesus-macaque-from-sars-coronavirus-challenge-by-recombinant-adenovirus-vaccine"><span class="header-section-number">10.19</span> Protection of Rhesus Macaque from SARS-Coronavirus challenge by recombinant adenovirus vaccine</h2>
<p>Chen et al. <em>bioRxiv</em> <span class="citation" data-cites="NPxs7bbq">[<a href="#ref-NPxs7bbq" role="doc-biblioref">781</a>]</span></p>
<h3 data-number="10.19.1" id="keywords-18"><span class="header-section-number">10.19.1</span> Keywords</h3>
<ul>
<li>SARS-CoV-1</li>
<li>rhesus macaque</li>
<li>recombinant adenovirus vaccine</li>
</ul>
<h3 data-number="10.19.2" id="main-findings-17"><span class="header-section-number">10.19.2</span> Main Findings</h3>
<p>Rhesus macaques were immunized intramuscularly twice (week 0 and
week 4) with SV8000 carrying the information to express a S1-orf8
fusion protein and the N protein from the BJ01 strain of SARS-CoV-1.
By week 8, immunized animals had signs of immunological protection
(IgG and neutralization titers) against SARS-CoV-1 and were
protected against challenge with the PUMC-1 strain, with fewer
detectable symptoms of respiratory distress, lower viral load,
shorter periods of viral persistence, and less pathology in the
lungs compared to non-immunized animals.</p>
<h3 data-number="10.19.3" id="limitations-18"><span class="header-section-number">10.19.3</span> Limitations</h3>
<p>The authors should write clearer descriptions of the methods used in
this article. They do not describe how the IgG titers or
neutralization titers were determined. There are some issues with
the presentation of data, for example, in Figure 1a, y-axis should
not be Vmax; forming cells and 1d would benefit from showing error
bars. Furthermore, although I inferred that the animals were
challenged at week 8, the authors did not explicitly detail when the
animals were challenged. The authors should explain the design of
their vaccine, including the choice of antigens and vector. The
authors also do not include a description of the ethical use of
animals in their study.</p>
<h3 data-number="10.19.4" id="significance-15"><span class="header-section-number">10.19.4</span> Significance</h3>
<p>The authors describe a vaccine for SARS-CoV-1 with no discussion of
possible implications for the current SARS-CoV-2 pandemic. Could a
similar vaccine be designed to protect against SARS-CoV-2 and would
the concerns regarding emerging viral mutations that the authors
describe as a limitation for SARS-CoV-1 also be true in the context
of SARS-CoV-2?</p>
<h3 data-number="10.19.5" id="credit-18"><span class="header-section-number">10.19.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.20" id="reduction-and-functional-exhaustion-of-t-cells-in-patients-with-coronavirus-disease-2019-covid-19"><span class="header-section-number">10.20</span> Reduction and Functional Exhaustion of T cells in Patients with Coronavirus Disease 2019 (COVID-19)</h2>
<p><span class="citation" data-cites="f50ahNxF">[<a href="#ref-f50ahNxF" role="doc-biblioref">782</a>]</span></p>
<h3 data-number="10.20.1" id="keywords-19"><span class="header-section-number">10.20.1</span> Keywords</h3>
<ul>
<li>T cell exhaustion</li>
<li>T cell lymphopenia</li>
<li>IL-6</li>
<li>IL-10</li>
<li>TNF-α</li>
</ul>
<h3 data-number="10.20.2" id="main-findings-18"><span class="header-section-number">10.20.2</span> Main Findings</h3>
<p>Based on a retrospective study of 522 COVID patients and 40
healthy controls from two hospitals in Wuhan, China, authors show both
age-dependent and clinical severity-dependent decrease in T cell numbers
with elderly patients and patients who are in ICU-care showing the most
dramatic decrease in T cell counts. Cytokine profiling of COVID patients
reveal that TNF-α, IL-6 and IL-10 are increased in infected patients
with patients in the ICU showing the highest levels. Interestingly,
these three cytokine levels were inversely correlated with T cell counts
and such inverse relationship was preserved throughout the disease
progression. Surface staining of exhaustion markers (PD-1 and Tim-3) and
flow cytometry of stained peripheral blood of 14 patients and 3 healthy
volunteers demonstrate that T cells of COVID patients have increased
expression of PD-1 with patients in ICU having the highest number of
CD8<sup>+</sup>PD-1<sup>+</sup> cells than their counterparts in non-ICU groups.</p>
<h3 data-number="10.20.3" id="limitations-19"><span class="header-section-number">10.20.3</span> Limitations</h3>
<p>Compared to the number of patients, number of control (n=
40) is small and is not controlled for age. Additional data linking
inflammatory cytokines and the quality of the adaptive response
including humoral and antigen specific T cell response is much needed. T
cell exhaustion study relies on marker-dependent labeling of T cell
functionality of a very limited sample size (n=17)—a
functional/mechanistic study of these T cells from PBMCs would have
bolstered their claims.</p>
<h3 data-number="10.20.4" id="significance-16"><span class="header-section-number">10.20.4</span> Significance</h3>
<p>Limited but contains interesting
implications. It is already known in literature that in the context of
acute respiratory viral infections CD8 T cells exhibit exhaustion-like
phenotypes which further underscores the importance of mechanistic
studies that can elucidate how COVID infection leads to lymphopenia and
T cell exhaustion-like phenotype.</p>
<p>However, as authors have noted, the data does point to an interesting
question: How these inflammatory cytokines (TNF-α, IL-6 and IL-10)
correlate with or affect effective viral immunity and what types of
cells produce these cytokines? Answering that question will help us
refine our targets for immune-modulatory therapies especially in
patients suffering from cytokine storms.</p>
<h3 data-number="10.20.5" id="credit-19"><span class="header-section-number">10.20.5</span> Credit</h3>
<p><em>This review by Chang Moon was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.21" id="clinical-characteristics-of-25-death-cases-infected-with-covid-19-pneumonia-a-retrospective-review-of-medical-records-in-a-single-medical-center-wuhan-china"><span class="header-section-number">10.21</span> Clinical Characteristics of 25 death cases infected with COVID-19 pneumonia: a retrospective review of medical records in a single medical center, Wuhan, China</h2>
<p><span class="citation" data-cites="EwNi3WJq">[<a href="#ref-EwNi3WJq" role="doc-biblioref">783</a>]</span></p>
<h3 data-number="10.21.1" id="keywords-20"><span class="header-section-number">10.21.1</span> Keywords</h3>
<ul>
<li>COVID-19</li>
<li>pneumonia</li>
<li>hypertension</li>
<li>diabetes</li>
<li>biomarker</li>
<li>neutrophilia</li>
<li>lymphopenia</li>
</ul>
<h3 data-number="10.21.2" id="main-findings-19"><span class="header-section-number">10.21.2</span> Main Findings</h3>
<p>Most common chronic conditions among 25 patients that died from
COVID-19 related respiratory failure were hypertension (64%) and
diabetes (40%). Disease progression was marked by progressive organ
failure, starting first with lung dysfunction, then heart (e.g.
increased cTnI and pro-BNP), followed by kidney (e.g. increased BUN,
Cr), and liver (e.g. ALT, AST). 72% of patients had neutrophilia and 88%
also had lymphopenia. General markers of inflammation were also
increased (e.g. PCT, D-Dimer, CRP, LDH, and SAA).</p>
<h3 data-number="10.21.3" id="limitations-20"><span class="header-section-number">10.21.3</span> Limitations</h3>
<p>The limitations of this study include small sample size and
lack of measurements for some tests for several patients. This study
would also have been stronger with comparison of the same measurements
to patients suffering from less severe disease to further validate and
correlate proposed biomarkers with disease severity.</p>
<h3 data-number="10.21.4" id="significance-17"><span class="header-section-number">10.21.4</span> Significance</h3>
<p>This study identifies chronic conditions (i.e. hypertension
and diabetes) that strongly correlates with disease severity. In
addition to general markers of inflammation, the authors also identify
concomitant neutrophilia and lymphopenia among their cohort of patients.
This is a potentially interesting immunological finding because we would
typically expect increased lymphocytes during a viral infection.
Neutrophilia may also be contributing to cytokine storm. In addition,
PCT was elevated in 90.5% of patients, suggesting a role for sepsis or
secondary bacterial infection in COVID-19 related respiratory failure.</p>
<h3 data-number="10.21.5" id="credit-20"><span class="header-section-number">10.21.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.22" id="sars-cov-2-infection-does-not-significantly-cause-acute-renal-injury-an-analysis-of-116-hospitalized-patients-with-covid-19-in-a-single-hospital-wuhan-china"><span class="header-section-number">10.22</span> SARS-CoV-2 infection does not significantly cause acute renal injury: an analysis of 116 hospitalized patients with COVID-19 in a single hospital, Wuhan, China</h2>
<p><span class="citation" data-cites="KG5Lg65z">[<a href="#ref-KG5Lg65z" role="doc-biblioref">784</a>]</span></p>
<h3 data-number="10.22.1" id="keywords-21"><span class="header-section-number">10.22.1</span> Keywords</h3>
<ul>
<li>CoVID-19</li>
<li>2019-nCoV</li>
<li>SARS-CoV-2</li>
<li>kidney</li>
<li>clinical</li>
<li>longitudinal</li>
</ul>
<h3 data-number="10.22.2" id="main-findings-20"><span class="header-section-number">10.22.2</span> Main Findings</h3>
<ul>
<li><p>Clinical data from 116 hospitalized CoVID-19 patients analyzed over
4 weeks for correlation with renal injury. Comorbidities included
chronic renal failure (CRF) in 5 patients (4.3%).</p></li>
<li><p>10.8% of patients with no prior kidney disease showed elevations in
blood urea or creatinine, and 7.2% of patients with no prior kidney
disease showed albuminuria.</p></li>
<li><p>Patients with pre-existing CRF underwent continuous renal
replacement therapy (CRRT) alongside CoVID-19 treatment. Renal
functions remained stable in these patients.</p></li>
<li><p>All 5 patients with CRF survived CoVID-19 therapy without
progression to ARDS or worsening of CRF.</p></li>
</ul>
<h3 data-number="10.22.3" id="limitations-21"><span class="header-section-number">10.22.3</span> Limitations</h3>
<ul>
<li><p>Renal injury biomarkers in patients with incipient kidney
abnormalities not tabulated separately, making overall data hard to
interpret. It will be critical to separately examine kidney function
(BUN, urine creatinine and eGFR) in patients that developed any
kidney abnormalities (7.2~10.8% of cohort).</p></li>
<li><p>No information on type of CoVID-19 therapy used across cohort; will
be useful to correlate how treatment modality influences kidney
function (and other parameters).</p></li>
<li><p>Invokes previous clinical-correlation studies that indicate low
instances of kidney damage<span class="citation" data-cites="VRoblZFJ bgrzawWp">[<a href="#ref-VRoblZFJ" role="doc-biblioref">785</a>,<a href="#ref-bgrzawWp" role="doc-biblioref">786</a>]</span>, but those studies did not track
longitudinal urine samples for acute renal injury markers and viral
shedding.</p></li>
<li><p>CRRT in patients with CRF is standard therapy irrespective of
CoVID-19 status; it will be important to compare clinical parameters
of these patients (n=5) with virus-naïve CRF patients (none in this
study) to make any meaningful conclusions.</p></li>
</ul>
<h3 data-number="10.22.4" id="significance-18"><span class="header-section-number">10.22.4</span> Significance</h3>
<ul>
<li><p>This study argues that renal impairment is uncommon in CoVID-19 and
not associated with high mortaility, in stark contrast with a
concurrent study <span class="citation" data-cites="1Acv5UMSt">[<a href="#ref-1Acv5UMSt" role="doc-biblioref">764</a>]</span>.
If supported by further studies, this argues kidney impairment is
secondary to cytokine storm/inflammation-induced organ failure, and
not due to direct viral replication.</p></li>
<li><p>Will be important to comprehensively characterize large-datasets of
CoVID-19 patients to conclude if kidney function actively disrupted
due to viral infection.</p></li>
</ul>
<h3 data-number="10.22.5" id="credit-21"><span class="header-section-number">10.22.5</span> Credit</h3>
<p><em>Review by Samarth Hegde as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.23" id="potential-t-cell-and-b-cell-epitopes-of-2019-ncov"><span class="header-section-number">10.23</span> Potential T-cell and B-Cell Epitopes of 2019-nCoV</h2>
<p><span class="citation" data-cites="hdSafq1o">[<a href="#ref-hdSafq1o" role="doc-biblioref">787</a>]</span></p>
<h3 data-number="10.23.1" id="keywords-22"><span class="header-section-number">10.23.1</span> Keywords</h3>
<ul>
<li>COVID-19</li>
<li>vaccine</li>
<li>epitopes</li>
<li>spike protein</li>
<li>MHC-I</li>
<li>MHC-II</li>
<li>neutralizing antibodies</li>
<li>ACE2</li>
</ul>
<h3 data-number="10.23.2" id="main-findings-21"><span class="header-section-number">10.23.2</span> Main Findings</h3>
<p>The authors use 2 neural network algorithms, NetMHCpan4 and
MARIA, to identify regions within the COVID-19 genome that are
presentable by HLA. They identify 405 viral epitopes that are
presentable on MHC-I and MHC-II and validate using known epitopes from
SARS-CoV. To determine whether immune surveillance drives viral
mutations to evade MHC presentation, the authors analyzed 68 viral
genomes from 4 continents. They identified 93 point mutations that
occurred preferentially in regions predicted to be presented by MHC-I
(p=0.02) suggesting viral evolution to evade CD8 T-cell mediated
killing. 2 nonsense mutations were also identified that resulted in loss
of presentation of an associated antigen (FGDSVEEVL) predicted to be
good antigen for presentation across multiple HLA alleles.</p>
<p>To identify potential sites of neutralizing antibody binding, the
authors used homology modeling to the SARS-CoV’s spike protein (S
protein) to determine the putative structure of the CoV2 spike protein.
They used Discotope2 to identify antibody binding sites on the protein
surface in both the down and up conformations of the S protein. The
authors validate this approach by first identifying antibody binding
site in SARS-CoV S protein. In both the down and up conformation of the
CoV2 S protein, the authors identified a potential antibody binding site
on the S protein receptor binding domain (RBD) of the ACE2 receptor
(residues 440-460, 494-506). While RBDs in both SARS-CoV and CoV2 spike
proteins may be important for antibody binding, the authors note that
SARS-CoV has larger attack surfaces than CoV2. These results were later
validated on published crystal structures of the CoV2 S protein RBD and
human ACE2. Furthermore, analysis of 68 viral genomes did not identify
any mutations in this potential antibody binding site in CoV2.</p>
<p>Finally, the authors compile a list of potential peptide vaccine
candidates across the viral genome that can be presented by multiple HLA
alleles. Several of the peptides showed homology to SARS-CoV T-cell and
B-cell epitopes.</p>
<h3 data-number="10.23.3" id="limitations-22"><span class="header-section-number">10.23.3</span> Limitations</h3>
<p>While the authors used computational methods of validation,
primarily through multiple comparisons to published SARS-CoV structures
and epitopes, future work should include experimental validation of
putative T-cell and B-cell epitopes.</p>
<h3 data-number="10.23.4" id="significance-19"><span class="header-section-number">10.23.4</span> Significance</h3>
<p>The authors identified potential T-cell and B-cell epitopes
that may be good candidates for peptide based vaccines against CoV2.
They also made interesting observations in comparing SARS-CoV and CoV2
potential antibody binding sites, noting that SARS-CoV had larger attack
surfaces for potential neutralizing antibody binding. One of the
highlights of this paper was the authors’ mutation analysis of 68 viral
genomes from 4 continents. This analysis not only validated their
computational method for identifying T-cell epitopes, but showed that
immune surveillance likely drives viral mutation in MHC-I binding
peptides. The smaller attack surface may point to potential mechanisms
of immune evasion by CoV2. However, absence of mutations in the RBD of
CoV2 and the small number of mutations in peptides presentable to T
cells suggests that vaccines against multiple epitopes could still
elicit robust immunity against CoV2.</p>
<h3 data-number="10.23.5" id="credit-22"><span class="header-section-number">10.23.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.24" id="structure-function-and-antigenicity-of-the-sarscov-2-spike-glycoprotein"><span class="header-section-number">10.24</span> Structure, Function, and Antigenicity of the SARSCoV-2 Spike Glycoprotein</h2>
<p>Walls et al. <em>bioRxiv</em>. <span class="citation" data-cites="8xnxgpag">[<a href="#ref-8xnxgpag" role="doc-biblioref">788</a>]</span> now <span class="citation" data-cites="qcVbT0w4">[<a href="#ref-qcVbT0w4" role="doc-biblioref">62</a>]</span></p>
<h3 data-number="10.24.1" id="keywords-23"><span class="header-section-number">10.24.1</span> Keywords</h3>
<ul>
<li>binding affinity</li>
<li>antigenicity</li>
<li>neutralizing antibody</li>
</ul>
<h3 data-number="10.24.2" id="main-findings-22"><span class="header-section-number">10.24.2</span> Main Findings</h3>
<p>The authors highlight a human angiotensin-converting enzyme 2
(hACE2), as a potential receptor used by the current Severe Acute
respiratory syndrome coronavirus-2 (SARS-CoV-2) as a host factor that
allows the virus target human cells. This virus-host interaction
facilitates the infection of human cells with a high affinity comparable
with SARS-CoV. The authors propose this mechanism as a probable
explanation of the efficient transmission of SARS-CoV-2 between humans.
Besides, Walls and colleagues described SARS-CoV-2 S glycoprotein S by
Cryo-EM along with neutralizing polyclonal response against SAR-CoV-2 S
from mice immunized with SAR-CoV and blocking SAR-CoV-2 S-mediated entry
into VeroE6 infected cells.**</p>
<h3 data-number="10.24.3" id="limitations-23"><span class="header-section-number">10.24.3</span> Limitations</h3>
<p>The SARS-CoV-2 depends on the cell factors ACE2 and TMPRSS2, this
last, according to a recent manuscript by Markus Hoffman et al., <em>Cell</em>,
2020. The authors used green monkey (VeroE6) and hamster (BHK) cell
lines in the experiments to drive its conclusions to humans; however, it
is well known the caucasian colon adenocarcinoma human cell line
(CaCo-2), highly express the hACE2 receptor as the TMPRSS2 protease as
well. In humans, ACE2 protein is highly expressed in the
gastrointestinal tract, which again, makes the CaCo-2 cell line suitable
for the following SARS-CoV-2 studies.</p>
<h3 data-number="10.24.4" id="significance-20"><span class="header-section-number">10.24.4</span> Significance</h3>
<p>The results propose a functional receptor used by SARS-CoV-2 to
infect humans worldwide and defining two distinct conformations of spike
(S) glycoprotein by cryogenic electron microscopy (Cryo-EM). This study
might help establish a precedent for initial drug design and treatment
of the current global human coronavirus epidemic.</p>
<h3 data-number="10.24.5" id="credit-23"><span class="header-section-number">10.24.5</span> Credit</h3>
<p><em>Review by postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.25" id="breadth-of-concomitant-immune-responses-underpinning-viral-clearance-and-patient-recovery-in-a-non-severe-case-of-covid-19"><span class="header-section-number">10.25</span> Breadth of concomitant immune responses underpinning viral clearance and patient recovery in a non-severe case of COVID-19</h2>
<p>Thevarajan et al. <em>medRxiv</em> <span class="citation" data-cites="QdimT8mN">[<a href="#ref-QdimT8mN" role="doc-biblioref">789</a>]</span></p>
<h3 data-number="10.25.1" id="keywords-24"><span class="header-section-number">10.25.1</span> Keywords</h3>
<ul>
<li>IgG</li>
<li>IgM</li>
<li>TfH cells</li>
<li>NK cells</li>
<li>SNP</li>
</ul>
<h3 data-number="10.25.2" id="main-findings-23"><span class="header-section-number">10.25.2</span> Main Findings</h3>
<p>The authors characterized the immune response in peripheral blood of a
47-year old COVID-19 patient.</p>
<p>SARS-CoV2 was detected in nasopharyngeal swab, sputum and faeces
samples, but not in urine, rectal swab, whole blood or throat swab. 7
days after symptom onset, the nasopharyngeal swab test turned negative,
at day 10 the radiography infiltrates were cleared and at day 13 the
patient became asymptomatic.</p>
<p>Immunofluorescence staining shows from day 7 the presence of
<strong>COVID-19-binding IgG and IgM</strong> antibodies in plasma, that increase
until day 20.</p>
<p>Flow cytometry on whole blood reveals a plasmablast peak at day 8, a
gradual increase in T follicular helper cells, stable HLA-DR<sup>+</sup> NK
frequencies and decreased monocyte frequencies compared to healthy
counterparts. The expression of CD38 and HLA-DR peaked on T cells at D9
and was associated with higher production of cytotoxic mediators by
CD8<sup>+</sup> T cells.</p>
<p>IL-6 and IL-8 were undetectable in plasma.</p>
<p>The authors further highlight the presence of the <strong>IFITM3
SNP-rs12252-C/C variant</strong> in this patient, which is associated with
higher susceptibility to influenza virus.</p>
<h3 data-number="10.25.3" id="limitations-24"><span class="header-section-number">10.25.3</span> Limitations</h3>
<p>These results need to be confirmed in additional patients.</p>
<p>COVID-19 patients have increased infiltration of macrophages in their
lungs <span class="citation" data-cites="tVkKuIJt">[<a href="#ref-tVkKuIJt" role="doc-biblioref">790</a>]</span>. Monitoring monocyte proportions in blood earlier in the
disease might help to evaluate their eventual migration to the lungs.</p>
<p>The stable concentration of HLA-DR<sup>+</sup> NK cells in blood from day 7 is
not sufficient to rule out NK cell activation upon SARS-CoV2 infection.
In response to influenza A virus, NK cells express higher levels of
activation markers CD69 and CD38, proliferate better and display higher
cytotoxicity <span class="citation" data-cites="87CJN2CG">[<a href="#ref-87CJN2CG" role="doc-biblioref">791</a>]</span>. Assessing these parameters in COVID-19 patients is
required to better understand NK cell role in clearing this infection.</p>
<p>Neutralization potential of the COVID-19-binding IgG and IgM antibodies
should be assessed in future studies.</p>
<p>This patient was able to clear the virus, while presenting a SNP
associated with severe outcome following influenza infection. The
association between this SNP and outcome upon SARS-CoV2 infection should
be further investigated.</p>
<h3 data-number="10.25.4" id="significance-21"><span class="header-section-number">10.25.4</span> Significance</h3>
<p>This study is among the first to describe the appearance of
COVID-19-binding IgG and IgM antibodies upon infection. The emergence of
new serological assays might contribute to monitor more precisely the
seroconversion kinetics of COVID-19 patients <span class="citation" data-cites="yJW1RVQb">[<a href="#ref-yJW1RVQb" role="doc-biblioref">318</a>]</span>. Further association
studies between IFITM3 SNP-rs12252-C/C variant and clinical data might
help to refine the COVID-19 outcome prediction tools.</p>
<h3 data-number="10.25.5" id="credit-24"><span class="header-section-number">10.25.5</span> Credit</h3>
<p><em>Review by Bérengère Salomé as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.26" id="the-landscape-of-lung-bronchoalveolar-immune-cells-in-covid-19-revealed-by-single-cell-rna-sequencing"><span class="header-section-number">10.26</span> The landscape of lung bronchoalveolar immune cells in COVID-19 revealed by single-cell RNA sequencing</h2>
<p>Liao et al. <em>medRxiv</em> <span class="citation" data-cites="tVkKuIJt">[<a href="#ref-tVkKuIJt" role="doc-biblioref">790</a>]</span></p>
<h3 data-number="10.26.1" id="keywords-25"><span class="header-section-number">10.26.1</span> Keywords</h3>
<ul>
<li>COVID-19</li>
<li>SARS-CoV-2</li>
<li>Broncho-alveolar lavage</li>
<li>macrophages</li>
<li>NK cells</li>
<li>T cells</li>
<li>cytokine storm</li>
<li>scRNAseq</li>
</ul>
<h3 data-number="10.26.2" id="main-findings-24"><span class="header-section-number">10.26.2</span> Main Findings</h3>
<p>The authors performed single-cell RNA sequencing (scRNAseq) on
bronchoalveolar lavage fluid (BAL) from 6 COVID-19 patients (n=3 mild
cases, n=3 severe cases). Data was compared to previously generated
scRNAseq data from healthy donor lung tissue (n=8).</p>
<p>Clustering analysis of the 6 patients revealed distinct immune cell
organization between mild and severe disease. Specifically, they found
that transcriptional clusters annotated as tissue resident alveolar
macrophages were strongly reduced while monocytes-derived FCN1<sup>+</sup>SPP1<sup>+</sup>
inflammatory macrophages dominated the BAL of patients with severe
COVID19 diseases. They show that inflammatory macrophages upregulated
interferon-signaling genes, monocytes recruiting chemokines including
CCL2, CCL3, CCL4 as well as IL-6, TNF, IL-8 and profibrotic cytokine
TGF-β, while alveolar macrophages expressed lipid metabolism genes, such
as PPARG.</p>
<p>The lymphoid compartment was overall enriched in lungs from patients.
Clonally expanded CD8 T cells were enriched in mild cases suggesting
that CD8 T cells contribute to viral clearance as in Flu infection,
whereas proliferating T cells were enriched in severe cases.</p>
<p>SARS-CoV-2 viral transcripts were detected in severe patients, but
considered here as ambient contaminations.</p>
<h3 data-number="10.26.3" id="limitations-25"><span class="header-section-number">10.26.3</span> Limitations</h3>
<p>These results are based on samples from 6 patients and should therefore
be confirmed in the future in additional patients. Longitudinal
monitoring of BAL during disease progression or resolution would have
been most useful.</p>
<p>The mechanisms underlying the skewing of the macrophage compartment in
patients towards inflammatory macrophages should be investigated in
future studies.</p>
<p>Deeper characterization of the lymphoid subsets is required. The
composition of the “proliferating” cluster and how these cells differ
from conventional T cell clusters should be assessed. NK and CD8 T cell
transcriptomic profile, in particular the expression of cytotoxic
mediator and immune checkpoint transcripts, should be compared between
healthy and diseased lesions.</p>
<h3 data-number="10.26.4" id="significance-22"><span class="header-section-number">10.26.4</span> Significance</h3>
<p>COVID-19 induces a robust inflammatory cytokine storm in patients that
contributes to severe lung tissue damage and ARDS <span class="citation" data-cites="vasHgjGH">[<a href="#ref-vasHgjGH" role="doc-biblioref">792</a>]</span>. Accumulation of
monocyte-derived inflammatory macrophages at the expense of Alveolar
macrophages known to play an anti-inflammatory role following
respiratory viral infection, in part through the PPARγ pathway <span class="citation" data-cites="9GEyqic3 WBN5f37b">[<a href="#ref-9GEyqic3" role="doc-biblioref">793</a>,<a href="#ref-WBN5f37b" role="doc-biblioref">794</a>]</span> are
likely contributing to lung tissue injuries. These data suggest that
reduction of monocyte accumulation in the lung tissues could help
modulate COVID-19-induced inflammation. Further analysis of lymphoid
subsets is required to understand the contribution of adaptive immunity
to disease outcome.</p>
<h3 data-number="10.26.5" id="credit-25"><span class="header-section-number">10.26.5</span> Credit</h3>
<p><em>Review by Bérengère Salomé and Assaf Magen as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.27" id="can-routine-laboratory-tests-discriminate-2019-novel-coronavirus-infected-pneumonia-from-other-community-acquired-pneumonia"><span class="header-section-number">10.27</span> Can routine laboratory tests discriminate 2019 novel coronavirus infected pneumonia from other community-acquired pneumonia?</h2>
<p>Pan et al. <em>medRxiv</em> <span class="citation" data-cites="FOVDaThy">[<a href="#ref-FOVDaThy" role="doc-biblioref">795</a>]</span></p>
<h3 data-number="10.27.1" id="keywords-26"><span class="header-section-number">10.27.1</span> Keywords</h3>
<ul>
<li>Routine laboratory testing</li>
</ul>
<h3 data-number="10.27.2" id="main-findings-25"><span class="header-section-number">10.27.2</span> Main Findings</h3>
<p>In an attempt to use standard laboratory testing for the discrimination
between “Novel Coronavirus Infected Pneumonia” (NCIP) and a usual
community acquired pneumonia (CAP), the authors compared laboratory
testing results of 84 NCIP patients with those of a historical group of
316 CAP patients from 2018 naturally COVID-19 negative. The authors
describe significantly lower white blood- as well as red blood- and
platelet counts in NCIP patients. When analyzing differential blood
counts, lower absolute counts were measured in all subsets of NCIP
patients. With regard to clinical chemistry parameters, they found
increased AST and bilirubin in NCIP patients as compared to CAP
patients.</p>
<h3 data-number="10.27.3" id="limitations-26"><span class="header-section-number">10.27.3</span> Limitations</h3>
<p>The authors claim to describe a simple method to rapidly assess a
pre-test probability for NCIP. However, the study has substantial
weakpoints. The deviation in clinical laboratory values in NCIP patients
described here can usually be observed in severely ill patients. The
authors do not comment on how severely ill the patients tested here were
in comparison to the historical control. Thus, the conclusion that the
tests discriminate between CAP and NCIP lacks justification.</p>
<h3 data-number="10.27.4" id="significance-23"><span class="header-section-number">10.27.4</span> Significance</h3>
<p>The article strives to compare initial laboratory testing results in
patients with COVID-19 pneumonia as compared to patients with a usual
community acquired pneumonia. The implications of this study for the
current clinical situation seem restricted due to a lack in clinical
information and the use of a control group that might not be
appropriate.</p>
<h3 data-number="10.27.5" id="credit-26"><span class="header-section-number">10.27.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.28" id="correlation-analysis-between-disease-severity-and-inflammation-related-parameters-in-patients-with-covid-19-pneumonia"><span class="header-section-number">10.28</span> Correlation Analysis Between Disease Severity and Inflammation-related Parameters in Patients with COVID-19 Pneumonia</h2>
<p><span class="citation" data-cites="1Du40IOOH">[<a href="#ref-1Du40IOOH" role="doc-biblioref">796</a>]</span></p>
<h3 data-number="10.28.1" id="keywords-27"><span class="header-section-number">10.28.1</span> Keywords</h3>
<ul>
<li>cytokine</li>
<li>COVID-19 pneumonia</li>
<li>severity</li>
<li>disease progression</li>
</ul>
<h3 data-number="10.28.2" id="main-findings-26"><span class="header-section-number">10.28.2</span> Main Findings</h3>
<p>This study is a cross-sectional analysis of 100 patients with
COVID-19 pneumonia, divided into mild (n = 34), severe (n = 34), and
critical (n = 32) disease status based on clinical definitions.</p>
<p>The criteria used to define disease severity are as follows:</p>
<ol type="1">
<li><p><em>Severe</em> – any of the following: respiratory distress or respiratory
rate ≥ 30 respirations/minute; oxygen saturation ≤ 93% at rest;
oxygen partial pressure (PaO2)/oxygen concentration (FiO2) in
arterial blood ≤ 300mmHg, progression of disease on imaging
to &gt;50% lung involvement in the short term.</p></li>
<li><p><em>Critical</em> – any of the following: respiratory failure that requires
mechanical ventilation; shock; other organ failure that requires
treatment in the ICU.</p></li>
<li><p>Patients with pneumonia who test positive for COVID-19 who do not
have the symptoms delineated above are considered <em>mild</em>.</p></li>
</ol>
<p>Peripheral blood inflammatory markers were correlated to disease status.
Disease severity was significantly associated with levels of IL-2R,
IL-6, IL-8, IL-10, TNF-α, CRP, ferroprotein, and procalcitonin. Total
WBC count, lymphocyte count, neutrophil count, and eosinophil count were
also significantly correlated with disease status. Since this is a
retrospective, cross-sectional study of clinical laboratory values,
these data may be extrapolated for clinical decision making, but without
studies of underlying cellular causes of these changes this study does
not contribute to a deeper understanding of SARS-CoV-2 interactions with
the immune system.</p>
<p>It is also notable that the mean age of patients in the mild group was
significantly different from the mean ages of patients designated as
severe or critical (p &lt; 0.001). The mean patient age was not
significantly different between the severe and critical groups. However,
IL-6, IL-8, procalcitonin (Table 2), CRP, ferroprotein (Figure 3A, 3B),
WBC count, and neutrophil count (Figure 4A, 4B) were all significantly
elevated in the critical group compared to severe. These data suggest
underlying differences in COVID-19 progression that is unrelated to age.</p>
<h3 data-number="10.28.3" id="significance-24"><span class="header-section-number">10.28.3</span> Significance</h3>
<p>Given the inflammatory profile outlined in this study,
patients who have mild or severe COVID-19 pneumonia, who <em>also</em> have any
elevations in the inflammatory biomarkers listed above, should be
closely monitored for potential progression to critical status.</p>
<h3 data-number="10.28.4" id="credit-27"><span class="header-section-number">10.28.4</span> Credit</h3>
<p><em>This review by JJF was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.29" id="an-effective-ctl-peptide-vaccine-for-ebola-zaire-based-on-survivors-cd8-targeting-of-a-particular-nucleocapsid-protein-epitope-with-potential-implications-for-covid-19-vaccine-design"><span class="header-section-number">10.29</span> An Effective CTL Peptide Vaccine for Ebola Zaire Based on Survivors’ CD8+ Targeting of a Particular Nucleocapsid Protein Epitope with Potential Implications for COVID-19 Vaccine Design</h2>
<p>Herst et al. <em>bioRxiv</em> <span class="citation" data-cites="1DTLMIDoM">[<a href="#ref-1DTLMIDoM" role="doc-biblioref">797</a>]</span></p>
<h3 data-number="10.29.1" id="keywords-28"><span class="header-section-number">10.29.1</span> Keywords</h3>
<ul>
<li>Peptide vaccine</li>
<li>Ebolavirus</li>
<li>nucleocapsid</li>
<li>epitope</li>
<li>vaccine design</li>
<li>microsphere</li>
</ul>
<h3 data-number="10.29.2" id="main-findings-27"><span class="header-section-number">10.29.2</span> Main Findings</h3>
<p>Vaccination of mice with a single dose of a 9-amino-acid peptide NP44-52
located in a conserved region of ebolavirus (EBOV) nucleocapsid protein
(NP) confers CD8+ T-cell-mediated immunity against mouse adapted EBOV
(maEBOV). Bioinformatic analyses predict multiple conserved CD8+ T cell
epitopes in the SARS-CoV-2 NP, suggesting that a similar approach may be
feasible for vaccine design against SARS-CoV-2.</p>
<p>The authors focus on a site within a 20-peptide region of EBOV NP which
was commonly targeted by CD8+ T cells in a group of EBOV survivors
carrying the HLA-A*30:01:01 allele. To justify the testing of specific
vaccine epitopes in a mouse challenge setting, the authors cite known
examples of human pathogen-derived peptide antigens that are also
recognized by C57BL/6 mice, as well as existing data surrounding known
mouse immunogenicity of peptides related to this EBOV NP region. Testing
3 distinct 9mer peptides over an 11 amino-acid window and comparing to
vaccination with the 11mer with a T-cell reactivity readout demonstrated
that optimizing peptide length and position for immunogenicity may be
crucial, likely due to suboptimal peptide processing and MHC-class-I
loading.</p>
<p>Vaccines for maEBOV challenge studies were constructed by packaging
NP44-52 in d,l poly(lactic-co-glycolic) acid microspheres. CpG was also
packaged within the microspheres, while Monophosphoryl Lipid A (a TLR4
ligand) was added to the injectate solution. A second peptide consisting
of a predicted MHC-II epitope from the EBOV VG19 protein was added using
a separate population of microspheres, and the formulation was injected
by intraperitoneal administration. The vaccine was protective against a
range of maEBOV doses up to at least 10,000 PFU. Survival was
anticorrelated with levels of IL6, MCP-1 (CCL2), IL9, and GM-CSF, which
recapitulated trends seen in human EBOV infection.</p>
<p>While HLA-A*30:01:01 is only present in a minority of humans, the
authors state that MHC binding algorithms predict NP44-52 to be a strong
binder of a set of more common HLA-A*02 alleles. The authors predict
that a peptide vaccine based on the proposed formulation could elicit
responses in up to 50% of people in Sudan or 30% of people in North
America.</p>
<p>SARS-CoV-2 NP, meanwhile, has conserved regions which may provide
peptide-vaccine candidates. Scanning the SARS-CoV-2 NP sequence for
HLA-binding 9mers identified 53 peptides with predicted binding affinity
&lt; 500nM, including peptides that are predicted to bind to HLA-class-I
alleles of 97% of humans, 7 of which have previously been tested
<em>in-vitro</em>.</p>
<p>The results support previously appreciated correlations between certain
cytokines and disease severity, specifically IL6 which relates to
multiple trial therapies. Prediction of HLA-class-I binding of
SARS-CoV-2 NP peptides suggests the plausibility of a peptide vaccine
targeting conserved regions of SARS-CoV-2 NP although further validation
in previously infected patient samples will be essential.</p>
<h3 data-number="10.29.3" id="credit-28"><span class="header-section-number">10.29.3</span> Credit</h3>
<p><em>Review by Andrew M. Leader as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.30" id="epitope-based-peptide-vaccines-predicted-against-novel-coronavirus-disease-caused-by-sars-cov-2"><span class="header-section-number">10.30</span> Epitope-based peptide vaccines predicted against novel coronavirus disease caused by SARS-CoV-2</h2>
<p>Li et al. <em>bioRxiv</em>. <span class="citation" data-cites="9spK49tt">[<a href="#ref-9spK49tt" role="doc-biblioref">798</a>]</span></p>
<h3 data-number="10.30.1" id="keywords-29"><span class="header-section-number">10.30.1</span> Keywords</h3>
<ul>
<li>SARS-CoV-2</li>
<li>immune-informatics</li>
<li>vaccine design</li>
<li>T cell epitope</li>
<li>B cell epitope</li>
</ul>
<h3 data-number="10.30.2" id="main-findings-28"><span class="header-section-number">10.30.2</span> Main Findings</h3>
<p>This study employs a series of bioinformatic
pipelines to identify T and B cell epitopes on spike (S) protein of
SARS-CoV-2 and assess their properties for vaccine potential. To
identify B cell epitopes, they assessed structural accessibility,
hydrophilicity, and beta-turn and flexibility which are all factors that
promote their targeting by antibodies. To identify T cell epitopes, they
filtered for peptides with high antigenicity score and capacity to bind
3 or more MHC alleles. Using the protein digest server, they also
demonstrated that their identified T and B cell epitopes are stable,
having multiple non-digesting enzymes per epitope. Epitopes were also
determined to be non-allergenic and non-toxin as assessed by Allergen FP
1.0 and ToxinPred, respectively. For T cell epitopes, they assessed the
strength of epitope-HLA interaction via PepSite. Overall, they predict
four B cell and eleven T cell epitopes (two MHC I and nine MHC II
binding) to pass stringent computational thresholds as candidates for
vaccine development. Furthermore, they performed sequence alignment
between all identified SARS-CoV-2 S protein mutations and predicted
epitopes, and showed that the epitopes are conserved across 134 isolates
from 38 locations worldwide. However, they report that these conserved
epitopes may soon become obsolete given the known mutation rate of
related SARS-CoV is estimated to be 4x10-4/site/year, underscoring the
urgency of anti-viral vaccine development.</p>
<h3 data-number="10.30.3" id="limitations-27"><span class="header-section-number">10.30.3</span> Limitations</h3>
<p>While spike (S) protein may have a
critical role in viral entry into host cells and their epitope
prediction criterion were comprehensive, this study did not examine
other candidate SARS-CoV-2 proteins. This point is particularly
important given that a single epitope may not be sufficient to induce
robust immune memory, and recent approaches involve multi-epitope
vaccine design. Furthermore, their study only included a direct
implementation of various published methods, but did not validate
individual bioinformatic tools with controls to demonstrate robustness.
Finally, it is critical that these predicted epitopes are experimentally
validated before any conclusions can be drawn about their potential as
vaccine candidates or their clinical efficacy.</p>
<h3 data-number="10.30.4" id="significance-25"><span class="header-section-number">10.30.4</span> Significance</h3>
<p>This study provides a computational framework to
rapidly identify epitopes that may serve as potential vaccine candidates
for treating SARS-CoV-2.</p>
<h3 data-number="10.30.5" id="credit-29"><span class="header-section-number">10.30.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.31" id="the-definition-and-risks-of-cytokine-release-syndrome-like-in-11-covid-19-infected-pneumonia-critically-ill-patients-disease-characteristics-and-retrospective-analysis"><span class="header-section-number">10.31</span> The definition and risks of Cytokine Release Syndrome-Like in 11 COVID-19-Infected Pneumonia critically ill patients: Disease Characteristics and Retrospective Analysis</h2>
<p>Wang Jr. et al. <em>medRxiv</em>.
<span class="citation" data-cites="qg1bmQng">[<a href="#ref-qg1bmQng" role="doc-biblioref">799</a>]</span></p>
<h3 data-number="10.31.1" id="keywords-30"><span class="header-section-number">10.31.1</span> Keywords</h3>
<ul>
<li>Cytokine release syndrome (CRS)</li>
<li>biomarkers</li>
<li>ARDS</li>
<li>IL-6</li>
<li>lymphopenia</li>
</ul>
<h3 data-number="10.31.2" id="main-findings-29"><span class="header-section-number">10.31.2</span> Main Findings</h3>
<p>This study describes the occurrence of a cytokine release syndrome-like
(CRSL) toxicity in ICU patients with COVID-19 pneumonia. The median time
from first symptom to acute respiratory distress syndrome (ARDS) was 10
days. All patients had decreased CD3, CD4 and CD8 cells, and a
significant increase of serum IL-6. Furthermore, 91% had decreased NK
cells. The changes in IL-6 levels preceded those in CD4 and CD8 cell
counts. All of these parameters correlated with the area of pulmonary
inflammation in CT scan images. Mechanical ventilation increased the
numbers of CD4 and CD8 cells, while decreasing the levels of IL-6, and
improving the immunological parameters.</p>
<h3 data-number="10.31.3" id="limitations-28"><span class="header-section-number">10.31.3</span> Limitations</h3>
<p>The number of patients included in this retrospective single center
study is small (n=11), and the follow-up period very short (25 days).
Eight of the eleven patients were described as having CRSL, and were
treated by intubation (7) or ECMO (2). Nine patients were still in the
intensive care unit at the time of publication of this article, so their
disease outcome is unknown.</p>
<h3 data-number="10.31.4" id="significance-26"><span class="header-section-number">10.31.4</span> Significance</h3>
<p>The authors define a cytokine release syndrome-like toxicity in patients
with COVID-19 with clinical radiological and immunological criteria: 1)
decrease of circulating CD4, CD8 and NK cells; 2) substantial increase
of IL-6 in peripheral blood; 3) continuous fever; 4) organ and tissue
damage. This event seems to occur very often in critically ill patients
with COVID-19 pneumonia. Interestingly, the increase of IL-6 in the
peripheral blood preceded other laboratory alterations, thus, IL-6 might
be an early biomarker for the severity of COVID-19 pneumonia. The
manuscript will require considerable editing for organization and
clarity.</p>
<h3 data-number="10.31.5" id="credit-30"><span class="header-section-number">10.31.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.32" id="clinical-characteristics-of-36-non-survivors-with-covid-19-in-wuhan-china"><span class="header-section-number">10.32</span> Clinical characteristics of 36 non-survivors with COVID-19 in Wuhan, China</h2>
<p>Huang et al. <em>medRxiv</em>. <span class="citation" data-cites="YODgfrtM">[<a href="#ref-YODgfrtM" role="doc-biblioref">800</a>]</span></p>
<h3 data-number="10.32.1" id="keywords-31"><span class="header-section-number">10.32.1</span> Keywords</h3>
<ul>
<li>Clinical Characteristics</li>
<li>Non Survivors</li>
<li>retrospective study</li>
</ul>
<h3 data-number="10.32.2" id="main-findings-30"><span class="header-section-number">10.32.2</span> Main Findings</h3>
<p>This is a simple study reporting clinical
characteristics of patients who did not survive COVID-19. All patients
(mean age=69.22 years) had acute respiratory distress syndrome (ARDS)
and their median time from onset to ARDS was 11 days. The median time
from onset to death was 17 days. Most patients were older male (70%
male) with co-morbidities and only 11 % were smokers. 75% patients
showed bilateral pneumonia. Many patients had chronic diseases,
including hypertension (58.33%). cardiovascular disease (22.22%) and
diabetes (19.44%). Typical clinical feature measured in these patients
includes lymphopenia and elevated markers of inflammation.</p>
<h3 data-number="10.32.3" id="limitations-29"><span class="header-section-number">10.32.3</span> Limitations</h3>
<p>As noted by the authors, the conclusions of this study are
very limited because this is single-centered study focusing on a small
cohort of patients who did not survive. Many clinical parameters
observed by the authors (such* as increase levels of serum CRP, PCT,
IL-6) have also been described in other COVID19 patients who survived
the infection</p>
<h3 data-number="10.32.4" id="significance-27"><span class="header-section-number">10.32.4</span> Significance</h3>
<p>This study is essentially descriptive and may be useful for
clinical teams monitoring COVID19 patients.</p>
<h3 data-number="10.32.5" id="credit-31"><span class="header-section-number">10.32.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.33" id="risk-factors-related-to-hepatic-injury-in-patients-with-corona-virus-disease-2019"><span class="header-section-number">10.33</span> Risk Factors Related to Hepatic Injury in Patients with Corona Virus Disease 2019</h2>
<p><span class="citation" data-cites="5CGbn9Jj">[<a href="#ref-5CGbn9Jj" role="doc-biblioref">801</a>]</span></p>
<h3 data-number="10.33.1" id="keywords-32"><span class="header-section-number">10.33.1</span> Keywords</h3>
<ul>
<li>COVID-related Hepatic Injury</li>
</ul>
<h3 data-number="10.33.2" id="main-findings-31"><span class="header-section-number">10.33.2</span> Main Findings</h3>
<p>Based on a retrospective study of 85 hospitalized COVID
patients in a Beijing hospital, authors showed that patients with
elevated ALT levels (n = 33) were characterized by significantly higher
levels of lactic acid and CRP as well as lymphopenia and hypoalbuminemia
compared to their counterparts with normal ALT levels. Proportion of
severe and critical patients in the ALT elevation group was
significantly higher than that of normal ALT group. Multivariate
logistic regression performed on clinical factors related to ALT
elevation showed that CRP <span class="math inline">\(\geq\)</span> 20mg/L and low lymphocyte count
(&lt;1.1*10^9 cells/L) were independently related to ALT elevation—a
finding that led the authors to suggest cytokine storm as a major
mechanism of liver damage.</p>
<h3 data-number="10.33.3" id="limitations-30"><span class="header-section-number">10.33.3</span> Limitations</h3>
<p>The article’s most attractive claim that liver damage seen in COVID
patients is caused by cytokine storm (rather than direct infection of
the liver) hinges solely on their multivariate regression analysis.
Without further mechanistic studies a) demonstrating how high levels of
inflammatory cytokines can induce liver damage and b) contrasting types
of liver damage incurred by direct infection of the liver vs.
system-wide elevation of inflammatory cytokines, their claim remains
thin. It is also worth noting that six of their elevated ALT group
(n=33) had a history of liver disease (i.e. HBV infection, alcoholic
liver disease, fatty liver) which can confound their effort to pin down
the cause of hepatic injury to COVID.</p>
<h3 data-number="10.33.4" id="significance-28"><span class="header-section-number">10.33.4</span> Significance</h3>
<p>Limited. This article confirms a rich body
of literature describing liver damage and lymphopenia in COVID patients.</p>
<h3 data-number="10.33.5" id="credit-32"><span class="header-section-number">10.33.5</span> Credit</h3>
<p><em>Review by Chang Moon as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.34" id="detectable-serum-sars-cov-2-viral-load-rnaaemia-is-closely-associated-with-drastically-elevated-interleukin-6-il-6-level-in-critically-ill-covid-19-patients"><span class="header-section-number">10.34</span> Detectable serum SARS-CoV-2 viral load (RNAaemia) is closely associated with drastically elevated interleukin 6 (IL-6) level in critically ill COVID-19 patients</h2>
<p><span class="citation" data-cites="T0BDaXkw">[<a href="#ref-T0BDaXkw" role="doc-biblioref">802</a>]</span></p>
<h3 data-number="10.34.1" id="keywords-33"><span class="header-section-number">10.34.1</span> Keywords</h3>
<ul>
<li>ARDS</li>
<li>interleukin-6 (IL-6)</li>
<li>procalcitonin (PCT)</li>
<li>pro-inflammatory cytokines</li>
<li>SARS-CoV-2 RNAaemia</li>
</ul>
<h3 data-number="10.34.2" id="main-findings-32"><span class="header-section-number">10.34.2</span> Main Findings</h3>
<p>48 adult patients diagnosed with Covid19 according to Chinese guidelines
for Covid19 diagnosis and treatment version 6 were included in this
study. Patients were further sub-divided into three groups based on
clinical symptoms and disease severity: (1) mild, positive Covid19 qPCR
with no or mild clinical symptoms (fever; respiratory; radiological
abnormalities); (2) severe, at least one of the following: shortness of
breath/respiratory rate &gt;30/min, oxygen saturation SaO<sub>2</sub>&lt;93%,
Horowitz index paO2/FiO<sub>2</sub> &lt; 300 mmHg (indicating moderate pulmonary
damage); and (3) critically ill, at least one additional complicating
factor: respiratory failure with need for mechanical ventilation;
systemic shock; multi-organ failure and transfer to ICU. Serum samples
and throat-swaps were collected from all 48 patients enrolled.
SARS-CoV-2 RNA was assessed by qPCR with positive results being defined
as Ct values &lt; 40, and serum interleukin-6 (IL-6) was quantified
using a commercially available detection kit. Briefly, patient
characteristics in this study confirm previous reports suggesting that
higher age and comorbidities are significant risk factors of clinical
severity. Of note, 5 out of 48 of patients (10.41%), all in the
critically ill category, were found to have detectable serum SARS-CoV-2
RNA levels, so-called RNAaemia. Moreover, serum IL-6 levels in these
patients were found to be substantially higher and this correlated with
the presence of detectable SARS-CoV-2 RNA levels. The authors
hypothesize that viral RNA might be released from acutely damages
tissues in moribund patients during the course of Covid19 and that
RNaemia along with IL-6 could potentially be used as a prognostic
marker.</p>
<h3 data-number="10.34.3" id="limitations-31"><span class="header-section-number">10.34.3</span> Limitations</h3>
<p>While this group’s report generally confirms some of the major findings
of a more extensive study, published in early February 2020, <span class="citation" data-cites="vasHgjGH">[<a href="#ref-vasHgjGH" role="doc-biblioref">792</a>]</span>,
there are limitations that should be taken into account. First, the
number of patients enrolled is relatively small; second, interpretation
of these data would benefit from inclusion of information about study
specifics as well as providing relevant data on the clinical course of
these patients other than the fact that some were admitted to ICU (i.e.
demographics on how many patients needed respiratory support, dialysis,
APACHE Ii/III or other standard ICU scores as robust prognostic markers
for mortality etc). It also remains unclear at which time point the
serum samples were taken, i.e. whether at admission, when the diagnosis
was made or during the course of the hospital stay (and potentially
after onset of therapy, which could have affected both IL-6 and RNA
levels). The methods section lacks important information on the qPCR
protocol employed, including primers and cycling conditions used. From a
technical point of view, Ct values &gt;35 seem somewhat non-specific
(although Ct &lt;40 was defined as the CDC cutoff as well) indicating
that serum RNA levels are probably very low, therefore stressing the
need for highly specific primers and high qPCR efficiency. In addition,
the statistical tests used (t-tests, according to the methods section)
do not seem appropriate as the organ-specific data such as BUN and
troponin T values seem to be not normally distributed across groups (n=
5 RNAaemia+ vs. n= 43 RNAaemia-). Given the range of standard deviations
and the differences in patient sample size, it is difficult to believe
that these data are statistically significantly different.</p>
<h3 data-number="10.34.4" id="significance-29"><span class="header-section-number">10.34.4</span> Significance</h3>
<p>This study is very rudimentary and lacks a lot of relevant clinical
details. However, it corroborates some previously published observations
regarding RNAemia and IL-6 by another group. Generally, regarding future
studies, it would be important to address the question of IL-6 and other
inflammatory cytokine dynamics in relation to Covid19 disease kinetics
(high levels of IL-6, IL-8 and plasma leukotriene were shown to have
prognostic value at the onset of ARDS ; serum IL-2 and IL-15 have been
associated with mortality; reviewed by Chen W &amp; Ware L, Clin Transl Med.
2015 <span class="citation" data-cites="l3CzJ5ni">[<a href="#ref-l3CzJ5ni" role="doc-biblioref">803</a>]</span>).</p>
<h3 data-number="10.34.5" id="credit-33"><span class="header-section-number">10.34.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.35" id="lymphopenia-predicts-disease-severity-of-covid-19-a-descriptive-and-predictive-study"><span class="header-section-number">10.35</span> Lymphopenia predicts disease severity of COVID-19: a descriptive and predictive study</h2>
<p><span class="citation" data-cites="uIwTzZkJ">[<a href="#ref-uIwTzZkJ" role="doc-biblioref">804</a>]</span></p>
<h3 data-number="10.35.1" id="keywords-34"><span class="header-section-number">10.35.1</span> Keywords</h3>
<ul>
<li>Lymphopenia</li>
</ul>
<h3 data-number="10.35.2" id="main-findings-33"><span class="header-section-number">10.35.2</span> Main Findings</h3>
<p>Based on a retrospective study of 162 COVID patients from a
local hospital in Wuhan, China, the authors show an inverse correlation
between lymphocyte % (LYM%) of patients and their disease severity. The
authors have also tracked LYM% of 70 cases (15 deaths; 15 severe; 40
moderate) throughout the disease progression with fatal cases showing no
recovery of lymphocytes ( &lt;5%) even after 17-19 days post-onset. The
temporal data of LYM % in COVID patients was used to construct a
Time-Lymphocyte% model which is used to categorize and predict patients’
disease severity and progression. The model was validated using 92
hospitalized cases and kappa statistic test was used to assess agreement
between predicted disease severity and the assigned clinical severity (k
= 0.49).</p>
<h3 data-number="10.35.3" id="limitations-32"><span class="header-section-number">10.35.3</span> Limitations</h3>
<p>Time-Lymphocyte % Model (TLM) that authors have proposed as
a predictive model for clinical severity is very simple in its
construction and derives from correlative data of 162 patients. In order
for the model to be of use, it needs validation using a far more robust
data set and possibly a mechanistic study on how COVID leads to
lymphopenia in the first place. In addition, it should be noted that no
statistical test assessing significance of LYM % values between disease
severities was performed.</p>
<h3 data-number="10.35.4" id="significance-30"><span class="header-section-number">10.35.4</span> Significance</h3>
<p>This article is of limited significance as
it simply reports similar descriptions of COVID patients made in
previous literature that severe cases are characterized by lymphopenia.</p>
<h3 data-number="10.35.5" id="credit-34"><span class="header-section-number">10.35.5</span> Credit</h3>
<p><em>Review by Chang Moon as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.36" id="the-potential-role-of-il-6-in-monitoring-severe-case-of-coronavirus-disease-2019"><span class="header-section-number">10.36</span> The potential role of IL-6 in monitoring severe case of coronavirus disease 2019</h2>
<p>Liu et al. <em>medRxiv</em>. <span class="citation" data-cites="1GMxHWnxN">[<a href="#ref-1GMxHWnxN" role="doc-biblioref">805</a>]</span></p>
<h3 data-number="10.36.1" id="keywords-35"><span class="header-section-number">10.36.1</span> Keywords</h3>
<ul>
<li>Cytokine Release Syndrome</li>
<li>lymphocytopenia</li>
<li>IL-6</li>
<li>CRP</li>
<li>COVID19</li>
<li>pneumonia</li>
</ul>
<h3 data-number="10.36.2" id="main-findings-34"><span class="header-section-number">10.36.2</span> Main Findings</h3>
<p>Study on blood biomarkers on 80 COVID19 patients (69 severe
and 11 non-severe). Patients with severe symptoms at admission
(baseline) showed obvious lymphocytopenia and significantly increased
interleukin-6 (IL-6) and CRP, which was positively correlated with
symptoms severity. IL-6 at baseline positively correlates with CRP, LDH,
ferritin and D-Dimer abundance in blood.</p>
<p>Longitudinal analysis of 30 patients (before and after treatment) showed
significant reduction of IL-6 in remission cases.</p>
<h3 data-number="10.36.3" id="limitations-33"><span class="header-section-number">10.36.3</span> Limitations</h3>
<p>Limited sample size at baseline, especially for the
non-severe leads to question on representativeness. The longitudinal
study method is not described in detail and suffers from
non-standardized treatment. Limited panel of pro-inflammatory cytokine
was analyzed. Patients with severe disease show a wide range of altered
blood composition and biomarkers of inflammation, as well as differences
in disease course (53.6% were cured, about 10% developed acute
respiratory distress syndrome). The authors comment on associations
between IL-6 levels and outcomes, but these were not statistically
significant (maybe due to the number of patients, non-standardized
treatments, etc.) and data is not shown. Prognostic biomarkers could
have been better explored. Study lacks multivariate analysis.</p>
<h3 data-number="10.36.4" id="significance-31"><span class="header-section-number">10.36.4</span> Significance</h3>
<p>IL-6 could be used as a pharmacodynamic
marker of disease severity. Cytokine Release Syndrome (CRS) is a
well-known side effect for CAR-T cancer therapy and there are several
effective drugs to manage CRS. Drugs used to manage CRS could be tested
to treat the most severe cases of COVID19.</p>
<h3 data-number="10.36.5" id="credit-35"><span class="header-section-number">10.36.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.37" id="clinical-and-laboratory-profiles-of-75-hospitalized-patients-with-novel-coronavirus-disease-2019-in-hefei-china"><span class="header-section-number">10.37</span> Clinical and Laboratory Profiles of 75 Hospitalized Patients with Novel Coronavirus Disease 2019 in Hefei, China</h2>
<p>Zhao et al. <em>medRxiv</em>. <span class="citation" data-cites="1B0vBwv1q">[<a href="#ref-1B0vBwv1q" role="doc-biblioref">806</a>]</span></p>
<h3 data-number="10.37.1" id="keywords-36"><span class="header-section-number">10.37.1</span> Keywords</h3>
<ul>
<li>Routine laboratory testing</li>
</ul>
<h3 data-number="10.37.2" id="main-findings-35"><span class="header-section-number">10.37.2</span> Main Findings</h3>
<p>The authors of this study provide a comprehensive analysis of clinical
laboratory assessments in 75 patients (median age 47 year old)
hospitalized for Corona virus infection in China measuring differential
blood counts including T-cell subsets (CD4, CD8), coagulation function,
basic blood chemistry, of infection-related biomarkers including CRP,
Procalcitonin (PCT) (Precursor of calcitonin that increases during
bacterial infection or tissue injury), IL-6 and erythrocyte
sedimentation rate as well as clinical parameters. Among the most common
hematological changes they found increased neutrophils, reduced CD4 and
CD8 lymphocytes, increased LDH, CRP and PCT</p>
<p>When looking at patients with elevated IL-6, the authors describe
significantly reduced CD4 and CD8 lymphocyte counts and elevated CRP and
PCT levels were significantly increased in infected patients suggesting
that increased IL-6 may correlate well with disease severity in COVID-19
infections</p>
<h3 data-number="10.37.3" id="limitations-34"><span class="header-section-number">10.37.3</span> Limitations</h3>
<p>The authors performed an early assessment of clinical standard
parameters in patients infected with COVID-19. Overall, the number of
cases (75) is rather low and the snapshot approach does not inform about
dynamics and thus potential relevance in the assessment of treatment
options in this group of patients.</p>
<h3 data-number="10.37.4" id="significance-32"><span class="header-section-number">10.37.4</span> Significance</h3>
<p>The article summarizes provides a good summary of some of the common
changes in immune cells inflammatory cytokines in patients with a
COVID-19 infection and. Understanding how these changes can help predict
severity of disease and guide therapy including IL-6 cytokine receptor
blockade using Tocilizumab or Sarilumab will be important to explore.</p>
<h3 data-number="10.37.5" id="credit-36"><span class="header-section-number">10.37.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.38" id="exuberant-elevation-of-ip-10-mcp-3-and-il-1ra-during-sars-cov-2-infection-is-associated-with-disease-severity-and-fatal-outcome"><span class="header-section-number">10.38</span> Exuberant elevation of IP-10, MCP-3 and IL-1ra during SARS-CoV-2 infection is associated with disease severity and fatal outcome</h2>
<p>Yang et al. <em>medRxiv</em> <span class="citation" data-cites="RyXNw87v">[<a href="#ref-RyXNw87v" role="doc-biblioref">807</a>]</span></p>
<h3 data-number="10.38.1" id="keywords-37"><span class="header-section-number">10.38.1</span> Keywords</h3>
<ul>
<li>cytokine</li>
<li>IP-10</li>
<li>MCP-3</li>
<li>IL-1Ra</li>
<li>lymphocyte</li>
<li>neutrophil</li>
<li>stratification</li>
<li>disease severity</li>
<li>viral load</li>
<li>lung function</li>
<li>complications</li>
<li>clinical data</li>
</ul>
<h3 data-number="10.38.2" id="summary-3"><span class="header-section-number">10.38.2</span> Summary</h3>
<p>Plasma cytokine analysis (48 cytokines) was
performed on COVID-19 patient plasma samples, who were sub-stratified as
severe (N=34), moderate (N=19), and compared to healthy controls (N=8).
Patients were monitored for up to 24 days after illness onset: viral
load (qRT-PCR), cytokine (multiplex on subset of patients), lab tests,
and epidemiological/clinical characteristics of patients were reported.</p>
<h3 data-number="10.38.3" id="main-findings-36"><span class="header-section-number">10.38.3</span> Main Findings</h3>
<ul>
<li><p>Many elevated cytokines with COVID-19 onset compared to healthy
controls (IFNy, IL-1Ra, IL-2Ra, IL-6, IL-10, IL-18, HGF, MCP-3, MIG,
M-CSF, G-CSF, MIG-1a, and IP-10).</p></li>
<li><p>IP-10, IL-1Ra, and MCP-3 (esp. together) were associated with
disease severity and fatal outcome.</p></li>
<li><p>IP-10 was correlated to patient viral load (r=0.3006, p=0.0075).</p></li>
<li><p>IP-10, IL-1Ra, and MCP-3 were correlated to loss of lung function
(PaO<sub>2</sub>/FaO<sub>2</sub> (arterial/atmospheric O2) and Murray Score (lung
injury) with MCP-3 being the most correlated (r=0.4104 p&lt;0.0001
and r=0.5107 p&lt;0.0001 respectively).</p></li>
<li><p>Viral load (Lower Ct Value from qRT-PCR) was associated with
upregulated IP-10 only (not IL-1Ra or MCP-3) and was mildly
correlated with decreased lung function: PaO<sub>2</sub>/FaO<sub>2</sub>
(arterial/atmospheric O2) and Murray Score (lung injury).</p></li>
<li><p>Lymphopenia (decreased CD4 and CD8 T cells) and increased neutrophil
correlated w/ severe patients.</p></li>
<li><p>Complications were associated with COVID severity (ARDS, hepatic
insufficiency, renal insufficiency).</p></li>
</ul>
<h3 data-number="10.38.4" id="limitations-35"><span class="header-section-number">10.38.4</span> Limitations</h3>
<p>Collection time of clinical data and lab results
not reported directly (likely 4 days (2,6) after illness onset), making
it very difficult to determine if cytokines were predictive of patient
outcome or reflective of patient compensatory immune response (likely
the latter). Small N for cytokine analysis (N=2 fatal and N=5
severe/critical, and N=7 moderate or discharged). Viral treatment
strategy not clearly outlined.</p>
<h3 data-number="10.38.5" id="expanded-results"><span class="header-section-number">10.38.5</span> Expanded Results</h3>
<p><em>NOTE</em>: Moderate COVID-19 was classified by fever, respiratory
manifestations, and radiological findings consistent with pneumonia
while severe patients had one or more of the following: 1) respiratory
distraction, resting O2 saturation, or 3) arterial PaO2/FiO2 <em>&lt;</em> 300
mmHg.</p>
<p>Cytokine Results (Human Cytokine Screening Panel, Bio-Rad):</p>
<ul>
<li><p><strong>Significant elevation of cytokines observed in COVID patients
compared to healthy controls: IFNy, IL-1Ra, IL-2Ra, IL-6, IL-10,
IL-18, HGF, MCP-3, MIG, M-CSF, G-CSF, MIG-1a, and IP-10.</strong></p></li>
<li><p>Severity was correlated <strong>with increase in measured IP-10, MCP-3,
and IL-Ra</strong> as measure by area under the curve analysis during
sample timecourse (2-24 days after illness onset).</p></li>
<li><p>IL-1Ra incr. significant 0-7 days after onset, MCP-3 signif.
upregulated throughout observation timecourse, and IP-10 increased
and upregulated throughout (trending downwards over time).</p></li>
<li><p><strong>The three cytokines together (IP-10, IL-1Ra, and MCP-3 AUC) served
as the best predictors of disease deterioration and fatal outcome.</strong></p></li>
<li><p>No significant differences between moderate/severe observed between
groups in IL-2Ra, IL-6, IL-10, IL-18, CTACK, G-CSF, HGF, M-CSF,
MIP-1a, MIG, and IFNy at any timepoints.</p></li>
<li><p><strong>Viral load (Lower Ct Value from qRT-PCR) was associated with
upregulated IP-10 only (not IL-1Ra or MCP-3) and was highly
correlated with decreased lung function: PaO<sub>2</sub>/FaO<sub>2</sub>
(arterial/atmospheric O2) and Murray Score (lung injury).</strong></p></li>
<li><p><strong>Antibodies against these cytokines (esp. anti-IP-10) may serve as
a potential treatment for amelioration of COVID-19 (and associated
ARDS).</strong></p></li>
</ul>
<p>Lab results:</p>
<ul>
<li><p><strong>Decreased lymphocytes (%) in all patients – lymphopenia corr. w/
severe patients</strong></p>
<ul>
<li><strong>Decreased CD4 and CD8 T cells</strong> – no monocyte or
eosino/basophil % measured</li>
</ul></li>
<li><p><strong>Increased neutrophils (%)</strong></p></li>
<li><p>Increased BUN (mmol/L) – other kidney markers, liver markers, and
LDH were not significantly different between groups and were not
compared to healthy controls.</p></li>
</ul>
<p>Clinical features (between moderate vs. severe patient groups):</p>
<ul>
<li><p><strong>Complications were associated with severity (ARDS, hepatic
insufficiency, renal insufficiency).</strong></p></li>
<li><p>Coexisting conditions between groups were not significantly
different (chronic heart/lung/renal/liver disease, diabetes, or
cancer) and patient time courses (onset to admission and onset to
viral tx) also not significantly different – 4 days (2, 6) on
average for admission and 4 (3,7) for antiviral.</p></li>
<li><p>Increased corticosteroids and mechanical/ invasive mechanical
ventilation in severe patients.</p></li>
<li><p>Increased median age in severe group (Median (Range = 63.5 (42-74)
vs. 51 (22-78)) and patients <em>&gt;</em> 60 yrs had higher ratio of
severe patients as compared patients 16-59 yrs.</p></li>
<li><p>Higher incidence of fever is severe patients (91.2 vs. 68.4%),
myalgia (57.7 vs. 48.1%), and chill (17.6% vs. 0%).</p></li>
<li><p>No differences in cough, headache, nausea/vomiting, or diarrhea.</p></li>
</ul>
<h3 data-number="10.38.6" id="significance-33"><span class="header-section-number">10.38.6</span> Significance</h3>
<p>Outline of pathological time course (implicating innate
immunity esp.) and identification key cytokines associated with disease
severity and prognosis (+ comorbidities). Anti-IP-10 as a possible
therapeutic intervention (ex: Eldelumab).</p>
<h3 data-number="10.38.7" id="credit-37"><span class="header-section-number">10.38.7</span> Credit</h3>
<p><em>Review by Natalie Vaninov as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.39" id="antibody-responses-to-sars-cov-2-in-patients-of-novel-coronavirus-disease-2019"><span class="header-section-number">10.39</span> Antibody responses to SARS-CoV-2 in patients of novel coronavirus disease 2019</h2>
<p>Zhao Jr. et al. <em>medRxiv</em>. <span class="citation" data-cites="5HXuCa5t">[<a href="#ref-5HXuCa5t" role="doc-biblioref">808</a>]</span></p>
<h3 data-number="10.39.1" id="keywords-38"><span class="header-section-number">10.39.1</span> Keywords</h3>
<ul>
<li>SARS-CoV-2 IgG</li>
<li>seroconversion rate</li>
<li>total Ab</li>
<li>Ig</li>
<li>IgM</li>
</ul>
<h3 data-number="10.39.2" id="main-findings-37"><span class="header-section-number">10.39.2</span> Main Findings</h3>
<p>This study examined antibody responses in
the blood of COVID-19 patients during the early SARS CoV2 outbreak in
China. Total 535 plasma samples were collected from 173 patients (51.4%
female) and were tested for seroconversion rate using ELISA. Authors
also compared the sensitivity of RNA and antibody tests over the course
of the disease . The key findings are:</p>
<ul>
<li><p>Among 173 patients, the seroconversion rates for total antibody
(Ab), IgM and IgG were 93.1% (161/173), 82.7% (143/173) and 64.7%
(112/173), respectively.</p></li>
<li><p>The seroconversion sequentially appeared for Ab, IgM and then IgG,
with a median time of 11, 12 and 14 days, respectively. Overall, the
seroconversion of Ab was significantly quicker than that of IgM (p =
0.012) and IgG (p &lt; 0.001). Comparisons of seroconversion rates
between critical and non-critical patients did not reveal any
significant differences.</p></li>
<li><p>RNA tests had higher sensitivity in early phase and within 7 days of
disease onset than antibody assays (66.7% Vs 38.3% respectively).</p></li>
<li><p>The sensitivity of the Ab assays was higher 8 days after disease
onset, reached 90% at day 13 and 100% at later time points (15-39
days). In contrast, RNA was only detectable in 45.5% of samples at
days 15-39.</p></li>
<li><p>In patients with undetectable RNA in nasal samples collected during
day 1-3, day 4-7, day 8-14 and day 15-39 since disease onset, 28.6%
(2/7), 53.6% (15/28), 98.2% (56/57) and 100% (30/30) had detectable
total Ab titers respectively Combining RNA and antibody tests
significantly raised the sensitivity for detecting COVID-19 patients
in different stages of the disease (p &lt; 0.001).</p></li>
<li><p>There was a strong positive correlation between clinical severity
and antibody titer 2-weeks after illness onset.</p></li>
<li><p>Dynamic profiling of viral RNA and antibodies in representative
COVID-19 patients (n=9) since onset of disease revealed that
antibodies may not be sufficient to clear the virus. It should be
noted that increases in of antibody titers were not always
accompanied by RNA clearance.</p></li>
</ul>
<h3 data-number="10.39.3" id="limitations-36"><span class="header-section-number">10.39.3</span> Limitations</h3>
<p>Because different types of ELISA assays were used for
determining antibody concentrations at different time points after
disease onset, sequential seroconversion of total Ab, IgM and IgG may
not represent actual temporal differences but rather differences in the
affinities of the assays used. Also, due to the lack of blood samples
collected from patients in the later stage of illness, how long the
antibodies could last remain unknown. For investigative dynamics of
antibodies, more samples were required.</p>
<h3 data-number="10.39.4" id="significance-34"><span class="header-section-number">10.39.4</span> Significance</h3>
<p>Total and IgG antibody titers could be used to understand
the epidemiology of SARS CoV-2 infection and to assist in determining
the level of humoral immune response in patients.</p>
<p>The findings provide strong clinical evidence for routine serological
and RNA testing in the diagnosis and clinical management of COVID-19
patients. The understanding of antibody responses and their half-life
during and after SARS CoV2 infection is important and warrants further
investigations.</p>
<h3 data-number="10.39.5" id="credit-38"><span class="header-section-number">10.39.5</span> Credit</h3>
<p><em>This review was undertaken by Zafar Mahmood and edited by K Alexandropoulos as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.40" id="restoration-of-leukomonocyte-counts-is-associated-with-viral-clearance-in-covid-19-hospitalized-patients"><span class="header-section-number">10.40</span> Restoration of leukomonocyte counts is associated with viral clearance in COVID-19 hospitalized patients</h2>
<p>Chen et al. <em>medRxiv</em> <span class="citation" data-cites="uCyoHtV3">[<a href="#ref-uCyoHtV3" role="doc-biblioref">809</a>]</span></p>
<h3 data-number="10.40.1" id="keywords-39"><span class="header-section-number">10.40.1</span> Keywords</h3>
<ul>
<li>COVID-19</li>
<li>T cell</li>
<li>B cell</li>
<li>NK cell</li>
<li>IL-6</li>
<li>pro-calcitonin</li>
<li>cytokine storm</li>
</ul>
<h3 data-number="10.40.2" id="main-findings-38"><span class="header-section-number">10.40.2</span> Main Findings</h3>
<p>The authors collected data on 25 COVID-19 patients (n=11 men, n=14
women) using standard laboratory tests and flow cytometry. All patients
were treated with antibiotics. Twenty-four of the 25 patients were also
treated with anti-viral Umefinovir and 14 of the patients were treated
with corticosteroids. 14 patients became negative for the virus after
8-14 days of treatment. The same treatment course was extended to 15-23
days for patients who were still positive for the virus at day 14.</p>
<p>The authors found a negative association between age and resolution of
infection. Patients with hypertension, diabetes, malignancy or chronic
liver disease were all unable to clear the virus at day 14, though not
statistically significant.</p>
<p>Elevated procalcitonin and a trend for increased IL-6 were also found in
peripheral blood prior to the treatment.</p>
<p>A trend for lower NK cell, T cell and B cell counts in patients was also
reported. B cell, CD4 and CD8 T cell counts were only increased upon
treatment in patients who cleared the virus. NK cell frequencies
remained unchanged after treatment in all the patients.</p>
<h3 data-number="10.40.3" id="limitations-37"><span class="header-section-number">10.40.3</span> Limitations</h3>
<p>73% of the patients who remained positive for SARS-CoV2 after the 1<sup>st</sup>
treatment, and 43% of all patients who cleared the virus were treated
with corticosteroids. Corticosteroids have strong effects on the immune
compartment in blood <span class="citation" data-cites="L6kwccjn">[<a href="#ref-L6kwccjn" role="doc-biblioref">810</a>]</span>. The authors should have accounted for
corticosteroid treatment when considering changes in T, NK and B cell
frequencies.</p>
<p>Assessing if IL-6 concentrations were back to baseline levels following
treatment would have provided insights into the COVID-19 cytokine storm
biology. Patients with higher baseline levels of IL-6 have been reported
to have lower CD8 and CD4 T cell frequencies <span class="citation" data-cites="1B0vBwv1q">[<a href="#ref-1B0vBwv1q" role="doc-biblioref">806</a>]</span>. Correlating IL-6 with
cell counts before and after treatment would thus have also been of
interest. The report of the laboratory measures in table 2 is incomplete
and should include the frequencies of patients with increased/decreased
levels for each parameter.</p>
<p>Correction is needed for the 1<sup>st</sup> paragraph of the discussion as data
does not support NK cell restoration upon treatment in patients who
cleared the virus. NK cells remain unchanged after the 1<sup>st</sup> treatment
course and only seem to increase in 2 out of 6 donors after the 2<sup>nd</sup>
treatment course in those patients.</p>
<h3 data-number="10.40.4" id="significance-35"><span class="header-section-number">10.40.4</span> Significance</h3>
<p>Previous reports suggest an association between disease severity and
elevated IL-6 or pro-calcitonin concentrations in COVID-19
patients <span class="citation" data-cites="T0BDaXkw K9G9QT8y">[<a href="#ref-T0BDaXkw" role="doc-biblioref">802</a>,<a href="#ref-K9G9QT8y" role="doc-biblioref">811</a>]</span>. IL-6 receptor blockade is also being administered to
patients enrolled in clinical trials (NCT04317092). This report thus
contributes to highlight elevated concentrations of these analytes in
COVID-19 patients. Mechanisms underlying the association between viral
clearance and restoration of the T cell and B cell frequencies suggests
viral-driven immune dysregulation, which needs to be investigated in
further studies.</p>
<h3 data-number="10.40.5" id="credit-39"><span class="header-section-number">10.40.5</span> Credit</h3>
<p><em>Review by Bérengère Salomé as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.41" id="clinical-findings-in-critically-ill-patients-infected-with-sars-cov-2-in-guangdong-province-china-a-multi-center-retrospective-observational-study"><span class="header-section-number">10.41</span> Clinical findings in critically ill patients infected with SARS-CoV-2 in Guangdong Province, China: a multi-center, retrospective, observational study</h2>
<p>Xu et al. <em>medRxiv</em>. <span class="citation" data-cites="USOnp7c0">[<a href="#ref-USOnp7c0" role="doc-biblioref">812</a>]</span></p>
<h3 data-number="10.41.1" id="keywords-40"><span class="header-section-number">10.41.1</span> Keywords</h3>
<ul>
<li>clinical outcomes</li>
<li>prognosis</li>
<li>critically ill patients</li>
<li>ICU</li>
<li>lymphopenia</li>
<li>LDH</li>
</ul>
<h3 data-number="10.41.2" id="main-findings-39"><span class="header-section-number">10.41.2</span> Main Findings:</h3>
<p>This work analyses laboratory and clinical data from 45 patients treated
in the in ICU in a single province in China. Overall, 44% of the
patients were intubated within 3 days of ICU admission with only 1
death.</p>
<p>Lymphopenia was noted in 91% of patient with an inverse correlation with
LDH.</p>
<p>Lymphocyte levels are negatively correlated with Sequential Organ
Failure Assessment (SOFA) score (clinical score, the higher the more
critical state), LDH levels are positively correlated to SOFA score.
Overall, older patients (&gt;60yo), with high SOFA score, high LDH
levels and low lymphocytes levels at ICU admission are at higher risk of
intubation.</p>
<p>Of note, convalescent plasma was administered to 6 patients but due to
limited sample size no conclusion can be made.</p>
<h3 data-number="10.41.3" id="limitations-38"><span class="header-section-number">10.41.3</span> Limitations</h3>
<p>While the study offers important insights into disease course and
clinical lab correlates of outcome, the cohort is relatively small and
is likely skewed towards a less-severe population compared to other ICU
reports given the outcomes observed. Analysis of laboratory values and
predictors of outcomes in larger cohorts will be important to make
triage and treatment decisions. As with many retrospective analyses,
pre-infection data is limited and thus it is not possible to understand
whether lymphopenia was secondary to underlying comorbidities or
infection.</p>
<p>Well-designed studies are necessary to evaluate the effect of
convalescent plasma administration.</p>
<h3 data-number="10.41.4" id="significance-36"><span class="header-section-number">10.41.4</span> Significance</h3>
<p>This clinical data enables the identification of at-risk
patients and gives guidance for research for treatment options. Indeed,
further work is needed to better understand the causes of the
lymphopenia and its correlation with outcome.</p>
<h3 data-number="10.41.5" id="credit-40"><span class="header-section-number">10.41.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.42" id="immune-multi-epitope-vaccine-design-using-an-immunoinformatics-approach-for-2019-novel-coronavirus-in-china-sars-cov-2"><span class="header-section-number">10.42</span> Immune Multi-epitope vaccine design using an immunoinformatics approach for 2019 novel coronavirus in China (SARS-CoV-2)</h2>
<p><span class="citation" data-cites="14OLQIYK0">[<a href="#ref-14OLQIYK0" role="doc-biblioref">813</a>]</span></p>
<h3 data-number="10.42.1" id="keywords-41"><span class="header-section-number">10.42.1</span> Keywords</h3>
<ul>
<li>Vaccine</li>
<li>in silico</li>
<li>B cell epitopes</li>
<li>T cell epitopes</li>
</ul>
<h3 data-number="10.42.2" id="main-findings-40"><span class="header-section-number">10.42.2</span> Main Findings</h3>
<p>Using in silico bioinformatic tools, this study identified putative
antigenic B-cell epitopes and HLA restricted T-cell epitopes from the
spike, envelope and membrane proteins of SARS-CoV-2, based on the genome
sequence available on the NCBI database. T cell epitopes were selected
based on predicted affinity for the more common HLA-I alleles in the
Chinese population. Subsequently, the authors designed vaccine peptides
by bridging selected B-cell epitopes and adjacent T-cell epitopes.
Vaccine peptides containing only T-cell epitopes were also generated.</p>
<p>From 61 predicted B-cell epitopes, only 19 were exposed on the surface
of the virion and had a high antigenicity score. A total of 499 T-cell
epitopes were predicted. Based on the 19 B-cell epitopes and their 121
adjacent T-cell epitopes, 17 candidate vaccine peptides were designed.
Additionally, another 102 vaccine peptides containing T-cell epitopes
only were generated. Based on the epitope counts and HLA score, 13 of
those were selected. Thus, a total of 30 peptide vaccine candidates were
designed.</p>
<h3 data-number="10.42.3" id="limitations-39"><span class="header-section-number">10.42.3</span> Limitations</h3>
<p>While this study provides candidates for the development of vaccines
against SARS-CoV-2, in vitro and in vivo trials are required to validate
the immunogenicity of the selected B and T cell epitopes. This could be
done using serum and cells from CoV-2-exposed individuals, and in
preclinical studies. The implication of this study for the current
epidemic are thus limited. Nevertheless, further research on this field
is greatly needed.</p>
<h3 data-number="10.42.4" id="credit-41"><span class="header-section-number">10.42.4</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.43" id="clinical-features-of-patients-infected-with-the-2019-novel-coronavirus-covid-19-in-shanghai-china"><span class="header-section-number">10.43</span> Clinical Features of Patients Infected with the 2019 Novel Coronavirus (COVID-19) in Shanghai, China</h2>
<p>Cao et al. <em>medRxiv</em> <span class="citation" data-cites="12ilBRq6I">[<a href="#ref-12ilBRq6I" role="doc-biblioref">814</a>]</span></p>
<h3 data-number="10.43.1" id="keywords-42"><span class="header-section-number">10.43.1</span> Keywords</h3>
<ul>
<li>Disease severity</li>
<li>clinical features</li>
<li>laboratory abnormalities</li>
</ul>
<h3 data-number="10.43.2" id="main-findings-41"><span class="header-section-number">10.43.2</span> Main Findings</h3>
<p>This single-center cohort study analyzes the clinical and
laboratory features of 198 patients with confirmed COVID-19 infection in
Shanghai, China and correlated these parameters with clinical disease
severity, including subsequent intensive care unit (ICU) admission. 19
cases (9.5%) required ICU admission after developing respiratory failure
or organ dysfunction. Age, male sex, underlying cardiovascular disease,
and high symptom severity (high fever, dyspnea) were all significantly
correlated with ICU admission. Additionally, ICU admission was more
common in patients who presented with lymphopenia and elevated
neutrophil counts, among other laboratory abnormalities. Flow cytometric
analysis revealed that patients admitted to the ICU had significantly
reduced circulating CD3+ T cell, CD4+ T cell, CD8+ T cell, and CD45+
leukocyte populations compared to the cohort of patients not requiring
ICU admission.</p>
<h3 data-number="10.43.3" id="limitations-40"><span class="header-section-number">10.43.3</span> Limitations</h3>
<p>The limitations of this study include the relatively
small sample size and lack of longitudinal testing. The authors also did
not assess whether respiratory comorbidity – such as asthma or chronic
obstructive lung disease – in addition to immunosuppression affected ICU
admission likelihood.</p>
<h3 data-number="10.43.4" id="significance-37"><span class="header-section-number">10.43.4</span> Significance</h3>
<p>COVID-19 has already sickened thousands across the globe,
though the severity of these infections is markedly diverse, ranging
from mild symptoms to respiratory failure requiring maximal
intervention. Understanding what clinical, laboratory, and immunologic
factors predict the clinical course of COVID-19 infection permits
frontline providers to distribute limited medical resources more
effectively.</p>
<h3 data-number="10.43.5" id="credit-42"><span class="header-section-number">10.43.5</span> Credit</h3>
<p><em>Review by Andrew Charap as part of a project by students, postdocs and
faculty at the Immunology Institute of the Icahn School of Medicine at
Mount Sinai.</em></p>
<h2 data-number="10.44" id="serological-detection-of-2019-ncov-respond-to-the-epidemic-a-useful-complement-to-nucleic-acid-testing"><span class="header-section-number">10.44</span> Serological detection of 2019-nCoV respond to the epidemic: A useful complement to nucleic acid testing</h2>
<p>Zhang et al. <em>medRxiv</em>. <span class="citation" data-cites="IPJURDxi">[<a href="#ref-IPJURDxi" role="doc-biblioref">815</a>]</span></p>
<h3 data-number="10.44.1" id="keywords-43"><span class="header-section-number">10.44.1</span> Keywords</h3>
<ul>
<li>Immunoassay</li>
<li>serum IgM and IgG</li>
<li>specific antibodies</li>
</ul>
<h3 data-number="10.44.2" id="main-finding"><span class="header-section-number">10.44.2</span> Main Finding</h3>
<p>This study showed that both anti-2019-nCov IgM and IgG were detected by
automated chemiluminescent immunoassay in the patients who had been
already confirmed as positive by nucleic acid detection, while single
positivity of IgM or IgG were detected in a very few cases in the other
population including 225 non-COVID-19 cases. In addition to the increase
of anti-2019-nCov IgM 7-12 days after morbidity, the increase of IgG was
detected in three patients with COVID-19 within a very short of time
(0-1 day).</p>
<h3 data-number="10.44.3" id="limitations-41"><span class="header-section-number">10.44.3</span> Limitations</h3>
<p>The limitation of this study is only
3 confirmed COVID-19 cases were included, so that the relationship
between anti-2019-nCov antibodies and disease progression might not be
clearly defined. Another limitation is that they did not show the course
of 2019-nCov specific antibodies in the cases with positive for COVID-19
but without clinical symptoms.</p>
<h3 data-number="10.44.4" id="significance-38"><span class="header-section-number">10.44.4</span> Significance</h3>
<p>The detection of anti-2019-nCov antibodies can be an alternative method
to diagnose and treat COVID-19 more comprehensively by distinguish non
COVID-19 patients. It may be helpful to understand the course of
individual cases with COVID-19 to predict the prognosis if more cases
will be evaluated.</p>
<h3 data-number="10.44.5" id="credit-43"><span class="header-section-number">10.44.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.45" id="human-kidney-is-a-target-for-novel-severe-acute-respiratory-syndrome-coronavirus-2-sars-cov-2-infection"><span class="header-section-number">10.45</span> Human Kidney is a Target for Novel Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) Infection</h2>
<p><span class="citation" data-cites="H0nvocJz">[<a href="#ref-H0nvocJz" role="doc-biblioref">816</a>]</span></p>
<h3 data-number="10.45.1" id="keywords-44"><span class="header-section-number">10.45.1</span> Keywords</h3>
<ul>
<li>Kidney/Renal Failure</li>
<li>Macrophage Infiltration</li>
<li>Complement Activation</li>
</ul>
<h3 data-number="10.45.2" id="main-finding-1"><span class="header-section-number">10.45.2</span> Main Finding</h3>
<p>Analyzing the eGFR (effective glomerular flow rate) of 85 Covid-19
patients and characterizing tissue damage and viral presence in
post-mortem kidney samples from 6 Covid-19 patients, the authors
conclude that significant damage occurs to the kidney, following
Covid-19 infection. This is in contrast to the SARS infection from the
2003 outbreak. They determine this damage to be more prevalent in
patients older than 60 years old, as determined by analysis of eGFR. H&amp;E
and IHC analysis in 6 Covid-19 patients revealed that damage was in the
tubules, not the glomeruli of the kidneys and suggested that macrophage
accumulation and C5b-9 deposition are key to this process. </p>
<h3 data-number="10.45.3" id="limitations-42"><span class="header-section-number">10.45.3</span> Limitations</h3>
<p>Severe limitations include that the H&amp;E and IHC samples were performed
on post-mortem samples of unknown age, thus we cannot assess how/if age
correlates with kidney damage, upon Covid-19 infection. Additionally,
eGFR was the only <em>in-vivo</em> measurement. Blood urea nitrogen and
proteinuria are amongst other measurements that could have been obtained
from patient records. An immune panel of the blood was not performed to
assess immune system activation. Additionally, patients are only from
one hospital. </p>
<h3 data-number="10.45.4" id="significance-39"><span class="header-section-number">10.45.4</span> Significance</h3>
<p>This report makes clear that kidney damage is prevalent
in Covid-19 patients and should be accounted for. </p>
<h3 data-number="10.45.5" id="credit-44"><span class="header-section-number">10.45.5</span> Credit</h3>
<p><em>Review by Dan Fu Ruan, Evan Cody and Venu Pothula as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine at Mount Sinai.</em></p>
<h2 data-number="10.46" id="covid-19-early-warning-score-a-multi-parameter-screening-tool-to-identify-highly-suspected-patients"><span class="header-section-number">10.46</span> COVID-19 early warning score: a multi-parameter screening tool to identify highly suspected patients</h2>
<p>Song et al. <em>medRxiv</em>. <span class="citation" data-cites="KHNnjP7M">[<a href="#ref-KHNnjP7M" role="doc-biblioref">817</a>]</span></p>
<h3 data-number="10.46.1" id="keywords-45"><span class="header-section-number">10.46.1</span> Keywords</h3>
<ul>
<li>retrospective</li>
<li>electronic health records</li>
<li>blood counts</li>
<li>diagnostic</li>
<li>prognostic</li>
<li>modeling</li>
</ul>
<h3 data-number="10.46.2" id="main-findings-42"><span class="header-section-number">10.46.2</span> Main Findings</h3>
<p>The aim of this study was to identify diagnostic or prognostic criteria
which could identify patients with COVID-19 and predict patients who
would go on to develop severe respiratory disease. The authors use EMR
data from individuals taking a COVID-19 test at Zhejiang hospital, China
in late January/Early February. A large number of clinical parameters
were different between individuals with COVID-19 and also between
‘severe’ and ‘non-severe’ infections and the authors combine these into
a multivariate linear model to derive a weighted score, presumably
intended for clinical use.</p>
<h3 data-number="10.46.3" id="limitations-43"><span class="header-section-number">10.46.3</span> Limitations</h3>
<p>Unfortunately, the paper is lacking a lot of crucial information, making
it impossible to determine the importance or relevance of the findings.
Most importantly, the timings of the clinical measurements are not
described relative to the disease course, so it is unclear if the
differences between ‘severe’ and ‘non-severe’ infections are occurring
before progression to severe disease (which would make them useful
prognostic markers), or after (which would not).</p>
<h3 data-number="10.46.4" id="significance-40"><span class="header-section-number">10.46.4</span> Significance</h3>
<p>This paper is one of many retrospective studies coming from hospitals in
China studying individuals with COVID-19. Because of the sparse
description of the study design, this paper offers little new
information. However, studies like this could be very valuable and we
would strongly encourage the authors to revise this manuscript to
include more information about the timeline of clinical measurements in
relation to disease onset and more details of patient outcomes.</p>
<h3 data-number="10.46.5" id="credit-45"><span class="header-section-number">10.46.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.47" id="ly6e-impairs-coronavirus-fusion-and-confers-immune-control-of-viral-disease"><span class="header-section-number">10.47</span> LY6E impairs coronavirus fusion and confers immune control of viral disease</h2>
<p>Pfaender et al. <em>bioRxiv</em>. <span class="citation" data-cites="19xwRoatu">[<a href="#ref-19xwRoatu" role="doc-biblioref">818</a>]</span></p>
<h3 data-number="10.47.1" id="keywords-46"><span class="header-section-number">10.47.1</span> Keywords</h3>
<ul>
<li>interferon-stimulated genes</li>
<li>antiviral interferons</li>
<li>human coronaviruses (CoV)</li>
<li>murine hepatitis virus (MHV)</li>
</ul>
<h3 data-number="10.47.2" id="main-findings-43"><span class="header-section-number">10.47.2</span> Main Findings</h3>
<p>Screening a cDNA library of &gt;350 human interferon-stimulated genes
for antiviral activity against endemic human coronavirus HCoV-229E
(associated with the common cold), Pfaender S &amp; Mar K <em>et al.</em> identify
lymphocyte antigen 6 complex, locus E (Ly6E) as an inhibitor of cellular
infection of Huh7 cells, a human hepatoma cell line susceptible to
HCoV-229E and other coronaviruses. In a series of consecutive <em>in vitro</em>
experiments including both stable Ly6E overexpression and
CRISPR-Cas9-mediated knockout the authors further demonstrate that Ly6E
reduces cellular infection by various other coronaviruses including
human SARS-CoV and SARS-CoV-2 as well as murine CoV mouse hepatitis
virus (MHV). Their experiments suggest that this effect is dependent on
Ly6E inhibition of CoV strain-specific spike protein-mediated membrane
fusion required for viral cell entry.</p>
<p>To address the function of Ly6E <em>in vivo</em>, hematopoietic stem
cell-specific Ly6E knock-out mice were generated by breeding Ly6E<sup>fl/fl</sup>
mice (referred to as functional wild-type mice) with transgenic
<em>Vav-iCre</em> mice (offspring referred to as Ly6E HSC ko mice); wild-type
and Ly6E HSC ko mice of both sexes were infected intraperitoneally with
varying doses of the natural murine coronavirus MHV, generally causing a
wide range of diseases in mice including hepatitis, enteritis and
encephalomyelitis. Briefly, compared to wild-type controls, mice lacking
hematopoietic cell-expressed Ly6E were found to present with a more
severe disease phenotype as based on serum ALT levels (prognostic of
liver damage), liver histopathology, and viral titers in the spleen.
Moreover, bulk RNAseq analysis of infected liver and spleen tissues
indicated changes in gene expression pathways related to tissue damage
and antiviral immune responses as well as a reduction of genes
associated with type I IFN response and inflammation. Finally, the
authors report substantial differences in the numbers of hepatic and
splenic APC subsets between wild-type and knockout mice following MHV
infection and show that Ly6E-deficient B cells and to a lesser extent
also DCs are particularly susceptible to MHV infection <em>in vitro</em>.</p>
<h3 data-number="10.47.3" id="limitations-44"><span class="header-section-number">10.47.3</span> Limitations</h3>
<p>Experiments and data in this study are presented in an overall logical
and coherent fashion; however, some observations and the conclusions
drawn are problematic and should be further addressed &amp; discussed by the
authors. Methodological &amp; formal limitations include relatively low
replicate numbers as well as missing technical replicates for some <em>in
vitro</em> experiments (<em>cf</em>. Fig. legend 1; Fig. legend 2e); the omission
of “outliers” in Fig. legend 2 without an apparent rationale as to why
this approach was chosen; the lack of detection of actual Ly6E protein
levels in Ly6E HSC ko or wild-type mice; and most importantly, missing
information on RNAseq data collection &amp; analysis in the method section
and throughout the paper. A more relevant concern though is that the
interpretation of the experimental data presented and the language used
tend to overrate and at times overgeneralize findings: for example,
while the authors demonstrate statistically significant, Ly6E-mediated
reduction of coronavirus titers in stable cells lines <em>in vitro</em>, it
remains unclear whether a viral titer reduction by one log decade would
be of actual biological relevance in face of high viral titers <em>in
vivo</em>. After high-dose intraperitoneal MHV infection <em>in vivo</em>, early
viral titers in Ly6E HSC knockout vs. wt mice only showed an elevation
in the spleen (~1.5 log decades) but not liver of the ko mice (other
tissue not evaluated), and while ko mice presented with only modestly
increased liver pathology, both male and female ko mice exhibited
significantly higher mortality. Thus, the manuscript tile statement that
“Ly6E … confers immune control of viral disease” is supported by only
limited <em>in vivo</em> data, and gain-of-function experiments (eg. Ly6E
overexpression) were not performed. Of additional note here, tissue
tropism and virulence differ greatly among various MHV strains and
isolates whereas dose, route of infection, age, genetic background and
sex of the mice used may additionally affect disease outcome and
phenotype (<em>cf</em>. Taguchi F &amp; Hirai-Yuki A,
<a href="https://doi.org/10.3389/fmicb.2012.00068" class="uri">https://doi.org/10.3389/fmicb.2012.00068</a>; Kanolkhar A et al,
<a href="https://jvi.asm.org/content/%2083/18/9258">https://jvi.asm.org/content/
83/18/9258</a>). Observations
attributed to hematopoietic stem cell-specific Ly6E deletion could
therefore be influenced by the different genetic backgrounds of floxed
and cre mice used, and although it appears that littermates wt and ko
littermates were used in the experiments, the potentially decisive
impact of strain differences should at least have been discussed. Along
these lines, it should also be taken into account that the majority of
human coronaviruses cause respiratory symptoms, which follow a different
clinical course engaging other primary cellular mediators than the
hepatotropic murine MHV disease studied here. It therefore remains
highly speculative how the findings reported in this study will
translate to human disease and it would therefore be important to test
other routes of MHV infection and doses that have been described to
produce a more comparable phenotype to human coronavirus disease (<em>cf.</em>
Kanolkhar A et al, <a href="https://jvi.asm.org/content/%2083/18/9258">https://jvi.asm.org/content/
83/18/9258</a>). Another
important shortcoming of this study is the lack of any information on
functional deficits or changes in Ly6E-deficient immune cells and how
this might relate to the phenotype observed. Overall, the <em>in vitro</em>
experiments are more convincing than the <em>in vivo</em> studies which appear
somewhat limited.</p>
<h3 data-number="10.47.4" id="significance-41"><span class="header-section-number">10.47.4</span> Significance</h3>
<p>Despite some shortcomings, the experiments performed in this study
suggest a novel and somewhat unexpected role of Ly6E in the protection
against coronaviruses across species. These findings are of relevance
and should be further explored in ongoing research on potential
coronavirus therapies. Yet an important caveat pertains to the authors’
suggestion that “therapeutic mimicking of Ly6E action” may constitute a
first line of defense against novel coronaviruses since their own prior
work demonstrated that Ly6E can enhance rather than curtail infection
with influenza A and other viruses.</p>
<h3 data-number="10.47.5" id="credit-46"><span class="header-section-number">10.47.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.48" id="a-preliminary-study-on-serological-assay-for-severe-acute-respiratory-syndrome-coronavirus-2-sars-cov-2-in-238-admitted-hospital-patients"><span class="header-section-number">10.48</span> A preliminary study on serological assay for severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) in 238 admitted hospital patients</h2>
<p>Liu et al. <em>medRxiv</em>. <span class="citation" data-cites="574aSa8C">[<a href="#ref-574aSa8C" role="doc-biblioref">819</a>]</span></p>
<h3 data-number="10.48.1" id="keywords-47"><span class="header-section-number">10.48.1</span> Keywords</h3>
<ul>
<li>diagnosis</li>
<li>serological assay</li>
<li>ELISA</li>
<li>RT-PCR</li>
</ul>
<h3 data-number="10.48.2" id="main-findings-44"><span class="header-section-number">10.48.2</span> Main Findings</h3>
<p>While RT-PCR is being used currently to routinely diagnose infection
with SARS-CoV-2, there are significant limitations to the use of a
nucleic acid test that lead to a high false-negative rate. This
article describes ELISAs that can measure IgM and IgG antibodies
against the N protein of SARS-CoV-2 to test samples from 238
patients (153 positive by RT-PCR and 85 negative by RT-PCR) at
different times after symptom onset. The positivity rate of the IgM
and/or IgG ELISAs was greater than that of the RT-PCR (81.5%
compared to 64.3%) with similar positive rates in the confirmed and
suspected cases (83% and 78.8%, respectively), suggesting that many
of the suspected but RT-PCR-negative cases were also infected. The
authors also found that the ELISAs have higher positive rates later
after symptom onset while RT-PCR is more effective as a diagnostic
test early during the infection.</p>
<h3 data-number="10.48.3" id="limitations-45"><span class="header-section-number">10.48.3</span> Limitations</h3>
<p>I cannot identify any limitations to this study.</p>
<h3 data-number="10.48.4" id="significance-42"><span class="header-section-number">10.48.4</span> Significance</h3>
<p>The authors make a strong case for using a combination of ELISA and
RT-PCR for diagnosis of infection with SARS-CoV-2, especially
considering the dynamics of positivity rates of RT-PCR and ELISA.
Fewer false-negative diagnoses would improve infection control and
patient management.</p>
<h3 data-number="10.48.5" id="credit-47"><span class="header-section-number">10.48.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.49" id="monoclonal-antibodies-for-the-s2-subunit-of-spike-of-sars-cov-cross-react-with-the-newly-emerged-sars-cov-2"><span class="header-section-number">10.49</span> Monoclonal antibodies for the S2 subunit of spike of SARS-CoV cross-react with the newly-emerged SARS-CoV-2</h2>
<p><span class="citation" data-cites="NT3T9a8H">[<a href="#ref-NT3T9a8H" role="doc-biblioref">820</a>]</span></p>
<h3 data-number="10.49.1" id="keywords-48"><span class="header-section-number">10.49.1</span> Keywords</h3>
<ul>
<li>Coronavirus Disease 2019 (COVID-19)</li>
<li>SARS-CoV-2</li>
<li>Spike protein</li>
<li>Cross- reactive antibodies</li>
</ul>
<h3 data-number="10.49.2" id="main-findings-45"><span class="header-section-number">10.49.2</span> Main Findings</h3>
<p>Whole genome sequencing-based comparisons of the 2003
Severe Acute Respiratory Syndrome Coronavirus (SARS-CoV) and the 2019
SARS-CoV-2 revealed conserved receptor binding domain (RBD) and host
cell receptor, angiotensin-converting enzyme 2 (ACE2). In line with
this, the authors tested cross-reactivity of murine monoclonal
antibodies (mAbs) previously generated against the SARS-CoV spike (S)
glycoprotein involved in viral entry. One of the screened mAb, 1A9, was
able to bind and cross-neutralize multiple strains of SARS-CoV, as well
as, detect the S protein in SARS-CoV-2-infected cells. mAb 1A9 was
generated using an immunogenic fragment in the S2 subunit of SARS-CoV
and binds through a novel epitope within the S2 subunit at amino acids
1111-1130. It is important to note that CD8+ T lymphocyte epitopes
overlap with these residues, suggesting that S2 subunit could be
involved in inducing both, humoral and cell-mediated immunity.</p>
<h3 data-number="10.49.3" id="limitations-46"><span class="header-section-number">10.49.3</span> Limitations</h3>
<p>The authors used previously generated mouse mAbs
against the S protein in SARS-CoV expressed in mammalian cell line.
Future experimental validation using COVID-19 patient samples is needed
to validate these findings. In addition, the results of these studies
are predominantly based on in vitro experiments and so, evaluating the
effects of the mAb 1A9 in an animal model infected with this virus will
help us better understand the host immune responses in COVID-19 and
potential therapeutic vaccines.</p>
<h3 data-number="10.49.4" id="significance-43"><span class="header-section-number">10.49.4</span> Significance</h3>
<p>This study identified mAbs that recognize the new
coronavirus, SARS-Cov-2. These cross-reactive mAbs will help in
developing diagnostic assays for COVID-19.</p>
<h3 data-number="10.49.5" id="credit-48"><span class="header-section-number">10.49.5</span> Credit</h3>
<p><em>This review was undertaken by Tamar Plitt and Katherine Lindblad as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.50" id="mortality-of-covid-19-is-associated-with-cellular-immune-function-compared-to-immune-function-in-chinese-han-population"><span class="header-section-number">10.50</span> Mortality of COVID-19 is Associated with Cellular Immune Function Compared to Immune Function in Chinese Han Population</h2>
<p>Zeng et al. <em>medRxiv</em>. <span class="citation" data-cites="1GEbBtBgh">[<a href="#ref-1GEbBtBgh" role="doc-biblioref">821</a>]</span></p>
<h3 data-number="10.50.1" id="keywords-49"><span class="header-section-number">10.50.1</span> Keywords</h3>
<ul>
<li>WBC</li>
<li>peripheral blood</li>
<li>CD4</li>
<li>CD8 T cells</li>
</ul>
<h3 data-number="10.50.2" id="main-findings-46"><span class="header-section-number">10.50.2</span> Main Findings</h3>
<p>Retrospective study of the clinical characteristics of 752
patients infected with COVID-19 at Chinese PLA General Hospital, Peking
Union Medical College Hospital, and affiliated hospitals at Shanghai
University of medicine &amp; Health Sciences. This study is the first one
that compares PB from healthy controls from the same regions in Shanghai
and Beijing, and infected COVID-19 patients to standardize a reference
range of WBCs of people at high risk.</p>
<h3 data-number="10.50.3" id="limitations-47"><span class="header-section-number">10.50.3</span> Limitations</h3>
<p>Lower levels of leukocyte counts -B cells, CD4 and CD8 T
cells- correlated with mortality (WBCs are significantly lower in severe
or critical UCI patients vs mild ones). Based on 14,117 normal controls
in Chinese Han population (ranging in age from 18-86) it is recommended
that reference ranges of people at high risk of COVID-19 infection are
CD3+ lymphocytes below 900 cells/mm3, CD4+ lymphocytes below 500
cells/mm3, and CD8+ lymphocytes below 300 cells/mm3. Importantly, this
study also reported that the levels of D-dimer, C-reactive protein and
IL-6 were elevated in COVID-19 pts., indicating clot formation, severe
inflammation and cytokine storm.</p>
<h3 data-number="10.50.4" id="significance-44"><span class="header-section-number">10.50.4</span> Significance</h3>
<p>This study sets a threshold to identify patients at risk
by analyzing their levels of leukocytes, which is an easy and fast
approach to stratify individuals that require hospitalization. Although
the study is limited (only counts of WBC are analyzed and not its
profile) the data is solid and statistically robust to correlate levels
of lymphopenia with mortality.</p>
<h3 data-number="10.50.5" id="credit-49"><span class="header-section-number">10.50.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.51" id="retrospective-analysis-of-clinical-features-in-101-death-cases-with-covid-19"><span class="header-section-number">10.51</span> Retrospective Analysis of Clinical Features in 101 Death Cases with COVID-19</h2>
<p>Chen et al. <em>medRxiv</em>. <span class="citation" data-cites="WuHcOc9T">[<a href="#ref-WuHcOc9T" role="doc-biblioref">822</a>]</span></p>
<h3 data-number="10.51.1" id="keywords-50"><span class="header-section-number">10.51.1</span> Keywords</h3>
<ul>
<li>death biomarkers</li>
<li>cardiac damage</li>
<li>Troponin</li>
<li>Blood type</li>
<li>respiratory failure</li>
<li>hypertension</li>
</ul>
<h3 data-number="10.51.2" id="main-findings-47"><span class="header-section-number">10.51.2</span> Main Findings</h3>
<p>This is a retrospective study involving 101 death cases with COVID-19 in
Wuhan Jinyintan Hospital. The aim was to describe clinical,
epidemiological and laboratory features of fatal cases in order to
identify the possible primary mortality causes related to COVID-19.</p>
<p>Among 101 death cases, 56.44% were confirmed by RT-PCR and 43.6% by
clinical diagnostics. Males dominated the number of deaths and the
average age was 65.46 years. All patients died of respiratory failure
and multiple organs failure, except one (acute coronary syndrome). The
predominant comorbidities were hypertension (42.57%) and diabetes
(22.77%). 25.74% of the patients presented more than two underlying
diseases. 82% of patients presented myocardial enzymes abnormalities at
admission and further increase in myocardial damage indicators with
disease progression: patients with elevated Troponin I progressed faster
to death. Alterations in coagulation were also detected. Indicators of
liver and kidney damage increased 48 hours before death. The authors
studied the deceased patients’ blood type and presented the following
results: type A (44.44%), type B (29.29%), type AB (8.08%) and type O
(18.19%), which is inconsistent with the distribution in Han population
in Wuhan.</p>
<p>Clinical analysis showed that the most common symptom was fever (91.9%),
followed by cough and dyspnea. The medium time from onset of symptoms to
acute respiratory distress syndrome (ARDS) development was 12 days.
Unlike SARS, only 2 patients with COVID-19 had diarrhea. 98% presented
abnormal lung imaging at admission and most had double-lung
abnormalities. Related to the laboratorial findings some inflammatory
indicators gradually increased during the disease progression, such as
IL-6 secretion in the circulation, procalcitonin (PCT) and C-reactive
protein (CRP), while platelets numbers decreased. The authors also
reported an initial lymphopenia that was followed by an increase in the
lymphocytes numbers. Neutrophil count increased with disease
progression.</p>
<p>The patients received different treatments such as antiviral drugs
(60.40%), glucocorticoids, thymosin and immunoglobulins. All patients
received antibiotic treatment and some received antifungal drugs. All
patients received oxygen therapy (invasive or non-invasive ones).</p>
<h3 data-number="10.51.3" id="limitations-48"><span class="header-section-number">10.51.3</span> Limitations</h3>
<p>This study involves just fatal patients, lacking comparisons with other
groups of patients e.g. patients that recovered from COVID-19. The
authors didn’t discuss the different approaches used for treatments and
how these may affect the several parameters measured. The possible
relationship between the increase of inflammatory indicators and
morbidities of COVID-19 are not discussed.</p>
<h3 data-number="10.51.4" id="significance-45"><span class="header-section-number">10.51.4</span> Significance</h3>
<p>This study has the largest cohort of fatal cases reported so far. The
authors show that COVID-19 causes fatal respiratory distress syndrome
and multiple organ failure. This study highlights prevalent myocardial
damage and indicates that cardiac function of COVID-19 patients should
be carefully monitored. The data suggest that Troponin I should be
further investigated as an early indicator of patients with high risk of
accelerated health deterioration. Secondary bacterial and fungal
infections were frequent in critically ill patients and these need to be
carefully monitored in severe COVID-19 patients. Differences in blood
type distribution were observed, suggesting that type A is detrimental
while type O is protective – but further studies are needed to confirm
these findings and elucidate if blood type influences infection or
disease severity. Several inflammatory indicators (neutrophils, PCT, CRP
and IL-6, D-dimer) increased according to disease severity and should be
assessed as biomarkers and to better understand the biology of
progression to severe disease.</p>
<h3 data-number="10.51.5" id="credit-50"><span class="header-section-number">10.51.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.52" id="relationship-between-the-abo-blood-group-and-the-covid-19-susceptibility"><span class="header-section-number">10.52</span> Relationship between the ABO Blood Group and the COVID-19 Susceptibility</h2>
<p>Zhao et al. <em>medRxiv</em>. <span class="citation" data-cites="1DbV444Pj">[<a href="#ref-1DbV444Pj" role="doc-biblioref">823</a>]</span></p>
<h3 data-number="10.52.1" id="keywords-51"><span class="header-section-number">10.52.1</span> Keywords</h3>
<ul>
<li>ABO blood group</li>
<li>COVID-19 susceptibility</li>
</ul>
<h3 data-number="10.52.2" id="main-findings-48"><span class="header-section-number">10.52.2</span> Main Findings</h3>
<p>These authors compared the ABO blood group of 2,173 patients with
RT-PCR-confirmed COVID-19 from hospitals in Wuhan and Shenzhen with the
ABO blood group distribution in unaffected people in the same cities
from previous studies (2015 and 2010 for Wuhan and Shenzhen,
respectively). They found that people with blood group A are
statistically over-represented in the number of those infected and who
succumb to death while those with blood group O are statistically
underrepresented with no influence of age or sex.</p>
<h3 data-number="10.52.3" id="limitations-49"><span class="header-section-number">10.52.3</span> Limitations</h3>
<p>This study compares patients with COVID-19 to the general population
but relies on data published 5 and 10 years ago for the control. The
mechanisms that the authors propose may underlie the differences they
observed require further study.</p>
<h3 data-number="10.52.4" id="significance-46"><span class="header-section-number">10.52.4</span> Significance</h3>
<p>Risk stratification based on blood group may be beneficial for
patients and also healthcare workers in infection control. Additionally,
investigating the mechanism behind these findings could lead to better
developing prophylactic and therapeutic targets for COVID-19.</p>
<h3 data-number="10.52.5" id="credit-51"><span class="header-section-number">10.52.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.53" id="the-inhaled-corticosteroid-ciclesonide-blocks-coronavirus-rna-replication-by-targeting-viral-nsp15"><span class="header-section-number">10.53</span> The inhaled corticosteroid ciclesonide blocks coronavirus RNA replication by targeting viral NSP15</h2>
<p>Matsuyama et al. <em>bioRxiv</em> <span class="citation" data-cites="JwL9G6bd">[<a href="#ref-JwL9G6bd" role="doc-biblioref">824</a>]</span></p>
<h3 data-number="10.53.1" id="keywords-52"><span class="header-section-number">10.53.1</span> Keywords</h3>
<ul>
<li>Corticosteroids</li>
<li>ciclesonide</li>
<li>mometasone</li>
<li>NSP15</li>
<li>MERS-CoV</li>
</ul>
<h3 data-number="10.53.2" id="main-findings-49"><span class="header-section-number">10.53.2</span> Main Findings</h3>
<p>This study reconsiders the use of inhaled corticosteroids in the
treatment of pneumonia by coronavirus. Corticosteroids were associated
with increased mortality for SARS in 2003 and for MERS in 2013, probably
due to that fact that systemic corticosteroids suppress the innate
immune system, resulting in increased viral replication. However, some
steroid compounds might block coronavirus replication. The authors
screened steroids from a chemical library and assessed the viral growth
suppression and drug cytotoxicity. Ciclesonide demonstrated low
cytotoxicity and potent suppression of MERS-CoV viral growth. The
commonly used systemic steroids cortisone, prednisolone and
dexamethasone did not suppress viral growth, nor did the commonly used
inhaled steroid fluticasone. To identify the drug target of virus
replication, the authors conducted 11 consecutive MERS-CoV passages in
the presence of ciclesonide or mometasone, and they could generate a
mutant virus that developed resistance to ciclesonide, but not to
mometasone. Afterwards, they performed next-generation sequencing and
identified an amino acid substitution in nonstructural protein 15
(NSP15) as the predicted mechanism for viral resistance to ciclesonide.
The authors were able to successfully generate a recombinant virus
carrying that amino acid substitution, which overcome the antiviral
effect of ciclesonide, suggesting that ciclosenide interacts with NSP15.
The mutant virus was inhibited by mometasone, suggesting that the
antiviral target of mometasone is different from that of ciclesonide.
Lastly, the effects of ciclesonide and mometason on suppressing the
replication of SARS-CoV-2 were evaluated. Both compounds were found to
suppress viral replication with a similar efficacy to lopinavir.</p>
<h3 data-number="10.53.3" id="limitations-50"><span class="header-section-number">10.53.3</span> Limitations</h3>
<p>Most of the experiments, including the identification of the mutation in
NSP15 were conducted with MERS-CoV. This is not the closest related
virus to SARS-CoV-2, as that would be SARS-CoV. Thus, to repeat the
initial experiments with SARS-CoV, or preferably SARS-CoV-2, is
essential. The manuscript should address this and, therefore, it will
require considerable editing for organization and clarity. Also, in
terms of cell immunogenic epitopes, while SARS-CoV-2 spike protein
contains several predicted B and T cell immunogenic epitopes that are
shared with other coronaviruses, some studies have shown critical
differences between MERS-CoV, SARS-CoV and SARS-CoV-2. A main criticism
is that the authors only used VeroE6/TMPRSS2 cells to gauge the direct
cytotoxic effects of viral replication. To evaluate this in other cell
lines, including human airway epithelial cells, is crucial, as the
infectivity of coronavirus strains greatly varies in different cell
lines,</p>
<h3 data-number="10.53.4" id="significance-47"><span class="header-section-number">10.53.4</span> Significance</h3>
<p>Nevertheless, these findings encourage evaluating ciclesonide and
mometasone as better options for patients with COVID-19 in need of
inhaled steroids, especially as an alternative to other corticosteroids
that have been shown to increase viral replication in vitro. This should
be evaluated in future clinical studies.</p>
<h3 data-number="10.53.5" id="credit-52"><span class="header-section-number">10.53.5</span> Credit</h3>
<p><em>This review was undertaken by Alvaro Moreira, MD as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.54" id="a-human-monoclonal-antibody-blocking-sars-cov-2-infection"><span class="header-section-number">10.54</span> A human monoclonal antibody blocking SARS-CoV-2 infection **</h2>
<p>Wang et al. <em>bioRxiv</em>. <span class="citation" data-cites="XcuzhxrJ">[<a href="#ref-XcuzhxrJ" role="doc-biblioref">609</a>]</span></p>
<h3 data-number="10.54.1" id="keywords-53"><span class="header-section-number">10.54.1</span> Keywords</h3>
<ul>
<li>Monoclonal antibodies</li>
<li>SARS-CoV2</li>
<li>cross-neutralization</li>
<li>potential treatment</li>
<li>spike receptor</li>
</ul>
<h3 data-number="10.54.2" id="main-findings-50"><span class="header-section-number">10.54.2</span> Main Findings</h3>
<p>The authors reported a human monoclonal
antibody that neutralizes SARS-CoV-2 and SARS-Cov which belong to same
family of corona viruses. For identifying mAbs, supernatants of a
collection of 51 hybridomas raised against the spike protein of SARS-CoV
(SARS-S) were screened by ELISA for cross-reactivity against the spike
protein of SARS-CoN2 (SARS2-S). Hybridomas were derived from immunized
transgenic H2L2 mice (chimeric for fully human VH-VL and rat constant
region). Four SARS-S hybridomas displayed cross-reactivity with SARS2-S,
one of which (47D11) exhibited cross-neutralizing activity for SARS-S
and SARS2-S pseudotyped VSV infection. A recombinant, fully human IgG1
isotype antibody was generated and used for further characterization.</p>
<p>The humanized 47D11 antibody inhibited infection of VeroE6 cells with
SARS-CoV and SARS-CoV-2 with IC50 values of 0.19 and 0.57 μg/ml
respectively. 47D11 mAb bound a conserved epitope on the spike receptor
binding domain (RBD) explaining its ability to cross-neutralize SARS-CoV
and SARS-CoV-2. 47D11 was shown to target the S1B RBD of SARS-S and
SARS2-S with similar affinities. Interestingly, binding of 47D11 to
SARS-S1B and SARS2-S1B did not interfere with S1B binding to ACE2
receptor-expressing cells assayed by flow cytometry.</p>
<h3 data-number="10.54.3" id="limitations-51"><span class="header-section-number">10.54.3</span> Limitations</h3>
<p>These results show that the human 47D11 antibody
neutralizes SARS-CoV and SARS-Cov2 infectivity via an as yet unknown
mechanism that is different from receptor binding interference.
Alternative mechanisms were proposed but these as yet remain to be
tested in the context of SARS-CoV2. From a therapeutic standpoint and in
the absence of in vivo data, it is unclear whether the 47D11 ab can
alter the course of infection in an infected host through virus
clearance or protect an uninfected host that is exposed to the virus.
There is a precedent for the latter possibility as it relates to
SARS-CoV that was cited by the authors and could turn out to be true for
SARS-CoV2.</p>
<h3 data-number="10.54.4" id="significance-48"><span class="header-section-number">10.54.4</span> Significance</h3>
<p>This study enabled the identification of novel neutralizing
antibody against COV-that could potentially be used as first line of
treatment in the near future to reduce the viral load and adverse
effects in infected patients. In addition, neutralizing antibodies such
as 47D11 represent promising reagents for developing
antigen-antibody-based detection test kits and assays.</p>
<h3 data-number="10.54.5" id="credit-53"><span class="header-section-number">10.54.5</span> Credit</h3>
<p><em>This review was edited by K. Alexandropoulos as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<p><strong>Heat inactivation of serum interferes with the immunoanalysis of
antibodies to SARS-CoV-2</strong></p>
<p>Heat inactivation, immunochromatography, diagnosis, serum antibodies,
IgM, IgG</p>
<p><em>Summary</em></p>
<p>The use of heat inactivation to neutralize pathogens in serum samples
collected from suspected COVID-19 patients reduces the sensitivity of a
fluorescent immunochromatographic assay to detect anti-SARS-CoV-2 IgM
and IgG.</p>
<p><em>Major findings</em></p>
<p>Coronaviruses can be killed by heat inactivation, and this is an
important safety precaution in laboratory manipulation of clinical
samples. However, the effect of this step on downstream
SARS-CoV-2-specific serum antibody assays has not been examined. The
authors tested the effect of heat inactivation (56 deg C for 30 minutes)
versus no heat inactivation on a fluorescence immunochromatography
assay. Heat inactivation reduced all IgM measurements by an average of
54% and most IgG measurements (22/36 samples, average reduction of 50%),
consistent with the lower thermal stability of IgM than that of IgG.
Heat inactivation caused a subset of IgM but not IgG readings to fall
below a specified positivity threshold.</p>
<p><em>Limitations</em></p>
<p>Limitations included the use of only one type of assay for testing heat
inactivated vs non-inactivated sera, and the use of the same baseline
for heat inactivated and non-inactivated sera. The results indicate that
heat inactivation affects the quantification of SARS-CoV-2-antibody
response, specially IgM, but still allows to distinguish positive
specific IgG. Therefore, the effect of heat inactivation should be
studied when designing assays that quantitatively associate
immunoglobulin levels (especially IgM) to immune state.</p>
<p><em>Review by Andrew M. Leader as part of a project by students, postdocs
and faculty at the Immunology Institute of the Icahn school of medicine,
Mount Sinai.</em></p>
<h2 data-number="10.55" id="immune-phenotyping-based-on-neutrophil-to-lymphocyte-ratio-and-igg-predicts-disease-severity-and-outcome-for-patients-with-covid-19"><span class="header-section-number">10.55</span> Immune phenotyping based on neutrophil-to-lymphocyte ratio and IgG predicts disease severity and outcome for patients with COVID-19</h2>
<p>Zhang et al. <em>medRxiv</em> <span class="citation" data-cites="1CwCTXPqj">[<a href="#ref-1CwCTXPqj" role="doc-biblioref">825</a>]</span></p>
<h3 data-number="10.55.1" id="keywords-54"><span class="header-section-number">10.55.1</span> Keywords</h3>
<ul>
<li>Biomarkers</li>
<li>cytokines</li>
<li>IgG</li>
<li>immune cells</li>
</ul>
<h3 data-number="10.55.2" id="main-findings-51"><span class="header-section-number">10.55.2</span> Main Findings</h3>
<p>In a cohort of 222 patients, anti-SARS-CoV-2 IgM and IgG levels were
analyzed during acute and convalescent phases (up to day 35) and
correlated to the diseases’ severity. The same was done with
neutrophil-to-lymphocyte ratio. High IgG levels and high
neutrophil-to-lymphocyte ratio in convalescence were both independently
associated to the severity of the disease. The simultaneous occurrence
of both of these laboratory findings correlated even stronger to the
diseases’ severity.</p>
<p>Severe cases with high neutrophil-to-lymphocyte ratios had clearly
higher levels of IL-6. The authors propose that a robust IgG response
leads to immune-mediated tissue damage, thus explaining the worse
outcome in patients with overexuberant antibody response.</p>
<h3 data-number="10.55.3" id="limitations-52"><span class="header-section-number">10.55.3</span> Limitations</h3>
<p>A main criticism is that the criteria for stratifying patients in severe
vs. non-severe are not described. The only reference related to this is
the difference between the percentage of patients who needed mechanical
ventilation, which was greater in patients with both high IgG levels and
high neutrophil-to-lymphocyte ratio. No patient with both low IgG levels
and low neutrophil-to-lymphocyte ratio was treated with mechanical
ventilation.</p>
<p>The proposed correlation of severity with IL-2 and IL-10 levels is not
very strong.</p>
<p>Furthermore, although mostly ignored in the paper’s discussion, one of
the most interesting findings is that an early increase in
anti-SARS-CoV-2 IgM levels also seems to correlate with severe disease.
However, as only median values are shown for antibody kinetics curves,
the extent of variation in acute phase cannot be assessed.</p>
<h3 data-number="10.55.4" id="significance-49"><span class="header-section-number">10.55.4</span> Significance</h3>
<p>Anti-SARS-CoV-2 IgG levels and with neutrophil-to-lymphocyte ratio
predict severity of COVID-19 independently of each other. An additive
predictive value of both variables is noticeable. Importantly, an
early-on increase in anti-SARS-CoV-2 IgM levels also seem to predict
outcome.</p>
<h3 data-number="10.55.5" id="credit-54"><span class="header-section-number">10.55.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.56" id="reinfection-could-not-occur-in-sars-2-cov-2-infected-rhesus-macaques"><span class="header-section-number">10.56</span> Reinfection could not occur in SARS-2 CoV-2 infected rhesus macaques</h2>
<p>Bao et al. <em>bioRxiv</em> <span class="citation" data-cites="3XqFhi8l">[<a href="#ref-3XqFhi8l" role="doc-biblioref">826</a>]</span></p>
<h3 data-number="10.56.1" id="keywords-55"><span class="header-section-number">10.56.1</span> Keywords</h3>
<ul>
<li>SARS-CoV-2</li>
<li>viral load</li>
<li>reinfection</li>
<li>relapse</li>
<li>non-human primate model</li>
</ul>
<h3 data-number="10.56.2" id="main-findings-52"><span class="header-section-number">10.56.2</span> Main Findings</h3>
<p>This study addresses the issue or acquired immunity after a primary
COVID-19 infection in rhesus monkeys. Four Chinese rhesus macaques were
intratracheally infected with SARS-CoV-2 and two out of the four were
re-infected at 28 days post initial infection (dpi) with the same viral
dose after confirming the recovery by the absence of clinical symptoms,
radiological abnormalities and viral detection (2 negative RT-PCR
tests). While the initial infection led the viral loads in nasal and
pharyngeal swabs that reach approximately 6.5 log<sub>10</sub> RNA copies/ml at 3
dpi in all four monkeys, viral loads in the swabs tested negative after
reinfection in the two reinfected monkeys. In addition, the necropsies
from a monkey (M1) at 7 days after primary infection, and another monkey
(M3) at 5 days post reinfection, revealed the histopathological damages
and viral replication in the examined tissues from M1, while no viral
replication as well as no histological damages were detected in the
tissues from M3. Furthermore, sera from three monkeys at 21 and 28 dpi
exhibited neutralizing activity against SARS-CoV-2 in vitro, suggesting
the production of protective neutralizing antibodies in these monkeys.
Overall, this study indicates that primary infection with SARS-CoV-2 may
protect from subsequent exposure to the same virus.</p>
<h3 data-number="10.56.3" id="limitations-53"><span class="header-section-number">10.56.3</span> Limitations</h3>
<p>In human, virus has been detected by nasopharyngeal swabs until 9 to 15
days after the onset of symptoms. In the infected monkeys in this study,
virus were detected from day 1 after the infection, declining to
undetectable level by day 15 post infection. It may suggest that there
is a faster viral clearance mechanism in monkeys, therefore the
conclusions of reinfection protection for humans need to be carefully
considered. In addition, only two monkeys were re-infected in this study
and the clinical signs of these monkeys were not similar: M3 did not
show weight loss and M4 showed relatively higher fever on the day of
infection and the day of re-challenge.</p>
<h3 data-number="10.56.4" id="significance-50"><span class="header-section-number">10.56.4</span> Significance</h3>
<p>This study showed clear viral clearance and no indications of relapse or
viremia after a secondary infection with SARS-CoV-2 in a Chinese rhesus
macaque model. These results support the idea that patients with full
recovery (two negative RT-PCR results) may also be protected from
secondary SARS-CoV-2 infection. Recovered patients may be able to
reintegrate to normal public life and provide protective serum perhaps
even if having had a mild infection. The results are also encouraging
for successful vaccine development against SARS-CoV-2.</p>
<h3 data-number="10.56.5" id="credit-55"><span class="header-section-number">10.56.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.57" id="a-highly-conserved-cryptic-epitope-in-the-receptor-binding-domains-of-sars-cov-2-and-sars-cov"><span class="header-section-number">10.57</span> A highly conserved cryptic epitope in the receptor-binding domains of SARS-CoV-2 and SARS-CoV</h2>
<p><span class="citation" data-cites="6gQxRkWM">[<a href="#ref-6gQxRkWM" role="doc-biblioref">827</a>]</span></p>
<h3 data-number="10.57.1" id="keywords-56"><span class="header-section-number">10.57.1</span> Keywords</h3>
<ul>
<li>neutralizing antibody</li>
<li>cross-reactivity</li>
</ul>
<h3 data-number="10.57.2" id="main-findings-53"><span class="header-section-number">10.57.2</span> Main Findings</h3>
<p>Given the sequence similarity of the surface spike glycoprotein (S) of
SARS-CoV-2 and SAR-CoV, Yuan et al. (2020) propose that neutralizing
antibodies isolated from convalescent SARS-CoV patients may offer
insight into cross-reactive antibodies targeting SARS-CoV-2. In
particular, they find that the receptor-binding domain (RBD) of
SARS-CoV-2 S protein shares 86% sequence similarity with the RBD of
SARS-CoV S protein that binds to the CR3022 neutralizing antibody.
CR3022 also displays increased affinity for the “up” conformation of the
SARS-CoV-2 S protein compared to the “down” conformation as it does for
the SARS-CoV S protein. Therefore, the authors propose that this
cross-reactive antibody may confer some degree of protection <em>in vivo</em>
even if it fails to neutralize <em>in vitro</em>.</p>
<h3 data-number="10.57.3" id="limitations-54"><span class="header-section-number">10.57.3</span> Limitations</h3>
<p>Although the authors offer a logical rationale for
identifying cross-reactive neutralizing antibodies derived from
SARS-CoV, their study using only CR3022 failed to demonstrate whether
this approach will be successful. After all, CR3022 failed to neutralize
<em>in vitro</em> despite the binding affinity to a similar epitope on
SARS-CoV-2. They would benefit from testing more candidates and using an
<em>in vivo</em> model to demonstrate their claim that protection may be
possible in the absence neutralization if combinations are used <em>in
vivo.</em></p>
<h3 data-number="10.57.4" id="significance-51"><span class="header-section-number">10.57.4</span> Significance</h3>
<p>The ability to make use of previously characterized
neutralizing antibodies for conserved epitopes can expedite drug design
and treatment options.</p>
<h3 data-number="10.57.5" id="credit-56"><span class="header-section-number">10.57.5</span> Credit</h3>
<p><em>This review was undertaken by Dan Fu Ruan, Evan Cody and Venu Pothula as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.58" id="highly-accurate-and-sensitive-diagnostic-detection-of-sars-cov-2-by-digital-pcr"><span class="header-section-number">10.58</span> Highly accurate and sensitive diagnostic detection of SARS-CoV-2 by digital PCR</h2>
<p>Dong et al. <em>medRxiv</em> <span class="citation" data-cites="UYD5T2mp">[<a href="#ref-UYD5T2mp" role="doc-biblioref">293</a>]</span></p>
<h3 data-number="10.58.1" id="keywords-57"><span class="header-section-number">10.58.1</span> Keywords</h3>
<ul>
<li>Diagnosis</li>
<li>digital PCR</li>
</ul>
<h3 data-number="10.58.2" id="main-findings-54"><span class="header-section-number">10.58.2</span> Main Findings</h3>
<p>The authors present a digital PCR (dPCR) diagnostic test for
SARS-CoV-2 infection. In 103 individuals that were confirmed in a
follow-up to be infected, the standard qPCR test had a positivity
rate of 28.2% while the dPCR test detected 87.4% of the infections
by detecting an additional 61 positive cases. The authors also
tested samples from close contacts (early in infection stage) and
convalescing individuals (late in infection stage) and were able to
detect SARS-CoV-2 nucleic acid in many more samples using dPCR
compared to qPCR.</p>
<h3 data-number="10.58.3" id="limitations-55"><span class="header-section-number">10.58.3</span> Limitations</h3>
<p>I did not detect limitations.</p>
<h3 data-number="10.58.4" id="significance-52"><span class="header-section-number">10.58.4</span> Significance</h3>
<p>The authors make a strong case for the need for a highly sensitive
and accurate confirmatory method for diagnosing COVID-19 during this
outbreak and present a potential addition to the diagnostic arsenal.
They propose a dPCR test that they present has a dramatically lower
false negative rate than the standard RT-qPCR tests and can be
especially beneficial in people with low viral load, whether they
are in the earlier or later stages of infection.</p>
<h3 data-number="10.58.5" id="credit-57"><span class="header-section-number">10.58.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.59" id="sars-cov-2-invades-host-cells-via-a-novel-route-cd147-spike-protein"><span class="header-section-number">10.59</span> SARS-CoV-2 invades host cells via a novel route: CD147-spike protein</h2>
<p>Wang et al. <em>bioRxiv</em> <span class="citation" data-cites="14y8UvAbH">[<a href="#ref-14y8UvAbH" role="doc-biblioref">828</a>]</span></p>
<h3 data-number="10.59.1" id="keywords-58"><span class="header-section-number">10.59.1</span> Keywords</h3>
<ul>
<li>spike protein</li>
<li>viral entry</li>
<li>CD147</li>
<li>SARS-CoV-2</li>
</ul>
<h3 data-number="10.59.2" id="main-findings-55"><span class="header-section-number">10.59.2</span> Main Findings</h3>
<p>The authors propose a novel mechanism of SARS-CoV-2 viral entry through
the interaction of the viral spike protein (SP) and the immunoglobulin
superfamily protein CD147 (also known as Basigin). Using an in-house
developed humanized antibody against CD147 (maplazumab), they show that
blocking CD147 decreases viral replication in Vero E6 cells. Using
surface plasmon resonance (SPR), ELISA, and Co-IP assays, they show that
the spike protein of SARS-CoV-2 directly interacts with CD147. Lastly,
they utilize immune-election microscopy to show spike protein and CD147
localize to viral inclusion bodies of Vero E6 cells.</p>
<h3 data-number="10.59.3" id="limitations-56"><span class="header-section-number">10.59.3</span> Limitations</h3>
<p>The authors claim that an anti-CD147 antibody (Meplazumab) inhibits
SARS-CoV-2 replication by testing cell growth and viral load in cells
infected with SARS-CoV-2, however there are key pieces of this
experiment that are missing. First, the authors fail to use a
non-specific antibody control. Second, the authors claim that viral
replication is inhibited, and that they test this by qPCR, however this
data is <strong>not shown.</strong> To further prove specificity, the authors should
introduce CD147 to non-susceptible cells and show that they become
permissive.</p>
<p>The authors claim that there is a direct interaction between CD147 and
SP through SPR, ELISA, and Co-IP, and this data seems generally
convincing. The electron microscopy provides further correlative
evidence that SARS-CoV-2 may interact with CD147 as they are both found
in the same viral inclusion body. A quantification of this data would
make the findings more robust.</p>
<p>Finally, the data in this paper lacks replicates, error bars, and
statistics to show that the data are reproducible and statistically
significant.</p>
<h3 data-number="10.59.4" id="significance-53"><span class="header-section-number">10.59.4</span> Significance</h3>
<p>It has been shown in various studies that SARS-CoV-2 binds to the cell
surface protein ACE2 for cell entry, yet ACE2 is highly expressed in
heart, kidney, and intestinal cells, raising the concern that blocking
ACE2 would result in harmful side effects <span class="citation" data-cites="BcOf65BE">[<a href="#ref-BcOf65BE" role="doc-biblioref">829</a>]</span> CD147 on the other hand
is highly expressed in various tumor types, inflamed tissues, and
pathogen infected cells, suggesting that the inhibition of CD147 would
not result in major side effects <span class="citation" data-cites="34nQJalJ KcThTnsm">[<a href="#ref-34nQJalJ" role="doc-biblioref">830</a>,<a href="#ref-KcThTnsm" role="doc-biblioref">831</a>]</span> The research in this paper has
resulted in an ongoing clinical trail in China to test the safety and
efficacy of anti-CD147 Meplazumab to treat COVID-19. (ClinicalTrails.gov
identifier NCT04275245).</p>
<h3 data-number="10.59.5" id="credit-58"><span class="header-section-number">10.59.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.60" id="blood-single-cell-immune-profiling-reveals-that-interferon-mapk-pathway-mediated-adaptive-immune-response-for-covid-19"><span class="header-section-number">10.60</span> Blood single cell immune profiling reveals that interferon-MAPK pathway mediated adaptive immune response for COVID-19</h2>
<p>Huang et al. <em>medRxiv</em> <span class="citation" data-cites="18l4l0nLX">[<a href="#ref-18l4l0nLX" role="doc-biblioref">832</a>]</span></p>
<h3 data-number="10.60.1" id="keywords-59"><span class="header-section-number">10.60.1</span> Keywords</h3>
<ul>
<li>COVID-19</li>
<li>SARS-CoV-2</li>
<li>PBMC</li>
<li>single cell</li>
<li>MAPK</li>
</ul>
<h3 data-number="10.60.2" id="main-findings-56"><span class="header-section-number">10.60.2</span> Main Findings</h3>
<p>The authors performed single-cell RNA sequencing (scRNAseq) of
peripheral blood mononuclear cells isolated from whole blood samples of
COVID-19 patients (n=10). Data was compared to scRNAseq of samples
collected from patients with influenza A (n=1), acute pharyngitis (n=1),
and cerebral infarction (n=1), as well as, three healthy controls.
COVID-19 patients were categorized into those with moderate (n=6),
severe (n=1), critical (n=1), and cured (n=2) disease. Analysis across
all COVID-19 disease levels revealed 56 different cellular subtypes,
among 17 immune cell types; comparisons between each category to the
normal controls revealed <strong>increased proportions of CD1c<sup>+</sup> dendritic
cells, CD8<sup>+</sup> CTLs, and plasmacytoid dendritic cells and a decrease in
proportions of B cells and CD4<sup>+</sup> T cells</strong>.</p>
<p><strong>TCR sequencing revealed that greater clonality is associated with
milder COVID-19 disease</strong>; BCR sequencing revealed that COVID-19
patients have circulating antibodies against known viral antigens,
including EBV, HIV, influenza A, and other RNA viruses. This may suggest
that the immune response to SARS-CoV-2 infection elicits production of
antibodies against known RNA viruses.</p>
<p>Excluding enriched pathways shared by COVID-19 patients and patients
with other conditions (influenza A, acute pharyngitis, and cerebral
infarction), the authors identified the <strong>interferon-MAPK signaling
pathway as a major response to SARS-CoV-2 infection</strong>. The authors
performed quantitative real-time reverse transcriptase polymerase chain
reaction (RT-PCR) for interferon-MAPK signaling genes: <em>IRF27, BST2,</em>
and <em>FOS</em>. These samples were collected from a separate cohort of
COVID-19 patients (critical, n=3; severe, n=3; moderate, n=19; mild,
n=3; and cured, n=10; and healthy controls, n=5). Notably, consistent
with the original scRNAseq data, <em>FOS</em> showed up-regulation in COVID-19
patients and down-regulation in cured patients. <strong>The authors propose
that <em>FOS</em> may be a candidate marker gene for curative COVID-19
disease</strong>.</p>
<h3 data-number="10.60.3" id="limitations-57"><span class="header-section-number">10.60.3</span> Limitations</h3>
<p>The sample size of this study is limited. To further delineate
differences in the immune profile of peripheral blood of COVID-19
patients, a greater sample size is needed, and longitudinal samples are
needed, as well. A better understanding of the immunological
interactions in cured patients, for example, would require a profile
before and after improvement.</p>
<p>Moreover, the conclusions drawn from this scRNAseq study point to
potential autoimmunity and immune deficiency to distinguish different
severities of COVID-19 disease. However, this requires an expanded
number of samples and a more robust organization of specific immune cell
subtypes that can be compared across different patients. Importantly,
this criterion is likely needed to ensure greater specificity in
identifying markers for COVID-19 infection and subsequent immune
response.</p>
<h3 data-number="10.60.4" id="significance-54"><span class="header-section-number">10.60.4</span> Significance</h3>
<p>At the single-cell level, COVID-19 disease has been characterized in the
lung, but a greater understanding of systemic immunological responses is
furthered in this study. Type I interferon is an important signaling
molecule for the anti-viral response. The identification of the
interferon-MAPK signaling pathway and the differential expression of
MAPK regulators between patients of differing COVID-19 severity and
compared to cured patients may underscore the importance of either
immune deficiency or autoimmunity in COVID-19 disease.</p>
<h3 data-number="10.60.5" id="credit-59"><span class="header-section-number">10.60.5</span> Credit</h3>
<p><em>This review was undertaken by Matthew D. Park as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.61" id="cross-reactive-antibody-response-between-sars-cov-2-and-sars-cov-infection."><span class="header-section-number">10.61</span> Cross-reactive antibody response between SARS-CoV-2 and SARS-CoV infection.</h2>
<p>Lv et al. <em>bioRxiv</em> <span class="citation" data-cites="dM1rOHmH">[<a href="#ref-dM1rOHmH" role="doc-biblioref">833</a>]</span></p>
<h3 data-number="10.61.1" id="keywords-60"><span class="header-section-number">10.61.1</span> Keywords</h3>
<p>SARS-CoV-2, SARS-CoV, spike protein, RBD, cross-reactivity,
cross-neutralization, antibody, human patients, mouse</p>
<h3 data-number="10.61.2" id="main-findings-57"><span class="header-section-number">10.61.2</span> Main Findings</h3>
<p>The authors explore the antigenic differences between SARS-CoV-2 and
SARS-CoV by analyzing plasma samples from SARS-CoV-2 (n = 15) and
SARS-CoV (n = 7) patients. Cross-reactivity in antibody binding to the
spike protein between SARS-CoV-2 and SARS-CoV was found to be common,
mostly targeting non-RBD regions in plasma from SARS-CoV-2 patients.
Only one SARS-CoV-2 plasma sample was able to cross-neutralize SARS-CoV,
with low neutralization activity. No cross-neutralization response was
detected in plasma from SARS-CoV patients.</p>
<p>To further investigate the cross-reactivity of antibody responses to
SARS-CoV-2 and SARS-CoV, the authors analyzed the antibody response of
plasma collected from mice infected or immunized with SARS-CoV-2 or
SARS-CoV (n = 5 or 6 per group). Plasma from mice immunized with
SARS-CoV-2 displayed cross-reactive responses to SARS-CoV S ectodomain
and, to a lesser extent, SARS-CoV RBD. Similarly, plasma from mice
immunized with SARS-CoV displayed cross-reactive responses to SARS-CoV-2
S ectodomain. Cross-neutralization activity was not detected in any of
the mouse plasma samples.</p>
<h3 data-number="10.61.3" id="limitations-58"><span class="header-section-number">10.61.3</span> Limitations</h3>
<p>The size of each patient cohort is insufficient to accurately determine
the frequency of cross-reactivity and cross-neutralization in the
current SARS-CoV-2 pandemic. Recruitment of additional patients from a
larger range of geographical regions and time points would also enable
exploration into the effect of the genetic diversity and evolution of
the SARS-CoV-2 virus on cross-reactivity. This work would also benefit
from the mapping of specific epitopes for each sample. Future studies
may determine whether the non-neutralizing antibody responses can confer
<em>in vitro</em> protection or lead to antibody-dependent disease enhancement.</p>
<h3 data-number="10.61.4" id="significance-55"><span class="header-section-number">10.61.4</span> Significance</h3>
<p>The cross-reactive antibody responses to S protein in the majority of
SARS-CoV-2 patients is an important consideration for development of
serological assays and vaccine development during the current outbreak.
The limited extent of cross-neutralization demonstrated in this study
indicates that vaccinating to cross-reactive conserved epitopes may have
limited efficacy, presenting a key concern for the development of a more
universal coronavirus vaccine to address the global health risk of novel
coronavirus outbreaks.</p>
<h3 data-number="10.61.5" id="credit-60"><span class="header-section-number">10.61.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.62" id="the-feasibility-of-convalescent-plasma-therapy-in-severe-covid-19-patients-a-pilot-study"><span class="header-section-number">10.62</span> The feasibility of convalescent plasma therapy in severe COVID-19 patients: a pilot study</h2>
<p>Duan et al. <em>medRxiv</em> <span class="citation" data-cites="UENoLkhz">[<a href="#ref-UENoLkhz" role="doc-biblioref">834</a>]</span></p>
<h3 data-number="10.62.1" id="keywords-61"><span class="header-section-number">10.62.1</span> Keywords</h3>
<ul>
<li>COVID-19</li>
<li>SARS-CoV-2</li>
<li>convalescent plasma</li>
<li>treatment outcome</li>
<li>pilot</li>
<li>therapy</li>
<li>transfusion</li>
</ul>
<h3 data-number="10.62.2" id="main-findings-58"><span class="header-section-number">10.62.2</span> Main Findings</h3>
<p>This is the first report to date of convalescent plasma therapy as a
therapeutic against COVID-19 disease. This is a feasibility pilot study.
The authors report the administration and clinical benefit of 200 mL of
convalescent plasma (CP) (1:640 titer) derived from recently cured
donors (CP selected among 40 donors based on high neutralizing titer and
ABO compatibility) to 10 severe COVID-19 patients with confirmed
viremia. The primary endpoint was the safety of CP transfusion. The
secondary endpoint were clinical signs of improvement based on symptoms
and laboratory parameters.</p>
<p>The authors reported use of methylene blue photochemistry to inactivate
any potential residual virus in the plasma samples, without compromising
neutralizing antibodies, and no virus was detected before transfusion.</p>
<p>The authors report the following:</p>
<ul>
<li><p>No adverse events were observed in all patients, except 1 patient
who exhibited transient facial red spotting.</p></li>
<li><p>All patients showed significant improvement in or complete
disappearance of clinical symptoms, including fever, cough,
shortness of breath, and chest pain after 3 days of CP therapy.</p></li>
<li><p>Reduction of pulmonary lesions revealed by chest CT.</p></li>
<li><p>Elevation of lymphocyte counts in patients with lymphocytopenia.</p></li>
<li><p>Increase in SaO<sub>2</sub> in all patients, indicative of recuperating lung
function.</p></li>
<li><p>Resolution of SARS-CoV-2 viremia in 7 patients and increase in
neutralizing antibody titers in 5 patients. Persistence of
neutralizing antibody levels in 4 patients.</p></li>
</ul>
<h3 data-number="10.62.3" id="limitations-59"><span class="header-section-number">10.62.3</span> Limitations</h3>
<p>It is important to note that most recipients had high neutralization
titers of antibodies before plasma transfusion and even without
transfusion it would be expected to see an increase in neutralizing
antibodies over time. In addition to the small sample set number (n=10),
there are additional limitations to this pilot study:</p>
<ol type="1">
<li><p>All patients received concurrent therapy, in addition to the CP
transfusion. Therefore, it is unclear whether a combinatorial or
synergistic effect between these standards of care and CP
transfusion contributed to the clearance of viremia and improvement
of symptoms in these COVID-19 patients.</p></li>
<li><p>The kinetics of viral clearance was not investigated, with respect
to the administration of CP transfusion. So, the definitive impact
of CP transfusion on immune dynamics and subsequent viral load is
not well defined.</p></li>
<li><p>Comparison with a small historical control group is not ideal.</p></li>
</ol>
<h3 data-number="10.62.4" id="significance-56"><span class="header-section-number">10.62.4</span> Significance</h3>
<p>For the first time, a pilot study provides promising results involving
the use of convalescent plasma from cured COVID-19 patients to treat
others with more severe disease. The authors report that the
administration of a single, high-dose of neutralizing antibodies is
safe. In addition, there were encouraging results with regards to the
reduction of viral load and improvement of clinical outcomes. It is,
therefore, necessary to expand this type of study with more
participants, in order to determine optimal dose and treatment kinetics.
It is important to note that CP has been studied to treat H1N1
influenza, SARS-CoV-1, and MERS-CoV, although it has not been proven to
be effective in treating these infections.</p>
<h3 data-number="10.62.5" id="credit-61"><span class="header-section-number">10.62.5</span> Credit</h3>
<p><em>Review by Matthew D. Park and revised by Alice O. Kamphorst and Maria A. Curotto de Lafaille as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.63" id="hydroxychloroquine-and-azithromycin-as-a-treatment-of-covid-19-results-of-an-open-label-non-randomized-clinical-trial"><span class="header-section-number">10.63</span> Hydroxychloroquine and azithromycin as a treatment of COVID-19: results of an open label non-randomized clinical trial</h2>
<p><span class="citation" data-cites="Xbkby2Jr">[<a href="#ref-Xbkby2Jr" role="doc-biblioref">835</a>]</span></p>
<h3 data-number="10.63.1" id="keywords-62"><span class="header-section-number">10.63.1</span> Keywords</h3>
<ul>
<li>hydroxychloroquine</li>
<li>clearance</li>
<li>viral load</li>
<li>clinical trial</li>
</ul>
<h3 data-number="10.63.2" id="main-findings-59"><span class="header-section-number">10.63.2</span> Main Findings</h3>
<p>This study was a single-arm, open label clinical trial with
600 mg hydroxychloroquine (HCQ) in the treatment arm (n = 20). Patients
who refused participation or patients from another center not treated
with HCQ were included as negative controls (n = 16). Among the patients
in the treatment arm, 6 received concomitant azithromycin to prevent
superimposed bacterial infection. The primary endpoint was respiratory
viral loads on day 6 post enrollment, measured by nasopharyngeal swab
followed by real-time reverse transcription-PCR.</p>
<p>HCQ alone was able to significantly reduce viral loads by day 6 (n =
8/14, 57.1% complete clearance, p &lt; 0.001); azithromycin appears to
be synergistic with HCQ, as 6/6 patients receiving combined treatment
had complete viral clearance (p &lt; 0.001).</p>
<h3 data-number="10.63.3" id="limitations-60"><span class="header-section-number">10.63.3</span> Limitations</h3>
<p>Despite what is outlined above, this study has a number
of limitations that must be considered. First, there were originally n =
26 patients in the treatment arm, with 6 lost to follow up for the
following reasons: 3 transferred to ICU, 1 discharge, 1
self-discontinued treatment d/t side effects, and 1 patient expired.
Total length of clinical follow up was 14 days, but the data beyond day
6 post-inclusion are not shown.</p>
<p>Strikingly, in supplementary table 1, results of the real-time RT-PCR
are listed for the control and treatment arms from D0 – D6. However, the
data are not reported in a standard way, with a mix of broadly positive
or negative result delineation with Ct (cycle threshold) values, the
standard output of real time PCR. It is impossible to compare what is
defined as a positive value between the patients in the control and
treatment arms without a standardized threshold for a positive test.
Further, the starting viral loads reported at D0 in the groups receiving
HCQ or HCQ + azithromycin were significantly different (ct of 25.3 vs
26.8 respectively), which could explain in part the differences observed
in the response to treatment between 2 groups. Finally, patients in the
control arm from outside the primary medical center in this study
(Marseille) did not actually have samples tested by PCR daily. Instead,
positive test results from every other day were extrapolated to mean
positive results on the day before and after testing as well (Table 2,
footnote <sup>a</sup>).</p>
<p>Taken together, the results of this study suggest that HCQ represents a
promising treatment avenue for COVID-19 patients. However, the limited
size of the trial, and the way in which the results were reported does
not allow for other medical centers to extrapolate a positive or
negative result in the treatment of their own patients with HCQ +/-
azithromycin. Further larger randomized clinical trials will be required
to ascertain the efficacy of HCQ +/- azithromycin in the treatment of
COVID-19.</p>
<h3 data-number="10.63.4" id="significance-57"><span class="header-section-number">10.63.4</span> Significance</h3>
<p>Chloroquine is thought to inhibit viral infection,
including SARS-Cov-2, by increasing pH within endosomes and lysosomes,
altering the biochemical conditions required for viral fusion <span class="citation" data-cites="IhdcvbzC BIlqPh6M">[<a href="#ref-IhdcvbzC" role="doc-biblioref">352</a>,<a href="#ref-BIlqPh6M" role="doc-biblioref">836</a>]</span>.
However, chloroquine also has immuno-modulatory effects that I think may
play a role. Chloroquine has been shown to increase CTLA-4 expression at
the cell surface by decreasing its degradation in the endo-lysosome
pathway; AP-1 traffics the cytoplasmic tail of CTLA-4 to lysosomes, but
in conditions of increased pH, the protein machinery required for
degradation is less functional <span class="citation" data-cites="Q9tXjG1R">[<a href="#ref-Q9tXjG1R" role="doc-biblioref">837</a>]</span>. As such, more CTLA-4 remains in
endosomes and is trafficked back to the cell surface. It is possible
that this may also contribute to patient recovery via reduction of
cytokine storm, in addition to the direct anti-viral effects of HCQ.</p>
<h3 data-number="10.63.5" id="credit-62"><span class="header-section-number">10.63.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.64" id="recapitulation-of-sars-cov-2-infection-and-cholangiocyte-damage-with-human-liver-organoids"><span class="header-section-number">10.64</span> Recapitulation of SARS-CoV-2 Infection and Cholangiocyte Damage with Human Liver Organoids</h2>
<p><span class="citation" data-cites="OuDMtCRl">[<a href="#ref-OuDMtCRl" role="doc-biblioref">761</a>]</span></p>
<h3 data-number="10.64.1" id="keywords-63"><span class="header-section-number">10.64.1</span> Keywords</h3>
<ul>
<li>CoVID-19</li>
<li>2019-nCoV</li>
<li>SARS-CoV-2</li>
<li>liver</li>
<li>organoids</li>
<li>Cholangiocyte</li>
</ul>
<h3 data-number="10.64.2" id="main-findings-60"><span class="header-section-number">10.64.2</span> Main Findings</h3>
<ul>
<li><p>Used human liver ductal organoids to determine ACE2+ cholangiocytes
in healthy liver (2.92% of all cells) are infectable and support
SARS-CoV-2 viral replication.</p></li>
<li><p>Plaque-purified SARS-CoV-2 viral infection disrupted organoid
barrier and bile transporting functions of cholangiocytes through
dysregulation of genes involved in tight junction formation (CLDN1)
and bile acid transportation (ASBT and CFTR).</p></li>
</ul>
<h3 data-number="10.64.3" id="limitations-61"><span class="header-section-number">10.64.3</span> Limitations:</h3>
<ul>
<li><p>Unclear if liver damage observed in patients due to direct
cholangiocyte infection or due to secondary immune/cytokine effects.
This study argues for direct damage as it lacks immune contexture;
but further studies needed with autopsy samples or organoid-immune
cell co-culture to conclude strongly.</p></li>
<li><p>Would be important to measure cholangiocyte-intrinsic anti-viral
response and alarmins secreted upon infection, and furthermore study
tropism of various immune cells to conditioned media from organoids
infected with SARS-CoV-2.</p></li>
<li><p>Does not address how cirrhotic liver or
alcohol/smoking/obesity-associated liver organoids respond to
SARS-CoV-2 infectivity and replication, worth pursuing to
experimentally address clinical data indicating co-morbidities.</p></li>
</ul>
<h3 data-number="10.64.4" id="significance-58"><span class="header-section-number">10.64.4</span> Significance</h3>
<ul>
<li><p>Useful model to rapidly study drug activity against SARS-CoV-2
infection in liver, while monitoring baseline liver damage.</p></li>
<li><p>Liver abnormality observed in &gt;50% of CoVID-19 patients; the
results from this study could explain the bile acid accumulation and
consequent liver damage observed.</p></li>
</ul>
<h3 data-number="10.64.5" id="credit-63"><span class="header-section-number">10.64.5</span> Credit</h3>
<p><em>Review by Samarth Hegde as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.65" id="the-sequence-of-human-ace2-is-suboptimal-for-binding-the-s-spike-protein-of-sars-coronavirus-2"><span class="header-section-number">10.65</span> The sequence of human ACE2 is suboptimal for binding the S spike protein of SARS coronavirus 2</h2>
<p><span class="citation" data-cites="3VPi509z">[<a href="#ref-3VPi509z" role="doc-biblioref">838</a>]</span></p>
<h3 data-number="10.65.1" id="keywords-64"><span class="header-section-number">10.65.1</span> Keywords</h3>
<ul>
<li>Coronavirus Disease 2019 (COVID-19)</li>
<li>SARS-CoV-2</li>
<li>Spike protein S</li>
<li>ACE2</li>
</ul>
<h3 data-number="10.65.2" id="main-findings-61"><span class="header-section-number">10.65.2</span> Main Findings</h3>
<p>Severe Acute Respiratory Syndrome Coronavirus 2
(SARS-CoV-2) infects cells through S spike glycoprotein binding
angiotensin-converting enzyme (ACE2) on host cells. S protein can bind
both membrane-bound ACE2 and soluble ACE2 (sACE2), which can serve as a
decoy that neutralizes infection. Recombinant sACE2 is now being tested
in clinical trials for COVID-19. To determine if a therapeutic sACE2
with higher affinity for S protein could be designed, authors generated
a library containing every amino acid substitution possible at the 117
sites spanning the binding interface with S protein. The ACE2 library
was expressed in human Expi293F cells and cells were incubated with
medium containing the receptor binding domain (RBD) of SARS-CoV-2 fused
to GFP. Cells with high or low affinity mutant ACE2 receptor compared to
affinity of wild type ACE2 for the RBD were FACS sorted and transcripts
from these sorted populations were deep sequenced. Deep mutagenesis
identified numerous mutations in ACE2 that enhance RBD binding. This
work serves to identify putative high affinity ACE2 therapeutics for the
treatment of CoV-2.</p>
<h3 data-number="10.65.3" id="limitations-62"><span class="header-section-number">10.65.3</span> Limitations</h3>
<p>The authors generated a large library of mutated
ACE2, expressed them in human Expi293F cells, and performed deep
mutagenesis to identify enhanced binders for the RBD of SARS-CoV-2 S
protein. While these data serve as a useful resource, the ability of the
high affinity ACE2 mutants identified to serve as therapeutics needs
further validation in terms of conformational stability when purified as
well as efficacy/safety both <em>in vitro</em> and <em>in vivo</em>. Additionally,
authors mentioned fusing the therapeutic ACE2 to Fc receptors to elicit
beneficial host immune responses, which would need further design and
validation.</p>
<h3 data-number="10.65.4" id="significance-59"><span class="header-section-number">10.65.4</span> Significance</h3>
<p>This study identified structural ACE2 mutants that
have potential to serve as therapeutics in the treatment of SARS-CoV-2
upon further testing and validation.</p>
<h3 data-number="10.65.5" id="credit-64"><span class="header-section-number">10.65.5</span> Credit</h3>
<p><em>This review was undertaken by Katherine Lindblad and Tamar Plitt as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<p><strong><em>Title</em></strong>:  A serological assay to detect SARS-Cov-2 seroconversion in
humans</p>
<p><strong><em>Immunology keywords</em></strong>:  specific serological assay - ELISA -
seroconversion - antibody titers</p>
<p><em>Note: the authors of this review work in the same institution as the
authors of the study</em></p>
<p><strong><em>Main findings</em></strong>:  </p>
<p>Production of recombinant whole Spike (S) protein and the smaller
Receptor Binding Domain (RBD) based on the sequence of Wuhan-Hu-1
SARS-CoV-2 isolate. The S protein was modified to allow trimerization
and increase stability. The authors compared the antibody reactivity of
59 banked human serum samples (non-exposed) and 3 serum samples from
confirmed SARS-CoV-2 infected patients. All Covid-19 patient sera
reacted to the S protein and RBD domain compared to the control sera.</p>
<p>The authors also characterized the antibody isotypes from the Covid-19
patients, and observed stronger IgG3 response than IgG1. IgM and IgA
responses were also prevalent.</p>
<p><strong><em>Limitations of the study</em></strong>:  </p>
<p>The authors analyzed a total of 59 control human serum samples, and
samples from only three different patients to test for reactivity
against the RBD domain and full-length spike protein. It will be
important to follow up with a larger number of patient samples to
confirm the data obtained. Furthermore, it would be interesting to
assess people at different age groups and determine whether unexposed
control kids have a higher “background”.</p>
<p>Applications of the assay described in this study in diagnosis are
limited, since antibody response should start to be detectable only one
to two weeks after infection. Future studies will be required to assess
how long after infection this assay allow to detect anti-CoV2
antibodies. Finally, while likely, the association of seroconversion
with protective immunity against SARS-Cov-2 infection still needs to be
fully established.</p>
<p><strong><em>Relevance</em></strong>:</p>
<p>This study has strong implications in the research against SARS-Cov-2.
First, it is now possible to perform serosurveys and determine who has
been infected, allowing a more accurate estimate of infection prevalence
and death rate. Second, if it is confirmed that re-infection does not
happen (or is rare), this assay can be used as a tool to screen
healthcare workers and prioritize immune ones to work with infected
patients. Third, potential convalescent plasma donors can now be
screened to help treating currently infected patients. Of note, this
assay does not involve live virus handling. experimentally, this is an
advantage as the assay does not require the precautions required by
manipulation of live virus. Finally, the recombinant proteins described
in this study represent new tools that can be used for further
applications, including vaccine development.</p>
<h2 data-number="10.66" id="comparative-pathogenesis-of-covid-19-mers-and-sars-in-a-non-human-primate-model"><span class="header-section-number">10.66</span> COMPARATIVE PATHOGENESIS OF COVID-19, MERS AND SARS IN A NON-HUMAN PRIMATE MODEL</h2>
<p><span class="citation" data-cites="1EcOLnVXk">[<a href="#ref-1EcOLnVXk" role="doc-biblioref">839</a>]</span></p>
<h3 data-number="10.66.1" id="keywords-65"><span class="header-section-number">10.66.1</span> Keywords</h3>
<ul>
<li>SARS-CoV2</li>
<li>cynomolgus macaque</li>
<li>SARS-CoV</li>
</ul>
<h3 data-number="10.66.2" id="main-findings-62"><span class="header-section-number">10.66.2</span> Main Findings</h3>
<p>This work assesses SARS-CoV-2 infection in young adult and aged
cynomolgus macaques. 4 macaques per age group were infected with
low-passage clinical sample of SARS-CoV-2 by intranasal and
intratracheal administration. Viral presence was assessed in nose,
throat and rectum through RT-PCR and viral culture. SARS-CoV-2
replication was confirmed in the respiratory track (including nasal
samples), and it was also detected in ileum. Viral nucleocapsid
detection by IHC showed infection of type I and II pneumocytes and
epithelia. Virus was found to peak between 2 and 4 days after
administration and reached higher levels in aged vs. young animals. The
early peak is consistent with data in patients and contrasts to SARS-CoV
replication. SARS-CoV-2 reached levels below detection between 8 and 21
days after inoculation and macaques established antibody immunity
against the virus by day 14. There were histopathological alteration in
lung, but no overt clinical signs. At day 4 post inoculation of
SARS-CoV-2, two of four animals presented foci of pulmonary
consolidation, with limited areas of alveolar edema and pneumonia, as
well as immune cell infiltration. In sum, cynomolgus macaques are
permissive to SARS- CoV-2 and develop lung pathology (less severe than
SARC-CoV, but more severe than MERS-CoV).</p>
<h3 data-number="10.66.3" id="limitations-63"><span class="header-section-number">10.66.3</span> Limitations</h3>
<p>Even though cynomolgus macaques were permissive to SARS-CoV-2
replication, it is unclear if the viral load reaches levels comparable
to humans and there wasn’t overt clinical pathology.</p>
<h3 data-number="10.66.4" id="significance-60"><span class="header-section-number">10.66.4</span> Significance</h3>
<p>The development of platforms in which to carry out relevant
experimentation on SARS-CoV-2 pathophysiology is of great urgency.
Cynomolgus macaques offer an environment in which viral replication can
happen, albeit in a limited way and without translating into clinically
relevant symptoms. Other groups are contributing to SARS-CoV2 literature
using this animal model <span class="citation" data-cites="3XqFhi8l">[<a href="#ref-3XqFhi8l" role="doc-biblioref">826</a>]</span>, potentially showing protection against
reinfection in cured macaques. Therefore, this platform could be used to
examine SARS-CoV2 pathophysiology while studies in other animal models
are also underway <span class="citation" data-cites="NzFgXYvQ MgLBA5m">[<a href="#ref-MgLBA5m" role="doc-biblioref">763</a>,<a href="#ref-NzFgXYvQ" role="doc-biblioref">840</a>]</span>.</p>
<h3 data-number="10.66.5" id="credit-65"><span class="header-section-number">10.66.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.67" id="investigating-the-impact-of-asymptomatic-carriers-on-covid-19-transmission"><span class="header-section-number">10.67</span> Investigating the Impact of Asymptomatic Carriers on COVID-19 Transmission</h2>
<p><span class="citation" data-cites="OvFueDLZ">[<a href="#ref-OvFueDLZ" role="doc-biblioref">841</a>]</span></p>
<h3 data-number="10.67.1" id="keywords-66"><span class="header-section-number">10.67.1</span> Keywords</h3>
<ul>
<li>Coronavirus Disease 2019 (COVID-19)</li>
<li>SARS-CoV-2</li>
<li>asymptomatic carriers</li>
<li>mathematical model</li>
<li>transmission</li>
</ul>
<h3 data-number="10.67.2" id="main-findings-63"><span class="header-section-number">10.67.2</span> Main Findings</h3>
<p>Multiple studies reported the same level of infectiousness
between symptomatic and asymptomatic carriers of SARS-CoV-2. Given that
asymptomatic and undocumented carriers escape public health surveillance
systems, a better mathematical model of transmission is needed to
determine a more accurate estimate of the basic reproductive number
(R<sub>0</sub>) of the virus to assess the contagiousness of virus. The authors
developed a SEYAR dynamical model for transmission of the new
coronavirus that takes into account asymptomatic and undocumented
carriers. The model was validated using data reported from thirteen
countries during the first three weeks of community transmission. While
current studies estimate R<sub>0</sub> to be around 3, this model indicates that
the value could range between 5.5 to 25.4.</p>
<h3 data-number="10.67.3" id="limitations-64"><span class="header-section-number">10.67.3</span> Limitations</h3>
<p>The SEYAR model realistically depicts
transmission of the virus only during the initial stages of the disease.
More data is necessary to better fit the model with current trends. In
addition, multiple factors (e.g. behavioral patterns, surveillance
capabilities, environmental and socioeconomic factors) affect
transmission of the virus and so, these factors must be taken into
consideration when estimating the R<sub>0</sub>.</p>
<h3 data-number="10.67.4" id="significance-61"><span class="header-section-number">10.67.4</span> Significance</h3>
<p>Public health authorities use the basic reproductive
number to determine the severity of disease. An accurate estimate of
R<sub>0</sub> will inform intervention strategies. This model can be applied to
different locations to assess the potential impact of COVID-19.</p>
<h3 data-number="10.67.5" id="credit-66"><span class="header-section-number">10.67.5</span> Credit</h3>
<p><em>This review was undertaken by Tamar Plitt and Katherine Lindblad as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.68" id="antibody-responses-to-sars-cov-2-in-covid-19-patients-the-perspective-application-of-serological-tests-in-clinical-practice"><span class="header-section-number">10.68</span> Antibody responses to SARS-CoV-2 in COVID-19 patients: the perspective application of serological tests in clinical practice</h2>
<p>Long et al. <em>medRxiv</em> <span class="citation" data-cites="Y9g1sQ1Y">[<a href="#ref-Y9g1sQ1Y" role="doc-biblioref">842</a>]</span></p>
<h3 data-number="10.68.1" id="keywords-67"><span class="header-section-number">10.68.1</span> Keywords</h3>
<ul>
<li>Serum antibodies</li>
<li>IgM</li>
<li>IgG</li>
<li>immunoassay</li>
<li>diagnosis</li>
<li>seroconversion</li>
</ul>
<h3 data-number="10.68.2" id="main-findings-64"><span class="header-section-number">10.68.2</span> Main Findings</h3>
<p>This study investigated the profile of the acute antibody response
against SARS-CoV-2 and provided proposals for serologic tests in
clinical practice. Magnetic Chemiluminescence Enzyme Immunoassay was
used to evaluate IgM and IgG seroconversion in 285 hospital admitted
patients who tested positive for SARS-CoV-2 by RT-PCR and in 52 COVID-19
suspected patients that tested negative by RT-PCR. A follow up study
with 63 patients was performed to investigate longitudinal effects. In
addition, IgG and IgM titers were evaluated in a cohort of close
contacts (164 persons) of an infected couple.</p>
<p>The median day of seroconversion for both IgG and IgM was 13 days after
symptom onset. Patients varied in the order of IgM/ IgG seroconversion
and there was no apparent correlation of order with age, severity, or
hospitalization time. This led the authors to conclude that for
diagnosis IgM and IgG should be detected simultaneously at the early
phase of infection.</p>
<p>IgG titers, but not IgM titers were higher in severe patients compared
to non-severe patients after controlling for days post-symptom onset.
Importantly, 12% of COVID-19 patients (RT-PCR confirmed) did not meet
the WHO serological diagnosis criterion of either seroconversion or
<em>&gt;</em> 4-fold increase in IgG titer in sequential samples. This suggests
the current serological criteria may be too stringent for COVID-19
diagnosis.</p>
<p>Of note, 4 patients from a group of 52 suspects (negative RT-PCR test)
had anti-SARS-Cov-2 IgM and IgG. Similarly, 4.3% (7/162) of “close
contacts” who had negative RT-PCR tests were positive for IgG and/or
IgM. This highlights the usefulness of a serological assay to identify
asymptomatic infections and/or infections that are missed by RT-PCR.</p>
<h3 data-number="10.68.3" id="limitations-65"><span class="header-section-number">10.68.3</span> Limitations</h3>
<p>This group’s report generally confirms the findings of others that have
evaluated the acute antibody response to SARS-Cov-2. However, these data
would benefit from inclusion of data on whether the participants had a
documented history of viral infection. Moreover, serum samples that were
collected prior to SARS-Cov-2 outbreak from patients with other viral
infections would serve as a useful negative control for their assay.
Methodological limitations include that only one serum sample per case
was tested as well as the heat inactivation of serum samples prior to
testing. It has previously been reported that heat inactivation
interferes with the level of antibodies to SARS-Cov-2 and their protocol
may have resulted in diminished quantification of IgM, specifically <span class="citation" data-cites="p1c9Nilh">[<a href="#ref-p1c9Nilh" role="doc-biblioref">843</a>]</span>.</p>
<h3 data-number="10.68.4" id="significance-62"><span class="header-section-number">10.68.4</span> Significance</h3>
<p>Understanding the features of the antibody responses against SARS-CoV is
useful in the development of a serological test for the diagnosis of
COVID-19. This paper addresses the need for additional screening methods
that can detect the presence of infection despite lower viral titers.
Detecting the production of antibodies, especially IgM, which are
produced rapidly after infection can be combined with PCR to enhance
detection sensitivity and accuracy and map the full spread of infection
in communities, Moreover, serologic assays would be useful to screen
health care workers in order to identify those with immunity to care for
patients with COVID19.</p>
<h3 data-number="10.68.5" id="credit-67"><span class="header-section-number">10.68.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.69" id="sars-cov-2-specific-antibody-responses-in-covid-19-patients"><span class="header-section-number">10.69</span> SARS-CoV-2 specific antibody responses in COVID-19 patients</h2>
<p><span class="citation" data-cites="1BLNvgEao">[<a href="#ref-1BLNvgEao" role="doc-biblioref">844</a>]</span></p>
<h3 data-number="10.69.1" id="keywords-68"><span class="header-section-number">10.69.1</span> Keywords</h3>
<ul>
<li>immunoassay</li>
<li>antibody specificity</li>
<li>serology</li>
<li>cross-reactivity</li>
</ul>
<h3 data-number="10.69.2" id="main-findings-65"><span class="header-section-number">10.69.2</span> Main findings</h3>
<p>Antibodies specific to SARS-CoV-2 S protein, the S1 subunit and the RBD (receptor-binding domain) were detected in all SARS-CoV-2 patient sera by 13 to 21 days post onset of disease. Antibodies specific to SARS-CoV N protein (90% similarity to SARS-CoV-2) were able to neutralize SARS-CoV-2 by PRNT (plaque reduction neutralizing test). SARS-CoV-2 serum cross-reacted with SARS-CoV S and S1 proteins, and to a lower extent with MERS-CoV S protein, but not with the MERS-CoV S1 protein, consistent with an analysis of genetic similarity. No reactivity to SARS-CoV-2 antigens was observed in serum from patients with ubiquitous human CoV infections (common cold) or to non-CoV viral respiratory infections.</p>
<h3 data-number="10.69.3" id="limitations-66"><span class="header-section-number">10.69.3</span> Limitations</h3>
<p>Authors describe development of a serological ELISA based assay for the detection of neutralizing antibodies towards regions of the spike and nucleocapsid domains of the SARS-CoV-2 virus. Serum samples were obtained from PCR-confirmed COVID-19 patients. Negative control samples include a cohort of patients with confirmed recent exposure to non-CoV infections (i.e. adenovirus, bocavirus, enterovirus, influenza, RSV, CMV, EBV) as well as a cohort of patients with confirmed infections with ubiquitous human CoV infections known to cause the common cold. The study also included serum from patients with previous MERS-CoV and SARS-CoV zoonotic infections. This impressive patient cohort allowed the authors to determine the sensitivity and specificity of the development of their in-house ELISA assay. Of note, seroconversion was observed as early as 13 days following COVID-19 onset but the authors were not clear how disease onset was determined.</p>
<h3 data-number="10.69.4" id="significance-63"><span class="header-section-number">10.69.4</span> Significance</h3>
<p>Validated serological tests are urgently needed to map the full spread of SARS-CoV-2 in the population and to determine the kinetics of the antibody response to SARS-CoV-2. Furthermore, clinical trials are ongoing using plasma from patients who have recovered from SARS-CoV-2 as a therapeutic option. An assay such as the one described in this study could be used to screen for strong antibody responses in recovered patients. Furthermore, the assay could be used to screen health care workers for antibody responses to SARS-CoV-2 as personal protective equipment continues to dwindle. The challenge going forward will be to standardize and scale-up the various in-house ELISA’s being developed in independent laboratories across the world.</p>
<h2 data-number="10.70" id="a-brief-review-of-antiviral-drugs-evaluated-in-registered-clinical-trials-for-covid-19"><span class="header-section-number">10.70</span> A brief review of antiviral drugs evaluated in registered clinical trials for COVID-19</h2>
<p>Belhadi et al. <span class="citation" data-cites="K41YFM2N">[<a href="#ref-K41YFM2N" role="doc-biblioref">845</a>]</span></p>
<h3 data-number="10.70.1" id="keywords-69"><span class="header-section-number">10.70.1</span> Keywords</h3>
<ul>
<li>Clinical trials</li>
<li>COVID-19</li>
<li>SARS CoV-2</li>
<li>2019-nCoV</li>
<li>SARS Cov-2</li>
<li>Hcov-19</li>
<li>novel coronal virus</li>
<li>new corona virus</li>
<li>antiviral drugs</li>
</ul>
<h3 data-number="10.70.2" id="main-findings-66"><span class="header-section-number">10.70.2</span> Main Findings</h3>
<p>Summary of clinical trials registered as
of March7, 2020 from U.S, Chinese, Korean, Iranian and European
registries. Out of the 353 studies identified, 115 were selected for
data extraction. 80% of the trials were randomized with parallel
assignment and the median number of planned inclusions was 63 (IRQ,
36-120). Most frequent therapies in the trials included; 1) antiviral
drugs [lopinavir/ritonavir (n-15); umifenovir (n=9); favipiravir (n=7);
redmesivir (n=5)]; 2) anti-malaria drugs [chloroquine (n-11);
hydroxychloroquine (n=7)}; immunosuppressant drugs [methylprednisolone
(n=5)]; and stem cell therapies (n=23). Medians of the total number of
planned inclusions per trial for these therapies were also included.
Stem cells and lopunavir/ritonavir were the most frequently evaluated
candidate therapies (23 and 15 trials respectively), whereas remdesivir
was only tested in 5 trials but these trials had the highest median
number of planned inclusions per trial (400, IQR 394-453). Most of the
agents used in the different trials were chosen based on preclinical
assessments of antiviral activity against SARS CoV and MERS Cov corona
viruses.</p>
<p>The primary outcomes of the studies were clinical (66%); virological
(23%); radiological (8%); or immunological (3%). The trials were
classified as those that included patients with severe disease only;
trials that included patients with moderate disease; and trials that
included patients with severe or moderate disease.</p>
<h3 data-number="10.70.3" id="limitations-67"><span class="header-section-number">10.70.3</span> Limitations</h3>
<p>The trials evaluated provided incomplete information:
23% of these were phase IV trials but the bulk of the trials (54%) did
not describe the phase of the study. Only 52% of the trials (n=60)
reported treatment dose and only 34% (n=39) reported the duration. A lot
of the trials included a small number of patients and the trials are
still ongoing, therefore no insight was provided on the outcome of the
trials.</p>
<h3 data-number="10.70.4" id="significance-64"><span class="header-section-number">10.70.4</span> Significance</h3>
<p>Nonetheless, this review serves as framework for
identifying COVID-19 related trials, which can be expanded upon as new
trials begin at an accelerated rate as the disease spreads around the
world.</p>
<h3 data-number="10.70.5" id="credit-68"><span class="header-section-number">10.70.5</span> Credit</h3>
<p><em>This review was undertaken by K Alexandropoulos as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.71" id="ace-2-expression-in-the-small-airway-epithelia-of-smokers-and-copd-patients-implications-for-covid-19"><span class="header-section-number">10.71</span> ACE-2 Expression in the Small Airway Epithelia of Smokers and COPD Patients: Implications for COVID-19</h2>
<p>Leung et al. <em>medRxiv</em>. <span class="citation" data-cites="eTsOvBVf">[<a href="#ref-eTsOvBVf" role="doc-biblioref">846</a>]</span></p>
<h3 data-number="10.71.1" id="keywords-70"><span class="header-section-number">10.71.1</span> Keywords</h3>
<ul>
<li>chronic obstructive pulmonary disease</li>
<li>COPD</li>
<li>smokingE-2</li>
<li>risk factors</li>
</ul>
<h3 data-number="10.71.2" id="main-findings-67"><span class="header-section-number">10.71.2</span> Main Findings</h3>
<p>In bronchial epithelial samples from 3 different cohorts of
individuals, ACE-2 gene expression was found to be significantly
increased in both COPD patients and smokers relative to healthy
controls. Across all test subjects, ACE-2 gene expression was also
highly correlated with decreased forced expiratory volume in 1 second
(FEV1), which may explain the increased COVID-19 disease severity in
COPD patients. Former smokers were also found to show decreased ACE2
expression relative to current smokers and had no significant difference
when compared to non-smokers.</p>
<h3 data-number="10.71.3" id="limitations-68"><span class="header-section-number">10.71.3</span> Limitations</h3>
<p>While the upregulation of ACE-2 is an interesting
hypothesis for COVID-19 disease severity in COPD patients, this study
leaves many more unanswered questions than it addresses. Further studies
are required to show whether the specific cell type isolated in these
studies is relevant to the pathophysiology of COVID-19. Furthermore,
there is no attempt to show whether that increased ACE-2 expression
contributes to greater disease severity. Does the increased ACE-2
expression lead to greater infectivity with SARS-CoV-2? There is no
mechanistic explanation for why ACE-2 levels are increased in COPD
patients. The authors could also have considered the impact of
co-morbidities and interventions such as corticosteroids or
bronchodilators on ACE-2 expression. Finally, given the extensive
sequencing performed, the authors could have conducted significantly
more in-depth analyses into gene signature differences.</p>
<h3 data-number="10.71.4" id="significance-65"><span class="header-section-number">10.71.4</span> Significance</h3>
<p>This study attempts to address an
important clinical finding that both smokers and COPD patients show
increased mortality from COVID-19. The novel finding that ACE-2
expression is induced in smokers and COPD patients suggests not only a
mechanism for the clinical observation, but also highlights the
potential benefit of smoking cessation in reducing the risk of severe
COVID-19 disease.</p>
<h3 data-number="10.71.5" id="credit-69"><span class="header-section-number">10.71.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.72" id="dynamic-profile-of-severe-or-critical-covid-19-cases"><span class="header-section-number">10.72</span> Dynamic profile of severe or critical COVID-19 cases</h2>
<p><span class="citation" data-cites="18KZGwJ0J">[<a href="#ref-18KZGwJ0J" role="doc-biblioref">847</a>]</span></p>
<h3 data-number="10.72.1" id="keywords-71"><span class="header-section-number">10.72.1</span> Keywords</h3>
<ul>
<li>Coronavirus Disease 2019 (COVID-19)</li>
<li>SARS-CoV-2</li>
<li>progressive lymphopenia (PLD)</li>
<li>T-lymphocytes</li>
<li>clinical data</li>
<li>co-infection</li>
<li>influenza A</li>
</ul>
<h3 data-number="10.72.2" id="main-findings-68"><span class="header-section-number">10.72.2</span> Main Findings</h3>
<p>Authors evaluate clinical correlates of 10 patients (6 male
and 4 female) hospitalized for severe Acute Respiratory Syndrome
Coronavirus 2 (SARS-CoV-2). All patients required oxygen support and
received broad spectrum antibiotics and 6 patients received anti-viral
drugs. Additionally, 40% of patients were co-infected with influenza A.
All 10 patients developed lymphopenia, two of which developed
progressive lymphopenia (PLD) and died. Peripheral blood (PB)
lymphocytes were analyzed – low CD4 and CD8 counts were noted in most
patients, though CD4:CD8 ratio remained normal.</p>
<h3 data-number="10.72.3" id="limitations-69"><span class="header-section-number">10.72.3</span> Limitations</h3>
<p>The authors evaluated a small cohort of severe
SARS-CoV-2 cases and found an association between T cell lymphopenia and
adverse outcomes. However, this is an extremely small and diverse cohort
(40% of patients were co-infected with influenza A). These findings need
to be validated in a larger cohort. Additionally, the value of this data
would be greatly increased by adding individual data points for each
patient as well as by adding error bars to each of the figures.</p>
<h3 data-number="10.72.4" id="significance-66"><span class="header-section-number">10.72.4</span> Significance</h3>
<p>This study provides a collection of clinical data and
tracks evolution of T lymphocyte in 10 patients hospitalized for
SARS-CoV-2, of which 4 patients were co-infected with influenza A.</p>
<h3 data-number="10.72.5" id="credit-70"><span class="header-section-number">10.72.5</span> Credit</h3>
<p><em>This review was undertaken by Katherine Lindblad and Tamar Plitt as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.73" id="association-between-clinical-laboratory-and-ct-characteristics-and-rt-pcr-results-in-the-follow-up-of-covid-19-patients"><span class="header-section-number">10.73</span> Association between Clinical, Laboratory and CT Characteristics and RT-PCR Results in the Follow-up of COVID-19 patients</h2>
<p>Fu et al. <em>medRxiv</em>. <span class="citation" data-cites="1BJXDZ04c">[<a href="#ref-1BJXDZ04c" role="doc-biblioref">848</a>]</span></p>
<h3 data-number="10.73.1" id="keywords-72"><span class="header-section-number">10.73.1</span> Keywords</h3>
<ul>
<li>COVID-19</li>
<li>clinical</li>
<li>lymphocyte</li>
<li>CRP</li>
<li>LDH</li>
<li>HSST TNT</li>
<li>PCR test</li>
<li>readmission</li>
<li>CT</li>
<li>GGO</li>
<li>disease progression</li>
</ul>
<h3 data-number="10.73.2" id="study-description"><span class="header-section-number">10.73.2</span> Study description</h3>
<p>Data analyzed from 52 COVID-19 patients admitted
and then discharged with COVID-19. Clinical, laboratory, and
radiological data were longitudinally recorded with illness timecourse
(PCR + to PCR-) and 7 patients (13.5%) were readmitted with a follow up
positive test (PCR+) within two weeks of discharge.</p>
<h3 data-number="10.73.3" id="main-findings-69"><span class="header-section-number">10.73.3</span> Main Findings</h3>
<ul>
<li><p>At admission:</p>
<ul>
<li><p>The majority of patients had increased CRP at admission (63.5%).</p></li>
<li><p>LDH, and HSST TNT were significantly increased at admission.</p></li>
<li><p>Radiographic signs via chest CT showed increased involvement in
lower lobes: right lower lobe (47 cases, 90.4%), left lower lobe
(37 cases, 71.2%).</p></li>
<li><p>GGO (90.4%), interlobular septal thickening (42.3%), vascular
enlargement (42.3%), and reticulation (11.5%) were most commonly
observed.</p></li>
</ul></li>
<li><p>After negative PCR test (discharge):</p>
<ul>
<li><p>CRP levels decreased lymphocyte counts (#/L) increased
significantly (CD3+, CD3+/8+ and CD3+/4+) after negative PCR.</p></li>
<li><p>Consolidation and mixed GGO observed in longitudinal CT imaging
w different extents of inflammatory exudation in lungs, with
overall tendency for improvement (except 2/7 patients that were
readmitted after discharge with re-positive test) after negative
PCR.</p></li>
</ul></li>
<li><p>Seven patients repeated positive RT-PCR test and were readmitted to
the hospital (9 to 17 day after initial discharge).</p>
<ul>
<li><p>Follow up CT necessary to monitor improvement during recovery
and patients with lesion progression should be given more
attention.</p></li>
<li><p>Dynamic CT in addition to negative test essential in clinical
diagnosis due to nasal swab PCR sampling bias (false-negatives).</p></li>
<li><p>Increase in CRP occurred in 2 readmitted patients (and decr. in
lymphocytes in one patient), but was not correlated with new
lesions or disease progression vs. improvement (very low N).</p></li>
<li><p>Patients readmitted attributed to false-negative PCR vs.
re-exposure.</p></li>
</ul></li>
</ul>
<h3 data-number="10.73.4" id="limitations-70"><span class="header-section-number">10.73.4</span> Limitations</h3>
<p>Patients sampled in this study were generally
younger (65.4% &lt; 50 yrs) and less critically ill/all discharged.
Small number of recovered patients (N=18). Time of follow up was
relatively short.. Limited clinical information available about patients
with re-positive test (except CRP and lymph tracking).</p>
<h3 data-number="10.73.5" id="extended-results"><span class="header-section-number">10.73.5</span> Extended Results</h3>
<p><em>NOTE</em>: Patients sampled in this study were generally younger (65.4%
&lt; 50 yrs) and less critically ill/all discharged. After two
consecutive negative PCR tests, patients were discharged.</p>
<p>Clinical Results at Admission</p>
<ul>
<li><p>Median interval disease onset to admission (5 days, IQR: 3-7)</p></li>
<li><p>Most common symptoms included fever, fatigue, dry cough, and
expectoration.</p></li>
<li><p>Fifteen patients had reduced lymphocyte counts (28.8%).</p></li>
<li><p>No change in WBC or Neutrophil counts.</p></li>
<li><p><strong>The majority of patients had increased CRP at admission (63.5%).</strong></p></li>
<li><p><strong>LDH, and HSST TNT were significantly increased at admission.</strong></p></li>
<li><p>Fibrinogen was trending high though not significant.</p></li>
<li><p>No major changes in liver function observed.</p></li>
<li><p><strong>Radiographic signs via chest CT showed increased involvement in
lower lobes: right lower lobe (47 cases, 90.4%), left lower lobe (37
cases, 71.2%).</strong></p></li>
<li><p><strong>GGO (90.4%), interlobular septal thickening (42.3%), vascular
enlargement (42.3%)</strong>, and reticulation (11.5%) were most commonly
observed.</p></li>
</ul>
<p>Change in Clinical Results following Negative Test</p>
<ul>
<li><p><strong>CRP levels decreased after negative PCR.</strong></p></li>
<li><p><strong>Lymphocyte counts (#/L) increased significantly (CD3+, CD3+/8+
and CD3+/4+).</strong></p></li>
<li><p>No significant change to CD4/8 ratio.</p></li>
<li><p>LDH, HSST TNT, and Fibronegin remained high throughout, though range
observed decreased over time.</p></li>
<li><p><strong>Consolidation and mixed GGO observed in longitudinal CT imaging.</strong></p></li>
<li><p><strong>Patients showed different extents of inflammatory exudation in
lungs, with overall tendency for improvement (except 2/7 patients
that were readmitted after discharge with re-positive test).</strong></p></li>
</ul>
<p>Patients Readmitted with PCR+ test</p>
<ul>
<li><p><strong>Seven patients repeated positive RT-PCR test and were readmitted
to the hospital (9 to 17 day after initial discharge).</strong></p></li>
<li><p>Improvement during readmission in 4 patients and observation of
segmental progression CT in 2 patients (2/18 or 11.1% - re-positive
9 and 10 days post-discharge).</p></li>
<li><p>Two patients showed new GGO, while others improved greatly.</p></li>
<li><p><strong>Follow up CT necessary to monitor improvement during recovery and
patients with lesion progression should be given more attention.</strong></p></li>
<li><p><strong>Dynamic CT in addition to negative test essential in clinical
diagnosis due to nasal swab PCR sampling bias (false-negatives).</strong></p></li>
<li><p><strong>Increase in CRP occurred in 2 readmitted patients (and decr. in
lymphocytes in one patient), but was not correlated with new lesions
or disease progression vs. improvement (very low N).</strong></p></li>
</ul>
<h3 data-number="10.73.6" id="significance-67"><span class="header-section-number">10.73.6</span> Significance</h3>
<p>Study tracked key clinical features associated with
disease progression, recovery, and determinants of clinical
diagnosis/management of COVID-19 patients.</p>
<h3 data-number="10.73.7" id="credit-71"><span class="header-section-number">10.73.7</span> Credit</h3>
<p><em>This review was undertaken by Natalie Vaninov as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.74" id="an-orally-bioavailable-broad-spectrum-antiviral-inhibits-sars-cov-2-and-multiple-2-endemic-epidemic-and-bat-coronavirus"><span class="header-section-number">10.74</span> An orally bioavailable broad-spectrum antiviral inhibits SARS-CoV-2 and multiple 2 endemic, epidemic and bat coronavirus</h2>
<p>Sheahan et al. <em>bioRxiv</em>. <span class="citation" data-cites="iMCz5Twj">[<a href="#ref-iMCz5Twj" role="doc-biblioref">849</a>]</span></p>
<h3 data-number="10.74.1" id="keywords-73"><span class="header-section-number">10.74.1</span> Keywords</h3>
<ul>
<li>Treatment</li>
<li>Antiviral</li>
<li>Broad spectrum antiviral</li>
<li>ribonucleoside analog β-D-N4 30 hydroxycytidine (NHC)</li>
<li>Remdesivir</li>
</ul>
<h3 data-number="10.74.2" id="main-findings-70"><span class="header-section-number">10.74.2</span> Main Findings</h3>
<p>β-D-N4 30 –hydroxycytidine (NHC, EIDD-1931) is an
orally bioavailable ribonucleoside with antiviral activity against
various RNA viruses including Ebola, Influenza and CoV. NHC activity
introduce<em>d</em><del>s</del> mutations in the viral (but not cellular) RNA in a
dose dependent <em>manner</em> that directly correlated with a decrease in
viral titers. Authors show that NHC inhibited multiple genetically
distinct Bat-CoV viruses in human primary epithelial cells <em>without
affecting</em> <em>cell viability even at high concentrations (100</em> <em>µM)</em>.
Prophylactic oral administration of NHC in C57BL/6 mice reduce lung
titers of SARS-CoV and prevented weight loss and hemorrhage. Therapeutic
administration of NHC in C57BL/6 mice 12 hours post infected with
SARS-CoV reduced acute lung injury, viral titer, and lung hemorrhage.
The degree of clinical benefit was dependent on the time of treatment
initiation post infection. The authors also demonstrate that NHC reduces
MERS-CoV infection titers, pathogenesis, and viral RNA in prophylactic
and therapeutic settings.</p>
<h3 data-number="10.74.3" id="limitations-71"><span class="header-section-number">10.74.3</span> Limitations</h3>
<p>Most of the experiments were conducted using MERS-CoV, and
SARS-CoV and a few experiments were conducted using other strains of CoV
as opposed to SARS-CoV-2. The authors note the core residues that make
up the RNA interaction sites (which constitutes the NHC interaction
sites) are highly conserved among CoV and because of this conservation
their understanding is that NHC can inhibit a broad-spectrum of CoV
including SARS-CoV-2.</p>
<p>The increased viral mutation rates associated with NHC activity may have
adverse effects if mutations cause the virus to become drug resistant,
more infectious or speed-up immune evasion. <em>In addition, the temporal
diminishing effectiveness of NHC on clinical outcome when NHC was used
therapeutically is concerning. However, the longer window (7-10 days)
for clinical disease onset in human patients from the time of infection
compared to that of mice (24-48 hours), may associate with increased NHC
effectiveness in the clinic.</em></p>
<h3 data-number="10.74.4" id="significance-68"><span class="header-section-number">10.74.4</span> Significance</h3>
<p>Prophylactic or therapeutic oral administration of NHC
reduces lung titers and prevents acute lung failure in C57B\6 mice
infected with CoV. Given its <em>broad-spectrum antiviral activity, NHC
could turn out to be a useful drug for treating current, emerging and
future corona virus outbreaks.</em>
 
#### Credit</p>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.75" id="identification-of-antiviral-drug-candidates-against-sars-cov-2-from-fda-approved-drugs"><span class="header-section-number">10.75</span> Identification of antiviral drug candidates against SARS-CoV-2 from FDA-approved drugs</h2>
<p>Sangeun Jeon et al. <span class="citation" data-cites="ZDP9vSpg">[<a href="#ref-ZDP9vSpg" role="doc-biblioref">850</a>]</span></p>
<h3 data-number="10.75.1" id="keywords-74"><span class="header-section-number">10.75.1</span> Keywords</h3>
<ul>
<li>COVID-19</li>
<li>SARS CoV-2</li>
<li>antiviral drugs</li>
<li>niclosamide</li>
<li>ciclesonide</li>
</ul>
<h3 data-number="10.75.2" id="main-findings-71"><span class="header-section-number">10.75.2</span> Main Findings</h3>
<p>A panel of ~3,000 FDA- and IND-approved
antiviral drugs were previously screened for inhibitory efficacy against
SARS CoV, a coronavirus related to the novel coronavirus SARS CoV-2
(79.5%) homology. 35 of these drugs along with another 15 (suggested by
infectious disease specialists) were tested in vitro for their ability
to inhibit SARS CoV-2 infectivity of Vero cells while preserving cell
viability. The infected cells were scored by immunofluorescence analysis
using an antibody against the N protein of SARS CoV-2. Chloroquine,
lopinavir and remdesivir were used as reference drugs.</p>
<p>Twenty four out of 50 drugs exhibited antiviral activity with IC50
values ranging from 0.1-10µM. Among these, two stood ou: 1) the-anti
helminthic drug niclosamide which exhibited potent antiviral activity
against SARS CoV-2 (IC50=0.28 µM). The broad-spectrum antiviral effect
of niclosamide against SARS and MERS-CoV have been previously documented
and recent evidence suggests that in may inhibit autophagy and reduce
MERS C0V replication. 2) Ciclesonide, a corticosteroid used to treat
asthma and allergic rhinitis, also exhibited antiviral efficacy but with
a lower IC50 (4.33µM) compared to niclosamide. The antiviral effects of
ciclesonide were directed against NSP15, a viral riboendonuclease which
is the molecular target of this drug.</p>
<h3 data-number="10.75.3" id="limitations-72"><span class="header-section-number">10.75.3</span> Limitations</h3>
<p>The drugs were tested against SARS CoV-2 infectivity in
vitro only, therefore preclinical studies in animals and clinical trials
in patients will be need for validation of these drugs as therapeutic
agents for COVID-19. In addition, niclosamide exhibits low adsorption
pharmatokinetically which could be alleviated with further development
of drug formulation to increase effective delivery of this drug to
target tissues. Nonetheless, niclosamide and ciclesonide represent
promising therapeutic agents against SARS CoV-2 given that other
compounds tested in the same study including favipiravir (currently used
in clinical trials) and atazanavir (predicted as the most potent
antiviral drug by AI-inference modeling) did not exhibit antiviral
activity in the current study.</p>
<h3 data-number="10.75.4" id="credit-72"><span class="header-section-number">10.75.4</span> Credit</h3>
<p><em>This review was undertaken by K Alexandropoulos as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.76" id="respiratory-disease-and-virus-shedding-in-rhesus-macaques-inoculated-with-sars-cov-2"><span class="header-section-number">10.76</span> Respiratory disease and virus shedding in rhesus macaques inoculated with SARS-CoV-2</h2>
<p>Munster et al. <em>bioRxiv</em>. <span class="citation" data-cites="LYFzvlzE">[<a href="#ref-LYFzvlzE" role="doc-biblioref">851</a>]</span></p>
<h3 data-number="10.76.1" id="keywords-75"><span class="header-section-number">10.76.1</span> Keywords</h3>
<p>animal model, pulmonary infiltrates, dynamic
of antibody response, cytokine</p>
<h3 data-number="10.76.2" id="main-findings-72"><span class="header-section-number">10.76.2</span> Main Findings</h3>
<p>Inoculation of 8 Resus macaques with SARS-Cov-2 , which
all showed clinical signs of infection (respiratory pattern, reduced
appetite, weight loss, elevated body temperature) resulting in moderate,
transient disease. Four animals were euthanized at 3dpi, the 4 others at
21 dpi. Study of viral loads in different organs showed that nose swab
and throat swabs were the most sensitive, with bronchio-alveolar lavage.
Interstitial pneumonia was visible in radiographies and at the
histological scale too. Clinically, the macaques had similar symptoms as
described in human patients with moderate disease.</p>
<p>Viral shedding was consistently detected in nose swabs and throat swabs
immediately after infection but less consistent thereafter which could
reflect virus administration route (intranasal, oral). Bronchoalveolar
lavages performed as a measure of virus replication in the lower
respiratory tract on animals maintained for 21 days, contained high
viral loads in 1 and 3dpi. The majority of the animals exhibited
pulmonary edema and mild to moderate interstitial pneumonia on terminal
bronchioles. In addition to the lung, viral RNA could also be detected
throughout the respiratory track where viral replication mainly
occurred.</p>
<p>Immunologic responses included leukocytosis, neutrophilia, monocytosis
and lymphopenia in the majority of the animals at 1dpi. Lymphocytes and
monocytes re-normalized at 2dpi. Neutrophils declined after 3dpi and
through 10dpi after which they started to recover. After infection,
serum analysis revealed significant increases in <strong>IL1ra, IL6, IL10,
IL15, MCP-1, MIP-1b, but quick normalization</strong> (3dpi). <strong>Antibody
response started around 7dpi, and the antibody titers stayed elevated
until 21dpi</strong> (day of animal euthanasia).</p>
<h3 data-number="10.76.3" id="limitations-73"><span class="header-section-number">10.76.3</span> Limitations</h3>
<p>The macaques were inoculated via a combination
of intratracheal, intranasal, ocular and oral routes, which might not
reproduce how humans get infected. Maybe this can lead to different
dynamics in the host immune response. Also, the authors noted that the
seroconversion was not directly followed by a decline in viral loads, as
observed in covid19 patients.</p>
<h3 data-number="10.76.4" id="significance-69"><span class="header-section-number">10.76.4</span> Significance</h3>
<p>This work confirms that rhesus
macaques can be a good model to study Covid-19, as it has been shown by
other groups <span class="citation" data-cites="3XqFhi8l unVgVIfU 1EcOLnVXk">[<a href="#ref-3XqFhi8l" role="doc-biblioref">826</a>,<a href="#ref-1EcOLnVXk" role="doc-biblioref">839</a>,<a href="#ref-unVgVIfU" role="doc-biblioref">852</a>]</span>. While these experiments recapitulate
moderate COVID-19 in humans, the mode of inoculation via a combination
of intratracheal, intranasal, ocular and oral routes, might not
reproduce how humans get infected and may lead to different dynamics in
the host immune response. For example, the authors noted that the
seroconversion was not directly followed by a decline in viral loads, as
observed in COVID-19 patients. Therefore, it will be interesting to
follow their antibody titers longer and further assess the
possibility/effect of reinfection in these macaques. It is essential to
be able to understand the dynamic of the disease and associated immune
responses, and to work on vaccine development and antiviral drug
testing.</p>
<h3 data-number="10.76.5" id="credit-73"><span class="header-section-number">10.76.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.77" id="ace2-expression-is-increased-in-the-lungs-of-patients-with-comorbidities-associated-with-severe-covid-19"><span class="header-section-number">10.77</span> ACE2 Expression is Increased in the Lungs of Patients with Comorbidities Associated with Severe COVID-19</h2>
<p><span class="citation" data-cites="wjtVy2v0">[<a href="#ref-wjtVy2v0" role="doc-biblioref">853</a>]</span></p>
<h3 data-number="10.77.1" id="keywords-76"><span class="header-section-number">10.77.1</span> Keywords</h3>
<ul>
<li>CoVID-19</li>
<li>2019-nCoV</li>
<li>SARS-CoV-2</li>
<li>lung</li>
<li>comorbidities</li>
<li>histone</li>
<li>epigenetics</li>
</ul>
<h3 data-number="10.77.2" id="main-findings-73"><span class="header-section-number">10.77.2</span> Main Findings</h3>
<ul>
<li><p>Transcriptomic analysis using systems-level meta-analysis and
network analysis of existing literature to determine ACE2 regulation
in patients who have frequent COVID-19 comorbidities [eg-
cardiovascular diseases, familial pulmonary hypertension, cancer].</p></li>
<li><p>Enrichment analyses indicated pathways associated with inflammation,
metabolism, macrophage autophagy, and ER stress.</p></li>
<li><p>ACE2 higher in adenocarcinoma compared to adjacent normal lung; ACE2
higher in COPD patients compared to normal.</p></li>
<li><p>Co-expression analysis identified genes important to viral entry
such as RAB1A, ADAM10, HMGBs, and TLR3 to be associated with ACE2 in
diseased lungs.</p></li>
<li><p>ACE2 expression could be potentially regulated by enzymes that
modify histones, including HAT1, HDAC2, and KDM5B.</p></li>
</ul>
<h3 data-number="10.77.3" id="limitations-74"><span class="header-section-number">10.77.3</span> Limitations:</h3>
<ul>
<li><p>Not actual CoVID-19 patients with co-morbidities, so interpretations
in this study need to be confirmed by analyzing upcoming
transcriptomics from CoVID-19 patients having co-morbidity metadata.</p></li>
<li><p>As mentioned by authors, study does not look at diabetes and
autoimmunity as risk factors in CoVID-19 patients due to lack of
data; would be useful to extend such analyses to those datasets when
available.</p></li>
<li><p>Co-expression analysis is perfunctory and needs
validation-experiments especially in CoVID-19 lung samples to mean
anything.</p></li>
<li><p>Epigenomic analyses are intriguing but incomplete, as existence of
histone marks does not necessarily mean occupancy. Would be
pertinent to check cell-line data (CCLE) or actual CoVID-19 patient
samples to confirm ACE2 epigenetic control.</p></li>
</ul>
<h3 data-number="10.77.4" id="significance-70"><span class="header-section-number">10.77.4</span> Significance</h3>
<ul>
<li><p>Study implies vulnerable populations have ACE2 upregulation that
could promote CoVID-19 severity. Shows important data-mining
strategy to find gene-networks associated with ACE2 upregulation in
co-morbid patients.</p></li>
<li><p>Several of the genes co-upregulated with ACE2 in diseased lung might
play an important role in CoVID-19 and can be preliminary targets
for therapeutics</p></li>
<li><p>If in silico findings hold true, epigenetic control of ACE2
expression could be a new target for CoVID-19 therapy with
strategies such as KDM5 demethylases.</p></li>
</ul>
<h3 data-number="10.77.5" id="credit-74"><span class="header-section-number">10.77.5</span> Credit</h3>
<p><em>Review by Samarth Hegde as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.78" id="meplazumab-treats-covid-19-pneumonia-an-open-labelled-concurrent-controlled-add-on-clinical-trial"><span class="header-section-number">10.78</span> Meplazumab treats COVID-19 pneumonia: an open-labelled, concurrent controlled add-on clinical trial</h2>
<p>Bian et al. <em>medRxiv</em>. <span class="citation" data-cites="DFhY5nfb">[<a href="#ref-DFhY5nfb" role="doc-biblioref">854</a>]</span></p>
<h3 data-number="10.78.1" id="keywords-77"><span class="header-section-number">10.78.1</span> Keywords</h3>
<ul>
<li>Meplazumab</li>
<li>CD147</li>
<li>humanized antibody</li>
<li>clinical trial</li>
</ul>
<h3 data-number="10.78.2" id="main-findings-74"><span class="header-section-number">10.78.2</span> Main Findings</h3>
<p>This work is based on previous work by the same group
that demonstrated that SARS-CoV-2can also enter host cells via CD147
(also called Basigin, part of the immunoglobulin superfamily, is
expressed by many cell types) consistent with their previous work with
SARS-CoV-1. <sup>1</sup> A prospective clinical trial was conducted with 17
patients receiving Meplazumab, a humanized anti-CD147 antibody, in
addition to all other treatments. 11 patients were included as a control
group (non-randomized).</p>
<p>They observed a faster overall improvement rate in the Meplazumab group
(e.g. at day 14 47% vs 17% improvement rate) compared to the control
patients. Also, virological clearance was more rapid with median of 3
days in the Meplazumab group vs 13 days in control group. In laboratory
values, a faster normalization of lymphocyte counts in the Meplazumab
group was observed, but no clear difference was observed for CRP levels.</p>
<h3 data-number="10.78.3" id="limitations-75"><span class="header-section-number">10.78.3</span> Limitations</h3>
<p>While the results from the study are encouraging, this
study was non-randomized, open-label and on a small number of patients,
all from the same hospital. It offers evidence to perform a larger scale
study. Selection bias as well as differences between treatment groups
(e.g. age 51yo vs 64yo) may have contributed to results. The authors
mention that there was no toxic effect to Meplazumab injection but more
patient and longer-term studies are necessary to assess this.</p>
<h3 data-number="10.78.4" id="significance-71"><span class="header-section-number">10.78.4</span> Significance</h3>
<p>These results seem promising as for now there are
limited treatments for Covid-19 patients, but a larger cohort of patient
is needed. CD147 has already been described to facilitate HIV <span class="citation" data-cites="yP8TXdq2">[<a href="#ref-yP8TXdq2" role="doc-biblioref">855</a>]</span>,
measles virus <span class="citation" data-cites="X8l1n4zY">[<a href="#ref-X8l1n4zY" role="doc-biblioref">856</a>]</span>, and malaria <span class="citation" data-cites="YcyNaIVC">[<a href="#ref-YcyNaIVC" role="doc-biblioref">857</a>]</span> entry into host cells. This group was
the first to describe the CD147-spike route of SARS-Cov-2 entry in host
cells <span class="citation" data-cites="14y8UvAbH">[<a href="#ref-14y8UvAbH" role="doc-biblioref">828</a>]</span> p147. Indeed, they had previously shown in 2005 that SARS-Cov
could enter host cells via this transmembrane protein <span class="citation" data-cites="m9yLJ5Da">[<a href="#ref-m9yLJ5Da" role="doc-biblioref">858</a>]</span>. Further
biological understanding of how SARS-CoV-2 can enter host cells and how
this integrates with ACE2R route of entry is needed. Also, the specific
cellular targets of the anti-CD147 antibody need to be assessed, as this
protein can be expressed by many cell types and has been shown to
involved in leukocytes aggregation <span class="citation" data-cites="J0ibHE3l">[<a href="#ref-J0ibHE3l" role="doc-biblioref">859</a>]</span>. Lastly, Meplazumab is not a
commercially-available drug and requires significant health resources to
generate and administer which might prevent rapid development and use.</p>
<h3 data-number="10.78.5" id="credit-75"><span class="header-section-number">10.78.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.79" id="potent-human-neutralizing-antibodies-elicited-1-by-sars-cov-2-infection"><span class="header-section-number">10.79</span> Potent human neutralizing antibodies elicited 1 by SARS-CoV-2 infection</h2>
<p>Ju et al. <em>bioRxiv</em>. <span class="citation" data-cites="GhJYjnft">[<a href="#ref-GhJYjnft" role="doc-biblioref">144</a>]</span></p>
<h3 data-number="10.79.1" id="keywords-78"><span class="header-section-number">10.79.1</span> Keywords</h3>
<ul>
<li>monoclonal antibodies</li>
<li>neutralization</li>
<li>antibody cross-reactivity</li>
<li>Receptor Binding Domain</li>
</ul>
<h3 data-number="10.79.2" id="main-findings-75"><span class="header-section-number">10.79.2</span> Main Findings</h3>
<p>In this study the authors report the affinity, cross reactivity (with
SARS-CoV and MERS-CoV virus) and viral neutralization capacity of 206
monoclonal antibodies engineered from isolated IgG memory B cells of
patients suffering from SARS-CoV-2 infection in Wuhan, China. All
patients but one recovered from disease. Interestingly, the patient that
did not recover had less SARS-CoV-2 specific B cells circulating
compared to other patients.</p>
<p>Plasma from all patients reacted to trimeric Spike proteins from
SARS-CoV-2, SARS-CoV and MERS-CoV but no HIV BG505 trimer. Furthermore,
plasma from patients recognized the receptor binding domain (RBD) from
SARS-CoV-2 but had little to no cross-reactivity against the RBD of
related viruses SARS-CoV and MERS-CoV, suggesting significant
differences between the RBDs of the different viruses. Negligible levels
of cross-neutralization using pseudoviruses bearing Spike proteins of
SARS-CoV-2, SARS-CoV or MERS-CoV, were observed, corroborating the ELISA
cross-reactivity assays on the RBDs.</p>
<p>SARS-CoV-2 RBD specific B cells constituted 0.005-0.065% of the total B
cell population and 0.023-0.329% of the memory subpopulation. SARS-CoV
specific IgG memory B cells were single cell sorted to sequence the
antibody genes that were subsequently expressed as recombinant IgG1
antibodies. From this library, 206 antibodies with different binding
capacities were obtained. No discernible patterns of VH usage were found
in the 206 antibodies suggesting immunologically distinct responses to
the infection. Nevertheless, most high-binding antibodies were derived
by clonal expansion. Further analyses in one of the patient derived
clones, showed that the antibodies from three different timepoints did
not group together in phylogenetic analysis, suggesting selection during
early infection.</p>
<p>Using surface plasmon resonance (SPR) 13 antibodies were found to have
10<sup>-8</sup> tp 10<sup>-9</sup> dissociation constants (Kd). Of the 13 antibodies, two
showed 98-99% blocking of SARS-CoV-2 RBD-ACE2 receptor binding in
competition assays. Thus, low Kd values alone did not predict ACE2
competing capacities. Consistent with competition assays the two
antibodies that show high ACE2 blocking (P2C-2F6 and P2C-1F11) were the
most capable of neutralizing pseudoviruses bearing SARS-CoV-2 spike
protein (IC<sub>50</sub> of 0.06 and 0.03 µg/mL, respectively). Finally, using
SPR the neutralizing antibodies were found to recognize both overlapping
and distinct epitopes of the RBD of SARS-CoV-2.</p>
<h3 data-number="10.79.3" id="limitations-76"><span class="header-section-number">10.79.3</span> Limitations</h3>
<ol type="1">
<li><p>Relatively low number of patients</p>
<ol type="a">
<li>No significant conclusion can be drawn about the possible
&gt; correlation between humoral response and disease severity</li>
</ol></li>
<li><p><em>In vitro</em> Cytopathic Effect Assay (CPE) for neutralization activity</p>
<ol type="a">
<li><p>Huh7 cells were used in neutralization assays with
&gt; pseudoviruses, and they may not entirely mimic what happens in
&gt; the upper respiratory tract</p></li>
<li><p>CPE assay is not quantitative</p></li>
</ol></li>
<li><p>Duplicated panel in Figure 4C reported (has been fixed in version 2)</p></li>
</ol>
<h3 data-number="10.79.4" id="significance-72"><span class="header-section-number">10.79.4</span> Significance</h3>
<p>This paper offers an explanation as to why
previously isolated antibodies against SARS-CoV do not effectively block
SARS-CoV-2. Also, it offers important insight into the development of
humoral responses at various time points during the first weeks of the
disease in small but clinically diverse group of patients. Furthermore,
it provides valuable information and well characterized antibody
candidates for the development of a recombinant antibody treatment for
SARS-CoV-2. Nevertheless, it also shows that plasmapheresis might have
variability in its effectiveness, depending on the donor’s antibody
repertoire at the time of donation.</p>
<h3 data-number="10.79.5" id="credit-76"><span class="header-section-number">10.79.5</span> Credit</h3>
<p><em>Review by Jovani Catalan-Dibene as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.80" id="characterisation-of-the-transcriptome-and-proteome-of-sars-cov-2-using-direct-rna-sequencing-and-tandem-mass-spectrometry-reveals-evidence-for-a-cell-passage-induced-in-frame-deletion-in-the-spike-glycoprotein-that-removes-the-furin-like-cleavage-site."><span class="header-section-number">10.80</span> Characterisation of the transcriptome and proteome of SARS-CoV-2 using direct RNA sequencing and tandem mass spectrometry reveals evidence for a cell passage induced in-frame deletion in the spike glycoprotein that removes the furin-like cleavage site.</h2>
<p>Davidson et al. <span class="citation" data-cites="bCYYaFKM">[<a href="#ref-bCYYaFKM" role="doc-biblioref">860</a>]</span></p>
<h3 data-number="10.80.1" id="keywords-79"><span class="header-section-number">10.80.1</span> Keywords</h3>
<ul>
<li>Transcription</li>
<li>RNA-seq</li>
<li>proteomics</li>
<li>mass spec</li>
<li>furin cleavage site</li>
<li>mutation</li>
<li>pathogenicity</li>
</ul>
<h3 data-number="10.80.2" id="main-findings-76"><span class="header-section-number">10.80.2</span> Main Findings</h3>
<p>The authors performed long read RNA sequencing using an Oxford Nanopore
MinION as well as tandem mass spec (MS) on Vero cells (a cell line
derived from kidney cells of the African green monkey that is deficient
in interferon) infected with SARS-CoV-2.</p>
<p>The authors found that passage of the virus in Vero cells gave rise to a
spontaneous 9 amino acid deletion (679-NSPRRARSV-687 to I) in the spike
(S) protein. The deleted sequence overlaps a predicted furin cleavage
site at the S1 / S2 domain boundary that is present in SARS-CoV-2 but
not SARS-CoV or the closely related bat coronavirus RaTG13, which are
cleaved at S1 / S2 by other proteases <span class="citation" data-cites="CqQVkaqj">[<a href="#ref-CqQVkaqj" role="doc-biblioref">49</a>]</span>. Furin cleavage sites at
similar positions in other viruses have been linked to increased
pathogenicity and greater cell tropism <span class="citation" data-cites="16tHVjVzu">[<a href="#ref-16tHVjVzu" role="doc-biblioref">861</a>]</span>. Loss of this site in
SARS-CoV-2 has also already been shown to increase viral entry into Vero
but not BHK cells (which are also interferon deficient) <span class="citation" data-cites="qcVbT0w4">[<a href="#ref-qcVbT0w4" role="doc-biblioref">62</a>]</span>. The
authors therefore make an important contribution in demonstrating that
passage in Vero cells may lead to spontaneous loss of a key
pathogenicity-conferring element in SARS-CoV-2.</p>
<h3 data-number="10.80.3" id="limitations-77"><span class="header-section-number">10.80.3</span> Limitations</h3>
<p>As the authors note, a similar study posted earlier by Kim et al., which
also passaged SARS-CoV-2 in Vero cells, did not identify any loss in the
S protein furin cleavage site <span class="citation" data-cites="hRVfFbS">[<a href="#ref-hRVfFbS" role="doc-biblioref">862</a>]</span>. It therefore remains to be
determined how likely it is that this mutation spontaneously arises. A
quantitative investigation using multiple experimental replicas to
understand the spontaneous viral mutation rate at this site and
elsewhere would be informative. Also, the mechanistic basis for the
higher viral fitness conferred by loss of the furin cleavage site in
Vero cells – but, evidently, not in vivo in humans, as this site is
maintained in all currently sequenced circulating isolates - remains to
be understood.</p>
<p>Due to the high base-call error rate of MinION sequencing, the authors’
bioinformatic pipeline required aligning transcripts to a reference to
correct sequencing artifacts. This presumably made it difficult or
impossible to identify other kinds of mutations, such as single
nucleotide substitutions, which may occur even more frequently than the
deletions identified in this work. Pairing long read sequencing with
higher-accuracy short-read sequencing may be one approach to overcome
this issue.</p>
<h3 data-number="10.80.4" id="significance-73"><span class="header-section-number">10.80.4</span> Significance</h3>
<p>As the authors suggest, animal studies using live virus challenge may
need to periodically verify the genomic integrity of the virus, or
potentially risk unknowingly using a likely less-pathogenic variant of
the virus.</p>
<p>More broadly, the results emphasize the complexity and plasticity of the
SARS-CoV-2 viral transcriptome and proteome. For example, the authors
found multiple versions of transcripts encoding the nucleocapsid (N)
protein, each with different small internal deletions, some of which
were verified for translation by MS. A number of peptides arising from
translation of unexpected rearrangements of transcripts were also
detected. Additionally, the authors identified phosphorylation of a
number of viral proteins (N, M, ORF 3a, nsp3, nsp9, nsp12 and S). For
any cases where these have functional consequences, targeting the
kinases responsible could be an avenue for drug development.
Understanding the functional consequences of the mutations, transcript
variations, and post translational modifications identified in this
study will be important future work.</p>
<h3 data-number="10.80.5" id="credit-77"><span class="header-section-number">10.80.5</span> Credit</h3>
<p><em>This review was undertaken by Tim O’Donnell, Maria Kuksin as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.81" id="a-sars-cov-2-human-protein-protein-interaction-map-reveals-drug-targets-and-potential-drug--repurposing"><span class="header-section-number">10.81</span> A SARS-CoV-2-Human Protein-Protein Interaction Map Reveals Drug Targets and Potential Drug- Repurposing</h2>
<p>Gordon et al. <em>bioRxiv</em> <span class="citation" data-cites="phJM8g2Y">[<a href="#ref-phJM8g2Y" role="doc-biblioref">147</a>]</span></p>
<h3 data-number="10.81.1" id="keywords-80"><span class="header-section-number">10.81.1</span> Keywords</h3>
<ul>
<li>protein-protein interactions</li>
<li>mass spectrometry</li>
<li>drug targets</li>
</ul>
<h3 data-number="10.81.2" id="main-findings-77"><span class="header-section-number">10.81.2</span> Main Findings</h3>
<p>Gordon et al cloned, tagged and expressed 26 of the 29
SARS-CoV-2 proteins individually in HEK293T cells and used mass
spectrometry to identify protein-protein interactions. They identified
332 viral-host protein-protein interactions. Furthermore, they used
these interactions to identify 66 existing drugs known to target host
proteins or host pathways (eg SARS-CoV-2 N and Orf8 proteins interact
with proteins regulated by the mTOR pathway, so mTOR inhibitors
Silmitasertib and Rapamycin are possible drug candidates).</p>
<h3 data-number="10.81.3" id="limitations-78"><span class="header-section-number">10.81.3</span> Limitations</h3>
<p>The main limitation of the study stems from
the reductionist model: overexpression of plasmids encoding individual
viral proteins in HEK293T cells. This precludes any interactions between
the viral proteins, or the combined effects of multiple proteins on the
host, as they are expressed individually. Moreover, HEK293T cells come
from primary embryonic kidney and therefore might not reflect how
SARS-CoV-2 interacts with its primary target, the lung. However, the
authors found that the proteins found to interact with viral proteins in
their experiments are enriched in lung tissue compared to HEK293Ts.</p>
<h3 data-number="10.81.4" id="significance-74"><span class="header-section-number">10.81.4</span> Significance</h3>
<p>The authors provide a “SARS-CoV-2 interaction map,”
which may provide potential hypotheses as to how the virus interacts
with the host. Further, they identified existing drugs that could
disrupt these host-viral interactions and curb SARS-CoV-2 infection.
Although these interactions have not been validated, this paper acts as
a valuable resource.</p>
<h3 data-number="10.81.5" id="credit-78"><span class="header-section-number">10.81.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.82" id="first-clinical-study-using-hcv-protease-inhibitor-danoprevir-to-treat-naïve-and-experienced-covid-19-patients"><span class="header-section-number">10.82</span> First Clinical Study Using HCV Protease Inhibitor Danoprevir to Treat Naïve and Experienced COVID-19 Patients</h2>
<p>Chen et al. <em>medRxiv</em>.
<span class="citation" data-cites="RTIk8DTo">[<a href="#ref-RTIk8DTo" role="doc-biblioref">863</a>]</span></p>
<h3 data-number="10.82.1" id="keywords-81"><span class="header-section-number">10.82.1</span> Keywords</h3>
<ul>
<li>Clinical study</li>
<li>HCV protease inhibitor</li>
<li>Danoprevir</li>
<li>Ritonavir</li>
<li>Covid19 treatment</li>
</ul>
<h3 data-number="10.82.2" id="main-findings-78"><span class="header-section-number">10.82.2</span> Main Findings</h3>
<p>The authors treated 11 Covid-19 patients with Danoprevir, a
commercialized HCV protease inhibitor <span class="citation" data-cites="YoCPtq6P">[<a href="#ref-YoCPtq6P" role="doc-biblioref">864</a>]</span><sup>(p4)</sup>, boosted by ritonavir <span class="citation" data-cites="10GnhBctk">[<a href="#ref-10GnhBctk" role="doc-biblioref">865</a>]</span>, a
CYP3A4 inhibitor (which enhances the plasma concentration and
bioavailabilty of Danoprevir). Two patients had never received
anti-viral therapy before (=naïve), whereas nine patients were on
Lopinavir/Ritonavir treatment before switching to Danoprevir/Ritonavir
(=experienced). The age ranged from 18 to 66yo.</p>
<p>Naïve patients that received Danoprevir/Ritonavir treatment had a
decreased hospitalization time. Patients treated with
Lopinavir/Ritonavir did not have a negative PCR test, while after
switching to Danoprevir/Ritonavir treatment, the first negative PCR test
occurred at a median of two days.</p>
<h3 data-number="10.82.3" id="limitations-79"><span class="header-section-number">10.82.3</span> Limitations</h3>
<p>The results of the study are very hard to interpret as there is no
control group not receiving Danoprevir/Ritonavir treatment. This was
especially true in naïve patients who seemed to have more mild symptoms
before the start of the study and were younger (18 and 44yo) compared to
the experienced patients (18 to 66yo). The possibility that the patients
would have recovered without Danoprevir/Ritonavir treatment cannot be
excluded.</p>
<h3 data-number="10.82.4" id="significance-75"><span class="header-section-number">10.82.4</span> Significance</h3>
<p>The authors of the study treated patients with Danoprevir, with the
rational to that this is an approved and well tolerated drug for HCV
patients <span class="citation" data-cites="10GnhBctk">[<a href="#ref-10GnhBctk" role="doc-biblioref">865</a>]</span>, and that it could also target the protease from SARS-CoV-2
(essential for viral replication and transcription). Indeed, homology
modelling data indicated that HCV protease inhibitors have the highest
binding affinity to Sars-Cov2 protease among other approved drugs <span class="citation" data-cites="7zBwK1wn">[<a href="#ref-7zBwK1wn" role="doc-biblioref">866</a>]</span>.</p>
<p>While this study shows that the combination of Danoprevir and Ritonavir
might be beneficial for Covid-19 patients, additional clinical trials
with more patients and with better methodology (randomization and
control group) are needed to make further conclusions.</p>
<h3 data-number="10.82.5" id="credit-79"><span class="header-section-number">10.82.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.83" id="efficacy-of-hydroxychloroquine-in-patients-with-covid-19-results-of-a-randomized-clinical-trial"><span class="header-section-number">10.83</span> Efficacy of hydroxychloroquine in patients with COVID-19: results of a randomized clinical trial</h2>
<p><span class="citation" data-cites="wzHb7Mj5">[<a href="#ref-wzHb7Mj5" role="doc-biblioref">402</a>]</span></p>
<h3 data-number="10.83.1" id="keywords-82"><span class="header-section-number">10.83.1</span> Keywords</h3>
<ul>
<li>hydroxychloroquine</li>
</ul>
<h3 data-number="10.83.2" id="study-description-1"><span class="header-section-number">10.83.2</span> Study Description</h3>
<p>This is a randomized clinical trial of hydroxychloroquine (HCQ) efficacy
in the treatment of COVID-19. From February 4 – February 28, 2020 142
COVID-19 positive patients were admitted to Renmin Hospital of Wuhan
University. 62 patients met inclusion criteria and were enrolled in a
double blind, randomized control trial, with 31 patients in each arm.</p>
<p>Inclusion criteria:</p>
<ol type="1">
<li><p>Age ≥ 18 years</p></li>
<li><p>Positive diagnosis COVID-19 by detection of SARS-CoV-2 by RT-PCR</p></li>
<li><p>Diagnosis of pneumonia on chest CT</p></li>
<li><p>Mild respiratory illness, defined by SaO<sub>2</sub>/SPO<sub>2</sub> ratio &gt; 93% or
PaO<sub>2</sub>/FIO<sub>2</sub> ratio &gt; 300 mmHg in hospital room conditions (Note:
relevant clinical references described below.)</p>
<ol type="a">
<li><p>Hypoxia is defined as an SpO2 of 85-94%; severe hypoxia &lt;
85%.</p></li>
<li><p>The PaO<sub>2</sub>/FIO<sub>2</sub> (ratio of arterial oxygen tension to fraction
of inspired oxygen) is used to classify the severity of acute
respiratory distress syndrome (ARDS). Mild ARDS has a
PaO<sub>2</sub>/FIO<sub>2</sub> of 200-300 mmHg, moderate is 100-200, and severe
&lt; 100.</p></li>
</ol></li>
<li><p>Willing to receive a random assignment to any designated treatment
group; not participating in another study at the same time</p></li>
</ol>
<p>Exclusion criteria:</p>
<ol type="1">
<li><p>Severe or critical respiratory illness (not explicitly defined,
presumed to be respiratory function worse than outlined in inclusion
criteria); or participation in trial does not meet patient’s maximum
benefit or safe follow up criteria</p></li>
<li><p>Retinopathy or other retinal diseases</p></li>
<li><p>Conduction block or other arrhythmias</p></li>
<li><p>Severe liver disease, defined by Child-Pugh score ≥ C or AST &gt;
twice the upper limit</p></li>
<li><p>Pregnant or breastfeeding</p></li>
<li><p>Severe renal failure, defined by eGFR ≤ 30 mL/min/1.73m<sup>2</sup>, or on
dialysis</p></li>
<li><p>Potential transfer to another hospital within 72h of enrollment</p></li>
<li><p>Received any trial treatment for COVID-19 within 30 days before the
current study</p></li>
</ol>
<p>All patients received the standard of care: oxygen therapy, antiviral
agents, antibacterial agents, and immunoglobulin, with or without
corticosteroids. Patients in the HCQ treatment group received additional
oral HCQ 400 mg/day, given as 200 mg 2x/day. HCQ was administered from
days 1-5 of the trial. The primary endpoint was 5 days post enrollment
or a severe adverse reaction to HCQ. The primary outcome evaluated was
time to clinical recovery (TTCR), defined as return to normal body
temperature and cough cessation for &gt; 72h. Chest CT were imaged on
days 0 and 6 of the trial for both groups; body temperature and patient
reports of cough were collected 3x/day from day 0 – 6. The mean age and
sex distribution between the HCQ and control arms were comparable.</p>
<h3 data-number="10.83.3" id="main-findings-79"><span class="header-section-number">10.83.3</span> Main Findings</h3>
<p>There were 2 patients showing mild secondary effects of HCQ treatment.
More importantly, while 4 patients in the control group progressed to
severe disease, none progressed in the treatment group.</p>
<p>TTCR was significantly decreased in the HCQ treatment arm; recovery from
fever was shortened by one day (3.2 days control vs. 2.2 days HCQ, p =
0.0008); time to cessation of cough was similarly reduced (3.1 days
control vs. 2.0 days HCQ, p = 0.0016).</p>
<p>Overall, it appears that HCQ treatment of patients with mild COVID-19
has a modest effect on clinical recovery (symptom relief on average 1
day earlier) but may be more potent in reducing the progression from
mild to severe disease.</p>
<h3 data-number="10.83.4" id="limitations-80"><span class="header-section-number">10.83.4</span> Limitations</h3>
<p>This study is limited in its inclusion of only patients with mild
disease, and exclusion of those on any treatment other than the standard
of care. It would also have been important to include the laboratory
values of positive RT-PCR detection of SARS-CoV-2 to compare the
baseline and evolution of the patients’ viral load.</p>
<h3 data-number="10.83.5" id="limitations-81"><span class="header-section-number">10.83.5</span> Limitations</h3>
<p>Despite its limitations, the study design has good rigor as a double
blind RCT and consistent symptom checks on each day of the trail. Now
that the FDA has approved HCQ for treatment of COVID-19 in the USA, this
study supports the efficacy of HCQ use early in treatment of patients
showing mild symptoms, to improve time to clinical recovery, and
possibly reduce disease progression. However, most of the current
applications of HCQ have been in patients with severe disease and for
compassionate use, which are out of the scope of the findings presented
in this trial. Several additional clinical trials to examine
<a href="https://twitter.com/hashtag/hydroxychloroquine?src=hashtag_click">hydroxychloroquine</a>
are now undergoing; their results will be critical to further validate
these findings.</p>
<h3 data-number="10.83.6" id="credit-80"><span class="header-section-number">10.83.6</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<p><strong>Structure-based modeling of SARS-CoV-2 peptide/HLA-A02 antigens</strong></p>
<p><a href="https://doi.org/10.1101/2020.03.23.004176" class="uri">https://doi.org/10.1101/2020.03.23.004176</a></p>
<p><em>Immunology keywords</em>:</p>
<p>CoVID-19, 2019-nCoV, SARS-CoV-2, comparative, homology, peptide,
modeling, simulation, HLA-A, antigen</p>
<p><em>Summary of Findings</em>:</p>
<ul>
<li><p>The authors utilize homology modeling to identify peptides from the
SARS-CoV-2 proteome that potentially bind HLA-A*02:01.</p></li>
<li><p>They utilize high-resolution X-ray structures of peptide/MHC
complexes on Protein Data Bank, substitute homologous peptides with
SARS-CoV-2 peptides, and calculate MHC/SARS-CoV-2 peptide Rosetta
binding energy.</p></li>
<li><p>They select MHC/SARS-CoV-2 complex models with highest binding
energy for further study and publish models in an online database
(https://rosettamhc.chemistry.ucsc.edu).</p></li>
</ul>
<p><em>Limitations</em>:</p>
<ul>
<li><p>The authors only utilize computational methods and predicted
SARS-CoV-2 peptides must be validated experimentally for
immunogenicity and clinical response.</p></li>
<li><p>Due to computational burden and limited availability of high
resolution X-ray structures on PDB, authors only simulate 9-mer and
10-mer peptide binding to HLA-A*02:01.</p></li>
<li><p>Since the authors compare select existing X-ray structures as a
starting point, backbone conformations that deviate significantly
between test and template peptides are not captured. Furthermore,
Rosetta modeling protocols do not capture all possible structures
and binding energy scoring does not fully recapitulate fundamental
forces.<sup>1,2</sup></p></li>
</ul>
<p><em>Importance/Relevance:</em></p>
<ul>
<li><p>The authors identify and publish high-scoring SARS-CoV-2 peptides
that may direct a targeted, experimental validation approach toward
a COVID-19 vaccine.</p></li>
<li><p>The authors utilize Rosetta simulation to further filter results
from NetMHCpan 4.0, supporting machine learning prediction with
structural analysis.</p></li>
<li><p>The authors develop RosettaMHC, a computationally efficient method
of leveraging existing X-ray structures for identification of
immunogenic peptides.</p></li>
</ul>
<p><em>References:</em></p>
<ol type="1">
<li><p>Bender, B. J., Cisneros, A., 3rd, Duran, A. M., Finn, J. A., Fu, D.,
Lokits, A. D., . . . Moretti, R. (2016). Protocols for Molecular
Modeling with Rosetta3 and RosettaScripts. Biochemistry, 55(34),
4748-4763. doi:10.1021/acs.biochem.6b00444</p></li>
<li><p>Alford, R. F., Leaver-Fay, A., Jeliazkov, J. R., O’Meara, M. J.,
DiMaio, F. P., Park, H., . . . Gray, J. J. (2017). The Rosetta
All-Atom Energy Function for Macromolecular Modeling and Design. J
Chem Theory Comput, 13(6), 3031-3048. doi:10.1021/acs.jctc.7b00125</p></li>
</ol>
<p><em>Review by Jonathan Chung as part of a project by students, postdocs and
faculty at the Immunology Institute of the Icahn school of medicine,
Mount Sinai.</em></p>
<h2 data-number="10.84" id="serology-characteristics-of-sars-cov-2-infection-since-the-exposure-and-post-symptoms-onset"><span class="header-section-number">10.84</span> Serology characteristics of SARS-CoV-2 infection since the exposure and post symptoms onset</h2>
<p>Lou et al. <em>medRxiv</em>. <span class="citation" data-cites="lnuNSHEW">[<a href="#ref-lnuNSHEW" role="doc-biblioref">867</a>]</span></p>
<h3 data-number="10.84.1" id="keywords-83"><span class="header-section-number">10.84.1</span> Keywords</h3>
<ul>
<li>Seroconversion rate</li>
<li>Total Antibody</li>
<li>Ab</li>
<li>IgG and IgM</li>
<li>antibody</li>
</ul>
<h3 data-number="10.84.2" id="main-findings-80"><span class="header-section-number">10.84.2</span> Main Findings</h3>
<p>Currently, the diagnosis of SARS-CoV-2 infection
entirely depends on the detection of viral RNA using polymerase chain
reaction (PCR) assays. False negative results are common, particularly
when the samples are collected from upper respiratory. Serological
detection may be useful as an additional testing strategy. In this study
the authors reported that a typical acute antibody response was induced
during the SARS-CoV-2 infection, which was discuss earlier<sup>1</sup>. The
seroconversion rate for Ab, IgM and IgG in COVID-19 patients was 98.8%
(79/80), 93.8% (75/80) and 93.8% (75/80), respectively. The first
detectible serology marker was total antibody followed by IgM and IgG,
with a median seroconversion time of 15, 18 and 20 days-post exposure
(d.p.e) or 9, 10- and 12-days post-onset (d.p.o). Seroconversion was
first detected at day 7d.p.e in 98.9% of the patients. Interestingly
they found that viral load declined as antibody levels increased. This
was in contrast to a previous study <span class="citation" data-cites="5HXuCa5t">[<a href="#ref-5HXuCa5t" role="doc-biblioref">808</a>]</span>, showing that increased antibody
titers did not always correlate with RNA clearance (low number of
patient sample).</p>
<h3 data-number="10.84.3" id="limitations-82"><span class="header-section-number">10.84.3</span> Limitations</h3>
<p>Current knowledge of the antibody response to SAR-CoV-2
infection and its mechanism is not yet well elucidated. Similar to the
RNA test, the absence of antibody titers in the early stage of illness
could not exclude the possibility of infection. A diagnostic test, which
is the aim of the authors, would not be useful at the early time points
of infection but it could be used to screen asymptomatic patients or
patients with mild disease at later times after exposure.</p>
<h3 data-number="10.84.4" id="significance-76"><span class="header-section-number">10.84.4</span> Significance</h3>
<p>Understanding the antibody responses against SARS-CoV2 is
useful in the development of a serological test for the diagnosis of
COVID-19. This manuscript discussed acute antibody responses which can
be deducted in plasma for diagnostic as well as prognostic purposes.
Thus, patient-derived plasma with known antibody titers may be used
therapeutically for treating COVID-19 patients with severe illness.
<span id="OLE_LINK3" class="anchor"><span id="OLE_LINK4" class="anchor"></span></span></p>
<h3 data-number="10.84.5" id="credit-81"><span class="header-section-number">10.84.5</span> Credit</h3>
<p><em>This review was undertaken and edited by Konstantina A as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.85" id="sars-cov-2-launches-a-unique-transcriptional-signature-from-in-vitro-ex-vivo-and-in-vivo-systems"><span class="header-section-number">10.85</span> SARS-CoV-2 launches a unique transcriptional signature from in vitro, ex vivo, and in vivo systems</h2>
<p>Blanco-Melo et al. <em>bioRxiv</em>. <span class="citation" data-cites="aNSdARwe">[<a href="#ref-aNSdARwe" role="doc-biblioref">868</a>]</span></p>
<h3 data-number="10.85.1" id="keywords-84"><span class="header-section-number">10.85.1</span> Keywords</h3>
<ul>
<li>host cellular response</li>
<li>host-pathogen interaction</li>
<li>type I interferon</li>
<li>type III interferon</li>
<li>inflammation</li>
<li>RNA-seq</li>
<li>comparative analysis</li>
</ul>
<h3 data-number="10.85.2" id="main-findings-81"><span class="header-section-number">10.85.2</span> Main Findings</h3>
<p>Given the high mortality rate of SARS-CoV-2 relative to
other respiratory viruses such seasonal IAV and RSV, there may be
underlying host-pathogen interactions specific to SARS-CoV-2 that
predispose to a worse clinical outcome. Using <em>in vivo</em>, <em>ex vivo</em>, and
<em>in vitro</em> systems, the authors profiled host cell transcriptional
responses to SARS-CoV-2 and to other common respiratory viruses
(seasonal IAV and RSV). SARS-CoV-2 infection <em>in vitro</em> led to an
induction of type I interferon response signaling and the upregulation
of cytokine/chemokines transcripts. In comparison with IAV and RSV
infection, SARS-CoV-2 <em>in vitro</em> appears to uniquely induce less type I
and type III interferon expression and higher levels of two cytokines
previously implicated in respiratory inflammation. Lastly, <em>in vivo</em>
data from ferrets showed a reduced induction of cytokines and chemokines
by SARS-CoV-2 infection relative to IAV infection.</p>
<h3 data-number="10.85.3" id="limitations-83"><span class="header-section-number">10.85.3</span> Limitations</h3>
<p>While these results are promising, there are several
key weaknesses of this paper. 1) As the authors point out, there is an
undetectable level of SARS-CoV-2 putative receptor (ACE2) and protease
(TMPRSS2) expression in the lung epithelial cell line used for the <em>in
vitro</em> studies. This raises the important question of whether viral
replication actually occurs in any of the models used, which may explain
the lack of interferon production observed <em>in vitro</em> in SARS-CoV-2
treated cells. Further studies characterizing viral titers across
timepoints are needed. 2) Furthermore, these studies only characterize
the host response at a single dose and timepoint per virus, and it is
unclear why these doses/timepoints were chosen. This leaves open the
possibility that the observed differences between viruses could be due
to differences in dose, timing, host response, or a combination of all
of these. 3) It is unclear whether ferrets are productively infected,
which cell types are infected, and the extent/timing of the clinical
course of infection. Moreover, the <em>in vitro</em> and <em>in vivo</em> data do not
strongly correlate and the reasons for this are unclear.</p>
<h3 data-number="10.85.4" id="significance-77"><span class="header-section-number">10.85.4</span> Significance</h3>
<p>This paper describes potentially unique
transcriptional signatures of host cells exposed to SARS-CoV-2. If
validated, these findings may help explain clinical outcomes and could
be targeted in future therapeutic interventions.</p>
<h3 data-number="10.85.5" id="potential-conflicts-of-interest-disclosure"><span class="header-section-number">10.85.5</span> Potential Conflicts of Interest Disclosure</h3>
<p>The reviewers are also researchers at the Icahn School
of Medicine at Mount Sinai.</p>
<h3 data-number="10.85.6" id="credit-82"><span class="header-section-number">10.85.6</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.86" id="a-new-predictor-of-disease-severity-in-patients-with-covid-19-in-wuhan-china"><span class="header-section-number">10.86</span> A New Predictor of Disease Severity in Patients with COVID-19 in Wuhan, China</h2>
<p>Zhou et al. <em>bioRxiv</em>. <span class="citation" data-cites="EWZmuLvu">[<a href="#ref-EWZmuLvu" role="doc-biblioref">869</a>]</span></p>
<h3 data-number="10.86.1" id="keywords-85"><span class="header-section-number">10.86.1</span> Keywords</h3>
<ul>
<li>disease severity</li>
<li>clinical data</li>
<li>Neutrophils/lymphocytes ratio</li>
<li>CRP</li>
<li>D-dimer</li>
</ul>
<h3 data-number="10.86.2" id="main-findings-82"><span class="header-section-number">10.86.2</span> Main Findings</h3>
<p>377 hospitalized patients were divided into two groups: severe and
non-severe pneumonia. The laboratory results of their first day of
admission were retrospectively analyzed to identify predictors of
disease severity.</p>
<p>After adjusting for confounding factors from chronic comorbidities (such
as high blood pressure, type 2 diabetes, coronary heart disease, and
chronic obstructive pulmonary disease), the independent risk factors
identified for severe pneumonia were <strong>age</strong>, the <strong>ratio of
neutrophil/lymphocytes counts</strong>, <strong>CRP</strong> and <strong>D-dimer</strong> levels.</p>
<p>To further increase the specificity and sensibility of these markers,
they showed that their multiplication <strong>[(Neutrophil/lymphocyte count)
* CRP * D-dimer]</strong> was a better predictor of disease severity, with
higher sensitivity (95.7%) and specificity (63.3%), with a cutoff value
of 2.68.</p>
<h3 data-number="10.86.3" id="limitations-84"><span class="header-section-number">10.86.3</span> Limitations</h3>
<p>This study included 377 hospitalized patients. Among them, 45.6%
patients tested positive for SARS-Cov-2 nucleic acid test results, and
others were included in the study based on clinically diagnosis even if
the molecular diagnosis was negative. Thus, additional studies are
needed to verify this on a larger number of covid-19 certified patients
and the cutoff value might be adjusted. Also, all the patients that did
not have the clinical characteristics of severe pneumonia were included
in the non-severe pneumonia group, but usually patients are also divided
into moderate and mild disease.</p>
<p>Also, studying different subset of lymphocytes could lead to a more
specific predictor. Another study showed that the neutrophils to CD8+ T
cells ratio was a strong predictor of disease severity <span class="citation" data-cites="zTGzuiTI">[<a href="#ref-zTGzuiTI" role="doc-biblioref">778</a>]</span>. Another more
precise study showed that the percentage of helper T cells and
regulatory T cells decrease but the percentage of naïve helper T cells
increases in severe cases <span class="citation" data-cites="6Y81C4P3">[<a href="#ref-6Y81C4P3" role="doc-biblioref">771</a>]</span>. Taking these subpopulations into account
might make the predictor more powerful.</p>
<p>Other studies also noted an inverse correlation between disease severity
and LDH <span class="citation" data-cites="USOnp7c0">[<a href="#ref-USOnp7c0" role="doc-biblioref">812</a>]</span> or IL6 <span class="citation" data-cites="1GEbBtBgh">[<a href="#ref-1GEbBtBgh" role="doc-biblioref">821</a>]</span> levels, but the authors here do not discuss LDH nor
IL6 levels, although this could help to strengthen the predictor.</p>
<p>The study is based on the results obtained on the first day of
admission, studying the dynamic of the changes in patients might also be
interesting to better predict disease severity.</p>
<h3 data-number="10.86.4" id="significance-78"><span class="header-section-number">10.86.4</span> Significance</h3>
<p>This study confirms that the neutrophil to lymphocyte ratio can be a
predictor of disease severity as shown by many others <span class="citation" data-cites="6Y81C4P3 VRoblZFJ gK5e7Yhl">[<a href="#ref-gK5e7Yhl" role="doc-biblioref">770</a>,<a href="#ref-6Y81C4P3" role="doc-biblioref">771</a>,<a href="#ref-VRoblZFJ" role="doc-biblioref">785</a>]</span>. The
novelty here is that they show that a combination with other markers can
enhance the specificity and sensibility of the predictor, although the
study could be improved by taking into account sub-populations of
lymphocytes and more biological factors from patients such as LDH and
IL6.</p>
<h3 data-number="10.86.5" id="credit-83"><span class="header-section-number">10.86.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.87" id="metabolic-disturbances-and-inflammatory-dysfunction-predict-severity-of-coronavirus-disease-2019-covid-19-a-retrospective-study"><span class="header-section-number">10.87</span> Metabolic disturbances and inflammatory dysfunction predict severity of coronavirus disease 2019 (COVID-19): a retrospective study</h2>
<p>Shuke Nie et al. <em>medRxiv</em>. <span class="citation" data-cites="95sQSGzx">[<a href="#ref-95sQSGzx" role="doc-biblioref">870</a>]</span></p>
<h3 data-number="10.87.1" id="keywords-86"><span class="header-section-number">10.87.1</span> Keywords</h3>
<ul>
<li>metabolism</li>
<li>fasting blood glucose</li>
<li>serum total protein</li>
<li>albumin</li>
<li>blood lipid</li>
<li>HDL-C</li>
<li>APOA1</li>
<li>lymphocytopenia</li>
<li>IL-6</li>
<li>CRP</li>
<li>severity prediction of COVID19</li>
</ul>
<h3 data-number="10.87.2" id="main-findings-83"><span class="header-section-number">10.87.2</span> Main Findings</h3>
<p>Retrospective Study on 97 COVID-19 hospitalized patients
(25 severe and 72 non-severe) analyzing clinical and laboratory
parameter to predict transition from mild to severe disease based on
more accessible indicators (such as fasting blood glucose, serum protein
or blood lipid) than inflammatory indicators. In accordance with other
studies, age and hypertension were risk factors for disease severity,
and lymphopenia and increased IL-6 was observed in severe patients. The
authors show that fasting blood glucose (FBG) was altered and patients
with severe disease were often hyperglycemic. Data presented support
that hypoproteinaemia, hypoalbuminemia, and reduction in
high-densitylipoprotein (HDL-C) and ApoA1 were associated with disease
severity.</p>
<h3 data-number="10.87.3" id="limitations-85"><span class="header-section-number">10.87.3</span> Limitations</h3>
<p>In this study non-severe patients were divided in two groups based on
average course of the disease: mild group1 (14 days, n=28) and mild
group 2 (30 days, n=44). However mild patients with a longer disease
course did not show an intermediate phenotype (between mild patients
with shorter disease course and severe patients), hence it is unclear
whether this was a useful and how it impacted the analysis. Furthermore,
the non-exclusion of co-morbidity factors in the analysis may bias the
results (e.g. diabetic patients and glucose tests) It is not clear at
what point in time the laboratory parameters are sampled. In table 3, it
would have been interesting to explore a multivariate multiple
regression. The correlation lacks of positive control to assess the
specificity of the correlation to the disease vs. correlation in any
inflammatory case. The dynamic study assessing the predictability of the
laboratory parameter is limited to 2 patients. Hence there are several
associations with disease severity, but larger studies are necessary to
test the independent predictive value of these potential biomarkers.</p>
<h3 data-number="10.87.4" id="significance-79"><span class="header-section-number">10.87.4</span> Significance</h3>
<p>As hospital are getting overwhelmed a set of
easily accessible laboratory indicators (such as serum total protein)
would potentially provide a triage methodology between potentially
severe cases and mild ones. This paper also opens the question regarding
metabolic deregulation and COVID-19 severity.</p>
<h3 data-number="10.87.5" id="credit-84"><span class="header-section-number">10.87.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.88" id="viral-kinetics-and-antibody-responses-in-patients-with-covid-19"><span class="header-section-number">10.88</span> Viral Kinetics and Antibody Responses in Patients with COVID-19</h2>
<p><span class="citation" data-cites="btVuoX6V">[<a href="#ref-btVuoX6V" role="doc-biblioref">871</a>]</span></p>
<h3 data-number="10.88.1" id="keywords-87"><span class="header-section-number">10.88.1</span> Keywords</h3>
<ul>
<li>CoVID-19</li>
<li>2019-nCoV</li>
<li>SARS-CoV-2</li>
<li>IgG</li>
<li>IgM</li>
<li>clinical</li>
<li>kinetics</li>
<li>antibodies</li>
</ul>
<h3 data-number="10.88.2" id="main-findings-84"><span class="header-section-number">10.88.2</span> Main Findings</h3>
<ul>
<li><p>Prospective cohort of 67 patients, clinical specimens taken and
follow-up conducted.</p></li>
<li><p>Viral shedding, serum IgM, IgG antibody against NP evaluated and
correlated to disease severity and clinical outcome</p></li>
<li><p>Viral RNA levels peaked at 1 week from febrile/cough symptom onset
in sputum, nasal swabs, and stool samples. Shedding ranged from
12-19 days (median ranges) and was longer in severe patients.</p></li>
<li><p>IgM and IgG titers stratified patients into three archetypes as
‘strong vs weak vs non-responders’. Strong responders (with higher
IgM/IgG titers) were significantly higher in severe patients.</p></li>
</ul>
<h3 data-number="10.88.3" id="limitations-86"><span class="header-section-number">10.88.3</span> Limitations</h3>
<p>Specific for immune monitoring.</p>
<ul>
<li><p>Not clear if stool RNA captured from live infection in
intestine/liver or from swallowed sputum. Transmission electron
microscopy (TEM) carried out on sputum samples as proof of concept,
but not stools. TEM unreasonable for actual clinical diagnosis.</p></li>
<li><p>Several patients had co-morbidities (such as pulmonary and liver
disease) that were not accounted for when tracking antibody
responses. Viral kinetics and IgM/IgG titers in subsets of patients
with underlying conditions/undergoing certain medication would be
informative.</p></li>
</ul>
<h3 data-number="10.88.4" id="significance-80"><span class="header-section-number">10.88.4</span> Significance</h3>
<ul>
<li><p>Three archetypes of antibody response to SARS-CoV-2 with different
disease progression and kinetics is useful to stratify patients, and
for future serological tests.</p></li>
<li><p>Strong spike-IgG levels often correlate with lymphopenia and
CoVID-19 disease severity <span class="citation" data-cites="LIf5SSUs">[<a href="#ref-LIf5SSUs" role="doc-biblioref">872</a>]</span>, similar to macaque
studies in SARS <span class="citation" data-cites="tC5vJmwj">[<a href="#ref-tC5vJmwj" role="doc-biblioref">616</a>]</span>. It would be critical to see if anti-NP or
anti-Spike IgG antibodies for SARS-CoV-2 also elicit similar
detrimental effects before clinical use.</p></li>
</ul>
<h3 data-number="10.88.5" id="credit-85"><span class="header-section-number">10.88.5</span> Credit</h3>
<p><em>Review by Samarth Hegde as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.89" id="covid-19-infection-induces-readily-detectable-morphological-and-inflammation-related-phenotypic-changes-in-peripheral-blood-monocytes-the-severity-of-which-correlate-with-patient-outcome"><span class="header-section-number">10.89</span> COVID-19 infection induces readily detectable morphological and inflammation-related phenotypic changes in peripheral blood monocytes, the severity of which correlate with patient outcome</h2>
<p><span class="citation" data-cites="1FZgghazg">[<a href="#ref-1FZgghazg" role="doc-biblioref">873</a>]</span></p>
<h3 data-number="10.89.1" id="keywords-88"><span class="header-section-number">10.89.1</span> Keywords</h3>
<ul>
<li>Monocytes</li>
<li>FSC-high</li>
<li>PBMC</li>
<li>ACE2</li>
<li>inflammatory cytokines</li>
</ul>
<h3 data-number="10.89.2" id="main-findings-85"><span class="header-section-number">10.89.2</span> Main Findings</h3>
<p>This study is based on flow cytometry immunophenotyping of
PBMCs from 28 patients diagnosed positive for SARS-Cov2 (COVID19). The
authors identify a population of abnormally large (FSC-hi) monocytes,
present in COVID19 patients, but absent in PBMCs of healthy volunteers
(n=16) or patients with different infections (AIDS, malaria, TB). This
FSC-hi monocytic population contains classical, intermediate and
non-classical (monocytes (based on CD14 and CD16 expression) that
produce inflammatory cytokines (IL-6, TNF and IL-10). The authors
suggest an association of FSC-hi monocytes with poor outcome and
correlate a high percentage of FSC-low monocytes, or higher ratio of
FSC-low/hi monocytes, with faster hospital discharge.</p>
<h3 data-number="10.89.3" id="limitations-87"><span class="header-section-number">10.89.3</span> Limitations</h3>
<p>While identification of the monocytic population based
on FSC is rather robust, the characterization of these cells remains
weak. A comprehensive comparison of FSC-hi monocytes with FSC-low
monocytes from patients and healthy controls would be of high value. It
is unclear if percentages in blood are among CD45+ cells. Furthermore,
it would have been important to include absolute numbers of different
monocytic populations (in table 1 there are not enough samples and it is
unclear what the authors show).</p>
<p>The authors show expression of the ACE2 receptor on the surface of the
monocytes, and highlight these cells as potential targets of SARS-Cov2.
However, appropriate controls are needed. CD16 has high affinity to
rabbit IgG and it is unclear whether the authors considered unspecific
binding of rabbit anti-ACE2 to Fc receptors. Gene expression of ACE-2 on
monocytes needs to be assessed. Furthermore, it would be important to
confirm infection of monocytes by presence of viral proteins or viral
particles by microscopy.</p>
<p>Considering the predictive role of FSC-hi monocytes on the development
of the disease and its severity, some data expected at this level are
neither present nor addressed. Although the cohort is small, it does
include 3 ICU patients. What about their ratio of FSC-low vs FSC-hi
monocytes in comparison to other patients? Was this apparent early in
the disease course? Does this population of FSC-hi monocytes differ
between ICU patients and others in terms of frequency, phenotype or
cytokine secretion?</p>
<p>In general, figures need to revised to make the data clear. For example,
in Fig. 5, according to the legend it seems that patients with FSC-high
monocytes are discharged faster from the hospital. However according to
description in the text, patients were grouped in high or low levels of
FSC-low monocytes.</p>
<h3 data-number="10.89.4" id="significance-81"><span class="header-section-number">10.89.4</span> Significance</h3>
<p>Despite the limitations of this study, the discovery of
a FSC-high monocyte population in COVID-19 patients is of great
interest. With similar implication, a the recent study by Zhou et al. <span class="citation" data-cites="Bgo7Cmj3">[<a href="#ref-Bgo7Cmj3" role="doc-biblioref">776</a>]</span>
identified a connection between an inflammatory CD14+CD16+ monocyte
population and pulmonary immunopathology leading to deleterious clinical
manifestations and even acute mortality after SARS-CoV-2 infections.
Although the presence of these monocytes in the lungs has yet to be
demonstrated, such results support the importance of monocytes in the
critical inflammation observed in some COVID19 patients.</p>
<h3 data-number="10.89.5" id="credit-86"><span class="header-section-number">10.89.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.90" id="correlation-between-universal-bcg-vaccination-policy-and-reduced-morbidity-and-mortality-for-covid-19-an-epidemiological-study"><span class="header-section-number">10.90</span> Correlation between universal BCG vaccination policy and reduced morbidity and mortality for COVID-19: an epidemiological study</h2>
<p>Miller et al. <em>medRxiv</em>. <span class="citation" data-cites="15gdkEnfn">[<a href="#ref-15gdkEnfn" role="doc-biblioref">874</a>]</span></p>
<h3 data-number="10.90.1" id="keywords-89"><span class="header-section-number">10.90.1</span> Keywords</h3>
<ul>
<li>BCG vaccine</li>
<li>epidemiology</li>
<li>vaccination policy</li>
</ul>
<h3 data-number="10.90.2" id="main-findings-86"><span class="header-section-number">10.90.2</span> Main Findings</h3>
<p>The authors compared middle and high income countries that never had a
universal BCG vaccination policy (Italy, Lebanon, Nederland, Belgium)
and countries with a current policy (low income countries were excluded
from the analysis as their number of cases and deaths might be
underreported for the moment). <strong>Countries that never implement BCG
vaccination have a higher mortality rate than countries which have a BCG
vaccination policy</strong> (16.38 deaths per million people vs 0.78). Next,
<strong>the authors show that an earlier start of vaccination correlates with
a lower number of deaths per million inhabitants</strong>. They interpret this
as the vaccine protecting a larger fraction of elderly people, which are
usually more affected by COVID-19. Moreover, higher number of COVID-19
<strong>cases</strong> were presented in countries that never implemented a universal
BCG vaccination policy.</p>
<h3 data-number="10.90.3" id="limitations-88"><span class="header-section-number">10.90.3</span> Limitations</h3>
<p>While this study aims to test an intriguing hypothesis unfortunately,
the data is not sufficient at this time to accurately make any
determinations. Several caveats must be noted including: not all
countries are in the same stage of the pandemic, the number of
cases/deaths is still changing very rapidly in a lot of countries and
thus the association may only reflect exposure to the virus. This
analysis would need to be re-evaluated when all the countries are passed
the pandemic and more accurate numbers are available. Additionally, very
few middle and high-income countries ever implemented universal BCG
vaccination, which can be a source of bias (5 countries, vs 55 that have
a BCG vaccine policy). Effective screening and social isolation policies
also varied considerable across the countries tested and may reflect
another important confounder. The authors could consider analyzing the
Case Fatality Rate (CFR, % of patients with COVID-19 that die), to more
correct for exposure although testing availability will still bias this
result. Variability in mortality within countries or cities with
variable vaccination and similar exposure could also be appropriate
although confounders will still be present.</p>
<h3 data-number="10.90.4" id="significance-82"><span class="header-section-number">10.90.4</span> Significance</h3>
<p>BCG vaccine is a live attenuated strain derived from <em>Mycobacterium
bovis</em> and used for a vaccine for tuberculosis (TB). This vaccine has
been proven to be efficient in preventing childhood meningitis TB, but
doesn’t prevent adult TB as efficiently. For this reason, several
countries are now only recommending this vaccine for at-risk population
only.</p>
<p><strong>This study shows that there is a correlation between BCG vaccination
policy and reduced mortality for Covid-19</strong>. Indeed, BCG vaccine has
been shown to protect against several viruses and enhance innate
immunity <span class="citation" data-cites="16FOT89K5">[<a href="#ref-16FOT89K5" role="doc-biblioref">875</a>]</span>, which could explain why it could protect against SARS-CoV-2
infection, but the exact mechanism is still unknown. <strong>Moreover, the
efficiency of adult/older people vaccination and protection against
Covid-19 still needs to be assessed.</strong> Regarding this, Australian
researchers are starting a clinical trial of BCG vaccine for healthcare
workers <span class="citation" data-cites="MCuIRGFE">[<a href="#ref-MCuIRGFE" role="doc-biblioref">876</a>]</span>, to assess if it can protect them against Covid-19.</p>
<h3 data-number="10.90.5" id="credit-87"><span class="header-section-number">10.90.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.91" id="non-neural-expression-of-sars-cov-2-entry-genes-in-the-olfactory-epithelium"><span class="header-section-number">10.91</span> Non-neural expression of SARS-CoV-2 entry genes in the olfactory epithelium</h2>
<p><span class="citation" data-cites="12oDb7aki">[<a href="#ref-12oDb7aki" role="doc-biblioref">877</a>]</span></p>
<h3 data-number="10.91.1" id="keywords-90"><span class="header-section-number">10.91.1</span> Keywords</h3>
<ul>
<li>CoVID-19</li>
<li>2019-nCoV</li>
<li>SARS-CoV-2</li>
<li>anosmia</li>
<li>olfaction</li>
<li>scRNAseq</li>
</ul>
<h3 data-number="10.91.2" id="main-findings-87"><span class="header-section-number">10.91.2</span> Main Findings</h3>
<ul>
<li><p>Study analyzed bulk and scRNAseq data of olfactory cell types from
publicly-available mouse, nonhuman primate and human datasets.</p></li>
<li><p>show that <em>ACE2</em> and <em>TMPRSS2</em> (genes involved in SARS-CoV-2 entry)
are expressed in olfactory epithelial (OE) cells, basal stem cells
and respiratory epithelium (RE), but not sensory neurons.</p></li>
<li><p>Comparison of human RE and OE datasets (Deprez et al. 2019; Durante
et al. 2020) revealed that <em>ACE2</em> and <em>TMPRSS2</em> expression in OE
sustentacular cells was similar to expression in the remainder of
the non-nasal respiratory tract.</p></li>
</ul>
<h3 data-number="10.91.3" id="limitations-89"><span class="header-section-number">10.91.3</span> Limitations</h3>
<ul>
<li><p>Transcript data alone from healthy respiratory/olfactory cells is
not sufficient to confirm infectivity of nasal passage, or to
indicate damage to epithelia.</p></li>
<li><p>No mechanism defined for anosmia; it is not clear if epithelial
injury leads to reduced sensitivity or increased inflammation and
altered immune contexture drives neural/epithelial dysfunction. Will
be critical to test this in CoVID-19 patient samples or mouse
models.</p></li>
</ul>
<h3 data-number="10.91.4" id="significance-83"><span class="header-section-number">10.91.4</span> Significance</h3>
<ul>
<li><p>Study provides possible rationale for anosmia observed in several
CoVID-19 patients.</p></li>
<li><p>Raises possibility that nasal respiratory goblet, ciliated cells,
and olfactory epithelia may serve as a viral reservoir after initial
SARS-CoV-2 infection.</p></li>
<li><p>Human olfactory sensory neurons express several other molecules
important to CoV (not CoV-19) entry such as <em>FURIN, ST6GAL1,
ST3GAL4</em>; this suggests wider mechanism of neuronal infectivity in
other coronaviruses.</p></li>
</ul>
<h3 data-number="10.91.5" id="credit-88"><span class="header-section-number">10.91.5</span> Credit</h3>
<p><em>Review by Samarth Hegde as part of a project by students, postdocs and
faculty at the Immunology Institute of the Icahn School of Medicine,
Mount Sinai.</em></p>
<p><strong>Title: </strong></p>
<p><strong>SARS-CoV-2 proteome microarray for mapping COVID-19 antibody
interactions at amino acid resolution</strong></p>
<p><strong>Immunology keywords:</strong> SARS-CoV-2, COVID-19, high throughput, peptide
microarray, antibody epitope screening</p>
<p><strong>The main finding of the article: </strong></p>
<p>This study screened the viral protein epitopes recognized by antibodies
in the serum of 10 COVID-19 patients using a new SARS-CoV-2 proteome
peptide microarray. The peptide library was constructed with 966 linear
peptides, each 15 amino acids long with a 5 amino acid overlap, based on
the protein sequences encoded by the genome of the Wuhan-Hu-1 strain.</p>
<p>To investigate crossreactivity between SARS-CoV-1 and SARS-CoV-2, they
tested rabbit monoclonal and polyclonal antibodies against SARS-CoV-1
nucleocapsid (N) in the microarray. Antibodies against SARS-CoV-1 N
displayed binding to the SARS-CoV-2 nucleocapsid (N) peptides.
Polyclonal antibodies showed some crossreactivity to other epitopes from
membrane (M), spike (S), ORF1ab and ORF8. This suggests that previous
exposure to SARS-CoV-1 may induced antibodies recognizing both viruses.</p>
<p>Screening of IgM and IgG antibodies from 10 COVID-19 patients showed
that many antibodies targeted peptides on M, N, S, Orf1ab, Orf3a, Orf7a,
and Orf8 from SARS-CoV-2, while immunodominant epitopes with antibodies
in more than 80 % COVID-19 patients were present in N, S and Orf3. It is
shown that the receptor binding domain (RBD) resides on S protein and
RBD is important for SARS-CoV-2 to enter the host cells via ACE2. Among
six epitopes on S protein, structural analysis predicted that three
epitopes were located at the surface and three epitopes were located
inside of the protein. Furthermore, some IgM antibodies from 1 patient
and IgG antibodies from 2 patients bound to the same epitope (residue
456-460, FRKSN) which resided within the RBD, and structural analysis
determined that this epitope was located in the region of the RBD loop
that engages with ACE2.</p>
<p><strong>Critical analysis of the study:</strong></p>
<p>In addition to the limitations mentioned in the manuscript, it would
have been informative to do the analysis over the course of the disease.
The pattern of antibody recognition, especially on S protein, and the
course of antibodies of different isotypes recognizing the same peptide
might correlate to the clinical course in these patients. It would alos
have been informative to analyze the presence of cross-reactive
antibodies from pateints previously exposed to SARS-CoV-1.</p>
<p><strong>The importance and implications for the current epidemics:</strong></p>
<p>This study identified linear immunodominant epitopes on SARS-CoV-2,
Wuhan-Hu-1 strain. This is a valuable information to design vaccines
that will elicit desirable immune responses.</p>
<p><strong>The Novel Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2)
Directly Decimates Human Spleens and Lymph Nodes</strong></p>
<p>Review by Matthew D. Park</p>
<p>Revised by Miriam Merad</p>
<p>Keywords: COVID-19, SARS-CoV-2, spleen, lymph node, ACE2, macrophage</p>
<p><em>Main findings</em></p>
<p>It has been previously reported that COVID-19 patients exhibit severe
lymphocytopenia, but the mechanism through which this depletion occurs
has not been described. In order to characterize the cause and process
of lymphocyte depletion in COVID-19 patients, the authors performed
gross anatomical and <em>in situ</em> immune-histochemical analyses of spleens
and lymph nodes (hilar and subscapular) obtained from post-mortem
autopsies of 6 patients with confirmed positive viremia and 3 healthy
controls (deceased due to vehicle accidents).</p>
<p>Primary gross observations noted significant splenic and LN atrophy,
hemorrhaging, and necrosis with congestion of interstitial blood vessels
and large accumulation of mononuclear cells and massive lymphocyte
death. They found that CD68<sup>+</sup> CD169<sup>+</sup> cells in the spleens, hilar and
subscapular LN, and capillaries of these secondary lymphoid organs
expressed the ACE2 receptor and stain positive for the SARS-CoV-2
nucleoprotein (NP) antigen, while CD3<sup>+</sup> T cells and B220<sup>+</sup> B cells
lacked both the ACE2 receptor and SARS-CoV-2 NP antigen. ACE2<sup>+</sup> NP<sup>+</sup>
CD169<sup>+</sup> macrophages were positioned in the splenic marginal zone (MZ)
and in the marginal sinuses of LN, which suggests that these macrophages
were positioned to encounter invading pathogens first and may contribute
to virus dissemination.</p>
<p>Since SARS-CoV-2 does not directly infect lymphocytes, the authors
hypothesized that the NP<sup>+</sup> CD169<sup>+</sup> macrophages are responsible for
persistent activation of lymphocytes via Fas::FasL interactions that
would mediate activation-induced cell death (AICD). Indeed, the
expression of Fas was significantly higher in virus-infected tissue than
that of healthy controls, and TUNEL staining showed significant
lymphocytic apoptosis. Since pro-inflammatory cytokines like IL-6 and
TNF-α can also engage cellular apoptosis and necrosis, the authors
interrogated the cytokine expression of the secondary lymphoid organs
from COVID-19 patients; IL-6, not TNF-α, was elevated in virus-infected
splenic and lymph node tissues, compared to those of healthy controls,
and immunofluorescent staining showed that IL-6 is primarily produced by
the infected macrophages. <em>In vitro</em> infection of THP1 cells with
SARS-CoV-2 spike protein resulted in selectively increased <em>Il6</em>
expression, as opposed to <em>Il1b</em> and <em>Tnfa</em> transcription. Collectively,
the authors concluded that a combination of Fas up-regulation and IL-6
production by NP<sup>+</sup> CD169<sup>+</sup> macrophages induce AICD in lymphocytes in
secondary lymphoid organs, resulting in lymphocytopenia.</p>
<p>In summary, this study reports that CD169<sup>+</sup> macrophages in the splenic
MZ, subscapular LN, and the lining capillaries of the secondary lymphoid
tissues express ACE2 and are susceptible to SARS-CoV-2 infection. The
findings point to the potential role of these macrophages in viral
dissemination, immunopathology of these secondary lymphoid organs,
hyperinflammation and lymphopenia.</p>
<p><em>Limitations</em></p>
<p><em>Technical</em></p>
<p>A notable technical limitation is the small number of samples (n=6);
moreover, the analysis of these samples using multiplexed
immunohistochemistry and immunofluorescence do not necessarily provide
the depth of unbiased interrogation needed to better identify the cell
types involved.</p>
<p><em>Biological</em></p>
<p>The available literature and ongoing unpublished studies, including
single-cell experiments of spleen and LN from organ donors, do not
indicate that ACE2 is expressed by macrophages; however, it remains
possible that ACE2 expression may be triggered by type I IFN in COVID-19
patients. Importantly, the SARS-CoV-2 NP staining of the macrophages
does not necessarily reflect direct infection of these macrophages;
instead, positive staining only indicates that these macrophages carry
SARS-CoV-2 NP as antigen cargo, which may have been phagocytosed. Direct
viral culture of macrophages isolated from the secondary lymphoid organs
with SARS-CoV-2 is required to confirm the potential for direct
infection of macrophages by SARS-CoV-2. Additionally, it is important to
note that the low to negligible viremia reported in COVID-19 patients
to-date does not favor a dissemination route via the blood, as suggested
by this study, which would be necessary to explain the presence of
virally infected cells in the spleen.</p>
<p><em>Relevance</em></p>
<p>Excess inflammation in response to SARS-CoV-2 infection is characterized
by cytokine storm in many COVID-19 patients. The contribution of this
pathology to the overall fatality rate due to COVID-19, not even
necessarily directly due to SARS-CoV-2 infection, is significant. A
better understanding of the full effect and source of some of these
major cytokines, like IL-6, as well as the deficient immune responses,
like lymphocytopenia, is urgently needed. In this study, the authors
report severe tissue damage in spleens and lymph nodes of COVID-19
patients and identify the role that CD169<sup>+</sup> macrophages may play in the
hyperinflammation and lymphocytopenia that are both characteristic of
the disease. It may, therefore, be important to note the effects that
IL-6 inhibitors like Tocilizumab and Sarilumab may specifically have on
splenic and LN function. It is important to note that similar
observations of severe splenic and LN necrosis and inflammation in
patients infected with SARS-CoV-1 further support the potential
importance and relevance of this study.</p>
<h2 data-number="10.92" id="cigarette-smoke-triggers-the-expansion-of-a-subpopulation-of-respiratory-epithelial-cells-that-express-the-sars-cov-2-receptor-ace2"><span class="header-section-number">10.92</span> Cigarette smoke triggers the expansion of a subpopulation of respiratory epithelial cells that express the SARS-CoV-2 receptor ACE2</h2>
<p><span class="citation" data-cites="t4vofknR">[<a href="#ref-t4vofknR" role="doc-biblioref">878</a>]</span></p>
<h3 data-number="10.92.1" id="keywords-91"><span class="header-section-number">10.92.1</span> Keywords</h3>
<ul>
<li>CoVID-19</li>
<li>2019-nCoV</li>
<li>SARS-CoV-2</li>
<li>respiration</li>
<li>cigarette</li>
<li>ACE2</li>
<li>lung</li>
</ul>
<h3 data-number="10.92.2" id="main-findings-88"><span class="header-section-number">10.92.2</span> Main Findings</h3>
<ul>
<li><p>Study uses scRNAseq, bulk seq data and air-liquid interface culture
experiments to show that cigarette smoke causes a dose-dependent
upregulation of ACE2 in mouse and human lungs (transplantation,
tumor resection, or IPF datasets).</p></li>
<li><p>ACE2 was not up-regulated in patients with asthma or
lung-sarcoidosis or in mouse models of cystic fibrosis or carcinogen
exposure.</p></li>
<li><p>Cathepsin B (alternate protease involved in viral entry) is
increased in smoke-exposed mouse or human lungs.</p></li>
<li><p>Smoke triggers a protective expansion of mucus-secreting MUC5AC+
goblet and SCGB1A1+ club cells; ACE2 presence in these cells is
increased upon smoke exposure.</p></li>
</ul>
<h3 data-number="10.92.3" id="limitations-90"><span class="header-section-number">10.92.3</span> Limitations:</h3>
<ul>
<li><p>Long-term smokers usually have several co-morbidities including
immune dysfunction, which can affect interpretation of CoV-2
susceptibility in these datasets. Ideally, analyses can control for
major co-morbidities across smokers and non-smokers (immune
suppression, cardiovascular disease and atherosclerosis).</p></li>
<li><p>Hyperplasia of ACE2+ goblet cells upon smoking needs to be separated
from ACE2 upregulation in existing goblet cells.</p></li>
<li><p>ACE2 expression increase alone does not confirm increased viral
entry into goblet cells; future studies with air-liquid interface
cultures testing CoV-2 infectivity in <em>ex vivo</em> epithelial cells
from human epithelial lines, ex vivo samples or hACE2 mice will be
very informative.</p></li>
</ul>
<h3 data-number="10.92.4" id="significance-84"><span class="header-section-number">10.92.4</span> Significance</h3>
<ul>
<li><p>This study may partially explain why smokers are more likely to
develop severe SARS-CoV-2 infections. Also, the reversibility of
ACE2 expression upon smoking cessation suggests that quitting
smoking could lessen CoV-2 susceptibility.</p></li>
<li><p>Absence of ACE2 upregulation in other lung inflammation pathologies
implies CoV-2 susceptibility might be smoking-specific, and not
fibrosis-specific.</p></li>
<li><p>Another preprint showed ACE2 expression increases in lung of
patients with CoV-2 co-morbidities such as hypertension <span class="citation" data-cites="wjtVy2v0">[<a href="#ref-wjtVy2v0" role="doc-biblioref">853</a>]</span>; these studies
collectively paint a better picture of CoV-2 susceptibility before
actual experiments can be carried out.</p></li>
</ul>
<h3 data-number="10.92.5" id="credit-89"><span class="header-section-number">10.92.5</span> Credit</h3>
<p><em>Review by Samarth Hegde as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="10.93" id="the-comparative-superiority-of-igm-igg-antibody-test-to-real-time-reverse-transcriptase-pcr-detection-for-sars-cov-2-infection-diagnosis"><span class="header-section-number">10.93</span> The comparative superiority of IgM-IgG antibody test to real-time reverse transcriptase PCR detection for SARS-CoV-2 infection diagnosis</h2>
<p>Liu et al. <em>medRxiv</em>.
<span class="citation" data-cites="k1E3DuFN">[<a href="#ref-k1E3DuFN" role="doc-biblioref">879</a>]</span></p>
<h3 data-number="10.93.1" id="keywords-92"><span class="header-section-number">10.93.1</span> Keywords</h3>
<ul>
<li>IgM/IgG antibody test</li>
<li>nucleic acid test</li>
<li>SARS-CoV-2 detection</li>
</ul>
<h3 data-number="10.93.2" id="main-findings-89"><span class="header-section-number">10.93.2</span> Main Findings</h3>
<p>The study compares IgM and IgG antibody testing to RT-PCR detection of
SARS-CoV-2 infection. 133 patients diagnosed with SARS-CoV-2 in Renmin
Hospital (Wuhan University, China) were analyzed. The positive ratio was
78.95% (105/133) in IgM antibody test (SARS-CoV-2 antibody detection kit
from YHLO Biotech) and 68.42% (91/133) in RT-PCR (SARS-CoV-2 ORF1ab/N
qPCR detection kit). There were no differences in the sensitivity of
SARS-CO-V2 diagnosis in patients grouped according to disease severity.
For example, IgG responses were detected in 93.18% of moderate cases,
100% of severe cases and 97.3% of critical cases. In sum, positive
ratios were higher in antibody testing compared to RT-PCR detection,
demonstrating a higher detection sensitivity of IgM-IgG testing for
patients hospitalized with COVID-19 symptoms.</p>
<h3 data-number="10.93.3" id="limitations-91"><span class="header-section-number">10.93.3</span> Limitations</h3>
<p>This analysis only included one-time point of 133 hospitalized patients,
and the time from symptom onset was not described. There was no
discussion about specificity of the tests and no healthy controls were
included. It would be important to perform similar studies with more
patients, including younger age groups and patients with mild symptoms
as well as asymptomatic individuals. It is critical to determine how
early after infection/symptom onset antibodies can be detected and the
duration of this immune response.</p>
<h3 data-number="10.93.4" id="significance-85"><span class="header-section-number">10.93.4</span> Significance</h3>
<p>The IgM-IgG combined testing is important to improve clinical
sensitivity and diagnose COVID-19 patients. The combined antibody test
shows higher sensitivity than individual IgM and IgG tests or nucleic
acid-based methods, at least in patients hospitalized with symptoms.</p>
<h3 data-number="10.93.5" id="credit-90"><span class="header-section-number">10.93.5</span> Credit</h3>
<p><em>Review by Erica Dalla as part of a project by students, postdocs and
faculty at the Immunology Institute of the Icahn School of Medicine,
Mount Sinai.</em></p>
<p><strong>Title:</strong> Lectin-like Intestinal Defensin Inhibits 2019-nCoV Spike
binding to ACE2</p>
<p><strong>Immunology keywords:</strong> defensins, spike protein, intestinal Paneth
cells, ACE2 binding</p>
<p><strong>Main Findings:</strong></p>
<p>Human ACE2 was previously identified as the host receptor for
SARS-CoV-2. Despite ACE2 being expressed in both lung alveolar
epithelial cells and small intestine enterocytes, respiratory problems
are the most common symptom after viral infection while intestinal
symptoms are much less frequent. Thus, the authors here investigate the
biology behind the observed protection of the intestinal epithelium from
SARS-CoV-2. Human defensin 5 (HD5), produced by Paneth cells in the
small intestine, was shown to interact with human ACE2, with a binding
affinity of 39.3 nM by biolayer interferometry (BLI). A blocking
experiment using different doses of HD5 coating ACE2 showed that HD5
lowered viral spike protein S1 binding to ACE2. Further, a molecular
dynamic simulation demonstrated a strong intermolecular interaction
between HD5 and the ACE2 ligand binding domain. To test HD5 inhibitory
effect on S1 binding to ACE2, human intestinal epithelium Caco-2 cells
were preincubated with HD5. Preincubation strongly reduced adherence of
S1 to surface of cells. HD5 was effective at a concentration as low as
10 µg/mL, comparable to the concentration found in the intestinal fluid.</p>
<p><strong>Limitations:</strong></p>
<p>The study focuses exclusively on intestinal cells. However, HD5 could
have been tested to block ACE2-S1 binding in human lung epithelial cells
as a potential treatment strategy. It would be useful to know whether
HD5 could also prevent viral entry in lung cells.</p>
<p><strong>Relevance:</strong></p>
<p>This work provides the first understanding of the different efficiency
of viral entry and infection among ACE2-expressing cells and tissues.
Specifically, the authors show that human defensin 5 produced in the
small intestine is able to block binding between S1 and ACE2 necessary
for viral entry into cells. The study provides a plausible explanation
on why few patients show intestinal symptoms and suggests that patients
with intestinal disease that decrease defensins’ production may be more
susceptible to SARS-CoV-2. It also indicates that HD5 could be used as a
molecule to be exogenously administered to patients to prevent viral
infection in lung epithelial cells.</p>
<p><strong>Title: </strong></p>
<p><strong>Susceptibility of ferrets, cats, dogs and different domestic animals
to SARS-coronavirus-2</strong></p>
<p><strong>Immunology keywords:</strong> SARS-CoV-2, ferret, cat, laboratory animal,
domestic animals</p>
<p><strong>The main finding of the article: </strong></p>
<p>This study evaluated the susceptibility of different model laboratory
animals (ferrets), as well as companion (cats and dogs), and domestic
animals (pigs, chickens and ducks) to SARS-CoV-2. They tested infection
with two SARS-CoV2 isolates, one from an environmental sample collected
in the Huanan Seafood Market in Wuhan (F13-E) and the other from a human
patient in Wuhan (CTan-H).</p>
<p><em>Ferrets</em> were inoculated with either of the two viruses by intranasal
route with 10<sup>5</sup> pfu, and the viral replication was evaluated. Two
ferrets from each group were euthanized on day 4 post infection (p.i.).
AT day 4 p.i., viral RNA and infectious viruses were detected only in
upper respiratory tract (nasal turbinate, upper palate, tonisls, but not
in the trachea, lungs or other tissues. Viral RNA and virus titer in the
remaining ferrets were monitored in nasal washes and rectal swabs on
days 2, 4, 6, 8 and 10 p.i. Viral RNA and infectious viruses were
detected in nasal washes until day 8 p.i. One ferret in each group
developed fever and loss of appetite on days 10 and 12 p.i., however,
viral RNA was practically undetactable. These two ferrets showed severe
lymphoplasmacytic perivasculitis and vasculitis in the lungs and lower
antibody titers compare to other 4 ferrets.</p>
<p><em>Cats</em>. Five <em>s</em>ubadult 8-month-old domestic cats were inoculated with
CTan-h virus and three uninfected cats were placed in a cage adjacent to
each of the infected cats to monitor respiratory droplet transmission.
Viral RNA was detected in the upper respiratory organs from all infected
cats and in one out of three exposed cats. All infected (inoculated and
exposed) cats developed elevated antibodies against SARS-CoV2. Viral
replication studies with juvenile cats (70-100 days) revealed massive
lesions in the nasal and tracheal mucosa epithelium and lungs of two
inoculated cats which died or were euthanized on day 3 p.i., and
infection in one out of three exposed cats. These results indicated
SARS-CoV2 could replicate in cats, that juvenile cats were more
susceptible that adults, and theat SARS-CoV2 could be transmit via
respiratory droplets between cats.</p>
<p><em>Dogs and others.</em> Five 3-month-old beagle dogs were inoculated and
housed with two uninoculated beagles in a room. Two virus inoculated
dogs seroconverted, but others including two contact dogs were all
seronegative for SARS-CoV2 and infectious virus was not detected in any
swabs collected. Viral RNA was not detected in swabs from pigs,
chickens, and ducks inoculated or contacted. These results indicated
that dogs, pigs, chickens, and ducks might have low or no susceptibility
to SARS-CoV2.</p>
<p><strong>Critical analysis of the study:</strong></p>
<p>This manuscript describes the viral replication and clinical symptoms of
SARS-CoV2 infection in ferrets, and the SARS-CoV2 infection and
transmission in cats. Clinical and pathological analysis was not
performed in cats, therefore the correlation of virus titer with
symptoms severity in the adult and juvenile cats could not be
determined.</p>
<p><strong>The importance and implications for the current epidemics:</strong></p>
<p>SARS-CoV-2 transmission to tigers, cats and dogs has been previously
reported. It should be noted that this manuscript did not evaluate the
transmission from cats to human. Nevertheless, it clearly showed higher
susceptibility of ferrets and domestic cats to SARS-CoV-2. This data
strongly indicates the need for surveillance of possible infection and
transmission of SARS-CoV-2 by domestic cats.</p>
<h2 data-number="10.94" id="virus-host-interactome-and-proteomic-survey-of-pmbcs-from-covid-19-patients-reveal-potential-virulence-factors-influencing-sars-cov-2-pathogenesis"><span class="header-section-number">10.94</span> Virus-host interactome and proteomic survey of PMBCs from COVID-19 patients reveal potential virulence factors influencing SARS-CoV-2 pathogenesis</h2>
<p>Li et al. <em>bioRxiv</em>.
<span class="citation" data-cites="NLoN4aYj">[<a href="#ref-NLoN4aYj" role="doc-biblioref">149</a>]</span></p>
<h3 data-number="10.94.1" id="keywords-93"><span class="header-section-number">10.94.1</span> Keywords</h3>
<ul>
<li>PBMC</li>
<li>virulence factors
– interaction network
– nsp9</li>
<li>nsp10
– NKRF</li>
</ul>
<h3 data-number="10.94.2" id="main-findings-90"><span class="header-section-number">10.94.2</span> Main findings</h3>
<p>The authors identified <strong>intra-viral protein-protein interactions</strong>
(PPI) with two different approaches: genome wide yeast-two hybrid (Y2H)
and co-immunoprecipitation (co-IP). A total of 58 distinct PPI were
characterized. A screen of <strong>viral-host PPI</strong> was also established by
overexpressing all the SARS-CoV-2 genes with a Flag epitope into HEK293
cells and purifying each protein complex. Interacting host proteins were
then identified by liquid chromatography and tandem mass spectrometry.
251 cellular proteins were identified, such as subunits of ATPase, 40S
ribosomal proteins, T complex proteins and proteasome related proteins,
for a total of 631 viral-host PPI. Several interactions suggesting
protein-mediated modulation of the immune response were identified,
highlighting the multiple ways SARS-CoV-2 might reprogram infected
cells.</p>
<p>Subsequently, the authors compared global proteome profiles of PBMCs
from healthy donors (n=6) with PBMC from COVID-19 patients with mild
(n=22) or severe (n=13) symptoms. 220 proteins were found to be
differentially expressed between <em>healthy donors and mild COVID-19
patients</em>, and a pathway analysis showed <strong>a general activation of the
innate immune response.</strong> 553 proteins were differentially expressed
between the PBMC of <em>mild and severe COVID-19 patients</em>, most of them
(95%) being downregulated in severe patients<em>.</em> Functional pathway
analysis indicated a defect of T cell activation and function in severe
COVID-19. There was also evidence suggesting reduced antibody secretion
by B cells. Together, these results suggest a <strong>functional decline of
adaptive immunity</strong>. A FACS analysis of PBMC from severe patients
indicated higher levels of IL6 and IL8 but not IL17 compared to mild
patients.</p>
<p>Finally, the authors focused on NKRF, an endogenous repressor of IL8/IL6
synthesis that was previously identified as interacting with SARS-Cov-2
nsp9,10,12,13 and 15. Individually expressed nsp9 and nsp10 (but not
nsp12, nsp13, nsp15) induced both IL6 and IL8 in lung epithelial A459
cells, indicating that nsp9 and nsp10 may be directly involved in the
induction of these pro-inflammatory cytokines. The authors finally argue
that nsp9 and nsp10 represent potential drug targets to prevent
over-production of IL6 and IL8 in infected cells, and reducing the
over-activation of neutrophils.</p>
<h3 data-number="10.94.3" id="limitations-92"><span class="header-section-number">10.94.3</span> Limitations</h3>
<p>First, the authors seem to have forgotten to include the extended data
in the manuscript, and their proteomic data does not seem to be publicly
available for the moment, which limits greatly our analysis of their
results.</p>
<p>While this work provides important data on host and viral PPI, only 19
interactions were identified by Y2H system but 52 with co-IP. The
authors do not comment about what could lead to such differences between
the two techniques and they don’t specify whether they detected the same
interactions using the two techniques.</p>
<p>Moreover, the PBMC protein quantification was performed comparing bulk
PBMC. Consequently, protein differences likely reflect differences in
cell populations rather than cell-intrinsic differences in protein
expression. While this analysis is still interesting, a similar
experiment performed on pre-sorted specific cell populations would allow
measuring proteome dynamics at a higher resolution.</p>
<p>Finally, the authors did not discussed their results in regards to
another SARS-CoV-2 interactome of host-viral PPI that had been published
previously<sup>1</sup>. This study reported 332 host-virus PPI, but no
interaction of viral proteins with NKRF was found. Some interactions
were found in both studies (eg. N and G3BP1, Orf6 and RAE1). However,
the time point used to lyse the cells were different (40h previously vs
72h here), which could explain some of the differences.</p>
<h3 data-number="10.94.4" id="relevance"><span class="header-section-number">10.94.4</span> Relevance</h3>
<p>The identification of many interactions between intra-viral and
host-virus PPI provides an overview of host protein and pathways that
are modulated by SARS-CoV-2, which can lead to the identification of
potential targets for drug development.</p>
<p>In the model proposed by the authors, nsp9 and nsp10 from SARS-Cov-2
induce an over-expression of IL6 and IL8 by lung epithelial cells, which
recruits neutrophils and could lead to an excess in lung infiltration.
Nsp9 has been shown to be essential for viral replication for
SARS-Cov-1<sup>2</sup>, and shares a 97% homology with nsp9 from SARS-Cov-2<sup>3</sup>.
Further, nsp9 crystal structure was recently solved<sup>3</sup>, which can help
to develop drug inhibitors if this protein is further confirmed as being
important for the virulence of SARS-Cov-2.</p>
<p>1. Gordon DE, Jang GM, Bouhaddou M, et al. A SARS-CoV-2-Human
Protein-Protein Interaction Map Reveals Drug Targets and Potential
Drug-Repurposing. <em>bioRxiv</em>. March 2020:2020.03.22.002386.
doi:10.1101/2020.03.22.002386</p>
<p>2. Miknis ZJ, Donaldson EF, Umland TC, Rimmer RA, Baric RS, Schultz LW.
Severe acute respiratory syndrome coronavirus nsp9 dimerization is
essential for efficient viral growth. <em>J Virol</em>. 2009;83(7):3007-3018.
doi:10.1128/JVI.01505-08</p>
<p>3. Littler DR, Gully BS, Colson RN, Rossjohn J. <em>Crystal Structure of
the SARS-CoV-2 Non-Structural Protein 9, Nsp9</em>. Molecular Biology; 2020.
doi:10.1101/2020.03.28.013920</p>
<p><strong><em>Title:</em></strong> Prediction and Evolution of B Cell Epitopes of Surface
Protein in SARS-CoV-2</p>
<p><strong><em>Keywords:</em></strong> SARS-CoV-2; Epitopes; Bioinformatics; Evolution</p>
<p><strong><em>Summary/Main findings: </em></strong></p>
<p>Lon et al. used a bioinformatic analysis of the published SARS-CoV-2
genomes in order to identify conserved linear and conformational B cell
epitopes found on the spike (S), envelope (E), and membrane (M)
proteins. The characterization of the surface proteins in this study
began with an assessment of the peptide sequences in order to identify
hydrophilicity indices and protein instability indices using the
Port-Param tool in ExPASy. All three surface proteins were calculated to
have an instability score under 40 indicating that they were stable.
Linear epitopes were identified on the basis of surface probability and
antigenicity, excluding regions of glycosylation. Using BepiPred 2.0
(with a cutoff value of 0.35) and ABCpred (with a cutoff value of 0.51),
4 linear B cell epitopes were predicted for the S protein, 1 epitope for
the E protein, and 1 epitope for the M protein. For structural analysis,
SARS-CoV assemblies published in the Protein Data Bank (PDB) acting as
scaffolds for the SARS-CoV-2 S and E amino acid sequences were used for
input into the SWISS-MODEL server in order to generate three-dimensional
structural models for the assessment of conformational epitopes. Using
Ellipro (cutoff value of 0.063) and SEPPA (cutoff value of 0.5), 1
conformational epitope was identified for the S protein and 1 epitope
was identified for the E protein, both of which are accessible on the
surface of the virus. Finally, the Consurf Server was used to assess the
conservation of these epitopes. All epitopes were conserved across the
published SARS-CoV-2 genomes and one epitope of the spike protein was
predicted to be the most stable across coronavirus phylogeny.</p>
<p><strong><em>Critical Analysis/Limitations:</em></strong></p>
<p>While this study provides a preliminary identification of potential
linear and conformational B cell epitopes, the translational value of
the epitopes described still needs extensive experimental validation to
ascertain whether these elicit a humoral immune response. The
conformational epitope analyses are also limited by the fact that they
are based off of predicted 3D structure from homology comparisons and
not direct crystal structures of the proteins themselves. Additionally,
since there was not a published M protein with a high homology to
SARS-CoV-2, no conformational epitopes were assessed for this protein.
Finally, while evolutionary conservation is an important consideration
in understanding the biology of the virus, conservation does not
necessarily imply that these sites neutralize the virus or aid in
non-neutralizing <em>in vivo</em> protection.</p>
<p><strong><em>Relevance/Implications:</em></strong></p>
<p>With further experimental validation that confirms that these epitopes
induce effective antibody responses to the virus, the epitopes described
can be used for the development of treatments and vaccines as well as
better characterize the viral structure to more deeply understand
pathogenesis.</p>
<!-- default theme -->

<style>
    /* import google fonts */
    @import url("https://fonts.googleapis.com/css?family=Open+Sans:400,600,700");
    @import url("https://fonts.googleapis.com/css?family=Source+Code+Pro");

    /* -------------------------------------------------- */
    /* global */
    /* -------------------------------------------------- */

    /* all elements */
    * {
        /* force sans-serif font unless specified otherwise */
        font-family: "Open Sans", "Helvetica", sans-serif;

        /* prevent text inflation on some mobile browsers */
        -webkit-text-size-adjust: none !important;
        -moz-text-size-adjust: none !important;
        -o-text-size-adjust: none !important;
        text-size-adjust: none !important;
    }

    @media only screen {
        /* "page" element */
        body {
            position: relative;
            box-sizing: border-box;
            font-size: 12pt;
            line-height: 1.5;
            max-width: 8.5in;
            margin: 20px auto;
            padding: 40px;
            border-radius: 5px;
            border: solid 1px #bdbdbd;
            box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
            background: #ffffff;
        }
    }

    /* when on screen < 8.5in wide */
    @media only screen and (max-width: 8.5in) {
        /* "page" element */
        body {
            padding: 20px;
            margin: 0;
            border-radius: 0;
            border: none;
            box-shadow: 0 0 20px rgba(0, 0, 0, 0.05) inset;
            background: none;
        }
    }

    /* -------------------------------------------------- */
    /* headings */
    /* -------------------------------------------------- */

    /* all headings */
    h1,
    h2,
    h3,
    h4,
    h5,
    h6 {
        margin: 20px 0;
        padding: 0;
        font-weight: bold;
    }

    /* biggest heading */
    h1 {
        margin: 40px 0;
    }

    /* second biggest heading */
    h2 {
        margin-top: 30px;
        padding-bottom: 5px;
        border-bottom: solid 1px #bdbdbd;
    }

    /* heading font sizes */
    h1 {
        font-size: 2em;
    }
    h2 {
        font-size: 1.5em;
    }
    h3{
        font-size: 1.35em;
    }
    h4 {
        font-size: 1.25em;
    }
    h5 {
        font-size: 1.15em;
    }
    h6 {
        font-size: 1em;
    }

    /* -------------------------------------------------- */
    /* manuscript header */
    /* -------------------------------------------------- */

    /* manuscript title */
    header > h1 {
        margin: 0;
        text-align: center;
    }

    /* manuscript title caption text (ie "automatically generated on") */
    header + p {
        text-align: center;
        margin-top: 10px;
    }

    /* -------------------------------------------------- */
    /* text elements */
    /* -------------------------------------------------- */

    /* links */
    a {
        color: #2196f3;
        overflow-wrap: break-word;
    }

    /* normal links (not empty, not button link, not syntax highlighting link) */
    a:not(:empty):not(.button):not(.sourceLine) {
        padding-left: 1px;
        padding-right: 1px;
    }

    /* superscripts and subscripts */
    sub,
    sup {
        /* prevent from affecting line height */
        line-height: 0;
    }

    /* unordered and ordered lists*/
    ul,
    ol {
        padding-left: 20px;
    }

    /* class for styling text semibold */
    .semibold {
        font-weight: 600;
    }

    /* class for styling elements horizontally left aligned */
    .left {
        display: block;
        text-align: left;
        margin-left: auto;
        margin-right: 0;
        justify-content: left;
    }

    /* class for styling elements horizontally centered */
    .center {
        display: block;
        text-align: center;
        margin-left: auto;
        margin-right: auto;
        justify-content: center;
    }

    /* class for styling elements horizontally right aligned */
    .right {
        display: block;
        text-align: right;
        margin-left: 0;
        margin-right: auto;
        justify-content: right;
    }

    /* -------------------------------------------------- */
    /* section elements */
    /* -------------------------------------------------- */

    /* horizontal divider line */
    hr {
        border: none;
        height: 1px;
        background: #bdbdbd;
    }

    /* paragraphs, horizontal dividers, figures, tables, code */
    p,
    hr,
    figure,
    table,
    pre {
        /* treat all as "paragraphs", with consistent vertical margins */
        margin-top: 20px;
        margin-bottom: 20px;
    }

    /* -------------------------------------------------- */
    /* figures */
    /* -------------------------------------------------- */

    /* figure */
    figure {
        max-width: 100%;
        margin-left: auto;
        margin-right: auto;
    }

    /* figure caption */
    figcaption {
        padding: 0;
        padding-top: 10px;
    }

    /* figure image element */
    figure > img,
    figure > svg {
        max-width: 100%;
        display: block;
        margin-left: auto;
        margin-right: auto;
    }

    /* figure auto-number */
    img + figcaption > span:first-of-type,
    svg + figcaption > span:first-of-type {
        font-weight: bold;
        margin-right: 5px;
    }

    /* -------------------------------------------------- */
    /* tables */
    /* -------------------------------------------------- */

    /* table */
    table {
        border-collapse: collapse;
        border-spacing: 0;
        width: 100%;
        margin-left: auto;
        margin-right: auto;
    }

    /* table cells */
    th,
    td {
        border: solid 1px #bdbdbd;
        padding: 10px;
        /* squash table if too wide for page by forcing line breaks */
        overflow-wrap: break-word;
        word-break: break-word;
    }

    /* header row and even rows */
    th,
    tr:nth-child(2n) {
        background-color: #fafafa;
    }

    /* odd rows */
    tr:nth-child(2n + 1) {
        background-color: #ffffff;
    }

    /* table caption */
    caption {
        text-align: left;
        padding: 0;
        padding-bottom: 10px;
    }

    /* table auto-number */
    table > caption > span:first-of-type,
    div.table_wrapper > table > caption > span:first-of-type {
        font-weight: bold;
        margin-right: 5px;
    }

    /* -------------------------------------------------- */
    /* code */
    /* -------------------------------------------------- */

    /* multi-line code block */
    pre {
        padding: 10px;
        background-color: #eeeeee;
        color: #000000;
        border-radius: 5px;
        break-inside: avoid;
        text-align: left;
    }

    /* inline code, ie code within normal text */
    :not(pre) > code {
        padding: 0 4px;
        background-color: #eeeeee;
        color: #000000;
        border-radius: 5px;
    }

    /* code text */
    /* apply all children, to reach syntax highlighting sub-elements */
    code,
    code * {
        /* force monospace font */
        font-family: "Source Code Pro", "Courier New", monospace;
    }

    /* -------------------------------------------------- */
    /* quotes */
    /* -------------------------------------------------- */

    /* quoted text */
    blockquote {
        margin: 0;
        padding: 0;
        border-left: 4px solid #bdbdbd;
        padding-left: 16px;
        break-inside: avoid;
    }

    /* -------------------------------------------------- */
    /* banners */
    /* -------------------------------------------------- */

    /* info banners */
    .banner {
        box-sizing: border-box;
        display: block;
        position: relative;
        width: 100%;
        margin-top: 20px;
        margin-bottom: 20px;
        padding: 20px;
        text-align: center;
    }

    /* paragraph in banner */
    .banner > p {
        margin: 0;
    }

    /* -------------------------------------------------- */
    /* highlight colors */
    /* -------------------------------------------------- */

    .white {
        background: #ffffff;
    }
    .lightgrey {
        background: #eeeeee;
    }
    .grey {
        background: #757575;
    }
    .darkgrey {
        background: #424242;
    }
    .black {
        background: #000000;
    }
    .lightred {
        background: #ffcdd2;
    }
    .lightyellow {
        background: #ffecb3;
    }
    .lightgreen {
        background: #dcedc8;
    }
    .lightblue {
        background: #e3f2fd;
    }
    .lightpurple {
        background: #f3e5f5;
    }
    .red {
        background: #f44336;
    }
    .orange {
        background: #ff9800;
    }
    .yellow {
        background: #ffeb3b;
    }
    .green {
        background: #4caf50;
    }
    .blue {
        background: #2196f3;
    }
    .purple {
        background: #9c27b0;
    }
    .white,
    .lightgrey,
    .lightred,
    .lightyellow,
    .lightgreen,
    .lightblue,
    .lightpurple,
    .orange,
    .yellow,
    .white a,
    .lightgrey a,
    .lightred a,
    .lightyellow a,
    .lightgreen a,
    .lightblue a,
    .lightpurple a,
    .orange a,
    .yellow a {
        color: #000000;
    }
    .grey,
    .darkgrey,
    .black,
    .red,
    .green,
    .blue,
    .purple,
    .grey a,
    .darkgrey a,
    .black a,
    .red a,
    .green a,
    .blue a,
    .purple a {
        color: #ffffff;
    }

    /* -------------------------------------------------- */
    /* buttons */
    /* -------------------------------------------------- */

    /* class for styling links like buttons */
    .button {
        display: inline-flex;
        justify-content: center;
        align-items: center;
        margin: 5px;
        padding: 10px 20px;
        font-size: 0.75em;
        font-weight: 600;
        text-transform: uppercase;
        text-decoration: none;
        letter-spacing: 1px;
        background: none;
        color: #2196f3;
        border: solid 1px #bdbdbd;
        border-radius: 5px;
    }

    /* buttons when hovered */
    .button:hover:not([disabled]),
    .icon_button:hover:not([disabled]) {
        cursor: pointer;
        background: #f5f5f5;
    }

    /* buttons when disabled */
    .button[disabled],
    .icon_button[disabled] {
        opacity: 0.35;
        pointer-events: none;
    }

    /* class for styling buttons containg only single icon */
    .icon_button {
        display: inline-flex;
        justify-content: center;
        align-items: center;
        text-decoration: none;
        margin: 0;
        padding: 0;
        background: none;
        border-radius: 5px;
        border: none;
        width: 20px;
        height: 20px;
        min-width: 20px;
        min-height: 20px;
    }

    /* icon button inner svg image */
    .icon_button > svg {
        height: 16px;
    }

    /* -------------------------------------------------- */
    /* icons */
    /* -------------------------------------------------- */

    /* class for styling icons inline with text */
    .inline_icon {
        height: 1em;
        position: relative;
        top: 0.125em;
    }

    /* -------------------------------------------------- */
    /* print control */
    /* -------------------------------------------------- */

    @media print {
        @page {
            /* suggested printing margin */
            margin: 0.5in;
        }

        /* document and "page" elements */
        html, body {
            margin: 0;
            padding: 0;
            width: 100%;
            height: 100%;
        }

        /* "page" element */
        body {
            font-size: 11pt !important;
            line-height: 1.35;
        }

        /* all headings */
        h1,
        h2,
        h3,
        h4,
        h5,
        h6 {
            margin: 15px 0;
        }

        /* figures and tables */
        figure, table {
            font-size: 0.85em;
        }

        /* table cells */
        th,
        td {
            padding: 5px;
        }

        /* shrink font awesome icons */
        i.fas,
        i.fab,
        i.far,
        i.fal {
            transform: scale(0.85);
        }

        /* decrease banner margins */
        .banner {
            margin-top: 15px;
            margin-bottom: 15px;
            padding: 15px;
        }

        /* class for centering an element vertically on its own page */
        .page_center {
            margin: auto;
            width: 100%;
            height: 100%;
            display: flex;
            align-items: center;
            vertical-align: middle;
            break-before: page;
            break-after: page;
        }

        /* always insert a page break before the element */
        .page_break_before {
            break-before: page;
        }

        /* always insert a page break after the element */
        .page_break_after {
            break-after: page;
        }

        /* avoid page break before the element */
        .page_break_before_avoid {
            break-before: avoid;
        }

        /* avoid page break after the element */
        .page_break_after_avoid {
            break-after: avoid;
        }

        /* avoid page break inside the element */
        .page_break_inside_avoid {
            break-inside: avoid;
        }
    }

    /* -------------------------------------------------- */
    /* override pandoc css quirks */
    /* -------------------------------------------------- */

    .sourceCode {
        /* prevent unsightly overflow in wide code blocks */
        overflow: auto !important;
    }

    div.sourceCode {
        /* prevent background fill on top-most code block  container */
        background: none !important;
    }

    .sourceCode * {
        /* force consistent line spacing */
        line-height: 1.5 !important;
    }

    div.sourceCode {
        /* style code block margins same as <pre> element */
        margin-top: 20px;
        margin-bottom: 20px;
    }

    /* -------------------------------------------------- */
    /* tablenos */
    /* -------------------------------------------------- */

    /* tablenos wrapper */
    .tablenos {
        /* show scrollbar on tables if necessary to prevent overflow */
        width: 100%;
        margin: 20px 0;
    }

    .tablenos > table {
        /* move margins from table to table_wrapper to allow margin collapsing */
        margin: 0;
    }

    @media only screen {
        /* tablenos wrapper */
        .tablenos {
            /* show scrollbar on tables if necessary to prevent overflow */
            overflow-x: auto !important;
        }

        .tablenos th,
        .tablenos td {
            overflow-wrap: unset !important;
            word-break: unset !important;
        }

        /* table in wrapper */
        .tablenos table,
        .tablenos table * {
            /* don't break table words */
            overflow-wrap: normal !important;
        }
    }

    /* -------------------------------------------------- */
    /* mathjax */
    /* -------------------------------------------------- */

    /* mathjax containers */
    .math.display > span:not(.MathJax_Preview) {
        /* turn inline element (no dimensions) into block (allows fixed width and thus scrolling) */
        display: flex !important;
        overflow-x: auto !important;
        overflow-y: hidden !important;
        justify-content: center;
        align-items: center;
        margin: 0 !important;
    }

    /* right click menu */
    .MathJax_Menu {
        border-radius: 5px !important;
        border: solid 1px #bdbdbd !important;
        box-shadow: none !important;
    }

    /* equation auto-number */
    span[id^="eq:"] > span.math.display + span {
        font-weight: 600;
    }

    /* equation */
    span[id^="eq:"] > span.math.display > span {
        /* nudge to make room for equation auto-number and anchor */
        margin-right: 60px !important;
    }

    /* -------------------------------------------------- */
    /* anchors plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* anchor button */
        .anchor {
            opacity: 0;
            margin-left: 5px;
        }

        /* anchor buttons within <h2>'s */
        h2 .anchor {
            margin-left: 10px;
        }

        /* anchor buttons when hovered/focused and anything containing an anchor button when hovered */
        *:hover > .anchor,
        .anchor:hover,
        .anchor:focus {
            opacity: 1;
        }

        /* anchor button when hovered */
        .anchor:hover {
            cursor: pointer;
        }
    }

    /* always show anchor button on devices with no mouse/hover ability */
    @media (hover: none) {
        .anchor {
            opacity: 1;
        }
    }

    /* always hide anchor button on print */
    @media only print {
        .anchor {
            display: none;
        }
    }

    /* -------------------------------------------------- */
    /* accordion plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* accordion arrow button */
        .accordion_arrow {
            margin-right: 10px;
        }

        /* arrow icon when <h2> data-collapsed attribute true */
        h2[data-collapsed="true"] > .accordion_arrow > svg {
            transform: rotate(-90deg);
        }

        /* all elements (except <h2>'s) when data-collapsed attribute true */
        *:not(h2)[data-collapsed="true"] {
            display: none;
        }

        /* accordion arrow button when hovered and <h2>'s when hovered */
        .accordion_arrow:hover,
        h2[data-collapsed="true"]:hover,
        h2[data-collapsed="false"]:hover {
            cursor: pointer;
        }
    }

    /* always hide accordion arrow button on print */
    @media only print {
        .accordion_arrow {
            display: none;
        }
    }

    /* -------------------------------------------------- */
    /* tooltips plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* tooltip container */
        #tooltip {
            position: absolute;
            width: 50%;
            min-width: 240px;
            max-width: 75%;
            z-index: 1;
        }

        /* tooltip content */
        #tooltip_content {
            margin-bottom: 5px;
            padding: 20px;
            border-radius: 5px;
            border: solid 1px #bdbdbd;
            box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
            background: #ffffff;
            overflow-wrap: break-word;
        }

        /* tooltip copy of paragraphs and figures */
        #tooltip_content > p,
        #tooltip_content > figure {
            margin: 0;
            max-height: 320px;
            overflow-y: auto;
        }

        /* tooltip copy of <img> */
        #tooltip_content > figure > img,
        #tooltip_content > figure > svg {
            max-height: 260px;
        }

        /* navigation bar */
        #tooltip_nav_bar {
            margin-top: 10px;
            text-align: center;
        }

        /* navigation bar previous/next buton */
        #tooltip_nav_bar > .icon_button {
            position: relative;
            top: 3px;
        }

        /* navigation bar previous button */
        #tooltip_nav_bar > .icon_button:first-of-type {
            margin-right: 5px;
        }

        /* navigation bar next button */
        #tooltip_nav_bar > .icon_button:last-of-type {
            margin-left: 5px;
        }
    }

    /* always hide tooltip on print */
    @media only print {
        #tooltip {
            display: none;
        }
    }

    /* -------------------------------------------------- */
    /* jump to first plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* jump button */
        .jump_arrow {
            position: relative;
            top: 0.125em;
            margin-right: 5px;
        }
    }

    /* always hide jump button on print */
    @media only print {
        .jump_arrow {
            display: none;
        }
    }

    /* -------------------------------------------------- */
    /* link highlight plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* anything with data-highlighted attribute true */
        [data-highlighted="true"] {
            background: #ffeb3b;
        }

        /* anything with data-selected attribute true */
        [data-selected="true"] {
            background: #ff8a65 !important;
        }

        /* animation definition for glow */
        @keyframes highlight_glow {
            0% {
                background: none;
            }
            10% {
                background: #bbdefb;
            }
            100% {
                background: none;
            }
        }

        /* anything with data-glow attribute true */
        [data-glow="true"] {
            animation: highlight_glow 2s;
        }
    }

    /* -------------------------------------------------- */
    /* table of contents plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* toc panel */
        #toc_panel {
            box-sizing: border-box;
            position: fixed;
            top: 0;
            left: 0;
            background: #ffffff;
            box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
            z-index: 2;
        }

        /* toc panel when closed */
        #toc_panel[data-open="false"] {
            min-width: 60px;
            width: 60px;
            height: 60px;
            border-right: solid 1px #bdbdbd;
            border-bottom: solid 1px #bdbdbd;
        }

        /* toc panel when open */
        #toc_panel[data-open="true"] {
            min-width: 260px;
            max-width: 480px;
            /* keep panel edge consistent distance away from "page" edge */
            width: calc(((100vw - 8.5in) / 2) - 30px - 40px);
            bottom: 0;
            border-right: solid 1px #bdbdbd;
        }

        /* toc panel header */
        #toc_header {
            box-sizing: border-box;
            display: flex;
            flex-direction: row;
            align-items: center;
            height: 60px;
            margin: 0;
            padding: 20px;
        }

        /* toc panel header when hovered */
        #toc_header:hover {
            cursor: pointer;
        }

        /* toc panel header when panel open */
        #toc_panel[data-open="true"] > #toc_header {
            border-bottom: solid 1px #bdbdbd;
        }

        /* toc open/close header button */
        #toc_button {
            margin-right: 20px;
        }

        /* hide toc list and header text when closed */
        #toc_panel[data-open="false"] > #toc_header > *:not(#toc_button),
        #toc_panel[data-open="false"] > #toc_list {
            display: none;
        }

        /* toc list of entries */
        #toc_list {
            box-sizing: border-box;
            width: 100%;
            padding: 20px;
            position: absolute;
            top: calc(60px + 1px);
            bottom: 0;
            overflow: auto;
        }

        /* toc entry, link to section in document */
        .toc_link {
            display: block;
            padding: 5px;
            position: relative;
            font-weight: 600;
            text-decoration: none;
        }

        /* toc entry when hovered or when "viewed" */
        .toc_link:hover,
        .toc_link[data-viewing="true"] {
            background: #f5f5f5;
        }

        /* toc entry, level 1 indentation */
        .toc_link[data-level="1"] {
            margin-left: 0;
        }

        /* toc entry, level 2 indentation */
        .toc_link[data-level="2"] {
            margin-left: 20px;
        }

        /* toc entry, level 3 indentation */
        .toc_link[data-level="3"] {
            margin-left: 40px;
        }

        /* toc entry, level 4 indentation */
        .toc_link[data-level="4"] {
            margin-left: 60px;
        }

        /* toc entry bullets */
        #toc_panel[data-bullets="true"] .toc_link[data-level]:before {
            position: absolute;
            left: -15px;
            top: -1px;
            font-size: 1.5em;
        }

        /* toc entry, level 2 bullet */
        #toc_panel[data-bullets="true"] .toc_link[data-level="2"]:before {
            content: "\2022";
        }

        /* toc entry, level 3 bullet */
        #toc_panel[data-bullets="true"] .toc_link[data-level="3"]:before {
            content: "\25AB";
        }

        /* toc entry, level 4 bullet */
        #toc_panel[data-bullets="true"] .toc_link[data-level="4"]:before {
            content: "-";
        }
    }

    /* when on screen < 8.5in wide */
    @media only screen and (max-width: 8.5in) {
        /* push <body> ("page") element down to make room for toc icon */
        .toc_body_nudge {
            padding-top: 60px;
        }

        /* toc icon when panel closed and not hovered */
        #toc_panel[data-open="false"]:not(:hover) {
            background: rgba(255, 255, 255, 0.75);
        }
    }

    /* always hide toc panel on print */
    @media only print {
        #toc_panel {
            display: none;
        }
    }

    /* -------------------------------------------------- */
    /* lightbox plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* regular <img> in document when hovered */
        img.lightbox_document_img:hover {
            cursor: pointer;
        }

        .body_no_scroll {
            overflow: hidden !important;
        }

        /* screen overlay */
        #lightbox_overlay {
            display: flex;
            flex-direction: column;
            position: fixed;
            left: 0;
            top: 0;
            right: 0;
            bottom: 0;
            background: rgba(0, 0, 0, 0.75);
            z-index: 3;
        }

        /* middle area containing lightbox image */
        #lightbox_image_container {
            flex-grow: 1;
            display: flex;
            justify-content: center;
            align-items: center;
            overflow: hidden;
            position: relative;
            padding: 20px;
        }

        /* bottom area containing caption */
        #lightbox_bottom_container {
            display: flex;
            justify-content: center;
            align-items: center;
            height: 100px;
            min-height: 100px;
            max-height: 100px;
            background: rgba(0, 0, 0, 0.5);
        }

        /* image number info text box */
        #lightbox_number_info {
            position: absolute;
            color: #ffffff;
            font-weight: 600;
            left: 2px;
            top: 0;
            z-index: 4;
        }

        /* zoom info text box */
        #lightbox_zoom_info {
            position: absolute;
            color: #ffffff;
            font-weight: 600;
            right: 2px;
            top: 0;
            z-index: 4;
        }

        /* copy of image caption */
        #lightbox_caption {
            box-sizing: border-box;
            display: inline-block;
            width: 100%;
            max-height: 100%;
            padding: 10px 0;
            text-align: center;
            overflow-y: auto;
            color: #ffffff;
        }

        /* navigation previous/next button */
        .lightbox_button {
            width: 100px;
            height: 100%;
            min-width: 100px;
            min-height: 100%;
            color: #ffffff;
        }

        /* navigation previous/next button when hovered */
        .lightbox_button:hover {
            background: none !important;
        }

        /* navigation button icon */
        .lightbox_button > svg {
            height: 25px;
        }

        /* figure auto-number */
        #lightbox_caption > span:first-of-type {
            font-weight: bold;
            margin-right: 5px;
        }

        /* lightbox image when hovered */
        #lightbox_img:hover {
            cursor: grab;
        }

        /* lightbox image when grabbed */
        #lightbox_img:active {
            cursor: grabbing;
        }
    }

    /* when on screen < 480px wide */
    @media only screen and (max-width: 480px) {
        /* make navigation buttons skinnier on small screens to make more room for caption text */
        .lightbox_button {
            width: 50px;
            min-width: 50px;
        }
    }

    /* always hide lightbox on print */
    @media only print {
        #lightbox_overlay {
            display: none;
        }
    }

    /* -------------------------------------------------- */
    /* hypothesis (annotations) plugin */
    /* -------------------------------------------------- */

    /* hypothesis activation button */
    #hypothesis_button {
        box-sizing: border-box;
        position: fixed;
        top: 0;
        right: 0;
        width: 60px;
        height: 60px;
        background: #ffffff;
        border-radius: 0;
        border-left: solid 1px #bdbdbd;
        border-bottom: solid 1px #bdbdbd;
        box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
        z-index: 2;
    }

    /* hypothesis button svg */
    #hypothesis_button > svg {
        position: relative;
        top: -4px;
    }

    /* hypothesis annotation count */
    #hypothesis_count {
        position: absolute;
        left: 0;
        right: 0;
        bottom: 5px;
    }

    /* side panel */
    .annotator-frame {
        width: 280px !important;
    }

    /* match highlight color to rest of theme */
    .annotator-highlights-always-on .annotator-hl {
        background-color: #ffeb3b !important;
    }

    /* match focused color to rest of theme */
    .annotator-hl.annotator-hl-focused {
        background-color: #ff8a65 !important;
    }

    /* match bucket bar color to rest of theme */
    .annotator-bucket-bar {
        background: #f5f5f5 !important;
    }

    /* always hide button, toolbar, and tooltip on print */
    @media only print {
        #hypothesis_button {
            display: none;
        }

        .annotator-frame {
            display: none !important;
        }

        hypothesis-adder {
            display: none !important;
        }
    }
</style>
<!-- anchors plugin -->

<script>
    (function() {
        // /////////////////////////
        // DESCRIPTION
        // /////////////////////////

        // This Manubot plugin adds an anchor next to each of a certain type
        // of element that provides a human-readable url to that specific
        // item/position in the document (eg "manuscript.html#abstract"). It
        // also makes it such that scrolling out of view of a target removes
        // its identifier from the url.

        // /////////////////////////
        // OPTIONS
        // /////////////////////////

        // plugin name prefix for url parameters
        const pluginName = 'anchors';

        // default plugin options
        const options = {
            // which types of elements to add anchors next to, in
            // "document.querySelector" format
            typesQuery: 'h1, h2, h3, [id^="fig:"], [id^="tbl:"], [id^="eq:"]',
            // whether plugin is on or not
            enabled: 'true'
        };

        // change options above, or override with url parameter, eg:
        // 'manuscript.html?pluginName-enabled=false'

        // /////////////////////////
        // SCRIPT
        // /////////////////////////

        // start script
        function start() {
            // add anchor to each element of specified types
            const elements = document.querySelectorAll(options.typesQuery);
            for (const element of elements)
                addAnchor(element);

            // attach scroll listener to window
            window.addEventListener('scroll', onScroll);
        }

        // when window is scrolled
        function onScroll() {
            // if url has hash and user has scrolled out of view of hash
            // target, remove hash from url
            const tolerance = 100;
            const target = getHashTarget();
            if (target) {
                if (
                    target.getBoundingClientRect().top >
                        window.innerHeight + tolerance ||
                    target.getBoundingClientRect().bottom < 0 - tolerance
                )
                    history.pushState(null, null, ' ');
            }
        }

        // add anchor to element
        function addAnchor(element) {
            let addTo; // element to add anchor button to

            // if figure or table, modify withId and addTo to get expected
            // elements
            if (element.id.indexOf('fig:') === 0) {
                addTo = element.querySelector('figcaption');
            } else if (element.id.indexOf('tbl:') === 0) {
                addTo = element.querySelector('caption');
            } else if (element.id.indexOf('eq:') === 0) {
                addTo = element.querySelector('.eqnos-number');
            }

            addTo = addTo || element;
            const id = element.id || null;

            // do not add anchor if element doesn't have assigned id.
            // id is generated by pandoc and is assumed to be unique and
            // human-readable
            if (!id)
                return;

            // create anchor button
            const anchor = document.createElement('a');
            anchor.innerHTML = document.querySelector('.icon_link').innerHTML;
            anchor.title = 'Link to this part of the document';
            anchor.classList.add('icon_button', 'anchor');
            anchor.dataset.ignore = 'true';
            anchor.href = '#' + id;
            addTo.appendChild(anchor);
        }

        // get element that is target of link or url hash
        function getHashTarget() {
            const hash = window.location.hash;
            const id = hash.slice(1);
            let target = document.querySelector('[id="' + id + '"]');
            if (!target)
                return;

            // if figure or table, modify target to get expected element
            if (id.indexOf('fig:') === 0)
                target = target.querySelector('figure');
            if (id.indexOf('tbl:') === 0)
                target = target.querySelector('table');

            return target;
        }

        // load options from url parameters
        function loadOptions() {
            const url = window.location.search;
            const params = new URLSearchParams(url);
            for (const optionName of Object.keys(options)) {
                const paramName = pluginName + '-' + optionName;
                const param = params.get(paramName);
                if (param !== '' && param !== null)
                    options[optionName] = param;
            }
        }
        loadOptions();

        // start script when document is finished loading
        if (options.enabled === 'true')
            window.addEventListener('load', start);
    })();
</script>

<!-- link icon -->

<template class="icon_link">
    <!-- modified from: https://fontawesome.com/icons/link -->
    <svg width="16" height="16" viewBox="0 0 512 512">
        <path
            fill="currentColor"
            d="M326.612 185.391c59.747 59.809 58.927 155.698.36 214.59-.11.12-.24.25-.36.37l-67.2 67.2c-59.27 59.27-155.699 59.262-214.96 0-59.27-59.26-59.27-155.7 0-214.96l37.106-37.106c9.84-9.84 26.786-3.3 27.294 10.606.648 17.722 3.826 35.527 9.69 52.721 1.986 5.822.567 12.262-3.783 16.612l-13.087 13.087c-28.026 28.026-28.905 73.66-1.155 101.96 28.024 28.579 74.086 28.749 102.325.51l67.2-67.19c28.191-28.191 28.073-73.757 0-101.83-3.701-3.694-7.429-6.564-10.341-8.569a16.037 16.037 0 0 1-6.947-12.606c-.396-10.567 3.348-21.456 11.698-29.806l21.054-21.055c5.521-5.521 14.182-6.199 20.584-1.731a152.482 152.482 0 0 1 20.522 17.197zM467.547 44.449c-59.261-59.262-155.69-59.27-214.96 0l-67.2 67.2c-.12.12-.25.25-.36.37-58.566 58.892-59.387 154.781.36 214.59a152.454 152.454 0 0 0 20.521 17.196c6.402 4.468 15.064 3.789 20.584-1.731l21.054-21.055c8.35-8.35 12.094-19.239 11.698-29.806a16.037 16.037 0 0 0-6.947-12.606c-2.912-2.005-6.64-4.875-10.341-8.569-28.073-28.073-28.191-73.639 0-101.83l67.2-67.19c28.239-28.239 74.3-28.069 102.325.51 27.75 28.3 26.872 73.934-1.155 101.96l-13.087 13.087c-4.35 4.35-5.769 10.79-3.783 16.612 5.864 17.194 9.042 34.999 9.69 52.721.509 13.906 17.454 20.446 27.294 10.606l37.106-37.106c59.271-59.259 59.271-155.699.001-214.959z"
        ></path>
    </svg>
</template>
<!-- accordion plugin -->

<script>
    (function() {
        // /////////////////////////
        // DESCRIPTION
        // /////////////////////////

        // This Manubot plugin allows sections of content under <h2> headings
        // to be collapsible.

        // /////////////////////////
        // OPTIONS
        // /////////////////////////

        // plugin name prefix for url parameters
        const pluginName = 'accordion';

        // default plugin options
        const options = {
            // whether to always start expanded ('false'), always start
            // collapsed ('true'), or start collapsed when screen small ('auto')
            startCollapsed: 'auto',
            // whether plugin is on or not
            enabled: 'true'
        };

        // change options above, or override with url parameter, eg:
        // 'manuscript.html?pluginName-enabled=false'

        // /////////////////////////
        // SCRIPT
        // /////////////////////////

        // start script
        function start() {
            // run through each <h2> heading
            const headings = document.querySelectorAll('h2');
            for (const heading of headings) {
                addArrow(heading);

                // start expanded/collapsed based on option
                if (
                    options.startCollapsed === 'true' ||
                    (options.startCollapsed === 'auto' && isSmallScreen())
                )
                    collapseHeading(heading);
                else
                    expandHeading(heading);
            }

            // attach hash change listener to window
            window.addEventListener('hashchange', onHashChange);
        }

        // when hash (eg manuscript.html#introduction) changes
        function onHashChange() {
            const target = getHashTarget();
            if (target)
                goToElement(target);
        }

        // add arrow to heading
        function addArrow(heading) {
            // add arrow button
            const arrow = document.createElement('button');
            arrow.innerHTML = document.querySelector(
                '.icon_angle_down'
            ).innerHTML;
            arrow.classList.add('icon_button', 'accordion_arrow');
            heading.insertBefore(arrow, heading.firstChild);

            // attach click listener to heading and button
            heading.addEventListener('click', onHeadingClick);
            arrow.addEventListener('click', onArrowClick);
        }

        // determine if on mobile-like device with small screen
        function isSmallScreen() {
            return Math.min(window.innerWidth, window.innerHeight) < 480;
        }

        // scroll to and focus element
        function goToElement(element, offset) {
            // expand accordion section if collapsed
            expandElement(element);
            const y =
                getRectInView(element).top -
                getRectInView(document.documentElement).top -
                (offset || 0);
            // trigger any function listening for "onscroll" event
            window.dispatchEvent(new Event('scroll'));
            window.scrollTo(0, y);
            document.activeElement.blur();
            element.focus();
        }

        // get element that is target of hash
        function getHashTarget(link) {
            const hash = link ? link.hash : window.location.hash;
            const id = hash.slice(1);
            let target = document.querySelector('[id="' + id + '"]');
            if (!target)
                return;

            // if figure or table, modify target to get expected element
            if (id.indexOf('fig:') === 0)
                target = target.querySelector('figure');
            if (id.indexOf('tbl:') === 0)
                target = target.querySelector('table');

            return target;
        }

        // when <h2> heading is clicked
        function onHeadingClick(event) {
            // only collapse if <h2> itself is target of click (eg, user did
            // not click on anchor within <h2>)
            if (event.target === this)
                toggleCollapse(this);
        }

        // when arrow button is clicked
        function onArrowClick() {
            toggleCollapse(this.parentNode);
        }

        // collapse section if expanded, expand if collapsed
        function toggleCollapse(heading) {
            if (heading.dataset.collapsed === 'false')
                collapseHeading(heading);
            else
                expandHeading(heading);
        }

        // elements to exclude from collapse, such as table of contents panel,
        // hypothesis panel, etc
        const exclude = '#toc_panel, div.annotator-frame, #lightbox_overlay';

        // collapse section
        function collapseHeading(heading) {
            heading.setAttribute('data-collapsed', 'true');
            const children = getChildren(heading);
            for (const child of children)
                child.setAttribute('data-collapsed', 'true');
        }

        // expand section
        function expandHeading(heading) {
            heading.setAttribute('data-collapsed', 'false');
            const children = getChildren(heading);
            for (const child of children)
                child.setAttribute('data-collapsed', 'false');
        }

        // get list of elements between this <h2> and next <h2> or <h1>
        // ("children" of the <h2> section)
        function getChildren(heading) {
            return nextUntil(heading, 'h2, h1', exclude);
        }

        // get position/dimensions of element or viewport
        function getRectInView(element) {
            let rect = {};
            rect.left = 0;
            rect.top = 0;
            rect.right = document.documentElement.clientWidth;
            rect.bottom = document.documentElement.clientHeight;
            let style = {};

            if (element instanceof HTMLElement) {
                rect = element.getBoundingClientRect();
                style = window.getComputedStyle(element);
            }

            const margin = {};
            margin.left = parseFloat(style.marginLeftWidth) || 0;
            margin.top = parseFloat(style.marginTopWidth) || 0;
            margin.right = parseFloat(style.marginRightWidth) || 0;
            margin.bottom = parseFloat(style.marginBottomWidth) || 0;

            const border = {};
            border.left = parseFloat(style.borderLeftWidth) || 0;
            border.top = parseFloat(style.borderTopWidth) || 0;
            border.right = parseFloat(style.borderRightWidth) || 0;
            border.bottom = parseFloat(style.borderBottomWidth) || 0;

            const newRect = {};
            newRect.left = rect.left + margin.left + border.left;
            newRect.top = rect.top + margin.top + border.top;
            newRect.right = rect.right + margin.right + border.right;
            newRect.bottom = rect.bottom + margin.bottom + border.bottom;
            newRect.width = newRect.right - newRect.left;
            newRect.height = newRect.bottom - newRect.top;

            return newRect;
        }

        // get list of elements after a start element up to element matching
        // query
        function nextUntil(element, query, exclude) {
            const elements = [];
            while (element = element.nextElementSibling, element) {
                if (element.matches(query))
                    break;
                if (!element.matches(exclude))
                    elements.push(element);
            }
            return elements;
        }

        // get closest element before specified element that matches query
        function firstBefore(element, query) {
            while (
                element &&
                element !== document.body &&
                !element.matches(query)
            )
                element = element.previousElementSibling || element.parentNode;

            return element;
        }

        // check if element is part of collapsed heading
        function isCollapsed(element) {
            while (element && element !== document.body) {
                if (element.dataset.collapsed === 'true')
                    return true;
                element = element.parentNode;
            }
            return false;
        }

        // expand heading containing element if necesary
        function expandElement(element) {
            if (isCollapsed(element)) {
                const heading = firstBefore(element, 'h2');
                if (heading)
                    heading.click();
            }
        }

        // load options from url parameters
        function loadOptions() {
            const url = window.location.search;
            const params = new URLSearchParams(url);
            for (const optionName of Object.keys(options)) {
                const paramName = pluginName + '-' + optionName;
                const param = params.get(paramName);
                if (param !== '' && param !== null)
                    options[optionName] = param;
            }
        }
        loadOptions();

        // start script when document is finished loading
        if (options.enabled === 'true')
            window.addEventListener('load', start);
    })();
</script>

<!-- angle down icon -->

<template class="icon_angle_down">
    <!-- modified from: https://fontawesome.com/icons/angle-down -->
    <svg width="16" height="16" viewBox="0 0 448 512">
        <path
            fill="currentColor"
            d="M207.029 381.476L12.686 187.132c-9.373-9.373-9.373-24.569 0-33.941l22.667-22.667c9.357-9.357 24.522-9.375 33.901-.04L224 284.505l154.745-154.021c9.379-9.335 24.544-9.317 33.901.04l22.667 22.667c9.373 9.373 9.373 24.569 0 33.941L240.971 381.476c-9.373 9.372-24.569 9.372-33.942 0z"
        ></path>
    </svg>
</template>
<!-- tooltips plugin -->

<script>
    (function() {
        // /////////////////////////
        // DESCRIPTION
        // /////////////////////////

        // This Manubot plugin makes it such that when the user hovers or
        // focuses a link to a citation or figure, a tooltip appears with a
        // preview of the reference content, along with arrows to navigate
        // between instances of the same reference in the document.

        // /////////////////////////
        // OPTIONS
        // /////////////////////////

        // plugin name prefix for url parameters
        const pluginName = 'tooltips';

        // default plugin options
        const options = {
            // whether user must click off to close tooltip instead of just
            // un-hovering
            clickClose: 'false',
            // delay (in ms) between opening and closing tooltip
            delay: '100',
            // whether plugin is on or not
            enabled: 'true'
        };

        // change options above, or override with url parameter, eg:
        // 'manuscript.html?pluginName-enabled=false'

        // /////////////////////////
        // SCRIPT
        // /////////////////////////

        // start script
        function start() {
            const links = getLinks();
            for (const link of links) {
                // attach hover and focus listeners to link
                link.addEventListener('mouseover', onLinkHover);
                link.addEventListener('mouseleave', onLinkUnhover);
                link.addEventListener('focus', onLinkFocus);
                link.addEventListener('touchend', onLinkTouch);
            }

            // attach mouse, key, and resize listeners to window
            window.addEventListener('mousedown', onClick);
            window.addEventListener('touchstart', onClick);
            window.addEventListener('keyup', onKeyUp);
            window.addEventListener('resize', onResize);
        }

        // when link is hovered
        function onLinkHover() {
            // function to open tooltip
            const delayOpenTooltip = function() {
                openTooltip(this);
            }.bind(this);

            // run open function after delay
            this.openTooltipTimer = window.setTimeout(
                delayOpenTooltip,
                options.delay
            );
        }

        // when mouse leaves link
        function onLinkUnhover() {
            // cancel opening tooltip
            window.clearTimeout(this.openTooltipTimer);

            // don't close on unhover if option specifies
            if (options.clickClose === 'true')
                return;

            // function to close tooltip
            const delayCloseTooltip = function() {
                // if tooltip open and if mouse isn't over tooltip, close
                const tooltip = document.getElementById('tooltip');
                if (tooltip && !tooltip.matches(':hover'))
                    closeTooltip();
            };

            // run close function after delay
            this.closeTooltipTimer = window.setTimeout(
                delayCloseTooltip,
                options.delay
            );
        }

        // when link is focused (tabbed to)
        function onLinkFocus(event) {
            openTooltip(this);
        }

        // when link is touched on touch screen
        function onLinkTouch(event) {
            // attempt to force hover state on first tap always, and trigger
            // regular link click (and navigation) on second tap
            if (event.target === document.activeElement)
                event.target.click();
            else {
                document.activeElement.blur();
                event.target.focus();
            }
            if (event.cancelable)
                event.preventDefault();
            event.stopPropagation();
            return false;
        }

        // when mouse is clicked anywhere in window
        function onClick(event) {
            closeTooltip();
        }

        // when key pressed
        function onKeyUp(event) {
            if (!event || !event.key)
                return;

            switch (event.key) {
                // trigger click of prev button
                case 'ArrowLeft':
                    const prevButton = document.getElementById(
                        'tooltip_prev_button'
                    );
                    if (prevButton)
                        prevButton.click();
                    break;
                // trigger click of next button
                case 'ArrowRight':
                    const nextButton = document.getElementById(
                        'tooltip_next_button'
                    );
                    if (nextButton)
                        nextButton.click();
                    break;
                // close on esc
                case 'Escape':
                    closeTooltip();
                    break;
            }
        }

        // when window is resized or zoomed
        function onResize() {
            closeTooltip();
        }

        // get all links of types we wish to handle
        function getLinks() {
            const queries = [];
            // exclude buttons, anchor links, toc links, etc
            const exclude =
                ':not(.button):not(.icon_button):not(.anchor):not(.toc_link)';
            queries.push('a[href^="#ref-"]' + exclude); // citation links
            queries.push('a[href^="#fig:"]' + exclude); // figure links
            const query = queries.join(', ');
            return document.querySelectorAll(query);
        }

        // get links with same target, get index of link in set, get total
        // same links
        function getSameLinks(link) {
            const sameLinks = [];
            const links = getLinks();
            for (const otherLink of links) {
                if (
                    otherLink.getAttribute('href') === link.getAttribute('href')
                )
                    sameLinks.push(otherLink);
            }

            return {
                elements: sameLinks,
                index: sameLinks.indexOf(link),
                total: sameLinks.length
            };
        }

        // open tooltip
        function openTooltip(link) {
            // delete tooltip if it exists, start fresh
            closeTooltip();

            // make tooltip element
            const tooltip = makeTooltip(link);

            // if source couldn't be found and tooltip not made, exit
            if (!tooltip)
                return;

            // make navbar elements
            const navBar = makeNavBar(link);
            if (navBar)
                tooltip.firstElementChild.appendChild(navBar);

            // attach tooltip to page
            document.body.appendChild(tooltip);

            // position tooltip
            const position = function() {
                positionTooltip(link);
            };
            position();

            // if tooltip contains images, position again after they've loaded
            const imgs = tooltip.querySelectorAll('img');
            for (const img of imgs)
                img.addEventListener('load', position);
        }

        // close (delete) tooltip
        function closeTooltip() {
            const tooltip = document.getElementById('tooltip');
            if (tooltip)
                tooltip.remove();
        }

        // make tooltip
        function makeTooltip(link) {
            // get target element that link points to
            const source = getSource(link);

            // if source can't be found, exit
            if (!source)
                return;

            // create new tooltip
            const tooltip = document.createElement('div');
            tooltip.id = 'tooltip';
            const tooltipContent = document.createElement('div');
            tooltipContent.id = 'tooltip_content';
            tooltip.appendChild(tooltipContent);

            // make copy of source node and put in tooltip
            const sourceCopy = makeCopy(source);
            tooltipContent.appendChild(sourceCopy);

            // attach mouse event listeners
            tooltip.addEventListener('click', onTooltipClick);
            tooltip.addEventListener('mousedown', onTooltipClick);
            tooltip.addEventListener('touchstart', onTooltipClick);
            tooltip.addEventListener('mouseleave', onTooltipUnhover);

            // (for interaction with lightbox plugin)
            // transfer click on tooltip copied img to original img
            const sourceImg = source.querySelector('img');
            const sourceCopyImg = sourceCopy.querySelector('img');
            if (sourceImg && sourceCopyImg) {
                const clickImg = function() {
                    sourceImg.click();
                    closeTooltip();
                };
                sourceCopyImg.addEventListener('click', clickImg);
            }

            return tooltip;
        }

        // make carbon copy of html dom element
        function makeCopy(source) {
            const sourceCopy = source.cloneNode(true);

            // delete elements marked with ignore (eg anchor and jump buttons)
            const deleteFromCopy = sourceCopy.querySelectorAll(
                '[data-ignore="true"]'
            );
            for (const element of deleteFromCopy)
                element.remove();

            // delete certain element attributes
            const attributes = [
                'id',
                'data-collapsed',
                'data-selected',
                'data-highlighted',
                'data-glow'
            ];
            for (const attribute of attributes) {
                sourceCopy.removeAttribute(attribute);
                const elements = sourceCopy.querySelectorAll(
                    '[' + attribute + ']'
                );
                for (const element of elements)
                    element.removeAttribute(attribute);
            }

            return sourceCopy;
        }

        // when tooltip is clicked
        function onTooltipClick(event) {
            // when user clicks on tooltip, stop click from transferring
            // outside of tooltip (eg, click off to close tooltip, or eg click
            // off to unhighlight same refs)
            event.stopPropagation();
        }

        // when tooltip is unhovered
        function onTooltipUnhover(event) {
            if (options.clickClose === 'true')
                return;

            // make sure new mouse/touch/focus no longer over tooltip or any
            // element within it
            const tooltip = document.getElementById('tooltip');
            if (!tooltip)
                return;
            if (this.contains(event.relatedTarget))
                return;

            closeTooltip();
        }

        // make nav bar to go betwen prev/next instances of same reference
        function makeNavBar(link) {
            // find other links to the same source
            const sameLinks = getSameLinks(link);

            // don't show nav bar when singular reference
            if (sameLinks.total <= 1)
                return;

            // find prev/next links with same target
            const prevLink = getPrevLink(link, sameLinks);
            const nextLink = getNextLink(link, sameLinks);

            // create nav bar
            const navBar = document.createElement('div');
            navBar.id = 'tooltip_nav_bar';
            const text = sameLinks.index + 1 + ' of ' + sameLinks.total;

            // create nav bar prev/next buttons
            const prevButton = document.createElement('button');
            const nextButton = document.createElement('button');
            prevButton.id = 'tooltip_prev_button';
            nextButton.id = 'tooltip_next_button';
            prevButton.title =
                'Jump to the previous occurence of this item in the document [←]';
            nextButton.title =
                'Jump to the next occurence of this item in the document [→]';
            prevButton.classList.add('icon_button');
            nextButton.classList.add('icon_button');
            prevButton.innerHTML = document.querySelector(
                '.icon_caret_left'
            ).innerHTML;
            nextButton.innerHTML = document.querySelector(
                '.icon_caret_right'
            ).innerHTML;
            navBar.appendChild(prevButton);
            navBar.appendChild(document.createTextNode(text));
            navBar.appendChild(nextButton);

            // attach click listeners to buttons
            prevButton.addEventListener('click', function() {
                onPrevNextClick(link, prevLink);
            });
            nextButton.addEventListener('click', function() {
                onPrevNextClick(link, nextLink);
            });

            return navBar;
        }

        // get previous link with same target
        function getPrevLink(link, sameLinks) {
            if (!sameLinks)
                sameLinks = getSameLinks(link);
            // wrap index to other side if < 1
            let index;
            if (sameLinks.index - 1 >= 0)
                index = sameLinks.index - 1;
            else
                index = sameLinks.total - 1;
            return sameLinks.elements[index];
        }

        // get next link with same target
        function getNextLink(link, sameLinks) {
            if (!sameLinks)
                sameLinks = getSameLinks(link);
            // wrap index to other side if > total
            let index;
            if (sameLinks.index + 1 <= sameLinks.total - 1)
                index = sameLinks.index + 1;
            else
                index = 0;
            return sameLinks.elements[index];
        }

        // get element that is target of link or url hash
        function getSource(link) {
            const hash = link ? link.hash : window.location.hash;
            const id = hash.slice(1);
            let target = document.querySelector('[id="' + id + '"]');
            if (!target)
                return;

            // if ref or figure, modify target to get expected element
            if (id.indexOf('ref-') === 0)
                target = target.querySelector('p');
            else if (id.indexOf('fig:') === 0)
                target = target.querySelector('figure');

            return target;
        }

        // when prev/next arrow button is clicked
        function onPrevNextClick(link, prevNextLink) {
            if (link && prevNextLink)
                goToElement(prevNextLink, window.innerHeight * 0.5);
        }

        // scroll to and focus element
        function goToElement(element, offset) {
            // expand accordion section if collapsed
            expandElement(element);
            const y =
                getRectInView(element).top -
                getRectInView(document.documentElement).top -
                (offset || 0);
            // trigger any function listening for "onscroll" event
            window.dispatchEvent(new Event('scroll'));
            window.scrollTo(0, y);
            document.activeElement.blur();
            element.focus();
        }

        // determine position to place tooltip based on link position in
        // viewport and tooltip size
        function positionTooltip(link, left, top) {
            const tooltipElement = document.getElementById('tooltip');
            if (!tooltipElement)
                return;

            // get convenient vars for position/dimensions of
            // link/tooltip/page/view
            link = getRectInPage(link);
            const tooltip = getRectInPage(tooltipElement);
            const view = getRectInPage();

            // horizontal positioning
            if (left)
                // use explicit value
                left = left;
            else if (link.left + tooltip.width < view.right)
                // fit tooltip to right of link
                left = link.left;
            else if (link.right - tooltip.width > view.left)
                // fit tooltip to left of link
                left = link.right - tooltip.width;
            // center tooltip in view
            else
                left = (view.right - view.left) / 2 - tooltip.width / 2;

            // vertical positioning
            if (top)
                // use explicit value
                top = top;
            else if (link.top - tooltip.height > view.top)
                // fit tooltip above link
                top = link.top - tooltip.height;
            else if (link.bottom + tooltip.height < view.bottom)
                // fit tooltip below link
                top = link.bottom;
            else {
                // center tooltip in view
                top = view.top + view.height / 2 - tooltip.height / 2;
                // nudge off of link to left/right if possible
                if (link.right + tooltip.width < view.right)
                    left = link.right;
                else if (link.left - tooltip.width > view.left)
                    left = link.left - tooltip.width;
            }

            tooltipElement.style.left = left + 'px';
            tooltipElement.style.top = top + 'px';
        }

        // get position/dimensions of element or viewport
        function getRectInView(element) {
            let rect = {};
            rect.left = 0;
            rect.top = 0;
            rect.right = document.documentElement.clientWidth;
            rect.bottom = document.documentElement.clientHeight;
            let style = {};

            if (element instanceof HTMLElement) {
                rect = element.getBoundingClientRect();
                style = window.getComputedStyle(element);
            }

            const margin = {};
            margin.left = parseFloat(style.marginLeftWidth) || 0;
            margin.top = parseFloat(style.marginTopWidth) || 0;
            margin.right = parseFloat(style.marginRightWidth) || 0;
            margin.bottom = parseFloat(style.marginBottomWidth) || 0;

            const border = {};
            border.left = parseFloat(style.borderLeftWidth) || 0;
            border.top = parseFloat(style.borderTopWidth) || 0;
            border.right = parseFloat(style.borderRightWidth) || 0;
            border.bottom = parseFloat(style.borderBottomWidth) || 0;

            const newRect = {};
            newRect.left = rect.left + margin.left + border.left;
            newRect.top = rect.top + margin.top + border.top;
            newRect.right = rect.right + margin.right + border.right;
            newRect.bottom = rect.bottom + margin.bottom + border.bottom;
            newRect.width = newRect.right - newRect.left;
            newRect.height = newRect.bottom - newRect.top;

            return newRect;
        }

        // get position of element relative to page
        function getRectInPage(element) {
            const rect = getRectInView(element);
            const body = getRectInView(document.body);

            const newRect = {};
            newRect.left = rect.left - body.left;
            newRect.top = rect.top - body.top;
            newRect.right = rect.right - body.left;
            newRect.bottom = rect.bottom - body.top;
            newRect.width = rect.width;
            newRect.height = rect.height;

            return newRect;
        }

        // (for interaction with accordion plugin)
        // get closest element before specified element that matches query
        function firstBefore(element, query) {
            while (
                element &&
                element !== document.body &&
                !element.matches(query)
            )
                element = element.previousElementSibling || element.parentNode;

            return element;
        }

        // (for interaction with accordion plugin)
        // check if element is part of collapsed heading
        function isCollapsed(element) {
            while (element && element !== document.body) {
                if (element.dataset.collapsed === 'true')
                    return true;
                element = element.parentNode;
            }
            return false;
        }

        // (for interaction with accordion plugin)
        // expand heading containing element if necesary
        function expandElement(element) {
            if (isCollapsed(element)) {
                const heading = firstBefore(element, 'h2');
                if (heading)
                    heading.click();
            }
        }

        // load options from url parameters
        function loadOptions() {
            const url = window.location.search;
            const params = new URLSearchParams(url);
            for (const optionName of Object.keys(options)) {
                const paramName = pluginName + '-' + optionName;
                const param = params.get(paramName);
                if (param !== '' && param !== null)
                    options[optionName] = param;
            }
        }
        loadOptions();

        // start script when document is finished loading
        if (options.enabled === 'true')
            window.addEventListener('load', start);
    })();
</script>

<!-- caret left icon -->

<template class="icon_caret_left">
    <!-- modified from: https://fontawesome.com/icons/caret-left -->
    <svg width="16" height="16" viewBox="0 0 192 512">
        <path
            fill="currentColor"
            d="M192 127.338v257.324c0 17.818-21.543 26.741-34.142 14.142L29.196 270.142c-7.81-7.81-7.81-20.474 0-28.284l128.662-128.662c12.599-12.6 34.142-3.676 34.142 14.142z"
        ></path>
    </svg>
</template>

<!-- caret right icon -->

<template class="icon_caret_right">
    <!-- modified from: https://fontawesome.com/icons/caret-right -->
    <svg width="16" height="16" viewBox="0 0 192 512">
        <path
            fill="currentColor"
            d="M0 384.662V127.338c0-17.818 21.543-26.741 34.142-14.142l128.662 128.662c7.81 7.81 7.81 20.474 0 28.284L34.142 398.804C21.543 411.404 0 402.48 0 384.662z"
        ></path>
    </svg>
</template>
<!-- jump to first plugin -->

<script>
    (function() {
        // /////////////////////////
        // DESCRIPTION
        // /////////////////////////

        // This Manubot plugin adds a button next to each reference entry,
        // figure, and table that jumps the page to the first occurrence of a
        // link to that item in the manuscript.

        // /////////////////////////
        // OPTIONS
        // /////////////////////////

        // plugin name prefix for url parameters
        const pluginName = 'jumpToFirst';

        // default plugin options
        const options = {
            // whether to add buttons next to reference entries
            references: 'true',
            // whether to add buttons next to figures
            figures: 'true',
            // whether to add buttons next to tables
            tables: 'true',
            // whether plugin is on or not
            enabled: 'true'
        };

        // change options above, or override with url parameter, eg:
        // 'manuscript.html?pluginName-enabled=false'

        // /////////////////////////
        // SCRIPT
        // /////////////////////////

        // start script
        function start() {
            if (options.references !== 'false')
                makeReferenceButtons();
            if (options.figures !== 'false')
                makeFigureButtons();
            if (options.tables !== 'false')
                makeTableButtons();
        }

        // when jump button clicked
        function onButtonClick() {
            const first = getFirstOccurrence(this.dataset.id);
            if (!first)
                return;

            // update url hash so navigating "back" in history will return
            // user to jump button
            window.location.hash = this.dataset.id;
            // scroll to link
            window.setTimeout(function() {
                goToElement(first, window.innerHeight * 0.5);
            }, 0);
        }

        // get first occurence of link to item in document
        function getFirstOccurrence(id) {
            let query = 'a';
            query += '[href="#' + id + '"]';
            // exclude buttons, anchor links, toc links, etc
            query +=
                ':not(.button):not(.icon_button):not(.anchor):not(.toc_link)';
            return document.querySelector(query);
        }

        // add button next to each reference entry
        function makeReferenceButtons() {
            const references = document.querySelectorAll('div[id^="ref-"]');
            for (const reference of references) {
                // get reference id and element to add button to
                const id = reference.id;
                const container = reference.firstElementChild;
                const first = getFirstOccurrence(id);

                // if can't find link to reference, ignore
                if (!first)
                    continue;

                // make jump button
                let button = document.createElement('button');
                button.classList.add('icon_button', 'jump_arrow');
                button.title =
                    'Jump to the first occurence of this reference in the document';
                button.innerHTML = document.querySelector(
                    '.icon_angle_double_up'
                ).innerHTML;
                button.dataset.id = id;
                button.dataset.ignore = 'true';
                container.innerHTML = button.outerHTML + container.innerHTML;
                button = container.firstElementChild;
                button.addEventListener('click', onButtonClick);
            }
        }

        // add button next to each figure
        function makeFigureButtons() {
            const figures = document.querySelectorAll('[id^="fig:"]');
            for (const figure of figures) {
                // get figure id and element to add button to
                const id = figure.id;
                const container = figure.querySelector('figcaption') || figure;
                const first = getFirstOccurrence(id);

                // if can't find link to figure, ignore
                if (!first)
                    continue;

                // make jump button
                const button = document.createElement('button');
                button.classList.add('icon_button', 'jump_arrow');
                button.title =
                    'Jump to the first occurence of this figure in the document';
                button.innerHTML = document.querySelector(
                    '.icon_angle_double_up'
                ).innerHTML;
                button.dataset.id = id;
                button.dataset.ignore = 'true';
                container.insertBefore(button, container.firstElementChild);
                button.addEventListener('click', onButtonClick);
            }
        }

        // add button next to each figure
        function makeTableButtons() {
            const tables = document.querySelectorAll('[id^="tbl:"]');
            for (const table of tables) {
                // get ref id and element to add button to
                const id = table.id;
                const container = table.querySelector('caption') || table;
                const first = getFirstOccurrence(id);

                // if can't find link to table, ignore
                if (!first)
                    continue;

                // make jump button
                const button = document.createElement('button');
                button.classList.add('icon_button', 'jump_arrow');
                button.title =
                    'Jump to the first occurence of this table in the document';
                button.innerHTML = document.querySelector(
                    '.icon_angle_double_up'
                ).innerHTML;
                button.dataset.id = id;
                button.dataset.ignore = 'true';
                container.insertBefore(button, container.firstElementChild);
                button.addEventListener('click', onButtonClick);
            }
        }

        // scroll to and focus element
        function goToElement(element, offset) {
            // expand accordion section if collapsed
            expandElement(element);
            const y =
                getRectInView(element).top -
                getRectInView(document.documentElement).top -
                (offset || 0);
            // trigger any function listening for "onscroll" event
            window.dispatchEvent(new Event('scroll'));
            window.scrollTo(0, y);
            document.activeElement.blur();
            element.focus();
        }

        // get position/dimensions of element or viewport
        function getRectInView(element) {
            let rect = {};
            rect.left = 0;
            rect.top = 0;
            rect.right = document.documentElement.clientWidth;
            rect.bottom = document.documentElement.clientHeight;
            let style = {};

            if (element instanceof HTMLElement) {
                rect = element.getBoundingClientRect();
                style = window.getComputedStyle(element);
            }

            const margin = {};
            margin.left = parseFloat(style.marginLeftWidth) || 0;
            margin.top = parseFloat(style.marginTopWidth) || 0;
            margin.right = parseFloat(style.marginRightWidth) || 0;
            margin.bottom = parseFloat(style.marginBottomWidth) || 0;

            const border = {};
            border.left = parseFloat(style.borderLeftWidth) || 0;
            border.top = parseFloat(style.borderTopWidth) || 0;
            border.right = parseFloat(style.borderRightWidth) || 0;
            border.bottom = parseFloat(style.borderBottomWidth) || 0;

            const newRect = {};
            newRect.left = rect.left + margin.left + border.left;
            newRect.top = rect.top + margin.top + border.top;
            newRect.right = rect.right + margin.right + border.right;
            newRect.bottom = rect.bottom + margin.bottom + border.bottom;
            newRect.width = newRect.right - newRect.left;
            newRect.height = newRect.bottom - newRect.top;

            return newRect;
        }

        // get closest element before specified element that matches query
        function firstBefore(element, query) {
            while (
                element &&
                element !== document.body &&
                !element.matches(query)
            )
                element = element.previousElementSibling || element.parentNode;

            return element;
        }

        // check if element is part of collapsed heading
        function isCollapsed(element) {
            while (element && element !== document.body) {
                if (element.dataset.collapsed === 'true')
                    return true;
                element = element.parentNode;
            }
            return false;
        }

        // (for interaction with accordion plugin)
        // expand heading containing element if necesary
        function expandElement(element) {
            if (isCollapsed(element)) {
                const heading = firstBefore(element, 'h2');
                if (heading)
                    heading.click();
            }
        }

        // load options from url parameters
        function loadOptions() {
            const url = window.location.search;
            const params = new URLSearchParams(url);
            for (const optionName of Object.keys(options)) {
                const paramName = pluginName + '-' + optionName;
                const param = params.get(paramName);
                if (param !== '' && param !== null)
                    options[optionName] = param;
            }
        }
        loadOptions();

        // start script when document is finished loading
        if (options.enabled === 'true')
            window.addEventListener('load', start);
    })();
</script>

<!-- angle double up icon -->

<template class="icon_angle_double_up">
    <!-- modified from: https://fontawesome.com/icons/angle-double-up -->
    <svg width="16" height="16" viewBox="0 0 320 512">
        <path
            fill="currentColor"
            d="M177 255.7l136 136c9.4 9.4 9.4 24.6 0 33.9l-22.6 22.6c-9.4 9.4-24.6 9.4-33.9 0L160 351.9l-96.4 96.4c-9.4 9.4-24.6 9.4-33.9 0L7 425.7c-9.4-9.4-9.4-24.6 0-33.9l136-136c9.4-9.5 24.6-9.5 34-.1zm-34-192L7 199.7c-9.4 9.4-9.4 24.6 0 33.9l22.6 22.6c9.4 9.4 24.6 9.4 33.9 0l96.4-96.4 96.4 96.4c9.4 9.4 24.6 9.4 33.9 0l22.6-22.6c9.4-9.4 9.4-24.6 0-33.9l-136-136c-9.2-9.4-24.4-9.4-33.8 0z"
        ></path>
    </svg>
</template>
<!-- link highlight plugin -->

<script>
    (function() {
        // /////////////////////////
        // DESCRIPTION
        // /////////////////////////

        // This Manubot plugin makes it such that when a user hovers or
        // focuses a link, other links that have the same target will be
        // highlighted. It also makes it such that when clicking a link, the
        // target of the link (eg reference, figure, table) is briefly
        // highlighted.

        // /////////////////////////
        // OPTIONS
        // /////////////////////////

        // plugin name prefix for url parameters
        const pluginName = 'linkHighlight';

        // default plugin options
        const options = {
            // whether to also highlight links that go to external urls
            externalLinks: 'false',
            // whether user must click off to unhighlight instead of just
            // un-hovering
            clickUnhighlight: 'false',
            // whether to also highlight links that are unique
            highlightUnique: 'true',
            // whether plugin is on or not
            enabled: 'true'
        };

        // change options above, or override with url parameter, eg:
        // 'manuscript.html?pluginName-enabled=false'

        // /////////////////////////
        // SCRIPT
        // /////////////////////////

        // start script
        function start() {
            const links = getLinks();
            for (const link of links) {
                // attach mouse and focus listeners to link
                link.addEventListener('mouseenter', onLinkFocus);
                link.addEventListener('focus', onLinkFocus);
                link.addEventListener('mouseleave', onLinkUnhover);
            }

            // attach click and hash change listeners to window
            window.addEventListener('click', onClick);
            window.addEventListener('touchstart', onClick);
            window.addEventListener('hashchange', onHashChange);

            // run hash change on window load in case user has navigated
            // directly to hash
            onHashChange();
        }

        // when link is focused (tabbed to) or hovered
        function onLinkFocus() {
            highlight(this);
        }

        // when link is unhovered
        function onLinkUnhover() {
            if (options.clickUnhighlight !== 'true')
                unhighlightAll();
        }

        // when the mouse is clicked anywhere in window
        function onClick(event) {
            unhighlightAll();
        }

        // when hash (eg manuscript.html#introduction) changes
        function onHashChange() {
            const target = getHashTarget();
            if (target)
                glowElement(target);
        }

        // get element that is target of link or url hash
        function getHashTarget(link) {
            const hash = link ? link.hash : window.location.hash;
            const id = hash.slice(1);
            let target = document.querySelector('[id="' + id + '"]');
            if (!target)
                return;

            return target;
        }

        // start glow sequence on an element
        function glowElement(element) {
            const startGlow = function() {
                onGlowEnd();
                element.dataset.glow = 'true';
                element.addEventListener('animationend', onGlowEnd);
            };
            const onGlowEnd = function() {
                element.removeAttribute('data-glow');
                element.removeEventListener('animationend', onGlowEnd);
            };
            startGlow();
        }

        // highlight link and all others with same target
        function highlight(link) {
            // force unhighlight all to start fresh
            unhighlightAll();

            // get links with same target
            if (!link)
                return;
            const sameLinks = getSameLinks(link);

            // if link unique and option is off, exit and don't highlight
            if (sameLinks.length <= 1 && options.highlightUnique !== 'true')
                return;

            // highlight all same links, and "select" (special highlight) this
            // one
            for (const sameLink of sameLinks) {
                if (sameLink === link)
                    sameLink.setAttribute('data-selected', 'true');
                else
                    sameLink.setAttribute('data-highlighted', 'true');
            }
        }

        // unhighlight all links
        function unhighlightAll() {
            const links = getLinks();
            for (const link of links) {
                link.setAttribute('data-selected', 'false');
                link.setAttribute('data-highlighted', 'false');
            }
        }

        // get links with same target
        function getSameLinks(link) {
            const results = [];
            const links = getLinks();
            for (const otherLink of links) {
                if (
                    otherLink.getAttribute('href') === link.getAttribute('href')
                )
                    results.push(otherLink);
            }
            return results;
        }

        // get all links of types we wish to handle
        function getLinks() {
            let query = 'a';
            if (options.externalLinks !== 'true')
                query += '[href^="#"]';
            // exclude buttons, anchor links, toc links, etc
            query +=
                ':not(.button):not(.icon_button):not(.anchor):not(.toc_link)';
            return document.querySelectorAll(query);
        }

        // load options from url parameters
        function loadOptions() {
            const url = window.location.search;
            const params = new URLSearchParams(url);
            for (const optionName of Object.keys(options)) {
                const paramName = pluginName + '-' + optionName;
                const param = params.get(paramName);
                if (param !== '' && param !== null)
                    options[optionName] = param;
            }
        }
        loadOptions();

        // start script when document is finished loading
        if (options.enabled === 'true')
            window.addEventListener('load', start);
    })();
</script>
<script>
    (function() {
        // /////////////////////////
        // DESCRIPTION
        // /////////////////////////

        // This Manubot plugin provides a "table of contents" (toc) panel on
        // the side of the document that allows the user to conveniently
        // navigate between sections of the document.

        // /////////////////////////
        // OPTIONS
        // /////////////////////////

        // plugin name prefix for url parameters
        const pluginName = 'tableOfContents';

        // default plugin options
        const options = {
            // which types of elements to add links for, in
            // "document.querySelector" format
            typesQuery: 'h1, h2, h3',
            // whether toc starts open. use 'true' or 'false', or 'auto' to
            // use 'true' behavior when screen wide enough and 'false' when not
            startOpen: 'false',
            // whether toc closes when clicking on toc link. use 'true' or
            // 'false', or 'auto' to use 'false' behavior when screen wide
            // enough and 'true' when not
            clickClose: 'auto',
            // if list item is more than this many characters, text will be
            // truncated
            charLimit: '50',
            // whether or not to show bullets next to each toc item
            bullets: 'false',
            // whether plugin is on or not
            enabled: 'true'
        };

        // change options above, or override with url parameter, eg:
        // 'manuscript.html?pluginName-enabled=false'

        // /////////////////////////
        // SCRIPT
        // /////////////////////////

        // start script
        function start() {
            // make toc panel and populate with entries (links to document
            // sections)
            const panel = makePanel();
            if (!panel)
                return;
            makeEntries(panel);
            // attach panel to document after making entries, so 'toc' heading
            // in panel isn't included in toc
            document.body.insertBefore(panel, document.body.firstChild);

            // initial panel state
            if (
                options.startOpen === 'true' ||
                (options.startOpen === 'auto' && !isSmallScreen())
            )
                openPanel();
            else
                closePanel();

            // attach click, scroll, and hash change listeners to window
            window.addEventListener('click', onClick);
            window.addEventListener('scroll', onScroll);
            window.addEventListener('hashchange', onScroll);
            window.addEventListener('keyup', onKeyUp);
            onScroll();

            // add class to push document body down out of way of toc button
            document.body.classList.add('toc_body_nudge');
        }

        // determine if screen wide enough to fit toc panel
        function isSmallScreen() {
            // in default theme:
            // 816px = 8.5in = width of "page" (<body>) element
            // 260px = min width of toc panel (*2 for both sides of <body>)
            return window.innerWidth < 816 + 260 * 2;
        }

        // when mouse is clicked anywhere in window
        function onClick() {
            if (isSmallScreen())
                closePanel();
        }

        // when window is scrolled or hash changed
        function onScroll() {
            highlightViewed();
        }

        // when key pressed
        function onKeyUp(event) {
            if (!event || !event.key)
                return;

            // close on esc
            if (event.key === 'Escape')
                closePanel();
        }

        // find entry of currently viewed document section in toc and highlight
        function highlightViewed() {
            const firstId = getFirstInView(options.typesQuery);

            // get toc entries (links), unhighlight all, then highlight viewed
            const list = document.getElementById('toc_list');
            if (!firstId || !list)
                return;
            const links = list.querySelectorAll('a');
            for (const link of links)
                link.dataset.viewing = 'false';
            const link = list.querySelector('a[href="#' + firstId + '"]');
            if (!link)
                return;
            link.dataset.viewing = 'true';
        }

        // get first or previous toc listed element in top half of view
        function getFirstInView(query) {
            // get all elements matching query and with id
            const elements = document.querySelectorAll(query);
            const elementsWithIds = [];
            for (const element of elements) {
                if (element.id)
                    elementsWithIds.push(element);
            }


            // get first or previous element in top half of view
            for (let i = 0; i < elementsWithIds.length; i++) {
                const element = elementsWithIds[i];
                const prevElement = elementsWithIds[Math.max(0, i - 1)];
                if (element.getBoundingClientRect().top >= 0) {
                    if (
                        element.getBoundingClientRect().top <
                        window.innerHeight / 2
                    )
                        return element.id;
                    else
                        return prevElement.id;
                }
            }
        }

        // make panel
        function makePanel() {
            // create panel
            const panel = document.createElement('div');
            panel.id = 'toc_panel';
            if (options.bullets === 'true')
                panel.dataset.bullets = 'true';

            // create header
            const header = document.createElement('div');
            header.id = 'toc_header';

            // create toc button
            const button = document.createElement('button');
            button.id = 'toc_button';
            button.innerHTML = document.querySelector('.icon_th_list').innerHTML;
            button.title = 'Table of Contents';
            button.classList.add('icon_button');

            // create header text
            const text = document.createElement('h4');
            text.innerHTML = 'Table of Contents';

            // create container for toc list
            const list = document.createElement('div');
            list.id = 'toc_list';

            // attach click listeners
            panel.addEventListener('click', onPanelClick);
            header.addEventListener('click', onHeaderClick);
            button.addEventListener('click', onButtonClick);

            // attach elements
            header.appendChild(button);
            header.appendChild(text);
            panel.appendChild(header);
            panel.appendChild(list);

            return panel;
        }

        // create toc entries (links) to each element of the specified types
        function makeEntries(panel) {
            const elements = document.querySelectorAll(options.typesQuery);
            for (const element of elements) {
                // do not add link if element doesn't have assigned id
                if (!element.id)
                    continue;

                // create link/list item
                const link = document.createElement('a');
                link.classList.add('toc_link');
                switch (element.tagName.toLowerCase()) {
                    case 'h1':
                        link.dataset.level = '1';
                        break;
                    case 'h2':
                        link.dataset.level = '2';
                        break;
                    case 'h3':
                        link.dataset.level = '3';
                        break;
                    case 'h4':
                        link.dataset.level = '4';
                        break;
                }
                link.title = element.innerText;
                let text = element.innerText;
                if (text.length > options.charLimit)
                    text = text.slice(0, options.charLimit) + '...';
                link.innerHTML = text;
                link.href = '#' + element.id;
                link.addEventListener('click', onLinkClick);

                // attach link
                panel.querySelector('#toc_list').appendChild(link);
            }
        }

        // when panel is clicked
        function onPanelClick(event) {
            // stop click from propagating to window/document and closing panel
            event.stopPropagation();
        }

        // when header itself is clicked
        function onHeaderClick(event) {
            togglePanel();
        }

        // when button is clicked
        function onButtonClick(event) {
            togglePanel();
            // stop header underneath button from also being clicked
            event.stopPropagation();
        }

        // when link is clicked
        function onLinkClick(event) {
            if (
                options.clickClose === 'true' ||
                (options.clickClose === 'auto' && isSmallScreen())
            )
                closePanel();
            else
                openPanel();
        }

        // open panel if closed, close if opened
        function togglePanel() {
            const panel = document.getElementById('toc_panel');
            if (!panel)
                return;

            if (panel.dataset.open === 'true')
                closePanel();
            else
                openPanel();
        }

        // open panel
        function openPanel() {
            const panel = document.getElementById('toc_panel');
            if (panel)
                panel.dataset.open = 'true';
        }

        // close panel
        function closePanel() {
            const panel = document.getElementById('toc_panel');
            if (panel)
                panel.dataset.open = 'false';
        }

        // load options from url parameters
        function loadOptions() {
            const url = window.location.search;
            const params = new URLSearchParams(url);
            for (const optionName of Object.keys(options)) {
                const paramName = pluginName + '-' + optionName;
                const param = params.get(paramName);
                if (param !== '' && param !== null)
                    options[optionName] = param;
            }
        }
        loadOptions();

        // start script when document is finished loading
        if (options.enabled === 'true')
            window.addEventListener('load', start);
    })();
</script>

<!-- th list icon -->

<template class="icon_th_list">
    <!-- modified from: https://fontawesome.com/icons/th-list -->
    <svg width="16" height="16" viewBox="0 0 512 512" tabindex="-1">
        <path
            fill="currentColor"
            d="M96 96c0 26.51-21.49 48-48 48S0 122.51 0 96s21.49-48 48-48 48 21.49 48 48zM48 208c-26.51 0-48 21.49-48 48s21.49 48 48 48 48-21.49 48-48-21.49-48-48-48zm0 160c-26.51 0-48 21.49-48 48s21.49 48 48 48 48-21.49 48-48-21.49-48-48-48zm96-236h352c8.837 0 16-7.163 16-16V76c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h352c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h352c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16z"
            tabindex="-1"
        ></path>
    </svg>
</template>
<!-- lightbox plugin -->

<script>
    (function() {
        // /////////////////////////
        // DESCRIPTION
        // /////////////////////////

        // This Manubot plugin makes it such that when a user clicks on an
        // image, the image fills the screen and the user can pan/drag/zoom
        // the image and navigate between other images in the document.

        // /////////////////////////
        // OPTIONS
        // /////////////////////////

        // plugin name prefix for url parameters
        const pluginName = 'lightbox';

        // default plugin options
        const options = {
            // list of possible zoom/scale factors
            zoomSteps:
                '0.1, 0.25, 0.333333, 0.5, 0.666666, 0.75, 1,' +
                '1.25, 1.5, 1.75, 2, 2.5, 3, 3.5, 4, 5, 6, 7, 8',
            // whether to fit image to view ('fit'), display at 100% and shrink
            // if necessary ('shrink'), or always display at 100% ('100')
            defaultZoom: 'fit',
            // whether to zoom in/out toward center of view ('true') or mouse
            // ('false')
            centerZoom: 'false',
            // whether plugin is on or not
            enabled: 'true'
        };

        // change options above, or override with url parameter, eg:
        // 'manuscript.html?pluginName-enabled=false'

        // /////////////////////////
        // SCRIPT
        // /////////////////////////

        // start script
        function start() {
            // run through each <img> element
            const imgs = document.querySelectorAll('figure > img');
            let count = 1;
            for (const img of imgs) {
                img.classList.add('lightbox_document_img');
                img.dataset.number = count;
                img.dataset.total = imgs.length;
                img.addEventListener('click', openLightbox);
                count++;
            }

            // attach mouse and key listeners to window
            window.addEventListener('mousemove', onWindowMouseMove);
            window.addEventListener('keyup', onKeyUp);
        }

        // when mouse is moved anywhere in window
        function onWindowMouseMove(event) {
            window.mouseX = event.clientX;
            window.mouseY = event.clientY;
        }

        // when key pressed
        function onKeyUp(event) {
            if (!event || !event.key)
                return;

            switch (event.key) {
                // trigger click of prev button
                case 'ArrowLeft':
                    const prevButton = document.getElementById(
                        'lightbox_prev_button'
                    );
                    if (prevButton)
                        prevButton.click();
                    break;
                // trigger click of next button
                case 'ArrowRight':
                    const nextButton = document.getElementById(
                        'lightbox_next_button'
                    );
                    if (nextButton)
                        nextButton.click();
                    break;
                // close on esc
                case 'Escape':
                    closeLightbox();
                    break;
            }
        }

        // open lightbox
        function openLightbox() {
            const lightbox = makeLightbox(this);
            if (!lightbox)
                return;

            blurBody(lightbox);
            document.body.appendChild(lightbox);
        }

        // make lightbox
        function makeLightbox(img) {
            // delete lightbox if it exists, start fresh
            closeLightbox();

            // create screen overlay containing lightbox
            const overlay = document.createElement('div');
            overlay.id = 'lightbox_overlay';

            // create image info boxes
            const numberInfo = document.createElement('div');
            const zoomInfo = document.createElement('div');
            numberInfo.id = 'lightbox_number_info';
            zoomInfo.id = 'lightbox_zoom_info';

            // create container for image
            const imageContainer = document.createElement('div');
            imageContainer.id = 'lightbox_image_container';
            const lightboxImg = makeLightboxImg(
                img,
                imageContainer,
                numberInfo,
                zoomInfo
            );
            imageContainer.appendChild(lightboxImg);

            // create bottom container for caption and navigation buttons
            const bottomContainer = document.createElement('div');
            bottomContainer.id = 'lightbox_bottom_container';
            const caption = makeCaption(img);
            const prevButton = makePrevButton(img);
            const nextButton = makeNextButton(img);
            bottomContainer.appendChild(prevButton);
            bottomContainer.appendChild(caption);
            bottomContainer.appendChild(nextButton);

            // attach top middle and bottom to overlay
            overlay.appendChild(numberInfo);
            overlay.appendChild(zoomInfo);
            overlay.appendChild(imageContainer);
            overlay.appendChild(bottomContainer);

            return overlay;
        }

        // make <img> object that is intuitively draggable and zoomable
        function makeLightboxImg(
            sourceImg,
            container,
            numberInfoBox,
            zoomInfoBox
        ) {
            // create copy of source <img>
            const img = sourceImg.cloneNode(true);
            img.classList.remove('lightbox_document_img');
            img.removeAttribute('id');
            img.removeAttribute('width');
            img.removeAttribute('height');
            img.style.position = 'unset';
            img.style.margin = '0';
            img.style.padding = '0';
            img.style.width = '';
            img.style.height = '';
            img.style.minWidth = '';
            img.style.minHeight = '';
            img.style.maxWidth = '';
            img.style.maxHeight = '';
            img.id = 'lightbox_img';

            // build sorted list of unique zoomSteps, always including a 100%
            let zoomSteps = [];
            const optionsZooms = options.zoomSteps.split(/[^0-9.]/);
            for (const optionZoom of optionsZooms) {
                const newZoom = parseFloat(optionZoom);
                if (newZoom && !zoomSteps.includes(newZoom))
                    zoomSteps.push(newZoom);
            }
            if (!zoomSteps.includes(1))
                zoomSteps.push(1);
            zoomSteps = zoomSteps.sort(function sortNumber(a, b) {
                return a - b;
            });

            // <img> object property variables
            let zoom = 1;
            let translateX = 0;
            let translateY = 0;
            let clickMouseX = undefined;
            let clickMouseY = undefined;
            let clickTranslateX = undefined;
            let clickTranslateY = undefined;

            updateNumberInfo();

            // update image numbers displayed in info box
            function updateNumberInfo() {
                numberInfoBox.innerHTML =
                    sourceImg.dataset.number + ' of ' + sourceImg.dataset.total;
            }

            // update zoom displayed in info box
            function updateZoomInfo() {
                let zoomInfo = zoom * 100;
                if (!Number.isInteger(zoomInfo))
                    zoomInfo = zoomInfo.toFixed(2);
                zoomInfoBox.innerHTML = zoomInfo + '%';
            }

            // move to closest zoom step above current zoom
            const zoomIn = function() {
                for (const zoomStep of zoomSteps) {
                    if (zoomStep > zoom) {
                        zoom = zoomStep;
                        break;
                    }
                }
                updateTransform();
            };

            // move to closest zoom step above current zoom
            const zoomOut = function() {
                zoomSteps.reverse();
                for (const zoomStep of zoomSteps) {
                    if (zoomStep < zoom) {
                        zoom = zoomStep;
                        break;
                    }
                }
                zoomSteps.reverse();

                updateTransform();
            };

            // update display of <img> based on scale/translate properties
            const updateTransform = function() {
                // set transform
                img.style.transform =
                    'translate(' +
                    (translateX || 0) +
                    'px,' +
                    (translateY || 0) +
                    'px) scale(' +
                    (zoom || 1) +
                    ')';

                // get new width/height after scale
                const rect = img.getBoundingClientRect();
                // limit translate
                translateX = Math.max(translateX, -rect.width / 2);
                translateX = Math.min(translateX, rect.width / 2);
                translateY = Math.max(translateY, -rect.height / 2);
                translateY = Math.min(translateY, rect.height / 2);

                // set transform
                img.style.transform =
                    'translate(' +
                    (translateX || 0) +
                    'px,' +
                    (translateY || 0) +
                    'px) scale(' +
                    (zoom || 1) +
                    ')';

                updateZoomInfo();
            };

            // fit <img> to container
            const fit = function() {
                // no x/y offset, 100% zoom by default
                translateX = 0;
                translateY = 0;
                zoom = 1;

                // widths of <img> and container
                const imgWidth = img.naturalWidth;
                const imgHeight = img.naturalHeight;
                const containerWidth = parseFloat(
                    window.getComputedStyle(container).width
                );
                const containerHeight = parseFloat(
                    window.getComputedStyle(container).height
                );

                // how much zooming is needed to fit <img> to container
                const xRatio = imgWidth / containerWidth;
                const yRatio = imgHeight / containerHeight;
                const maxRatio = Math.max(xRatio, yRatio);
                const newZoom = 1 / maxRatio;

                // fit <img> to container according to option
                if (options.defaultZoom === 'shrink') {
                    if (maxRatio > 1)
                        zoom = newZoom;
                } else if (options.defaultZoom === 'fit')
                    zoom = newZoom;

                updateTransform();
            };

            // when mouse wheel is rolled anywhere in container
            const onContainerWheel = function(event) {
                if (!event)
                    return;

                // let ctrl + mouse wheel to zoom behave as normal
                if (event.ctrlKey)
                    return;

                // prevent normal scroll behavior
                event.preventDefault();
                event.stopPropagation();

                // point around which to scale img
                const viewRect = container.getBoundingClientRect();
                const viewX = (viewRect.left + viewRect.right) / 2;
                const viewY = (viewRect.top + viewRect.bottom) / 2;
                const originX = options.centerZoom === 'true' ? viewX : mouseX;
                const originY = options.centerZoom === 'true' ? viewY : mouseY;

                // get point on image under origin
                const oldRect = img.getBoundingClientRect();
                const oldPercentX = (originX - oldRect.left) / oldRect.width;
                const oldPercentY = (originY - oldRect.top) / oldRect.height;

                // increment/decrement zoom
                if (event.deltaY < 0)
                    zoomIn();
                if (event.deltaY > 0)
                    zoomOut();

                // get offset between previous image point and origin
                const newRect = img.getBoundingClientRect();
                const offsetX =
                    originX - (newRect.left + newRect.width * oldPercentX);
                const offsetY =
                    originY - (newRect.top + newRect.height * oldPercentY);

                // translate image to keep image point under origin
                translateX += offsetX;
                translateY += offsetY;

                // perform translate
                updateTransform();
            };

            // when container is clicked
            function onContainerClick(event) {
                // if container itself is target of click, and not other
                // element above it
                if (event.target === this)
                    closeLightbox();
            }

            // when mouse button is pressed on image
            const onImageMouseDown = function(event) {
                // store original mouse position relative to image
                clickMouseX = window.mouseX;
                clickMouseY = window.mouseY;
                clickTranslateX = translateX;
                clickTranslateY = translateY;
                event.stopPropagation();
                event.preventDefault();
            };

            // when mouse button is released anywhere in window
            const onWindowMouseUp = function(event) {
                // reset original mouse position
                clickMouseX = undefined;
                clickMouseY = undefined;
                clickTranslateX = undefined;
                clickTranslateY = undefined;

                // remove global listener if lightbox removed from document
                if (!document.body.contains(container))
                    window.removeEventListener('mouseup', onWindowMouseUp);
            };

            // when mouse is moved anywhere in window
            const onWindowMouseMove = function(event) {
                if (
                    clickMouseX === undefined ||
                    clickMouseY === undefined ||
                    clickTranslateX === undefined ||
                    clickTranslateY === undefined
                )
                    return;

                // offset image based on original and current mouse position
                translateX = clickTranslateX + window.mouseX - clickMouseX;
                translateY = clickTranslateY + window.mouseY - clickMouseY;
                updateTransform();
                event.preventDefault();

                // remove global listener if lightbox removed from document
                if (!document.body.contains(container))
                    window.removeEventListener('mousemove', onWindowMouseMove);
            };

            // when window is resized
            const onWindowResize = function(event) {
                fit();

                // remove global listener if lightbox removed from document
                if (!document.body.contains(container))
                    window.removeEventListener('resize', onWindowResize);
            };

            // attach the necessary event listeners
            img.addEventListener('dblclick', fit);
            img.addEventListener('mousedown', onImageMouseDown);
            container.addEventListener('wheel', onContainerWheel);
            container.addEventListener('mousedown', onContainerClick);
            container.addEventListener('touchstart', onContainerClick);
            window.addEventListener('mouseup', onWindowMouseUp);
            window.addEventListener('mousemove', onWindowMouseMove);
            window.addEventListener('resize', onWindowResize);

            // run fit() after lightbox atttached to document and <img> Loaded
            // so needed container and img dimensions available
            img.addEventListener('load', fit);

            return img;
        }

        // make caption
        function makeCaption(img) {
            const caption = document.createElement('div');
            caption.id = 'lightbox_caption';
            const captionSource = img.nextElementSibling;
            if (captionSource.tagName.toLowerCase() === 'figcaption') {
                const captionCopy = makeCopy(captionSource);
                caption.innerHTML = captionCopy.innerHTML;
            }

            caption.addEventListener('touchstart', function(event) {
                event.stopPropagation();
            });

            return caption;
        }

        // make carbon copy of html dom element
        function makeCopy(source) {
            const sourceCopy = source.cloneNode(true);

            // delete elements marked with ignore (eg anchor and jump buttons)
            const deleteFromCopy = sourceCopy.querySelectorAll(
                '[data-ignore="true"]'
            );
            for (const element of deleteFromCopy)
                element.remove();

            // delete certain element attributes
            const attributes = [
                'id',
                'data-collapsed',
                'data-selected',
                'data-highlighted',
                'data-glow'
            ];
            for (const attribute of attributes) {
                sourceCopy.removeAttribute(attribute);
                const elements = sourceCopy.querySelectorAll(
                    '[' + attribute + ']'
                );
                for (const element of elements)
                    element.removeAttribute(attribute);
            }

            return sourceCopy;
        }

        // make button to jump to previous image in document
        function makePrevButton(img) {
            const prevButton = document.createElement('button');
            prevButton.id = 'lightbox_prev_button';
            prevButton.title = 'Jump to the previous image in the document [←]';
            prevButton.classList.add('icon_button', 'lightbox_button');
            prevButton.innerHTML = document.querySelector(
                '.icon_caret_left'
            ).innerHTML;

            // attach click listeners to button
            prevButton.addEventListener('click', function() {
                getPrevImg(img).click();
            });

            return prevButton;
        }

        // make button to jump to next image in document
        function makeNextButton(img) {
            const nextButton = document.createElement('button');
            nextButton.id = 'lightbox_next_button';
            nextButton.title = 'Jump to the next image in the document [→]';
            nextButton.classList.add('icon_button', 'lightbox_button');
            nextButton.innerHTML = document.querySelector(
                '.icon_caret_right'
            ).innerHTML;

            // attach click listeners to button
            nextButton.addEventListener('click', function() {
                getNextImg(img).click();
            });

            return nextButton;
        }

        // get previous image in document
        function getPrevImg(img) {
            const imgs = document.querySelectorAll('.lightbox_document_img');

            // find index of provided img
            let index;
            for (index = 0; index < imgs.length; index++) {
                if (imgs[index] === img)
                    break;
            }


            // wrap index to other side if < 1
            if (index - 1 >= 0)
                index--;
            else
                index = imgs.length - 1;
            return imgs[index];
        }

        // get next image in document
        function getNextImg(img) {
            const imgs = document.querySelectorAll('.lightbox_document_img');

            // find index of provided img
            let index;
            for (index = 0; index < imgs.length; index++) {
                if (imgs[index] === img)
                    break;
            }


            // wrap index to other side if > total
            if (index + 1 <= imgs.length - 1)
                index++;
            else
                index = 0;
            return imgs[index];
        }

        // close lightbox
        function closeLightbox() {
            focusBody();

            const lightbox = document.getElementById('lightbox_overlay');
            if (lightbox)
                lightbox.remove();
        }

        // make all elements behind lightbox non-focusable
        function blurBody(overlay) {
            const all = document.querySelectorAll('*');
            for (const element of all)
                element.tabIndex = -1;
            document.body.classList.add('body_no_scroll');
        }

        // make all elements focusable again
        function focusBody() {
            const all = document.querySelectorAll('*');
            for (const element of all)
                element.removeAttribute('tabIndex');
            document.body.classList.remove('body_no_scroll');
        }

        // load options from url parameters
        function loadOptions() {
            const url = window.location.search;
            const params = new URLSearchParams(url);
            for (const optionName of Object.keys(options)) {
                const paramName = pluginName + '-' + optionName;
                const param = params.get(paramName);
                if (param !== '' && param !== null)
                    options[optionName] = param;
            }
        }
        loadOptions();

        // start script when document is finished loading
        if (options.enabled === 'true')
            window.addEventListener('load', start);
    })();
</script>

<!-- caret left icon -->

<template class="icon_caret_left">
    <!-- modified from: https://fontawesome.com/icons/caret-left -->
    <svg width="16" height="16" viewBox="0 0 192 512">
        <path
            fill="currentColor"
            d="M192 127.338v257.324c0 17.818-21.543 26.741-34.142 14.142L29.196 270.142c-7.81-7.81-7.81-20.474 0-28.284l128.662-128.662c12.599-12.6 34.142-3.676 34.142 14.142z"
        ></path>
    </svg>
</template>

<!-- caret right icon -->

<template class="icon_caret_right">
    <!-- modified from: https://fontawesome.com/icons/caret-right -->
    <svg width="16" height="16" viewBox="0 0 192 512">
        <path
            fill="currentColor"
            d="M0 384.662V127.338c0-17.818 21.543-26.741 34.142-14.142l128.662 128.662c7.81 7.81 7.81 20.474 0 28.284L34.142 398.804C21.543 411.404 0 402.48 0 384.662z"
        ></path>
    </svg>
</template>
<!-- attributes plugin -->

<script>
    (function() {
        // /////////////////////////
        // DESCRIPTION
        // /////////////////////////

        // This Manubot plugin allows arbitrary HTML attributes to be attached
        // to (almost) any element. Place an HTML comment inside or next to the
        // desired element in the format <!-- $attribute="value" -->

        // /////////////////////////
        // OPTIONS
        // /////////////////////////

        // plugin name prefix for url parameters
        const pluginName = 'attributes';

        // default plugin options
        const options = {
            // whether plugin is on or not
            enabled: 'true'
        };

        // change options above, or override with url parameter, eg:
        // 'manuscript.html?pluginName-enabled=false'

        // /////////////////////////
        // SCRIPT
        // /////////////////////////

        // start script
        function start() {
            // get list of comments in document
            const comments = findComments();

            for(const comment of comments)
                if (comment.parentElement)
                    addAttributes(
                        comment.parentElement,
                        comment.nodeValue.trim()
                    );
        }

        // add html attributes to specified element based on string of 
        // html attributes and values
        function addAttributes(element, text) {
            // regex's for finding attribute/value pairs in the format of
            // attribute="value" or attribute='value
            const regex2 = /\$([a-zA-Z\-]+)?=\"(.+?)\"/;
            const regex1 = /\$([a-zA-Z\-]+)?=\'(.+?)\'/;

            // loop through attribute/value pairs
            let match;
            while(match = text.match(regex2) || text.match(regex1)) {
                // get attribute and value from regex capture groups
                let attribute = match[1];
                let value = match[2];

                // remove from string
                text = text.substring(match.index + match[0].length);

                if (!attribute || !value)
                    break;

                // set attribute of parent element
                try {
                    element.setAttribute(attribute, value);
                } catch(error) {
                    console.log(error);
                }

                // special case for colspan
                if (attribute === 'colspan')
                    removeTableCells(element, value);
            }
        }

        // get list of comment elements in document
        function findComments() {
            const comments = [];

            // iterate over comment nodes in document
            function acceptNode(node) {
                return NodeFilter.FILTER_ACCEPT;
            }
            const iterator = document.createNodeIterator(
                document.body,
                NodeFilter.SHOW_COMMENT,
                acceptNode
            );
            let node;
            while(node = iterator.nextNode())
                comments.push(node);

            return comments;
        }

        // remove certain number of cells after specified cell
        function removeTableCells(cell, number) {
            number = parseInt(number);
            if (!number)
                return;

            // remove elements
            for(; number > 1; number--) {
                if (cell.nextElementSibling)
                    cell.nextElementSibling.remove();
            }
        }

        // load options from url parameters
        function loadOptions() {
            const url = window.location.search;
            const params = new URLSearchParams(url);
            for (const optionName of Object.keys(options)) {
                const paramName = pluginName + '-' + optionName;
                const param = params.get(paramName);
                if (param !== '' && param !== null)
                    options[optionName] = param;
            }
        }
        loadOptions();

        // start script when document is finished loading
        if (options.enabled === 'true')
            window.addEventListener('load', start);
    })();
</script>
<!-- mathjax plugin configuration -->

<script type="text/x-mathjax-config">
    MathJax.Hub.Config({
        "CommonHTML": { linebreaks: { automatic: true } },
        "HTML-CSS": { linebreaks: { automatic: true } },
        "SVG": { linebreaks: { automatic: true } },
        "fast-preview": { disabled: true }
  });
</script>

<!-- mathjax plugin -->

<script
    src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML"
    integrity="sha512-0doc9hKxR3PYwso42RD1p5ySZpzzuDiOwMrdCEh2WdJZCjcmFKc/wEnL+z8fBQrnHoiNWbo+3fiGkOYXBdQp4A=="
    crossorigin="anonymous"
>
    // /////////////////////////
    // DESCRIPTION
    // /////////////////////////

    // This third-party plugin 'MathJax' allows the proper rendering of
    // math/equations written in LaTeX.

    // https://www.mathjax.org/
</script>
<!-- annotations plugin -->

<script>
    // /////////////////////////
    // DESCRIPTION
    // /////////////////////////

    // This third-party plugin 'Hypothesis' allows public annotation of the
    // manuscript.

    // https://web.hypothes.is/

    // plugin configuration
    window.hypothesisConfig = function() {
        return {
            branding: {
                accentColor: '#2196f3',
                appBackgroundColor: '#f8f8f8',
                ctaBackgroundColor: '#f8f8f8',
                ctaTextColor: '#000000',
                selectionFontFamily: 'Open Sans, Helvetica, sans serif',
                annotationFontFamily: 'Open Sans, Helvetica, sans serif'
            }
        };
    };

    // hypothesis client script
    const embed = 'https://hypothes.is/embed.js';
    // hypothesis annotation count query url
    const query = 'https://api.hypothes.is/api/search?limit=0&url='

    
    // start script
    function start() {
        const button = makeButton();
        document.body.insertBefore(button, document.body.firstChild);
        insertCount(button);
    }

    // make button
    function makeButton() {
        // create button
        const button = document.createElement('button');
        button.id = 'hypothesis_button';
        button.innerHTML = document.querySelector('.icon_hypothesis').innerHTML;
        button.title = 'Hypothesis annotations';
        button.classList.add('icon_button');

        function onClick(event) {
            onButtonClick(event, button);
        }

        // attach click listeners
        button.addEventListener('click', onClick);

        return button;
    }

    // insert annotations count
    async function insertCount(button) {
        // get annotation count from Hypothesis based on url
        let count = '-';
        try {
            const canonical = document.querySelector('link[rel="canonical"]');
            const location = window.location;
            const url = encodeURIComponent((canonical || location).href);
            const response = await fetch(query + url);
            const json = await response.json();
            count = json.total || '-';
        } catch(error) {
            console.log(error);
        }
        
        // put count into button
        const counter = document.createElement('span');
        counter.id = 'hypothesis_count';
        counter.innerHTML = count;
        button.title = 'View ' + count + ' Hypothesis annotations';
        button.append(counter);
    }

    // when button is clicked
    function onButtonClick(event, button) {
        const script = document.createElement('script');
        script.src = embed;
        document.body.append(script);
        button.remove();
    }

    window.addEventListener('load', start);
</script>

<!-- hypothesis icon -->

<template class="icon_hypothesis">
    <!-- modified from: https://simpleicons.org/icons/hypothesis.svg / https://git.io/Jf1VB -->
    <svg width="16" height="16" viewBox="0 0 24 24" tabindex="-1">
        <path
            fill="currentColor"
            d="M3.43 0C2.5 0 1.72 .768 1.72 1.72V18.86C1.72 19.8 2.5 20.57 3.43 20.57H9.38L12 24L14.62 20.57H20.57C21.5 20.57 22.29 19.8 22.29 18.86V1.72C22.29 .77 21.5 0 20.57 0H3.43M5.14 3.43H7.72V9.43S8.58 7.72 10.28 7.72C12 7.72 13.74 8.57 13.74 11.24V17.14H11.16V12C11.16 10.61 10.28 10.07 9.43 10.29C8.57 10.5 7.72 11.41 7.72 13.29V17.14H5.14V3.43M18 13.72C18.95 13.72 19.72 14.5 19.72 15.42A1.71 1.71 0 0 1 18 17.13A1.71 1.71 0 0 1 16.29 15.42C16.29 14.5 17.05 13.71 18 13.71Z"
            tabindex="-1"
        ></path>
    </svg>
</template>
<!-- analytics plugin -->

<!-- copy and paste code from Google Analytics or similar service here -->
</body>
</html>
